{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "05dd9806",
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "import json\n",
    "import pandas as pd\n",
    "from pathlib import Path\n",
    "from typing import Optional\n",
    "import sys\n",
    "import os\n",
    "\n",
    "\n",
    "from tools.clinical_db_fetcher.clients.clinvar_fetcher import (\n",
    "    search_clinvar_by_hgvs_g,\n",
    "    fetch_clinvar_details_by_id,\n",
    ")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "dcc9b5c5",
   "metadata": {},
   "outputs": [],
   "source": [
    "gene_symbol = 'FBN1'\n",
    "model_name = 'all-mpnet-base-v2'\n",
    "annotation_method = 'vep'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "4bac17d2",
   "metadata": {},
   "outputs": [],
   "source": [
    "existing_variants_path = os.path.join(\n",
    "    \"data_user\", \"user_query\", \"results\", \n",
    "    gene_symbol, f\"{model_name}_{annotation_method}\", \"existing_variants.json\"\n",
    ")\n",
    "\n",
    "with open(existing_variants_path, 'r') as f:\n",
    "    existing_variant_ids = set(json.load(f).get('variants', []))\n",
    "\n",
    "\n",
    "metadata_path = os.path.join(\n",
    "    \"data_user\", \"training_embedding_results\", \n",
    "    \"metadata\", gene_symbol, \"metadata.json\"\n",
    ")\n",
    "\n",
    "with open(metadata_path, 'r') as f:\n",
    "    metadata = json.load(f)\n",
    "metadata_variants = metadata.get('variants')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "6ca377b7",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_metadata = pd.DataFrame(metadata_variants)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "1a67b587",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Method 4: Vectorized filtering (most efficient)\n",
      "                 hgvs_genomic_38             hgvs_coding  hgvs_protein  \\\n",
      "33    NC_000015.10:g.48411280G>A   NM_000138.5:c.8326C>T  p.Arg2776Ter   \n",
      "36    NC_000015.10:g.48411338C>T   NM_000138.5:c.8268G>A  p.Trp2756Ter   \n",
      "54    NC_000015.10:g.48412715G>A   NM_000138.5:c.8080C>T  p.Arg2694Ter   \n",
      "61    NC_000015.10:g.48415549G>A   NM_000138.5:c.8038C>T  p.Arg2680Cys   \n",
      "70    NC_000015.10:g.48415585C>A   NM_000138.5:c.8002G>T  p.Gly2668Cys   \n",
      "...                          ...                     ...           ...   \n",
      "1442  NC_000015.10:g.48596360C>G    NM_000138.5:c.461G>C   p.Cys154Ser   \n",
      "1462  NC_000015.10:g.48600213C>T    NM_000138.5:c.368G>A   p.Cys123Tyr   \n",
      "1463  NC_000015.10:g.48600217G>A    NM_000138.5:c.364C>T   p.Arg122Cys   \n",
      "1489  NC_000015.10:g.48613009C>T  NM_000138.5:c.247+1G>A                 \n",
      "1498  NC_000015.10:g.48613073G>A    NM_000138.5:c.184C>T    p.Arg62Cys   \n",
      "\n",
      "     chromosome  position ref_allele alt_allele gene_symbol  \\\n",
      "33           15  48411280          G          A        FBN1   \n",
      "36           15  48411338          C          T        FBN1   \n",
      "54           15  48412715          G          A        FBN1   \n",
      "61           15  48415549          G          A        FBN1   \n",
      "70           15  48415585          C          A        FBN1   \n",
      "...         ...       ...        ...        ...         ...   \n",
      "1442         15  48596360          C          G        FBN1   \n",
      "1462         15  48600213          C          T        FBN1   \n",
      "1463         15  48600217          G          A        FBN1   \n",
      "1489         15  48613009          C          T        FBN1   \n",
      "1498         15  48613073          G          A        FBN1   \n",
      "\n",
      "     most_severe_consequence pathogenicity_original  \n",
      "33               stop_gained             pathogenic  \n",
      "36               stop_gained             pathogenic  \n",
      "54               stop_gained             pathogenic  \n",
      "61          missense_variant   pathogenic_or_likely  \n",
      "70          missense_variant      likely_pathogenic  \n",
      "...                      ...                    ...  \n",
      "1442        missense_variant   pathogenic_or_likely  \n",
      "1462        missense_variant             pathogenic  \n",
      "1463        missense_variant             pathogenic  \n",
      "1489    splice_donor_variant             pathogenic  \n",
      "1498        missense_variant             pathogenic  \n",
      "\n",
      "[154 rows x 10 columns]\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/ds/x3ps1_x1547fythw390d70680000gr/T/ipykernel_26811/2482771176.py:9: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  df_existing_variants.drop(columns=['variant_id', 'variant_hash', 'vcf_string'], inplace=True)\n"
     ]
    }
   ],
   "source": [
    "# Filter and select in one step (most efficient)\n",
    "# df_result = df_metadata.loc[\n",
    "#     df_metadata['variant_id'].isin(existing_variant_ids),\n",
    "#     ['variant_id', 'hgvs_genomic_38', 'pathogenicity_original', ]\n",
    "# ]\n",
    "\n",
    "print(\"Method 4: Vectorized filtering (most efficient)\")\n",
    "df_existing_variants = df_metadata[df_metadata['variant_id'].isin(existing_variant_ids)]\n",
    "df_existing_variants.drop(columns=['variant_id', 'variant_hash', 'vcf_string'], inplace=True)\n",
    "\n",
    "print(df_existing_variants)\n",
    "print()\n",
    "\n",
    "# # If you want to preserve the order of existing_variant_ids:\n",
    "# df_result_ordered = df_metadata.set_index('variant_id').loc[\n",
    "#     existing_variant_ids,\n",
    "#     ['pathogenicity_original', 'hgvs_genomic_38']\n",
    "# ].reset_index()\n",
    "\n",
    "# print(\"Method 4b: Preserving input order\")\n",
    "# print(df_result_ordered)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "f1b2f04a",
   "metadata": {},
   "outputs": [],
   "source": [
    "pred_result_df_path = f\"data_user/user_query/processed/{gene_symbol}/{annotation_method}/prediction_results_k{5}_combined.csv\"\n",
    "\n",
    "pred_result_df = pd.read_csv(pred_result_df_path)\n",
    "pred_result_df.drop(columns=['variant_id'], inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "d1bc0f35",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>hgvs_genomic_38</th>\n",
       "      <th>hgvs_coding</th>\n",
       "      <th>hgvs_protein</th>\n",
       "      <th>chromosome</th>\n",
       "      <th>position</th>\n",
       "      <th>ref_allele</th>\n",
       "      <th>alt_allele</th>\n",
       "      <th>gene_symbol</th>\n",
       "      <th>most_severe_consequence</th>\n",
       "      <th>pathogenicity_original</th>\n",
       "      <th>confidence_score_all-mpnet-base-v2</th>\n",
       "      <th>pred_result_all-mpnet-base-v2</th>\n",
       "      <th>confidence_score_google-embedding</th>\n",
       "      <th>pred_result_google-embedding</th>\n",
       "      <th>confidence_score_MedEmbed-large-v0.1</th>\n",
       "      <th>pred_result_MedEmbed-large-v0.1</th>\n",
       "      <th>top_similar_variants</th>\n",
       "      <th>clinvar_id</th>\n",
       "      <th>clinvar_url</th>\n",
       "      <th>germline_classification</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NC_000015.10:g.48411280G&gt;A</td>\n",
       "      <td>NM_000138.5:c.8326C&gt;T</td>\n",
       "      <td>p.Arg2776Ter</td>\n",
       "      <td>15</td>\n",
       "      <td>48411280</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>pathogenic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NC_000015.10:g.48411338C&gt;T</td>\n",
       "      <td>NM_000138.5:c.8268G&gt;A</td>\n",
       "      <td>p.Trp2756Ter</td>\n",
       "      <td>15</td>\n",
       "      <td>48411338</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>pathogenic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NC_000015.10:g.48412715G&gt;A</td>\n",
       "      <td>NM_000138.5:c.8080C&gt;T</td>\n",
       "      <td>p.Arg2694Ter</td>\n",
       "      <td>15</td>\n",
       "      <td>48412715</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>pathogenic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NC_000015.10:g.48415549G&gt;A</td>\n",
       "      <td>NM_000138.5:c.8038C&gt;T</td>\n",
       "      <td>p.Arg2680Cys</td>\n",
       "      <td>15</td>\n",
       "      <td>48415549</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>pathogenic_or_likely</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NC_000015.10:g.48415585C&gt;A</td>\n",
       "      <td>NM_000138.5:c.8002G&gt;T</td>\n",
       "      <td>p.Gly2668Cys</td>\n",
       "      <td>15</td>\n",
       "      <td>48415585</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>likely_pathogenic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>651</th>\n",
       "      <td>NC_000015.10:g.48412639_48412640del</td>\n",
       "      <td>NM_000138.5:c.8155_8156del</td>\n",
       "      <td>p.Lys2719AspfsTer18</td>\n",
       "      <td>15</td>\n",
       "      <td>48412638</td>\n",
       "      <td>CTT</td>\n",
       "      <td>C</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>frameshift_variant</td>\n",
       "      <td>unknown</td>\n",
       "      <td>0.8</td>\n",
       "      <td>likely_pathogenic</td>\n",
       "      <td>0.8</td>\n",
       "      <td>likely_pathogenic</td>\n",
       "      <td>1.0</td>\n",
       "      <td>potentially_benign</td>\n",
       "      <td>NC_000015.10:g.48411228T&gt;C (likely_pathogenic)...</td>\n",
       "      <td>40243.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/clinvar/variation...</td>\n",
       "      <td>Pathogenic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>652</th>\n",
       "      <td>NC_000015.10:g.48412573A&gt;G</td>\n",
       "      <td>NM_000138.5:c.8222T&gt;C</td>\n",
       "      <td>p.Ile2741Thr</td>\n",
       "      <td>15</td>\n",
       "      <td>48412573</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>unknown</td>\n",
       "      <td>1.0</td>\n",
       "      <td>potentially_pathogenic</td>\n",
       "      <td>0.8</td>\n",
       "      <td>likely_pathogenic</td>\n",
       "      <td>1.0</td>\n",
       "      <td>potentially_pathogenic</td>\n",
       "      <td>NC_000015.10:g.48415573A&gt;C (likely_pathogenic)...</td>\n",
       "      <td>2933163.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/clinvar/variation...</td>\n",
       "      <td>Conflicting classifications of pathogenicity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>653</th>\n",
       "      <td>NC_000015.10:g.48412568C&gt;A</td>\n",
       "      <td>NM_000138.5:c.8226+1G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>15</td>\n",
       "      <td>48412568</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>splice_donor_variant</td>\n",
       "      <td>unknown</td>\n",
       "      <td>1.0</td>\n",
       "      <td>potentially_pathogenic</td>\n",
       "      <td>1.0</td>\n",
       "      <td>potentially_pathogenic</td>\n",
       "      <td>1.0</td>\n",
       "      <td>potentially_pathogenic</td>\n",
       "      <td>NC_000015.10:g.48412564C&gt;T (pathogenic_or_like...</td>\n",
       "      <td>40245.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/clinvar/variation...</td>\n",
       "      <td>Pathogenic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>654</th>\n",
       "      <td>NC_000015.10:g.48411369_48411370del</td>\n",
       "      <td>NM_000138.5:c.8238_8239del</td>\n",
       "      <td>p.Glu2746AspfsTer13</td>\n",
       "      <td>15</td>\n",
       "      <td>48411368</td>\n",
       "      <td>CTC</td>\n",
       "      <td>C</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>frameshift_variant</td>\n",
       "      <td>unknown</td>\n",
       "      <td>0.8</td>\n",
       "      <td>likely_pathogenic</td>\n",
       "      <td>0.6</td>\n",
       "      <td>likely_pathogenic</td>\n",
       "      <td>1.0</td>\n",
       "      <td>potentially_benign</td>\n",
       "      <td>NC_000015.10:g.48411383G&gt;A (benign_or_likely),...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>655</th>\n",
       "      <td>NC_000015.10:g.48411007G&gt;A</td>\n",
       "      <td>NM_000138.5:c.8599C&gt;T</td>\n",
       "      <td>p.Gln2867Ter</td>\n",
       "      <td>15</td>\n",
       "      <td>48411007</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>unknown</td>\n",
       "      <td>1.0</td>\n",
       "      <td>potentially_pathogenic</td>\n",
       "      <td>1.0</td>\n",
       "      <td>potentially_pathogenic</td>\n",
       "      <td>1.0</td>\n",
       "      <td>potentially_pathogenic</td>\n",
       "      <td>NC_000015.10:g.48411085C&gt;A (pathogenic_or_like...</td>\n",
       "      <td>3893252.0</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/clinvar/variation...</td>\n",
       "      <td>Likely pathogenic</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>656 rows Ã— 20 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         hgvs_genomic_38                 hgvs_coding  \\\n",
       "0             NC_000015.10:g.48411280G>A       NM_000138.5:c.8326C>T   \n",
       "1             NC_000015.10:g.48411338C>T       NM_000138.5:c.8268G>A   \n",
       "2             NC_000015.10:g.48412715G>A       NM_000138.5:c.8080C>T   \n",
       "3             NC_000015.10:g.48415549G>A       NM_000138.5:c.8038C>T   \n",
       "4             NC_000015.10:g.48415585C>A       NM_000138.5:c.8002G>T   \n",
       "..                                   ...                         ...   \n",
       "651  NC_000015.10:g.48412639_48412640del  NM_000138.5:c.8155_8156del   \n",
       "652           NC_000015.10:g.48412573A>G       NM_000138.5:c.8222T>C   \n",
       "653           NC_000015.10:g.48412568C>A     NM_000138.5:c.8226+1G>T   \n",
       "654  NC_000015.10:g.48411369_48411370del  NM_000138.5:c.8238_8239del   \n",
       "655           NC_000015.10:g.48411007G>A       NM_000138.5:c.8599C>T   \n",
       "\n",
       "            hgvs_protein chromosome  position ref_allele alt_allele  \\\n",
       "0           p.Arg2776Ter         15  48411280          G          A   \n",
       "1           p.Trp2756Ter         15  48411338          C          T   \n",
       "2           p.Arg2694Ter         15  48412715          G          A   \n",
       "3           p.Arg2680Cys         15  48415549          G          A   \n",
       "4           p.Gly2668Cys         15  48415585          C          A   \n",
       "..                   ...        ...       ...        ...        ...   \n",
       "651  p.Lys2719AspfsTer18         15  48412638        CTT          C   \n",
       "652         p.Ile2741Thr         15  48412573          A          G   \n",
       "653                  NaN         15  48412568          C          A   \n",
       "654  p.Glu2746AspfsTer13         15  48411368        CTC          C   \n",
       "655         p.Gln2867Ter         15  48411007          G          A   \n",
       "\n",
       "    gene_symbol most_severe_consequence pathogenicity_original  \\\n",
       "0          FBN1             stop_gained             pathogenic   \n",
       "1          FBN1             stop_gained             pathogenic   \n",
       "2          FBN1             stop_gained             pathogenic   \n",
       "3          FBN1        missense_variant   pathogenic_or_likely   \n",
       "4          FBN1        missense_variant      likely_pathogenic   \n",
       "..          ...                     ...                    ...   \n",
       "651        FBN1      frameshift_variant                unknown   \n",
       "652        FBN1        missense_variant                unknown   \n",
       "653        FBN1    splice_donor_variant                unknown   \n",
       "654        FBN1      frameshift_variant                unknown   \n",
       "655        FBN1             stop_gained                unknown   \n",
       "\n",
       "     confidence_score_all-mpnet-base-v2 pred_result_all-mpnet-base-v2  \\\n",
       "0                                   NaN                           NaN   \n",
       "1                                   NaN                           NaN   \n",
       "2                                   NaN                           NaN   \n",
       "3                                   NaN                           NaN   \n",
       "4                                   NaN                           NaN   \n",
       "..                                  ...                           ...   \n",
       "651                                 0.8             likely_pathogenic   \n",
       "652                                 1.0        potentially_pathogenic   \n",
       "653                                 1.0        potentially_pathogenic   \n",
       "654                                 0.8             likely_pathogenic   \n",
       "655                                 1.0        potentially_pathogenic   \n",
       "\n",
       "     confidence_score_google-embedding pred_result_google-embedding  \\\n",
       "0                                  NaN                          NaN   \n",
       "1                                  NaN                          NaN   \n",
       "2                                  NaN                          NaN   \n",
       "3                                  NaN                          NaN   \n",
       "4                                  NaN                          NaN   \n",
       "..                                 ...                          ...   \n",
       "651                                0.8            likely_pathogenic   \n",
       "652                                0.8            likely_pathogenic   \n",
       "653                                1.0       potentially_pathogenic   \n",
       "654                                0.6            likely_pathogenic   \n",
       "655                                1.0       potentially_pathogenic   \n",
       "\n",
       "     confidence_score_MedEmbed-large-v0.1 pred_result_MedEmbed-large-v0.1  \\\n",
       "0                                     NaN                             NaN   \n",
       "1                                     NaN                             NaN   \n",
       "2                                     NaN                             NaN   \n",
       "3                                     NaN                             NaN   \n",
       "4                                     NaN                             NaN   \n",
       "..                                    ...                             ...   \n",
       "651                                   1.0              potentially_benign   \n",
       "652                                   1.0          potentially_pathogenic   \n",
       "653                                   1.0          potentially_pathogenic   \n",
       "654                                   1.0              potentially_benign   \n",
       "655                                   1.0          potentially_pathogenic   \n",
       "\n",
       "                                  top_similar_variants  clinvar_id  \\\n",
       "0                                                  NaN         NaN   \n",
       "1                                                  NaN         NaN   \n",
       "2                                                  NaN         NaN   \n",
       "3                                                  NaN         NaN   \n",
       "4                                                  NaN         NaN   \n",
       "..                                                 ...         ...   \n",
       "651  NC_000015.10:g.48411228T>C (likely_pathogenic)...     40243.0   \n",
       "652  NC_000015.10:g.48415573A>C (likely_pathogenic)...   2933163.0   \n",
       "653  NC_000015.10:g.48412564C>T (pathogenic_or_like...     40245.0   \n",
       "654  NC_000015.10:g.48411383G>A (benign_or_likely),...         NaN   \n",
       "655  NC_000015.10:g.48411085C>A (pathogenic_or_like...   3893252.0   \n",
       "\n",
       "                                           clinvar_url  \\\n",
       "0                                                  NaN   \n",
       "1                                                  NaN   \n",
       "2                                                  NaN   \n",
       "3                                                  NaN   \n",
       "4                                                  NaN   \n",
       "..                                                 ...   \n",
       "651  https://www.ncbi.nlm.nih.gov/clinvar/variation...   \n",
       "652  https://www.ncbi.nlm.nih.gov/clinvar/variation...   \n",
       "653  https://www.ncbi.nlm.nih.gov/clinvar/variation...   \n",
       "654                                                NaN   \n",
       "655  https://www.ncbi.nlm.nih.gov/clinvar/variation...   \n",
       "\n",
       "                          germline_classification  \n",
       "0                                             NaN  \n",
       "1                                             NaN  \n",
       "2                                             NaN  \n",
       "3                                             NaN  \n",
       "4                                             NaN  \n",
       "..                                            ...  \n",
       "651                                    Pathogenic  \n",
       "652  Conflicting classifications of pathogenicity  \n",
       "653                                    Pathogenic  \n",
       "654                                           NaN  \n",
       "655                             Likely pathogenic  \n",
       "\n",
       "[656 rows x 20 columns]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pred_result_df = pd.concat([df_existing_variants, pred_result_df], axis=0, join='outer', ignore_index=True)\n",
    "pred_result_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "5dcd2726",
   "metadata": {},
   "outputs": [],
   "source": [
    "pred_result_save_path = f\"data_user/user_query/processed/{gene_symbol}/{annotation_method}/prediction_results_k{5}_combined_with_existing_variants.csv\"\n",
    "pred_result_df.to_csv(pred_result_save_path, index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "1112a801",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'gene_symbol': 'FBN1',\n",
       " 'successful': {'results_count': 348,\n",
       "  'results': [{'variant_id': '15-48644711-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48644711\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48644711',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.59A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48644711T>C',\n",
       "     'hgvs_protein': 'p.Tyr20Cys',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"allele_string\": \"T/C\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48644711-T-C\", \"input\": \"NC_000015.10:g.48644711T>C\", \"seq_region_name\": \"15\", \"minimised\": 1, \"colocated_variants\": [{\"start\": 48644711, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48644711, \"phenotype_or_disease\": 1, \"id\": \"CM054704\"}, {\"frequencies\": {\"C\": {\"gnomadg_ami\": 0, \"eas\": 0, \"gnomadg_asj\": 0, \"gnomadg_remaining\": 0, \"gnomade_remaining\": 0.0001987, \"gnomadg_nfe\": 0.0002941, \"gnomadg_amr\": 0, \"sas\": 0, \"gnomadg_sas\": 0.000207, \"gnomadg_fin\": 9.416e-05, \"gnomade_mid\": 0, \"afr\": 0, \"gnomade_fin\": 1.874e-05, \"gnomade_sas\": 8.115e-05, \"af\": 0.0002, \"amr\": 0, \"gnomadg_eas\": 0, \"eur\": 0.001, \"gnomadg_mid\": 0, \"gnomadg\": 0.0001642, \"gnomade_afr\": 5.974e-05, \"gnomade\": 0.0002558, \"gnomadg_afr\": 7.216e-05, \"gnomade_nfe\": 0.0003156, \"gnomade_eas\": 2.519e-05, \"gnomade_amr\": 0, \"gnomade_asj\": 0}}, \"end\": 48644711, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000181402\", \"RCV000148501\", \"VCV000161245\", \"RCV000604732\", \"RCV000587403\", \"RCV000545272\"], \"UniProt\": [\"VAR_023859\"]}, \"clin_sig\": [\"uncertain_significance\", \"likely_benign\"], \"strand\": 1, \"allele_string\": \"T/C\", \"start\": 48644711, \"clin_sig_allele\": \"C:uncertain_significance;C:likely_benign;C:conflicting_classifications_of_pathogenicity\", \"pubmed\": [24033266, 26017485, 24941995, 16222657, 17740842], \"id\": \"rs201309310\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48644711T>C\", \"start\": 48644711, \"most_severe_consequence\": \"missense_variant\", \"end\": 48644711, \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.0647, \"am_class\": \"likely_benign\"}, \"mastermind_mmid3\": \"FBN1:Y20C\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_start\": 375, \"cdna_end\": 375, \"geno2mp_hpo_count\": 7, \"conservation\": 4.1, \"loeuf\": 0.105, \"protein_end\": 20, \"sift_score\": 0.19, \"phaplo\": 0.99897615427687, \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 20, \"cds_end\": 59, \"sift_prediction\": \"tolerated_low_confidence\", \"polyphen_score\": 0.251, \"geno2mp_url\": \"https://geno2mp.gs.washington.edu/Geno2MP/#/variant/15/48936908/T%3EC/snp\", \"amino_acids\": \"Y/C\", \"mutationtaster_pred\": \"N,N\", \"strand\": -1, \"cds_start\": 59, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"clinpred\": 0.0942895953798942, \"polyphen_prediction\": \"benign\", \"location\": \"15:48644711\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48644710, \"end\": 48644711, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Tyr20Cys\", \"cadd_raw\": 3.322437, \"given_ref\": \"T\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"hgvsc\": \"NM_000138.5:c.59A>G\", \"biotype\": \"protein_coding\", \"canonical\": 1, \"codons\": \"tAc/tGc\", \"mastermind_counts\": \"12|14|14\", \"cadd_phred\": 22.2, \"spliceai\": {\"DP_DL\": 6, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AG\": 6, \"DS_DL\": 0, \"DP_AL\": -2, \"DP_DG\": -40, \"DS_AG\": 0.02, \"DS_DG\": 0}, \"aa\": \"T\", \"blosum62\": -2, \"exon\": \"2/66\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"C\", \"gene_symbol_source\": \"EntrezGene\"}]}',\n",
       "    'coordinates': [{'pca_x': 0.0008667430864158221,\n",
       "      'pca_y': 0.12348069003439519},\n",
       "     {'t-sne_x': 8.953833705554766, 't-sne_y': 24.99688381473437},\n",
       "     {'umap_x': -7.979295253753662, 'umap_y': 12.485420227050781}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48610775-C-T',\n",
       "      'distance': 0.008603574708104134,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48534131-A-C',\n",
       "      'distance': 0.00936666876077652,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48600174-C-T',\n",
       "      'distance': 0.009431283921003342,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600147-C-T',\n",
       "      'distance': 0.009702414274215698,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526207-C-T',\n",
       "      'distance': 0.009744660928845406,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48600180-C-T',\n",
       "      'distance': 0.01046510599553585,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600226-A-C',\n",
       "      'distance': 0.010795209556818008,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48613018-C-T',\n",
       "      'distance': 0.011092938482761383,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644738-A-C',\n",
       "      'distance': 0.011181041598320007,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48600174-C-A',\n",
       "      'distance': 0.011354206129908562,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644691-C-T',\n",
       "      'distance': 0.011591584421694279,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48610736-G-C',\n",
       "      'distance': 0.011702872812747955,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48610820-C-T',\n",
       "      'distance': 0.01200261153280735,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537707-C-T',\n",
       "      'distance': 0.012042801827192307,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610800-C-T',\n",
       "      'distance': 0.012324085459113121,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48600148-A-G',\n",
       "      'distance': 0.012605901807546616,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48537647-C-A',\n",
       "      'distance': 0.013315415009856224,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48411064-T-G',\n",
       "      'distance': 0.013388678431510925,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48613058-A-G',\n",
       "      'distance': 0.014501884579658508,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.0427304282784462,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48613069-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48613069\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48613069',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.188A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48613069T>C',\n",
       "     'hgvs_protein': 'p.Tyr63Cys',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"allele_string\": \"T/C\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48613069-T-C\", \"input\": \"NC_000015.10:g.48613069T>C\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM074834\", \"end\": 48613069, \"start\": 48613069, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\"}, {\"clin_sig_allele\": \"C:pathogenic;C:uncertain_significance\", \"phenotype_or_disease\": 1, \"id\": \"rs1303389437\", \"pubmed\": [17657824], \"end\": 48613069, \"strand\": 1, \"allele_string\": \"T/C\", \"seq_region_name\": \"15\", \"clin_sig\": [\"uncertain_significance\", \"pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV001756116\", \"RCV000659499\", \"VCV000547296\", \"RCV001855388\"], \"UniProt\": [\"VAR_075987\"]}, \"start\": 48613069}], \"id\": \"NC_000015.10:g.48613069T>C\", \"start\": 48613069, \"end\": 48613069, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"mutationtaster_pred\": \"A\", \"cds_start\": 188, \"mutfunc\": {\"exp\": {\"dG_mt\": 111.260799, \"dG_wt_sd\": 0.000547, \"ddG\": 0.341199, \"dG_wt\": 110.9196, \"dG_mt_sd\": 0.000447, \"ddG_sd\": 0.000447}}, \"amino_acids\": \"Y/C\", \"cadd_raw\": 4.500084, \"given_ref\": \"T\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48613068, \"end\": 48613069}}}, \"hgvsp\": \"NP_000129.3:p.Tyr63Cys\", \"polyphen_prediction\": \"possibly_damaging\", \"clinpred\": 0.971140205860138, \"location\": \"15:48613069\", \"aa\": \"T\", \"blosum62\": -2, \"exon\": \"3/66\", \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_DG\": -50, \"DP_AL\": 50, \"DS_DL\": 0, \"DP_AG\": 23, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 17}, \"codons\": \"tAt/tGt\", \"canonical\": 1, \"cadd_phred\": 25.5, \"mastermind_counts\": \"9|9|9\", \"hgvsc\": \"NM_000138.5:c.188A>G\", \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"C\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"alphamissense\": {\"am_class\": \"ambiguous\", \"am_pathogenicity\": 0.561}, \"transcript_id\": \"NM_000138.5\", \"cdna_start\": 504, \"cdna_end\": 504, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:Y63C\", \"sift_score\": 0.01, \"phaplo\": 0.99897615427687, \"protein_end\": 63, \"loeuf\": 0.105, \"conservation\": 3.6, \"polyphen_score\": 0.894, \"protein_start\": 63, \"cds_end\": 188, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\"}]}',\n",
       "    'coordinates': [{'pca_x': 0.0336967762169999,\n",
       "      'pca_y': 0.07923168782673953},\n",
       "     {'t-sne_x': -0.06498631658938216, 't-sne_y': 3.923508938022403},\n",
       "     {'umap_x': 4.755611896514893, 'umap_y': 2.681628465652466}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463207-T-C',\n",
       "      'distance': 0.0017777078319340944,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420719-T-C',\n",
       "      'distance': 0.0019189838785678148,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415671-T-C',\n",
       "      'distance': 0.001922742580063641,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472607-T-C',\n",
       "      'distance': 0.0020311791449785233,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415605-T-C',\n",
       "      'distance': 0.0020510288886725903,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430727-T-C',\n",
       "      'distance': 0.0020572016946971416,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.0020630829967558384,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48537676-C-T',\n",
       "      'distance': 0.002256041159853339,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468468-T-C',\n",
       "      'distance': 0.0022787966299802065,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446795-C-T',\n",
       "      'distance': 0.0024325731210410595,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497298-T-C',\n",
       "      'distance': 0.0024383473210036755,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481687-T-C',\n",
       "      'distance': 0.002603178843855858,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487107-T-C',\n",
       "      'distance': 0.002648907480761409,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456729-C-T',\n",
       "      'distance': 0.0026590051129460335,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48537718-C-T',\n",
       "      'distance': 0.0026663867756724358,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427602-C-T',\n",
       "      'distance': 0.0026795603334903717,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497322-T-C',\n",
       "      'distance': 0.0026795980520546436,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437872-C-T',\n",
       "      'distance': 0.0026855492033064365,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.0027112732641398907,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432976-C-T',\n",
       "      'distance': 0.00273498916067183,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48613054-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48613054\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48613054',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.203G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48613054C>G',\n",
       "     'hgvs_protein': 'p.Cys68Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 68, \"cds_end\": 203, \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.883, \"conservation\": 5.92, \"loeuf\": 0.105, \"protein_end\": 68, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"mastermind_mmid3\": \"FBN1:C68S\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_end\": 519, \"cdna_start\": 519, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.999, \"am_class\": \"likely_pathogenic\"}, \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"G\", \"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.203G>C\", \"codons\": \"tGc/tCc\", \"canonical\": 1, \"cadd_phred\": 28.8, \"mastermind_counts\": \"4|6|9\", \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_DG\": 32, \"DP_AL\": 38, \"DS_DL\": 0, \"DP_AG\": -35, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -48}, \"aa\": \"C\", \"exon\": \"3/66\", \"blosum62\": -1, \"location\": \"15:48613054\", \"polyphen_prediction\": \"possibly_damaging\", \"clinpred\": 0.996410667896271, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48613054, \"start\": 48613053, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}}, \"hgvsp\": \"NP_000129.3:p.Cys68Ser\", \"given_ref\": \"C\", \"cadd_raw\": 5.139022, \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"amino_acids\": \"C/S\", \"strand\": -1, \"mutationtaster_pred\": \"D\", \"cds_start\": 203, \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.22594, \"dG_mt_sd\": 0.221784, \"dG_wt\": 110.9124, \"ddG\": 1.8788, \"dG_wt_sd\": 0.01099, \"dG_mt\": 112.7912}}, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\"}], \"end\": 48613054, \"start\": 48613054, \"id\": \"NC_000015.10:g.48613054C>G\", \"colocated_variants\": [{\"start\": 48613054, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48613054, \"phenotype_or_disease\": 1, \"id\": \"CM073061\"}, {\"id\": \"CM074845\", \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48613054, \"end\": 48613054}], \"variant_class\": \"SNV\", \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48613054C>G\", \"vcf_string\": \"15-48613054-C-G\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/G\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.03206784045467176,\n",
       "      'pca_y': 0.08874210905952848},\n",
       "     {'t-sne_x': 9.8520583175083, 't-sne_y': 15.46810971669273},\n",
       "     {'umap_x': 8.64865779876709, 'umap_y': 7.231654167175293}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.0024933721870183945,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.0036946050822734833,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.004183313809335232,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.007763073779642582,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537790-C-T',\n",
       "      'distance': 0.007778463885188103,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.007975682616233826,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.008014954626560211,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.008018014952540398,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.008187400177121162,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.008572237566113472,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432910-C-G',\n",
       "      'distance': 0.008657574653625488,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.008782001212239265,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.008791869506239891,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.009036055766046047,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-A',\n",
       "      'distance': 0.009113956242799759,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.009152484126389027,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48537718-C-T',\n",
       "      'distance': 0.009173965081572533,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.009297024458646774,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.00930339191108942,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.009380742907524109,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48596292-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48596292\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48596292',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.529T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48596292A>G',\n",
       "     'hgvs_protein': 'p.Cys177Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48596292A>G\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48596292-A-G\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/G\", \"start\": 48596292, \"most_severe_consequence\": \"missense_variant\", \"end\": 48596292, \"transcript_consequences\": [{\"hgvsp\": \"NP_000129.3:p.Cys177Arg\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48596292, \"start\": 48596291}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"cadd_raw\": 5.521929, \"given_ref\": \"A\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"clinpred\": 0.999780833721161, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48596292\", \"mutationtaster_pred\": \"A\", \"strand\": -1, \"cds_start\": 529, \"mutfunc\": {\"exp\": {\"dG_wt\": 108.2838, \"dG_mt_sd\": 1.144924, \"ddG_sd\": 0.985986, \"dG_mt\": 120.094, \"dG_wt_sd\": 0.601882, \"ddG\": 11.8102}}, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"amino_acids\": \"C/R\", \"gene_symbol_source\": \"EntrezGene\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"G\", \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_DG\": -9, \"DP_AL\": -9, \"DS_DL\": 0, \"DP_AG\": 18, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 2}, \"aa\": \"A\", \"blosum62\": -3, \"exon\": \"6/66\", \"hgvsc\": \"NM_000138.5:c.529T>C\", \"biotype\": \"protein_coding\", \"codons\": \"Tgt/Cgt\", \"canonical\": 1, \"cadd_phred\": 32, \"mastermind_counts\": \"7|9|9\", \"lrt_pred\": \"invalid_field\", \"cdna_end\": 845, \"cdna_start\": 845, \"mastermind_mmid3\": \"FBN1:C177R\", \"mane\": [\"MANE_Select\"], \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9997}, \"polyphen_score\": 0.963, \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 529, \"protein_start\": 177, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 177, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"loeuf\": 0.105, \"conservation\": 3.79}], \"minimised\": 1, \"id\": \"NC_000015.10:g.48596292A>G\", \"colocated_variants\": [{\"end\": 48596292, \"start\": 48596292, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"id\": \"CM054702\", \"phenotype_or_disease\": 1}, {\"phenotype_or_disease\": 1, \"id\": \"CM085416\", \"end\": 48596292, \"start\": 48596292, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\"}, {\"start\": 48596292, \"allele_string\": \"A/G\", \"strand\": 1, \"clin_sig\": [\"likely_pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"UniProt\": [\"VAR_023861\"], \"ClinVar\": [\"RCV000035222\", \"VCV000042380\"]}, \"end\": 48596292, \"phenotype_or_disease\": 1, \"id\": \"rs363853\", \"pubmed\": [24033266, 10486319, 16222657, 18435798], \"clin_sig_allele\": \"G:likely_pathogenic\"}], \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.04277271782013789,\n",
       "      'pca_y': 0.07275940449820018},\n",
       "     {'t-sne_x': 34.824809260376384, 't-sne_y': 4.137305105641391},\n",
       "     {'umap_x': 1.1839848756790161, 'umap_y': 6.459169387817383}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.0008672833209857345,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.0009359978721477091,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.001118034590035677,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0011579982237890363,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0011761401547119021,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.001260636723600328,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0012732874602079391,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.0012801105622202158,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.001366081414744258,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0014316129963845015,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48452673-A-G',\n",
       "      'distance': 0.0014635708648711443,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.0014775667805224657,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526223-A-G',\n",
       "      'distance': 0.0014942185953259468,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.001504222396761179,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.0015158600872382522,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0015331989852711558,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596373-A-G',\n",
       "      'distance': 0.001574410474859178,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0015819747932255268,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.001603057375177741,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.0016273939982056618,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48520711-G-C',\n",
       "    'metadata': {'vcf_string': '15\\t48520711\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48520711',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1095C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48520711G>C',\n",
       "     'hgvs_protein': 'p.Cys365Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48520711, \"end\": 48520711, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"spliceai\": {\"DP_AG\": -45, \"DS_DL\": 0, \"DP_DL\": -2, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_AG\": 0, \"DS_DG\": 0, \"DP_AL\": 50, \"DP_DG\": 1}, \"aa\": \"G\", \"exon\": \"10/66\", \"blosum62\": -2, \"hgvsc\": \"NM_000138.5:c.1095C>G\", \"biotype\": \"protein_coding\", \"canonical\": 1, \"codons\": \"tgC/tgG\", \"mastermind_counts\": \"4|4|4\", \"cadd_phred\": 26.9, \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"C\", \"mutationtaster_pred\": \"D\", \"strand\": -1, \"cds_start\": 1095, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"amino_acids\": \"C/W\", \"hgvsp\": \"NP_000129.3:p.Cys365Trp\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48520711, \"start\": 48520710, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"given_ref\": \"G\", \"cadd_raw\": 4.809158, \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"clinpred\": 0.999355733394623, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48520711\", \"protein_end\": 365, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"loeuf\": 0.105, \"conservation\": 3.95, \"polyphen_score\": 0.939, \"used_ref\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 1095, \"protein_start\": 365, \"sift_prediction\": \"deleterious_low_confidence\", \"eve_score\": 0.767781474755326, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9981}, \"lrt_pred\": \"invalid_field\", \"cdna_end\": 1411, \"cdna_start\": 1411, \"mastermind_mmid3\": \"FBN1:C365W\", \"mane\": [\"MANE_Select\"]}], \"minimised\": 1, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"end\": 48520711, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48520711, \"phenotype_or_disease\": 1, \"id\": \"CM074854\"}, {\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48520711, \"end\": 48520711, \"id\": \"CM145703\", \"phenotype_or_disease\": 1}], \"id\": \"NC_000015.10:g.48520711G>C\", \"input\": \"NC_000015.10:g.48520711G>C\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"G/C\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48520711-G-C\"}',\n",
       "    'coordinates': [{'pca_x': 0.03575134373941041,\n",
       "      'pca_y': 0.09513970461598398},\n",
       "     {'t-sne_x': 19.130634715101532, 't-sne_y': -7.732079426372001},\n",
       "     {'umap_x': 8.3751802444458, 'umap_y': 7.383533477783203}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.009495558217167854,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.00978129543364048,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.010076274164021015,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520708-C-G',\n",
       "      'distance': 0.010233529843389988,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468455-G-C',\n",
       "      'distance': 0.010364411398768425,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492562-G-C',\n",
       "      'distance': 0.010580179281532764,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.010703478939831257,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.010722139850258827,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505081-T-C',\n",
       "      'distance': 0.010727855376899242,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.010805870406329632,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.010990415699779987,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437347-G-C',\n",
       "      'distance': 0.011033049784600735,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.011047612875699997,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.011136592365801334,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437784-G-C',\n",
       "      'distance': 0.011155558750033379,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495162-C-G',\n",
       "      'distance': 0.011175565421581268,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.011237839236855507,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441798-C-T',\n",
       "      'distance': 0.011275341734290123,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.01131155714392662,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487333-G-C',\n",
       "      'distance': 0.011376140639185905,\n",
       "      'pathogenicity': 'benign'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48508633-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48508633\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48508633',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1786T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48508633A>G',\n",
       "     'hgvs_protein': 'p.Cys596Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48508633-A-G\", \"allele_string\": \"A/G\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48508633A>G\", \"id\": \"NC_000015.10:g.48508633A>G\", \"colocated_variants\": [{\"end\": 48508633, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48508633, \"phenotype_or_disease\": 1, \"id\": \"CM074802\"}, {\"phenotype_or_disease\": 1, \"id\": \"CM117993\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48508633, \"end\": 48508633}, {\"clin_sig_allele\": \"G:pathogenic;C:pathogenic\", \"id\": \"rs1057520131\", \"phenotype_or_disease\": 1, \"pubmed\": [25741868, 17657824, 17701892], \"end\": 48508633, \"allele_string\": \"A/C/G\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV001958689\", \"VCV001458822\", \"RCV001861486\", \"RCV000439856\", \"VCV000377096\"], \"UniProt\": [\"VAR_076015\"]}, \"seq_region_name\": \"15\", \"clin_sig\": [\"pathogenic\"], \"start\": 48508633}], \"variant_class\": \"SNV\", \"minimised\": 1, \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.9996, \"am_class\": \"likely_pathogenic\"}, \"eve_score\": 0.718293121397678, \"mastermind_mmid3\": \"FBN1:C596R\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_start\": 2102, \"cdna_end\": 2102, \"conservation\": 3.95, \"loeuf\": 0.105, \"protein_end\": 596, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 1786, \"protein_start\": 596, \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.994, \"amino_acids\": \"C/R\", \"strand\": -1, \"mutationtaster_pred\": \"D\", \"cds_start\": 1786, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48508633\", \"clinpred\": 0.999857425689697, \"hgvsp\": \"NP_000129.3:p.Cys596Arg\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48508632, \"end\": 48508633, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"cadd_raw\": 5.331608, \"given_ref\": \"A\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"hgvsc\": \"NM_000138.5:c.1786T>C\", \"biotype\": \"protein_coding\", \"codons\": \"Tgt/Cgt\", \"canonical\": 1, \"cadd_phred\": 30, \"mastermind_counts\": \"6|6|6\", \"spliceai\": {\"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -50, \"DS_DL\": 0, \"DP_AG\": 14, \"DP_DG\": -1, \"DP_AL\": -24, \"DS_DG\": 0, \"DS_AG\": 0}, \"aa\": \"A\", \"exon\": \"15/66\", \"blosum62\": -3, \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"G\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\"}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48508633, \"start\": 48508633}',\n",
       "    'coordinates': [{'pca_x': 0.040249336891101106,\n",
       "      'pca_y': 0.10286175766800806},\n",
       "     {'t-sne_x': -19.19225580968423, 't-sne_y': 34.0918894362825},\n",
       "     {'umap_x': 2.1353530883789062, 'umap_y': 20.673154830932617}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48513627-A-G',\n",
       "      'distance': 0.002106759697198868,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513631-A-G',\n",
       "      'distance': 0.002106759697198868,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48463148-A-G',\n",
       "      'distance': 0.0021609398536384106,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456643-A-G',\n",
       "      'distance': 0.0023303190246224403,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505136-A-G',\n",
       "      'distance': 0.0025720405392348766,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452637-A-G',\n",
       "      'distance': 0.0026502057444304228,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513600-A-G',\n",
       "      'distance': 0.002894874196499586,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515434-C-A',\n",
       "      'distance': 0.003170803189277649,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445454-A-G',\n",
       "      'distance': 0.003173891454935074,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472644-A-G',\n",
       "      'distance': 0.0036057266406714916,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495216-A-G',\n",
       "      'distance': 0.0036819116212427616,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510050-A-G',\n",
       "      'distance': 0.0037378277629613876,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.003931066021323204,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515465-G-A',\n",
       "      'distance': 0.004275234416127205,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495243-A-T',\n",
       "      'distance': 0.0043226489797234535,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.004330805037170649,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508632-C-T',\n",
       "      'distance': 0.004539160057902336,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495155-G-A',\n",
       "      'distance': 0.0048418897204101086,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-G',\n",
       "      'distance': 0.004946446977555752,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505117-C-A',\n",
       "      'distance': 0.004989561624825001,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48490037-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48490037\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48490037',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2896G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48490037C>A',\n",
       "     'hgvs_protein': 'p.Glu966Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"id\": \"NC_000015.10:g.48490037C>A\", \"colocated_variants\": [{\"end\": 48490037, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48490037, \"id\": \"CM074825\", \"phenotype_or_disease\": 1}, {\"end\": 48490037, \"start\": 48490037, \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"seq_region_name\": \"15\", \"var_synonyms\": {\"COSMIC\": [\"COSM5901900\"]}, \"somatic\": 1, \"id\": \"COSV57323688\", \"phenotype_or_disease\": 1}, {\"end\": 48490037, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000600367\", \"VCV000517122\", \"RCV001228545\"]}, \"clin_sig\": [\"pathogenic\"], \"strand\": 1, \"allele_string\": \"C/A\", \"start\": 48490037, \"clin_sig_allele\": \"A:pathogenic\", \"pubmed\": [17657824], \"id\": \"rs1555398830\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\", \"start\": 48490037, \"transcript_consequences\": [{\"amino_acids\": \"E/*\", \"cds_start\": 2896, \"mutationtaster_pred\": \"A\", \"strand\": -1, \"impact\": \"HIGH\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"location\": \"15:48490037\", \"hgvsp\": \"NP_000129.3:p.Glu966Ter\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48490037, \"start\": 48490036, \"type\": \"SimpleInterval\"}}, \"type\": \"Allele\"}, \"allele_num\": 1, \"consequence_terms\": [\"stop_gained\"], \"given_ref\": \"C\", \"cadd_raw\": 9.187479, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.2896G>T\", \"mastermind_counts\": \"4|4|4\", \"cadd_phred\": 40, \"canonical\": 1, \"codons\": \"Gag/Tag\", \"spliceai\": {\"DP_DL\": 8, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AG\": -3, \"DS_DL\": 0, \"DP_AL\": 6, \"DP_DG\": 14, \"DS_AG\": 0, \"DS_DG\": 0}, \"lof\": \"HC\", \"exon\": \"25/66\", \"aa\": \"C\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"A\", \"lof_info\": \"PERCENTILE:0.336118848653668,GERP_DIST:6053.72533015311,BP_DIST:5679,DIST_FROM_LAST_EXON:5290,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol_source\": \"EntrezGene\", \"transcript_id\": \"NM_000138.5\", \"mastermind_mmid3\": \"FBN1:E966X\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_end\": 3212, \"cdna_start\": 3212, \"loeuf\": 0.105, \"conservation\": 3.76, \"protein_end\": 966, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"cds_end\": 2896, \"protein_start\": 966}], \"most_severe_consequence\": \"stop_gained\", \"end\": 48490037, \"strand\": 1, \"vcf_string\": \"15-48490037-C-A\", \"allele_string\": \"C/A\", \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48490037C>A\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.016910945696111276,\n",
       "      'pca_y': 0.1000453884440893},\n",
       "     {'t-sne_x': 21.93277300116523, 't-sne_y': 42.87478785227668},\n",
       "     {'umap_x': -9.440598487854004, 'umap_y': 4.609256744384766}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48505139-C-A',\n",
       "      'distance': 0.0021538985893130302,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.002520448062568903,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.0025255901273339987,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.0027197953313589096,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.0027542721945792437,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411085-C-A',\n",
       "      'distance': 0.00277898320928216,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488233-C-A',\n",
       "      'distance': 0.0032615340314805508,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483878-C-A',\n",
       "      'distance': 0.0035684588365256786,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452676-C-A',\n",
       "      'distance': 0.003574883099645376,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.003932581748813391,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48534170-G-A',\n",
       "      'distance': 0.003980087116360664,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437837-C-A',\n",
       "      'distance': 0.003985814284533262,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.0040715401992201805,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415675-G-A',\n",
       "      'distance': 0.00412964029237628,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.004220058210194111,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515397-C-T',\n",
       "      'distance': 0.004376812372356653,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48495219-G-A',\n",
       "      'distance': 0.004401305690407753,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513615-G-A',\n",
       "      'distance': 0.004473998211324215,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.004504370037466288,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468080-A-C',\n",
       "      'distance': 0.00455758161842823,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48489964-GTACCCCAG-G',\n",
       "    'metadata': {'vcf_string': '15\\t48489964\\t.\\tGTACCCCAG\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48489964',\n",
       "     'ref_allele': 'GTACCCCAG',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2964_2971del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48489965_48489972del',\n",
       "     'hgvs_protein': 'p.Trp988Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48489965_48489972del\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"TACCCCAG/-\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48489964-GTACCCCAG-G\", \"start\": 48489965, \"most_severe_consequence\": \"stop_gained\", \"transcript_consequences\": [{\"lof_info\": \"PERCENTILE:0.344823584029712,GERP_DIST:6053.72533015311,BP_DIST:5607,DIST_FROM_LAST_EXON:5218,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"canonical\": 1, \"codons\": \"tgGGGTACTGag/tgag\", \"hgvsc\": \"NM_000138.5:c.2964_2971del\", \"biotype\": \"protein_coding\", \"exon\": \"25/66\", \"aa\": \"TACCCCAG\", \"lof\": \"HC\", \"location\": \"15:48489962-48489969\", \"hgvs_offset\": -3, \"consequence_terms\": [\"stop_gained\", \"frameshift_variant\"], \"allele_num\": 1, \"given_ref\": \"TACCCCAG\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"start\": 48489964, \"end\": 48489972, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}}, \"hgvsp\": \"NP_000129.3:p.Trp988Ter\", \"amino_acids\": \"WGTE/*\", \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"genesplicer\": \"loss/donor/48489967-48489968/Medium/4.394877,gain/acceptor/48489966-48489965/Medium/4.079870\", \"cds_start\": 2964, \"strand\": -1, \"protein_start\": 988, \"cds_end\": 2971, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"CAGTACCC\", \"loeuf\": 0.105, \"conservation\": 1.865, \"phaplo\": 0.99897615427687, \"protein_end\": 991, \"mane\": [\"MANE_Select\"], \"cdna_start\": 3280, \"cdna_end\": 3287, \"transcript_id\": \"NM_000138.5\"}], \"end\": 48489972, \"minimised\": 1, \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48489965_48489972del\"}',\n",
       "    'coordinates': [{'pca_x': 0.018536894756500805,\n",
       "      'pca_y': 0.08450050866935607},\n",
       "     {'t-sne_x': 19.02829569407806, 't-sne_y': 48.781413653568755},\n",
       "     {'umap_x': -9.582863807678223, 'umap_y': 5.857517719268799}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.080416239798069,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.08189146220684052,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.08214472979307175,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.08235123753547668,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.08278235793113708,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520764-G-A',\n",
       "      'distance': 0.08291727304458618,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.08336129039525986,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.0839184820652008,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.0842205062508583,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.08426953852176666,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508612-G-A',\n",
       "      'distance': 0.08478406816720963,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.08493877202272415,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.08500424772500992,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.08501638472080231,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48516225-G-A',\n",
       "      'distance': 0.08502660691738129,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.08535483479499817,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.08535642176866531,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489921-G-T',\n",
       "      'distance': 0.0853736400604248,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537704-G-A',\n",
       "      'distance': 0.08540767431259155,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437912-G-A',\n",
       "      'distance': 0.08544915914535522,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488192-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48488192\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488192',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3258T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488192A>T',\n",
       "     'hgvs_protein': 'p.Cys1086Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"vcf_string\": \"15-48488192-A-T\", \"allele_string\": \"A/T\", \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48488192A>T\", \"seq_region_name\": \"15\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48488192A>T\", \"colocated_variants\": [{\"id\": \"CM074866\", \"phenotype_or_disease\": 1, \"start\": 48488192, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"end\": 48488192}, {\"id\": \"CM950444\", \"phenotype_or_disease\": 1, \"start\": 48488192, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"end\": 48488192}, {\"end\": 48488192, \"var_synonyms\": {\"COSMIC\": [\"COSM9241817\"]}, \"seq_region_name\": \"15\", \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"start\": 48488192, \"somatic\": 1, \"id\": \"COSV104413955\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\", \"start\": 48488192, \"transcript_consequences\": [{\"gene_symbol_source\": \"EntrezGene\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"T\", \"lof_info\": \"PERCENTILE:0.378133704735376,GERP_DIST:6053.72533015311,BP_DIST:5319,DIST_FROM_LAST_EXON:4930,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"lof\": \"HC\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 9, \"SYMBOL\": \"FBN1\", \"DP_DL\": 2, \"DS_AL\": 0, \"DS_DG\": 0, \"DS_AG\": 0.01, \"DP_AL\": 49, \"DP_DG\": 4}, \"aa\": \"A\", \"exon\": \"27/66\", \"hgvsc\": \"NM_000138.5:c.3258T>A\", \"biotype\": \"protein_coding\", \"codons\": \"tgT/tgA\", \"canonical\": 1, \"cadd_phred\": 35, \"mastermind_counts\": \"4|4|4\", \"hgvsp\": \"NP_000129.3:p.Cys1086Ter\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48488192, \"start\": 48488191}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"given_ref\": \"A\", \"cadd_raw\": 7.226374, \"consequence_terms\": [\"stop_gained\"], \"allele_num\": 1, \"location\": \"15:48488192\", \"mutationtaster_pred\": \"A\", \"strand\": -1, \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.023822, \"dG_mt_sd\": 0.023822, \"dG_wt\": 175.402, \"dG_wt_sd\": 0, \"ddG\": -0.14599, \"dG_mt\": 175.25601}}, \"cds_start\": 3258, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"amino_acids\": \"C/*\", \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 1086, \"cds_end\": 3258, \"protein_end\": 1086, \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"conservation\": 1.53, \"lrt_pred\": \"invalid_field\", \"cdna_start\": 3574, \"cdna_end\": 3574, \"mastermind_mmid3\": \"FBN1:C1086X\", \"mane\": [\"MANE_Select\"], \"transcript_id\": \"NM_000138.5\"}], \"most_severe_consequence\": \"stop_gained\", \"end\": 48488192}',\n",
       "    'coordinates': [{'pca_x': 0.024982752743204023,\n",
       "      'pca_y': 0.09141335032677281},\n",
       "     {'t-sne_x': 25.680117282260046, 't-sne_y': 37.56017230283464},\n",
       "     {'umap_x': -9.862269401550293, 'umap_y': 2.5474283695220947}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.0009592833812348545,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537615-A-T',\n",
       "      'distance': 0.0014047878794372082,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48467954-A-T',\n",
       "      'distance': 0.0020133848302066326,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.00275564007461071,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.00335276173427701,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.003703922498971224,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489970-C-T',\n",
       "      'distance': 0.00453002005815506,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48467975-C-T',\n",
       "      'distance': 0.0046282680705189705,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.004681774415075779,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488233-C-A',\n",
       "      'distance': 0.004702693782746792,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437805-C-T',\n",
       "      'distance': 0.004740844946354628,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411338-C-T',\n",
       "      'distance': 0.004829273093491793,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.004879935644567013,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.004959233570843935,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485483-G-T',\n",
       "      'distance': 0.00497338455170393,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489969-C-T',\n",
       "      'distance': 0.005044678226113319,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495241-G-T',\n",
       "      'distance': 0.005175513215363026,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495575-G-T',\n",
       "      'distance': 0.005242970772087574,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420700-C-T',\n",
       "      'distance': 0.00531751848757267,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421995-G-T',\n",
       "      'distance': 0.005324927158653736,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48465859-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48465859\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48465859',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4748-1G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48465859C>T',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48465859C>T\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48465859-C-T\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\", \"start\": 48465859, \"end\": 48465859, \"most_severe_consequence\": \"splice_acceptor_variant\", \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"rf_score\": 0.94, \"mastermind_mmid3\": \"FBN1:S1583sa\", \"intron\": \"38/65\", \"transcript_id\": \"NM_000138.5\", \"maxentscan_alt\": -2.22, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"conservation\": 3.95, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"location\": \"15:48465859\", \"maxentscan_diff\": 8.75, \"cadd_raw\": 5.318078, \"given_ref\": \"C\", \"consequence_terms\": [\"splice_acceptor_variant\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48465859, \"start\": 48465858}, \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"ada_score\": 0.999987670894945, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"strand\": -1, \"genesplicer\": \"loss/acceptor/48465858-48465859/Medium/3.228909\", \"lof_info\": \"INTRON_SIZE:2079\", \"variant_allele\": \"T\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"maxentscan_ref\": 6.53, \"canonical\": 1, \"cadd_phred\": 29.9, \"mastermind_counts\": \"5|5|6\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.4748-1G>A\", \"aa\": \"C\", \"lof\": \"HC\", \"spliceai\": {\"DP_AL\": -1, \"DP_DG\": -6, \"DS_AG\": 0.01, \"DS_DG\": 0, \"DP_DL\": -33, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0.98, \"DP_AG\": -6, \"DS_DL\": 0}}], \"minimised\": 1, \"id\": \"NC_000015.10:g.48465859C>T\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"end\": 48465859, \"start\": 48465859, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"id\": \"CS075138\"}, {\"id\": \"CS154790\", \"phenotype_or_disease\": 1, \"end\": 48465859, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48465859}, {\"phenotype_or_disease\": 1, \"id\": \"rs2141270027\", \"pubmed\": [17657824], \"clin_sig_allele\": \"T:likely_pathogenic\", \"start\": 48465859, \"allele_string\": \"C/T\", \"strand\": 1, \"clin_sig\": [\"likely_pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV001377265\", \"VCV001066300\"]}, \"end\": 48465859}]}',\n",
       "    'coordinates': [{'pca_x': -0.237444999348768,\n",
       "      'pca_y': -0.0348091737694267},\n",
       "     {'t-sne_x': -28.965767564008473, 't-sne_y': -18.741097991935913},\n",
       "     {'umap_x': 11.849842071533203, 'umap_y': 18.744897842407227}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.0014952090568840504,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.0020637516863644123,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.002264021895825863,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.0025317836552858353,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.002558428095653653,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.002611436415463686,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.0026729260571300983,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.002897983882576227,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.003194281831383705,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.0033472515642642975,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.0033551338128745556,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.0035052881576120853,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.0036571600940078497,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.0036674472503364086,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.00412236712872982,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.004206816665828228,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-A',\n",
       "      'distance': 0.005111355800181627,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.005497637204825878,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470757-C-T',\n",
       "      'distance': 0.006140821613371372,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446823-C-G',\n",
       "      'distance': 0.006506213918328285,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48465612-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48465612\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48465612',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4898G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48465612C>G',\n",
       "     'hgvs_protein': 'p.Cys1633Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"C/G\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48465612-C-G\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48465612C>G\", \"colocated_variants\": [{\"id\": \"CM074862\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48465612, \"end\": 48465612}, {\"somatic\": 1, \"id\": \"COSV57311447\", \"phenotype_or_disease\": 1, \"end\": 48465612, \"var_synonyms\": {\"COSMIC\": [\"COSM317347\"]}, \"seq_region_name\": \"15\", \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"start\": 48465612}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48465612C>G\", \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"end\": 48465612, \"transcript_consequences\": [{\"cdna_start\": 5214, \"cdna_end\": 5214, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C1633S\", \"eve_score\": 0.797752234798084, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9994}, \"transcript_id\": \"NM_000138.5\", \"polyphen_score\": 0.999, \"cds_end\": 4898, \"protein_start\": 1633, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 1633, \"loeuf\": 0.105, \"conservation\": 3.95, \"cadd_raw\": 5.156867, \"given_ref\": \"C\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"start\": 48465611, \"end\": 48465612, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Cys1633Ser\", \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48465612\", \"clinpred\": 0.999774038791656, \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"strand\": -1, \"mutationtaster_pred\": \"D\", \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 0.098322, \"ddG_sd\": 0.096996, \"dG_wt\": 22.68068, \"dG_mt\": 26.88268, \"dG_wt_sd\": 0.013351, \"ddG\": 4.202}, \"mod\": {\"dG_wt\": 81.13572, \"ddG_sd\": 0.021301, \"dG_mt_sd\": 0.018128, \"ddG\": 3.91476, \"dG_wt_sd\": 0.009138, \"dG_mt\": 85.05048}}, \"cds_start\": 4898, \"amino_acids\": \"C/S\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"variant_allele\": \"G\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"aa\": \"C\", \"blosum62\": -1, \"exon\": \"40/66\", \"spliceai\": {\"DS_AG\": 0, \"DS_DG\": 0, \"DP_DG\": -44, \"DP_AL\": 7, \"DP_AG\": -27, \"DS_DL\": 0.02, \"DS_AL\": 0, \"DP_DL\": -34, \"SYMBOL\": \"FBN1\"}, \"canonical\": 1, \"codons\": \"tGt/tCt\", \"cadd_phred\": 28.9, \"mastermind_counts\": \"4|4|4\", \"hgvsc\": \"NM_000138.5:c.4898G>C\", \"biotype\": \"protein_coding\"}], \"start\": 48465612}',\n",
       "    'coordinates': [{'pca_x': 0.042087546584589075,\n",
       "      'pca_y': 0.0914652474721083},\n",
       "     {'t-sne_x': 17.18011259689652, 't-sne_y': -5.552613301520325},\n",
       "     {'umap_x': 8.655729293823242, 'umap_y': 7.027880668640137}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48432910-C-G',\n",
       "      'distance': 0.0012898811837658286,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.0022462077904492617,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.0025653024204075336,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.002571074292063713,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.002904998604208231,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489991-C-G',\n",
       "      'distance': 0.0030527771450579166,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437064-G-C',\n",
       "      'distance': 0.003061004914343357,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.003153200726956129,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0035188256297260523,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.0035224747844040394,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.0035589360632002354,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.0035703126341104507,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415716-T-C',\n",
       "      'distance': 0.0035981652326881886,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.003695641178637743,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.003730400465428829,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48463129-C-T',\n",
       "      'distance': 0.003801121609285474,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434627-C-T',\n",
       "      'distance': 0.003810194553807378,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472670-T-C',\n",
       "      'distance': 0.003812646260485053,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.0038603818975389004,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421652-G-C',\n",
       "      'distance': 0.003861946053802967,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48463232-TTCT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48463232\\t.\\tTTCT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48463232',\n",
       "     'ref_allele': 'TTCT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5076_5078del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48463233_48463235del',\n",
       "     'hgvs_protein': 'p.Arg1692del',\n",
       "     'most_severe_consequence': 'inframe_deletion'},\n",
       "    'annotation_raw': '{\"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48463233_48463235del\", \"minimised\": 1, \"transcript_consequences\": [{\"amino_acids\": \"RS/S\", \"strand\": -1, \"cds_start\": 5076, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"hgvs_offset\": -5, \"location\": \"15:48463228-48463230\", \"hgvsp\": \"NP_000129.3:p.Arg1692del\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48463235, \"start\": 48463232}, \"type\": \"SequenceLocation\"}}, \"given_ref\": \"TCT\", \"consequence_terms\": [\"inframe_deletion\"], \"allele_num\": 1, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.5076_5078del\", \"codons\": \"agAAGt/agt\", \"canonical\": 1, \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_DG\": -5, \"DP_AL\": -2, \"DS_DL\": 0, \"DP_AG\": -5, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -3}, \"aa\": \"TCT\", \"exon\": \"42/66\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"-\", \"gene_symbol_source\": \"EntrezGene\", \"transcript_id\": \"NM_000138.5\", \"mane\": [\"MANE_Select\"], \"cdna_start\": 5392, \"cdna_end\": 5394, \"loeuf\": 0.105, \"conservation\": 3.18, \"protein_end\": 1693, \"phaplo\": 0.99897615427687, \"used_ref\": \"CTT\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 5078, \"protein_start\": 1692}], \"most_severe_consequence\": \"inframe_deletion\", \"end\": 48463235, \"start\": 48463233, \"assembly_name\": \"GRCh38\", \"allele_string\": \"TCT/-\", \"vcf_string\": \"15-48463232-TTCT-T\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48463233_48463235del\"}',\n",
       "    'coordinates': [{'pca_x': 0.017055253988597335,\n",
       "      'pca_y': 0.06089558594802511},\n",
       "     {'t-sne_x': 5.015814259466448, 't-sne_y': 3.1100226657339967},\n",
       "     {'umap_x': 4.105144023895264, 'umap_y': 4.343475341796875}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430676-C-T',\n",
       "      'distance': 0.09696464985609055,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465825-C-T',\n",
       "      'distance': 0.09725016355514526,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.09775212407112122,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596325-A-T',\n",
       "      'distance': 0.09831666946411133,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.09838521480560303,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613073-G-A',\n",
       "      'distance': 0.09840114414691925,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48464009-C-T',\n",
       "      'distance': 0.09885773062705994,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.09886472672224045,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.0988674908876419,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09887609630823135,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434701-C-T',\n",
       "      'distance': 0.09897412359714508,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.09905672073364258,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.0990959107875824,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470684-C-T',\n",
       "      'distance': 0.09910464286804199,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09910997748374939,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.09941988438367844,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437872-C-T',\n",
       "      'distance': 0.09945950657129288,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.09949292242527008,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48436966-C-T',\n",
       "      'distance': 0.09951722621917725,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.09954096376895905,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48446800-AT-A',\n",
       "    'metadata': {'vcf_string': '15\\t48446800\\t.\\tAT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48446800',\n",
       "     'ref_allele': 'AT',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5693del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48446801del',\n",
       "     'hgvs_protein': 'p.Asp1898ValfsTer32',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"T/-\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48446800-AT-A\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48446801del\", \"colocated_variants\": [{\"start\": 48446801, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"end\": 48446801, \"id\": \"CD075411\", \"phenotype_or_disease\": 1}], \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48446801del\", \"minimised\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"cds_start\": 5693, \"amino_acids\": \"D/X\", \"given_ref\": \"T\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48446800, \"end\": 48446801, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"hgvsp\": \"NP_000129.3:p.Asp1898ValfsTer32\", \"location\": \"15:48446801\", \"aa\": \"T\", \"exon\": \"47/66\", \"lof\": \"HC\", \"spliceai\": {\"DP_AL\": 22, \"DP_DG\": 22, \"DS_DG\": 0, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 2, \"DS_AL\": 0.02, \"DS_DL\": 0, \"DP_AG\": 3}, \"codons\": \"gAt/gt\", \"canonical\": 1, \"mastermind_counts\": \"3|3|3\", \"hgvsc\": \"NM_000138.5:c.5693del\", \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"lof_info\": \"PERCENTILE:0.660747446610956,GERP_DIST:3337.29206062555,BP_DIST:2904,DIST_FROM_LAST_EXON:2515,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"transcript_id\": \"NM_000138.5\", \"cdna_end\": 6009, \"cdna_start\": 6009, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:D1898fs\", \"phaplo\": 0.99897615427687, \"protein_end\": 1898, \"conservation\": 3.95, \"loeuf\": 0.105, \"protein_start\": 1898, \"cds_end\": 5693, \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\"}], \"end\": 48446801, \"start\": 48446801}',\n",
       "    'coordinates': [{'pca_x': 0.05106461596207884,\n",
       "      'pca_y': 0.03035764497267395},\n",
       "     {'t-sne_x': 1.8022726229631754, 't-sne_y': 9.603204929019746},\n",
       "     {'umap_x': 3.7097368240356445, 'umap_y': 3.313716173171997}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.0990307554602623,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10061010718345642,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10201097279787064,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.10203676670789719,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.10234487056732178,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.1032857820391655,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.10331220924854279,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.10346169024705887,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.10371625423431396,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.10372410714626312,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445419-G-T',\n",
       "      'distance': 0.10400344431400299,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.1040215864777565,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10410286486148834,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.10412649810314178,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.10417353361845016,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10430798679590225,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.10456936806440353,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.10477758944034576,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.10488717257976532,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.10495888441801071,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48444616-TACCTGGT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48444616\\t.\\tTACCTGGT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48444616',\n",
       "     'ref_allele': 'TACCTGGT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5958_5964del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48444617_48444623del',\n",
       "     'hgvs_protein': 'p.Gly1987ValfsTer70',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48444617_48444623del\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"ACCTGGT/-\", \"vcf_string\": \"15-48444616-TACCTGGT-T\", \"strand\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"cds_end\": 5964, \"protein_start\": 1986, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"GGTACCT\", \"conservation\": 0.73, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"protein_end\": 1988, \"mane\": [\"MANE_Select\"], \"cdna_start\": 6274, \"cdna_end\": 6280, \"transcript_id\": \"NM_000138.5\", \"lof_info\": \"PERCENTILE:0.692200557103064,GERP_DIST:3144.30482168049,BP_DIST:2638,DIST_FROM_LAST_EXON:2249,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"gene_symbol_source\": \"EntrezGene\", \"codons\": \"ccAGGTACC/cc\", \"canonical\": 1, \"hgvsc\": \"NM_000138.5:c.5958_5964del\", \"biotype\": \"protein_coding\", \"exon\": \"49/66\", \"aa\": \"ACCTGGT\", \"lof\": \"HC\", \"location\": \"15:48444614-48444620\", \"hgvs_offset\": -3, \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"given_ref\": \"ACCTGGT\", \"hgvsp\": \"NP_000129.3:p.Gly1987ValfsTer70\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48444616, \"end\": 48444623}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"amino_acids\": \"PGT/X\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"genesplicer\": \"loss/donor/48444619-48444620/Medium/2.233884\", \"cds_start\": 5958, \"strand\": -1}], \"end\": 48444623, \"start\": 48444617, \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48444617_48444623del\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.03783982814617301,\n",
       "      'pca_y': 0.04333388746108841},\n",
       "     {'t-sne_x': 39.64431079863496, 't-sne_y': 6.567856869752452},\n",
       "     {'umap_x': 1.1100642681121826, 'umap_y': 7.360523700714111}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48444585-C-T',\n",
       "      'distance': 0.09441835433244705,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445382-A-G',\n",
       "      'distance': 0.09577460587024689,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444611-A-C',\n",
       "      'distance': 0.09580889344215393,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.0968349277973175,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.09907980263233185,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445408-T-C',\n",
       "      'distance': 0.0998205691576004,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446768-A-G',\n",
       "      'distance': 0.10002665221691132,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.10041249543428421,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.10067570209503174,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.10124732553958893,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445493-A-G',\n",
       "      'distance': 0.10136167705059052,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446811-A-G',\n",
       "      'distance': 0.10158374160528183,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.10191459953784943,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.10228648036718369,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.10265832394361496,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.1027657687664032,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.10301992297172546,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48452589-G-A',\n",
       "      'distance': 0.1034051924943924,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.10343022644519806,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.10355204343795776,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48444586-AT-A',\n",
       "    'metadata': {'vcf_string': '15\\t48444586\\t.\\tAT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48444586',\n",
       "     'ref_allele': 'AT',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5991del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48444587del',\n",
       "     'hgvs_protein': 'p.Arg1997SerfsTer62',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"start\": 48444587, \"transcript_consequences\": [{\"protein_end\": 1997, \"phaplo\": 0.99897615427687, \"conservation\": -4.26, \"loeuf\": 0.105, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"T\", \"cds_end\": 5991, \"protein_start\": 1997, \"transcript_id\": \"NM_000138.5\", \"cdna_start\": 6307, \"cdna_end\": 6307, \"mastermind_mmid3\": \"FBN1:R1997fs\", \"mane\": [\"MANE_Select\"], \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": -45, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": 0, \"DP_AL\": 1, \"DP_DG\": 33, \"DS_DG\": 0.04, \"DS_AG\": 0}, \"lof\": \"HC\", \"exon\": \"49/66\", \"aa\": \"T\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.5991del\", \"mastermind_counts\": \"3|3|3\", \"codons\": \"agA/ag\", \"canonical\": 1, \"gene_symbol_source\": \"EntrezGene\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"-\", \"lof_info\": \"PERCENTILE:0.69533426183844,GERP_DIST:3096.98392118812,BP_DIST:2608,DIST_FROM_LAST_EXON:2219,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"cds_start\": 5991, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"amino_acids\": \"R/X\", \"hgvsp\": \"NP_000129.3:p.Arg1997SerfsTer62\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"start\": 48444586, \"end\": 48444587}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"given_ref\": \"T\", \"location\": \"15:48444587\"}], \"most_severe_consequence\": \"frameshift_variant\", \"end\": 48444587, \"minimised\": 1, \"colocated_variants\": [{\"end\": 48444587, \"start\": 48444587, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"id\": \"CD075406\", \"phenotype_or_disease\": 1}], \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48444587del\", \"input\": \"NC_000015.10:g.48444587del\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"T/-\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48444586-AT-A\"}',\n",
       "    'coordinates': [{'pca_x': 0.05515070422294349,\n",
       "      'pca_y': 0.02995089886689003},\n",
       "     {'t-sne_x': 15.753444340642162, 't-sne_y': 4.906455184847852},\n",
       "     {'umap_x': 3.5328848361968994, 'umap_y': 3.758267641067505}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.10989328473806381,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.11136306822299957,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.11289370059967041,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.11381540447473526,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.11517682671546936,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.11525040864944458,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445419-G-T',\n",
       "      'distance': 0.1155344694852829,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.11612847447395325,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.11680489778518677,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.1168566420674324,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.11703251302242279,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.1172436997294426,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.1173461303114891,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445430-G-A',\n",
       "      'distance': 0.11745737493038177,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.11768067628145218,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520711-G-T',\n",
       "      'distance': 0.11775797605514526,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510065-G-A',\n",
       "      'distance': 0.11776989698410034,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.11791298538446426,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.11794356256723404,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537704-G-A',\n",
       "      'distance': 0.11815021932125092,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48437918-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48437918\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48437918',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6164-1G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48437918C>A',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"vcf_string\": \"15-48437918-C-A\", \"allele_string\": \"C/A\", \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48437918C>A\", \"seq_region_name\": \"15\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48437918C>A\", \"colocated_variants\": [{\"id\": \"COSV57315691\", \"phenotype_or_disease\": 1, \"somatic\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM962372\"]}, \"seq_region_name\": \"15\", \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"start\": 48437918, \"end\": 48437918}, {\"start\": 48437918, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48437918, \"phenotype_or_disease\": 1, \"id\": \"CS055584\"}, {\"end\": 48437918, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48437918, \"phenotype_or_disease\": 1, \"id\": \"CS075140\"}, {\"end\": 48437918, \"start\": 48437918, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"id\": \"CS1312107\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\", \"start\": 48437918, \"transcript_consequences\": [{\"maxentscan_alt\": -1.637, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"conservation\": 3.72, \"intron\": \"50/65\", \"mane\": [\"MANE_Select\"], \"rf_score\": 0.944, \"mastermind_mmid3\": \"FBN1:D2055sa\", \"transcript_id\": \"NM_000138.5\", \"gene_symbol_source\": \"EntrezGene\", \"lof_info\": \"INTRON_SIZE:3803\", \"variant_allele\": \"A\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"aa\": \"C\", \"lof\": \"HC\", \"spliceai\": {\"DP_AG\": 36, \"DS_DL\": 0, \"DP_DL\": -1, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0.96, \"DS_AG\": 0.03, \"DS_DG\": 0, \"DP_AL\": -1, \"DP_DG\": 41}, \"canonical\": 1, \"maxentscan_ref\": 6.961, \"mastermind_counts\": \"3|3|11\", \"cadd_phred\": 29.9, \"hgvsc\": \"NM_000138.5:c.6164-1G>T\", \"biotype\": \"protein_coding\", \"cadd_raw\": 5.309527, \"given_ref\": \"C\", \"allele_num\": 1, \"consequence_terms\": [\"splice_acceptor_variant\"], \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48437918, \"start\": 48437917, \"type\": \"SimpleInterval\"}}, \"type\": \"Allele\"}, \"location\": \"15:48437918\", \"maxentscan_diff\": 8.598, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"ada_score\": 0.999989125585301}], \"most_severe_consequence\": \"splice_acceptor_variant\", \"end\": 48437918}',\n",
       "    'coordinates': [{'pca_x': -0.2351843341478938,\n",
       "      'pca_y': -0.028360226001357326},\n",
       "     {'t-sne_x': -28.749045514951124, 't-sne_y': -18.929528570266218},\n",
       "     {'umap_x': 11.825786590576172, 'umap_y': 18.76871109008789}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.0036669508554041386,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.003921429626643658,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.004060880281031132,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.004176287911832333,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.00462390948086977,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.004813436418771744,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.004892006050795317,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.00524806696921587,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.005315120331943035,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.005370941013097763,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.005969918332993984,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-A',\n",
       "      'distance': 0.006279729772359133,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.006362613290548325,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.0063912891782820225,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.006960134021937847,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.007831932976841927,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446823-C-G',\n",
       "      'distance': 0.008598335087299347,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.009175071492791176,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470757-C-T',\n",
       "      'distance': 0.009792453609406948,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.010917183011770248,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48430793-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48430793\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48430793',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6749A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48430793T>C',\n",
       "     'hgvs_protein': 'p.Glu2250Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48430793T>C\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"T/C\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48430793-T-C\", \"start\": 48430793, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"eve_score\": 0.808882900766818, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.9733, \"am_class\": \"likely_pathogenic\"}, \"lrt_pred\": \"invalid_field\", \"cdna_end\": 7065, \"cdna_start\": 7065, \"mastermind_mmid3\": \"FBN1:E2250G\", \"mane\": [\"MANE_Select\"], \"protein_end\": 2250, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"loeuf\": 0.105, \"conservation\": 3.7, \"polyphen_score\": 0.992, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"T\", \"sift_prediction\": \"deleterious_low_confidence\", \"cds_end\": 6749, \"protein_start\": 2250, \"cds_start\": 6749, \"mutationtaster_pred\": \"D\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"amino_acids\": \"E/G\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48430793, \"start\": 48430792}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Glu2250Gly\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"cadd_raw\": 5.597205, \"given_ref\": \"T\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.999408364295959, \"location\": \"15:48430793\", \"spliceai\": {\"DP_DG\": 0, \"DP_AL\": 9, \"DS_AG\": 0, \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": -2, \"SYMBOL\": \"FBN1\", \"DP_AG\": -9, \"DS_DL\": 0}, \"exon\": \"56/66\", \"blosum62\": -2, \"aa\": \"T\", \"hgvsc\": \"NM_000138.5:c.6749A>G\", \"biotype\": \"protein_coding\", \"cadd_phred\": 32, \"mastermind_counts\": \"6|7|7\", \"canonical\": 1, \"codons\": \"gAg/gGg\", \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"C\"}], \"end\": 48430793, \"minimised\": 1, \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM074820\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48430793, \"end\": 48430793}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48430793T>C\"}',\n",
       "    'coordinates': [{'pca_x': 0.03458655616333517,\n",
       "      'pca_y': 0.08075749979298316},\n",
       "     {'t-sne_x': 8.544383651139205, 't-sne_y': -1.277540824490216},\n",
       "     {'umap_x': 4.872214317321777, 'umap_y': 3.98791241645813}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48420798-C-T',\n",
       "      'distance': 0.002222915180027485,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434707-T-C',\n",
       "      'distance': 0.0026993504725396633,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468474-C-T',\n",
       "      'distance': 0.0032195569947361946,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485490-T-C',\n",
       "      'distance': 0.0032796647865325212,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470684-C-T',\n",
       "      'distance': 0.0033633909188210964,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488439-T-C',\n",
       "      'distance': 0.003489653579890728,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48467996-G-C',\n",
       "      'distance': 0.003502527717500925,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472670-T-C',\n",
       "      'distance': 0.003539770608767867,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437872-C-T',\n",
       "      'distance': 0.0035478880163282156,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489977-C-T',\n",
       "      'distance': 0.0036392174661159515,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48430676-C-T',\n",
       "      'distance': 0.0036613349802792072,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.0037030247040092945,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487107-T-C',\n",
       "      'distance': 0.00374114653095603,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489991-C-G',\n",
       "      'distance': 0.003849982749670744,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437784-G-C',\n",
       "      'distance': 0.0038528931327164173,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.0038570610340684652,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428459-C-T',\n",
       "      'distance': 0.003858284093439579,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425483-C-T',\n",
       "      'distance': 0.0038831960409879684,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.00392613559961319,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420719-T-C',\n",
       "      'distance': 0.004003499634563923,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48425480-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48425480\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48425480',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7342T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48425480A>G',\n",
       "     'hgvs_protein': 'p.Cys2448Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48425480, \"end\": 48425480, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"cadd_phred\": 29.5, \"mastermind_counts\": \"7|7|7\", \"codons\": \"Tgc/Cgc\", \"canonical\": 1, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.7342T>C\", \"blosum62\": -3, \"exon\": \"60/66\", \"aa\": \"A\", \"spliceai\": {\"DP_DG\": -8, \"DP_AL\": -46, \"DS_AG\": 0, \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": 38, \"SYMBOL\": \"FBN1\", \"DP_AG\": 11, \"DS_DL\": 0}, \"variant_allele\": \"G\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"C/R\", \"impact\": \"MODERATE\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"cds_start\": 7342, \"mutationtaster_pred\": \"A\", \"strand\": -1, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48425480\", \"clinpred\": 0.999889254570007, \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"given_ref\": \"A\", \"cadd_raw\": 5.250465, \"hgvsp\": \"NP_000129.3:p.Cys2448Arg\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48425480, \"start\": 48425479}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"loeuf\": 0.105, \"conservation\": 3.95, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 2448, \"sift_prediction\": \"deleterious_low_confidence\", \"cds_end\": 7342, \"protein_start\": 2448, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"A\", \"polyphen_score\": 0.994, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9993}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.77266545519059, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C2448R\", \"cdna_end\": 7658, \"cdna_start\": 7658, \"lrt_pred\": \"invalid_field\"}], \"minimised\": 1, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48425480, \"end\": 48425480, \"id\": \"CM062711\", \"phenotype_or_disease\": 1}, {\"start\": 48425480, \"allele_string\": \"A/G\", \"strand\": 1, \"var_synonyms\": {\"UniProt\": [\"VAR_076141\"], \"ClinVar\": [\"RCV000696700\", \"VCV000574697\"]}, \"seq_region_name\": \"15\", \"clin_sig\": [\"pathogenic\"], \"end\": 48425480, \"phenotype_or_disease\": 1, \"id\": \"rs1566892757\", \"pubmed\": [17657824, 17701892], \"clin_sig_allele\": \"G:pathogenic\"}], \"id\": \"NC_000015.10:g.48425480A>G\", \"input\": \"NC_000015.10:g.48425480A>G\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"A/G\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48425480-A-G\"}',\n",
       "    'coordinates': [{'pca_x': 0.034536812454344784,\n",
       "      'pca_y': 0.1052908681687719},\n",
       "     {'t-sne_x': -19.024089009344546, 't-sne_y': 32.81816435342411},\n",
       "     {'umap_x': 2.296086311340332, 'umap_y': 20.67706871032715}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48495216-A-G',\n",
       "      'distance': 0.0012022124137729406,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472644-A-G',\n",
       "      'distance': 0.0013505311217159033,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.001366433221846819,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.0014147949405014515,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463148-A-G',\n",
       "      'distance': 0.0016198366647586226,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494210-A-G',\n",
       "      'distance': 0.0018461471190676093,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456643-A-G',\n",
       "      'distance': 0.0018877482507377863,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513627-A-G',\n",
       "      'distance': 0.0021982830949127674,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513631-A-G',\n",
       "      'distance': 0.0021982830949127674,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48513600-A-G',\n",
       "      'distance': 0.0023057907819747925,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445454-A-G',\n",
       "      'distance': 0.0023658527061343193,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515434-C-A',\n",
       "      'distance': 0.003098096465691924,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510050-A-G',\n",
       "      'distance': 0.003212051000446081,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495243-A-T',\n",
       "      'distance': 0.0032690870575606823,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48452637-A-G',\n",
       "      'distance': 0.0032952807378023863,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.0033424734137952328,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515465-G-A',\n",
       "      'distance': 0.0033523966558277607,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474345-A-C',\n",
       "      'distance': 0.0036609501112252474,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415585-C-A',\n",
       "      'distance': 0.0036942334845662117,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505136-A-G',\n",
       "      'distance': 0.003788209520280361,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48415531-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48415531\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415531',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8051+5G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415531C>T',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_5th_base_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48415531C>T\", \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48415531-C-T\", \"strand\": 1, \"most_severe_consequence\": \"splice_donor_5th_base_variant\", \"transcript_consequences\": [{\"strand\": -1, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"LOW\", \"ada_score\": 0.999898957920387, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48415531, \"start\": 48415530}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"consequence_terms\": [\"splice_donor_5th_base_variant\", \"intron_variant\"], \"allele_num\": 1, \"given_ref\": \"C\", \"cadd_raw\": 2.82721, \"maxentscan_diff\": 4.588, \"location\": \"15:48415531\", \"spliceai\": {\"DP_AG\": 5, \"DS_DL\": 0.42, \"DS_AL\": 0, \"DP_DL\": 5, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DG\": 0.02, \"DP_DG\": -20, \"DP_AL\": 11}, \"aa\": \"C\", \"hgvsc\": \"NM_000138.5:c.8051+5G>A\", \"biotype\": \"protein_coding\", \"cadd_phred\": 20.5, \"mastermind_counts\": \"8|9|24\", \"maxentscan_ref\": 9.04, \"canonical\": 1, \"gene_symbol_source\": \"EntrezGene\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"T\", \"transcript_id\": \"NM_000138.5\", \"intron\": \"64/65\", \"mastermind_mmid3\": \"FBN1:G2684int,FBN1:G2684intd,FBN1:G2684srd\", \"rf_score\": 0.918, \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"conservation\": 5.15, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"maxentscan_alt\": 4.452}], \"end\": 48415531, \"start\": 48415531, \"colocated_variants\": [{\"end\": 48415531, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48415531, \"phenotype_or_disease\": 1, \"id\": \"CS930807\"}, {\"clin_sig\": [\"uncertain_significance\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV001176127\", \"RCV000557828\", \"VCV000457268\", \"RCV002245998\", \"RCV004525961\", \"RCV004696933\"]}, \"allele_string\": \"C/T\", \"strand\": 1, \"start\": 48415531, \"end\": 48415531, \"frequencies\": {\"T\": {\"gnomadg_fin\": 0, \"gnomade_mid\": 0, \"gnomadg_sas\": 0, \"gnomade_fin\": 0, \"gnomade_sas\": 0, \"gnomadg_nfe\": 2.939e-05, \"gnomadg_remaining\": 0, \"gnomade_remaining\": 1.665e-05, \"gnomadg_asj\": 0, \"gnomadg_ami\": 0, \"gnomadg_amr\": 0, \"gnomade_afr\": 0, \"gnomadg\": 1.314e-05, \"gnomade_eas\": 0, \"gnomade_asj\": 0, \"gnomade_amr\": 0, \"gnomadg_afr\": 0, \"gnomade\": 6.056e-05, \"gnomade_nfe\": 7.879e-05, \"gnomadg_eas\": 0, \"gnomadg_mid\": 0}}, \"pubmed\": [12938084, 17657824], \"id\": \"rs767384075\", \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"T:conflicting_classifications_of_pathogenicity;T:uncertain_significance\"}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48415531C>T\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.24265733297701492,\n",
       "      'pca_y': -0.03841136760570965},\n",
       "     {'t-sne_x': -18.79866070099826, 't-sne_y': -33.343721657546965},\n",
       "     {'umap_x': 13.654034614562988, 'umap_y': 12.353372573852539}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48412564-C-T',\n",
       "      'distance': 0.0011919382959604263,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508577-C-T',\n",
       "      'distance': 0.0029135034419596195,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446701-C-T',\n",
       "      'distance': 0.0034745815210044384,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460241-C-T',\n",
       "      'distance': 0.003604544559493661,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515385-A-G',\n",
       "      'distance': 0.003632210660725832,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446701-C-A',\n",
       "      'distance': 0.00437938142567873,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437765-T-C',\n",
       "      'distance': 0.011914282105863094,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437318-T-C',\n",
       "      'distance': 0.012396220117807388,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48489845-T-C',\n",
       "      'distance': 0.013348313048481941,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48445370-A-G',\n",
       "      'distance': 0.013968870043754578,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487309-T-C',\n",
       "      'distance': 0.014095843769609928,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48427561-A-C',\n",
       "      'distance': 0.01435135304927826,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48472544-C-A',\n",
       "      'distance': 0.020616034045815468,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520652-C-T',\n",
       "      'distance': 0.021670421585440636,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.022664368152618408,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.023155447095632553,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.023276161402463913,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497265-C-T',\n",
       "      'distance': 0.02358189970254898,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.02376057580113411,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441720-C-T',\n",
       "      'distance': 0.02402089536190033,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.6},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7}}},\n",
       "   {'variant_id': '15-48412641-AC-A',\n",
       "    'metadata': {'vcf_string': '15\\t48412641\\t.\\tAC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48412641',\n",
       "     'ref_allele': 'AC',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8153del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48412642del',\n",
       "     'hgvs_protein': 'p.Cys2718LeufsTer34',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"frameshift_variant\", \"end\": 48412642, \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"mastermind_mmid3\": \"FBN1:C2718fs\", \"mane\": [\"MANE_Select\"], \"cdna_end\": 8469, \"cdna_start\": 8469, \"conservation\": 5.35, \"loeuf\": 0.105, \"protein_end\": 2718, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"protein_start\": 2718, \"cds_end\": 8153, \"amino_acids\": \"C/X\", \"cds_start\": 8153, \"genesplicer\": \"loss/donor/48412642-48412643/Medium/0.662595\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"location\": \"15:48412642\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48412642, \"start\": 48412641, \"type\": \"SimpleInterval\"}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}}, \"hgvsp\": \"NP_000129.3:p.Cys2718LeufsTer34\", \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"given_ref\": \"C\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.8153del\", \"mastermind_counts\": \"2|2|2\", \"canonical\": 1, \"codons\": \"tGt/tt\", \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": 2, \"SYMBOL\": \"FBN1\", \"DP_AG\": 14, \"DS_DL\": 0, \"DP_DG\": 4, \"DP_AL\": 16, \"DS_AG\": 0, \"DS_DG\": 0}, \"lof\": \"HC\", \"exon\": \"65/66\", \"aa\": \"C\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"-\", \"lof_info\": \"PERCENTILE:0.946262766945218,GERP_DIST:455.113940283656,BP_DIST:462,DIST_FROM_LAST_EXON:73,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol_source\": \"EntrezGene\"}], \"start\": 48412642, \"id\": \"NC_000015.10:g.48412642del\", \"variant_class\": \"deletion\", \"colocated_variants\": [{\"end\": 48412642, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48412642, \"phenotype_or_disease\": 1, \"id\": \"CD075412\"}], \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48412642del\", \"vcf_string\": \"15-48412641-AC-A\", \"allele_string\": \"C/-\", \"assembly_name\": \"GRCh38\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.0066017216908547255,\n",
       "      'pca_y': 0.043040234094503686},\n",
       "     {'t-sne_x': 1.3929781078888337, 't-sne_y': 10.329285164748777},\n",
       "     {'umap_x': 3.7263565063476562, 'umap_y': 3.225414514541626}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.10086636245250702,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.10130142420530319,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.10201650857925415,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.10352513194084167,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.10445773601531982,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.1046433225274086,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.10497650504112244,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.10509738326072693,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446747-C-T',\n",
       "      'distance': 0.10517971962690353,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48452567-C-T',\n",
       "      'distance': 0.10537177324295044,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.1053796261548996,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434701-C-T',\n",
       "      'distance': 0.10555065423250198,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596291-C-T',\n",
       "      'distance': 0.10555915534496307,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.10558198392391205,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.10568294674158096,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445408-T-C',\n",
       "      'distance': 0.10577686876058578,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.10580246150493622,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48441798-C-T',\n",
       "      'distance': 0.10581381618976593,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48537718-C-T',\n",
       "      'distance': 0.10589438676834106,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456714-C-T',\n",
       "      'distance': 0.10602171719074249,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48412619-G-A',\n",
       "    'metadata': {'vcf_string': '15\\t48412619\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48412619',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8176C>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48412619G>A',\n",
       "     'hgvs_protein': 'p.Arg2726Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"G/A\", \"vcf_string\": \"15-48412619-G-A\", \"input\": \"NC_000015.10:g.48412619G>A\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"start\": 48412619, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48412619, \"phenotype_or_disease\": 1, \"id\": \"CM950453\"}, {\"seq_region_name\": \"15\", \"var_synonyms\": {\"UniProt\": [\"VAR_002348\"], \"OMIM\": [134797.0023], \"ClinVar\": [\"RCV000029787\", \"VCV000016445\", \"RCV000306003\", \"RCV000399653\", \"RCV000394448\", \"RCV000337198\", \"RCV000181628\", \"RCV000243696\", \"RCV000279856\", \"RCV000617106\", \"RCV000586191\", \"RCV001081571\", \"RCV005007863\"]}, \"clin_sig\": [\"uncertain_significance\", \"likely_benign\"], \"strand\": 1, \"allele_string\": \"G/A/C\", \"start\": 48412619, \"frequencies\": {\"A\": {\"gnomadg_eas\": 0, \"eur\": 0, \"amr\": 0.0014, \"af\": 0.0002, \"gnomadg_mid\": 0.003401, \"gnomade_afr\": 0.0001195, \"gnomadg\": 0.0007092, \"gnomade_eas\": 0, \"gnomade_amr\": 0.001655, \"gnomade_asj\": 0, \"gnomadg_afr\": 0.0004571, \"gnomade\": 0.0007613, \"gnomade_nfe\": 0.0008714, \"gnomadg_nfe\": 0.0007792, \"gnomade_remaining\": 0.0007285, \"gnomadg_remaining\": 0.00142, \"gnomadg_asj\": 0, \"eas\": 0, \"gnomadg_ami\": 0, \"gnomadg_amr\": 0.002092, \"afr\": 0, \"gnomade_mid\": 0.0008669, \"gnomadg_fin\": 0, \"gnomadg_sas\": 0, \"sas\": 0, \"gnomade_fin\": 7.488e-05, \"gnomade_sas\": 0.0001507}}, \"end\": 48412619, \"pubmed\": [25741868, 25637381, 27930701, 24941995, 16333834, 11700157, 8563763, 16222657, 18435798, 17024, 7738200, 10756346, 12915484, 15598221, 19059503, 17657824, 17701892, 20375004, 23278365, 20979188, 21594992, 21594993, 24039054, 26875674, 24564502], \"id\": \"rs61746008\", \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"A:conflicting_classifications_of_pathogenicity;A:likely_benign\"}], \"id\": \"NC_000015.10:g.48412619G>A\", \"start\": 48412619, \"end\": 48412619, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"exon\": \"65/66\", \"blosum62\": -3, \"aa\": \"G\", \"spliceai\": {\"DP_AL\": 38, \"DP_DG\": 24, \"DS_DG\": 0.02, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -50, \"DS_AL\": 0.07, \"DS_DL\": 0.01, \"DP_AG\": -8}, \"cadd_phred\": 26.9, \"mastermind_counts\": \"37|68|68\", \"canonical\": 1, \"codons\": \"Cgg/Tgg\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.8176C>T\", \"eve_class\": \"Benign\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"A\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"cds_start\": 8176, \"mutationtaster_pred\": \"N\", \"strand\": -1, \"amino_acids\": \"R/W\", \"geno2mp_url\": \"https://geno2mp.gs.washington.edu/Geno2MP/#/variant/15/48704816/G%3EA/snp\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"given_ref\": \"G\", \"cadd_raw\": 4.825706, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48412618, \"end\": 48412619}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}}, \"hgvsp\": \"NP_000129.3:p.Arg2726Trp\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.0705803102813939, \"location\": \"15:48412619\", \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 2726, \"conservation\": 2.14, \"loeuf\": 0.105, \"geno2mp_hpo_count\": 22, \"polyphen_score\": 0.972, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 2726, \"cds_end\": 8176, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"G\", \"eve_score\": 0.195085055945207, \"alphamissense\": {\"am_pathogenicity\": 0.1993, \"am_class\": \"likely_benign\"}, \"transcript_id\": \"NM_000138.5\", \"cdna_start\": 8492, \"cdna_end\": 8492, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:R2726W\"}]}',\n",
       "    'coordinates': [{'pca_x': 0.04346414512768143,\n",
       "      'pca_y': 0.0883185957177487},\n",
       "     {'t-sne_x': 30.764886713689428, 't-sne_y': 10.357741675672619},\n",
       "     {'umap_x': 1.9526809453964233, 'umap_y': 7.997565269470215}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48415549-G-A',\n",
       "      'distance': 0.00124024145770818,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.0014968428295105696,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411243-G-A',\n",
       "      'distance': 0.0016554677858948708,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48490013-G-A',\n",
       "      'distance': 0.0017784200608730316,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428427-G-A',\n",
       "      'distance': 0.0017953277565538883,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437379-G-A',\n",
       "      'distance': 0.0018654002342373133,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427768-G-A',\n",
       "      'distance': 0.001971377059817314,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437009-G-A',\n",
       "      'distance': 0.002145719015970826,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430710-G-A',\n",
       "      'distance': 0.0026795505546033382,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48481723-G-A',\n",
       "      'distance': 0.0028067068196833134,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48537629-G-A',\n",
       "      'distance': 0.0028255837969481945,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490073-G-A',\n",
       "      'distance': 0.0030471771024167538,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.003053754335269332,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468097-G-A',\n",
       "      'distance': 0.0032442472875118256,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472665-A-G',\n",
       "      'distance': 0.0033061944413930178,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.0033516662660986185,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497332-G-A',\n",
       "      'distance': 0.003403428941965103,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48490050-C-A',\n",
       "      'distance': 0.0034215589985251427,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48434600-A-G',\n",
       "      'distance': 0.0034462157636880875,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.0035544224083423615,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8}}},\n",
       "   {'variant_id': '15-48644625-CTC-C',\n",
       "    'metadata': {'vcf_string': '15\\t48644625\\t.\\tCTC\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48644625',\n",
       "     'ref_allele': 'CTC',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.144_145del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48644626_48644627del',\n",
       "     'hgvs_protein': 'p.Gly49ThrfsTer5',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"start\": 48644626, \"end\": 48644627, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"amino_acids\": \"GG/GX\", \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"strand\": -1, \"cds_start\": 144, \"mutfunc\": {\"exp\": {\"dG_wt_sd\": 0, \"ddG\": 2.158799, \"dG_mt\": 113.078799, \"dG_wt\": 110.92, \"ddG_sd\": 0.002683, \"dG_mt_sd\": 0.002683}}, \"hgvs_offset\": -1, \"location\": \"15:48644625-48644626\", \"given_ref\": \"TC\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48644627, \"start\": 48644625}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"hgvsp\": \"NP_000129.3:p.Gly49ThrfsTer5\", \"canonical\": 1, \"codons\": \"ggAGga/ggga\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.144_145del\", \"aa\": \"TC\", \"exon\": \"2/66\", \"lof\": \"HC\", \"spliceai\": {\"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -19, \"DS_DL\": 0.04, \"DP_AG\": -19, \"DP_DG\": 6, \"DP_AL\": 39, \"DS_DG\": 0, \"DS_AG\": 0}, \"variant_allele\": \"-\", \"lof_info\": \"PERCENTILE:0.0168291550603528,GERP_DIST:8531.86538855145,BP_DIST:8408,DIST_FROM_LAST_EXON:8019,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"gene_symbol_source\": \"EntrezGene\", \"transcript_id\": \"NM_000138.5\", \"mane\": [\"MANE_Select\"], \"cdna_end\": 461, \"cdna_start\": 460, \"conservation\": 2.267, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"protein_end\": 49, \"cds_end\": 145, \"protein_start\": 48, \"used_ref\": \"CT\", \"mane_select\": \"ENST00000316623.10\"}], \"minimised\": 1, \"id\": \"NC_000015.10:g.48644626_48644627del\", \"variant_class\": \"deletion\", \"input\": \"NC_000015.10:g.48644626_48644627del\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48644625-CTC-C\", \"allele_string\": \"TC/-\", \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': 0.036650795950782356,\n",
       "      'pca_y': 0.04298640672058023},\n",
       "     {'t-sne_x': 7.632425621797569, 't-sne_y': 4.1971484280381945},\n",
       "     {'umap_x': 3.9425742626190186, 'umap_y': 3.1053264141082764}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10383427143096924,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.1047443300485611,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.10487684607505798,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10545738786458969,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.10594262182712555,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.10606741905212402,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10627011209726334,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.10630688071250916,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.10651910305023193,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10655686259269714,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.10684558004140854,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.10692892968654633,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.10695567727088928,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.10697172582149506,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.10739384591579437,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.10742325335741043,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.10757914185523987,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.10759842395782471,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489921-G-T',\n",
       "      'distance': 0.10760898143053055,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.10762961208820343,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48613036-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48613036\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48613036',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.221T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48613036A>T',\n",
       "     'hgvs_protein': 'p.Leu74Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48613036A>T\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48613036-A-T\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/T\", \"start\": 48613036, \"end\": 48613036, \"most_severe_consequence\": \"stop_gained\", \"transcript_consequences\": [{\"protein_end\": 74, \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"conservation\": 5.92, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"A\", \"cds_end\": 221, \"protein_start\": 74, \"transcript_id\": \"NM_000138.5\", \"lrt_pred\": \"invalid_field\", \"cdna_end\": 537, \"cdna_start\": 537, \"mastermind_mmid3\": \"FBN1:L74X\", \"mane\": [\"MANE_Select\"], \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -10, \"SYMBOL\": \"FBN1\", \"DP_DL\": -30, \"DS_AL\": 0, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": -17, \"DP_DG\": 3}, \"lof\": \"HC\", \"exon\": \"3/66\", \"aa\": \"A\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.221T>A\", \"cadd_phred\": 37, \"mastermind_counts\": \"2|2|2\", \"canonical\": 1, \"codons\": \"tTa/tAa\", \"gene_symbol_source\": \"EntrezGene\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"T\", \"lof_info\": \"PERCENTILE:0.0256499535747447,GERP_DIST:8432.24327185312,BP_DIST:8332,DIST_FROM_LAST_EXON:7943,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"cds_start\": 221, \"mutfunc\": {\"exp\": {\"dG_wt\": 110.5834, \"dG_mt_sd\": 0.561585, \"ddG_sd\": 0.315987, \"dG_mt\": 112.16, \"ddG\": 1.5766, \"dG_wt_sd\": 0.462889}}, \"strand\": -1, \"mutationtaster_pred\": \"A\", \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"amino_acids\": \"L/*\", \"hgvsp\": \"NP_000129.3:p.Leu74Ter\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48613036, \"start\": 48613035, \"type\": \"SimpleInterval\"}}, \"type\": \"Allele\"}, \"consequence_terms\": [\"stop_gained\"], \"allele_num\": 1, \"given_ref\": \"A\", \"cadd_raw\": 8.485294, \"location\": \"15:48613036\"}], \"minimised\": 1, \"id\": \"NC_000015.10:g.48613036A>T\", \"colocated_variants\": [{\"id\": \"CM131911\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48613036, \"end\": 48613036}], \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.020701107115129902,\n",
       "      'pca_y': 0.08908514581522328},\n",
       "     {'t-sne_x': 25.628930529586267, 't-sne_y': 37.618662399383275},\n",
       "     {'umap_x': -9.801517486572266, 'umap_y': 2.723545789718628}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48537615-A-T',\n",
       "      'distance': 0.002464604564011097,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.0030111053492873907,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.003112862352281809,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48467954-A-T',\n",
       "      'distance': 0.0033638097811490297,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.004783410578966141,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.005243356805294752,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.005583575926721096,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.006414362229406834,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437837-C-A',\n",
       "      'distance': 0.006423968356102705,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.006425332278013229,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437805-C-T',\n",
       "      'distance': 0.00648329546675086,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.006583276204764843,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.0066307662054896355,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.006641797721385956,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489921-G-T',\n",
       "      'distance': 0.006756450980901718,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48467975-C-T',\n",
       "      'distance': 0.0067869215272367,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.006828427314758301,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.006949060130864382,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.006978202611207962,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.007022390142083168,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48610829-G-C',\n",
       "    'metadata': {'vcf_string': '15\\t48610829\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48610829',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.248-3C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48610829G>C',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_region_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48610829G>C\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48610829-G-C\", \"allele_string\": \"G/C\", \"assembly_name\": \"GRCh38\", \"start\": 48610829, \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"LOW\", \"strand\": -1, \"ada_score\": 0.999914886326596, \"given_ref\": \"G\", \"cadd_raw\": 3.4232, \"allele_num\": 1, \"consequence_terms\": [\"splice_region_variant\", \"splice_polypyrimidine_tract_variant\", \"intron_variant\"], \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48610829, \"start\": 48610828}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"location\": \"15:48610829\", \"maxentscan_diff\": 11.489, \"aa\": \"G\", \"spliceai\": {\"DP_AL\": -3, \"DP_DG\": -1, \"DS_AG\": 0.19, \"DS_DG\": 0, \"DP_DL\": -3, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0.87, \"DP_AG\": -1, \"DS_DL\": 0}, \"maxentscan_ref\": 6.392, \"canonical\": 1, \"cadd_phred\": 22.5, \"mastermind_counts\": \"6|17|42\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.248-3C>G\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"C\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"transcript_id\": \"NM_000138.5\", \"intron\": \"3/65\", \"mane\": [\"MANE_Select\"], \"rf_score\": 0.982, \"mastermind_mmid3\": \"FBN1:P83int,FBN1:P83inta,FBN1:P83sra\", \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"conservation\": 4.94, \"maxentscan_alt\": -5.097, \"used_ref\": \"G\", \"mane_select\": \"ENST00000316623.10\"}], \"most_severe_consequence\": \"splice_region_variant\", \"end\": 48610829, \"minimised\": 1, \"id\": \"NC_000015.10:g.48610829G>C\", \"colocated_variants\": [{\"start\": 48610829, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48610829, \"phenotype_or_disease\": 1, \"id\": \"CS013964\"}, {\"clin_sig_allele\": \"C:likely_pathogenic\", \"phenotype_or_disease\": 1, \"id\": \"rs1555405537\", \"end\": 48610829, \"start\": 48610829, \"seq_region_name\": \"15\", \"clin_sig\": [\"likely_pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV000663548\", \"VCV000549089\"]}, \"strand\": 1, \"allele_string\": \"G/A/C\"}], \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': -0.2548793955307227,\n",
       "      'pca_y': -0.061624297468086245},\n",
       "     {'t-sne_x': -18.37078584396484, 't-sne_y': -39.78983987932761},\n",
       "     {'umap_x': 14.64780044555664, 'umap_y': 12.869588851928711}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48472684-C-G',\n",
       "      'distance': 0.0026583995204418898,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48468107-T-C',\n",
       "      'distance': 0.003740574698895216,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48490082-T-C',\n",
       "      'distance': 0.004695220850408077,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520823-G-T',\n",
       "      'distance': 0.005381627939641476,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48492594-G-A',\n",
       "      'distance': 0.00562267005443573,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48497397-C-T',\n",
       "      'distance': 0.005967282690107822,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48495596-A-G',\n",
       "      'distance': 0.0067195468582212925,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48465861-A-G',\n",
       "      'distance': 0.007039917167276144,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48505153-G-A',\n",
       "      'distance': 0.007916439324617386,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48510177-C-A',\n",
       "      'distance': 0.008206319063901901,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48468542-C-T',\n",
       "      'distance': 0.008293937891721725,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490086-A-G',\n",
       "      'distance': 0.00839720107614994,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48465864-G-A',\n",
       "      'distance': 0.008701205253601074,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48474385-A-G',\n",
       "      'distance': 0.009221641346812248,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48434587-G-C',\n",
       "      'distance': 0.009729165583848953,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48441854-A-T',\n",
       "      'distance': 0.01064610667526722,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48427560-G-C',\n",
       "      'distance': 0.012274861335754395,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48520652-C-T',\n",
       "      'distance': 0.0131513811647892,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48474658-A-G',\n",
       "      'distance': 0.014597536996006966,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48472544-C-A',\n",
       "      'distance': 0.014949450269341469,\n",
       "      'pathogenicity': 'likely_benign'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_benign',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_benign', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_benign',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_benign', 'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48610730-G-C',\n",
       "    'metadata': {'vcf_string': '15\\t48610730\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48610730',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.344C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48610730G>C',\n",
       "     'hgvs_protein': 'p.Ser115Cys',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"id\": \"NC_000015.10:g.48610730G>C\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"start\": 48610730, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48610730, \"phenotype_or_disease\": 1, \"id\": \"CM021993\"}], \"start\": 48610730, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"clinpred\": 0.954315423965454, \"polyphen_prediction\": \"possibly_damaging\", \"location\": \"15:48610730\", \"maxentscan_diff\": 0.026, \"cadd_raw\": 5.120226, \"given_ref\": \"G\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48610730, \"start\": 48610729}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}}, \"hgvsp\": \"NP_000129.3:p.Ser115Cys\", \"amino_acids\": \"S/C\", \"ada_score\": 0.828085286158523, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"D\", \"strand\": -1, \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.027395, \"dG_mt_sd\": 0.027395, \"dG_wt\": 110.92, \"ddG\": 0.480999, \"dG_wt_sd\": 0, \"dG_mt\": 111.400999}}, \"cds_start\": 344, \"variant_allele\": \"C\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"maxentscan_ref\": 9.092, \"codons\": \"tCc/tGc\", \"canonical\": 1, \"mastermind_counts\": \"6|13|13\", \"cadd_phred\": 28.7, \"hgvsc\": \"NM_000138.5:c.344C>G\", \"biotype\": \"protein_coding\", \"aa\": \"G\", \"exon\": \"4/66\", \"blosum62\": -1, \"spliceai\": {\"DP_DG\": 31, \"DP_AL\": 25, \"DS_DG\": 0, \"DS_AG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -2, \"DS_DL\": 0.09, \"DP_AG\": -1}, \"mane\": [\"MANE_Select\"], \"rf_score\": 0.624, \"mastermind_mmid3\": \"FBN1:I116srd,FBN1:S115C\", \"cdna_end\": 660, \"cdna_start\": 660, \"lrt_pred\": \"invalid_field\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.8307}, \"transcript_id\": \"NM_000138.5\", \"intact\": [{\"interaction_ac\": \"EBI-5400038\", \"feature_type\": \"mutation decreasing\"}], \"cds_end\": 344, \"protein_start\": 115, \"sift_prediction\": \"deleterious_low_confidence\", \"maxentscan_alt\": 9.066, \"used_ref\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.894, \"conservation\": 6.95, \"loeuf\": 0.105, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 115}], \"end\": 48610730, \"strand\": 1, \"vcf_string\": \"15-48610730-G-C\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"G/C\", \"input\": \"NC_000015.10:g.48610730G>C\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': -0.03930889460268576,\n",
       "      'pca_y': 0.0557053079657504},\n",
       "     {'t-sne_x': -13.168598133719565, 't-sne_y': -3.7956210665261043},\n",
       "     {'umap_x': 0.317466676235199, 'umap_y': 15.105649948120117}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.005648949649184942,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.005839512217789888,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.006324311718344688,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.006863688118755817,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.006932259537279606,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.007279584184288979,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.007360199466347694,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.00777808902785182,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.007814433425664902,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.007842982187867165,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.007847275584936142,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.00797234382480383,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.008171122521162033,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.008572838269174099,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.008682873100042343,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.009112859144806862,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.00912199355661869,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.01002911850810051,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.01010102778673172,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468534-T-C',\n",
       "      'distance': 0.010736685246229172,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48596292-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48596292\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48596292',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.529T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48596292A>T',\n",
       "     'hgvs_protein': 'p.Cys177Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"vcf_string\": \"15-48596292-A-T\", \"allele_string\": \"A/T\", \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48596292A>T\", \"seq_region_name\": \"15\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48596292A>T\", \"colocated_variants\": [{\"id\": \"CM054702\", \"phenotype_or_disease\": 1, \"end\": 48596292, \"start\": 48596292, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\"}, {\"phenotype_or_disease\": 1, \"id\": \"CM085416\", \"start\": 48596292, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48596292}], \"variant_class\": \"SNV\", \"start\": 48596292, \"most_severe_consequence\": \"missense_variant\", \"end\": 48596292, \"transcript_consequences\": [{\"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"T\", \"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.529T>A\", \"codons\": \"Tgt/Agt\", \"canonical\": 1, \"cadd_phred\": 29.8, \"mastermind_counts\": \"3|3|3\", \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": -6, \"SYMBOL\": \"FBN1\", \"DP_AG\": -2, \"DS_DL\": 0, \"DP_DG\": -15, \"DP_AL\": 0, \"DS_AG\": 0, \"DS_DG\": 0}, \"aa\": \"A\", \"blosum62\": -1, \"exon\": \"6/66\", \"polyphen_prediction\": \"possibly_damaging\", \"clinpred\": 0.998102724552155, \"location\": \"15:48596292\", \"hgvsp\": \"NP_000129.3:p.Cys177Ser\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48596292, \"start\": 48596291}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"given_ref\": \"A\", \"cadd_raw\": 5.298692, \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"amino_acids\": \"C/S\", \"strand\": -1, \"mutationtaster_pred\": \"D\", \"mutfunc\": {\"exp\": {\"dG_wt_sd\": 0.061398, \"ddG\": 0.417399, \"dG_mt\": 108.619999, \"dG_wt\": 108.2026, \"ddG_sd\": 0.001673, \"dG_mt_sd\": 0.060295}}, \"cds_start\": 529, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 529, \"protein_start\": 177, \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.883, \"loeuf\": 0.105, \"conservation\": 3.79, \"protein_end\": 177, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"mastermind_mmid3\": \"FBN1:C177S\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_end\": 845, \"cdna_start\": 845, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.9997, \"am_class\": \"likely_pathogenic\"}}]}',\n",
       "    'coordinates': [{'pca_x': 0.039319078718209416,\n",
       "      'pca_y': 0.07935653345698249},\n",
       "     {'t-sne_x': 15.966483050375476, 't-sne_y': 8.711989067710942},\n",
       "     {'umap_x': 3.2598979473114014, 'umap_y': 4.489316940307617}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48596325-A-T',\n",
       "      'distance': 0.00172464142087847,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.002583082765340805,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596291-C-T',\n",
       "      'distance': 0.0025973194278776646,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428439-A-T',\n",
       "      'distance': 0.0026499321684241295,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430791-A-T',\n",
       "      'distance': 0.002716377144679427,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.0028800717554986477,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596342-C-T',\n",
       "      'distance': 0.0030672517605125904,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596324-C-T',\n",
       "      'distance': 0.0031136763282120228,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432976-C-T',\n",
       "      'distance': 0.0031310557387769222,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.003142076078802347,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-T',\n",
       "      'distance': 0.003161383792757988,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.003271764609962702,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.003405881579965353,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537721-C-A',\n",
       "      'distance': 0.003444461850449443,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-T',\n",
       "      'distance': 0.0034475503489375114,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.0034650680609047413,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496177-C-T',\n",
       "      'distance': 0.0034861068706959486,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.0034888857044279575,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.0035308711230754852,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596373-A-G',\n",
       "      'distance': 0.003539085853844881,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48537698-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48537698\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48537698',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.649T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48537698A>C',\n",
       "     'hgvs_protein': 'p.Trp217Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"allele_string\": \"A/C\", \"vcf_string\": \"15-48537698-A-C\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48537698A>C\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48537698, \"end\": 48537698, \"phenotype_or_disease\": 1, \"id\": \"CM940757\"}, {\"pubmed\": [11826022, 8136837, 8791520, 11143906, 17568394, 7977366], \"id\": \"rs193922224\", \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"C:likely_pathogenic\", \"clin_sig\": [\"likely_pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"UniProt\": [\"VAR_002281\"], \"ClinVar\": [\"RCV000029763\", \"VCV000036101\"]}, \"allele_string\": \"A/C\", \"strand\": 1, \"start\": 48537698, \"end\": 48537698, \"frequencies\": {\"C\": {\"gnomade_mid\": 0, \"gnomade_afr\": 0, \"gnomade\": 6.84e-07, \"gnomade_nfe\": 8.993e-07, \"gnomade_fin\": 0, \"gnomade_sas\": 0, \"gnomade_eas\": 0, \"gnomade_asj\": 0, \"gnomade_amr\": 0, \"gnomade_remaining\": 0}}}], \"id\": \"NC_000015.10:g.48537698A>C\", \"minimised\": 1, \"transcript_consequences\": [{\"polyphen_score\": 1, \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 217, \"cds_end\": 649, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 217, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"conservation\": 3.79, \"loeuf\": 0.105, \"lrt_pred\": \"invalid_field\", \"cdna_start\": 965, \"cdna_end\": 965, \"mastermind_mmid3\": \"FBN1:W217G\", \"mane\": [\"MANE_Select\"], \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.9877, \"am_class\": \"likely_pathogenic\"}, \"gene_symbol_source\": \"EntrezGene\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"C\", \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": 43, \"SYMBOL\": \"FBN1\", \"DP_AG\": 11, \"DS_DL\": 0, \"DP_DG\": -12, \"DP_AL\": 43, \"DS_AG\": 0, \"DS_DG\": 0}, \"aa\": \"A\", \"exon\": \"7/66\", \"blosum62\": -2, \"hgvsc\": \"NM_000138.5:c.649T>G\", \"biotype\": \"protein_coding\", \"codons\": \"Tgg/Ggg\", \"canonical\": 1, \"mastermind_counts\": \"10|16|16\", \"cadd_phred\": 32, \"hgvsp\": \"NP_000129.3:p.Trp217Gly\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48537698, \"start\": 48537697}}, \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"given_ref\": \"A\", \"cadd_raw\": 5.474448, \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.999765694141388, \"location\": \"15:48537698\", \"mutationtaster_pred\": \"A\", \"strand\": -1, \"cds_start\": 649, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"amino_acids\": \"W/G\"}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48537698, \"start\": 48537698}',\n",
       "    'coordinates': [{'pca_x': 0.04344218518149756,\n",
       "      'pca_y': 0.08197650489722061},\n",
       "     {'t-sne_x': 19.904928957515978, 't-sne_y': 2.21390392035975},\n",
       "     {'umap_x': 3.6625237464904785, 'umap_y': 6.188014984130859}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48436967-A-C',\n",
       "      'distance': 0.002322881016880274,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520677-A-C',\n",
       "      'distance': 0.0029824115335941315,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.00361656048335135,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.0037840292789041996,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.003823649836704135,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.003991551697254181,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437873-A-G',\n",
       "      'distance': 0.004069914110004902,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420795-A-G',\n",
       "      'distance': 0.004117812030017376,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470679-A-G',\n",
       "      'distance': 0.0041373856365680695,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489977-C-T',\n",
       "      'distance': 0.004275287967175245,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.004287031479179859,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.004418375436216593,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437872-C-T',\n",
       "      'distance': 0.004588025156408548,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.004638229962438345,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474567-A-G',\n",
       "      'distance': 0.004642917774617672,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.004686690401285887,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.004798206500709057,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.004801744595170021,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.004837446380406618,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434707-T-C',\n",
       "      'distance': 0.0048657916486263275,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48534206-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48534206\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48534206',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.737-1G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48534206C>T',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48534206C>T\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"id\": \"CS020807\", \"phenotype_or_disease\": 1, \"end\": 48534206, \"start\": 48534206, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1}], \"minimised\": 1, \"most_severe_consequence\": \"splice_acceptor_variant\", \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"intron\": \"7/65\", \"mane\": [\"MANE_Select\"], \"rf_score\": 0.944, \"mastermind_mmid3\": \"FBN1:D246sa\", \"phaplo\": 0.99897615427687, \"conservation\": 3.55, \"loeuf\": 0.105, \"maxentscan_alt\": -0.007, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"genesplicer\": \"loss/acceptor/48534205-48534206/Medium/7.151667\", \"ada_score\": 0.999989125585301, \"cadd_raw\": 6.68419, \"given_ref\": \"C\", \"allele_num\": 1, \"consequence_terms\": [\"splice_acceptor_variant\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"start\": 48534205, \"end\": 48534206, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"location\": \"15:48534206\", \"maxentscan_diff\": 8.75, \"aa\": \"C\", \"lof\": \"HC\", \"spliceai\": {\"DP_AG\": -18, \"DS_DL\": 0, \"DP_DL\": -1, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0.99, \"DS_AG\": 0.53, \"DS_DG\": 0, \"DP_AL\": -1, \"DP_DG\": -18}, \"canonical\": 1, \"maxentscan_ref\": 8.743, \"cadd_phred\": 35, \"mastermind_counts\": \"1|1|1\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.737-1G>A\", \"gene_symbol_source\": \"EntrezGene\", \"lof_info\": \"INTRON_SIZE:3405\", \"variant_allele\": \"T\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\"}], \"end\": 48534206, \"start\": 48534206, \"vcf_string\": \"15-48534206-C-T\", \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48534206C>T\"}',\n",
       "    'coordinates': [{'pca_x': -0.23992400127346813,\n",
       "      'pca_y': -0.038323853648919715},\n",
       "     {'t-sne_x': -28.95909373129908, 't-sne_y': -18.64649262764384},\n",
       "     {'umap_x': 11.54886531829834, 'umap_y': 19.046472549438477}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.0008564737509004772,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.0013986666454002261,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.0014700288884341717,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.0014898263616487384,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.002011359203606844,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.002244440373033285,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.0023889034055173397,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.00259336456656456,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.0027041814755648375,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.002736019901931286,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.003050645114853978,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.003206202294677496,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.003393536899238825,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.0036105832550674677,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.003980761859565973,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.004059511702507734,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470757-C-T',\n",
       "      'distance': 0.0050664786249399185,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-A',\n",
       "      'distance': 0.005467658396810293,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446823-C-G',\n",
       "      'distance': 0.0055181533098220825,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452686-T-C',\n",
       "      'distance': 0.006206643767654896,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48526158-G-GT',\n",
       "    'metadata': {'vcf_string': '15\\t48526158\\t.\\tG\\tGT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48526158',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'GT',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.959dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48526159dup',\n",
       "     'hgvs_protein': 'p.Tyr320Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48526159dup\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"-/T\", \"vcf_string\": \"15-48526158-G-GT\", \"strand\": 1, \"transcript_consequences\": [{\"protein_end\": 320, \"phaplo\": 0.99897615427687, \"conservation\": 2.935, \"loeuf\": 0.105, \"used_ref\": \"-\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 320, \"cds_end\": 960, \"transcript_id\": \"NM_000138.5\", \"cdna_start\": 1275, \"cdna_end\": 1276, \"mastermind_mmid3\": \"FBN1:Y320X\", \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"spliceai\": {\"DP_DG\": -28, \"DP_AL\": -28, \"DS_DG\": 0, \"DS_AG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -13, \"DS_DL\": 0, \"DP_AG\": -11}, \"exon\": \"9/66\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.959dup\", \"codons\": \"tac/taAc\", \"canonical\": 1, \"mastermind_counts\": \"2|2|3\", \"gene_symbol_source\": \"EntrezGene\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.111304549675023,GERP_DIST:7439.82062975327,BP_DIST:7600,DIST_FROM_LAST_EXON:7211,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"T\", \"strand\": -1, \"cds_start\": 959, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"amino_acids\": \"Y/*\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48526158, \"end\": 48526158, \"type\": \"SimpleInterval\"}}}, \"hgvsp\": \"NP_000129.3:p.Tyr320Ter\", \"given_ref\": \"-\", \"consequence_terms\": [\"stop_gained\", \"frameshift_variant\"], \"allele_num\": 1, \"location\": \"15:48526158-48526159\"}], \"most_severe_consequence\": \"stop_gained\", \"end\": 48526158, \"start\": 48526159, \"colocated_variants\": [{\"end\": 48526159, \"start\": 48526159, \"allele_string\": \"T/TT\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000664037\", \"VCV000549473\"]}, \"seq_region_name\": \"15\", \"clin_sig\": [\"likely_pathogenic\"], \"phenotype_or_disease\": 1, \"id\": \"rs1555401005\"}], \"variant_class\": \"insertion\", \"id\": \"NC_000015.10:g.48526159dup\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.002835845024924677,\n",
       "      'pca_y': 0.05943157257026937},\n",
       "     {'t-sne_x': 25.578938676368114, 't-sne_y': 39.31665492901256},\n",
       "     {'umap_x': -9.85350513458252, 'umap_y': 3.015258312225342}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.07409802824258804,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.07413576543331146,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.07413609325885773,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.0744088739156723,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.07446092367172241,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.07507544755935669,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.07508139312267303,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.07539080083370209,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.07552289962768555,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.07562164962291718,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.07566189765930176,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.07576199620962143,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.07581369578838348,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.07599100470542908,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.07602386176586151,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508612-G-A',\n",
       "      'distance': 0.07612736523151398,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520764-G-A',\n",
       "      'distance': 0.07630890607833862,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.07633234560489655,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.07654910534620285,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465820-G-A',\n",
       "      'distance': 0.07668271660804749,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48526141-GT-G',\n",
       "    'metadata': {'vcf_string': '15\\t48526141\\t.\\tGT\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48526141',\n",
       "     'ref_allele': 'GT',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.976del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48526142del',\n",
       "     'hgvs_protein': 'p.Thr326ProfsTer4',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48526141-GT-G\", \"allele_string\": \"T/-\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48526142del\", \"id\": \"NC_000015.10:g.48526142del\", \"variant_class\": \"deletion\", \"minimised\": 1, \"end\": 48526142, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"cds_start\": 976, \"amino_acids\": \"T/X\", \"given_ref\": \"T\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"hgvsp\": \"NP_000129.3:p.Thr326ProfsTer4\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48526141, \"end\": 48526142}}}, \"location\": \"15:48526142\", \"aa\": \"T\", \"exon\": \"9/66\", \"lof\": \"HC\", \"spliceai\": {\"DP_AL\": 6, \"DP_DG\": -11, \"DS_DG\": 0, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 4, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": 47}, \"canonical\": 1, \"codons\": \"Acc/cc\", \"mastermind_counts\": \"1|1|1\", \"hgvsc\": \"NM_000138.5:c.976del\", \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"lof_info\": \"PERCENTILE:0.113277623026927,GERP_DIST:7426.10362903038,BP_DIST:7583,DIST_FROM_LAST_EXON:7194,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"transcript_id\": \"NM_000138.5\", \"cdna_end\": 1292, \"cdna_start\": 1292, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:T326fs\", \"phaplo\": 0.99897615427687, \"protein_end\": 326, \"loeuf\": 0.105, \"conservation\": 3.79, \"cds_end\": 976, \"protein_start\": 326, \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\"}], \"start\": 48526142}',\n",
       "    'coordinates': [{'pca_x': -0.010138137450701593,\n",
       "      'pca_y': 0.031664615240522546},\n",
       "     {'t-sne_x': 3.5806285404278517, 't-sne_y': 4.371958677394344},\n",
       "     {'umap_x': 3.8358099460601807, 'umap_y': 2.975564956665039}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.11084660142660141,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.11087767034769058,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.11152297258377075,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.1118992418050766,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.11197593808174133,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.11212849617004395,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48408466-T-C',\n",
       "      'distance': 0.11223199218511581,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48408950-A-G',\n",
       "      'distance': 0.11231778562068939,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.11258477717638016,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.11305780708789825,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.11328518390655518,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.11369213461875916,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48596360-C-T',\n",
       "      'distance': 0.1137026995420456,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.11372815072536469,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48409622-T-C',\n",
       "      'distance': 0.11377762258052826,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.11396713554859161,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.11397950351238251,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.11400122940540314,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.11408261209726334,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48446747-C-T',\n",
       "      'distance': 0.11411864310503006,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.6666666666666666},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7}}},\n",
       "   {'variant_id': '15-48515510-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48515510\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48515510',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1345G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48515510C>T',\n",
       "     'hgvs_protein': 'p.Val449Ile',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48515510, \"end\": 48515510, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"impact\": \"MODERATE\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"mutationtaster_pred\": \"N\", \"cds_start\": 1345, \"amino_acids\": \"V/I\", \"geno2mp_url\": \"https://geno2mp.gs.washington.edu/Geno2MP/#/variant/15/48807707/C%3ET/snp\", \"cadd_raw\": 1.129437, \"given_ref\": \"C\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48515510, \"start\": 48515509}, \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}}, \"hgvsp\": \"NP_000129.3:p.Val449Ile\", \"location\": \"15:48515510\", \"polyphen_prediction\": \"benign\", \"clinpred\": 0.0405718471463709, \"aa\": \"C\", \"exon\": \"12/66\", \"blosum62\": 3, \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -18, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 1, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_DG\": 7, \"DP_AL\": 5}, \"codons\": \"Gtt/Att\", \"canonical\": 1, \"mastermind_counts\": \"9|15|15\", \"cadd_phred\": 11.63, \"hgvsc\": \"NM_000138.5:c.1345G>A\", \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Benign\", \"variant_allele\": \"T\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"eve_score\": 0.123385318895426, \"alphamissense\": {\"am_class\": \"likely_benign\", \"am_pathogenicity\": 0.0612}, \"transcript_id\": \"NM_000138.5\", \"cdna_start\": 1661, \"cdna_end\": 1661, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:V449I\", \"phaplo\": 0.99897615427687, \"sift_score\": 1, \"protein_end\": 449, \"geno2mp_hpo_count\": 3, \"loeuf\": 0.105, \"conservation\": 1.81, \"polyphen_score\": 0, \"protein_start\": 449, \"cds_end\": 1345, \"sift_prediction\": \"tolerated_low_confidence\", \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\"}], \"minimised\": 1, \"colocated_variants\": [{\"id\": \"CM022783\", \"phenotype_or_disease\": 1, \"start\": 48515510, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"end\": 48515510}, {\"id\": \"rs139058991\", \"phenotype_or_disease\": 1, \"pubmed\": [24033266, 25637381, 24941995], \"clin_sig_allele\": \"T:benign;T:conflicting_classifications_of_pathogenicity;T:uncertain_significance\", \"allele_string\": \"C/T\", \"strand\": 1, \"clin_sig\": [\"uncertain_significance\", \"benign\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000455446\", \"RCV000148499\", \"VCV000161243\", \"RCV001071912\", \"RCV000771785\", \"RCV003328558\"], \"UniProt\": [\"VAR_076007\"]}, \"start\": 48515510, \"end\": 48515510, \"frequencies\": {\"T\": {\"gnomade_afr\": 0, \"gnomadg\": 9.199e-05, \"gnomade_asj\": 0, \"gnomade_eas\": 0, \"gnomade_amr\": 0, \"gnomadg_afr\": 4.826e-05, \"gnomade\": 4.242e-05, \"gnomade_nfe\": 5.396e-05, \"gnomadg_eas\": 0, \"gnomadg_mid\": 0, \"gnomadg_fin\": 0, \"gnomade_mid\": 0.0001734, \"gnomadg_sas\": 0, \"gnomade_fin\": 1.873e-05, \"gnomade_sas\": 0, \"gnomadg_remaining\": 0, \"gnomadg_asj\": 0, \"gnomade_remaining\": 0, \"gnomadg_nfe\": 0.0001617, \"gnomadg_ami\": 0, \"gnomadg_amr\": 6.541e-05}}}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48515510C>T\", \"input\": \"NC_000015.10:g.48515510C>T\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48515510-C-T\"}',\n",
       "    'coordinates': [{'pca_x': 0.024275273013100285,\n",
       "      'pca_y': 0.11105495078759506},\n",
       "     {'t-sne_x': -12.628951797883305, 't-sne_y': 33.662185944222585},\n",
       "     {'umap_x': 0.8006159663200378, 'umap_y': 21.636972427368164}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48472601-C-T',\n",
       "      'distance': 0.0036100507713854313,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505135-C-T',\n",
       "      'distance': 0.003666323609650135,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472556-C-T',\n",
       "      'distance': 0.00385805731639266,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513661-A-G',\n",
       "      'distance': 0.003906966187059879,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48495156-C-T',\n",
       "      'distance': 0.003916341811418533,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.003916924819350243,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.004078526981174946,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513554-C-T',\n",
       "      'distance': 0.00408572144806385,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508632-C-T',\n",
       "      'distance': 0.00409665796905756,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-C',\n",
       "      'distance': 0.0042260740883648396,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.004271331708878279,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515440-T-T',\n",
       "      'distance': 0.004306099843233824,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.00432671420276165,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425458-C-T',\n",
       "      'distance': 0.0043291328474879265,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463949-C-T',\n",
       "      'distance': 0.004404909443110228,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492538-C-T',\n",
       "      'distance': 0.004431356675922871,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472628-C-T',\n",
       "      'distance': 0.004445783793926239,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510049-C-T',\n",
       "      'distance': 0.004499444738030434,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.004523470997810364,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.004533971659839153,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48515433-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48515433\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48515433',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1422T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48515433A>C',\n",
       "     'hgvs_protein': 'p.Cys474Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48515433A>C\", \"vcf_string\": \"15-48515433-A-C\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/C\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"cadd_phred\": 24.2, \"mastermind_counts\": \"8|9|9\", \"canonical\": 1, \"codons\": \"tgT/tgG\", \"hgvsc\": \"NM_000138.5:c.1422T>G\", \"biotype\": \"protein_coding\", \"blosum62\": -2, \"exon\": \"12/66\", \"aa\": \"A\", \"spliceai\": {\"DP_AG\": 9, \"DS_DL\": 0.07, \"DP_DL\": 2, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_AG\": 0, \"DS_DG\": 0, \"DP_AL\": 2, \"DP_DG\": -40}, \"variant_allele\": \"C\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"C/W\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"cds_start\": 1422, \"strand\": -1, \"mutationtaster_pred\": \"A\", \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48515433\", \"clinpred\": 0.999908208847046, \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"cadd_raw\": 4.052748, \"given_ref\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys474Trp\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"end\": 48515433, \"start\": 48515432, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"type\": \"Allele\"}, \"loeuf\": 0.105, \"conservation\": -1.57, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 474, \"sift_prediction\": \"deleterious_low_confidence\", \"cds_end\": 1422, \"protein_start\": 474, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"A\", \"polyphen_score\": 0.987, \"alphamissense\": {\"am_pathogenicity\": 0.9903, \"am_class\": \"likely_pathogenic\"}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.691722119662613, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C474W\", \"cdna_start\": 1738, \"cdna_end\": 1738, \"lrt_pred\": \"invalid_field\"}], \"end\": 48515433, \"start\": 48515433, \"id\": \"NC_000015.10:g.48515433A>C\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48515433, \"end\": 48515433, \"id\": \"CM012598\", \"phenotype_or_disease\": 1}, {\"id\": \"rs1555400378\", \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"C:likely_pathogenic\", \"allele_string\": \"A/C\", \"strand\": 1, \"clin_sig\": [\"likely_pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000612430\", \"VCV000517124\"], \"UniProt\": [\"VAR_076008\"]}, \"start\": 48515433, \"end\": 48515433}], \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.03014230721219671,\n",
       "      'pca_y': 0.1137738000968036},\n",
       "     {'t-sne_x': -17.5923843034391, 't-sne_y': 30.263357714200488},\n",
       "     {'umap_x': 2.053609609603882, 'umap_y': 21.4997615814209}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48505136-A-C',\n",
       "      'distance': 0.001972380792722106,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508633-A-C',\n",
       "      'distance': 0.002160845324397087,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463950-A-C',\n",
       "      'distance': 0.0031697838567197323,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474345-A-C',\n",
       "      'distance': 0.003389716148376465,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515429-A-G',\n",
       "      'distance': 0.0035980248358100653,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495512-A-C',\n",
       "      'distance': 0.0037626191042363644,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515434-C-A',\n",
       "      'distance': 0.003946828190237284,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441764-A-C',\n",
       "      'distance': 0.004195146728307009,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508660-A-C',\n",
       "      'distance': 0.004279834683984518,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515434-C-T',\n",
       "      'distance': 0.0049262745305895805,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505135-C-T',\n",
       "      'distance': 0.005013694521039724,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463949-C-T',\n",
       "      'distance': 0.0051360479556024075,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463949-C-A',\n",
       "      'distance': 0.005168462637811899,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452637-A-G',\n",
       "      'distance': 0.005216177552938461,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505136-A-G',\n",
       "      'distance': 0.00526300398632884,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472601-C-T',\n",
       "      'distance': 0.005403532646596432,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425413-C-A',\n",
       "      'distance': 0.005599538329988718,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425401-T-C',\n",
       "      'distance': 0.005605184473097324,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415566-C-T',\n",
       "      'distance': 0.005616405978798866,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515440-T-T',\n",
       "      'distance': 0.005674594081938267,\n",
       "      'pathogenicity': 'benign'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48515423-T-A',\n",
       "    'metadata': {'vcf_string': '15\\t48515423\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48515423',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1432A>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48515423T>A',\n",
       "     'hgvs_protein': 'p.Lys478Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48515423T>A\", \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48515423, \"end\": 48515423, \"phenotype_or_disease\": 1, \"id\": \"CM098606\"}], \"variant_class\": \"SNV\", \"minimised\": 1, \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"mutationtaster_pred\": \"A\", \"cds_start\": 1432, \"amino_acids\": \"K/*\", \"given_ref\": \"T\", \"cadd_raw\": 8.551835, \"allele_num\": 1, \"consequence_terms\": [\"stop_gained\"], \"hgvsp\": \"NP_000129.3:p.Lys478Ter\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48515423, \"start\": 48515422, \"type\": \"SimpleInterval\"}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"type\": \"Allele\"}, \"location\": \"15:48515423\", \"aa\": \"T\", \"exon\": \"12/66\", \"lof\": \"HC\", \"spliceai\": {\"DP_DL\": -36, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AG\": 12, \"DS_DL\": 0.01, \"DP_AL\": 23, \"DP_DG\": 12, \"DS_AG\": 0, \"DS_DG\": 0.15}, \"codons\": \"Aaa/Taa\", \"canonical\": 1, \"mastermind_counts\": \"2|2|2\", \"cadd_phred\": 38, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.1432A>T\", \"gene_symbol_source\": \"EntrezGene\", \"lof_info\": \"PERCENTILE:0.166202414113278,GERP_DIST:7078.01753313104,BP_DIST:7130,DIST_FROM_LAST_EXON:6741,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"A\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"transcript_id\": \"NM_000138.5\", \"cdna_start\": 1748, \"cdna_end\": 1748, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:K478X\", \"phaplo\": 0.99897615427687, \"protein_end\": 478, \"conservation\": 2.76, \"loeuf\": 0.105, \"protein_start\": 478, \"cds_end\": 1432, \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\"}], \"most_severe_consequence\": \"stop_gained\", \"end\": 48515423, \"start\": 48515423, \"vcf_string\": \"15-48515423-T-A\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"T/A\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48515423T>A\"}',\n",
       "    'coordinates': [{'pca_x': 0.01855140664082941,\n",
       "      'pca_y': 0.09582299207305875},\n",
       "     {'t-sne_x': 25.50815380712363, 't-sne_y': 37.8672095123438},\n",
       "     {'umap_x': -10.007628440856934, 'umap_y': 2.8959288597106934}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.0025945082306861877,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.003088063094764948,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537615-A-T',\n",
       "      'distance': 0.004045192152261734,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48467954-A-T',\n",
       "      'distance': 0.0040548196993768215,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644643-T-A',\n",
       "      'distance': 0.004187296144664288,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.004716617055237293,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.00511980801820755,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513553-G-T',\n",
       "      'distance': 0.005322277080267668,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.005364920012652874,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.005502401851117611,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.005540383048355579,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415540-G-A',\n",
       "      'distance': 0.005692741367965937,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.005820498336106539,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.0058770691975951195,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.005914255511015654,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505139-C-A',\n",
       "      'distance': 0.005938221700489521,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415610-G-T',\n",
       "      'distance': 0.00598792964592576,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.006017365492880344,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48534170-G-A',\n",
       "      'distance': 0.006050892174243927,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421601-G-T',\n",
       "      'distance': 0.006159912329167128,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48513667-AT-A',\n",
       "    'metadata': {'vcf_string': '15\\t48513667\\t.\\tAT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48513667',\n",
       "     'ref_allele': 'AT',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1469del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48513668del',\n",
       "     'hgvs_protein': 'p.Asp490ValfsTer89',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"deletion\", \"colocated_variants\": [{\"end\": 48513668, \"start\": 48513668, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"id\": \"CD040175\", \"phenotype_or_disease\": 1}, {\"id\": \"COSV57315360\", \"phenotype_or_disease\": 1, \"somatic\": 1, \"seq_region_name\": \"15\", \"var_synonyms\": {\"COSMIC\": [\"COSM3729091\"]}, \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"start\": 48513668, \"end\": 48513668}], \"id\": \"NC_000015.10:g.48513668del\", \"minimised\": 1, \"transcript_consequences\": [{\"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48513667, \"end\": 48513668, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Asp490ValfsTer89\", \"consequence_terms\": [\"frameshift_variant\", \"splice_region_variant\"], \"allele_num\": 1, \"given_ref\": \"T\", \"location\": \"15:48513668\", \"cds_start\": 1469, \"strand\": -1, \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"amino_acids\": \"D/X\", \"gene_symbol_source\": \"EntrezGene\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.170496750232126,GERP_DIST:7022.03453291217,BP_DIST:7094,DIST_FROM_LAST_EXON:6705,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"spliceai\": {\"DP_DG\": 46, \"DP_AL\": 1, \"DS_AG\": 1, \"DS_DG\": 0, \"DS_AL\": 1, \"DP_DL\": -34, \"SYMBOL\": \"FBN1\", \"DP_AG\": 0, \"DS_DL\": 0}, \"lof\": \"HC\", \"exon\": \"13/66\", \"aa\": \"T\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.1469del\", \"mastermind_counts\": \"2|16|18\", \"canonical\": 1, \"codons\": \"gAt/gt\", \"cdna_start\": 1785, \"cdna_end\": 1785, \"mastermind_mmid3\": \"FBN1:D490fs,FBN1:D490sra\", \"mane\": [\"MANE_Select\"], \"transcript_id\": \"NM_000138.5\", \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"T\", \"cds_end\": 1469, \"protein_start\": 490, \"protein_end\": 490, \"phaplo\": 0.99897615427687, \"conservation\": 3.7, \"loeuf\": 0.105}], \"most_severe_consequence\": \"frameshift_variant\", \"end\": 48513668, \"start\": 48513668, \"allele_string\": \"T/-\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48513667-AT-A\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48513668del\"}',\n",
       "    'coordinates': [{'pca_x': -0.03066447131604179,\n",
       "      'pca_y': 0.006933387700962884},\n",
       "     {'t-sne_x': -12.908309562116703, 't-sne_y': -2.8271558064915925},\n",
       "     {'umap_x': 0.3810303211212158, 'umap_y': 15.098091125488281}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.08021894097328186,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.08067099004983902,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.08117285370826721,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.08182523399591446,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.08184594660997391,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.08274923264980316,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.08277181535959244,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.0829739049077034,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.08332918584346771,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.08366456627845764,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.08420507609844208,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.08421352505683899,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.08426812291145325,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.08435723185539246,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.08472782373428345,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.08528454601764679,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.08555584400892258,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.08690820634365082,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.0874403566122055,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610825-G-T',\n",
       "      'distance': 0.08754149079322815,\n",
       "      'pathogenicity': 'likely_benign'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48513592-G-C',\n",
       "    'metadata': {'vcf_string': '15\\t48513592\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48513592',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1545C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48513592G>C',\n",
       "     'hgvs_protein': 'p.Cys515Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"vcf_string\": \"15-48513592-G-C\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"G/C\", \"input\": \"NC_000015.10:g.48513592G>C\", \"seq_region_name\": \"15\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48513592G>C\", \"variant_class\": \"SNV\", \"start\": 48513592, \"most_severe_consequence\": \"missense_variant\", \"end\": 48513592, \"transcript_consequences\": [{\"variant_allele\": \"C\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"codons\": \"tgC/tgG\", \"canonical\": 1, \"mastermind_counts\": \"1|1|1\", \"cadd_phred\": 27.6, \"hgvsc\": \"NM_000138.5:c.1545C>G\", \"biotype\": \"protein_coding\", \"aa\": \"G\", \"exon\": \"13/66\", \"blosum62\": -2, \"spliceai\": {\"DS_DG\": 0.01, \"DS_AG\": 0, \"DP_DG\": 41, \"DP_AL\": 2, \"DS_DL\": 0, \"DP_AG\": 20, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -43}, \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.999904274940491, \"location\": \"15:48513592\", \"cadd_raw\": 4.95396, \"given_ref\": \"G\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48513592, \"start\": 48513591, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Cys515Trp\", \"amino_acids\": \"C/W\", \"impact\": \"MODERATE\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"D\", \"strand\": -1, \"cds_start\": 1545, \"protein_start\": 515, \"cds_end\": 1545, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.997, \"loeuf\": 0.105, \"conservation\": 1.95, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"protein_end\": 515, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C515W\", \"cdna_end\": 1861, \"cdna_start\": 1861, \"lrt_pred\": \"invalid_field\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9964}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.71022984168313}]}',\n",
       "    'coordinates': [{'pca_x': 0.04569911686899519,\n",
       "      'pca_y': 0.0887629865900593},\n",
       "     {'t-sne_x': 19.0816411385764, 't-sne_y': -7.846182382199166},\n",
       "     {'umap_x': 8.307821273803711, 'umap_y': 7.361911296844482}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468455-G-C',\n",
       "      'distance': 0.002927040448412299,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.0034490288235247135,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.003571538021788001,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.003607934806495905,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.003853593487292528,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492562-G-C',\n",
       "      'distance': 0.004541341215372086,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.004585138987749815,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.004708900116384029,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48513599-C-G',\n",
       "      'distance': 0.004722547717392445,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437784-G-C',\n",
       "      'distance': 0.00479836855083704,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495162-C-G',\n",
       "      'distance': 0.004834512248635292,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.004938979633152485,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508673-G-C',\n",
       "      'distance': 0.004947247449308634,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487333-G-C',\n",
       "      'distance': 0.004950850736349821,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.005009612534195185,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.005077039822936058,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437347-G-C',\n",
       "      'distance': 0.005119554232805967,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.005272414069622755,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505081-T-C',\n",
       "      'distance': 0.005310130305588245,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428356-G-C',\n",
       "      'distance': 0.005375667009502649,\n",
       "      'pathogenicity': 'benign_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48513591-G-C',\n",
       "    'metadata': {'vcf_string': '15\\t48513591\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48513591',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1546C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48513591G>C',\n",
       "     'hgvs_protein': 'p.Arg516Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"end\": 48513591, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48513591, \"id\": \"CM054693\", \"phenotype_or_disease\": 1}, {\"id\": \"CX102327\", \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48513591, \"end\": 48513591}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48513591G>C\", \"minimised\": 1, \"transcript_consequences\": [{\"codons\": \"Cga/Gga\", \"canonical\": 1, \"cadd_phred\": 22.6, \"mastermind_counts\": \"2|2|2\", \"hgvsc\": \"NM_000138.5:c.1546C>G\", \"biotype\": \"protein_coding\", \"aa\": \"G\", \"exon\": \"13/66\", \"blosum62\": -2, \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_DG\": -47, \"DP_AL\": -34, \"DS_DL\": 0, \"DP_AG\": 21, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 42}, \"variant_allele\": \"C\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Benign\", \"amino_acids\": \"R/G\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"strand\": -1, \"mutationtaster_pred\": \"N\", \"cds_start\": 1546, \"polyphen_prediction\": \"benign\", \"location\": \"15:48513591\", \"clinpred\": 0.8266242146492, \"cadd_raw\": 3.465534, \"given_ref\": \"G\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48513591, \"start\": 48513590, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Arg516Gly\", \"conservation\": 3.7, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"sift_score\": 0.05, \"protein_end\": 516, \"protein_start\": 516, \"cds_end\": 1546, \"sift_prediction\": \"tolerated_low_confidence\", \"used_ref\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.399, \"alphamissense\": {\"am_class\": \"likely_benign\", \"am_pathogenicity\": 0.1357}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.161694925352466, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:R516G\", \"cdna_start\": 1862, \"cdna_end\": 1862, \"lrt_pred\": \"invalid_field\"}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48513591, \"start\": 48513591, \"allele_string\": \"G/C\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48513591-G-C\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48513591G>C\"}',\n",
       "    'coordinates': [{'pca_x': 0.03975196548335025,\n",
       "      'pca_y': 0.0991225106690393},\n",
       "     {'t-sne_x': -15.32507139392668, 't-sne_y': 28.24755287221748},\n",
       "     {'umap_x': 1.7560102939605713, 'umap_y': 22.004093170166016}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48503845-G-C',\n",
       "      'distance': 0.005892328917980194,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515428-C-T',\n",
       "      'distance': 0.006218786351382732,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425412-G-C',\n",
       "      'distance': 0.007056259550154209,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505102-C-G',\n",
       "      'distance': 0.007292422465980053,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515429-A-G',\n",
       "      'distance': 0.008306456729769707,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445454-A-G',\n",
       "      'distance': 0.0090055912733078,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481729-T-C',\n",
       "      'distance': 0.009116079658269882,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48513600-A-G',\n",
       "      'distance': 0.009418830275535583,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.009445784613490105,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456643-A-G',\n",
       "      'distance': 0.009675145149230957,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508665-C-T',\n",
       "      'distance': 0.009734019637107849,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48463949-C-G',\n",
       "      'distance': 0.009831677190959454,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421652-G-C',\n",
       "      'distance': 0.009876780211925507,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505101-G-C',\n",
       "      'distance': 0.009947814047336578,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510128-C-T',\n",
       "      'distance': 0.00995081476867199,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508653-T-C',\n",
       "      'distance': 0.010053597390651703,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441764-A-C',\n",
       "      'distance': 0.010076640173792839,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513627-A-G',\n",
       "      'distance': 0.010322443209588528,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513631-A-G',\n",
       "      'distance': 0.010322443209588528,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48483823-C-G',\n",
       "      'distance': 0.010363951325416565,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48508644-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48508644\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48508644',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1775G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48508644C>T',\n",
       "     'hgvs_protein': 'p.Gly592Asp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"vcf_string\": \"15-48508644-C-T\", \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48508644C>T\", \"seq_region_name\": \"15\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48508644C>T\", \"colocated_variants\": [{\"end\": 48508644, \"start\": 48508644, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"id\": \"CM013919\", \"phenotype_or_disease\": 1}, {\"end\": 48508644, \"start\": 48508644, \"seq_region_name\": \"15\", \"clin_sig\": [\"likely_pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV000663491\", \"VCV000549043\"], \"UniProt\": [\"VAR_017976\"]}, \"strand\": 1, \"allele_string\": \"C/T\", \"clin_sig_allele\": \"T:likely_pathogenic\", \"phenotype_or_disease\": 1, \"id\": \"rs1555399959\"}], \"variant_class\": \"SNV\", \"start\": 48508644, \"transcript_consequences\": [{\"eve_score\": 0.7787230800494, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9981}, \"transcript_id\": \"NM_000138.5\", \"cdna_end\": 2091, \"cdna_start\": 2091, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:G592D\", \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"protein_end\": 592, \"conservation\": 3.95, \"loeuf\": 0.105, \"polyphen_score\": 0.995, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 592, \"cds_end\": 1775, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"impact\": \"MODERATE\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"cds_start\": 1775, \"mutationtaster_pred\": \"A\", \"strand\": -1, \"amino_acids\": \"G/D\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"given_ref\": \"C\", \"cadd_raw\": 5.241085, \"hgvsp\": \"NP_000129.3:p.Gly592Asp\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48508643, \"end\": 48508644}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.993719816207886, \"location\": \"15:48508644\", \"blosum62\": -1, \"exon\": \"15/66\", \"aa\": \"C\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -35, \"SYMBOL\": \"FBN1\", \"DP_DL\": -50, \"DS_AL\": 0, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": 3, \"DP_DG\": 16}, \"cadd_phred\": 29.4, \"mastermind_counts\": \"4|6|6\", \"codons\": \"gGc/gAc\", \"canonical\": 1, \"hgvsc\": \"NM_000138.5:c.1775G>A\", \"biotype\": \"protein_coding\", \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"T\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\"}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48508644}',\n",
       "    'coordinates': [{'pca_x': 0.030754465718210833,\n",
       "      'pca_y': 0.1110048013994411},\n",
       "     {'t-sne_x': -11.480699088000094, 't-sne_y': 28.825055436523492},\n",
       "     {'umap_x': 1.385931134223938, 'umap_y': 22.15540313720703}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48483935-C-T',\n",
       "      'distance': 0.003105593379586935,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495519-C-G',\n",
       "      'distance': 0.004140000324696302,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508660-A-C',\n",
       "      'distance': 0.00442871917039156,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481690-C-T',\n",
       "      'distance': 0.0046141790226101875,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508665-C-T',\n",
       "      'distance': 0.004642331041395664,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48513656-C-T',\n",
       "      'distance': 0.004782575648277998,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437038-C-T',\n",
       "      'distance': 0.004944185260683298,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415566-C-T',\n",
       "      'distance': 0.005098563618957996,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513643-G-A',\n",
       "      'distance': 0.005155701190233231,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48472601-C-T',\n",
       "      'distance': 0.005171525292098522,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510121-C-T',\n",
       "      'distance': 0.00542322127148509,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.00544527405872941,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495156-C-T',\n",
       "      'distance': 0.005445510149002075,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.005487811751663685,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.005700381472706795,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48441765-C-T',\n",
       "      'distance': 0.005725689232349396,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.005754433106631041,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508692-C-T',\n",
       "      'distance': 0.00576560664921999,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.0057914177887141705,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495131-C-T',\n",
       "      'distance': 0.005806395784020424,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48505139-CGTT-C',\n",
       "    'metadata': {'vcf_string': '15\\t48505139\\t.\\tCGTT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48505139',\n",
       "     'ref_allele': 'CGTT',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1843_1845del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48505140_48505142del',\n",
       "     'hgvs_protein': 'p.Asn615del',\n",
       "     'most_severe_consequence': 'inframe_deletion'},\n",
       "    'annotation_raw': '{\"start\": 48505140, \"most_severe_consequence\": \"inframe_deletion\", \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"cds_start\": 1843, \"strand\": -1, \"amino_acids\": \"N/-\", \"allele_num\": 1, \"consequence_terms\": [\"inframe_deletion\"], \"given_ref\": \"GTT\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48505139, \"end\": 48505142, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Asn615del\", \"location\": \"15:48505140-48505142\", \"exon\": \"16/66\", \"aa\": \"GTT\", \"spliceai\": {\"DP_AG\": 8, \"DS_DL\": 0, \"DS_AL\": 0, \"DP_DL\": 26, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DG\": 0, \"DP_DG\": 8, \"DP_AL\": 26}, \"mastermind_counts\": \"5|5|6\", \"canonical\": 1, \"codons\": \"AAC/-\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.1843_1845del\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"-\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"transcript_id\": \"NM_000138.5\", \"cdna_start\": 2159, \"cdna_end\": 2161, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:N615del\", \"phaplo\": 0.99897615427687, \"protein_end\": 615, \"loeuf\": 0.105, \"conservation\": 1.06, \"protein_start\": 615, \"cds_end\": 1845, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"GTT\"}], \"end\": 48505142, \"minimised\": 1, \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"seq_region_name\": \"15\", \"var_synonyms\": {\"COSMIC\": [\"COSM9471203\"]}, \"start\": 48505140, \"end\": 48505142, \"phenotype_or_disease\": 1, \"id\": \"COSV104608689\", \"somatic\": 1}, {\"end\": 48505142, \"start\": 48505140, \"allele_string\": \"GTT/-\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000663502\", \"VCV000549053\"]}, \"seq_region_name\": \"15\", \"clin_sig\": [\"likely_pathogenic\"], \"phenotype_or_disease\": 1, \"id\": \"rs1555399837\"}], \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48505140_48505142del\", \"input\": \"NC_000015.10:g.48505140_48505142del\", \"seq_region_name\": \"15\", \"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"GTT/-\", \"vcf_string\": \"15-48505139-CGTT-C\"}',\n",
       "    'coordinates': [{'pca_x': 0.02139628117240966,\n",
       "      'pca_y': 0.06371660898097145},\n",
       "     {'t-sne_x': 32.47331153403021, 't-sne_y': 11.69968754949686},\n",
       "     {'umap_x': 1.9620742797851562, 'umap_y': 7.976778984069824}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48613073-G-A',\n",
       "      'distance': 0.0911422073841095,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.09188170731067657,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.09221968054771423,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452589-G-A',\n",
       "      'distance': 0.0929386094212532,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.09295842051506042,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430710-G-A',\n",
       "      'distance': 0.09308014810085297,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.0932718962430954,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465825-C-T',\n",
       "      'distance': 0.0934276133775711,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09350953251123428,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.09360937774181366,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.09384991228580475,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411243-G-A',\n",
       "      'distance': 0.09394150972366333,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48437009-G-A',\n",
       "      'distance': 0.09404340386390686,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430676-C-T',\n",
       "      'distance': 0.09421883523464203,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481723-G-A',\n",
       "      'distance': 0.09439325332641602,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48490013-G-A',\n",
       "      'distance': 0.09459953010082245,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422024-A-G',\n",
       "      'distance': 0.09497058391571045,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503850-A-G',\n",
       "      'distance': 0.09503313153982162,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.09508734941482544,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48434701-C-T',\n",
       "      'distance': 0.09513111412525177,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8}}},\n",
       "   {'variant_id': '15-48505066-48505082-del-48cee38c',\n",
       "    'metadata': {'vcf_string': '15\\t48505066\\t.\\tAAGCATTCACATCTGTA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48505066',\n",
       "     'ref_allele': 'AAGCATTCACATCTGTA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1904_1919del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48505067_48505082del',\n",
       "     'hgvs_protein': 'p.Tyr635SerfsTer78',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"vcf_string\": \"15-48505066-AAGCATTCACATCTGTA-A\", \"allele_string\": \"AGCATTCACATCTGTA/-\", \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48505067_48505082del\", \"seq_region_name\": \"15\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48505067_48505082del\", \"variant_class\": \"deletion\", \"start\": 48505067, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"protein_start\": 635, \"cds_end\": 1919, \"used_ref\": \"AAGCATTCACATCTGT\", \"mane_select\": \"ENST00000316623.10\", \"phaplo\": 0.99897615427687, \"protein_end\": 640, \"conservation\": 1.717, \"loeuf\": 0.105, \"cdna_start\": 2220, \"cdna_end\": 2235, \"mane\": [\"MANE_Select\"], \"transcript_id\": \"NM_000138.5\", \"gene_symbol_source\": \"EntrezGene\", \"lof_info\": \"PERCENTILE:0.222725162488394,GERP_DIST:6702.96444040435,BP_DIST:6648,DIST_FROM_LAST_EXON:6259,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"aa\": \"AGCATTCACATCTGTA\", \"exon\": \"16/66\", \"lof\": \"HC\", \"canonical\": 1, \"codons\": \"tACAGATGTGAATGCTTc/tc\", \"hgvsc\": \"NM_000138.5:c.1904_1919del\", \"biotype\": \"protein_coding\", \"given_ref\": \"AGCATTCACATCTGTA\", \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"hgvsp\": \"NP_000129.3:p.Tyr635SerfsTer78\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48505082, \"start\": 48505066}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"hgvs_offset\": -1, \"location\": \"15:48505066-48505081\", \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"genesplicer\": \"loss/donor/48505073-48505074/Medium/1.447211\", \"cds_start\": 1904, \"amino_acids\": \"YRCECF/X\"}], \"end\": 48505082}',\n",
       "    'coordinates': [{'pca_x': 0.033313264327743175,\n",
       "      'pca_y': 0.04191051223747446},\n",
       "     {'t-sne_x': 33.08112367784699, 't-sne_y': 6.975532741101421},\n",
       "     {'umap_x': 1.699413537979126, 'umap_y': 6.994204521179199}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.0987013503909111,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.09875712543725967,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420752-A-G',\n",
       "      'distance': 0.09893631190061569,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446768-A-G',\n",
       "      'distance': 0.09956123679876328,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-G',\n",
       "      'distance': 0.09981343895196915,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.10031120479106903,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.1005924642086029,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.10062848031520844,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445382-A-G',\n",
       "      'distance': 0.10063066333532333,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503850-A-G',\n",
       "      'distance': 0.10065726935863495,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.10069960355758667,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.1009184792637825,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.10097062587738037,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.10099932551383972,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.10148708522319794,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.10150070488452911,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.10160049796104431,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.1016300618648529,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48488433-A-G',\n",
       "      'distance': 0.10163045674562454,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446811-A-G',\n",
       "      'distance': 0.10163125395774841,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48505030-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48505030\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48505030',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1955G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48505030C>T',\n",
       "     'hgvs_protein': 'p.Cys652Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48505030C>T\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM023916\", \"start\": 48505030, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48505030}, {\"phenotype_or_disease\": 1, \"id\": \"CM040034\", \"end\": 48505030, \"start\": 48505030, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\"}, {\"phenotype_or_disease\": 1, \"id\": \"CM147404\", \"end\": 48505030, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48505030}, {\"id\": \"COSV100354827\", \"phenotype_or_disease\": 1, \"somatic\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM8786099\"]}, \"seq_region_name\": \"15\", \"start\": 48505030, \"end\": 48505030}, {\"clin_sig_allele\": \"T:pathogenic\", \"pubmed\": [12446365, 17701892], \"id\": \"rs2043696418\", \"phenotype_or_disease\": 1, \"end\": 48505030, \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV001063817\", \"VCV000858022\"], \"UniProt\": [\"VAR_076018\"]}, \"allele_string\": \"C/T\", \"strand\": 1, \"start\": 48505030}], \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"phaplo\": 0.99897615427687, \"sift_score\": 0, \"protein_end\": 652, \"loeuf\": 0.105, \"conservation\": 3.95, \"polyphen_score\": 0.952, \"protein_start\": 652, \"cds_end\": 1955, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"eve_score\": 0.784001538331679, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9993}, \"transcript_id\": \"NM_000138.5\", \"cdna_end\": 2271, \"cdna_start\": 2271, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C652Y\", \"aa\": \"C\", \"blosum62\": -2, \"exon\": \"16/66\", \"spliceai\": {\"DP_DL\": -1, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AG\": 47, \"DS_DL\": 0.01, \"DP_AL\": -5, \"DP_DG\": 3, \"DS_AG\": 0, \"DS_DG\": 0.1}, \"codons\": \"tGt/tAt\", \"canonical\": 1, \"mastermind_counts\": \"6|8|8\", \"cadd_phred\": 32, \"hgvsc\": \"NM_000138.5:c.1955G>A\", \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"variant_allele\": \"T\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"impact\": \"MODERATE\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"A\", \"strand\": -1, \"cds_start\": 1955, \"amino_acids\": \"C/Y\", \"cadd_raw\": 5.514354, \"given_ref\": \"C\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48505029, \"end\": 48505030, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"hgvsp\": \"NP_000129.3:p.Cys652Tyr\", \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48505030\", \"clinpred\": 0.999886512756348}], \"end\": 48505030, \"start\": 48505030, \"vcf_string\": \"15-48505030-C-T\", \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48505030C>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.0331664959621453,\n",
       "      'pca_y': 0.10940434031487081},\n",
       "     {'t-sne_x': -12.556184321121567, 't-sne_y': 35.0117640050679},\n",
       "     {'umap_x': 0.9967969059944153, 'umap_y': 21.61537742614746}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48505135-C-T',\n",
       "      'distance': 0.002382294274866581,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510121-C-T',\n",
       "      'distance': 0.0037465051282197237,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48513554-C-T',\n",
       "      'distance': 0.0038887052796781063,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48510136-C-T',\n",
       "      'distance': 0.0040160794742405415,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510094-C-T',\n",
       "      'distance': 0.0040384698659181595,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472601-C-T',\n",
       "      'distance': 0.0043367501348257065,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510088-C-T',\n",
       "      'distance': 0.0043739210814237595,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515429-A-C',\n",
       "      'distance': 0.004394275136291981,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515440-T-T',\n",
       "      'distance': 0.004482226446270943,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48445453-C-T',\n",
       "      'distance': 0.004725442733615637,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508632-C-T',\n",
       "      'distance': 0.004826385527849197,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.0049496004357934,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510157-C-T',\n",
       "      'distance': 0.0050071836449205875,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463949-C-T',\n",
       "      'distance': 0.0050424267537891865,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513661-A-G',\n",
       "      'distance': 0.005070788785815239,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48472628-C-T',\n",
       "      'distance': 0.005196230486035347,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452654-C-T',\n",
       "      'distance': 0.00524442782625556,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.005285617429763079,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48444627-C-T',\n",
       "      'distance': 0.005371151026338339,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.005383343435823917,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48503877-ACAAA-A',\n",
       "    'metadata': {'vcf_string': '15\\t48503877\\t.\\tACAAA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48503877',\n",
       "     'ref_allele': 'ACAAA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2023_2026del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48503878_48503881del',\n",
       "     'hgvs_protein': 'p.Phe675ValfsTer42',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48503878_48503881del\", \"start\": 48503878, \"transcript_consequences\": [{\"loeuf\": 0.105, \"conservation\": 1.41, \"protein_end\": 676, \"phaplo\": 0.99897615427687, \"used_ref\": \"CAAA\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 2026, \"protein_start\": 675, \"transcript_id\": \"NM_000138.5\", \"mane\": [\"MANE_Select\"], \"cdna_end\": 2342, \"cdna_start\": 2339, \"hgvsc\": \"NM_000138.5:c.2023_2026del\", \"biotype\": \"protein_coding\", \"canonical\": 1, \"codons\": \"TTTGgt/gt\", \"lof\": \"HC\", \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": -38, \"DP_DG\": 44, \"DS_DL\": 0, \"DP_AG\": -41, \"SYMBOL\": \"FBN1\", \"DP_DL\": -43, \"DS_AL\": 0}, \"aa\": \"CAAA\", \"exon\": \"17/66\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"-\", \"lof_info\": \"PERCENTILE:0.235143918291551,GERP_DIST:6586.11709438125,BP_DIST:6545,DIST_FROM_LAST_EXON:6156,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"FG/X\", \"strand\": -1, \"cds_start\": 2023, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"hgvs_offset\": -4, \"location\": \"15:48503874-48503877\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48503877, \"end\": 48503881}}}, \"hgvsp\": \"NP_000129.3:p.Phe675ValfsTer42\", \"given_ref\": \"CAAA\", \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"]}], \"most_severe_consequence\": \"frameshift_variant\", \"end\": 48503881, \"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"CAAA/-\", \"vcf_string\": \"15-48503877-ACAAA-A\", \"input\": \"NC_000015.10:g.48503878_48503881del\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.044440099908131026,\n",
       "      'pca_y': 0.042304385768840456},\n",
       "     {'t-sne_x': 35.92958854392316, 't-sne_y': 8.031298743014414},\n",
       "     {'umap_x': 1.461431622505188, 'umap_y': 7.1810832023620605}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.10094384104013443,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.10189128667116165,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10276491940021515,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.1033308357000351,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420714-G-A',\n",
       "      'distance': 0.1034596860408783,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.10362027585506439,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.10377361625432968,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.10424342751502991,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.10425807535648346,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520764-G-A',\n",
       "      'distance': 0.10455623269081116,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10472337156534195,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.10480351746082306,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.10480717569589615,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.10498671233654022,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.1050407737493515,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437912-G-A',\n",
       "      'distance': 0.10512865334749222,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.10539059340953827,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.10545290261507034,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.10562780499458313,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.10563656687736511,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48503851-G-T',\n",
       "    'metadata': {'vcf_string': '15\\t48503851\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48503851',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2049C>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48503851G>T',\n",
       "     'hgvs_protein': 'p.Cys683Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"vcf_string\": \"15-48503851-G-T\", \"allele_string\": \"G/T\", \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48503851G>T\", \"seq_region_name\": \"15\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48503851G>T\", \"variant_class\": \"SNV\", \"start\": 48503851, \"end\": 48503851, \"most_severe_consequence\": \"stop_gained\", \"transcript_consequences\": [{\"loeuf\": 0.105, \"conservation\": 2.18, \"protein_end\": 683, \"phaplo\": 0.99897615427687, \"used_ref\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 2049, \"protein_start\": 683, \"transcript_id\": \"NM_000138.5\", \"mastermind_mmid3\": \"FBN1:C683X\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_start\": 2365, \"cdna_end\": 2365, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.2049C>A\", \"codons\": \"tgC/tgA\", \"canonical\": 1, \"cadd_phred\": 36, \"mastermind_counts\": \"3|3|3\", \"lof\": \"HC\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -6, \"SYMBOL\": \"FBN1\", \"DP_DL\": -17, \"DS_AL\": 0, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": -12, \"DP_DG\": 23}, \"aa\": \"G\", \"exon\": \"17/66\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"T\", \"lof_info\": \"PERCENTILE:0.237813370473538,GERP_DIST:6601.72370574335,BP_DIST:6518,DIST_FROM_LAST_EXON:6129,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"C/*\", \"mutationtaster_pred\": \"A\", \"strand\": -1, \"cds_start\": 2049, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"location\": \"15:48503851\", \"hgvsp\": \"NP_000129.3:p.Cys683Ter\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48503850, \"end\": 48503851}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"type\": \"Allele\"}, \"cadd_raw\": 7.964577, \"given_ref\": \"G\", \"allele_num\": 1, \"consequence_terms\": [\"stop_gained\"]}]}',\n",
       "    'coordinates': [{'pca_x': 0.021093253521011684,\n",
       "      'pca_y': 0.09265069691958963},\n",
       "     {'t-sne_x': 29.4178631440192, 't-sne_y': 38.3930704021563},\n",
       "     {'umap_x': -10.376983642578125, 'umap_y': 2.2674341201782227}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48503845-G-T',\n",
       "      'distance': 0.0008513064822182059,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421601-G-T',\n",
       "      'distance': 0.0008545387536287308,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495241-G-T',\n",
       "      'distance': 0.0009479615837335587,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495130-G-T',\n",
       "      'distance': 0.0010247343452647328,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474565-G-T',\n",
       "      'distance': 0.0010277549736201763,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421652-G-T',\n",
       "      'distance': 0.0011078087845817208,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415610-G-T',\n",
       "      'distance': 0.001291316351853311,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513553-G-T',\n",
       "      'distance': 0.0014113003853708506,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495575-G-T',\n",
       "      'distance': 0.0015036630211398005,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.001788584515452385,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485483-G-T',\n",
       "      'distance': 0.0019331874791532755,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.002063108142465353,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425416-G-T',\n",
       "      'distance': 0.002212847350165248,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.002264433540403843,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.0023942277766764164,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421995-G-T',\n",
       "      'distance': 0.002401323989033699,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.0025574674364179373,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.002956732641905546,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445419-G-T',\n",
       "      'distance': 0.003017583629116416,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489921-G-T',\n",
       "      'distance': 0.00351513153873384,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48503787-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48503787\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48503787',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2113G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48503787C>T',\n",
       "     'hgvs_protein': 'p.Ala705Thr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48503787C>T\", \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48503787-C-T\", \"strand\": 1, \"end\": 48503787, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"maxentscan_alt\": 7.443, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 705, \"cds_end\": 2113, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"polyphen_score\": 0.43, \"loeuf\": 0.105, \"conservation\": 3.95, \"sift_score\": 0.01, \"phaplo\": 0.99897615427687, \"protein_end\": 705, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:A705T,FBN1:A705srd\", \"rf_score\": 0.994, \"cdna_end\": 2429, \"cdna_start\": 2429, \"lrt_pred\": \"invalid_field\", \"alphamissense\": {\"am_pathogenicity\": 0.0947, \"am_class\": \"likely_benign\"}, \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"T\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"mastermind_counts\": \"9|17|19\", \"cadd_phred\": 29.9, \"maxentscan_ref\": 9.802, \"canonical\": 1, \"codons\": \"Gcg/Acg\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.2113G>A\", \"blosum62\": 0, \"exon\": \"17/66\", \"aa\": \"C\", \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": 21, \"DS_DL\": 0.74, \"DP_DG\": 47, \"DP_AL\": 6, \"DS_AG\": 0, \"DS_DG\": 0}, \"location\": \"15:48503787\", \"polyphen_prediction\": \"benign\", \"clinpred\": 0.873474549468182, \"maxentscan_diff\": 2.359, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"allele_num\": 1, \"cadd_raw\": 5.317808, \"given_ref\": \"C\", \"hgvsp\": \"NP_000129.3:p.Ala705Thr\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48503786, \"end\": 48503787, \"type\": \"SimpleInterval\"}}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}}, \"amino_acids\": \"A/T\", \"ada_score\": 0.999960824517021, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"cds_start\": 2113, \"mutationtaster_pred\": \"A\", \"strand\": -1}], \"start\": 48503787, \"colocated_variants\": [{\"id\": \"CM960626\", \"phenotype_or_disease\": 1, \"end\": 48503787, \"start\": 48503787, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\"}, {\"phenotype_or_disease\": 1, \"id\": \"CS098590\", \"end\": 48503787, \"start\": 48503787, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\"}, {\"start\": 48503787, \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000796267\", \"VCV000642750\"]}, \"allele_string\": \"C/T\", \"strand\": 1, \"end\": 48503787, \"phenotype_or_disease\": 1, \"id\": \"rs1597574156\", \"clin_sig_allele\": \"T:pathogenic\"}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48503787C>T\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.03378764066374178,\n",
       "      'pca_y': 0.04929603259346007},\n",
       "     {'t-sne_x': -13.173641668785596, 't-sne_y': -3.071139348715293},\n",
       "     {'umap_x': 0.34649890661239624, 'umap_y': 15.143325805664062}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.0021140221506357193,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.002305773086845875,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.002984838094562292,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.003075933549553156,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.003419519867748022,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.0034411102533340454,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.003496727207675576,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.00370002374984324,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.003840017132461071,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.003999526612460613,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.004046399146318436,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.00405688863247633,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.004238581750541925,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.0042822412215173244,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.004362574778497219,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.004529615864157677,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.00553913414478302,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.006052293349057436,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.006137493532150984,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.006412281654775143,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48498991-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48498991\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48498991',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2161G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48498991C>A',\n",
       "     'hgvs_protein': 'p.Gly721Cys',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM085404\", \"start\": 48498991, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48498991}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48498991C>A\", \"start\": 48498991, \"transcript_consequences\": [{\"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"A\", \"spliceai\": {\"DS_AG\": 0.07, \"DS_DG\": 0.04, \"DP_AL\": -1, \"DP_DG\": -6, \"DP_AG\": 47, \"DS_DL\": 0, \"DP_DL\": 0, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0}, \"aa\": \"C\", \"exon\": \"18/66\", \"blosum62\": -3, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.2161G>T\", \"codons\": \"Ggc/Tgc\", \"canonical\": 1, \"cadd_phred\": 28.3, \"mastermind_counts\": \"3|3|3\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"end\": 48498991, \"start\": 48498990, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}}, \"hgvsp\": \"NP_000129.3:p.Gly721Cys\", \"cadd_raw\": 5.065718, \"given_ref\": \"C\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"location\": \"15:48498991\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.98537677526474, \"strand\": -1, \"mutationtaster_pred\": \"D\", \"cds_start\": 2161, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"amino_acids\": \"G/C\", \"polyphen_score\": 0.996, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 2161, \"protein_start\": 721, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 721, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"conservation\": 3.45, \"lrt_pred\": \"invalid_field\", \"cdna_start\": 2477, \"cdna_end\": 2477, \"mastermind_mmid3\": \"FBN1:G721C\", \"mane\": [\"MANE_Select\"], \"eve_score\": 0.655833550694436, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.889}}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48498991, \"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/A\", \"vcf_string\": \"15-48498991-C-A\", \"input\": \"NC_000015.10:g.48498991C>A\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.03945702876525864,\n",
       "      'pca_y': 0.08221377851186454},\n",
       "     {'t-sne_x': 22.458139513081846, 't-sne_y': 14.257725819984564},\n",
       "     {'umap_x': 4.23883581161499, 'umap_y': 7.360351085662842}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48415582-C-A',\n",
       "      'distance': 0.002433564979583025,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.00275979982689023,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468097-G-A',\n",
       "      'distance': 0.002890553791075945,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415695-C-A',\n",
       "      'distance': 0.0029055860359221697,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.0030070680659264326,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.0030753419268876314,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.0031215723138302565,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.00317238736897707,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489896-C-T',\n",
       "      'distance': 0.0031752842478454113,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596318-C-A',\n",
       "      'distance': 0.0031985167879611254,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463904-C-A',\n",
       "      'distance': 0.0032828322146087885,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.0032940139062702656,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.003310756990686059,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.0033197104930877686,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.003373736049979925,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489956-A-G',\n",
       "      'distance': 0.0034153107553720474,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.0034210500307381153,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497358-C-A',\n",
       "      'distance': 0.0034291904885321856,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.003435459453612566,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.003437052248045802,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48497393-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48497393\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48497393',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2168-2A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48497393T>C',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"allele_string\": \"T/C\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48497393-T-C\", \"input\": \"NC_000015.10:g.48497393T>C\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48497393, \"end\": 48497393, \"id\": \"CS022103\", \"phenotype_or_disease\": 1}, {\"end\": 48497393, \"start\": 48497393, \"strand\": 1, \"allele_string\": \"T/C\", \"seq_region_name\": \"15\", \"clin_sig\": [\"pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV002314322\", \"VCV000519725\"]}, \"clin_sig_allele\": \"C:pathogenic\", \"id\": \"rs1555399388\", \"phenotype_or_disease\": 1}], \"id\": \"NC_000015.10:g.48497393T>C\", \"start\": 48497393, \"transcript_consequences\": [{\"loeuf\": 0.105, \"conservation\": 3.95, \"phaplo\": 0.99897615427687, \"maxentscan_alt\": 1.066, \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\", \"transcript_id\": \"NM_000138.5\", \"mane\": [\"MANE_Select\"], \"rf_score\": 0.944, \"mastermind_mmid3\": \"FBN1:D723sa\", \"intron\": \"18/65\", \"canonical\": 1, \"maxentscan_ref\": 9.021, \"cadd_phred\": 34, \"mastermind_counts\": \"3|3|10\", \"hgvsc\": \"NM_000138.5:c.2168-2A>G\", \"biotype\": \"protein_coding\", \"aa\": \"T\", \"lof\": \"HC\", \"spliceai\": {\"DP_DG\": 5, \"DP_AL\": -2, \"DS_DG\": 0, \"DS_AG\": 0.68, \"DS_AL\": 1, \"SYMBOL\": \"FBN1\", \"DP_DL\": -2, \"DS_DL\": 0, \"DP_AG\": -23}, \"variant_allele\": \"C\", \"lof_info\": \"INTRON_SIZE:1593\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"ada_score\": 0.999988247793601, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"genesplicer\": \"loss/acceptor/48497393-48497394/Medium/8.489863\", \"location\": \"15:48497393\", \"maxentscan_diff\": 7.955, \"given_ref\": \"T\", \"cadd_raw\": 6.376195, \"consequence_terms\": [\"splice_acceptor_variant\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48497392, \"end\": 48497393}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}}}], \"most_severe_consequence\": \"splice_acceptor_variant\", \"end\": 48497393}',\n",
       "    'coordinates': [{'pca_x': -0.24004744772030734,\n",
       "      'pca_y': -0.03451050392730705},\n",
       "     {'t-sne_x': -29.41344329761187, 't-sne_y': -19.158923019837914},\n",
       "     {'umap_x': 11.644562721252441, 'umap_y': 18.949920654296875}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.0015579058090224862,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.0016233142232522368,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.0018090361263602972,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.0019698343239724636,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.0023598307743668556,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.002561895176768303,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.0025957755278795958,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.0026035127229988575,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.0026419854257255793,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.002802882809191942,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.002897090744227171,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.0032192219514399767,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.0032257605344057083,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.0037517687305808067,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.004008772782981396,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452686-T-C',\n",
       "      'distance': 0.004062645137310028,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.004379297140985727,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.004894609097391367,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470757-C-T',\n",
       "      'distance': 0.0055656395852565765,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446823-C-G',\n",
       "      'distance': 0.0055982195772230625,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48497266-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48497266\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48497266',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2293G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48497266C>T',\n",
       "     'hgvs_protein': 'p.Asp765Asn',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"lrt_pred\": \"invalid_field\", \"cdna_end\": 2609, \"cdna_start\": 2609, \"mastermind_mmid3\": \"FBN1:D765N,FBN1:D765srd\", \"rf_score\": 0.992, \"mane\": [\"MANE_Select\"], \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9656}, \"polyphen_score\": 0.996, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"sift_prediction\": \"deleterious_low_confidence\", \"maxentscan_alt\": 0.426, \"protein_start\": 765, \"cds_end\": 2293, \"protein_end\": 765, \"phaplo\": 0.99897615427687, \"sift_score\": 0.01, \"conservation\": 3.95, \"loeuf\": 0.105, \"hgvsp\": \"NP_000129.3:p.Asp765Asn\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48497265, \"end\": 48497266}, \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"allele_num\": 1, \"given_ref\": \"C\", \"cadd_raw\": 8.282834, \"maxentscan_diff\": 5.859, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48497266\", \"clinpred\": 0.991911266367166, \"cds_start\": 2293, \"mutationtaster_pred\": \"A\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"ada_score\": 0.999997489571784, \"amino_acids\": \"D/N\", \"gene_symbol_source\": \"EntrezGene\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"T\", \"spliceai\": {\"DS_DL\": 0.41, \"DP_AG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 0, \"DS_DG\": 0.88, \"DS_AG\": 0, \"DP_DG\": 4, \"DP_AL\": 35}, \"blosum62\": 1, \"exon\": \"19/66\", \"aa\": \"C\", \"hgvsc\": \"NM_000138.5:c.2293G>A\", \"biotype\": \"protein_coding\", \"mastermind_counts\": \"4|5|5\", \"cadd_phred\": 37, \"codons\": \"Gat/Aat\", \"canonical\": 1, \"maxentscan_ref\": 6.286}], \"end\": 48497266, \"start\": 48497266, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CS127289\", \"end\": 48497266, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48497266}, {\"end\": 48497266, \"allele_string\": \"C/T\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV001812313\", \"VCV000993417\"]}, \"seq_region_name\": \"15\", \"clin_sig\": [\"likely_pathogenic\"], \"start\": 48497266, \"clin_sig_allele\": \"T:likely_pathogenic\", \"id\": \"rs2043615915\", \"phenotype_or_disease\": 1}], \"id\": \"NC_000015.10:g.48497266C>T\", \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48497266C>T\", \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48497266-C-T\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.04167985295261655,\n",
       "      'pca_y': 0.04962400661823373},\n",
       "     {'t-sne_x': -12.691802445370527, 't-sne_y': -2.7157887174270763},\n",
       "     {'umap_x': 0.46973368525505066, 'umap_y': 15.252174377441406}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.0013212868943810463,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.0020836552139371634,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.0022364305332303047,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.002302161417901516,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.0024173532146960497,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.002594033256173134,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.0025985506363213062,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.0027049118652939796,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.0028269097674638033,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.00286741741001606,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.0029025026597082615,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.0032793900463730097,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.003503313520923257,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.0035931861493736506,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.003843064187094569,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.003903630655258894,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.0056223031133413315,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.006047538481652737,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.006089949980378151,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.006193605717271566,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48496226-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48496226\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48496226',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2294-1G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48496226C>T',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48496226C>T\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48496226-C-T\", \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"start\": 48496226, \"transcript_consequences\": [{\"maxentscan_diff\": 8.75, \"location\": \"15:48496226\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48496226, \"start\": 48496225}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"allele_num\": 1, \"consequence_terms\": [\"splice_acceptor_variant\"], \"given_ref\": \"C\", \"cadd_raw\": 6.456859, \"ada_score\": 0.999990361384395, \"genesplicer\": \"loss/acceptor/48496225-48496226/Medium/6.251633\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"T\", \"lof_info\": \"INTRON_SIZE:1040\", \"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.2294-1G>A\", \"mastermind_counts\": \"1|1|4\", \"cadd_phred\": 34, \"canonical\": 1, \"maxentscan_ref\": 8.759, \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -26, \"DS_AL\": 0.99, \"SYMBOL\": \"FBN1\", \"DP_DL\": -14, \"DS_DG\": 0, \"DS_AG\": 0.61, \"DP_DG\": 47, \"DP_AL\": -1}, \"lof\": \"HC\", \"aa\": \"C\", \"mastermind_mmid3\": \"FBN1:D765sa\", \"rf_score\": 0.946, \"mane\": [\"MANE_Select\"], \"intron\": \"19/65\", \"transcript_id\": \"NM_000138.5\", \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"maxentscan_alt\": 0.008, \"loeuf\": 0.105, \"conservation\": 3.95, \"phaplo\": 0.99897615427687}], \"most_severe_consequence\": \"splice_acceptor_variant\", \"end\": 48496226, \"minimised\": 1, \"id\": \"NC_000015.10:g.48496226C>T\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"end\": 48496226, \"start\": 48496226, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"id\": \"CS040143\"}]}',\n",
       "    'coordinates': [{'pca_x': -0.2436438654411192,\n",
       "      'pca_y': -0.038382028499704},\n",
       "     {'t-sne_x': -29.00277689985133, 't-sne_y': -18.43306000433556},\n",
       "     {'umap_x': 11.744808197021484, 'umap_y': 18.851085662841797}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.0007270780624821782,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.0007340263691730797,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.001109367934986949,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.0011968822218477726,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.0021718547213822603,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.002409395296126604,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.002463705837726593,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.002719690091907978,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.002817947417497635,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.0029175295494496822,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.0030365418642759323,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.003130936762318015,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.003248834051191807,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.003351869760081172,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.0034780839923769236,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470757-C-T',\n",
       "      'distance': 0.003670591162517667,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.0038400422781705856,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446823-C-G',\n",
       "      'distance': 0.004269271623343229,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-A',\n",
       "      'distance': 0.004603235051035881,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.004979190416634083,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48496226-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48496226\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48496226',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2294-1G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48496226C>G',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48496226-C-G\", \"allele_string\": \"C/G\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48496226C>G\", \"id\": \"NC_000015.10:g.48496226C>G\", \"colocated_variants\": [{\"id\": \"CS040143\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48496226, \"end\": 48496226}], \"variant_class\": \"SNV\", \"minimised\": 1, \"end\": 48496226, \"most_severe_consequence\": \"splice_acceptor_variant\", \"transcript_consequences\": [{\"genesplicer\": \"loss/acceptor/48496225-48496226/Medium/6.251633\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"ada_score\": 0.999990361384395, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48496226, \"start\": 48496225}, \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}}, \"allele_num\": 1, \"consequence_terms\": [\"splice_acceptor_variant\"], \"given_ref\": \"C\", \"cadd_raw\": 6.313392, \"maxentscan_diff\": 8.064, \"location\": \"15:48496226\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -26, \"SYMBOL\": \"FBN1\", \"DP_DL\": -14, \"DS_AL\": 0.89, \"DS_DG\": 0, \"DS_AG\": 0.61, \"DP_AL\": -1, \"DP_DG\": 47}, \"lof\": \"HC\", \"aa\": \"C\", \"hgvsc\": \"NM_000138.5:c.2294-1G>C\", \"biotype\": \"protein_coding\", \"cadd_phred\": 34, \"mastermind_counts\": \"3|3|4\", \"maxentscan_ref\": 8.759, \"canonical\": 1, \"gene_symbol_source\": \"EntrezGene\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"G\", \"lof_info\": \"INTRON_SIZE:1040\", \"transcript_id\": \"NM_000138.5\", \"intron\": \"19/65\", \"mastermind_mmid3\": \"FBN1:D765sa\", \"rf_score\": 0.946, \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"conservation\": 3.95, \"loeuf\": 0.105, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"maxentscan_alt\": 0.694}], \"start\": 48496226}',\n",
       "    'coordinates': [{'pca_x': -0.24308907660275395,\n",
       "      'pca_y': -0.03490314020732149},\n",
       "     {'t-sne_x': -28.33412318808519, 't-sne_y': -18.802128204827813},\n",
       "     {'umap_x': 11.776309967041016, 'umap_y': 18.819067001342773}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.0010191724868491292,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446823-C-G',\n",
       "      'distance': 0.002919696271419525,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.003147023031488061,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.0031646646093577147,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.0032829996198415756,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.0032956923823803663,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.0035322739277035,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.003627524711191654,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.003707307856529951,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.0038971344474703074,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490079-C-G',\n",
       "      'distance': 0.003921056166291237,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.0039978669956326485,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.004017113242298365,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.004099169280380011,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.004237999673932791,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.0043325708247721195,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.005856258329004049,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.005866263993084431,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.005872425623238087,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470757-C-T',\n",
       "      'distance': 0.006855738814920187,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48496192-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48496192\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48496192',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2327G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48496192C>T',\n",
       "     'hgvs_protein': 'p.Cys776Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"vcf_string\": \"15-48496192-C-T\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\", \"input\": \"NC_000015.10:g.48496192C>T\", \"seq_region_name\": \"15\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48496192C>T\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"end\": 48496192, \"start\": 48496192, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"id\": \"CM013920\", \"phenotype_or_disease\": 1}, {\"phenotype_or_disease\": 1, \"id\": \"CM098661\", \"end\": 48496192, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48496192}, {\"end\": 48496192, \"strand\": 1, \"allele_string\": \"C/T\", \"seq_region_name\": \"15\", \"clin_sig\": [\"pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV000663529\", \"VCV000549073\"], \"UniProt\": [\"VAR_017989\"]}, \"start\": 48496192, \"clin_sig_allele\": \"T:pathogenic\", \"phenotype_or_disease\": 1, \"id\": \"rs1555399273\"}], \"start\": 48496192, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"amino_acids\": \"C/Y\", \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"cds_start\": 2327, \"mutationtaster_pred\": \"A\", \"strand\": -1, \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.998278919094129, \"location\": \"15:48496192\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"cadd_raw\": 5.202935, \"given_ref\": \"C\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"location\": {\"interval\": {\"start\": 48496191, \"end\": 48496192, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Cys776Tyr\", \"mastermind_counts\": \"5|11|11\", \"cadd_phred\": 29.2, \"codons\": \"tGt/tAt\", \"canonical\": 1, \"hgvsc\": \"NM_000138.5:c.2327G>A\", \"biotype\": \"protein_coding\", \"exon\": \"20/66\", \"blosum62\": -2, \"aa\": \"C\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -19, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 8, \"DS_DG\": 0, \"DS_AG\": 0.01, \"DP_DG\": 3, \"DP_AL\": 8}, \"variant_allele\": \"T\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9985}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.747016461752086, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C776Y\", \"cdna_start\": 2643, \"cdna_end\": 2643, \"lrt_pred\": \"invalid_field\", \"loeuf\": 0.105, \"conservation\": 3.95, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"protein_end\": 776, \"sift_prediction\": \"deleterious_low_confidence\", \"cds_end\": 2327, \"protein_start\": 776, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"polyphen_score\": 0.997}], \"end\": 48496192}',\n",
       "    'coordinates': [{'pca_x': 0.038283086761929874,\n",
       "      'pca_y': 0.08300948414521592},\n",
       "     {'t-sne_x': 6.09404399465346, 't-sne_y': 6.8768957196836356},\n",
       "     {'umap_x': 3.58998703956604, 'umap_y': 2.385993480682373}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48496213-C-T',\n",
       "      'distance': 0.0004401570186018944,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496177-C-T',\n",
       "      'distance': 0.0005953178042545915,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-T',\n",
       "      'distance': 0.0005996433319523931,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495176-C-T',\n",
       "      'distance': 0.0006411970825865865,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495173-C-T',\n",
       "      'distance': 0.0006441117729991674,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496144-C-T',\n",
       "      'distance': 0.0006558301975019276,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497379-C-T',\n",
       "      'distance': 0.0007382453186437488,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.0008422331884503365,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.0008593103266321123,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537718-C-T',\n",
       "      'distance': 0.0008607323979958892,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497358-C-T',\n",
       "      'distance': 0.0008708422537893057,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.0008711983682587743,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.0009707636781968176,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497343-C-T',\n",
       "      'distance': 0.0010029845871031284,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0010061883367598057,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520733-C-T',\n",
       "      'distance': 0.0010097805643454194,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.0010133436881005764,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432976-C-T',\n",
       "      'distance': 0.0010560029186308384,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485379-C-T',\n",
       "      'distance': 0.001087477896362543,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596342-C-T',\n",
       "      'distance': 0.001087951473891735,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48495540-TT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48495540\\t.\\tTT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48495540',\n",
       "     'ref_allele': 'TT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2468del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48495541del',\n",
       "     'hgvs_protein': 'p.Asn823ThrfsTer24',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"start\": 48495541, \"transcript_consequences\": [{\"hgvsp\": \"NP_000129.3:p.Asn823ThrfsTer24\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48495541, \"start\": 48495540}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"given_ref\": \"T\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"hgvs_offset\": -1, \"location\": \"15:48495540\", \"strand\": -1, \"mutfunc\": {\"exp\": {\"dG_mt\": 36.464079, \"dG_wt_sd\": 0.022443, \"ddG\": 0.380699, \"dG_wt\": 36.08338, \"dG_mt_sd\": 0.011484, \"ddG_sd\": 0.021653}}, \"cds_start\": 2468, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"amino_acids\": \"N/X\", \"gene_symbol_source\": \"EntrezGene\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"lof_info\": \"PERCENTILE:0.28644382544104,GERP_DIST:6053.72533015311,BP_DIST:6104,DIST_FROM_LAST_EXON:5715,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"lof\": \"HC\", \"spliceai\": {\"DP_AG\": -40, \"DS_DL\": 0, \"DS_AL\": 0, \"DP_DL\": -38, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DG\": 0, \"DP_DG\": -20, \"DP_AL\": 48}, \"aa\": \"T\", \"exon\": \"21/66\", \"hgvsc\": \"NM_000138.5:c.2468del\", \"biotype\": \"protein_coding\", \"canonical\": 1, \"codons\": \"aAc/ac\", \"cdna_end\": 2784, \"cdna_start\": 2784, \"mane\": [\"MANE_Select\"], \"transcript_id\": \"NM_000138.5\", \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 2468, \"protein_start\": 823, \"protein_end\": 823, \"phaplo\": 0.99897615427687, \"conservation\": 3.95, \"loeuf\": 0.105}], \"most_severe_consequence\": \"frameshift_variant\", \"end\": 48495541, \"minimised\": 1, \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48495541del\", \"input\": \"NC_000015.10:g.48495541del\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"T/-\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48495540-TT-T\"}',\n",
       "    'coordinates': [{'pca_x': -0.040536817642194786,\n",
       "      'pca_y': 0.02641272668280171},\n",
       "     {'t-sne_x': -12.924126723879134, 't-sne_y': -3.133336516453335},\n",
       "     {'umap_x': 0.4805017411708832, 'umap_y': 15.29361629486084}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.0942118763923645,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.09567512571811676,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.0964544266462326,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.09670601785182953,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.09740930795669556,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.09784526377916336,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.09796909242868423,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.09809991717338562,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.09817793220281601,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.098509281873703,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.09852445125579834,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.09894898533821106,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.09933119267225266,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.09947091341018677,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.10014647245407104,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.10046054422855377,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.10055646300315857,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.10112859308719635,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.10284779965877533,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487075-C-T',\n",
       "      'distance': 0.10302335023880005,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48495519-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48495519\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48495519',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2489G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48495519C>T',\n",
       "     'hgvs_protein': 'p.Cys830Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48495519-C-T\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48495519C>T\", \"id\": \"NC_000015.10:g.48495519C>T\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"end\": 48495519, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48495519, \"phenotype_or_disease\": 1, \"id\": \"CM065179\"}, {\"end\": 48495519, \"start\": 48495519, \"allele_string\": \"C/G/T\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000697797\", \"RCV000035144\", \"VCV000042310\", \"RCV002426546\", \"RCV000663549\", \"VCV000549090\"]}, \"seq_region_name\": \"15\", \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"clin_sig_allele\": \"T:pathogenic;G:pathogenic;G:likely_pathogenic\", \"phenotype_or_disease\": 1, \"id\": \"rs397515774\", \"pubmed\": [24033266, 19293843, 10486319, 17701892]}], \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"polyphen_prediction\": \"possibly_damaging\", \"location\": \"15:48495519\", \"clinpred\": 0.999493956565857, \"hgvsp\": \"NP_000129.3:p.Cys830Tyr\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"start\": 48495518, \"end\": 48495519}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"cadd_raw\": 5.286076, \"given_ref\": \"C\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"amino_acids\": \"C/Y\", \"strand\": -1, \"mutationtaster_pred\": \"A\", \"mutfunc\": {\"exp\": {\"dG_wt\": 36.02324, \"dG_mt_sd\": 2.161243, \"ddG_sd\": 2.206806, \"dG_mt\": 55.65862, \"ddG\": 19.63538, \"dG_wt_sd\": 0.057632}}, \"cds_start\": 2489, \"impact\": \"MODERATE\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"T\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.2489G>A\", \"biotype\": \"protein_coding\", \"codons\": \"tGt/tAt\", \"canonical\": 1, \"cadd_phred\": 29.7, \"mastermind_counts\": \"3|4|4\", \"spliceai\": {\"DP_DG\": 13, \"DP_AL\": 17, \"DS_DG\": 0, \"DS_AG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -50, \"DS_DL\": 0, \"DP_AG\": 47}, \"aa\": \"C\", \"exon\": \"21/66\", \"blosum62\": -2, \"mastermind_mmid3\": \"FBN1:C830Y\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_end\": 2805, \"cdna_start\": 2805, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9977}, \"eve_score\": 0.768528356707554, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 830, \"cds_end\": 2489, \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.892, \"loeuf\": 0.105, \"conservation\": 3.95, \"protein_end\": 830, \"sift_score\": 0, \"phaplo\": 0.99897615427687}], \"end\": 48495519, \"start\": 48495519}',\n",
       "    'coordinates': [{'pca_x': 0.028768561359695147,\n",
       "      'pca_y': 0.11171643270115061},\n",
       "     {'t-sne_x': -11.840089552955376, 't-sne_y': 32.45487990225016},\n",
       "     {'umap_x': 1.014746069908142, 'umap_y': 22.115568161010742}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48495131-C-T',\n",
       "      'distance': 0.0006485098274424672,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.0007087743142619729,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.0009547773515805602,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492538-C-T',\n",
       "      'distance': 0.0010653571225702763,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.0011203138856217265,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.0011382853845134377,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472628-C-T',\n",
       "      'distance': 0.001165306894108653,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.0012112571857869625,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.001263870857656002,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472601-C-T',\n",
       "      'distance': 0.0014153013471513987,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.0014483288396149874,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472556-C-T',\n",
       "      'distance': 0.001531711546704173,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.0015767149161547422,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495156-C-T',\n",
       "      'distance': 0.0016777236014604568,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-C',\n",
       "      'distance': 0.0016813324764370918,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463949-C-T',\n",
       "      'distance': 0.0017942094709724188,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421990-C-T',\n",
       "      'distance': 0.0018876696703955531,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.001999479718506336,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441765-C-T',\n",
       "      'distance': 0.002035706304013729,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452654-C-T',\n",
       "      'distance': 0.0023361756466329098,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48495213-C-CA',\n",
       "    'metadata': {'vcf_string': '15\\t48495213\\t.\\tC\\tCA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48495213',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'CA',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2586dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48495214dup',\n",
       "     'hgvs_protein': 'p.Glu863Ter',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"allele_string\": \"-/A\", \"vcf_string\": \"15-48495213-C-CA\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48495214dup\", \"variant_class\": \"insertion\", \"colocated_variants\": [{\"end\": 48495214, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000663559\", \"VCV000549098\"]}, \"clin_sig\": [\"pathogenic\"], \"strand\": 1, \"allele_string\": \"A/AA\", \"start\": 48495214, \"id\": \"rs1555399160\", \"phenotype_or_disease\": 1}], \"id\": \"NC_000015.10:g.48495214dup\", \"minimised\": 1, \"end\": 48495213, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"cdna_end\": 2903, \"cdna_start\": 2902, \"mastermind_mmid3\": \"FBN1:E863X\", \"mane\": [\"MANE_Select\"], \"transcript_id\": \"NM_000138.5\", \"used_ref\": \"-\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 862, \"cds_end\": 2587, \"protein_end\": 863, \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"conservation\": 2.72, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48495213, \"end\": 48495213}, \"type\": \"SequenceLocation\"}}, \"hgvsp\": \"NP_000129.3:p.Glu863Ter\", \"given_ref\": \"-\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"location\": \"15:48495213-48495214\", \"strand\": -1, \"cds_start\": 2586, \"mutfunc\": {\"exp\": {\"dG_wt\": 36.12824, \"dG_mt_sd\": 1.532928, \"ddG_sd\": 1.531065, \"dG_mt\": 51.0413, \"ddG\": 14.91306, \"dG_wt_sd\": 0.061298}}, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"amino_acids\": \"-/X\", \"gene_symbol_source\": \"EntrezGene\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"A\", \"lof_info\": \"PERCENTILE:0.300139275766017,GERP_DIST:6053.72533015311,BP_DIST:5986,DIST_FROM_LAST_EXON:5597,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"lof\": \"HC\", \"spliceai\": {\"DP_AL\": 47, \"DP_DG\": 47, \"DS_AG\": 0, \"DS_DG\": 0, \"DP_DL\": 3, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AG\": 36, \"DS_DL\": 0}, \"exon\": \"22/66\", \"hgvsc\": \"NM_000138.5:c.2586dup\", \"biotype\": \"protein_coding\", \"codons\": \"-/T\", \"canonical\": 1, \"mastermind_counts\": \"4|4|4\"}], \"start\": 48495214}',\n",
       "    'coordinates': [{'pca_x': -0.01745440137682222,\n",
       "      'pca_y': 0.0141272922155991},\n",
       "     {'t-sne_x': -8.143171677799517, 't-sne_y': -11.90813360394345},\n",
       "     {'umap_x': 1.5245307683944702, 'umap_y': 11.697088241577148}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410123-C-A',\n",
       "      'distance': 0.08999498188495636,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48408466-T-C',\n",
       "      'distance': 0.09176433831453323,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48409031-C-T',\n",
       "      'distance': 0.09259563684463501,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48409622-T-C',\n",
       "      'distance': 0.09294633567333221,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48408950-A-G',\n",
       "      'distance': 0.09308289736509323,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48409001-G-A',\n",
       "      'distance': 0.09425829350948334,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48409428-C-A',\n",
       "      'distance': 0.09527599066495895,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48410933-T-C',\n",
       "      'distance': 0.0956801176071167,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.09583662450313568,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09639203548431396,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644804-G-A',\n",
       "      'distance': 0.09713931381702423,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09786957502365112,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.09793280065059662,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.0991055890917778,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.09964156150817871,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.0999264046549797,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.10017096251249313,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.10023181140422821,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.10031502693891525,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432907-G-A',\n",
       "      'distance': 0.10032378137111664,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_benign',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'likely_benign', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_benign',\n",
       "      'confidence_score': 0.7333333333333333},\n",
       "     '20': {'pred_result': 'uncertain', 'confidence_score': 0.55}}},\n",
       "   {'variant_id': '15-48495177-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48495177\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48495177',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2623T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48495177A>G',\n",
       "     'hgvs_protein': 'p.Cys875Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"vcf_string\": \"15-48495177-A-G\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/G\", \"input\": \"NC_000015.10:g.48495177A>G\", \"seq_region_name\": \"15\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48495177A>G\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"start\": 48495177, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"end\": 48495177, \"id\": \"CM098668\", \"phenotype_or_disease\": 1}], \"start\": 48495177, \"most_severe_consequence\": \"missense_variant\", \"end\": 48495177, \"transcript_consequences\": [{\"polyphen_score\": 0.997, \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 875, \"cds_end\": 2623, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 875, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"loeuf\": 0.105, \"conservation\": 3.95, \"lrt_pred\": \"invalid_field\", \"cdna_start\": 2939, \"cdna_end\": 2939, \"mastermind_mmid3\": \"FBN1:C875R\", \"mane\": [\"MANE_Select\"], \"eve_score\": 0.769543669212004, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.9993, \"am_class\": \"likely_pathogenic\"}, \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"G\", \"spliceai\": {\"DP_AG\": 8, \"DS_DL\": 0, \"DP_DL\": 39, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_AG\": 0, \"DS_DG\": 0, \"DP_AL\": 6, \"DP_DG\": -50}, \"aa\": \"A\", \"blosum62\": -3, \"exon\": \"22/66\", \"hgvsc\": \"NM_000138.5:c.2623T>C\", \"biotype\": \"protein_coding\", \"canonical\": 1, \"codons\": \"Tgc/Cgc\", \"cadd_phred\": 28.6, \"mastermind_counts\": \"2|3|3\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48495176, \"end\": 48495177, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"hgvsp\": \"NP_000129.3:p.Cys875Arg\", \"cadd_raw\": 5.110344, \"given_ref\": \"A\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"location\": \"15:48495177\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.999746263027191, \"strand\": -1, \"mutationtaster_pred\": \"D\", \"cds_start\": 2623, \"mutfunc\": {\"exp\": {\"dG_wt\": 35.11562, \"dG_mt_sd\": 4.514333, \"ddG_sd\": 3.498243, \"dG_mt\": 56.4125, \"dG_wt_sd\": 1.374668, \"ddG\": 21.29688}}, \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"amino_acids\": \"C/R\"}]}',\n",
       "    'coordinates': [{'pca_x': 0.04525268395867377,\n",
       "      'pca_y': 0.07908950823336983},\n",
       "     {'t-sne_x': 34.503315395976024, 't-sne_y': 2.8423466208317008},\n",
       "     {'umap_x': 1.2218199968338013, 'umap_y': 6.501911640167236}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48497311-A-G',\n",
       "      'distance': 0.0007361314492300153,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.0007448560791090131,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.0007937463233247399,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.0009358120150864124,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.000949293258599937,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0010483309160917997,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.0010916523169726133,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.001103327376767993,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.0012163713108748198,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.0013009583344683051,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0014439650112763047,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526223-A-G',\n",
       "      'distance': 0.0014491182519122958,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.001476122415624559,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0015125095378607512,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.0015191464917734265,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.0015350632602348924,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0016376806888729334,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.0016496932366862893,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0016505607636645436,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.001723429886624217,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48495149-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48495149\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48495149',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2651G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48495149C>T',\n",
       "     'hgvs_protein': 'p.Gly884Glu',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"start\": 48495149, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48495149, \"phenotype_or_disease\": 1, \"id\": \"CM085414\"}, {\"id\": \"rs1555399149\", \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"T:pathogenic;T:likely_pathogenic\", \"strand\": 1, \"allele_string\": \"C/T\", \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"UniProt\": [\"VAR_076030\"], \"ClinVar\": [\"RCV000632001\", \"VCV000527201\", \"RCV005049623\"]}, \"start\": 48495149, \"end\": 48495149}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48495149C>T\", \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"loeuf\": 0.105, \"conservation\": 3.06, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"protein_end\": 884, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 884, \"cds_end\": 2651, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"polyphen_score\": 0.999, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9788}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.765199187712666, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:G884E\", \"cdna_end\": 2967, \"cdna_start\": 2967, \"lrt_pred\": \"invalid_field\", \"mastermind_counts\": \"4|4|4\", \"cadd_phred\": 26.6, \"codons\": \"gGa/gAa\", \"canonical\": 1, \"hgvsc\": \"NM_000138.5:c.2651G>A\", \"biotype\": \"protein_coding\", \"blosum62\": -2, \"exon\": \"22/66\", \"aa\": \"C\", \"spliceai\": {\"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -23, \"DS_DL\": 0, \"DP_AG\": 36, \"DP_DG\": -22, \"DP_AL\": -4, \"DS_DG\": 0.01, \"DS_AG\": 0}, \"variant_allele\": \"T\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"G/E\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"cds_start\": 2651, \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.607283, \"dG_mt_sd\": 0.494535, \"dG_wt\": 35.09816, \"ddG\": 8.17948, \"dG_wt_sd\": 0.208917, \"dG_mt\": 43.27764}}, \"strand\": -1, \"mutationtaster_pred\": \"D\", \"clinpred\": 0.987068393413014, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48495149\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"cadd_raw\": 4.762764, \"given_ref\": \"C\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48495149, \"start\": 48495148, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Gly884Glu\"}], \"end\": 48495149, \"start\": 48495149, \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48495149-C-T\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48495149C>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.034484615635215515,\n",
       "      'pca_y': 0.10588660107800957},\n",
       "     {'t-sne_x': -11.496375772165925, 't-sne_y': 30.71542102895444},\n",
       "     {'umap_x': 1.2894374132156372, 'umap_y': 22.327930450439453}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48437038-C-T',\n",
       "      'distance': 0.0017259707674384117,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481690-C-T',\n",
       "      'distance': 0.0019201573450118303,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508665-C-T',\n",
       "      'distance': 0.002013539196923375,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495156-C-T',\n",
       "      'distance': 0.0021363322157412767,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.002192629501223564,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481729-T-C',\n",
       "      'distance': 0.0029310835525393486,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.003090600250288844,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421990-C-T',\n",
       "      'distance': 0.0031376879196614027,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495131-C-T',\n",
       "      'distance': 0.003146575763821602,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.0032954153139144182,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441765-C-T',\n",
       "      'distance': 0.0033410578034818172,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-C',\n",
       "      'distance': 0.003524595173075795,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.0035517429932951927,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.0035952571779489517,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.003733284305781126,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.0037697742227464914,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495519-C-A',\n",
       "      'distance': 0.003771840361878276,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.0037902002222836018,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.0038182528223842382,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472601-C-T',\n",
       "      'distance': 0.003835911164060235,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48494245-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48494245\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48494245',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2687G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48494245C>T',\n",
       "     'hgvs_protein': 'p.Cys896Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"end\": 48494245, \"start\": 48494245, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM098669\", \"phenotype_or_disease\": 1}, {\"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48494245, \"end\": 48494245, \"phenotype_or_disease\": 1, \"id\": \"CX020910\"}, {\"clin_sig_allele\": \"T:pathogenic\", \"phenotype_or_disease\": 1, \"id\": \"rs2141302442\", \"pubmed\": [17701892], \"end\": 48494245, \"start\": 48494245, \"allele_string\": \"C/T\", \"strand\": 1, \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV001993169\", \"VCV001451245\"]}}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48494245C>T\", \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"end\": 48494245, \"transcript_consequences\": [{\"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48494245\", \"clinpred\": 0.99845278263092, \"cadd_raw\": 5.200631, \"given_ref\": \"C\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"start\": 48494244, \"end\": 48494245}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Cys896Tyr\", \"amino_acids\": \"C/Y\", \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"D\", \"strand\": -1, \"mutfunc\": {\"exp\": {\"dG_wt\": 35.86574, \"ddG_sd\": 2.115174, \"dG_mt_sd\": 2.179972, \"ddG\": 33.00286, \"dG_wt_sd\": 0.203042, \"dG_mt\": 68.8686}}, \"cds_start\": 2687, \"variant_allele\": \"T\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"codons\": \"tGt/tAt\", \"canonical\": 1, \"mastermind_counts\": \"2|2|2\", \"cadd_phred\": 29.2, \"hgvsc\": \"NM_000138.5:c.2687G>A\", \"biotype\": \"protein_coding\", \"aa\": \"C\", \"exon\": \"23/66\", \"blosum62\": -2, \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": -41, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": 2, \"DP_AL\": 9, \"DP_DG\": -2, \"DS_DG\": 0, \"DS_AG\": 0}, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C896Y\", \"cdna_start\": 3003, \"cdna_end\": 3003, \"lrt_pred\": \"invalid_field\", \"alphamissense\": {\"am_pathogenicity\": 0.9971, \"am_class\": \"likely_pathogenic\"}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.780417810793591, \"cds_end\": 2687, \"protein_start\": 896, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.993, \"loeuf\": 0.105, \"conservation\": 3.85, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 896}], \"start\": 48494245, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\", \"vcf_string\": \"15-48494245-C-T\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48494245C>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.030250462472045853,\n",
       "      'pca_y': 0.10625689939529442},\n",
       "     {'t-sne_x': -10.42682818890124, 't-sne_y': 32.52581979982527},\n",
       "     {'umap_x': 0.9175955653190613, 'umap_y': 22.262737274169922}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48421990-C-T',\n",
       "      'distance': 0.0009943880140781403,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.001268360996618867,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474260-C-T',\n",
       "      'distance': 0.0013615838252007961,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48441765-C-T',\n",
       "      'distance': 0.0013980793301016092,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492538-C-T',\n",
       "      'distance': 0.001399919972755015,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495131-C-T',\n",
       "      'distance': 0.001566539634950459,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.0015850262716412544,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495140-C-T',\n",
       "      'distance': 0.0016212298069149256,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.0016850647516548634,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.0017835134640336037,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.0018409962067380548,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.0018829701002687216,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452654-C-T',\n",
       "      'distance': 0.0019027271773666143,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-C',\n",
       "      'distance': 0.0019895490258932114,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.0020726248621940613,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508692-C-T',\n",
       "      'distance': 0.002107449574396014,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.0026617012917995453,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481690-C-T',\n",
       "      'distance': 0.002707959618419409,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515440-T-T',\n",
       "      'distance': 0.0027578379958868027,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48495156-C-T',\n",
       "      'distance': 0.0028656814247369766,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48492577-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48492577\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48492577',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2738A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48492577T>C',\n",
       "     'hgvs_protein': 'p.Glu913Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48492577T>C\", \"colocated_variants\": [{\"id\": \"CM013922\", \"phenotype_or_disease\": 1, \"end\": 48492577, \"start\": 48492577, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\"}, {\"id\": \"rs1555398995\", \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"G:likely_pathogenic;C:likely_pathogenic\", \"start\": 48492577, \"var_synonyms\": {\"ClinVar\": [\"RCV000559175\", \"VCV000457180\", \"RCV000663576\", \"VCV000549109\"], \"UniProt\": [\"VAR_017994\"]}, \"seq_region_name\": \"15\", \"clin_sig\": [\"likely_pathogenic\"], \"strand\": 1, \"allele_string\": \"T/C/G\", \"end\": 48492577}], \"variant_class\": \"SNV\", \"minimised\": 1, \"transcript_consequences\": [{\"protein_end\": 913, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"conservation\": 3.76, \"polyphen_score\": 0.997, \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 2738, \"protein_start\": 913, \"sift_prediction\": \"deleterious_low_confidence\", \"eve_score\": 0.809440872372077, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9923}, \"lrt_pred\": \"invalid_field\", \"cdna_start\": 3054, \"cdna_end\": 3054, \"mastermind_mmid3\": \"FBN1:E913G\", \"mane\": [\"MANE_Select\"], \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0.01, \"DP_AL\": -18, \"DP_DG\": -19, \"DS_DL\": 0, \"DP_AG\": 3, \"SYMBOL\": \"FBN1\", \"DP_DL\": -2, \"DS_AL\": 0}, \"aa\": \"T\", \"exon\": \"24/66\", \"blosum62\": -2, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.2738A>G\", \"codons\": \"gAa/gGa\", \"canonical\": 1, \"cadd_phred\": 29.8, \"mastermind_counts\": \"3|9|9\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"C\", \"mutationtaster_pred\": \"A\", \"strand\": -1, \"mutfunc\": {\"exp\": {\"dG_wt_sd\": 0.40196, \"ddG\": 1.60932, \"dG_mt\": 35.89902, \"ddG_sd\": 0.408949, \"dG_mt_sd\": 0.050205, \"dG_wt\": 34.2897}}, \"cds_start\": 2738, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"amino_acids\": \"E/G\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"start\": 48492576, \"end\": 48492577}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"hgvsp\": \"NP_000129.3:p.Glu913Gly\", \"cadd_raw\": 5.301818, \"given_ref\": \"T\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"location\": \"15:48492577\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.99958473443985}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48492577, \"start\": 48492577, \"vcf_string\": \"15-48492577-T-C\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"T/C\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48492577T>C\"}',\n",
       "    'coordinates': [{'pca_x': 0.04117721834400545,\n",
       "      'pca_y': 0.07773584825657318},\n",
       "     {'t-sne_x': 5.66101760920751, 't-sne_y': 1.8397808712933887},\n",
       "     {'umap_x': 4.505111217498779, 'umap_y': 3.2893004417419434}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48497382-T-C',\n",
       "      'distance': 0.0014660414308309555,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492578-C-T',\n",
       "      'distance': 0.002018613973632455,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488233-C-T',\n",
       "      'distance': 0.003161383792757988,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.003275846363976598,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421981-C-T',\n",
       "      'distance': 0.003276384435594082,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.003374354215338826,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474584-C-T',\n",
       "      'distance': 0.0035122826229780912,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415707-C-T',\n",
       "      'distance': 0.0037981716450303793,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487107-T-C',\n",
       "      'distance': 0.0038254628889262676,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537706-C-T',\n",
       "      'distance': 0.0038300759624689817,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425483-C-T',\n",
       "      'distance': 0.004018641076982021,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481687-T-C',\n",
       "      'distance': 0.004091751296073198,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421594-C-T',\n",
       "      'distance': 0.004123158287256956,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421982-C-T',\n",
       "      'distance': 0.004131955560296774,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489896-C-T',\n",
       "      'distance': 0.004154185764491558,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448851-C-T',\n",
       "      'distance': 0.004229324869811535,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492574-C-T',\n",
       "      'distance': 0.004333630204200745,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.00434358324855566,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.004390158224850893,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434707-T-C',\n",
       "      'distance': 0.004399208817631006,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48492506-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48492506\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48492506',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2809T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48492506A>G',\n",
       "     'hgvs_protein': 'p.Cys937Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"allele_string\": \"A/G\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48492506-A-G\", \"input\": \"NC_000015.10:g.48492506A>G\", \"seq_region_name\": \"15\", \"minimised\": 1, \"colocated_variants\": [{\"start\": 48492506, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48492506, \"id\": \"CM085418\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48492506A>G\", \"start\": 48492506, \"most_severe_consequence\": \"missense_variant\", \"end\": 48492506, \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C937R\", \"cdna_start\": 3125, \"cdna_end\": 3125, \"lrt_pred\": \"invalid_field\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9985}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.706384639445649, \"protein_start\": 937, \"cds_end\": 2809, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.995, \"loeuf\": 0.105, \"conservation\": 3.95, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"protein_end\": 937, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48492506\", \"clinpred\": 0.999979972839355, \"cadd_raw\": 5.145345, \"given_ref\": \"A\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"hgvsp\": \"NP_000129.3:p.Cys937Arg\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48492506, \"start\": 48492505}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"amino_acids\": \"C/R\", \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"mutationtaster_pred\": \"D\", \"mutfunc\": {\"exp\": {\"dG_wt\": 35.86958, \"ddG_sd\": 0.398297, \"dG_mt_sd\": 0.106997, \"dG_wt_sd\": 0.377874, \"ddG\": 5.31708, \"dG_mt\": 41.18666}}, \"cds_start\": 2809, \"variant_allele\": \"G\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"canonical\": 1, \"codons\": \"Tgt/Cgt\", \"cadd_phred\": 28.8, \"mastermind_counts\": \"5|5|5\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.2809T>C\", \"aa\": \"A\", \"exon\": \"24/66\", \"blosum62\": -3, \"spliceai\": {\"DP_AL\": 6, \"DP_DG\": -45, \"DS_DG\": 0, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 6, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": 19}}]}',\n",
       "    'coordinates': [{'pca_x': 0.04494283154355033,\n",
       "      'pca_y': 0.07211255892870788},\n",
       "     {'t-sne_x': 34.68535750488156, 't-sne_y': 3.3553008759235325},\n",
       "     {'umap_x': 1.2369579076766968, 'umap_y': 6.53375768661499}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.0006320093525573611,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.000639094621874392,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.0007757621351629496,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0007892180001363158,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.0007964540272951126,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0009563384810462594,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0010039816843345761,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0010284874588251114,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0010358602739870548,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.0010434540454298258,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.0010675376979634166,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.0010792145039886236,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0011223272886127234,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.0011246999492868781,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430770-A-G',\n",
       "      'distance': 0.0011490887263789773,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0011752522550523281,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427603-A-G',\n",
       "      'distance': 0.0012737130746245384,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.0012978017330169678,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497311-A-G',\n",
       "      'distance': 0.0013587083667516708,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.0014078202657401562,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48489983-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48489983\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48489983',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2950G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48489983C>T',\n",
       "     'hgvs_protein': 'p.Val984Ile',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48489983C>T\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"start\": 48489983, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"end\": 48489983, \"id\": \"CM960628\", \"phenotype_or_disease\": 1}, {\"somatic\": 1, \"phenotype_or_disease\": 1, \"id\": \"COSV100353780\", \"end\": 48489983, \"start\": 48489983, \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM7553097\"]}, \"seq_region_name\": \"15\"}, {\"somatic\": 1, \"id\": \"COSV57319967\", \"phenotype_or_disease\": 1, \"end\": 48489983, \"start\": 48489983, \"var_synonyms\": {\"COSMIC\": [\"COSM3501826\"]}, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\"}, {\"phenotype_or_disease\": 1, \"id\": \"rs747713929\", \"clin_sig_allele\": \"T:uncertain_significance;T:conflicting_classifications_of_pathogenicity;T:likely_benign\", \"allele_string\": \"C/A/G/T\", \"strand\": 1, \"seq_region_name\": \"15\", \"clin_sig\": [\"uncertain_significance\", \"likely_benign\"], \"var_synonyms\": {\"ClinVar\": [\"RCV000536950\", \"VCV000457184\", \"RCV001121900\", \"RCV001121899\", \"RCV001121898\", \"RCV001121897\", \"RCV001121896\", \"RCV001115325\", \"RCV001115324\", \"RCV001311040\"], \"UniProt\": [\"VAR_002296\"]}, \"start\": 48489983, \"end\": 48489983, \"frequencies\": {\"T\": {\"gnomadg_amr\": 0, \"gnomadg_ami\": 0, \"gnomadg_remaining\": 0, \"gnomadg_asj\": 0, \"gnomade_remaining\": 1.656e-05, \"gnomadg_nfe\": 4.41e-05, \"gnomade_fin\": 0, \"gnomade_sas\": 0, \"gnomadg_sas\": 0, \"gnomade_mid\": 0, \"gnomadg_fin\": 0, \"gnomadg_mid\": 0, \"gnomadg_eas\": 0, \"gnomade_nfe\": 1.619e-05, \"gnomadg_afr\": 2.415e-05, \"gnomade\": 1.368e-05, \"gnomade_amr\": 0, \"gnomade_eas\": 2.519e-05, \"gnomade_asj\": 0, \"gnomadg\": 2.63e-05, \"gnomade_afr\": 0}}}], \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"end\": 48489983, \"transcript_consequences\": [{\"hgvsp\": \"NP_000129.3:p.Val984Ile\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48489983, \"start\": 48489982}}, \"type\": \"Allele\"}, \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"cadd_raw\": 4.054893, \"given_ref\": \"C\", \"location\": \"15:48489983\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.270402098172735, \"cds_start\": 2950, \"strand\": -1, \"mutationtaster_pred\": \"N\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"amino_acids\": \"V/I\", \"eve_class\": \"Benign\", \"gene_symbol_source\": \"EntrezGene\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"T\", \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": -10, \"DP_DG\": -4, \"DS_DL\": 0, \"DP_AG\": -2, \"SYMBOL\": \"FBN1\", \"DP_DL\": -15, \"DS_AL\": 0}, \"blosum62\": 3, \"exon\": \"25/66\", \"aa\": \"C\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.2950G>A\", \"mastermind_counts\": \"8|12|12\", \"cadd_phred\": 24.2, \"codons\": \"Gtc/Atc\", \"canonical\": 1, \"lrt_pred\": \"invalid_field\", \"cdna_start\": 3266, \"cdna_end\": 3266, \"mastermind_mmid3\": \"FBN1:V984I\", \"mane\": [\"MANE_Select\"], \"eve_score\": 0.260832912104145, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_benign\", \"am_pathogenicity\": 0.0818}, \"polyphen_score\": 0.958, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 984, \"cds_end\": 2950, \"protein_end\": 984, \"sift_score\": 0.04, \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"conservation\": 3.76}], \"start\": 48489983, \"vcf_string\": \"15-48489983-C-T\", \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48489983C>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.038515939712854, 'pca_y': 0.08733128471744386},\n",
       "     {'t-sne_x': 9.280472498168546, 't-sne_y': -1.8202772174593902},\n",
       "     {'umap_x': 4.12098503112793, 'umap_y': 3.968780040740967}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48489977-C-T',\n",
       "      'distance': 0.0020886037964373827,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48470684-C-T',\n",
       "      'distance': 0.002561114961281419,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472595-C-T',\n",
       "      'distance': 0.002602535765618086,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.002606073161587119,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.0026867100968956947,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.002724261023104191,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495176-C-T',\n",
       "      'distance': 0.0027571835089474916,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488452-C-T',\n",
       "      'distance': 0.002795818727463484,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430676-C-T',\n",
       "      'distance': 0.0028206338174641132,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-T',\n",
       "      'distance': 0.0028605780098587275,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468474-C-T',\n",
       "      'distance': 0.002864193171262741,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485379-C-T',\n",
       "      'distance': 0.002943028463050723,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474299-C-T',\n",
       "      'distance': 0.002951106522232294,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.0029703378677368164,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48495173-C-T',\n",
       "      'distance': 0.0029960109386593103,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520733-C-T',\n",
       "      'distance': 0.003007199615240097,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489980-C-T',\n",
       "      'distance': 0.003014847170561552,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.003030134830623865,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.00304041663184762,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428459-C-T',\n",
       "      'distance': 0.003053558524698019,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48489953-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48489953\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48489953',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2980G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48489953C>A',\n",
       "     'hgvs_protein': 'p.Glu994Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48489953C>A\", \"vcf_string\": \"15-48489953-C-A\", \"allele_string\": \"C/A\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"most_severe_consequence\": \"stop_gained\", \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"strand\": -1, \"mutationtaster_pred\": \"A\", \"cds_start\": 2980, \"amino_acids\": \"E/*\", \"cadd_raw\": 9.654639, \"given_ref\": \"C\", \"allele_num\": 1, \"consequence_terms\": [\"stop_gained\"], \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48489953, \"start\": 48489952, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Glu994Ter\", \"location\": \"15:48489953\", \"aa\": \"C\", \"exon\": \"25/66\", \"lof\": \"HC\", \"spliceai\": {\"DP_DL\": -28, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AG\": -3, \"DS_DL\": 0, \"DP_AL\": 20, \"DP_DG\": 15, \"DS_AG\": 0, \"DS_DG\": 0}, \"codons\": \"Gag/Tag\", \"canonical\": 1, \"cadd_phred\": 42, \"mastermind_counts\": \"7|8|8\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.2980G>T\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"A\", \"lof_info\": \"PERCENTILE:0.345868152274838,GERP_DIST:6053.72533015311,BP_DIST:5595,DIST_FROM_LAST_EXON:5206,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"transcript_id\": \"NM_000138.5\", \"cdna_end\": 3296, \"cdna_start\": 3296, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:E994X\", \"phaplo\": 0.99897615427687, \"protein_end\": 994, \"loeuf\": 0.105, \"conservation\": 3.76, \"protein_start\": 994, \"cds_end\": 2980, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\"}], \"end\": 48489953, \"start\": 48489953, \"id\": \"NC_000015.10:g.48489953C>A\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM980726\", \"end\": 48489953, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48489953}, {\"end\": 48489953, \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"var_synonyms\": {\"COSMIC\": [\"COSM4055318\"]}, \"start\": 48489953, \"somatic\": 1, \"phenotype_or_disease\": 1, \"id\": \"COSV57314812\"}], \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.017951454840654454,\n",
       "      'pca_y': 0.09689904933848173},\n",
       "     {'t-sne_x': 21.353254646424514, 't-sne_y': 41.68829863249821},\n",
       "     {'umap_x': -9.518149375915527, 'umap_y': 4.567053318023682}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.0011003660038113594,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488233-C-A',\n",
       "      'distance': 0.002052688505500555,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452676-C-A',\n",
       "      'distance': 0.002500715432688594,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437837-C-A',\n",
       "      'distance': 0.002504676580429077,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.002537601627409458,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.002722474280744791,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.002789228456094861,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515397-C-T',\n",
       "      'distance': 0.00286218267865479,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.0029304688796401024,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.0029879556968808174,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.00302451872266829,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483878-C-A',\n",
       "      'distance': 0.0031948615796864033,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505139-C-A',\n",
       "      'distance': 0.003453270299360156,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.0034874083939939737,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411085-C-A',\n",
       "      'distance': 0.003494037315249443,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.00365958409383893,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483899-G-A',\n",
       "      'distance': 0.004078458063304424,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427630-G-A',\n",
       "      'distance': 0.004278814420104027,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.004283365327864885,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487129-G-A',\n",
       "      'distance': 0.004321525804698467,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48489890-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48489890\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48489890',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3043G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48489890C>G',\n",
       "     'hgvs_protein': 'p.Ala1015Pro',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48489890C>G\", \"seq_region_name\": \"15\", \"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/G\", \"vcf_string\": \"15-48489890-C-G\", \"start\": 48489890, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 1015, \"cds_end\": 3043, \"polyphen_score\": 0.435, \"conservation\": 3.76, \"loeuf\": 0.105, \"protein_end\": 1015, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"mastermind_mmid3\": \"FBN1:A1015P\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_start\": 3359, \"cdna_end\": 3359, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.935, \"am_class\": \"likely_pathogenic\"}, \"eve_score\": 0.262593487745468, \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"G\", \"eve_class\": \"Benign\", \"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3043G>C\", \"cadd_phred\": 24.6, \"mastermind_counts\": \"5|5|5\", \"codons\": \"Gcc/Ccc\", \"canonical\": 1, \"spliceai\": {\"DS_AG\": 0, \"DS_DG\": 0, \"DP_AL\": 11, \"DP_DG\": 35, \"DP_AG\": -5, \"DS_DL\": 0, \"DP_DL\": -39, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0}, \"exon\": \"25/66\", \"blosum62\": -1, \"aa\": \"C\", \"location\": \"15:48489890\", \"polyphen_prediction\": \"benign\", \"clinpred\": 0.937435388565063, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48489890, \"start\": 48489889}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Ala1015Pro\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"cadd_raw\": 4.218743, \"given_ref\": \"C\", \"amino_acids\": \"A/P\", \"cds_start\": 3043, \"mutationtaster_pred\": \"D\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\"}], \"end\": 48489890, \"minimised\": 1, \"colocated_variants\": [{\"start\": 48489890, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"end\": 48489890, \"phenotype_or_disease\": 1, \"id\": \"CM062715\"}, {\"phenotype_or_disease\": 1, \"id\": \"CM118911\", \"end\": 48489890, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48489890}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48489890C>G\"}',\n",
       "    'coordinates': [{'pca_x': 0.03284203849699413,\n",
       "      'pca_y': 0.07830397287453786},\n",
       "     {'t-sne_x': 20.52189286194806, 't-sne_y': -6.992447116032973},\n",
       "     {'umap_x': 8.46282958984375, 'umap_y': 7.242745876312256}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.001623291987925768,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415708-C-G',\n",
       "      'distance': 0.0024359440430998802,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487333-G-C',\n",
       "      'distance': 0.002491349820047617,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.002543875016272068,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.0026008067652583122,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.002639174461364746,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.002760025206953287,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.0028044814243912697,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.0028097520116716623,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495162-C-G',\n",
       "      'distance': 0.002845446579158306,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492562-G-C',\n",
       "      'distance': 0.0028583346866071224,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.003101399401202798,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520708-C-G',\n",
       "      'distance': 0.0031077731400728226,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428356-G-C',\n",
       "      'distance': 0.0031963340006768703,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.0032751443795859814,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.0032868555281311274,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0033118845894932747,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489896-C-T',\n",
       "      'distance': 0.003375825472176075,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.003386799246072769,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437347-G-C',\n",
       "      'distance': 0.003387885168194771,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48488493-T-A',\n",
       "    'metadata': {'vcf_string': '15\\t48488493\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488493',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3083A>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488493T>A',\n",
       "     'hgvs_protein': 'p.Asp1028Val',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"T/A\", \"vcf_string\": \"15-48488493-T-A\", \"input\": \"NC_000015.10:g.48488493T>A\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"end\": 48488493, \"start\": 48488493, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"id\": \"CM014706\"}, {\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48488493, \"end\": 48488493, \"id\": \"CM074821\", \"phenotype_or_disease\": 1}, {\"end\": 48488493, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV001984309\", \"VCV001493355\", \"RCV003444556\", \"RCV001281013\", \"RCV001248380\", \"RCV000494510\", \"VCV000429319\"]}, \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"strand\": 1, \"allele_string\": \"T/A/C\", \"start\": 48488493, \"clin_sig_allele\": \"C:likely_pathogenic;A:pathogenic\", \"pubmed\": [17657824], \"id\": \"rs1131691317\", \"phenotype_or_disease\": 1}], \"id\": \"NC_000015.10:g.48488493T>A\", \"start\": 48488493, \"most_severe_consequence\": \"missense_variant\", \"end\": 48488493, \"transcript_consequences\": [{\"polyphen_score\": 0.967, \"cds_end\": 3083, \"protein_start\": 1028, \"sift_prediction\": \"deleterious_low_confidence\", \"maxentscan_alt\": 5.513, \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\", \"sift_score\": 0.01, \"phaplo\": 0.99897615427687, \"protein_end\": 1028, \"loeuf\": 0.105, \"conservation\": 3.95, \"cdna_end\": 3399, \"cdna_start\": 3399, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"rf_score\": 0.662, \"mastermind_mmid3\": \"FBN1:D1028V,FBN1:D1028sra\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9969}, \"transcript_id\": \"NM_000138.5\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"A\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"aa\": \"T\", \"exon\": \"26/66\", \"blosum62\": -3, \"spliceai\": {\"DP_AG\": 18, \"DS_DL\": 0, \"DS_AL\": 0.02, \"DP_DL\": -41, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DG\": 0, \"DP_DG\": 0, \"DP_AL\": 0}, \"codons\": \"gAt/gTt\", \"canonical\": 1, \"maxentscan_ref\": 6.86, \"mastermind_counts\": \"9|11|15\", \"cadd_phred\": 33, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3083A>T\", \"given_ref\": \"T\", \"cadd_raw\": 5.792213, \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"ga4gh_vrs\": {\"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48488493, \"start\": 48488492, \"type\": \"SimpleInterval\"}}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Asp1028Val\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.997650563716888, \"location\": \"15:48488493\", \"maxentscan_diff\": 1.347, \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"mutationtaster_pred\": \"D\", \"strand\": -1, \"cds_start\": 3083, \"amino_acids\": \"D/V\", \"ada_score\": 0.977326488465211}]}',\n",
       "    'coordinates': [{'pca_x': -0.03793474480187773,\n",
       "      'pca_y': 0.05297359468444895},\n",
       "     {'t-sne_x': -12.52394309677042, 't-sne_y': -3.855488099272111},\n",
       "     {'umap_x': 0.2785949110984802, 'umap_y': 15.11567497253418}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.004925518296658993,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.00499038677662611,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.005618035793304443,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.005628096871078014,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.005647385027259588,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.005843047052621841,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.006220497190952301,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.006446711719036102,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.006446969695389271,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.006492847576737404,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.0066713979467749596,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.006896878592669964,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.007051754277199507,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.007143566384911537,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.007589076180011034,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.007602487690746784,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.008335469290614128,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.008789505809545517,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.009424075484275818,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487075-C-T',\n",
       "      'distance': 0.010275734588503838,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48489850-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48489850\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48489850',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3082+1G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48489850C>A',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48489850C>A\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"C/A\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48489850-C-A\", \"start\": 48489850, \"end\": 48489850, \"most_severe_consequence\": \"splice_donor_variant\", \"transcript_consequences\": [{\"intron\": \"25/65\", \"mastermind_mmid3\": \"FBN1:D1028sd\", \"rf_score\": 0.916, \"mane\": [\"MANE_Select\"], \"transcript_id\": \"NM_000138.5\", \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"maxentscan_alt\": -1.679, \"phaplo\": 0.99897615427687, \"conservation\": 3.95, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"start\": 48489849, \"end\": 48489850, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"consequence_terms\": [\"splice_donor_variant\"], \"allele_num\": 1, \"cadd_raw\": 6.413517, \"given_ref\": \"C\", \"maxentscan_diff\": 8.504, \"location\": \"15:48489850\", \"genesplicer\": \"loss/donor/48489850-48489851/Medium/5.093927\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"ada_score\": 0.999985532734767, \"gene_symbol_source\": \"EntrezGene\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"A\", \"lof_info\": \"INTRON_SIZE:1357\", \"spliceai\": {\"DP_AL\": -41, \"DP_DG\": -6, \"DS_DG\": 0.29, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 1, \"DS_AL\": 0, \"DS_DL\": 0.99, \"DP_AG\": -6}, \"lof\": \"HC\", \"aa\": \"C\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3082+1G>T\", \"mastermind_counts\": \"3|3|4\", \"cadd_phred\": 34, \"maxentscan_ref\": 6.825, \"canonical\": 1}], \"minimised\": 1, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CS011042\", \"end\": 48489850, \"start\": 48489850, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\"}], \"id\": \"NC_000015.10:g.48489850C>A\"}',\n",
       "    'coordinates': [{'pca_x': -0.24570612850714846,\n",
       "      'pca_y': -0.026566895588922355},\n",
       "     {'t-sne_x': -16.195791424633047, 't-sne_y': -28.416037803716375},\n",
       "     {'umap_x': 8.17761516571045, 'umap_y': 12.914263725280762}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48465789-C-A',\n",
       "      'distance': 0.0011372696608304977,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-A',\n",
       "      'distance': 0.001326738391071558,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-A',\n",
       "      'distance': 0.0016594817861914635,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508581-C-A',\n",
       "      'distance': 0.0018379053799435496,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.0031337651889771223,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437320-A-C',\n",
       "      'distance': 0.0034110182896256447,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445375-C-A',\n",
       "      'distance': 0.0034530183766037226,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.0038590207695961,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.004238510970026255,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.0042676483280956745,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474254-C-T',\n",
       "      'distance': 0.004390106536448002,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.004676768556237221,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-T',\n",
       "      'distance': 0.004816566593945026,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496099-C-T',\n",
       "      'distance': 0.004881967790424824,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497265-C-T',\n",
       "      'distance': 0.005259486380964518,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428345-C-T',\n",
       "      'distance': 0.005286103114485741,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.005354580469429493,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.0053633833304047585,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488112-C-T',\n",
       "      'distance': 0.005364258773624897,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.005439053289592266,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488449-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48488449\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488449',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3127A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488449T>C',\n",
       "     'hgvs_protein': 'p.Lys1043Glu',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48488449T>C\", \"vcf_string\": \"15-48488449-T-C\", \"allele_string\": \"T/C\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"phaplo\": 0.99897615427687, \"sift_score\": 0, \"protein_end\": 1043, \"conservation\": 3.95, \"loeuf\": 0.105, \"polyphen_score\": 0.988, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 1043, \"cds_end\": 3127, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"T\", \"eve_score\": 0.255439814949189, \"alphamissense\": {\"am_pathogenicity\": 0.5176, \"am_class\": \"ambiguous\"}, \"transcript_id\": \"NM_000138.5\", \"cdna_end\": 3443, \"cdna_start\": 3443, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:K1043E\", \"exon\": \"26/66\", \"blosum62\": 1, \"aa\": \"T\", \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": 3, \"SYMBOL\": \"FBN1\", \"DP_AG\": 44, \"DS_DL\": 0, \"DP_DG\": -34, \"DP_AL\": 27, \"DS_AG\": 0.01, \"DS_DG\": 0.01}, \"mastermind_counts\": \"4|4|4\", \"cadd_phred\": 26.4, \"canonical\": 1, \"codons\": \"Aag/Gag\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3127A>G\", \"eve_class\": \"Benign\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"C\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"cds_start\": 3127, \"mutationtaster_pred\": \"D\", \"strand\": -1, \"amino_acids\": \"K/E\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"cadd_raw\": 4.712131, \"given_ref\": \"T\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48488448, \"end\": 48488449}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Lys1043Glu\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.869398951530457, \"location\": \"15:48488449\"}], \"end\": 48488449, \"start\": 48488449, \"id\": \"NC_000015.10:g.48488449T>C\", \"variant_class\": \"SNV\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.033382472440385114,\n",
       "      'pca_y': 0.08162233702582458},\n",
       "     {'t-sne_x': 2.0165299776648893, 't-sne_y': 2.3981206471274024},\n",
       "     {'umap_x': 4.870207786560059, 'umap_y': 2.7586283683776855}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487107-T-C',\n",
       "      'distance': 0.0017327787354588509,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.001846876461058855,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481687-T-C',\n",
       "      'distance': 0.0018731496529653668,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.002153792418539524,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487131-T-C',\n",
       "      'distance': 0.0022921180352568626,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.002329053822904825,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497298-T-C',\n",
       "      'distance': 0.00237026484683156,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0024263043887913227,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488148-T-C',\n",
       "      'distance': 0.0025148612912744284,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.002524314448237419,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430727-T-C',\n",
       "      'distance': 0.0025347378104925156,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.0025444121565669775,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497322-T-C',\n",
       "      'distance': 0.0025679785758256912,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.0025784887839108706,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472607-T-C',\n",
       "      'distance': 0.0025925487279891968,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415707-C-T',\n",
       "      'distance': 0.0026057360228151083,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492574-C-T',\n",
       "      'distance': 0.002622157335281372,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487317-C-T',\n",
       "      'distance': 0.0026494809426367283,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497343-C-T',\n",
       "      'distance': 0.0026518190279603004,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.002655122196301818,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488445-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48488445\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488445',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3131G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488445C>T',\n",
       "     'hgvs_protein': 'p.Cys1044Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"start\": 48488445, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48488445, \"id\": \"CM040035\", \"phenotype_or_disease\": 1}, {\"start\": 48488445, \"var_synonyms\": {\"COSMIC\": [\"COSM4055316\"]}, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"end\": 48488445, \"id\": \"COSV57332471\", \"phenotype_or_disease\": 1, \"somatic\": 1}, {\"phenotype_or_disease\": 1, \"id\": \"rs730880100\", \"pubmed\": [14695540, 17701892], \"clin_sig_allele\": \"A:likely_pathogenic;T:pathogenic;T:likely_pathogenic\", \"start\": 48488445, \"allele_string\": \"C/A/T\", \"strand\": 1, \"var_synonyms\": {\"UniProt\": [\"VAR_017997\"], \"ClinVar\": [\"RCV000157226\", \"VCV000180353\", \"RCV002515046\", \"RCV000520265\", \"VCV000449897\"]}, \"seq_region_name\": \"15\", \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"end\": 48488445}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48488445C>T\", \"minimised\": 1, \"transcript_consequences\": [{\"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48488445\", \"clinpred\": 0.998602330904887, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"location\": {\"interval\": {\"end\": 48488445, \"start\": 48488444, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"hgvsp\": \"NP_000129.3:p.Cys1044Tyr\", \"given_ref\": \"C\", \"cadd_raw\": 5.218393, \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"amino_acids\": \"C/Y\", \"mutationtaster_pred\": \"A\", \"strand\": -1, \"cds_start\": 3131, \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"T\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3131G>A\", \"codons\": \"tGc/tAc\", \"canonical\": 1, \"cadd_phred\": 29.3, \"mastermind_counts\": \"11|17|17\", \"spliceai\": {\"DS_AG\": 0, \"DS_DG\": 0, \"DP_AL\": 49, \"DP_DG\": 7, \"DP_AG\": 48, \"DS_DL\": 0, \"DP_DL\": -30, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0}, \"aa\": \"C\", \"blosum62\": -2, \"exon\": \"26/66\", \"mastermind_mmid3\": \"FBN1:C1044Y\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_end\": 3447, \"cdna_start\": 3447, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9978}, \"eve_score\": 0.791469042319403, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 3131, \"protein_start\": 1044, \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.996, \"loeuf\": 0.105, \"conservation\": 3.95, \"protein_end\": 1044, \"phaplo\": 0.99897615427687, \"sift_score\": 0}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48488445, \"start\": 48488445, \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48488445-C-T\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48488445C>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.03543382553220059,\n",
       "      'pca_y': 0.0808782356220693},\n",
       "     {'t-sne_x': 7.643123552919436, 't-sne_y': 6.734471053918076},\n",
       "     {'umap_x': 3.277925968170166, 'umap_y': 2.2074363231658936}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48492574-C-T',\n",
       "      'distance': 0.0006986658554524183,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0007591366884298623,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.0008440962992608547,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.0008463336853310466,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.0009130837861448526,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.0009398382971994579,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487317-C-T',\n",
       "      'distance': 0.0010040078777819872,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.0010046906536445022,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497343-C-T',\n",
       "      'distance': 0.0010415122378617525,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497358-C-T',\n",
       "      'distance': 0.001049481681548059,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.0010562047827988863,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.0010834763525053859,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-T',\n",
       "      'distance': 0.0011094033252447844,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496144-C-T',\n",
       "      'distance': 0.0011116393143311143,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0011516902595758438,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0011551224160939455,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.0011551245115697384,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496213-C-T',\n",
       "      'distance': 0.0011616309639066458,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.0011682934127748013,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481699-C-T',\n",
       "      'distance': 0.0012159517500549555,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488384-TT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48488384\\t.\\tTT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488384',\n",
       "     'ref_allele': 'TT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3192del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488385del',\n",
       "     'hgvs_protein': 'p.Glu1065LysfsTer23',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"allele_string\": \"T/-\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48488384-TT-T\", \"input\": \"NC_000015.10:g.48488385del\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"deletion\", \"colocated_variants\": [{\"id\": \"rs1131692050\", \"phenotype_or_disease\": 1, \"pubmed\": [9101298], \"strand\": 1, \"allele_string\": \"TT/T\", \"var_synonyms\": {\"ClinVar\": [\"RCV000017911\", \"VCV000016448\"], \"OMIM\": [134797.0029]}, \"seq_region_name\": \"15\", \"clin_sig\": [\"pathogenic\"], \"start\": 48488384, \"end\": 48488385}], \"id\": \"NC_000015.10:g.48488385del\", \"start\": 48488385, \"end\": 48488385, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"aa\": \"T\", \"exon\": \"26/66\", \"lof\": \"HC\", \"spliceai\": {\"DS_AG\": 0, \"DS_DG\": 0.03, \"DP_DG\": 31, \"DP_AL\": 30, \"DP_AG\": -16, \"DS_DL\": 0.07, \"DS_AL\": 0, \"DP_DL\": -16, \"SYMBOL\": \"FBN1\"}, \"codons\": \"gaA/ga\", \"canonical\": 1, \"hgvsc\": \"NM_000138.5:c.3192del\", \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"-\", \"lof_info\": \"PERCENTILE:0.370473537604457,GERP_DIST:6053.72533015311,BP_DIST:5385,DIST_FROM_LAST_EXON:4996,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"cds_start\": 3192, \"amino_acids\": \"E/X\", \"given_ref\": \"T\", \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48488384, \"end\": 48488385}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Glu1065LysfsTer23\", \"hgvs_offset\": -1, \"location\": \"15:48488384\", \"phaplo\": 0.99897615427687, \"protein_end\": 1064, \"loeuf\": 0.105, \"conservation\": 3.95, \"protein_start\": 1064, \"cds_end\": 3192, \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\", \"transcript_id\": \"NM_000138.5\", \"cdna_start\": 3508, \"cdna_end\": 3508, \"mane\": [\"MANE_Select\"]}]}',\n",
       "    'coordinates': [{'pca_x': 0.003952478221043373,\n",
       "      'pca_y': 0.019754331491357656},\n",
       "     {'t-sne_x': 7.060394407312921, 't-sne_y': 3.028648928127204},\n",
       "     {'umap_x': 4.272100925445557, 'umap_y': 3.435042142868042}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.09580332040786743,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09752510488033295,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.09756822884082794,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09850536286830902,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.09899464249610901,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.0990951806306839,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.0992870181798935,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437069-C-T',\n",
       "      'distance': 0.0995335578918457,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.09958577901124954,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.10011506080627441,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446814-C-T',\n",
       "      'distance': 0.10014287382364273,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492578-C-T',\n",
       "      'distance': 0.10030794143676758,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421981-C-T',\n",
       "      'distance': 0.10031405091285706,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.10042308270931244,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.10048751533031464,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.10056804120540619,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495162-C-T',\n",
       "      'distance': 0.10060572624206543,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.10062401741743088,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448851-C-T',\n",
       "      'distance': 0.10071791708469391,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.1007915586233139,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48488372-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48488372\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488372',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3204C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488372G>C',\n",
       "     'hgvs_protein': 'p.Cys1068Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48488372G>C\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48488372-G-C\", \"allele_string\": \"G/C\", \"assembly_name\": \"GRCh38\", \"start\": 48488372, \"end\": 48488372, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"mastermind_mmid3\": \"FBN1:C1068W\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_end\": 3520, \"cdna_start\": 3520, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.9995, \"am_class\": \"likely_pathogenic\"}, \"eve_score\": 0.760069501514769, \"used_ref\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 1068, \"cds_end\": 3204, \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.98, \"conservation\": 3.09, \"loeuf\": 0.105, \"protein_end\": 1068, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.999875783920288, \"location\": \"15:48488372\", \"hgvsp\": \"NP_000129.3:p.Cys1068Trp\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48488372, \"start\": 48488371, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"cadd_raw\": 4.388932, \"given_ref\": \"G\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"amino_acids\": \"C/W\", \"mutationtaster_pred\": \"D\", \"strand\": -1, \"cds_start\": 3204, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"C\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3204C>G\", \"codons\": \"tgC/tgG\", \"canonical\": 1, \"mastermind_counts\": \"5|5|5\", \"cadd_phred\": 25.1, \"spliceai\": {\"DS_AG\": 0, \"DS_DG\": 0.01, \"DP_AL\": 47, \"DP_DG\": 43, \"DP_AG\": -4, \"DS_DL\": 0.01, \"DP_DL\": -4, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0}, \"aa\": \"G\", \"exon\": \"26/66\", \"blosum62\": -2}], \"minimised\": 1, \"id\": \"NC_000015.10:g.48488372G>C\", \"colocated_variants\": [{\"end\": 48488372, \"start\": 48488372, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CP044516\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.04035730503692034,\n",
       "      'pca_y': 0.08512346145652108},\n",
       "     {'t-sne_x': 19.182398673655513, 't-sne_y': -7.834960771658369},\n",
       "     {'umap_x': 8.31930923461914, 'umap_y': 7.342785358428955}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.001217156182974577,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.0013286066241562366,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468455-G-C',\n",
       "      'distance': 0.001482897438108921,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.0018309857696294785,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492562-G-C',\n",
       "      'distance': 0.00223877327516675,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48487333-G-C',\n",
       "      'distance': 0.0023887422867119312,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.002514813095331192,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.0026231214869767427,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.0027377058286219835,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428356-G-C',\n",
       "      'distance': 0.002759702503681183,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48437784-G-C',\n",
       "      'distance': 0.0027847730088979006,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520708-C-G',\n",
       "      'distance': 0.0027932277880609035,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495162-C-G',\n",
       "      'distance': 0.002862198743969202,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.002865210175514221,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415708-C-G',\n",
       "      'distance': 0.002878268714994192,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.002996657742187381,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48467996-G-C',\n",
       "      'distance': 0.0031421068124473095,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437064-G-C',\n",
       "      'distance': 0.0032196366228163242,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.003239850979298353,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.003260442754253745,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48487311-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48487311\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487311',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3463+1G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487311C>T',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\", \"vcf_string\": \"15-48487311-C-T\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48487311C>T\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48487311, \"end\": 48487311, \"id\": \"CS1612730\", \"phenotype_or_disease\": 1}, {\"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48487311, \"end\": 48487311, \"phenotype_or_disease\": 1, \"id\": \"CS982191\"}, {\"id\": \"rs397515792\", \"phenotype_or_disease\": 1, \"pubmed\": [24033266, 17657824, 9452033], \"clin_sig_allele\": \"A:pathogenic;T:pathogenic\", \"strand\": 1, \"allele_string\": \"C/A/T\", \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV001038405\", \"VCV000837133\", \"RCV000035173\", \"VCV000042336\"]}, \"start\": 48487311, \"end\": 48487311}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48487311C>T\", \"minimised\": 1, \"end\": 48487311, \"most_severe_consequence\": \"splice_donor_variant\", \"transcript_consequences\": [{\"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48487310, \"end\": 48487311}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"given_ref\": \"C\", \"cadd_raw\": 6.296284, \"consequence_terms\": [\"splice_donor_variant\"], \"allele_num\": 1, \"maxentscan_diff\": 8.182, \"location\": \"15:48487311\", \"strand\": -1, \"genesplicer\": \"loss/donor/48487311-48487312/Medium/5.945611\", \"impact\": \"HIGH\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"ada_score\": 0.999985532734767, \"gene_symbol_source\": \"EntrezGene\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"lof_info\": \"INTRON_SIZE:111\", \"variant_allele\": \"T\", \"lof\": \"HC\", \"spliceai\": {\"DP_AG\": -30, \"DS_DL\": 1, \"DP_DL\": 1, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_AG\": 0, \"DS_DG\": 0.85, \"DP_AL\": 28, \"DP_DG\": -33}, \"aa\": \"C\", \"hgvsc\": \"NM_000138.5:c.3463+1G>A\", \"biotype\": \"protein_coding\", \"maxentscan_ref\": 10.481, \"canonical\": 1, \"cadd_phred\": 34, \"mastermind_counts\": \"7|7|9\", \"intron\": \"28/65\", \"rf_score\": 0.934, \"mastermind_mmid3\": \"FBN1:D1155sd\", \"mane\": [\"MANE_Select\"], \"transcript_id\": \"NM_000138.5\", \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"maxentscan_alt\": 2.299, \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"conservation\": 3.85}], \"start\": 48487311}',\n",
       "    'coordinates': [{'pca_x': -0.24526190183064345,\n",
       "      'pca_y': -0.03064982098506194},\n",
       "     {'t-sne_x': -17.251659123654992, 't-sne_y': -28.21412679089851},\n",
       "     {'umap_x': 7.760351657867432, 'umap_y': 12.670188903808594}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.0010037918109446764,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.0014493056805804372,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428345-C-T',\n",
       "      'distance': 0.0015913023380562663,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488112-C-T',\n",
       "      'distance': 0.0017263054614886642,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.0017668323125690222,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463898-C-T',\n",
       "      'distance': 0.0017881345702335238,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496099-C-T',\n",
       "      'distance': 0.002016060519963503,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.0020168405026197433,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474254-C-T',\n",
       "      'distance': 0.002073791576549411,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.0020882501266896725,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.002120149787515402,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.002141986507922411,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497265-C-T',\n",
       "      'distance': 0.0022678265813738108,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.0024483725428581238,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.0024730809964239597,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441720-C-T',\n",
       "      'distance': 0.0025803116150200367,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465789-C-A',\n",
       "      'distance': 0.002666609361767769,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444540-C-T',\n",
       "      'distance': 0.002758147194981575,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613009-C-A',\n",
       "      'distance': 0.0030151698738336563,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-A',\n",
       "      'distance': 0.003547191619873047,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48487201-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48487201\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487201',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3464-1G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487201C>T',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48487201C>T\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48487201-C-T\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\", \"start\": 48487201, \"most_severe_consequence\": \"splice_acceptor_variant\", \"transcript_consequences\": [{\"loeuf\": 0.105, \"conservation\": 3.95, \"phaplo\": 0.99897615427687, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"maxentscan_alt\": 1.476, \"transcript_id\": \"NM_000138.5\", \"rf_score\": 0.938, \"mastermind_mmid3\": \"FBN1:D1155sa\", \"mane\": [\"MANE_Select\"], \"intron\": \"28/65\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3464-1G>A\", \"maxentscan_ref\": 10.227, \"canonical\": 1, \"cadd_phred\": 27.8, \"mastermind_counts\": \"6|6|7\", \"lof\": \"HC\", \"spliceai\": {\"DP_AL\": -1, \"DP_DG\": -3, \"DS_DG\": 0, \"DS_AG\": 0.03, \"SYMBOL\": \"FBN1\", \"DP_DL\": -12, \"DS_AL\": 1, \"DS_DL\": 0, \"DP_AG\": -18}, \"aa\": \"C\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"T\", \"lof_info\": \"INTRON_SIZE:111\", \"gene_symbol_source\": \"EntrezGene\", \"ada_score\": 0.999990361384395, \"strand\": -1, \"genesplicer\": \"loss/acceptor/48487200-48487201/Medium/11.200271\", \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"maxentscan_diff\": 8.75, \"location\": \"15:48487201\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48487201, \"start\": 48487200}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"type\": \"Allele\"}, \"given_ref\": \"C\", \"cadd_raw\": 4.975838, \"allele_num\": 1, \"consequence_terms\": [\"splice_acceptor_variant\"]}], \"end\": 48487201, \"minimised\": 1, \"id\": \"NC_000015.10:g.48487201C>T\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"start\": 48487201, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48487201, \"id\": \"CS098574\", \"phenotype_or_disease\": 1}, {\"start\": 48487201, \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV001993165\", \"VCV001451239\"]}, \"allele_string\": \"C/T\", \"strand\": 1, \"end\": 48487201, \"pubmed\": [17657824], \"phenotype_or_disease\": 1, \"id\": \"rs2141293336\", \"clin_sig_allele\": \"T:pathogenic\"}]}',\n",
       "    'coordinates': [{'pca_x': -0.2451526271765423,\n",
       "      'pca_y': -0.03483915761287683},\n",
       "     {'t-sne_x': -28.947214408847376, 't-sne_y': -18.500116616687716},\n",
       "     {'umap_x': 11.578680038452148, 'umap_y': 19.015769958496094}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.001033805776387453,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.0010683834552764893,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.001181487925350666,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.0012807573657482862,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.0020785070955753326,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.0021519733127206564,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.002252880483865738,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.002476376947015524,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.002494946587830782,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.0025606374256312847,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.0026486204005777836,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.002810864243656397,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.0028135348111391068,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.002865083049982786,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.003581494791433215,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.0038859625346958637,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470757-C-T',\n",
       "      'distance': 0.004469284787774086,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-A',\n",
       "      'distance': 0.004935843870043755,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446823-C-G',\n",
       "      'distance': 0.00497910100966692,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.0057283444330096245,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48487151-G-C',\n",
       "    'metadata': {'vcf_string': '15\\t48487151\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487151',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3513C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487151G>C',\n",
       "     'hgvs_protein': 'p.Cys1171Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"cadd_raw\": 2.815204, \"given_ref\": \"G\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48487150, \"end\": 48487151}}}, \"hgvsp\": \"NP_000129.3:p.Cys1171Trp\", \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48487151\", \"clinpred\": 0.999895215034485, \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"cds_start\": 3513, \"strand\": -1, \"mutationtaster_pred\": \"A\", \"amino_acids\": \"C/W\", \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"C\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"blosum62\": -2, \"exon\": \"29/66\", \"aa\": \"G\", \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_DG\": 38, \"DP_AL\": -14, \"DS_DL\": 0, \"DP_AG\": 46, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -17}, \"cadd_phred\": 20.4, \"mastermind_counts\": \"8|11|11\", \"canonical\": 1, \"codons\": \"tgC/tgG\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3513C>G\", \"cdna_end\": 3829, \"cdna_start\": 3829, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C1171W\", \"eve_score\": 0.692664377384527, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9986}, \"transcript_id\": \"NM_000138.5\", \"polyphen_score\": 0.974, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 1171, \"cds_end\": 3513, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"G\", \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"protein_end\": 1171, \"conservation\": -7.89, \"loeuf\": 0.105}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48487151, \"start\": 48487151, \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48487151, \"end\": 48487151, \"phenotype_or_disease\": 1, \"id\": \"CM095637\"}, {\"end\": 48487151, \"start\": 48487151, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM970512\"}, {\"end\": 48487151, \"allele_string\": \"G/A/C/T\", \"strand\": 1, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000586423\", \"RCV000584722\", \"VCV000492828\", \"RCV000181489\", \"VCV000200019\", \"RCV004003784\", \"RCV000527758\", \"VCV000457195\", \"RCV001186667\"], \"UniProt\": [\"VAR_002319\"]}, \"clin_sig\": [\"likely_benign\", \"likely_pathogenic\", \"pathogenic\"], \"start\": 48487151, \"clin_sig_allele\": \"A:likely_benign;T:pathogenic;C:pathogenic;C:likely_pathogenic\", \"phenotype_or_disease\": 1, \"id\": \"rs775417975\", \"pubmed\": [11175294, 9399842]}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48487151G>C\", \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48487151G>C\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"G/C\", \"vcf_string\": \"15-48487151-G-C\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.039540244052118256,\n",
       "      'pca_y': 0.0826942316142101},\n",
       "     {'t-sne_x': 19.152041053701286, 't-sne_y': -7.829916712382125},\n",
       "     {'umap_x': 8.279695510864258, 'umap_y': 7.324182033538818}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.0008726860396564007,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.0009598543401807547,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.0012961305910721421,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468455-G-C',\n",
       "      'distance': 0.001392457983456552,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437784-G-C',\n",
       "      'distance': 0.002362303202971816,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492562-G-C',\n",
       "      'distance': 0.0024179616011679173,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.002557232044637203,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.002560310997068882,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0026138084940612316,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.002621478633955121,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437064-G-C',\n",
       "      'distance': 0.0028102919459342957,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0028128784615546465,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520708-C-G',\n",
       "      'distance': 0.002897768747061491,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487333-G-C',\n",
       "      'distance': 0.002956502605229616,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.0029750377871096134,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.002976624993607402,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.0029777814634144306,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.0029854581225663424,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437347-G-C',\n",
       "      'distance': 0.0029929415322840214,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428356-G-C',\n",
       "      'distance': 0.003015555441379547,\n",
       "      'pathogenicity': 'benign_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48483895-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48483895\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48483895',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3761G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48483895C>G',\n",
       "     'hgvs_protein': 'p.Cys1254Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48483895, \"end\": 48483895, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9994}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.78027183268008, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C1254S\", \"cdna_end\": 4077, \"cdna_start\": 4077, \"lrt_pred\": \"invalid_field\", \"loeuf\": 0.105, \"conservation\": 3.95, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"protein_end\": 1254, \"cds_end\": 3761, \"protein_start\": 1254, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.944, \"amino_acids\": \"C/S\", \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"D\", \"strand\": -1, \"cds_start\": 3761, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48483895\", \"clinpred\": 0.999958753585815, \"given_ref\": \"C\", \"cadd_raw\": 5.2609, \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"hgvsp\": \"NP_000129.3:p.Cys1254Ser\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48483894, \"end\": 48483895, \"type\": \"SimpleInterval\"}}, \"type\": \"Allele\"}, \"codons\": \"tGc/tCc\", \"canonical\": 1, \"cadd_phred\": 29.6, \"mastermind_counts\": \"2|3|6\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3761G>C\", \"aa\": \"C\", \"blosum62\": -1, \"exon\": \"31/66\", \"spliceai\": {\"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 7, \"DS_DL\": 0, \"DP_AG\": 48, \"DP_DG\": -24, \"DP_AL\": 37, \"DS_DG\": 0, \"DS_AG\": 0}, \"variant_allele\": \"G\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\"}], \"minimised\": 1, \"colocated_variants\": [{\"end\": 48483895, \"start\": 48483895, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM098682\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48483895C>G\", \"input\": \"NC_000015.10:g.48483895C>G\", \"seq_region_name\": \"15\", \"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/G\", \"vcf_string\": \"15-48483895-C-G\"}',\n",
       "    'coordinates': [{'pca_x': 0.04020675892845332,\n",
       "      'pca_y': 0.0810727882741771},\n",
       "     {'t-sne_x': 20.52680090560495, 't-sne_y': -5.209779147932192},\n",
       "     {'umap_x': 8.576754570007324, 'umap_y': 6.9361724853515625}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.0003962198388762772,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.0010223343269899487,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.0010745265753939748,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.001141571905463934,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0018212422728538513,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0020286929793655872,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489991-C-G',\n",
       "      'distance': 0.002133316360414028,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.0022132638841867447,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.002267224481329322,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.0023798616603016853,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.002483244752511382,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.0024931090883910656,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.0025612569879740477,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.0026564528234302998,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.002767540281638503,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.0029715485870838165,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.003005389589816332,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-G',\n",
       "      'distance': 0.003075480693951249,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.003275641705840826,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487333-G-C',\n",
       "      'distance': 0.003286826889961958,\n",
       "      'pathogenicity': 'benign'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48483890-TT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48483890\\t.\\tTT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48483890',\n",
       "     'ref_allele': 'TT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3767del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48483891del',\n",
       "     'hgvs_protein': 'p.Asn1256IlefsTer20',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48483891del\", \"allele_string\": \"T/-\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48483890-TT-T\", \"strand\": 1, \"end\": 48483891, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"canonical\": 1, \"codons\": \"aAt/at\", \"hgvsc\": \"NM_000138.5:c.3767del\", \"biotype\": \"protein_coding\", \"exon\": \"31/66\", \"aa\": \"T\", \"spliceai\": {\"DP_AG\": -20, \"DS_DL\": 0, \"DS_AL\": 0, \"DP_DL\": 12, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DG\": 0, \"DP_DG\": -19, \"DP_AL\": 42}, \"lof\": \"HC\", \"variant_allele\": \"-\", \"lof_info\": \"PERCENTILE:0.437209842154132,GERP_DIST:5782.37535538673,BP_DIST:4816,DIST_FROM_LAST_EXON:4427,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"N/X\", \"impact\": \"HIGH\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"cds_start\": 3767, \"strand\": -1, \"location\": \"15:48483889\", \"hgvs_offset\": -2, \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"given_ref\": \"T\", \"hgvsp\": \"NP_000129.3:p.Asn1256IlefsTer20\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"end\": 48483891, \"start\": 48483890, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}}, \"conservation\": -1.25, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"protein_end\": 1256, \"cds_end\": 3767, \"protein_start\": 1256, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"T\", \"transcript_id\": \"NM_000138.5\", \"mane\": [\"MANE_Select\"], \"cdna_end\": 4083, \"cdna_start\": 4083}], \"start\": 48483891, \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48483891del\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.04469063049974453,\n",
       "      'pca_y': 0.029792368838567805},\n",
       "     {'t-sne_x': 4.135825881336874, 't-sne_y': 4.542674658435152},\n",
       "     {'umap_x': 3.9364147186279297, 'umap_y': 3.0343363285064697}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.10297106206417084,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10320756584405899,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10403946787118912,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10415226966142654,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.10468372702598572,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.10496361553668976,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.10496647655963898,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.10510006546974182,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10515733063220978,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.10519716143608093,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.10542164742946625,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.10553006082773209,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.10569368302822113,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.10583256185054779,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489921-G-T',\n",
       "      'distance': 0.10606080293655396,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.10612928122282028,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.10625199973583221,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.10639487206935883,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48441797-G-T',\n",
       "      'distance': 0.10649632662534714,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.1066591739654541,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48483863-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48483863\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48483863',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3793T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48483863A>G',\n",
       "     'hgvs_protein': 'p.Cys1265Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"polyphen_score\": 0.983, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"A\", \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 1265, \"cds_end\": 3793, \"protein_end\": 1265, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"conservation\": 3.95, \"loeuf\": 0.105, \"lrt_pred\": \"invalid_field\", \"cdna_start\": 4109, \"cdna_end\": 4109, \"mastermind_mmid3\": \"FBN1:C1265R\", \"mane\": [\"MANE_Select\"], \"eve_score\": 0.778970878593191, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9994}, \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"G\", \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": -41, \"DP_DG\": -45, \"DS_DL\": 0, \"DP_AG\": 33, \"SYMBOL\": \"FBN1\", \"DP_DL\": -41, \"DS_AL\": 0}, \"exon\": \"31/66\", \"blosum62\": -3, \"aa\": \"A\", \"hgvsc\": \"NM_000138.5:c.3793T>C\", \"biotype\": \"protein_coding\", \"cadd_phred\": 29.4, \"mastermind_counts\": \"9|13|13\", \"codons\": \"Tgt/Cgt\", \"canonical\": 1, \"hgvsp\": \"NP_000129.3:p.Cys1265Arg\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"start\": 48483862, \"end\": 48483863, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"cadd_raw\": 5.238583, \"given_ref\": \"A\", \"clinpred\": 0.999938368797302, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48483863\", \"cds_start\": 3793, \"strand\": -1, \"mutationtaster_pred\": \"A\", \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"amino_acids\": \"C/R\"}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48483863, \"start\": 48483863, \"id\": \"NC_000015.10:g.48483863A>G\", \"colocated_variants\": [{\"end\": 48483863, \"start\": 48483863, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"id\": \"CM098683\", \"phenotype_or_disease\": 1}, {\"id\": \"CM980729\", \"phenotype_or_disease\": 1, \"start\": 48483863, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"end\": 48483863}, {\"clin_sig_allele\": \"G:pathogenic\", \"id\": \"rs137854474\", \"phenotype_or_disease\": 1, \"pubmed\": [9837823], \"end\": 48483863, \"strand\": 1, \"allele_string\": \"A/G\", \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"UniProt\": [\"VAR_018320\"], \"OMIM\": [134797.0031], \"ClinVar\": [\"RCV000017913\", \"VCV000016450\"]}, \"start\": 48483863}], \"variant_class\": \"SNV\", \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48483863A>G\", \"vcf_string\": \"15-48483863-A-G\", \"allele_string\": \"A/G\", \"assembly_name\": \"GRCh38\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.04765385322625948,\n",
       "      'pca_y': 0.07407894506409013},\n",
       "     {'t-sne_x': 35.71704386033352, 't-sne_y': 3.808856868175411},\n",
       "     {'umap_x': 1.014329433441162, 'umap_y': 6.6890668869018555}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0006018118583597243,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0008171273511834443,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0008767058607190847,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0008996081305667758,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.0009654175955802202,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0009828200563788414,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468418-A-G',\n",
       "      'distance': 0.000997523427940905,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.0010197153314948082,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.0010421210899949074,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.0010452456772327423,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0010554218897596002,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0010823489865288138,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.001096602063626051,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.0011044779093936086,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596373-A-G',\n",
       "      'distance': 0.001116723520681262,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.001174431061372161,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452673-A-G',\n",
       "      'distance': 0.0013230879558250308,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.0013261267449706793,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430770-A-G',\n",
       "      'distance': 0.001352318562567234,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.0013571733143180609,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48481660-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48481660\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48481660',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3959G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48481660C>T',\n",
       "     'hgvs_protein': 'p.Cys1320Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"colocated_variants\": [{\"end\": 48481660, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48481660, \"id\": \"CM1312098\", \"phenotype_or_disease\": 1}, {\"end\": 48481660, \"start\": 48481660, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"id\": \"CM137732\"}], \"id\": \"NC_000015.10:g.48481660C>T\", \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"hgvsc\": \"NM_000138.5:c.3959G>A\", \"biotype\": \"protein_coding\", \"cadd_phred\": 32, \"mastermind_counts\": \"6|6|6\", \"canonical\": 1, \"codons\": \"tGt/tAt\", \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": 1, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": 40, \"DP_AL\": -5, \"DP_DG\": -5, \"DS_DG\": 0, \"DS_AG\": 0}, \"exon\": \"32/66\", \"blosum62\": -2, \"aa\": \"C\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"T\", \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"C/Y\", \"cds_start\": 3959, \"strand\": -1, \"mutationtaster_pred\": \"D\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"location\": \"15:48481660\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.999963521957397, \"hgvsp\": \"NP_000129.3:p.Cys1320Tyr\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48481659, \"end\": 48481660, \"type\": \"SimpleInterval\"}}, \"type\": \"Allele\"}, \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"given_ref\": \"C\", \"cadd_raw\": 5.547335, \"loeuf\": 0.105, \"conservation\": 3.85, \"protein_end\": 1320, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 1320, \"cds_end\": 3959, \"polyphen_score\": 0.994, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9977}, \"eve_score\": 0.808097197417918, \"mastermind_mmid3\": \"FBN1:C1320Y\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_start\": 4275, \"cdna_end\": 4275}], \"end\": 48481660, \"start\": 48481660, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\", \"vcf_string\": \"15-48481660-C-T\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48481660C>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.03848047856133205,\n",
       "      'pca_y': 0.08615053072797998},\n",
       "     {'t-sne_x': 5.396269368221804, 't-sne_y': -7.8786384186948215},\n",
       "     {'umap_x': 5.319488525390625, 'umap_y': 5.033369541168213}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48437032-C-T',\n",
       "      'distance': 0.001273533795028925,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421653-C-T',\n",
       "      'distance': 0.0012764794519171119,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503846-C-T',\n",
       "      'distance': 0.0014929616590961814,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463943-C-T',\n",
       "      'distance': 0.0017177097033709288,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483931-C-T',\n",
       "      'distance': 0.0018945221090689301,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474638-C-T',\n",
       "      'distance': 0.0019657739903777838,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474533-C-T',\n",
       "      'distance': 0.0020074178464710712,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48464009-C-T',\n",
       "      'distance': 0.002021635416895151,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434701-C-T',\n",
       "      'distance': 0.0020751822739839554,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432910-C-T',\n",
       "      'distance': 0.002150998217985034,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-T',\n",
       "      'distance': 0.0021673571318387985,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425419-C-T',\n",
       "      'distance': 0.002196425339207053,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425816-C-T',\n",
       "      'distance': 0.0022102626971900463,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415701-T-C',\n",
       "      'distance': 0.0022523701190948486,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432916-C-T',\n",
       "      'distance': 0.0023451163433492184,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472595-C-T',\n",
       "      'distance': 0.0023825252428650856,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474344-C-T',\n",
       "      'distance': 0.002383897081017494,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437369-C-T',\n",
       "      'distance': 0.0024236023891717196,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503849-C-T',\n",
       "      'distance': 0.0024528461508452892,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474572-C-T',\n",
       "      'distance': 0.002507920376956463,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48474652-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48474652\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474652',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3965-2A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474652T>C',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"maxentscan_alt\": -0.241, \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\", \"conservation\": 3.95, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"rf_score\": 0.928, \"mastermind_mmid3\": \"FBN1:D1322sa\", \"intron\": \"32/65\", \"transcript_id\": \"NM_000138.5\", \"lof_info\": \"INTRON_SIZE:7004\", \"variant_allele\": \"C\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"gene_symbol_source\": \"EntrezGene\", \"canonical\": 1, \"maxentscan_ref\": 7.714, \"mastermind_counts\": \"4|4|9\", \"cadd_phred\": 31, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3965-2A>G\", \"aa\": \"T\", \"lof\": \"HC\", \"spliceai\": {\"DP_AL\": -2, \"DP_DG\": 43, \"DS_AG\": 0.03, \"DS_DG\": 0, \"DP_DL\": -48, \"SYMBOL\": \"FBN1\", \"DS_AL\": 1, \"DP_AG\": 43, \"DS_DL\": 0}, \"location\": \"15:48474652\", \"maxentscan_diff\": 7.955, \"given_ref\": \"T\", \"cadd_raw\": 5.394107, \"consequence_terms\": [\"splice_acceptor_variant\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48474651, \"end\": 48474652}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"ada_score\": 0.999988247793601, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"genesplicer\": \"loss/acceptor/48474652-48474653/Medium/6.752076\"}], \"most_severe_consequence\": \"splice_acceptor_variant\", \"end\": 48474652, \"start\": 48474652, \"id\": \"NC_000015.10:g.48474652T>C\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"end\": 48474652, \"start\": 48474652, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"id\": \"CS055582\", \"phenotype_or_disease\": 1}, {\"id\": \"CS951411\", \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48474652, \"end\": 48474652}], \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48474652T>C\", \"vcf_string\": \"15-48474652-T-C\", \"allele_string\": \"T/C\", \"assembly_name\": \"GRCh38\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.239881804522445,\n",
       "      'pca_y': -0.043452075438226975},\n",
       "     {'t-sne_x': -30.23301511022024, 't-sne_y': -18.19639071683493},\n",
       "     {'umap_x': 11.655366897583008, 'umap_y': 18.938932418823242}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48452686-T-C',\n",
       "      'distance': 0.0008793734014034271,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.002111545531079173,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.0027394071221351624,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.003120227949693799,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.003217678517103195,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.0038551846519112587,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.004039713181555271,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.0044910842552781105,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.004599861800670624,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.00485245930030942,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-A',\n",
       "      'distance': 0.004865679889917374,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.005075039342045784,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.00520029291510582,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470757-C-T',\n",
       "      'distance': 0.005372376646846533,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.005799580365419388,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490079-C-G',\n",
       "      'distance': 0.006188630126416683,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.006216675043106079,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.00644941721111536,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.006994524970650673,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.0076976558193564415,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48474642-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48474642\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474642',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3973G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474642C>G',\n",
       "     'hgvs_protein': 'p.Glu1325Gln',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3973G>C\", \"canonical\": 1, \"codons\": \"Gaa/Caa\", \"cadd_phred\": 28.2, \"mastermind_counts\": \"4|7|7\", \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": -38, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": -3, \"DP_AL\": 5, \"DP_DG\": -3, \"DS_DG\": 0, \"DS_AG\": 0}, \"aa\": \"C\", \"exon\": \"33/66\", \"blosum62\": 2, \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"G\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"amino_acids\": \"E/Q\", \"strand\": -1, \"mutationtaster_pred\": \"D\", \"cds_start\": 3973, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"polyphen_prediction\": \"possibly_damaging\", \"clinpred\": 0.997959017753601, \"location\": \"15:48474642\", \"hgvsp\": \"NP_000129.3:p.Glu1325Gln\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48474642, \"start\": 48474641}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"cadd_raw\": 5.046272, \"given_ref\": \"C\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"conservation\": 3.95, \"loeuf\": 0.105, \"protein_end\": 1325, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 1325, \"cds_end\": 3973, \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.639, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.9793, \"am_class\": \"likely_pathogenic\"}, \"eve_score\": 0.806985546375727, \"mastermind_mmid3\": \"FBN1:E1325Q\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_end\": 4289, \"cdna_start\": 4289}], \"end\": 48474642, \"start\": 48474642, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"id\": \"CM010041\", \"phenotype_or_disease\": 1, \"end\": 48474642, \"start\": 48474642, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\"}, {\"id\": \"CM040036\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48474642, \"end\": 48474642}, {\"end\": 48474642, \"start\": 48474642, \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"seq_region_name\": \"15\", \"var_synonyms\": {\"COSMIC\": [\"COSM9225437\"]}, \"somatic\": 1, \"id\": \"COSV104608679\", \"phenotype_or_disease\": 1}], \"id\": \"NC_000015.10:g.48474642C>G\", \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48474642C>G\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/G\", \"vcf_string\": \"15-48474642-C-G\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.04215526302815534,\n",
       "      'pca_y': 0.08540930819942952},\n",
       "     {'t-sne_x': 6.93346262742288, 't-sne_y': -0.4824204801042194},\n",
       "     {'umap_x': 7.970897674560547, 'umap_y': 6.9192609786987305}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468526-C-T',\n",
       "      'distance': 0.0047567132860422134,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437064-G-C',\n",
       "      'distance': 0.005418848711997271,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432910-C-G',\n",
       "      'distance': 0.00542584341019392,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434627-C-T',\n",
       "      'distance': 0.005433172453194857,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437039-C-T',\n",
       "      'distance': 0.0057158321142196655,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.005806970410048962,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.00591584388166666,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.006008949596434832,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.006030963733792305,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428356-G-C',\n",
       "      'distance': 0.00604373449459672,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48474369-C-T',\n",
       "      'distance': 0.006049930118024349,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465825-C-T',\n",
       "      'distance': 0.006054229568690062,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425483-C-T',\n",
       "      'distance': 0.006153556983917952,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.006224651355296373,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495162-C-G',\n",
       "      'distance': 0.0062301624566316605,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.006257029250264168,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.0062607815489172935,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425823-C-T',\n",
       "      'distance': 0.006287663243710995,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437347-G-C',\n",
       "      'distance': 0.0063365548849105835,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-G',\n",
       "      'distance': 0.006394500844180584,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48474255-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48474255\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474255',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4210G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474255C>A',\n",
       "     'hgvs_protein': 'p.Asp1404Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48474255, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"loeuf\": 0.105, \"conservation\": 3.95, \"protein_end\": 1404, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"maxentscan_alt\": 5.714, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 1404, \"cds_end\": 4210, \"polyphen_score\": 0.998, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.9852, \"am_class\": \"likely_pathogenic\"}, \"mastermind_mmid3\": \"FBN1:D1404Y,FBN1:D1404srd\", \"rf_score\": 0.986, \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_start\": 4526, \"cdna_end\": 4526, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.4210G>T\", \"cadd_phred\": 37, \"mastermind_counts\": \"7|15|21\", \"canonical\": 1, \"maxentscan_ref\": 9.802, \"codons\": \"Gac/Tac\", \"spliceai\": {\"DP_DL\": 0, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AG\": 0, \"DS_DL\": 0.88, \"DP_AL\": 29, \"DP_DG\": 21, \"DS_AG\": 0, \"DS_DG\": 0.84}, \"blosum62\": -3, \"exon\": \"34/66\", \"aa\": \"C\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"A\", \"gene_symbol_source\": \"EntrezGene\", \"ada_score\": 0.999830074688166, \"amino_acids\": \"D/Y\", \"cds_start\": 4210, \"strand\": -1, \"mutationtaster_pred\": \"D\", \"impact\": \"MODERATE\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"maxentscan_diff\": 4.088, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48474255\", \"clinpred\": 0.983426928520203, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48474255, \"start\": 48474254, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}}, \"hgvsp\": \"NP_000129.3:p.Asp1404Tyr\", \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"allele_num\": 1, \"given_ref\": \"C\", \"cadd_raw\": 8.059727}], \"end\": 48474255, \"minimised\": 1, \"id\": \"NC_000015.10:g.48474255C>A\", \"colocated_variants\": [{\"end\": 48474255, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48474255, \"phenotype_or_disease\": 1, \"id\": \"CM154835\"}, {\"phenotype_or_disease\": 1, \"id\": \"CM970514\", \"end\": 48474255, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48474255}], \"variant_class\": \"SNV\", \"input\": \"NC_000015.10:g.48474255C>A\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48474255-C-A\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/A\"}',\n",
       "    'coordinates': [{'pca_x': -0.0407118697454394,\n",
       "      'pca_y': 0.05928422225125254},\n",
       "     {'t-sne_x': -12.436834300926007, 't-sne_y': -3.8127411847258563},\n",
       "     {'umap_x': 0.30058202147483826, 'umap_y': 15.102937698364258}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.0027856382075697184,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.00280563673004508,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.0031270647887140512,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.003625068813562393,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.00455279229208827,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.005113552324473858,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.005197376944124699,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.005411548539996147,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.005663683637976646,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.0058137038722634315,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.005856706295162439,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.0059021105989813805,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487075-C-T',\n",
       "      'distance': 0.00668476615101099,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.006741181947290897,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.006816238164901733,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.007020644843578339,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.007380069233477116,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468534-T-C',\n",
       "      'distance': 0.007430810481309891,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.007524934597313404,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.007916026748716831,\n",
       "      'pathogenicity': 'benign'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48472629-ACTGGCCA-A',\n",
       "    'metadata': {'vcf_string': '15\\t48472629\\t.\\tACTGGCCA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48472629',\n",
       "     'ref_allele': 'ACTGGCCA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4253_4259del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48472630_48472636del',\n",
       "     'hgvs_protein': 'p.Gly1418AlafsTer55',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48472630_48472636del\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"CTGGCCA/-\", \"vcf_string\": \"15-48472629-ACTGGCCA-A\", \"strand\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"end\": 48472636, \"transcript_consequences\": [{\"aa\": \"CTGGCCA\", \"exon\": \"35/66\", \"lof\": \"HC\", \"codons\": \"gGCCAGTGc/gc\", \"canonical\": 1, \"hgvsc\": \"NM_000138.5:c.4253_4259del\", \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"lof_info\": \"PERCENTILE:0.494312906220984,GERP_DIST:5173.0222873956,BP_DIST:4328,DIST_FROM_LAST_EXON:3939,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"impact\": \"HIGH\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"cds_start\": 4253, \"amino_acids\": \"GQC/X\", \"given_ref\": \"CTGGCCA\", \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"hgvsp\": \"NP_000129.3:p.Gly1418AlafsTer55\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48472629, \"end\": 48472636, \"type\": \"SimpleInterval\"}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}}, \"hgvs_offset\": -2, \"location\": \"15:48472628-48472634\", \"phaplo\": 0.99897615427687, \"protein_end\": 1420, \"loeuf\": 0.105, \"conservation\": 1.724, \"cds_end\": 4259, \"protein_start\": 1418, \"used_ref\": \"CACTGGC\", \"mane_select\": \"ENST00000316623.10\", \"transcript_id\": \"NM_000138.5\", \"cdna_end\": 4575, \"cdna_start\": 4569, \"mane\": [\"MANE_Select\"]}], \"start\": 48472630, \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48472630_48472636del\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.04227679743981752,\n",
       "      'pca_y': 0.03802199123904728},\n",
       "     {'t-sne_x': 34.45907343625409, 't-sne_y': 11.36896418293317},\n",
       "     {'umap_x': 1.4058769941329956, 'umap_y': 7.426647186279297}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.1029931902885437,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.10522305220365524,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10585172474384308,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.1068604588508606,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.10706356167793274,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.10753914713859558,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10771655291318893,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10796006768941879,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437912-G-A',\n",
       "      'distance': 0.1083221584558487,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.10838218778371811,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.10844855010509491,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.10853371024131775,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.10869478434324265,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48441797-G-T',\n",
       "      'distance': 0.1092093363404274,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48516225-G-A',\n",
       "      'distance': 0.1092512309551239,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.10947194695472717,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.1094733327627182,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.1095895916223526,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.10982421040534973,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.11004577577114105,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48467965-ins23-ffcf544b',\n",
       "    'metadata': {'vcf_string': '15\\t48467965\\t.\\tA\\tAAGGAGTACCCCAGGCTTTACCCA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48467965',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'AAGGAGTACCCCAGGCTTTACCCA',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4699_4721dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48467966_48467988dup',\n",
       "     'hgvs_protein': 'p.Cys1574TrpfsTer15',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48467966_48467988dup\", \"seq_region_name\": \"15\", \"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"-/AGGAGTACCCCAGGCTTTACCCA\", \"vcf_string\": \"15-48467965-A-AAGGAGTACCCCAGGCTTTACCCA\", \"start\": 48467966, \"end\": 48467965, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"protein_start\": 1574, \"cds_end\": 4722, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"-\", \"phaplo\": 0.99897615427687, \"protein_end\": 1574, \"loeuf\": 0.105, \"conservation\": 0.195, \"cdna_end\": 5038, \"cdna_start\": 5037, \"mane\": [\"MANE_Select\"], \"transcript_id\": \"NM_000138.5\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"CAAGGAGTACCCCAGGCTTTACC\", \"lof_info\": \"PERCENTILE:0.547934076137419,GERP_DIST:4669.65904123262,BP_DIST:3869,DIST_FROM_LAST_EXON:3480,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"exon\": \"38/66\", \"lof\": \"HC\", \"codons\": \"tgt/tgGGTAAAGCCTGGGGTACTCCTTGt\", \"canonical\": 1, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.4699_4721dup\", \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"given_ref\": \"-\", \"hgvsp\": \"NP_000129.3:p.Cys1574TrpfsTer15\", \"location\": \"15:48467963-48467964\", \"hgvs_offset\": -2, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"cds_start\": 4721, \"mutfunc\": {\"mod\": {\"dG_wt\": 81.14262, \"ddG_sd\": 0.001293, \"dG_mt_sd\": 0.00076, \"dG_wt_sd\": 0.001572, \"ddG\": 2.347139, \"dG_mt\": 83.489759}, \"exp\": {\"dG_wt\": 22.6955, \"ddG_sd\": 0.001368, \"dG_mt_sd\": 0.001368, \"ddG\": 1.85738, \"dG_wt_sd\": 0, \"dG_mt\": 24.55288}}, \"strand\": -1, \"amino_acids\": \"C/WVKPGVLLX\"}], \"minimised\": 1, \"variant_class\": \"insertion\", \"id\": \"NC_000015.10:g.48467966_48467988dup\"}',\n",
       "    'coordinates': [{'pca_x': 0.019530987129808136,\n",
       "      'pca_y': 0.01781110117037319},\n",
       "     {'t-sne_x': 7.274552724291325, 't-sne_y': 8.66030780239305},\n",
       "     {'umap_x': 3.8142952919006348, 'umap_y': 2.7075095176696777}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.10035169869661331,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.10123034566640854,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.10170760750770569,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.10187356173992157,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.10211624950170517,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.10224732756614685,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468504-C-T',\n",
       "      'distance': 0.10225869715213776,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427602-C-T',\n",
       "      'distance': 0.1022847443819046,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.10235953330993652,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415707-C-T',\n",
       "      'distance': 0.10236513614654541,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.10242089629173279,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48467997-C-T',\n",
       "      'distance': 0.1025465577840805,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.10258247703313828,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-T',\n",
       "      'distance': 0.10261773318052292,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.10265251994132996,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487317-C-T',\n",
       "      'distance': 0.10289707779884338,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.10290463268756866,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-G',\n",
       "      'distance': 0.10294823348522186,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437069-C-T',\n",
       "      'distance': 0.10306092351675034,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.1032162606716156,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48465683-CT-C',\n",
       "    'metadata': {'vcf_string': '15\\t48465683\\t.\\tCT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48465683',\n",
       "     'ref_allele': 'CT',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4826del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48465684del',\n",
       "     'hgvs_protein': 'p.Glu1609GlyfsTer31',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48465684del\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48465683-CT-C\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"T/-\", \"start\": 48465684, \"transcript_consequences\": [{\"aa\": \"T\", \"exon\": \"40/66\", \"lof\": \"HC\", \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": 10, \"SYMBOL\": \"FBN1\", \"DP_AG\": 10, \"DS_DL\": 0, \"DP_DG\": -37, \"DP_AL\": 36, \"DS_AG\": 0, \"DS_DG\": 0}, \"canonical\": 1, \"codons\": \"gAg/gg\", \"mastermind_counts\": \"4|4|4\", \"hgvsc\": \"NM_000138.5:c.4826del\", \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"-\", \"lof_info\": \"PERCENTILE:0.560120705663881,GERP_DIST:4542.90322713554,BP_DIST:3764,DIST_FROM_LAST_EXON:3375,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"impact\": \"HIGH\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"mutfunc\": {\"exp\": {\"dG_wt_sd\": 0.638785, \"ddG\": 2.08954, \"dG_mt\": 24.10116, \"ddG_sd\": 0.200975, \"dG_mt_sd\": 0.834268, \"dG_wt\": 22.01162}, \"mod\": {\"dG_mt\": 78.5191, \"ddG\": 2.09298, \"dG_wt_sd\": 0.481533, \"dG_wt\": 76.42612, \"dG_mt_sd\": 0.580523, \"ddG_sd\": 0.135096}}, \"cds_start\": 4826, \"amino_acids\": \"E/X\", \"given_ref\": \"T\", \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"hgvsp\": \"NP_000129.3:p.Glu1609GlyfsTer31\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48465684, \"start\": 48465683}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"location\": \"15:48465684\", \"phaplo\": 0.99897615427687, \"protein_end\": 1609, \"loeuf\": 0.105, \"conservation\": 2.81, \"protein_start\": 1609, \"cds_end\": 4826, \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\", \"transcript_id\": \"NM_000138.5\", \"cdna_end\": 5142, \"cdna_start\": 5142, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:E1609fs\"}], \"most_severe_consequence\": \"frameshift_variant\", \"end\": 48465684, \"minimised\": 1, \"id\": \"NC_000015.10:g.48465684del\", \"variant_class\": \"deletion\", \"colocated_variants\": [{\"start\": 48465684, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48465684, \"id\": \"CD991707\", \"phenotype_or_disease\": 1}]}',\n",
       "    'coordinates': [{'pca_x': 0.02094218941696044,\n",
       "      'pca_y': 0.02622316311483492},\n",
       "     {'t-sne_x': 9.640617397926027, 't-sne_y': 15.229981355040932},\n",
       "     {'umap_x': 3.7789833545684814, 'umap_y': 3.4379382133483887}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.12101683020591736,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.12193422019481659,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.12263207137584686,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.1228364109992981,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.12289394438266754,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.1231420636177063,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.12344379723072052,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.12349043786525726,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.1239333301782608,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.12394842505455017,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.12420496344566345,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.12452097237110138,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.124686598777771,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.12487754970788956,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48445419-G-T',\n",
       "      'distance': 0.12498928606510162,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.12511077523231506,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.12520334124565125,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445430-G-A',\n",
       "      'distance': 0.12536615133285522,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.12549637258052826,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.12550944089889526,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48463222-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48463222\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48463222',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5084G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48463222C>T',\n",
       "     'hgvs_protein': 'p.Cys1695Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48463222C>T\", \"vcf_string\": \"15-48463222-C-T\", \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"transcript_consequences\": [{\"loeuf\": 0.105, \"conservation\": 3.95, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 1695, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 1695, \"cds_end\": 5084, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"polyphen_score\": 0.983, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9997}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.796289063305112, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C1695Y\", \"cdna_end\": 5400, \"cdna_start\": 5400, \"lrt_pred\": \"invalid_field\", \"cadd_phred\": 27.9, \"mastermind_counts\": \"4|6|6\", \"codons\": \"tGc/tAc\", \"canonical\": 1, \"hgvsc\": \"NM_000138.5:c.5084G>A\", \"biotype\": \"protein_coding\", \"blosum62\": -2, \"exon\": \"42/66\", \"aa\": \"C\", \"spliceai\": {\"DP_AG\": -7, \"DS_DL\": 0, \"DP_DL\": 5, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_AG\": 0, \"DS_DG\": 0, \"DP_AL\": 5, \"DP_DG\": 7}, \"variant_allele\": \"T\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"C/Y\", \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"cds_start\": 5084, \"strand\": -1, \"mutationtaster_pred\": \"D\", \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48463222\", \"clinpred\": 0.99141354913147, \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"given_ref\": \"C\", \"cadd_raw\": 5.001993, \"hgvsp\": \"NP_000129.3:p.Cys1695Tyr\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48463222, \"start\": 48463221, \"type\": \"SimpleInterval\"}}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48463222, \"start\": 48463222, \"id\": \"NC_000015.10:g.48463222C>T\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM138475\", \"end\": 48463222, \"start\": 48463222, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\"}, {\"somatic\": 1, \"id\": \"COSV57312693\", \"phenotype_or_disease\": 1, \"end\": 48463222, \"start\": 48463222, \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"seq_region_name\": \"15\", \"var_synonyms\": {\"COSMIC\": [\"COSM366174\"]}}, {\"somatic\": 1, \"id\": \"COSV57329613\", \"phenotype_or_disease\": 1, \"end\": 48463222, \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"var_synonyms\": {\"COSMIC\": [\"COSM6077365\"]}, \"seq_region_name\": \"15\", \"start\": 48463222}], \"variant_class\": \"SNV\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.03940579273921857,\n",
       "      'pca_y': 0.09884433153822211},\n",
       "     {'t-sne_x': 6.09810369280161, 't-sne_y': -6.4744619852365375},\n",
       "     {'umap_x': 5.244020938873291, 'umap_y': 4.773061275482178}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48425852-C-T',\n",
       "      'distance': 0.0010660236002877355,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503849-C-T',\n",
       "      'distance': 0.0011476781219244003,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432916-C-T',\n",
       "      'distance': 0.0011579180136322975,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48464009-C-T',\n",
       "      'distance': 0.0011781182838603854,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415632-C-T',\n",
       "      'distance': 0.0012671402655541897,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425446-C-T',\n",
       "      'distance': 0.0013798470608890057,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432955-C-T',\n",
       "      'distance': 0.0014025638811290264,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432910-C-T',\n",
       "      'distance': 0.0014047182630747557,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425816-C-T',\n",
       "      'distance': 0.0014227053616195917,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474344-C-T',\n",
       "      'distance': 0.0017120025586336851,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.001725267618894577,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483931-C-T',\n",
       "      'distance': 0.0017994536319747567,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503846-C-T',\n",
       "      'distance': 0.001919458620250225,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463943-C-T',\n",
       "      'distance': 0.0020061293616890907,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474638-C-T',\n",
       "      'distance': 0.0021009754855185747,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474533-C-T',\n",
       "      'distance': 0.0021043261513113976,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463219-T-C',\n",
       "      'distance': 0.002135862596333027,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437032-C-T',\n",
       "      'distance': 0.00213884306140244,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434701-C-T',\n",
       "      'distance': 0.0021515400148928165,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437039-C-T',\n",
       "      'distance': 0.002188142854720354,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48463103-G-A',\n",
       "    'metadata': {'vcf_string': '15\\t48463103\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48463103',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5203C>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48463103G>A',\n",
       "     'hgvs_protein': 'p.Gln1735Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM085401\", \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48463103, \"end\": 48463103}, {\"id\": \"COSV57318043\", \"phenotype_or_disease\": 1, \"somatic\": 1, \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"var_synonyms\": {\"COSMIC\": [\"COSM6231903\"]}, \"seq_region_name\": \"15\", \"start\": 48463103, \"end\": 48463103}], \"id\": \"NC_000015.10:g.48463103G>A\", \"start\": 48463103, \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:Q1735X\", \"cdna_start\": 5519, \"cdna_end\": 5519, \"lrt_pred\": \"invalid_field\", \"loeuf\": 0.105, \"conservation\": 3.95, \"phaplo\": 0.99897615427687, \"protein_end\": 1735, \"protein_start\": 1735, \"cds_end\": 5203, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"G\", \"amino_acids\": \"Q/*\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"cds_start\": 5203, \"mutationtaster_pred\": \"A\", \"strand\": -1, \"location\": \"15:48463103\", \"allele_num\": 1, \"consequence_terms\": [\"stop_gained\"], \"cadd_raw\": 10.076559, \"given_ref\": \"G\", \"hgvsp\": \"NP_000129.3:p.Gln1735Ter\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48463103, \"start\": 48463102}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"cadd_phred\": 45, \"mastermind_counts\": \"4|4|4\", \"codons\": \"Cag/Tag\", \"canonical\": 1, \"hgvsc\": \"NM_000138.5:c.5203C>T\", \"biotype\": \"protein_coding\", \"exon\": \"42/66\", \"aa\": \"G\", \"spliceai\": {\"DP_DG\": -25, \"DP_AL\": -3, \"DS_DG\": 0, \"DS_AG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -15, \"DS_DL\": 0, \"DP_AG\": 26}, \"lof\": \"HC\", \"lof_info\": \"PERCENTILE:0.603876508820798,GERP_DIST:3974.84112178832,BP_DIST:3389,DIST_FROM_LAST_EXON:3000,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"A\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"gene_symbol_source\": \"EntrezGene\"}], \"most_severe_consequence\": \"stop_gained\", \"end\": 48463103, \"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"G/A\", \"vcf_string\": \"15-48463103-G-A\", \"input\": \"NC_000015.10:g.48463103G>A\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.02136882689900449,\n",
       "      'pca_y': 0.09693168712859448},\n",
       "     {'t-sne_x': 16.659580323923183, 't-sne_y': 48.27087164412318},\n",
       "     {'umap_x': -9.445549964904785, 'umap_y': 5.560941219329834}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48425474-G-A',\n",
       "      'distance': 0.0009532130789011717,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520737-G-A',\n",
       "      'distance': 0.0009564191568642855,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437376-G-A',\n",
       "      'distance': 0.0010021212510764599,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513579-G-A',\n",
       "      'distance': 0.0010322814341634512,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.001054404303431511,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465679-G-A',\n",
       "      'distance': 0.0011518967803567648,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.00124355498701334,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488197-G-A',\n",
       "      'distance': 0.0012519460869953036,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432971-G-A',\n",
       "      'distance': 0.0013270849594846368,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428361-G-A',\n",
       "      'distance': 0.001382462796755135,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465622-G-A',\n",
       "      'distance': 0.0014501941623166203,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520764-G-A',\n",
       "      'distance': 0.0014702260959893465,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487129-G-A',\n",
       "      'distance': 0.0015452816151082516,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.001609110739082098,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483899-G-A',\n",
       "      'distance': 0.0016173257026821375,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427630-G-A',\n",
       "      'distance': 0.001722919405438006,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452670-G-A',\n",
       "      'distance': 0.0017279937164857984,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472632-G-A',\n",
       "      'distance': 0.001775027485564351,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485416-G-A',\n",
       "      'distance': 0.0019633949268609285,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495237-G-A',\n",
       "      'distance': 0.0019779372960329056,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48460300-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48460300\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48460300',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5242T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48460300A>G',\n",
       "     'hgvs_protein': 'p.Cys1748Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48460300A>G\", \"vcf_string\": \"15-48460300-A-G\", \"allele_string\": \"A/G\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"end\": 48460300, \"transcript_consequences\": [{\"aa\": \"A\", \"blosum62\": -3, \"exon\": \"43/66\", \"spliceai\": {\"DP_DG\": -50, \"DP_AL\": 14, \"DS_DG\": 0, \"DS_AG\": 0.01, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 17, \"DS_DL\": 0, \"DP_AG\": 17}, \"canonical\": 1, \"codons\": \"Tgt/Cgt\", \"cadd_phred\": 26, \"mastermind_counts\": \"5|10|10\", \"hgvsc\": \"NM_000138.5:c.5242T>C\", \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"variant_allele\": \"G\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"impact\": \"MODERATE\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"mutationtaster_pred\": \"D\", \"cds_start\": 5242, \"amino_acids\": \"C/R\", \"cadd_raw\": 4.622591, \"given_ref\": \"A\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"hgvsp\": \"NP_000129.3:p.Cys1748Arg\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48460300, \"start\": 48460299, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}}, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48460300\", \"clinpred\": 0.999710977077484, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 1748, \"loeuf\": 0.105, \"conservation\": 3.95, \"polyphen_score\": 0.983, \"cds_end\": 5242, \"protein_start\": 1748, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"eve_score\": 0.707867328319397, \"alphamissense\": {\"am_pathogenicity\": 0.9995, \"am_class\": \"likely_pathogenic\"}, \"transcript_id\": \"NM_000138.5\", \"cdna_end\": 5558, \"cdna_start\": 5558, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C1748R\"}], \"start\": 48460300, \"id\": \"NC_000015.10:g.48460300A>G\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"end\": 48460300, \"start\": 48460300, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"id\": \"CM138474\"}], \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.04878065462476433,\n",
       "      'pca_y': 0.0704439831656945},\n",
       "     {'t-sne_x': 36.745192716453495, 't-sne_y': 1.5911965329216051},\n",
       "     {'umap_x': 0.9216873049736023, 'umap_y': 6.830440998077393}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.0015956229763105512,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468490-A-G',\n",
       "      'distance': 0.0016931826248764992,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437370-A-G',\n",
       "      'distance': 0.0017381744692102075,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489956-A-G',\n",
       "      'distance': 0.0018665752140805125,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456688-A-G',\n",
       "      'distance': 0.0019120464567095041,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596319-A-G',\n",
       "      'distance': 0.0019740078132599592,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468418-A-G',\n",
       "      'distance': 0.0021619913168251514,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452673-A-G',\n",
       "      'distance': 0.0021903435699641705,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.002218343783169985,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448882-A-G',\n",
       "      'distance': 0.0022498895414173603,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.002546676667407155,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.002565995790064335,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.0026132557541131973,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0026595965027809143,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0027459128759801388,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0027554826810956,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-G',\n",
       "      'distance': 0.0027751473244279623,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.002775818109512329,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.0028212741017341614,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.002909139497205615,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48456690-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48456690\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48456690',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5369G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48456690C>G',\n",
       "     'hgvs_protein': 'p.Arg1790Pro',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48456690C>G\", \"seq_region_name\": \"15\", \"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/G\", \"vcf_string\": \"15-48456690-C-G\", \"start\": 48456690, \"most_severe_consequence\": \"missense_variant\", \"end\": 48456690, \"transcript_consequences\": [{\"loeuf\": 0.105, \"conservation\": 3.95, \"protein_end\": 1790, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 1790, \"cds_end\": 5369, \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.994, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9982}, \"eve_score\": 0.639178510911963, \"mastermind_mmid3\": \"FBN1:R1790P\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_start\": 5685, \"cdna_end\": 5685, \"hgvsc\": \"NM_000138.5:c.5369G>C\", \"biotype\": \"protein_coding\", \"codons\": \"cGa/cCa\", \"canonical\": 1, \"mastermind_counts\": \"8|8|8\", \"cadd_phred\": 31, \"spliceai\": {\"DP_AG\": 29, \"DS_DL\": 0, \"DS_AL\": 0, \"DP_DL\": -43, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DG\": 0, \"DP_DG\": 29, \"DP_AL\": 5}, \"aa\": \"C\", \"exon\": \"44/66\", \"blosum62\": -2, \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"G\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Uncertain\", \"amino_acids\": \"R/P\", \"strand\": -1, \"mutationtaster_pred\": \"A\", \"cds_start\": 5369, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"location\": \"15:48456690\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.996263921260834, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48456690, \"start\": 48456689, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"hgvsp\": \"NP_000129.3:p.Arg1790Pro\", \"given_ref\": \"C\", \"cadd_raw\": 5.400951, \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"]}], \"minimised\": 1, \"colocated_variants\": [{\"start\": 48456690, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48456690, \"phenotype_or_disease\": 1, \"id\": \"CM013932\"}, {\"end\": 48456690, \"start\": 48456690, \"clin_sig\": [\"uncertain_significance\", \"likely_pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV003420163\", \"RCV000663777\", \"VCV000549272\", \"RCV000773641\", \"RCV000788925\", \"RCV001868188\", \"RCV004533458\", \"RCV000663778\", \"VCV000549273\"], \"UniProt\": [\"VAR_018016\"]}, \"allele_string\": \"C/G/T\", \"strand\": 1, \"clin_sig_allele\": \"G:likely_pathogenic;T:conflicting_classifications_of_pathogenicity;T:uncertain_significance\", \"phenotype_or_disease\": 1, \"id\": \"rs1555396428\"}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48456690C>G\"}',\n",
       "    'coordinates': [{'pca_x': 0.0319161510006058,\n",
       "      'pca_y': 0.10064082978194766},\n",
       "     {'t-sne_x': -14.510507559696741, 't-sne_y': 28.431272643354347},\n",
       "     {'umap_x': 1.9963408708572388, 'umap_y': 21.586397171020508}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463949-C-G',\n",
       "      'distance': 0.003268572734668851,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503845-G-C',\n",
       "      'distance': 0.003381395014002919,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425412-G-C',\n",
       "      'distance': 0.003577181836590171,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472644-A-G',\n",
       "      'distance': 0.003735352773219347,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-G',\n",
       "      'distance': 0.0038125524297356606,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.003850571345537901,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463148-A-G',\n",
       "      'distance': 0.003930985927581787,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.003999809734523296,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495216-A-G',\n",
       "      'distance': 0.004026595037430525,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.004034996032714844,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48494210-A-G',\n",
       "      'distance': 0.004150526598095894,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483823-C-G',\n",
       "      'distance': 0.004207883961498737,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456643-A-G',\n",
       "      'distance': 0.004316282458603382,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515428-C-T',\n",
       "      'distance': 0.004364115186035633,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481690-C-T',\n",
       "      'distance': 0.004450863227248192,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437027-T-G',\n",
       "      'distance': 0.004508974030613899,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445454-A-G',\n",
       "      'distance': 0.004617414437234402,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437038-C-T',\n",
       "      'distance': 0.004748832434415817,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508665-C-T',\n",
       "      'distance': 0.004782520234584808,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495156-C-T',\n",
       "      'distance': 0.0047855861485004425,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48456642-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48456642\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48456642',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5417G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48456642C>T',\n",
       "     'hgvs_protein': 'p.Cys1806Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\", \"vcf_string\": \"15-48456642-C-T\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48456642C>T\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"end\": 48456642, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48456642, \"id\": \"CM022004\", \"phenotype_or_disease\": 1}, {\"phenotype_or_disease\": 1, \"id\": \"CM022785\", \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48456642, \"end\": 48456642}], \"id\": \"NC_000015.10:g.48456642C>T\", \"minimised\": 1, \"transcript_consequences\": [{\"given_ref\": \"C\", \"cadd_raw\": 5.121525, \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48456641, \"end\": 48456642, \"type\": \"SimpleInterval\"}}}, \"hgvsp\": \"NP_000129.3:p.Cys1806Tyr\", \"location\": \"15:48456642\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.999909639358521, \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"mutationtaster_pred\": \"D\", \"strand\": -1, \"cds_start\": 5417, \"amino_acids\": \"C/Y\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"variant_allele\": \"T\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"aa\": \"C\", \"exon\": \"44/66\", \"blosum62\": -2, \"spliceai\": {\"DP_AL\": 5, \"DP_DG\": -5, \"DS_DG\": 0, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 5, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": -11}, \"canonical\": 1, \"codons\": \"tGt/tAt\", \"cadd_phred\": 28.7, \"mastermind_counts\": \"7|10|10\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.5417G>A\", \"cdna_start\": 5733, \"cdna_end\": 5733, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C1806Y\", \"eve_score\": 0.728852469380898, \"alphamissense\": {\"am_pathogenicity\": 0.9992, \"am_class\": \"likely_pathogenic\"}, \"transcript_id\": \"NM_000138.5\", \"polyphen_score\": 0.996, \"protein_start\": 1806, \"cds_end\": 5417, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"protein_end\": 1806, \"loeuf\": 0.105, \"conservation\": 3.95}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48456642, \"start\": 48456642}',\n",
       "    'coordinates': [{'pca_x': 0.029419496835379394,\n",
       "      'pca_y': 0.10813843514891555},\n",
       "     {'t-sne_x': -11.039894054295141, 't-sne_y': 34.44440236677071},\n",
       "     {'umap_x': 0.8425163626670837, 'umap_y': 22.07597541809082}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48513661-A-G',\n",
       "      'distance': 0.0014489991590380669,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.0015683970414102077,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474260-C-T',\n",
       "      'distance': 0.0017973588546738029,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425458-C-T',\n",
       "      'distance': 0.0019175156485289335,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492538-C-T',\n",
       "      'distance': 0.001929456484504044,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.001940075191669166,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495140-C-T',\n",
       "      'distance': 0.001950095291249454,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48513554-C-T',\n",
       "      'distance': 0.001965465024113655,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.001991848461329937,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515440-T-T',\n",
       "      'distance': 0.0019924482330679893,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48445453-C-T',\n",
       "      'distance': 0.002098601311445236,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452654-C-T',\n",
       "      'distance': 0.0021830485202372074,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.0022113153245300055,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.002213725820183754,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-C',\n",
       "      'distance': 0.002216032240539789,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.0023399661295115948,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.002380798803642392,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510049-C-T',\n",
       "      'distance': 0.002512128558009863,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495131-C-T',\n",
       "      'distance': 0.002532180165871978,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.0025490373373031616,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48456642-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48456642\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48456642',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5417G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48456642C>G',\n",
       "     'hgvs_protein': 'p.Cys1806Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48456642, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"mastermind_counts\": \"6|9|9\", \"cadd_phred\": 28.2, \"codons\": \"tGt/tCt\", \"canonical\": 1, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.5417G>C\", \"exon\": \"44/66\", \"blosum62\": -1, \"aa\": \"C\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -6, \"SYMBOL\": \"FBN1\", \"DP_DL\": 5, \"DS_AL\": 0, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": 5, \"DP_DG\": -5}, \"variant_allele\": \"G\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"C/S\", \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"cds_start\": 5417, \"mutationtaster_pred\": \"D\", \"strand\": -1, \"clinpred\": 0.999077439308167, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48456642\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"given_ref\": \"C\", \"cadd_raw\": 5.040034, \"hgvsp\": \"NP_000129.3:p.Cys1806Ser\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48456641, \"end\": 48456642, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}}, \"loeuf\": 0.105, \"conservation\": 3.95, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 1806, \"sift_prediction\": \"deleterious_low_confidence\", \"cds_end\": 5417, \"protein_start\": 1806, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"polyphen_score\": 0.985, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9992}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.775409802674971, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C1806S\", \"cdna_end\": 5733, \"cdna_start\": 5733, \"lrt_pred\": \"invalid_field\"}], \"end\": 48456642, \"minimised\": 1, \"id\": \"NC_000015.10:g.48456642C>G\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM022004\", \"end\": 48456642, \"start\": 48456642, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\"}, {\"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48456642, \"end\": 48456642, \"phenotype_or_disease\": 1, \"id\": \"CM022785\"}], \"input\": \"NC_000015.10:g.48456642C>G\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48456642-C-G\", \"allele_string\": \"C/G\", \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': 0.02773060544973638,\n",
       "      'pca_y': 0.10772800217002079},\n",
       "     {'t-sne_x': -14.074022845840986, 't-sne_y': 28.534735557321362},\n",
       "     {'umap_x': 1.6061501502990723, 'umap_y': 21.73468589782715}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463949-C-G',\n",
       "      'distance': 0.0015664432430639863,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-G',\n",
       "      'distance': 0.0022788485512137413,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483823-C-G',\n",
       "      'distance': 0.002880677580833435,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425412-G-C',\n",
       "      'distance': 0.0031697116792201996,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503845-G-C',\n",
       "      'distance': 0.0034112068824470043,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.0037729342002421618,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456642-C-A',\n",
       "      'distance': 0.0038878880441188812,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456643-A-G',\n",
       "      'distance': 0.0040224166586995125,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463148-A-G',\n",
       "      'distance': 0.004026033915579319,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48513600-A-G',\n",
       "      'distance': 0.004027160350233316,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463949-C-T',\n",
       "      'distance': 0.0041228774935007095,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472556-C-T',\n",
       "      'distance': 0.0041754660196602345,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472628-C-T',\n",
       "      'distance': 0.004188518039882183,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.004207509104162455,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.004343783482909203,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.0043562594801187515,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472601-C-T',\n",
       "      'distance': 0.0044272225350141525,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437026-T-C',\n",
       "      'distance': 0.004438186064362526,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.004517612978816032,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452654-C-T',\n",
       "      'distance': 0.0045308442786335945,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48452685-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48452685\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48452685',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5423-1G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48452685C>A',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"splice_acceptor_variant\", \"end\": 48452685, \"transcript_consequences\": [{\"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"maxentscan_alt\": -1.245, \"conservation\": 3.95, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"rf_score\": 0.936, \"mastermind_mmid3\": \"FBN1:D1808sa\", \"mane\": [\"MANE_Select\"], \"intron\": \"44/65\", \"transcript_id\": \"NM_000138.5\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"A\", \"lof_info\": \"INTRON_SIZE:3952\", \"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.5423-1G>T\", \"canonical\": 1, \"maxentscan_ref\": 7.354, \"cadd_phred\": 33, \"mastermind_counts\": \"2|2|4\", \"lof\": \"HC\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 16, \"SYMBOL\": \"FBN1\", \"DP_DL\": -1, \"DS_AL\": 0.99, \"DS_DG\": 0, \"DS_AG\": 0.68, \"DP_AL\": -1, \"DP_DG\": -12}, \"aa\": \"C\", \"maxentscan_diff\": 8.598, \"location\": \"15:48452685\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48452685, \"start\": 48452684}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"cadd_raw\": 5.815235, \"given_ref\": \"C\", \"allele_num\": 1, \"consequence_terms\": [\"splice_acceptor_variant\"], \"ada_score\": 0.999980428708689, \"strand\": -1, \"genesplicer\": \"loss/acceptor/48452684-48452685/High/14.953520\", \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\"}], \"start\": 48452685, \"colocated_variants\": [{\"end\": 48452685, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48452685, \"phenotype_or_disease\": 1, \"id\": \"CS022104\"}, {\"end\": 48452685, \"start\": 48452685, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"id\": \"CS075139\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48452685C>A\", \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48452685C>A\", \"allele_string\": \"C/A\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48452685-C-A\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.23660881136159861,\n",
       "      'pca_y': -0.03312404800727351},\n",
       "     {'t-sne_x': -29.916020562159897, 't-sne_y': -17.754852273037663},\n",
       "     {'umap_x': 11.61347770690918, 'umap_y': 18.98130226135254}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463241-C-A',\n",
       "      'distance': 0.003976981621235609,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.005485718138515949,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.006263205781579018,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.006782738491892815,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.006828468292951584,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.006921305786818266,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.007062045391649008,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.0073158335871994495,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.0073368907906115055,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.007363060489296913,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.007559488527476788,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.007576426025480032,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.007916328497231007,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.008156944066286087,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452686-T-C',\n",
       "      'distance': 0.008296064101159573,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.00837085209786892,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.008575898595154285,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.009115254506468773,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.009580863639712334,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470757-C-T',\n",
       "      'distance': 0.010472698137164116,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48452607-C-CA',\n",
       "    'metadata': {'vcf_string': '15\\t48452607\\t.\\tC\\tCA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48452607',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'CA',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5499dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48452608dup',\n",
       "     'hgvs_protein': 'p.Asp1834Ter',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"end\": 48452607, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"loeuf\": 0.105, \"conservation\": -1.165, \"protein_end\": 1834, \"phaplo\": 0.99897615427687, \"used_ref\": \"-\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 1833, \"cds_end\": 5500, \"transcript_id\": \"NM_000138.5\", \"mastermind_mmid3\": \"FBN1:D1834X\", \"mane\": [\"MANE_Select\"], \"cdna_end\": 5816, \"cdna_start\": 5815, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.5499dup\", \"canonical\": 1, \"codons\": \"-/T\", \"mastermind_counts\": \"3|3|3\", \"lof\": \"HC\", \"spliceai\": {\"DP_AG\": -34, \"DS_DL\": 0, \"DP_DL\": -3, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_AG\": 0, \"DS_DG\": 0, \"DP_AL\": -3, \"DP_DG\": -37}, \"exon\": \"45/66\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"A\", \"lof_info\": \"PERCENTILE:0.638231197771588,GERP_DIST:3589.02818443701,BP_DIST:3096,DIST_FROM_LAST_EXON:2707,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"-/X\", \"strand\": -1, \"cds_start\": 5499, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"location\": \"15:48452607-48452608\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"interval\": {\"end\": 48452607, \"start\": 48452607, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"hgvsp\": \"NP_000129.3:p.Asp1834Ter\", \"given_ref\": \"-\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1}], \"start\": 48452608, \"id\": \"NC_000015.10:g.48452608dup\", \"colocated_variants\": [{\"id\": \"rs1555396199\", \"phenotype_or_disease\": 1, \"end\": 48452608, \"start\": 48452608, \"strand\": 1, \"allele_string\": \"A/AA\", \"seq_region_name\": \"15\", \"clin_sig\": [\"likely_pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV000663789\", \"VCV000549281\"]}}], \"variant_class\": \"insertion\", \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48452608dup\", \"vcf_string\": \"15-48452607-C-CA\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"-/A\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.016446412624804924,\n",
       "      'pca_y': 0.005051533143464446},\n",
       "     {'t-sne_x': 9.859287502134235, 't-sne_y': 15.420500782389434},\n",
       "     {'umap_x': 3.6944286823272705, 'umap_y': 3.594531774520874}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.10183224827051163,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.10227629542350769,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.10361383855342865,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.10438461601734161,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452595-C-A',\n",
       "      'distance': 0.10495398938655853,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.10520945489406586,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.10578083992004395,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.10589094460010529,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452567-C-T',\n",
       "      'distance': 0.10608042776584625,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446814-C-T',\n",
       "      'distance': 0.10609906911849976,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.10623328387737274,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446768-A-G',\n",
       "      'distance': 0.10627315193414688,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.1063678190112114,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.10645236074924469,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446811-A-G',\n",
       "      'distance': 0.10650903731584549,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.10655701905488968,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.10669250786304474,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.10684187710285187,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.10704164206981659,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445492-C-T',\n",
       "      'distance': 0.10707931220531464,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48452598-G-A',\n",
       "    'metadata': {'vcf_string': '15\\t48452598\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48452598',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5509C>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48452598G>A',\n",
       "     'hgvs_protein': 'p.Pro1837Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48452598G>A\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"G/A\", \"vcf_string\": \"15-48452598-G-A\", \"strand\": 1, \"end\": 48452598, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"mastermind_mmid3\": \"FBN1:P1837S\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_start\": 5825, \"cdna_end\": 5825, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_benign\", \"am_pathogenicity\": 0.0948}, \"eve_score\": 0.22272607422477, \"used_ref\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 5509, \"protein_start\": 1837, \"sift_prediction\": \"tolerated_low_confidence\", \"polyphen_score\": 0.996, \"conservation\": 3.62, \"loeuf\": 0.105, \"protein_end\": 1837, \"phaplo\": 0.99897615427687, \"sift_score\": 0.06, \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.656174063682556, \"location\": \"15:48452598\", \"hgvsp\": \"NP_000129.3:p.Pro1837Ser\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48452597, \"end\": 48452598, \"type\": \"SimpleInterval\"}}, \"type\": \"Allele\"}, \"cadd_raw\": 5.857833, \"given_ref\": \"G\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"amino_acids\": \"P/S\", \"mutationtaster_pred\": \"D\", \"strand\": -1, \"cds_start\": 5509, \"impact\": \"MODERATE\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"A\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Benign\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.5509C>T\", \"canonical\": 1, \"codons\": \"Ccc/Tcc\", \"cadd_phred\": 33, \"mastermind_counts\": \"7|10|10\", \"spliceai\": {\"DS_AG\": 0, \"DS_DG\": 0.7, \"DP_DG\": 6, \"DP_AL\": -25, \"DP_AG\": -36, \"DS_DL\": 0.04, \"DS_AL\": 0, \"DP_DL\": -36, \"SYMBOL\": \"FBN1\"}, \"aa\": \"G\", \"exon\": \"45/66\", \"blosum62\": -1}], \"start\": 48452598, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"id\": \"CM960638\", \"phenotype_or_disease\": 1, \"end\": 48452598, \"start\": 48452598, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\"}, {\"phenotype_or_disease\": 1, \"id\": \"COSV100355041\", \"somatic\": 1, \"start\": 48452598, \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM8102508\"]}, \"seq_region_name\": \"15\", \"end\": 48452598}, {\"clin_sig_allele\": \"A:uncertain_significance\", \"phenotype_or_disease\": 1, \"id\": \"rs755430984\", \"pubmed\": [24033266], \"end\": 48452598, \"frequencies\": {\"A\": {\"gnomadg_amr\": 0, \"gnomadg_remaining\": 0, \"gnomadg_asj\": 0, \"gnomade_remaining\": 0, \"gnomadg_nfe\": 1.47e-05, \"gnomadg_ami\": 0, \"gnomade_sas\": 0, \"gnomade_fin\": 0, \"gnomade_mid\": 0, \"gnomadg_fin\": 0, \"gnomadg_sas\": 0, \"gnomadg_mid\": 0, \"gnomadg_eas\": 0, \"gnomade_eas\": 0, \"gnomade_asj\": 0, \"gnomade_amr\": 0, \"gnomadg_afr\": 0, \"gnomade_nfe\": 1.799e-06, \"gnomade\": 1.368e-06, \"gnomade_afr\": 0, \"gnomadg\": 6.57e-06}}, \"allele_string\": \"G/A\", \"strand\": 1, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV004805019\", \"RCV000455082\", \"VCV000402848\", \"RCV001524795\"], \"UniProt\": [\"VAR_002330\"]}, \"clin_sig\": [\"uncertain_significance\"], \"start\": 48452598}], \"id\": \"NC_000015.10:g.48452598G>A\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.04399898796440216,\n",
       "      'pca_y': 0.08099995539528601},\n",
       "     {'t-sne_x': 34.9642634624393, 't-sne_y': 9.436983296374795},\n",
       "     {'umap_x': 1.5176725387573242, 'umap_y': 7.6803460121154785}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487353-G-A',\n",
       "      'distance': 0.0023703190963715315,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48432907-G-A',\n",
       "      'distance': 0.0025066668167710304,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48452589-G-A',\n",
       "      'distance': 0.002681134734302759,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.0034598256461322308,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468097-G-A',\n",
       "      'distance': 0.0036411250475794077,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490073-G-A',\n",
       "      'distance': 0.0036905992310494184,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537629-G-A',\n",
       "      'distance': 0.0037448194343596697,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456688-A-G',\n",
       "      'distance': 0.004163672216236591,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508588-A-G',\n",
       "      'distance': 0.00423115398734808,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497332-G-A',\n",
       "      'distance': 0.004340200684964657,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430710-G-A',\n",
       "      'distance': 0.004346681293100119,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.004412348847836256,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596319-A-G',\n",
       "      'distance': 0.0044480059295892715,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.004482761956751347,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48452595-C-A',\n",
       "      'distance': 0.004497206769883633,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.0045300694182515144,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.004587201401591301,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452673-A-G',\n",
       "      'distance': 0.004627899266779423,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600217-G-A',\n",
       "      'distance': 0.004644066095352173,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446811-A-G',\n",
       "      'distance': 0.004648913629353046,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8}}},\n",
       "   {'variant_id': '15-48448861-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48448861\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48448861',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5578T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48448861A>G',\n",
       "     'hgvs_protein': 'p.Cys1860Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"allele_string\": \"A/G\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48448861-A-G\", \"input\": \"NC_000015.10:g.48448861A>G\", \"seq_region_name\": \"15\", \"minimised\": 1, \"colocated_variants\": [{\"start\": 48448861, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48448861, \"phenotype_or_disease\": 1, \"id\": \"CM085406\"}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48448861A>G\", \"start\": 48448861, \"end\": 48448861, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C1860R\", \"cdna_end\": 5894, \"cdna_start\": 5894, \"lrt_pred\": \"invalid_field\", \"alphamissense\": {\"am_pathogenicity\": 0.9994, \"am_class\": \"likely_pathogenic\"}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.75271356366282, \"cds_end\": 5578, \"protein_start\": 1860, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.996, \"conservation\": 3.95, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"protein_end\": 1860, \"location\": \"15:48448861\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.999954462051392, \"given_ref\": \"A\", \"cadd_raw\": 5.154516, \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48448860, \"end\": 48448861}, \"type\": \"SequenceLocation\"}}, \"hgvsp\": \"NP_000129.3:p.Cys1860Arg\", \"amino_acids\": \"C/R\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"strand\": -1, \"mutationtaster_pred\": \"D\", \"cds_start\": 5578, \"variant_allele\": \"G\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"canonical\": 1, \"codons\": \"Tgc/Cgc\", \"cadd_phred\": 28.9, \"mastermind_counts\": \"6|7|7\", \"hgvsc\": \"NM_000138.5:c.5578T>C\", \"biotype\": \"protein_coding\", \"aa\": \"A\", \"blosum62\": -3, \"exon\": \"46/66\", \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": 8, \"SYMBOL\": \"FBN1\", \"DP_AG\": 32, \"DS_DL\": 0, \"DP_DG\": 6, \"DP_AL\": 49, \"DS_AG\": 0, \"DS_DG\": 0}}]}',\n",
       "    'coordinates': [{'pca_x': 0.04756603239769641,\n",
       "      'pca_y': 0.08000153644941707},\n",
       "     {'t-sne_x': 38.428823444009325, 't-sne_y': 4.583680974881519},\n",
       "     {'umap_x': 0.8824970126152039, 'umap_y': 6.828773021697998}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48446811-A-G',\n",
       "      'distance': 0.0018414725782349706,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456688-A-G',\n",
       "      'distance': 0.0019326311303302646,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596319-A-G',\n",
       "      'distance': 0.0019344780594110489,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452673-A-G',\n",
       "      'distance': 0.0020024406258016825,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.0020118507090955973,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448813-A-G',\n",
       "      'distance': 0.002057037316262722,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.002057954901829362,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.002069644397124648,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0020790761336684227,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596373-A-G',\n",
       "      'distance': 0.0021408514585345984,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.002185326535254717,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.0022057881578803062,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468418-A-G',\n",
       "      'distance': 0.0022116079926490784,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468490-A-G',\n",
       "      'distance': 0.0022198958322405815,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.002269304823130369,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0022889948450028896,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448882-A-G',\n",
       "      'distance': 0.0023646499030292034,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0024060546420514584,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.002428076695650816,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497311-A-G',\n",
       "      'distance': 0.002492467639967799,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48448825-AG-A',\n",
       "    'metadata': {'vcf_string': '15\\t48448825\\t.\\tAG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48448825',\n",
       "     'ref_allele': 'AG',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5613del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48448826del',\n",
       "     'hgvs_protein': 'p.Tyr1873IlefsTer20',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"deletion\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CD040180\", \"start\": 48448826, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48448826}, {\"end\": 48448826, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000706237\", \"VCV000582223\"]}, \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"G/-\", \"strand\": 1, \"start\": 48448826, \"pubmed\": [17657824], \"phenotype_or_disease\": 1, \"id\": \"rs1566899541\"}], \"id\": \"NC_000015.10:g.48448826del\", \"minimised\": 1, \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"mastermind_mmid3\": \"FBN1:Y1873fs\", \"mane\": [\"MANE_Select\"], \"cdna_start\": 5929, \"cdna_end\": 5929, \"loeuf\": 0.105, \"conservation\": -2.73, \"protein_end\": 1871, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"G\", \"protein_start\": 1871, \"cds_end\": 5613, \"amino_acids\": \"S/X\", \"cds_start\": 5613, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"location\": \"15:48448826\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48448826, \"start\": 48448825, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Tyr1873IlefsTer20\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"given_ref\": \"G\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.5613del\", \"mastermind_counts\": \"3|3|3\", \"canonical\": 1, \"codons\": \"agC/ag\", \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": 43, \"SYMBOL\": \"FBN1\", \"DP_AG\": 50, \"DS_DL\": 0, \"DP_DG\": -49, \"DP_AL\": -46, \"DS_AG\": 0, \"DS_DG\": 0}, \"lof\": \"HC\", \"exon\": \"46/66\", \"aa\": \"G\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"-\", \"lof_info\": \"PERCENTILE:0.651462395543176,GERP_DIST:3445.00193004087,BP_DIST:2983,DIST_FROM_LAST_EXON:2594,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol_source\": \"EntrezGene\"}], \"most_severe_consequence\": \"frameshift_variant\", \"end\": 48448826, \"start\": 48448826, \"allele_string\": \"G/-\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48448825-AG-A\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48448826del\"}',\n",
       "    'coordinates': [{'pca_x': 0.0399254873343973,\n",
       "      'pca_y': 0.02686077362139217},\n",
       "     {'t-sne_x': 38.36770014110373, 't-sne_y': 8.786066537835307},\n",
       "     {'umap_x': 1.4256315231323242, 'umap_y': 7.348462104797363}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.08832946419715881,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.08990921080112457,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445382-A-G',\n",
       "      'distance': 0.09037186950445175,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.09049248695373535,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.09063401818275452,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.09069397300481796,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.09098529815673828,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.09114581346511841,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.09133268147706985,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09154835343360901,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.0916471928358078,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.09172060340642929,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445493-A-G',\n",
       "      'distance': 0.09176497161388397,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.0917777419090271,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430710-G-A',\n",
       "      'distance': 0.09182872623205185,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48452595-C-A',\n",
       "      'distance': 0.09183479845523834,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.09190933406352997,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09201283752918243,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452673-A-G',\n",
       "      'distance': 0.0920143723487854,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446811-A-G',\n",
       "      'distance': 0.09205345064401627,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48446815-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48446815\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48446815',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5679T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48446815A>T',\n",
       "     'hgvs_protein': 'p.Asn1893Lys',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"start\": 48446815, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"end\": 48446815, \"id\": \"CD1312139\", \"phenotype_or_disease\": 1}, {\"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48446815, \"end\": 48446815, \"id\": \"CM970516\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48446815A>T\", \"minimised\": 1, \"transcript_consequences\": [{\"mastermind_mmid3\": \"FBN1:N1893K\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_end\": 5995, \"cdna_start\": 5995, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.9743, \"am_class\": \"likely_pathogenic\"}, \"eve_score\": 0.205857283560712, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"A\", \"sift_prediction\": \"deleterious_low_confidence\", \"cds_end\": 5679, \"protein_start\": 1893, \"polyphen_score\": 0.992, \"conservation\": -7.63, \"loeuf\": 0.105, \"protein_end\": 1893, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.981910526752472, \"location\": \"15:48446815\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"start\": 48446814, \"end\": 48446815}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"hgvsp\": \"NP_000129.3:p.Asn1893Lys\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"cadd_raw\": 3.108458, \"given_ref\": \"A\", \"amino_acids\": \"N/K\", \"cds_start\": 5679, \"mutationtaster_pred\": \"D\", \"strand\": -1, \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"T\", \"eve_class\": \"Benign\", \"gene_symbol_source\": \"EntrezGene\", \"hgvsc\": \"NM_000138.5:c.5679T>A\", \"biotype\": \"protein_coding\", \"cadd_phred\": 21.4, \"mastermind_counts\": \"6|6|6\", \"codons\": \"aaT/aaA\", \"canonical\": 1, \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": 6, \"SYMBOL\": \"FBN1\", \"DP_AG\": -2, \"DS_DL\": 0, \"DP_DG\": -2, \"DP_AL\": -12, \"DS_AG\": 0.02, \"DS_DG\": 0}, \"exon\": \"47/66\", \"blosum62\": 0, \"aa\": \"A\"}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48446815, \"start\": 48446815, \"allele_string\": \"A/T\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48446815-A-T\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48446815A>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.04226413479736468,\n",
       "      'pca_y': 0.0731404931751429},\n",
       "     {'t-sne_x': 16.451858252083433, 't-sne_y': 8.92777090258289},\n",
       "     {'umap_x': 3.3923795223236084, 'umap_y': 4.911489486694336}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468527-A-T',\n",
       "      'distance': 0.0034199757501482964,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.0034968857653439045,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-T',\n",
       "      'distance': 0.0035199509002268314,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430791-A-T',\n",
       "      'distance': 0.0036617000587284565,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596325-A-T',\n",
       "      'distance': 0.0038828435353934765,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428439-A-T',\n",
       "      'distance': 0.003978357650339603,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446811-A-G',\n",
       "      'distance': 0.004002120345830917,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.004382519517093897,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.0045095691457390785,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.004554232582449913,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.004644349217414856,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415684-C-T',\n",
       "      'distance': 0.004758041352033615,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.004790103062987328,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48444579-C-A',\n",
       "      'distance': 0.0048545063473284245,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446768-A-G',\n",
       "      'distance': 0.005047865677624941,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.0050523229874670506,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446747-C-T',\n",
       "      'distance': 0.005103124305605888,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520677-A-G',\n",
       "      'distance': 0.005132526624947786,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.005134546663612127,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.005139521323144436,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48446738-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48446738\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48446738',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5756G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48446738C>T',\n",
       "     'hgvs_protein': 'p.Gly1919Asp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48446738, \"most_severe_consequence\": \"missense_variant\", \"end\": 48446738, \"transcript_consequences\": [{\"amino_acids\": \"G/D\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"strand\": -1, \"mutationtaster_pred\": \"D\", \"cds_start\": 5756, \"location\": \"15:48446738\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.997098445892334, \"cadd_raw\": 5.302102, \"given_ref\": \"C\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48446738, \"start\": 48446737}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"hgvsp\": \"NP_000129.3:p.Gly1919Asp\", \"canonical\": 1, \"codons\": \"gGt/gAt\", \"mastermind_counts\": \"6|6|6\", \"cadd_phred\": 29.8, \"hgvsc\": \"NM_000138.5:c.5756G>A\", \"biotype\": \"protein_coding\", \"aa\": \"C\", \"exon\": \"47/66\", \"blosum62\": -1, \"spliceai\": {\"DP_AL\": 1, \"DP_DG\": -26, \"DS_AG\": 0, \"DS_DG\": 0, \"DP_DL\": 1, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AG\": -33, \"DS_DL\": 0}, \"variant_allele\": \"T\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"alphamissense\": {\"am_pathogenicity\": 0.9875, \"am_class\": \"likely_pathogenic\"}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.767057574490232, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:G1919D\", \"cdna_end\": 6072, \"cdna_start\": 6072, \"lrt_pred\": \"invalid_field\", \"conservation\": 3.95, \"loeuf\": 0.105, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 1919, \"protein_start\": 1919, \"cds_end\": 5756, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.999}], \"minimised\": 1, \"colocated_variants\": [{\"id\": \"CM022787\", \"phenotype_or_disease\": 1, \"end\": 48446738, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48446738}, {\"phenotype_or_disease\": 1, \"id\": \"CM096458\", \"end\": 48446738, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48446738}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48446738C>T\", \"input\": \"NC_000015.10:g.48446738C>T\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48446738-C-T\"}',\n",
       "    'coordinates': [{'pca_x': 0.037381318767226224,\n",
       "      'pca_y': 0.0795610445842912},\n",
       "     {'t-sne_x': 9.099300954013712, 't-sne_y': 2.649806736293779},\n",
       "     {'umap_x': 4.137150764465332, 'umap_y': 3.124417543411255}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.0011625871993601322,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.0014552847715094686,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48537706-C-T',\n",
       "      'distance': 0.0015768035082146525,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48441771-C-T',\n",
       "      'distance': 0.0017740698531270027,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446747-C-T',\n",
       "      'distance': 0.0017835460603237152,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456729-C-T',\n",
       "      'distance': 0.0017867969581857324,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.0018291581654921174,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421981-C-T',\n",
       "      'distance': 0.0018322805408388376,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421594-C-T',\n",
       "      'distance': 0.001858619274571538,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421982-C-T',\n",
       "      'distance': 0.001931958831846714,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0019437847658991814,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496213-C-T',\n",
       "      'distance': 0.0019440290052443743,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495162-C-T',\n",
       "      'distance': 0.0019715384114533663,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456714-C-T',\n",
       "      'distance': 0.001974912825971842,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415707-C-T',\n",
       "      'distance': 0.0019897674210369587,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448851-C-T',\n",
       "      'distance': 0.0019941425416618586,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.0020453480537980795,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.0020474232733249664,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.0020549921318888664,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.002111300826072693,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48446711-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48446711\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48446711',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5783G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48446711C>G',\n",
       "     'hgvs_protein': 'p.Cys1928Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48446711C>G\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48446711, \"end\": 48446711, \"phenotype_or_disease\": 1, \"id\": \"CM055257\"}, {\"id\": \"CM090179\", \"phenotype_or_disease\": 1, \"end\": 48446711, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48446711}], \"variant_class\": \"SNV\", \"minimised\": 1, \"transcript_consequences\": [{\"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48446711\", \"clinpred\": 0.998521625995636, \"hgvsp\": \"NP_000129.3:p.Cys1928Ser\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48446711, \"start\": 48446710}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"given_ref\": \"C\", \"cadd_raw\": 5.233691, \"amino_acids\": \"C/S\", \"cds_start\": 5783, \"strand\": -1, \"mutationtaster_pred\": \"D\", \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"G\", \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"hgvsc\": \"NM_000138.5:c.5783G>C\", \"biotype\": \"protein_coding\", \"cadd_phred\": 29.4, \"mastermind_counts\": \"4|8|8\", \"canonical\": 1, \"codons\": \"tGt/tCt\", \"spliceai\": {\"DP_AL\": -5, \"DP_DG\": -5, \"DS_DG\": 0, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 1, \"DS_AL\": 0, \"DS_DL\": 0.01, \"DP_AG\": -6}, \"blosum62\": -1, \"exon\": \"47/66\", \"aa\": \"C\", \"mastermind_mmid3\": \"FBN1:C1928S\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_end\": 6099, \"cdna_start\": 6099, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.9997, \"am_class\": \"likely_pathogenic\"}, \"eve_score\": 0.736065095173503, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 1928, \"cds_end\": 5783, \"polyphen_score\": 0.985, \"conservation\": 3.95, \"loeuf\": 0.105, \"protein_end\": 1928, \"sift_score\": 0, \"phaplo\": 0.99897615427687}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48446711, \"start\": 48446711, \"vcf_string\": \"15-48446711-C-G\", \"allele_string\": \"C/G\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48446711C>G\"}',\n",
       "    'coordinates': [{'pca_x': 0.04165857922639188,\n",
       "      'pca_y': 0.07925860410142774},\n",
       "     {'t-sne_x': 19.977426174691356, 't-sne_y': -5.569147377037053},\n",
       "     {'umap_x': 8.709061622619629, 'umap_y': 6.982685089111328}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0024238168261945248,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.0026384973898530006,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.0026789074763655663,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0028920853510499,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.0029099665116518736,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.0029385974630713463,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.0030961991287767887,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.0032153441570699215,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.003266536630690098,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.003596337977796793,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445492-C-T',\n",
       "      'distance': 0.0037122841458767653,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.003727983683347702,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446747-C-T',\n",
       "      'distance': 0.00374586065299809,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446775-T-C',\n",
       "      'distance': 0.003950945101678371,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446816-T-C',\n",
       "      'distance': 0.004123925697058439,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.004144249018281698,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.004165560007095337,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.00427408330142498,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.004281673580408096,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513599-C-G',\n",
       "      'distance': 0.004309909883886576,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48445493-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48445493\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48445493',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5800T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48445493A>C',\n",
       "     'hgvs_protein': 'p.Cys1934Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48445493A>C\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48445493-A-C\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/C\", \"start\": 48445493, \"most_severe_consequence\": \"missense_variant\", \"end\": 48445493, \"transcript_consequences\": [{\"cadd_raw\": 5.13438, \"given_ref\": \"A\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"location\": {\"interval\": {\"start\": 48445492, \"end\": 48445493, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Cys1934Gly\", \"clinpred\": 0.999714910984039, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48445493\", \"impact\": \"MODERATE\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"D\", \"strand\": -1, \"cds_start\": 5800, \"amino_acids\": \"C/G\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"variant_allele\": \"C\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"aa\": \"A\", \"exon\": \"48/66\", \"blosum62\": -3, \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -1, \"SYMBOL\": \"FBN1\", \"DP_DL\": -2, \"DS_AL\": 0, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": 34, \"DP_DG\": 0}, \"canonical\": 1, \"codons\": \"Tgt/Ggt\", \"mastermind_counts\": \"4|4|4\", \"cadd_phred\": 28.8, \"hgvsc\": \"NM_000138.5:c.5800T>G\", \"biotype\": \"protein_coding\", \"cdna_end\": 6116, \"cdna_start\": 6116, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C1934G\", \"eve_score\": 0.806912727222808, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9933}, \"transcript_id\": \"NM_000138.5\", \"polyphen_score\": 0.985, \"cds_end\": 5800, \"protein_start\": 1934, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 1934, \"loeuf\": 0.105, \"conservation\": 3.93}], \"minimised\": 1, \"id\": \"NC_000015.10:g.48445493A>C\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM085407\", \"end\": 48445493, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48445493}, {\"phenotype_or_disease\": 1, \"id\": \"CM117726\", \"end\": 48445493, \"start\": 48445493, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\"}, {\"start\": 48445493, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48445493, \"id\": \"CM157692\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.04735588857248185,\n",
       "      'pca_y': 0.08016272238947361},\n",
       "     {'t-sne_x': 21.750294506057557, 't-sne_y': 6.048984213265983},\n",
       "     {'umap_x': 3.6610872745513916, 'umap_y': 6.141015529632568}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48445493-A-G',\n",
       "      'distance': 0.0037957357708364725,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445492-C-T',\n",
       "      'distance': 0.004045521840453148,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.00437664519995451,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508693-A-C',\n",
       "      'distance': 0.004598118364810944,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.004822521470487118,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.005045819096267223,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.005162190645933151,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445432-A-C',\n",
       "      'distance': 0.005190807394683361,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.005209839902818203,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.00547465356066823,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488419-A-C',\n",
       "      'distance': 0.005479112733155489,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.0055718738585710526,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.005586815997958183,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.005616744980216026,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.005679706111550331,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448813-A-G',\n",
       "      'distance': 0.0056826225481927395,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452595-C-A',\n",
       "      'distance': 0.005778225604444742,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.0058662681840360165,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.006310733500868082,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446816-T-C',\n",
       "      'distance': 0.0066373879089951515,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48444560-AA-A',\n",
       "    'metadata': {'vcf_string': '15\\t48444560\\t.\\tAA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48444560',\n",
       "     'ref_allele': 'AA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6018del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48444561del',\n",
       "     'hgvs_protein': 'p.Gln2007LysfsTer52',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"vcf_string\": \"15-48444560-AA-A\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/-\", \"input\": \"NC_000015.10:g.48444561del\", \"seq_region_name\": \"15\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48444561del\", \"variant_class\": \"deletion\", \"start\": 48444561, \"most_severe_consequence\": \"frameshift_variant\", \"end\": 48444561, \"transcript_consequences\": [{\"cds_start\": 6018, \"strand\": -1, \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"amino_acids\": \"L/X\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48444561, \"start\": 48444560}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Gln2007LysfsTer52\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"given_ref\": \"A\", \"location\": \"15:48444560\", \"hgvs_offset\": -1, \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": 27, \"DP_DG\": -11, \"DS_DL\": 0, \"DP_AG\": -29, \"SYMBOL\": \"FBN1\", \"DP_DL\": -19, \"DS_AL\": 0}, \"lof\": \"HC\", \"exon\": \"49/66\", \"aa\": \"A\", \"hgvsc\": \"NM_000138.5:c.6018del\", \"biotype\": \"protein_coding\", \"codons\": \"ctT/ct\", \"canonical\": 1, \"gene_symbol_source\": \"EntrezGene\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"lof_info\": \"PERCENTILE:0.698467966573816,GERP_DIST:3068.29392101169,BP_DIST:2582,DIST_FROM_LAST_EXON:2193,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"transcript_id\": \"NM_000138.5\", \"cdna_end\": 6334, \"cdna_start\": 6334, \"mane\": [\"MANE_Select\"], \"protein_end\": 2006, \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"conservation\": 3.95, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"A\", \"cds_end\": 6018, \"protein_start\": 2006}]}',\n",
       "    'coordinates': [{'pca_x': -0.04472526838264515,\n",
       "      'pca_y': 0.044609203362278686},\n",
       "     {'t-sne_x': -12.903281345306343, 't-sne_y': -3.25094921750324},\n",
       "     {'umap_x': 0.3899432122707367, 'umap_y': 15.175230026245117}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.09780149161815643,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.0999089777469635,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.1009768545627594,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.10141374170780182,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.1020914763212204,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.102307990193367,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.10287120938301086,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.10323704779148102,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.10352832078933716,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.10353472083806992,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.10389156639575958,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.10390843451023102,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.103939950466156,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.10472776740789413,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.10510134696960449,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.10521809011697769,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.10524668544530869,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.10637307167053223,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.1063755676150322,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48444585-C-T',\n",
       "      'distance': 0.10713338851928711,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48441724-G-A',\n",
       "    'metadata': {'vcf_string': '15\\t48441724\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48441724',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6160C>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48441724G>A',\n",
       "     'hgvs_protein': 'p.Gln2054Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48441724-G-A\", \"allele_string\": \"G/A\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48441724G>A\", \"id\": \"NC_000015.10:g.48441724G>A\", \"colocated_variants\": [{\"end\": 48441724, \"start\": 48441724, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"id\": \"CM013937\", \"phenotype_or_disease\": 1}, {\"phenotype_or_disease\": 1, \"id\": \"rs1555395456\", \"clin_sig_allele\": \"A:pathogenic\", \"start\": 48441724, \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000663850\", \"VCV000549331\"]}, \"allele_string\": \"G/A\", \"strand\": 1, \"end\": 48441724}], \"variant_class\": \"SNV\", \"minimised\": 1, \"most_severe_consequence\": \"stop_gained\", \"end\": 48441724, \"transcript_consequences\": [{\"consequence_terms\": [\"stop_gained\"], \"allele_num\": 1, \"given_ref\": \"G\", \"cadd_raw\": 9.769051, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48441724, \"start\": 48441723}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Gln2054Ter\", \"location\": \"15:48441724\", \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"cds_start\": 6160, \"strand\": -1, \"mutationtaster_pred\": \"A\", \"amino_acids\": \"Q/*\", \"gene_symbol_source\": \"EntrezGene\", \"lof_info\": \"PERCENTILE:0.714948932219127,GERP_DIST:2888.70375534296,BP_DIST:2440,DIST_FROM_LAST_EXON:2051,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"A\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"exon\": \"50/66\", \"aa\": \"G\", \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": -3, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": -20, \"DP_AL\": 35, \"DP_DG\": 5, \"DS_DG\": 0, \"DS_AG\": 0}, \"lof\": \"HC\", \"cadd_phred\": 43, \"mastermind_counts\": \"6|7|7\", \"codons\": \"Caa/Taa\", \"canonical\": 1, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.6160C>T\", \"cdna_end\": 6476, \"cdna_start\": 6476, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:Q2054X\", \"transcript_id\": \"NM_000138.5\", \"cds_end\": 6160, \"protein_start\": 2054, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"G\", \"phaplo\": 0.99897615427687, \"protein_end\": 2054, \"conservation\": 3.09, \"loeuf\": 0.105}], \"start\": 48441724}',\n",
       "    'coordinates': [{'pca_x': 0.01960131960751199,\n",
       "      'pca_y': 0.09693782776216747},\n",
       "     {'t-sne_x': 18.660129333278554, 't-sne_y': 46.166136764545904},\n",
       "     {'umap_x': -9.21691608428955, 'umap_y': 6.0132622718811035}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48415540-G-A',\n",
       "      'distance': 0.0008663283078931272,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425423-G-A',\n",
       "      'distance': 0.001135130412876606,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.0011905825231224298,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485416-G-A',\n",
       "      'distance': 0.001374746672809124,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465622-G-A',\n",
       "      'distance': 0.001436177408322692,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415675-G-A',\n",
       "      'distance': 0.0015056238044053316,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.0015508930664509535,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472632-G-A',\n",
       "      'distance': 0.0015553748235106468,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495237-G-A',\n",
       "      'distance': 0.0016296980902552605,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445430-G-A',\n",
       "      'distance': 0.0016446227673441172,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.001667646924033761,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427630-G-A',\n",
       "      'distance': 0.0018053523963317275,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432947-G-A',\n",
       "      'distance': 0.001810448826290667,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425829-G-A',\n",
       "      'distance': 0.001820078119635582,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513615-G-A',\n",
       "      'distance': 0.0018549112137407064,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437376-G-A',\n",
       "      'distance': 0.0019306058529764414,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.0019312648801133037,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465679-G-A',\n",
       "      'distance': 0.00200370023958385,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468064-G-A',\n",
       "      'distance': 0.0020200235303491354,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465820-G-A',\n",
       "      'distance': 0.002033545868471265,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48441723-T-G',\n",
       "    'metadata': {'vcf_string': '15\\t48441723\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48441723',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6161A>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48441723T>G',\n",
       "     'hgvs_protein': 'p.Gln2054Pro',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"T/G\", \"vcf_string\": \"15-48441723-T-G\", \"input\": \"NC_000015.10:g.48441723T>G\", \"seq_region_name\": \"15\", \"minimised\": 1, \"colocated_variants\": [{\"end\": 48441723, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48441723, \"id\": \"CM003139\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48441723T>G\", \"start\": 48441723, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"sift_prediction\": \"tolerated_low_confidence\", \"maxentscan_alt\": 10.858, \"cds_end\": 6161, \"protein_start\": 2054, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"T\", \"polyphen_score\": 0.552, \"loeuf\": 0.105, \"conservation\": 2.79, \"phaplo\": 0.99897615427687, \"sift_score\": 0.2, \"protein_end\": 2054, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:D2055srd,FBN1:Q2054P\", \"rf_score\": 0.634, \"cdna_end\": 6477, \"cdna_start\": 6477, \"lrt_pred\": \"invalid_field\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9582}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.141081064789179, \"variant_allele\": \"G\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"eve_class\": \"Benign\", \"gene_symbol_source\": \"EntrezGene\", \"cadd_phred\": 24.9, \"mastermind_counts\": \"5|10|20\", \"codons\": \"cAa/cCa\", \"canonical\": 1, \"maxentscan_ref\": 10.998, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.6161A>C\", \"exon\": \"50/66\", \"blosum62\": -1, \"aa\": \"T\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -3, \"SYMBOL\": \"FBN1\", \"DP_DL\": -2, \"DS_AL\": 0, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": 36, \"DP_DG\": 6}, \"polyphen_prediction\": \"possibly_damaging\", \"location\": \"15:48441723\", \"clinpred\": 0.777485430240631, \"maxentscan_diff\": 0.14, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"allele_num\": 1, \"given_ref\": \"T\", \"cadd_raw\": 4.314369, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48441723, \"start\": 48441722}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Gln2054Pro\", \"amino_acids\": \"Q/P\", \"ada_score\": 0.864805939962064, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"cds_start\": 6161, \"mutationtaster_pred\": \"D\", \"strand\": -1}], \"end\": 48441723}',\n",
       "    'coordinates': [{'pca_x': -0.038820627959663866,\n",
       "      'pca_y': 0.04485867909059265},\n",
       "     {'t-sne_x': -13.505736111459525, 't-sne_y': -3.366886724219164},\n",
       "     {'umap_x': 0.33822086453437805, 'umap_y': 15.103503227233887}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.0042463792487978935,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.0047111366875469685,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.005530943628400564,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.005544283427298069,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.005604439415037632,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.005865094251930714,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.005890099331736565,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.006032796576619148,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.006157873198390007,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.0062273843213915825,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.00627465033903718,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.006326080299913883,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.006362593732774258,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.0066004302352666855,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.006639312021434307,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.006642885506153107,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.006812818814069033,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.007119214162230492,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.009280066005885601,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.009704364463686943,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48437891-T-A',\n",
       "    'metadata': {'vcf_string': '15\\t48437891\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48437891',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6190A>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48437891T>A',\n",
       "     'hgvs_protein': 'p.Lys2064Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"stop_gained\", \"end\": 48437891, \"transcript_consequences\": [{\"amino_acids\": \"K/*\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"strand\": -1, \"mutationtaster_pred\": \"A\", \"cds_start\": 6190, \"location\": \"15:48437891\", \"given_ref\": \"T\", \"cadd_raw\": 9.589022, \"allele_num\": 1, \"consequence_terms\": [\"stop_gained\"], \"ga4gh_vrs\": {\"location\": {\"interval\": {\"start\": 48437890, \"end\": 48437891, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Lys2064Ter\", \"codons\": \"Aag/Tag\", \"canonical\": 1, \"cadd_phred\": 42, \"mastermind_counts\": \"4|4|4\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.6190A>T\", \"aa\": \"T\", \"exon\": \"51/66\", \"lof\": \"HC\", \"spliceai\": {\"DS_AL\": 0.25, \"DP_DL\": -12, \"SYMBOL\": \"FBN1\", \"DP_AG\": -3, \"DS_DL\": 0, \"DP_DG\": 26, \"DP_AL\": 26, \"DS_AG\": 0.01, \"DS_DG\": 0}, \"variant_allele\": \"A\", \"lof_info\": \"PERCENTILE:0.718430826369545,GERP_DIST:2833.20045496225,BP_DIST:2411,DIST_FROM_LAST_EXON:2022,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"transcript_id\": \"NM_000138.5\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:K2064X\", \"cdna_start\": 6506, \"cdna_end\": 6506, \"lrt_pred\": \"invalid_field\", \"conservation\": 3.95, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"protein_end\": 2064, \"protein_start\": 2064, \"cds_end\": 6190, \"used_ref\": \"T\", \"mane_select\": \"ENST00000316623.10\"}], \"start\": 48437891, \"id\": \"NC_000015.10:g.48437891T>A\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM085412\", \"end\": 48437891, \"start\": 48437891, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1}], \"variant_class\": \"SNV\", \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48437891T>A\", \"vcf_string\": \"15-48437891-T-A\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"T/A\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.018969765145006187,\n",
       "      'pca_y': 0.0953084745880412},\n",
       "     {'t-sne_x': 18.07831967744226, 't-sne_y': 49.92491260322327},\n",
       "     {'umap_x': -9.47569751739502, 'umap_y': 5.574869632720947}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48448879-G-A',\n",
       "      'distance': 0.003917176276445389,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428361-G-A',\n",
       "      'distance': 0.004296570550650358,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644643-T-A',\n",
       "      'distance': 0.004372989758849144,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465679-G-A',\n",
       "      'distance': 0.0046358732506632805,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437376-G-A',\n",
       "      'distance': 0.004679277539253235,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.0047081345692276955,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513579-G-A',\n",
       "      'distance': 0.004787798039615154,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425474-G-A',\n",
       "      'distance': 0.0048949262127280235,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420714-G-A',\n",
       "      'distance': 0.004945278633385897,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520737-G-A',\n",
       "      'distance': 0.004958792589604855,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432971-G-A',\n",
       "      'distance': 0.0049754842184484005,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427630-G-A',\n",
       "      'distance': 0.005109146237373352,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.005357092246413231,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48467954-A-T',\n",
       "      'distance': 0.00539219519123435,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.00569380447268486,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.00573063874617219,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520764-G-A',\n",
       "      'distance': 0.005744258873164654,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465622-G-A',\n",
       "      'distance': 0.005776769481599331,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.005855233874171972,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445430-G-A',\n",
       "      'distance': 0.005996575579047203,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48437322-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48437322\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48437322',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6379G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48437322C>A',\n",
       "     'hgvs_protein': 'p.Asp2127Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48437322C>A\", \"vcf_string\": \"15-48437322-C-A\", \"allele_string\": \"C/A\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"end\": 48437322, \"transcript_consequences\": [{\"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 2127, \"cds_end\": 6379, \"sift_prediction\": \"deleterious_low_confidence\", \"maxentscan_alt\": -2.374, \"polyphen_score\": 0.999, \"conservation\": 3.95, \"loeuf\": 0.105, \"protein_end\": 2127, \"phaplo\": 0.99897615427687, \"sift_score\": 0.01, \"rf_score\": 0.986, \"mastermind_mmid3\": \"FBN1:D2127Y,FBN1:D2127srd\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_start\": 6695, \"cdna_end\": 6695, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.948}, \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"A\", \"gene_symbol_source\": \"EntrezGene\", \"hgvsc\": \"NM_000138.5:c.6379G>T\", \"biotype\": \"protein_coding\", \"canonical\": 1, \"maxentscan_ref\": 5.069, \"codons\": \"Gat/Tat\", \"mastermind_counts\": \"8|21|27\", \"cadd_phred\": 35, \"spliceai\": {\"DP_DG\": -35, \"DP_AL\": 0, \"DS_AG\": 0, \"DS_DG\": 0.01, \"DS_AL\": 0, \"DP_DL\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": -35, \"DS_DL\": 0.28}, \"aa\": \"C\", \"exon\": \"52/66\", \"blosum62\": -3, \"maxentscan_diff\": 7.444, \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.986808203304025, \"location\": \"15:48437322\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48437322, \"start\": 48437321, \"type\": \"SimpleInterval\"}}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Asp2127Tyr\", \"cadd_raw\": 6.772929, \"given_ref\": \"C\", \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"allele_num\": 1, \"ada_score\": 0.999980866958532, \"amino_acids\": \"D/Y\", \"mutationtaster_pred\": \"D\", \"strand\": -1, \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.048319, \"dG_mt_sd\": 0.001923, \"dG_wt\": 189.4806, \"ddG\": 0.1656, \"dG_wt_sd\": 0.048345, \"dG_mt\": 189.6462}}, \"cds_start\": 6379, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396}], \"start\": 48437322, \"id\": \"NC_000015.10:g.48437322C>A\", \"colocated_variants\": [{\"end\": 48437322, \"start\": 48437322, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"id\": \"CM003140\"}, {\"end\": 48437322, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48437322, \"id\": \"CM154850\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.041773819783007135,\n",
       "      'pca_y': 0.06133586169209814},\n",
       "     {'t-sne_x': -12.589626152710158, 't-sne_y': -4.196973859330111},\n",
       "     {'umap_x': 0.35451018810272217, 'umap_y': 15.067383766174316}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.003238423727452755,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.004575969651341438,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.004968026652932167,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.005052007269114256,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.005438539199531078,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.006039607338607311,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.006142430007457733,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.006267853081226349,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.006354938726872206,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.006501025520265102,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.0066245486959815025,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.007007400505244732,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.007292980793863535,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.0074784159660339355,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.0075316037982702255,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.0076300399377942085,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.008166561834514141,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.008426256477832794,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.008698818273842335,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.00918971560895443,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48437076-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48437076\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48437076',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6381T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48437076A>T',\n",
       "     'hgvs_protein': 'p.Asp2127Glu',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48437076A>T\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM940768\", \"end\": 48437076, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48437076}], \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"variant_allele\": \"T\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"codons\": \"gaT/gaA\", \"maxentscan_ref\": 6.738, \"canonical\": 1, \"mastermind_counts\": \"8|21|27\", \"cadd_phred\": 20.2, \"hgvsc\": \"NM_000138.5:c.6381T>A\", \"biotype\": \"protein_coding\", \"aa\": \"A\", \"exon\": \"53/66\", \"blosum62\": 2, \"spliceai\": {\"DP_DL\": -24, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AG\": -17, \"DS_DL\": 0, \"DP_AL\": 1, \"DP_DG\": 1, \"DS_AG\": 0, \"DS_DG\": 0}, \"location\": \"15:48437076\", \"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.976185739040375, \"maxentscan_diff\": 1.212, \"given_ref\": \"A\", \"cadd_raw\": 2.769444, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"allele_num\": 1, \"hgvsp\": \"NP_000129.3:p.Asp2127Glu\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"start\": 48437075, \"end\": 48437076, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"amino_acids\": \"D/E\", \"ada_score\": 6.84744934471113e-05, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"strand\": -1, \"mutationtaster_pred\": \"D\", \"cds_start\": 6381, \"mutfunc\": {\"exp\": {\"dG_mt\": 189.861, \"ddG\": 0.4094, \"dG_wt_sd\": 0.038591, \"dG_mt_sd\": 0.013601, \"ddG_sd\": 0.045019, \"dG_wt\": 189.4516}}, \"cds_end\": 6381, \"protein_start\": 2127, \"sift_prediction\": \"deleterious_low_confidence\", \"maxentscan_alt\": 5.526, \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.993, \"conservation\": -5.51, \"loeuf\": 0.105, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 2127, \"mane\": [\"MANE_Select\"], \"rf_score\": 0.148, \"mastermind_mmid3\": \"FBN1:D2127E,FBN1:D2127sra\", \"cdna_end\": 6697, \"cdna_start\": 6697, \"lrt_pred\": \"invalid_field\", \"alphamissense\": {\"am_pathogenicity\": 0.9767, \"am_class\": \"likely_pathogenic\"}, \"transcript_id\": \"NM_000138.5\"}], \"end\": 48437076, \"start\": 48437076, \"vcf_string\": \"15-48437076-A-T\", \"allele_string\": \"A/T\", \"assembly_name\": \"GRCh38\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48437076A>T\"}',\n",
       "    'coordinates': [{'pca_x': -0.03053756405158052,\n",
       "      'pca_y': 0.04848446492235913},\n",
       "     {'t-sne_x': -12.799769117978968, 't-sne_y': -2.3151799471136227},\n",
       "     {'umap_x': 0.3940017819404602, 'umap_y': 15.179849624633789}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.005427972879260778,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.005862894933670759,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.005906186997890472,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.005984783638268709,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.006230891682207584,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.006660319399088621,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.006955934688448906,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.006961541716009378,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.007091201841831207,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.007263904437422752,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.007304513361304998,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.007743408903479576,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.007870258763432503,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.008564373478293419,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.008614836260676384,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.009437578730285168,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.009459668770432472,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.009475257247686386,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.010757096111774445,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.011655235663056374,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48434630-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48434630\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48434630',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6580G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48434630C>A',\n",
       "     'hgvs_protein': 'p.Glu2194Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"most_severe_consequence\": \"stop_gained\", \"variant_class\": \"SNV\", \"vcf_string\": \"15-48434630-C-A\", \"minimised\": 1, \"allele_string\": \"C/A\", \"strand\": 1, \"end\": 48434630, \"input\": \"NC_000015.10:g.48434630C>A\", \"id\": \"NC_000015.10:g.48434630C>A\", \"transcript_consequences\": [{\"loeuf\": 0.105, \"variant_allele\": \"A\", \"amino_acids\": \"E/*\", \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"protein_start\": 2194, \"codons\": \"Gag/Tag\", \"cds_end\": 6580, \"lrt_pred\": \"invalid_field\", \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"impact\": \"HIGH\", \"conservation\": 3.95, \"exon\": \"54/66\", \"cadd_phred\": 46, \"hgvsc\": \"NM_000138.5:c.6580G>T\", \"gene_id\": \"2200\", \"mutationtaster_pred\": \"A\", \"mastermind_mmid3\": \"FBN1:E2194X\", \"lof_info\": \"PERCENTILE:0.763695450324977,GERP_DIST:2287.11669742465,BP_DIST:2024,DIST_FROM_LAST_EXON:1635,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"protein_end\": 2194, \"used_ref\": \"C\", \"cds_start\": 6580, \"cdna_start\": 6896, \"consequence_terms\": [\"stop_gained\"], \"location\": \"15:48434630\", \"hgvsp\": \"NP_000129.3:p.Glu2194Ter\", \"aa\": \"C\", \"cdna_end\": 6896, \"cadd_raw\": 10.217942, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"allele_num\": 1, \"mastermind_counts\": \"4|4|4\", \"mutfunc\": {\"exp\": {\"dG_wt_sd\": 0.000447, \"dG_mt_sd\": 0.096185, \"dG_mt\": 191.7902, \"dG_wt\": 192.3158, \"ddG\": -0.5256, \"ddG_sd\": 0.096632}}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"start\": 48434629, \"type\": \"SimpleInterval\", \"end\": 48434630}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}}, \"given_ref\": \"C\", \"spliceai\": {\"DS_AG\": 0, \"DP_DL\": -36, \"DP_DG\": 2, \"DS_DL\": 0.02, \"DP_AL\": 33, \"DP_AG\": -36, \"DS_AL\": 0, \"DS_DG\": 0.01, \"SYMBOL\": \"FBN1\"}, \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"gene_symbol_source\": \"EntrezGene\"}], \"start\": 48434630, \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM116657\", \"end\": 48434630, \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48434630, \"phenotype_or_disease\": 1}, {\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48434630, \"end\": 48434630, \"id\": \"COSV99075941\", \"allele_string\": \"COSMIC_MUTATION\", \"somatic\": 1, \"strand\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM7367852\"]}}]}',\n",
       "    'coordinates': [{'pca_x': 0.015078809854792763,\n",
       "      'pca_y': 0.09555462488116362},\n",
       "     {'t-sne_x': 21.69556702307692, 't-sne_y': 42.744220556851495},\n",
       "     {'umap_x': -9.581673622131348, 'umap_y': 4.490502834320068}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48505139-C-A',\n",
       "      'distance': 0.0017590952338650823,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.0018357874359935522,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.002099142409861088,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452676-C-A',\n",
       "      'distance': 0.0021200734190642834,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411085-C-A',\n",
       "      'distance': 0.002178216353058815,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.0022618353832513094,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515397-C-T',\n",
       "      'distance': 0.00256253476254642,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.002863472793251276,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.0028730938211083412,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437837-C-A',\n",
       "      'distance': 0.002989994129166007,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.0032699061557650566,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483878-C-A',\n",
       "      'distance': 0.003305502235889435,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.0034654263872653246,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488233-C-A',\n",
       "      'distance': 0.0036565689370036125,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48534170-G-A',\n",
       "      'distance': 0.003722875379025936,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415675-G-A',\n",
       "      'distance': 0.0038011549040675163,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415540-G-A',\n",
       "      'distance': 0.0038953206967562437,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.004109037108719349,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.00417025201022625,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48513615-G-A',\n",
       "      'distance': 0.004201946780085564,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48434627-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48434627\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48434627',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6583G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48434627C>A',\n",
       "     'hgvs_protein': 'p.Gly2195Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48434627C>A\", \"end\": 48434627, \"id\": \"NC_000015.10:g.48434627C>A\", \"allele_string\": \"C/A\", \"strand\": 1, \"colocated_variants\": [{\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48434627, \"id\": \"CM013939\", \"end\": 48434627, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1}, {\"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM074826\", \"end\": 48434627, \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48434627, \"phenotype_or_disease\": 1}, {\"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"allele_string\": \"C/A/T\", \"end\": 48434627, \"id\": \"rs886038976\", \"strand\": 1, \"var_synonyms\": {\"UniProt\": [\"VAR_076123\"], \"ClinVar\": [\"RCV001388990\", \"RCV000663881\", \"VCV000549351\", \"RCV000700740\", \"RCV000663880\", \"VCV000263922\", \"RCV002310907\"]}, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"pubmed\": [11700157, 17657824], \"start\": 48434627, \"clin_sig_allele\": \"T:pathogenic;A:pathogenic;T:likely_pathogenic\"}], \"transcript_consequences\": [{\"given_ref\": \"C\", \"spliceai\": {\"DP_AL\": 3, \"DS_DL\": 0, \"DP_DG\": -27, \"DS_AG\": 0, \"DP_DL\": -33, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0, \"DP_AG\": 36}, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"interval\": {\"end\": 48434627, \"type\": \"SimpleInterval\", \"start\": 48434626}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"mutfunc\": {\"exp\": {\"dG_wt_sd\": 0.002738, \"dG_mt_sd\": 0.003563, \"dG_wt\": 192.312, \"dG_mt\": 192.505799, \"ddG\": 0.193799, \"ddG_sd\": 0.001303}}, \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_end\": 6899, \"location\": \"15:48434627\", \"hgvsp\": \"NP_000129.3:p.Gly2195Ter\", \"aa\": \"C\", \"mastermind_counts\": \"6|7|7\", \"allele_num\": 1, \"cadd_raw\": 9.878431, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cdna_start\": 6899, \"consequence_terms\": [\"stop_gained\"], \"used_ref\": \"C\", \"cds_start\": 6583, \"mastermind_mmid3\": \"FBN1:G2195X\", \"gene_id\": \"2200\", \"mutationtaster_pred\": \"A\", \"protein_end\": 2195, \"lof_info\": \"PERCENTILE:0.764043639740019,GERP_DIST:2283.99669753909,BP_DIST:2021,DIST_FROM_LAST_EXON:1632,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"cadd_phred\": 43, \"impact\": \"HIGH\", \"conservation\": 3.95, \"exon\": \"54/66\", \"hgvsc\": \"NM_000138.5:c.6583G>T\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"cds_end\": 6583, \"protein_start\": 2195, \"codons\": \"Gga/Tga\", \"lrt_pred\": \"invalid_field\", \"loeuf\": 0.105, \"variant_allele\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"G/*\"}], \"start\": 48434627, \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"stop_gained\", \"minimised\": 1, \"vcf_string\": \"15-48434627-C-A\"}',\n",
       "    'coordinates': [{'pca_x': 0.01831489491233368,\n",
       "      'pca_y': 0.10023719484439203},\n",
       "     {'t-sne_x': 21.46991317948357, 't-sne_y': 40.32147542084052},\n",
       "     {'umap_x': -9.444323539733887, 'umap_y': 5.172585487365723}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.0027599819004535675,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.0033556241542100906,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.003860867116600275,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.004045099951326847,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.004754746798425913,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.004790583625435829,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415540-G-A',\n",
       "      'distance': 0.004866335541009903,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425829-G-A',\n",
       "      'distance': 0.004896092228591442,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.004950238391757011,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.0050046080723404884,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465820-G-A',\n",
       "      'distance': 0.005130586214363575,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.00517461821436882,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.005200780462473631,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432947-G-A',\n",
       "      'distance': 0.005206324625760317,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427630-G-A',\n",
       "      'distance': 0.005259444937109947,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48412715-G-A',\n",
       "      'distance': 0.005268947221338749,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415675-G-A',\n",
       "      'distance': 0.005308225750923157,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.00534074055030942,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468064-G-A',\n",
       "      'distance': 0.0054212091490626335,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425423-G-A',\n",
       "      'distance': 0.0054306755773723125,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48430769-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48430769\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48430769',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6773G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48430769C>T',\n",
       "     'hgvs_protein': 'p.Cys2258Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48430769-C-T\", \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"cds_start\": 6773, \"eve_score\": 0.787055525993301, \"used_ref\": \"C\", \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.998073600287496, \"cdna_start\": 7089, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cadd_raw\": 5.202957, \"mastermind_counts\": \"8|9|9\", \"allele_num\": 1, \"aa\": \"C\", \"location\": \"15:48430769\", \"hgvsp\": \"NP_000129.3:p.Cys2258Tyr\", \"cdna_end\": 7089, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48430768, \"type\": \"SimpleInterval\", \"end\": 48430769}}}, \"spliceai\": {\"DP_DL\": 1, \"DS_AG\": 0, \"DP_DG\": 3, \"DP_AL\": 33, \"DS_DL\": 0, \"DP_AG\": 22, \"DS_DG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\"}, \"given_ref\": \"C\", \"amino_acids\": \"C/Y\", \"canonical\": 1, \"polyphen_score\": 0.996, \"mane_select\": \"ENST00000316623.10\", \"loeuf\": 0.105, \"variant_allele\": \"T\", \"lrt_pred\": \"invalid_field\", \"protein_start\": 2258, \"cds_end\": 6773, \"codons\": \"tGt/tAt\", \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"sift_prediction\": \"deleterious_low_confidence\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"hgvsc\": \"NM_000138.5:c.6773G>A\", \"eve_class\": \"Pathogenic\", \"exon\": \"56/66\", \"conservation\": 3.95, \"impact\": \"MODERATE\", \"polyphen_prediction\": \"probably_damaging\", \"cadd_phred\": 29.2, \"sift_score\": 0, \"alphamissense\": {\"am_pathogenicity\": 0.9984, \"am_class\": \"likely_pathogenic\"}, \"gene_symbol\": \"FBN1\", \"protein_end\": 2258, \"mutationtaster_pred\": \"A\", \"gene_id\": \"2200\", \"blosum62\": -2, \"mastermind_mmid3\": \"FBN1:C2258Y\"}], \"start\": 48430769, \"colocated_variants\": [{\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48430769, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48430769, \"id\": \"CD152330\", \"strand\": 1}, {\"strand\": 1, \"end\": 48430769, \"id\": \"CM993161\", \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"start\": 48430769, \"seq_region_name\": \"15\"}, {\"seq_region_name\": \"15\", \"start\": 48430769, \"phenotype_or_disease\": 1, \"id\": \"COSV100356076\", \"end\": 48430769, \"allele_string\": \"COSMIC_MUTATION\", \"somatic\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM7669690\"]}, \"strand\": 1}, {\"somatic\": 1, \"strand\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM6505615\"]}, \"end\": 48430769, \"id\": \"COSV57328175\", \"allele_string\": \"COSMIC_MUTATION\", \"start\": 48430769, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"phenotype_or_disease\": 1, \"start\": 48430769, \"clin_sig_allele\": \"T:likely_pathogenic\", \"seq_region_name\": \"15\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV002310990\", \"VCV000264106\", \"RCV004725131\"], \"UniProt\": [\"VAR_076131\"]}, \"clin_sig\": [\"likely_pathogenic\"], \"allele_string\": \"C/T\", \"end\": 48430769, \"id\": \"rs886039047\"}], \"allele_string\": \"C/T\", \"strand\": 1, \"id\": \"NC_000015.10:g.48430769C>T\", \"input\": \"NC_000015.10:g.48430769C>T\", \"end\": 48430769}',\n",
       "    'coordinates': [{'pca_x': 0.037240656348638546,\n",
       "      'pca_y': 0.08116129636351344},\n",
       "     {'t-sne_x': 8.047000775134931, 't-sne_y': 11.541716442138814},\n",
       "     {'umap_x': 3.0202858448028564, 'umap_y': 2.9191014766693115}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48420794-C-T',\n",
       "      'distance': 0.0011757061583921313,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437827-C-T',\n",
       "      'distance': 0.001456632511690259,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434599-C-T',\n",
       "      'distance': 0.0017900331877171993,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48467997-C-T',\n",
       "      'distance': 0.001797902281396091,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537676-C-T',\n",
       "      'distance': 0.0021274560131132603,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415695-C-T',\n",
       "      'distance': 0.0022524797823280096,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.002307749819010496,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430676-C-T',\n",
       "      'distance': 0.0023631167132407427,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428390-C-T',\n",
       "      'distance': 0.0024129608646035194,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420737-C-T',\n",
       "      'distance': 0.002442330587655306,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.0024630750995129347,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468456-C-T',\n",
       "      'distance': 0.002464109333232045,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.0024907016195356846,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468462-C-T',\n",
       "      'distance': 0.002537167863920331,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427602-C-T',\n",
       "      'distance': 0.00261647324077785,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.002639255253598094,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470684-C-T',\n",
       "      'distance': 0.002710767090320587,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.0027127100620418787,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456687-C-T',\n",
       "      'distance': 0.0027393950149416924,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0027398099191486835,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48428437-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48428437\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48428437',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6906T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48428437A>T',\n",
       "     'hgvs_protein': 'p.Cys2302Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"mastermind_mmid3\": \"FBN1:C2302X\", \"mutationtaster_pred\": \"A\", \"gene_id\": \"2200\", \"protein_end\": 2302, \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.801532033426184,GERP_DIST:1937.2890933156,BP_DIST:1701,DIST_FROM_LAST_EXON:1312,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"cadd_phred\": 36, \"conservation\": 0.08, \"exon\": \"57/66\", \"impact\": \"HIGH\", \"hgvsc\": \"NM_000138.5:c.6906T>A\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"lrt_pred\": \"invalid_field\", \"cds_end\": 6906, \"codons\": \"tgT/tgA\", \"protein_start\": 2302, \"loeuf\": 0.105, \"variant_allele\": \"T\", \"genesplicer\": \"loss/donor/48428436-48428437/Medium/2.773375\", \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"C/*\", \"spliceai\": {\"DP_AG\": 36, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0.01, \"DS_AL\": 0.01, \"DP_DL\": 2, \"DS_AG\": 0, \"DP_AL\": 34, \"DS_DL\": 0, \"DP_DG\": -40}, \"given_ref\": \"A\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48428436, \"end\": 48428437, \"type\": \"SimpleInterval\"}}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}}, \"gene_symbol_source\": \"EntrezGene\", \"lof\": \"HC\", \"mane\": [\"MANE_Select\"], \"cdna_end\": 7222, \"aa\": \"A\", \"location\": \"15:48428437\", \"hgvsp\": \"NP_000129.3:p.Cys2302Ter\", \"allele_num\": 1, \"mastermind_counts\": \"3|3|3\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cadd_raw\": 7.519888, \"cdna_start\": 7222, \"consequence_terms\": [\"stop_gained\"], \"used_ref\": \"A\", \"cds_start\": 6906}], \"start\": 48428437, \"colocated_variants\": [{\"id\": \"CM118909\", \"end\": 48428437, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48428437}], \"strand\": 1, \"allele_string\": \"A/T\", \"id\": \"NC_000015.10:g.48428437A>T\", \"end\": 48428437, \"input\": \"NC_000015.10:g.48428437A>T\", \"vcf_string\": \"15-48428437-A-T\", \"minimised\": 1, \"most_severe_consequence\": \"stop_gained\", \"seq_region_name\": \"15\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.02548566421415563,\n",
       "      'pca_y': 0.08819068031010746},\n",
       "     {'t-sne_x': 25.130622069112814, 't-sne_y': 37.323798551561},\n",
       "     {'umap_x': -9.813299179077148, 'umap_y': 2.6221399307250977}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.001493021147325635,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537615-A-T',\n",
       "      'distance': 0.0029224962927401066,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.003591889748349786,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48467954-A-T',\n",
       "      'distance': 0.003769317641854286,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537696-C-T',\n",
       "      'distance': 0.004038355778902769,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.004126116633415222,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.005591375753283501,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483878-C-A',\n",
       "      'distance': 0.005776537582278252,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437805-C-T',\n",
       "      'distance': 0.006358706392347813,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.00656126718968153,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.006639366969466209,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.0067020561546087265,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.006761620286852121,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.0069241710007190704,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421601-G-T',\n",
       "      'distance': 0.006989415735006332,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.007102101109921932,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.0071055893786251545,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.007254589349031448,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485483-G-T',\n",
       "      'distance': 0.00726445484906435,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495241-G-T',\n",
       "      'distance': 0.0073478445410728455,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48428424-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48428424\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48428424',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6919T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48428424A>G',\n",
       "     'hgvs_protein': 'p.Cys2307Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"vcf_string\": \"15-48428424-A-G\", \"variant_class\": \"SNV\", \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"colocated_variants\": [{\"start\": 48428424, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48428424, \"id\": \"CM098719\", \"allele_string\": \"HGMD_MUTATION\"}], \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"variant_allele\": \"G\", \"loeuf\": 0.105, \"polyphen_score\": 0.997, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"C/R\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"sift_prediction\": \"deleterious_low_confidence\", \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"codons\": \"Tgc/Cgc\", \"cds_end\": 6919, \"protein_start\": 2307, \"lrt_pred\": \"invalid_field\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9987}, \"sift_score\": 0, \"cadd_phred\": 28.9, \"impact\": \"MODERATE\", \"polyphen_prediction\": \"probably_damaging\", \"exon\": \"57/66\", \"conservation\": 3.95, \"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.6919T>C\", \"mastermind_mmid3\": \"FBN1:C2307R\", \"gene_id\": \"2200\", \"blosum62\": -3, \"mutationtaster_pred\": \"D\", \"protein_end\": 2307, \"gene_symbol\": \"FBN1\", \"used_ref\": \"A\", \"eve_score\": 0.78517222711324, \"cds_start\": 6919, \"cdna_start\": 7235, \"clinpred\": 0.999852538108826, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 7235, \"location\": \"15:48428424\", \"hgvsp\": \"NP_000129.3:p.Cys2307Arg\", \"aa\": \"A\", \"allele_num\": 1, \"mastermind_counts\": \"4|5|5\", \"cadd_raw\": 5.161177, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"given_ref\": \"A\", \"spliceai\": {\"DP_AG\": 47, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0.01, \"DS_AL\": 0, \"DS_AG\": 0, \"DP_DL\": -33, \"DP_AL\": 48, \"DS_DL\": 0, \"DP_DG\": -27}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48428423, \"type\": \"SimpleInterval\", \"end\": 48428424}}}, \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\"}], \"start\": 48428424, \"id\": \"NC_000015.10:g.48428424A>G\", \"input\": \"NC_000015.10:g.48428424A>G\", \"end\": 48428424, \"allele_string\": \"A/G\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.046853919775553476,\n",
       "      'pca_y': 0.0781673793060817},\n",
       "     {'t-sne_x': 30.019791366815312, 't-sne_y': 4.684361668487362},\n",
       "     {'umap_x': 2.2123501300811768, 'umap_y': 6.910488605499268}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48434702-A-G',\n",
       "      'distance': 0.0012852093204855919,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.0013014208525419235,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.0013262477004900575,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.0013266170863062143,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.0014154808595776558,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.0014334123115986586,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.0015187383396551013,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0015496739652007818,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420795-A-G',\n",
       "      'distance': 0.0015732134925201535,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0016182921826839447,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427603-A-G',\n",
       "      'distance': 0.0016496411990374327,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.0017627931665629148,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.001763826934620738,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470679-A-G',\n",
       "      'distance': 0.0017704716883599758,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.0018099015578627586,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515396-C-A',\n",
       "      'distance': 0.001817492302507162,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.0018626560922712088,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474567-A-G',\n",
       "      'distance': 0.0018645928939804435,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437873-A-G',\n",
       "      'distance': 0.0018779071979224682,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0018981900066137314,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48427730-CAT-C',\n",
       "    'metadata': {'vcf_string': '15\\t48427730\\t.\\tCAT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48427730',\n",
       "     'ref_allele': 'CAT',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7039_7040del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48427731_48427732del',\n",
       "     'hgvs_protein': 'p.Met2347ValfsTer19',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"end\": 48427732, \"strand\": 1, \"colocated_variants\": [{\"id\": \"rs794728319\", \"start\": 48427731, \"pubmed\": [11826022, 17627385, 18435798, 19863550, 17657824, 19618372], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV001192804\", \"RCV001775090\", \"RCV000755190\", \"RCV000613882\", \"RCV000208062\", \"RCV000181674\", \"VCV000200171\", \"RCV000524503\"]}, \"end\": 48427732, \"strand\": 1, \"allele_string\": \"AT/-\", \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"phenotype_or_disease\": 1}], \"start\": 48427731, \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"vcf_string\": \"15-48427730-CAT-C\", \"minimised\": 1, \"allele_string\": \"AT/-\", \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"strand\": -1, \"protein_start\": 2347, \"cdna_start\": 7355, \"hgvsc\": \"NM_000138.5:c.7039_7040del\", \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"58/66\", \"conservation\": 2.79, \"biotype\": \"protein_coding\", \"cds_end\": 7040, \"protein_end\": 2347, \"hgvsp\": \"NP_000129.3:p.Met2347ValfsTer19\", \"canonical\": 1, \"codons\": \"ATg/g\", \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"aa\": \"AT\", \"cds_start\": 7039, \"mastermind_mmid3\": \"FBN1:M2347fs\", \"phaplo\": 0.99897615427687, \"protein_id\": \"NP_000129.3\", \"geno2mp_hpo_count\": 1, \"mastermind_counts\": \"24|26|26\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"given_ref\": \"AT\", \"gene_symbol\": \"FBN1\", \"amino_acids\": \"M/X\", \"lof_info\": \"PERCENTILE:0.817084493964717,GERP_DIST:1844.90151258707,BP_DIST:1568,DIST_FROM_LAST_EXON:1179,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"location\": \"15:48427731-48427732\", \"loeuf\": 0.105, \"gene_id\": \"2200\", \"spliceai\": {\"DS_AG\": 0, \"DS_DG\": 0, \"DP_DL\": 1, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AL\": 43, \"SYMBOL\": \"FBN1\", \"DP_DG\": -1, \"DP_AG\": -18}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48427732, \"type\": \"SimpleInterval\", \"start\": 48427730}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"geno2mp_url\": \"https://geno2mp.gs.washington.edu/Geno2MP/#/variant/15/48719927/CAT%3EC/indel\", \"impact\": \"HIGH\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"AT\", \"cdna_end\": 7356, \"variant_allele\": \"-\"}], \"input\": \"NC_000015.10:g.48427731_48427732del\", \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48427731_48427732del\"}',\n",
       "    'coordinates': [{'pca_x': 0.03916983530021617,\n",
       "      'pca_y': 0.028234853668981534},\n",
       "     {'t-sne_x': 7.562905060803371, 't-sne_y': 4.300049100392648},\n",
       "     {'umap_x': 4.0162272453308105, 'umap_y': 3.0683913230895996}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.10678260773420334,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10703763365745544,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.10723038762807846,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.1075318306684494,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10817430913448334,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10834024846553802,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.1086883470416069,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.10903526842594147,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.1094226986169815,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.10949748754501343,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.10959359258413315,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.10967011749744415,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.10972066223621368,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.10995172709226608,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.10999564826488495,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.11012028157711029,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.11031933128833771,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.11034991592168808,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.11040788888931274,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.11046856641769409,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48427729-ACA-A',\n",
       "    'metadata': {'vcf_string': '15\\t48427729\\t.\\tACA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48427729',\n",
       "     'ref_allele': 'ACA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7043_7044del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48427730_48427731del',\n",
       "     'hgvs_protein': 'p.Cys2348SerfsTer18',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48427730_48427731del\", \"variant_class\": \"deletion\", \"transcript_consequences\": [{\"protein_start\": 2348, \"hgvsc\": \"NM_000138.5:c.7043_7044del\", \"cdna_start\": 7359, \"strand\": -1, \"hgvs_offset\": -3, \"cds_end\": 7044, \"protein_end\": 2348, \"hgvsp\": \"NP_000129.3:p.Cys2348SerfsTer18\", \"canonical\": 1, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"58/66\", \"conservation\": 2.915, \"biotype\": \"protein_coding\", \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"aa\": \"CA\", \"codons\": \"tGT/t\", \"protein_id\": \"NP_000129.3\", \"cds_start\": 7043, \"phaplo\": 0.99897615427687, \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"C/X\", \"lof_info\": \"PERCENTILE:0.817548746518106,GERP_DIST:1843.47838779688,BP_DIST:1567,DIST_FROM_LAST_EXON:1178,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"CA\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"spliceai\": {\"DS_AG\": 0, \"DS_DG\": 0, \"DP_DL\": 2, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AL\": 44, \"SYMBOL\": \"FBN1\", \"DP_DG\": 0, \"DP_AG\": -17}, \"location\": \"15:48427727-48427728\", \"loeuf\": 0.105, \"used_ref\": \"AC\", \"variant_allele\": \"-\", \"cdna_end\": 7360, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48427729, \"type\": \"SimpleInterval\", \"end\": 48427731}, \"type\": \"SequenceLocation\"}}, \"impact\": \"HIGH\", \"mane\": [\"MANE_Select\"]}], \"most_severe_consequence\": \"frameshift_variant\", \"input\": \"NC_000015.10:g.48427730_48427731del\", \"allele_string\": \"CA/-\", \"minimised\": 1, \"vcf_string\": \"15-48427729-ACA-A\", \"seq_region_name\": \"15\", \"assembly_name\": \"GRCh38\", \"start\": 48427730, \"strand\": 1, \"end\": 48427731}',\n",
       "    'coordinates': [{'pca_x': 0.03995681228412956,\n",
       "      'pca_y': 0.03330208138952153},\n",
       "     {'t-sne_x': 4.98752130707101, 't-sne_y': 8.425235099033019},\n",
       "     {'umap_x': 3.603438377380371, 'umap_y': 2.9907898902893066}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.09398685395717621,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.0980551540851593,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.09846927970647812,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.0995558574795723,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.09976667910814285,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.09994066506624222,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.10037664324045181,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10047371685504913,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.10057127475738525,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.10059395432472229,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.10093297064304352,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.10110613703727722,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10115456581115723,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.10125041007995605,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.10143371671438217,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48436970-C-A',\n",
       "      'distance': 0.10153274238109589,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.10155254602432251,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.10158468782901764,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.10164320468902588,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10167770087718964,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48421680-TT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48421680\\t.\\tTT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48421680',\n",
       "     'ref_allele': 'TT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7577del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48421681del',\n",
       "     'hgvs_protein': 'p.Asn2526MetfsTer156',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48421680-TT-T\", \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"start\": 48421681, \"colocated_variants\": [{\"var_synonyms\": {\"ClinVar\": [\"RCV000663964\", \"VCV000549417\"]}, \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48421681, \"id\": \"rs1555394218\", \"start\": 48421680, \"phenotype_or_disease\": 1, \"allele_string\": \"TT/T\", \"clin_sig\": [\"likely_pathogenic\"]}], \"strand\": 1, \"end\": 48421681, \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48421681del\", \"transcript_consequences\": [{\"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"exon\": \"62/66\", \"conservation\": 3.76, \"cds_end\": 7577, \"protein_end\": 2526, \"hgvsp\": \"NP_000129.3:p.Asn2526MetfsTer156\", \"canonical\": 1, \"hgvs_offset\": -1, \"strand\": -1, \"protein_start\": 2526, \"hgvsc\": \"NM_000138.5:c.7577del\", \"cdna_start\": 7893, \"cds_start\": 7577, \"phaplo\": 0.99897615427687, \"protein_id\": \"NP_000129.3\", \"codons\": \"aAt/at\", \"lof\": \"HC\", \"aa\": \"T\", \"transcript_id\": \"NM_000138.5\", \"gene_symbol\": \"FBN1\", \"given_ref\": \"T\", \"amino_acids\": \"N/X\", \"lof_info\": \"PERCENTILE:0.879410399257196,GERP_DIST:1163.96800452471,BP_DIST:1036,DIST_FROM_LAST_EXON:647,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48421681, \"start\": 48421680, \"type\": \"SimpleInterval\"}}}, \"mane\": [\"MANE_Select\"], \"impact\": \"HIGH\", \"used_ref\": \"T\", \"cdna_end\": 7893, \"variant_allele\": \"-\", \"location\": \"15:48421680\", \"loeuf\": 0.105, \"gene_id\": \"2200\", \"spliceai\": {\"DP_DG\": 6, \"DP_AG\": -10, \"SYMBOL\": \"FBN1\", \"DP_AL\": -38, \"DS_DL\": 0, \"DS_AL\": 0, \"DS_DG\": 0, \"DP_DL\": -38, \"DS_AG\": 0}}], \"most_severe_consequence\": \"frameshift_variant\", \"input\": \"NC_000015.10:g.48421681del\", \"allele_string\": \"T/-\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.043870781794298486,\n",
       "      'pca_y': 0.03090985262517732},\n",
       "     {'t-sne_x': 3.862373771158229, 't-sne_y': 8.15822709117375},\n",
       "     {'umap_x': 3.719743251800537, 'umap_y': 3.1980459690093994}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.09951155632734299,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.09993655979633331,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10080105811357498,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.10147732496261597,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.10201804339885712,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.10212597250938416,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10233042389154434,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.10241295397281647,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10249761492013931,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.10250000655651093,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.10268443077802658,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.10283266007900238,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.10283426940441132,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.10286031663417816,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.10340900719165802,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.10349113494157791,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.10352544486522675,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.1037161648273468,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489921-G-T',\n",
       "      'distance': 0.10375151038169861,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.10384945571422577,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48421616-GC-G',\n",
       "    'metadata': {'vcf_string': '15\\t48421616\\t.\\tGC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48421616',\n",
       "     'ref_allele': 'GC',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7640del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48421617del',\n",
       "     'hgvs_protein': 'p.Ser2547ThrfsTer135',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"start\": 48421617, \"vcf_string\": \"15-48421616-GC-G\", \"end\": 48421617, \"colocated_variants\": [{\"strand\": 1, \"end\": 48421617, \"seq_region_name\": \"15\", \"id\": \"CD040182\", \"start\": 48421617, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\"}], \"strand\": 1, \"id\": \"NC_000015.10:g.48421617del\", \"variant_class\": \"deletion\", \"allele_string\": \"C/-\", \"minimised\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"mastermind_counts\": \"1|1|1\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"given_ref\": \"C\", \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.886722376973073,GERP_DIST:1098.49708523005,BP_DIST:972,DIST_FROM_LAST_EXON:583,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"amino_acids\": \"S/X\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"location\": \"15:48421617\", \"gene_id\": \"2200\", \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DG\": 40, \"DP_AG\": 26, \"DP_DL\": 26, \"DS_DG\": 0, \"DS_AG\": 0.01, \"DP_AL\": -40, \"DS_AL\": 0, \"DS_DL\": 0}, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48421616, \"type\": \"SimpleInterval\", \"end\": 48421617}}}, \"impact\": \"HIGH\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"C\", \"variant_allele\": \"-\", \"cdna_end\": 7956, \"strand\": -1, \"protein_start\": 2547, \"cdna_start\": 7956, \"hgvsc\": \"NM_000138.5:c.7640del\", \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"62/66\", \"conservation\": 3.76, \"biotype\": \"protein_coding\", \"protein_end\": 2547, \"cds_end\": 7640, \"hgvsp\": \"NP_000129.3:p.Ser2547ThrfsTer135\", \"canonical\": 1, \"codons\": \"aGc/ac\", \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"aa\": \"C\", \"cds_start\": 7640, \"mastermind_mmid3\": \"FBN1:S2547fs\", \"phaplo\": 0.99897615427687, \"protein_id\": \"NP_000129.3\"}], \"input\": \"NC_000015.10:g.48421617del\"}',\n",
       "    'coordinates': [{'pca_x': -0.010628914946952561,\n",
       "      'pca_y': 0.040077749312992926},\n",
       "     {'t-sne_x': 3.850453704934879, 't-sne_y': 3.7612705002786133},\n",
       "     {'umap_x': 1.4507094621658325, 'umap_y': 11.60727596282959}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.1019091010093689,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48408466-T-C',\n",
       "      'distance': 0.10359830409288406,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.10385842621326447,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.10417163372039795,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.10457693785429001,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48409622-T-C',\n",
       "      'distance': 0.10512789338827133,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48408950-A-G',\n",
       "      'distance': 0.10540448129177094,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48644804-G-A',\n",
       "      'distance': 0.1056823879480362,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48409428-C-A',\n",
       "      'distance': 0.10574158281087875,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.10598684847354889,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430710-G-A',\n",
       "      'distance': 0.10640934854745865,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48410933-T-C',\n",
       "      'distance': 0.10659144818782806,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48411243-G-A',\n",
       "      'distance': 0.10661087185144424,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48410123-C-A',\n",
       "      'distance': 0.10666529834270477,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.10671694576740265,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.10673828423023224,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.10681018233299255,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.10682445764541626,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.10690651834011078,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.1069529801607132,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.6},\n",
       "     '10': {'pred_result': 'likely_benign', 'confidence_score': 0.6},\n",
       "     '15': {'pred_result': 'likely_benign',\n",
       "      'confidence_score': 0.6666666666666666},\n",
       "     '20': {'pred_result': 'uncertain', 'confidence_score': 0.5}}},\n",
       "   {'variant_id': '15-48420764-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48420764\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48420764',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7742G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48420764C>A',\n",
       "     'hgvs_protein': 'p.Cys2581Phe',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"end\": 48420764, \"strand\": 1, \"colocated_variants\": [{\"id\": \"CM013942\", \"start\": 48420764, \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48420764, \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1}, {\"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"end\": 48420764, \"seq_region_name\": \"15\", \"start\": 48420764, \"id\": \"CM077269\"}, {\"allele_string\": \"C/A/G/T\", \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"phenotype_or_disease\": 1, \"id\": \"rs1555394149\", \"start\": 48420764, \"pubmed\": [11826022, 11700157, 17701892], \"var_synonyms\": {\"ClinVar\": [\"RCV002314343\", \"VCV000519750\", \"RCV001898380\", \"VCV001393814\", \"RCV002397349\", \"RCV000663974\", \"VCV000549426\"], \"UniProt\": [\"VAR_018039\"]}, \"seq_region_name\": \"15\", \"strand\": 1, \"clin_sig_allele\": \"G:pathogenic;A:likely_pathogenic;T:likely_pathogenic\", \"end\": 48420764}], \"start\": 48420764, \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"vcf_string\": \"15-48420764-C-A\", \"minimised\": 1, \"allele_string\": \"C/A\", \"input\": \"NC_000015.10:g.48420764C>A\", \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"eve_class\": \"Pathogenic\", \"mutationtaster_pred\": \"A\", \"mastermind_counts\": \"2|6|6\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"blosum62\": -2, \"gene_symbol\": \"FBN1\", \"given_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"C/F\", \"ptriplo\": 0.984699554922396, \"alphamissense\": {\"am_pathogenicity\": 0.9981, \"am_class\": \"likely_pathogenic\"}, \"loeuf\": 0.105, \"location\": \"15:48420764\", \"gene_id\": \"2200\", \"eve_score\": 0.753149921256752, \"cadd_phred\": 31, \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DG\": -24, \"DP_AG\": 42, \"DS_AG\": 0, \"DP_DL\": 42, \"DS_DG\": 0, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AL\": 25}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48420763, \"type\": \"SimpleInterval\", \"end\": 48420764}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"mane\": [\"MANE_Select\"], \"impact\": \"MODERATE\", \"used_ref\": \"C\", \"variant_allele\": \"A\", \"cdna_end\": 8058, \"strand\": -1, \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 2581, \"sift_score\": 0, \"hgvsc\": \"NM_000138.5:c.7742G>T\", \"cdna_start\": 8058, \"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"conservation\": 3.77, \"exon\": \"63/66\", \"protein_end\": 2581, \"cds_end\": 7742, \"canonical\": 1, \"polyphen_score\": 0.996, \"hgvsp\": \"NP_000129.3:p.Cys2581Phe\", \"codons\": \"tGc/tTc\", \"aa\": \"C\", \"transcript_id\": \"NM_000138.5\", \"cadd_raw\": 5.38678, \"sift_prediction\": \"deleterious_low_confidence\", \"clinpred\": 0.996861696243286, \"mastermind_mmid3\": \"FBN1:C2581F\", \"lrt_pred\": \"invalid_field\", \"cds_start\": 7742, \"phaplo\": 0.99897615427687, \"protein_id\": \"NP_000129.3\"}], \"id\": \"NC_000015.10:g.48420764C>A\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.03814783106402969, 'pca_y': 0.098895173296985},\n",
       "     {'t-sne_x': 21.10367269181675, 't-sne_y': 11.274338935595242},\n",
       "     {'umap_x': 4.239465236663818, 'umap_y': 7.1208391189575195}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48425783-C-A',\n",
       "      'distance': 0.0014190612128004432,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468522-C-A',\n",
       "      'distance': 0.0019775014370679855,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.002026254776865244,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537721-C-A',\n",
       "      'distance': 0.002083532977849245,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465840-C-A',\n",
       "      'distance': 0.0020981905981898308,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.0021969545632600784,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485463-C-A',\n",
       "      'distance': 0.0023045623674988747,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472628-C-A',\n",
       "      'distance': 0.002405150793492794,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.0024276720359921455,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497358-C-A',\n",
       "      'distance': 0.0024594434071332216,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.0025003282353281975,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.00252167833968997,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463904-C-A',\n",
       "      'distance': 0.0025849551893770695,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503849-C-A',\n",
       "      'distance': 0.0026060035452246666,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.002609224058687687,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495513-C-A',\n",
       "      'distance': 0.0026110827457159758,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.002747079823166132,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48444579-C-A',\n",
       "      'distance': 0.0030459428671747446,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.0031277399975806475,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.003238509176298976,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48420715-G-GA',\n",
       "    'metadata': {'vcf_string': '15\\t48420715\\t.\\tG\\tGA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48420715',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'GA',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7790dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48420716dup',\n",
       "     'hgvs_protein': 'p.Gln2598ProfsTer10',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"allele_string\": \"-/A\", \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"codons\": \"ctc/ctTc\", \"biotype\": \"protein_coding\", \"amino_acids\": \"L/LX\", \"variant_allele\": \"A\", \"lof_info\": \"PERCENTILE:0.904131847725163,GERP_DIST:841.365824449062,BP_DIST:823,DIST_FROM_LAST_EXON:434,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"allele_num\": 1, \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"gene_id\": \"2200\", \"spliceai\": {\"DP_DG\": -31, \"DS_AG\": 0, \"DP_DL\": -28, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": 24, \"DS_AL\": 0, \"DP_AL\": -32, \"DS_DL\": 0.04}, \"hgvsp\": \"NP_000129.3:p.Gln2598ProfsTer10\", \"mastermind_mmid3\": \"FBN1:Q2598fs\", \"protein_id\": \"NP_000129.3\", \"protein_start\": 2597, \"cdna_start\": 8106, \"canonical\": 1, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48420715, \"type\": \"SimpleInterval\", \"start\": 48420715}, \"type\": \"SequenceLocation\"}}, \"exon\": \"63/66\", \"loeuf\": 0.105, \"cds_start\": 7790, \"cds_end\": 7791, \"strand\": -1, \"given_ref\": \"-\", \"conservation\": 0.955, \"mastermind_counts\": \"3|3|4\", \"ptriplo\": 0.984699554922396, \"location\": \"15:48420715-48420716\", \"phaplo\": 0.99897615427687, \"used_ref\": \"-\", \"consequence_terms\": [\"frameshift_variant\"], \"hgvsc\": \"NM_000138.5:c.7790dup\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_end\": 8107, \"impact\": \"HIGH\", \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"protein_end\": 2597}], \"most_severe_consequence\": \"frameshift_variant\", \"minimised\": 1, \"input\": \"NC_000015.10:g.48420716dup\", \"seq_region_name\": \"15\", \"start\": 48420716, \"id\": \"NC_000015.10:g.48420716dup\", \"end\": 48420715, \"vcf_string\": \"15-48420715-G-GA\", \"variant_class\": \"insertion\", \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': 0.027554486635473135,\n",
       "      'pca_y': 0.01477607901233146},\n",
       "     {'t-sne_x': 22.75963943808611, 't-sne_y': 15.768251291319919},\n",
       "     {'umap_x': 3.868267297744751, 'umap_y': 6.974175453186035}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.08566085994243622,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.08619062602519989,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.08654093742370605,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.08659852296113968,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.08689910173416138,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.08720064163208008,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452595-C-A',\n",
       "      'distance': 0.0873103141784668,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.0875357910990715,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.0876709520816803,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.08771700412034988,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-G',\n",
       "      'distance': 0.08774787187576294,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.08774973452091217,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.08805446326732635,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.08809062838554382,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437069-C-T',\n",
       "      'distance': 0.08818425238132477,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.08822644501924515,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.08833082020282745,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48421981-C-T',\n",
       "      'distance': 0.08834561705589294,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520708-C-A',\n",
       "      'distance': 0.08835865557193756,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.08837871253490448,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48420682-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48420682\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48420682',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7819+5G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48420682C>T',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_5th_base_variant'},\n",
       "    'annotation_raw': '{\"start\": 48420682, \"minimised\": 1, \"most_severe_consequence\": \"splice_donor_5th_base_variant\", \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48420682C>T\", \"transcript_consequences\": [{\"strand\": -1, \"given_ref\": \"C\", \"location\": \"15:48420682\", \"conservation\": 5.35, \"mastermind_counts\": \"5|8|35\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"maxentscan_diff\": 5.536, \"cadd_raw\": 1.987962, \"consequence_terms\": [\"splice_donor_5th_base_variant\", \"intron_variant\"], \"used_ref\": \"C\", \"cadd_phred\": 16.7, \"gene_symbol_source\": \"EntrezGene\", \"impact\": \"LOW\", \"hgvsc\": \"NM_000138.5:c.7819+5G>A\", \"mane_select\": \"ENST00000316623.10\", \"aa\": \"C\", \"gene_symbol\": \"FBN1\", \"mane\": [\"MANE_Select\"], \"biotype\": \"protein_coding\", \"variant_allele\": \"T\", \"maxentscan_alt\": -2.819, \"allele_num\": 1, \"intron\": \"63/65\", \"gene_id\": \"2200\", \"transcript_id\": \"NM_000138.5\", \"spliceai\": {\"DP_DG\": -36, \"DP_DL\": 5, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_DL\": 0.76, \"DS_AL\": 0, \"DP_AL\": 1, \"DP_AG\": -22}, \"maxentscan_ref\": 2.717, \"protein_id\": \"NP_000129.3\", \"mastermind_mmid3\": \"FBN1:D2607int,FBN1:D2607intd,FBN1:D2607srd\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48420682, \"type\": \"SimpleInterval\", \"start\": 48420681}, \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"loeuf\": 0.105, \"canonical\": 1}], \"allele_string\": \"C/T\", \"strand\": 1, \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"start\": 48420682, \"id\": \"CS085784\", \"end\": 48420682, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\"}, {\"id\": \"rs2141220566\", \"end\": 48420682, \"clin_sig_allele\": \"T:uncertain_significance;T:likely_pathogenic\", \"start\": 48420682, \"clin_sig\": [\"uncertain_significance\", \"likely_pathogenic\"], \"phenotype_or_disease\": 1, \"allele_string\": \"C/T\", \"var_synonyms\": {\"ClinVar\": [\"RCV001756598\", \"VCV001303099\", \"RCV005050402\"]}, \"seq_region_name\": \"15\", \"strand\": 1}], \"vcf_string\": \"15-48420682-C-T\", \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48420682C>T\", \"end\": 48420682}',\n",
       "    'coordinates': [{'pca_x': -0.2470440312842316,\n",
       "      'pca_y': -0.04123221938707643},\n",
       "     {'t-sne_x': -18.819617505924093, 't-sne_y': -33.38923836493636},\n",
       "     {'umap_x': 13.654227256774902, 'umap_y': 12.35094165802002}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48446701-C-T',\n",
       "      'distance': 0.00226938771083951,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508577-C-T',\n",
       "      'distance': 0.002562406938523054,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515385-A-G',\n",
       "      'distance': 0.0028738495893776417,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460241-C-T',\n",
       "      'distance': 0.0031662785913795233,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48412564-C-T',\n",
       "      'distance': 0.0036427555605769157,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446701-C-A',\n",
       "      'distance': 0.003711951896548271,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437318-T-C',\n",
       "      'distance': 0.01298649050295353,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48437765-T-C',\n",
       "      'distance': 0.013009948655962944,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489845-T-C',\n",
       "      'distance': 0.013690884225070477,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48427561-A-C',\n",
       "      'distance': 0.014607856050133705,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48487309-T-C',\n",
       "      'distance': 0.014856405556201935,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48445370-A-G',\n",
       "      'distance': 0.015801923349499702,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520652-C-T',\n",
       "      'distance': 0.02135402150452137,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48472544-C-A',\n",
       "      'distance': 0.02227068319916725,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.024155333638191223,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.02440589666366577,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.024472003802657127,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.024866407737135887,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497265-C-T',\n",
       "      'distance': 0.025073755532503128,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463898-C-T',\n",
       "      'distance': 0.02508135885000229,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.6},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7}}},\n",
       "   {'variant_id': '15-48415719-T-G',\n",
       "    'metadata': {'vcf_string': '15\\t48415719\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415719',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7868A>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415719T>G',\n",
       "     'hgvs_protein': 'p.His2623Pro',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"T/G\", \"transcript_consequences\": [{\"given_ref\": \"T\", \"strand\": -1, \"cds_end\": 7868, \"cds_start\": 7868, \"eve_score\": 0.403057733016133, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.818}, \"phaplo\": 0.99897615427687, \"location\": \"15:48415719\", \"mastermind_counts\": \"4|4|4\", \"ptriplo\": 0.984699554922396, \"conservation\": 5.35, \"consequence_terms\": [\"missense_variant\"], \"cadd_phred\": 27.4, \"used_ref\": \"T\", \"sift_prediction\": \"deleterious_low_confidence\", \"mutationtaster_pred\": \"D\", \"cadd_raw\": 4.905885, \"aa\": \"T\", \"protein_end\": 2623, \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -2, \"gene_symbol\": \"FBN1\", \"impact\": \"MODERATE\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_end\": 8184, \"hgvsc\": \"NM_000138.5:c.7868A>C\", \"variant_allele\": \"G\", \"amino_acids\": \"H/P\", \"biotype\": \"protein_coding\", \"codons\": \"cAc/cCc\", \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"sift_score\": 0, \"hgvsp\": \"NP_000129.3:p.His2623Pro\", \"eve_class\": \"Uncertain\", \"spliceai\": {\"DP_DL\": -45, \"DS_AG\": 0, \"DP_DG\": 48, \"DS_DL\": 0, \"DP_AL\": 48, \"DS_AL\": 0, \"DP_AG\": -47, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0}, \"gene_id\": \"2200\", \"polyphen_score\": 0.764, \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.981656706479832, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48415718, \"type\": \"SimpleInterval\", \"end\": 48415719}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"exon\": \"64/66\", \"canonical\": 1, \"lrt_pred\": \"invalid_field\", \"cdna_start\": 8184, \"protein_start\": 2623, \"protein_id\": \"NP_000129.3\", \"polyphen_prediction\": \"possibly_damaging\", \"mastermind_mmid3\": \"FBN1:H2623P\"}], \"strand\": 1, \"start\": 48415719, \"input\": \"NC_000015.10:g.48415719T>G\", \"most_severe_consequence\": \"missense_variant\", \"minimised\": 1, \"seq_region_name\": \"15\", \"variant_class\": \"SNV\", \"vcf_string\": \"15-48415719-T-G\", \"id\": \"NC_000015.10:g.48415719T>G\", \"end\": 48415719, \"colocated_variants\": [{\"start\": 48415719, \"end\": 48415719, \"id\": \"CM970522\", \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1}], \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': 0.04363762558379927, 'pca_y': 0.077101201897144},\n",
       "     {'t-sne_x': 6.107773294642821, 't-sne_y': -4.073826653804099},\n",
       "     {'umap_x': 5.047346115112305, 'umap_y': 4.48164176940918}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463129-C-T',\n",
       "      'distance': 0.0026312980335205793,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.003039394738152623,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434627-C-T',\n",
       "      'distance': 0.0030457437969744205,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465825-C-T',\n",
       "      'distance': 0.0031398599967360497,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432910-C-T',\n",
       "      'distance': 0.003207108471542597,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437039-C-T',\n",
       "      'distance': 0.0032978407107293606,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.003364068688824773,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432955-C-T',\n",
       "      'distance': 0.003501639701426029,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432916-C-T',\n",
       "      'distance': 0.003515389980748296,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.0035241632722318172,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.0035532540641725063,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468474-C-T',\n",
       "      'distance': 0.0035937149077653885,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425816-C-T',\n",
       "      'distance': 0.0035967505536973476,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425823-C-T',\n",
       "      'distance': 0.0036120570730417967,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432910-C-G',\n",
       "      'distance': 0.0036124801263213158,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.003663885872811079,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-T',\n",
       "      'distance': 0.0037365988828241825,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415716-T-C',\n",
       "      'distance': 0.003747696755453944,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.0037697898223996162,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.003821642603725195,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48415715-G-C',\n",
       "    'metadata': {'vcf_string': '15\\t48415715\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415715',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7872C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415715G>C',\n",
       "     'hgvs_protein': 'p.Asn2624Lys',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48415715, \"most_severe_consequence\": \"missense_variant\", \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48415715G>C\", \"allele_string\": \"G/C\", \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"codons\": \"aaC/aaG\", \"biotype\": \"protein_coding\", \"variant_allele\": \"C\", \"amino_acids\": \"N/K\", \"sift_score\": 0, \"allele_num\": 1, \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.997092485427856, \"polyphen_score\": 0.428, \"gene_id\": \"2200\", \"spliceai\": {\"DP_DG\": -50, \"DP_DL\": -41, \"DS_AG\": 0, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": -1, \"DP_AL\": -43, \"DS_AL\": 0, \"DS_DL\": 0}, \"eve_class\": \"Pathogenic\", \"hgvsp\": \"NP_000129.3:p.Asn2624Lys\", \"polyphen_prediction\": \"benign\", \"mastermind_mmid3\": \"FBN1:N2624K\", \"protein_id\": \"NP_000129.3\", \"protein_start\": 2624, \"cdna_start\": 8188, \"lrt_pred\": \"invalid_field\", \"canonical\": 1, \"exon\": \"64/66\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48415714, \"end\": 48415715, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}}, \"loeuf\": 0.105, \"alphamissense\": {\"am_pathogenicity\": 0.9991, \"am_class\": \"likely_pathogenic\"}, \"cds_start\": 7872, \"eve_score\": 0.710310725044965, \"cds_end\": 7872, \"strand\": -1, \"given_ref\": \"G\", \"conservation\": 4.43, \"ptriplo\": 0.984699554922396, \"mastermind_counts\": \"3|5|5\", \"location\": \"15:48415715\", \"phaplo\": 0.99897615427687, \"cadd_raw\": 3.779168, \"sift_prediction\": \"deleterious_low_confidence\", \"mutationtaster_pred\": \"D\", \"used_ref\": \"G\", \"cadd_phred\": 23.5, \"consequence_terms\": [\"missense_variant\"], \"hgvsc\": \"NM_000138.5:c.7872C>G\", \"impact\": \"MODERATE\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_end\": 8188, \"gene_symbol\": \"FBN1\", \"blosum62\": 0, \"mane_select\": \"ENST00000316623.10\", \"aa\": \"G\", \"protein_end\": 2624}], \"strand\": 1, \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"end\": 48415715, \"id\": \"CM015153\", \"start\": 48415715}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"vcf_string\": \"15-48415715-G-C\", \"end\": 48415715, \"id\": \"NC_000015.10:g.48415715G>C\"}',\n",
       "    'coordinates': [{'pca_x': 0.039489720882476126,\n",
       "      'pca_y': 0.08436616117824236},\n",
       "     {'t-sne_x': 17.265760643773124, 't-sne_y': -7.357328651781544},\n",
       "     {'umap_x': 8.165770530700684, 'umap_y': 6.942580699920654}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48437064-G-C',\n",
       "      'distance': 0.002609910210594535,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.002868467243388295,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415716-T-C',\n",
       "      'distance': 0.0029133649077266455,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432910-C-G',\n",
       "      'distance': 0.0033109886571764946,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428356-G-C',\n",
       "      'distance': 0.003503575921058655,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48488439-T-C',\n",
       "      'distance': 0.0035574333742260933,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421680-T-C',\n",
       "      'distance': 0.003563208505511284,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437347-G-C',\n",
       "      'distance': 0.0036589184310287237,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.003733154619112611,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.0037680179812014103,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.0037802429869771004,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437784-G-C',\n",
       "      'distance': 0.0038057316560298204,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.003873704466968775,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421652-G-C',\n",
       "      'distance': 0.003969421610236168,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48467996-G-C',\n",
       "      'distance': 0.0039741299115121365,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.003981709945946932,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468455-G-C',\n",
       "      'distance': 0.004049653187394142,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.004127563443034887,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472670-T-C',\n",
       "      'distance': 0.004152257461100817,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.00419636769220233,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48415696-ACTTG-A',\n",
       "    'metadata': {'vcf_string': '15\\t48415696\\t.\\tACTTG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415696',\n",
       "     'ref_allele': 'ACTTG',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7887_7890del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415697_48415700del',\n",
       "     'hgvs_protein': 'p.Lys2630AlafsTer51',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"transcript_consequences\": [{\"cds_end\": 7890, \"given_ref\": \"CTTG\", \"strand\": -1, \"cds_start\": 7887, \"phaplo\": 0.99897615427687, \"mastermind_counts\": \"5|5|6\", \"ptriplo\": 0.984699554922396, \"conservation\": 4.652, \"location\": \"15:48415697-48415700\", \"used_ref\": \"CTTG\", \"consequence_terms\": [\"frameshift_variant\"], \"gene_symbol\": \"FBN1\", \"aa\": \"CTTG\", \"protein_end\": 2630, \"mane_select\": \"ENST00000316623.10\", \"hgvsc\": \"NM_000138.5:c.7887_7890del\", \"impact\": \"HIGH\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_end\": 8206, \"amino_acids\": \"YK/X\", \"variant_allele\": \"-\", \"biotype\": \"protein_coding\", \"codons\": \"taCAAG/ta\", \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"lof_info\": \"PERCENTILE:0.915738161559889,GERP_DIST:706.048844635487,BP_DIST:724,DIST_FROM_LAST_EXON:335,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"hgvsp\": \"NP_000129.3:p.Lys2630AlafsTer51\", \"spliceai\": {\"DS_AL\": 0, \"DP_AL\": -24, \"DP_AG\": 18, \"DS_DL\": 0, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DP_DL\": -31, \"DP_DG\": -22}, \"transcript_id\": \"NM_000138.5\", \"lof\": \"HC\", \"gene_id\": \"2200\", \"canonical\": 1, \"loeuf\": 0.105, \"exon\": \"64/66\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48415700, \"type\": \"SimpleInterval\", \"start\": 48415696}}}, \"mastermind_mmid3\": \"FBN1:K2630fs\", \"cdna_start\": 8203, \"protein_start\": 2629, \"protein_id\": \"NP_000129.3\"}], \"allele_string\": \"CTTG/-\", \"most_severe_consequence\": \"frameshift_variant\", \"minimised\": 1, \"input\": \"NC_000015.10:g.48415697_48415700del\", \"seq_region_name\": \"15\", \"start\": 48415697, \"end\": 48415700, \"id\": \"NC_000015.10:g.48415697_48415700del\", \"vcf_string\": \"15-48415696-ACTTG-A\", \"variant_class\": \"deletion\", \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"end\": 48415700, \"id\": \"rs1170632495\", \"start\": 48415697, \"frequencies\": {\"-\": {\"gnomade_sas\": 0, \"gnomade_nfe\": 8.993e-07, \"gnomade_fin\": 0, \"gnomade_remaining\": 0, \"gnomade_eas\": 0, \"gnomade_amr\": 0, \"gnomade_asj\": 0, \"gnomade_mid\": 0, \"gnomade_afr\": 0, \"gnomade\": 6.841e-07}}, \"seq_region_name\": \"15\", \"allele_string\": \"CTTG/-\", \"strand\": 1}]}',\n",
       "    'coordinates': [{'pca_x': 0.04039501345819352,\n",
       "      'pca_y': 0.03705044469628948},\n",
       "     {'t-sne_x': 4.3080953273134215, 't-sne_y': 8.481839197039083},\n",
       "     {'umap_x': 3.689502000808716, 'umap_y': 3.1348602771759033}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.09752204269170761,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.0988774523139,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.0990791916847229,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.09987731277942657,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.09991589188575745,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.10003343224525452,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.10011454671621323,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.1001775711774826,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.10055197775363922,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10089165717363358,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.10095645487308502,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.10099916160106659,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.10112713277339935,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.10120105743408203,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437912-G-A',\n",
       "      'distance': 0.10144960880279541,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.1014624685049057,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.10162246227264404,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537704-G-A',\n",
       "      'distance': 0.1017555519938469,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48516225-G-A',\n",
       "      'distance': 0.10179787874221802,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.10187847912311554,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48415672-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48415672\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415672',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7915T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415672A>T',\n",
       "     'hgvs_protein': 'p.Tyr2639Asn',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"id\": \"CM020695\", \"phenotype_or_disease\": 1, \"start\": 48415672, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"end\": 48415672}], \"id\": \"NC_000015.10:g.48415672A>T\", \"start\": 48415672, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"polyphen_score\": 0.888, \"cds_end\": 7915, \"protein_start\": 2639, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"protein_end\": 2639, \"conservation\": 5.35, \"loeuf\": 0.105, \"cdna_end\": 8231, \"cdna_start\": 8231, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:Y2639N\", \"eve_score\": 0.271511492402584, \"alphamissense\": {\"am_class\": \"ambiguous\", \"am_pathogenicity\": 0.466}, \"transcript_id\": \"NM_000138.5\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Benign\", \"variant_allele\": \"T\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"aa\": \"A\", \"blosum62\": -2, \"exon\": \"64/66\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 2, \"DS_AL\": 0, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": 2, \"DP_DG\": -7}, \"codons\": \"Tat/Aat\", \"canonical\": 1, \"cadd_phred\": 27.3, \"mastermind_counts\": \"2|2|2\", \"hgvsc\": \"NM_000138.5:c.7915T>A\", \"biotype\": \"protein_coding\", \"given_ref\": \"A\", \"cadd_raw\": 4.894648, \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48415672, \"start\": 48415671, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Tyr2639Asn\", \"polyphen_prediction\": \"possibly_damaging\", \"location\": \"15:48415672\", \"clinpred\": 0.99155855178833, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"strand\": -1, \"mutationtaster_pred\": \"D\", \"cds_start\": 7915, \"genesplicer\": \"loss/donor/48415671-48415672/Medium/0.707877\", \"amino_acids\": \"Y/N\"}], \"end\": 48415672, \"strand\": 1, \"allele_string\": \"A/T\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48415672-A-T\", \"input\": \"NC_000015.10:g.48415672A>T\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.03752888303893637,\n",
       "      'pca_y': 0.0754682197157005},\n",
       "     {'t-sne_x': 14.975390352913086, 't-sne_y': 9.243919982448407},\n",
       "     {'umap_x': 3.3923869132995605, 'umap_y': 4.555396556854248}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.002157502807676792,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430791-A-T',\n",
       "      'distance': 0.00247741281054914,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428439-A-T',\n",
       "      'distance': 0.002626784611493349,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-T',\n",
       "      'distance': 0.0030872179195284843,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468527-A-T',\n",
       "      'distance': 0.003295010654255748,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596325-A-T',\n",
       "      'distance': 0.0033283005468547344,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427684-A-T',\n",
       "      'distance': 0.0036271228455007076,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432976-C-T',\n",
       "      'distance': 0.003685424104332924,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.003706255229189992,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415684-C-T',\n",
       "      'distance': 0.003753430675715208,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.004011821001768112,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596324-C-T',\n",
       "      'distance': 0.004042685031890869,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.004047672264277935,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.004082207567989826,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.0041929613798856735,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-G',\n",
       "      'distance': 0.004202553071081638,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.004228382371366024,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.004253491759300232,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.004255744628608227,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.004266069270670414,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48412738-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48412738\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48412738',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8057G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48412738C>A',\n",
       "     'hgvs_protein': 'p.Cys2686Phe',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48412738, \"end\": 48412738, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"polyphen_prediction\": \"probably_damaging\", \"clinpred\": 0.999560654163361, \"location\": \"15:48412738\", \"cadd_raw\": 5.053584, \"given_ref\": \"C\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48412737, \"end\": 48412738}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}}, \"hgvsp\": \"NP_000129.3:p.Cys2686Phe\", \"amino_acids\": \"C/F\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"mutationtaster_pred\": \"D\", \"strand\": -1, \"cds_start\": 8057, \"variant_allele\": \"A\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Uncertain\", \"codons\": \"tGt/tTt\", \"canonical\": 1, \"cadd_phred\": 28.2, \"mastermind_counts\": \"5|5|5\", \"hgvsc\": \"NM_000138.5:c.8057G>T\", \"biotype\": \"protein_coding\", \"aa\": \"C\", \"blosum62\": -2, \"exon\": \"65/66\", \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": -43, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": 27, \"DP_AL\": 5, \"DP_DG\": 5, \"DS_DG\": 0, \"DS_AG\": 0}, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C2686F\", \"cdna_start\": 8373, \"cdna_end\": 8373, \"lrt_pred\": \"invalid_field\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9969}, \"transcript_id\": \"NM_000138.5\", \"eve_score\": 0.517642608503216, \"cds_end\": 8057, \"protein_start\": 2686, \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.99, \"loeuf\": 0.105, \"conservation\": 5.35, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 2686}], \"minimised\": 1, \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM972823\", \"end\": 48412738, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48412738}, {\"phenotype_or_disease\": 1, \"id\": \"rs2141211820\", \"pubmed\": [17701892], \"clin_sig_allele\": \"A:pathogenic\", \"start\": 48412738, \"allele_string\": \"C/A\", \"strand\": 1, \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV001383754\", \"VCV001071312\"]}, \"end\": 48412738}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48412738C>A\", \"input\": \"NC_000015.10:g.48412738C>A\", \"seq_region_name\": \"15\", \"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/A\", \"vcf_string\": \"15-48412738-C-A\"}',\n",
       "    'coordinates': [{'pca_x': 0.04047595353408079,\n",
       "      'pca_y': 0.09491290097003088},\n",
       "     {'t-sne_x': 21.57939851128493, 't-sne_y': 10.172107420193798},\n",
       "     {'umap_x': 4.461117744445801, 'umap_y': 6.879871368408203}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468522-C-A',\n",
       "      'distance': 0.0010710086207836866,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465840-C-A',\n",
       "      'distance': 0.0012015345273539424,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425783-C-A',\n",
       "      'distance': 0.0017659473232924938,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472628-C-A',\n",
       "      'distance': 0.0018212435534223914,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503849-C-A',\n",
       "      'distance': 0.0020987880416214466,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495513-C-A',\n",
       "      'distance': 0.0022454573772847652,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415632-C-T',\n",
       "      'distance': 0.0025317235849797726,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.002757345326244831,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.0027721829246729612,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.0028659875970333815,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.0029060975648462772,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485463-C-A',\n",
       "      'distance': 0.0029318761080503464,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432910-C-T',\n",
       "      'distance': 0.0029457956552505493,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463904-C-A',\n",
       "      'distance': 0.002970887813717127,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432955-C-T',\n",
       "      'distance': 0.002982271835207939,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425816-C-T',\n",
       "      'distance': 0.002986625535413623,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490050-C-A',\n",
       "      'distance': 0.0031156232580542564,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48474638-C-T',\n",
       "      'distance': 0.003135467180982232,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537721-C-A',\n",
       "      'distance': 0.0031676902435719967,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425852-C-T',\n",
       "      'distance': 0.0031922843772917986,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48412646-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48412646\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48412646',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8149G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48412646C>A',\n",
       "     'hgvs_protein': 'p.Glu2717Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"allele_string\": \"C/A\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48412646-C-A\", \"input\": \"NC_000015.10:g.48412646C>A\", \"seq_region_name\": \"15\", \"minimised\": 1, \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM013944\", \"end\": 48412646, \"start\": 48412646, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1}, {\"end\": 48412646, \"start\": 48412646, \"clin_sig\": [\"uncertain_significance\", \"likely_benign\", \"likely_pathogenic\", \"pathogenic\"], \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000664013\", \"RCV000247812\", \"VCV000263433\", \"RCV001706243\", \"RCV000229179\", \"VCV000237106\", \"RCV000273444\", \"RCV000270822\", \"RCV000365739\", \"RCV000387643\", \"RCV000362980\", \"RCV000330801\", \"RCV000506463\", \"RCV000295866\"]}, \"allele_string\": \"C/A/T\", \"strand\": 1, \"clin_sig_allele\": \"T:uncertain_significance;T:likely_benign;T:conflicting_classifications_of_pathogenicity;A:pathogenic/likely_pathogenic;A:pathogenic\", \"phenotype_or_disease\": 1, \"id\": \"rs187553035\"}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48412646C>A\", \"start\": 48412646, \"most_severe_consequence\": \"stop_gained\", \"transcript_consequences\": [{\"location\": \"15:48412646\", \"hgvsp\": \"NP_000129.3:p.Glu2717Ter\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48412645, \"end\": 48412646}, \"type\": \"SequenceLocation\"}}, \"allele_num\": 1, \"consequence_terms\": [\"stop_gained\"], \"cadd_raw\": 10.231526, \"given_ref\": \"C\", \"amino_acids\": \"E/*\", \"cds_start\": 8149, \"mutationtaster_pred\": \"A\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"lof_info\": \"PERCENTILE:0.945798514391829,GERP_DIST:465.950940355659,BP_DIST:466,DIST_FROM_LAST_EXON:77,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"A\", \"gene_symbol_source\": \"EntrezGene\", \"hgvsc\": \"NM_000138.5:c.8149G>T\", \"biotype\": \"protein_coding\", \"mastermind_counts\": \"6|7|7\", \"cadd_phred\": 46, \"canonical\": 1, \"codons\": \"Gag/Tag\", \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": -30, \"DS_AL\": 0.1, \"DS_DL\": 0, \"DP_AG\": -3, \"DP_AL\": 11, \"DP_DG\": -3, \"DS_DG\": 0.02, \"DS_AG\": 0}, \"lof\": \"HC\", \"exon\": \"65/66\", \"aa\": \"C\", \"mastermind_mmid3\": \"FBN1:E2717X\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_end\": 8465, \"cdna_start\": 8465, \"transcript_id\": \"NM_000138.5\", \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"cds_end\": 8149, \"protein_start\": 2717, \"conservation\": 5.35, \"loeuf\": 0.105, \"protein_end\": 2717, \"phaplo\": 0.99897615427687}], \"end\": 48412646}',\n",
       "    'coordinates': [{'pca_x': 0.01667410601514477,\n",
       "      'pca_y': 0.09715003553651305},\n",
       "     {'t-sne_x': 22.10052654230834, 't-sne_y': 42.80849781081619},\n",
       "     {'umap_x': -9.546883583068848, 'umap_y': 4.660379886627197}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48505139-C-A',\n",
       "      'distance': 0.0017941502155736089,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411085-C-A',\n",
       "      'distance': 0.002004183828830719,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.0021733168978244066,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.0025021014735102654,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.00258575938642025,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452676-C-A',\n",
       "      'distance': 0.002852843841537833,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.003197507932782173,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515397-C-T',\n",
       "      'distance': 0.003388350596651435,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48415675-G-A',\n",
       "      'distance': 0.0037018940784037113,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.0037302267737686634,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48534170-G-A',\n",
       "      'distance': 0.003749065101146698,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488233-C-A',\n",
       "      'distance': 0.0038343591149896383,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.0038445021491497755,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437837-C-A',\n",
       "      'distance': 0.0038568708114326,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.003981795161962509,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.003999626729637384,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483878-C-A',\n",
       "      'distance': 0.004059066064655781,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415540-G-A',\n",
       "      'distance': 0.004181340336799622,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468070-G-A',\n",
       "      'distance': 0.004617988597601652,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513615-G-A',\n",
       "      'distance': 0.004712117835879326,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48411286-T-A',\n",
       "    'metadata': {'vcf_string': '15\\t48411286\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48411286',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8320A>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48411286T>A',\n",
       "     'hgvs_protein': 'p.Lys2774Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"id\": \"CM085408\", \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48411286, \"end\": 48411286}], \"id\": \"NC_000015.10:g.48411286T>A\", \"start\": 48411286, \"end\": 48411286, \"most_severe_consequence\": \"stop_gained\", \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"mastermind_mmid3\": \"FBN1:K2774X\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_end\": 8636, \"cdna_start\": 8636, \"conservation\": 5.31, \"loeuf\": 0.105, \"protein_end\": 2774, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"T\", \"protein_start\": 2774, \"cds_end\": 8320, \"amino_acids\": \"K/*\", \"cds_start\": 8320, \"nmd\": \"NMD_escaping_variant\", \"mutationtaster_pred\": \"D\", \"strand\": -1, \"impact\": \"HIGH\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"location\": \"15:48411286\", \"hgvsp\": \"NP_000129.3:p.Lys2774Ter\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48411285, \"end\": 48411286}}}, \"consequence_terms\": [\"stop_gained\"], \"allele_num\": 1, \"cadd_raw\": 9.744227, \"given_ref\": \"T\", \"hgvsc\": \"NM_000138.5:c.8320A>T\", \"biotype\": \"protein_coding\", \"cadd_phred\": 43, \"mastermind_counts\": \"3|4|4\", \"codons\": \"Aag/Tag\", \"canonical\": 1, \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0.02, \"DP_AL\": -2, \"DP_DG\": 10, \"DS_DL\": 0, \"DP_AG\": 4, \"SYMBOL\": \"FBN1\", \"DP_DL\": -2, \"DS_AL\": 0}, \"lof\": \"HC\", \"exon\": \"66/66\", \"aa\": \"T\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"lof_info\": \"PERCENTILE:0.965645311049211,GERP_DIST:452.010367393494,BP_DIST:296,DIST_FROM_LAST_EXON:-93,50_BP_RULE:FAIL,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"A\", \"gene_symbol_source\": \"EntrezGene\"}], \"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"T/A\", \"vcf_string\": \"15-48411286-T-A\", \"input\": \"NC_000015.10:g.48411286T>A\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.010765270588023484,\n",
       "      'pca_y': 0.09579383900233601},\n",
       "     {'t-sne_x': 14.932765207026058, 't-sne_y': 47.01858688248253},\n",
       "     {'umap_x': -8.991454124450684, 'umap_y': 5.701269149780273}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48644643-T-A',\n",
       "      'distance': 0.0032127085141837597,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.003804967273026705,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.0043346332386136055,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415675-G-A',\n",
       "      'distance': 0.005034426227211952,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48534170-G-A',\n",
       "      'distance': 0.005208012647926807,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415540-G-A',\n",
       "      'distance': 0.005224690772593021,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.005308587569743395,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48467954-A-T',\n",
       "      'distance': 0.005406477022916079,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537615-A-T',\n",
       "      'distance': 0.005447745323181152,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.005461180582642555,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.005611035507172346,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427630-G-A',\n",
       "      'distance': 0.0056128427386283875,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425423-G-A',\n",
       "      'distance': 0.005664794240146875,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452670-G-A',\n",
       "      'distance': 0.005754483863711357,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.005774230230599642,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445430-G-A',\n",
       "      'distance': 0.005803072825074196,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.00582133186981082,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513615-G-A',\n",
       "      'distance': 0.0058273570612072945,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411280-G-A',\n",
       "      'distance': 0.0058451006188988686,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505139-C-A',\n",
       "      'distance': 0.005924557335674763,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48411079-TAAGTT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48411079\\t.\\tTAAGTT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48411079',\n",
       "     'ref_allele': 'TAAGTT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8525_8529del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48411080_48411084del',\n",
       "     'hgvs_protein': 'p.Leu2842ProfsTer7',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"allele_string\": \"AAGTT/-\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48411079-TAAGTT-T\", \"input\": \"NC_000015.10:g.48411080_48411084del\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48411080_48411084del\", \"start\": 48411080, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"cds_end\": 8529, \"protein_start\": 2842, \"used_ref\": \"GTTAA\", \"mane_select\": \"ENST00000316623.10\", \"phaplo\": 0.99897615427687, \"protein_end\": 2843, \"conservation\": 1.459, \"loeuf\": 0.105, \"cdna_start\": 8841, \"cdna_end\": 8845, \"mane\": [\"MANE_Select\"], \"transcript_id\": \"NM_000138.5\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"-\", \"lof_info\": \"PERCENTILE:0.989902506963788,GERP_DIST:172.296126556396,BP_DIST:90,DIST_FROM_LAST_EXON:-299,50_BP_RULE:FAIL,PHYLOCSF_TOO_SHORT\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"aa\": \"AAGTT\", \"exon\": \"66/66\", \"lof\": \"HC\", \"codons\": \"cTTAAC/c\", \"canonical\": 1, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.8525_8529del\", \"given_ref\": \"AAGTT\", \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"hgvsp\": \"NP_000129.3:p.Leu2842ProfsTer7\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48411084, \"start\": 48411079, \"type\": \"SimpleInterval\"}}}, \"hgvs_offset\": -3, \"location\": \"15:48411077-48411081\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"strand\": -1, \"nmd\": \"NMD_escaping_variant\", \"cds_start\": 8525, \"amino_acids\": \"LN/X\"}], \"end\": 48411084}',\n",
       "    'coordinates': [{'pca_x': 0.0334531675427053,\n",
       "      'pca_y': 0.034979634713467954},\n",
       "     {'t-sne_x': 8.130311525277904, 't-sne_y': 3.5574377923334843},\n",
       "     {'umap_x': 3.923224449157715, 'umap_y': 3.4792168140411377}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.10280127823352814,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10344391316175461,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10372896492481232,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.10387112200260162,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10440235584974289,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.10503434389829636,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.10585781186819077,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.10592931509017944,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.10641156882047653,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.10641375184059143,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441797-G-T',\n",
       "      'distance': 0.10644116997718811,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.10661676526069641,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10670040547847748,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.10681374371051788,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.10686901956796646,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.10689512640237808,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.10690955817699432,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.10696469247341156,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.10702396929264069,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.10736502707004547,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48411071-T-TA',\n",
       "    'metadata': {'vcf_string': '15\\t48411071\\t.\\tT\\tTA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48411071',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'TA',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8534dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48411072dup',\n",
       "     'hgvs_protein': 'p.Glu2846ArgfsTer5',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"minimised\": 1, \"start\": 48411072, \"input\": \"NC_000015.10:g.48411072dup\", \"most_severe_consequence\": \"frameshift_variant\", \"end\": 48411071, \"vcf_string\": \"15-48411071-T-TA\", \"strand\": 1, \"seq_region_name\": \"15\", \"transcript_consequences\": [{\"variant_allele\": \"A\", \"impact\": \"HIGH\", \"phaplo\": 0.99897615427687, \"cdna_start\": 8850, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48411071, \"start\": 48411071}}}, \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.8534dup\", \"canonical\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"spliceai\": {\"DS_AL\": 0, \"DS_AG\": 0, \"DP_AL\": 36, \"DS_DG\": 0, \"DS_DL\": 0, \"DP_AG\": 17, \"DP_DG\": 21, \"DP_DL\": -40, \"SYMBOL\": \"FBN1\"}, \"mane_select\": \"ENST00000316623.10\", \"given_ref\": \"-\", \"cds_start\": 8534, \"used_ref\": \"-\", \"cdna_end\": 8851, \"protein_id\": \"NP_000129.3\", \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"L/LX\", \"mane\": [\"MANE_Select\"], \"loeuf\": 0.105, \"lof_info\": \"PERCENTILE:0.990482822655525,GERP_DIST:157.60678768158,BP_DIST:82,DIST_FROM_LAST_EXON:-307,50_BP_RULE:FAIL,PHYLOCSF_TOO_SHORT\", \"exon\": \"66/66\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"codons\": \"cta/ctTa\", \"allele_num\": 1, \"biotype\": \"protein_coding\", \"location\": \"15:48411071-48411072\", \"gene_id\": \"2200\", \"conservation\": 3.56, \"cds_end\": 8535, \"lof\": \"HC\", \"protein_start\": 2845, \"hgvsp\": \"NP_000129.3:p.Glu2846ArgfsTer5\", \"transcript_id\": \"NM_000138.5\", \"protein_end\": 2845, \"nmd\": \"NMD_escaping_variant\"}], \"id\": \"NC_000015.10:g.48411072dup\", \"allele_string\": \"-/A\", \"variant_class\": \"insertion\"}',\n",
       "    'coordinates': [{'pca_x': 0.02995707825741159,\n",
       "      'pca_y': 0.011627465061339486},\n",
       "     {'t-sne_x': 24.759920704440937, 't-sne_y': 1.6341152755872759},\n",
       "     {'umap_x': 3.9304556846618652, 'umap_y': 3.824249267578125}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48470689-G-A',\n",
       "      'distance': 0.09771375358104706,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520690-G-A',\n",
       "      'distance': 0.0993163138628006,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48470722-G-A',\n",
       "      'distance': 0.0995408445596695,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48420718-G-A',\n",
       "      'distance': 0.10010673850774765,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48456731-G-A',\n",
       "      'distance': 0.10069297254085541,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48487142-G-A',\n",
       "      'distance': 0.10072864592075348,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48613083-G-T',\n",
       "      'distance': 0.10108061134815216,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520693-A-G',\n",
       "      'distance': 0.10116396844387054,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48489981-G-A',\n",
       "      'distance': 0.1015177071094513,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48420778-G-A',\n",
       "      'distance': 0.10177840292453766,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48420763-G-A',\n",
       "      'distance': 0.10182224214076996,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48487157-G-A',\n",
       "      'distance': 0.10191546380519867,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48460250-G-A',\n",
       "      'distance': 0.10202623158693314,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520675-A-G',\n",
       "      'distance': 0.10217776149511337,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48427613-G-A',\n",
       "      'distance': 0.10229752212762833,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48427700-G-A',\n",
       "      'distance': 0.10230711847543716,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48446797-G-A',\n",
       "      'distance': 0.10265146195888519,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48412674-G-A',\n",
       "      'distance': 0.10279406607151031,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48526242-G-A',\n",
       "      'distance': 0.10308291018009186,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48470680-C-T',\n",
       "      'distance': 0.10308604687452316,\n",
       "      'pathogenicity': 'likely_benign'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_benign',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_benign', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_benign', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_benign', 'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48411059-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48411059\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48411059',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8547T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48411059A>C',\n",
       "     'hgvs_protein': 'p.Tyr2849Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48411059A>C\", \"minimised\": 1, \"assembly_name\": \"GRCh38\", \"start\": 48411059, \"colocated_variants\": [{\"end\": 48411059, \"id\": \"CM107867\", \"start\": 48411059, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\"}], \"id\": \"NC_000015.10:g.48411059A>C\", \"allele_string\": \"A/C\", \"variant_class\": \"SNV\", \"end\": 48411059, \"vcf_string\": \"15-48411059-A-C\", \"most_severe_consequence\": \"stop_gained\", \"transcript_consequences\": [{\"hgvsp\": \"NP_000129.3:p.Tyr2849Ter\", \"cds_end\": 8547, \"conservation\": 4.24, \"lof\": \"HC\", \"protein_start\": 2849, \"nmd\": \"NMD_escaping_variant\", \"protein_end\": 2849, \"transcript_id\": \"NM_000138.5\", \"cadd_phred\": 37, \"codons\": \"taT/taG\", \"allele_num\": 1, \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"exon\": \"66/66\", \"mastermind_counts\": \"5|7|7\", \"loeuf\": 0.105, \"lof_info\": \"PERCENTILE:0.991991643454039,GERP_DIST:154.142000645399,BP_DIST:69,DIST_FROM_LAST_EXON:-320,50_BP_RULE:FAIL,PHYLOCSF_TOO_SHORT\", \"location\": \"15:48411059\", \"gene_id\": \"2200\", \"biotype\": \"protein_coding\", \"cds_start\": 8547, \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": -28, \"DP_DG\": 50, \"DP_AG\": 29, \"DS_DL\": 0, \"DS_DG\": 0, \"DP_AL\": -43, \"DS_AG\": 0, \"DS_AL\": 0}, \"given_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"cadd_raw\": 8.06853, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"amino_acids\": \"Y/*\", \"cdna_end\": 8863, \"protein_id\": \"NP_000129.3\", \"used_ref\": \"A\", \"lrt_pred\": \"invalid_field\", \"phaplo\": 0.99897615427687, \"variant_allele\": \"C\", \"mastermind_mmid3\": \"FBN1:Y2849X\", \"impact\": \"HIGH\", \"mutationtaster_pred\": \"D\", \"canonical\": 1, \"consequence_terms\": [\"stop_gained\"], \"aa\": \"A\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48411059, \"start\": 48411058}}}, \"cdna_start\": 8863, \"hgvsc\": \"NM_000138.5:c.8547T>G\", \"strand\": -1}], \"strand\": 1, \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.01359229744793268,\n",
       "      'pca_y': 0.0928795513096762},\n",
       "     {'t-sne_x': 25.56347208083217, 't-sne_y': 43.602216123775484},\n",
       "     {'umap_x': -9.762850761413574, 'umap_y': 4.015021800994873}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468080-A-C',\n",
       "      'distance': 0.0011009452864527702,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437362-A-C',\n",
       "      'distance': 0.002236597239971161,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48412647-G-C',\n",
       "      'distance': 0.004935993812978268,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.005101935938000679,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.005539651494473219,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.005561596713960171,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.005685965530574322,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487106-G-C',\n",
       "      'distance': 0.0059195407666265965,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474554-C-T',\n",
       "      'distance': 0.006053907331079245,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505139-C-A',\n",
       "      'distance': 0.006082212086766958,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437901-G-C',\n",
       "      'distance': 0.006086783483624458,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420706-G-C',\n",
       "      'distance': 0.006140705198049545,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411085-C-A',\n",
       "      'distance': 0.006141550838947296,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489921-G-C',\n",
       "      'distance': 0.006203247234225273,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.006454354152083397,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411338-C-T',\n",
       "      'distance': 0.006531556136906147,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.006685170345008373,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437837-C-A',\n",
       "      'distance': 0.006706824526190758,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.006746700033545494,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488233-C-A',\n",
       "      'distance': 0.006769213825464249,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48644722-AA-A',\n",
       "    'metadata': {'vcf_string': '15\\t48644722\\t.\\tAA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48644722',\n",
       "     'ref_allele': 'AA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.50del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48644723del',\n",
       "     'hgvs_protein': 'p.Leu17Ter',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48644723del\", \"start\": 48644723, \"assembly_name\": \"GRCh38\", \"minimised\": 1, \"variant_class\": \"deletion\", \"allele_string\": \"A/-\", \"id\": \"NC_000015.10:g.48644723del\", \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"nmd\": \"NMD_escaping_variant\", \"protein_end\": 17, \"conservation\": 5.2, \"cds_end\": 50, \"protein_start\": 17, \"lof\": \"HC\", \"hgvs_offset\": -3, \"hgvsp\": \"NP_000129.3:p.Leu17Ter\", \"biotype\": \"protein_coding\", \"location\": \"15:48644720\", \"gene_id\": \"2200\", \"lof_info\": \"PERCENTILE:0.00580315691736305,GERP_DIST:8614.25134617248,BP_DIST:8505,DIST_FROM_LAST_EXON:8116,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"loeuf\": 0.105, \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"exon\": \"2/66\", \"codons\": \"tTa/ta\", \"allele_num\": 1, \"used_ref\": \"A\", \"protein_id\": \"NP_000129.3\", \"cdna_end\": 366, \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"L/X\", \"mane\": [\"MANE_Select\"], \"spliceai\": {\"DP_DL\": -5, \"SYMBOL\": \"FBN1\", \"DP_AG\": -18, \"DP_DG\": 13, \"DS_AG\": 0, \"DP_AL\": -5, \"DS_DL\": 0, \"DS_DG\": 0, \"DS_AL\": 0}, \"mane_select\": \"ENST00000316623.10\", \"given_ref\": \"A\", \"cds_start\": 50, \"cdna_start\": 366, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"end\": 48644723, \"start\": 48644722, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}}, \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.50del\", \"canonical\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"aa\": \"A\", \"variant_allele\": \"-\", \"impact\": \"HIGH\", \"phaplo\": 0.99897615427687}], \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48644722-AA-A\", \"end\": 48644723, \"most_severe_consequence\": \"frameshift_variant\"}',\n",
       "    'coordinates': [{'pca_x': -0.038065542365829326,\n",
       "      'pca_y': 0.0365010036396732},\n",
       "     {'t-sne_x': -12.99909369745171, 't-sne_y': -3.3087060618088064},\n",
       "     {'umap_x': 0.34633591771125793, 'umap_y': 15.103670120239258}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.0935441255569458,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.09466998279094696,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.09711311012506485,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.09723623096942902,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.09754198789596558,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.09794452786445618,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.09819351136684418,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.09838740527629852,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.0986291915178299,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.09874530881643295,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.09983161836862564,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.10003134608268738,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.10040142387151718,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.10077308118343353,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.10112379491329193,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.10130620002746582,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.10169621556997299,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.10230269283056259,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.10238800942897797,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.10322929918766022,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48537677-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48537677\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48537677',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.670T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48537677A>G',\n",
       "     'hgvs_protein': 'p.Cys224Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"minimised\": 1, \"colocated_variants\": [{\"id\": \"rs1555401676\", \"end\": 48537677, \"clin_sig_allele\": \"G:likely_pathogenic\", \"start\": 48537677, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000663894\", \"VCV000549362\"], \"UniProt\": [\"VAR_023862\"]}, \"allele_string\": \"A/G\", \"clin_sig\": [\"likely_pathogenic\"], \"strand\": 1, \"seq_region_name\": \"15\"}], \"start\": 48537677, \"input\": \"NC_000015.10:g.48537677A>G\", \"end\": 48537677, \"vcf_string\": \"15-48537677-A-G\", \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"hgvsc\": \"NM_000138.5:c.670T>C\", \"strand\": -1, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48537676, \"end\": 48537677}}}, \"cdna_start\": 986, \"aa\": \"A\", \"canonical\": 1, \"consequence_terms\": [\"missense_variant\"], \"impact\": \"MODERATE\", \"mutationtaster_pred\": \"A\", \"mastermind_mmid3\": \"FBN1:C224R\", \"variant_allele\": \"G\", \"lrt_pred\": \"invalid_field\", \"phaplo\": 0.99897615427687, \"cdna_end\": 986, \"protein_id\": \"NP_000129.3\", \"used_ref\": \"A\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"amino_acids\": \"C/R\", \"cadd_raw\": 5.31231, \"gene_symbol_source\": \"EntrezGene\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9992}, \"given_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"spliceai\": {\"DS_AG\": 0, \"DP_AL\": -12, \"DS_DL\": 0, \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": 9, \"SYMBOL\": \"FBN1\", \"DP_AG\": 32, \"DP_DG\": 4}, \"cds_start\": 670, \"clinpred\": 0.999820530414581, \"blosum62\": -3, \"sift_score\": 0, \"biotype\": \"protein_coding\", \"gene_id\": \"2200\", \"location\": \"15:48537677\", \"polyphen_prediction\": \"probably_damaging\", \"exon\": \"7/66\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"polyphen_score\": 0.993, \"mastermind_counts\": \"3|7|7\", \"allele_num\": 1, \"codons\": \"Tgt/Cgt\", \"transcript_id\": \"NM_000138.5\", \"cadd_phred\": 29.9, \"protein_end\": 224, \"protein_start\": 224, \"cds_end\": 670, \"conservation\": 3.79, \"hgvsp\": \"NP_000129.3:p.Cys224Arg\"}], \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"A/G\", \"id\": \"NC_000015.10:g.48537677A>G\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.04850069839928339,\n",
       "      'pca_y': 0.07856638780642769},\n",
       "     {'t-sne_x': 30.169328185036147, 't-sne_y': 3.1917464168744742},\n",
       "     {'umap_x': 2.3333723545074463, 'umap_y': 6.934187412261963}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48420795-A-G',\n",
       "      'distance': 0.0009695982444100082,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470679-A-G',\n",
       "      'distance': 0.0012565376237034798,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437873-A-G',\n",
       "      'distance': 0.0015863971784710884,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.0020515774376690388,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434702-A-G',\n",
       "      'distance': 0.0021215989254415035,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.0022342863958328962,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.0022632614709436893,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474567-A-G',\n",
       "      'distance': 0.0022724259179085493,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515396-C-A',\n",
       "      'distance': 0.00234096497297287,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.0024698430206626654,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.002476995112374425,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.00263319187797606,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432944-A-G',\n",
       "      'distance': 0.0026858444325625896,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415567-A-G',\n",
       "      'distance': 0.0027299849316477776,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.0027487394399940968,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596325-A-G',\n",
       "      'distance': 0.0027631153352558613,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.002911594230681658,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427603-A-G',\n",
       "      'distance': 0.002941333455964923,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520677-A-G',\n",
       "      'distance': 0.0029583419673144817,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.0029851861763745546,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48534129-G-T',\n",
       "    'metadata': {'vcf_string': '15\\t48534129\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48534129',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.813C>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48534129G>T',\n",
       "     'hgvs_protein': 'p.Cys271Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48534129G>T\", \"assembly_name\": \"GRCh38\", \"minimised\": 1, \"start\": 48534129, \"colocated_variants\": [{\"start\": 48534129, \"end\": 48534129, \"id\": \"CM132727\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1}, {\"start\": 48534129, \"id\": \"CM166347\", \"end\": 48534129, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"phenotype_or_disease\": 1}, {\"clin_sig_allele\": \"C:likely_pathogenic\", \"start\": 48534129, \"end\": 48534129, \"id\": \"rs765692335\", \"clin_sig\": [\"likely_pathogenic\"], \"strand\": 1, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV002422635\", \"RCV000755188\", \"VCV000617870\"]}, \"allele_string\": \"G/C/T\", \"phenotype_or_disease\": 1}], \"id\": \"NC_000015.10:g.48534129G>T\", \"allele_string\": \"G/T\", \"variant_class\": \"SNV\", \"vcf_string\": \"15-48534129-G-T\", \"end\": 48534129, \"most_severe_consequence\": \"stop_gained\", \"transcript_consequences\": [{\"cadd_raw\": 7.622651, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"amino_acids\": \"C/*\", \"cdna_end\": 1129, \"protein_id\": \"NP_000129.3\", \"used_ref\": \"G\", \"cds_start\": 813, \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": -49, \"DP_DG\": -45, \"DP_AG\": 38, \"DS_DL\": 0, \"DS_DG\": 0, \"DP_AL\": -32, \"DS_AG\": 0, \"DS_AL\": 0}, \"given_ref\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"consequence_terms\": [\"stop_gained\"], \"aa\": \"G\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48534128, \"end\": 48534129}}}, \"cdna_start\": 1129, \"hgvsc\": \"NM_000138.5:c.813C>A\", \"strand\": -1, \"phaplo\": 0.99897615427687, \"lrt_pred\": \"invalid_field\", \"variant_allele\": \"T\", \"mastermind_mmid3\": \"FBN1:C271X\", \"impact\": \"HIGH\", \"mutationtaster_pred\": \"A\", \"protein_end\": 271, \"transcript_id\": \"NM_000138.5\", \"cadd_phred\": 36, \"hgvsp\": \"NP_000129.3:p.Cys271Ter\", \"cds_end\": 813, \"conservation\": 2.96, \"lof\": \"HC\", \"protein_start\": 271, \"location\": \"15:48534129\", \"gene_id\": \"2200\", \"biotype\": \"protein_coding\", \"codons\": \"tgC/tgA\", \"allele_num\": 1, \"gene_symbol\": \"FBN1\", \"exon\": \"8/66\", \"ptriplo\": 0.984699554922396, \"mastermind_counts\": \"2|2|2\", \"loeuf\": 0.105, \"lof_info\": \"PERCENTILE:0.0943593314763231,GERP_DIST:7616.11479106406,BP_DIST:7745,DIST_FROM_LAST_EXON:7356,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\"}], \"strand\": 1, \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.02007320214458043, 'pca_y': 0.092355101291969},\n",
       "     {'t-sne_x': 29.321279418836458, 't-sne_y': 38.42929512469843},\n",
       "     {'umap_x': -10.335972785949707, 'umap_y': 2.3318560123443604}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48421652-G-T',\n",
       "      'distance': 0.0011025338899344206,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421601-G-T',\n",
       "      'distance': 0.0011990212369710207,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495241-G-T',\n",
       "      'distance': 0.001244019833393395,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495130-G-T',\n",
       "      'distance': 0.0013569036964327097,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495575-G-T',\n",
       "      'distance': 0.0014123886357992887,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503845-G-T',\n",
       "      'distance': 0.0014350325800478458,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474565-G-T',\n",
       "      'distance': 0.00160258321557194,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415610-G-T',\n",
       "      'distance': 0.0019793862011283636,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513553-G-T',\n",
       "      'distance': 0.0020302915945649147,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485483-G-T',\n",
       "      'distance': 0.002059753518551588,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.002137822099030018,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.0026352228596806526,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425416-G-T',\n",
       "      'distance': 0.002660741563886404,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.0026717642322182655,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421995-G-T',\n",
       "      'distance': 0.002728111110627651,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.003154005156829953,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.0032060472294688225,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445419-G-T',\n",
       "      'distance': 0.0033306176774203777,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.00350948772393167,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411338-C-T',\n",
       "      'distance': 0.003528027795255184,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48513626-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48513626\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48513626',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1511G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48513626C>A',\n",
       "     'hgvs_protein': 'p.Cys504Phe',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"C/A\", \"id\": \"NC_000015.10:g.48513626C>A\", \"transcript_consequences\": [{\"protein_start\": 504, \"alphamissense\": {\"am_pathogenicity\": 0.9933, \"am_class\": \"likely_pathogenic\"}, \"consequence_terms\": [\"missense_variant\"], \"given_ref\": \"C\", \"impact\": \"MODERATE\", \"eve_class\": \"Pathogenic\", \"ptriplo\": 0.984699554922396, \"mastermind_mmid3\": \"FBN1:C504F\", \"exon\": \"13/66\", \"spliceai\": {\"DS_DL\": 0, \"DS_AL\": 0, \"DP_DG\": 7, \"DS_AG\": 0, \"DP_DL\": 1, \"DS_DG\": 0, \"DP_AG\": -14, \"SYMBOL\": \"FBN1\", \"DP_AL\": 42}, \"polyphen_prediction\": \"probably_damaging\", \"mastermind_counts\": \"8|12|12\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 1827, \"aa\": \"C\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48513626, \"start\": 48513625}, \"type\": \"SequenceLocation\"}}, \"sift_score\": 0, \"mane_select\": \"ENST00000316623.10\", \"conservation\": 3.7, \"eve_score\": 0.755189131304482, \"mane\": [\"MANE_Select\"], \"cdna_end\": 1827, \"protein_end\": 504, \"cadd_raw\": 5.028604, \"mutationtaster_pred\": \"D\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.999936103820801, \"hgvsp\": \"NP_000129.3:p.Cys504Phe\", \"polyphen_score\": 0.996, \"biotype\": \"protein_coding\", \"loeuf\": 0.105, \"variant_allele\": \"A\", \"cadd_phred\": 28.1, \"allele_num\": 1, \"canonical\": 1, \"cds_start\": 1511, \"location\": \"15:48513626\", \"blosum62\": -2, \"hgvsc\": \"NM_000138.5:c.1511G>T\", \"gene_id\": \"2200\", \"codons\": \"tGt/tTt\", \"cds_end\": 1511, \"lrt_pred\": \"invalid_field\", \"used_ref\": \"C\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"sift_prediction\": \"deleterious_low_confidence\", \"amino_acids\": \"C/F\", \"phaplo\": 0.99897615427687, \"gene_symbol\": \"FBN1\"}], \"minimised\": 1, \"strand\": 1, \"assembly_name\": \"GRCh38\", \"start\": 48513626, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48513626C>A\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"end\": 48513626, \"start\": 48513626, \"id\": \"CM077275\"}, {\"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"end\": 48513626, \"start\": 48513626, \"id\": \"CM990586\", \"phenotype_or_disease\": 1, \"strand\": 1}, {\"phenotype_or_disease\": 1, \"strand\": 1, \"clin_sig_allele\": \"T:pathogenic;G:likely_pathogenic\", \"seq_region_name\": \"15\", \"var_synonyms\": {\"UniProt\": [\"VAR_010776\"], \"ClinVar\": [\"RCV002388173\", \"RCV000663465\", \"VCV000549021\", \"RCV000586374\", \"VCV000495557\"]}, \"allele_string\": \"C/A/G/T\", \"frequencies\": {\"A\": {\"gnomade_mid\": 0, \"gnomade_remaining\": 0, \"gnomade_asj\": 0, \"gnomade_nfe\": 8.994e-07, \"gnomade_eas\": 0, \"gnomade_sas\": 0, \"gnomade_afr\": 0, \"gnomade\": 6.841e-07, \"gnomade_amr\": 0, \"gnomade_fin\": 0}}, \"end\": 48513626, \"start\": 48513626, \"id\": \"rs1156747241\", \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"]}], \"end\": 48513626, \"vcf_string\": \"15-48513626-C-A\", \"variant_class\": \"SNV\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.032384387298876494,\n",
       "      'pca_y': 0.11220493466755646},\n",
       "     {'t-sne_x': -15.528024337312717, 't-sne_y': 35.48414731656477},\n",
       "     {'umap_x': 1.7570418119430542, 'umap_y': 21.345643997192383}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48515434-C-T',\n",
       "      'distance': 0.001314772991463542,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505117-C-A',\n",
       "      'distance': 0.0015695022884756327,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456642-C-A',\n",
       "      'distance': 0.0020427426788955927,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463949-C-A',\n",
       "      'distance': 0.0020555141381919384,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495519-C-A',\n",
       "      'distance': 0.0026705749332904816,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425413-C-A',\n",
       "      'distance': 0.002825664123520255,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474566-C-A',\n",
       "      'distance': 0.0033015732187777758,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513627-A-G',\n",
       "      'distance': 0.0035681859590113163,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513631-A-G',\n",
       "      'distance': 0.0035681859590113163,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48513600-A-G',\n",
       "      'distance': 0.003667389042675495,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415585-C-A',\n",
       "      'distance': 0.003719439497217536,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515429-A-C',\n",
       "      'distance': 0.0037707535084336996,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465619-A-T',\n",
       "      'distance': 0.003783040912821889,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.003922296687960625,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515429-A-G',\n",
       "      'distance': 0.003963448107242584,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472556-C-T',\n",
       "      'distance': 0.00403523538261652,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463950-A-C',\n",
       "      'distance': 0.004055473022162914,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472601-C-T',\n",
       "      'distance': 0.004075055010616779,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.004137028008699417,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.00414340989664197,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48515482-T-G',\n",
       "    'metadata': {'vcf_string': '15\\t48515482\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48515482',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1373A>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48515482T>G',\n",
       "     'hgvs_protein': 'p.Tyr458Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"allele_string\": \"T/G\", \"id\": \"NC_000015.10:g.48515482T>G\", \"strand\": 1, \"seq_region_name\": \"15\", \"transcript_consequences\": [{\"hgvsc\": \"NM_000138.5:c.1373A>C\", \"strand\": -1, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"end\": 48515482, \"start\": 48515481, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}}, \"cdna_start\": 1689, \"aa\": \"T\", \"consequence_terms\": [\"missense_variant\"], \"canonical\": 1, \"impact\": \"MODERATE\", \"mastermind_mmid3\": \"FBN1:Y458S\", \"mutationtaster_pred\": \"D\", \"variant_allele\": \"G\", \"lrt_pred\": \"invalid_field\", \"phaplo\": 0.99897615427687, \"cdna_end\": 1689, \"protein_id\": \"NP_000129.3\", \"used_ref\": \"T\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"amino_acids\": \"Y/S\", \"gene_symbol_source\": \"EntrezGene\", \"cadd_raw\": 3.570068, \"given_ref\": \"T\", \"alphamissense\": {\"am_pathogenicity\": 0.0565, \"am_class\": \"likely_benign\"}, \"mane_select\": \"ENST00000316623.10\", \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": -45, \"DP_DG\": -47, \"DP_AG\": 10, \"DS_DL\": 0, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": 33, \"DS_AL\": 0}, \"eve_score\": 0.106813313195603, \"cds_start\": 1373, \"clinpred\": 0.461333642157878, \"blosum62\": -2, \"sift_score\": 0.03, \"biotype\": \"protein_coding\", \"gene_id\": \"2200\", \"location\": \"15:48515482\", \"polyphen_prediction\": \"benign\", \"gene_symbol\": \"FBN1\", \"exon\": \"12/66\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"polyphen_score\": 0.003, \"mastermind_counts\": \"2|2|2\", \"allele_num\": 1, \"eve_class\": \"Benign\", \"codons\": \"tAt/tCt\", \"transcript_id\": \"NM_000138.5\", \"cadd_phred\": 22.9, \"protein_end\": 458, \"protein_start\": 458, \"cds_end\": 1373, \"conservation\": 3.95, \"hgvsp\": \"NP_000129.3:p.Tyr458Ser\"}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48515482, \"vcf_string\": \"15-48515482-T-G\", \"input\": \"NC_000015.10:g.48515482T>G\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"CM098642\", \"end\": 48515482, \"start\": 48515482}], \"start\": 48515482, \"assembly_name\": \"GRCh38\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.042002509835035756,\n",
       "      'pca_y': 0.074544006226414},\n",
       "     {'t-sne_x': 3.9251345621524636, 't-sne_y': 9.40314151216143},\n",
       "     {'umap_x': 3.759929656982422, 'umap_y': 3.2619032859802246}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.00292675057426095,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456687-C-T',\n",
       "      'distance': 0.003328160848468542,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495162-C-T',\n",
       "      'distance': 0.0033708408009260893,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456729-C-T',\n",
       "      'distance': 0.003418037900701165,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.0034649986773729324,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432976-C-T',\n",
       "      'distance': 0.003505062311887741,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596324-C-T',\n",
       "      'distance': 0.003605856327340007,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.0036128819920122623,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515476-C-T',\n",
       "      'distance': 0.0036131099332123995,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468462-C-T',\n",
       "      'distance': 0.0036160233430564404,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-T',\n",
       "      'distance': 0.0036510382778942585,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456714-C-T',\n",
       "      'distance': 0.003654800821095705,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.0036741108633577824,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421982-C-T',\n",
       "      'distance': 0.0036779693327844143,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.003686540760099888,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468456-C-T',\n",
       "      'distance': 0.003691092599183321,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465573-C-T',\n",
       "      'distance': 0.0037166988477110863,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-T',\n",
       "      'distance': 0.0037469319067895412,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537676-C-T',\n",
       "      'distance': 0.003754518460482359,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0037732708733528852,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48515386-48515434-del-ecce9618',\n",
       "    'metadata': {'vcf_string': '15\\t48515386\\t.\\tCCAATACACTCCCCACGGAGGTCCAGCTGGAACCCTTTGTTGCACTCAC\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48515386',\n",
       "     'ref_allele': 'CCAATACACTCCCCACGGAGGTCCAGCTGGAACCCTTTGTTGCACTCAC',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1423_1468+2del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48515387_48515434del',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"splice_donor_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48515387_48515434del\", \"start\": 48515387, \"seq_region_name\": \"15\", \"minimised\": 1, \"vcf_string\": \"15-48515386-CCAATACACTCCCCACGGAGGTCCAGCTGGAACCCTTTGTTGCACTCAC-C\", \"end\": 48515434, \"allele_string\": \"CAATACACTCCCCACGGAGGTCCAGCTGGAACCCTTTGTTGCACTCAC/-\", \"variant_class\": \"deletion\", \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"strand\": -1, \"canonical\": 1, \"protein_id\": \"NP_000129.3\", \"cds_end\": 1423, \"protein_start\": 475, \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"gene_symbol\": \"FBN1\", \"impact\": \"HIGH\", \"mane_select\": \"ENST00000316623.10\", \"variant_allele\": \"-\", \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"12/66\", \"phaplo\": 0.99897615427687, \"cdna_start\": 1739, \"loeuf\": 0.105, \"given_ref\": \"CAATACACTCCCCACGGAGGTCCAGCTGGAACCCTTTGTTGCACTCAC\", \"ptriplo\": 0.984699554922396, \"genesplicer\": \"gain/acceptor/48515402-48515401/Medium/1.100001\", \"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"end\": 48515434, \"type\": \"SimpleInterval\", \"start\": 48515386}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}}, \"conservation\": 2.5, \"protein_end\": 475, \"aa\": \"CAATACACTCCCCACGGAGGTCCAGCTGGAACCCTTTGTTGCACTCAC\", \"used_ref\": \"CAATACACTCCCCACGGAGGTCCAGCTGGAACCCTTTGTTGCACTCAC\", \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.1423_1468+2del\", \"gene_id\": \"2200\", \"allele_num\": 1, \"consequence_terms\": [\"splice_donor_variant\", \"coding_sequence_variant\"], \"cds_start\": 1423, \"cdna_end\": 1739, \"hgvs_offset\": -2, \"location\": \"15:48515385-48515432\"}], \"id\": \"NC_000015.10:g.48515387_48515434del\"}',\n",
       "    'coordinates': [{'pca_x': -0.18053256121627892,\n",
       "      'pca_y': 0.004284265301673897},\n",
       "     {'t-sne_x': -21.92920957315066, 't-sne_y': -25.00469710730107},\n",
       "     {'umap_x': 8.33222484588623, 'umap_y': 12.866095542907715}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48494203-C-G',\n",
       "      'distance': 0.10935357958078384,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425368-C-T',\n",
       "      'distance': 0.11209425330162048,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600138-C-G',\n",
       "      'distance': 0.11241946369409561,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488112-C-T',\n",
       "      'distance': 0.11366914212703705,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428345-C-T',\n",
       "      'distance': 0.11442349851131439,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496099-C-T',\n",
       "      'distance': 0.11484284698963165,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644605-C-G',\n",
       "      'distance': 0.1149248406291008,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.11510081589221954,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.11522044986486435,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.11613578349351883,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427566-C-A',\n",
       "      'distance': 0.11620713770389557,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.11633626371622086,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483816-A-G',\n",
       "      'distance': 0.11650899052619934,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.1165742576122284,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497265-C-T',\n",
       "      'distance': 0.11731576919555664,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.11818936467170715,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460244-A-C',\n",
       "      'distance': 0.11863352358341217,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456635-A-G',\n",
       "      'distance': 0.11929045617580414,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.11953271180391312,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.11955081671476364,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48510124-CGTCCA-C',\n",
       "    'metadata': {'vcf_string': '15\\t48510124\\t.\\tCGTCCA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48510124',\n",
       "     'ref_allele': 'CGTCCA',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1629_1633del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48510125_48510129del',\n",
       "     'hgvs_protein': 'p.Asn543LysfsTer14',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"start\": 48510125, \"minimised\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"input\": \"NC_000015.10:g.48510125_48510129del\", \"seq_region_name\": \"15\", \"transcript_consequences\": [{\"cds_end\": 1633, \"given_ref\": \"GTCCA\", \"strand\": -1, \"cds_start\": 1629, \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"mastermind_counts\": \"3|3|3\", \"conservation\": -0.544, \"location\": \"15:48510125-48510129\", \"used_ref\": \"GTCCA\", \"consequence_terms\": [\"frameshift_variant\"], \"gene_symbol\": \"FBN1\", \"protein_end\": 545, \"aa\": \"GTCCA\", \"mane_select\": \"ENST00000316623.10\", \"hgvsc\": \"NM_000138.5:c.1629_1633del\", \"gene_symbol_source\": \"EntrezGene\", \"impact\": \"HIGH\", \"cdna_end\": 1949, \"variant_allele\": \"-\", \"amino_acids\": \"NGR/KX\", \"biotype\": \"protein_coding\", \"codons\": \"aaTGGACgc/aagc\", \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"lof_info\": \"PERCENTILE:0.189531104921077,GERP_DIST:6958.70058792392,BP_DIST:6931,DIST_FROM_LAST_EXON:6542,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"hgvsp\": \"NP_000129.3:p.Asn543LysfsTer14\", \"transcript_id\": \"NM_000138.5\", \"lof\": \"HC\", \"gene_id\": \"2200\", \"canonical\": 1, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48510124, \"type\": \"SimpleInterval\", \"end\": 48510129}, \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"exon\": \"14/66\", \"mastermind_mmid3\": \"FBN1:N543fs\", \"cdna_start\": 1945, \"protein_id\": \"NP_000129.3\", \"protein_start\": 543}], \"allele_string\": \"GTCCA/-\", \"strand\": 1, \"assembly_name\": \"GRCh38\", \"variant_class\": \"deletion\", \"vcf_string\": \"15-48510124-CGTCCA-C\", \"id\": \"NC_000015.10:g.48510125_48510129del\", \"end\": 48510129}',\n",
       "    'coordinates': [{'pca_x': 0.04634929953883382,\n",
       "      'pca_y': 0.03977533331106959},\n",
       "     {'t-sne_x': 0.8501858151827619, 't-sne_y': 9.695890589792704},\n",
       "     {'umap_x': 3.533071517944336, 'umap_y': 3.4574196338653564}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.09915796667337418,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.10206044465303421,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.10382377356290817,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445419-G-T',\n",
       "      'distance': 0.10718872398138046,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445430-G-A',\n",
       "      'distance': 0.10723041743040085,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10732227563858032,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.1074444130063057,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10763248801231384,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.1078844964504242,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.10803069919347763,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10834380984306335,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.1089322566986084,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10903872549533844,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.10909591615200043,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.10919766873121262,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.10928493738174438,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.10931868851184845,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508612-G-A',\n",
       "      'distance': 0.10953617095947266,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.10954155772924423,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.10960403084754944,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48505101-G-T',\n",
       "    'metadata': {'vcf_string': '15\\t48505101\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48505101',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1884C>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48505101G>T',\n",
       "     'hgvs_protein': 'p.Cys628Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"start\": 48505101, \"allele_string\": \"G/T\", \"id\": \"NC_000015.10:g.48505101G>T\", \"seq_region_name\": \"15\", \"minimised\": 1, \"end\": 48505101, \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"hgvsp\": \"NP_000129.3:p.Cys628Ter\", \"gene_id\": \"2200\", \"cdna_end\": 2200, \"cdna_start\": 2200, \"impact\": \"HIGH\", \"cds_start\": 1884, \"consequence_terms\": [\"stop_gained\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"end\": 48505101, \"start\": 48505100, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"loeuf\": 0.105, \"protein_start\": 628, \"aa\": \"G\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48505101\", \"lof_info\": \"PERCENTILE:0.21866295264624,GERP_DIST:6738.2260405113,BP_DIST:6682,DIST_FROM_LAST_EXON:6293,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"used_ref\": \"G\", \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"mutationtaster_pred\": \"A\", \"biotype\": \"protein_coding\", \"cadd_raw\": 5.248905, \"cds_end\": 1884, \"given_ref\": \"G\", \"protein_end\": 628, \"cadd_phred\": 29.5, \"lof\": \"HC\", \"hgvsc\": \"NM_000138.5:c.1884C>A\", \"mastermind_counts\": \"4|4|4\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"variant_allele\": \"T\", \"canonical\": 1, \"spliceai\": {\"DP_DL\": -25, \"DP_AL\": 46, \"DS_DG\": 0, \"DS_AL\": 0.11, \"DP_DG\": -48, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AG\": -24}, \"exon\": \"16/66\", \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"conservation\": -2.58, \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"C/*\", \"codons\": \"tgC/tgA\", \"mastermind_mmid3\": \"FBN1:C628X\"}], \"colocated_variants\": [{\"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM021591\", \"phenotype_or_disease\": 1, \"strand\": 1, \"start\": 48505101, \"end\": 48505101}, {\"seq_region_name\": \"15\", \"strand\": 1, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM4683784\"]}, \"id\": \"COSV57318415\", \"allele_string\": \"COSMIC_MUTATION\", \"somatic\": 1, \"end\": 48505101, \"start\": 48505101}, {\"end\": 48505101, \"start\": 48505101, \"id\": \"CX022936\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}], \"vcf_string\": \"15-48505101-G-T\", \"most_severe_consequence\": \"stop_gained\", \"input\": \"NC_000015.10:g.48505101G>T\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.02509554351183982,\n",
       "      'pca_y': 0.09626269493323028},\n",
       "     {'t-sne_x': 29.893317619198893, 't-sne_y': 38.144765391943054},\n",
       "     {'umap_x': -10.32343578338623, 'umap_y': 2.414207935333252}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48421601-G-T',\n",
       "      'distance': 0.0018138717859983444,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513553-G-T',\n",
       "      'distance': 0.001820814679376781,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503845-G-T',\n",
       "      'distance': 0.002149679698050022,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495241-G-T',\n",
       "      'distance': 0.0022034309804439545,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421652-G-T',\n",
       "      'distance': 0.0022037485614418983,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445419-G-T',\n",
       "      'distance': 0.0023263676557689905,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415610-G-T',\n",
       "      'distance': 0.0025120177306234837,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495130-G-T',\n",
       "      'distance': 0.002618397120386362,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.0026218174025416374,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474565-G-T',\n",
       "      'distance': 0.002662152284756303,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.0027705924585461617,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485483-G-T',\n",
       "      'distance': 0.002833792008459568,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421995-G-T',\n",
       "      'distance': 0.0028406521305441856,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.0029908670112490654,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.0031896319705992937,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.0031928052194416523,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495575-G-T',\n",
       "      'distance': 0.0032701706513762474,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425416-G-T',\n",
       "      'distance': 0.0034984129015356302,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.0037015951238572598,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489921-G-T',\n",
       "      'distance': 0.004188552498817444,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48505079-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48505079\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48505079',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1906A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48505079T>C',\n",
       "     'hgvs_protein': 'p.Arg636Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"start\": 48505079, \"end\": 48505079, \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM141655\", \"seq_region_name\": \"15\"}], \"transcript_consequences\": [{\"protein_end\": 636, \"polyphen_score\": 0.057, \"cadd_phred\": 19.43, \"eve_class\": \"Uncertain\", \"sift_prediction\": \"deleterious_low_confidence\", \"mastermind_counts\": \"4|5|5\", \"strand\": -1, \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"C\", \"clinpred\": 0.982994318008423, \"hgvsc\": \"NM_000138.5:c.1906A>G\", \"spliceai\": {\"DP_DL\": -3, \"DP_AL\": -2, \"DS_DG\": 0.01, \"DS_AL\": 0, \"DP_DG\": -50, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AG\": -20}, \"blosum62\": -2, \"exon\": \"16/66\", \"lrt_pred\": \"invalid_field\", \"canonical\": 1, \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"conservation\": -3.43, \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"R/G\", \"mastermind_mmid3\": \"FBN1:R636G\", \"codons\": \"Aga/Gga\", \"cdna_end\": 2222, \"cdna_start\": 2222, \"hgvsp\": \"NP_000129.3:p.Arg636Gly\", \"gene_id\": \"2200\", \"cds_start\": 1906, \"eve_score\": 0.441774753464896, \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\"], \"protein_start\": 636, \"polyphen_prediction\": \"benign\", \"protein_id\": \"NP_000129.3\", \"aa\": \"T\", \"location\": \"15:48505079\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48505079, \"start\": 48505078, \"type\": \"SimpleInterval\"}}}, \"genesplicer\": \"loss/acceptor/48505079-48505080/Medium/1.364758\", \"loeuf\": 0.105, \"used_ref\": \"T\", \"alphamissense\": {\"am_class\": \"likely_benign\", \"am_pathogenicity\": 0.3308}, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"N\", \"sift_score\": 0, \"given_ref\": \"T\", \"cadd_raw\": 2.599624, \"biotype\": \"protein_coding\", \"cds_end\": 1906}], \"input\": \"NC_000015.10:g.48505079T>C\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"vcf_string\": \"15-48505079-T-C\", \"allele_string\": \"T/C\", \"assembly_name\": \"GRCh38\", \"start\": 48505079, \"minimised\": 1, \"end\": 48505079, \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48505079T>C\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.04268702124749346,\n",
       "      'pca_y': 0.07471039926323925},\n",
       "     {'t-sne_x': 2.2219012163263776, 't-sne_y': 1.5618238878823119},\n",
       "     {'umap_x': 4.785599708557129, 'umap_y': 2.9082579612731934}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.002673062961548567,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446816-T-C',\n",
       "      'distance': 0.0026851161383092403,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-T',\n",
       "      'distance': 0.0027537813875824213,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505081-T-C',\n",
       "      'distance': 0.0028573614545166492,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446775-T-C',\n",
       "      'distance': 0.0030395532958209515,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430727-T-C',\n",
       "      'distance': 0.00325160869397223,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489980-C-T',\n",
       "      'distance': 0.003322743345052004,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488148-T-C',\n",
       "      'distance': 0.0033354535698890686,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.0033754250034689903,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.0033833212219178677,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.0034344103187322617,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415671-T-C',\n",
       "      'distance': 0.0034363032318651676,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.003477917518466711,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48463207-T-C',\n",
       "      'distance': 0.0034884836059063673,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472607-T-C',\n",
       "      'distance': 0.003495211713016033,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497298-T-C',\n",
       "      'distance': 0.0035246657207608223,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497382-T-C',\n",
       "      'distance': 0.0035255898255854845,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.003528257831931114,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48415605-T-C',\n",
       "      'distance': 0.003574226750060916,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495162-C-T',\n",
       "      'distance': 0.0035805210936814547,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48497264-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48497264\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48497264',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2293+2T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48497264A>G',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48497264A>G\", \"variant_class\": \"SNV\", \"end\": 48497264, \"minimised\": 1, \"start\": 48497264, \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/G\", \"vcf_string\": \"15-48497264-A-G\", \"most_severe_consequence\": \"splice_donor_variant\", \"input\": \"NC_000015.10:g.48497264A>G\", \"strand\": 1, \"transcript_consequences\": [{\"cadd_phred\": 27.2, \"intron\": \"19/65\", \"lof\": \"HC\", \"canonical\": 1, \"spliceai\": {\"DS_AL\": 0, \"DS_DG\": 0.01, \"DP_AL\": 37, \"DP_DL\": 2, \"DP_AG\": -4, \"DS_DL\": 0.97, \"DP_DG\": -5, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\"}, \"hgvsc\": \"NM_000138.5:c.2293+2T>C\", \"variant_allele\": \"G\", \"strand\": -1, \"transcript_id\": \"NM_000138.5\", \"mastermind_counts\": \"4|4|6\", \"mastermind_mmid3\": \"FBN1:D765sd\", \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"impact\": \"HIGH\", \"rf_score\": 0.952, \"maxentscan_diff\": 7.754, \"gene_id\": \"2200\", \"loeuf\": 0.105, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48497263, \"end\": 48497264}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}}, \"protein_id\": \"NP_000129.3\", \"location\": \"15:48497264\", \"aa\": \"A\", \"consequence_terms\": [\"splice_donor_variant\"], \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"INTRON_SIZE:1040\", \"used_ref\": \"A\", \"biotype\": \"protein_coding\", \"cadd_raw\": 4.881488, \"ada_score\": 0.999990713472995, \"given_ref\": \"A\", \"maxentscan_alt\": -1.468, \"maxentscan_ref\": 6.286}], \"colocated_variants\": [{\"end\": 48497264, \"start\": 48497264, \"seq_region_name\": \"15\", \"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CS961584\", \"allele_string\": \"HGMD_MUTATION\"}]}',\n",
       "    'coordinates': [{'pca_x': -0.23911898032722506,\n",
       "      'pca_y': -0.03407695566148679},\n",
       "     {'t-sne_x': -15.099220435403632, 't-sne_y': -28.41094641660698},\n",
       "     {'umap_x': 8.26136589050293, 'umap_y': 13.102280616760254}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48437320-A-G',\n",
       "      'distance': 0.0007683661533519626,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465788-A-G',\n",
       "      'distance': 0.0012077116407454014,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437320-A-C',\n",
       "      'distance': 0.003412112593650818,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613009-C-A',\n",
       "      'distance': 0.004432858899235725,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508581-C-A',\n",
       "      'distance': 0.0047012269496917725,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465789-C-A',\n",
       "      'distance': 0.004849633201956749,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-A',\n",
       "      'distance': 0.005199050065129995,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-A',\n",
       "      'distance': 0.005200335290282965,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510042-A-G',\n",
       "      'distance': 0.005962883587926626,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.006283335387706757,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-G',\n",
       "      'distance': 0.006323481909930706,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.0063832346349954605,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515386-C-T',\n",
       "      'distance': 0.006560346111655235,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.0066216085106134415,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.006689836736768484,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.006807116791605949,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474254-C-T',\n",
       "      'distance': 0.006822989322245121,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.006863867864012718,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488112-C-T',\n",
       "      'distance': 0.006983778439462185,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.007043158635497093,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48495474-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48495474\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48495474',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2534G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48495474C>T',\n",
       "     'hgvs_protein': 'p.Cys845Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48495474C>T\", \"vcf_string\": \"15-48495474-C-T\", \"transcript_consequences\": [{\"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 845, \"eve_class\": \"Pathogenic\", \"cadd_phred\": 29, \"polyphen_score\": 0.972, \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C845Y\", \"codons\": \"tGc/tAc\", \"amino_acids\": \"C/Y\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"hgvsc\": \"NM_000138.5:c.2534G>A\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"T\", \"clinpred\": 0.999407052993774, \"strand\": -1, \"mastermind_counts\": \"3|3|3\", \"canonical\": 1, \"blosum62\": -2, \"lrt_pred\": \"invalid_field\", \"exon\": \"21/66\", \"spliceai\": {\"DP_AL\": -5, \"DP_DL\": -7, \"DS_AL\": 0, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DG\": -5, \"DS_AG\": 0, \"DP_AG\": 5, \"DS_DL\": 0}, \"consequence_terms\": [\"missense_variant\"], \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"type\": \"Allele\", \"location\": {\"interval\": {\"end\": 48495474, \"type\": \"SimpleInterval\", \"start\": 48495473}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"location\": \"15:48495474\", \"protein_id\": \"NP_000129.3\", \"aa\": \"C\", \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 845, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys845Tyr\", \"cdna_start\": 2850, \"cdna_end\": 2850, \"impact\": \"MODERATE\", \"eve_score\": 0.762964772534144, \"cds_start\": 2534, \"mutationtaster_pred\": \"D\", \"cds_end\": 2534, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.172084, \"sift_score\": 0, \"given_ref\": \"C\", \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.480381, \"dG_mt\": 62.29526, \"dG_wt\": 36.12738, \"dG_mt_sd\": 0.494562, \"dG_wt_sd\": 0.029683, \"ddG\": 26.16788}}, \"used_ref\": \"C\", \"alphamissense\": {\"am_pathogenicity\": 0.9982, \"am_class\": \"likely_pathogenic\"}, \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\"}], \"colocated_variants\": [{\"end\": 48495474, \"start\": 48495474, \"seq_region_name\": \"15\", \"id\": \"CM1311111\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1}], \"minimised\": 1, \"end\": 48495474, \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48495474C>T\", \"seq_region_name\": \"15\", \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"start\": 48495474}',\n",
       "    'coordinates': [{'pca_x': 0.03667804331903532,\n",
       "      'pca_y': 0.08396426863439374},\n",
       "     {'t-sne_x': 6.5485865618632575, 't-sne_y': 7.050710164338365},\n",
       "     {'umap_x': 3.4168107509613037, 'umap_y': 2.2418015003204346}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48495173-C-T',\n",
       "      'distance': 0.0005195253761485219,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.0005554492818191648,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495176-C-T',\n",
       "      'distance': 0.0006196537287905812,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496144-C-T',\n",
       "      'distance': 0.0006681724335066974,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496213-C-T',\n",
       "      'distance': 0.0007202387787401676,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.0007414420251734555,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497358-C-T',\n",
       "      'distance': 0.0007536960183642805,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492574-C-T',\n",
       "      'distance': 0.0007993958424776793,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0008871408645063639,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-T',\n",
       "      'distance': 0.0008912262273952365,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.0008981679566204548,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.0009057668503373861,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0009072509710676968,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.0009090957464650273,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497379-C-T',\n",
       "      'distance': 0.0009311724570579827,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497343-C-T',\n",
       "      'distance': 0.0009341776603832841,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.0009406866156496108,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520733-C-T',\n",
       "      'distance': 0.0009512525284662843,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.0009597037569619715,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.0010029769036918879,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48489979-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48489979\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48489979',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2954G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48489979C>T',\n",
       "     'hgvs_protein': 'p.Gly985Glu',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48489979C>T\", \"end\": 48489979, \"variant_class\": \"SNV\", \"minimised\": 1, \"start\": 48489979, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\", \"vcf_string\": \"15-48489979-C-T\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48489979C>T\", \"transcript_consequences\": [{\"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 985, \"cadd_phred\": 25, \"polyphen_score\": 0.999, \"eve_class\": \"Pathogenic\", \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"conservation\": 3.76, \"gene_symbol_source\": \"EntrezGene\", \"mastermind_mmid3\": \"FBN1:G985E\", \"codons\": \"gGg/gAg\", \"amino_acids\": \"G/E\", \"mastermind_counts\": \"10|12|12\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.999841809272766, \"variant_allele\": \"T\", \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.2954G>A\", \"spliceai\": {\"DS_AL\": 0, \"DS_DG\": 0, \"DP_AL\": -6, \"DP_DL\": 0, \"DP_AG\": -2, \"DS_DL\": 0, \"DP_DG\": -11, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\"}, \"exon\": \"25/66\", \"lrt_pred\": \"invalid_field\", \"blosum62\": -2, \"canonical\": 1, \"consequence_terms\": [\"missense_variant\"], \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 985, \"location\": \"15:48489979\", \"protein_id\": \"NP_000129.3\", \"aa\": \"C\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48489979, \"start\": 48489978, \"type\": \"SimpleInterval\"}}}, \"loeuf\": 0.105, \"cdna_end\": 3270, \"cdna_start\": 3270, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Gly985Glu\", \"cds_start\": 2954, \"eve_score\": 0.722640523926707, \"impact\": \"MODERATE\", \"mutationtaster_pred\": \"A\", \"given_ref\": \"C\", \"sift_score\": 0, \"cadd_raw\": 4.349507, \"biotype\": \"protein_coding\", \"cds_end\": 2954, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9937}, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396}], \"colocated_variants\": [{\"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM990588\", \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"end\": 48489979, \"start\": 48489979}, {\"seq_region_name\": \"15\", \"strand\": 1, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM8792514\"]}, \"id\": \"COSV100356110\", \"allele_string\": \"COSMIC_MUTATION\", \"somatic\": 1, \"end\": 48489979, \"start\": 48489979}, {\"seq_region_name\": \"15\", \"allele_string\": \"C/T\", \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"pubmed\": [12938084, 11700157, 9399842, 10441597], \"id\": \"rs137854477\", \"var_synonyms\": {\"ClinVar\": [\"RCV000017916\", \"VCV000016453\", \"RCV002433459\"], \"UniProt\": [\"VAR_018319\"], \"OMIM\": [134797.0034]}, \"phenotype_or_disease\": 1, \"strand\": 1, \"clin_sig_allele\": \"T:pathogenic;T:likely_pathogenic\", \"start\": 48489979, \"end\": 48489979}]}',\n",
       "    'coordinates': [{'pca_x': 0.03755076889170206,\n",
       "      'pca_y': 0.08627551980652173},\n",
       "     {'t-sne_x': 9.126509403639629, 't-sne_y': -1.7750942917081507},\n",
       "     {'umap_x': 4.4874138832092285, 'umap_y': 4.26547384262085}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48489977-C-T',\n",
       "      'distance': 0.0016594200860708952,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48420798-C-T',\n",
       "      'distance': 0.002114469651132822,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468474-C-T',\n",
       "      'distance': 0.0021803819108754396,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489896-C-T',\n",
       "      'distance': 0.0023682971950620413,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470684-C-T',\n",
       "      'distance': 0.0025417646393179893,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428459-C-T',\n",
       "      'distance': 0.0025956868194043636,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472595-C-T',\n",
       "      'distance': 0.002622624859213829,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425479-C-T',\n",
       "      'distance': 0.002627807669341564,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474299-C-T',\n",
       "      'distance': 0.002697490621358156,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488452-C-T',\n",
       "      'distance': 0.002715371549129486,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430676-C-T',\n",
       "      'distance': 0.002816582564264536,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434627-C-T',\n",
       "      'distance': 0.0029555046930909157,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421594-C-T',\n",
       "      'distance': 0.002969036577269435,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474572-C-T',\n",
       "      'distance': 0.002970322733744979,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.0029718568548560143,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427755-C-T',\n",
       "      'distance': 0.0029909792356193066,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.0030695709865540266,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425374-C-T',\n",
       "      'distance': 0.0030817226506769657,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425483-C-T',\n",
       "      'distance': 0.003081845585256815,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.003083245363086462,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48489857-AG-A',\n",
       "    'metadata': {'vcf_string': '15\\t48489857\\t.\\tAG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48489857',\n",
       "     'ref_allele': 'AG',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3075del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48489858del',\n",
       "     'hgvs_protein': 'p.Phe1026SerfsTer9',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"allele_string\": \"G/-\", \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"impact\": \"HIGH\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48489857, \"type\": \"SimpleInterval\", \"end\": 48489858}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"variant_allele\": \"-\", \"cdna_end\": 3391, \"used_ref\": \"G\", \"location\": \"15:48489858\", \"loeuf\": 0.105, \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DG\": -2, \"DP_AG\": 6, \"DS_DG\": 0, \"DP_DL\": -13, \"DS_AG\": 0, \"DP_AL\": -7, \"DS_AL\": 0, \"DS_DL\": 0}, \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"given_ref\": \"G\", \"ptriplo\": 0.984699554922396, \"lof_info\": \"PERCENTILE:0.356894150417827,GERP_DIST:6053.72533015311,BP_DIST:5500,DIST_FROM_LAST_EXON:5111,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"amino_acids\": \"F/X\", \"mane_select\": \"ENST00000316623.10\", \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"mastermind_counts\": \"2|2|2\", \"phaplo\": 0.99897615427687, \"mastermind_mmid3\": \"FBN1:F1026fs\", \"cds_start\": 3075, \"protein_id\": \"NP_000129.3\", \"codons\": \"ttC/tt\", \"aa\": \"G\", \"transcript_id\": \"NM_000138.5\", \"lof\": \"HC\", \"biotype\": \"protein_coding\", \"conservation\": 0.079, \"exon\": \"25/66\", \"gene_symbol_source\": \"EntrezGene\", \"hgvsp\": \"NP_000129.3:p.Phe1026SerfsTer9\", \"canonical\": 1, \"cds_end\": 3075, \"protein_end\": 1025, \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.3075del\", \"cdna_start\": 3391, \"protein_start\": 1025}], \"input\": \"NC_000015.10:g.48489858del\", \"id\": \"NC_000015.10:g.48489858del\", \"variant_class\": \"deletion\", \"end\": 48489858, \"strand\": 1, \"colocated_variants\": [{\"seq_region_name\": \"15\", \"end\": 48489858, \"strand\": 1, \"start\": 48489858, \"id\": \"CD102325\", \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\"}, {\"id\": \"COSV100355313\", \"start\": 48489858, \"var_synonyms\": {\"COSMIC\": [\"COSM9068478\"]}, \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48489858, \"allele_string\": \"COSMIC_MUTATION\", \"somatic\": 1, \"phenotype_or_disease\": 1}], \"start\": 48489858, \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"vcf_string\": \"15-48489857-AG-A\"}',\n",
       "    'coordinates': [{'pca_x': 0.04765073685565335,\n",
       "      'pca_y': 0.04001614475140324},\n",
       "     {'t-sne_x': 32.290261245999375, 't-sne_y': 11.561992408534488},\n",
       "     {'umap_x': 1.8736271858215332, 'umap_y': 7.694654941558838}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.09506836533546448,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.0950910747051239,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09571357816457748,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.09617225080728531,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.09621940553188324,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.09633209556341171,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613073-G-A',\n",
       "      'distance': 0.09669283777475357,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481723-G-A',\n",
       "      'distance': 0.09670588374137878,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.09674052894115448,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430710-G-A',\n",
       "      'distance': 0.09675093740224838,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.09680239856243134,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.09682600200176239,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.0969008132815361,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.09712471067905426,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.09731431305408478,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468097-G-A',\n",
       "      'distance': 0.09740295261144638,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.09742720425128937,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411243-G-A',\n",
       "      'distance': 0.09748415648937225,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.09755280613899231,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490073-G-A',\n",
       "      'distance': 0.09755340218544006,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.6666666666666666},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7}}},\n",
       "   {'variant_id': '15-48488464-G-A',\n",
       "    'metadata': {'vcf_string': '15\\t48488464\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488464',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3112C>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488464G>A',\n",
       "     'hgvs_protein': 'p.Leu1038Phe',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48488464, \"id\": \"CM136187\", \"start\": 48488464, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\"}, {\"var_synonyms\": {\"ClinVar\": [\"RCV001804537\", \"VCV001332021\"]}, \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48488464, \"clin_sig_allele\": \"A:uncertain_significance\", \"start\": 48488464, \"id\": \"rs1297814403\", \"phenotype_or_disease\": 1, \"frequencies\": {\"A\": {\"gnomade_remaining\": 0, \"gnomade_nfe\": 8.993e-07, \"gnomade\": 6.841e-07, \"gnomade_sas\": 0, \"gnomade_eas\": 0, \"gnomade_afr\": 0, \"gnomade_asj\": 0, \"gnomade_mid\": 0, \"gnomade_amr\": 0, \"gnomade_fin\": 0}}, \"clin_sig\": [\"uncertain_significance\"], \"allele_string\": \"G/A\"}], \"strand\": 1, \"end\": 48488464, \"vcf_string\": \"15-48488464-G-A\", \"seq_region_name\": \"15\", \"assembly_name\": \"GRCh38\", \"start\": 48488464, \"input\": \"NC_000015.10:g.48488464G>A\", \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"gene_id\": \"2200\", \"eve_score\": 0.552865790102707, \"cadd_phred\": 27, \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DG\": -49, \"DP_AG\": 29, \"DP_AL\": 47, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_DL\": 29, \"DS_DG\": 0, \"DS_AG\": 0}, \"alphamissense\": {\"am_class\": \"ambiguous\", \"am_pathogenicity\": 0.5264}, \"location\": \"15:48488464\", \"loeuf\": 0.105, \"used_ref\": \"G\", \"cdna_end\": 3428, \"variant_allele\": \"A\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48488463, \"end\": 48488464}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"type\": \"Allele\"}, \"mane\": [\"MANE_Select\"], \"impact\": \"MODERATE\", \"mastermind_counts\": \"4|4|4\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"eve_class\": \"Uncertain\", \"mutationtaster_pred\": \"D\", \"amino_acids\": \"L/F\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"blosum62\": 0, \"gene_symbol\": \"FBN1\", \"given_ref\": \"G\", \"aa\": \"G\", \"transcript_id\": \"NM_000138.5\", \"cadd_raw\": 4.843469, \"sift_prediction\": \"deleterious_low_confidence\", \"codons\": \"Ctc/Ttc\", \"protein_id\": \"NP_000129.3\", \"clinpred\": 0.935195446014404, \"lrt_pred\": \"invalid_field\", \"mastermind_mmid3\": \"FBN1:L1038F\", \"cds_start\": 3112, \"phaplo\": 0.99897615427687, \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 1038, \"sift_score\": 0, \"hgvsc\": \"NM_000138.5:c.3112C>T\", \"cdna_start\": 3428, \"strand\": -1, \"cds_end\": 3112, \"protein_end\": 1038, \"polyphen_score\": 0.997, \"hgvsp\": \"NP_000129.3:p.Leu1038Phe\", \"canonical\": 1, \"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"exon\": \"26/66\", \"conservation\": 3.95}], \"allele_string\": \"G/A\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48488464G>A\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.04301641302176751,\n",
       "      'pca_y': 0.08513755296984746},\n",
       "     {'t-sne_x': 31.21075902639593, 't-sne_y': 10.805996050722664},\n",
       "     {'umap_x': 1.8777800798416138, 'umap_y': 7.953482151031494}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430710-G-A',\n",
       "      'distance': 0.002322663087397814,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.002374769654124975,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437379-G-A',\n",
       "      'distance': 0.002393679227679968,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48490073-G-A',\n",
       "      'distance': 0.002405914943665266,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428427-G-A',\n",
       "      'distance': 0.002535445848479867,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481723-G-A',\n",
       "      'distance': 0.002677302807569504,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48411243-G-A',\n",
       "      'distance': 0.00272058486007154,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48497332-G-A',\n",
       "      'distance': 0.0027341493405401707,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48490013-G-A',\n",
       "      'distance': 0.002835717983543873,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487353-G-A',\n",
       "      'distance': 0.002900910098105669,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48537629-G-A',\n",
       "      'distance': 0.003002310171723366,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437009-G-A',\n",
       "      'distance': 0.003136517945677042,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415549-G-A',\n",
       "      'distance': 0.003226621076464653,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.003260737517848611,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.0032663869205862284,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.003380353795364499,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468097-G-A',\n",
       "      'distance': 0.003467717906460166,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.00347803789190948,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.003671790938824415,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432907-G-A',\n",
       "      'distance': 0.0037344684824347496,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.6},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.6666666666666666},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.75}}},\n",
       "   {'variant_id': '15-48488448-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48488448\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488448',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3128A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488448T>C',\n",
       "     'hgvs_protein': 'p.Lys1043Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48488448T>C\", \"input\": \"NC_000015.10:g.48488448T>C\", \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"amino_acids\": \"K/R\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"blosum62\": 2, \"gene_symbol\": \"FBN1\", \"given_ref\": \"T\", \"mastermind_counts\": \"19|25|25\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"eve_class\": \"Benign\", \"mutationtaster_pred\": \"A\", \"used_ref\": \"T\", \"variant_allele\": \"C\", \"cdna_end\": 3444, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48488447, \"type\": \"SimpleInterval\", \"end\": 48488448}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"type\": \"Allele\"}, \"mane\": [\"MANE_Select\"], \"impact\": \"MODERATE\", \"gene_id\": \"2200\", \"eve_score\": 0.132112941694326, \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DG\": 0, \"DP_AG\": -1, \"DS_DG\": 0, \"DP_DL\": -33, \"DS_AG\": 0.09, \"DP_AL\": 45, \"DS_AL\": 0.04, \"DS_DL\": 0}, \"cadd_phred\": 24, \"alphamissense\": {\"am_pathogenicity\": 0.0865, \"am_class\": \"likely_benign\"}, \"location\": \"15:48488448\", \"loeuf\": 0.105, \"protein_end\": 1043, \"cds_end\": 3128, \"polyphen_score\": 0.988, \"canonical\": 1, \"hgvsp\": \"NP_000129.3:p.Lys1043Arg\", \"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"conservation\": 3.95, \"exon\": \"26/66\", \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 1043, \"sift_score\": 0.3, \"hgvsc\": \"NM_000138.5:c.3128A>G\", \"cdna_start\": 3444, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"lrt_pred\": \"invalid_field\", \"mastermind_mmid3\": \"FBN1:K1043R\", \"clinpred\": 0.246092687496215, \"cds_start\": 3128, \"phaplo\": 0.99897615427687, \"aa\": \"T\", \"sift_prediction\": \"tolerated_low_confidence\", \"transcript_id\": \"NM_000138.5\", \"cadd_raw\": 3.985424, \"codons\": \"aAg/aGg\"}], \"allele_string\": \"T/C\", \"minimised\": 1, \"vcf_string\": \"15-48488448-T-C\", \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"start\": 48488448, \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"start\": 48488448, \"id\": \"CM970510\", \"seq_region_name\": \"15\", \"end\": 48488448, \"strand\": 1}, {\"strand\": 1, \"clin_sig_allele\": \"C:pathogenic;C:uncertain_significance;C:likely_benign\", \"end\": 48488448, \"var_synonyms\": {\"UniProt\": [\"VAR_002300\"], \"OMIM\": [134797.0027], \"ClinVar\": [\"RCV002321486\", \"RCV000017909\", \"VCV000016446\", \"RCV003764584\"]}, \"seq_region_name\": \"15\", \"pubmed\": [26741492, 9101298], \"start\": 48488448, \"id\": \"rs137854472\", \"phenotype_or_disease\": 1, \"frequencies\": {\"C\": {\"gnomade\": 4.788e-06, \"gnomade_nfe\": 0, \"gnomade_remaining\": 0, \"gnomade_eas\": 0.0001763, \"gnomade_afr\": 0, \"gnomade_sas\": 0, \"gnomade_asj\": 0, \"gnomade_amr\": 0, \"gnomade_mid\": 0, \"gnomade_fin\": 0}}, \"clin_sig\": [\"uncertain_significance\", \"likely_benign\", \"pathogenic\"], \"allele_string\": \"T/C\"}], \"strand\": 1, \"end\": 48488448}',\n",
       "    'coordinates': [{'pca_x': 0.03800531599231047,\n",
       "      'pca_y': 0.0818744346190978},\n",
       "     {'t-sne_x': 2.5342914638024685, 't-sne_y': 2.004389146447163},\n",
       "     {'umap_x': 4.606714248657227, 'umap_y': 2.7219350337982178}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.0019521450158208609,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487107-T-C',\n",
       "      'distance': 0.002081652171909809,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0021182361524552107,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481687-T-C',\n",
       "      'distance': 0.002150491112843156,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488148-T-C',\n",
       "      'distance': 0.002158623654395342,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472607-T-C',\n",
       "      'distance': 0.002185429912060499,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489980-C-T',\n",
       "      'distance': 0.002235859865322709,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497298-T-C',\n",
       "      'distance': 0.0023208700586110353,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0023447794374078512,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430727-T-C',\n",
       "      'distance': 0.0023571313358843327,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.0023991172201931477,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.002409365028142929,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0024434763472527266,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.0024478419218212366,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.0024678553454577923,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495176-C-T',\n",
       "      'distance': 0.0024813623167574406,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.0024847988970577717,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487131-T-C',\n",
       "      'distance': 0.0025036977604031563,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.0025191637687385082,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.002537185326218605,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488238-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48488238\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488238',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3212T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488238A>C',\n",
       "     'hgvs_protein': 'p.Ile1071Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"end\": 48488238, \"strand\": 1, \"colocated_variants\": [{\"end\": 48488238, \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"CM960632\", \"start\": 48488238, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\"}, {\"phenotype_or_disease\": 1, \"allele_string\": \"A/C\", \"clin_sig\": [\"uncertain_significance\"], \"clin_sig_allele\": \"C:uncertain_significance\", \"end\": 48488238, \"strand\": 1, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000663614\", \"VCV000549142\"]}, \"id\": \"rs1555398647\", \"start\": 48488238}], \"start\": 48488238, \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"vcf_string\": \"15-48488238-A-C\", \"minimised\": 1, \"allele_string\": \"A/C\", \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48488238A>C\", \"transcript_consequences\": [{\"strand\": -1, \"polyphen_prediction\": \"possibly_damaging\", \"protein_start\": 1071, \"sift_score\": 0, \"hgvsc\": \"NM_000138.5:c.3212T>G\", \"cdna_start\": 3528, \"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"conservation\": 3.95, \"exon\": \"27/66\", \"cds_end\": 3212, \"protein_end\": 1071, \"hgvsp\": \"NP_000129.3:p.Ile1071Ser\", \"polyphen_score\": 0.61, \"canonical\": 1, \"codons\": \"aTt/aGt\", \"aa\": \"A\", \"transcript_id\": \"NM_000138.5\", \"cadd_raw\": 6.401024, \"sift_prediction\": \"deleterious_low_confidence\", \"mastermind_mmid3\": \"FBN1:I1071S\", \"lrt_pred\": \"invalid_field\", \"clinpred\": 0.995168209075928, \"cds_start\": 3212, \"phaplo\": 0.99897615427687, \"protein_id\": \"NP_000129.3\", \"eve_class\": \"Pathogenic\", \"mutationtaster_pred\": \"D\", \"mastermind_counts\": \"12|16|16\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"gene_symbol\": \"FBN1\", \"blosum62\": -2, \"given_ref\": \"A\", \"amino_acids\": \"I/S\", \"mane_select\": \"ENST00000316623.10\", \"mutfunc\": {\"exp\": {\"dG_mt\": 170.5644, \"ddG_sd\": 0.033206, \"dG_mt_sd\": 0.018889, \"dG_wt_sd\": 0.033729, \"ddG\": -1.3598, \"dG_wt\": 171.9242}}, \"ptriplo\": 0.984699554922396, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9786}, \"loeuf\": 0.105, \"location\": \"15:48488238\", \"gene_id\": \"2200\", \"eve_score\": 0.653834110561083, \"spliceai\": {\"DS_AL\": 0, \"DS_DL\": 0, \"DP_AL\": -44, \"DS_AG\": 0.55, \"DP_DL\": -37, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": -1, \"DP_DG\": -1}, \"cadd_phred\": 34, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"end\": 48488238, \"start\": 48488237, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"mane\": [\"MANE_Select\"], \"impact\": \"MODERATE\", \"used_ref\": \"A\", \"cdna_end\": 3528, \"variant_allele\": \"C\"}], \"id\": \"NC_000015.10:g.48488238A>C\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.03866297681447556,\n",
       "      'pca_y': 0.0830560737086927},\n",
       "     {'t-sne_x': 20.35190828108938, 't-sne_y': 3.6783691636676847},\n",
       "     {'umap_x': 3.9176743030548096, 'umap_y': 6.3572187423706055}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.002196413930505514,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.002237408421933651,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.0023370631970465183,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.0025269847828894854,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.0025632414035499096,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.00261777825653553,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520677-A-C',\n",
       "      'distance': 0.0027543387841433287,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.0030088371131569147,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488419-A-C',\n",
       "      'distance': 0.0032233060337603092,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.0034333965741097927,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.00348097737878561,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.0035741243045777082,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415582-C-A',\n",
       "      'distance': 0.003663456067442894,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.0036695494782179594,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.003706029849126935,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497358-C-A',\n",
       "      'distance': 0.003713259007781744,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.0037150918506085873,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.0037151360884308815,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.003890776773914695,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.003984200302511454,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488231-T-A',\n",
       "    'metadata': {'vcf_string': '15\\t48488231\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488231',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3219A>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488231T>A',\n",
       "     'hgvs_protein': 'p.Glu1073Asp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48488231T>A\", \"minimised\": 1, \"allele_string\": \"T/A\", \"input\": \"NC_000015.10:g.48488231T>A\", \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"cds_start\": 3219, \"lrt_pred\": \"invalid_field\", \"clinpred\": 0.975277125835419, \"mastermind_mmid3\": \"FBN1:E1073D\", \"phaplo\": 0.99897615427687, \"cadd_raw\": 3.548228, \"transcript_id\": \"NM_000138.5\", \"sift_prediction\": \"deleterious_low_confidence\", \"aa\": \"T\", \"codons\": \"gaA/gaT\", \"cds_end\": 3219, \"protein_end\": 1073, \"polyphen_score\": 0.821, \"canonical\": 1, \"hgvsp\": \"NP_000129.3:p.Glu1073Asp\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": -4.82, \"exon\": \"27/66\", \"biotype\": \"protein_coding\", \"protein_start\": 1073, \"polyphen_prediction\": \"possibly_damaging\", \"cdna_start\": 3535, \"hgvsc\": \"NM_000138.5:c.3219A>T\", \"sift_score\": 0, \"strand\": -1, \"used_ref\": \"T\", \"variant_allele\": \"A\", \"cdna_end\": 3535, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48488231, \"start\": 48488230, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}}, \"impact\": \"MODERATE\", \"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"cadd_phred\": 22.8, \"spliceai\": {\"DS_AL\": 0, \"DS_DL\": 0, \"DP_AL\": -30, \"DS_AG\": 0, \"DP_DL\": -35, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DG\": -8, \"DP_AG\": 10}, \"eve_score\": 0.730334547709533, \"loeuf\": 0.105, \"location\": \"15:48488231\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9963}, \"mutfunc\": {\"exp\": {\"dG_wt\": 171.8736, \"ddG\": 0.9788, \"dG_mt_sd\": 0.059142, \"dG_wt_sd\": 0.077661, \"ddG_sd\": 0.02716, \"dG_mt\": 172.8524}}, \"amino_acids\": \"E/D\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"T\", \"blosum62\": 2, \"gene_symbol\": \"FBN1\", \"mastermind_counts\": \"11|17|17\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"eve_class\": \"Pathogenic\", \"mutationtaster_pred\": \"D\"}], \"start\": 48488231, \"seq_region_name\": \"15\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48488231-T-A\", \"end\": 48488231, \"strand\": 1, \"colocated_variants\": [{\"strand\": 1, \"end\": 48488231, \"seq_region_name\": \"15\", \"start\": 48488231, \"id\": \"CM960634\", \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\"}]}',\n",
       "    'coordinates': [{'pca_x': 0.040226327732644146,\n",
       "      'pca_y': 0.07958386635997017},\n",
       "     {'t-sne_x': 25.09420190908584, 't-sne_y': 13.2199434714967},\n",
       "     {'umap_x': 3.8390121459960938, 'umap_y': 7.436699390411377}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48488233-C-T',\n",
       "      'distance': 0.004235980100929737,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490073-G-A',\n",
       "      'distance': 0.00449850270524621,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.00453247269615531,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.004569056443870068,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415582-C-A',\n",
       "      'distance': 0.004614135250449181,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446814-C-T',\n",
       "      'distance': 0.00462151225656271,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497332-G-A',\n",
       "      'distance': 0.004722601734101772,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487353-G-A',\n",
       "      'distance': 0.00474843243137002,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.00476276408880949,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.004784220363944769,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.004793483763933182,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.004805976990610361,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.004886494483798742,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537629-G-A',\n",
       "      'distance': 0.004963225219398737,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.005016970448195934,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437069-C-T',\n",
       "      'distance': 0.005033861845731735,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.0050828345119953156,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430791-A-T',\n",
       "      'distance': 0.005103921517729759,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497358-C-A',\n",
       "      'distance': 0.005107802338898182,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.005131312180310488,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48487393-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48487393\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487393',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3382G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487393C>T',\n",
       "     'hgvs_protein': 'p.Val1128Ile',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"eve_score\": 0.0850046882240264, \"alphamissense\": {\"am_pathogenicity\": 0.0571, \"am_class\": \"likely_benign\"}, \"transcript_id\": \"NM_000138.5\", \"cdna_start\": 3698, \"cdna_end\": 3698, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:V1128I\", \"sift_score\": 0.31, \"phaplo\": 0.99897615427687, \"protein_end\": 1128, \"conservation\": -0.054, \"loeuf\": 0.105, \"polyphen_score\": 0.003, \"protein_start\": 1128, \"cds_end\": 3382, \"sift_prediction\": \"tolerated_low_confidence\", \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"impact\": \"MODERATE\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"D\", \"strand\": -1, \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.009396, \"dG_mt_sd\": 0.010358, \"dG_wt\": 174.8118, \"ddG\": -0.55839, \"dG_wt_sd\": 0.003898, \"dG_mt\": 174.25341}}, \"cds_start\": 3382, \"amino_acids\": \"V/I\", \"cadd_raw\": 0.246113, \"given_ref\": \"C\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"hgvsp\": \"NP_000129.3:p.Val1128Ile\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48487392, \"end\": 48487393}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"polyphen_prediction\": \"benign\", \"clinpred\": 0.136089533567429, \"location\": \"15:48487393\", \"aa\": \"C\", \"exon\": \"28/66\", \"blosum62\": 3, \"spliceai\": {\"DP_DG\": 6, \"DP_AL\": -16, \"DS_AG\": 0, \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": -17, \"SYMBOL\": \"FBN1\", \"DP_AG\": 27, \"DS_DL\": 0}, \"canonical\": 1, \"codons\": \"Gtt/Att\", \"cadd_phred\": 2.713, \"mastermind_counts\": \"5|11|11\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3382G>A\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Benign\", \"variant_allele\": \"T\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\"}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48487393, \"start\": 48487393, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM013926\", \"end\": 48487393, \"start\": 48487393, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1}, {\"phenotype_or_disease\": 1, \"id\": \"COSV57325336\", \"somatic\": 1, \"start\": 48487393, \"var_synonyms\": {\"COSMIC\": [\"COSM1373267\"]}, \"seq_region_name\": \"15\", \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"end\": 48487393}], \"id\": \"NC_000015.10:g.48487393C>T\", \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48487393C>T\", \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48487393-C-T\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.03797047616845789,\n",
       "      'pca_y': 0.08059780683290269},\n",
       "     {'t-sne_x': 7.86781815779347, 't-sne_y': 7.512536200217028},\n",
       "     {'umap_x': 3.1608123779296875, 'umap_y': 2.471991539001465}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.001753706601448357,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.0019988587591797113,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.0021429930347949266,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.002147907391190529,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.002189120277762413,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.00226925709284842,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.002310552168637514,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481699-C-T',\n",
       "      'distance': 0.002324966248124838,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415684-C-T',\n",
       "      'distance': 0.002332503441721201,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.002349353861063719,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495173-C-T',\n",
       "      'distance': 0.0024030283093452454,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485379-C-T',\n",
       "      'distance': 0.00241054967045784,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520733-C-T',\n",
       "      'distance': 0.0024179837200790644,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.002435085829347372,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496213-C-T',\n",
       "      'distance': 0.002444349229335785,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.0024456593673676252,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.002453687833622098,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.0024697938933968544,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.0024741948582232,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492574-C-T',\n",
       "      'distance': 0.0024784195702522993,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48487367-G-C',\n",
       "    'metadata': {'vcf_string': '15\\t48487367\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487367',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3408C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487367G>C',\n",
       "     'hgvs_protein': 'p.Tyr1136Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"id\": \"NC_000015.10:g.48487367G>C\", \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48487367, \"end\": 48487367, \"id\": \"CD117753\", \"phenotype_or_disease\": 1}, {\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48487367, \"end\": 48487367, \"id\": \"CM085419\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\", \"start\": 48487367, \"most_severe_consequence\": \"stop_gained\", \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"mastermind_mmid3\": \"FBN1:Y1136X\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_end\": 3724, \"cdna_start\": 3724, \"conservation\": 2.2, \"loeuf\": 0.105, \"protein_end\": 1136, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"G\", \"protein_start\": 1136, \"cds_end\": 3408, \"amino_acids\": \"Y/*\", \"cds_start\": 3408, \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.067843, \"dG_mt_sd\": 0.055964, \"dG_wt\": 174.7894, \"dG_wt_sd\": 0.042104, \"ddG\": 0.036599, \"dG_mt\": 174.825999}}, \"mutationtaster_pred\": \"A\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"location\": \"15:48487367\", \"hgvsp\": \"NP_000129.3:p.Tyr1136Ter\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48487367, \"start\": 48487366, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"consequence_terms\": [\"stop_gained\"], \"allele_num\": 1, \"given_ref\": \"G\", \"cadd_raw\": 8.240099, \"hgvsc\": \"NM_000138.5:c.3408C>G\", \"biotype\": \"protein_coding\", \"cadd_phred\": 37, \"mastermind_counts\": \"7|7|7\", \"codons\": \"taC/taG\", \"canonical\": 1, \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": 10, \"DP_DG\": -49, \"DS_DL\": 0, \"DP_AG\": 31, \"SYMBOL\": \"FBN1\", \"DP_DL\": 10, \"DS_AL\": 0}, \"lof\": \"HC\", \"exon\": \"28/66\", \"aa\": \"G\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"C\", \"lof_info\": \"PERCENTILE:0.395543175487465,GERP_DIST:6053.72533015311,BP_DIST:5170,DIST_FROM_LAST_EXON:4781,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol_source\": \"EntrezGene\"}], \"end\": 48487367, \"strand\": 1, \"vcf_string\": \"15-48487367-G-C\", \"allele_string\": \"G/C\", \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48487367G>C\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.014993798327142316,\n",
       "      'pca_y': 0.0940698499349055},\n",
       "     {'t-sne_x': 28.599473239968432, 't-sne_y': 43.90448821332245},\n",
       "     {'umap_x': -10.107992172241211, 'umap_y': 3.379448413848877}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487106-G-C',\n",
       "      'distance': 0.0007870844565331936,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489921-G-C',\n",
       "      'distance': 0.0010942235821858048,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437901-G-C',\n",
       "      'distance': 0.0013532794546335936,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420706-G-C',\n",
       "      'distance': 0.0014696142170578241,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48412647-G-C',\n",
       "      'distance': 0.0019640731625258923,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596311-G-C',\n",
       "      'distance': 0.00208043260499835,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463209-G-C',\n",
       "      'distance': 0.003437757957726717,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48613075-G-C',\n",
       "      'distance': 0.004047169350087643,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495570-G-C',\n",
       "      'distance': 0.005019358359277248,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437362-A-C',\n",
       "      'distance': 0.005033071152865887,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.0051000420935451984,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485483-G-T',\n",
       "      'distance': 0.005744637455791235,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495241-G-T',\n",
       "      'distance': 0.0059654973447322845,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415610-G-T',\n",
       "      'distance': 0.006122093182057142,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489921-G-T',\n",
       "      'distance': 0.006177403032779694,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474565-G-T',\n",
       "      'distance': 0.006179230287671089,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468080-A-C',\n",
       "      'distance': 0.0061921607702970505,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495130-G-T',\n",
       "      'distance': 0.00632213382050395,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.006361912004649639,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.006378724705427885,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48485484-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48485484\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48485484',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3602G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48485484C>T',\n",
       "     'hgvs_protein': 'p.Cys1201Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"start\": 48485484, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48485484, \"phenotype_or_disease\": 1, \"id\": \"CM081261\"}, {\"start\": 48485484, \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM962392\"]}, \"seq_region_name\": \"15\", \"end\": 48485484, \"phenotype_or_disease\": 1, \"id\": \"COSV57328496\", \"somatic\": 1}, {\"end\": 48485484, \"start\": 48485484, \"strand\": 1, \"allele_string\": \"C/T\", \"seq_region_name\": \"15\", \"clin_sig\": [\"likely_pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV000663654\", \"VCV000549176\"]}, \"clin_sig_allele\": \"T:likely_pathogenic\", \"id\": \"rs1555398413\", \"phenotype_or_disease\": 1}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48485484C>T\", \"minimised\": 1, \"transcript_consequences\": [{\"hgvsp\": \"NP_000129.3:p.Cys1201Tyr\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48485484, \"start\": 48485483, \"type\": \"SimpleInterval\"}}}, \"cadd_raw\": 5.205581, \"given_ref\": \"C\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.99993908405304, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48485484\", \"strand\": -1, \"mutationtaster_pred\": \"A\", \"cds_start\": 3602, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"ptriplo\": 0.984699554922396, \"amino_acids\": \"C/Y\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"T\", \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": 5, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": -4, \"DP_AL\": 12, \"DP_DG\": 38, \"DS_DG\": 0, \"DS_AG\": 0}, \"aa\": \"C\", \"blosum62\": -2, \"exon\": \"30/66\", \"hgvsc\": \"NM_000138.5:c.3602G>A\", \"biotype\": \"protein_coding\", \"codons\": \"tGc/tAc\", \"canonical\": 1, \"cadd_phred\": 29.2, \"mastermind_counts\": \"7|7|7\", \"lrt_pred\": \"invalid_field\", \"cdna_start\": 3918, \"cdna_end\": 3918, \"mastermind_mmid3\": \"FBN1:C1201Y\", \"mane\": [\"MANE_Select\"], \"eve_score\": 0.665662217352091, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.9974, \"am_class\": \"likely_pathogenic\"}, \"polyphen_score\": 0.988, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"protein_start\": 1201, \"cds_end\": 3602, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 1201, \"sift_score\": 0, \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"conservation\": 3.85}], \"most_severe_consequence\": \"missense_variant\", \"end\": 48485484, \"start\": 48485484, \"allele_string\": \"C/T\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48485484-C-T\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48485484C>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.03821626920870591,\n",
       "      'pca_y': 0.08934021752772495},\n",
       "     {'t-sne_x': 7.492375583504148, 't-sne_y': 11.224309684564597},\n",
       "     {'umap_x': 3.037745475769043, 'umap_y': 2.7838335037231445}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48434599-C-T',\n",
       "      'distance': 0.0015491914236918092,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437827-C-T',\n",
       "      'distance': 0.0017076642252504826,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415695-C-T',\n",
       "      'distance': 0.0019380011362954974,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468462-C-T',\n",
       "      'distance': 0.0020390001591295004,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0020685310009866953,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468456-C-T',\n",
       "      'distance': 0.0021582678891718388,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420794-C-T',\n",
       "      'distance': 0.002199865411967039,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.002401582896709442,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468489-C-T',\n",
       "      'distance': 0.0025708249304443598,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.0025931093841791153,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.002607586095109582,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596318-C-T',\n",
       "      'distance': 0.002649960806593299,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.0026510502211749554,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.002676942851394415,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485379-C-T',\n",
       "      'distance': 0.0026906549464911222,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.002702455036342144,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481699-C-T',\n",
       "      'distance': 0.0027126427739858627,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0027463457081466913,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.002756012137979269,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488451-C-T',\n",
       "      'distance': 0.0027577581349760294,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48474624-G-GTG',\n",
       "    'metadata': {'vcf_string': '15\\t48474624\\t.\\tG\\tGTG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474624',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'GTG',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3993_3994dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474625_48474626dup',\n",
       "     'hgvs_protein': 'p.Asn1332ThrfsTer82',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"start\": 48474625, \"end\": 48474624, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"mane\": [\"MANE_Select\"], \"cdna_end\": 4311, \"cdna_start\": 4310, \"conservation\": -1.97, \"loeuf\": 0.105, \"protein_end\": 1332, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"-\", \"protein_start\": 1332, \"cds_end\": 3995, \"amino_acids\": \"N/TX\", \"cds_start\": 3994, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"location\": \"15:48474620-48474621\", \"hgvs_offset\": -4, \"hgvsp\": \"NP_000129.3:p.Asn1332ThrfsTer82\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"start\": 48474624, \"end\": 48474624}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"TG\", \"type\": \"SequenceState\"}}, \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"given_ref\": \"-\", \"hgvsc\": \"NM_000138.5:c.3993_3994dup\", \"biotype\": \"protein_coding\", \"codons\": \"aac/aCAac\", \"canonical\": 1, \"lof\": \"HC\", \"exon\": \"33/66\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"TG\", \"lof_info\": \"PERCENTILE:0.46355617455896,GERP_DIST:5411.35726435483,BP_DIST:4593,DIST_FROM_LAST_EXON:4204,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol_source\": \"EntrezGene\"}], \"minimised\": 1, \"id\": \"NC_000015.10:g.48474625_48474626dup\", \"variant_class\": \"insertion\", \"input\": \"NC_000015.10:g.48474625_48474626dup\", \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48474624-G-GTG\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"-/TG\"}',\n",
       "    'coordinates': [{'pca_x': 0.02727280476591465,\n",
       "      'pca_y': 0.00767071921228351},\n",
       "     {'t-sne_x': 11.40569442532571, 't-sne_y': 6.8311820851062555},\n",
       "     {'umap_x': 3.2314529418945312, 'umap_y': 3.6177423000335693}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.08823668211698532,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09063879400491714,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.09101936966180801,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.09128948301076889,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.09138251841068268,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09160499274730682,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.0917280912399292,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468504-C-T',\n",
       "      'distance': 0.09176763147115707,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.09198561310768127,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.09207775443792343,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.09230170398950577,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.09241393953561783,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.09245720505714417,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446768-A-G',\n",
       "      'distance': 0.09253845363855362,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.09260490536689758,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437807-A-G',\n",
       "      'distance': 0.09260725229978561,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430770-A-G',\n",
       "      'distance': 0.09267128258943558,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0926833301782608,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.09269137680530548,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415684-C-T',\n",
       "      'distance': 0.09269975870847702,\n",
       "      'pathogenicity': 'benign_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.6},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8}}},\n",
       "   {'variant_id': '15-48474603-CA-C',\n",
       "    'metadata': {'vcf_string': '15\\t48474603\\t.\\tCA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474603',\n",
       "     'ref_allele': 'CA',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4011del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474604del',\n",
       "     'hgvs_protein': 'p.Val1338TyrfsTer75',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"id\": \"NC_000015.10:g.48474604del\", \"variant_class\": \"deletion\", \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48474604, \"end\": 48474604, \"phenotype_or_disease\": 1, \"id\": \"CD982648\"}, {\"pubmed\": [17657824], \"phenotype_or_disease\": 1, \"id\": \"rs2043404959\", \"end\": 48474604, \"seq_region_name\": \"15\", \"clin_sig\": [\"pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV001059506\", \"VCV000854447\"]}, \"allele_string\": \"A/-\", \"strand\": 1, \"start\": 48474604}], \"start\": 48474604, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:V1338fs\", \"cdna_start\": 4327, \"cdna_end\": 4327, \"transcript_id\": \"NM_000138.5\", \"protein_start\": 1337, \"cds_end\": 4011, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"A\", \"conservation\": -3.19, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"protein_end\": 1337, \"location\": \"15:48474604\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"given_ref\": \"A\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"start\": 48474603, \"end\": 48474604}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"hgvsp\": \"NP_000129.3:p.Val1338TyrfsTer75\", \"amino_acids\": \"A/X\", \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"genesplicer\": \"gain/donor/48474604-48474605/Medium/2.651054\", \"cds_start\": 4011, \"strand\": -1, \"lof_info\": \"PERCENTILE:0.465529247910864,GERP_DIST:5391.99287199974,BP_DIST:4572,DIST_FROM_LAST_EXON:4183,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"mastermind_counts\": \"5|5|5\", \"codons\": \"gcT/gc\", \"canonical\": 1, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.4011del\", \"exon\": \"33/66\", \"aa\": \"A\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 23, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 1, \"DS_DG\": 0.01, \"DS_AG\": 0, \"DP_DG\": 2, \"DP_AL\": 47}, \"lof\": \"HC\"}], \"end\": 48474604, \"strand\": 1, \"vcf_string\": \"15-48474603-CA-C\", \"allele_string\": \"A/-\", \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48474604del\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.049342894689975825,\n",
       "      'pca_y': 0.042510305212568635},\n",
       "     {'t-sne_x': 11.064141854580164, 't-sne_y': 14.941756741464573},\n",
       "     {'umap_x': 2.329277992248535, 'umap_y': 6.316009044647217}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09117302298545837,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.0922192856669426,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09230741113424301,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.09273801743984222,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.09309923648834229,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09353430569171906,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.09362705796957016,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.09371338039636612,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.09385673701763153,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420752-A-G',\n",
       "      'distance': 0.09391292929649353,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.0939260870218277,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.09393331408500671,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.09395284950733185,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.09396609663963318,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.09397191554307938,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.09403407573699951,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.09412560611963272,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.09416064620018005,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.09442397952079773,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.09444357454776764,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48474320-T-A',\n",
       "    'metadata': {'vcf_string': '15\\t48474320\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474320',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4145A>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474320T>A',\n",
       "     'hgvs_protein': 'p.Asn1382Ile',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"id\": \"NC_000015.10:g.48474320T>A\", \"transcript_consequences\": [{\"cadd_phred\": 29.9, \"biotype\": \"protein_coding\", \"variant_allele\": \"A\", \"loeuf\": 0.105, \"hgvsp\": \"NP_000129.3:p.Asn1382Ile\", \"polyphen_score\": 0.928, \"clinpred\": 0.999683737754822, \"transcript_id\": \"NM_000138.5\", \"blosum62\": -3, \"location\": \"15:48474320\", \"allele_num\": 1, \"canonical\": 1, \"cds_start\": 4145, \"used_ref\": \"T\", \"lrt_pred\": \"invalid_field\", \"hgvsc\": \"NM_000138.5:c.4145A>T\", \"gene_id\": \"2200\", \"codons\": \"aAt/aTt\", \"cds_end\": 4145, \"phaplo\": 0.99897615427687, \"gene_symbol\": \"FBN1\", \"sift_prediction\": \"deleterious_low_confidence\", \"amino_acids\": \"N/I\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"eve_class\": \"Pathogenic\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\"], \"given_ref\": \"T\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9952}, \"protein_start\": 1382, \"aa\": \"T\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 4461, \"polyphen_prediction\": \"probably_damaging\", \"spliceai\": {\"DP_AG\": 20, \"DP_AL\": 16, \"SYMBOL\": \"FBN1\", \"DP_DL\": -44, \"DS_AG\": 0, \"DP_DG\": 20, \"DS_AL\": 0, \"DS_DL\": 0, \"DS_DG\": 0}, \"mastermind_counts\": \"5|5|5\", \"exon\": \"34/66\", \"mastermind_mmid3\": \"FBN1:N1382I\", \"sift_score\": 0, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48474320, \"start\": 48474319}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}}, \"mane_select\": \"ENST00000316623.10\", \"conservation\": 3.95, \"cadd_raw\": 5.319837, \"mutationtaster_pred\": \"D\", \"protein_end\": 1382, \"cdna_end\": 4461, \"mane\": [\"MANE_Select\"], \"eve_score\": 0.76789242892003}], \"allele_string\": \"T/A\", \"start\": 48474320, \"assembly_name\": \"GRCh38\", \"strand\": 1, \"vcf_string\": \"15-48474320-T-A\", \"end\": 48474320, \"colocated_variants\": [{\"strand\": 1, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"id\": \"CM077926\", \"start\": 48474320, \"end\": 48474320}, {\"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"end\": 48474320, \"id\": \"CM950447\", \"start\": 48474320}, {\"phenotype_or_disease\": 1, \"clin_sig_allele\": \"A:likely_benign\", \"strand\": 1, \"end\": 48474320, \"start\": 48474320, \"clin_sig\": [\"likely_benign\"], \"id\": \"rs1555397661\", \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000659539\", \"VCV000547318\"]}, \"allele_string\": \"T/A\"}], \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48474320T>A\", \"variant_class\": \"SNV\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.038153115528987036,\n",
       "      'pca_y': 0.08895075978139672},\n",
       "     {'t-sne_x': 16.948038828000506, 't-sne_y': 8.7267714231908},\n",
       "     {'umap_x': 3.5717110633850098, 'umap_y': 5.54335880279541}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468527-A-T',\n",
       "      'distance': 0.002491689519956708,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.0040305862203240395,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48411243-G-A',\n",
       "      'distance': 0.004181492142379284,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48490050-C-A',\n",
       "      'distance': 0.004232534673064947,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48472628-C-A',\n",
       "      'distance': 0.004471177235245705,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495513-C-A',\n",
       "      'distance': 0.0044723376631736755,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434701-C-T',\n",
       "      'distance': 0.004741666838526726,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465840-C-A',\n",
       "      'distance': 0.004770340397953987,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-T',\n",
       "      'distance': 0.004902808461338282,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596325-A-T',\n",
       "      'distance': 0.004910345654934645,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468522-C-A',\n",
       "      'distance': 0.004927669186145067,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465825-C-T',\n",
       "      'distance': 0.005132828373461962,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415549-G-A',\n",
       "      'distance': 0.005222740583121777,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430710-G-A',\n",
       "      'distance': 0.005257380194962025,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48481723-G-A',\n",
       "      'distance': 0.005287590436637402,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48422024-A-G',\n",
       "      'distance': 0.00533636100590229,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437379-G-A',\n",
       "      'distance': 0.005343837197870016,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468001-A-G',\n",
       "      'distance': 0.005369722843170166,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415632-C-T',\n",
       "      'distance': 0.0054165953770279884,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432911-A-G',\n",
       "      'distance': 0.005418054293841124,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.6},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.7333333333333333},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8}}},\n",
       "   {'variant_id': '15-48474277-TGTGTATCCT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48474277\\t.\\tTGTGTATCCT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474277',\n",
       "     'ref_allele': 'TGTGTATCCT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4182_4190del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474278_48474286del',\n",
       "     'hgvs_protein': 'p.Tyr1395_Gly1397del',\n",
       "     'most_severe_consequence': 'inframe_deletion'},\n",
       "    'annotation_raw': '{\"allele_string\": \"GTGTATCCT/-\", \"id\": \"NC_000015.10:g.48474278_48474286del\", \"transcript_consequences\": [{\"used_ref\": \"CCTGTGTAT\", \"codons\": \"ggATACACAGGt/ggt\", \"gene_id\": \"2200\", \"cds_end\": 4190, \"hgvsc\": \"NM_000138.5:c.4182_4190del\", \"phaplo\": 0.99897615427687, \"gene_symbol\": \"FBN1\", \"amino_acids\": \"GYTG/G\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"variant_allele\": \"-\", \"loeuf\": 0.105, \"biotype\": \"protein_coding\", \"hgvs_offset\": -3, \"hgvsp\": \"NP_000129.3:p.Tyr1395_Gly1397del\", \"transcript_id\": \"NM_000138.5\", \"location\": \"15:48474275-48474283\", \"cds_start\": 4182, \"canonical\": 1, \"allele_num\": 1, \"conservation\": 1.752, \"mane_select\": \"ENST00000316623.10\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"end\": 48474286, \"type\": \"SimpleInterval\", \"start\": 48474277}}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}}, \"protein_end\": 1397, \"mane\": [\"MANE_Select\"], \"cdna_end\": 4506, \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"consequence_terms\": [\"inframe_deletion\"], \"given_ref\": \"GTGTATCCT\", \"protein_start\": 1394, \"aa\": \"GTGTATCCT\", \"cdna_start\": 4498, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"34/66\"}], \"minimised\": 1, \"assembly_name\": \"GRCh38\", \"strand\": 1, \"start\": 48474278, \"input\": \"NC_000015.10:g.48474278_48474286del\", \"most_severe_consequence\": \"inframe_deletion\", \"end\": 48474286, \"vcf_string\": \"15-48474277-TGTGTATCCT-T\", \"variant_class\": \"deletion\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.011531322001176791,\n",
       "      'pca_y': 0.04416025372345075},\n",
       "     {'t-sne_x': 11.532182674247986, 't-sne_y': 1.7648513022861023},\n",
       "     {'umap_x': 2.414891242980957, 'umap_y': 6.200529098510742}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48420752-A-G',\n",
       "      'distance': 0.12125041335821152,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-G',\n",
       "      'distance': 0.1224113181233406,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420798-C-T',\n",
       "      'distance': 0.12274114787578583,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.12275853753089905,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613073-G-A',\n",
       "      'distance': 0.12340481579303741,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503850-A-G',\n",
       "      'distance': 0.12342073023319244,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415707-C-T',\n",
       "      'distance': 0.1234952062368393,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.12356312572956085,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420795-A-G',\n",
       "      'distance': 0.12375088036060333,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.12377822399139404,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474584-C-T',\n",
       "      'distance': 0.12388642877340317,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421594-C-T',\n",
       "      'distance': 0.12401571124792099,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.12414302676916122,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468097-G-A',\n",
       "      'distance': 0.12415559589862823,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430676-C-T',\n",
       "      'distance': 0.12418165802955627,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.12435004115104675,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481723-G-A',\n",
       "      'distance': 0.12438968569040298,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.12441812455654144,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.12443539500236511,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.12444669008255005,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48472617-G-C',\n",
       "    'metadata': {'vcf_string': '15\\t48472617\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48472617',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4270C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48472617G>C',\n",
       "     'hgvs_protein': 'p.Pro1424Ala',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"end\": 48472617, \"vcf_string\": \"15-48472617-G-C\", \"input\": \"NC_000015.10:g.48472617G>C\", \"most_severe_consequence\": \"missense_variant\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"end\": 48472617, \"id\": \"CD067546\", \"start\": 48472617, \"phenotype_or_disease\": 1, \"strand\": 1}, {\"phenotype_or_disease\": 1, \"strand\": 1, \"end\": 48472617, \"start\": 48472617, \"id\": \"CM054685\", \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\"}, {\"phenotype_or_disease\": 1, \"strand\": 1, \"end\": 48472617, \"start\": 48472617, \"id\": \"CM983868\", \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\"}, {\"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000035194\", \"VCV000042355\", \"RCV000586163\", \"RCV000755199\", \"RCV000771903\", \"RCV000765222\", \"RCV000226865\", \"RCV000161118\", \"RCV001121324\", \"RCV001121323\", \"RCV001121322\", \"RCV001121321\", \"RCV001116398\", \"RCV001386703\", \"VCV001073640\"], \"UniProt\": [\"VAR_018010\"]}, \"allele_string\": \"G/A/C\", \"frequencies\": {\"C\": {\"gnomadg\": 0.0003089, \"gnomade_fin\": 0, \"gnomadg_sas\": 0, \"gnomadg_nfe\": 0.0001764, \"gnomade\": 0.0003133, \"gnomade_afr\": 0.0002389, \"gnomadg_eas\": 0, \"gnomadg_afr\": 7.241e-05, \"gnomade_sas\": 0, \"gnomadg_amr\": 0.002029, \"gnomadg_remaining\": 0.0004789, \"gnomade_nfe\": 0.0003606, \"gnomade_amr\": 0.0006931, \"gnomadg_mid\": 0, \"gnomadg_asj\": 0, \"gnomadg_fin\": 0, \"gnomadg_ami\": 0, \"gnomade_eas\": 0, \"gnomade_mid\": 0, \"gnomade_remaining\": 0.000298, \"gnomade_asj\": 0}}, \"end\": 48472617, \"start\": 48472617, \"id\": \"rs201273753\", \"clin_sig\": [\"uncertain_significance\", \"benign\", \"likely_benign\", \"pathogenic\"], \"pubmed\": [24033266, 24941995, 12938084, 14695540, 11524736, 16222657, 17657824, 9399842], \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"C:uncertain_significance;C:likely_benign;C:benign;A:pathogenic;C:conflicting_classifications_of_pathogenicity\", \"strand\": 1}], \"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48472617G>C\", \"transcript_consequences\": [{\"mastermind_counts\": \"28|33|33\", \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_DL\": -15, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_DG\": -1, \"SYMBOL\": \"FBN1\", \"DP_AL\": -4, \"DP_AG\": -1}, \"polyphen_prediction\": \"probably_damaging\", \"mastermind_mmid3\": \"FBN1:P1424A\", \"exon\": \"35/66\", \"aa\": \"G\", \"cdna_start\": 4586, \"gene_symbol_source\": \"EntrezGene\", \"given_ref\": \"G\", \"consequence_terms\": [\"missense_variant\"], \"alphamissense\": {\"am_class\": \"likely_benign\", \"am_pathogenicity\": 0.0962}, \"protein_start\": 1424, \"ptriplo\": 0.984699554922396, \"eve_class\": \"Benign\", \"geno2mp_hpo_count\": 11, \"impact\": \"MODERATE\", \"cdna_end\": 4586, \"mane\": [\"MANE_Select\"], \"eve_score\": 0.224979964176667, \"cadd_raw\": 4.832984, \"mutationtaster_pred\": \"D\", \"protein_end\": 1424, \"conservation\": 3.95, \"sift_score\": 0.01, \"mane_select\": \"ENST00000316623.10\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"interval\": {\"start\": 48472616, \"type\": \"SimpleInterval\", \"end\": 48472617}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}}, \"cds_start\": 4270, \"canonical\": 1, \"allele_num\": 1, \"blosum62\": -1, \"location\": \"15:48472617\", \"polyphen_score\": 0.993, \"hgvsp\": \"NP_000129.3:p.Pro1424Ala\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.481280416250229, \"cadd_phred\": 27, \"loeuf\": 0.105, \"variant_allele\": \"C\", \"geno2mp_url\": \"https://geno2mp.gs.washington.edu/Geno2MP/#/variant/15/48764814/G%3EC/snp\", \"biotype\": \"protein_coding\", \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"gene_symbol\": \"FBN1\", \"phaplo\": 0.99897615427687, \"sift_prediction\": \"deleterious_low_confidence\", \"amino_acids\": \"P/A\", \"lrt_pred\": \"invalid_field\", \"codons\": \"Cca/Gca\", \"gene_id\": \"2200\", \"cds_end\": 4270, \"hgvsc\": \"NM_000138.5:c.4270C>G\", \"used_ref\": \"G\"}], \"minimised\": 1, \"allele_string\": \"G/C\", \"start\": 48472617, \"assembly_name\": \"GRCh38\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.027398084872515882,\n",
       "      'pca_y': 0.1069262973019231},\n",
       "     {'t-sne_x': -15.457452610808039, 't-sne_y': 28.232172419096525},\n",
       "     {'umap_x': 1.8693805932998657, 'umap_y': 21.844451904296875}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48515428-C-T',\n",
       "      'distance': 0.0029552197083830833,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425412-G-C',\n",
       "      'distance': 0.003518529701977968,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503845-G-C',\n",
       "      'distance': 0.0037621627561748028,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505101-G-C',\n",
       "      'distance': 0.004193063825368881,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515434-C-A',\n",
       "      'distance': 0.004450405947864056,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508633-A-C',\n",
       "      'distance': 0.004901097156107426,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474345-A-C',\n",
       "      'distance': 0.005018058232963085,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.005054309964179993,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-G',\n",
       "      'distance': 0.005196946207433939,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463949-C-G',\n",
       "      'distance': 0.005418224260210991,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437026-T-C',\n",
       "      'distance': 0.005493371747434139,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505136-A-C',\n",
       "      'distance': 0.005496225319802761,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48441764-A-C',\n",
       "      'distance': 0.005582081153988838,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452637-A-G',\n",
       "      'distance': 0.0056662652641534805,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437038-C-T',\n",
       "      'distance': 0.005775827914476395,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472628-C-T',\n",
       "      'distance': 0.005825186613947153,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481729-T-C',\n",
       "      'distance': 0.00584037508815527,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48472556-C-T',\n",
       "      'distance': 0.005900670308619738,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.005965270102024078,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463950-A-C',\n",
       "      'distance': 0.005976046435534954,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48468417-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48468417\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48468417',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4577G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48468417C>G',\n",
       "     'hgvs_protein': 'p.Cys1526Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48468417, \"strand\": 1, \"assembly_name\": \"GRCh38\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48468417C>G\", \"transcript_consequences\": [{\"eve_class\": \"Pathogenic\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\"], \"given_ref\": \"C\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9997}, \"protein_start\": 1526, \"aa\": \"C\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 4893, \"polyphen_prediction\": \"possibly_damaging\", \"spliceai\": {\"DP_DG\": -5, \"DS_DL\": 0.01, \"DS_AL\": 0, \"DP_DL\": -1, \"DS_AG\": 0, \"DS_DG\": 0, \"DP_AG\": 20, \"DP_AL\": -1, \"SYMBOL\": \"FBN1\"}, \"mastermind_counts\": \"1|2|2\", \"mastermind_mmid3\": \"FBN1:C1526S\", \"exon\": \"37/66\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48468417, \"start\": 48468416}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"sift_score\": 0, \"mane_select\": \"ENST00000316623.10\", \"conservation\": 3.95, \"cadd_raw\": 5.15187, \"mutationtaster_pred\": \"D\", \"protein_end\": 1526, \"cdna_end\": 4893, \"mane\": [\"MANE_Select\"], \"eve_score\": 0.747048882791876, \"mutfunc\": {\"mod\": {\"dG_mt\": 85.59206, \"ddG\": 5.08706, \"dG_wt\": 80.505, \"dG_wt_sd\": 0.018523, \"dG_mt_sd\": 0.103415, \"ddG_sd\": 0.113954}, \"exp\": {\"ddG_sd\": 0.043571, \"ddG\": 5.09522, \"dG_mt\": 27.70658, \"dG_mt_sd\": 0.035978, \"dG_wt\": 22.61136, \"dG_wt_sd\": 0.025125}}, \"cadd_phred\": 28.9, \"biotype\": \"protein_coding\", \"loeuf\": 0.105, \"variant_allele\": \"G\", \"hgvsp\": \"NP_000129.3:p.Cys1526Ser\", \"polyphen_score\": 0.461, \"clinpred\": 0.999948978424072, \"transcript_id\": \"NM_000138.5\", \"blosum62\": -1, \"location\": \"15:48468417\", \"allele_num\": 1, \"cds_start\": 4577, \"canonical\": 1, \"used_ref\": \"C\", \"lrt_pred\": \"invalid_field\", \"hgvsc\": \"NM_000138.5:c.4577G>C\", \"gene_id\": \"2200\", \"codons\": \"tGt/tCt\", \"cds_end\": 4577, \"gene_symbol\": \"FBN1\", \"phaplo\": 0.99897615427687, \"sift_prediction\": \"deleterious_low_confidence\", \"amino_acids\": \"C/S\", \"strand\": -1, \"protein_id\": \"NP_000129.3\"}], \"allele_string\": \"C/G\", \"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"vcf_string\": \"15-48468417-C-G\", \"end\": 48468417, \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"end\": 48468417, \"start\": 48468417, \"id\": \"CM077928\"}, {\"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM096448\", \"start\": 48468417, \"end\": 48468417, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\"}], \"input\": \"NC_000015.10:g.48468417C>G\", \"most_severe_consequence\": \"missense_variant\"}',\n",
       "    'coordinates': [{'pca_x': 0.03791449613782272,\n",
       "      'pca_y': 0.07575007989030683},\n",
       "     {'t-sne_x': 19.982924739428498, 't-sne_y': -5.667388576930791},\n",
       "     {'umap_x': 8.631585121154785, 'umap_y': 7.0901312828063965}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0008170827059075236,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.0008993649389594793,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0010089706629514694,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.0010867260862141848,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.0011943503050133586,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.0013632173649966717,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.0014361791545525193,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.0014880998060107231,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.00164904841221869,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.0018017800757661462,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.002081863349303603,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468455-G-C',\n",
       "      'distance': 0.0021225963719189167,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.002164744306355715,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415708-C-G',\n",
       "      'distance': 0.0022852884139865637,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.002367083914577961,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.0024116283748298883,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489991-C-G',\n",
       "      'distance': 0.002441877033561468,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.0024853090289980173,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.0025442165788263083,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520708-C-G',\n",
       "      'distance': 0.0026212423108518124,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48468096-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48468096\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48468096',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4589G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48468096C>T',\n",
       "     'hgvs_protein': 'p.Arg1530His',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"strand\": 1, \"end\": 48468096, \"start\": 48468096, \"id\": \"CM136793\", \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\"}, {\"start\": 48468096, \"id\": \"COSV57308525\", \"end\": 48468096, \"allele_string\": \"COSMIC_MUTATION\", \"seq_region_name\": \"15\", \"somatic\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM1678442\"]}, \"strand\": 1, \"phenotype_or_disease\": 1}, {\"allele_string\": \"C/A/G/T\", \"var_synonyms\": {\"ClinVar\": [\"RCV001188772\", \"VCV000926280\", \"RCV003048412\", \"VCV002134989\", \"RCV002246002\", \"RCV002314296\", \"RCV001220153\", \"VCV000519692\"]}, \"seq_region_name\": \"15\", \"clin_sig\": [\"uncertain_significance\", \"likely_pathogenic\"], \"id\": \"rs778710767\", \"start\": 48468096, \"end\": 48468096, \"frequencies\": {\"T\": {\"gnomade_mid\": 0, \"gnomade_remaining\": 0, \"gnomade_nfe\": 1.799e-06, \"gnomade_asj\": 0, \"gnomade_eas\": 0, \"gnomade_sas\": 1.159e-05, \"gnomade_afr\": 0, \"gnomade\": 2.052e-06, \"gnomade_amr\": 0, \"gnomade_fin\": 0}}, \"strand\": 1, \"clin_sig_allele\": \"T:uncertain_significance;G:likely_pathogenic;A:likely_pathogenic\", \"phenotype_or_disease\": 1, \"pubmed\": [11700157]}], \"input\": \"NC_000015.10:g.48468096C>T\", \"most_severe_consequence\": \"missense_variant\", \"vcf_string\": \"15-48468096-C-T\", \"end\": 48468096, \"strand\": 1, \"assembly_name\": \"GRCh38\", \"start\": 48468096, \"allele_string\": \"C/T\", \"minimised\": 1, \"transcript_consequences\": [{\"used_ref\": \"C\", \"lrt_pred\": \"invalid_field\", \"gene_id\": \"2200\", \"codons\": \"cGc/cAc\", \"cds_end\": 4589, \"hgvsc\": \"NM_000138.5:c.4589G>A\", \"phaplo\": 0.99897615427687, \"gene_symbol\": \"FBN1\", \"sift_prediction\": \"deleterious_low_confidence\", \"amino_acids\": \"R/H\", \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_phred\": 32, \"mutfunc\": {\"mod\": {\"ddG_sd\": 0.099985, \"ddG\": 2.0307, \"dG_mt\": 82.04484, \"dG_mt_sd\": 0.093605, \"dG_wt\": 80.01414, \"dG_wt_sd\": 0.024524}, \"exp\": {\"ddG_sd\": 0.094328, \"dG_wt\": 22.69488, \"dG_wt_sd\": 0.000376, \"dG_mt_sd\": 0.094219, \"dG_mt\": 25.15066, \"ddG\": 2.45578}}, \"loeuf\": 0.105, \"variant_allele\": \"T\", \"biotype\": \"protein_coding\", \"polyphen_score\": 0.949, \"hgvsp\": \"NP_000129.3:p.Arg1530His\", \"clinpred\": 0.997008860111237, \"transcript_id\": \"NM_000138.5\", \"blosum62\": 0, \"location\": \"15:48468096\", \"canonical\": 1, \"cds_start\": 4589, \"allele_num\": 1, \"conservation\": 3.7, \"sift_score\": 0, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48468095, \"type\": \"SimpleInterval\", \"end\": 48468096}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"mane_select\": \"ENST00000316623.10\", \"cadd_raw\": 5.532377, \"mutationtaster_pred\": \"D\", \"protein_end\": 1530, \"cdna_end\": 4905, \"mane\": [\"MANE_Select\"], \"eve_score\": 0.800929422303928, \"ptriplo\": 0.984699554922396, \"eve_class\": \"Pathogenic\", \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\"], \"given_ref\": \"C\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.7647}, \"protein_start\": 1530, \"aa\": \"C\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 4905, \"mastermind_counts\": \"3|4|4\", \"spliceai\": {\"DP_AG\": -36, \"DP_AL\": 6, \"SYMBOL\": \"FBN1\", \"DP_DL\": -10, \"DS_AG\": 0.01, \"DP_DG\": 15, \"DS_DL\": 0, \"DS_AL\": 0.01, \"DS_DG\": 0}, \"polyphen_prediction\": \"probably_damaging\", \"exon\": \"38/66\", \"mastermind_mmid3\": \"FBN1:R1530H\"}], \"id\": \"NC_000015.10:g.48468096C>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.040854310054869725,\n",
       "      'pca_y': 0.08026977302740851},\n",
       "     {'t-sne_x': 9.069817330121795, 't-sne_y': -4.328405214175792},\n",
       "     {'umap_x': 4.311956882476807, 'umap_y': 4.262296199798584}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48474299-C-T',\n",
       "      'distance': 0.0020126020535826683,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472595-C-T',\n",
       "      'distance': 0.002036836463958025,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428459-C-T',\n",
       "      'distance': 0.002115790266543627,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470684-C-T',\n",
       "      'distance': 0.002165931509807706,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.00220126798376441,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48430676-C-T',\n",
       "      'distance': 0.0022815645206719637,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468474-C-T',\n",
       "      'distance': 0.0023189582861959934,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.0025401879101991653,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48436966-C-T',\n",
       "      'distance': 0.0025706402957439423,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-T',\n",
       "      'distance': 0.0026809971313923597,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437872-C-T',\n",
       "      'distance': 0.0026913597248494625,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489977-C-T',\n",
       "      'distance': 0.0027100376319140196,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48437369-C-T',\n",
       "      'distance': 0.0027229436673223972,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.0027910680510103703,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425479-C-T',\n",
       "      'distance': 0.0027970459777861834,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495176-C-T',\n",
       "      'distance': 0.002797867404296994,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497379-C-T',\n",
       "      'distance': 0.0028060837648808956,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434627-C-T',\n",
       "      'distance': 0.0028095035813748837,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.002824974013492465,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432916-C-T',\n",
       "      'distance': 0.002839053515344858,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48465853-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48465853\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48465853',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4753T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48465853A>T',\n",
       "     'hgvs_protein': 'p.Tyr1585Asn',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48465853A>T\", \"allele_string\": \"A/T\", \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48465853A>T\", \"transcript_consequences\": [{\"impact\": \"MODERATE\", \"mane\": [\"MANE_Select\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48465852, \"type\": \"SimpleInterval\", \"end\": 48465853}}}, \"cdna_end\": 5069, \"variant_allele\": \"T\", \"used_ref\": \"A\", \"location\": \"15:48465853\", \"loeuf\": 0.105, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9147}, \"spliceai\": {\"DP_DG\": -27, \"DP_AG\": 5, \"SYMBOL\": \"FBN1\", \"DS_DL\": 0, \"DS_AL\": 0, \"DP_AL\": -28, \"DS_AG\": 0, \"DP_DL\": 5, \"DS_DG\": 0}, \"cadd_phred\": 27.6, \"eve_score\": 0.68493285759494, \"gene_id\": \"2200\", \"given_ref\": \"A\", \"blosum62\": -2, \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"mutfunc\": {\"mod\": {\"dG_mt\": 83.168279, \"ddG_sd\": 0.057027, \"dG_mt_sd\": 0.071139, \"dG_wt_sd\": 0.017732, \"dG_wt\": 81.13642, \"ddG\": 2.031859}, \"exp\": {\"dG_wt_sd\": 0.000223, \"dG_mt_sd\": 0.01557, \"dG_wt\": 22.6954, \"ddG\": 3.46552, \"dG_mt\": 26.16092, \"ddG_sd\": 0.015702}}, \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"Y/N\", \"mutationtaster_pred\": \"D\", \"eve_class\": \"Pathogenic\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"mastermind_counts\": \"3|3|3\", \"phaplo\": 0.99897615427687, \"cds_start\": 4753, \"clinpred\": 0.998860359191895, \"lrt_pred\": \"invalid_field\", \"mastermind_mmid3\": \"FBN1:Y1585N\", \"protein_id\": \"NP_000129.3\", \"codons\": \"Tac/Aac\", \"transcript_id\": \"NM_000138.5\", \"sift_prediction\": \"deleterious_low_confidence\", \"cadd_raw\": 4.950156, \"aa\": \"A\", \"exon\": \"39/66\", \"conservation\": 3.95, \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"canonical\": 1, \"polyphen_score\": 1, \"hgvsp\": \"NP_000129.3:p.Tyr1585Asn\", \"protein_end\": 1585, \"cds_end\": 4753, \"strand\": -1, \"cdna_start\": 5069, \"hgvsc\": \"NM_000138.5:c.4753T>A\", \"sift_score\": 0, \"protein_start\": 1585, \"polyphen_prediction\": \"probably_damaging\"}], \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"start\": 48465853, \"vcf_string\": \"15-48465853-A-T\", \"end\": 48465853, \"colocated_variants\": [{\"id\": \"CM127267\", \"start\": 48465853, \"seq_region_name\": \"15\", \"end\": 48465853, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1}], \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.040667023247945855,\n",
       "      'pca_y': 0.08579044498577931},\n",
       "     {'t-sne_x': 5.783751611998748, 't-sne_y': -0.7908296894155334},\n",
       "     {'umap_x': 3.995269298553467, 'umap_y': 4.735254287719727}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468527-A-T',\n",
       "      'distance': 0.001999291591346264,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596325-A-T',\n",
       "      'distance': 0.003589936764910817,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428439-A-T',\n",
       "      'distance': 0.0036963033489882946,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-T',\n",
       "      'distance': 0.003711789846420288,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465825-C-T',\n",
       "      'distance': 0.003871580585837364,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465840-C-A',\n",
       "      'distance': 0.0038758190348744392,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425446-C-T',\n",
       "      'distance': 0.00419576745480299,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.004230655264109373,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.004271261394023895,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434627-C-T',\n",
       "      'distance': 0.004278414882719517,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415632-C-T',\n",
       "      'distance': 0.004379950929433107,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427684-A-T',\n",
       "      'distance': 0.004514552652835846,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432955-C-T',\n",
       "      'distance': 0.004534037783741951,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432916-C-T',\n",
       "      'distance': 0.0046896133571863174,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425816-C-T',\n",
       "      'distance': 0.004767265170812607,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432910-C-T',\n",
       "      'distance': 0.00488235941156745,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445421-A-T',\n",
       "      'distance': 0.004923808388411999,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430791-A-T',\n",
       "      'distance': 0.0049982815980911255,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425852-C-T',\n",
       "      'distance': 0.005079355556517839,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468522-C-A',\n",
       "      'distance': 0.005105690564960241,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48465694-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48465694\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48465694',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4817-1G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48465694C>T',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"assembly_name\": \"GRCh38\", \"start\": 48465694, \"vcf_string\": \"15-48465694-C-T\", \"end\": 48465694, \"colocated_variants\": [{\"end\": 48465694, \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"CS022871\", \"start\": 48465694, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\"}, {\"start\": 48465694, \"id\": \"rs2141269787\", \"var_synonyms\": {\"ClinVar\": [\"RCV001550700\", \"VCV001190111\"]}, \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48465694, \"clin_sig_allele\": \"T:pathogenic\", \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"C/T\", \"phenotype_or_disease\": 1}], \"strand\": 1, \"id\": \"NC_000015.10:g.48465694C>T\", \"variant_class\": \"SNV\", \"allele_string\": \"C/T\", \"minimised\": 1, \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"phaplo\": 0.99897615427687, \"mastermind_mmid3\": \"FBN1:D1606sa\", \"transcript_id\": \"NM_000138.5\", \"cadd_raw\": 5.741221, \"maxentscan_diff\": 8.75, \"aa\": \"C\", \"lof\": \"HC\", \"intron\": \"39/65\", \"canonical\": 1, \"conservation\": 3.95, \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"hgvsc\": \"NM_000138.5:c.4817-1G>A\", \"rf_score\": 0.938, \"maxentscan_ref\": 7.347, \"strand\": -1, \"ada_score\": 0.999990361384395, \"variant_allele\": \"T\", \"used_ref\": \"C\", \"maxentscan_alt\": -1.403, \"impact\": \"HIGH\", \"mane\": [\"MANE_Select\"], \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48465693, \"type\": \"SimpleInterval\", \"end\": 48465694}}, \"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}}, \"cadd_phred\": 33, \"spliceai\": {\"DS_DL\": 0, \"DS_AL\": 0.99, \"DP_AL\": -1, \"DS_AG\": 0.23, \"DS_DG\": 0, \"DP_DL\": -48, \"DP_AG\": -18, \"DP_DG\": -18, \"SYMBOL\": \"FBN1\"}, \"genesplicer\": \"loss/acceptor/48465693-48465694/Medium/3.108769\", \"gene_id\": \"2200\", \"location\": \"15:48465694\", \"loeuf\": 0.105, \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"INTRON_SIZE:96\", \"given_ref\": \"C\", \"gene_symbol\": \"FBN1\", \"consequence_terms\": [\"splice_acceptor_variant\"], \"allele_num\": 1, \"mastermind_counts\": \"5|5|13\"}], \"most_severe_consequence\": \"splice_acceptor_variant\", \"input\": \"NC_000015.10:g.48465694C>T\"}',\n",
       "    'coordinates': [{'pca_x': -0.24236745228524859,\n",
       "      'pca_y': -0.04004492804556252},\n",
       "     {'t-sne_x': -29.009380623627923, 't-sne_y': -18.43560953714732},\n",
       "     {'umap_x': 11.867774963378906, 'umap_y': 18.726680755615234}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.0007599701639264822,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.0008293765131384134,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.0011529037728905678,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.001154748722910881,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.0017994936788454652,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.0021594595164060593,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.0023186441976577044,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.0025468836538493633,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.002602695021778345,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.002655611140653491,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.002665857784450054,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.002921285340562463,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.0030701416544616222,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.003225035732612014,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.0035052422899752855,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.0035688350908458233,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470757-C-T',\n",
       "      'distance': 0.0038341539911925793,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446823-C-G',\n",
       "      'distance': 0.004348161164671183,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-A',\n",
       "      'distance': 0.0046243066899478436,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.004959460347890854,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48456715-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48456715\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48456715',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5344T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48456715A>G',\n",
       "     'hgvs_protein': 'p.Cys1782Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"end\": 48456715, \"strand\": 1, \"colocated_variants\": [{\"start\": 48456715, \"id\": \"CM045410\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48456715, \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1}], \"start\": 48456715, \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"vcf_string\": \"15-48456715-A-G\", \"minimised\": 1, \"allele_string\": \"A/G\", \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"sift_prediction\": \"deleterious_low_confidence\", \"transcript_id\": \"NM_000138.5\", \"cadd_raw\": 4.998151, \"aa\": \"A\", \"codons\": \"Tgt/Cgt\", \"protein_id\": \"NP_000129.3\", \"cds_start\": 5344, \"lrt_pred\": \"invalid_field\", \"clinpred\": 0.999884843826294, \"mastermind_mmid3\": \"FBN1:C1782R\", \"phaplo\": 0.99897615427687, \"protein_start\": 1782, \"polyphen_prediction\": \"probably_damaging\", \"hgvsc\": \"NM_000138.5:c.5344T>C\", \"cdna_start\": 5660, \"sift_score\": 0, \"strand\": -1, \"protein_end\": 1782, \"cds_end\": 5344, \"polyphen_score\": 0.996, \"canonical\": 1, \"hgvsp\": \"NP_000129.3:p.Cys1782Arg\", \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"44/66\", \"conservation\": 3.95, \"biotype\": \"protein_coding\", \"gene_id\": \"2200\", \"cadd_phred\": 27.9, \"spliceai\": {\"DP_AL\": -20, \"DS_AL\": 0, \"DS_DL\": 0, \"DS_DG\": 0, \"DP_DL\": 19, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DG\": 4, \"DP_AG\": 47}, \"eve_score\": 0.794797046557595, \"location\": \"15:48456715\", \"loeuf\": 0.105, \"alphamissense\": {\"am_pathogenicity\": 0.9997, \"am_class\": \"likely_pathogenic\"}, \"used_ref\": \"A\", \"variant_allele\": \"G\", \"cdna_end\": 5660, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48456715, \"start\": 48456714, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"impact\": \"MODERATE\", \"mane\": [\"MANE_Select\"], \"mastermind_counts\": \"4|7|7\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"eve_class\": \"Pathogenic\", \"mutationtaster_pred\": \"D\", \"amino_acids\": \"C/R\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"A\", \"blosum62\": -3, \"gene_symbol\": \"FBN1\"}], \"input\": \"NC_000015.10:g.48456715A>G\", \"id\": \"NC_000015.10:g.48456715A>G\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.04593718282768433,\n",
       "      'pca_y': 0.07238043088405371},\n",
       "     {'t-sne_x': 37.00731726868847, 't-sne_y': 4.151327022532274},\n",
       "     {'umap_x': 1.0431448221206665, 'umap_y': 6.521518707275391}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0007196306833066046,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452673-A-G',\n",
       "      'distance': 0.0009065180784091353,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.0009201699867844582,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0009885933250188828,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.0010288037592545152,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0010447625536471605,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.0010776197304949164,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.0011329902336001396,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0011857148492708802,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0011977760586887598,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0012436143588274717,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.0012884296011179686,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.0012907044729217887,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.0013643349520862103,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0013662691926583648,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.0013769069919362664,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456688-A-G',\n",
       "      'distance': 0.0013964768731966615,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.001412884215824306,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.001421733875758946,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.001428601797670126,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48452640-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48452640\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48452640',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5467G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48452640C>A',\n",
       "     'hgvs_protein': 'p.Glu1823Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"assembly_name\": \"GRCh38\", \"start\": 48452640, \"vcf_string\": \"15-48452640-C-A\", \"end\": 48452640, \"colocated_variants\": [{\"start\": 48452640, \"id\": \"CM980732\", \"seq_region_name\": \"15\", \"end\": 48452640, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1}], \"strand\": 1, \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48452640C>A\", \"allele_string\": \"C/A\", \"minimised\": 1, \"most_severe_consequence\": \"stop_gained\", \"input\": \"NC_000015.10:g.48452640C>A\", \"transcript_consequences\": [{\"lrt_pred\": \"invalid_field\", \"mastermind_mmid3\": \"FBN1:E1823X\", \"cds_start\": 5467, \"phaplo\": 0.99897615427687, \"protein_id\": \"NP_000129.3\", \"codons\": \"Gaa/Taa\", \"lof\": \"HC\", \"aa\": \"C\", \"transcript_id\": \"NM_000138.5\", \"cadd_raw\": 9.930735, \"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"exon\": \"45/66\", \"conservation\": 3.95, \"protein_end\": 1823, \"cds_end\": 5467, \"hgvsp\": \"NP_000129.3:p.Glu1823Ter\", \"canonical\": 1, \"strand\": -1, \"protein_start\": 1823, \"cdna_start\": 5783, \"hgvsc\": \"NM_000138.5:c.5467G>T\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48452640, \"type\": \"SimpleInterval\", \"start\": 48452639}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"mane\": [\"MANE_Select\"], \"impact\": \"HIGH\", \"used_ref\": \"C\", \"cdna_end\": 5783, \"variant_allele\": \"A\", \"loeuf\": 0.105, \"location\": \"15:48452640\", \"gene_id\": \"2200\", \"spliceai\": {\"DP_AG\": -19, \"DP_DG\": 44, \"SYMBOL\": \"FBN1\", \"DP_DL\": -36, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": 44, \"DS_DL\": 0.06, \"DS_AL\": 0.02}, \"cadd_phred\": 44, \"gene_symbol\": \"FBN1\", \"given_ref\": \"C\", \"amino_acids\": \"E/*\", \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"PERCENTILE:0.634517177344475,GERP_DIST:3650.65685455622,BP_DIST:3128,DIST_FROM_LAST_EXON:2739,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"A\", \"mastermind_counts\": \"5|5|5\", \"allele_num\": 1, \"consequence_terms\": [\"stop_gained\"]}]}',\n",
       "    'coordinates': [{'pca_x': 0.019444042282700594,\n",
       "      'pca_y': 0.09583462010987973},\n",
       "     {'t-sne_x': 22.696916956262562, 't-sne_y': 42.13871064117466},\n",
       "     {'umap_x': -9.617778778076172, 'umap_y': 4.522611618041992}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.0012615448795258999,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.00127123249694705,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411085-C-A',\n",
       "      'distance': 0.0013189015444368124,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.001491681905463338,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437837-C-A',\n",
       "      'distance': 0.0022199004888534546,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488233-C-A',\n",
       "      'distance': 0.0024350909516215324,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.0024932236410677433,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48452676-C-A',\n",
       "      'distance': 0.0027206852100789547,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505139-C-A',\n",
       "      'distance': 0.003229886759072542,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.003234405070543289,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48436970-C-A',\n",
       "      'distance': 0.0039858086965978146,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.00398784875869751,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515397-C-T',\n",
       "      'distance': 0.004432780668139458,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48483878-C-A',\n",
       "      'distance': 0.004883124493062496,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.004941883962601423,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.00532903429120779,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.005532319191843271,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468070-G-A',\n",
       "      'distance': 0.005554010160267353,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.005607041530311108,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.005707140080630779,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48446771-G-A',\n",
       "    'metadata': {'vcf_string': '15\\t48446771\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48446771',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5723C>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48446771G>A',\n",
       "     'hgvs_protein': 'p.Thr1908Ile',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"allele_string\": \"G/A\", \"input\": \"NC_000015.10:g.48446771G>A\", \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"impact\": \"MODERATE\", \"mane\": [\"MANE_Select\"], \"ga4gh_vrs\": {\"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"start\": 48446770, \"end\": 48446771}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"variant_allele\": \"A\", \"cdna_end\": 6039, \"used_ref\": \"G\", \"location\": \"15:48446771\", \"loeuf\": 0.105, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.6227}, \"cadd_phred\": 29, \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_AG\": -32, \"DP_DG\": -32, \"DP_AL\": 32, \"DS_AL\": 0, \"DS_DL\": 0, \"DS_DG\": 0, \"DP_DL\": 50, \"DS_AG\": 0}, \"eve_score\": 0.297111682171284, \"gene_id\": \"2200\", \"given_ref\": \"G\", \"gene_symbol\": \"FBN1\", \"blosum62\": -1, \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"T/I\", \"mutationtaster_pred\": \"D\", \"eve_class\": \"Benign\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"mastermind_counts\": \"6|7|7\", \"phaplo\": 0.99897615427687, \"cds_start\": 5723, \"clinpred\": 0.902041614055634, \"mastermind_mmid3\": \"FBN1:T1908I\", \"lrt_pred\": \"invalid_field\", \"protein_id\": \"NP_000129.3\", \"codons\": \"aCa/aTa\", \"sift_prediction\": \"deleterious_low_confidence\", \"transcript_id\": \"NM_000138.5\", \"cadd_raw\": 5.166115, \"aa\": \"G\", \"conservation\": 3.95, \"exon\": \"47/66\", \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"canonical\": 1, \"polyphen_score\": 0.997, \"hgvsp\": \"NP_000129.3:p.Thr1908Ile\", \"cds_end\": 5723, \"protein_end\": 1908, \"strand\": -1, \"cdna_start\": 6039, \"hgvsc\": \"NM_000138.5:c.5723C>T\", \"sift_score\": 0, \"protein_start\": 1908, \"polyphen_prediction\": \"probably_damaging\"}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48446771G>A\", \"end\": 48446771, \"strand\": 1, \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"id\": \"CM022786\", \"start\": 48446771, \"seq_region_name\": \"15\", \"end\": 48446771, \"strand\": 1}, {\"id\": \"COSV57320220\", \"start\": 48446771, \"seq_region_name\": \"15\", \"var_synonyms\": {\"COSMIC\": [\"COSM5569369\"]}, \"end\": 48446771, \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"somatic\": 1, \"phenotype_or_disease\": 1}, {\"id\": \"rs1304811982\", \"start\": 48446771, \"strand\": 1, \"end\": 48446771, \"clin_sig_allele\": \"A:uncertain_significance\", \"var_synonyms\": {\"ClinVar\": [\"RCV004008342\", \"RCV003325548\", \"RCV001182983\", \"VCV000922756\"], \"UniProt\": [\"VAR_076101\"]}, \"seq_region_name\": \"15\", \"allele_string\": \"G/A\", \"clin_sig\": [\"uncertain_significance\"], \"frequencies\": {\"A\": {\"gnomade_remaining\": 0, \"gnomade\": 2.053e-06, \"gnomade_nfe\": 2.699e-06, \"gnomade_sas\": 0, \"gnomade_eas\": 0, \"gnomade_afr\": 0, \"gnomade_asj\": 0, \"gnomade_mid\": 0, \"gnomade_fin\": 0, \"gnomade_amr\": 0}}, \"phenotype_or_disease\": 1}], \"start\": 48446771, \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"vcf_string\": \"15-48446771-G-A\"}',\n",
       "    'coordinates': [{'pca_x': 0.040760789655830354,\n",
       "      'pca_y': 0.0792270468971874},\n",
       "     {'t-sne_x': 34.59437783985946, 't-sne_y': 10.4182704922234},\n",
       "     {'umap_x': 1.4372375011444092, 'umap_y': 7.5859880447387695}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.0018252063309773803,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.002186317928135395,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468097-G-A',\n",
       "      'distance': 0.0026842935476452112,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432907-G-A',\n",
       "      'distance': 0.002782063325867057,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48537629-G-A',\n",
       "      'distance': 0.0028251861222088337,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446768-A-G',\n",
       "      'distance': 0.002848214004188776,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.002949278801679611,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487353-G-A',\n",
       "      'distance': 0.003140116576105356,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48490073-G-A',\n",
       "      'distance': 0.003162612207233906,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497332-G-A',\n",
       "      'distance': 0.0034390566870570183,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446811-A-G',\n",
       "      'distance': 0.003505846718326211,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.0035160935949534178,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0036630311515182257,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.003712515579536557,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485397-A-G',\n",
       "      'distance': 0.0037340810522437096,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48613073-G-A',\n",
       "      'distance': 0.003750762203708291,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.003761943429708481,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0037948794197291136,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520677-A-G',\n",
       "      'distance': 0.003810692811384797,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.0038757543079555035,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8}}},\n",
       "   {'variant_id': '15-48446705-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48446705\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48446705',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5788+1G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48446705C>A',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"splice_donor_variant\", \"end\": 48446705, \"transcript_consequences\": [{\"conservation\": 3.95, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"maxentscan_alt\": -1.902, \"transcript_id\": \"NM_000138.5\", \"mastermind_mmid3\": \"FBN1:D1930sd\", \"rf_score\": 0.938, \"mane\": [\"MANE_Select\"], \"intron\": \"47/65\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.5788+1G>T\", \"mastermind_counts\": \"2|2|9\", \"cadd_phred\": 35, \"maxentscan_ref\": 6.602, \"canonical\": 1, \"spliceai\": {\"DP_AG\": -32, \"DS_DL\": 1, \"DP_DL\": 1, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_AG\": 0, \"DS_DG\": 0.6, \"DP_AL\": 1, \"DP_DG\": -32}, \"lof\": \"HC\", \"aa\": \"C\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"variant_allele\": \"A\", \"lof_info\": \"INTRON_SIZE:1201\", \"gene_symbol_source\": \"EntrezGene\", \"ada_score\": 0.999990361384395, \"genesplicer\": \"loss/donor/48446705-48446706/Medium/0.624984\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"maxentscan_diff\": 8.504, \"location\": \"15:48446705\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48446705, \"start\": 48446704, \"type\": \"SimpleInterval\"}}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}}, \"consequence_terms\": [\"splice_donor_variant\"], \"allele_num\": 1, \"given_ref\": \"C\", \"cadd_raw\": 6.79501}], \"start\": 48446705, \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48446705, \"end\": 48446705, \"id\": \"CS013455\", \"phenotype_or_disease\": 1}, {\"phenotype_or_disease\": 1, \"id\": \"CS065549\", \"end\": 48446705, \"start\": 48446705, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\"}, {\"end\": 48446705, \"start\": 48446705, \"allele_string\": \"C/A/T\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV001662733\", \"RCV000663818\", \"VCV000549304\", \"RCV000017919\", \"VCV000016456\"], \"OMIM\": [134797.0037]}, \"seq_region_name\": \"15\", \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"clin_sig_allele\": \"A:likely_pathogenic;A:pathogenic;T:pathogenic\", \"phenotype_or_disease\": 1, \"id\": \"rs1555395819\", \"pubmed\": [11702223]}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48446705C>A\", \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48446705C>A\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/A\", \"vcf_string\": \"15-48446705-C-A\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.24194806630874366,\n",
       "      'pca_y': -0.027990079224564236},\n",
       "     {'t-sne_x': -16.11719692985573, 't-sne_y': -28.24836056355555},\n",
       "     {'umap_x': 8.207076072692871, 'umap_y': 12.91186237335205}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48613009-C-A',\n",
       "      'distance': 0.0010926243849098682,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465789-C-A',\n",
       "      'distance': 0.0015346381114795804,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508581-C-A',\n",
       "      'distance': 0.0016851101536303759,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-A',\n",
       "      'distance': 0.0018077472923323512,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-A',\n",
       "      'distance': 0.0018438778351992369,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437320-A-C',\n",
       "      'distance': 0.0024249209091067314,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.0029053054749965668,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.0030424422584474087,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.0032158447429537773,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-T',\n",
       "      'distance': 0.0032542371191084385,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.003290607128292322,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.0033200830221176147,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.0033951872028410435,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.0034238125663250685,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444540-C-T',\n",
       "      'distance': 0.0035411142744123936,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48441720-C-T',\n",
       "      'distance': 0.0035429149866104126,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.003550436347723007,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474254-C-T',\n",
       "      'distance': 0.003557879477739334,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.003582616103813052,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488112-C-T',\n",
       "      'distance': 0.0036372251342982054,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48445500-AACA-A',\n",
       "    'metadata': {'vcf_string': '15\\t48445500\\t.\\tAACA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48445500',\n",
       "     'ref_allele': 'AACA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5792_5794del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48445501_48445503del',\n",
       "     'hgvs_protein': 'p.Val1931del',\n",
       "     'most_severe_consequence': 'inframe_deletion'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"vcf_string\": \"15-48445500-AACA-A\", \"allele_string\": \"ACA/-\", \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48445501_48445503del\", \"seq_region_name\": \"15\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48445501_48445503del\", \"variant_class\": \"deletion\", \"start\": 48445501, \"most_severe_consequence\": \"inframe_deletion\", \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"cdna_end\": 6110, \"cdna_start\": 6108, \"transcript_id\": \"NM_000138.5\", \"used_ref\": \"CAA\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 5794, \"protein_start\": 1931, \"conservation\": 3.633, \"loeuf\": 0.105, \"protein_end\": 1932, \"phaplo\": 0.99897615427687, \"hgvs_offset\": -2, \"location\": \"15:48445499-48445501\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48445500, \"end\": 48445503}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Val1931del\", \"given_ref\": \"ACA\", \"consequence_terms\": [\"inframe_deletion\"], \"allele_num\": 1, \"amino_acids\": \"VD/D\", \"strand\": -1, \"cds_start\": 5792, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"MODERATE\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"-\", \"gene_symbol_source\": \"EntrezGene\", \"hgvsc\": \"NM_000138.5:c.5792_5794del\", \"biotype\": \"protein_coding\", \"codons\": \"gTTGat/gat\", \"canonical\": 1, \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": 2, \"SYMBOL\": \"FBN1\", \"DP_AG\": 27, \"DS_DL\": 0, \"DP_DG\": 5, \"DP_AL\": 4, \"DS_AG\": 0.01, \"DS_DG\": 0}, \"aa\": \"ACA\", \"exon\": \"48/66\"}], \"end\": 48445503}',\n",
       "    'coordinates': [{'pca_x': 0.025794140927353992,\n",
       "      'pca_y': 0.05002003333574145},\n",
       "     {'t-sne_x': 24.251460453146937, 't-sne_y': 14.604235290346237},\n",
       "     {'umap_x': 1.590286135673523, 'umap_y': 7.483298301696777}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.09813850373029709,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.09826062619686127,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.09911627322435379,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.09977772831916809,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432907-G-A',\n",
       "      'distance': 0.09994666278362274,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.1002463698387146,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.10031824558973312,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.10032228380441666,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.10036956518888474,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446814-C-T',\n",
       "      'distance': 0.10061655938625336,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48452595-C-A',\n",
       "      'distance': 0.10064327716827393,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.10064515471458435,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446768-A-G',\n",
       "      'distance': 0.10066136717796326,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.10067705810070038,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.10069592297077179,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445382-A-G',\n",
       "      'distance': 0.10081307590007782,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430791-A-T',\n",
       "      'distance': 0.10092934966087341,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420752-A-G',\n",
       "      'distance': 0.10093000531196594,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.10103558748960495,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-T',\n",
       "      'distance': 0.10118666291236877,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48444647-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48444647\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48444647',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5931T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48444647A>C',\n",
       "     'hgvs_protein': 'p.Cys1977Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"colocated_variants\": [{\"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"start\": 48444647, \"end\": 48444647, \"id\": \"CM085432\", \"phenotype_or_disease\": 1}, {\"id\": \"CM161840\", \"phenotype_or_disease\": 1, \"start\": 48444647, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"end\": 48444647}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48444647A>C\", \"start\": 48444647, \"most_severe_consequence\": \"missense_variant\", \"end\": 48444647, \"transcript_consequences\": [{\"sift_score\": 0, \"phaplo\": 0.99897615427687, \"protein_end\": 1977, \"loeuf\": 0.105, \"conservation\": 3.95, \"polyphen_score\": 0.997, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 1977, \"cds_end\": 5931, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"A\", \"eve_score\": 0.780563470337522, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9979}, \"transcript_id\": \"NM_000138.5\", \"cdna_start\": 6247, \"cdna_end\": 6247, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C1977W\", \"exon\": \"49/66\", \"blosum62\": -2, \"aa\": \"A\", \"spliceai\": {\"DS_AL\": 0.01, \"DP_DL\": -8, \"SYMBOL\": \"FBN1\", \"DP_AG\": 13, \"DS_DL\": 0, \"DP_DG\": -28, \"DP_AL\": -8, \"DS_AG\": 0, \"DS_DG\": 0.02}, \"cadd_phred\": 27.6, \"mastermind_counts\": \"5|5|5\", \"codons\": \"tgT/tgG\", \"canonical\": 1, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.5931T>G\", \"eve_class\": \"Pathogenic\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"C\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"MODERATE\", \"cds_start\": 5931, \"strand\": -1, \"mutationtaster_pred\": \"D\", \"amino_acids\": \"C/W\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"cadd_raw\": 4.943756, \"given_ref\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys1977Trp\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48444647, \"start\": 48444646}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}}, \"clinpred\": 0.999905586242676, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48444647\"}], \"strand\": 1, \"allele_string\": \"A/C\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48444647-A-C\", \"input\": \"NC_000015.10:g.48444647A>C\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.03778088783977346,\n",
       "      'pca_y': 0.10540202182594746},\n",
       "     {'t-sne_x': -12.237620617584971, 't-sne_y': 38.72901181927286},\n",
       "     {'umap_x': 1.783734917640686, 'umap_y': 21.493310928344727}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48444648-C-T',\n",
       "      'distance': 0.004833783023059368,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-G',\n",
       "      'distance': 0.006615116260945797,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508632-C-T',\n",
       "      'distance': 0.006797524634748697,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508633-A-C',\n",
       "      'distance': 0.0068072848953306675,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495512-A-C',\n",
       "      'distance': 0.006898526102304459,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444612-C-A',\n",
       "      'distance': 0.006910647265613079,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505136-A-C',\n",
       "      'distance': 0.00705302320420742,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463950-A-C',\n",
       "      'distance': 0.007259089034050703,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505117-C-A',\n",
       "      'distance': 0.007569970563054085,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444628-A-G',\n",
       "      'distance': 0.007854828611016273,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513661-A-G',\n",
       "      'distance': 0.007954525761306286,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48445454-A-G',\n",
       "      'distance': 0.008348913863301277,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48441764-A-C',\n",
       "      'distance': 0.008359028957784176,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445453-C-T',\n",
       "      'distance': 0.008436497300863266,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463148-A-G',\n",
       "      'distance': 0.008600913919508457,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474345-A-C',\n",
       "      'distance': 0.008635366335511208,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48444627-C-T',\n",
       "      'distance': 0.008821907453238964,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510094-C-A',\n",
       "      'distance': 0.008919455111026764,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463949-C-T',\n",
       "      'distance': 0.00899106077849865,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452637-A-G',\n",
       "      'distance': 0.009104223921895027,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48441726-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48441726\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48441726',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6158G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48441726C>A',\n",
       "     'hgvs_protein': 'p.Cys2053Phe',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"end\": 48441726, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"mastermind_mmid3\": \"FBN1:C2053F\", \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"cdna_start\": 6474, \"cdna_end\": 6474, \"transcript_id\": \"NM_000138.5\", \"alphamissense\": {\"am_pathogenicity\": 0.9968, \"am_class\": \"likely_pathogenic\"}, \"eve_score\": 0.647003135454768, \"used_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 6158, \"protein_start\": 2053, \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.104, \"conservation\": 3.95, \"loeuf\": 0.105, \"protein_end\": 2053, \"phaplo\": 0.99897615427687, \"sift_score\": 0, \"clinpred\": 0.991711434496321, \"polyphen_prediction\": \"benign\", \"location\": \"15:48441726\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48441725, \"end\": 48441726}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Cys2053Phe\", \"given_ref\": \"C\", \"cadd_raw\": 4.382424, \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"amino_acids\": \"C/F\", \"mutationtaster_pred\": \"D\", \"strand\": -1, \"cds_start\": 6158, \"impact\": \"MODERATE\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"A\", \"gene_symbol_source\": \"EntrezGene\", \"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.6158G>T\", \"biotype\": \"protein_coding\", \"canonical\": 1, \"codons\": \"tGc/tTc\", \"mastermind_counts\": \"8|10|10\", \"cadd_phred\": 25.1, \"spliceai\": {\"DS_AG\": 0, \"DS_DG\": 0, \"DP_AL\": 33, \"DP_DG\": -9, \"DP_AG\": -5, \"DS_DL\": 0, \"DP_DL\": 3, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0}, \"aa\": \"C\", \"blosum62\": -2, \"exon\": \"50/66\"}], \"start\": 48441726, \"id\": \"NC_000015.10:g.48441726C>A\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"start\": 48441726, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48441726, \"phenotype_or_disease\": 1, \"id\": \"CM154849\"}, {\"phenotype_or_disease\": 1, \"id\": \"CM992330\", \"end\": 48441726, \"start\": 48441726, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\"}, {\"pubmed\": [17701892], \"phenotype_or_disease\": 1, \"id\": \"rs363805\", \"clin_sig_allele\": \"T:pathogenic\", \"start\": 48441726, \"var_synonyms\": {\"ClinVar\": [\"RCV000631911\", \"VCV000527145\"], \"UniProt\": [\"VAR_055731\"]}, \"seq_region_name\": \"15\", \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"C/A/T\", \"strand\": 1, \"end\": 48441726}], \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48441726C>A\", \"vcf_string\": \"15-48441726-C-A\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/A\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.039287296327494614,\n",
       "      'pca_y': 0.10433467428751564},\n",
       "     {'t-sne_x': 21.63936638369781, 't-sne_y': 10.024744905289323},\n",
       "     {'umap_x': 4.271248817443848, 'umap_y': 6.924519062042236}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48465840-C-A',\n",
       "      'distance': 0.0017229770310223103,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468522-C-A',\n",
       "      'distance': 0.001748077105730772,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472628-C-A',\n",
       "      'distance': 0.0019129179418087006,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503849-C-A',\n",
       "      'distance': 0.0022773235104978085,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425783-C-A',\n",
       "      'distance': 0.0024178242310881615,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495513-C-A',\n",
       "      'distance': 0.002465018769726157,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425852-C-T',\n",
       "      'distance': 0.002955819945782423,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537721-C-A',\n",
       "      'distance': 0.00301175145432353,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415632-C-T',\n",
       "      'distance': 0.003177968319505453,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485463-C-A',\n",
       "      'distance': 0.003322338219732046,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432955-C-T',\n",
       "      'distance': 0.003363809548318386,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.0033789551816880703,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.0034156180918216705,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596318-C-A',\n",
       "      'distance': 0.0034492388367652893,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490050-C-A',\n",
       "      'distance': 0.0034756106324493885,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.003602280281484127,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48464009-C-T',\n",
       "      'distance': 0.0036173711996525526,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.0036501064896583557,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434701-C-T',\n",
       "      'distance': 0.003693951293826103,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463904-C-A',\n",
       "      'distance': 0.0037561492063105106,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48437895-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48437895\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48437895',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6186T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48437895A>T',\n",
       "     'hgvs_protein': 'p.Tyr2062Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"allele_string\": \"A/T\", \"vcf_string\": \"15-48437895-A-T\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48437895A>T\", \"colocated_variants\": [{\"id\": \"CD096461\", \"phenotype_or_disease\": 1, \"end\": 48437895, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48437895}, {\"phenotype_or_disease\": 1, \"id\": \"CM085420\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48437895, \"end\": 48437895}, {\"id\": \"CM152326\", \"phenotype_or_disease\": 1, \"start\": 48437895, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"end\": 48437895}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48437895A>T\", \"minimised\": 1, \"transcript_consequences\": [{\"protein_start\": 2062, \"cds_end\": 6186, \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"conservation\": -6.17, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"protein_end\": 2062, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:Y2062X\", \"cdna_end\": 6502, \"cdna_start\": 6502, \"lrt_pred\": \"invalid_field\", \"transcript_id\": \"NM_000138.5\", \"lof_info\": \"PERCENTILE:0.717966573816156,GERP_DIST:2840.1086550355,BP_DIST:2415,DIST_FROM_LAST_EXON:2026,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"T\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"gene_symbol_source\": \"EntrezGene\", \"codons\": \"taT/taA\", \"canonical\": 1, \"cadd_phred\": 22.9, \"mastermind_counts\": \"3|3|9\", \"hgvsc\": \"NM_000138.5:c.6186T>A\", \"biotype\": \"protein_coding\", \"aa\": \"A\", \"exon\": \"51/66\", \"lof\": \"HC\", \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_DG\": 22, \"DP_AL\": 22, \"DS_DL\": 0, \"DP_AG\": -2, \"DS_AL\": 0.05, \"SYMBOL\": \"FBN1\", \"DP_DL\": -16}, \"location\": \"15:48437895\", \"cadd_raw\": 3.559312, \"given_ref\": \"A\", \"consequence_terms\": [\"stop_gained\"], \"allele_num\": 1, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48437894, \"end\": 48437895}}}, \"hgvsp\": \"NP_000129.3:p.Tyr2062Ter\", \"amino_acids\": \"Y/*\", \"impact\": \"HIGH\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"strand\": -1, \"mutationtaster_pred\": \"A\", \"cds_start\": 6186}], \"most_severe_consequence\": \"stop_gained\", \"end\": 48437895, \"start\": 48437895}',\n",
       "    'coordinates': [{'pca_x': 0.020480066832317706,\n",
       "      'pca_y': 0.09822016118936491},\n",
       "     {'t-sne_x': 25.691880284129045, 't-sne_y': 37.59484762929739},\n",
       "     {'umap_x': -10.151270866394043, 'umap_y': 2.567741870880127}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48467954-A-T',\n",
       "      'distance': 0.004037295002490282,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.004465308506041765,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537615-A-T',\n",
       "      'distance': 0.004643771797418594,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.004850800149142742,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.005566701292991638,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437805-C-T',\n",
       "      'distance': 0.006152288056910038,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.006185699254274368,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489921-G-T',\n",
       "      'distance': 0.006220286712050438,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.006223583593964577,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.006419817917048931,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.006609463132917881,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.00723519129678607,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448879-G-A',\n",
       "      'distance': 0.0072781722992658615,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489969-C-T',\n",
       "      'distance': 0.0074006784707307816,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.007577949203550816,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415610-G-T',\n",
       "      'distance': 0.00769067695364356,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537696-C-T',\n",
       "      'distance': 0.007732573430985212,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474554-C-T',\n",
       "      'distance': 0.00791870430111885,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425416-G-T',\n",
       "      'distance': 0.008006248623132706,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.008041363209486008,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48437830-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48437830\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48437830',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6251G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48437830C>T',\n",
       "     'hgvs_protein': 'p.Cys2084Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"C/T\", \"end\": 48437830, \"vcf_string\": \"15-48437830-C-T\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48437830C>T\", \"transcript_consequences\": [{\"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.719524564739135, \"spliceai\": {\"DP_AL\": 7, \"DS_AG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 10, \"DP_AG\": -14, \"DS_DL\": 0, \"DP_DG\": -2, \"DS_DG\": 0}, \"codons\": \"tGc/tAc\", \"cds_start\": 6251, \"cdna_end\": 6567, \"cadd_phred\": 28.7, \"cadd_raw\": 5.124953, \"location\": \"15:48437830\", \"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48437829, \"type\": \"SimpleInterval\", \"end\": 48437830}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"type\": \"Allele\"}, \"alphamissense\": {\"am_pathogenicity\": 0.9989, \"am_class\": \"likely_pathogenic\"}, \"conservation\": 3.95, \"protein_end\": 2084, \"used_ref\": \"C\", \"aa\": \"C\", \"hgvsp\": \"NP_000129.3:p.Cys2084Tyr\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.6251G>A\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"impact\": \"MODERATE\", \"blosum62\": -2, \"mane_select\": \"ENST00000316623.10\", \"variant_allele\": \"T\", \"phaplo\": 0.99897615427687, \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 6567, \"exon\": \"51/66\", \"loeuf\": 0.105, \"eve_class\": \"Pathogenic\", \"given_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"polyphen_prediction\": \"possibly_damaging\", \"polyphen_score\": 0.8, \"strand\": -1, \"mastermind_mmid3\": \"FBN1:C2084Y\", \"canonical\": 1, \"clinpred\": 0.999409794807434, \"mastermind_counts\": \"3|3|3\", \"sift_score\": 0, \"mutationtaster_pred\": \"A\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 6251, \"protein_start\": 2084, \"amino_acids\": \"C/Y\", \"transcript_id\": \"NM_000138.5\"}], \"colocated_variants\": [{\"id\": \"CM085424\", \"phenotype_or_disease\": 1, \"start\": 48437830, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48437830}, {\"end\": 48437830, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"start\": 48437830, \"id\": \"CM127286\"}, {\"end\": 48437830, \"clin_sig_allele\": \"T:likely_pathogenic;A:likely_pathogenic\", \"allele_string\": \"C/A/T\", \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"rs794728245\", \"var_synonyms\": {\"ClinVar\": [\"RCV000663856\", \"VCV000549336\", \"RCV000181561\", \"VCV000200078\"], \"UniProt\": [\"VAR_076114\"]}, \"phenotype_or_disease\": 1, \"clin_sig\": [\"likely_pathogenic\"], \"start\": 48437830}], \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"start\": 48437830, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48437830C>T\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\"}',\n",
       "    'coordinates': [{'pca_x': 0.034393556364192224,\n",
       "      'pca_y': 0.08237831630366607},\n",
       "     {'t-sne_x': 6.562896791161212, 't-sne_y': 8.945316663425857},\n",
       "     {'umap_x': 3.40039324760437, 'umap_y': 2.220594882965088}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.000684197002556175,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.0007061978103592992,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.000899522565305233,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.000945487292483449,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427602-C-T',\n",
       "      'distance': 0.0009578054305166006,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-T',\n",
       "      'distance': 0.0009936066344380379,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.0010049054399132729,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487317-C-T',\n",
       "      'distance': 0.0010068158153444529,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.0010109763825312257,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497358-C-T',\n",
       "      'distance': 0.0010297468397766352,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496213-C-T',\n",
       "      'distance': 0.0010359922889620066,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.0010433944407850504,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492574-C-T',\n",
       "      'distance': 0.0010555380722507834,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.0010625917930155993,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.0010821030009537935,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441771-C-T',\n",
       "      'distance': 0.0010859868489205837,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497343-C-T',\n",
       "      'distance': 0.0010973673779517412,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0011099855182692409,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.001118397107347846,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496144-C-T',\n",
       "      'distance': 0.001130509888753295,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48437050-A-ACA',\n",
       "    'metadata': {'vcf_string': '15\\t48437050\\t.\\tA\\tACA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48437050',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'ACA',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6406_6407dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48437051_48437052dup',\n",
       "     'hgvs_protein': 'p.Cys2137SerfsTer24',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"start\": 48437051, \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48437051_48437052dup\", \"transcript_consequences\": [{\"used_ref\": \"-\", \"hgvsp\": \"NP_000129.3:p.Cys2137SerfsTer24\", \"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.6406_6407dup\", \"biotype\": \"protein_coding\", \"conservation\": 3.03, \"protein_end\": 2136, \"consequence_terms\": [\"frameshift_variant\"], \"codons\": \"gtc/gtGTc\", \"cds_start\": 6407, \"cdna_end\": 6724, \"hgvs_offset\": -1, \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.022774, \"dG_wt_sd\": 0.000447, \"dG_mt\": 190.4456, \"ddG\": -0.5032, \"dG_mt_sd\": 0.022733, \"dG_wt\": 190.9488}}, \"location\": \"15:48437049-48437050\", \"allele_num\": 1, \"protein_id\": \"NP_000129.3\", \"cds_end\": 6408, \"protein_start\": 2136, \"lof\": \"HC\", \"amino_acids\": \"V/VX\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"canonical\": 1, \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 6723, \"phaplo\": 0.99897615427687, \"exon\": \"53/66\", \"loeuf\": 0.105, \"given_ref\": \"-\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"impact\": \"HIGH\", \"lof_info\": \"PERCENTILE:0.743616527390901,GERP_DIST:2545.13655850291,BP_DIST:2197,DIST_FROM_LAST_EXON:1808,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mane_select\": \"ENST00000316623.10\", \"variant_allele\": \"AC\"}], \"id\": \"NC_000015.10:g.48437051_48437052dup\", \"variant_class\": \"insertion\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48437050-A-ACA\", \"end\": 48437050, \"allele_string\": \"-/CA\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.027183699681216728,\n",
       "      'pca_y': 0.013237290199371872},\n",
       "     {'t-sne_x': 11.00520205329021, 't-sne_y': 15.098137392944063},\n",
       "     {'umap_x': 3.8013250827789307, 'umap_y': 3.8644087314605713}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.08406074345111847,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.08510160446166992,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.0856548622250557,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.08610369265079498,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.08612315356731415,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.0864240899682045,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.08648129552602768,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.08655744791030884,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452595-C-A',\n",
       "      'distance': 0.08663378655910492,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.08674019575119019,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437069-C-T',\n",
       "      'distance': 0.08678479492664337,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.08680187165737152,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.08701437711715698,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446814-C-T',\n",
       "      'distance': 0.08711080253124237,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.08717282116413116,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.08717525005340576,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430770-A-G',\n",
       "      'distance': 0.08725875616073608,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468504-C-T',\n",
       "      'distance': 0.08734560757875443,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.08738027513027191,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.08742228150367737,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48434595-48434715-del-36342bf4',\n",
       "    'metadata': {'vcf_string': '15\\t48434595\\t.\\tTTCACATGTCATCATTGGACCGGGCTCAAATCCCTCCTCGCAGGTGCATTCAAAACCTCCAATCACATTCTTGCAGGTTCCATTTCCACAAGGATTGCCAACAGAACATTCATCAGTATCT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48434595',\n",
       "     'ref_allele': 'TTCACATGTCATCATTGGACCGGGCTCAAATCCCTCCTCGCAGGTGCATTCAAAACCTCCAATCACATTCTTGCAGGTTCCATTTCCACAAGGATTGCCAACAGAACATTCATCAGTATCT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6497_6616del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48434596_48434715del',\n",
       "     'hgvs_protein': 'p.Thr2167_Asp2206del',\n",
       "     'most_severe_consequence': 'inframe_deletion'},\n",
       "    'annotation_raw': '{\"variant_class\": \"deletion\", \"assembly_name\": \"GRCh38\", \"id\": \"NC_000015.10:g.48434596_48434715del\", \"transcript_consequences\": [{\"mane_select\": \"ENST00000316623.10\", \"impact\": \"MODERATE\", \"variant_allele\": \"-\", \"gene_symbol\": \"FBN1\", \"given_ref\": \"TCACATGTCATCATTGGACCGGGCTCAAATCCCTCCTCGCAGGTGCATTCAAAACCTCCAATCACATTCTTGCAGGTTCCATTTCCACAAGGATTGCCAACAGAACATTCATCAGTATCT\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"exon\": \"54/66\", \"cdna_start\": 6813, \"canonical\": 1, \"strand\": -1, \"protein_start\": 2166, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"DTDECSVGNPCGNGTCKNVIGGFECTCEEGFEPGPMMTCED/D\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 6616, \"allele_num\": 1, \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 0.069874, \"dG_wt\": 189.6428, \"ddG\": -0.28079, \"dG_mt\": 189.36201, \"dG_wt_sd\": 0.172622, \"ddG_sd\": 0.183115}}, \"location\": \"15:48434594-48434713\", \"hgvs_offset\": -2, \"consequence_terms\": [\"inframe_deletion\", \"splice_region_variant\", \"start_retained_variant\"], \"cdna_end\": 6932, \"codons\": \"gATACTGATGAATGTTCTGTTGGCAATCCTTGTGGAAATGGAACCTGCAAGAATGTGATTGGAGGTTTTGAATGCACCTGCGAGGAGGGATTTGAGCCCGGTCCAATGATGACATGTGAAGat/gat\", \"cds_start\": 6497, \"conservation\": 2.839, \"protein_end\": 2206, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48434595, \"end\": 48434715}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"type\": \"Allele\"}, \"biotype\": \"protein_coding\", \"genesplicer\": \"loss/acceptor/48434640-48434639/Medium/4.959640,gain/acceptor/48434596-48434597/Medium/8.343897\", \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.6497_6616del\", \"gene_id\": \"2200\", \"intron\": \"53/65\", \"hgvsp\": \"NP_000129.3:p.Thr2167_Asp2206del\", \"aa\": \"TCACATGTCATCATTGGACCGGGCTCAAATCCCTCCTCgCAGGTGCATTCAAAACCTCCAATCACATTCTTGCAGGTTCCATTTCCACAAGGATTGCCAACAGAACATTCATCAGTATCT\", \"used_ref\": \"CTTCACATGTCATCATTGGACCGGGCTCAAATCCCTCCTCGCAGGTGCATTCAAAACCTCCAATCACATTCTTGCAGGTTCCATTTCCACAAGGATTGCCAACAGAACATTCATCAGTAT\"}], \"minimised\": 1, \"allele_string\": \"TCACATGTCATCATTGGACCGGGCTCAAATCCCTCCTCGCAGGTGCATTCAAAACCTCCAATCACATTCTTGCAGGTTCCATTTCCACAAGGATTGCCAACAGAACATTCATCAGTATCT/-\", \"vcf_string\": \"15-48434595-TTCACATGTCATCATTGGACCGGGCTCAAATCCCTCCTCGCAGGTGCATTCAAAACCTCCAATCACATTCTTGCAGGTTCCATTTCCACAAGGATTGCCAACAGAACATTCATCAGTATCT-T\", \"end\": 48434715, \"input\": \"NC_000015.10:g.48434596_48434715del\", \"strand\": 1, \"most_severe_consequence\": \"inframe_deletion\", \"start\": 48434596, \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': -0.08798458034903177,\n",
       "      'pca_y': 0.021583349872854506},\n",
       "     {'t-sne_x': -12.63112630552781, 't-sne_y': -3.004065069044725},\n",
       "     {'umap_x': 0.44047868251800537, 'umap_y': 15.049419403076172}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.09077447652816772,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.09168408811092377,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.0933791846036911,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.0934319794178009,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.09439200907945633,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.09445519745349884,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.0945141389966011,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.09458476305007935,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.09486445784568787,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.09521275758743286,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.09533166885375977,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.09658412635326385,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.09700389206409454,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.0970117524266243,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.09735868871212006,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487075-C-T',\n",
       "      'distance': 0.09868305921554565,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.09898537397384644,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.09964972734451294,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.09993310272693634,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.10018058121204376,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48432944-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48432944\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48432944',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6661T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48432944A>C',\n",
       "     'hgvs_protein': 'p.Cys2221Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"end\": 48432944, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"CM022006\", \"phenotype_or_disease\": 1, \"start\": 48432944}, {\"end\": 48432944, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48432944, \"phenotype_or_disease\": 1, \"id\": \"CM992331\"}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48432944A>C\", \"transcript_consequences\": [{\"location\": \"15:48432944\", \"cadd_raw\": 5.221198, \"cadd_phred\": 29.3, \"eve_score\": 0.778449668629877, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 6977, \"codons\": \"Tgt/Ggt\", \"spliceai\": {\"DP_AL\": 44, \"DS_AG\": 0, \"DS_AL\": 0, \"DP_DL\": 24, \"SYMBOL\": \"FBN1\", \"DP_AG\": -1, \"DS_DL\": 0, \"DP_DG\": 0, \"DS_DG\": 0}, \"cds_start\": 6661, \"allele_num\": 1, \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.6661T>G\", \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys2221Gly\", \"aa\": \"A\", \"used_ref\": \"A\", \"conservation\": 3.95, \"alphamissense\": {\"am_pathogenicity\": 0.9909, \"am_class\": \"likely_pathogenic\"}, \"protein_end\": 2221, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48432944, \"type\": \"SimpleInterval\", \"start\": 48432943}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"biotype\": \"protein_coding\", \"genesplicer\": \"gain/acceptor/48432943-48432944/Medium/3.378630\", \"eve_class\": \"Pathogenic\", \"given_ref\": \"A\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 6977, \"exon\": \"55/66\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -3, \"impact\": \"MODERATE\", \"variant_allele\": \"C\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"protein_start\": 2221, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/G\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 6661, \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:C2221G\", \"clinpred\": 0.999596893787384, \"mutationtaster_pred\": \"D\", \"sift_score\": 0, \"mastermind_counts\": \"5|9|9\", \"polyphen_prediction\": \"probably_damaging\", \"polyphen_score\": 0.985, \"strand\": -1}], \"minimised\": 1, \"allele_string\": \"A/C\", \"end\": 48432944, \"vcf_string\": \"15-48432944-A-C\", \"input\": \"NC_000015.10:g.48432944A>C\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48432944}',\n",
       "    'coordinates': [{'pca_x': 0.03768145474406143,\n",
       "      'pca_y': 0.08732516379713945},\n",
       "     {'t-sne_x': 19.613445316331603, 't-sne_y': 3.566114389398412},\n",
       "     {'umap_x': 3.7865097522735596, 'umap_y': 6.241121768951416}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.002504840027540922,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445432-A-C',\n",
       "      'distance': 0.0025204895064234734,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.00254455185495317,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48436967-A-C',\n",
       "      'distance': 0.002583632944151759,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483822-A-C',\n",
       "      'distance': 0.002729848027229309,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432944-A-G',\n",
       "      'distance': 0.0028545833192765713,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.0030186299700289965,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.0031155948527157307,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.0032514999620616436,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.0032673010136932135,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415716-T-C',\n",
       "      'distance': 0.0032741944305598736,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520677-A-C',\n",
       "      'distance': 0.0032895635813474655,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.003294333117082715,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.0034338003024458885,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468522-C-A',\n",
       "      'distance': 0.003517929930239916,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.003522387007251382,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.0035388553515076637,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.0036314758472144604,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.0036347261629998684,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.003671677317470312,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48430671-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48430671\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48430671',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6871G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48430671C>T',\n",
       "     'hgvs_protein': 'p.Asp2291Asn',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"C/T\", \"vcf_string\": \"15-48430671-C-T\", \"end\": 48430671, \"minimised\": 1, \"id\": \"NC_000015.10:g.48430671C>T\", \"transcript_consequences\": [{\"ptriplo\": 0.984699554922396, \"given_ref\": \"C\", \"loeuf\": 0.105, \"cdna_start\": 7187, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"exon\": \"56/66\", \"variant_allele\": \"T\", \"blosum62\": 1, \"mane_select\": \"ENST00000316623.10\", \"impact\": \"MODERATE\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"D/N\", \"protein_start\": 2291, \"cds_end\": 6871, \"maxentscan_ref\": 9.144, \"protein_id\": \"NP_000129.3\", \"mutationtaster_pred\": \"A\", \"mastermind_counts\": \"5|8|16\", \"sift_score\": 0, \"mastermind_mmid3\": \"FBN1:D2291N,FBN1:D2291srd\", \"canonical\": 1, \"clinpred\": 0.993515946731822, \"ada_score\": 0.999981290965904, \"strand\": -1, \"rf_score\": 1, \"polyphen_prediction\": \"probably_damaging\", \"polyphen_score\": 0.995, \"maxentscan_alt\": 5.167, \"cadd_raw\": 7.563763, \"location\": \"15:48430671\", \"cadd_phred\": 36, \"cdna_end\": 7187, \"spliceai\": {\"DS_DG\": 0.69, \"DP_DG\": 4, \"DS_DL\": 0.77, \"DP_AG\": 27, \"DP_DL\": 0, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_AG\": 0, \"DP_AL\": 4}, \"codons\": \"Gat/Aat\", \"cds_start\": 6871, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"allele_num\": 1, \"maxentscan_diff\": 3.977, \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.6871G>A\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"hgvsp\": \"NP_000129.3:p.Asp2291Asn\", \"aa\": \"C\", \"used_ref\": \"C\", \"alphamissense\": {\"am_pathogenicity\": 0.9518, \"am_class\": \"likely_pathogenic\"}, \"protein_end\": 2291, \"conservation\": 3.95, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48430670, \"end\": 48430671}}}, \"biotype\": \"protein_coding\"}], \"variant_class\": \"SNV\", \"colocated_variants\": [{\"id\": \"CM074797\", \"phenotype_or_disease\": 1, \"start\": 48430671, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48430671}, {\"phenotype_or_disease\": 1, \"start\": 48430671, \"id\": \"CM106075\", \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"end\": 48430671}, {\"phenotype_or_disease\": 1, \"start\": 48430671, \"id\": \"CS077361\", \"end\": 48430671, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\"}, {\"end\": 48430671, \"clin_sig_allele\": \"T:pathogenic\", \"allele_string\": \"C/T\", \"strand\": 1, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000181580\", \"VCV000200094\"]}, \"id\": \"rs794728257\", \"phenotype_or_disease\": 1, \"start\": 48430671, \"clin_sig\": [\"pathogenic\"]}], \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"start\": 48430671, \"input\": \"NC_000015.10:g.48430671C>T\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.03597550040704099,\n",
       "      'pca_y': 0.05368489366712811},\n",
       "     {'t-sne_x': -12.302857820273317, 't-sne_y': -2.6623762556209467},\n",
       "     {'umap_x': 0.32221415638923645, 'umap_y': 15.058182716369629}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.002366330474615097,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.003320632968097925,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.0039736200124025345,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.004829297307878733,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.004995784722268581,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.0051724533550441265,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.005397752393037081,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.005556774325668812,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.0056519475765526295,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.005913895089179277,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.0062159886583685875,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.006417793221771717,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.006548834033310413,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.006941952742636204,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487075-C-T',\n",
       "      'distance': 0.0071056922897696495,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.007135281339287758,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.007452935446053743,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.007817914709448814,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.008073108270764351,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.00860593281686306,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48415660-TG-T',\n",
       "    'metadata': {'vcf_string': '15\\t48415660\\t.\\tTG\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415660',\n",
       "     'ref_allele': 'TG',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7926del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415661del',\n",
       "     'hgvs_protein': 'p.Phe2642LeufsTer40',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"G/-\", \"start\": 48415661, \"assembly_name\": \"GRCh38\", \"end\": 48415661, \"variant_class\": \"deletion\", \"minimised\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48415661del\", \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"id\": \"CD012682\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48415661, \"end\": 48415661}], \"transcript_consequences\": [{\"protein_start\": 2642, \"aa\": \"G\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48415661\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48415661, \"type\": \"SimpleInterval\", \"start\": 48415660}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}}, \"loeuf\": 0.105, \"consequence_terms\": [\"frameshift_variant\"], \"cds_start\": 7926, \"impact\": \"HIGH\", \"cdna_end\": 8242, \"cdna_start\": 8242, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Phe2642LeufsTer40\", \"given_ref\": \"G\", \"biotype\": \"protein_coding\", \"cds_end\": 7926, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"used_ref\": \"G\", \"lof_info\": \"PERCENTILE:0.91991643454039,GERP_DIST:642.344442284107,BP_DIST:688,DIST_FROM_LAST_EXON:299,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"lof\": \"HC\", \"protein_end\": 2642, \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 5.35, \"amino_acids\": \"F/X\", \"mastermind_mmid3\": \"FBN1:F2642fs\", \"codons\": \"ttC/tt\", \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"spliceai\": {\"DS_DG\": 0.01, \"DS_AL\": 0, \"DP_DL\": -27, \"DP_AL\": 12, \"DS_DL\": 0, \"DP_AG\": 3, \"DP_DG\": 5, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0}, \"exon\": \"64/66\", \"canonical\": 1, \"mastermind_counts\": \"7|7|7\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"variant_allele\": \"-\", \"hgvsc\": \"NM_000138.5:c.7926del\"}], \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48415661del\", \"vcf_string\": \"15-48415660-TG-T\"}',\n",
       "    'coordinates': [{'pca_x': 0.03985922351042931,\n",
       "      'pca_y': 0.03101010545205559},\n",
       "     {'t-sne_x': 11.064236934962178, 't-sne_y': 15.040020798524443},\n",
       "     {'umap_x': 3.8388655185699463, 'umap_y': 3.5502800941467285}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.11211413890123367,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.1126512810587883,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.1132814809679985,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.11339683830738068,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.11389724910259247,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.11470543593168259,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.11507128179073334,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.11512386053800583,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.11513365060091019,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.11517303436994553,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.11521914601325989,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537704-G-A',\n",
       "      'distance': 0.11530593037605286,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.11544115841388702,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.11545135825872421,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.11561976373195648,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48516225-G-A',\n",
       "      'distance': 0.11566025018692017,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.11574853211641312,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437912-G-A',\n",
       "      'distance': 0.1157626137137413,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.11578056961297989,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.11582174897193909,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48415599-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48415599\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415599',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7988G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415599C>G',\n",
       "     'hgvs_protein': 'p.Cys2663Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48415599, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/G\", \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48415599C>G\", \"variant_class\": \"SNV\", \"end\": 48415599, \"minimised\": 1, \"colocated_variants\": [{\"start\": 48415599, \"end\": 48415599, \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM043997\", \"seq_region_name\": \"15\"}, {\"end\": 48415599, \"start\": 48415599, \"id\": \"CM161867\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"start\": 48415599, \"end\": 48415599, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"clin_sig_allele\": \"A:pathogenic;T:pathogenic;G:pathogenic\", \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"C/A/G/T\", \"var_synonyms\": {\"ClinVar\": [\"RCV003229032\", \"RCV001247322\", \"VCV000971524\", \"RCV001939335\", \"VCV001453782\", \"RCV000818844\", \"VCV000661431\"], \"UniProt\": [\"VAR_023915\"]}, \"pubmed\": [17701892], \"id\": \"rs1597509631\"}], \"transcript_consequences\": [{\"sift_prediction\": \"deleterious_low_confidence\", \"eve_class\": \"Pathogenic\", \"cadd_phred\": 25.2, \"polyphen_score\": 0.144, \"protein_end\": 2663, \"mastermind_mmid3\": \"FBN1:C2663S\", \"codons\": \"tGt/tCt\", \"amino_acids\": \"C/S\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 5.35, \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"blosum62\": -1, \"exon\": \"64/66\", \"spliceai\": {\"DP_DG\": -24, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AG\": 19, \"DP_DL\": 34, \"DP_AL\": 8, \"DS_DG\": 0, \"DS_AL\": 0}, \"canonical\": 1, \"variant_allele\": \"G\", \"strand\": -1, \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.99974912405014, \"mastermind_counts\": \"2|7|7\", \"hgvsc\": \"NM_000138.5:c.7988G>C\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48415599\", \"aa\": \"C\", \"polyphen_prediction\": \"benign\", \"protein_start\": 2663, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48415599, \"start\": 48415598, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.748726247852938, \"cds_start\": 7988, \"impact\": \"MODERATE\", \"cdna_start\": 8304, \"cdna_end\": 8304, \"hgvsp\": \"NP_000129.3:p.Cys2663Ser\", \"gene_id\": \"2200\", \"given_ref\": \"C\", \"sift_score\": 0, \"cds_end\": 7988, \"cadd_raw\": 4.421439, \"biotype\": \"protein_coding\", \"mutationtaster_pred\": \"A\", \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.999}, \"used_ref\": \"C\"}], \"vcf_string\": \"15-48415599-C-G\", \"input\": \"NC_000015.10:g.48415599C>G\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.0371398938755903,\n",
       "      'pca_y': 0.08065907890052851},\n",
       "     {'t-sne_x': 20.424866936770453, 't-sne_y': -5.243390586029139},\n",
       "     {'umap_x': 8.70821762084961, 'umap_y': 7.044143199920654}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.0009288151632063091,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.0009597369935363531,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.0010029978584498167,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0010284904856234789,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.0014997631078585982,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.0015619364567101002,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0017236827407032251,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.0017793530132621527,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.001889352104626596,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.0019546253606677055,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415708-C-G',\n",
       "      'distance': 0.0021999196615070105,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.002275413367897272,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.002324697794392705,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489991-C-G',\n",
       "      'distance': 0.002377739641815424,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.0024790996685624123,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.0027376720681786537,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.0027728178538382053,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.0028154740575701,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.0029648724012076855,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.002985316328704357,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48412745-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48412745\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48412745',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8052-2A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48412745T>C',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"start\": 48412745, \"end\": 48412745, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CS930806\"}], \"transcript_consequences\": [{\"aa\": \"T\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48412745\", \"loeuf\": 0.105, \"genesplicer\": \"loss/acceptor/48412745-48412746/Medium/8.059364\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"end\": 48412745, \"start\": 48412744, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"consequence_terms\": [\"splice_acceptor_variant\"], \"impact\": \"HIGH\", \"maxentscan_diff\": 7.955, \"rf_score\": 0.93, \"gene_id\": \"2200\", \"ada_score\": 0.999988247793601, \"given_ref\": \"T\", \"cadd_raw\": 5.178896, \"biotype\": \"protein_coding\", \"maxentscan_ref\": 9.419, \"maxentscan_alt\": 1.464, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"used_ref\": \"T\", \"lof_info\": \"INTRON_SIZE:2792\", \"intron\": \"64/65\", \"lof\": \"HC\", \"cadd_phred\": 29, \"mastermind_mmid3\": \"FBN1:G2684sa\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 5.35, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"spliceai\": {\"DS_AL\": 0.98, \"DS_DG\": 0, \"DP_AL\": -2, \"DP_DL\": -50, \"DP_AG\": -34, \"DS_DL\": 0, \"DP_DG\": -34, \"DS_AG\": 0.01, \"SYMBOL\": \"FBN1\"}, \"canonical\": 1, \"variant_allele\": \"C\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"mastermind_counts\": \"5|5|5\", \"hgvsc\": \"NM_000138.5:c.8052-2A>G\"}], \"vcf_string\": \"15-48412745-T-C\", \"most_severe_consequence\": \"splice_acceptor_variant\", \"input\": \"NC_000015.10:g.48412745T>C\", \"strand\": 1, \"start\": 48412745, \"assembly_name\": \"GRCh38\", \"allele_string\": \"T/C\", \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48412745T>C\", \"end\": 48412745, \"variant_class\": \"SNV\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.23014325603523528,\n",
       "      'pca_y': -0.03950986847174026},\n",
       "     {'t-sne_x': -29.975714318254393, 't-sne_y': -19.062975674581075},\n",
       "     {'umap_x': 11.634404182434082, 'umap_y': 18.964012145996094}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.0011490760371088982,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.0013524398673325777,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.002142490353435278,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.002169269137084484,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.002416803501546383,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.0024605216458439827,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.0026151661295443773,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.0027793659828603268,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.0031842810567468405,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.0035884492099285126,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452686-T-C',\n",
       "      'distance': 0.0035900971852242947,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.003813748713582754,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.004228572361171246,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.004335764795541763,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.004587050527334213,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.004687040112912655,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.004793204367160797,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.005406500771641731,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446823-C-G',\n",
       "      'distance': 0.006732046138495207,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-A',\n",
       "      'distance': 0.007156009785830975,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48613054-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48613054\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48613054',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.203G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48613054C>A',\n",
       "     'hgvs_protein': 'p.Cys68Phe',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"end\": 48613054, \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48613054C>A\", \"seq_region_name\": \"15\", \"allele_string\": \"C/A\", \"assembly_name\": \"GRCh38\", \"start\": 48613054, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48613054C>A\", \"strand\": 1, \"vcf_string\": \"15-48613054-C-A\", \"transcript_consequences\": [{\"mastermind_counts\": \"4|4|4\", \"strand\": -1, \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"A\", \"clinpred\": 0.996172487735748, \"hgvsc\": \"NM_000138.5:c.203G>T\", \"spliceai\": {\"DP_AL\": 38, \"DP_DL\": -48, \"DS_AL\": 0, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_DG\": -35, \"SYMBOL\": \"FBN1\", \"DP_AG\": -35, \"DS_DL\": 0}, \"lrt_pred\": \"invalid_field\", \"exon\": \"3/66\", \"blosum62\": -2, \"canonical\": 1, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"conservation\": 5.92, \"gene_symbol_source\": \"EntrezGene\", \"mastermind_mmid3\": \"FBN1:C68F\", \"amino_acids\": \"C/F\", \"codons\": \"tGc/tTc\", \"protein_end\": 68, \"cadd_phred\": 30, \"polyphen_score\": 0.963, \"sift_prediction\": \"deleterious_low_confidence\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9975}, \"used_ref\": \"C\", \"mutfunc\": {\"exp\": {\"ddG\": 3.0288, \"dG_wt_sd\": 0.012194, \"dG_mt_sd\": 0.289827, \"dG_wt\": 110.8992, \"ddG_sd\": 0.277797, \"dG_mt\": 113.928}}, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"A\", \"given_ref\": \"C\", \"sift_score\": 0, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.337224, \"cds_end\": 203, \"cdna_end\": 519, \"cdna_start\": 519, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys68Phe\", \"cds_start\": 203, \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\"], \"protein_start\": 68, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48613054\", \"protein_id\": \"NP_000129.3\", \"aa\": \"C\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48613053, \"type\": \"SimpleInterval\", \"end\": 48613054}}}, \"loeuf\": 0.105}], \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM073061\", \"seq_region_name\": \"15\", \"start\": 48613054, \"end\": 48613054}, {\"start\": 48613054, \"end\": 48613054, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM074845\"}, {\"strand\": 1, \"clin_sig_allele\": \"T:likely_pathogenic;A:pathogenic;T:pathogenic\", \"phenotype_or_disease\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV001258095\", \"RCV001044191\", \"VCV000841878\", \"RCV000804019\", \"VCV000649153\"]}, \"pubmed\": [17657824, 17701892], \"id\": \"rs1597633163\", \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"allele_string\": \"C/A/T\", \"seq_region_name\": \"15\", \"end\": 48613054, \"start\": 48613054}]}',\n",
       "    'coordinates': [{'pca_x': 0.04002011523869037,\n",
       "      'pca_y': 0.09709269497512094},\n",
       "     {'t-sne_x': 20.246518896570223, 't-sne_y': 14.066874385252618},\n",
       "     {'umap_x': 4.368235111236572, 'umap_y': 7.285407543182373}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48610808-C-A',\n",
       "      'distance': 0.0011330705601722002,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537721-C-A',\n",
       "      'distance': 0.0024432032369077206,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485463-C-A',\n",
       "      'distance': 0.0028120914939790964,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463904-C-A',\n",
       "      'distance': 0.002891749609261751,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.0030264430679380894,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.0031380485743284225,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596318-C-A',\n",
       "      'distance': 0.0031760195270180702,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.003293488174676895,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.003365742973983288,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.0034944594372063875,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.003541754325851798,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497358-C-A',\n",
       "      'distance': 0.0036577684804797173,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425783-C-A',\n",
       "      'distance': 0.003806278109550476,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.00405177753418684,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415582-C-A',\n",
       "      'distance': 0.0041696662083268166,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444579-C-A',\n",
       "      'distance': 0.004233482293784618,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415695-C-A',\n",
       "      'distance': 0.004343711771070957,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537790-C-T',\n",
       "      'distance': 0.004441738128662109,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468522-C-A',\n",
       "      'distance': 0.004673486575484276,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.004746836610138416,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48610773-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48610773\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48610773',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.301A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48610773T>C',\n",
       "     'hgvs_protein': 'p.Thr101Ala',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48610773T>C\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"vcf_string\": \"15-48610773-T-C\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM098635\", \"phenotype_or_disease\": 1, \"strand\": 1, \"start\": 48610773, \"end\": 48610773}], \"transcript_consequences\": [{\"used_ref\": \"T\", \"alphamissense\": {\"am_class\": \"likely_benign\", \"am_pathogenicity\": 0.3392}, \"mutfunc\": {\"exp\": {\"dG_mt\": 111.210199, \"ddG_sd\": 0.051662, \"dG_wt\": 110.7562, \"dG_mt_sd\": 0.065212, \"dG_wt_sd\": 0.022185, \"ddG\": 0.453999}}, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"D\", \"given_ref\": \"T\", \"sift_score\": 0.01, \"biotype\": \"protein_coding\", \"cadd_raw\": 4.486779, \"cds_end\": 301, \"cdna_end\": 617, \"cdna_start\": 617, \"hgvsp\": \"NP_000129.3:p.Thr101Ala\", \"gene_id\": \"2200\", \"cds_start\": 301, \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\"], \"protein_start\": 101, \"polyphen_prediction\": \"possibly_damaging\", \"location\": \"15:48610773\", \"protein_id\": \"NP_000129.3\", \"aa\": \"T\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"interval\": {\"end\": 48610773, \"start\": 48610772, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"loeuf\": 0.105, \"mastermind_counts\": \"2|6|6\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.906832635402679, \"variant_allele\": \"C\", \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.301A>G\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 13, \"DP_DG\": -45, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DG\": 0.04, \"DS_AL\": 0, \"DP_DL\": -12, \"DP_AL\": -18}, \"blosum62\": 0, \"lrt_pred\": \"invalid_field\", \"exon\": \"4/66\", \"canonical\": 1, \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"conservation\": 5.92, \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"T/A\", \"codons\": \"Act/Gct\", \"mastermind_mmid3\": \"FBN1:T101A\", \"protein_end\": 101, \"intact\": [{\"interaction_ac\": \"EBI-5400038\", \"feature_type\": \"mutation decreasing\"}], \"polyphen_score\": 0.903, \"cadd_phred\": 25.5, \"sift_prediction\": \"deleterious_low_confidence\"}], \"minimised\": 1, \"end\": 48610773, \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48610773T>C\", \"seq_region_name\": \"15\", \"allele_string\": \"T/C\", \"assembly_name\": \"GRCh38\", \"start\": 48610773}',\n",
       "    'coordinates': [{'pca_x': 0.03159177986969813,\n",
       "      'pca_y': 0.07737327772938982},\n",
       "     {'t-sne_x': 2.7947593396903163, 't-sne_y': 1.867444690254868},\n",
       "     {'umap_x': 4.237123966217041, 'umap_y': 2.920579195022583}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.003538752207532525,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489896-C-T',\n",
       "      'distance': 0.00362053862772882,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537676-C-T',\n",
       "      'distance': 0.0036351450253278017,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446775-T-C',\n",
       "      'distance': 0.003682501148432493,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415684-C-T',\n",
       "      'distance': 0.0037208101712167263,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48489977-C-T',\n",
       "      'distance': 0.0037532877177000046,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.0038029898423701525,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421594-C-T',\n",
       "      'distance': 0.003826938569545746,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485379-C-T',\n",
       "      'distance': 0.003920545801520348,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495162-C-T',\n",
       "      'distance': 0.003947393968701363,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415671-T-C',\n",
       "      'distance': 0.003953586332499981,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497298-T-C',\n",
       "      'distance': 0.003958581946790218,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537718-C-T',\n",
       "      'distance': 0.004005396738648415,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.004063772037625313,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.004083230160176754,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.004117792006582022,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488439-T-C',\n",
       "      'distance': 0.004122869577258825,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481687-T-C',\n",
       "      'distance': 0.004130663350224495,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487107-T-C',\n",
       "      'distance': 0.004135488998144865,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446816-T-C',\n",
       "      'distance': 0.004142485558986664,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48515501-AA-A',\n",
       "    'metadata': {'vcf_string': '15\\t48515501\\t.\\tAA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48515501',\n",
       "     'ref_allele': 'AA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1354del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48515502del',\n",
       "     'hgvs_protein': 'p.Tyr452ThrfsTer127',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"allele_string\": \"A/-\", \"vcf_string\": \"15-48515501-AA-A\", \"end\": 48515502, \"variant_class\": \"deletion\", \"assembly_name\": \"GRCh38\", \"id\": \"NC_000015.10:g.48515502del\", \"transcript_consequences\": [{\"lof_info\": \"PERCENTILE:0.157149489322191,GERP_DIST:7172.58313425342,BP_DIST:7209,DIST_FROM_LAST_EXON:6820,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mane_select\": \"ENST00000316623.10\", \"impact\": \"HIGH\", \"variant_allele\": \"-\", \"gene_symbol\": \"FBN1\", \"given_ref\": \"A\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"exon\": \"12/66\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 1670, \"canonical\": 1, \"strand\": -1, \"protein_start\": 452, \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"Y/X\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 1354, \"allele_num\": 1, \"location\": \"15:48515501\", \"hgvs_offset\": -1, \"consequence_terms\": [\"frameshift_variant\"], \"cdna_end\": 1670, \"spliceai\": {\"DP_DL\": 14, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AL\": -9, \"DS_AG\": 0, \"DS_DG\": 0, \"DP_DG\": -7, \"DP_AG\": 14, \"DS_DL\": 0}, \"codons\": \"Tac/ac\", \"cds_start\": 1354, \"protein_end\": 452, \"conservation\": -0.011, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"end\": 48515502, \"start\": 48515501, \"type\": \"SimpleInterval\"}}}, \"biotype\": \"protein_coding\", \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.1354del\", \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Tyr452ThrfsTer127\", \"used_ref\": \"A\", \"aa\": \"A\"}], \"input\": \"NC_000015.10:g.48515502del\", \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"start\": 48515502, \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.010805125246699188,\n",
       "      'pca_y': 0.01899730114367674},\n",
       "     {'t-sne_x': 1.9125607710882808, 't-sne_y': 6.316481535576118},\n",
       "     {'umap_x': 3.5844385623931885, 'umap_y': 3.675462484359741}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.0999809056520462,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.10021006315946579,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.10050904750823975,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.10095946490764618,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.10115708410739899,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.10154891014099121,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.10166963934898376,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.10169717669487,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.10178792476654053,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.10202084481716156,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452567-C-T',\n",
       "      'distance': 0.1020527333021164,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.10206586122512817,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.1021408662199974,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.1021566316485405,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.10220855474472046,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.102230966091156,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420752-A-G',\n",
       "      'distance': 0.10224451124668121,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.102284274995327,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.10242617130279541,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430736-A-G',\n",
       "      'distance': 0.10247249156236649,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48497301-CCTGAATT-C',\n",
       "    'metadata': {'vcf_string': '15\\t48497301\\t.\\tCCTGAATT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48497301',\n",
       "     'ref_allele': 'CCTGAATT',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2251_2257del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48497302_48497308del',\n",
       "     'hgvs_protein': 'p.Asn751AspfsTer19',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"start\": 48497302, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48497302_48497308del\", \"strand\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"allele_string\": \"CTGAATT/-\", \"vcf_string\": \"15-48497301-CCTGAATT-C\", \"end\": 48497308, \"minimised\": 1, \"id\": \"NC_000015.10:g.48497302_48497308del\", \"transcript_consequences\": [{\"ga4gh_vrs\": {\"location\": {\"interval\": {\"end\": 48497308, \"type\": \"SimpleInterval\", \"start\": 48497301}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"conservation\": 2.184, \"protein_end\": 753, \"biotype\": \"protein_coding\", \"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.2251_2257del\", \"aa\": \"CTGAATT\", \"used_ref\": \"CTGAATT\", \"hgvsp\": \"NP_000129.3:p.Asn751AspfsTer19\", \"allele_num\": 1, \"location\": \"15:48497302-48497308\", \"consequence_terms\": [\"frameshift_variant\"], \"codons\": \"AATTCAGga/ga\", \"cds_start\": 2251, \"cdna_end\": 2573, \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:N751fs\", \"mastermind_counts\": \"2|2|2\", \"strand\": -1, \"protein_start\": 751, \"lof\": \"HC\", \"amino_acids\": \"NSG/X\", \"transcript_id\": \"NM_000138.5\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 2257, \"impact\": \"HIGH\", \"lof_info\": \"PERCENTILE:0.261954503249768,GERP_DIST:6252.16473448426,BP_DIST:6312,DIST_FROM_LAST_EXON:5923,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mane_select\": \"ENST00000316623.10\", \"variant_allele\": \"-\", \"gene_symbol\": \"FBN1\", \"given_ref\": \"CTGAATT\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 2567, \"exon\": \"19/66\", \"loeuf\": 0.105}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"deletion\"}',\n",
       "    'coordinates': [{'pca_x': 0.03727898675115615,\n",
       "      'pca_y': 0.042680999910997074},\n",
       "     {'t-sne_x': 34.788724857342636, 't-sne_y': 7.310177385984071},\n",
       "     {'umap_x': 1.6739771366119385, 'umap_y': 6.896965980529785}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.09762142598628998,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.09800834953784943,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.09826818108558655,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.09887774288654327,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.09889308363199234,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.09923691302537918,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.0992928147315979,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.09936193376779556,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.09999039769172668,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.10009592771530151,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10016892105340958,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.1002495065331459,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437912-G-A',\n",
       "      'distance': 0.10029497742652893,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.1003928929567337,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.10065142810344696,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.10076513141393661,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10077972710132599,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516225-G-A',\n",
       "      'distance': 0.1008702963590622,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.10088331997394562,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537704-G-A',\n",
       "      'distance': 0.10096120834350586,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48489947-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48489947\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48489947',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2986T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48489947A>G',\n",
       "     'hgvs_protein': 'p.Cys996Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"end\": 48489947, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48489947, \"phenotype_or_disease\": 1, \"id\": \"CM154825\"}, {\"end\": 48489947, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48489947, \"phenotype_or_disease\": 1, \"id\": \"CM983864\"}, {\"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"A/G/T\", \"clin_sig_allele\": \"T:pathogenic;G:pathogenic\", \"end\": 48489947, \"pubmed\": [9399842, 17701892], \"frequencies\": {\"G\": {\"gnomadg_sas\": 0, \"gnomadg_remaining\": 0, \"gnomadg_fin\": 0, \"gnomadg_nfe\": 1.47e-05, \"gnomadg_afr\": 0, \"gnomadg\": 6.576e-06, \"gnomadg_amr\": 0, \"gnomadg_asj\": 0, \"gnomadg_mid\": 0, \"gnomadg_eas\": 0, \"gnomadg_ami\": 0}}, \"clin_sig\": [\"pathogenic\"], \"start\": 48489947, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"UniProt\": [\"VAR_002297\"], \"ClinVar\": [\"RCV001064051\", \"VCV000858220\", \"RCV000631920\", \"VCV000527151\"]}, \"id\": \"rs140592\"}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48489947A>G\", \"transcript_consequences\": [{\"biotype\": \"protein_coding\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9984}, \"conservation\": 3.76, \"protein_end\": 996, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"end\": 48489947, \"start\": 48489946, \"type\": \"SimpleInterval\"}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Cys996Arg\", \"aa\": \"A\", \"used_ref\": \"A\", \"hgvsc\": \"NM_000138.5:c.2986T>C\", \"gene_id\": \"2200\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"cdna_end\": 3302, \"cds_start\": 2986, \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 26, \"DP_DG\": -22, \"DS_DG\": 0, \"DP_AL\": 21, \"DS_AG\": 0, \"DS_AL\": 0, \"DP_DL\": 21, \"SYMBOL\": \"FBN1\"}, \"codons\": \"Tgt/Cgt\", \"eve_score\": 0.792255411403384, \"consequence_terms\": [\"missense_variant\"], \"location\": \"15:48489947\", \"cadd_raw\": 5.178646, \"cadd_phred\": 29, \"strand\": -1, \"polyphen_prediction\": \"probably_damaging\", \"polyphen_score\": 0.994, \"mutationtaster_pred\": \"A\", \"sift_score\": 0, \"mastermind_counts\": \"7|12|12\", \"clinpred\": 0.999889135360718, \"mastermind_mmid3\": \"FBN1:C996R\", \"canonical\": 1, \"cds_end\": 2986, \"protein_id\": \"NP_000129.3\", \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/R\", \"protein_start\": 996, \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -3, \"impact\": \"MODERATE\", \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 3302, \"exon\": \"25/66\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"eve_class\": \"Pathogenic\", \"given_ref\": \"A\"}], \"minimised\": 1, \"allele_string\": \"A/G\", \"end\": 48489947, \"vcf_string\": \"15-48489947-A-G\", \"input\": \"NC_000015.10:g.48489947A>G\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"start\": 48489947}',\n",
       "    'coordinates': [{'pca_x': 0.04646095789998872,\n",
       "      'pca_y': 0.08089165164897155},\n",
       "     {'t-sne_x': 35.97874200973319, 't-sne_y': 1.364225343043101},\n",
       "     {'umap_x': 1.425195336341858, 'umap_y': 6.683753490447998}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48489956-A-G',\n",
       "      'distance': 0.001560418400913477,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.0018136862199753523,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437370-A-G',\n",
       "      'distance': 0.001974965911358595,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0022463698405772448,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0022947010584175587,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.002397596836090088,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0024024960584938526,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.002424719277769327,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.0025472212582826614,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596325-A-G',\n",
       "      'distance': 0.002621293533593416,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.0026225168257951736,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.002638349076732993,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596319-A-G',\n",
       "      'distance': 0.002639663638547063,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.002670049201697111,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.002681538462638855,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.002713463269174099,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.002715541049838066,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434702-A-G',\n",
       "      'distance': 0.002741336589679122,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.002797124907374382,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596373-A-G',\n",
       "      'distance': 0.0028168512508273125,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488458-TGCAGAGGCTGGGT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48488458\\t.\\tTGCAGAGGCTGGGT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488458',\n",
       "     'ref_allele': 'TGCAGAGGCTGGGT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3110_3122del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488459_48488471del',\n",
       "     'hgvs_protein': 'p.Ser1037ThrfsTer47',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"strand\": -1, \"canonical\": 1, \"protein_id\": \"NP_000129.3\", \"cds_end\": 3122, \"lof\": \"HC\", \"protein_start\": 1037, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"SLCTH/X\", \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"PERCENTILE:0.362349117920149,GERP_DIST:6053.72533015311,BP_DIST:5459,DIST_FROM_LAST_EXON:5070,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"impact\": \"HIGH\", \"variant_allele\": \"-\", \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"26/66\", \"cdna_start\": 3426, \"given_ref\": \"GCAGAGGCTGGGT\", \"ptriplo\": 0.984699554922396, \"biotype\": \"protein_coding\", \"conservation\": 1.933, \"protein_end\": 1041, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"start\": 48488458, \"type\": \"SimpleInterval\", \"end\": 48488471}}}, \"hgvsp\": \"NP_000129.3:p.Ser1037ThrfsTer47\", \"used_ref\": \"TGGGTGCAGAGGC\", \"aa\": \"GCAGAGGCTGGGT\", \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.3110_3122del\", \"gene_id\": \"2200\", \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"cdna_end\": 3438, \"codons\": \"aGCCTCTGCACCCAc/ac\", \"cds_start\": 3110, \"location\": \"15:48488454-48488466\", \"hgvs_offset\": -5}], \"id\": \"NC_000015.10:g.48488459_48488471del\", \"variant_class\": \"deletion\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48488458-TGCAGAGGCTGGGT-T\", \"end\": 48488471, \"allele_string\": \"GCAGAGGCTGGGT/-\", \"minimised\": 1, \"seq_region_name\": \"15\", \"start\": 48488459, \"strand\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"input\": \"NC_000015.10:g.48488459_48488471del\"}',\n",
       "    'coordinates': [{'pca_x': 0.029508192245440254,\n",
       "      'pca_y': 0.027616505108902366},\n",
       "     {'t-sne_x': 33.14841463263826, 't-sne_y': 6.189726511466886},\n",
       "     {'umap_x': 1.7103499174118042, 'umap_y': 6.431199550628662}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.11306308954954147,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.11306702345609665,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.11450353264808655,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.11461278796195984,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.11506607383489609,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.11522731184959412,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.11531756818294525,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.11554167419672012,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437912-G-A',\n",
       "      'distance': 0.11595819890499115,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48516225-G-A',\n",
       "      'distance': 0.11625171452760696,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.11637500673532486,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.11660952866077423,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.11671132594347,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468427-G-A',\n",
       "      'distance': 0.116941437125206,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.11695750057697296,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.11702439188957214,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420714-G-A',\n",
       "      'distance': 0.11739916354417801,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.11743466556072235,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537704-G-A',\n",
       "      'distance': 0.1174614280462265,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.11755363643169403,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488363-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48488363\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488363',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3208+5G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488363C>A',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_5th_base_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"C/A\", \"vcf_string\": \"15-48488363-C-A\", \"end\": 48488363, \"minimised\": 1, \"id\": \"NC_000015.10:g.48488363C>A\", \"transcript_consequences\": [{\"variant_allele\": \"A\", \"impact\": \"LOW\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"C\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"mastermind_counts\": \"6|11|24\", \"mastermind_mmid3\": \"FBN1:D1070int,FBN1:D1070intd,FBN1:D1070srd\", \"canonical\": 1, \"strand\": -1, \"ada_score\": 0.999511111782114, \"maxentscan_alt\": 4.445, \"rf_score\": 0.932, \"transcript_id\": \"NM_000138.5\", \"protein_id\": \"NP_000129.3\", \"maxentscan_ref\": 8.941, \"allele_num\": 1, \"maxentscan_diff\": 4.496, \"cadd_phred\": 18.77, \"location\": \"15:48488363\", \"cadd_raw\": 2.449286, \"spliceai\": {\"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 5, \"DP_AL\": 15, \"DS_AG\": 0, \"DP_DG\": -31, \"DS_DG\": 0, \"DP_AG\": -27, \"DS_DL\": 0.76}, \"consequence_terms\": [\"splice_donor_5th_base_variant\", \"intron_variant\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48488362, \"end\": 48488363}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"conservation\": 3.95, \"biotype\": \"protein_coding\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3208+5G>T\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"C\", \"aa\": \"C\", \"intron\": \"26/65\"}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48488363, \"id\": \"CS941484\", \"phenotype_or_disease\": 1, \"start\": 48488363}], \"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"start\": 48488363, \"input\": \"NC_000015.10:g.48488363C>A\", \"strand\": 1, \"most_severe_consequence\": \"splice_donor_5th_base_variant\"}',\n",
       "    'coordinates': [{'pca_x': -0.23665180990333495,\n",
       "      'pca_y': -0.03557352693958141},\n",
       "     {'t-sne_x': -18.834151166948743, 't-sne_y': -33.41312703256291},\n",
       "     {'umap_x': 13.642850875854492, 'umap_y': 12.331378936767578}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48446701-C-A',\n",
       "      'distance': 0.0011266934452578425,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508577-C-T',\n",
       "      'distance': 0.0018600369803607464,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460241-C-T',\n",
       "      'distance': 0.0019949516281485558,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446701-C-T',\n",
       "      'distance': 0.002183794742450118,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515385-A-G',\n",
       "      'distance': 0.0035397293977439404,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48412564-C-T',\n",
       "      'distance': 0.0036766361445188522,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427561-A-C',\n",
       "      'distance': 0.01274157501757145,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48437318-T-C',\n",
       "      'distance': 0.013289199210703373,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48445370-A-G',\n",
       "      'distance': 0.0148176159709692,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437765-T-C',\n",
       "      'distance': 0.015583026222884655,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489845-T-C',\n",
       "      'distance': 0.015634968876838684,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48487309-T-C',\n",
       "      'distance': 0.016841163858771324,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48472544-C-A',\n",
       "      'distance': 0.018041308969259262,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520652-C-T',\n",
       "      'distance': 0.020588770508766174,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48434587-G-C',\n",
       "      'distance': 0.023862365633249283,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48445375-C-A',\n",
       "      'distance': 0.024679549038410187,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508581-C-A',\n",
       "      'distance': 0.024721559137105942,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-A',\n",
       "      'distance': 0.024732805788517,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437320-A-G',\n",
       "      'distance': 0.024976471439003944,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-A',\n",
       "      'distance': 0.025123659521341324,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'uncertain',\n",
       "      'confidence_score': 0.5333333333333333},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.65}}},\n",
       "   {'variant_id': '15-48488229-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48488229\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488229',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3221G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488229C>T',\n",
       "     'hgvs_protein': 'p.Cys1074Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"end\": 48488229, \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"start\": 48488229, \"id\": \"HM971939\", \"strand\": 1, \"end\": 48488229, \"seq_region_name\": \"15\"}, {\"start\": 48488229, \"id\": \"rs1555398645\", \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000507217\", \"VCV000439705\"], \"UniProt\": [\"VAR_076046\"]}, \"clin_sig_allele\": \"T:pathogenic\", \"end\": 48488229, \"strand\": 1, \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"C/T\", \"phenotype_or_disease\": 1}], \"strand\": 1, \"seq_region_name\": \"15\", \"assembly_name\": \"GRCh38\", \"start\": 48488229, \"vcf_string\": \"15-48488229-C-T\", \"allele_string\": \"C/T\", \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48488229C>T\", \"transcript_consequences\": [{\"codons\": \"tGc/tAc\", \"sift_prediction\": \"deleterious_low_confidence\", \"transcript_id\": \"NM_000138.5\", \"cadd_raw\": 5.645875, \"aa\": \"C\", \"cds_start\": 3221, \"lrt_pred\": \"invalid_field\", \"mastermind_mmid3\": \"FBN1:C1074Y\", \"clinpred\": 0.999947428703308, \"phaplo\": 0.99897615427687, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"protein_start\": 1074, \"polyphen_prediction\": \"probably_damaging\", \"cdna_start\": 3537, \"hgvsc\": \"NM_000138.5:c.3221G>A\", \"sift_score\": 0, \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"exon\": \"27/66\", \"biotype\": \"protein_coding\", \"cds_end\": 3221, \"protein_end\": 1074, \"polyphen_score\": 0.944, \"canonical\": 1, \"hgvsp\": \"NP_000129.3:p.Cys1074Tyr\", \"loeuf\": 0.105, \"location\": \"15:48488229\", \"alphamissense\": {\"am_pathogenicity\": 0.999, \"am_class\": \"likely_pathogenic\"}, \"gene_id\": \"2200\", \"cadd_phred\": 33, \"spliceai\": {\"DP_DL\": -4, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": -28, \"DS_DL\": 0, \"DS_AL\": 0, \"DP_AG\": -30, \"DP_DG\": 3, \"SYMBOL\": \"FBN1\"}, \"eve_score\": 0.761305890362243, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48488229, \"type\": \"SimpleInterval\", \"start\": 48488228}}, \"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}}, \"impact\": \"MODERATE\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"C\", \"variant_allele\": \"T\", \"cdna_end\": 3537, \"eve_class\": \"Pathogenic\", \"mutationtaster_pred\": \"A\", \"mastermind_counts\": \"8|8|8\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"given_ref\": \"C\", \"gene_symbol\": \"FBN1\", \"blosum62\": -2, \"mutfunc\": {\"exp\": {\"ddG\": -1.4108, \"dG_wt\": 173.9986, \"dG_mt_sd\": 0.456181, \"dG_wt_sd\": 0.228295, \"ddG_sd\": 0.302708, \"dG_mt\": 172.5878}}, \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"C/Y\", \"ptriplo\": 0.984699554922396}], \"id\": \"NC_000015.10:g.48488229C>T\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.039016855651786087,\n",
       "      'pca_y': 0.08195096977260993},\n",
       "     {'t-sne_x': 7.497928520610581, 't-sne_y': 9.917716017998883},\n",
       "     {'umap_x': 3.0175740718841553, 'umap_y': 2.5252106189727783}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0006790084298700094,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415695-C-T',\n",
       "      'distance': 0.0008485236903652549,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434599-C-T',\n",
       "      'distance': 0.000985042890533805,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.0010295152897015214,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.0010554397013038397,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0010874480940401554,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474293-C-T',\n",
       "      'distance': 0.0011030880268663168,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.0011540950508788228,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.0011550886556506157,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468456-C-T',\n",
       "      'distance': 0.0011725772637873888,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.0012231244472786784,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.0012370292097330093,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428390-C-T',\n",
       "      'distance': 0.001326749799773097,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.0013891442213207483,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495173-C-T',\n",
       "      'distance': 0.0014178077690303326,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437827-C-T',\n",
       "      'distance': 0.0014337787870317698,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.0014353534206748009,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481699-C-T',\n",
       "      'distance': 0.0014812285080552101,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492574-C-T',\n",
       "      'distance': 0.0014962372370064259,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.0014986839378252625,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48463224-CAAACTTC-C',\n",
       "    'metadata': {'vcf_string': '15\\t48463224\\t.\\tCAAACTTC\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48463224',\n",
       "     'ref_allele': 'CAAACTTC',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5076_5082del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48463225_48463231del',\n",
       "     'hgvs_protein': 'p.Arg1692SerfsTer21',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48463225_48463231del\", \"most_severe_consequence\": \"frameshift_variant\", \"input\": \"NC_000015.10:g.48463225_48463231del\", \"transcript_consequences\": [{\"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"amino_acids\": \"RSL/X\", \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"PERCENTILE:0.58983286908078,GERP_DIST:4213.94907216728,BP_DIST:3511,DIST_FROM_LAST_EXON:3122,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"given_ref\": \"AAACTTC\", \"gene_id\": \"2200\", \"location\": \"15:48463224-48463230\", \"loeuf\": 0.105, \"used_ref\": \"CAAACTT\", \"variant_allele\": \"-\", \"cdna_end\": 5398, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"type\": \"Allele\", \"location\": {\"interval\": {\"start\": 48463224, \"type\": \"SimpleInterval\", \"end\": 48463231}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"mane\": [\"MANE_Select\"], \"impact\": \"HIGH\", \"protein_start\": 1692, \"hgvsc\": \"NM_000138.5:c.5076_5082del\", \"cdna_start\": 5392, \"hgvs_offset\": -1, \"strand\": -1, \"protein_end\": 1694, \"cds_end\": 5082, \"canonical\": 1, \"hgvsp\": \"NP_000129.3:p.Arg1692SerfsTer21\", \"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"conservation\": 2.922, \"exon\": \"42/66\", \"lof\": \"HC\", \"aa\": \"AAACTTC\", \"transcript_id\": \"NM_000138.5\", \"codons\": \"agAAGTTTG/ag\", \"protein_id\": \"NP_000129.3\", \"cds_start\": 5076, \"phaplo\": 0.99897615427687}], \"minimised\": 1, \"allele_string\": \"AAACTTC/-\", \"vcf_string\": \"15-48463224-CAAACTTC-C\", \"start\": 48463225, \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48463231}',\n",
       "    'coordinates': [{'pca_x': 0.039309842167400966,\n",
       "      'pca_y': 0.04121791523049865},\n",
       "     {'t-sne_x': 29.60944298980886, 't-sne_y': 14.224971475184617},\n",
       "     {'umap_x': 1.7252811193466187, 'umap_y': 7.412687301635742}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.10045907646417618,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.10057184845209122,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10069781541824341,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.10123100876808167,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.10135851055383682,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.10191217809915543,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.10235483944416046,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.10259205102920532,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.10274273157119751,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.1033296138048172,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.10336942970752716,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10349389910697937,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.10354959964752197,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.1035810187458992,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.10362100601196289,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420714-G-A',\n",
       "      'distance': 0.10363276302814484,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445430-G-A',\n",
       "      'distance': 0.10374394804239273,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.10379939526319504,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.10384240746498108,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437912-G-A',\n",
       "      'distance': 0.10385122895240784,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48463145-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48463145\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48463145',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5161T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48463145A>C',\n",
       "     'hgvs_protein': 'p.Cys1721Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"id\": \"CM1211214\", \"start\": 48463145, \"end\": 48463145, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1}], \"strand\": 1, \"end\": 48463145, \"vcf_string\": \"15-48463145-A-C\", \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"start\": 48463145, \"transcript_consequences\": [{\"strand\": -1, \"protein_start\": 1721, \"polyphen_prediction\": \"probably_damaging\", \"cdna_start\": 5477, \"hgvsc\": \"NM_000138.5:c.5161T>G\", \"sift_score\": 0, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"42/66\", \"conservation\": 3.95, \"biotype\": \"protein_coding\", \"protein_end\": 1721, \"cds_end\": 5161, \"hgvsp\": \"NP_000129.3:p.Cys1721Gly\", \"polyphen_score\": 0.985, \"canonical\": 1, \"codons\": \"Tgt/Ggt\", \"transcript_id\": \"NM_000138.5\", \"sift_prediction\": \"deleterious_low_confidence\", \"cadd_raw\": 5.068686, \"aa\": \"A\", \"cds_start\": 5161, \"lrt_pred\": \"invalid_field\", \"mastermind_mmid3\": \"FBN1:C1721G\", \"clinpred\": 0.99894243478775, \"phaplo\": 0.99897615427687, \"protein_id\": \"NP_000129.3\", \"eve_class\": \"Pathogenic\", \"mutationtaster_pred\": \"D\", \"mastermind_counts\": \"2|3|3\", \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"given_ref\": \"A\", \"gene_symbol\": \"FBN1\", \"blosum62\": -3, \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"C/G\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"location\": \"15:48463145\", \"alphamissense\": {\"am_pathogenicity\": 0.9929, \"am_class\": \"likely_pathogenic\"}, \"gene_id\": \"2200\", \"spliceai\": {\"DP_AG\": 12, \"DP_DG\": 0, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DP_DL\": -43, \"DS_DG\": 0, \"DS_DL\": 0, \"DS_AL\": 0, \"DP_AL\": -33}, \"cadd_phred\": 28.3, \"eve_score\": 0.790628069032541, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"type\": \"Allele\", \"location\": {\"interval\": {\"end\": 48463145, \"type\": \"SimpleInterval\", \"start\": 48463144}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"impact\": \"MODERATE\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"A\", \"variant_allele\": \"C\", \"cdna_end\": 5477}], \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48463145A>C\", \"allele_string\": \"A/C\", \"minimised\": 1, \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48463145A>C\"}',\n",
       "    'coordinates': [{'pca_x': 0.04482481773635305,\n",
       "      'pca_y': 0.09004516144132002},\n",
       "     {'t-sne_x': 19.080837012111587, 't-sne_y': 4.266136136573814},\n",
       "     {'umap_x': 3.7537920475006104, 'umap_y': 6.28398323059082}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48445432-A-C',\n",
       "      'distance': 0.0024401829577982426,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483822-A-C',\n",
       "      'distance': 0.0026299760211259127,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.002886031288653612,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.003132348880171776,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.003371079219505191,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.003385685384273529,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.00350005179643631,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.0035292617976665497,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415716-T-C',\n",
       "      'distance': 0.003637793706730008,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508693-A-C',\n",
       "      'distance': 0.003648291574791074,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48436967-A-C',\n",
       "      'distance': 0.0037136636674404144,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.003761765779927373,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468522-C-A',\n",
       "      'distance': 0.0038030524738132954,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.0038077228236943483,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415545-A-C',\n",
       "      'distance': 0.0038599157705903053,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.004001116845756769,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520677-A-C',\n",
       "      'distance': 0.004054665099829435,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434707-T-C',\n",
       "      'distance': 0.004055536352097988,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437011-T-C',\n",
       "      'distance': 0.0041448455303907394,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.0041530542075634,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48415736-G-C',\n",
       "    'metadata': {'vcf_string': '15\\t48415736\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415736',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7851C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415736G>C',\n",
       "     'hgvs_protein': 'p.Cys2617Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"end\": 48415736, \"strand\": 1, \"colocated_variants\": [{\"id\": \"CM098730\", \"start\": 48415736, \"seq_region_name\": \"15\", \"end\": 48415736, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1}, {\"somatic\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"phenotype_or_disease\": 1, \"start\": 48415736, \"id\": \"COSV57331096\", \"end\": 48415736, \"strand\": 1, \"seq_region_name\": \"15\", \"var_synonyms\": {\"COSMIC\": [\"COSM1373252\"]}}], \"start\": 48415736, \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"vcf_string\": \"15-48415736-G-C\", \"minimised\": 1, \"allele_string\": \"G/C\", \"input\": \"NC_000015.10:g.48415736G>C\", \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"gene_symbol\": \"FBN1\", \"blosum62\": -2, \"given_ref\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"C/W\", \"ptriplo\": 0.984699554922396, \"eve_class\": \"Pathogenic\", \"mutationtaster_pred\": \"D\", \"mastermind_counts\": \"1|1|1\", \"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48415736, \"type\": \"SimpleInterval\", \"start\": 48415735}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"type\": \"Allele\"}, \"mane\": [\"MANE_Select\"], \"impact\": \"MODERATE\", \"used_ref\": \"G\", \"variant_allele\": \"C\", \"cdna_end\": 8167, \"alphamissense\": {\"am_pathogenicity\": 0.9982, \"am_class\": \"likely_pathogenic\"}, \"location\": \"15:48415736\", \"loeuf\": 0.105, \"gene_id\": \"2200\", \"eve_score\": 0.782527455507016, \"spliceai\": {\"DP_DG\": 31, \"DP_AG\": 34, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DP_DL\": -38, \"DS_DG\": 0, \"DS_DL\": 0, \"DS_AL\": 0.02, \"DP_AL\": 31}, \"cadd_phred\": 12.66, \"gene_symbol_source\": \"EntrezGene\", \"biotype\": \"protein_coding\", \"conservation\": -10.7, \"exon\": \"64/66\", \"cds_end\": 7851, \"protein_end\": 2617, \"canonical\": 1, \"polyphen_score\": 0.935, \"hgvsp\": \"NP_000129.3:p.Cys2617Trp\", \"strand\": -1, \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 2617, \"sift_score\": 0, \"hgvsc\": \"NM_000138.5:c.7851C>G\", \"cdna_start\": 8167, \"mastermind_mmid3\": \"FBN1:C2617W\", \"clinpred\": 0.998922526836395, \"lrt_pred\": \"invalid_field\", \"cds_start\": 7851, \"phaplo\": 0.99897615427687, \"protein_id\": \"NP_000129.3\", \"codons\": \"tgC/tgG\", \"aa\": \"G\", \"cadd_raw\": 1.26415, \"transcript_id\": \"NM_000138.5\", \"sift_prediction\": \"deleterious_low_confidence\"}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48415736G>C\"}',\n",
       "    'coordinates': [{'pca_x': 0.04007475327360838,\n",
       "      'pca_y': 0.09088396803503748},\n",
       "     {'t-sne_x': 17.23943855767039, 't-sne_y': -7.399742935486912},\n",
       "     {'umap_x': 8.266603469848633, 'umap_y': 7.0840229988098145}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48437064-G-C',\n",
       "      'distance': 0.0012000193819403648,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.0015756897628307343,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421652-G-C',\n",
       "      'distance': 0.0019535990431904793,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432910-C-G',\n",
       "      'distance': 0.0024890825152397156,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48467996-G-C',\n",
       "      'distance': 0.0030563008040189743,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.0031656953506171703,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468455-G-C',\n",
       "      'distance': 0.00321213249117136,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.003222026862204075,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.00340458145365119,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.0034775901585817337,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.0036178489681333303,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437784-G-C',\n",
       "      'distance': 0.003623243886977434,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415716-T-C',\n",
       "      'distance': 0.0036464855074882507,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.0037861960008740425,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508673-G-C',\n",
       "      'distance': 0.0039510950446128845,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.003952797502279282,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472670-T-C',\n",
       "      'distance': 0.004147321917116642,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.004153904039412737,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437347-G-C',\n",
       "      'distance': 0.004168629180639982,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488439-T-C',\n",
       "      'distance': 0.004172398708760738,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48644605-48644769-del-aa8c294c',\n",
       "    'metadata': {'vcf_string': '15\\t48644605\\t.\\tCCCTTTAAGCGCGTCGTGTCCTCCACCGCCTCTTCTCTTGGCCCGACTGGCTCTGGTTTCCTTCACGTTCCCAGCCTCCAAATTGGCGTCCGCCCCATGGCTCGTGTAGGACGCTAAAAGCACGGTAAATCCCAGGGCGATCTCCAGCAGACGCCCTCGACGCAT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48644605',\n",
       "     'ref_allele': 'CCCTTTAAGCGCGTCGTGTCCTCCACCGCCTCTTCTCTTGGCCCGACTGGCTCTGGTTTCCTTCACGTTCCCAGCCTCCAAATTGGCGTCCGCCCCATGGCTCGTGTAGGACGCTAAAAGCACGGTAAATCCCAGGGCGATCTCCAGCAGACGCCCTCGACGCAT',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1_164del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48644606_48644769del',\n",
       "     'hgvs_protein': 'p.Met1_?55',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"start\": 48644606, \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"vcf_string\": \"15-48644605-CCCTTTAAGCGCGTCGTGTCCTCCACCGCCTCTTCTCTTGGCCCGACTGGCTCTGGTTTCCTTCACGTTCCCAGCCTCCAAATTGGCGTCCGCCCCATGGCTCGTGTAGGACGCTAAAAGCACGGTAAATCCCAGGGCGATCTCCAGCAGACGCCCTCGACGCAT-C\", \"end\": 48644769, \"strand\": 1, \"id\": \"NC_000015.10:g.48644606_48644769del\", \"variant_class\": \"deletion\", \"minimised\": 1, \"allele_string\": \"CCTTTAAGCGCGTCGTGTCCTCCACCGCCTCTTCTCTTGGCCCGACTGGCTCTGGTTTCCTTCACGTTCCCAGCCTCCAAATTGGCGTCCGCCCCATGGCTCGTGTAGGACGCTAAAAGCACGGTAAATCCCAGGGCGATCTCCAGCAGACGCCCTCGACGCAT/-\", \"most_severe_consequence\": \"frameshift_variant\", \"input\": \"NC_000015.10:g.48644606_48644769del\", \"transcript_consequences\": [{\"codons\": \"ATGCGTCGAGGGCGTCTGCTGGAGATCGCCCTGGGATTTACCGTGCTTTTAGCGTCCTACACGAGCCATGGGGCGGACGCCAATTTGGAGGCTGGGAACGTGAAGGAAACCAGAGCCAGTCGGGCCAAGAGAAGAGGCGGTGGAGGACACGACGCGCTTAAAGGa/a\", \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"aa\": \"CCTTTAAGCGCGTCGTGTCCTCCACCGCCTCTTCTCTTGGCCCGACTGGCTCTGGTTTCCTTCACGTTCCCAGCCTCCAAATTGGCGTCCGCCCCATGGCTCGTGTAGGACGCTAAAAGCACGGTAAATCCCAGGACGATCTCCAGCAGACgCCCTCGACGCAT\", \"cds_start\": 1, \"mastermind_mmid3\": \"FBN1:G55srd,FBN1:M1del\", \"phaplo\": 0.99897615427687, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"protein_start\": 1, \"cdna_start\": 317, \"hgvsc\": \"NM_000138.5:c.1_164del\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.803, \"exon\": \"2/66\", \"biotype\": \"protein_coding\", \"protein_end\": 55, \"cds_end\": 164, \"canonical\": 1, \"hgvsp\": \"NP_000129.3:p.Met1_?55\", \"location\": \"15:48644606-48644769\", \"loeuf\": 0.105, \"gene_id\": \"2200\", \"genesplicer\": \"loss/donor/48644646-48644645/Medium/6.120963,loss/acceptor/48644698-48644697/Medium/4.824589\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48644605, \"type\": \"SimpleInterval\", \"end\": 48644769}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"impact\": \"HIGH\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"CCTTTAAGCGCGTCGTGTCCTCCACCGCCTCTTCTCTTGGCCCGACTGGCTCTGGTTTCCTTCACGTTCCCAGCCTCCAAATTGGCGTCCGCCCCATGGCTCGTGTAGGACGCTAAAAGCACGGTAAATCCCAGGGCGATCTCCAGCAGACGCCCTCGACGCAT\", \"variant_allele\": \"-\", \"cdna_end\": 480, \"mastermind_counts\": \"1|13|45\", \"consequence_terms\": [\"frameshift_variant\", \"start_lost\", \"splice_region_variant\"], \"allele_num\": 1, \"given_ref\": \"CCTTTAAGCGCGTCGTGTCCTCCACCGCCTCTTCTCTTGGCCCGACTGGCTCTGGTTTCCTTCACGTTCCCAGCCTCCAAATTGGCGTCCGCCCCATGGCTCGTGTAGGACGCTAAAAGCACGGTAAATCCCAGGGCGATCTCCAGCAGACGCCCTCGACGCAT\", \"gene_symbol\": \"FBN1\", \"amino_acids\": \"MRRGRLLEIALGFTVLLASYTSHGADANLEAGNVKETRASRAKRRGGGGHDALKG/X\", \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"PERCENTILE:0.0190343546889508,GERP_DIST:8502.4442873613,BP_DIST:8388,DIST_FROM_LAST_EXON:7999,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"ptriplo\": 0.984699554922396}]}',\n",
       "    'coordinates': [{'pca_x': -0.04065593134933799,\n",
       "      'pca_y': 0.03387641257938619},\n",
       "     {'t-sne_x': -12.713546083428103, 't-sne_y': -3.146197147977157},\n",
       "     {'umap_x': 0.20331697165966034, 'umap_y': 15.03797435760498}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.0855531096458435,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.08620679378509521,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.08792391419410706,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.08854848146438599,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.08863769471645355,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.08864764869213104,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.08927233517169952,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.08928070962429047,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.08958099037408829,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.08998757600784302,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.09003009647130966,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.09009762108325958,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.09055778384208679,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.09103122353553772,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.09104374051094055,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.09121817350387573,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.09168210625648499,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.09195372462272644,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.09199216961860657,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.09363557398319244,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48600215-GCG-G',\n",
       "    'metadata': {'vcf_string': '15\\t48600215\\t.\\tGCG\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48600215',\n",
       "     'ref_allele': 'GCG',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.365_366del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48600216_48600217del',\n",
       "     'hgvs_protein': 'p.Arg122LeufsTer6',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48600215-GCG-G\", \"minimised\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"seq_region_name\": \"15\", \"variant_class\": \"deletion\", \"assembly_name\": \"GRCh38\", \"start\": 48600216, \"transcript_consequences\": [{\"exon\": \"5/66\", \"conservation\": 6.95, \"impact\": \"HIGH\", \"hgvsc\": \"NM_000138.5:c.365_366del\", \"gene_id\": \"2200\", \"protein_end\": 122, \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.0424791086350975,GERP_DIST:8250.25925771752,BP_DIST:8190,DIST_FROM_LAST_EXON:7801,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"loeuf\": 0.105, \"genesplicer\": \"gain/acceptor/48600217-48600216/Medium/1.487214\", \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"R/X\", \"canonical\": 1, \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"protein_start\": 122, \"cds_end\": 366, \"codons\": \"cGC/c\", \"cdna_end\": 682, \"aa\": \"CG\", \"hgvsp\": \"NP_000129.3:p.Arg122LeufsTer6\", \"location\": \"15:48600215-48600216\", \"allele_num\": 1, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"spliceai\": {\"DP_AG\": -14, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0.28, \"DP_DL\": -13, \"DS_AG\": 0, \"DP_AL\": 19, \"DS_DL\": 0, \"DP_DG\": 19}, \"given_ref\": \"CG\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"start\": 48600215, \"type\": \"SimpleInterval\", \"end\": 48600217}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}}, \"mutfunc\": {\"exp\": {\"ddG\": 0.3222, \"ddG_sd\": 0.327615, \"dG_wt_sd\": 0.031785, \"dG_mt_sd\": 0.350123, \"dG_wt\": 110.8844, \"dG_mt\": 111.2066}}, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"used_ref\": \"GC\", \"hgvs_offset\": -1, \"cds_start\": 365, \"cdna_start\": 681, \"consequence_terms\": [\"frameshift_variant\"]}], \"allele_string\": \"CG/-\", \"strand\": 1, \"id\": \"NC_000015.10:g.48600216_48600217del\", \"end\": 48600217, \"input\": \"NC_000015.10:g.48600216_48600217del\"}',\n",
       "    'coordinates': [{'pca_x': 0.04664973123852473,\n",
       "      'pca_y': 0.03193261565263369},\n",
       "     {'t-sne_x': 35.51708271107232, 't-sne_y': 11.334038741456562},\n",
       "     {'umap_x': 1.4754642248153687, 'umap_y': 7.595690727233887}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.10530491173267365,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520711-G-T',\n",
       "      'distance': 0.10680145025253296,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10691286623477936,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10879898071289062,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.10903844982385635,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.10904812812805176,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10927335172891617,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.10947544872760773,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10986975580453873,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441797-G-T',\n",
       "      'distance': 0.10988853126764297,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.1101948693394661,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.1102386862039566,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.11024490743875504,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.1103736013174057,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.11043562740087509,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.11051910370588303,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.11071726679801941,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.11128436774015427,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537704-G-A',\n",
       "      'distance': 0.11128569394350052,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437912-G-A',\n",
       "      'distance': 0.11132600158452988,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48600195-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48600195\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48600195',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.386G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48600195C>T',\n",
       "     'hgvs_protein': 'p.Cys129Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"end\": 48600195, \"vcf_string\": \"15-48600195-C-T\", \"allele_string\": \"C/T\", \"minimised\": 1, \"transcript_consequences\": [{\"sift_score\": 0, \"mastermind_counts\": \"11|21|21\", \"mutationtaster_pred\": \"A\", \"canonical\": 1, \"clinpred\": 0.99924635887146, \"mastermind_mmid3\": \"FBN1:C129Y\", \"strand\": -1, \"polyphen_score\": 0.963, \"polyphen_prediction\": \"probably_damaging\", \"amino_acids\": \"C/Y\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 129, \"cds_end\": 386, \"protein_id\": \"NP_000129.3\", \"variant_allele\": \"T\", \"impact\": \"MODERATE\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -2, \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"C\", \"exon\": \"5/66\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 702, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48600194, \"type\": \"SimpleInterval\", \"end\": 48600195}, \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"alphamissense\": {\"am_pathogenicity\": 0.9951, \"am_class\": \"likely_pathogenic\"}, \"protein_end\": 129, \"conservation\": 6.95, \"biotype\": \"protein_coding\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.386G>A\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"C\", \"aa\": \"C\", \"hgvsp\": \"NP_000129.3:p.Cys129Tyr\", \"allele_num\": 1, \"cadd_phred\": 27.6, \"location\": \"15:48600195\", \"mutfunc\": {\"exp\": {\"dG_mt\": 112.8372, \"ddG\": 1.9222, \"dG_wt_sd\": 0.005522, \"ddG_sd\": 0.171844, \"dG_wt\": 110.915, \"dG_mt_sd\": 0.171533}}, \"cadd_raw\": 4.938436, \"cds_start\": 386, \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -30, \"DS_DG\": 0, \"DP_DG\": 7, \"DS_AG\": 0, \"DP_AL\": 39, \"SYMBOL\": \"FBN1\", \"DP_DL\": 19, \"DS_AL\": 0.08}, \"codons\": \"tGc/tAc\", \"cdna_end\": 702, \"consequence_terms\": [\"missense_variant\"]}], \"id\": \"NC_000015.10:g.48600195C>T\", \"colocated_variants\": [{\"id\": \"CM950436\", \"phenotype_or_disease\": 1, \"start\": 48600195, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48600195}, {\"somatic\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"end\": 48600195, \"start\": 48600195, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM6001826\"]}, \"id\": \"COSV57318510\"}, {\"start\": 48600195, \"clin_sig\": [\"pathogenic\"], \"phenotype_or_disease\": 1, \"var_synonyms\": {\"UniProt\": [\"VAR_002278\"], \"ClinVar\": [\"RCV000696593\", \"VCV000574616\", \"RCV002007277\", \"VCV001451553\"]}, \"id\": \"rs1566935517\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"C/A/T\", \"clin_sig_allele\": \"A:pathogenic;T:pathogenic\", \"pubmed\": [22539340, 7611299, 17701892], \"end\": 48600195}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"start\": 48600195, \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48600195C>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.038256456442669895,\n",
       "      'pca_y': 0.07753068881388667},\n",
       "     {'t-sne_x': 8.411371287823789, 't-sne_y': 10.097087347668424},\n",
       "     {'umap_x': 3.0125985145568848, 'umap_y': 2.6539647579193115}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48600213-C-T',\n",
       "      'distance': 0.0009572833077982068,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415695-C-T',\n",
       "      'distance': 0.0011854578042402864,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596318-C-T',\n",
       "      'distance': 0.001291534979827702,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.001298672636039555,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.0013078830670565367,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.0013243805151432753,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.0014060449320822954,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0014951087068766356,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427602-C-T',\n",
       "      'distance': 0.001554669113829732,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.0015653696609660983,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468456-C-T',\n",
       "      'distance': 0.0015749686863273382,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.0015937576536089182,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-T',\n",
       "      'distance': 0.0016334154643118382,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428390-C-T',\n",
       "      'distance': 0.0016375237610191107,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.0016402166802436113,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496144-C-T',\n",
       "      'distance': 0.0016598731745034456,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492574-C-T',\n",
       "      'distance': 0.0016777788987383246,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.0016853794222697616,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0017075298819690943,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496213-C-T',\n",
       "      'distance': 0.001714655663818121,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48600175-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48600175\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48600175',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.406T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48600175A>T',\n",
       "     'hgvs_protein': 'p.Cys136Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"CM085405\", \"end\": 48600175, \"start\": 48600175}, {\"pubmed\": [17701892], \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"strand\": 1, \"seq_region_name\": \"15\", \"var_synonyms\": {\"UniProt\": [\"VAR_075991\"], \"ClinVar\": [\"RCV000663690\", \"VCV000549206\", \"RCV000663689\", \"VCV000549205\", \"RCV000529377\", \"VCV000457203\", \"RCV000663691\"]}, \"allele_string\": \"A/C/G/T\", \"phenotype_or_disease\": 1, \"start\": 48600175, \"clin_sig_allele\": \"C:pathogenic;T:likely_pathogenic;C:likely_pathogenic;G:likely_pathogenic\", \"end\": 48600175, \"id\": \"rs1555405041\"}], \"start\": 48600175, \"input\": \"NC_000015.10:g.48600175A>T\", \"vcf_string\": \"15-48600175-A-T\", \"end\": 48600175, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"amino_acids\": \"C/S\", \"cadd_raw\": 4.650982, \"gene_symbol_source\": \"EntrezGene\", \"cdna_end\": 722, \"protein_id\": \"NP_000129.3\", \"used_ref\": \"A\", \"sift_prediction\": \"deleterious_low_confidence\", \"cds_start\": 406, \"given_ref\": \"A\", \"alphamissense\": {\"am_pathogenicity\": 0.9983, \"am_class\": \"likely_pathogenic\"}, \"mane_select\": \"ENST00000316623.10\", \"spliceai\": {\"DS_DL\": 0.05, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": -6, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -36, \"DP_DG\": -33, \"DP_AG\": 22}, \"aa\": \"A\", \"consequence_terms\": [\"missense_variant\"], \"canonical\": 1, \"hgvsc\": \"NM_000138.5:c.406T>A\", \"strand\": -1, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"start\": 48600174, \"end\": 48600175, \"type\": \"SimpleInterval\"}}}, \"cdna_start\": 722, \"phaplo\": 0.99897615427687, \"lrt_pred\": \"invalid_field\", \"mutationtaster_pred\": \"A\", \"impact\": \"MODERATE\", \"mastermind_mmid3\": \"FBN1:C136S\", \"variant_allele\": \"T\", \"protein_end\": 136, \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.00228, \"dG_wt\": 110.914, \"dG_mt\": 111.675199, \"ddG\": 0.761199, \"dG_mt_sd\": 0.000447, \"dG_wt_sd\": 0.002549}}, \"transcript_id\": \"NM_000138.5\", \"cadd_phred\": 26.1, \"hgvsp\": \"NP_000129.3:p.Cys136Ser\", \"protein_start\": 136, \"cds_end\": 406, \"conservation\": 6.95, \"gene_id\": \"2200\", \"location\": \"15:48600175\", \"polyphen_prediction\": \"possibly_damaging\", \"clinpred\": 0.999417424201965, \"biotype\": \"protein_coding\", \"sift_score\": 0, \"blosum62\": -1, \"allele_num\": 1, \"codons\": \"Tgc/Agc\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"exon\": \"5/66\", \"polyphen_score\": 0.883, \"mastermind_counts\": \"3|3|3\", \"loeuf\": 0.105}], \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48600175A>T\", \"allele_string\": \"A/T\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.012306324497722343,\n",
       "      'pca_y': 0.11325124820161997},\n",
       "     {'t-sne_x': 9.672517032140915, 't-sne_y': 25.626287266675327},\n",
       "     {'umap_x': -8.202656745910645, 'umap_y': 12.51375961303711}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48600180-C-T',\n",
       "      'distance': 0.005098555702716112,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600147-C-T',\n",
       "      'distance': 0.006603301502764225,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613018-C-T',\n",
       "      'distance': 0.0067393360659480095,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48534131-A-C',\n",
       "      'distance': 0.0076192691922187805,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48600174-C-T',\n",
       "      'distance': 0.007749514654278755,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600174-C-A',\n",
       "      'distance': 0.007941112853586674,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600148-A-G',\n",
       "      'distance': 0.008348101750016212,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48610775-C-T',\n",
       "      'distance': 0.008501454256474972,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537707-C-T',\n",
       "      'distance': 0.008598757907748222,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610820-C-T',\n",
       "      'distance': 0.009034223854541779,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48526207-C-T',\n",
       "      'distance': 0.009460709057748318,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48537647-C-A',\n",
       "      'distance': 0.009597282856702805,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48600226-A-C',\n",
       "      'distance': 0.009848156943917274,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48610800-C-T',\n",
       "      'distance': 0.00985704641789198,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644738-A-C',\n",
       "      'distance': 0.010388724505901337,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613058-A-G',\n",
       "      'distance': 0.010814499109983444,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644691-C-T',\n",
       "      'distance': 0.010827160440385342,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48411064-T-G',\n",
       "      'distance': 0.013920167461037636,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48610736-G-C',\n",
       "      'distance': 0.014121051877737045,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.036938562989234924,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48596297-GG-G',\n",
       "    'metadata': {'vcf_string': '15\\t48596297\\t.\\tGG\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48596297',\n",
       "     'ref_allele': 'GG',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.526del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48596298del',\n",
       "     'hgvs_protein': 'p.Gln176SerfsTer14',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"allele_string\": \"G/-\", \"vcf_string\": \"15-48596297-GG-G\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48596298del\", \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48596298del\", \"minimised\": 1, \"end\": 48596298, \"most_severe_consequence\": \"frameshift_variant\", \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"mane\": [\"MANE_Select\"], \"cdna_start\": 842, \"cdna_end\": 842, \"loeuf\": 0.105, \"conservation\": 6.95, \"protein_end\": 176, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"G\", \"cds_end\": 526, \"protein_start\": 176, \"amino_acids\": \"Q/X\", \"mutfunc\": {\"exp\": {\"dG_wt\": 110.9196, \"ddG_sd\": 0.001732, \"dG_mt_sd\": 0.001516, \"ddG\": 1.272, \"dG_wt_sd\": 0.000547, \"dG_mt\": 112.1916}}, \"cds_start\": 526, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"location\": \"15:48596295\", \"hgvs_offset\": -3, \"hgvsp\": \"NP_000129.3:p.Gln176SerfsTer14\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48596297, \"end\": 48596298, \"type\": \"SimpleInterval\"}}}, \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"given_ref\": \"G\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.526del\", \"canonical\": 1, \"codons\": \"Cag/ag\", \"spliceai\": {\"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -14, \"DS_DL\": 0.01, \"DP_AG\": 48, \"DP_DG\": 48, \"DP_AL\": -5, \"DS_DG\": 0, \"DS_AG\": 0}, \"lof\": \"HC\", \"exon\": \"6/66\", \"aa\": \"G\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"-\", \"lof_info\": \"PERCENTILE:0.0610492107706592,GERP_DIST:8089.97090554396,BP_DIST:8033,DIST_FROM_LAST_EXON:7644,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol_source\": \"EntrezGene\"}], \"start\": 48596298}',\n",
       "    'coordinates': [{'pca_x': 0.003108403008949573,\n",
       "      'pca_y': 0.012991972371389979},\n",
       "     {'t-sne_x': -13.438679680513895, 't-sne_y': -2.247799092556624},\n",
       "     {'umap_x': 1.1178432703018188, 'umap_y': 7.964522361755371}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.08903533965349197,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.09046211838722229,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.09131571650505066,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09136208891868591,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.09188544750213623,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.09227567166090012,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.09246368706226349,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.0929395779967308,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.0934886634349823,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.09354183077812195,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.09361125528812408,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.09363815933465958,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.09378555417060852,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.09385194629430771,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.09389214962720871,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.0939217358827591,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.09393833577632904,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.09395010769367218,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.09408612549304962,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.09411703050136566,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48537761-G-A',\n",
       "    'metadata': {'vcf_string': '15\\t48537761\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48537761',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.586C>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48537761G>A',\n",
       "     'hgvs_protein': 'p.Gln196Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"id\": \"CM043995\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48537761, \"end\": 48537761}], \"id\": \"NC_000015.10:g.48537761G>A\", \"start\": 48537761, \"most_severe_consequence\": \"stop_gained\", \"end\": 48537761, \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"cdna_end\": 902, \"cdna_start\": 902, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:Q196X\", \"phaplo\": 0.99897615427687, \"protein_end\": 196, \"conservation\": 3.79, \"loeuf\": 0.105, \"cds_end\": 586, \"protein_start\": 196, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"G\", \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"cds_start\": 586, \"mutationtaster_pred\": \"A\", \"strand\": -1, \"amino_acids\": \"Q/*\", \"allele_num\": 1, \"consequence_terms\": [\"stop_gained\"], \"cadd_raw\": 9.4206, \"given_ref\": \"G\", \"hgvsp\": \"NP_000129.3:p.Gln196Ter\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48537760, \"end\": 48537761}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"location\": \"15:48537761\", \"exon\": \"7/66\", \"aa\": \"G\", \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_DG\": 47, \"DP_AL\": 47, \"DS_DL\": 0, \"DP_AG\": -14, \"DS_AL\": 0.02, \"SYMBOL\": \"FBN1\", \"DP_DL\": -20}, \"lof\": \"HC\", \"mastermind_counts\": \"3|3|3\", \"cadd_phred\": 41, \"codons\": \"Cag/Tag\", \"canonical\": 1, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.586C>T\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"A\", \"lof_info\": \"PERCENTILE:0.0680129990714949,GERP_DIST:7984.32306484142,BP_DIST:7971,DIST_FROM_LAST_EXON:7582,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\"}], \"strand\": 1, \"allele_string\": \"G/A\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48537761-G-A\", \"input\": \"NC_000015.10:g.48537761G>A\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.026449055596260623,\n",
       "      'pca_y': 0.09420932262655843},\n",
       "     {'t-sne_x': 16.915267404719447, 't-sne_y': 48.26096620505379},\n",
       "     {'umap_x': -9.526056289672852, 'umap_y': 5.625954627990723}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48437376-G-A',\n",
       "      'distance': 0.003316309303045273,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427630-G-A',\n",
       "      'distance': 0.00344554940238595,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425474-G-A',\n",
       "      'distance': 0.003510891692712903,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428361-G-A',\n",
       "      'distance': 0.003520083613693714,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488197-G-A',\n",
       "      'distance': 0.0035318899899721146,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432971-G-A',\n",
       "      'distance': 0.0035566817969083786,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.0036390696186572313,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.003647575154900551,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465679-G-A',\n",
       "      'distance': 0.0036617890000343323,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513579-G-A',\n",
       "      'distance': 0.003699331544339657,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520737-G-A',\n",
       "      'distance': 0.0038386713713407516,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.003908874001353979,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487129-G-A',\n",
       "      'distance': 0.003959439694881439,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448879-G-A',\n",
       "      'distance': 0.003968021832406521,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483899-G-A',\n",
       "      'distance': 0.003971396014094353,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.004146372899413109,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520764-G-A',\n",
       "      'distance': 0.004277007654309273,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.004291504621505737,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420714-G-A',\n",
       "      'distance': 0.004385082051157951,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.0044010104611516,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48537712-GTG-G',\n",
       "    'metadata': {'vcf_string': '15\\t48537712\\t.\\tGTG\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48537712',\n",
       "     'ref_allele': 'GTG',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.635_636del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48537713_48537714del',\n",
       "     'hgvs_protein': 'p.Thr212SerfsTer10',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"frameshift_variant\", \"end\": 48537714, \"transcript_consequences\": [{\"amino_acids\": \"T/X\", \"cds_start\": 635, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"location\": \"15:48537711-48537712\", \"hgvs_offset\": -2, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"interval\": {\"end\": 48537714, \"start\": 48537712, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Thr212SerfsTer10\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"given_ref\": \"TG\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.635_636del\", \"canonical\": 1, \"codons\": \"aCA/a\", \"spliceai\": {\"DS_DG\": 0, \"DS_AG\": 0, \"DP_DG\": 7, \"DP_AL\": -3, \"DS_DL\": 0, \"DP_AG\": 14, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -26}, \"lof\": \"HC\", \"exon\": \"7/66\", \"aa\": \"TG\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"lof_info\": \"PERCENTILE:0.0738161559888579,GERP_DIST:7895.51028102795,BP_DIST:7923,DIST_FROM_LAST_EXON:7534,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"gene_symbol_source\": \"EntrezGene\", \"transcript_id\": \"NM_000138.5\", \"mane\": [\"MANE_Select\"], \"cdna_end\": 952, \"cdna_start\": 951, \"conservation\": 2.915, \"loeuf\": 0.105, \"protein_end\": 212, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"TG\", \"protein_start\": 212, \"cds_end\": 636}], \"start\": 48537713, \"id\": \"NC_000015.10:g.48537713_48537714del\", \"variant_class\": \"deletion\", \"minimised\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48537713_48537714del\", \"vcf_string\": \"15-48537712-GTG-G\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"TG/-\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.046839193225146736,\n",
       "      'pca_y': 0.03059170917701704},\n",
       "     {'t-sne_x': 32.551188535081664, 't-sne_y': 11.238068954788405},\n",
       "     {'umap_x': 1.7524570226669312, 'umap_y': 7.606172561645508}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.09547640383243561,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.09581970423460007,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.09609454870223999,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09611023962497711,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.09625476598739624,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.09682689607143402,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.0971650630235672,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.09754596650600433,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48445382-A-G',\n",
       "      'distance': 0.09758521616458893,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.09759734570980072,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430710-G-A',\n",
       "      'distance': 0.09777811169624329,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.0978538915514946,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.09789780527353287,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437807-A-G',\n",
       "      'distance': 0.09813905507326126,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411243-G-A',\n",
       "      'distance': 0.09820552170276642,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.09820594638586044,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.0982879176735878,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.09844635426998138,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.09861992299556732,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.09862138330936432,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.6},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.6666666666666666},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.75}}},\n",
       "   {'variant_id': '15-48537690-GT-G',\n",
       "    'metadata': {'vcf_string': '15\\t48537690\\t.\\tGT\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48537690',\n",
       "     'ref_allele': 'GT',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.656del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48537691del',\n",
       "     'hgvs_protein': 'p.His219ProfsTer111',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"allele_string\": \"T/-\", \"vcf_string\": \"15-48537690-GT-G\", \"strand\": 1, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48537691del\", \"colocated_variants\": [{\"start\": 48537691, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"end\": 48537691, \"phenotype_or_disease\": 1, \"id\": \"CD021818\"}, {\"end\": 48537691, \"start\": 48537691, \"allele_string\": \"T/-\", \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"rs25390\"}], \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48537691del\", \"minimised\": 1, \"transcript_consequences\": [{\"location\": \"15:48537691\", \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"given_ref\": \"T\", \"hgvsp\": \"NP_000129.3:p.His219ProfsTer111\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48537691, \"start\": 48537690, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}}, \"amino_acids\": \"H/X\", \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"ptriplo\": 0.984699554922396, \"cds_start\": 656, \"strand\": -1, \"variant_allele\": \"-\", \"lof_info\": \"PERCENTILE:0.0761374187558032,GERP_DIST:7864.66383556286,BP_DIST:7901,DIST_FROM_LAST_EXON:7512,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"gene_symbol_source\": \"EntrezGene\", \"mastermind_counts\": \"4|4|5\", \"codons\": \"cAc/cc\", \"canonical\": 1, \"hgvsc\": \"NM_000138.5:c.656del\", \"biotype\": \"protein_coding\", \"exon\": \"7/66\", \"aa\": \"T\", \"spliceai\": {\"DP_AL\": 19, \"DP_DG\": -4, \"DS_DG\": 0, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 21, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": -10}, \"lof\": \"HC\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:H219fs\", \"cdna_start\": 972, \"cdna_end\": 972, \"transcript_id\": \"NM_000138.5\", \"cds_end\": 656, \"protein_start\": 219, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"T\", \"conservation\": 3.79, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"protein_end\": 219}], \"most_severe_consequence\": \"frameshift_variant\", \"end\": 48537691, \"start\": 48537691}',\n",
       "    'coordinates': [{'pca_x': 0.048884761220738815,\n",
       "      'pca_y': 0.03396984453856019},\n",
       "     {'t-sne_x': 8.084888703324918, 't-sne_y': 14.787709230595715},\n",
       "     {'umap_x': 3.8873212337493896, 'umap_y': 3.097744941711426}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.10507744550704956,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10816896706819534,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10869026184082031,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.1089911237359047,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10964097082614899,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441797-G-T',\n",
       "      'distance': 0.11055617779493332,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.1106405258178711,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.11088554561138153,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.11155936866998672,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.1117279976606369,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.11180408298969269,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.11212889850139618,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.11226232349872589,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.11227291822433472,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.1124127209186554,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.11262236535549164,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.1126382127404213,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.11268427222967148,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.11306366324424744,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.11313128471374512,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48534197-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48534197\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48534197',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.745G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48534197C>A',\n",
       "     'hgvs_protein': 'p.Glu249Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"vcf_string\": \"15-48534197-C-A\", \"allele_string\": \"C/A\", \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48534197C>A\", \"seq_region_name\": \"15\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48534197C>A\", \"colocated_variants\": [{\"end\": 48534197, \"start\": 48534197, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"id\": \"CM021590\"}], \"variant_class\": \"SNV\", \"start\": 48534197, \"most_severe_consequence\": \"stop_gained\", \"end\": 48534197, \"transcript_consequences\": [{\"phaplo\": 0.99897615427687, \"protein_end\": 249, \"loeuf\": 0.105, \"conservation\": 3.79, \"cds_end\": 745, \"protein_start\": 249, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"transcript_id\": \"NM_000138.5\", \"cdna_start\": 1061, \"cdna_end\": 1061, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:E249X\", \"exon\": \"8/66\", \"aa\": \"C\", \"spliceai\": {\"DS_AL\": 0.02, \"DP_DL\": 15, \"SYMBOL\": \"FBN1\", \"DP_AG\": -9, \"DS_DL\": 0, \"DP_DG\": -30, \"DP_AL\": 8, \"DS_AG\": 0, \"DS_DG\": 0}, \"lof\": \"HC\", \"mastermind_counts\": \"2|2|2\", \"cadd_phred\": 40, \"codons\": \"Gaa/Taa\", \"canonical\": 1, \"hgvsc\": \"NM_000138.5:c.745G>T\", \"biotype\": \"protein_coding\", \"gene_symbol_source\": \"EntrezGene\", \"variant_allele\": \"A\", \"lof_info\": \"PERCENTILE:0.0864670380687094,GERP_DIST:7767.11523910919,BP_DIST:7813,DIST_FROM_LAST_EXON:7424,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"impact\": \"HIGH\", \"cds_start\": 745, \"mutationtaster_pred\": \"A\", \"strand\": -1, \"amino_acids\": \"E/*\", \"allele_num\": 1, \"consequence_terms\": [\"stop_gained\"], \"given_ref\": \"C\", \"cadd_raw\": 9.144778, \"hgvsp\": \"NP_000129.3:p.Glu249Ter\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48534197, \"start\": 48534196}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"location\": \"15:48534197\"}]}',\n",
       "    'coordinates': [{'pca_x': 0.02047871601473969,\n",
       "      'pca_y': 0.09781407618886925},\n",
       "     {'t-sne_x': 22.234582286375673, 't-sne_y': 41.59960304578235},\n",
       "     {'umap_x': -9.457404136657715, 'umap_y': 4.490847110748291}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48437837-C-A',\n",
       "      'distance': 0.001141608809120953,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.0014039829839020967,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488233-C-A',\n",
       "      'distance': 0.0014638687716796994,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.0017183294985443354,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.0017352711874991655,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.002073400653898716,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411085-C-A',\n",
       "      'distance': 0.002413125243037939,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.002556407358497381,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.0025582602247595787,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48436970-C-A',\n",
       "      'distance': 0.003093153238296509,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515397-C-T',\n",
       "      'distance': 0.0036518520209938288,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48452676-C-A',\n",
       "      'distance': 0.003674524137750268,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.00414192583411932,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.004507717676460743,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483878-C-A',\n",
       "      'distance': 0.00455198809504509,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.00459909625351429,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.004668395034968853,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537615-A-T',\n",
       "      'distance': 0.005003316327929497,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.005263491999357939,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505139-C-A',\n",
       "      'distance': 0.005306357517838478,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48526187-A-AA',\n",
       "    'metadata': {'vcf_string': '15\\t48526187\\t.\\tA\\tAA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48526187',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'AA',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.931dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48526188dup',\n",
       "     'hgvs_protein': 'p.Tyr311LeufsTer37',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"location\": \"15:48526186-48526187\", \"conservation\": 1.085, \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"cds_start\": 931, \"strand\": -1, \"given_ref\": \"-\", \"cds_end\": 932, \"impact\": \"HIGH\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_end\": 1248, \"hgvsc\": \"NM_000138.5:c.931dup\", \"mane_select\": \"ENST00000316623.10\", \"protein_end\": 311, \"gene_symbol\": \"FBN1\", \"hgvs_offset\": -1, \"consequence_terms\": [\"frameshift_variant\"], \"used_ref\": \"-\", \"lof_info\": \"PERCENTILE:0.1080547818013,GERP_DIST:7466.95348069733,BP_DIST:7629,DIST_FROM_LAST_EXON:7240,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"codons\": \"tac/tTac\", \"biotype\": \"protein_coding\", \"variant_allele\": \"A\", \"amino_acids\": \"Y/LX\", \"protein_start\": 311, \"protein_id\": \"NP_000129.3\", \"cdna_start\": 1247, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48526187, \"type\": \"SimpleInterval\", \"start\": 48526187}}}, \"exon\": \"9/66\", \"loeuf\": 0.105, \"canonical\": 1, \"gene_id\": \"2200\", \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"spliceai\": {\"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_AL\": -32, \"DS_AL\": 0, \"DP_AG\": -40, \"DS_DL\": 0, \"DP_DG\": 27, \"DP_DL\": -42, \"DS_AG\": 0}, \"hgvsp\": \"NP_000129.3:p.Tyr311LeufsTer37\"}], \"allele_string\": \"-/A\", \"strand\": 1, \"start\": 48526188, \"most_severe_consequence\": \"frameshift_variant\", \"seq_region_name\": \"15\", \"minimised\": 1, \"input\": \"NC_000015.10:g.48526188dup\", \"variant_class\": \"insertion\", \"vcf_string\": \"15-48526187-A-AA\", \"end\": 48526187, \"id\": \"NC_000015.10:g.48526188dup\", \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': 0.032256972368801096,\n",
       "      'pca_y': 0.012419304257113413},\n",
       "     {'t-sne_x': 10.35314845668756, 't-sne_y': 15.382810358760006},\n",
       "     {'umap_x': 4.15068244934082, 'umap_y': 7.232395648956299}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09618374705314636,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09660264849662781,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.09692750871181488,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.09848083555698395,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.09919101744890213,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520708-C-A',\n",
       "      'distance': 0.09964178502559662,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.09964501857757568,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.09978285431861877,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.09979414194822311,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.09994210302829742,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452595-C-A',\n",
       "      'distance': 0.09997069090604782,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.10007426142692566,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.10019327700138092,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.10022453218698502,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446814-C-T',\n",
       "      'distance': 0.10024328529834747,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.10029508918523788,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-G',\n",
       "      'distance': 0.10032373666763306,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.1003398597240448,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.10037268698215485,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.10041752457618713,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48520742-TT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48520742\\t.\\tTT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48520742',\n",
       "     'ref_allele': 'TT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1066del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48520743del',\n",
       "     'hgvs_protein': 'p.Met356CysfsTer39',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"end\": 48520743, \"id\": \"NC_000015.10:g.48520743del\", \"variant_class\": \"deletion\", \"vcf_string\": \"15-48520742-TT-T\", \"input\": \"NC_000015.10:g.48520743del\", \"most_severe_consequence\": \"frameshift_variant\", \"minimised\": 1, \"seq_region_name\": \"15\", \"start\": 48520743, \"strand\": 1, \"allele_string\": \"T/-\", \"transcript_consequences\": [{\"allele_num\": 1, \"lof_info\": \"PERCENTILE:0.12372330547818,GERP_DIST:7346.03570910851,BP_DIST:7497,DIST_FROM_LAST_EXON:7108,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"biotype\": \"protein_coding\", \"amino_acids\": \"M/X\", \"variant_allele\": \"-\", \"mane\": [\"MANE_Select\"], \"codons\": \"Atg/tg\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48520742, \"end\": 48520743, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"exon\": \"10/66\", \"loeuf\": 0.105, \"canonical\": 1, \"protein_id\": \"NP_000129.3\", \"protein_start\": 356, \"cdna_start\": 1382, \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_DL\": 0, \"DP_AG\": 48, \"DS_AL\": 0, \"DP_AL\": -8, \"DP_DG\": 48, \"DS_AG\": 0, \"DP_DL\": -43}, \"hgvsp\": \"NP_000129.3:p.Met356CysfsTer39\", \"gene_id\": \"2200\", \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"phaplo\": 0.99897615427687, \"location\": \"15:48520740\", \"conservation\": 3.79, \"ptriplo\": 0.984699554922396, \"strand\": -1, \"given_ref\": \"T\", \"cds_end\": 1066, \"cds_start\": 1066, \"mane_select\": \"ENST00000316623.10\", \"protein_end\": 356, \"aa\": \"T\", \"gene_symbol\": \"FBN1\", \"cdna_end\": 1382, \"gene_symbol_source\": \"EntrezGene\", \"impact\": \"HIGH\", \"hgvsc\": \"NM_000138.5:c.1066del\", \"consequence_terms\": [\"frameshift_variant\"], \"used_ref\": \"T\", \"hgvs_offset\": -3}]}',\n",
       "    'coordinates': [{'pca_x': -0.020530896440251134,\n",
       "      'pca_y': 0.029589974297872766},\n",
       "     {'t-sne_x': -13.121106617721319, 't-sne_y': -2.8933459642336383},\n",
       "     {'umap_x': 0.4367993175983429, 'umap_y': 15.048958778381348}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.09540379047393799,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.09544595330953598,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.09555132687091827,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.09665900468826294,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.09676319360733032,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.09799881279468536,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.09871869534254074,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.0989239364862442,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.09903344511985779,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.0990743488073349,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.09922732412815094,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.09942984580993652,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.0994628369808197,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09998267143964767,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.10003039985895157,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.10031140595674515,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.10058288276195526,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.10081294924020767,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.10098789632320404,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.1009930670261383,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48515422-T-TT',\n",
       "    'metadata': {'vcf_string': '15\\t48515422\\t.\\tT\\tTT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48515422',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'TT',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1434dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48515423dup',\n",
       "     'hgvs_protein': 'p.Gly479ArgfsTer12',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"transcript_consequences\": [{\"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"protein_end\": 479, \"hgvsc\": \"NM_000138.5:c.1434dup\", \"cdna_end\": 1751, \"gene_symbol_source\": \"EntrezGene\", \"impact\": \"HIGH\", \"used_ref\": \"-\", \"consequence_terms\": [\"frameshift_variant\"], \"hgvs_offset\": -2, \"phaplo\": 0.99897615427687, \"conservation\": 2.76, \"ptriplo\": 0.984699554922396, \"location\": \"15:48515420-48515421\", \"cds_end\": 1435, \"strand\": -1, \"given_ref\": \"-\", \"cds_start\": 1434, \"canonical\": 1, \"exon\": \"12/66\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"end\": 48515422, \"type\": \"SimpleInterval\", \"start\": 48515422}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"loeuf\": 0.105, \"protein_id\": \"NP_000129.3\", \"protein_start\": 478, \"cdna_start\": 1750, \"spliceai\": {\"DP_DG\": 13, \"DS_AG\": 0, \"DP_DL\": 15, \"DS_DG\": 0.01, \"SYMBOL\": \"FBN1\", \"DP_AG\": 44, \"DS_AL\": 0, \"DP_AL\": 13, \"DS_DL\": 0}, \"hgvsp\": \"NP_000129.3:p.Gly479ArgfsTer12\", \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"gene_id\": \"2200\", \"allele_num\": 1, \"lof_info\": \"PERCENTILE:0.166434540389972,GERP_DIST:7078.01753313104,BP_DIST:7130,DIST_FROM_LAST_EXON:6741,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"biotype\": \"protein_coding\", \"amino_acids\": \"-/X\", \"variant_allele\": \"T\", \"mane\": [\"MANE_Select\"], \"codons\": \"-/A\"}], \"allele_string\": \"-/T\", \"most_severe_consequence\": \"frameshift_variant\", \"input\": \"NC_000015.10:g.48515423dup\", \"minimised\": 1, \"seq_region_name\": \"15\", \"start\": 48515423, \"end\": 48515422, \"id\": \"NC_000015.10:g.48515423dup\", \"vcf_string\": \"15-48515422-T-TT\", \"variant_class\": \"insertion\", \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': -0.05518609277766794,\n",
       "      'pca_y': -0.005558776737921288},\n",
       "     {'t-sne_x': -13.259460660976263, 't-sne_y': -2.6010766599656425},\n",
       "     {'umap_x': 0.4165351986885071, 'umap_y': 15.062503814697266}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.08862406760454178,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.08987100422382355,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.09047368168830872,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.0909925326704979,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.09202957153320312,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.09244565665721893,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.09314462542533875,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.09364378452301025,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.09395132958889008,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.09411901980638504,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.09486109018325806,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.09537011384963989,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.09565874189138412,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.09623008966445923,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.096441850066185,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48408837-T-C',\n",
       "      'distance': 0.0972379595041275,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.09776327759027481,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.09786135703325272,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487199-G-A',\n",
       "      'distance': 0.0979582741856575,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48409762-A-G',\n",
       "      'distance': 0.09827377647161484,\n",
       "      'pathogenicity': 'benign_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8}}},\n",
       "   {'variant_id': '15-48515391-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48515391\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48515391',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1464T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48515391A>T',\n",
       "     'hgvs_protein': 'p.Cys488Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48515391A>T\", \"end\": 48515391, \"variant_class\": \"SNV\", \"vcf_string\": \"15-48515391-A-T\", \"colocated_variants\": [{\"start\": 48515391, \"end\": 48515391, \"id\": \"CM095758\", \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1}], \"assembly_name\": \"GRCh38\", \"strand\": 1, \"allele_string\": \"A/T\", \"transcript_consequences\": [{\"conservation\": -1.54, \"mastermind_counts\": \"3|3|3\", \"ptriplo\": 0.984699554922396, \"location\": \"15:48515391\", \"phaplo\": 0.99897615427687, \"cds_start\": 1464, \"cds_end\": 1464, \"strand\": -1, \"given_ref\": \"A\", \"hgvsc\": \"NM_000138.5:c.1464T>A\", \"impact\": \"HIGH\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_end\": 1780, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"aa\": \"A\", \"protein_end\": 488, \"cadd_raw\": 6.607655, \"mutationtaster_pred\": \"A\", \"used_ref\": \"A\", \"cadd_phred\": 35, \"consequence_terms\": [\"stop_gained\"], \"lof_info\": \"PERCENTILE:0.16991643454039,GERP_DIST:7020.29453290263,BP_DIST:7098,DIST_FROM_LAST_EXON:6709,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"codons\": \"tgT/tgA\", \"biotype\": \"protein_coding\", \"variant_allele\": \"T\", \"amino_acids\": \"C/*\", \"mastermind_mmid3\": \"FBN1:C488X\", \"protein_id\": \"NP_000129.3\", \"protein_start\": 488, \"cdna_start\": 1780, \"lrt_pred\": \"invalid_field\", \"canonical\": 1, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48515391, \"type\": \"SimpleInterval\", \"start\": 48515390}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}}, \"exon\": \"12/66\", \"loeuf\": 0.105, \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"gene_id\": \"2200\", \"spliceai\": {\"DP_DG\": -22, \"DS_AG\": 0, \"DP_DL\": 44, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0.01, \"DS_DL\": 0.02, \"DP_AG\": -22, \"DS_AL\": 0, \"DP_AL\": 44}, \"hgvsp\": \"NP_000129.3:p.Cys488Ter\"}], \"most_severe_consequence\": \"stop_gained\", \"minimised\": 1, \"input\": \"NC_000015.10:g.48515391A>T\", \"seq_region_name\": \"15\", \"start\": 48515391}',\n",
       "    'coordinates': [{'pca_x': 0.02505042106813475,\n",
       "      'pca_y': 0.09272950132675756},\n",
       "     {'t-sne_x': 25.71259047971529, 't-sne_y': 37.59761805225115},\n",
       "     {'umap_x': -9.916093826293945, 'umap_y': 2.5866918563842773}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48537615-A-T',\n",
       "      'distance': 0.0016376596176996827,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.0022780662402510643,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48467954-A-T',\n",
       "      'distance': 0.0023318578023463488,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.0035637617111206055,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.0037393481470644474,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513553-G-T',\n",
       "      'distance': 0.0049436441622674465,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.0050972215831279755,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.005252336151897907,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.005361568182706833,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.005472513847053051,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48467975-C-T',\n",
       "      'distance': 0.0058066509664058685,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437805-C-T',\n",
       "      'distance': 0.005902928300201893,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421995-G-T',\n",
       "      'distance': 0.005929445382207632,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485483-G-T',\n",
       "      'distance': 0.006217215210199356,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411338-C-T',\n",
       "      'distance': 0.0062610795721411705,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515397-C-T',\n",
       "      'distance': 0.006345759145915508,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48420700-C-T',\n",
       "      'distance': 0.006411888170987368,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.006468705367296934,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.006495400331914425,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415610-G-T',\n",
       "      'distance': 0.006512422114610672,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48513642-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48513642\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48513642',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1495T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48513642A>G',\n",
       "     'hgvs_protein': 'p.Cys499Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"end\": 48513642, \"id\": \"CM087769\", \"start\": 48513642, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\"}, {\"end\": 48513642, \"id\": \"CM992320\", \"start\": 48513642, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1}, {\"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"phenotype_or_disease\": 1, \"id\": \"rs2043778680\", \"end\": 48513642, \"clin_sig_allele\": \"T:pathogenic;G:pathogenic;C:likely_pathogenic\", \"start\": 48513642, \"pubmed\": [17701892], \"strand\": 1, \"allele_string\": \"A/C/G/T\", \"var_synonyms\": {\"ClinVar\": [\"RCV001050328\", \"VCV000846906\", \"RCV003783669\", \"VCV002925575\", \"RCV001449963\", \"VCV001120215\"]}, \"seq_region_name\": \"15\"}], \"variant_class\": \"SNV\", \"vcf_string\": \"15-48513642-A-G\", \"id\": \"NC_000015.10:g.48513642A>G\", \"end\": 48513642, \"start\": 48513642, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48513642A>G\", \"minimised\": 1, \"seq_region_name\": \"15\", \"transcript_consequences\": [{\"gene_symbol\": \"FBN1\", \"aa\": \"A\", \"protein_end\": 499, \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -3, \"hgvsc\": \"NM_000138.5:c.1495T>C\", \"gene_symbol_source\": \"EntrezGene\", \"impact\": \"MODERATE\", \"cdna_end\": 1811, \"cadd_phred\": 28.2, \"used_ref\": \"A\", \"consequence_terms\": [\"missense_variant\"], \"mutationtaster_pred\": \"D\", \"sift_prediction\": \"deleterious_low_confidence\", \"cadd_raw\": 5.045633, \"phaplo\": 0.99897615427687, \"mastermind_counts\": \"5|7|7\", \"ptriplo\": 0.984699554922396, \"conservation\": 3.7, \"location\": \"15:48513642\", \"cds_end\": 1495, \"given_ref\": \"A\", \"strand\": -1, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9952}, \"cds_start\": 1495, \"eve_score\": 0.762065338994412, \"canonical\": 1, \"lrt_pred\": \"invalid_field\", \"loeuf\": 0.105, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48513642, \"type\": \"SimpleInterval\", \"start\": 48513641}}}, \"exon\": \"13/66\", \"mastermind_mmid3\": \"FBN1:C499R\", \"polyphen_prediction\": \"probably_damaging\", \"cdna_start\": 1811, \"protein_start\": 499, \"protein_id\": \"NP_000129.3\", \"hgvsp\": \"NP_000129.3:p.Cys499Arg\", \"eve_class\": \"Pathogenic\", \"spliceai\": {\"DP_DG\": -9, \"DP_DL\": -6, \"DS_AG\": 0, \"DS_DG\": 0.01, \"SYMBOL\": \"FBN1\", \"DP_AG\": 26, \"DS_AL\": 0, \"DP_AL\": -48, \"DS_DL\": 0}, \"clinpred\": 0.999857664108276, \"transcript_id\": \"NM_000138.5\", \"gene_id\": \"2200\", \"polyphen_score\": 0.996, \"allele_num\": 1, \"sift_score\": 0, \"variant_allele\": \"G\", \"amino_acids\": \"C/R\", \"biotype\": \"protein_coding\", \"codons\": \"Tgt/Cgt\", \"mane\": [\"MANE_Select\"]}], \"allele_string\": \"A/G\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.03619459379560258,\n",
       "      'pca_y': 0.09677093978999488},\n",
       "     {'t-sne_x': -19.108982315632304, 't-sne_y': 34.109206156002216},\n",
       "     {'umap_x': 2.2457993030548096, 'umap_y': 20.743650436401367}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48513627-A-G',\n",
       "      'distance': 0.0010177779477089643,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513631-A-G',\n",
       "      'distance': 0.0010177779477089643,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48463148-A-G',\n",
       "      'distance': 0.0014611384831368923,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48513600-A-G',\n",
       "      'distance': 0.0019286551978439093,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472644-A-G',\n",
       "      'distance': 0.0022369595244526863,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505136-A-G',\n",
       "      'distance': 0.0022554034367203712,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445454-A-G',\n",
       "      'distance': 0.0023068105801939964,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.002460777759552002,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495216-A-G',\n",
       "      'distance': 0.0026742564514279366,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452637-A-G',\n",
       "      'distance': 0.0027579821180552244,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515434-C-A',\n",
       "      'distance': 0.00276260357350111,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.002818626817315817,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510050-A-G',\n",
       "      'distance': 0.002849270822480321,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456643-A-G',\n",
       "      'distance': 0.002969999797642231,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472557-A-T',\n",
       "      'distance': 0.0036140610463917255,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48494210-A-G',\n",
       "      'distance': 0.003733938792720437,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515465-G-A',\n",
       "      'distance': 0.004100345075130463,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495243-A-T',\n",
       "      'distance': 0.004146147985011339,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48463950-A-C',\n",
       "      'distance': 0.004190836567431688,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465646-A-G',\n",
       "      'distance': 0.004351229406893253,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48513566-G-GT',\n",
       "    'metadata': {'vcf_string': '15\\t48513566\\t.\\tG\\tGT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48513566',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'GT',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1570dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48513567dup',\n",
       "     'hgvs_protein': 'p.Thr524AsnfsTer9',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48513567dup\", \"start\": 48513567, \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000497892\", \"VCV000431982\", \"RCV000539674\", \"RCV000766241\"]}, \"allele_string\": \"T/TT\", \"pubmed\": [17657824], \"end\": 48513567, \"id\": \"rs1555400274\", \"start\": 48513567}], \"minimised\": 1, \"assembly_name\": \"GRCh38\", \"variant_class\": \"insertion\", \"id\": \"NC_000015.10:g.48513567dup\", \"allele_string\": \"-/T\", \"transcript_consequences\": [{\"cdna_end\": 1887, \"protein_id\": \"NP_000129.3\", \"used_ref\": \"-\", \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"amino_acids\": \"T/NX\", \"spliceai\": {\"DP_DL\": -22, \"SYMBOL\": \"FBN1\", \"DP_AG\": 46, \"DP_DG\": -17, \"DS_AG\": 0, \"DP_AL\": 28, \"DS_DL\": 0, \"DS_DG\": 0, \"DS_AL\": 0}, \"given_ref\": \"-\", \"mane_select\": \"ENST00000316623.10\", \"cds_start\": 1570, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48513566, \"end\": 48513566}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"cdna_start\": 1886, \"hgvsc\": \"NM_000138.5:c.1570dup\", \"strand\": -1, \"consequence_terms\": [\"frameshift_variant\"], \"canonical\": 1, \"variant_allele\": \"T\", \"impact\": \"HIGH\", \"mastermind_mmid3\": \"FBN1:T524fs\", \"phaplo\": 0.99897615427687, \"transcript_id\": \"NM_000138.5\", \"protein_end\": 524, \"cds_end\": 1571, \"conservation\": 3.7, \"protein_start\": 524, \"lof\": \"HC\", \"hgvsp\": \"NP_000129.3:p.Thr524AsnfsTer9\", \"biotype\": \"protein_coding\", \"location\": \"15:48513566-48513567\", \"gene_id\": \"2200\", \"exon\": \"13/66\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"mastermind_counts\": \"3|4|6\", \"lof_info\": \"PERCENTILE:0.1822191272052,GERP_DIST:6981.9841912345,BP_DIST:6993,DIST_FROM_LAST_EXON:6604,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"loeuf\": 0.105, \"codons\": \"acg/aAcg\", \"allele_num\": 1}], \"seq_region_name\": \"15\", \"strand\": 1, \"vcf_string\": \"15-48513566-G-GT\", \"end\": 48513566, \"most_severe_consequence\": \"frameshift_variant\"}',\n",
       "    'coordinates': [{'pca_x': 0.005028133984349092,\n",
       "      'pca_y': 0.008728236739863969},\n",
       "     {'t-sne_x': 9.849925797114201, 't-sne_y': 15.42833851769791},\n",
       "     {'umap_x': 3.6191599369049072, 'umap_y': 3.44642972946167}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.10779131948947906,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.10864710807800293,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.10886932164430618,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.10955046117305756,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445408-T-C',\n",
       "      'distance': 0.11064264178276062,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.11182677000761032,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.11192011088132858,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.11209015548229218,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48444585-C-T',\n",
       "      'distance': 0.11258360743522644,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445382-A-G',\n",
       "      'distance': 0.11281482130289078,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.11283731460571289,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446814-C-T',\n",
       "      'distance': 0.11283943057060242,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446811-A-G',\n",
       "      'distance': 0.11315304040908813,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446747-C-T',\n",
       "      'distance': 0.11315597593784332,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.11318806558847427,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.1131976991891861,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445492-C-T',\n",
       "      'distance': 0.11324973404407501,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452567-C-T',\n",
       "      'distance': 0.11344735324382782,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.11347617208957672,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.1134876012802124,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48510158-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48510158\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48510158',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1600T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48510158A>G',\n",
       "     'hgvs_protein': 'p.Cys534Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"end\": 48510158, \"id\": \"CM098646\", \"start\": 48510158, \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\"}], \"start\": 48510158, \"input\": \"NC_000015.10:g.48510158A>G\", \"most_severe_consequence\": \"missense_variant\", \"end\": 48510158, \"vcf_string\": \"15-48510158-A-G\", \"seq_region_name\": \"15\", \"strand\": 1, \"transcript_consequences\": [{\"cds_start\": 1600, \"eve_score\": 0.726107247142163, \"spliceai\": {\"DP_DL\": -9, \"SYMBOL\": \"FBN1\", \"DP_AG\": 41, \"DP_DG\": -12, \"DP_AL\": -9, \"DS_AG\": 0, \"DS_DG\": 0, \"DS_DL\": 0, \"DS_AL\": 0}, \"given_ref\": \"A\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9994}, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol_source\": \"EntrezGene\", \"cadd_raw\": 5.063968, \"mane\": [\"MANE_Select\"], \"amino_acids\": \"C/R\", \"sift_prediction\": \"deleterious_low_confidence\", \"cdna_end\": 1916, \"protein_id\": \"NP_000129.3\", \"used_ref\": \"A\", \"lrt_pred\": \"invalid_field\", \"phaplo\": 0.99897615427687, \"variant_allele\": \"G\", \"impact\": \"MODERATE\", \"mutationtaster_pred\": \"D\", \"mastermind_mmid3\": \"FBN1:C534R\", \"canonical\": 1, \"consequence_terms\": [\"missense_variant\"], \"aa\": \"A\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"start\": 48510157, \"end\": 48510158}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"cdna_start\": 1916, \"hgvsc\": \"NM_000138.5:c.1600T>C\", \"strand\": -1, \"hgvsp\": \"NP_000129.3:p.Cys534Arg\", \"cds_end\": 1600, \"conservation\": 3.95, \"protein_start\": 534, \"protein_end\": 534, \"transcript_id\": \"NM_000138.5\", \"cadd_phred\": 28.3, \"codons\": \"Tgt/Cgt\", \"allele_num\": 1, \"eve_class\": \"Pathogenic\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"exon\": \"14/66\", \"polyphen_score\": 0.996, \"mastermind_counts\": \"3|3|3\", \"loeuf\": 0.105, \"location\": \"15:48510158\", \"polyphen_prediction\": \"probably_damaging\", \"gene_id\": \"2200\", \"clinpred\": 0.999912977218628, \"biotype\": \"protein_coding\", \"sift_score\": 0, \"blosum62\": -3}], \"allele_string\": \"A/G\", \"id\": \"NC_000015.10:g.48510158A>G\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.03714152219627423,\n",
       "      'pca_y': 0.11213725954808107},\n",
       "     {'t-sne_x': -18.87771492251606, 't-sne_y': 34.12700717663631},\n",
       "     {'umap_x': 2.102566957473755, 'umap_y': 20.942073822021484}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48513600-A-G',\n",
       "      'distance': 0.006034303456544876,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510050-A-G',\n",
       "      'distance': 0.00663011334836483,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513627-A-G',\n",
       "      'distance': 0.006839124020189047,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513631-A-G',\n",
       "      'distance': 0.006839124020189047,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48463148-A-G',\n",
       "      'distance': 0.007520486135035753,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445454-A-G',\n",
       "      'distance': 0.008068833500146866,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456643-A-G',\n",
       "      'distance': 0.00809550005942583,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452637-A-G',\n",
       "      'distance': 0.008284181356430054,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463949-C-G',\n",
       "      'distance': 0.00832103006541729,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505136-A-G',\n",
       "      'distance': 0.00832307804375887,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510157-C-T',\n",
       "      'distance': 0.008575264364480972,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515434-C-A',\n",
       "      'distance': 0.008689185604453087,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472644-A-G',\n",
       "      'distance': 0.00914507545530796,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465619-A-T',\n",
       "      'distance': 0.009248441085219383,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48463949-C-A',\n",
       "      'distance': 0.009257003664970398,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.009320052340626717,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505102-C-G',\n",
       "      'distance': 0.009356727823615074,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48463950-A-C',\n",
       "      'distance': 0.009397763758897781,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495216-A-G',\n",
       "      'distance': 0.009415432810783386,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505117-C-A',\n",
       "      'distance': 0.00946750771254301,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48510080-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48510080\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48510080',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1678G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48510080C>T',\n",
       "     'hgvs_protein': 'p.Gly560Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"start\": 48510080, \"end\": 48510080, \"id\": \"CM013917\"}, {\"end\": 48510080, \"id\": \"rs1064794283\", \"start\": 48510080, \"clin_sig_allele\": \"T:pathogenic;A:likely_pathogenic;T:likely_pathogenic;T:uncertain_significance\", \"phenotype_or_disease\": 1, \"pubmed\": [11700157], \"strand\": 1, \"seq_region_name\": \"15\", \"clin_sig\": [\"uncertain_significance\", \"likely_pathogenic\", \"pathogenic\"], \"allele_string\": \"C/A/T\", \"var_synonyms\": {\"ClinVar\": [\"RCV002525829\", \"RCV001377914\", \"VCV001066814\", \"RCV000481478\", \"VCV000420087\", \"RCV000663480\"], \"UniProt\": [\"VAR_017974\"]}}], \"start\": 48510080, \"input\": \"NC_000015.10:g.48510080C>T\", \"most_severe_consequence\": \"missense_variant\", \"end\": 48510080, \"vcf_string\": \"15-48510080-C-T\", \"strand\": 1, \"seq_region_name\": \"15\", \"transcript_consequences\": [{\"cds_start\": 1678, \"mane_select\": \"ENST00000316623.10\", \"given_ref\": \"C\", \"alphamissense\": {\"am_pathogenicity\": 0.9431, \"am_class\": \"likely_pathogenic\"}, \"eve_score\": 0.752463636490434, \"spliceai\": {\"DP_DG\": -36, \"DP_AG\": 29, \"SYMBOL\": \"FBN1\", \"DP_DL\": 13, \"DS_AL\": 0, \"DS_DG\": 0, \"DS_DL\": 0, \"DP_AL\": -19, \"DS_AG\": 0}, \"amino_acids\": \"G/S\", \"mane\": [\"MANE_Select\"], \"cadd_raw\": 4.974075, \"gene_symbol_source\": \"EntrezGene\", \"used_ref\": \"C\", \"protein_id\": \"NP_000129.3\", \"cdna_end\": 1994, \"sift_prediction\": \"deleterious_low_confidence\", \"lrt_pred\": \"invalid_field\", \"phaplo\": 0.99897615427687, \"impact\": \"MODERATE\", \"mastermind_mmid3\": \"FBN1:G560S\", \"mutationtaster_pred\": \"D\", \"variant_allele\": \"T\", \"aa\": \"C\", \"consequence_terms\": [\"missense_variant\"], \"canonical\": 1, \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.1678G>A\", \"cdna_start\": 1994, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48510079, \"end\": 48510080}}}, \"hgvsp\": \"NP_000129.3:p.Gly560Ser\", \"protein_start\": 560, \"conservation\": 3.95, \"cds_end\": 1678, \"protein_end\": 560, \"cadd_phred\": 27.8, \"transcript_id\": \"NM_000138.5\", \"eve_class\": \"Pathogenic\", \"allele_num\": 1, \"codons\": \"Ggc/Agc\", \"polyphen_score\": 0.998, \"mastermind_counts\": \"5|9|9\", \"loeuf\": 0.105, \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"exon\": \"14/66\", \"gene_id\": \"2200\", \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48510080\", \"blosum62\": 0, \"sift_score\": 0, \"biotype\": \"protein_coding\", \"clinpred\": 0.998717784881592}], \"allele_string\": \"C/T\", \"id\": \"NC_000015.10:g.48510080C>T\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.03883624674738012,\n",
       "      'pca_y': 0.11229935024885221},\n",
       "     {'t-sne_x': -10.93478344809602, 't-sne_y': 36.79493694876625},\n",
       "     {'umap_x': 1.1260069608688354, 'umap_y': 21.766916275024414}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48510128-C-T',\n",
       "      'distance': 0.0037179903592914343,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510136-C-T',\n",
       "      'distance': 0.004255939740687609,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510094-C-A',\n",
       "      'distance': 0.00439478037878871,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508665-C-T',\n",
       "      'distance': 0.0046104546636343,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48510088-C-T',\n",
       "      'distance': 0.00539795495569706,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445453-C-T',\n",
       "      'distance': 0.005471634212881327,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-C',\n",
       "      'distance': 0.005543072242289782,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495156-C-T',\n",
       "      'distance': 0.0055669574066996574,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505117-C-A',\n",
       "      'distance': 0.005679006688296795,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510094-C-T',\n",
       "      'distance': 0.00599893182516098,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508653-T-C',\n",
       "      'distance': 0.00607177522033453,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.0061066667549312115,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510049-C-T',\n",
       "      'distance': 0.006163604557514191,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495131-C-T',\n",
       "      'distance': 0.0061774225905537605,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.006187936291098595,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.006188360042870045,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510125-G-A',\n",
       "      'distance': 0.006244204007089138,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515434-C-T',\n",
       "      'distance': 0.006284553557634354,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472601-C-T',\n",
       "      'distance': 0.006295844912528992,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.006365220062434673,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48508675-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48508675\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48508675',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1744T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48508675A>G',\n",
       "     'hgvs_protein': 'p.Cys582Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"minimised\": 1, \"start\": 48508675, \"colocated_variants\": [{\"end\": 48508675, \"id\": \"CM085415\", \"start\": 48508675, \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\"}, {\"end\": 48508675, \"id\": \"rs2141321636\", \"start\": 48508675, \"clin_sig_allele\": \"G:pathogenic\", \"phenotype_or_disease\": 1, \"pubmed\": [17701892], \"strand\": 1, \"seq_region_name\": \"15\", \"clin_sig\": [\"pathogenic\"], \"var_synonyms\": {\"UniProt\": [\"VAR_076014\"], \"ClinVar\": [\"RCV001956154\", \"VCV001458007\"]}, \"allele_string\": \"A/G\"}], \"input\": \"NC_000015.10:g.48508675A>G\", \"vcf_string\": \"15-48508675-A-G\", \"end\": 48508675, \"most_severe_consequence\": \"missense_variant\", \"transcript_consequences\": [{\"sift_prediction\": \"deleterious_low_confidence\", \"cdna_end\": 2060, \"protein_id\": \"NP_000129.3\", \"used_ref\": \"A\", \"gene_symbol_source\": \"EntrezGene\", \"cadd_raw\": 5.198097, \"mane\": [\"MANE_Select\"], \"amino_acids\": \"C/R\", \"spliceai\": {\"DS_AG\": 0, \"DP_AL\": 49, \"DS_DL\": 0, \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": 49, \"SYMBOL\": \"FBN1\", \"DP_AG\": 29, \"DP_DG\": -42}, \"eve_score\": 0.723466968720646, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9995}, \"given_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"cds_start\": 1744, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"type\": \"Allele\", \"location\": {\"interval\": {\"end\": 48508675, \"start\": 48508674, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"cdna_start\": 2060, \"hgvsc\": \"NM_000138.5:c.1744T>C\", \"strand\": -1, \"canonical\": 1, \"consequence_terms\": [\"missense_variant\"], \"aa\": \"A\", \"variant_allele\": \"G\", \"mastermind_mmid3\": \"FBN1:C582R\", \"impact\": \"MODERATE\", \"mutationtaster_pred\": \"D\", \"phaplo\": 0.99897615427687, \"lrt_pred\": \"invalid_field\", \"transcript_id\": \"NM_000138.5\", \"cadd_phred\": 29.2, \"protein_end\": 582, \"cds_end\": 1744, \"conservation\": 3.95, \"protein_start\": 582, \"hgvsp\": \"NP_000129.3:p.Cys582Arg\", \"clinpred\": 0.999876260757446, \"blosum62\": -3, \"sift_score\": 0, \"biotype\": \"protein_coding\", \"location\": \"15:48508675\", \"polyphen_prediction\": \"probably_damaging\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"exon\": \"15/66\", \"mastermind_counts\": \"3|3|3\", \"polyphen_score\": 0.975, \"loeuf\": 0.105, \"codons\": \"Tgc/Cgc\", \"allele_num\": 1, \"eve_class\": \"Pathogenic\"}], \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48508675A>G\", \"allele_string\": \"A/G\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.04125054799231698,\n",
       "      'pca_y': 0.10193748129464889},\n",
       "     {'t-sne_x': -18.687636207658926, 't-sne_y': 33.84818983115958},\n",
       "     {'umap_x': 2.237635850906372, 'umap_y': 20.740140914916992}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48445454-A-G',\n",
       "      'distance': 0.001719277584925294,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48513600-A-G',\n",
       "      'distance': 0.001745011773891747,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495216-A-G',\n",
       "      'distance': 0.0019346928456798196,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510050-A-G',\n",
       "      'distance': 0.0019588603172451258,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513627-A-G',\n",
       "      'distance': 0.0020220752339810133,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513631-A-G',\n",
       "      'distance': 0.0020220752339810133,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48463148-A-G',\n",
       "      'distance': 0.00207396037876606,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.0021525626070797443,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472644-A-G',\n",
       "      'distance': 0.002323705470189452,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.0023799529299139977,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456643-A-G',\n",
       "      'distance': 0.00267346715554595,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494210-A-G',\n",
       "      'distance': 0.002880592131987214,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505136-A-G',\n",
       "      'distance': 0.0034562349319458008,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452637-A-G',\n",
       "      'distance': 0.003566296771168709,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515434-C-A',\n",
       "      'distance': 0.0035850307904183865,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508665-C-T',\n",
       "      'distance': 0.0036301533691585064,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456642-C-A',\n",
       "      'distance': 0.004048503004014492,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505102-C-G',\n",
       "      'distance': 0.004381743725389242,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515465-G-A',\n",
       "      'distance': 0.004500390030443668,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415585-C-A',\n",
       "      'distance': 0.004625244066119194,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48508659-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48508659\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48508659',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1760G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48508659C>T',\n",
       "     'hgvs_protein': 'p.Cys587Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"missense_variant\", \"end\": 48508659, \"vcf_string\": \"15-48508659-C-T\", \"strand\": 1, \"seq_region_name\": \"15\", \"transcript_consequences\": [{\"phaplo\": 0.99897615427687, \"lrt_pred\": \"invalid_field\", \"variant_allele\": \"T\", \"impact\": \"MODERATE\", \"mastermind_mmid3\": \"FBN1:C587Y\", \"mutationtaster_pred\": \"A\", \"canonical\": 1, \"consequence_terms\": [\"missense_variant\"], \"aa\": \"C\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48508659, \"start\": 48508658}}}, \"cdna_start\": 2076, \"hgvsc\": \"NM_000138.5:c.1760G>A\", \"strand\": -1, \"cds_start\": 1760, \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": 29, \"DP_DG\": 3, \"DP_AG\": -12, \"DS_DG\": 0, \"DS_DL\": 0, \"DS_AG\": 0, \"DP_AL\": 23, \"DS_AL\": 0}, \"eve_score\": 0.769489058830674, \"alphamissense\": {\"am_pathogenicity\": 0.9991, \"am_class\": \"likely_pathogenic\"}, \"given_ref\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"cadd_raw\": 5.402239, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"amino_acids\": \"C/Y\", \"sift_prediction\": \"deleterious_low_confidence\", \"cdna_end\": 2076, \"protein_id\": \"NP_000129.3\", \"used_ref\": \"C\", \"codons\": \"tGt/tAt\", \"allele_num\": 1, \"eve_class\": \"Pathogenic\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"exon\": \"15/66\", \"loeuf\": 0.105, \"polyphen_score\": 0.975, \"mastermind_counts\": \"9|17|17\", \"location\": \"15:48508659\", \"polyphen_prediction\": \"probably_damaging\", \"gene_id\": \"2200\", \"clinpred\": 0.999868512153625, \"biotype\": \"protein_coding\", \"sift_score\": 0, \"blosum62\": -2, \"hgvsp\": \"NP_000129.3:p.Cys587Tyr\", \"cds_end\": 1760, \"conservation\": 3.95, \"protein_start\": 587, \"protein_end\": 587, \"transcript_id\": \"NM_000138.5\", \"cadd_phred\": 31}], \"allele_string\": \"C/T\", \"id\": \"NC_000015.10:g.48508659C>T\", \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"minimised\": 1, \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"CM970508\", \"end\": 48508659, \"start\": 48508659}, {\"clin_sig_allele\": \"T:pathogenic\", \"start\": 48508659, \"end\": 48508659, \"id\": \"rs1555399963\", \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000663488\", \"VCV000549040\"], \"UniProt\": [\"VAR_002286\"]}, \"allele_string\": \"C/T\", \"phenotype_or_disease\": 1}], \"start\": 48508659, \"input\": \"NC_000015.10:g.48508659C>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.03128812825279967,\n",
       "      'pca_y': 0.1096461617277717},\n",
       "     {'t-sne_x': -11.70148508559974, 't-sne_y': 32.92873573010263},\n",
       "     {'umap_x': 0.9141893982887268, 'umap_y': 21.971487045288086}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.0014593335799872875,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.0014765106607228518,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492538-C-T',\n",
       "      'distance': 0.0015132758999243379,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515429-A-C',\n",
       "      'distance': 0.0015959481243044138,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.0017091422341763973,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.0017105857841670513,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472556-C-T',\n",
       "      'distance': 0.0017777017783373594,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495131-C-T',\n",
       "      'distance': 0.0019086303655058146,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472601-C-T',\n",
       "      'distance': 0.0019788220524787903,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.001998499035835266,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445453-C-T',\n",
       "      'distance': 0.002037509810179472,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.0020389428827911615,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472628-C-T',\n",
       "      'distance': 0.0020397091284394264,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452654-C-T',\n",
       "      'distance': 0.002091151662170887,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510049-C-T',\n",
       "      'distance': 0.002125300234183669,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.00213355990126729,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513661-A-G',\n",
       "      'distance': 0.00217242818325758,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48508692-C-T',\n",
       "      'distance': 0.002182899508625269,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463949-C-T',\n",
       "      'distance': 0.002294505015015602,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.0023226430639624596,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48508625-G-C',\n",
       "    'metadata': {'vcf_string': '15\\t48508625\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48508625',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1794C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48508625G>C',\n",
       "     'hgvs_protein': 'p.Cys598Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM1612542\", \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"end\": 48508625, \"start\": 48508625}, {\"id\": \"CM983862\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"end\": 48508625, \"start\": 48508625}, {\"end\": 48508625, \"start\": 48508625, \"seq_region_name\": \"15\", \"clin_sig_allele\": \"C:pathogenic\", \"strand\": 1, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000663493\", \"VCV000549045\"], \"UniProt\": [\"VAR_017978\"]}, \"id\": \"rs1555399954\", \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"G/C\"}], \"transcript_consequences\": [{\"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.02, \"amino_acids\": \"C/W\", \"codons\": \"tgC/tgG\", \"mastermind_mmid3\": \"FBN1:C598W\", \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 22, \"SYMBOL\": \"FBN1\", \"DP_DG\": -43, \"DS_AG\": 0, \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": 8, \"DP_AL\": -16}, \"exon\": \"15/66\", \"blosum62\": -2, \"lrt_pred\": \"invalid_field\", \"canonical\": 1, \"mastermind_counts\": \"5|10|10\", \"variant_allele\": \"C\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.999832630157471, \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.1794C>G\", \"sift_prediction\": \"deleterious_low_confidence\", \"cadd_phred\": 26.7, \"polyphen_score\": 0.995, \"eve_class\": \"Pathogenic\", \"protein_end\": 598, \"given_ref\": \"G\", \"sift_score\": 0, \"biotype\": \"protein_coding\", \"cadd_raw\": 4.778127, \"cds_end\": 1794, \"mutationtaster_pred\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"alphamissense\": {\"am_pathogenicity\": 0.9992, \"am_class\": \"likely_pathogenic\"}, \"used_ref\": \"G\", \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 598, \"location\": \"15:48508625\", \"protein_id\": \"NP_000129.3\", \"aa\": \"G\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48508625, \"type\": \"SimpleInterval\", \"start\": 48508624}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}}, \"loeuf\": 0.105, \"consequence_terms\": [\"missense_variant\"], \"cds_start\": 1794, \"eve_score\": 0.671568960233001, \"impact\": \"MODERATE\", \"cdna_end\": 2110, \"cdna_start\": 2110, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys598Trp\"}], \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48508625G>C\", \"vcf_string\": \"15-48508625-G-C\", \"allele_string\": \"G/C\", \"assembly_name\": \"GRCh38\", \"start\": 48508625, \"minimised\": 1, \"end\": 48508625, \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48508625G>C\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.03258852415781116,\n",
       "      'pca_y': 0.1063278967309855},\n",
       "     {'t-sne_x': -15.331818336494239, 't-sne_y': 28.248789050236997},\n",
       "     {'umap_x': 1.8358662128448486, 'umap_y': 21.8528995513916}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48503845-G-C',\n",
       "      'distance': 0.0017978688701987267,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425412-G-C',\n",
       "      'distance': 0.0024504491593688726,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515428-C-T',\n",
       "      'distance': 0.002458547241985798,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505102-C-G',\n",
       "      'distance': 0.00347405974753201,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505101-G-C',\n",
       "      'distance': 0.003868989646434784,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-G',\n",
       "      'distance': 0.004069490358233452,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463949-C-G',\n",
       "      'distance': 0.004086337052285671,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508653-T-C',\n",
       "      'distance': 0.004783314652740955,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437026-T-C',\n",
       "      'distance': 0.004934521391987801,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474345-A-C',\n",
       "      'distance': 0.005257526878267527,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463950-A-C',\n",
       "      'distance': 0.005395997781306505,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472556-C-T',\n",
       "      'distance': 0.005424722097814083,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495512-A-C',\n",
       "      'distance': 0.0054524922743439674,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.005502804182469845,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463148-A-G',\n",
       "      'distance': 0.0056058792397379875,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.005663128569722176,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513627-A-G',\n",
       "      'distance': 0.005688452161848545,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513631-A-G',\n",
       "      'distance': 0.005688452161848545,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.005718397442251444,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505117-C-A',\n",
       "      'distance': 0.005725073628127575,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48508584-TT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48508584\\t.\\tTT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48508584',\n",
       "     'ref_allele': 'TT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1836del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48508585del',\n",
       "     'hgvs_protein': 'p.Asp613ThrfsTer12',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"used_ref\": \"T\", \"lof_info\": \"PERCENTILE:0.213091922005571,GERP_DIST:6750.5258361415,BP_DIST:6731,DIST_FROM_LAST_EXON:6342,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"given_ref\": \"T\", \"cds_end\": 1836, \"biotype\": \"protein_coding\", \"cds_start\": 1836, \"impact\": \"HIGH\", \"cdna_start\": 2152, \"cdna_end\": 2152, \"hgvsp\": \"NP_000129.3:p.Asp613ThrfsTer12\", \"gene_id\": \"2200\", \"protein_id\": \"NP_000129.3\", \"aa\": \"T\", \"location\": \"15:48508583\", \"protein_start\": 612, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48508584, \"type\": \"SimpleInterval\", \"end\": 48508585}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}}, \"consequence_terms\": [\"frameshift_variant\", \"splice_region_variant\"], \"exon\": \"15/66\", \"spliceai\": {\"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": -2, \"DP_AL\": 15, \"DS_DL\": 0, \"DP_AG\": 25, \"SYMBOL\": \"FBN1\", \"DP_DG\": 4, \"DS_AG\": 0}, \"canonical\": 1, \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"-\", \"hgvs_offset\": -2, \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.1836del\", \"codons\": \"aaA/aa\", \"amino_acids\": \"K/X\", \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"protein_end\": 612, \"lof\": \"HC\"}], \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48508585del\", \"vcf_string\": \"15-48508584-TT-T\", \"allele_string\": \"T/-\", \"start\": 48508585, \"assembly_name\": \"GRCh38\", \"end\": 48508585, \"variant_class\": \"deletion\", \"minimised\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48508585del\"}',\n",
       "    'coordinates': [{'pca_x': -0.03095552598067762,\n",
       "      'pca_y': 0.017088665865871287},\n",
       "     {'t-sne_x': -12.492051382032805, 't-sne_y': -3.222339777820653},\n",
       "     {'umap_x': 0.38395166397094727, 'umap_y': 15.077980041503906}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.08185367286205292,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.08205966651439667,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.08274348080158234,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.08325006067752838,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.08345147222280502,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.08407740294933319,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.08410079032182693,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.08461064845323563,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.08494921028614044,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.08576257526874542,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.08604425191879272,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.08651763200759888,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.08661890029907227,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.08708538860082626,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.0871686115860939,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.08719958364963531,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.08749036490917206,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.08824272453784943,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48610825-G-T',\n",
       "      'distance': 0.0886693149805069,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.08904679119586945,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48505076-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48505076\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48505076',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1909T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48505076A>G',\n",
       "     'hgvs_protein': 'p.Cys637Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"end\": 48505076, \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48505076A>G\", \"seq_region_name\": \"15\", \"allele_string\": \"A/G\", \"assembly_name\": \"GRCh38\", \"start\": 48505076, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48505076A>G\", \"strand\": 1, \"vcf_string\": \"15-48505076-A-G\", \"transcript_consequences\": [{\"impact\": \"MODERATE\", \"cds_start\": 1909, \"eve_score\": 0.657154115077657, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys637Arg\", \"cdna_end\": 2225, \"cdna_start\": 2225, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48505075, \"type\": \"SimpleInterval\", \"end\": 48505076}}}, \"loeuf\": 0.105, \"protein_start\": 637, \"polyphen_prediction\": \"probably_damaging\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48505076\", \"aa\": \"A\", \"consequence_terms\": [\"missense_variant\"], \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"used_ref\": \"A\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9986}, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.377331, \"cds_end\": 1909, \"sift_score\": 0, \"given_ref\": \"A\", \"mutationtaster_pred\": \"A\", \"cadd_phred\": 31, \"polyphen_score\": 0.987, \"eve_class\": \"Pathogenic\", \"protein_end\": 637, \"sift_prediction\": \"deleterious_low_confidence\", \"canonical\": 1, \"spliceai\": {\"DP_DL\": 0, \"DP_AL\": 1, \"DS_DG\": 0.01, \"DS_AL\": 0, \"DS_AG\": 0, \"DP_DG\": -47, \"SYMBOL\": \"FBN1\", \"DS_DL\": 0, \"DP_AG\": -23}, \"exon\": \"16/66\", \"lrt_pred\": \"invalid_field\", \"blosum62\": -3, \"hgvsc\": \"NM_000138.5:c.1909T>C\", \"mastermind_counts\": \"11|12|12\", \"strand\": -1, \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"G\", \"clinpred\": 0.999855399131775, \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"codons\": \"Tgt/Cgt\", \"amino_acids\": \"C/R\", \"mastermind_mmid3\": \"FBN1:C637R\", \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"]}], \"colocated_variants\": [{\"start\": 48505076, \"end\": 48505076, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM992322\"}, {\"start\": 48505076, \"end\": 48505076, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"G:likely_pathogenic\", \"strand\": 1, \"clin_sig\": [\"likely_pathogenic\"], \"allele_string\": \"A/G\", \"var_synonyms\": {\"ClinVar\": [\"RCV000208087\", \"VCV000222601\"]}, \"id\": \"rs869025404\", \"pubmed\": [10486319, 21895641]}]}',\n",
       "    'coordinates': [{'pca_x': 0.051449642064739065,\n",
       "      'pca_y': 0.07538234430335364},\n",
       "     {'t-sne_x': 38.511612232391435, 't-sne_y': 4.721703376483296},\n",
       "     {'umap_x': 1.0038994550704956, 'umap_y': 6.641388416290283}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48446811-A-G',\n",
       "      'distance': 0.0013565612025558949,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452673-A-G',\n",
       "      'distance': 0.0017477520741522312,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.001867213286459446,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0018894280074164271,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.0018925120821222663,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.0019638994708657265,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497311-A-G',\n",
       "      'distance': 0.0021034819073975086,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.002115905750542879,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.002117325086146593,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.002138267271220684,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.002156514674425125,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.002176210517063737,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0021864366717636585,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.002198441419750452,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.0022030863910913467,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.002224667463451624,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.002237797249108553,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448813-A-G',\n",
       "      'distance': 0.002246252726763487,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456688-A-G',\n",
       "      'distance': 0.002302885986864567,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.0024101249873638153,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48503929-CAT-C',\n",
       "    'metadata': {'vcf_string': '15\\t48503929\\t.\\tCAT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48503929',\n",
       "     'ref_allele': 'CAT',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.1969_1970del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48503930_48503931del',\n",
       "     'hgvs_protein': 'p.Met657AlafsTer23',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"AT/-\", \"start\": 48503930, \"assembly_name\": \"GRCh38\", \"variant_class\": \"deletion\", \"end\": 48503931, \"minimised\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48503930_48503931del\", \"transcript_consequences\": [{\"cdna_start\": 2285, \"cdna_end\": 2286, \"hgvsp\": \"NP_000129.3:p.Met657AlafsTer23\", \"gene_id\": \"2200\", \"cds_start\": 1969, \"impact\": \"HIGH\", \"consequence_terms\": [\"frameshift_variant\"], \"location\": \"15:48503930-48503931\", \"protein_id\": \"NP_000129.3\", \"aa\": \"AT\", \"protein_start\": 657, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48503929, \"end\": 48503931}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}}, \"used_ref\": \"AT\", \"lof_info\": \"PERCENTILE:0.228644382544104,GERP_DIST:6661.1612442092,BP_DIST:6597,DIST_FROM_LAST_EXON:6208,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"AT\", \"cds_end\": 1970, \"biotype\": \"protein_coding\", \"protein_end\": 657, \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"-\", \"strand\": -1, \"mastermind_counts\": \"2|2|4\", \"hgvsc\": \"NM_000138.5:c.1969_1970del\", \"exon\": \"17/66\", \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DG\": 10, \"DS_AG\": 0, \"DP_AG\": 10, \"DS_DL\": 0, \"DP_AL\": 45, \"DP_DL\": 47, \"DS_AL\": 0, \"DS_DG\": 0}, \"canonical\": 1, \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"codons\": \"ATg/g\", \"amino_acids\": \"M/X\", \"mastermind_mmid3\": \"FBN1:M657fs\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95}], \"colocated_variants\": [{\"allele_string\": \"AT/-\", \"clin_sig\": [\"pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV002246092\", \"VCV001325480\"]}, \"id\": \"rs2141315944\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48503930, \"end\": 48503931}], \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48503930_48503931del\", \"vcf_string\": \"15-48503929-CAT-C\"}',\n",
       "    'coordinates': [{'pca_x': 0.059292696077739515,\n",
       "      'pca_y': 0.028286953207952},\n",
       "     {'t-sne_x': 14.482352467292689, 't-sne_y': 3.676542863751652},\n",
       "     {'umap_x': 3.574568748474121, 'umap_y': 3.668790340423584}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.10446904599666595,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.10742904245853424,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.1111757680773735,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.11140286177396774,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445419-G-T',\n",
       "      'distance': 0.11281931400299072,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.11282475292682648,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.11299041658639908,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.11316036432981491,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.11378955096006393,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445430-G-A',\n",
       "      'distance': 0.11430883407592773,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.11445663124322891,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.11447804421186447,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.11471423506736755,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.11492423713207245,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.11515350639820099,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.11525320261716843,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503905-G-C',\n",
       "      'distance': 0.11530580371618271,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.11538296937942505,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.11541622877120972,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427646-A-T',\n",
       "      'distance': 0.11554712057113647,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48503843-G-T',\n",
       "    'metadata': {'vcf_string': '15\\t48503843\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48503843',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2057C>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48503843G>T',\n",
       "     'hgvs_protein': 'p.Ala686Asp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48503843, \"assembly_name\": \"GRCh38\", \"allele_string\": \"G/T\", \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48503843G>T\", \"end\": 48503843, \"variant_class\": \"SNV\", \"minimised\": 1, \"colocated_variants\": [{\"start\": 48503843, \"end\": 48503843, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM098656\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\"}, {\"seq_region_name\": \"15\", \"id\": \"rs193922186\", \"var_synonyms\": {\"ClinVar\": [\"RCV004821965\", \"RCV000029705\", \"VCV000036045\"]}, \"clin_sig\": [\"uncertain_significance\", \"likely_pathogenic\"], \"allele_string\": \"G/A/T\", \"clin_sig_allele\": \"T:likely_pathogenic;T:uncertain_significance\", \"strand\": 1, \"frequencies\": {\"T\": {\"gnomadg_eas\": 0, \"gnomadg_nfe\": 1.47e-05, \"gnomadg\": 6.57e-06, \"gnomadg_sas\": 0, \"gnomadg_amr\": 0, \"gnomadg_afr\": 0, \"gnomadg_ami\": 0, \"gnomadg_asj\": 0, \"gnomadg_fin\": 0, \"gnomadg_mid\": 0, \"gnomadg_remaining\": 0}}, \"phenotype_or_disease\": 1, \"end\": 48503843, \"start\": 48503843}], \"transcript_consequences\": [{\"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 686, \"cadd_phred\": 28.6, \"polyphen_score\": 0.879, \"eve_class\": \"Pathogenic\", \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"mastermind_mmid3\": \"FBN1:A686D\", \"codons\": \"gCc/gAc\", \"amino_acids\": \"A/D\", \"hgvsc\": \"NM_000138.5:c.2057C>A\", \"mastermind_counts\": \"7|8|8\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"T\", \"clinpred\": 0.978694397551989, \"strand\": -1, \"canonical\": 1, \"spliceai\": {\"DS_AL\": 0, \"DS_DG\": 0, \"DP_AL\": -11, \"DP_DL\": 49, \"DP_AG\": -4, \"DS_DL\": 0, \"DP_DG\": -9, \"DS_AG\": 0.02, \"SYMBOL\": \"FBN1\"}, \"blosum62\": -2, \"lrt_pred\": \"invalid_field\", \"exon\": \"17/66\", \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48503842, \"end\": 48503843}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"loeuf\": 0.105, \"protein_start\": 686, \"polyphen_prediction\": \"possibly_damaging\", \"aa\": \"G\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48503843\", \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Ala686Asp\", \"cdna_end\": 2373, \"cdna_start\": 2373, \"impact\": \"MODERATE\", \"cds_start\": 2057, \"eve_score\": 0.671821305618943, \"mutationtaster_pred\": \"A\", \"biotype\": \"protein_coding\", \"cadd_raw\": 5.118564, \"cds_end\": 2057, \"given_ref\": \"G\", \"sift_score\": 0, \"used_ref\": \"G\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.7472}, \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\"}], \"vcf_string\": \"15-48503843-G-T\", \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48503843G>T\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.034528582895626686,\n",
       "      'pca_y': 0.1049957037660679},\n",
       "     {'t-sne_x': -11.412935024784561, 't-sne_y': 30.568209499199114},\n",
       "     {'umap_x': 1.3599332571029663, 'umap_y': 22.23385238647461}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48481690-C-T',\n",
       "      'distance': 0.003594084642827511,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437038-C-T',\n",
       "      'distance': 0.0037116981111466885,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.004172525368630886,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508665-C-T',\n",
       "      'distance': 0.004406890831887722,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415566-C-T',\n",
       "      'distance': 0.004437071271240711,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495156-C-T',\n",
       "      'distance': 0.004751705564558506,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494210-A-G',\n",
       "      'distance': 0.004754412919282913,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.004759593401104212,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48441765-C-T',\n",
       "      'distance': 0.00486400630325079,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515434-C-A',\n",
       "      'distance': 0.004928469657897949,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495216-A-G',\n",
       "      'distance': 0.004999779164791107,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.0051975492388010025,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437027-T-G',\n",
       "      'distance': 0.005292949266731739,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.005325060337781906,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483935-C-T',\n",
       "      'distance': 0.005396267399191856,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.005486174952238798,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421990-C-T',\n",
       "      'distance': 0.005496441386640072,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415585-C-A',\n",
       "      'distance': 0.005527520552277565,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.005535946227610111,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472644-A-G',\n",
       "      'distance': 0.005622870288789272,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48499039-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48499039\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48499039',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2114-1G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48499039C>G',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48499039-C-G\", \"input\": \"NC_000015.10:g.48499039C>G\", \"most_severe_consequence\": \"splice_acceptor_variant\", \"strand\": 1, \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"id\": \"CS013966\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48499039, \"end\": 48499039}, {\"end\": 48499039, \"start\": 48499039, \"strand\": 1, \"clin_sig_allele\": \"G:pathogenic\", \"phenotype_or_disease\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000663517\", \"VCV000549064\"]}, \"id\": \"rs1555399484\", \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"C/G\", \"seq_region_name\": \"15\"}], \"transcript_consequences\": [{\"canonical\": 1, \"spliceai\": {\"DP_DL\": -48, \"DP_AL\": -1, \"DS_DG\": 0, \"DS_AL\": 0.84, \"DP_DG\": -12, \"DS_AG\": 0.01, \"SYMBOL\": \"FBN1\", \"DS_DL\": 0, \"DP_AG\": -12}, \"hgvsc\": \"NM_000138.5:c.2114-1G>C\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"G\", \"strand\": -1, \"mastermind_counts\": \"3|3|10\", \"mastermind_mmid3\": \"FBN1:A705sa\", \"conservation\": 3.45, \"gene_symbol_source\": \"EntrezGene\", \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"cadd_phred\": 32, \"intron\": \"17/65\", \"lof\": \"HC\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"INTRON_SIZE:4748\", \"used_ref\": \"C\", \"biotype\": \"protein_coding\", \"cadd_raw\": 5.48896, \"ada_score\": 0.999983677848485, \"given_ref\": \"C\", \"maxentscan_alt\": -0.22, \"maxentscan_ref\": 7.844, \"impact\": \"HIGH\", \"rf_score\": 0.942, \"maxentscan_diff\": 8.064, \"gene_id\": \"2200\", \"loeuf\": 0.105, \"genesplicer\": \"loss/acceptor/48499038-48499039/Medium/6.533947\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"start\": 48499038, \"end\": 48499039}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}}, \"protein_id\": \"NP_000129.3\", \"aa\": \"C\", \"location\": \"15:48499039\", \"consequence_terms\": [\"splice_acceptor_variant\"]}], \"id\": \"NC_000015.10:g.48499039C>G\", \"seq_region_name\": \"15\", \"minimised\": 1, \"end\": 48499039, \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"start\": 48499039, \"allele_string\": \"C/G\"}',\n",
       "    'coordinates': [{'pca_x': -0.23385376051596596,\n",
       "      'pca_y': -0.04216777042043955},\n",
       "     {'t-sne_x': -29.47177033045866, 't-sne_y': -17.920077234030142},\n",
       "     {'umap_x': 11.782332420349121, 'umap_y': 18.812036514282227}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.0024467059411108494,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.0028533930890262127,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.003138248808681965,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446823-C-G',\n",
       "      'distance': 0.003355537075549364,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.004023712128400803,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490079-C-G',\n",
       "      'distance': 0.004054100252687931,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.004369887989014387,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.004372862633317709,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.004409633111208677,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.004442651756107807,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.0045507485046982765,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.0047589498572051525,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.0048508006148040295,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.004939459264278412,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.004996202886104584,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.005075238179415464,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.005131984129548073,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.005782777443528175,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.0059781246818602085,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48452686-T-C',\n",
       "      'distance': 0.006404911633580923,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48499020-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48499020\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48499020',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2132G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48499020C>T',\n",
       "     'hgvs_protein': 'p.Cys711Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48499020C>T\", \"end\": 48499020, \"variant_class\": \"SNV\", \"minimised\": 1, \"start\": 48499020, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\", \"vcf_string\": \"15-48499020-C-T\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48499020C>T\", \"colocated_variants\": [{\"id\": \"CM960627\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"end\": 48499020, \"start\": 48499020}], \"transcript_consequences\": [{\"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.45, \"mastermind_mmid3\": \"FBN1:C711Y\", \"codons\": \"tGc/tAc\", \"amino_acids\": \"C/Y\", \"hgvsc\": \"NM_000138.5:c.2132G>A\", \"mastermind_counts\": \"8|12|12\", \"clinpred\": 0.997156262397766, \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"T\", \"strand\": -1, \"canonical\": 1, \"spliceai\": {\"DP_AL\": 7, \"DP_DL\": -35, \"DS_AL\": 0, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DG\": -32, \"DS_AG\": 0.01, \"DP_AG\": 18, \"DS_DL\": 0}, \"blosum62\": -2, \"lrt_pred\": \"invalid_field\", \"exon\": \"18/66\", \"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 711, \"cadd_phred\": 28.2, \"polyphen_score\": 0.987, \"eve_class\": \"Pathogenic\", \"mutationtaster_pred\": \"D\", \"biotype\": \"protein_coding\", \"cadd_raw\": 5.05114, \"cds_end\": 2132, \"sift_score\": 0, \"given_ref\": \"C\", \"used_ref\": \"C\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9981}, \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48499020, \"type\": \"SimpleInterval\", \"start\": 48499019}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"loeuf\": 0.105, \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 711, \"location\": \"15:48499020\", \"protein_id\": \"NP_000129.3\", \"aa\": \"C\", \"hgvsp\": \"NP_000129.3:p.Cys711Tyr\", \"gene_id\": \"2200\", \"cdna_end\": 2448, \"cdna_start\": 2448, \"impact\": \"MODERATE\", \"cds_start\": 2132, \"eve_score\": 0.784967140696906}]}',\n",
       "    'coordinates': [{'pca_x': 0.037346373333747185,\n",
       "      'pca_y': 0.08433867265357228},\n",
       "     {'t-sne_x': 5.5055827981666425, 't-sne_y': -7.918505340426879},\n",
       "     {'umap_x': 5.150539875030518, 'umap_y': 4.947110652923584}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48421653-C-T',\n",
       "      'distance': 0.0012867050245404243,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503846-C-T',\n",
       "      'distance': 0.0015004075830802321,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437032-C-T',\n",
       "      'distance': 0.0016824137419462204,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425419-C-T',\n",
       "      'distance': 0.002334209391847253,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503849-C-T',\n",
       "      'distance': 0.002422978635877371,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483931-C-T',\n",
       "      'distance': 0.0024491813965141773,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425816-C-T',\n",
       "      'distance': 0.002482140436768532,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463943-C-T',\n",
       "      'distance': 0.00248327711597085,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432910-C-T',\n",
       "      'distance': 0.0025789933279156685,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432916-C-T',\n",
       "      'distance': 0.0026074526831507683,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474533-C-T',\n",
       "      'distance': 0.0026092324405908585,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474299-C-T',\n",
       "      'distance': 0.0027519240975379944,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472595-C-T',\n",
       "      'distance': 0.00275383610278368,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48464009-C-T',\n",
       "      'distance': 0.002795134438201785,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415701-T-C',\n",
       "      'distance': 0.0028206214774399996,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425374-C-T',\n",
       "      'distance': 0.002907545305788517,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474638-C-T',\n",
       "      'distance': 0.0029937676154077053,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437369-C-T',\n",
       "      'distance': 0.00299726240336895,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425479-C-T',\n",
       "      'distance': 0.0030261175706982613,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474572-C-T',\n",
       "      'distance': 0.0030281818471848965,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48498985-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48498985\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48498985',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2167G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48498985C>T',\n",
       "     'hgvs_protein': 'p.Asp723Asn',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"start\": 48498985, \"end\": 48498985, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CS106073\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\"}], \"transcript_consequences\": [{\"maxentscan_alt\": 3.378, \"maxentscan_ref\": 8.366, \"mutationtaster_pred\": \"D\", \"cds_end\": 2167, \"cadd_raw\": 6.624876, \"biotype\": \"protein_coding\", \"ada_score\": 0.999986947114126, \"sift_score\": 0.02, \"given_ref\": \"C\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.8107}, \"used_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48498984, \"end\": 48498985}}, \"type\": \"Allele\"}, \"location\": \"15:48498985\", \"protein_id\": \"NP_000129.3\", \"aa\": \"C\", \"protein_start\": 723, \"polyphen_prediction\": \"probably_damaging\", \"hgvsp\": \"NP_000129.3:p.Asp723Asn\", \"gene_id\": \"2200\", \"cdna_start\": 2483, \"cdna_end\": 2483, \"impact\": \"MODERATE\", \"maxentscan_diff\": 4.988, \"rf_score\": 0.996, \"cds_start\": 2167, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mastermind_mmid3\": \"FBN1:D723N,FBN1:D723srd\", \"amino_acids\": \"D/N\", \"codons\": \"Gat/Aat\", \"conservation\": 3.45, \"gene_symbol_source\": \"EntrezGene\", \"hgvsc\": \"NM_000138.5:c.2167G>A\", \"variant_allele\": \"T\", \"strand\": -1, \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.983883559703827, \"mastermind_counts\": \"3|15|17\", \"canonical\": 1, \"lrt_pred\": \"invalid_field\", \"exon\": \"18/66\", \"blosum62\": 1, \"spliceai\": {\"DP_AG\": -32, \"DS_DL\": 0.77, \"DP_DG\": -32, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_DG\": 0, \"DP_AL\": 42, \"DP_DL\": 0}, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 723, \"polyphen_score\": 0.996, \"cadd_phred\": 35}], \"vcf_string\": \"15-48498985-C-T\", \"input\": \"NC_000015.10:g.48498985C>T\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"start\": 48498985, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\", \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48498985C>T\", \"variant_class\": \"SNV\", \"end\": 48498985, \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.0392158307621635,\n",
       "      'pca_y': 0.04937164614375634},\n",
       "     {'t-sne_x': -13.097488194571856, 't-sne_y': -3.0238455321534863},\n",
       "     {'umap_x': 0.43315163254737854, 'umap_y': 15.235814094543457}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.0015501243760809302,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.0018668262055143714,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.002005078364163637,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.0024419878609478474,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.0027956964913755655,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.002796670887619257,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.0028095983434468508,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.0029908709693700075,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.003165939822793007,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.0032249740324914455,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.0032296222634613514,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.0033430620096623898,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.0034411405213177204,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.0036036926321685314,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.0038828407414257526,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.004469919949769974,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.0058355387300252914,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.005907760001718998,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.005990367382764816,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.006421827711164951,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48497391-T-A',\n",
       "    'metadata': {'vcf_string': '15\\t48497391\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48497391',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2168A>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48497391T>A',\n",
       "     'hgvs_protein': 'p.Asp723Val',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"variant_class\": \"SNV\", \"end\": 48497391, \"id\": \"NC_000015.10:g.48497391T>A\", \"seq_region_name\": \"15\", \"allele_string\": \"T/A\", \"assembly_name\": \"GRCh38\", \"start\": 48497391, \"input\": \"NC_000015.10:g.48497391T>A\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"vcf_string\": \"15-48497391-T-A\", \"transcript_consequences\": [{\"aa\": \"T\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48497391\", \"protein_start\": 723, \"polyphen_prediction\": \"probably_damaging\", \"loeuf\": 0.105, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48497391, \"type\": \"SimpleInterval\", \"start\": 48497390}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}}, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"cds_start\": 2168, \"impact\": \"MODERATE\", \"rf_score\": 0.566, \"maxentscan_diff\": 1.604, \"cdna_start\": 2484, \"cdna_end\": 2484, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Asp723Val\", \"ada_score\": 0.833717545201685, \"given_ref\": \"T\", \"sift_score\": 0, \"cds_end\": 2168, \"cadd_raw\": 5.580161, \"biotype\": \"protein_coding\", \"maxentscan_ref\": 9.021, \"mutationtaster_pred\": \"D\", \"maxentscan_alt\": 7.417, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"used_ref\": \"T\", \"alphamissense\": {\"am_pathogenicity\": 0.9807, \"am_class\": \"likely_pathogenic\"}, \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.998, \"cadd_phred\": 32, \"protein_end\": 723, \"codons\": \"gAt/gTt\", \"amino_acids\": \"D/V\", \"mastermind_mmid3\": \"FBN1:D723V,FBN1:D723sra\", \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"lrt_pred\": \"invalid_field\", \"blosum62\": -3, \"exon\": \"19/66\", \"spliceai\": {\"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": -11, \"DP_AL\": 0, \"DS_DL\": 0, \"DP_AG\": -21, \"DP_DG\": 0, \"DS_AG\": 0.01, \"SYMBOL\": \"FBN1\"}, \"canonical\": 1, \"clinpred\": 0.9979567527771, \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"A\", \"strand\": -1, \"mastermind_counts\": \"6|14|27\", \"hgvsc\": \"NM_000138.5:c.2168A>T\"}], \"colocated_variants\": [{\"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM021999\", \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"end\": 48497391, \"start\": 48497391}, {\"end\": 48497391, \"start\": 48497391, \"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM930243\", \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\"}]}',\n",
       "    'coordinates': [{'pca_x': -0.03426869776695484,\n",
       "      'pca_y': 0.04955080414217344},\n",
       "     {'t-sne_x': -13.435772558496685, 't-sne_y': -3.00176960081951},\n",
       "     {'umap_x': 0.36906689405441284, 'umap_y': 15.069920539855957}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.005071509163826704,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.005377213936299086,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.005540955811738968,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.005687860772013664,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.0057664536871016026,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.0064206114038825035,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.006491563282907009,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.006756648421287537,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.00708059873431921,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.0072933584451675415,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.0072934566996991634,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.007480456493794918,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.00769668398424983,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.008241210132837296,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.00871373899281025,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.008927782997488976,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.00920393317937851,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.009538328275084496,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.011022092774510384,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.011195845901966095,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48497311-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48497311\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48497311',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2248T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48497311A>T',\n",
       "     'hgvs_protein': 'p.Cys750Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48497311-A-T\", \"input\": \"NC_000015.10:g.48497311A>T\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"colocated_variants\": [{\"end\": 48497311, \"start\": 48497311, \"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM098659\", \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\"}, {\"end\": 48497311, \"start\": 48497311, \"seq_region_name\": \"15\", \"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM940761\", \"allele_string\": \"HGMD_MUTATION\"}], \"transcript_consequences\": [{\"cds_end\": 2248, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.040136, \"given_ref\": \"A\", \"sift_score\": 0, \"mutationtaster_pred\": \"D\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"A\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9974}, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48497310, \"end\": 48497311}}, \"type\": \"Allele\"}, \"protein_id\": \"NP_000129.3\", \"location\": \"15:48497311\", \"aa\": \"A\", \"protein_start\": 750, \"polyphen_prediction\": \"probably_damaging\", \"consequence_terms\": [\"missense_variant\"], \"impact\": \"MODERATE\", \"eve_score\": 0.795486317427967, \"cds_start\": 2248, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys750Ser\", \"cdna_start\": 2564, \"cdna_end\": 2564, \"amino_acids\": \"C/S\", \"codons\": \"Tgc/Agc\", \"mastermind_mmid3\": \"FBN1:C750S\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"canonical\": 1, \"exon\": \"19/66\", \"lrt_pred\": \"invalid_field\", \"blosum62\": -1, \"spliceai\": {\"DP_DL\": 6, \"DP_AL\": -10, \"DS_DG\": 0.01, \"DS_AL\": 0, \"DP_DG\": -41, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AG\": -2}, \"hgvsc\": \"NM_000138.5:c.2248T>A\", \"variant_allele\": \"T\", \"clinpred\": 0.999855399131775, \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"mastermind_counts\": \"3|3|3\", \"sift_prediction\": \"deleterious_low_confidence\", \"eve_class\": \"Pathogenic\", \"cadd_phred\": 28.2, \"polyphen_score\": 0.992, \"protein_end\": 750}], \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48497311A>T\", \"end\": 48497311, \"variant_class\": \"SNV\", \"minimised\": 1, \"start\": 48497311, \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/T\"}',\n",
       "    'coordinates': [{'pca_x': 0.04118551041665669,\n",
       "      'pca_y': 0.08095477073932353},\n",
       "     {'t-sne_x': 15.71776300831612, 't-sne_y': 9.311417337066105},\n",
       "     {'umap_x': 3.201324224472046, 'umap_y': 4.959694862365723}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.0022501552011817694,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465574-A-T',\n",
       "      'distance': 0.0024209669791162014,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430791-A-T',\n",
       "      'distance': 0.00311967171728611,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596325-A-T',\n",
       "      'distance': 0.0031825529877096415,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.0032317345030605793,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497311-A-G',\n",
       "      'distance': 0.0033086922485381365,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428439-A-T',\n",
       "      'distance': 0.003435081336647272,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0035492382012307644,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.0035660264547914267,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.003570186672732234,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.0036533097736537457,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596373-A-G',\n",
       "      'distance': 0.003716793842613697,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.0037232674658298492,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.0037690664175897837,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.003790938528254628,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.003804711624979973,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.0038650245405733585,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.003904312150552869,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415684-C-T',\n",
       "      'distance': 0.00390629144385457,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48497358-C-A',\n",
       "      'distance': 0.003909615334123373,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48497311-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48497311\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48497311',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2248T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48497311A>C',\n",
       "     'hgvs_protein': 'p.Cys750Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48497311-A-C\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48497311A>C\", \"colocated_variants\": [{\"end\": 48497311, \"start\": 48497311, \"id\": \"CM098659\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"end\": 48497311, \"start\": 48497311, \"seq_region_name\": \"15\", \"id\": \"CM940761\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1}], \"transcript_consequences\": [{\"used_ref\": \"A\", \"alphamissense\": {\"am_pathogenicity\": 0.9856, \"am_class\": \"likely_pathogenic\"}, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"D\", \"sift_score\": 0, \"given_ref\": \"A\", \"cadd_raw\": 5.242112, \"biotype\": \"protein_coding\", \"cds_end\": 2248, \"cdna_end\": 2564, \"cdna_start\": 2564, \"hgvsp\": \"NP_000129.3:p.Cys750Gly\", \"gene_id\": \"2200\", \"cds_start\": 2248, \"eve_score\": 0.790586655127818, \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\"], \"protein_start\": 750, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48497311\", \"protein_id\": \"NP_000129.3\", \"aa\": \"A\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"start\": 48497310, \"type\": \"SimpleInterval\", \"end\": 48497311}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"loeuf\": 0.105, \"mastermind_counts\": \"6|17|17\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"C\", \"strand\": -1, \"clinpred\": 0.999728500843048, \"hgvsc\": \"NM_000138.5:c.2248T>G\", \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DG\": -41, \"DS_AG\": 0, \"DP_AG\": -1, \"DS_DL\": 0, \"DP_AL\": -10, \"DP_DL\": 6, \"DS_AL\": 0, \"DS_DG\": 0.02}, \"blosum62\": -3, \"lrt_pred\": \"invalid_field\", \"exon\": \"19/66\", \"canonical\": 1, \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"mastermind_mmid3\": \"FBN1:C750G\", \"codons\": \"Tgc/Ggc\", \"amino_acids\": \"C/G\", \"protein_end\": 750, \"cadd_phred\": 29.4, \"polyphen_score\": 0.995, \"eve_class\": \"Pathogenic\", \"sift_prediction\": \"deleterious_low_confidence\"}], \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48497311A>C\", \"variant_class\": \"SNV\", \"end\": 48497311, \"minimised\": 1, \"start\": 48497311, \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/C\"}',\n",
       "    'coordinates': [{'pca_x': 0.04187072846969761,\n",
       "      'pca_y': 0.08390327630310604},\n",
       "     {'t-sne_x': 20.41961806751495, 't-sne_y': 3.5445437435908906},\n",
       "     {'umap_x': 3.608490467071533, 'umap_y': 6.34119987487793}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.001631460851058364,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.0016946954419836402,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.0018271286971867085,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488419-A-C',\n",
       "      'distance': 0.00202355207875371,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.002077084733173251,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.0022036710288375616,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.002388750435784459,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.0025379895232617855,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508693-A-C',\n",
       "      'distance': 0.0028057494200766087,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520677-A-C',\n",
       "      'distance': 0.003398377448320389,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.0037368549965322018,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497311-A-G',\n",
       "      'distance': 0.0037853841204196215,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483822-A-C',\n",
       "      'distance': 0.003874524962157011,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.003971775062382221,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.004006671719253063,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415582-C-A',\n",
       "      'distance': 0.004010994918644428,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.004097283817827702,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.004158196039497852,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.004214071203023195,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.00423359964042902,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48496105-CAT-C',\n",
       "    'metadata': {'vcf_string': '15\\t48496105\\t.\\tCAT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48496105',\n",
       "     'ref_allele': 'CAT',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2412_2413del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48496106_48496107del',\n",
       "     'hgvs_protein': 'p.Cys805Ter',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"strand\": 1, \"phenotype_or_disease\": 1, \"pubmed\": [24033266, 10533071, 17657824, 11453977], \"id\": \"rs397515769\", \"var_synonyms\": {\"ClinVar\": [\"RCV003764662\", \"RCV000035138\", \"VCV000042304\"]}, \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"AT/-\", \"seq_region_name\": \"15\", \"end\": 48496107, \"start\": 48496106}], \"transcript_consequences\": [{\"consequence_terms\": [\"frameshift_variant\"], \"protein_start\": 804, \"protein_id\": \"NP_000129.3\", \"aa\": \"AT\", \"location\": \"15:48496106-48496107\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48496107, \"type\": \"SimpleInterval\", \"start\": 48496105}}}, \"loeuf\": 0.105, \"cdna_end\": 2729, \"cdna_start\": 2728, \"hgvsp\": \"NP_000129.3:p.Cys805Ter\", \"gene_id\": \"2200\", \"cds_start\": 2412, \"impact\": \"HIGH\", \"given_ref\": \"AT\", \"biotype\": \"protein_coding\", \"cds_end\": 2413, \"used_ref\": \"AT\", \"lof_info\": \"PERCENTILE:0.280060352831941,GERP_DIST:6053.72533015311,BP_DIST:6157,DIST_FROM_LAST_EXON:5768,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"lof\": \"HC\", \"protein_end\": 805, \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"conservation\": 1.42, \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"TC/TX\", \"mastermind_mmid3\": \"FBN1:C805X\", \"codons\": \"acATgt/acgt\", \"mastermind_counts\": \"7|7|7\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"-\", \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.2412_2413del\", \"spliceai\": {\"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": 1, \"DP_AL\": 1, \"DS_DL\": 0, \"DP_AG\": 31, \"DP_DG\": 3, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0}, \"exon\": \"20/66\", \"canonical\": 1}], \"vcf_string\": \"15-48496105-CAT-C\", \"input\": \"NC_000015.10:g.48496106_48496107del\", \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"assembly_name\": \"GRCh38\", \"start\": 48496106, \"allele_string\": \"AT/-\", \"id\": \"NC_000015.10:g.48496106_48496107del\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"deletion\", \"end\": 48496107}',\n",
       "    'coordinates': [{'pca_x': 0.03685022455281689,\n",
       "      'pca_y': 0.029218298427542234},\n",
       "     {'t-sne_x': 7.530062359375885, 't-sne_y': 4.862941101318317},\n",
       "     {'umap_x': 3.800694704055786, 'umap_y': 2.7942068576812744}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.10631033778190613,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10744529962539673,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10744722932577133,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.1075134351849556,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10771510004997253,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.1079334244132042,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.10850855708122253,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.10885274410247803,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.10901582986116409,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.10922393202781677,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.10934785008430481,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.1094583198428154,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.10955019295215607,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.10962146520614624,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.10964101552963257,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.10996664315462112,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427646-A-T',\n",
       "      'distance': 0.11003255844116211,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.11012528091669083,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437805-C-T',\n",
       "      'distance': 0.11014239490032196,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.1102527379989624,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48495561-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48495561\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48495561',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2447G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48495561C>G',\n",
       "     'hgvs_protein': 'p.Cys816Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"C/G\", \"start\": 48495561, \"assembly_name\": \"GRCh38\", \"end\": 48495561, \"variant_class\": \"SNV\", \"minimised\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48495561C>G\", \"colocated_variants\": [{\"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM067402\", \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"end\": 48495561, \"start\": 48495561}, {\"end\": 48495561, \"start\": 48495561, \"seq_region_name\": \"15\", \"id\": \"CM983863\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1}, {\"allele_string\": \"COSMIC_MUTATION\", \"id\": \"COSV100356701\", \"var_synonyms\": {\"COSMIC\": [\"COSM7799926\"]}, \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48495561, \"end\": 48495561, \"somatic\": 1}, {\"end\": 48495561, \"start\": 48495561, \"strand\": 1, \"clin_sig_allele\": \"A:uncertain_significance;G:likely_pathogenic\", \"phenotype_or_disease\": 1, \"pubmed\": [24033266, 14695540, 9399842], \"id\": \"rs397515770\", \"var_synonyms\": {\"ClinVar\": [\"RCV000769649\", \"VCV000626600\", \"RCV000035139\", \"VCV000042305\"], \"UniProt\": [\"VAR_017991\"]}, \"clin_sig\": [\"uncertain_significance\", \"likely_pathogenic\"], \"allele_string\": \"C/A/G\", \"seq_region_name\": \"15\"}], \"transcript_consequences\": [{\"protein_end\": 816, \"eve_class\": \"Pathogenic\", \"polyphen_score\": 0.881, \"cadd_phred\": 28, \"sift_prediction\": \"deleterious_low_confidence\", \"hgvsc\": \"NM_000138.5:c.2447G>C\", \"variant_allele\": \"G\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"clinpred\": 0.999230265617371, \"mastermind_counts\": \"6|9|9\", \"canonical\": 1, \"exon\": \"21/66\", \"lrt_pred\": \"invalid_field\", \"blosum62\": -1, \"spliceai\": {\"DP_AG\": 27, \"DS_DL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DG\": -10, \"DS_AG\": 0, \"DS_AL\": 0.01, \"DS_DG\": 0, \"DP_AL\": 44, \"DP_DL\": -41}, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"amino_acids\": \"C/S\", \"codons\": \"tGc/tCc\", \"mastermind_mmid3\": \"FBN1:C816S\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys816Ser\", \"cdna_start\": 2763, \"cdna_end\": 2763, \"impact\": \"MODERATE\", \"eve_score\": 0.729070174354608, \"cds_start\": 2447, \"consequence_terms\": [\"missense_variant\"], \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48495560, \"type\": \"SimpleInterval\", \"end\": 48495561}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"protein_id\": \"NP_000129.3\", \"location\": \"15:48495561\", \"aa\": \"C\", \"polyphen_prediction\": \"possibly_damaging\", \"protein_start\": 816, \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 0.018957, \"dG_wt\": 36.0551, \"dG_wt_sd\": 0.059088, \"ddG\": 5.04584, \"ddG_sd\": 0.044529, \"dG_mt\": 41.10094}}, \"used_ref\": \"C\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9984}, \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"mutationtaster_pred\": \"A\", \"cds_end\": 2447, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.021031, \"sift_score\": 0, \"given_ref\": \"C\"}], \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48495561C>G\", \"vcf_string\": \"15-48495561-C-G\"}',\n",
       "    'coordinates': [{'pca_x': 0.03875581377505338,\n",
       "      'pca_y': 0.08272918602782925},\n",
       "     {'t-sne_x': 20.004321018925108, 't-sne_y': -5.605744584007776},\n",
       "     {'umap_x': 8.46474838256836, 'umap_y': 7.048567771911621}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0009506000205874443,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0009527311194688082,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.0010498142801225185,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.0010809197556227446,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.001123016350902617,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.001202884828671813,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.0012535452842712402,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.001289348816499114,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.002070453017950058,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.002207252662628889,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.0022103311493992805,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.0022213184274733067,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.002321518026292324,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489991-C-G',\n",
       "      'distance': 0.00236193323507905,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.002452075481414795,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.0024783657863736153,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520708-C-G',\n",
       "      'distance': 0.0026813377626240253,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.0027139256708323956,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-G',\n",
       "      'distance': 0.002739511663094163,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.0028030737303197384,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48495243-AA-A',\n",
       "    'metadata': {'vcf_string': '15\\t48495243\\t.\\tAA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48495243',\n",
       "     'ref_allele': 'AA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2557del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48495244del',\n",
       "     'hgvs_protein': 'p.Cys853AlafsTer19',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48495243-AA-A\", \"strand\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"input\": \"NC_000015.10:g.48495244del\", \"colocated_variants\": [{\"end\": 48495244, \"start\": 48495244, \"seq_region_name\": \"15\", \"id\": \"CD111081\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1}], \"transcript_consequences\": [{\"lof\": \"HC\", \"protein_end\": 853, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"amino_acids\": \"C/X\", \"codons\": \"Tgc/gc\", \"conservation\": 1.41, \"gene_symbol_source\": \"EntrezGene\", \"strand\": -1, \"variant_allele\": \"-\", \"transcript_id\": \"NM_000138.5\", \"hgvs_offset\": -1, \"hgvsc\": \"NM_000138.5:c.2557del\", \"exon\": \"22/66\", \"spliceai\": {\"DP_DL\": 13, \"DP_AL\": 48, \"DS_DG\": 0, \"DS_AL\": 0, \"DS_AG\": 0, \"DP_DG\": -27, \"SYMBOL\": \"FBN1\", \"DS_DL\": 0, \"DP_AG\": 6}, \"canonical\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"protein_id\": \"NP_000129.3\", \"location\": \"15:48495243\", \"aa\": \"A\", \"protein_start\": 853, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48495244, \"type\": \"SimpleInterval\", \"start\": 48495243}, \"type\": \"SequenceLocation\"}}, \"cdna_start\": 2873, \"cdna_end\": 2873, \"hgvsp\": \"NP_000129.3:p.Cys853AlafsTer19\", \"gene_id\": \"2200\", \"cds_start\": 2557, \"impact\": \"HIGH\", \"given_ref\": \"A\", \"cds_end\": 2557, \"biotype\": \"protein_coding\", \"used_ref\": \"A\", \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 0.151864, \"dG_wt\": 36.13536, \"dG_wt_sd\": 0.069732, \"ddG\": -0.72182, \"dG_mt\": 35.41354, \"ddG_sd\": 0.154268}}, \"lof_info\": \"PERCENTILE:0.296773444753946,GERP_DIST:6053.72533015311,BP_DIST:6016,DIST_FROM_LAST_EXON:5627,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396}], \"id\": \"NC_000015.10:g.48495244del\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"deletion\", \"end\": 48495244, \"assembly_name\": \"GRCh38\", \"start\": 48495244, \"allele_string\": \"A/-\"}',\n",
       "    'coordinates': [{'pca_x': -0.035144400108226115,\n",
       "      'pca_y': 0.037623572558424},\n",
       "     {'t-sne_x': -12.992111404905788, 't-sne_y': -3.1751422608890283},\n",
       "     {'umap_x': 0.4257781207561493, 'umap_y': 15.061488151550293}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.09041780233383179,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.09276339411735535,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.09327783435583115,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.09391849488019943,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.09425894916057587,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.09443049877882004,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.09526868164539337,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.0955238789319992,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.09568680077791214,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.09589342772960663,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.09618392586708069,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.09640984237194061,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.09655947983264923,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.09658980369567871,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.09719396382570267,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.09791731834411621,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.09794262051582336,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.09835348278284073,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.09871132671833038,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09997833520174026,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48495229-G-GA',\n",
       "    'metadata': {'vcf_string': '15\\t48495229\\t.\\tG\\tGA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48495229',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'GA',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2570dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48495230dup',\n",
       "     'hgvs_protein': 'p.Ile858HisfsTer2',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"frameshift_variant\", \"input\": \"NC_000015.10:g.48495230dup\", \"strand\": 1, \"vcf_string\": \"15-48495229-G-GA\", \"transcript_consequences\": [{\"mastermind_counts\": \"2|2|2\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"A\", \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.2570dup\", \"spliceai\": {\"DS_AL\": 0, \"DS_DG\": 0, \"DP_AL\": 31, \"DP_DL\": -13, \"DP_AG\": 20, \"DS_DL\": 0, \"DS_AG\": 0, \"DP_DG\": 20, \"SYMBOL\": \"FBN1\"}, \"exon\": \"22/66\", \"canonical\": 1, \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"conservation\": 1.412, \"gene_symbol_source\": \"EntrezGene\", \"codons\": \"gtc/gtTc\", \"amino_acids\": \"V/VX\", \"mastermind_mmid3\": \"FBN1:I858fs\", \"protein_end\": 857, \"lof\": \"HC\", \"used_ref\": \"-\", \"lof_info\": \"PERCENTILE:0.298282265552461,GERP_DIST:6053.72533015311,BP_DIST:6002,DIST_FROM_LAST_EXON:5613,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mutfunc\": {\"exp\": {\"dG_mt\": 36.08256, \"ddG_sd\": 0.040042, \"dG_mt_sd\": 0.040556, \"dG_wt\": 36.14068, \"dG_wt_sd\": 0.041411, \"ddG\": -0.05812}}, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"-\", \"biotype\": \"protein_coding\", \"cds_end\": 2571, \"cdna_end\": 2887, \"cdna_start\": 2886, \"hgvsp\": \"NP_000129.3:p.Ile858HisfsTer2\", \"gene_id\": \"2200\", \"cds_start\": 2570, \"impact\": \"HIGH\", \"consequence_terms\": [\"frameshift_variant\"], \"protein_start\": 857, \"location\": \"15:48495229-48495230\", \"protein_id\": \"NP_000129.3\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48495229, \"start\": 48495229, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}}, \"loeuf\": 0.105}], \"end\": 48495229, \"variant_class\": \"insertion\", \"minimised\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48495230dup\", \"allele_string\": \"-/A\", \"start\": 48495230, \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': -0.01817600419244328,\n",
       "      'pca_y': 0.018234007909288273},\n",
       "     {'t-sne_x': 11.140162572905275, 't-sne_y': 15.90757064139023},\n",
       "     {'umap_x': 1.4943526983261108, 'umap_y': 11.661099433898926}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410123-C-A',\n",
       "      'distance': 0.08950875699520111,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48408950-A-G',\n",
       "      'distance': 0.0908544510602951,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.09154927730560303,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48408466-T-C',\n",
       "      'distance': 0.09184764325618744,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.09248782694339752,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48409622-T-C',\n",
       "      'distance': 0.0927184671163559,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09281006455421448,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48409031-C-T',\n",
       "      'distance': 0.09293018281459808,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09369491040706635,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48409001-G-A',\n",
       "      'distance': 0.09390954673290253,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48409428-C-A',\n",
       "      'distance': 0.09399434179067612,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48432907-G-A',\n",
       "      'distance': 0.09430685639381409,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.09457151591777802,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.09460276365280151,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.09497925639152527,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.09516933560371399,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.09526999294757843,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.09531283378601074,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437807-A-G',\n",
       "      'distance': 0.09540075808763504,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644804-G-A',\n",
       "      'distance': 0.09541149437427521,\n",
       "      'pathogenicity': 'benign_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_benign',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_benign', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_benign', 'confidence_score': 0.6},\n",
       "     '20': {'pred_result': 'uncertain', 'confidence_score': 0.5}}},\n",
       "   {'variant_id': '15-48495132-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48495132\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48495132',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2668T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48495132A>G',\n",
       "     'hgvs_protein': 'p.Cys890Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48495132A>G\", \"end\": 48495132, \"variant_class\": \"SNV\", \"minimised\": 1, \"start\": 48495132, \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/G\", \"vcf_string\": \"15-48495132-A-G\", \"input\": \"NC_000015.10:g.48495132A>G\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"transcript_consequences\": [{\"sift_score\": 0, \"given_ref\": \"A\", \"biotype\": \"protein_coding\", \"cadd_raw\": 4.87165, \"cds_end\": 2668, \"mutationtaster_pred\": \"D\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9979}, \"used_ref\": \"A\", \"mutfunc\": {\"exp\": {\"dG_mt\": 36.80974, \"ddG_sd\": 0.299942, \"ddG\": 2.1617, \"dG_wt\": 34.64804, \"dG_mt_sd\": 0.338382, \"dG_wt_sd\": 0.05471}}, \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 890, \"location\": \"15:48495132\", \"protein_id\": \"NP_000129.3\", \"aa\": \"A\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48495132, \"type\": \"SimpleInterval\", \"start\": 48495131}}, \"type\": \"Allele\"}, \"loeuf\": 0.105, \"consequence_terms\": [\"missense_variant\"], \"cds_start\": 2668, \"eve_score\": 0.75917911318384, \"impact\": \"MODERATE\", \"cdna_end\": 2984, \"cdna_start\": 2984, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys890Arg\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"amino_acids\": \"C/R\", \"codons\": \"Tgc/Cgc\", \"mastermind_mmid3\": \"FBN1:C890R\", \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"spliceai\": {\"DP_DL\": -5, \"DP_AL\": 30, \"DS_DG\": 0, \"DS_AL\": 0, \"DS_AG\": 0, \"DP_DG\": -9, \"SYMBOL\": \"FBN1\", \"DS_DL\": 0.12, \"DP_AG\": -5}, \"exon\": \"22/66\", \"lrt_pred\": \"invalid_field\", \"blosum62\": -3, \"canonical\": 1, \"mastermind_counts\": \"6|10|10\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.999750316143036, \"variant_allele\": \"G\", \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.2668T>C\", \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.909, \"cadd_phred\": 27.2, \"eve_class\": \"Pathogenic\", \"protein_end\": 890}], \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM054688\", \"seq_region_name\": \"15\", \"start\": 48495132, \"end\": 48495132}, {\"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM952168\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48495132, \"end\": 48495132}]}',\n",
       "    'coordinates': [{'pca_x': 0.03319678788453814,\n",
       "      'pca_y': 0.10307009863262692},\n",
       "     {'t-sne_x': -19.01965946695287, 't-sne_y': 32.88250746607552},\n",
       "     {'umap_x': 2.278346061706543, 'umap_y': 20.714296340942383}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48495216-A-G',\n",
       "      'distance': 0.0010959922801703215,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.0012473700335249305,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463148-A-G',\n",
       "      'distance': 0.0013037424068897963,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.0013440249022096395,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472644-A-G',\n",
       "      'distance': 0.0015219170600175858,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48513627-A-G',\n",
       "      'distance': 0.0017399312928318977,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513631-A-G',\n",
       "      'distance': 0.0017399312928318977,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48494210-A-G',\n",
       "      'distance': 0.0023036603815853596,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513600-A-G',\n",
       "      'distance': 0.00237592332996428,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456643-A-G',\n",
       "      'distance': 0.0024096183478832245,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445454-A-G',\n",
       "      'distance': 0.002502304734662175,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515434-C-A',\n",
       "      'distance': 0.0026078466325998306,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515465-G-A',\n",
       "      'distance': 0.0027570556849241257,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495243-A-T',\n",
       "      'distance': 0.0028404768090695143,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48452637-A-G',\n",
       "      'distance': 0.0029980898834764957,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510050-A-G',\n",
       "      'distance': 0.0031023002229630947,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505136-A-G',\n",
       "      'distance': 0.0032377629540860653,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474345-A-C',\n",
       "      'distance': 0.0034407838247716427,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.0034660007804632187,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472557-A-T',\n",
       "      'distance': 0.003795132739469409,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48495123-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48495123\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48495123',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2677G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48495123C>T',\n",
       "     'hgvs_protein': 'p.Asp893Asn',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"start\": 48495123, \"allele_string\": \"C/T\", \"id\": \"NC_000015.10:g.48495123C>T\", \"seq_region_name\": \"15\", \"minimised\": 1, \"end\": 48495123, \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"canonical\": 1, \"spliceai\": {\"DP_DG\": 4, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DL\": 0.5, \"DP_AG\": 0, \"DP_DL\": 0, \"DP_AL\": 4, \"DS_DG\": 0.76, \"DS_AL\": 0}, \"lrt_pred\": \"invalid_field\", \"blosum62\": 1, \"exon\": \"22/66\", \"hgvsc\": \"NM_000138.5:c.2677G>A\", \"mastermind_counts\": \"4|7|15\", \"variant_allele\": \"T\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"clinpred\": 0.944041550159454, \"conservation\": 3.72, \"gene_symbol_source\": \"EntrezGene\", \"mastermind_mmid3\": \"FBN1:D893N,FBN1:D893srd\", \"codons\": \"Gat/Aat\", \"amino_acids\": \"D/N\", \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"polyphen_score\": 0.959, \"cadd_phred\": 36, \"protein_end\": 893, \"sift_prediction\": \"deleterious_low_confidence\", \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 0.02709, \"dG_wt\": 35.69164, \"dG_wt_sd\": 0.036383, \"ddG\": 0.600219, \"ddG_sd\": 0.036955, \"dG_mt\": 36.291859}}, \"used_ref\": \"C\", \"alphamissense\": {\"am_pathogenicity\": 0.8016, \"am_class\": \"likely_pathogenic\"}, \"cadd_raw\": 7.949912, \"biotype\": \"protein_coding\", \"cds_end\": 2677, \"sift_score\": 0.01, \"given_ref\": \"C\", \"ada_score\": 0.999984702697481, \"maxentscan_alt\": 3.783, \"maxentscan_ref\": 8.846, \"mutationtaster_pred\": \"D\", \"rf_score\": 0.992, \"maxentscan_diff\": 5.062, \"impact\": \"MODERATE\", \"cds_start\": 2677, \"hgvsp\": \"NP_000129.3:p.Asp893Asn\", \"gene_id\": \"2200\", \"cdna_end\": 2993, \"cdna_start\": 2993, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48495123, \"type\": \"SimpleInterval\", \"start\": 48495122}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"loeuf\": 0.105, \"protein_start\": 893, \"polyphen_prediction\": \"probably_damaging\", \"aa\": \"C\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48495123\", \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"]}], \"colocated_variants\": [{\"start\": 48495123, \"end\": 48495123, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CS013967\"}, {\"clin_sig_allele\": \"T:uncertain_significance;T:likely_pathogenic;G:likely_pathogenic;A:likely_pathogenic;A:uncertain_significance\", \"strand\": 1, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV002564295\", \"RCV001509498\", \"VCV001163973\", \"RCV002440407\", \"RCV000663568\", \"VCV000549101\", \"RCV000029716\", \"VCV000036056\"]}, \"id\": \"rs193922193\", \"pubmed\": [11700157], \"allele_string\": \"C/A/G/T\", \"clin_sig\": [\"uncertain_significance\", \"likely_pathogenic\"], \"seq_region_name\": \"15\", \"end\": 48495123, \"start\": 48495123}], \"vcf_string\": \"15-48495123-C-T\", \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48495123C>T\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.0414238278049867,\n",
       "      'pca_y': 0.05260231752333067},\n",
       "     {'t-sne_x': -13.066289071391761, 't-sne_y': -3.2146524102622056},\n",
       "     {'umap_x': 0.4302431344985962, 'umap_y': 15.012742042541504}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.0017413803143426776,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.0017974840011447668,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.002250976860523224,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.0028065836522728205,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.0028089587576687336,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.0035282596945762634,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.003672656137496233,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.0036967319902032614,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.004115348681807518,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.0041335211135447025,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.004199046641588211,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.0042920904234051704,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.004634217359125614,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.005101077724248171,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.005443671252578497,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.0054696365259587765,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.005628528539091349,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.006496764719486237,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487075-C-T',\n",
       "      'distance': 0.006881361827254295,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.007068391889333725,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48492575-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48492575\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48492575',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2740T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48492575A>T',\n",
       "     'hgvs_protein': 'p.Cys914Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 914, \"polyphen_score\": 0.998, \"cadd_phred\": 28.7, \"eve_class\": \"Pathogenic\", \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.76, \"codons\": \"Tgt/Agt\", \"mastermind_mmid3\": \"FBN1:C914S\", \"amino_acids\": \"C/S\", \"mastermind_counts\": \"5|7|7\", \"strand\": -1, \"variant_allele\": \"T\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.999589502811432, \"hgvsc\": \"NM_000138.5:c.2740T>A\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 5, \"DP_DG\": 0, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": -7, \"DP_AL\": 11}, \"lrt_pred\": \"invalid_field\", \"blosum62\": -1, \"exon\": \"24/66\", \"canonical\": 1, \"consequence_terms\": [\"missense_variant\"], \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 914, \"protein_id\": \"NP_000129.3\", \"location\": \"15:48492575\", \"aa\": \"A\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48492574, \"end\": 48492575}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"loeuf\": 0.105, \"cdna_end\": 3056, \"cdna_start\": 3056, \"hgvsp\": \"NP_000129.3:p.Cys914Ser\", \"gene_id\": \"2200\", \"cds_start\": 2740, \"eve_score\": 0.783816066637441, \"impact\": \"MODERATE\", \"mutationtaster_pred\": \"A\", \"sift_score\": 0, \"given_ref\": \"A\", \"cadd_raw\": 5.121446, \"biotype\": \"protein_coding\", \"cds_end\": 2740, \"used_ref\": \"A\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9991}, \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 0.010189, \"dG_wt\": 36.10714, \"dG_wt_sd\": 0.042362, \"ddG\": 2.19726, \"dG_mt\": 38.3044, \"ddG_sd\": 0.047095}}, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396}], \"colocated_variants\": [{\"start\": 48492575, \"end\": 48492575, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM098670\", \"phenotype_or_disease\": 1, \"strand\": 1}, {\"end\": 48492575, \"start\": 48492575, \"seq_region_name\": \"15\", \"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM1612716\", \"allele_string\": \"HGMD_MUTATION\"}, {\"start\": 48492575, \"end\": 48492575, \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"allele_string\": \"A/C/G/T\", \"var_synonyms\": {\"ClinVar\": [\"RCV000619511\", \"VCV000520236\", \"RCV000663577\", \"VCV000549110\", \"RCV000807768\", \"VCV000652247\"]}, \"pubmed\": [17701892], \"id\": \"rs1555398994\", \"phenotype_or_disease\": 1, \"strand\": 1, \"clin_sig_allele\": \"T:likely_pathogenic;C:likely_pathogenic;G:pathogenic\", \"seq_region_name\": \"15\"}], \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48492575A>T\", \"vcf_string\": \"15-48492575-A-T\", \"allele_string\": \"A/T\", \"assembly_name\": \"GRCh38\", \"start\": 48492575, \"minimised\": 1, \"end\": 48492575, \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48492575A>T\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.031010197730836908,\n",
       "      'pca_y': 0.09935669760048343},\n",
       "     {'t-sne_x': -14.91766236200353, 't-sne_y': 33.37660462545809},\n",
       "     {'umap_x': 1.3287934064865112, 'umap_y': 21.82013511657715}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48472557-A-T',\n",
       "      'distance': 0.0011348482221364975,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495243-A-T',\n",
       "      'distance': 0.0015911380760371685,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465619-A-T',\n",
       "      'distance': 0.001920436741784215,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425745-A-T',\n",
       "      'distance': 0.0024603295605629683,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48422024-A-T',\n",
       "      'distance': 0.0025897957384586334,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.0029101695399731398,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.0029245491605252028,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492538-C-T',\n",
       "      'distance': 0.0029417623300105333,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.0031051216647028923,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495131-C-T',\n",
       "      'distance': 0.0031371540389955044,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.003153066849336028,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.003249256405979395,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.0034501426853239536,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.00345632154494524,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495156-C-T',\n",
       "      'distance': 0.003486259374767542,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495216-A-G',\n",
       "      'distance': 0.003546398598700762,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.0035820116754621267,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494210-A-G',\n",
       "      'distance': 0.0036882495041936636,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.0037617026828229427,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.0037647897843271494,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48492554-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48492554\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48492554',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2761T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48492554A>C',\n",
       "     'hgvs_protein': 'p.Cys921Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"A/C\", \"assembly_name\": \"GRCh38\", \"start\": 48492554, \"minimised\": 1, \"end\": 48492554, \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48492554A>C\", \"seq_region_name\": \"15\", \"colocated_variants\": [{\"id\": \"CM022003\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"end\": 48492554, \"start\": 48492554}], \"transcript_consequences\": [{\"consequence_terms\": [\"missense_variant\"], \"protein_id\": \"NP_000129.3\", \"aa\": \"A\", \"location\": \"15:48492554\", \"protein_start\": 921, \"polyphen_prediction\": \"probably_damaging\", \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48492554, \"type\": \"SimpleInterval\", \"start\": 48492553}}}, \"cdna_start\": 3077, \"cdna_end\": 3077, \"hgvsp\": \"NP_000129.3:p.Cys921Gly\", \"gene_id\": \"2200\", \"eve_score\": 0.782243243472092, \"cds_start\": 2761, \"impact\": \"MODERATE\", \"mutationtaster_pred\": \"D\", \"given_ref\": \"A\", \"sift_score\": 0, \"cds_end\": 2761, \"cadd_raw\": 5.24729, \"biotype\": \"protein_coding\", \"used_ref\": \"A\", \"alphamissense\": {\"am_pathogenicity\": 0.9929, \"am_class\": \"likely_pathogenic\"}, \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.035031, \"dG_mt\": 40.6766, \"dG_wt\": 36.1043, \"dG_mt_sd\": 0.007814, \"dG_wt_sd\": 0.03781, \"ddG\": 4.5723}}, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 921, \"eve_class\": \"Pathogenic\", \"polyphen_score\": 0.999, \"cadd_phred\": 29.5, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"codons\": \"Tgt/Ggt\", \"mastermind_mmid3\": \"FBN1:C921G\", \"amino_acids\": \"C/G\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.76, \"variant_allele\": \"C\", \"strand\": -1, \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.999918818473816, \"mastermind_counts\": \"5|8|8\", \"hgvsc\": \"NM_000138.5:c.2761T>G\", \"lrt_pred\": \"invalid_field\", \"exon\": \"24/66\", \"blosum62\": -3, \"spliceai\": {\"DS_DG\": 0.01, \"DS_AL\": 0, \"DP_DL\": -13, \"DP_AL\": 26, \"DS_DL\": 0, \"DP_AG\": 5, \"SYMBOL\": \"FBN1\", \"DP_DG\": 0, \"DS_AG\": 0}, \"canonical\": 1}], \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48492554A>C\", \"strand\": 1, \"vcf_string\": \"15-48492554-A-C\"}',\n",
       "    'coordinates': [{'pca_x': 0.03428857322973555,\n",
       "      'pca_y': 0.10602131378299406},\n",
       "     {'t-sne_x': -17.850597614486645, 't-sne_y': 29.967545454855276},\n",
       "     {'umap_x': 1.9951374530792236, 'umap_y': 21.614547729492188}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48508660-A-C',\n",
       "      'distance': 0.0025966258253902197,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508633-A-C',\n",
       "      'distance': 0.004196688067167997,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505136-A-C',\n",
       "      'distance': 0.004401856567710638,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474345-A-C',\n",
       "      'distance': 0.004437997005879879,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463950-A-C',\n",
       "      'distance': 0.00480850413441658,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495512-A-C',\n",
       "      'distance': 0.004848719108849764,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-G',\n",
       "      'distance': 0.005339550785720348,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483935-C-T',\n",
       "      'distance': 0.005450577475130558,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495519-C-G',\n",
       "      'distance': 0.005463153589516878,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513643-G-A',\n",
       "      'distance': 0.0060233487747609615,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48513656-C-T',\n",
       "      'distance': 0.006107139866799116,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495519-C-A',\n",
       "      'distance': 0.006318600382655859,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441764-A-C',\n",
       "      'distance': 0.006328989751636982,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465646-A-G',\n",
       "      'distance': 0.006513223983347416,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515434-C-A',\n",
       "      'distance': 0.006782165728509426,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415585-C-A',\n",
       "      'distance': 0.006861705798655748,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.00705287279561162,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463949-C-A',\n",
       "      'distance': 0.007089332677423954,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.007098148576915264,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.007164963521063328,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48492492-CATTC-C',\n",
       "    'metadata': {'vcf_string': '15\\t48492492\\t.\\tCATTC\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48492492',\n",
       "     'ref_allele': 'CATTC',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2821_2824del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48492493_48492496del',\n",
       "     'hgvs_protein': 'p.Met941LeufsTer19',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48492492-CATTC-C\", \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48492493_48492496del\", \"transcript_consequences\": [{\"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"used_ref\": \"TCAT\", \"lof_info\": \"PERCENTILE:0.327762302692665,GERP_DIST:6053.72533015311,BP_DIST:5752,DIST_FROM_LAST_EXON:5363,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.007321, \"dG_mt\": 40.04436, \"ddG\": 3.87396, \"dG_wt_sd\": 0, \"dG_mt_sd\": 0.007321, \"dG_wt\": 36.1704}}, \"given_ref\": \"ATTC\", \"biotype\": \"protein_coding\", \"cds_end\": 2824, \"cds_start\": 2821, \"impact\": \"HIGH\", \"cdna_end\": 3140, \"cdna_start\": 3137, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Met941LeufsTer19\", \"protein_start\": 941, \"aa\": \"ATTC\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48492491-48492494\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"start\": 48492492, \"type\": \"SimpleInterval\", \"end\": 48492496}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}}, \"loeuf\": 0.105, \"consequence_terms\": [\"frameshift_variant\"], \"spliceai\": {\"DS_AL\": 0, \"DS_DG\": 0, \"DP_AL\": 33, \"DP_DL\": -34, \"DP_AG\": -21, \"DS_DL\": 0, \"DP_DG\": -31, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\"}, \"exon\": \"24/66\", \"canonical\": 1, \"hgvs_offset\": -2, \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"-\", \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.2821_2824del\", \"conservation\": 3.146, \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"MT/X\", \"codons\": \"ATGAct/ct\", \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"protein_end\": 942, \"lof\": \"HC\"}], \"id\": \"NC_000015.10:g.48492493_48492496del\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"deletion\", \"end\": 48492496, \"assembly_name\": \"GRCh38\", \"start\": 48492493, \"allele_string\": \"ATTC/-\"}',\n",
       "    'coordinates': [{'pca_x': 0.045241669204384743,\n",
       "      'pca_y': 0.03816072135519414},\n",
       "     {'t-sne_x': 4.4008100633501375, 't-sne_y': 7.853018617244995},\n",
       "     {'umap_x': 3.665795087814331, 'umap_y': 3.04789400100708}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.10151475667953491,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.10286469757556915,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.10307516157627106,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.10409072041511536,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.1042013168334961,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.10430825501680374,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.1044238805770874,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.10442391037940979,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.10449590533971786,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.10451562702655792,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-T',\n",
       "      'distance': 0.10452593863010406,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.10454560816287994,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468504-C-T',\n",
       "      'distance': 0.1045861542224884,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.10461744666099548,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.10462629795074463,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.1046682745218277,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.10466830432415009,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.10467171669006348,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446768-A-G',\n",
       "      'distance': 0.10474493354558945,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.1048813909292221,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48489969-CC-C',\n",
       "    'metadata': {'vcf_string': '15\\t48489969\\t.\\tCC\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48489969',\n",
       "     'ref_allele': 'CC',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.2966del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48489970del',\n",
       "     'hgvs_protein': 'p.Gly989ValfsTer10',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48489970, \"start\": 48489969, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}}, \"loeuf\": 0.105, \"protein_start\": 989, \"aa\": \"C\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48489967\", \"consequence_terms\": [\"frameshift_variant\"], \"impact\": \"HIGH\", \"cds_start\": 2966, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Gly989ValfsTer10\", \"cdna_end\": 3282, \"cdna_start\": 3282, \"biotype\": \"protein_coding\", \"cds_end\": 2966, \"given_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.344243268337976,GERP_DIST:6053.72533015311,BP_DIST:5612,DIST_FROM_LAST_EXON:5223,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"used_ref\": \"C\", \"lof\": \"HC\", \"protein_end\": 989, \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.76, \"codons\": \"gGt/gt\", \"amino_acids\": \"G/X\", \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"canonical\": 1, \"spliceai\": {\"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": -1, \"DP_AL\": -5, \"DS_DL\": 0, \"DP_AG\": 4, \"DP_DG\": -44, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\"}, \"exon\": \"25/66\", \"hgvsc\": \"NM_000138.5:c.2966del\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"-\", \"hgvs_offset\": -3, \"strand\": -1}], \"colocated_variants\": [{\"start\": 48489970, \"end\": 48489970, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM1514024\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\"}], \"strand\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"input\": \"NC_000015.10:g.48489970del\", \"vcf_string\": \"15-48489969-CC-C\", \"allele_string\": \"C/-\", \"start\": 48489970, \"assembly_name\": \"GRCh38\", \"variant_class\": \"deletion\", \"end\": 48489970, \"minimised\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48489970del\"}',\n",
       "    'coordinates': [{'pca_x': 0.04835159823421098,\n",
       "      'pca_y': 0.03793634451596397},\n",
       "     {'t-sne_x': 7.3446136852229245, 't-sne_y': 3.8686118408044456},\n",
       "     {'umap_x': 4.1101861000061035, 'umap_y': 3.2436630725860596}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.11121433228254318,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.11150221526622772,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.11165308952331543,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.11187766492366791,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.1122874915599823,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.11263269186019897,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.11280811578035355,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.11287987232208252,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.11309727281332016,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.11332955211400986,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.11341924965381622,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.1135098934173584,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.11361183971166611,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.11376295238733292,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463209-G-C',\n",
       "      'distance': 0.1137704849243164,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.11381205916404724,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.11415205895900726,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.11416783928871155,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489921-G-T',\n",
       "      'distance': 0.11421175301074982,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.11425057053565979,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48489910-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48489910\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48489910',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3023G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48489910C>T',\n",
       "     'hgvs_protein': 'p.Cys1008Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48489910-C-T\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48489910C>T\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM085429\", \"phenotype_or_disease\": 1, \"strand\": 1, \"start\": 48489910, \"end\": 48489910}], \"transcript_consequences\": [{\"polyphen_score\": 0.975, \"cadd_phred\": 28.5, \"eve_class\": \"Pathogenic\", \"protein_end\": 1008, \"sift_prediction\": \"deleterious_low_confidence\", \"canonical\": 1, \"spliceai\": {\"DP_DG\": 3, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": 7, \"DS_DL\": 0, \"DP_AL\": 15, \"DP_DL\": 15, \"DS_AL\": 0, \"DS_DG\": 0}, \"exon\": \"25/66\", \"lrt_pred\": \"invalid_field\", \"blosum62\": -2, \"hgvsc\": \"NM_000138.5:c.3023G>A\", \"mastermind_counts\": \"5|5|5\", \"clinpred\": 0.998569369316101, \"variant_allele\": \"T\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"conservation\": 3.76, \"gene_symbol_source\": \"EntrezGene\", \"codons\": \"tGt/tAt\", \"amino_acids\": \"C/Y\", \"mastermind_mmid3\": \"FBN1:C1008Y\", \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"impact\": \"MODERATE\", \"cds_start\": 3023, \"eve_score\": 0.729793828098189, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys1008Tyr\", \"cdna_end\": 3339, \"cdna_start\": 3339, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48489909, \"type\": \"SimpleInterval\", \"end\": 48489910}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"loeuf\": 0.105, \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 1008, \"location\": \"15:48489910\", \"protein_id\": \"NP_000129.3\", \"aa\": \"C\", \"consequence_terms\": [\"missense_variant\"], \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9963}, \"used_ref\": \"C\", \"biotype\": \"protein_coding\", \"cadd_raw\": 5.088638, \"cds_end\": 3023, \"sift_score\": 0, \"given_ref\": \"C\", \"mutationtaster_pred\": \"D\"}], \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48489910C>T\", \"variant_class\": \"SNV\", \"end\": 48489910, \"minimised\": 1, \"start\": 48489910, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\"}',\n",
       "    'coordinates': [{'pca_x': 0.03598019887313231,\n",
       "      'pca_y': 0.08306592416888939},\n",
       "     {'t-sne_x': 7.962644026853779, 't-sne_y': 8.704762477968442},\n",
       "     {'umap_x': 3.127246141433716, 'umap_y': 2.3124265670776367}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.0008236198336817324,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.0009101465111598372,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.000910400238353759,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.0009209817508235574,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.0010126307606697083,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.001030046259984374,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481699-C-T',\n",
       "      'distance': 0.0011122836731374264,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0011206890922039747,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.001144632464274764,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.0011589627247303724,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.0011628838256001472,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415695-C-T',\n",
       "      'distance': 0.0011678963201120496,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0011943860445171595,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.0012258810456842184,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492574-C-T',\n",
       "      'distance': 0.0012314408086240292,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487317-C-T',\n",
       "      'distance': 0.0012612983118742704,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.0012696759076789021,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.0012819458497688174,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.001293986919336021,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496213-C-T',\n",
       "      'distance': 0.001297383802011609,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48489864-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48489864\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48489864',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3069G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48489864C>G',\n",
       "     'hgvs_protein': 'p.Lys1023Asn',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"start\": 48489864, \"allele_string\": \"C/G\", \"id\": \"NC_000015.10:g.48489864C>G\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"SNV\", \"end\": 48489864, \"colocated_variants\": [{\"id\": \"CM940762\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"end\": 48489864, \"start\": 48489864}], \"transcript_consequences\": [{\"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"alphamissense\": {\"am_pathogenicity\": 0.7584, \"am_class\": \"likely_pathogenic\"}, \"used_ref\": \"C\", \"cds_end\": 3069, \"biotype\": \"protein_coding\", \"cadd_raw\": 2.83325, \"given_ref\": \"C\", \"sift_score\": 0, \"mutationtaster_pred\": \"D\", \"impact\": \"MODERATE\", \"eve_score\": 0.555255955914815, \"cds_start\": 3069, \"hgvsp\": \"NP_000129.3:p.Lys1023Asn\", \"gene_id\": \"2200\", \"cdna_start\": 3385, \"cdna_end\": 3385, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48489863, \"end\": 48489864}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"aa\": \"C\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48489864\", \"polyphen_prediction\": \"possibly_damaging\", \"protein_start\": 1023, \"consequence_terms\": [\"missense_variant\"], \"canonical\": 1, \"blosum62\": 0, \"exon\": \"25/66\", \"lrt_pred\": \"invalid_field\", \"spliceai\": {\"DP_DG\": -20, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AG\": -14, \"DP_DL\": 18, \"DP_AL\": -1, \"DS_DG\": 0, \"DS_AL\": 0}, \"hgvsc\": \"NM_000138.5:c.3069G>C\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"variant_allele\": \"G\", \"clinpred\": 0.850444257259369, \"mastermind_counts\": \"7|17|19\", \"mastermind_mmid3\": \"FBN1:K1023N\", \"amino_acids\": \"K/N\", \"codons\": \"aaG/aaC\", \"conservation\": -1.23, \"gene_symbol_source\": \"EntrezGene\", \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"eve_class\": \"Uncertain\", \"polyphen_score\": 0.784, \"cadd_phred\": 20.5, \"protein_end\": 1023, \"sift_prediction\": \"deleterious_low_confidence\"}], \"vcf_string\": \"15-48489864-C-G\", \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48489864C>G\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.03669573440936674,\n",
       "      'pca_y': 0.08821492621241106},\n",
       "     {'t-sne_x': 17.398519494676982, 't-sne_y': -5.456073924778392},\n",
       "     {'umap_x': 8.526586532592773, 'umap_y': 7.08863639831543}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48432910-C-G',\n",
       "      'distance': 0.0026995192747563124,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.0028567169792950153,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.002968297805637121,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.0032998528331518173,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.0033634935971349478,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495162-C-G',\n",
       "      'distance': 0.0033641213085502386,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.0033779297955334187,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489991-C-G',\n",
       "      'distance': 0.00348075944930315,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.003557187505066395,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.0035585183650255203,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.003635549917817116,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0036893938668072224,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488439-T-C',\n",
       "      'distance': 0.003746968461200595,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.0037518274039030075,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.003804400796070695,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.0038192870561033487,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.003821253078058362,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.0038394914008677006,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.003948461264371872,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.004042666871100664,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48489851-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48489851\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48489851',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3082G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48489851C>A',\n",
       "     'hgvs_protein': 'p.Asp1028Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"start\": 48489851, \"end\": 48489851, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM106076\", \"phenotype_or_disease\": 1, \"strand\": 1}], \"transcript_consequences\": [{\"ada_score\": 0.999992895158708, \"given_ref\": \"C\", \"sift_score\": 0, \"cds_end\": 3082, \"biotype\": \"protein_coding\", \"cadd_raw\": 7.082247, \"mutationtaster_pred\": \"D\", \"maxentscan_ref\": 6.825, \"maxentscan_alt\": -4.118, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"used_ref\": \"C\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.982}, \"location\": \"15:48489851\", \"protein_id\": \"NP_000129.3\", \"aa\": \"C\", \"protein_start\": 1028, \"polyphen_prediction\": \"probably_damaging\", \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48489850, \"end\": 48489851}}}, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"cds_start\": 3082, \"impact\": \"MODERATE\", \"rf_score\": 1, \"maxentscan_diff\": 10.943, \"cdna_start\": 3398, \"cdna_end\": 3398, \"hgvsp\": \"NP_000129.3:p.Asp1028Tyr\", \"gene_id\": \"2200\", \"mastermind_mmid3\": \"FBN1:D1028Y,FBN1:D1028srd\", \"amino_acids\": \"D/Y\", \"codons\": \"Gat/Tat\", \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"blosum62\": -3, \"exon\": \"25/66\", \"lrt_pred\": \"invalid_field\", \"spliceai\": {\"DS_DG\": 0.16, \"DS_AL\": 0, \"DP_DL\": 0, \"DP_AL\": -1, \"DS_DL\": 0.44, \"DP_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DG\": -7, \"DS_AG\": 0}, \"canonical\": 1, \"variant_allele\": \"A\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.999338567256927, \"strand\": -1, \"mastermind_counts\": \"5|10|14\", \"hgvsc\": \"NM_000138.5:c.3082G>T\", \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.984, \"cadd_phred\": 35, \"protein_end\": 1028}], \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48489851C>A\", \"vcf_string\": \"15-48489851-C-A\", \"allele_string\": \"C/A\", \"assembly_name\": \"GRCh38\", \"start\": 48489851, \"minimised\": 1, \"end\": 48489851, \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48489851C>A\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': -0.039475673030629294,\n",
       "      'pca_y': 0.060181601857175825},\n",
       "     {'t-sne_x': -12.617038160455401, 't-sne_y': -4.222766730169553},\n",
       "     {'umap_x': 0.2317955195903778, 'umap_y': 15.114107131958008}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.002566469367593527,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.003920316696166992,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.004113798029720783,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.004210241138935089,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.004824069328606129,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.004825351294130087,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.0048255096189677715,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.004929321818053722,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.005383022129535675,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.005489714443683624,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.005607034545391798,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.0060643041506409645,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.006140314042568207,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.006323222070932388,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.00658731535077095,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.006902594584971666,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.007366175297647715,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.007515536621212959,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.007603096775710583,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487075-C-T',\n",
       "      'distance': 0.007749815471470356,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48488481-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48488481\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488481',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3095G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488481C>T',\n",
       "     'hgvs_protein': 'p.Cys1032Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48488481C>T\", \"end\": 48488481, \"variant_class\": \"SNV\", \"minimised\": 1, \"start\": 48488481, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\", \"vcf_string\": \"15-48488481-C-T\", \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48488481C>T\", \"strand\": 1, \"colocated_variants\": [{\"end\": 48488481, \"start\": 48488481, \"id\": \"CM010036\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"seq_region_name\": \"15\", \"var_synonyms\": {\"COSMIC\": [\"COSM433983\"]}, \"id\": \"COSV57321074\", \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"somatic\": 1, \"end\": 48488481, \"start\": 48488481}, {\"seq_region_name\": \"15\", \"strand\": 1, \"clin_sig_allele\": \"T:pathogenic\", \"phenotype_or_disease\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000532010\", \"RCV000017926\", \"VCV000016462\"], \"UniProt\": [\"VAR_076044\"], \"OMIM\": [134797.0043]}, \"pubmed\": [17701892, 15287423], \"id\": \"rs137854481\", \"allele_string\": \"C/T\", \"clin_sig\": [\"pathogenic\"], \"end\": 48488481, \"start\": 48488481}], \"transcript_consequences\": [{\"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 1032, \"protein_id\": \"NP_000129.3\", \"aa\": \"C\", \"location\": \"15:48488481\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"start\": 48488480, \"type\": \"SimpleInterval\", \"end\": 48488481}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"loeuf\": 0.105, \"consequence_terms\": [\"missense_variant\"], \"cds_start\": 3095, \"eve_score\": 0.791285703937675, \"impact\": \"MODERATE\", \"cdna_end\": 3411, \"cdna_start\": 3411, \"hgvsp\": \"NP_000129.3:p.Cys1032Tyr\", \"gene_id\": \"2200\", \"sift_score\": 0, \"given_ref\": \"C\", \"biotype\": \"protein_coding\", \"cadd_raw\": 5.302928, \"cds_end\": 3095, \"mutationtaster_pred\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"used_ref\": \"C\", \"alphamissense\": {\"am_pathogenicity\": 0.9991, \"am_class\": \"likely_pathogenic\"}, \"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.996, \"cadd_phred\": 29.8, \"eve_class\": \"Pathogenic\", \"protein_end\": 1032, \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"C/Y\", \"mastermind_mmid3\": \"FBN1:C1032Y\", \"codons\": \"tGc/tAc\", \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 12, \"DS_AG\": 0, \"DP_DG\": 8, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": 12, \"DP_AL\": -16}, \"blosum62\": -2, \"exon\": \"26/66\", \"lrt_pred\": \"invalid_field\", \"canonical\": 1, \"mastermind_counts\": \"13|15|15\", \"variant_allele\": \"T\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.998610583227561, \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.3095G>A\"}]}',\n",
       "    'coordinates': [{'pca_x': 0.03775444691479207,\n",
       "      'pca_y': 0.08149037222125449},\n",
       "     {'t-sne_x': 7.689176629256104, 't-sne_y': 9.552956797729633},\n",
       "     {'umap_x': 3.0339088439941406, 'umap_y': 2.4027774333953857}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0007641569245606661,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.0008187376079149544,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.0008884940179996192,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415695-C-T',\n",
       "      'distance': 0.0009012500522658229,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.0010026930831372738,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.0010231399210169911,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.0011084969155490398,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.001136214006692171,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0011994385858997703,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468456-C-T',\n",
       "      'distance': 0.001219515223056078,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474293-C-T',\n",
       "      'distance': 0.0012981499312445521,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.0013193836202844977,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492574-C-T',\n",
       "      'distance': 0.0013251653872430325,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.0013544437242671847,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.0013604654232040048,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.0014046046417206526,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0014147285837680101,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434599-C-T',\n",
       "      'distance': 0.0014397480990737677,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.0014559626579284668,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481699-C-T',\n",
       "      'distance': 0.00146795017644763,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488460-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48488460\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488460',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3116G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488460C>T',\n",
       "     'hgvs_protein': 'p.Cys1039Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"location\": \"15:48488460\", \"aa\": \"C\", \"protein_start\": 1039, \"polyphen_prediction\": \"probably_damaging\", \"loeuf\": 0.105, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48488459, \"type\": \"SimpleInterval\", \"end\": 48488460}}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}}, \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.775898066435472, \"cds_start\": 3116, \"impact\": \"MODERATE\", \"cdna_start\": 3432, \"cdna_end\": 3432, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys1039Tyr\", \"sift_score\": 0, \"given_ref\": \"C\", \"cds_end\": 3116, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.115309, \"mutationtaster_pred\": \"D\", \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"used_ref\": \"C\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9993}, \"sift_prediction\": \"deleterious_low_confidence\", \"eve_class\": \"Pathogenic\", \"polyphen_score\": 0.974, \"cadd_phred\": 28.6, \"protein_end\": 1039, \"amino_acids\": \"C/Y\", \"mastermind_mmid3\": \"FBN1:C1039Y\", \"codons\": \"tGc/tAc\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"lrt_pred\": \"invalid_field\", \"exon\": \"26/66\", \"blosum62\": -2, \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 33, \"DP_DG\": -8, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0.01, \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": 33, \"DP_AL\": 16}, \"canonical\": 1, \"variant_allele\": \"T\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.989441659924185, \"strand\": -1, \"mastermind_counts\": \"8|19|19\", \"hgvsc\": \"NM_000138.5:c.3116G>A\"}], \"colocated_variants\": [{\"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM972803\", \"phenotype_or_disease\": 1, \"strand\": 1, \"start\": 48488460, \"end\": 48488460}, {\"start\": 48488460, \"end\": 48488460, \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"T:pathogenic\", \"strand\": 1, \"allele_string\": \"C/T\", \"clin_sig\": [\"pathogenic\"], \"id\": \"rs2141295277\", \"pubmed\": [17701892], \"var_synonyms\": {\"ClinVar\": [\"RCV001381677\", \"VCV001069719\"]}, \"seq_region_name\": \"15\"}], \"vcf_string\": \"15-48488460-C-T\", \"input\": \"NC_000015.10:g.48488460C>T\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"assembly_name\": \"GRCh38\", \"start\": 48488460, \"allele_string\": \"C/T\", \"id\": \"NC_000015.10:g.48488460C>T\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"SNV\", \"end\": 48488460}',\n",
       "    'coordinates': [{'pca_x': 0.036110020395370424,\n",
       "      'pca_y': 0.08001050343715138},\n",
       "     {'t-sne_x': 7.422318519033603, 't-sne_y': 9.227770136113485},\n",
       "     {'umap_x': 3.218602180480957, 'umap_y': 2.5375819206237793}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.0005155563703738153,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0005252714618109167,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.0006499814335256815,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415695-C-T',\n",
       "      'distance': 0.0008291095145978034,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.0008608894422650337,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.000872151053044945,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.0008988047484308481,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.0009051390225067735,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.000908997724764049,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.0009224626119248569,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468456-C-T',\n",
       "      'distance': 0.0009470690856687725,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.000997563824057579,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495176-C-T',\n",
       "      'distance': 0.0010299498680979013,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495173-C-T',\n",
       "      'distance': 0.00103258085437119,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.001032639411278069,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0010368884541094303,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428390-C-T',\n",
       "      'distance': 0.0010378679726272821,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468504-C-T',\n",
       "      'distance': 0.0010413770796731114,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520733-C-T',\n",
       "      'distance': 0.0010530224535614252,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481699-C-T',\n",
       "      'distance': 0.0011248198570683599,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488419-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48488419\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488419',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3157T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488419A>G',\n",
       "     'hgvs_protein': 'p.Cys1053Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"A/G\", \"start\": 48488419, \"assembly_name\": \"GRCh38\", \"end\": 48488419, \"variant_class\": \"SNV\", \"minimised\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48488419A>G\", \"transcript_consequences\": [{\"hgvsp\": \"NP_000129.3:p.Cys1053Arg\", \"gene_id\": \"2200\", \"cdna_end\": 3473, \"cdna_start\": 3473, \"impact\": \"MODERATE\", \"cds_start\": 3157, \"eve_score\": 0.718370339563934, \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48488418, \"end\": 48488419}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"loeuf\": 0.105, \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 1053, \"protein_id\": \"NP_000129.3\", \"aa\": \"A\", \"location\": \"15:48488419\", \"used_ref\": \"A\", \"alphamissense\": {\"am_pathogenicity\": 0.9994, \"am_class\": \"likely_pathogenic\"}, \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"mutationtaster_pred\": \"D\", \"cadd_raw\": 5.171442, \"biotype\": \"protein_coding\", \"cds_end\": 3157, \"sift_score\": 0, \"given_ref\": \"A\", \"protein_end\": 1053, \"polyphen_score\": 0.918, \"cadd_phred\": 29, \"eve_class\": \"Pathogenic\", \"sift_prediction\": \"deleterious_low_confidence\", \"hgvsc\": \"NM_000138.5:c.3157T>C\", \"mastermind_counts\": \"13|18|18\", \"variant_allele\": \"G\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.999938368797302, \"strand\": -1, \"canonical\": 1, \"spliceai\": {\"DP_AG\": -4, \"DS_DL\": 0, \"DP_DG\": -1, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_AL\": 0, \"DS_DG\": 0, \"DP_AL\": 0, \"DP_DL\": 33}, \"exon\": \"26/66\", \"blosum62\": -3, \"lrt_pred\": \"invalid_field\", \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"C/R\", \"mastermind_mmid3\": \"FBN1:C1053R\", \"codons\": \"Tgc/Cgc\"}], \"colocated_variants\": [{\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM960630\", \"start\": 48488419, \"end\": 48488419}], \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48488419A>G\", \"vcf_string\": \"15-48488419-A-G\"}',\n",
       "    'coordinates': [{'pca_x': 0.04526976846761788,\n",
       "      'pca_y': 0.0776869963767633},\n",
       "     {'t-sne_x': 35.66575029221654, 't-sne_y': 1.9481827119297601},\n",
       "     {'umap_x': 1.1170628070831299, 'umap_y': 6.639495372772217}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.000744304561521858,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489956-A-G',\n",
       "      'distance': 0.0008130824426189065,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0008987183682620525,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0009482967434450984,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.001037676353007555,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.0010459336917847395,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.0010952057782560587,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468418-A-G',\n",
       "      'distance': 0.0011282149935141206,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0011700143804773688,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0011854261392727494,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596319-A-G',\n",
       "      'distance': 0.0011937522795051336,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0012972349068149924,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.00131797487847507,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.0013517769984900951,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0013820743188261986,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0013867730740457773,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.0013883933424949646,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0014337766915559769,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.0014829467982053757,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456688-A-G',\n",
       "      'distance': 0.0014902425464242697,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488413-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48488413\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488413',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3163T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488413A>C',\n",
       "     'hgvs_protein': 'p.Cys1055Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"protein_end\": 1055, \"eve_class\": \"Pathogenic\", \"polyphen_score\": 0.985, \"cadd_phred\": 29.6, \"sift_prediction\": \"deleterious_low_confidence\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"C\", \"clinpred\": 0.999852418899536, \"strand\": -1, \"mastermind_counts\": \"11|15|15\", \"hgvsc\": \"NM_000138.5:c.3163T>G\", \"blosum62\": -3, \"lrt_pred\": \"invalid_field\", \"exon\": \"26/66\", \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DG\": -45, \"DS_AG\": 0, \"DS_DL\": 0.01, \"DP_AG\": 35, \"DP_DL\": 2, \"DP_AL\": 6, \"DS_DG\": 0, \"DS_AL\": 0}, \"canonical\": 1, \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"codons\": \"Tgt/Ggt\", \"mastermind_mmid3\": \"FBN1:C1055G\", \"amino_acids\": \"C/G\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"cdna_start\": 3479, \"cdna_end\": 3479, \"hgvsp\": \"NP_000129.3:p.Cys1055Gly\", \"gene_id\": \"2200\", \"eve_score\": 0.703549149770621, \"cds_start\": 3163, \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\"], \"aa\": \"A\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48488413\", \"protein_start\": 1055, \"polyphen_prediction\": \"probably_damaging\", \"loeuf\": 0.105, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48488412, \"end\": 48488413}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}}, \"used_ref\": \"A\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9882}, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"D\", \"given_ref\": \"A\", \"sift_score\": 0, \"cds_end\": 3163, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.260434}], \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM056958\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48488413, \"end\": 48488413}, {\"seq_region_name\": \"15\", \"id\": \"CM960631\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"end\": 48488413, \"start\": 48488413}, {\"end\": 48488413, \"start\": 48488413, \"seq_region_name\": \"15\", \"strand\": 1, \"var_synonyms\": {\"UniProt\": [\"VAR_002304\"]}, \"id\": \"rs1597564258\", \"allele_string\": \"A/C\"}], \"vcf_string\": \"15-48488413-A-C\", \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48488413A>C\", \"strand\": 1, \"assembly_name\": \"GRCh38\", \"start\": 48488413, \"allele_string\": \"A/C\", \"id\": \"NC_000015.10:g.48488413A>C\", \"seq_region_name\": \"15\", \"minimised\": 1, \"end\": 48488413, \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.03903294991087308,\n",
       "      'pca_y': 0.08034408428929293},\n",
       "     {'t-sne_x': 20.29171159065879, 't-sne_y': 3.533085243178586},\n",
       "     {'umap_x': 3.6681571006774902, 'umap_y': 6.250564098358154}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.0009457850246690214,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.0009545122156850994,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.000975236005615443,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488419-A-C',\n",
       "      'distance': 0.0011837320635095239,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.0011944888392463326,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.0012782004196196795,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.0014550641644746065,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.0014633233658969402,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520677-A-C',\n",
       "      'distance': 0.0016037870664149523,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.0028894986025989056,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.003110974794253707,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415582-C-A',\n",
       "      'distance': 0.0031244494020938873,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-G',\n",
       "      'distance': 0.0031807946506887674,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.003233792260289192,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.003234144765883684,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.003274358343333006,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0033284532837569714,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0033904912415891886,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.003504138672724366,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497358-C-A',\n",
       "      'distance': 0.0035306084901094437,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488411-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48488411\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488411',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3165T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488411A>C',\n",
       "     'hgvs_protein': 'p.Cys1055Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"used_ref\": \"A\", \"alphamissense\": {\"am_pathogenicity\": 0.9984, \"am_class\": \"likely_pathogenic\"}, \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"mutationtaster_pred\": \"A\", \"cadd_raw\": 3.856326, \"biotype\": \"protein_coding\", \"cds_end\": 3165, \"sift_score\": 0, \"given_ref\": \"A\", \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys1055Trp\", \"cdna_end\": 3481, \"cdna_start\": 3481, \"impact\": \"MODERATE\", \"cds_start\": 3165, \"eve_score\": 0.770900723861499, \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48488411, \"type\": \"SimpleInterval\", \"start\": 48488410}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}}, \"loeuf\": 0.105, \"protein_start\": 1055, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48488411\", \"protein_id\": \"NP_000129.3\", \"aa\": \"A\", \"hgvsc\": \"NM_000138.5:c.3165T>G\", \"mastermind_counts\": \"12|18|18\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"C\", \"clinpred\": 0.999903440475464, \"strand\": -1, \"canonical\": 1, \"spliceai\": {\"DP_DG\": 4, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": 4, \"DS_DL\": 0.01, \"DP_AL\": 8, \"DP_DL\": -43, \"DS_AL\": 0, \"DS_DG\": 0.04}, \"exon\": \"26/66\", \"lrt_pred\": \"invalid_field\", \"blosum62\": -2, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"gene_symbol_source\": \"EntrezGene\", \"conservation\": -2.21, \"codons\": \"tgT/tgG\", \"amino_acids\": \"C/W\", \"mastermind_mmid3\": \"FBN1:C1055W\", \"protein_end\": 1055, \"cadd_phred\": 23.7, \"polyphen_score\": 0.997, \"eve_class\": \"Pathogenic\", \"sift_prediction\": \"deleterious_low_confidence\"}], \"colocated_variants\": [{\"start\": 48488411, \"end\": 48488411, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM013923\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\"}, {\"start\": 48488411, \"end\": 48488411, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"clin_sig_allele\": \"T:pathogenic;T:likely_pathogenic;C:likely_pathogenic\", \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"allele_string\": \"A/C/T\", \"var_synonyms\": {\"UniProt\": [\"VAR_017998\"], \"ClinVar\": [\"RCV001782936\", \"RCV001384246\", \"RCV000476172\", \"VCV000406294\", \"RCV000663608\", \"VCV000549136\"]}, \"pubmed\": [17657824], \"id\": \"rs1060501040\"}], \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48488411A>C\", \"strand\": 1, \"vcf_string\": \"15-48488411-A-C\", \"allele_string\": \"A/C\", \"start\": 48488411, \"assembly_name\": \"GRCh38\", \"end\": 48488411, \"variant_class\": \"SNV\", \"minimised\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48488411A>C\"}',\n",
       "    'coordinates': [{'pca_x': 0.03878446283013087,\n",
       "      'pca_y': 0.07990761197919818},\n",
       "     {'t-sne_x': 20.408256044575783, 't-sne_y': 3.9844751769910842},\n",
       "     {'umap_x': 3.8441715240478516, 'umap_y': 6.288352966308594}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.000979261239990592,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.001329566352069378,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.0015710998559370637,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.00165493693202734,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.0017280977917835116,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488419-A-C',\n",
       "      'distance': 0.001927815261296928,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.0019379780860617757,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.0020962944254279137,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520677-A-C',\n",
       "      'distance': 0.0024452011566609144,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.0026877070777118206,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.003471146337687969,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.0035009775310754776,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415582-C-A',\n",
       "      'distance': 0.0035683480091392994,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497358-C-A',\n",
       "      'distance': 0.0037312288768589497,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.0038165294099599123,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.0038778744637966156,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.003892492735758424,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0039037223905324936,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.003999300766736269,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.004000316374003887,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488374-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48488374\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488374',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3202T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488374A>C',\n",
       "     'hgvs_protein': 'p.Cys1068Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"A/C\", \"start\": 48488374, \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"end\": 48488374, \"minimised\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48488374A>C\", \"colocated_variants\": [{\"id\": \"CM098515\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"end\": 48488374, \"start\": 48488374}, {\"start\": 48488374, \"end\": 48488374, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM111083\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\"}, {\"end\": 48488374, \"start\": 48488374, \"seq_region_name\": \"15\", \"strand\": 1, \"clin_sig_allele\": \"G:pathogenic\", \"frequencies\": {\"C\": {\"gnomade_sas\": 0, \"gnomade_mid\": 0, \"gnomade\": 6.84e-07, \"gnomade_asj\": 0, \"gnomade_nfe\": 0, \"gnomade_afr\": 0, \"gnomade_remaining\": 1.656e-05, \"gnomade_amr\": 0, \"gnomade_eas\": 0, \"gnomade_fin\": 0}}, \"phenotype_or_disease\": 1, \"pubmed\": [17701892], \"id\": \"rs1293095681\", \"var_synonyms\": {\"UniProt\": [\"VAR_064503\"], \"ClinVar\": [\"RCV001067486\", \"RCV000663609\", \"VCV000549137\"]}, \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"A/C/G\"}], \"transcript_consequences\": [{\"mastermind_counts\": \"7|10|10\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"C\", \"strand\": -1, \"clinpred\": 0.999897956848145, \"hgvsc\": \"NM_000138.5:c.3202T>G\", \"spliceai\": {\"DP_DG\": 41, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AG\": 45, \"DP_DL\": -6, \"DP_AL\": 4, \"DS_DG\": 0, \"DS_AL\": 0}, \"lrt_pred\": \"invalid_field\", \"exon\": \"26/66\", \"blosum62\": -3, \"canonical\": 1, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"mastermind_mmid3\": \"FBN1:C1068G\", \"amino_acids\": \"C/G\", \"codons\": \"Tgc/Ggc\", \"protein_end\": 1068, \"cadd_phred\": 28.5, \"polyphen_score\": 0.684, \"eve_class\": \"Pathogenic\", \"sift_prediction\": \"deleterious_low_confidence\", \"alphamissense\": {\"am_pathogenicity\": 0.9948, \"am_class\": \"likely_pathogenic\"}, \"used_ref\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"D\", \"sift_score\": 0, \"given_ref\": \"A\", \"biotype\": \"protein_coding\", \"cadd_raw\": 5.087589, \"cds_end\": 3202, \"cdna_end\": 3518, \"cdna_start\": 3518, \"hgvsp\": \"NP_000129.3:p.Cys1068Gly\", \"gene_id\": \"2200\", \"cds_start\": 3202, \"eve_score\": 0.753491830498725, \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\"], \"polyphen_prediction\": \"possibly_damaging\", \"protein_start\": 1068, \"aa\": \"A\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48488374\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48488374, \"start\": 48488373, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}}, \"loeuf\": 0.105}], \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48488374A>C\", \"vcf_string\": \"15-48488374-A-C\"}',\n",
       "    'coordinates': [{'pca_x': 0.04286254191588834,\n",
       "      'pca_y': 0.08433012161262565},\n",
       "     {'t-sne_x': 20.432669977885315, 't-sne_y': 3.649222645820782},\n",
       "     {'umap_x': 3.753136396408081, 'umap_y': 6.156163215637207}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.0011994729284197092,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.0013245036825537682,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488419-A-C',\n",
       "      'distance': 0.001348426565527916,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.001366825308650732,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.001621278002858162,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.0019753966480493546,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.0020059181842952967,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.0021582655608654022,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.0024478076957166195,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520677-A-C',\n",
       "      'distance': 0.002567342249676585,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.0035404646769165993,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.0037148017436265945,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.0037256034556776285,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.003750513307750225,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489875-T-C',\n",
       "      'distance': 0.0037714517675340176,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48489956-A-G',\n",
       "      'distance': 0.0038346045184880495,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415695-C-A',\n",
       "      'distance': 0.0039148349314928055,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.004024475812911987,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508693-A-C',\n",
       "      'distance': 0.004034689161926508,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596319-A-G',\n",
       "      'distance': 0.004075806587934494,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48488368-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48488368\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488368',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3208G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488368C>G',\n",
       "     'hgvs_protein': 'p.Asp1070His',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48488368C>G\", \"variant_class\": \"SNV\", \"end\": 48488368, \"minimised\": 1, \"start\": 48488368, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/G\", \"vcf_string\": \"15-48488368-C-G\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48488368C>G\", \"transcript_consequences\": [{\"protein_end\": 1070, \"polyphen_score\": 0.989, \"cadd_phred\": 35, \"sift_prediction\": \"deleterious_low_confidence\", \"hgvsc\": \"NM_000138.5:c.3208G>C\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.996496021747589, \"variant_allele\": \"G\", \"strand\": -1, \"mastermind_counts\": \"6|12|17\", \"canonical\": 1, \"exon\": \"26/66\", \"blosum62\": -1, \"lrt_pred\": \"invalid_field\", \"spliceai\": {\"DS_DG\": 0.06, \"DS_AL\": 0, \"DP_DL\": 0, \"DP_AL\": 46, \"DS_DL\": 0.94, \"DP_AG\": -32, \"DP_DG\": 47, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\"}, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"mastermind_mmid3\": \"FBN1:D1070H,FBN1:D1070srd\", \"amino_acids\": \"D/H\", \"codons\": \"Gac/Cac\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"hgvsp\": \"NP_000129.3:p.Asp1070His\", \"gene_id\": \"2200\", \"cdna_start\": 3524, \"cdna_end\": 3524, \"impact\": \"MODERATE\", \"rf_score\": 1, \"maxentscan_diff\": 3.568, \"cds_start\": 3208, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"interval\": {\"start\": 48488367, \"type\": \"SimpleInterval\", \"end\": 48488368}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"protein_id\": \"NP_000129.3\", \"location\": \"15:48488368\", \"aa\": \"C\", \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 1070, \"mutfunc\": {\"exp\": {\"dG_wt\": 170.0534, \"dG_mt_sd\": 0.120207, \"dG_wt_sd\": 0.186272, \"ddG\": 1.698, \"dG_mt\": 171.7514, \"ddG_sd\": 0.179496}}, \"used_ref\": \"C\", \"alphamissense\": {\"am_pathogenicity\": 0.9971, \"am_class\": \"likely_pathogenic\"}, \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"maxentscan_alt\": 5.373, \"maxentscan_ref\": 8.941, \"mutationtaster_pred\": \"D\", \"cds_end\": 3208, \"biotype\": \"protein_coding\", \"cadd_raw\": 6.720847, \"ada_score\": 0.999996241550158, \"sift_score\": 0, \"given_ref\": \"C\"}], \"colocated_variants\": [{\"end\": 48488368, \"start\": 48488368, \"somatic\": 1, \"id\": \"COSV100354052\", \"var_synonyms\": {\"COSMIC\": [\"COSM8241919\"]}, \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"start\": 48488368, \"end\": 48488368, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CS102148\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\"}]}',\n",
       "    'coordinates': [{'pca_x': -0.03948940420116759,\n",
       "      'pca_y': 0.053431564864145624},\n",
       "     {'t-sne_x': -13.199766905209382, 't-sne_y': -3.813082846719672},\n",
       "     {'umap_x': 0.350563645362854, 'umap_y': 15.12125301361084}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.004549699369817972,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.0052553340792655945,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.005442276131361723,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.005758208222687244,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.005811826325953007,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.005938400514423847,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.006004880182445049,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.006415714975446463,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.006517579313367605,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.006636884529143572,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.006813647225499153,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.0072521427646279335,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.00803367979824543,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.008063830435276031,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.008360039442777634,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.00872498657554388,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.009017382748425007,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.00924019142985344,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468534-T-C',\n",
       "      'distance': 0.009380027651786804,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.009747076779603958,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488241-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48488241\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488241',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3209A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488241T>C',\n",
       "     'hgvs_protein': 'p.Asp1070Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48488241-T-C\", \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48488241T>C\", \"strand\": 1, \"transcript_consequences\": [{\"cds_end\": 3209, \"cadd_raw\": 5.72662, \"biotype\": \"protein_coding\", \"ada_score\": 0.756627169070259, \"sift_score\": 0, \"given_ref\": \"T\", \"maxentscan_alt\": 7.954, \"mutationtaster_pred\": \"A\", \"maxentscan_ref\": 7.533, \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 0.052041, \"dG_wt\": 170.0118, \"dG_wt_sd\": 0.042481, \"ddG\": 1.034599, \"ddG_sd\": 0.038102, \"dG_mt\": 171.046399}}, \"alphamissense\": {\"am_pathogenicity\": 0.9948, \"am_class\": \"likely_pathogenic\"}, \"used_ref\": \"T\", \"loeuf\": 0.105, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48488240, \"end\": 48488241}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}}, \"aa\": \"T\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48488241\", \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 1070, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"impact\": \"MODERATE\", \"maxentscan_diff\": -0.421, \"rf_score\": 0.644, \"cds_start\": 3209, \"hgvsp\": \"NP_000129.3:p.Asp1070Gly\", \"gene_id\": \"2200\", \"cdna_start\": 3525, \"cdna_end\": 3525, \"mastermind_mmid3\": \"FBN1:D1070G,FBN1:D1070sra\", \"amino_acids\": \"D/G\", \"codons\": \"gAc/gGc\", \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"canonical\": 1, \"lrt_pred\": \"invalid_field\", \"exon\": \"27/66\", \"blosum62\": -1, \"spliceai\": {\"DP_DL\": -41, \"DP_AL\": -40, \"DS_DG\": 0, \"DS_AL\": 0.01, \"SYMBOL\": \"FBN1\", \"DP_DG\": 1, \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AG\": 0}, \"hgvsc\": \"NM_000138.5:c.3209A>G\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"C\", \"clinpred\": 0.99284090563433, \"strand\": -1, \"mastermind_counts\": \"10|24|26\", \"sift_prediction\": \"deleterious_low_confidence\", \"cadd_phred\": 33, \"polyphen_score\": 0.951, \"protein_end\": 1070}], \"colocated_variants\": [{\"start\": 48488241, \"end\": 48488241, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM020686\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\"}, {\"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"var_synonyms\": {\"COSMIC\": [\"COSM5507425\"]}, \"id\": \"COSV57317979\", \"seq_region_name\": \"15\", \"start\": 48488241, \"end\": 48488241, \"somatic\": 1}, {\"end\": 48488241, \"start\": 48488241, \"seq_region_name\": \"15\", \"strand\": 1, \"clin_sig_allele\": \"C:likely_pathogenic\", \"phenotype_or_disease\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV003330874\", \"RCV000663613\", \"VCV000549141\"]}, \"id\": \"rs1555398648\", \"clin_sig\": [\"likely_pathogenic\"], \"allele_string\": \"T/C\"}], \"id\": \"NC_000015.10:g.48488241T>C\", \"seq_region_name\": \"15\", \"minimised\": 1, \"end\": 48488241, \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"start\": 48488241, \"allele_string\": \"T/C\"}',\n",
       "    'coordinates': [{'pca_x': -0.04041607102721605,\n",
       "      'pca_y': 0.05134965875202931},\n",
       "     {'t-sne_x': -13.043579630435561, 't-sne_y': -3.822446617887351},\n",
       "     {'umap_x': 0.3508422374725342, 'umap_y': 15.06040096282959}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.0009515685960650444,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.0010975564364343882,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.0028456361033022404,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.0029039019718766212,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.0029053327161818743,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.0029681511223316193,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.003207542933523655,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.0034196344204247,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.003437221050262451,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.0038697225973010063,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.003969382960349321,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.004178700968623161,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.004441285040229559,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468534-T-C',\n",
       "      'distance': 0.004453070927411318,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.004668152891099453,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.005020590499043465,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.0051325904205441475,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.005234406795352697,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.005560205318033695,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.005747135728597641,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488235-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48488235\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488235',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3215A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488235T>C',\n",
       "     'hgvs_protein': 'p.Asp1072Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"start\": 48488235, \"allele_string\": \"T/C\", \"id\": \"NC_000015.10:g.48488235T>C\", \"seq_region_name\": \"15\", \"minimised\": 1, \"end\": 48488235, \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"codons\": \"gAc/gGc\", \"amino_acids\": \"D/G\", \"mastermind_mmid3\": \"FBN1:D1072G\", \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 39, \"SYMBOL\": \"FBN1\", \"DP_DG\": -12, \"DS_AG\": 0, \"DS_DG\": 0, \"DS_AL\": 0.01, \"DP_DL\": -34, \"DP_AL\": -34}, \"lrt_pred\": \"invalid_field\", \"exon\": \"27/66\", \"blosum62\": -1, \"canonical\": 1, \"mastermind_counts\": \"11|13|13\", \"variant_allele\": \"C\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"clinpred\": 0.986658751964569, \"hgvsc\": \"NM_000138.5:c.3215A>G\", \"sift_prediction\": \"deleterious_low_confidence\", \"cadd_phred\": 29.7, \"polyphen_score\": 0.928, \"eve_class\": \"Uncertain\", \"protein_end\": 1072, \"given_ref\": \"T\", \"sift_score\": 0, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.286296, \"cds_end\": 3215, \"mutationtaster_pred\": \"D\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9114}, \"used_ref\": \"T\", \"mutfunc\": {\"exp\": {\"ddG\": 0.615799, \"dG_wt_sd\": 0.109506, \"dG_mt_sd\": 0.184655, \"dG_wt\": 171.6208, \"ddG_sd\": 0.129871, \"dG_mt\": 172.236599}}, \"protein_start\": 1072, \"polyphen_prediction\": \"probably_damaging\", \"protein_id\": \"NP_000129.3\", \"aa\": \"T\", \"location\": \"15:48488235\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"end\": 48488235, \"start\": 48488234, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"loeuf\": 0.105, \"consequence_terms\": [\"missense_variant\"], \"cds_start\": 3215, \"eve_score\": 0.529685055265953, \"impact\": \"MODERATE\", \"cdna_end\": 3531, \"cdna_start\": 3531, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Asp1072Gly\"}], \"colocated_variants\": [{\"start\": 48488235, \"end\": 48488235, \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM960633\", \"seq_region_name\": \"15\"}], \"vcf_string\": \"15-48488235-T-C\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48488235T>C\"}',\n",
       "    'coordinates': [{'pca_x': 0.03724415252543528,\n",
       "      'pca_y': 0.08103528528313922},\n",
       "     {'t-sne_x': 2.5811835061628483, 't-sne_y': 1.8891960809391044},\n",
       "     {'umap_x': 5.111954689025879, 'umap_y': 2.9932053089141846}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.0006971551338210702,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487107-T-C',\n",
       "      'distance': 0.0015935349510982633,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.0017244594637304544,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497298-T-C',\n",
       "      'distance': 0.0018607396632432938,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434707-T-C',\n",
       "      'distance': 0.00196444452740252,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487131-T-C',\n",
       "      'distance': 0.002063358435407281,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488148-T-C',\n",
       "      'distance': 0.002074713120236993,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488439-T-C',\n",
       "      'distance': 0.0021098200231790543,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481687-T-C',\n",
       "      'distance': 0.0021123341284692287,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446775-T-C',\n",
       "      'distance': 0.002178985858336091,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485490-T-C',\n",
       "      'distance': 0.0022081825882196426,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472607-T-C',\n",
       "      'distance': 0.002222900278866291,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430727-T-C',\n",
       "      'distance': 0.0022372568491846323,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0023813825100660324,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446816-T-C',\n",
       "      'distance': 0.0023907581344246864,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497382-T-C',\n",
       "      'distance': 0.002397826872766018,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0024470635689795017,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497322-T-C',\n",
       "      'distance': 0.002457191701978445,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0024809641763567924,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489896-C-T',\n",
       "      'distance': 0.0025122498627752066,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488230-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48488230\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488230',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3220T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488230A>G',\n",
       "     'hgvs_protein': 'p.Cys1074Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"start\": 48488230, \"allele_string\": \"A/G\", \"id\": \"NC_000015.10:g.48488230A>G\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"SNV\", \"end\": 48488230, \"transcript_consequences\": [{\"eve_class\": \"Pathogenic\", \"polyphen_score\": 0.918, \"cadd_phred\": 32, \"protein_end\": 1074, \"sift_prediction\": \"deleterious_low_confidence\", \"canonical\": 1, \"lrt_pred\": \"invalid_field\", \"blosum62\": -3, \"exon\": \"27/66\", \"spliceai\": {\"DP_DG\": -30, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AG\": 11, \"DP_DL\": -29, \"DP_AL\": -29, \"DS_DG\": 0, \"DS_AL\": 0.01}, \"hgvsc\": \"NM_000138.5:c.3220T>C\", \"clinpred\": 0.999938011169434, \"variant_allele\": \"G\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"mastermind_counts\": \"17|22|22\", \"mastermind_mmid3\": \"FBN1:C1074R\", \"codons\": \"Tgc/Cgc\", \"amino_acids\": \"C/R\", \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"impact\": \"MODERATE\", \"eve_score\": 0.76957096406466, \"cds_start\": 3220, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys1074Arg\", \"cdna_start\": 3536, \"cdna_end\": 3536, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48488230, \"start\": 48488229, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}}, \"aa\": \"A\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48488230\", \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 1074, \"consequence_terms\": [\"missense_variant\"], \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"mutfunc\": {\"exp\": {\"ddG\": -0.0862, \"dG_wt_sd\": 0.06289, \"dG_mt_sd\": 0.130426, \"dG_wt\": 172.7772, \"ddG_sd\": 0.07757, \"dG_mt\": 172.691}}, \"used_ref\": \"A\", \"alphamissense\": {\"am_pathogenicity\": 0.9995, \"am_class\": \"likely_pathogenic\"}, \"cds_end\": 3220, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.534887, \"sift_score\": 0, \"given_ref\": \"A\", \"mutationtaster_pred\": \"A\"}], \"colocated_variants\": [{\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM154827\", \"start\": 48488230, \"end\": 48488230}, {\"start\": 48488230, \"end\": 48488230, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM950443\", \"phenotype_or_disease\": 1, \"strand\": 1}, {\"allele_string\": \"COSMIC_MUTATION\", \"var_synonyms\": {\"COSMIC\": [\"COSM5445516\"]}, \"id\": \"COSV57311437\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48488230, \"end\": 48488230, \"somatic\": 1}, {\"clin_sig_allele\": \"G:pathogenic\", \"strand\": 1, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"OMIM\": [134797.0016], \"UniProt\": [\"VAR_002308\"], \"ClinVar\": [\"RCV000017900\", \"VCV000016438\"]}, \"id\": \"rs137854465\", \"pubmed\": [8136837, 8428751], \"allele_string\": \"A/G\", \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"end\": 48488230, \"start\": 48488230}], \"vcf_string\": \"15-48488230-A-G\", \"input\": \"NC_000015.10:g.48488230A>G\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.04741296188215338,\n",
       "      'pca_y': 0.07616678149026856},\n",
       "     {'t-sne_x': 36.28222355277545, 't-sne_y': 1.7261656101426255},\n",
       "     {'umap_x': 1.0793160200119019, 'umap_y': 6.697263240814209}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.0010055601596832275,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.0010725210886448622,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.00113398814573884,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.0011585899628698826,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0011800677748396993,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489956-A-G',\n",
       "      'distance': 0.0012176544405519962,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596319-A-G',\n",
       "      'distance': 0.001261314144358039,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.001280313590541482,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0013527213595807552,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0014196152333170176,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468418-A-G',\n",
       "      'distance': 0.0014443553518503904,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0015304868575185537,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.0015313453041017056,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456688-A-G',\n",
       "      'distance': 0.0015415172092616558,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430770-A-G',\n",
       "      'distance': 0.0015546028735116124,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.001618948532268405,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0016245153965428472,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468490-A-G',\n",
       "      'distance': 0.0016260603442788124,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.0016675116494297981,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0016918466426432133,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488212-GG-G',\n",
       "    'metadata': {'vcf_string': '15\\t48488212\\t.\\tGG\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488212',\n",
       "     'ref_allele': 'GG',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3238del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488213del',\n",
       "     'hgvs_protein': 'p.Leu1080SerfsTer8',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48488212-GG-G\", \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48488213del\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"GG/G\", \"clin_sig\": [\"pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV000255667\", \"VCV000265445\"]}, \"id\": \"rs886039549\", \"start\": 48488212, \"end\": 48488213}], \"transcript_consequences\": [{\"lof\": \"HC\", \"protein_end\": 1080, \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"codons\": \"Ctc/tc\", \"amino_acids\": \"L/X\", \"hgvsc\": \"NM_000138.5:c.3238del\", \"variant_allele\": \"-\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"hgvs_offset\": -1, \"canonical\": 1, \"spliceai\": {\"DP_AG\": -11, \"DS_DL\": 0, \"DP_DG\": -11, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0.04, \"DS_AL\": 0, \"DS_DG\": 0, \"DP_AL\": 29, \"DP_DL\": -12}, \"exon\": \"27/66\", \"consequence_terms\": [\"frameshift_variant\"], \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48488212, \"end\": 48488213}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"loeuf\": 0.105, \"protein_start\": 1080, \"aa\": \"G\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48488212\", \"hgvsp\": \"NP_000129.3:p.Leu1080SerfsTer8\", \"gene_id\": \"2200\", \"cdna_end\": 3554, \"cdna_start\": 3554, \"impact\": \"HIGH\", \"cds_start\": 3238, \"biotype\": \"protein_coding\", \"cds_end\": 3238, \"given_ref\": \"G\", \"lof_info\": \"PERCENTILE:0.375812441968431,GERP_DIST:6053.72533015311,BP_DIST:5340,DIST_FROM_LAST_EXON:4951,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mutfunc\": {\"exp\": {\"dG_wt\": 175.2114, \"dG_mt_sd\": 0.006906, \"dG_wt_sd\": 0.015372, \"ddG\": -0.4012, \"dG_mt\": 174.8102, \"ddG_sd\": 0.009884}}, \"used_ref\": \"G\", \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\"}], \"id\": \"NC_000015.10:g.48488213del\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"deletion\", \"end\": 48488213, \"assembly_name\": \"GRCh38\", \"start\": 48488213, \"allele_string\": \"G/-\"}',\n",
       "    'coordinates': [{'pca_x': 0.04702381167737796,\n",
       "      'pca_y': 0.02938259585097635},\n",
       "     {'t-sne_x': 9.417687531901656, 't-sne_y': 2.7842028747230034},\n",
       "     {'umap_x': 4.289947986602783, 'umap_y': 3.327329635620117}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09296958148479462,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421981-C-T',\n",
       "      'distance': 0.09417195618152618,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.09430626034736633,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09436236321926117,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.09446030855178833,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.09446150809526443,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.0945071205496788,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448851-C-T',\n",
       "      'distance': 0.0945904552936554,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415707-C-T',\n",
       "      'distance': 0.09486159682273865,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.09487994015216827,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474584-C-T',\n",
       "      'distance': 0.09503854066133499,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.09505333006381989,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495162-C-T',\n",
       "      'distance': 0.09525782614946365,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492578-C-T',\n",
       "      'distance': 0.09535763412714005,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.0953984335064888,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437069-C-T',\n",
       "      'distance': 0.09539897739887238,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.0954655334353447,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.09546883404254913,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432907-G-A',\n",
       "      'distance': 0.09547943621873856,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0955435186624527,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48488209-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48488209\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488209',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3241T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488209A>C',\n",
       "     'hgvs_protein': 'p.Cys1081Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48488209A>C\", \"variant_class\": \"SNV\", \"end\": 48488209, \"minimised\": 1, \"start\": 48488209, \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/C\", \"vcf_string\": \"15-48488209-A-C\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48488209A>C\", \"transcript_consequences\": [{\"cdna_end\": 3557, \"cdna_start\": 3557, \"hgvsp\": \"NP_000129.3:p.Cys1081Gly\", \"gene_id\": \"2200\", \"cds_start\": 3241, \"eve_score\": 0.767781474755326, \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\"], \"polyphen_prediction\": \"possibly_damaging\", \"protein_start\": 1081, \"aa\": \"A\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48488209\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48488209, \"start\": 48488208, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}}, \"loeuf\": 0.105, \"used_ref\": \"A\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9943}, \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.054112, \"dG_mt\": 174.36901, \"ddG\": -1.00879, \"dG_mt_sd\": 0, \"dG_wt_sd\": 0.054112, \"dG_wt\": 175.3778}}, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"D\", \"sift_score\": 0, \"given_ref\": \"A\", \"biotype\": \"protein_coding\", \"cadd_raw\": 5.312648, \"cds_end\": 3241, \"protein_end\": 1081, \"cadd_phred\": 29.9, \"polyphen_score\": 0.83, \"eve_class\": \"Pathogenic\", \"sift_prediction\": \"deleterious_low_confidence\", \"mastermind_counts\": \"6|6|6\", \"variant_allele\": \"C\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"clinpred\": 0.999852418899536, \"hgvsc\": \"NM_000138.5:c.3241T>G\", \"spliceai\": {\"DP_DL\": -8, \"DP_AL\": -8, \"DS_DG\": 0, \"DS_AL\": 0.02, \"SYMBOL\": \"FBN1\", \"DP_DG\": 0, \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AG\": 32}, \"blosum62\": -3, \"exon\": \"27/66\", \"lrt_pred\": \"invalid_field\", \"canonical\": 1, \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"codons\": \"Tgt/Ggt\", \"mastermind_mmid3\": \"FBN1:C1081G\", \"amino_acids\": \"C/G\"}], \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM020687\", \"seq_region_name\": \"15\", \"start\": 48488209, \"end\": 48488209}]}',\n",
       "    'coordinates': [{'pca_x': 0.04239714868021016,\n",
       "      'pca_y': 0.08266567208608488},\n",
       "     {'t-sne_x': 20.520946195736613, 't-sne_y': 3.7309540617958423},\n",
       "     {'umap_x': 3.834273099899292, 'umap_y': 6.12905216217041}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.0010285339085385203,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488419-A-C',\n",
       "      'distance': 0.0012023414019495249,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.0013098790077492595,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.0015240679495036602,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.0016815310809761286,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.0017844708636403084,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.0017980493139475584,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.002152472035959363,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.002277533058077097,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520677-A-C',\n",
       "      'distance': 0.002622223226353526,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.0030502660665661097,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489956-A-G',\n",
       "      'distance': 0.0033132503740489483,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0033307448029518127,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.003397179301828146,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.0035710560623556376,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0036320840008556843,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415695-C-A',\n",
       "      'distance': 0.0036452533677220345,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.0036760964430868626,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489980-C-T',\n",
       "      'distance': 0.003715939586982131,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.003765876404941082,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488194-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48488194\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488194',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3256T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488194A>G',\n",
       "     'hgvs_protein': 'p.Cys1086Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48488194, \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/G\", \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48488194A>G\", \"variant_class\": \"SNV\", \"end\": 48488194, \"minimised\": 1, \"colocated_variants\": [{\"start\": 48488194, \"end\": 48488194, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM078031\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\"}, {\"seq_region_name\": \"15\", \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"A/C/G\", \"var_synonyms\": {\"ClinVar\": [\"RCV004702270\", \"RCV001216980\", \"VCV000946172\", \"RCV000663619\", \"VCV000549147\"]}, \"id\": \"rs1555398637\", \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"G:pathogenic;C:pathogenic\", \"strand\": 1, \"start\": 48488194, \"end\": 48488194}], \"transcript_consequences\": [{\"eve_class\": \"Pathogenic\", \"polyphen_score\": 0.918, \"cadd_phred\": 32, \"protein_end\": 1086, \"sift_prediction\": \"deleterious_low_confidence\", \"blosum62\": -3, \"exon\": \"27/66\", \"lrt_pred\": \"invalid_field\", \"spliceai\": {\"DP_DL\": 6, \"DP_AL\": 5, \"DS_DG\": 0, \"DS_AL\": 0, \"DS_AG\": 0, \"DP_DG\": -2, \"SYMBOL\": \"FBN1\", \"DS_DL\": 0, \"DP_AG\": 7}, \"canonical\": 1, \"strand\": -1, \"variant_allele\": \"G\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.999938368797302, \"mastermind_counts\": \"9|10|10\", \"hgvsc\": \"NM_000138.5:c.3256T>C\", \"amino_acids\": \"C/R\", \"codons\": \"Tgt/Cgt\", \"mastermind_mmid3\": \"FBN1:C1086R\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"eve_score\": 0.791732662847737, \"cds_start\": 3256, \"impact\": \"MODERATE\", \"cdna_start\": 3572, \"cdna_end\": 3572, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys1086Arg\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48488194\", \"aa\": \"A\", \"protein_start\": 1086, \"polyphen_prediction\": \"probably_damaging\", \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48488193, \"type\": \"SimpleInterval\", \"end\": 48488194}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"consequence_terms\": [\"missense_variant\"], \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"alphamissense\": {\"am_pathogenicity\": 0.9994, \"am_class\": \"likely_pathogenic\"}, \"used_ref\": \"A\", \"mutfunc\": {\"exp\": {\"dG_mt\": 177.9088, \"ddG_sd\": 0.730931, \"ddG\": 4.1086, \"dG_wt\": 173.8002, \"dG_mt_sd\": 1.369917, \"dG_wt_sd\": 1.161649}}, \"given_ref\": \"A\", \"sift_score\": 0, \"cds_end\": 3256, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.446846, \"mutationtaster_pred\": \"A\"}], \"vcf_string\": \"15-48488194-A-G\", \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48488194A>G\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.052865959786096095,\n",
       "      'pca_y': 0.07780393843608646},\n",
       "     {'t-sne_x': 34.521878839357065, 't-sne_y': 3.7698005253230398},\n",
       "     {'umap_x': 1.396363615989685, 'umap_y': 6.467901706695557}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0019065662054345012,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.001926581608131528,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526223-A-G',\n",
       "      'distance': 0.001942370319738984,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.0020229117944836617,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.002101870719343424,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0021159304305911064,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489956-A-G',\n",
       "      'distance': 0.002209136262536049,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.002289143390953541,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515396-C-A',\n",
       "      'distance': 0.002336017321795225,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.0023898580111563206,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0024001989513635635,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.002408233005553484,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.002430180087685585,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.0025064931251108646,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596325-A-G',\n",
       "      'distance': 0.002511365106329322,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0025552744045853615,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488433-A-G',\n",
       "      'distance': 0.0025755283422768116,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434702-A-G',\n",
       "      'distance': 0.0026444231625646353,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481733-A-G',\n",
       "      'distance': 0.002648136578500271,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.0026610475033521652,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48488193-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48488193\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48488193',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3257G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48488193C>T',\n",
       "     'hgvs_protein': 'p.Cys1086Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"start\": 48488193, \"end\": 48488193, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM992323\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\"}, {\"clin_sig_allele\": \"T:pathogenic\", \"strand\": 1, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000017929\", \"VCV000016465\"], \"OMIM\": [134797.0046]}, \"id\": \"rs137854484\", \"pubmed\": [17366579], \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"C/T\", \"seq_region_name\": \"15\", \"end\": 48488193, \"start\": 48488193}], \"transcript_consequences\": [{\"mutfunc\": {\"exp\": {\"ddG\": 11.2506, \"dG_mt_sd\": 1.29467, \"dG_wt_sd\": 1.461054, \"dG_wt\": 173.6842, \"dG_mt\": 184.9348, \"ddG_sd\": 0.970855}}, \"alphamissense\": {\"am_pathogenicity\": 0.999, \"am_class\": \"likely_pathogenic\"}, \"used_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"mutationtaster_pred\": \"A\", \"cadd_raw\": 5.526362, \"biotype\": \"protein_coding\", \"cds_end\": 3257, \"sift_score\": 0, \"given_ref\": \"C\", \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys1086Tyr\", \"cdna_end\": 3573, \"cdna_start\": 3573, \"impact\": \"MODERATE\", \"cds_start\": 3257, \"eve_score\": 0.737104821510196, \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"end\": 48488193, \"start\": 48488192, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}}, \"loeuf\": 0.105, \"polyphen_prediction\": \"probably_damaging\", \"protein_start\": 1086, \"location\": \"15:48488193\", \"protein_id\": \"NP_000129.3\", \"aa\": \"C\", \"hgvsc\": \"NM_000138.5:c.3257G>A\", \"mastermind_counts\": \"12|13|13\", \"variant_allele\": \"T\", \"strand\": -1, \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.999957799911499, \"canonical\": 1, \"spliceai\": {\"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": 1, \"DP_AL\": 6, \"DS_DL\": 0, \"DP_AG\": 8, \"SYMBOL\": \"FBN1\", \"DP_DG\": 3, \"DS_AG\": 0.01}, \"blosum62\": -2, \"lrt_pred\": \"invalid_field\", \"exon\": \"27/66\", \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"mastermind_mmid3\": \"FBN1:C1086Y\", \"codons\": \"tGt/tAt\", \"amino_acids\": \"C/Y\", \"protein_end\": 1086, \"polyphen_score\": 0.918, \"cadd_phred\": 32, \"eve_class\": \"Pathogenic\", \"sift_prediction\": \"deleterious_low_confidence\"}], \"vcf_string\": \"15-48488193-C-T\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48488193C>T\", \"assembly_name\": \"GRCh38\", \"start\": 48488193, \"allele_string\": \"C/T\", \"id\": \"NC_000015.10:g.48488193C>T\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"SNV\", \"end\": 48488193}',\n",
       "    'coordinates': [{'pca_x': 0.04218411630769209,\n",
       "      'pca_y': 0.0818154776831507},\n",
       "     {'t-sne_x': 7.926329025920214, 't-sne_y': 8.478714578673168},\n",
       "     {'umap_x': 3.125066041946411, 'umap_y': 2.3786299228668213}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.0010436727898195386,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.001223900937475264,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0012937100837007165,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.0013405362842604518,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0014197059208527207,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.0014425842091441154,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.0014527958119288087,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0014660342130810022,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481699-C-T',\n",
       "      'distance': 0.0015264727408066392,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.0015264898538589478,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.001543236430734396,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.0015535785350948572,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492574-C-T',\n",
       "      'distance': 0.0016128618735820055,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-T',\n",
       "      'distance': 0.0016456980956718326,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.0016513911541551352,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415695-C-T',\n",
       "      'distance': 0.0016700502019375563,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474293-C-T',\n",
       "      'distance': 0.0017037757206708193,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465573-C-T',\n",
       "      'distance': 0.0017407566774636507,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.0017771365819498897,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497343-C-T',\n",
       "      'distance': 0.001779941376298666,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48487426-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48487426\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487426',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3349T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487426A>G',\n",
       "     'hgvs_protein': 'p.Cys1117Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"A/G\", \"assembly_name\": \"GRCh38\", \"start\": 48487426, \"minimised\": 1, \"variant_class\": \"SNV\", \"end\": 48487426, \"id\": \"NC_000015.10:g.48487426A>G\", \"seq_region_name\": \"15\", \"transcript_consequences\": [{\"consequence_terms\": [\"missense_variant\"], \"loeuf\": 0.105, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48487426, \"type\": \"SimpleInterval\", \"start\": 48487425}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"protein_id\": \"NP_000129.3\", \"aa\": \"A\", \"location\": \"15:48487426\", \"protein_start\": 1117, \"polyphen_prediction\": \"probably_damaging\", \"hgvsp\": \"NP_000129.3:p.Cys1117Arg\", \"gene_id\": \"2200\", \"cdna_start\": 3665, \"cdna_end\": 3665, \"impact\": \"MODERATE\", \"eve_score\": 0.761041986696184, \"cds_start\": 3349, \"mutationtaster_pred\": \"D\", \"cds_end\": 3349, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.315862, \"given_ref\": \"A\", \"sift_score\": 0, \"mutfunc\": {\"exp\": {\"ddG\": 1.3674, \"dG_wt_sd\": 0.345477, \"dG_mt_sd\": 0.436904, \"dG_wt\": 169.5208, \"dG_mt\": 170.8882, \"ddG_sd\": 0.1611}}, \"used_ref\": \"A\", \"alphamissense\": {\"am_pathogenicity\": 0.9989, \"am_class\": \"likely_pathogenic\"}, \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 1117, \"eve_class\": \"Pathogenic\", \"cadd_phred\": 29.9, \"polyphen_score\": 0.997, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"mastermind_mmid3\": \"FBN1:C1117R\", \"codons\": \"Tgt/Cgt\", \"amino_acids\": \"C/R\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"hgvsc\": \"NM_000138.5:c.3349T>C\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.999883413314819, \"variant_allele\": \"G\", \"strand\": -1, \"mastermind_counts\": \"12|13|13\", \"canonical\": 1, \"blosum62\": -3, \"lrt_pred\": \"invalid_field\", \"exon\": \"28/66\", \"spliceai\": {\"DP_DL\": 0, \"DP_AL\": -49, \"DS_DG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DG\": -27, \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AG\": -6}}], \"colocated_variants\": [{\"end\": 48487426, \"start\": 48487426, \"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM111085\", \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\"}, {\"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM960635\", \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"end\": 48487426, \"start\": 48487426}, {\"start\": 48487426, \"end\": 48487426, \"seq_region_name\": \"15\", \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"A/G\", \"pubmed\": [8281141, 17701892], \"id\": \"rs2141293770\", \"var_synonyms\": {\"UniProt\": [\"VAR_076049\"], \"ClinVar\": [\"RCV001890903\", \"VCV001390245\"]}, \"phenotype_or_disease\": 1, \"strand\": 1, \"clin_sig_allele\": \"G:pathogenic\"}], \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48487426A>G\", \"vcf_string\": \"15-48487426-A-G\"}',\n",
       "    'coordinates': [{'pca_x': 0.04434352726964469,\n",
       "      'pca_y': 0.07371406537824479},\n",
       "     {'t-sne_x': 34.749200021583206, 't-sne_y': 3.9139412140142493},\n",
       "     {'umap_x': 1.2895326614379883, 'umap_y': 6.375193119049072}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0005121847498230636,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0005465518916025758,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.0007245211745612323,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0008327236864715815,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0008509872714057565,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.0009123318595811725,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.0009348199237138033,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.000989701016806066,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.0009903291938826442,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0010085176909342408,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0010359291918575764,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.0010738254059106112,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.0011471530888229609,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.0011589745990931988,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415567-A-G',\n",
       "      'distance': 0.0011893667979165912,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0012264831457287073,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0012659815838560462,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.0012729496229439974,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.0013052746653556824,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-G',\n",
       "      'distance': 0.0013180015375837684,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48487425-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48487425\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487425',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3350G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487425C>T',\n",
       "     'hgvs_protein': 'p.Cys1117Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48487425-C-T\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48487425C>T\", \"transcript_consequences\": [{\"cds_start\": 3350, \"eve_score\": 0.757591201085127, \"impact\": \"MODERATE\", \"cdna_end\": 3666, \"cdna_start\": 3666, \"hgvsp\": \"NP_000129.3:p.Cys1117Tyr\", \"gene_id\": \"2200\", \"protein_start\": 1117, \"polyphen_prediction\": \"probably_damaging\", \"protein_id\": \"NP_000129.3\", \"aa\": \"C\", \"location\": \"15:48487425\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"end\": 48487425, \"start\": 48487424, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"loeuf\": 0.105, \"consequence_terms\": [\"missense_variant\"], \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"used_ref\": \"C\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9983}, \"mutfunc\": {\"exp\": {\"ddG\": 3.8648, \"dG_mt_sd\": 0.929884, \"dG_wt\": 172.8234, \"dG_wt_sd\": 1.002146, \"dG_mt\": 176.6882, \"ddG_sd\": 1.726177}}, \"given_ref\": \"C\", \"sift_score\": 0, \"cadd_raw\": 5.364661, \"biotype\": \"protein_coding\", \"cds_end\": 3350, \"mutationtaster_pred\": \"A\", \"cadd_phred\": 31, \"polyphen_score\": 0.997, \"eve_class\": \"Pathogenic\", \"protein_end\": 1117, \"sift_prediction\": \"deleterious_low_confidence\", \"spliceai\": {\"DP_AG\": -27, \"DS_DL\": 0, \"DP_DG\": 3, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_AL\": 0, \"DS_DG\": 0, \"DP_AL\": 1, \"DP_DL\": -26}, \"blosum62\": -2, \"lrt_pred\": \"invalid_field\", \"exon\": \"28/66\", \"canonical\": 1, \"mastermind_counts\": \"14|28|28\", \"variant_allele\": \"T\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"clinpred\": 0.999876379966736, \"hgvsc\": \"NM_000138.5:c.3350G>A\", \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"codons\": \"tGt/tAt\", \"mastermind_mmid3\": \"FBN1:C1117Y\", \"amino_acids\": \"C/Y\", \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"]}], \"colocated_variants\": [{\"end\": 48487425, \"start\": 48487425, \"seq_region_name\": \"15\", \"id\": \"CM930245\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1}, {\"clin_sig\": [\"pathogenic\"], \"allele_string\": \"C/T\", \"pubmed\": [8281141, 17701892], \"id\": \"rs137854470\", \"var_synonyms\": {\"ClinVar\": [\"RCV003764581\", \"RCV000017890\", \"VCV000016428\"], \"UniProt\": [\"VAR_002311\"], \"OMIM\": [134797.0025]}, \"phenotype_or_disease\": 1, \"strand\": 1, \"clin_sig_allele\": \"T:pathogenic\", \"seq_region_name\": \"15\", \"start\": 48487425, \"end\": 48487425}], \"id\": \"NC_000015.10:g.48487425C>T\", \"seq_region_name\": \"15\", \"minimised\": 1, \"end\": 48487425, \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"start\": 48487425, \"allele_string\": \"C/T\"}',\n",
       "    'coordinates': [{'pca_x': 0.03627397892714779,\n",
       "      'pca_y': 0.08248917888795246},\n",
       "     {'t-sne_x': 7.851703800592903, 't-sne_y': 7.089043529228739},\n",
       "     {'umap_x': 3.466264486312866, 'umap_y': 2.2692408561706543}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.0004943351959809661,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487317-C-T',\n",
       "      'distance': 0.0006343111163005233,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0006373269716277719,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.0006999162142165005,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0007222199346870184,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0007329682121053338,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-T',\n",
       "      'distance': 0.0007555602351203561,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.0007707769982516766,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.0007821179460734129,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.0007839315221644938,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496213-C-T',\n",
       "      'distance': 0.0008073002099990845,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497358-C-T',\n",
       "      'distance': 0.0008117049583233893,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481699-C-T',\n",
       "      'distance': 0.0008126840111799538,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496177-C-T',\n",
       "      'distance': 0.0008418048964813352,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496144-C-T',\n",
       "      'distance': 0.0008480235701426864,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497343-C-T',\n",
       "      'distance': 0.0008746234234422445,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.0008778631454333663,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485379-C-T',\n",
       "      'distance': 0.0008845766424201429,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.0009123305790126324,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497379-C-T',\n",
       "      'distance': 0.00094663753407076,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48487418-TCTCT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48487418\\t.\\tTCTCT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487418',\n",
       "     'ref_allele': 'TCTCT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3356_3359del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487419_48487422del',\n",
       "     'hgvs_protein': 'p.Arg1119IlefsTer42',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"end\": 48487422, \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48487419_48487422del\", \"seq_region_name\": \"15\", \"allele_string\": \"CTCT/-\", \"assembly_name\": \"GRCh38\", \"start\": 48487419, \"strand\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"input\": \"NC_000015.10:g.48487419_48487422del\", \"vcf_string\": \"15-48487418-TCTCT-T\", \"transcript_consequences\": [{\"consequence_terms\": [\"frameshift_variant\"], \"loeuf\": 0.105, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48487418, \"end\": 48487422}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}}, \"location\": \"15:48487416-48487419\", \"protein_id\": \"NP_000129.3\", \"aa\": \"CTCT\", \"protein_start\": 1119, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Arg1119IlefsTer42\", \"cdna_start\": 3672, \"cdna_end\": 3675, \"impact\": \"HIGH\", \"cds_start\": 3356, \"cds_end\": 3359, \"biotype\": \"protein_coding\", \"given_ref\": \"CTCT\", \"mutfunc\": {\"exp\": {\"dG_wt_sd\": 0.135182, \"dG_mt_sd\": 0.000836, \"dG_wt\": 174.6694, \"ddG\": -0.4036, \"dG_mt\": 174.2658, \"ddG_sd\": 0.134606}}, \"lof_info\": \"PERCENTILE:0.389856081708449,GERP_DIST:6053.72533015311,BP_DIST:5222,DIST_FROM_LAST_EXON:4833,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"used_ref\": \"TCTC\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"lof\": \"HC\", \"protein_end\": 1120, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"amino_acids\": \"RD/X\", \"codons\": \"aGAGAt/at\", \"conservation\": 2.448, \"gene_symbol_source\": \"EntrezGene\", \"hgvsc\": \"NM_000138.5:c.3356_3359del\", \"variant_allele\": \"-\", \"strand\": -1, \"transcript_id\": \"NM_000138.5\", \"hgvs_offset\": -3, \"canonical\": 1, \"exon\": \"28/66\", \"spliceai\": {\"DS_AG\": 0, \"DP_DG\": 8, \"SYMBOL\": \"FBN1\", \"DP_AG\": -20, \"DS_DL\": 0, \"DP_AL\": -41, \"DP_DL\": -19, \"DS_AL\": 0, \"DS_DG\": 0}}]}',\n",
       "    'coordinates': [{'pca_x': 0.040895894214275336,\n",
       "      'pca_y': 0.036044410966843514},\n",
       "     {'t-sne_x': 4.923307372054697, 't-sne_y': 8.317668147751364},\n",
       "     {'umap_x': 3.684486150741577, 'umap_y': 3.112826108932495}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09244340658187866,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09350210428237915,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09386367350816727,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.09395183622837067,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-T',\n",
       "      'distance': 0.09433089941740036,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468504-C-T',\n",
       "      'distance': 0.09437835961580276,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.09441890567541122,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.09443651884794235,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.09447941184043884,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.09452132135629654,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.09466636925935745,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.09467807412147522,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.09470310807228088,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.09476173669099808,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.09483011811971664,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.09500794112682343,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.09502338618040085,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.09510573744773865,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.09517830610275269,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.09518297016620636,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48487396-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48487396\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487396',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3379G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487396C>T',\n",
       "     'hgvs_protein': 'p.Gly1127Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48487396-C-T\", \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48487396C>T\", \"strand\": 1, \"colocated_variants\": [{\"start\": 48487396, \"end\": 48487396, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM950445\"}, {\"seq_region_name\": \"15\", \"pubmed\": [12938084, 7762551, 8882780, 9236141, 9404258, 16571647, 18423350, 34201307], \"id\": \"rs137854468\", \"var_synonyms\": {\"UniProt\": [\"VAR_002312\"], \"OMIM\": [134797.0021], \"ClinVar\": [\"RCV000590660\", \"RCV000631963\", \"RCV000017905\", \"VCV000016443\"]}, \"clin_sig\": [\"uncertain_significance\", \"likely_pathogenic\", \"pathogenic\"], \"allele_string\": \"C/T\", \"strand\": 1, \"clin_sig_allele\": \"T:pathogenic;T:conflicting_classifications_of_pathogenicity;T:likely_pathogenic\", \"phenotype_or_disease\": 1, \"end\": 48487396, \"start\": 48487396}], \"transcript_consequences\": [{\"sift_prediction\": \"deleterious_low_confidence\", \"protein_end\": 1127, \"eve_class\": \"Pathogenic\", \"polyphen_score\": 1, \"cadd_phred\": 28.1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"codons\": \"Ggt/Agt\", \"mastermind_mmid3\": \"FBN1:G1127S\", \"amino_acids\": \"G/S\", \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"hgvsc\": \"NM_000138.5:c.3379G>A\", \"strand\": -1, \"variant_allele\": \"T\", \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.994862258434296, \"mastermind_counts\": \"9|49|49\", \"canonical\": 1, \"lrt_pred\": \"invalid_field\", \"blosum62\": 0, \"exon\": \"28/66\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -2, \"DP_DG\": 3, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": -20, \"DP_AL\": -19}, \"consequence_terms\": [\"missense_variant\"], \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"start\": 48487395, \"type\": \"SimpleInterval\", \"end\": 48487396}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"location\": \"15:48487396\", \"protein_id\": \"NP_000129.3\", \"aa\": \"C\", \"protein_start\": 1127, \"polyphen_prediction\": \"probably_damaging\", \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Gly1127Ser\", \"cdna_start\": 3695, \"cdna_end\": 3695, \"impact\": \"MODERATE\", \"eve_score\": 0.773273240536784, \"cds_start\": 3379, \"mutationtaster_pred\": \"A\", \"cds_end\": 3379, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.022721, \"sift_score\": 0, \"given_ref\": \"C\", \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 0.062874, \"dG_wt_sd\": 0, \"dG_wt\": 174.816, \"ddG\": 7.565199, \"ddG_sd\": 0.062874, \"dG_mt\": 182.381199}}, \"used_ref\": \"C\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.8732}, \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\"}], \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48487396C>T\", \"variant_class\": \"SNV\", \"end\": 48487396, \"minimised\": 1, \"start\": 48487396, \"assembly_name\": \"GRCh38\", \"allele_string\": \"C/T\"}',\n",
       "    'coordinates': [{'pca_x': 0.03483767441163377,\n",
       "      'pca_y': 0.08006408492743386},\n",
       "     {'t-sne_x': 9.00977152445497, 't-sne_y': 2.951324277095814},\n",
       "     {'umap_x': 3.880946159362793, 'umap_y': 2.723837375640869}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.0009915228001773357,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495162-C-T',\n",
       "      'distance': 0.001194793381728232,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.0012493615504354239,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0013003399362787604,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485379-C-T',\n",
       "      'distance': 0.0013762088492512703,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421594-C-T',\n",
       "      'distance': 0.0013787251664325595,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.001452093943953514,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.0015117577277123928,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48415684-C-T',\n",
       "      'distance': 0.0015134750865399837,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.0015158638125285506,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495176-C-T',\n",
       "      'distance': 0.0015290196752175689,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496213-C-T',\n",
       "      'distance': 0.0016370066441595554,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489896-C-T',\n",
       "      'distance': 0.0016801077872514725,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.0016993801109492779,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432976-C-T',\n",
       "      'distance': 0.0017045249696820974,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.0017100241966545582,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537706-C-T',\n",
       "      'distance': 0.0017188802594318986,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.0017304171342402697,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0017365452367812395,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415707-C-T',\n",
       "      'distance': 0.0017588707851246,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48487389-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48487389\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487389',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3386G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487389C>T',\n",
       "     'hgvs_protein': 'p.Cys1129Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"end\": 48487389, \"start\": 48487389, \"strand\": 1, \"phenotype_or_disease\": 1, \"id\": \"CM990589\", \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\"}, {\"strand\": 1, \"clin_sig_allele\": \"T:pathogenic\", \"phenotype_or_disease\": 1, \"var_synonyms\": {\"OMIM\": [134797.0044], \"UniProt\": [\"VAR_010777\"], \"ClinVar\": [\"RCV002514108\", \"RCV000017927\", \"VCV000016463\"]}, \"pubmed\": [17701892, 10425041], \"id\": \"rs137854482\", \"allele_string\": \"C/T\", \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"end\": 48487389, \"start\": 48487389}], \"transcript_consequences\": [{\"mutfunc\": {\"exp\": {\"ddG\": 1.861, \"dG_wt_sd\": 1.455188, \"dG_mt_sd\": 1.232444, \"dG_wt\": 170.6984, \"dG_mt\": 172.5594, \"ddG_sd\": 0.373359}}, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9985}, \"used_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"mutationtaster_pred\": \"A\", \"cadd_raw\": 4.947292, \"biotype\": \"protein_coding\", \"cds_end\": 3386, \"given_ref\": \"C\", \"sift_score\": 0, \"hgvsp\": \"NP_000129.3:p.Cys1129Tyr\", \"gene_id\": \"2200\", \"cdna_end\": 3702, \"cdna_start\": 3702, \"impact\": \"MODERATE\", \"cds_start\": 3386, \"eve_score\": 0.767947893265842, \"consequence_terms\": [\"missense_variant\"], \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48487388, \"type\": \"SimpleInterval\", \"end\": 48487389}, \"type\": \"SequenceLocation\"}}, \"loeuf\": 0.105, \"protein_start\": 1129, \"polyphen_prediction\": \"probably_damaging\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48487389\", \"aa\": \"C\", \"hgvsc\": \"NM_000138.5:c.3386G>A\", \"mastermind_counts\": \"10|21|21\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"T\", \"clinpred\": 0.999868988990784, \"strand\": -1, \"canonical\": 1, \"spliceai\": {\"DP_AG\": -12, \"DS_DL\": 0, \"DP_DG\": 7, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DS_AL\": 0, \"DS_DG\": 0, \"DP_AL\": 9, \"DP_DL\": 10}, \"exon\": \"28/66\", \"blosum62\": -2, \"lrt_pred\": \"invalid_field\", \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"conservation\": 3.95, \"gene_symbol_source\": \"EntrezGene\", \"mastermind_mmid3\": \"FBN1:C1129Y\", \"amino_acids\": \"C/Y\", \"codons\": \"tGc/tAc\", \"protein_end\": 1129, \"polyphen_score\": 0.999, \"cadd_phred\": 27.6, \"eve_class\": \"Pathogenic\", \"sift_prediction\": \"deleterious_low_confidence\"}], \"vcf_string\": \"15-48487389-C-T\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48487389C>T\", \"assembly_name\": \"GRCh38\", \"start\": 48487389, \"allele_string\": \"C/T\", \"id\": \"NC_000015.10:g.48487389C>T\", \"seq_region_name\": \"15\", \"minimised\": 1, \"variant_class\": \"SNV\", \"end\": 48487389}',\n",
       "    'coordinates': [{'pca_x': 0.03860764193288013,\n",
       "      'pca_y': 0.07956086104847715},\n",
       "     {'t-sne_x': 7.910096279790424, 't-sne_y': 7.858878000875421},\n",
       "     {'umap_x': 3.317070722579956, 'umap_y': 2.3066723346710205}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.0005460315151140094,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-T',\n",
       "      'distance': 0.0005937762325629592,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.000702570250723511,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.0007097416091710329,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0007443047361448407,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.0007482676301151514,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0007810338865965605,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.0008432958275079727,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481699-C-T',\n",
       "      'distance': 0.0008607429917901754,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415695-C-T',\n",
       "      'distance': 0.0008785738609731197,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487317-C-T',\n",
       "      'distance': 0.0008796543115749955,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.0008928395691327751,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.0009162091300822794,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492574-C-T',\n",
       "      'distance': 0.0009346874430775642,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-T',\n",
       "      'distance': 0.0009364346624352038,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.0009519075392745435,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.0009676226763986051,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497343-C-T',\n",
       "      'distance': 0.0009883653838187456,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468456-C-T',\n",
       "      'distance': 0.0010417974554002285,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.0010554131586104631,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48487387-GG-G',\n",
       "    'metadata': {'vcf_string': '15\\t48487387\\t.\\tGG\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487387',\n",
       "     'ref_allele': 'GG',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3388del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487388del',\n",
       "     'hgvs_protein': 'p.His1130IlefsTer32',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"canonical\": 1, \"exon\": \"28/66\", \"spliceai\": {\"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": -10, \"DP_AL\": -10, \"DS_DL\": 0, \"DP_AG\": 33, \"DP_DG\": 12, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\"}, \"hgvsc\": \"NM_000138.5:c.3388del\", \"variant_allele\": \"-\", \"hgvs_offset\": -1, \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"codons\": \"Cat/at\", \"amino_acids\": \"H/X\", \"conservation\": 1.17, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"protein_end\": 1130, \"lof\": \"HC\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 1.232444, \"dG_wt_sd\": 1.455188, \"dG_wt\": 170.6984, \"ddG\": 1.861, \"dG_mt\": 172.5594, \"ddG_sd\": 0.373359}}, \"lof_info\": \"PERCENTILE:0.39322191272052,GERP_DIST:6053.72533015311,BP_DIST:5191,DIST_FROM_LAST_EXON:4802,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"used_ref\": \"G\", \"cds_end\": 3388, \"biotype\": \"protein_coding\", \"given_ref\": \"G\", \"impact\": \"HIGH\", \"cds_start\": 3388, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.His1130IlefsTer32\", \"cdna_start\": 3704, \"cdna_end\": 3704, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48487387, \"end\": 48487388}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"protein_id\": \"NP_000129.3\", \"location\": \"15:48487387\", \"aa\": \"G\", \"protein_start\": 1130, \"consequence_terms\": [\"frameshift_variant\"]}], \"colocated_variants\": [{\"start\": 48487387, \"end\": 48487388, \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"GG/G\", \"clin_sig\": [\"pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV000663636\", \"VCV000549162\"]}, \"id\": \"rs1555398572\", \"seq_region_name\": \"15\"}], \"vcf_string\": \"15-48487387-GG-G\", \"input\": \"NC_000015.10:g.48487388del\", \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"assembly_name\": \"GRCh38\", \"start\": 48487388, \"allele_string\": \"G/-\", \"id\": \"NC_000015.10:g.48487388del\", \"seq_region_name\": \"15\", \"minimised\": 1, \"end\": 48487388, \"variant_class\": \"deletion\"}',\n",
       "    'coordinates': [{'pca_x': 0.013574326950508794,\n",
       "      'pca_y': 0.0119788315703259},\n",
       "     {'t-sne_x': 7.7640620592540035, 't-sne_y': 3.8893815533898546},\n",
       "     {'umap_x': 1.9110500812530518, 'umap_y': 5.985403060913086}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.09199444949626923,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09204325079917908,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.09285381436347961,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.09297806769609451,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.09332060068845749,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.09369783103466034,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.09371902048587799,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.09415940940380096,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.09430995583534241,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09432828426361084,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.09445200860500336,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446768-A-G',\n",
       "      'distance': 0.09450598061084747,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.09464138001203537,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.0946597158908844,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.09469953179359436,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.0947640910744667,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432907-G-A',\n",
       "      'distance': 0.09476509690284729,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48490073-G-A',\n",
       "      'distance': 0.09487336874008179,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495162-C-T',\n",
       "      'distance': 0.09487685561180115,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.09488842636346817,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48487365-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48487365\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487365',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3410G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487365C>G',\n",
       "     'hgvs_protein': 'p.Arg1137Pro',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48487365C>G\", \"strand\": 1, \"vcf_string\": \"15-48487365-C-G\", \"colocated_variants\": [{\"start\": 48487365, \"end\": 48487365, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM910154\"}, {\"id\": \"COSV57315928\", \"var_synonyms\": {\"COSMIC\": [\"COSM2190380\"]}, \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"end\": 48487365, \"start\": 48487365, \"somatic\": 1}, {\"end\": 48487365, \"start\": 48487365, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV004802349\", \"RCV001360976\", \"RCV001538393\", \"RCV001182733\", \"RCV000659525\", \"VCV000547308\", \"RCV002235656\", \"RCV002499136\", \"RCV000017883\", \"VCV000016421\"], \"UniProt\": [\"VAR_002314\"], \"OMIM\": [134797.0001]}, \"pubmed\": [8281141, 7842017, 1852208], \"id\": \"rs137854456\", \"allele_string\": \"C/A/G/T\", \"clin_sig\": [\"uncertain_significance\", \"likely_benign\", \"pathogenic\"], \"strand\": 1, \"clin_sig_allele\": \"G:pathogenic;T:likely_benign;T:uncertain_significance\", \"phenotype_or_disease\": 1}], \"transcript_consequences\": [{\"protein_end\": 1137, \"eve_class\": \"Benign\", \"cadd_phred\": 29.6, \"polyphen_score\": 0.834, \"sift_prediction\": \"deleterious_low_confidence\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"variant_allele\": \"G\", \"clinpred\": 0.950022459030151, \"mastermind_counts\": \"11|31|31\", \"hgvsc\": \"NM_000138.5:c.3410G>C\", \"lrt_pred\": \"invalid_field\", \"exon\": \"28/66\", \"blosum62\": -2, \"spliceai\": {\"DS_AG\": 0, \"DP_DG\": -47, \"SYMBOL\": \"FBN1\", \"DS_DL\": 0, \"DP_AG\": -26, \"DP_DL\": -50, \"DP_AL\": 12, \"DS_DG\": 0, \"DS_AL\": 0}, \"canonical\": 1, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"amino_acids\": \"R/P\", \"codons\": \"cGc/cCc\", \"mastermind_mmid3\": \"FBN1:R1137P\", \"conservation\": 3.06, \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 3726, \"cdna_end\": 3726, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Arg1137Pro\", \"eve_score\": 0.257902836873016, \"cds_start\": 3410, \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\"], \"aa\": \"C\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48487365\", \"polyphen_prediction\": \"possibly_damaging\", \"protein_start\": 1137, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48487364, \"end\": 48487365}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"alphamissense\": {\"am_pathogenicity\": 0.8742, \"am_class\": \"likely_pathogenic\"}, \"used_ref\": \"C\", \"mutfunc\": {\"exp\": {\"ddG_sd\": 0.011562, \"dG_mt\": 175.1756, \"dG_wt_sd\": 0.942392, \"dG_mt_sd\": 0.951291, \"dG_wt\": 173.7928, \"ddG\": 1.3828}}, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"mutationtaster_pred\": \"A\", \"given_ref\": \"C\", \"sift_score\": 0, \"cds_end\": 3410, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.261397}], \"variant_class\": \"SNV\", \"end\": 48487365, \"minimised\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48487365C>G\", \"allele_string\": \"C/G\", \"start\": 48487365, \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': 0.03394186020630302,\n",
       "      'pca_y': 0.07982295071721762},\n",
       "     {'t-sne_x': 20.490275034533703, 't-sne_y': -6.93265592963942},\n",
       "     {'umap_x': 8.499530792236328, 'umap_y': 7.402284145355225}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.001017795060761273,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.0019406704232096672,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-G',\n",
       "      'distance': 0.002052051480859518,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.0020727142691612244,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0023025344125926495,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.002312181517481804,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492562-G-C',\n",
       "      'distance': 0.002368172165006399,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.0024764349218457937,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.002485980512574315,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495162-C-G',\n",
       "      'distance': 0.0025274804793298244,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.0025743322912603617,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487333-G-C',\n",
       "      'distance': 0.0026118545792996883,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.0026233363896608353,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520708-C-G',\n",
       "      'distance': 0.002631640527397394,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.0026613324880599976,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.002717136638239026,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437347-G-C',\n",
       "      'distance': 0.00284405960701406,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.0029211295768618584,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428356-G-C',\n",
       "      'distance': 0.0029593128710985184,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.0030336263589560986,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48487363-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48487363\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487363',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3412T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487363A>G',\n",
       "     'hgvs_protein': 'p.Cys1138Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"A/G\", \"assembly_name\": \"GRCh38\", \"start\": 48487363, \"minimised\": 1, \"end\": 48487363, \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48487363A>G\", \"seq_region_name\": \"15\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"id\": \"CM098516\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1, \"end\": 48487363, \"start\": 48487363}, {\"end\": 48487363, \"start\": 48487363, \"seq_region_name\": \"15\", \"id\": \"CM098675\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"phenotype_or_disease\": 1}, {\"end\": 48487363, \"start\": 48487363, \"id\": \"rs1131691806\", \"var_synonyms\": {\"ClinVar\": [\"RCV000663638\", \"RCV000493457\", \"VCV000430152\"]}, \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"allele_string\": \"A/G\", \"clin_sig_allele\": \"G:likely_pathogenic;G:pathogenic\", \"strand\": 1, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}], \"transcript_consequences\": [{\"sift_prediction\": \"deleterious_low_confidence\", \"polyphen_score\": 0.999, \"cadd_phred\": 28.9, \"eve_class\": \"Pathogenic\", \"protein_end\": 1138, \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"mastermind_mmid3\": \"FBN1:C1138R\", \"codons\": \"Tgt/Cgt\", \"amino_acids\": \"C/R\", \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"canonical\": 1, \"spliceai\": {\"DP_DL\": -45, \"DP_AL\": -24, \"DS_DG\": 0, \"DS_AL\": 0, \"DS_AG\": 0, \"DP_DG\": -42, \"SYMBOL\": \"FBN1\", \"DS_DL\": 0, \"DP_AG\": 14}, \"lrt_pred\": \"invalid_field\", \"blosum62\": -3, \"exon\": \"28/66\", \"hgvsc\": \"NM_000138.5:c.3412T>C\", \"mastermind_counts\": \"6|7|7\", \"variant_allele\": \"G\", \"strand\": -1, \"transcript_id\": \"NM_000138.5\", \"clinpred\": 0.999862790107727, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48487363, \"start\": 48487362, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"loeuf\": 0.105, \"protein_start\": 1138, \"polyphen_prediction\": \"probably_damaging\", \"location\": \"15:48487363\", \"protein_id\": \"NP_000129.3\", \"aa\": \"A\", \"consequence_terms\": [\"missense_variant\"], \"impact\": \"MODERATE\", \"cds_start\": 3412, \"eve_score\": 0.789646256149404, \"hgvsp\": \"NP_000129.3:p.Cys1138Arg\", \"gene_id\": \"2200\", \"cdna_end\": 3728, \"cdna_start\": 3728, \"cadd_raw\": 5.153626, \"biotype\": \"protein_coding\", \"cds_end\": 3412, \"given_ref\": \"A\", \"sift_score\": 0, \"mutationtaster_pred\": \"A\", \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 3.125613, \"dG_wt\": 173.4904, \"dG_wt_sd\": 1.219719, \"ddG\": 13.372, \"dG_mt\": 186.8624, \"ddG_sd\": 2.2433}}, \"used_ref\": \"A\", \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9979}}], \"input\": \"NC_000015.10:g.48487363A>G\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"vcf_string\": \"15-48487363-A-G\"}',\n",
       "    'coordinates': [{'pca_x': 0.047318317371550817,\n",
       "      'pca_y': 0.07809553726487309},\n",
       "     {'t-sne_x': 35.8197799794147, 't-sne_y': 1.6695762667151108},\n",
       "     {'umap_x': 1.1432452201843262, 'umap_y': 6.450561046600342}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.0009529375238344073,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489956-A-G',\n",
       "      'distance': 0.0009863423183560371,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.0012213492300361395,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0013678065733984113,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0013842161279171705,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0014075415674597025,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437370-A-G',\n",
       "      'distance': 0.0014234501868486404,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0015494563849642873,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0015721863601356745,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468418-A-G',\n",
       "      'distance': 0.0016066597308963537,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596319-A-G',\n",
       "      'distance': 0.0016713396180421114,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.001739563187584281,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.0017760088667273521,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0017899821978062391,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456688-A-G',\n",
       "      'distance': 0.0017956164665520191,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468490-A-G',\n",
       "      'distance': 0.0018008565530180931,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0018071257509291172,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.001855827053077519,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.001860014395788312,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.0018862471915781498,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48487363-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48487363\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487363',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3412T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487363A>C',\n",
       "     'hgvs_protein': 'p.Cys1138Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"A/C\", \"start\": 48487363, \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"end\": 48487363, \"minimised\": 1, \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48487363A>C\", \"transcript_consequences\": [{\"eve_class\": \"Pathogenic\", \"cadd_phred\": 28.4, \"polyphen_score\": 0.998, \"protein_end\": 1138, \"sift_prediction\": \"deleterious_low_confidence\", \"exon\": \"28/66\", \"blosum62\": -3, \"lrt_pred\": \"invalid_field\", \"spliceai\": {\"DS_AL\": 0, \"DS_DG\": 0, \"DP_AL\": -24, \"DP_DL\": -45, \"DP_AG\": 14, \"DS_DL\": 0, \"DP_DG\": 0, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\"}, \"canonical\": 1, \"clinpred\": 0.999766051769257, \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"C\", \"strand\": -1, \"mastermind_counts\": \"3|3|3\", \"hgvsc\": \"NM_000138.5:c.3412T>G\", \"amino_acids\": \"C/G\", \"mastermind_mmid3\": \"FBN1:C1138G\", \"codons\": \"Tgt/Ggt\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95, \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"eve_score\": 0.763195884605577, \"cds_start\": 3412, \"impact\": \"MODERATE\", \"cdna_start\": 3728, \"cdna_end\": 3728, \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys1138Gly\", \"protein_id\": \"NP_000129.3\", \"aa\": \"A\", \"location\": \"15:48487363\", \"protein_start\": 1138, \"polyphen_prediction\": \"probably_damaging\", \"loeuf\": 0.105, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48487362, \"type\": \"SimpleInterval\", \"end\": 48487363}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}}, \"consequence_terms\": [\"missense_variant\"], \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"used_ref\": \"A\", \"alphamissense\": {\"am_pathogenicity\": 0.9844, \"am_class\": \"likely_pathogenic\"}, \"mutfunc\": {\"exp\": {\"ddG\": 1.2126, \"dG_wt_sd\": 0.463763, \"dG_mt_sd\": 1.03463, \"dG_wt\": 174.3092, \"dG_mt\": 175.5218, \"ddG_sd\": 0.572689}}, \"sift_score\": 0, \"given_ref\": \"A\", \"cds_end\": 3412, \"biotype\": \"protein_coding\", \"cadd_raw\": 5.077902, \"mutationtaster_pred\": \"D\"}], \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM098516\", \"phenotype_or_disease\": 1, \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48487363, \"end\": 48487363}, {\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM098675\", \"start\": 48487363, \"end\": 48487363}], \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48487363A>C\", \"vcf_string\": \"15-48487363-A-C\"}',\n",
       "    'coordinates': [{'pca_x': 0.04057386303402735,\n",
       "      'pca_y': 0.08030626805370561},\n",
       "     {'t-sne_x': 20.272820623557145, 't-sne_y': 3.5329512760639634},\n",
       "     {'umap_x': 3.672018527984619, 'umap_y': 6.095444202423096}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.0010860426118597388,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.0011325436644256115,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.0012715535704046488,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.0014859761577099562,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488419-A-C',\n",
       "      'distance': 0.0015253938036039472,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.0015299853403121233,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.0016063344664871693,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.001634495914913714,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520677-A-C',\n",
       "      'distance': 0.001904497155919671,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.002698717638850212,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.0028893924318253994,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.003334311069920659,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481687-T-C',\n",
       "      'distance': 0.003463478060439229,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0035110306926071644,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0036107036285102367,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488412-C-T',\n",
       "      'distance': 0.0036132263485342264,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497298-T-C',\n",
       "      'distance': 0.0036160103045403957,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415567-A-G',\n",
       "      'distance': 0.0036160158924758434,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.00362113444134593,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.003635534318163991,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48487185-48487202-del-e7070a26',\n",
       "    'metadata': {'vcf_string': '15\\t48487185\\t.\\tTCACATTCATTGATGTCT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487185',\n",
       "     'ref_allele': 'TCACATTCATTGATGTCT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3464_3480del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487186_48487202del',\n",
       "     'hgvs_protein': 'p.Asp1155AlafsTer32',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"input\": \"NC_000015.10:g.48487186_48487202del\", \"vcf_string\": \"15-48487185-TCACATTCATTGATGTCT-T\", \"transcript_consequences\": [{\"exon\": \"29/66\", \"canonical\": 1, \"hgvs_offset\": -2, \"variant_allele\": \"-\", \"transcript_id\": \"NM_000138.5\", \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.3464_3480del\", \"conservation\": 2.827, \"gene_symbol_source\": \"EntrezGene\", \"amino_acids\": \"DINECE/X\", \"codons\": \"gACATCAATGAATGTGAG/g\", \"phaplo\": 0.99897615427687, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"protein_end\": 1160, \"intron\": \"28/65\", \"lof\": \"HC\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"used_ref\": \"CTCACATTCATTGATGT\", \"lof_info\": \"PERCENTILE:0.403899721448468,GERP_DIST:6053.72533015311,BP_DIST:5101,DIST_FROM_LAST_EXON:4712,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT,INTRON_SIZE:111\", \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 0.298768, \"dG_wt\": 174.157, \"dG_wt_sd\": 0.352217, \"ddG\": -0.9286, \"dG_mt\": 173.2284, \"ddG_sd\": 0.056826}}, \"given_ref\": \"CACATTCATTGATGTCT\", \"biotype\": \"protein_coding\", \"cds_end\": 3480, \"cds_start\": 3464, \"impact\": \"HIGH\", \"cdna_end\": 3796, \"cdna_start\": 3780, \"hgvsp\": \"NP_000129.3:p.Asp1155AlafsTer32\", \"gene_id\": \"2200\", \"protein_start\": 1155, \"aa\": \"CACATTCATTGATGTCT\", \"protein_id\": \"NP_000129.3\", \"location\": \"15:48487184-48487200\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48487185, \"end\": 48487202}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}}, \"genesplicer\": \"gain/acceptor/48487186-48487187/Medium/8.458943\", \"loeuf\": 0.105, \"consequence_terms\": [\"frameshift_variant\", \"splice_region_variant\", \"start_retained_variant\"]}], \"minimised\": 1, \"variant_class\": \"deletion\", \"end\": 48487202, \"id\": \"NC_000015.10:g.48487186_48487202del\", \"seq_region_name\": \"15\", \"allele_string\": \"CACATTCATTGATGTCT/-\", \"assembly_name\": \"GRCh38\", \"start\": 48487186}',\n",
       "    'coordinates': [{'pca_x': -0.06787917315277,\n",
       "      'pca_y': 0.015342415033173067},\n",
       "     {'t-sne_x': -12.859052764428867, 't-sne_y': -2.90509279386529},\n",
       "     {'umap_x': 0.41206449270248413, 'umap_y': 15.211341857910156}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.09725967049598694,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.09829556941986084,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.09878450632095337,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.09946379065513611,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.09981097280979156,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.1001773476600647,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.10037118196487427,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.10075104236602783,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.1007828414440155,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.1010114848613739,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.10134606063365936,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.10138756036758423,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.10164259374141693,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.10243629664182663,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.10253331065177917,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.10290846228599548,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.1032949686050415,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.10363554954528809,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487075-C-T',\n",
       "      'distance': 0.10398595035076141,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.10580679774284363,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48487153-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48487153\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487153',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3511T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487153A>G',\n",
       "     'hgvs_protein': 'p.Cys1171Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48487153, \"assembly_name\": \"GRCh38\", \"allele_string\": \"A/G\", \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48487153A>G\", \"variant_class\": \"SNV\", \"end\": 48487153, \"minimised\": 1, \"transcript_consequences\": [{\"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys1171Arg\", \"cdna_start\": 3827, \"cdna_end\": 3827, \"impact\": \"MODERATE\", \"eve_score\": 0.808941966143544, \"cds_start\": 3511, \"consequence_terms\": [\"missense_variant\"], \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48487153, \"type\": \"SimpleInterval\", \"start\": 48487152}}, \"type\": \"Allele\"}, \"protein_id\": \"NP_000129.3\", \"aa\": \"A\", \"location\": \"15:48487153\", \"polyphen_prediction\": \"possibly_damaging\", \"protein_start\": 1171, \"used_ref\": \"A\", \"alphamissense\": {\"am_pathogenicity\": 0.999, \"am_class\": \"likely_pathogenic\"}, \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"mutationtaster_pred\": \"D\", \"cds_end\": 3511, \"cadd_raw\": 5.045064, \"biotype\": \"protein_coding\", \"given_ref\": \"A\", \"sift_score\": 0, \"protein_end\": 1171, \"eve_class\": \"Pathogenic\", \"cadd_phred\": 28.2, \"polyphen_score\": 0.896, \"sift_prediction\": \"deleterious_low_confidence\", \"hgvsc\": \"NM_000138.5:c.3511T>C\", \"transcript_id\": \"NM_000138.5\", \"variant_allele\": \"G\", \"strand\": -1, \"clinpred\": 0.99986743927002, \"mastermind_counts\": \"10|12|12\", \"canonical\": 1, \"blosum62\": -3, \"lrt_pred\": \"invalid_field\", \"exon\": \"29/66\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 47, \"SYMBOL\": \"FBN1\", \"DP_DG\": -19, \"DS_AG\": 0, \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": 36, \"DP_AL\": -16}, \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:C1171R\", \"codons\": \"Tgc/Cgc\", \"amino_acids\": \"C/R\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": 3.95}], \"colocated_variants\": [{\"start\": 48487153, \"end\": 48487153, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM992324\"}, {\"strand\": 1, \"clin_sig_allele\": \"G:pathogenic\", \"phenotype_or_disease\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV001381675\", \"VCV001069718\"]}, \"pubmed\": [17701892], \"id\": \"rs2141293252\", \"allele_string\": \"A/G\", \"clin_sig\": [\"pathogenic\"], \"seq_region_name\": \"15\", \"end\": 48487153, \"start\": 48487153}], \"vcf_string\": \"15-48487153-A-G\", \"input\": \"NC_000015.10:g.48487153A>G\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.04273744962203607,\n",
       "      'pca_y': 0.07925469063635801},\n",
       "     {'t-sne_x': 35.48352739110737, 't-sne_y': 2.4463491989504047},\n",
       "     {'umap_x': 1.0645402669906616, 'umap_y': 6.6980719566345215}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.0009564004139974713,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.001025855541229248,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0010801837779581547,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0012201422359794378,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.0012622223002836108,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0012949936790391803,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.001304152188822627,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0013247422175481915,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.0013346802443265915,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0013612564653158188,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0013969669817015529,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.0014027064898982644,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.0014540450647473335,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.0014553319197148085,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452673-A-G',\n",
       "      'distance': 0.001520247315056622,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0016042104689404368,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0016583034303039312,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497311-A-G',\n",
       "      'distance': 0.0016718085389584303,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.001674695173278451,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596319-A-G',\n",
       "      'distance': 0.0016918941400945187,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48487138-C-CT',\n",
       "    'metadata': {'vcf_string': '15\\t48487138\\t.\\tC\\tCT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487138',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'CT',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3525dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487139dup',\n",
       "     'hgvs_protein': 'p.Gly1176ArgfsTer17',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"start\": 48487139, \"assembly_name\": \"GRCh38\", \"allele_string\": \"-/T\", \"seq_region_name\": \"15\", \"id\": \"NC_000015.10:g.48487139dup\", \"end\": 48487138, \"variant_class\": \"insertion\", \"minimised\": 1, \"transcript_consequences\": [{\"consequence_terms\": [\"frameshift_variant\"], \"protein_id\": \"NP_000129.3\", \"location\": \"15:48487138-48487139\", \"protein_start\": 1175, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48487138, \"type\": \"SimpleInterval\", \"end\": 48487138}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"cdna_start\": 3841, \"cdna_end\": 3842, \"hgvsp\": \"NP_000129.3:p.Gly1176ArgfsTer17\", \"gene_id\": \"2200\", \"cds_start\": 3525, \"impact\": \"HIGH\", \"given_ref\": \"-\", \"cds_end\": 3526, \"biotype\": \"protein_coding\", \"used_ref\": \"-\", \"lof_info\": \"PERCENTILE:0.409122562674095,GERP_DIST:6053.72533015311,BP_DIST:5054,DIST_FROM_LAST_EXON:4665,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"ptriplo\": 0.984699554922396, \"lof\": \"HC\", \"protein_end\": 1176, \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"mane\": [\"MANE_Select\"], \"amino_acids\": \"-/X\", \"mastermind_mmid3\": \"FBN1:G1176fs\", \"codons\": \"-/A\", \"gene_symbol_source\": \"EntrezGene\", \"conservation\": -0.575, \"strand\": -1, \"variant_allele\": \"T\", \"transcript_id\": \"NM_000138.5\", \"mastermind_counts\": \"4|4|4\", \"hgvsc\": \"NM_000138.5:c.3525dup\", \"exon\": \"29/66\", \"spliceai\": {\"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": 47, \"DP_AL\": -1, \"DS_DL\": 0, \"DP_AG\": -50, \"DP_DG\": 0, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0}, \"canonical\": 1}], \"vcf_string\": \"15-48487138-C-CT\", \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48487139dup\"}',\n",
       "    'coordinates': [{'pca_x': 0.030264881972740937,\n",
       "      'pca_y': 0.003108911529835591},\n",
       "     {'t-sne_x': 7.509340464482734, 't-sne_y': 8.321796979913746},\n",
       "     {'umap_x': 4.009030818939209, 'umap_y': 2.995234251022339}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48520693-A-G',\n",
       "      'distance': 0.10044705867767334,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48470689-G-A',\n",
       "      'distance': 0.10046660155057907,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48420718-G-A',\n",
       "      'distance': 0.10208939015865326,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520675-A-G',\n",
       "      'distance': 0.1022927463054657,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48613083-G-T',\n",
       "      'distance': 0.10249777138233185,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48427700-G-A',\n",
       "      'distance': 0.10263761878013611,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48489981-G-T',\n",
       "      'distance': 0.10290593653917313,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48487142-G-A',\n",
       "      'distance': 0.10310573875904083,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48487157-G-A',\n",
       "      'distance': 0.10329461842775345,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48460250-G-A',\n",
       "      'distance': 0.10349749028682709,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48470722-G-A',\n",
       "      'distance': 0.10369890928268433,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520690-G-A',\n",
       "      'distance': 0.10377676784992218,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48456731-G-A',\n",
       "      'distance': 0.10386878252029419,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48470680-C-T',\n",
       "      'distance': 0.10390699654817581,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48489981-G-A',\n",
       "      'distance': 0.10413157194852829,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48460259-G-A',\n",
       "      'distance': 0.1044517457485199,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48487184-C-T',\n",
       "      'distance': 0.10448411852121353,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48420778-G-A',\n",
       "      'distance': 0.10454375296831131,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48427613-G-A',\n",
       "      'distance': 0.10460101068019867,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48489897-G-A',\n",
       "      'distance': 0.10461591184139252,\n",
       "      'pathogenicity': 'likely_benign'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_benign',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_benign', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_benign', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_benign', 'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48487126-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48487126\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487126',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3538T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487126A>G',\n",
       "     'hgvs_protein': 'p.Cys1180Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"id\": \"CM098678\", \"start\": 48487126, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48487126}, {\"clin_sig_allele\": \"G:pathogenic\", \"pubmed\": [17701892], \"end\": 48487126, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"A/G\", \"clin_sig\": [\"pathogenic\"], \"start\": 48487126, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV001951501\", \"VCV001460238\"]}, \"id\": \"rs2141293199\"}], \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"eve_score\": 0.756835610200807, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 3854, \"cds_start\": 3538, \"spliceai\": {\"DP_AL\": 0, \"DS_AG\": 0, \"DP_DL\": -47, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": 11, \"DS_DG\": 0, \"DP_DG\": -45}, \"codons\": \"Tgt/Cgt\", \"location\": \"15:48487126\", \"cadd_raw\": 5.209751, \"cadd_phred\": 29.2, \"allele_num\": 1, \"hgvsp\": \"NP_000129.3:p.Cys1180Arg\", \"used_ref\": \"A\", \"aa\": \"A\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3538T>C\", \"biotype\": \"protein_coding\", \"protein_end\": 1180, \"alphamissense\": {\"am_pathogenicity\": 0.9982, \"am_class\": \"likely_pathogenic\"}, \"conservation\": 3.95, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"end\": 48487126, \"start\": 48487125, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}}, \"loeuf\": 0.105, \"cdna_start\": 3854, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"exon\": \"29/66\", \"given_ref\": \"A\", \"eve_class\": \"Pathogenic\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -3, \"impact\": \"MODERATE\", \"variant_allele\": \"G\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 3538, \"protein_start\": 1180, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/R\", \"polyphen_score\": 0.853, \"polyphen_prediction\": \"possibly_damaging\", \"strand\": -1, \"mastermind_mmid3\": \"FBN1:C1180R\", \"clinpred\": 0.999744117259979, \"canonical\": 1, \"mutationtaster_pred\": \"D\", \"mastermind_counts\": \"3|3|3\", \"sift_score\": 0}], \"id\": \"NC_000015.10:g.48487126A>G\", \"minimised\": 1, \"vcf_string\": \"15-48487126-A-G\", \"end\": 48487126, \"allele_string\": \"A/G\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48487126A>G\", \"start\": 48487126, \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.04502105881716961,\n",
       "      'pca_y': 0.07407434643675279},\n",
       "     {'t-sne_x': 34.80478218242253, 't-sne_y': 3.8968765909084726},\n",
       "     {'umap_x': 1.2423478364944458, 'umap_y': 6.400403022766113}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0004884529043920338,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0006312202312983572,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0006506069330498576,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.0006708867731504142,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0008074643556028605,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.0008163152961060405,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.0008607443887740374,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.0009113200940191746,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.0009212529985234141,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0009449083590880036,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0009814458899199963,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.0010242881253361702,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.001058576162904501,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.001084228977560997,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.001086030388250947,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0010981280356645584,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0011473087361082435,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.001192591735161841,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-G',\n",
       "      'distance': 0.0012132675619795918,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.0012149256654083729,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48487119-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48487119\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487119',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3545G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487119C>G',\n",
       "     'hgvs_protein': 'p.Cys1182Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48487119, \"seq_region_name\": \"15\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48487119C>G\", \"transcript_consequences\": [{\"canonical\": 1, \"mastermind_mmid3\": \"FBN1:C1182S\", \"clinpred\": 0.99928230047226, \"mutationtaster_pred\": \"D\", \"sift_score\": 0, \"mastermind_counts\": \"8|9|9\", \"polyphen_score\": 0.644, \"polyphen_prediction\": \"possibly_damaging\", \"strand\": -1, \"protein_start\": 1182, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/S\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 3545, \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -1, \"impact\": \"MODERATE\", \"variant_allele\": \"G\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"eve_class\": \"Pathogenic\", \"given_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"29/66\", \"cdna_start\": 3861, \"phaplo\": 0.99897615427687, \"conservation\": 3.95, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9974}, \"protein_end\": 1182, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"start\": 48487118, \"type\": \"SimpleInterval\", \"end\": 48487119}}, \"type\": \"Allele\", \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}}, \"biotype\": \"protein_coding\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3545G>C\", \"hgvsp\": \"NP_000129.3:p.Cys1182Ser\", \"used_ref\": \"C\", \"aa\": \"C\", \"allele_num\": 1, \"cadd_raw\": 5.084241, \"location\": \"15:48487119\", \"cadd_phred\": 28.4, \"eve_score\": 0.781191727470512, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 3861, \"codons\": \"tGc/tCc\", \"spliceai\": {\"DS_DG\": 0, \"DP_DG\": -40, \"DP_AG\": -31, \"DS_DL\": 0, \"DP_DL\": -38, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AL\": 18, \"DS_AG\": 0}, \"cds_start\": 3545}], \"id\": \"NC_000015.10:g.48487119C>G\", \"colocated_variants\": [{\"id\": \"CM154830\", \"start\": 48487119, \"phenotype_or_disease\": 1, \"end\": 48487119, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1}, {\"end\": 48487119, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"CM980728\", \"phenotype_or_disease\": 1, \"start\": 48487119}], \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48487119-C-G\", \"end\": 48487119, \"allele_string\": \"C/G\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.0372337949830841,\n",
       "      'pca_y': 0.08087008546286473},\n",
       "     {'t-sne_x': 20.46836161751613, 't-sne_y': -5.217107325715363},\n",
       "     {'umap_x': 8.582740783691406, 'umap_y': 7.122677326202393}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.0007770736119709909,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.0009365689475089312,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.0009695091284811497,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0010925414972007275,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.001159223960712552,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.0012355017242953181,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0013147054705768824,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.0015389086911454797,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.0015840843552723527,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.0019308917690068483,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.0019447366939857602,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489991-C-G',\n",
       "      'distance': 0.0020324685610830784,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.002094501396641135,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.002310769632458687,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.0024973731487989426,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.002558153122663498,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.0025971895083785057,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.0026368743274360895,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415708-C-G',\n",
       "      'distance': 0.002655408112332225,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468455-G-C',\n",
       "      'distance': 0.002846670104190707,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48487118-G-C',\n",
       "    'metadata': {'vcf_string': '15\\t48487118\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48487118',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3546C>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48487118G>C',\n",
       "     'hgvs_protein': 'p.Cys1182Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48487118G>C\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"start\": 48487118, \"seq_region_name\": \"15\", \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"start\": 48487118, \"id\": \"CM145738\", \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"end\": 48487118}, {\"id\": \"CM972857\", \"phenotype_or_disease\": 1, \"start\": 48487118, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48487118}, {\"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"somatic\": 1, \"seq_region_name\": \"15\", \"end\": 48487118, \"var_synonyms\": {\"COSMIC\": [\"COSM5807823\"]}, \"id\": \"COSV57308588\", \"phenotype_or_disease\": 1, \"start\": 48487118}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48487118G>C\", \"transcript_consequences\": [{\"hgvsp\": \"NP_000129.3:p.Cys1182Trp\", \"used_ref\": \"G\", \"aa\": \"G\", \"hgvsc\": \"NM_000138.5:c.3546C>G\", \"gene_id\": \"2200\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"biotype\": \"protein_coding\", \"conservation\": 3.09, \"protein_end\": 1182, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9979}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48487117, \"end\": 48487118}, \"type\": \"SequenceLocation\"}}, \"cdna_end\": 3862, \"codons\": \"tgC/tgG\", \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": 16, \"SYMBOL\": \"FBN1\", \"DP_AL\": -30, \"DS_AG\": 0, \"DP_DG\": -39, \"DS_DG\": 0, \"DS_DL\": 0, \"DP_AG\": 19}, \"cds_start\": 3546, \"eve_score\": 0.755770678321203, \"consequence_terms\": [\"missense_variant\"], \"cadd_raw\": 4.505252, \"location\": \"15:48487118\", \"cadd_phred\": 25.5, \"allele_num\": 1, \"cds_end\": 3546, \"protein_id\": \"NP_000129.3\", \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/W\", \"protein_start\": 1182, \"strand\": -1, \"polyphen_prediction\": \"probably_damaging\", \"polyphen_score\": 0.983, \"mutationtaster_pred\": \"D\", \"mastermind_counts\": \"7|8|8\", \"sift_score\": 0, \"mastermind_mmid3\": \"FBN1:C1182W\", \"canonical\": 1, \"clinpred\": 0.999764859676361, \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 3862, \"phaplo\": 0.99897615427687, \"exon\": \"29/66\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"G\", \"eve_class\": \"Pathogenic\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -2, \"impact\": \"MODERATE\"}], \"minimised\": 1, \"allele_string\": \"G/C\", \"end\": 48487118, \"vcf_string\": \"15-48487118-G-C\"}',\n",
       "    'coordinates': [{'pca_x': 0.04095471888843294,\n",
       "      'pca_y': 0.08330597138648452},\n",
       "     {'t-sne_x': 19.165437330412466, 't-sne_y': -7.850982729142578},\n",
       "     {'umap_x': 8.363123893737793, 'umap_y': 7.21900749206543}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.0007673058426007628,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.0008464337443001568,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468455-G-C',\n",
       "      'distance': 0.0010640529217198491,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.001199106452986598,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492562-G-C',\n",
       "      'distance': 0.0017288500675931573,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48437784-G-C',\n",
       "      'distance': 0.001904108445160091,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415708-C-G',\n",
       "      'distance': 0.0022401134483516216,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.0022536227479577065,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428356-G-C',\n",
       "      'distance': 0.0022760131396353245,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48487333-G-C',\n",
       "      'distance': 0.0023441656958311796,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.002347787842154503,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.0023948103189468384,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.0024979214649647474,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.002558633219450712,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437347-G-C',\n",
       "      'distance': 0.00263903196901083,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.002684075850993395,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0026963644195348024,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.0027239741757512093,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.0028586098924279213,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520708-C-G',\n",
       "      'distance': 0.002860681852325797,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48485487-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48485487\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48485487',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3599A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48485487T>C',\n",
       "     'hgvs_protein': 'p.Glu1200Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"vcf_string\": \"15-48485487-T-C\", \"end\": 48485487, \"allele_string\": \"T/C\", \"colocated_variants\": [{\"start\": 48485487, \"phenotype_or_disease\": 1, \"id\": \"CM990590\", \"end\": 48485487, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\"}], \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"start\": 48485486, \"type\": \"SimpleInterval\", \"end\": 48485487}}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}}, \"protein_end\": 1200, \"conservation\": 3.85, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9926}, \"used_ref\": \"T\", \"aa\": \"T\", \"hgvsp\": \"NP_000129.3:p.Glu1200Gly\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3599A>G\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"allele_num\": 1, \"spliceai\": {\"DP_AL\": -7, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 35, \"DS_AL\": 0, \"DP_AG\": 9, \"DS_DL\": 0, \"DS_DG\": 0, \"DP_DG\": -17}, \"codons\": \"gAa/gGa\", \"cds_start\": 3599, \"cdna_end\": 3915, \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.68095838467804, \"cadd_phred\": 29.8, \"cadd_raw\": 5.295606, \"location\": \"15:48485487\", \"strand\": -1, \"polyphen_prediction\": \"probably_damaging\", \"polyphen_score\": 0.978, \"sift_score\": 0, \"mastermind_counts\": \"8|14|14\", \"mutationtaster_pred\": \"D\", \"mastermind_mmid3\": \"FBN1:E1200G\", \"clinpred\": 0.999067008495331, \"canonical\": 1, \"cds_end\": 3599, \"protein_id\": \"NP_000129.3\", \"amino_acids\": \"E/G\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 1200, \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"C\", \"impact\": \"MODERATE\", \"blosum62\": -2, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"exon\": \"30/66\", \"cdna_start\": 3915, \"loeuf\": 0.105, \"ptriplo\": 0.984699554922396, \"given_ref\": \"T\", \"eve_class\": \"Pathogenic\"}], \"id\": \"NC_000015.10:g.48485487T>C\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48485487T>C\", \"start\": 48485487, \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.03787047131707777,\n",
       "      'pca_y': 0.08335262994642889},\n",
       "     {'t-sne_x': 5.455682517137034, 't-sne_y': 1.9488746259289806},\n",
       "     {'umap_x': 4.5918755531311035, 'umap_y': 3.169506549835205}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48497382-T-C',\n",
       "      'distance': 0.0014257633592933416,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.002500478643923998,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.0025706617161631584,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488233-C-T',\n",
       "      'distance': 0.0028635773342102766,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.002913906704634428,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421594-C-T',\n",
       "      'distance': 0.0029268087819218636,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421982-C-T',\n",
       "      'distance': 0.0029324544593691826,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495162-C-T',\n",
       "      'distance': 0.003059693379327655,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487107-T-C',\n",
       "      'distance': 0.003085440956056118,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489896-C-T',\n",
       "      'distance': 0.00310823367908597,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437069-C-T',\n",
       "      'distance': 0.0031565609388053417,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472607-T-C',\n",
       "      'distance': 0.00316251371987164,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420719-T-C',\n",
       "      'distance': 0.0031644897535443306,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481687-T-C',\n",
       "      'distance': 0.003297652117908001,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.0033177477307617664,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497298-T-C',\n",
       "      'distance': 0.003376510925590992,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421981-C-T',\n",
       "      'distance': 0.0033979862928390503,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485379-C-T',\n",
       "      'distance': 0.003426835872232914,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415707-C-T',\n",
       "      'distance': 0.0034673702903091908,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.0035198295954614878,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48485419-48485485-del-729f3122',\n",
       "    'metadata': {'vcf_string': '15\\t48485419\\t.\\tAGCTACATTCATAGCTGCCTTCAGAGTTTGTGCAGAAGGTTTCACAACCACCATTCATTATGCTGCA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48485419',\n",
       "     'ref_allele': 'AGCTACATTCATAGCTGCCTTCAGAGTTTGTGCAGAAGGTTTCACAACCACCATTCATTATGCTGCA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3603_3668del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48485420_48485485del',\n",
       "     'hgvs_protein': 'p.Ser1202_Cys1223del',\n",
       "     'most_severe_consequence': 'inframe_deletion'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"start\": 48485420, \"input\": \"NC_000015.10:g.48485420_48485485del\", \"most_severe_consequence\": \"inframe_deletion\", \"strand\": 1, \"allele_string\": \"GCTACATTCATAGCTGCCTTCAGAGTTTGTGCAGAAGGTTTCACAACCACCATTCATTATGCTGCA/-\", \"end\": 48485485, \"vcf_string\": \"15-48485419-AGCTACATTCATAGCTGCCTTCAGAGTTTGTGCAGAAGGTTTCACAACCACCATTCATTATGCTGCA-A\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48485420_48485485del\", \"transcript_consequences\": [{\"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"end\": 48485485, \"type\": \"SimpleInterval\", \"start\": 48485419}}}, \"protein_end\": 1223, \"conservation\": 2.519, \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.3603_3668del\", \"gene_id\": \"2200\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"CAGCTACATTCATAGCTGCCTTCAGAGTTTGTGCAGAAGGTTTCACAACCACCATTCATTATGCTG\", \"aa\": \"GCTACATTCATAGCTGCCTTCAGAGTTTGTGCAGAAGGTTTCACAACCACCATTCATTATGCTGCA\", \"hgvsp\": \"NP_000129.3:p.Ser1202_Cys1223del\", \"allele_num\": 1, \"hgvs_offset\": -2, \"location\": \"15:48485418-48485483\", \"cds_start\": 3603, \"codons\": \"tgCAGCATAATGAATGGTGGTTGTGAAACCTTCTGCACAAACTCTGAAGGCAGCTATGAATGTAGCTGt/tgt\", \"cdna_end\": 3984, \"consequence_terms\": [\"inframe_deletion\"], \"canonical\": 1, \"strand\": -1, \"amino_acids\": \"CSIMNGGCETFCTNSEGSYECSC/C\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 1201, \"cds_end\": 3668, \"protein_id\": \"NP_000129.3\", \"variant_allele\": \"-\", \"impact\": \"MODERATE\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"GCTACATTCATAGCTGCCTTCAGAGTTTGTGCAGAAGGTTTCACAACCACCATTCATTATGCTGCA\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"exon\": \"30/66\", \"cdna_start\": 3919, \"loeuf\": 0.105}], \"variant_class\": \"deletion\", \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': -0.014768735041985143,\n",
       "      'pca_y': 0.09047523818677211},\n",
       "     {'t-sne_x': 9.013591261262118, 't-sne_y': 1.1562923106391452},\n",
       "     {'umap_x': 4.144603252410889, 'umap_y': 3.89439058303833}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.1592663675546646,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.1594962179660797,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.1603417843580246,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48436970-C-A',\n",
       "      'distance': 0.16077716648578644,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.16104470193386078,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516225-G-A',\n",
       "      'distance': 0.1614203155040741,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.16186952590942383,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.1619890034198761,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.162122905254364,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.16228565573692322,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.16248413920402527,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.16265258193016052,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.1628405749797821,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.1630588173866272,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.1631247103214264,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.1632489264011383,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441797-G-T',\n",
       "      'distance': 0.16341498494148254,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.16349488496780396,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503845-G-T',\n",
       "      'distance': 0.1637623906135559,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.16382983326911926,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48485462-AC-A',\n",
       "    'metadata': {'vcf_string': '15\\t48485462\\t.\\tAC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48485462',\n",
       "     'ref_allele': 'AC',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3623del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48485463del',\n",
       "     'hgvs_protein': 'p.Cys1208LeufsTer22',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"allele_string\": \"C/-\", \"vcf_string\": \"15-48485462-AC-A\", \"end\": 48485463, \"colocated_variants\": [{\"end\": 48485463, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48485463, \"phenotype_or_disease\": 1, \"id\": \"CD983901\"}, {\"allele_string\": \"C/-\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48485463, \"id\": \"rs1555398407\", \"var_synonyms\": {\"ClinVar\": [\"RCV000663655\", \"VCV000549177\"]}, \"phenotype_or_disease\": 1, \"clin_sig\": [\"pathogenic\"], \"start\": 48485463}], \"variant_class\": \"deletion\", \"assembly_name\": \"GRCh38\", \"id\": \"NC_000015.10:g.48485463del\", \"transcript_consequences\": [{\"ptriplo\": 0.984699554922396, \"given_ref\": \"C\", \"loeuf\": 0.105, \"exon\": \"30/66\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 3939, \"variant_allele\": \"-\", \"lof_info\": \"PERCENTILE:0.420496750232126,GERP_DIST:6012.14673394263,BP_DIST:4957,DIST_FROM_LAST_EXON:4568,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mane_select\": \"ENST00000316623.10\", \"impact\": \"HIGH\", \"gene_symbol\": \"FBN1\", \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/X\", \"protein_start\": 1208, \"lof\": \"HC\", \"cds_end\": 3623, \"protein_id\": \"NP_000129.3\", \"mastermind_counts\": \"6|6|6\", \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:C1208fs\", \"strand\": -1, \"location\": \"15:48485463\", \"cdna_end\": 3939, \"codons\": \"tGt/tt\", \"cds_start\": 3623, \"spliceai\": {\"DS_AG\": 0, \"DP_AL\": 18, \"SYMBOL\": \"FBN1\", \"DP_DL\": 8, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": -30, \"DS_DG\": 0, \"DP_DG\": 18}, \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1, \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3623del\", \"mane\": [\"MANE_Select\"], \"hgvsp\": \"NP_000129.3:p.Cys1208LeufsTer22\", \"aa\": \"C\", \"used_ref\": \"C\", \"conservation\": 2.97, \"protein_end\": 1208, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48485462, \"end\": 48485463}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}}, \"biotype\": \"protein_coding\"}], \"input\": \"NC_000015.10:g.48485463del\", \"strand\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"start\": 48485463, \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.044590216351539036,\n",
       "      'pca_y': 0.0422543531760659},\n",
       "     {'t-sne_x': 1.8845915771786026, 't-sne_y': 9.51095516008019},\n",
       "     {'umap_x': 3.8208200931549072, 'umap_y': 2.9127111434936523}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.1033773422241211,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.10629843920469284,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.10676389932632446,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.1067848652601242,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.10690857470035553,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.10699416697025299,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.10782436281442642,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.10788322985172272,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427646-A-T',\n",
       "      'distance': 0.1080671101808548,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48441797-G-T',\n",
       "      'distance': 0.10818110406398773,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.1083923727273941,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485483-G-T',\n",
       "      'distance': 0.10852660238742828,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.10861727595329285,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445419-G-T',\n",
       "      'distance': 0.10902050137519836,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.10907997190952301,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.1090969443321228,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427659-C-T',\n",
       "      'distance': 0.10934477299451828,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.10935239493846893,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.10943125188350677,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.10946868360042572,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48485424-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48485424\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48485424',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3662G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48485424C>T',\n",
       "     'hgvs_protein': 'p.Cys1221Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48485424C>T\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"start\": 48485424, \"seq_region_name\": \"15\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"end\": 48485424, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"start\": 48485424, \"id\": \"CM060012\"}, {\"clin_sig_allele\": \"T:pathogenic\", \"end\": 48485424, \"pubmed\": [16333834], \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"C/T\", \"start\": 48485424, \"clin_sig\": [\"pathogenic\"], \"phenotype_or_disease\": 1, \"id\": \"rs137854483\", \"var_synonyms\": {\"ClinVar\": [\"RCV000030943\", \"VCV000016464\"], \"OMIM\": [134797.0045]}}], \"assembly_name\": \"GRCh38\", \"id\": \"NC_000015.10:g.48485424C>T\", \"transcript_consequences\": [{\"blosum62\": -2, \"mane_select\": \"ENST00000316623.10\", \"impact\": \"MODERATE\", \"variant_allele\": \"T\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"given_ref\": \"C\", \"eve_class\": \"Pathogenic\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 3978, \"phaplo\": 0.99897615427687, \"exon\": \"30/66\", \"clinpred\": 0.998571877008692, \"mastermind_mmid3\": \"FBN1:C1221Y\", \"canonical\": 1, \"mutationtaster_pred\": \"A\", \"mastermind_counts\": \"7|9|9\", \"sift_score\": 0, \"polyphen_prediction\": \"probably_damaging\", \"polyphen_score\": 0.997, \"strand\": -1, \"protein_start\": 1221, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/Y\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 3662, \"allele_num\": 1, \"location\": \"15:48485424\", \"cadd_raw\": 5.116765, \"cadd_phred\": 28.6, \"eve_score\": 0.763656690731502, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 3978, \"codons\": \"tGt/tAt\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 19, \"DP_DG\": -50, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": -4, \"DS_AL\": 0, \"DP_DL\": -49, \"SYMBOL\": \"FBN1\"}, \"cds_start\": 3662, \"alphamissense\": {\"am_pathogenicity\": 0.9963, \"am_class\": \"likely_pathogenic\"}, \"conservation\": 3.85, \"protein_end\": 1221, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"end\": 48485424, \"type\": \"SimpleInterval\", \"start\": 48485423}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}}, \"biotype\": \"protein_coding\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.3662G>A\", \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys1221Tyr\", \"used_ref\": \"C\", \"aa\": \"C\"}], \"minimised\": 1, \"allele_string\": \"C/T\", \"vcf_string\": \"15-48485424-C-T\", \"end\": 48485424}',\n",
       "    'coordinates': [{'pca_x': 0.03710971795072596,\n",
       "      'pca_y': 0.0901960413686545},\n",
       "     {'t-sne_x': 7.164384465891087, 't-sne_y': -5.954130289534887},\n",
       "     {'umap_x': 4.8680572509765625, 'umap_y': 4.807281017303467}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48474638-C-T',\n",
       "      'distance': 0.0007917119655758142,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425816-C-T',\n",
       "      'distance': 0.0008649514056742191,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474533-C-T',\n",
       "      'distance': 0.0009505703928880394,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432910-C-T',\n",
       "      'distance': 0.0009893972892314196,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.0009950012899935246,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432916-C-T',\n",
       "      'distance': 0.0010492217261344194,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415632-C-T',\n",
       "      'distance': 0.0012741419486701488,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503849-C-T',\n",
       "      'distance': 0.0012781424447894096,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483931-C-T',\n",
       "      'distance': 0.0013452551793307066,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437369-C-T',\n",
       "      'distance': 0.0014687476214021444,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472595-C-T',\n",
       "      'distance': 0.001480036647990346,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474344-C-T',\n",
       "      'distance': 0.0015012476360425353,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432955-C-T',\n",
       "      'distance': 0.0015116201248019934,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48464009-C-T',\n",
       "      'distance': 0.0015859110280871391,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-T',\n",
       "      'distance': 0.0016066250391304493,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425852-C-T',\n",
       "      'distance': 0.001647867145948112,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425479-C-T',\n",
       "      'distance': 0.0017591535579413176,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474572-C-T',\n",
       "      'distance': 0.0018393851350992918,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465825-C-T',\n",
       "      'distance': 0.0018900248687714338,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434627-C-T',\n",
       "      'distance': 0.0019314968958497047,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48485418-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48485418\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48485418',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3668G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48485418C>T',\n",
       "     'hgvs_protein': 'p.Cys1223Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"allele_string\": \"C/T\", \"vcf_string\": \"15-48485418-C-T\", \"end\": 48485418, \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"id\": \"CM940764\", \"start\": 48485418, \"phenotype_or_disease\": 1, \"end\": 48485418, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1}, {\"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"start\": 48485418, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"OMIM\": [134797.0022], \"UniProt\": [\"VAR_002321\"], \"ClinVar\": [\"RCV000242083\", \"RCV000017906\", \"VCV000016444\", \"RCV000181691\", \"VCV000200184\"]}, \"id\": \"rs137854469\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"C/A/T\", \"clin_sig_allele\": \"T:pathogenic;A:likely_pathogenic\", \"pubmed\": [8430317, 8563763, 8071963], \"end\": 48485418}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48485418C>T\", \"transcript_consequences\": [{\"allele_num\": 1, \"cadd_phred\": 31, \"cadd_raw\": 5.430991, \"location\": \"15:48485418\", \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.750511819915878, \"cds_start\": 3668, \"codons\": \"tGt/tAt\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 2, \"DP_DG\": 7, \"DS_DG\": 0, \"DP_AL\": 18, \"DS_AG\": 0, \"DS_AL\": 0, \"DP_DL\": -48, \"SYMBOL\": \"FBN1\"}, \"cdna_end\": 3984, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"start\": 48485417, \"end\": 48485418}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}}, \"protein_end\": 1223, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9976}, \"conservation\": 3.85, \"biotype\": \"protein_coding\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"hgvsc\": \"NM_000138.5:c.3668G>A\", \"gene_id\": \"2200\", \"aa\": \"C\", \"used_ref\": \"C\", \"hgvsp\": \"NP_000129.3:p.Cys1223Tyr\", \"impact\": \"MODERATE\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -2, \"variant_allele\": \"T\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"eve_class\": \"Pathogenic\", \"given_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"30/66\", \"cdna_start\": 3984, \"loeuf\": 0.105, \"canonical\": 1, \"clinpred\": 0.998598543675359, \"mastermind_mmid3\": \"FBN1:C1223Y\", \"mastermind_counts\": \"15|37|37\", \"sift_score\": 0, \"mutationtaster_pred\": \"A\", \"polyphen_score\": 0.997, \"polyphen_prediction\": \"probably_damaging\", \"strand\": -1, \"protein_start\": 1223, \"amino_acids\": \"C/Y\", \"transcript_id\": \"NM_000138.5\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 3668}], \"input\": \"NC_000015.10:g.48485418C>T\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48485418}',\n",
       "    'coordinates': [{'pca_x': 0.03895166409424988,\n",
       "      'pca_y': 0.08978005293097388},\n",
       "     {'t-sne_x': 7.126413831940483, 't-sne_y': -5.997056579680961},\n",
       "     {'umap_x': 5.007726669311523, 'umap_y': 4.718027114868164}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48474638-C-T',\n",
       "      'distance': 0.0007437615422531962,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425816-C-T',\n",
       "      'distance': 0.0007884862716309726,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474533-C-T',\n",
       "      'distance': 0.0009912960231304169,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432910-C-T',\n",
       "      'distance': 0.001033990760333836,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.0011029564775526524,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432916-C-T',\n",
       "      'distance': 0.0011409013532102108,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483931-C-T',\n",
       "      'distance': 0.001178485807031393,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503849-C-T',\n",
       "      'distance': 0.001181888859719038,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48464009-C-T',\n",
       "      'distance': 0.0013177987420931458,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472595-C-T',\n",
       "      'distance': 0.00142310606315732,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474344-C-T',\n",
       "      'distance': 0.0014537101378664374,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415632-C-T',\n",
       "      'distance': 0.0014744594227522612,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432955-C-T',\n",
       "      'distance': 0.001488277455791831,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-T',\n",
       "      'distance': 0.0014948102179914713,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437369-C-T',\n",
       "      'distance': 0.0015384869184345007,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425852-C-T',\n",
       "      'distance': 0.001550127286463976,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425479-C-T',\n",
       "      'distance': 0.0017918397206813097,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503846-C-T',\n",
       "      'distance': 0.001794740674085915,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474572-C-T',\n",
       "      'distance': 0.0018316200003027916,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428459-C-T',\n",
       "      'distance': 0.0019954857416450977,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48485410-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48485410\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48485410',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3676G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48485410C>A',\n",
       "     'hgvs_protein': 'p.Gly1226Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48485410-C-A\", \"end\": 48485410, \"allele_string\": \"C/A\", \"minimised\": 1, \"transcript_consequences\": [{\"cds_end\": 3676, \"protein_id\": \"NP_000129.3\", \"amino_acids\": \"G/*\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 1226, \"lof\": \"HC\", \"strand\": -1, \"mastermind_counts\": \"3|3|3\", \"mutationtaster_pred\": \"A\", \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:G1226X\", \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"30/66\", \"phaplo\": 0.99897615427687, \"cdna_start\": 3992, \"loeuf\": 0.105, \"ptriplo\": 0.984699554922396, \"given_ref\": \"C\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"A\", \"impact\": \"HIGH\", \"lof_info\": \"PERCENTILE:0.426648096564531,GERP_DIST:5915.79569898546,BP_DIST:4904,DIST_FROM_LAST_EXON:4515,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"C\", \"aa\": \"C\", \"hgvsp\": \"NP_000129.3:p.Gly1226Ter\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3676G>T\", \"mane\": [\"MANE_Select\"], \"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48485409, \"end\": 48485410}}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}}, \"protein_end\": 1226, \"conservation\": 3.85, \"codons\": \"Gga/Tga\", \"cds_start\": 3676, \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": -36, \"SYMBOL\": \"FBN1\", \"DP_AL\": -1, \"DS_AG\": 0, \"DP_DG\": -40, \"DS_DG\": 0, \"DP_AG\": 26, \"DS_DL\": 0}, \"cdna_end\": 3992, \"consequence_terms\": [\"stop_gained\"], \"cadd_phred\": 40, \"cadd_raw\": 9.223816, \"location\": \"15:48485410\", \"allele_num\": 1}], \"id\": \"NC_000015.10:g.48485410C>A\", \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48485410, \"id\": \"CM118912\", \"phenotype_or_disease\": 1, \"start\": 48485410}, {\"somatic\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"end\": 48485410, \"start\": 48485410, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM4397758\"]}, \"id\": \"COSV57315537\"}], \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"start\": 48485410, \"seq_region_name\": \"15\", \"strand\": 1, \"most_severe_consequence\": \"stop_gained\", \"input\": \"NC_000015.10:g.48485410C>A\"}',\n",
       "    'coordinates': [{'pca_x': 0.016982175266189198,\n",
       "      'pca_y': 0.09952102606051641},\n",
       "     {'t-sne_x': 21.535225031493137, 't-sne_y': 40.38362671620274},\n",
       "     {'umap_x': -9.41271686553955, 'umap_y': 4.94071626663208}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.0014608174096792936,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.0020052683539688587,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.0028810761868953705,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.0028895526193082333,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.0029111900366842747,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.0032799774780869484,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411085-C-A',\n",
       "      'distance': 0.0036033603828400373,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.0037093970458954573,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488233-C-A',\n",
       "      'distance': 0.003791575552895665,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437837-C-A',\n",
       "      'distance': 0.003792918985709548,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425829-G-A',\n",
       "      'distance': 0.0038561527617275715,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.003905337769538164,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48412715-G-A',\n",
       "      'distance': 0.003940812312066555,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495219-G-A',\n",
       "      'distance': 0.003964062314480543,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468070-G-A',\n",
       "      'distance': 0.00398043729364872,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411280-G-A',\n",
       "      'distance': 0.004043318796902895,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415540-G-A',\n",
       "      'distance': 0.004105288535356522,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.00410835724323988,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.004113491158932447,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465820-G-A',\n",
       "      'distance': 0.004145654384046793,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48485383-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48485383\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48485383',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3703T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48485383A>G',\n",
       "     'hgvs_protein': 'p.Ser1235Pro',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48485383, \"start\": 48485383, \"phenotype_or_disease\": 1, \"id\": \"CM117624\"}, {\"phenotype_or_disease\": 1, \"clin_sig\": [\"uncertain_significance\", \"pathogenic\"], \"start\": 48485383, \"var_synonyms\": {\"ClinVar\": [\"RCV000659531\", \"VCV000547313\", \"RCV003485624\"]}, \"id\": \"rs1555398382\", \"clin_sig_allele\": \"G:pathogenic;G:uncertain_significance\", \"end\": 48485383, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"A/G\"}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48485383A>G\", \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"cds_end\": 3703, \"protein_start\": 1235, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"S/P\", \"polyphen_prediction\": \"possibly_damaging\", \"polyphen_score\": 0.669, \"strand\": -1, \"mastermind_mmid3\": \"FBN1:S1235P\", \"canonical\": 1, \"clinpred\": 0.976584076881409, \"mutationtaster_pred\": \"A\", \"sift_score\": 0, \"mastermind_counts\": \"4|5|5\", \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 4019, \"exon\": \"30/66\", \"given_ref\": \"A\", \"eve_class\": \"Uncertain\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -1, \"impact\": \"MODERATE\", \"variant_allele\": \"G\", \"hgvsp\": \"NP_000129.3:p.Ser1235Pro\", \"aa\": \"A\", \"used_ref\": \"A\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.3703T>C\", \"gene_id\": \"2200\", \"biotype\": \"protein_coding\", \"alphamissense\": {\"am_pathogenicity\": 0.9462, \"am_class\": \"likely_pathogenic\"}, \"conservation\": 3.85, \"protein_end\": 1235, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48485382, \"end\": 48485383}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"eve_score\": 0.614432262186655, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 4019, \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": -17, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DP_AL\": 37, \"DP_DG\": -13, \"DS_DG\": 0, \"DS_DL\": 0, \"DP_AG\": 3}, \"cds_start\": 3703, \"codons\": \"Tca/Cca\", \"cadd_raw\": 4.606296, \"location\": \"15:48485383\", \"cadd_phred\": 25.9, \"allele_num\": 1}], \"minimised\": 1, \"allele_string\": \"A/G\", \"end\": 48485383, \"vcf_string\": \"15-48485383-A-G\", \"input\": \"NC_000015.10:g.48485383A>G\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48485383}',\n",
       "    'coordinates': [{'pca_x': 0.0387664512285959,\n",
       "      'pca_y': 0.08107975937249415},\n",
       "     {'t-sne_x': 36.25405698504863, 't-sne_y': 6.554444874961687},\n",
       "     {'umap_x': 1.3292235136032104, 'umap_y': 6.594879627227783}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48485397-A-G',\n",
       "      'distance': 0.0027924180030822754,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.002935245865955949,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.003257879987359047,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489911-A-G',\n",
       "      'distance': 0.0035789445973932743,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.003600968513637781,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520677-A-G',\n",
       "      'distance': 0.0036087920889258385,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.003666314762085676,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596373-A-G',\n",
       "      'distance': 0.003691134974360466,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0037369709461927414,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.00375959649682045,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468418-A-G',\n",
       "      'distance': 0.0039047945756465197,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.003961506299674511,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430770-A-G',\n",
       "      'distance': 0.003984159789979458,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.003987772855907679,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456688-A-G',\n",
       "      'distance': 0.004008070565760136,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.004120195284485817,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474567-A-G',\n",
       "      'distance': 0.004127931781113148,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.0041313935071229935,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437370-A-G',\n",
       "      'distance': 0.004137118346989155,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.004164590500295162,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.6},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48485380-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48485380\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48485380',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3706T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48485380A>G',\n",
       "     'hgvs_protein': 'p.Cys1236Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48485380A>G\", \"seq_region_name\": \"15\", \"start\": 48485380, \"minimised\": 1, \"end\": 48485380, \"vcf_string\": \"15-48485380-A-G\", \"allele_string\": \"A/G\", \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48485380, \"id\": \"CM055941\", \"start\": 48485380, \"phenotype_or_disease\": 1}, {\"clin_sig_allele\": \"G:likely_pathogenic;C:likely_pathogenic\", \"end\": 48485380, \"pubmed\": [17701892], \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"A/C/G\", \"phenotype_or_disease\": 1, \"start\": 48485380, \"clin_sig\": [\"likely_pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV002039454\", \"VCV001350034\", \"RCV000663661\", \"VCV000549182\"]}, \"id\": \"rs1555398380\"}], \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"allele_num\": 1, \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.746431365129081, \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": 4, \"DP_DG\": -10, \"DS_DG\": 0.01, \"DS_AG\": 0, \"DP_AL\": -6, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -3}, \"cds_start\": 3706, \"codons\": \"Tgc/Cgc\", \"cdna_end\": 4022, \"cadd_phred\": 28.4, \"cadd_raw\": 5.074485, \"location\": \"15:48485380\", \"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"end\": 48485380, \"start\": 48485379, \"type\": \"SimpleInterval\"}}}, \"conservation\": 3.85, \"protein_end\": 1236, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9989}, \"used_ref\": \"A\", \"aa\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys1236Arg\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.3706T>C\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"impact\": \"MODERATE\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -3, \"variant_allele\": \"G\", \"exon\": \"30/66\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 4022, \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"given_ref\": \"A\", \"eve_class\": \"Pathogenic\", \"ptriplo\": 0.984699554922396, \"polyphen_score\": 0.996, \"polyphen_prediction\": \"probably_damaging\", \"strand\": -1, \"clinpred\": 0.999850869178772, \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:C1236R\", \"mastermind_counts\": \"10|10|10\", \"sift_score\": 0, \"mutationtaster_pred\": \"A\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 3706, \"protein_start\": 1236, \"amino_acids\": \"C/R\", \"transcript_id\": \"NM_000138.5\"}], \"id\": \"NC_000015.10:g.48485380A>G\"}',\n",
       "    'coordinates': [{'pca_x': 0.047345599357762724,\n",
       "      'pca_y': 0.08938627330168447},\n",
       "     {'t-sne_x': 27.696464655384453, 't-sne_y': 6.5113848801985466},\n",
       "     {'umap_x': 2.603677272796631, 'umap_y': 7.12608003616333}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468001-A-G',\n",
       "      'distance': 0.0009242906235158443,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503850-A-G',\n",
       "      'distance': 0.001368425553664565,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425420-A-G',\n",
       "      'distance': 0.0014517463278025389,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415723-A-G',\n",
       "      'distance': 0.0017393981106579304,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.0018280819058418274,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481733-A-G',\n",
       "      'distance': 0.0018649329431355,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472665-A-G',\n",
       "      'distance': 0.0020540955010801554,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.0021874886006116867,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432911-A-G',\n",
       "      'distance': 0.0021889577619731426,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434600-A-G',\n",
       "      'distance': 0.002246642718091607,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432944-A-G',\n",
       "      'distance': 0.0022701236885041,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425414-A-G',\n",
       "      'distance': 0.0023352319840341806,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495153-A-G',\n",
       "      'distance': 0.002517447341233492,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.0025559579953551292,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.0025594066828489304,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.0025897379964590073,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.0026030545122921467,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474567-A-G',\n",
       "      'distance': 0.002749030478298664,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422024-A-G',\n",
       "      'distance': 0.002823697170242667,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.002959977835416794,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48483943-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48483943\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48483943',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3713A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48483943T>C',\n",
       "     'hgvs_protein': 'p.Asp1238Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"vcf_string\": \"15-48483943-T-C\", \"end\": 48483943, \"allele_string\": \"T/C\", \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48483943, \"id\": \"CM010039\", \"phenotype_or_disease\": 1, \"start\": 48483943}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"location\": \"15:48483943\", \"cadd_raw\": 5.726448, \"cadd_phred\": 33, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"cdna_end\": 4029, \"codons\": \"gAc/gGc\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -11, \"DS_DG\": 0, \"DP_DG\": 1, \"DP_AL\": 47, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -41, \"DS_AL\": 0}, \"cds_start\": 3713, \"maxentscan_diff\": -0.138, \"allele_num\": 1, \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.3713A>G\", \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Asp1238Gly\", \"aa\": \"T\", \"used_ref\": \"T\", \"conservation\": 3.95, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9941}, \"protein_end\": 1238, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48483943, \"start\": 48483942, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"biotype\": \"protein_coding\", \"given_ref\": \"T\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"31/66\", \"cdna_start\": 4029, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -1, \"impact\": \"MODERATE\", \"variant_allele\": \"C\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"protein_start\": 1238, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"D/G\", \"maxentscan_ref\": 7.711, \"protein_id\": \"NP_000129.3\", \"cds_end\": 3713, \"clinpred\": 0.995173156261444, \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:D1238G,FBN1:D1238sra\", \"mutationtaster_pred\": \"D\", \"sift_score\": 0.01, \"mastermind_counts\": \"6|15|18\", \"rf_score\": 0.762, \"polyphen_score\": 0.995, \"polyphen_prediction\": \"probably_damaging\", \"maxentscan_alt\": 7.849, \"ada_score\": 0.805755131882353, \"strand\": -1}], \"id\": \"NC_000015.10:g.48483943T>C\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48483943T>C\", \"start\": 48483943, \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': -0.0408719726582218,\n",
       "      'pca_y': 0.05262301501318855},\n",
       "     {'t-sne_x': -13.006490556339662, 't-sne_y': -3.844601531094112},\n",
       "     {'umap_x': 0.36959823966026306, 'umap_y': 15.037339210510254}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.0012673221062868834,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.0018028734484687448,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.0024693969171494246,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.002552993828430772,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.0029683990869671106,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.0036771234590560198,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487437-T-C',\n",
       "      'distance': 0.004255316220223904,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.004514053463935852,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468534-T-C',\n",
       "      'distance': 0.004598195664584637,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.004804817959666252,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.004993868060410023,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.005107291974127293,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.0051192608661949635,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.005141484551131725,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.005271732807159424,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.006013087462633848,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487075-C-T',\n",
       "      'distance': 0.006192355882376432,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.006448663771152496,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.006782229989767075,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.006845172960311174,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48483910-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48483910\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48483910',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3746G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48483910C>G',\n",
       "     'hgvs_protein': 'p.Cys1249Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"allele_string\": \"C/G\", \"end\": 48483910, \"vcf_string\": \"15-48483910-C-G\", \"colocated_variants\": [{\"id\": \"CM950446\", \"start\": 48483910, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48483910}, {\"phenotype_or_disease\": 1, \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"start\": 48483910, \"id\": \"rs137854458\", \"var_synonyms\": {\"UniProt\": [\"VAR_002323\"], \"ClinVar\": [\"RCV000017887\", \"VCV000016425\", \"RCV002345247\"], \"OMIM\": [134797.0005]}, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"C/G\", \"clin_sig_allele\": \"G:likely_pathogenic;G:pathogenic\", \"pubmed\": [1301946], \"end\": 48483910}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48483910C>G\", \"transcript_consequences\": [{\"cds_end\": 3746, \"protein_id\": \"NP_000129.3\", \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/S\", \"protein_start\": 1249, \"strand\": -1, \"polyphen_score\": 0.944, \"polyphen_prediction\": \"probably_damaging\", \"mutationtaster_pred\": \"A\", \"sift_score\": 0, \"mastermind_counts\": \"9|15|15\", \"canonical\": 1, \"clinpred\": 0.985155362711449, \"mastermind_mmid3\": \"FBN1:C1249S\", \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"exon\": \"31/66\", \"cdna_start\": 4062, \"ptriplo\": 0.984699554922396, \"given_ref\": \"C\", \"eve_class\": \"Pathogenic\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -1, \"impact\": \"MODERATE\", \"hgvsp\": \"NP_000129.3:p.Cys1249Ser\", \"used_ref\": \"C\", \"aa\": \"C\", \"hgvsc\": \"NM_000138.5:c.3746G>C\", \"gene_id\": \"2200\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"biotype\": \"protein_coding\", \"protein_end\": 1249, \"conservation\": 3.95, \"alphamissense\": {\"am_pathogenicity\": 0.9987, \"am_class\": \"likely_pathogenic\"}, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48483910, \"start\": 48483909, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"cdna_end\": 4062, \"codons\": \"tGt/tCt\", \"spliceai\": {\"DP_DG\": -8, \"DS_DG\": 0, \"DS_DL\": 0, \"DP_AG\": -14, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -39, \"DS_AG\": 0, \"DP_AL\": 33}, \"cds_start\": 3746, \"eve_score\": 0.794909700737277, \"consequence_terms\": [\"missense_variant\"], \"cadd_raw\": 5.142204, \"location\": \"15:48483910\", \"cadd_phred\": 28.8, \"allele_num\": 1}], \"input\": \"NC_000015.10:g.48483910C>G\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"start\": 48483910, \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.032890769348754484,\n",
       "      'pca_y': 0.10276695128467936},\n",
       "     {'t-sne_x': -13.865112002091031, 't-sne_y': 28.466889823477185},\n",
       "     {'umap_x': 1.668426752090454, 'umap_y': 22.15416145324707}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463949-C-G',\n",
       "      'distance': 0.0014517444651573896,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483823-C-G',\n",
       "      'distance': 0.0014893418410792947,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-G',\n",
       "      'distance': 0.0029244106262922287,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425412-G-C',\n",
       "      'distance': 0.003065410302951932,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503845-G-C',\n",
       "      'distance': 0.003492739051580429,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.0035064932890236378,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.0037974310107529163,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481690-C-T',\n",
       "      'distance': 0.0038325877394527197,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.0038681100122630596,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472628-C-T',\n",
       "      'distance': 0.0038997831288725138,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.0039518303237855434,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.0039785392582416534,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472644-A-G',\n",
       "      'distance': 0.0040529523976147175,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.004080366808921099,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494210-A-G',\n",
       "      'distance': 0.004090555012226105,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437026-T-C',\n",
       "      'distance': 0.004105263389647007,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495131-C-T',\n",
       "      'distance': 0.004106201697140932,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.004132529255002737,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495519-C-G',\n",
       "      'distance': 0.004191308747977018,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.004206416197121143,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48483817-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48483817\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48483817',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3838+1G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48483817C>T',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48483817C>T\", \"transcript_consequences\": [{\"strand\": -1, \"ada_score\": 0.999990361384395, \"rf_score\": 0.938, \"maxentscan_alt\": 0.711, \"mastermind_counts\": \"9|9|10\", \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:D1280sd\", \"maxentscan_ref\": 8.893, \"protein_id\": \"NP_000129.3\", \"transcript_id\": \"NM_000138.5\", \"lof\": \"HC\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"T\", \"lof_info\": \"INTRON_SIZE:2037\", \"mane_select\": \"ENST00000316623.10\", \"impact\": \"HIGH\", \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"given_ref\": \"C\", \"biotype\": \"protein_coding\", \"conservation\": 3.95, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48483817, \"start\": 48483816, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}}, \"used_ref\": \"C\", \"aa\": \"C\", \"intron\": \"31/65\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3838+1G>A\", \"mane\": [\"MANE_Select\"], \"allele_num\": 1, \"maxentscan_diff\": 8.182, \"spliceai\": {\"DS_AG\": 0, \"DP_AL\": 5, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 1, \"DS_DL\": 0.99, \"DP_AG\": -34, \"DP_DG\": -34, \"DS_DG\": 0.64}, \"consequence_terms\": [\"splice_donor_variant\"], \"location\": \"15:48483817\", \"cadd_raw\": 6.740862, \"cadd_phred\": 35}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"start\": 48483817, \"phenotype_or_disease\": 1, \"id\": \"CS098575\", \"end\": 48483817, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\"}, {\"id\": \"CS105215\", \"phenotype_or_disease\": 1, \"start\": 48483817, \"end\": 48483817, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\"}], \"variant_class\": \"SNV\", \"allele_string\": \"C/T\", \"vcf_string\": \"15-48483817-C-T\", \"end\": 48483817, \"minimised\": 1, \"start\": 48483817, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48483817C>T\", \"strand\": 1, \"most_severe_consequence\": \"splice_donor_variant\"}',\n",
       "    'coordinates': [{'pca_x': -0.2435104432979021,\n",
       "      'pca_y': -0.03029435660582034},\n",
       "     {'t-sne_x': -16.892650353848122, 't-sne_y': -26.90302596551247},\n",
       "     {'umap_x': 7.895468711853027, 'umap_y': 12.851375579833984}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48474254-C-T',\n",
       "      'distance': 0.0006409529014490545,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444540-C-T',\n",
       "      'distance': 0.0011481575202196836,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48441720-C-T',\n",
       "      'distance': 0.0012294237967580557,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.001316626789048314,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-T',\n",
       "      'distance': 0.0013322621816769242,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.0014299775939434767,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.001987189520150423,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496099-C-T',\n",
       "      'distance': 0.002025470370426774,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.002046954818069935,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.002122445497661829,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428345-C-T',\n",
       "      'distance': 0.002183289034292102,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.002192651154473424,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.0023383155930787325,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497265-C-T',\n",
       "      'distance': 0.0023411656729876995,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488112-C-T',\n",
       "      'distance': 0.0023791524581611156,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.0026523680426180363,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-A',\n",
       "      'distance': 0.00270628510043025,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432865-C-T',\n",
       "      'distance': 0.0031020804308354855,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425738-C-T',\n",
       "      'distance': 0.003108044620603323,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.0031282855197787285,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48483815-T-A',\n",
       "    'metadata': {'vcf_string': '15\\t48483815\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48483815',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3838+3A>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48483815T>A',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_region_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"maxentscan_diff\": 6.316, \"allele_num\": 1, \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": 3, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DP_AL\": 20, \"DP_DG\": 7, \"DS_DG\": 0.57, \"DS_DL\": 0.05, \"DP_AG\": 3}, \"consequence_terms\": [\"splice_donor_region_variant\", \"intron_variant\"], \"location\": \"15:48483815\", \"cadd_raw\": 5.048395, \"cadd_phred\": 28.2, \"biotype\": \"protein_coding\", \"conservation\": 3.72, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48483815, \"type\": \"SimpleInterval\", \"start\": 48483814}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"aa\": \"T\", \"used_ref\": \"T\", \"intron\": \"31/65\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3838+3A>T\", \"mane\": [\"MANE_Select\"], \"gene_symbol\": \"FBN1\", \"variant_allele\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"impact\": \"LOW\", \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"given_ref\": \"T\", \"ada_score\": 0.999383870999031, \"strand\": -1, \"rf_score\": 0.942, \"maxentscan_alt\": 2.577, \"mastermind_counts\": \"3|3|15\", \"mastermind_mmid3\": \"FBN1:D1280int,FBN1:D1280intd,FBN1:D1280srd\", \"canonical\": 1, \"maxentscan_ref\": 8.893, \"protein_id\": \"NP_000129.3\", \"transcript_id\": \"NM_000138.5\"}], \"id\": \"NC_000015.10:g.48483815T>A\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"start\": 48483815, \"id\": \"CS040145\", \"end\": 48483815, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\"}, {\"phenotype_or_disease\": 1, \"start\": 48483815, \"clin_sig\": [\"uncertain_significance\"], \"var_synonyms\": {\"ClinVar\": [\"RCV000663668\", \"VCV000549188\"]}, \"id\": \"rs1555398265\", \"clin_sig_allele\": \"A:uncertain_significance\", \"end\": 48483815, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"T/A\"}], \"assembly_name\": \"GRCh38\", \"end\": 48483815, \"vcf_string\": \"15-48483815-T-A\", \"allele_string\": \"T/A\", \"minimised\": 1, \"seq_region_name\": \"15\", \"start\": 48483815, \"most_severe_consequence\": \"splice_donor_region_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48483815T>A\"}',\n",
       "    'coordinates': [{'pca_x': -0.2502465656761303,\n",
       "      'pca_y': -0.051186499011514223},\n",
       "     {'t-sne_x': -17.184868667079527, 't-sne_y': -33.8209332783799},\n",
       "     {'umap_x': 13.691357612609863, 'umap_y': 12.390631675720215}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48445370-A-G',\n",
       "      'distance': 0.004227661527693272,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427561-A-C',\n",
       "      'distance': 0.005951208062469959,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48437318-T-C',\n",
       "      'distance': 0.006096808239817619,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48437765-T-C',\n",
       "      'distance': 0.006565380841493607,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489845-T-C',\n",
       "      'distance': 0.008283276110887527,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48487309-T-C',\n",
       "      'distance': 0.009928558021783829,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48412564-C-T',\n",
       "      'distance': 0.012097994796931744,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508577-C-T',\n",
       "      'distance': 0.013149132952094078,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460241-C-T',\n",
       "      'distance': 0.013627586886286736,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446701-C-A',\n",
       "      'distance': 0.01403987780213356,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515385-A-G',\n",
       "      'distance': 0.014074547216296196,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446701-C-T',\n",
       "      'distance': 0.014172995463013649,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472544-C-A',\n",
       "      'distance': 0.01468227431178093,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.01759759709239006,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-A',\n",
       "      'distance': 0.01771366596221924,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.01843745820224285,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520652-C-T',\n",
       "      'distance': 0.018489962443709373,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.018592504784464836,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441720-C-T',\n",
       "      'distance': 0.018839556723833084,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508581-C-A',\n",
       "      'distance': 0.01886506751179695,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_benign',\n",
       "      'confidence_score': 0.6},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.6},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.6666666666666666},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.7}}},\n",
       "   {'variant_id': '15-48481781-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48481781\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48481781',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3839-1G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48481781C>A',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"start\": 48481781, \"strand\": 1, \"most_severe_consequence\": \"splice_acceptor_variant\", \"input\": \"NC_000015.10:g.48481781C>A\", \"end\": 48481781, \"vcf_string\": \"15-48481781-C-A\", \"allele_string\": \"C/A\", \"minimised\": 1, \"transcript_consequences\": [{\"given_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"INTRON_SIZE:2037\", \"impact\": \"HIGH\", \"variant_allele\": \"A\", \"gene_symbol\": \"FBN1\", \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"maxentscan_ref\": 6.442, \"protein_id\": \"NP_000129.3\", \"mastermind_mmid3\": \"FBN1:D1280sa\", \"canonical\": 1, \"mastermind_counts\": \"7|7|12\", \"rf_score\": 0.926, \"maxentscan_alt\": -2.156, \"strand\": -1, \"ada_score\": 0.999989072007339, \"cadd_raw\": 5.815611, \"location\": \"15:48481781\", \"cadd_phred\": 33, \"consequence_terms\": [\"splice_acceptor_variant\"], \"spliceai\": {\"DS_DG\": 0, \"DP_DG\": -2, \"DS_DL\": 0, \"DP_AG\": 1, \"DP_DL\": -1, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0.99, \"DP_AL\": -1, \"DS_AG\": 0.04}, \"allele_num\": 1, \"maxentscan_diff\": 8.598, \"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3839-1G>T\", \"intron\": \"31/65\", \"aa\": \"C\", \"used_ref\": \"C\", \"conservation\": 3.85, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48481781, \"start\": 48481780, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"type\": \"Allele\"}, \"biotype\": \"protein_coding\", \"genesplicer\": \"loss/acceptor/48481780-48481781/Medium/6.905423\"}], \"id\": \"NC_000015.10:g.48481781C>A\", \"colocated_variants\": [{\"id\": \"CS961585\", \"start\": 48481781, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48481781}, {\"clin_sig_allele\": \"A:pathogenic\", \"end\": 48481781, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"C/A\", \"start\": 48481781, \"clin_sig\": [\"pathogenic\"], \"phenotype_or_disease\": 1, \"id\": \"rs794728211\", \"var_synonyms\": {\"ClinVar\": [\"RCV000181497\", \"VCV000200026\"]}}], \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': -0.2392527920075174,\n",
       "      'pca_y': -0.03387304109210877},\n",
       "     {'t-sne_x': -28.923483735001668, 't-sne_y': -18.51886267471719},\n",
       "     {'umap_x': 11.879687309265137, 'umap_y': 18.71509552001953}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.0028456272557377815,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.003168545663356781,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.0032593058422207832,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.0033277077600359917,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-A',\n",
       "      'distance': 0.0033390913158655167,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.004364874213933945,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.004814230836927891,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.004907302092760801,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.004988356493413448,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.004994165617972612,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.005162662826478481,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.005178287625312805,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.005210262723267078,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.005250272341072559,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.005425526760518551,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.006071255076676607,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470757-C-T',\n",
       "      'distance': 0.006503025069832802,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.006543870083987713,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446823-C-G',\n",
       "      'distance': 0.006773343775421381,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.007669324055314064,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48481721-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48481721\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48481721',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3898A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48481721T>C',\n",
       "     'hgvs_protein': 'p.Lys1300Glu',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48481721, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48481721T>C\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"id\": \"NC_000015.10:g.48481721T>C\", \"transcript_consequences\": [{\"clinpred\": 0.956307053565979, \"mastermind_mmid3\": \"FBN1:K1300E\", \"canonical\": 1, \"sift_score\": 0, \"mastermind_counts\": \"6|10|10\", \"mutationtaster_pred\": \"D\", \"polyphen_score\": 0.982, \"polyphen_prediction\": \"probably_damaging\", \"strand\": -1, \"protein_start\": 1300, \"amino_acids\": \"K/E\", \"transcript_id\": \"NM_000138.5\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 3898, \"impact\": \"MODERATE\", \"blosum62\": 1, \"mane_select\": \"ENST00000316623.10\", \"variant_allele\": \"C\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"given_ref\": \"T\", \"eve_class\": \"Uncertain\", \"ptriplo\": 0.984699554922396, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"32/66\", \"phaplo\": 0.99897615427687, \"cdna_start\": 4214, \"loeuf\": 0.105, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48481720, \"end\": 48481721}, \"type\": \"SequenceLocation\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}}, \"alphamissense\": {\"am_class\": \"ambiguous\", \"am_pathogenicity\": 0.4725}, \"protein_end\": 1300, \"conservation\": 3.85, \"biotype\": \"protein_coding\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3898A>G\", \"used_ref\": \"T\", \"aa\": \"T\", \"hgvsp\": \"NP_000129.3:p.Lys1300Glu\", \"allele_num\": 1, \"cadd_phred\": 25.9, \"cadd_raw\": 4.609044, \"location\": \"15:48481721\", \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.465835422874742, \"cds_start\": 3898, \"codons\": \"Aaa/Gaa\", \"spliceai\": {\"DS_AG\": 0, \"DP_AL\": -4, \"SYMBOL\": \"FBN1\", \"DP_DL\": -48, \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": 19, \"DS_DG\": 0, \"DP_DG\": -21}, \"cdna_end\": 4214}], \"colocated_variants\": [{\"id\": \"CM010040\", \"phenotype_or_disease\": 1, \"start\": 48481721, \"end\": 48481721, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\"}, {\"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"end\": 48481721, \"phenotype_or_disease\": 1, \"start\": 48481721, \"id\": \"CM161832\"}], \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"T/C\", \"vcf_string\": \"15-48481721-T-C\", \"end\": 48481721, \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.03249772587163108,\n",
       "      'pca_y': 0.10588818428287824},\n",
       "     {'t-sne_x': -11.43081712778249, 't-sne_y': 29.93094520928127},\n",
       "     {'umap_x': 1.0133767127990723, 'umap_y': 22.323667526245117}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48481729-T-C',\n",
       "      'distance': 0.0024115312844514847,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48481690-C-T',\n",
       "      'distance': 0.0029727357905358076,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.0029777083545923233,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.003049948252737522,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441765-C-T',\n",
       "      'distance': 0.0032476126216351986,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437026-T-C',\n",
       "      'distance': 0.00335673987865448,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.0033847959712147713,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492538-C-T',\n",
       "      'distance': 0.0033927231561392546,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495156-C-T',\n",
       "      'distance': 0.0034274649806320667,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.003460160456597805,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495131-C-T',\n",
       "      'distance': 0.0034970708657056093,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.003509996924549341,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-C',\n",
       "      'distance': 0.0035115471109747887,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.003540606237947941,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421990-C-T',\n",
       "      'distance': 0.003542256774380803,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.0035907297860831022,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421651-C-T',\n",
       "      'distance': 0.003635847009718418,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472628-C-T',\n",
       "      'distance': 0.0036986605264246464,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437038-C-T',\n",
       "      'distance': 0.0037904465571045876,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474260-C-T',\n",
       "      'distance': 0.00379074364900589,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48481661-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48481661\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48481661',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3958T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48481661A>T',\n",
       "     'hgvs_protein': 'p.Cys1320Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"start\": 48481661, \"input\": \"NC_000015.10:g.48481661A>T\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"allele_string\": \"A/T\", \"end\": 48481661, \"vcf_string\": \"15-48481661-A-T\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48481661A>T\", \"transcript_consequences\": [{\"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9987}, \"protein_end\": 1320, \"conservation\": 3.85, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48481660, \"end\": 48481661}}}, \"biotype\": \"protein_coding\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3958T>A\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"hgvsp\": \"NP_000129.3:p.Cys1320Ser\", \"used_ref\": \"A\", \"aa\": \"A\", \"allele_num\": 1, \"location\": \"15:48481661\", \"cadd_raw\": 5.279688, \"cadd_phred\": 29.7, \"cdna_end\": 4274, \"codons\": \"Tgt/Agt\", \"cds_start\": 3958, \"spliceai\": {\"DP_DL\": -6, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AL\": 11, \"DS_AG\": 0, \"DS_DG\": 0, \"DP_DG\": 0, \"DP_AG\": -2, \"DS_DL\": 0}, \"eve_score\": 0.779488841920983, \"consequence_terms\": [\"missense_variant\"], \"mutationtaster_pred\": \"D\", \"mastermind_counts\": \"8|13|14\", \"sift_score\": 0, \"mastermind_mmid3\": \"FBN1:C1320S\", \"clinpred\": 0.999770939350128, \"canonical\": 1, \"strand\": -1, \"polyphen_prediction\": \"probably_damaging\", \"polyphen_score\": 0.978, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/S\", \"protein_start\": 1320, \"cds_end\": 3958, \"protein_id\": \"NP_000129.3\", \"variant_allele\": \"T\", \"blosum62\": -1, \"mane_select\": \"ENST00000316623.10\", \"impact\": \"MODERATE\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"A\", \"eve_class\": \"Pathogenic\", \"loeuf\": 0.105, \"cdna_start\": 4274, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"exon\": \"32/66\"}], \"variant_class\": \"SNV\", \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48481661, \"id\": \"CM085410\", \"phenotype_or_disease\": 1, \"start\": 48481661}, {\"id\": \"CM972808\", \"start\": 48481661, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48481661}], \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': 0.04186034207512701,\n",
       "      'pca_y': 0.07548625210189511},\n",
       "     {'t-sne_x': 14.92654352364115, 't-sne_y': 9.215922146809742},\n",
       "     {'umap_x': 3.2001521587371826, 'umap_y': 4.705179691314697}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.0011399921495467424,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428439-A-T',\n",
       "      'distance': 0.0019296938553452492,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430791-A-T',\n",
       "      'distance': 0.0020791252609342337,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.0028276965022087097,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427603-A-G',\n",
       "      'distance': 0.002975564682856202,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415567-A-G',\n",
       "      'distance': 0.003090867307037115,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.003101394744589925,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427684-A-T',\n",
       "      'distance': 0.003111201338469982,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.003115254221484065,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0031386595219373703,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596325-A-T',\n",
       "      'distance': 0.003153448924422264,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.0032065603882074356,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.00326708797365427,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487317-C-T',\n",
       "      'distance': 0.003272272413596511,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481699-C-T',\n",
       "      'distance': 0.003290136344730854,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.003294190624728799,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463904-C-A',\n",
       "      'distance': 0.003326871432363987,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.003329896368086338,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.0033338330686092377,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.003336475230753422,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48481654-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48481654\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48481654',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3964+1G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48481654C>T',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48481654C>T\", \"strand\": 1, \"most_severe_consequence\": \"splice_donor_variant\", \"start\": 48481654, \"seq_region_name\": \"15\", \"minimised\": 1, \"allele_string\": \"C/T\", \"vcf_string\": \"15-48481654-C-T\", \"end\": 48481654, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"start\": 48481654, \"id\": \"CS111088\", \"end\": 48481654, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\"}, {\"id\": \"CS991391\", \"start\": 48481654, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48481654}, {\"end\": 48481654, \"clin_sig_allele\": \"T:pathogenic\", \"allele_string\": \"C/T\", \"strand\": 1, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV000181500\", \"VCV000200028\"]}, \"id\": \"rs794728212\", \"phenotype_or_disease\": 1, \"clin_sig\": [\"pathogenic\"], \"start\": 48481654}], \"assembly_name\": \"GRCh38\", \"id\": \"NC_000015.10:g.48481654C>T\", \"transcript_consequences\": [{\"allele_num\": 1, \"maxentscan_diff\": 8.182, \"consequence_terms\": [\"splice_donor_variant\"], \"spliceai\": {\"DP_DG\": 7, \"DS_DG\": 0.02, \"DP_AG\": -13, \"DS_DL\": 0.99, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 1, \"DS_AG\": 0, \"DP_AL\": 1}, \"location\": \"15:48481654\", \"cadd_raw\": 6.011004, \"cadd_phred\": 34, \"biotype\": \"protein_coding\", \"conservation\": 3.85, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"end\": 48481654, \"start\": 48481653, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}}, \"intron\": \"32/65\", \"aa\": \"C\", \"used_ref\": \"C\", \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.3964+1G>A\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"lof_info\": \"INTRON_SIZE:7004\", \"mane_select\": \"ENST00000316623.10\", \"impact\": \"HIGH\", \"variant_allele\": \"T\", \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"given_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"rf_score\": 0.928, \"maxentscan_alt\": -1.191, \"ada_score\": 0.999990361384395, \"strand\": -1, \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:D1322sd\", \"mastermind_counts\": \"7|7|12\", \"maxentscan_ref\": 6.991, \"protein_id\": \"NP_000129.3\", \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\"}]}',\n",
       "    'coordinates': [{'pca_x': -0.24816157935328828,\n",
       "      'pca_y': -0.026506253655030322},\n",
       "     {'t-sne_x': -18.231606138436504, 't-sne_y': -25.52318300412368},\n",
       "     {'umap_x': 8.08526611328125, 'umap_y': 13.248522758483887}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48421557-C-T',\n",
       "      'distance': 0.0008771413704380393,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474254-C-T',\n",
       "      'distance': 0.0026381216011941433,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432865-C-T',\n",
       "      'distance': 0.0033541463781148195,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441720-C-T',\n",
       "      'distance': 0.0034929336979985237,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.003513422328978777,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48444540-C-T',\n",
       "      'distance': 0.003533774521201849,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456636-C-T',\n",
       "      'distance': 0.0035365985240787268,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.0036814059130847454,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425738-C-T',\n",
       "      'distance': 0.0037190988659858704,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427566-C-T',\n",
       "      'distance': 0.003731284523382783,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.00387406162917614,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-T',\n",
       "      'distance': 0.003877410665154457,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515392-C-T',\n",
       "      'distance': 0.003994550555944443,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.004157843068242073,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428345-C-T',\n",
       "      'distance': 0.004184633027762175,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.00420564878731966,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503786-C-T',\n",
       "      'distance': 0.004277252126485109,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.004372996278107166,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.004478094168007374,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.0045549217611551285,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48474652-T-A',\n",
       "    'metadata': {'vcf_string': '15\\t48474652\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474652',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3965-2A>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474652T>A',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"splice_acceptor_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48474652T>A\", \"start\": 48474652, \"seq_region_name\": \"15\", \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"end\": 48474652, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"CS055582\", \"phenotype_or_disease\": 1, \"start\": 48474652}, {\"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48474652, \"start\": 48474652, \"phenotype_or_disease\": 1, \"id\": \"CS951411\"}, {\"id\": \"rs387906547\", \"var_synonyms\": {\"OMIM\": [134797.0019], \"ClinVar\": [\"RCV000017903\", \"VCV000016441\"]}, \"phenotype_or_disease\": 1, \"start\": 48474652, \"clin_sig\": [\"pathogenic\"], \"end\": 48474652, \"pubmed\": [7633409], \"clin_sig_allele\": \"A:pathogenic\", \"allele_string\": \"T/A\", \"strand\": 1, \"seq_region_name\": \"15\"}], \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"gene_symbol_source\": \"EntrezGene\", \"given_ref\": \"T\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"INTRON_SIZE:7004\", \"impact\": \"HIGH\", \"variant_allele\": \"A\", \"maxentscan_ref\": 7.714, \"protein_id\": \"NP_000129.3\", \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"rf_score\": 0.928, \"maxentscan_alt\": -0.653, \"strand\": -1, \"ada_score\": 0.999988247793601, \"mastermind_mmid3\": \"FBN1:D1322sa\", \"canonical\": 1, \"mastermind_counts\": \"5|5|9\", \"consequence_terms\": [\"splice_acceptor_variant\"], \"spliceai\": {\"DP_DG\": 7, \"DS_DG\": 0, \"DP_AG\": -26, \"DS_DL\": 0, \"DS_AL\": 1, \"SYMBOL\": \"FBN1\", \"DP_DL\": -48, \"DP_AL\": -2, \"DS_AG\": 0.02}, \"cadd_raw\": 5.321607, \"location\": \"15:48474652\", \"cadd_phred\": 29.9, \"allele_num\": 1, \"maxentscan_diff\": 8.367, \"intron\": \"32/65\", \"used_ref\": \"T\", \"aa\": \"T\", \"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3965-2A>T\", \"biotype\": \"protein_coding\", \"genesplicer\": \"loss/acceptor/48474652-48474653/Medium/6.752076\", \"conservation\": 3.95, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48474652, \"start\": 48474651, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}}}], \"id\": \"NC_000015.10:g.48474652T>A\", \"minimised\": 1, \"vcf_string\": \"15-48474652-T-A\", \"end\": 48474652, \"allele_string\": \"T/A\"}',\n",
       "    'coordinates': [{'pca_x': -0.23152720917263694,\n",
       "      'pca_y': -0.03892141940860539},\n",
       "     {'t-sne_x': -29.921628510162687, 't-sne_y': -18.852934426636587},\n",
       "     {'umap_x': 11.916655540466309, 'umap_y': 18.678564071655273}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48503940-C-T',\n",
       "      'distance': 0.004178179427981377,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596379-C-T',\n",
       "      'distance': 0.005229135509580374,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430803-C-T',\n",
       "      'distance': 0.00585842365399003,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446823-C-T',\n",
       "      'distance': 0.005987121257930994,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-A',\n",
       "      'distance': 0.006131540518254042,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472677-C-T',\n",
       "      'distance': 0.0063833934254944324,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425866-T-C',\n",
       "      'distance': 0.0068901050835847855,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470758-T-C',\n",
       "      'distance': 0.006910392083227634,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503941-T-C',\n",
       "      'distance': 0.007251183968037367,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492588-T-C',\n",
       "      'distance': 0.007273015100508928,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437919-T-C',\n",
       "      'distance': 0.0072818901389837265,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463241-C-T',\n",
       "      'distance': 0.0073799858801066875,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460319-T-C',\n",
       "      'distance': 0.007561381906270981,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48516364-T-C',\n",
       "      'distance': 0.00784831028431654,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508706-T-C',\n",
       "      'distance': 0.00825319904834032,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520818-C-G',\n",
       "      'distance': 0.008591923862695694,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600236-T-C',\n",
       "      'distance': 0.008916143327951431,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470757-C-T',\n",
       "      'distance': 0.009757463820278645,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48464022-C-G',\n",
       "      'distance': 0.010399946011602879,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452686-T-C',\n",
       "      'distance': 0.01173735223710537,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48474642-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48474642\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474642',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3973G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474642C>A',\n",
       "     'hgvs_protein': 'p.Glu1325Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"start\": 48474642, \"input\": \"NC_000015.10:g.48474642C>A\", \"most_severe_consequence\": \"stop_gained\", \"strand\": 1, \"allele_string\": \"C/A\", \"vcf_string\": \"15-48474642-C-A\", \"end\": 48474642, \"minimised\": 1, \"id\": \"NC_000015.10:g.48474642C>A\", \"transcript_consequences\": [{\"loeuf\": 0.105, \"cdna_start\": 4289, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"33/66\", \"phaplo\": 0.99897615427687, \"given_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"PERCENTILE:0.461118848653668,GERP_DIST:5440.17400687039,BP_DIST:4610,DIST_FROM_LAST_EXON:4221,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"impact\": \"HIGH\", \"variant_allele\": \"A\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 3973, \"protein_start\": 1325, \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"E/*\", \"strand\": -1, \"mastermind_mmid3\": \"FBN1:E1325X\", \"canonical\": 1, \"mutationtaster_pred\": \"A\", \"mastermind_counts\": \"6|6|6\", \"consequence_terms\": [\"stop_gained\"], \"cdna_end\": 4289, \"spliceai\": {\"DP_AG\": -16, \"DS_DL\": 0, \"DP_DG\": -3, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": 8, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -38}, \"cds_start\": 3973, \"codons\": \"Gaa/Taa\", \"cadd_raw\": 9.953323, \"location\": \"15:48474642\", \"cadd_phred\": 44, \"allele_num\": 1, \"hgvsp\": \"NP_000129.3:p.Glu1325Ter\", \"used_ref\": \"C\", \"aa\": \"C\", \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.3973G>T\", \"gene_id\": \"2200\", \"biotype\": \"protein_coding\", \"conservation\": 3.95, \"protein_end\": 1325, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"end\": 48474642, \"type\": \"SimpleInterval\", \"start\": 48474641}}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}}}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"end\": 48474642, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"id\": \"CM010041\", \"start\": 48474642, \"phenotype_or_disease\": 1}, {\"end\": 48474642, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"CM040036\", \"phenotype_or_disease\": 1, \"start\": 48474642}, {\"end\": 48474642, \"allele_string\": \"COSMIC_MUTATION\", \"somatic\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM9225437\"]}, \"id\": \"COSV104608679\", \"start\": 48474642, \"phenotype_or_disease\": 1}, {\"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"C/A\", \"clin_sig_allele\": \"A:pathogenic\", \"pubmed\": [11175294, 17657824], \"end\": 48474642, \"clin_sig\": [\"pathogenic\"], \"start\": 48474642, \"phenotype_or_disease\": 1, \"id\": \"rs2141280296\", \"var_synonyms\": {\"ClinVar\": [\"RCV001386705\", \"VCV001073642\"]}}], \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.0205422569877087,\n",
       "      'pca_y': 0.09602965900231078},\n",
       "     {'t-sne_x': 22.615219757267205, 't-sne_y': 42.016174615795435},\n",
       "     {'umap_x': -9.555595397949219, 'umap_y': 4.626873016357422}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.0008152592927217484,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.001139261992648244,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411085-C-A',\n",
       "      'distance': 0.0013900056947022676,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437837-C-A',\n",
       "      'distance': 0.0014154216041788459,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.001415754435583949,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488233-C-A',\n",
       "      'distance': 0.0016750902868807316,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.002005974994972348,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.0025007922668009996,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.003111753612756729,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48483878-C-A',\n",
       "      'distance': 0.0036192364059388638,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452676-C-A',\n",
       "      'distance': 0.0036330160219222307,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505139-C-A',\n",
       "      'distance': 0.003703879192471504,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48436970-C-A',\n",
       "      'distance': 0.003873503999784589,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.004296694882214069,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489869-C-A',\n",
       "      'distance': 0.004398710560053587,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515397-C-T',\n",
       "      'distance': 0.004426994360983372,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.004521803930401802,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.004525299184024334,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415540-G-A',\n",
       "      'distance': 0.004566011019051075,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.004748311825096607,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48474639-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48474639\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474639',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.3976T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474639A>G',\n",
       "     'hgvs_protein': 'p.Cys1326Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"A/G\", \"end\": 48474639, \"vcf_string\": \"15-48474639-A-G\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48474639A>G\", \"transcript_consequences\": [{\"mastermind_counts\": \"10|12|12\", \"sift_score\": 0, \"mutationtaster_pred\": \"D\", \"clinpred\": 0.999844193458557, \"mastermind_mmid3\": \"FBN1:C1326R\", \"canonical\": 1, \"strand\": -1, \"polyphen_prediction\": \"possibly_damaging\", \"polyphen_score\": 0.666, \"amino_acids\": \"C/R\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 1326, \"cds_end\": 3976, \"protein_id\": \"NP_000129.3\", \"variant_allele\": \"G\", \"impact\": \"MODERATE\", \"blosum62\": -3, \"mane_select\": \"ENST00000316623.10\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"A\", \"eve_class\": \"Pathogenic\", \"cdna_start\": 4292, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"exon\": \"33/66\", \"loeuf\": 0.105, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48474639, \"type\": \"SimpleInterval\", \"start\": 48474638}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"alphamissense\": {\"am_pathogenicity\": 0.9986, \"am_class\": \"likely_pathogenic\"}, \"conservation\": 3.95, \"protein_end\": 1326, \"biotype\": \"protein_coding\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.3976T>C\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"used_ref\": \"A\", \"aa\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys1326Arg\", \"allele_num\": 1, \"cadd_phred\": 29.2, \"cadd_raw\": 5.198335, \"location\": \"15:48474639\", \"codons\": \"Tgt/Cgt\", \"spliceai\": {\"DS_DG\": 0, \"DP_DG\": -35, \"DS_DL\": 0, \"DP_AG\": 11, \"SYMBOL\": \"FBN1\", \"DP_DL\": 11, \"DS_AL\": 0, \"DP_AL\": -13, \"DS_AG\": 0}, \"cds_start\": 3976, \"cdna_end\": 4292, \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.793224310943385}], \"colocated_variants\": [{\"end\": 48474639, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"id\": \"CM992325\", \"phenotype_or_disease\": 1, \"start\": 48474639}, {\"id\": \"rs2043405278\", \"var_synonyms\": {\"UniProt\": [\"VAR_076063\"], \"ClinVar\": [\"RCV003041233\", \"VCV002137695\"]}, \"start\": 48474639, \"clin_sig\": [\"pathogenic\"], \"phenotype_or_disease\": 1, \"end\": 48474639, \"pubmed\": [17657824, 17701892], \"clin_sig_allele\": \"G:pathogenic\", \"allele_string\": \"A/G\", \"seq_region_name\": \"15\", \"strand\": 1}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"start\": 48474639, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48474639A>G\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\"}',\n",
       "    'coordinates': [{'pca_x': 0.04974596893963357,\n",
       "      'pca_y': 0.08045590107038153},\n",
       "     {'t-sne_x': 28.933543269203458, 't-sne_y': 5.227139034781744},\n",
       "     {'umap_x': 2.4421703815460205, 'umap_y': 6.990270614624023}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.0007275867974385619,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.0010368071962147951,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432944-A-G',\n",
       "      'distance': 0.0010519545758143067,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434600-A-G',\n",
       "      'distance': 0.0014046181458979845,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.0014119480038061738,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481733-A-G',\n",
       "      'distance': 0.0014231333043426275,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472665-A-G',\n",
       "      'distance': 0.0015299315564334393,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.001625180710107088,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.0016440781764686108,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432911-A-G',\n",
       "      'distance': 0.0017421925440430641,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.0018245077226310968,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434702-A-G',\n",
       "      'distance': 0.001836872659623623,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470679-A-G',\n",
       "      'distance': 0.0020064380951225758,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474567-A-G',\n",
       "      'distance': 0.0020605858881026506,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437873-A-G',\n",
       "      'distance': 0.00215766578912735,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.0021898651029914618,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415723-A-G',\n",
       "      'distance': 0.0022474813740700483,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452655-A-G',\n",
       "      'distance': 0.0023411205038428307,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596325-A-G',\n",
       "      'distance': 0.0023423272650688887,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425414-A-G',\n",
       "      'distance': 0.00259475689381361,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48474606-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48474606\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474606',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4009G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474606C>G',\n",
       "     'hgvs_protein': 'p.Ala1337Pro',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48474606, \"seq_region_name\": \"15\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48474606C>G\", \"transcript_consequences\": [{\"hgvsp\": \"NP_000129.3:p.Ala1337Pro\", \"used_ref\": \"C\", \"aa\": \"C\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.4009G>C\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"biotype\": \"protein_coding\", \"alphamissense\": {\"am_pathogenicity\": 0.9807, \"am_class\": \"likely_pathogenic\"}, \"conservation\": 3.95, \"protein_end\": 1337, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48474606, \"start\": 48474605, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"type\": \"Allele\"}, \"cdna_end\": 4325, \"spliceai\": {\"DP_AG\": 44, \"DS_DL\": 0, \"DS_DG\": 0, \"DP_DG\": -2, \"DP_AL\": 20, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 1, \"DS_AL\": 0}, \"cds_start\": 4009, \"codons\": \"Gct/Cct\", \"eve_score\": 0.739062151407264, \"consequence_terms\": [\"missense_variant\"], \"location\": \"15:48474606\", \"cadd_raw\": 4.916884, \"cadd_phred\": 27.4, \"allele_num\": 1, \"cds_end\": 4009, \"protein_id\": \"NP_000129.3\", \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"A/P\", \"protein_start\": 1337, \"strand\": -1, \"polyphen_score\": 0.808, \"polyphen_prediction\": \"possibly_damaging\", \"mutationtaster_pred\": \"D\", \"mastermind_counts\": \"4|8|8\", \"sift_score\": 0, \"mastermind_mmid3\": \"FBN1:A1337P\", \"canonical\": 1, \"clinpred\": 0.997421979904175, \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"33/66\", \"cdna_start\": 4325, \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"given_ref\": \"C\", \"eve_class\": \"Pathogenic\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -1, \"impact\": \"MODERATE\"}], \"id\": \"NC_000015.10:g.48474606C>G\", \"variant_class\": \"SNV\", \"colocated_variants\": [{\"end\": 48474606, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48474606, \"phenotype_or_disease\": 1, \"id\": \"CM983867\"}, {\"clin_sig_allele\": \"G:uncertain_significance\", \"end\": 48474606, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"C/G/T\", \"phenotype_or_disease\": 1, \"start\": 48474606, \"clin_sig\": [\"uncertain_significance\"], \"id\": \"rs753648789\", \"var_synonyms\": {\"ClinVar\": [\"RCV000663681\", \"VCV000549199\"], \"UniProt\": [\"VAR_018004\"]}}], \"assembly_name\": \"GRCh38\", \"vcf_string\": \"15-48474606-C-G\", \"end\": 48474606, \"allele_string\": \"C/G\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.03893924999911297,\n",
       "      'pca_y': 0.07736632490459089},\n",
       "     {'t-sne_x': 20.466391498688324, 't-sne_y': -6.963026468455042},\n",
       "     {'umap_x': 8.4617919921875, 'umap_y': 7.3172712326049805}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.0016018133610486984,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.002305955160409212,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-G',\n",
       "      'distance': 0.002419940894469619,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0026486795395612717,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0026810462586581707,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495162-C-G',\n",
       "      'distance': 0.0027272808365523815,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437347-G-C',\n",
       "      'distance': 0.0027394513599574566,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.0028547835536301136,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.002932074246928096,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.0029620632994920015,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.0029643408488482237,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487333-G-C',\n",
       "      'distance': 0.0029785828664898872,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.0029822587966918945,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.003007611259818077,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.0030697821639478207,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492562-G-C',\n",
       "      'distance': 0.003123870585113764,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48428356-G-C',\n",
       "      'distance': 0.0031427480280399323,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.003170917509123683,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520708-C-G',\n",
       "      'distance': 0.003187159076333046,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.003222143743187189,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48474599-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48474599\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474599',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4016G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474599C>T',\n",
       "     'hgvs_protein': 'p.Cys1339Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48474599C>T\", \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"cds_end\": 4016, \"protein_start\": 1339, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/Y\", \"polyphen_prediction\": \"possibly_damaging\", \"polyphen_score\": 0.86, \"strand\": -1, \"clinpred\": 0.999927520751953, \"mastermind_mmid3\": \"FBN1:C1339Y\", \"canonical\": 1, \"mutationtaster_pred\": \"A\", \"mastermind_counts\": \"7|14|14\", \"sift_score\": 0, \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"33/66\", \"phaplo\": 0.99897615427687, \"cdna_start\": 4332, \"given_ref\": \"C\", \"eve_class\": \"Pathogenic\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -2, \"impact\": \"MODERATE\", \"variant_allele\": \"T\", \"hgvsp\": \"NP_000129.3:p.Cys1339Tyr\", \"used_ref\": \"C\", \"aa\": \"C\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.4016G>A\", \"biotype\": \"protein_coding\", \"alphamissense\": {\"am_pathogenicity\": 0.9984, \"am_class\": \"likely_pathogenic\"}, \"protein_end\": 1339, \"conservation\": 3.95, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"start\": 48474598, \"end\": 48474599}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}}, \"eve_score\": 0.80563974915104, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 4332, \"spliceai\": {\"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 5, \"DS_AG\": 0, \"DP_AL\": -12, \"DP_DG\": -26, \"DS_DG\": 0, \"DP_AG\": -31, \"DS_DL\": 0}, \"cds_start\": 4016, \"codons\": \"tGt/tAt\", \"location\": \"15:48474599\", \"cadd_raw\": 4.941219, \"cadd_phred\": 27.6, \"allele_num\": 1}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"id\": \"CM013927\", \"phenotype_or_disease\": 1, \"start\": 48474599, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48474599}, {\"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48474599, \"start\": 48474599, \"phenotype_or_disease\": 1, \"id\": \"CM145760\"}, {\"id\": \"rs397515798\", \"var_synonyms\": {\"UniProt\": [\"VAR_018005\"], \"ClinVar\": [\"RCV000035185\", \"VCV000042346\", \"RCV001052318\", \"RCV000663683\", \"VCV000549200\"]}, \"phenotype_or_disease\": 1, \"start\": 48474599, \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"allele_string\": \"C/G/T\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48474599, \"pubmed\": [24033266, 11700157, 10486319, 17701892], \"clin_sig_allele\": \"T:pathogenic;G:pathogenic/likely_pathogenic;T:likely_pathogenic\"}], \"variant_class\": \"SNV\", \"allele_string\": \"C/T\", \"vcf_string\": \"15-48474599-C-T\", \"end\": 48474599, \"minimised\": 1, \"start\": 48474599, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48474599C>T\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.03783308519029635,\n",
       "      'pca_y': 0.08641925808251907},\n",
       "     {'t-sne_x': 8.629036030514483, 't-sne_y': -4.173925050356323},\n",
       "     {'umap_x': 4.506664276123047, 'umap_y': 4.5513763427734375}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48474572-C-T',\n",
       "      'distance': 0.0005972990766167641,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-T',\n",
       "      'distance': 0.000769892823882401,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425479-C-T',\n",
       "      'distance': 0.0009337415685877204,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472595-C-T',\n",
       "      'distance': 0.001099935034289956,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437369-C-T',\n",
       "      'distance': 0.0011124515440315008,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427755-C-T',\n",
       "      'distance': 0.0011325193336233497,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425374-C-T',\n",
       "      'distance': 0.0011711525730788708,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428459-C-T',\n",
       "      'distance': 0.0011796745238825679,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470684-C-T',\n",
       "      'distance': 0.0014216512208804488,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474533-C-T',\n",
       "      'distance': 0.0014636048581451178,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474638-C-T',\n",
       "      'distance': 0.0014833678724244237,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432916-C-T',\n",
       "      'distance': 0.0016529207350686193,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497343-C-T',\n",
       "      'distance': 0.0016682320274412632,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48436966-C-T',\n",
       "      'distance': 0.0017635916592553258,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468474-C-T',\n",
       "      'distance': 0.00176821812056005,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497316-C-T',\n",
       "      'distance': 0.0017952998168766499,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485379-C-T',\n",
       "      'distance': 0.0018488243222236633,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497379-C-T',\n",
       "      'distance': 0.0018677329644560814,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487167-C-T',\n",
       "      'distance': 0.00186829490121454,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425816-C-T',\n",
       "      'distance': 0.0018702002707868814,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48474578-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48474578\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474578',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4037T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474578A>C',\n",
       "     'hgvs_protein': 'p.Phe1346Cys',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48474578, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48474578A>C\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"id\": \"NC_000015.10:g.48474578A>C\", \"transcript_consequences\": [{\"cdna_end\": 4353, \"cds_start\": 4037, \"spliceai\": {\"DP_DG\": -50, \"DS_DG\": 0, \"DP_AG\": 9, \"DS_DL\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 26, \"DP_AL\": 48, \"DS_AG\": 0}, \"codons\": \"tTc/tGc\", \"eve_score\": 0.732231177540549, \"consequence_terms\": [\"missense_variant\"], \"cadd_raw\": 5.186944, \"location\": \"15:48474578\", \"cadd_phred\": 29.1, \"allele_num\": 1, \"hgvsp\": \"NP_000129.3:p.Phe1346Cys\", \"aa\": \"A\", \"used_ref\": \"A\", \"hgvsc\": \"NM_000138.5:c.4037T>G\", \"gene_id\": \"2200\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"biotype\": \"protein_coding\", \"alphamissense\": {\"am_pathogenicity\": 0.9967, \"am_class\": \"likely_pathogenic\"}, \"protein_end\": 1346, \"conservation\": 3.95, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48474578, \"start\": 48474577, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}}, \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"exon\": \"33/66\", \"cdna_start\": 4353, \"ptriplo\": 0.984699554922396, \"given_ref\": \"A\", \"eve_class\": \"Pathogenic\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"C\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -2, \"impact\": \"MODERATE\", \"cds_end\": 4037, \"protein_id\": \"NP_000129.3\", \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"F/C\", \"protein_start\": 1346, \"strand\": -1, \"polyphen_score\": 0.892, \"polyphen_prediction\": \"possibly_damaging\", \"mutationtaster_pred\": \"D\", \"mastermind_counts\": \"4|4|4\", \"sift_score\": 0, \"canonical\": 1, \"clinpred\": 0.998922348022461, \"mastermind_mmid3\": \"FBN1:F1346C\"}], \"colocated_variants\": [{\"end\": 48474578, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"id\": \"CM118910\", \"start\": 48474578, \"phenotype_or_disease\": 1}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"allele_string\": \"A/C\", \"vcf_string\": \"15-48474578-A-C\", \"end\": 48474578, \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.04124961444582159,\n",
       "      'pca_y': 0.0883864284591353},\n",
       "     {'t-sne_x': 20.258197124129197, 't-sne_y': 3.504012540910354},\n",
       "     {'umap_x': 3.8087480068206787, 'umap_y': 6.14070987701416}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.0023855881299823523,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.002687613945454359,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.0029019773937761784,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445432-A-C',\n",
       "      'distance': 0.00298461620695889,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.0030188176315277815,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48436967-A-C',\n",
       "      'distance': 0.0030241007916629314,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520677-A-C',\n",
       "      'distance': 0.003072442952543497,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.0030894142109900713,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.003147952724248171,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488419-A-C',\n",
       "      'distance': 0.0032915377523750067,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.0033592437393963337,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488439-T-C',\n",
       "      'distance': 0.003363593015819788,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434707-T-C',\n",
       "      'distance': 0.003558744676411152,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474572-C-T',\n",
       "      'distance': 0.0037844961043447256,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483822-A-C',\n",
       "      'distance': 0.003810383379459381,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508693-A-C',\n",
       "      'distance': 0.0038229594938457012,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425783-C-A',\n",
       "      'distance': 0.003890970256179571,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497298-T-C',\n",
       "      'distance': 0.004070810042321682,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485490-T-C',\n",
       "      'distance': 0.004102783277630806,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.004110878799110651,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48474369-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48474369\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474369',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4096G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474369C>A',\n",
       "     'hgvs_protein': 'p.Glu1366Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"allele_string\": \"C/A\", \"vcf_string\": \"15-48474369-C-A\", \"end\": 48474369, \"minimised\": 1, \"id\": \"NC_000015.10:g.48474369C>A\", \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.4096G>T\", \"hgvsp\": \"NP_000129.3:p.Glu1366Ter\", \"used_ref\": \"C\", \"aa\": \"C\", \"conservation\": 3.95, \"protein_end\": 1366, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48474368, \"end\": 48474369}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\", \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}}, \"biotype\": \"protein_coding\", \"cadd_raw\": 9.651167, \"location\": \"15:48474369\", \"cadd_phred\": 42, \"consequence_terms\": [\"stop_gained\"], \"cdna_end\": 4412, \"spliceai\": {\"DP_DL\": 9, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AL\": 8, \"DS_AG\": 0.06, \"DS_DG\": 0, \"DP_DG\": -29, \"DP_AG\": -29, \"DS_DL\": 0}, \"codons\": \"Gaa/Taa\", \"cds_start\": 4096, \"allele_num\": 1, \"lof\": \"HC\", \"protein_start\": 1366, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"E/*\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 4096, \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:E1366X\", \"mutationtaster_pred\": \"A\", \"mastermind_counts\": \"4|4|4\", \"strand\": -1, \"given_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 4412, \"exon\": \"34/66\", \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"PERCENTILE:0.475394614670381,GERP_DIST:5357.29826996625,BP_DIST:4488,DIST_FROM_LAST_EXON:4099,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"impact\": \"HIGH\", \"variant_allele\": \"A\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\"}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"start\": 48474369, \"id\": \"CM040037\", \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"end\": 48474369}, {\"id\": \"CM043996\", \"phenotype_or_disease\": 1, \"start\": 48474369, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48474369}, {\"start\": 48474369, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM3782884\"]}, \"id\": \"COSV57328127\", \"end\": 48474369, \"somatic\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"COSMIC_MUTATION\"}], \"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"start\": 48474369, \"input\": \"NC_000015.10:g.48474369C>A\", \"most_severe_consequence\": \"stop_gained\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.022907110059340624,\n",
       "      'pca_y': 0.09843231131883645},\n",
       "     {'t-sne_x': 20.98633766488151, 't-sne_y': 41.55191562222068},\n",
       "     {'umap_x': -9.437003135681152, 'umap_y': 4.626110553741455}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48483878-C-A',\n",
       "      'distance': 0.00180406856816262,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.0031145107932388783,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437837-C-A',\n",
       "      'distance': 0.003436561906710267,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.0038052734453231096,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.004093368537724018,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.004336881451308727,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488233-C-A',\n",
       "      'distance': 0.004378754645586014,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48436970-C-A',\n",
       "      'distance': 0.004482019692659378,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.004508924670517445,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411085-C-A',\n",
       "      'distance': 0.004707524552941322,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.0048387739807367325,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.0048659564927220345,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.005017091054469347,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.005338981281965971,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.005346257239580154,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.0055307019501924515,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515397-C-T',\n",
       "      'distance': 0.0058693112805485725,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.005955852568149567,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452676-C-A',\n",
       "      'distance': 0.005968545097857714,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48516225-G-A',\n",
       "      'distance': 0.006030597724020481,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48474344-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48474344\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474344',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4121G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474344C>G',\n",
       "     'hgvs_protein': 'p.Cys1374Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48474344C>G\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"start\": 48474344, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"id\": \"CM040038\", \"start\": 48474344, \"phenotype_or_disease\": 1, \"end\": 48474344, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1}, {\"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48474344, \"id\": \"CM098687\", \"start\": 48474344, \"phenotype_or_disease\": 1}], \"assembly_name\": \"GRCh38\", \"id\": \"NC_000015.10:g.48474344C>G\", \"transcript_consequences\": [{\"mane_select\": \"ENST00000316623.10\", \"blosum62\": -1, \"impact\": \"MODERATE\", \"variant_allele\": \"G\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"eve_class\": \"Pathogenic\", \"given_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 4437, \"phaplo\": 0.99897615427687, \"exon\": \"34/66\", \"mastermind_mmid3\": \"FBN1:C1374S\", \"clinpred\": 0.999879479408264, \"canonical\": 1, \"mutationtaster_pred\": \"D\", \"sift_score\": 0, \"mastermind_counts\": \"5|8|8\", \"polyphen_score\": 0.294, \"polyphen_prediction\": \"benign\", \"strand\": -1, \"protein_start\": 1374, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/S\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 4121, \"allele_num\": 1, \"location\": \"15:48474344\", \"cadd_raw\": 4.706952, \"cadd_phred\": 26.4, \"eve_score\": 0.779217940902521, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 4437, \"codons\": \"tGc/tCc\", \"spliceai\": {\"DP_AL\": -4, \"DS_AG\": 0, \"DP_DL\": 34, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AG\": -8, \"DS_DL\": 0, \"DS_DG\": 0, \"DP_DG\": 33}, \"cds_start\": 4121, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9981}, \"protein_end\": 1374, \"conservation\": 3.95, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"end\": 48474344, \"start\": 48474343, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"biotype\": \"protein_coding\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"hgvsc\": \"NM_000138.5:c.4121G>C\", \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys1374Ser\", \"used_ref\": \"C\", \"aa\": \"C\"}], \"minimised\": 1, \"allele_string\": \"C/G\", \"vcf_string\": \"15-48474344-C-G\", \"end\": 48474344}',\n",
       "    'coordinates': [{'pca_x': 0.04116654333264799,\n",
       "      'pca_y': 0.0939093796476615},\n",
       "     {'t-sne_x': 17.239355996249856, 't-sne_y': -5.498887770019767},\n",
       "     {'umap_x': 8.371875762939453, 'umap_y': 6.899507999420166}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48432910-C-G',\n",
       "      'distance': 0.0018331832252442837,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.0024813273921608925,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.0025947708636522293,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.002759551163762808,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474344-C-T',\n",
       "      'distance': 0.0032578783575445414,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437064-G-C',\n",
       "      'distance': 0.003477497026324272,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0037053734995424747,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489991-C-G',\n",
       "      'distance': 0.0038297369610518217,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.0038305423222482204,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.004100371152162552,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.004101159982383251,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.0042282892391085625,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474533-C-T',\n",
       "      'distance': 0.004231323953717947,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.004287762567400932,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483931-C-T',\n",
       "      'distance': 0.004532254301011562,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481733-A-G',\n",
       "      'distance': 0.004548961762338877,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415716-T-C',\n",
       "      'distance': 0.004571608267724514,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.004633190575987101,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.004652620758861303,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.0046529159881174564,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48474300-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48474300\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474300',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4165T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474300A>G',\n",
       "     'hgvs_protein': 'p.Cys1389Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"id\": \"CM040039\", \"start\": 48474300, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48474300}, {\"id\": \"rs193922203\", \"var_synonyms\": {\"UniProt\": [\"VAR_018008\"], \"ClinVar\": [\"RCV000029735\", \"VCV000036073\"]}, \"start\": 48474300, \"clin_sig\": [\"likely_pathogenic\"], \"phenotype_or_disease\": 1, \"allele_string\": \"A/C/G\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48474300, \"clin_sig_allele\": \"C:likely_pathogenic\"}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"G\", \"impact\": \"MODERATE\", \"blosum62\": -3, \"mane_select\": \"ENST00000316623.10\", \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 4481, \"exon\": \"34/66\", \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"ptriplo\": 0.984699554922396, \"given_ref\": \"A\", \"eve_class\": \"Pathogenic\", \"strand\": -1, \"polyphen_score\": 0.86, \"polyphen_prediction\": \"possibly_damaging\", \"mastermind_counts\": \"4|7|7\", \"sift_score\": 0, \"mutationtaster_pred\": \"D\", \"mastermind_mmid3\": \"FBN1:C1389R\", \"canonical\": 1, \"clinpred\": 0.999865174293518, \"cds_end\": 4165, \"protein_id\": \"NP_000129.3\", \"amino_acids\": \"C/R\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 1389, \"allele_num\": 1, \"codons\": \"Tgt/Cgt\", \"spliceai\": {\"DP_DG\": -45, \"DS_DG\": 0, \"DP_AG\": 40, \"DS_DL\": 0.01, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -24, \"DS_AG\": 0, \"DP_AL\": -12}, \"cds_start\": 4165, \"cdna_end\": 4481, \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.756592944687923, \"cadd_phred\": 31, \"cadd_raw\": 5.425618, \"location\": \"15:48474300\", \"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48474300, \"start\": 48474299, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"alphamissense\": {\"am_pathogenicity\": 0.9991, \"am_class\": \"likely_pathogenic\"}, \"conservation\": 3.95, \"protein_end\": 1389, \"used_ref\": \"A\", \"aa\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys1389Arg\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.4165T>C\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"]}], \"id\": \"NC_000015.10:g.48474300A>G\", \"minimised\": 1, \"end\": 48474300, \"vcf_string\": \"15-48474300-A-G\", \"allele_string\": \"A/G\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48474300A>G\", \"seq_region_name\": \"15\", \"start\": 48474300}',\n",
       "    'coordinates': [{'pca_x': 0.046058631185191184,\n",
       "      'pca_y': 0.08531006400477023},\n",
       "     {'t-sne_x': 29.12739320223507, 't-sne_y': 6.137261592605872},\n",
       "     {'umap_x': 2.5875844955444336, 'umap_y': 7.129839897155762}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48434600-A-G',\n",
       "      'distance': 0.0009482249733991921,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481733-A-G',\n",
       "      'distance': 0.0010992983588948846,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432911-A-G',\n",
       "      'distance': 0.0011041284305974841,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472665-A-G',\n",
       "      'distance': 0.001188539550639689,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432944-A-G',\n",
       "      'distance': 0.001462640124373138,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.0014943552669137716,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.0016423495253548026,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.00167175754904747,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.0016780220903456211,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.001681696274317801,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425414-A-G',\n",
       "      'distance': 0.0016976543702185154,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422024-A-G',\n",
       "      'distance': 0.001768465619534254,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596325-A-G',\n",
       "      'distance': 0.001831890782341361,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.001845866208896041,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434702-A-G',\n",
       "      'distance': 0.0018691527657210827,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474567-A-G',\n",
       "      'distance': 0.001972070662304759,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452655-A-G',\n",
       "      'distance': 0.0022075786255300045,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48503850-A-G',\n",
       "      'distance': 0.002221708884462714,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.0022261766716837883,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468001-A-G',\n",
       "      'distance': 0.0022761027794331312,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48474286-TT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48474286\\t.\\tTT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474286',\n",
       "     'ref_allele': 'TT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4179del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474287del',\n",
       "     'hgvs_protein': 'p.Gly1394AspfsTer19',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"start\": 48474287, \"input\": \"NC_000015.10:g.48474287del\", \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"id\": \"NC_000015.10:g.48474287del\", \"transcript_consequences\": [{\"allele_num\": 1, \"cdna_end\": 4495, \"spliceai\": {\"DS_AG\": 0, \"DP_AL\": -2, \"DS_AL\": 0, \"DP_DL\": -25, \"SYMBOL\": \"FBN1\", \"DS_DL\": 0, \"DP_AG\": 50, \"DP_DG\": -10, \"DS_DG\": 0.12}, \"cds_start\": 4179, \"codons\": \"gaA/ga\", \"consequence_terms\": [\"frameshift_variant\"], \"location\": \"15:48474286\", \"hgvs_offset\": -1, \"biotype\": \"protein_coding\", \"protein_end\": 1393, \"conservation\": 3.95, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"end\": 48474287, \"start\": 48474286, \"type\": \"SimpleInterval\"}}, \"type\": \"Allele\", \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}}, \"hgvsp\": \"NP_000129.3:p.Gly1394AspfsTer19\", \"used_ref\": \"T\", \"aa\": \"T\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.4179del\", \"mane\": [\"MANE_Select\"], \"gene_symbol\": \"FBN1\", \"variant_allele\": \"-\", \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"PERCENTILE:0.485027855153203,GERP_DIST:5257.55255156756,BP_DIST:4406,DIST_FROM_LAST_EXON:4017,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"impact\": \"HIGH\", \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 4495, \"exon\": \"34/66\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"T\", \"strand\": -1, \"canonical\": 1, \"cds_end\": 4179, \"protein_id\": \"NP_000129.3\", \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"E/X\", \"lof\": \"HC\", \"protein_start\": 1393}], \"variant_class\": \"deletion\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"T/-\", \"end\": 48474287, \"vcf_string\": \"15-48474286-TT-T\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.0010604364533877003,\n",
       "      'pca_y': 0.01352861892474401},\n",
       "     {'t-sne_x': -13.010034496691024, 't-sne_y': -2.7501945575896367},\n",
       "     {'umap_x': 0.5482549071311951, 'umap_y': 14.948619842529297}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.09702170640230179,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.09769092500209808,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.09967266023159027,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.09971774369478226,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.10041011869907379,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.10055197030305862,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.1006263867020607,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.10073274374008179,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.1010361760854721,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.10112383216619492,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.10113924741744995,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.1016184538602829,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.10167834162712097,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.10176924616098404,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.10185070335865021,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.10220621526241302,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.10230150818824768,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.1027793139219284,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.10294129699468613,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.10301351547241211,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48474259-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48474259\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474259',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4206T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474259A>C',\n",
       "     'hgvs_protein': 'p.Cys1402Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48474259A>C\", \"seq_region_name\": \"15\", \"start\": 48474259, \"minimised\": 1, \"vcf_string\": \"15-48474259-A-C\", \"end\": 48474259, \"allele_string\": \"A/C\", \"colocated_variants\": [{\"start\": 48474259, \"phenotype_or_disease\": 1, \"id\": \"CM992327\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48474259}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"codons\": \"tgT/tgG\", \"cds_start\": 4206, \"spliceai\": {\"DP_AG\": -4, \"DS_DL\": 0.03, \"DP_DG\": -8, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": -5, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 17}, \"cdna_end\": 4522, \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.780344855322332, \"cadd_phred\": 22.4, \"location\": \"15:48474259\", \"cadd_raw\": 3.407398, \"allele_num\": 1, \"aa\": \"A\", \"used_ref\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys1402Trp\", \"hgvsc\": \"NM_000138.5:c.4206T>G\", \"gene_id\": \"2200\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"start\": 48474258, \"type\": \"SimpleInterval\", \"end\": 48474259}}}, \"protein_end\": 1402, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9985}, \"conservation\": -2.82, \"phaplo\": 0.99897615427687, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"34/66\", \"cdna_start\": 4522, \"loeuf\": 0.105, \"ptriplo\": 0.984699554922396, \"eve_class\": \"Pathogenic\", \"given_ref\": \"A\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"C\", \"impact\": \"MODERATE\", \"blosum62\": -2, \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 4206, \"protein_id\": \"NP_000129.3\", \"amino_acids\": \"C/W\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 1402, \"strand\": -1, \"polyphen_score\": 0.845, \"polyphen_prediction\": \"possibly_damaging\", \"sift_score\": 0, \"mastermind_counts\": \"5|5|5\", \"mutationtaster_pred\": \"D\", \"canonical\": 1, \"clinpred\": 0.999784529209137, \"mastermind_mmid3\": \"FBN1:C1402W\"}], \"id\": \"NC_000015.10:g.48474259A>C\"}',\n",
       "    'coordinates': [{'pca_x': 0.04147543957410732,\n",
       "      'pca_y': 0.0908217839873989},\n",
       "     {'t-sne_x': 22.968542110689764, 't-sne_y': 5.364524333997793},\n",
       "     {'umap_x': 3.603998899459839, 'umap_y': 6.395431041717529}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48483822-A-C',\n",
       "      'distance': 0.0017916669603437185,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.0032508186995983124,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445432-A-C',\n",
       "      'distance': 0.0035222647711634636,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415545-A-C',\n",
       "      'distance': 0.0035760654136538506,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.003676268272101879,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.003695415100082755,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415716-T-C',\n",
       "      'distance': 0.003723591100424528,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468522-C-A',\n",
       "      'distance': 0.0037372393999248743,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474344-C-T',\n",
       "      'distance': 0.0038593849167227745,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.0038882249500602484,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483931-C-T',\n",
       "      'distance': 0.003976359497755766,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472665-A-G',\n",
       "      'distance': 0.00403627147898078,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432944-A-G',\n",
       "      'distance': 0.004075625911355019,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481733-A-G',\n",
       "      'distance': 0.004080458544194698,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415632-C-T',\n",
       "      'distance': 0.004081243183463812,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472628-C-A',\n",
       "      'distance': 0.004086795262992382,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.004203048534691334,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.004219106398522854,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432955-C-T',\n",
       "      'distance': 0.004233190789818764,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.004240994341671467,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48474253-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48474253\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48474253',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4210+2T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48474253A>G',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"id\": \"NC_000015.10:g.48474253A>G\", \"transcript_consequences\": [{\"cadd_raw\": 6.099842, \"location\": \"15:48474253\", \"cadd_phred\": 34, \"consequence_terms\": [\"splice_donor_variant\"], \"spliceai\": {\"DP_DG\": 23, \"DS_DG\": 0.8, \"DS_DL\": 0.93, \"DP_AG\": 2, \"DS_AL\": 0, \"DP_DL\": 2, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DP_AL\": 31}, \"allele_num\": 1, \"maxentscan_diff\": 7.754, \"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.4210+2T>C\", \"intron\": \"34/65\", \"aa\": \"A\", \"used_ref\": \"A\", \"conservation\": 3.95, \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48474252, \"end\": 48474253}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"biotype\": \"protein_coding\", \"genesplicer\": \"loss/donor/48474252-48474253/Medium/6.144281\", \"given_ref\": \"A\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"gene_symbol_source\": \"EntrezGene\", \"lof_info\": \"INTRON_SIZE:1578\", \"mane_select\": \"ENST00000316623.10\", \"impact\": \"HIGH\", \"variant_allele\": \"G\", \"gene_symbol\": \"FBN1\", \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"maxentscan_ref\": 9.802, \"protein_id\": \"NP_000129.3\", \"mastermind_mmid3\": \"FBN1:D1404sd\", \"canonical\": 1, \"mastermind_counts\": \"3|3|10\", \"rf_score\": 0.64, \"maxentscan_alt\": 2.048, \"ada_score\": 0.988163425542712, \"strand\": -1}], \"minimised\": 1, \"allele_string\": \"A/G\", \"end\": 48474253, \"vcf_string\": \"15-48474253-A-G\", \"input\": \"NC_000015.10:g.48474253A>G\", \"strand\": 1, \"most_severe_consequence\": \"splice_donor_variant\", \"seq_region_name\": \"15\", \"start\": 48474253}',\n",
       "    'coordinates': [{'pca_x': -0.23954168656335884,\n",
       "      'pca_y': -0.03207417713482608},\n",
       "     {'t-sne_x': -14.882258704667267, 't-sne_y': -27.99243661194193},\n",
       "     {'umap_x': 8.200759887695312, 'umap_y': 13.187573432922363}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48465788-A-G',\n",
       "      'distance': 0.002193132881075144,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510042-A-G',\n",
       "      'distance': 0.0025187938008457422,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437320-A-G',\n",
       "      'distance': 0.0026072748005390167,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515386-C-T',\n",
       "      'distance': 0.0026775337755680084,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445375-C-A',\n",
       "      'distance': 0.003748734015971422,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474254-C-T',\n",
       "      'distance': 0.005086115561425686,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437320-A-C',\n",
       "      'distance': 0.0050902473740279675,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.005200237035751343,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465789-C-A',\n",
       "      'distance': 0.005305997096002102,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441720-C-T',\n",
       "      'distance': 0.0053820861503481865,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613009-C-A',\n",
       "      'distance': 0.005392921157181263,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.005462313536554575,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-G',\n",
       "      'distance': 0.005483994260430336,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444540-C-T',\n",
       "      'distance': 0.005535039119422436,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508581-C-A',\n",
       "      'distance': 0.005566134117543697,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.005825329106301069,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.00594183336943388,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.0060091568157076836,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.006127060391008854,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503785-A-C',\n",
       "      'distance': 0.006197336129844189,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48472664-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48472664\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48472664',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4223G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48472664C>A',\n",
       "     'hgvs_protein': 'p.Cys1408Phe',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48472664C>A\", \"start\": 48472664, \"seq_region_name\": \"15\", \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"end\": 48472664, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"start\": 48472664, \"id\": \"CM010043\"}], \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"cadd_phred\": 29.3, \"location\": \"15:48472664\", \"cadd_raw\": 5.218521, \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.750194535425561, \"spliceai\": {\"DS_AG\": 0, \"DP_AL\": 12, \"DP_DL\": 0, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_DL\": 0, \"DP_AG\": -35, \"DS_DG\": 0, \"DP_DG\": 2}, \"cds_start\": 4223, \"codons\": \"tGc/tTc\", \"cdna_end\": 4539, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.4223G>T\", \"aa\": \"C\", \"used_ref\": \"C\", \"hgvsp\": \"NP_000129.3:p.Cys1408Phe\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"end\": 48472664, \"start\": 48472663, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}}, \"alphamissense\": {\"am_pathogenicity\": 0.9957, \"am_class\": \"likely_pathogenic\"}, \"protein_end\": 1408, \"conservation\": 3.95, \"biotype\": \"protein_coding\", \"given_ref\": \"C\", \"eve_class\": \"Pathogenic\", \"ptriplo\": 0.984699554922396, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 4539, \"exon\": \"35/66\", \"loeuf\": 0.105, \"impact\": \"MODERATE\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -2, \"variant_allele\": \"A\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"protein_start\": 1408, \"amino_acids\": \"C/F\", \"transcript_id\": \"NM_000138.5\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 4223, \"clinpred\": 0.999939322471619, \"mastermind_mmid3\": \"FBN1:C1408F\", \"canonical\": 1, \"sift_score\": 0, \"mastermind_counts\": \"6|8|8\", \"mutationtaster_pred\": \"D\", \"polyphen_prediction\": \"probably_damaging\", \"polyphen_score\": 0.994, \"strand\": -1}], \"id\": \"NC_000015.10:g.48472664C>A\", \"minimised\": 1, \"vcf_string\": \"15-48472664-C-A\", \"end\": 48472664, \"allele_string\": \"C/A\"}',\n",
       "    'coordinates': [{'pca_x': 0.029251630019844765,\n",
       "      'pca_y': 0.1184359619485433},\n",
       "     {'t-sne_x': -16.017262224168825, 't-sne_y': 34.82646861280899},\n",
       "     {'umap_x': 1.6430268287658691, 'umap_y': 21.457319259643555}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48495519-C-A',\n",
       "      'distance': 0.0015444114105775952,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515434-C-T',\n",
       "      'distance': 0.00211097439751029,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474566-C-A',\n",
       "      'distance': 0.0021405857987701893,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456642-C-A',\n",
       "      'distance': 0.0021661228965967894,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463949-C-A',\n",
       "      'distance': 0.002243509516119957,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425413-C-A',\n",
       "      'distance': 0.002247184980660677,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415585-C-A',\n",
       "      'distance': 0.0022519915364682674,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.00294672092422843,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472601-C-T',\n",
       "      'distance': 0.0035776409786194563,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.0035815455485135317,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.003610718995332718,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505117-C-A',\n",
       "      'distance': 0.003661814145743847,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472628-C-T',\n",
       "      'distance': 0.003671234007924795,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495243-A-T',\n",
       "      'distance': 0.003710589837282896,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472556-C-T',\n",
       "      'distance': 0.0037571589928120375,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465619-A-T',\n",
       "      'distance': 0.0038334568962454796,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495156-C-T',\n",
       "      'distance': 0.003875748487189412,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.004064428620040417,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.004079722799360752,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425458-C-T',\n",
       "      'distance': 0.004112761467695236,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48472602-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48472602\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48472602',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4285T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48472602A>T',\n",
       "     'hgvs_protein': 'p.Cys1429Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"vcf_string\": \"15-48472602-A-T\", \"end\": 48472602, \"allele_string\": \"A/T\", \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"start\": 48472602, \"id\": \"CM983869\", \"end\": 48472602, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\"}, {\"phenotype_or_disease\": 1, \"start\": 48472602, \"clin_sig\": [\"pathogenic\"], \"var_synonyms\": {\"UniProt\": [\"VAR_018011\"], \"ClinVar\": [\"RCV000663706\", \"VCV000549215\"]}, \"id\": \"rs1555397546\", \"clin_sig_allele\": \"T:pathogenic\", \"end\": 48472602, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"A/T\"}], \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"cds_end\": 4285, \"protein_id\": \"NP_000129.3\", \"amino_acids\": \"C/S\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 1429, \"strand\": -1, \"polyphen_prediction\": \"probably_damaging\", \"polyphen_score\": 0.978, \"sift_score\": 0, \"mastermind_counts\": \"5|12|13\", \"mutationtaster_pred\": \"A\", \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:C1429S\", \"clinpred\": 0.99992835521698, \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 4601, \"phaplo\": 0.99897615427687, \"exon\": \"35/66\", \"loeuf\": 0.105, \"ptriplo\": 0.984699554922396, \"eve_class\": \"Pathogenic\", \"given_ref\": \"A\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"T\", \"impact\": \"MODERATE\", \"blosum62\": -1, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"A\", \"aa\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys1429Ser\", \"hgvsc\": \"NM_000138.5:c.4285T>A\", \"gene_id\": \"2200\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"start\": 48472601, \"type\": \"SimpleInterval\", \"end\": 48472602}}}, \"conservation\": 3.95, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9984}, \"protein_end\": 1429, \"spliceai\": {\"DP_DG\": -45, \"DS_DG\": 0, \"DP_AG\": -2, \"DS_DL\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -47, \"DS_AG\": 0, \"DP_AL\": 27}, \"codons\": \"Tgt/Agt\", \"cds_start\": 4285, \"cdna_end\": 4601, \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.777598946553316, \"cadd_phred\": 29.6, \"location\": \"15:48472602\", \"cadd_raw\": 5.268, \"allele_num\": 1}], \"id\": \"NC_000015.10:g.48472602A>T\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48472602A>T\", \"start\": 48472602, \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.04346549859817056,\n",
       "      'pca_y': 0.0804931497091386},\n",
       "     {'t-sne_x': 15.937424280909266, 't-sne_y': 8.744636532324021},\n",
       "     {'umap_x': 3.3781492710113525, 'umap_y': 4.85257625579834}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.000928020104765892,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428439-A-T',\n",
       "      'distance': 0.001680153189226985,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596325-A-T',\n",
       "      'distance': 0.0023170914500951767,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427684-A-T',\n",
       "      'distance': 0.003403827315196395,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.003615566762164235,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-T',\n",
       "      'distance': 0.0036434088833630085,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465574-A-T',\n",
       "      'distance': 0.003740739542990923,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415723-A-G',\n",
       "      'distance': 0.004042032640427351,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468001-A-G',\n",
       "      'distance': 0.004064749926328659,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468527-A-T',\n",
       "      'distance': 0.004096153192222118,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434701-C-T',\n",
       "      'distance': 0.004107974469661713,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434702-A-G',\n",
       "      'distance': 0.004161329939961433,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425783-C-A',\n",
       "      'distance': 0.004223879426717758,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472628-C-A',\n",
       "      'distance': 0.004240530543029308,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503849-C-A',\n",
       "      'distance': 0.004244016483426094,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430791-A-T',\n",
       "      'distance': 0.004275288432836533,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.0042771389707922935,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432944-A-G',\n",
       "      'distance': 0.004320248495787382,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437032-C-T',\n",
       "      'distance': 0.004334823228418827,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503850-A-G',\n",
       "      'distance': 0.004344099201261997,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48472550-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48472550\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48472550',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4336+1G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48472550C>T',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"start\": 48472550, \"phenotype_or_disease\": 1, \"id\": \"CS054893\", \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48472550}, {\"phenotype_or_disease\": 1, \"start\": 48472550, \"id\": \"CS098577\", \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"end\": 48472550}, {\"var_synonyms\": {\"ClinVar\": [\"RCV001972363\", \"VCV001456344\", \"RCV001956135\", \"VCV001457971\"]}, \"id\": \"rs2141277821\", \"phenotype_or_disease\": 1, \"start\": 48472550, \"clin_sig\": [\"pathogenic\"], \"allele_string\": \"C/A/T\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48472550, \"pubmed\": [17657824], \"clin_sig_allele\": \"T:pathogenic;A:pathogenic\"}], \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"mastermind_counts\": \"4|4|6\", \"mastermind_mmid3\": \"FBN1:D1446sd\", \"canonical\": 1, \"strand\": -1, \"ada_score\": 0.999990361384395, \"maxentscan_alt\": 0.827, \"rf_score\": 0.938, \"transcript_id\": \"NM_000138.5\", \"lof\": \"HC\", \"protein_id\": \"NP_000129.3\", \"maxentscan_ref\": 9.009, \"variant_allele\": \"T\", \"impact\": \"HIGH\", \"lof_info\": \"INTRON_SIZE:1794\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"C\", \"phaplo\": 0.99897615427687, \"gene_symbol_source\": \"EntrezGene\", \"loeuf\": 0.105, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48472550, \"start\": 48472549, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}}, \"conservation\": 3.95, \"genesplicer\": \"loss/donor/48472550-48472551/Medium/10.043071\", \"biotype\": \"protein_coding\", \"hgvsc\": \"NM_000138.5:c.4336+1G>A\", \"gene_id\": \"2200\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"C\", \"aa\": \"C\", \"intron\": \"35/65\", \"allele_num\": 1, \"maxentscan_diff\": 8.182, \"cadd_phred\": 35, \"cadd_raw\": 7.096972, \"location\": \"15:48472550\", \"spliceai\": {\"DS_DG\": 0.61, \"DP_DG\": -49, \"DP_AG\": 7, \"DS_DL\": 0.95, \"DP_DL\": 1, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AL\": 47, \"DS_AG\": 0}, \"consequence_terms\": [\"splice_donor_variant\"]}], \"id\": \"NC_000015.10:g.48472550C>T\", \"minimised\": 1, \"end\": 48472550, \"vcf_string\": \"15-48472550-C-T\", \"allele_string\": \"C/T\", \"most_severe_consequence\": \"splice_donor_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48472550C>T\", \"seq_region_name\": \"15\", \"start\": 48472550}',\n",
       "    'coordinates': [{'pca_x': -0.24662168976093604,\n",
       "      'pca_y': -0.02605248006953884},\n",
       "     {'t-sne_x': -18.192120822758575, 't-sne_y': -25.598245369784998},\n",
       "     {'umap_x': 7.97327995300293, 'umap_y': 13.29992389678955}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48421557-C-T',\n",
       "      'distance': 0.001574343303218484,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474254-C-T',\n",
       "      'distance': 0.0021248748525977135,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441720-C-T',\n",
       "      'distance': 0.0029529749881476164,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48444540-C-T',\n",
       "      'distance': 0.0029880935326218605,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.0031910655088722706,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432865-C-T',\n",
       "      'distance': 0.003342783311381936,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.0034075086005032063,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.003569013671949506,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.0036559777799993753,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503786-C-T',\n",
       "      'distance': 0.0036792545579373837,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463898-C-T',\n",
       "      'distance': 0.003689838107675314,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456636-C-T',\n",
       "      'distance': 0.003757116850465536,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425738-C-T',\n",
       "      'distance': 0.0038404010701924562,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515392-C-T',\n",
       "      'distance': 0.003891162108629942,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427566-C-T',\n",
       "      'distance': 0.003956733737140894,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.00406277272850275,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.0040719881653785706,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.0041105905547738075,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.00424172542989254,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.004288755357265472,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48470727-AGA-A',\n",
       "    'metadata': {'vcf_string': '15\\t48470727\\t.\\tAGA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48470727',\n",
       "     'ref_allele': 'AGA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4365_4366del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48470728_48470729del',\n",
       "     'hgvs_protein': 'p.Ile1455MetfsTer16',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48470728_48470729del\", \"strand\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"start\": 48470728, \"seq_region_name\": \"15\", \"assembly_name\": \"GRCh38\", \"variant_class\": \"deletion\", \"id\": \"NC_000015.10:g.48470728_48470729del\", \"transcript_consequences\": [{\"amino_acids\": \"IC/MX\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 1455, \"lof\": \"HC\", \"cds_end\": 4366, \"protein_id\": \"NP_000129.3\", \"canonical\": 1, \"strand\": -1, \"ptriplo\": 0.984699554922396, \"given_ref\": \"GA\", \"cdna_start\": 4681, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"36/66\", \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"variant_allele\": \"-\", \"impact\": \"HIGH\", \"lof_info\": \"PERCENTILE:0.506731662024141,GERP_DIST:5068.5425852567,BP_DIST:4221,DIST_FROM_LAST_EXON:3832,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.4365_4366del\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"AG\", \"aa\": \"GA\", \"hgvsp\": \"NP_000129.3:p.Ile1455MetfsTer16\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48470729, \"type\": \"SimpleInterval\", \"start\": 48470727}, \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"type\": \"Allele\"}, \"conservation\": 3.52, \"protein_end\": 1456, \"biotype\": \"protein_coding\", \"hgvs_offset\": -1, \"location\": \"15:48470727-48470728\", \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": -42, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DP_AL\": 29, \"DP_DG\": -48, \"DS_DG\": 0, \"DP_AG\": -48, \"DS_DL\": 0}, \"cds_start\": 4365, \"codons\": \"atCTgt/atgt\", \"cdna_end\": 4682, \"consequence_terms\": [\"frameshift_variant\"], \"allele_num\": 1}], \"minimised\": 1, \"allele_string\": \"GA/-\", \"end\": 48470729, \"vcf_string\": \"15-48470727-AGA-A\"}',\n",
       "    'coordinates': [{'pca_x': 0.04208893926084034,\n",
       "      'pca_y': 0.034063038597522126},\n",
       "     {'t-sne_x': 32.692845239868426, 't-sne_y': 6.7238353338396895},\n",
       "     {'umap_x': 1.8390624523162842, 'umap_y': 7.27304744720459}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.09789957106113434,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427591-G-A',\n",
       "      'distance': 0.09900357574224472,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.09908854961395264,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.10009750723838806,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.10018739104270935,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.10020066797733307,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.10031625628471375,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.10035321861505508,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.10073710978031158,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437912-G-A',\n",
       "      'distance': 0.10104238986968994,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48516225-G-A',\n",
       "      'distance': 0.10161426663398743,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537704-G-A',\n",
       "      'distance': 0.10165611654520035,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.10169699788093567,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420714-G-A',\n",
       "      'distance': 0.10185420513153076,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.10212165862321854,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10222926735877991,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.10226621478796005,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.10246782004833221,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468427-G-A',\n",
       "      'distance': 0.10248424857854843,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520764-G-A',\n",
       "      'distance': 0.1025102287530899,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48468509-GG-G',\n",
       "    'metadata': {'vcf_string': '15\\t48468509\\t.\\tGG\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48468509',\n",
       "     'ref_allele': 'GG',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4485del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48468510del',\n",
       "     'hgvs_protein': 'p.Thr1496ArgfsTer24',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"start\": 48468510, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48468510del\", \"strand\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"allele_string\": \"G/-\", \"end\": 48468510, \"vcf_string\": \"15-48468509-GG-G\", \"minimised\": 1, \"id\": \"NC_000015.10:g.48468510del\", \"transcript_consequences\": [{\"gene_symbol_source\": \"EntrezGene\", \"exon\": \"37/66\", \"cdna_start\": 4801, \"phaplo\": 0.99897615427687, \"loeuf\": 0.105, \"ptriplo\": 0.984699554922396, \"given_ref\": \"G\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"-\", \"impact\": \"HIGH\", \"lof_info\": \"PERCENTILE:0.520543175487465,GERP_DIST:4954.79349325001,BP_DIST:4103,DIST_FROM_LAST_EXON:3714,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mane_select\": \"ENST00000316623.10\", \"cds_end\": 4485, \"protein_id\": \"NP_000129.3\", \"amino_acids\": \"T/X\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 1495, \"lof\": \"HC\", \"strand\": -1, \"canonical\": 1, \"spliceai\": {\"DS_DG\": 0, \"DP_DG\": -2, \"DP_AG\": -12, \"DS_DL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 1, \"DS_AL\": 0, \"DP_AL\": 25, \"DS_AG\": 0}, \"codons\": \"acC/ac\", \"cds_start\": 4485, \"cdna_end\": 4801, \"consequence_terms\": [\"frameshift_variant\"], \"hgvs_offset\": -1, \"mutfunc\": {\"mod\": {\"dG_wt\": 80.51846, \"dG_mt_sd\": 0.002349, \"dG_wt_sd\": 0.000492, \"ddG_sd\": 0.002131, \"dG_mt\": 81.43132, \"ddG\": 0.91286}, \"exp\": {\"dG_wt\": 22.61096, \"dG_mt_sd\": 0.00326, \"ddG_sd\": 0.003892, \"dG_wt_sd\": 0.00177, \"ddG\": 0.7385, \"dG_mt\": 23.34946}}, \"location\": \"15:48468509\", \"allele_num\": 1, \"aa\": \"G\", \"used_ref\": \"G\", \"hgvsp\": \"NP_000129.3:p.Thr1496ArgfsTer24\", \"hgvsc\": \"NM_000138.5:c.4485del\", \"gene_id\": \"2200\", \"mane\": [\"MANE_Select\"], \"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"end\": 48468510, \"start\": 48468509, \"type\": \"SimpleInterval\"}}}, \"protein_end\": 1495, \"conservation\": 3.95}], \"variant_class\": \"deletion\", \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': -0.015312056198337934,\n",
       "      'pca_y': 0.023968473674066675},\n",
       "     {'t-sne_x': -13.050442337943519, 't-sne_y': -2.913596177729926},\n",
       "     {'umap_x': 0.20856180787086487, 'umap_y': 15.000701904296875}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.0915486067533493,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.09171565622091293,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.0941944569349289,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.09471626579761505,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.09516823291778564,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.09518103301525116,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.09567408263683319,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.09582871198654175,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.09596709907054901,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.09611707925796509,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09664244949817657,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.09694193303585052,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.09702363610267639,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.09711870551109314,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.0971929132938385,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.09733256697654724,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.09828783571720123,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.09830456972122192,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.09837841987609863,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468097-G-A',\n",
       "      'distance': 0.09891514480113983,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48468504-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48468504\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48468504',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4490G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48468504C>G',\n",
       "     'hgvs_protein': 'p.Cys1497Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48468504C>G\", \"seq_region_name\": \"15\", \"start\": 48468504, \"colocated_variants\": [{\"end\": 48468504, \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"start\": 48468504, \"id\": \"CM074842\"}, {\"id\": \"CM980730\", \"start\": 48468504, \"phenotype_or_disease\": 1, \"end\": 48468504, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1}, {\"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"C/G/T\", \"clin_sig_allele\": \"T:pathogenic/likely_pathogenic;G:pathogenic\", \"end\": 48468504, \"start\": 48468504, \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"phenotype_or_disease\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000663733\", \"VCV000549237\", \"RCV000659543\", \"VCV000547321\"]}, \"id\": \"rs1555397212\"}], \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"eve_score\": 0.8011857153877, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 4806, \"codons\": \"tGc/tCc\", \"cds_start\": 4490, \"spliceai\": {\"DP_DL\": 3, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AL\": 43, \"DS_AG\": 0, \"DS_DG\": 0, \"DP_DG\": 5, \"DS_DL\": 0, \"DP_AG\": -7}, \"cadd_raw\": 4.762339, \"location\": \"15:48468504\", \"mutfunc\": {\"mod\": {\"dG_mt_sd\": 0.777722, \"dG_wt\": 80.1528, \"ddG\": 3.65626, \"dG_mt\": 83.80906, \"ddG_sd\": 0.045687, \"dG_wt_sd\": 0.819518}, \"exp\": {\"dG_wt\": 22.6461, \"dG_mt_sd\": 0.010776, \"ddG_sd\": 0.037466, \"dG_wt_sd\": 0.038039, \"ddG\": 4.30596, \"dG_mt\": 26.95206}}, \"cadd_phred\": 26.6, \"allele_num\": 1, \"hgvsp\": \"NP_000129.3:p.Cys1497Ser\", \"aa\": \"C\", \"used_ref\": \"C\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.4490G>C\", \"biotype\": \"protein_coding\", \"protein_end\": 1497, \"conservation\": 3.95, \"alphamissense\": {\"am_pathogenicity\": 0.9994, \"am_class\": \"likely_pathogenic\"}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48468503, \"end\": 48468504}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"loeuf\": 0.105, \"exon\": \"37/66\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 4806, \"eve_class\": \"Pathogenic\", \"given_ref\": \"C\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"blosum62\": -1, \"mane_select\": \"ENST00000316623.10\", \"impact\": \"MODERATE\", \"variant_allele\": \"G\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 4490, \"protein_start\": 1497, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/S\", \"polyphen_score\": 0.886, \"polyphen_prediction\": \"possibly_damaging\", \"strand\": -1, \"mastermind_mmid3\": \"FBN1:C1497S\", \"clinpred\": 0.999963521957397, \"canonical\": 1, \"mutationtaster_pred\": \"A\", \"sift_score\": 0, \"mastermind_counts\": \"6|6|6\"}], \"id\": \"NC_000015.10:g.48468504C>G\", \"minimised\": 1, \"end\": 48468504, \"vcf_string\": \"15-48468504-C-G\", \"allele_string\": \"C/G\"}',\n",
       "    'coordinates': [{'pca_x': 0.037127895017478485,\n",
       "      'pca_y': 0.07878277013234594},\n",
       "     {'t-sne_x': 20.52181756670015, 't-sne_y': -5.30559234106351},\n",
       "     {'umap_x': 8.538939476013184, 'umap_y': 7.185831069946289}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.001065300777554512,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.0013305405154824257,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.001414524856954813,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0014195339754223824,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.0014216674026101828,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0014309176476672292,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.0014559299452230334,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.0016040944028645754,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.0017477888613939285,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.002046229550614953,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.002106588799506426,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495162-C-G',\n",
       "      'distance': 0.0023586079478263855,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468455-G-C',\n",
       "      'distance': 0.002364052925258875,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.0025707741733640432,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.0025768596678972244,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520708-C-G',\n",
       "      'distance': 0.002608738373965025,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.0026990892365574837,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.002703638281673193,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489991-C-G',\n",
       "      'distance': 0.002777158748358488,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437004-G-C',\n",
       "      'distance': 0.002796248998492956,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48468457-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48468457\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48468457',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4537T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48468457A>G',\n",
       "     'hgvs_protein': 'p.Cys1513Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48468457A>G\", \"transcript_consequences\": [{\"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.4537T>C\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"hgvsp\": \"NP_000129.3:p.Cys1513Arg\", \"aa\": \"A\", \"used_ref\": \"A\", \"alphamissense\": {\"am_pathogenicity\": 0.9995, \"am_class\": \"likely_pathogenic\"}, \"conservation\": 3.95, \"protein_end\": 1513, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48468456, \"type\": \"SimpleInterval\", \"end\": 48468457}, \"type\": \"SequenceLocation\"}}, \"biotype\": \"protein_coding\", \"location\": \"15:48468457\", \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 0.451115, \"dG_wt\": 22.5537, \"dG_wt_sd\": 0.091722, \"ddG_sd\": 0.479315, \"dG_mt\": 27.17336, \"ddG\": 4.61966}, \"mod\": {\"dG_wt_sd\": 0.024181, \"ddG_sd\": 1.043286, \"dG_mt\": 86.27644, \"ddG\": 5.79634, \"dG_wt\": 80.4801, \"dG_mt_sd\": 1.031676}}, \"cadd_raw\": 5.211337, \"cadd_phred\": 29.2, \"cdna_end\": 4853, \"spliceai\": {\"DP_AG\": 17, \"DS_DL\": 0, \"DP_DG\": -45, \"DS_DG\": 0, \"DP_AL\": 49, \"DS_AG\": 0, \"DS_AL\": 0, \"DP_DL\": -39, \"SYMBOL\": \"FBN1\"}, \"codons\": \"Tgc/Cgc\", \"cds_start\": 4537, \"eve_score\": 0.754605141584401, \"consequence_terms\": [\"missense_variant\"], \"allele_num\": 1, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/R\", \"protein_start\": 1513, \"cds_end\": 4537, \"protein_id\": \"NP_000129.3\", \"mutationtaster_pred\": \"D\", \"mastermind_counts\": \"6|10|10\", \"sift_score\": 0, \"mastermind_mmid3\": \"FBN1:C1513R\", \"canonical\": 1, \"clinpred\": 0.999896883964539, \"strand\": -1, \"polyphen_prediction\": \"possibly_damaging\", \"polyphen_score\": 0.615, \"ptriplo\": 0.984699554922396, \"given_ref\": \"A\", \"eve_class\": \"Pathogenic\", \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 4853, \"phaplo\": 0.99897615427687, \"exon\": \"37/66\", \"variant_allele\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -3, \"impact\": \"MODERATE\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\"}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"end\": 48468457, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"id\": \"CM940766\", \"start\": 48468457, \"phenotype_or_disease\": 1}, {\"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48468457, \"id\": \"CX100485\", \"start\": 48468457, \"phenotype_or_disease\": 1}, {\"phenotype_or_disease\": 1, \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"start\": 48468457, \"var_synonyms\": {\"UniProt\": [\"VAR_002326\"], \"ClinVar\": [\"RCV003775914\", \"RCV002340074\", \"VCV001741306\"]}, \"id\": \"rs112723282\", \"clin_sig_allele\": \"C:pathogenic;C:likely_pathogenic\", \"end\": 48468457, \"pubmed\": [8136837, 17701892], \"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"A/C/G\"}], \"variant_class\": \"SNV\", \"allele_string\": \"A/G\", \"end\": 48468457, \"vcf_string\": \"15-48468457-A-G\", \"minimised\": 1, \"seq_region_name\": \"15\", \"start\": 48468457, \"input\": \"NC_000015.10:g.48468457A>G\", \"strand\": 1, \"most_severe_consequence\": \"missense_variant\"}',\n",
       "    'coordinates': [{'pca_x': 0.045946107857899654,\n",
       "      'pca_y': 0.07201305863869922},\n",
       "     {'t-sne_x': 36.73301789540879, 't-sne_y': 3.636332758458336},\n",
       "     {'umap_x': 1.084974765777588, 'umap_y': 6.523336410522461}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.0004438405157998204,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468418-A-G',\n",
       "      'distance': 0.0007606294238939881,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.0009593183058314025,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0009964390192180872,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0010315291583538055,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.001169858267530799,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0011901443358510733,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.001222698949277401,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0012476514093577862,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0012507530627772212,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468490-A-G',\n",
       "      'distance': 0.00126023986376822,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.001316990703344345,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0013455308508127928,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.001368437078781426,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456688-A-G',\n",
       "      'distance': 0.0013746039476245642,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.0014524792786687613,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.0014910870231688023,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.0015048208879306912,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-G',\n",
       "      'distance': 0.001505799125880003,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0015120443422347307,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48468083-G-T',\n",
       "    'metadata': {'vcf_string': '15\\t48468083\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48468083',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4602C>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48468083G>T',\n",
       "     'hgvs_protein': 'p.Cys1534Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"allele_string\": \"G/T\", \"vcf_string\": \"15-48468083-G-T\", \"end\": 48468083, \"colocated_variants\": [{\"id\": \"CM085426\", \"start\": 48468083, \"phenotype_or_disease\": 1, \"end\": 48468083, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1}, {\"id\": \"COSV57327626\", \"var_synonyms\": {\"COSMIC\": [\"COSM5519305\"]}, \"start\": 48468083, \"phenotype_or_disease\": 1, \"end\": 48468083, \"allele_string\": \"COSMIC_MUTATION\", \"seq_region_name\": \"15\", \"somatic\": 1, \"strand\": 1}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48468083G>T\", \"transcript_consequences\": [{\"aa\": \"G\", \"used_ref\": \"G\", \"hgvsp\": \"NP_000129.3:p.Cys1534Ter\", \"hgvsc\": \"NM_000138.5:c.4602C>A\", \"gene_id\": \"2200\", \"mane\": [\"MANE_Select\"], \"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48468082, \"type\": \"SimpleInterval\", \"end\": 48468083}, \"type\": \"SequenceLocation\"}}, \"protein_end\": 1534, \"conservation\": 3.7, \"cds_start\": 4602, \"codons\": \"tgC/tgA\", \"spliceai\": {\"DS_AG\": 0.03, \"DP_AL\": 19, \"DP_DL\": -7, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AG\": -23, \"DS_DL\": 0, \"DS_DG\": 0, \"DP_DG\": 28}, \"cdna_end\": 4918, \"consequence_terms\": [\"stop_gained\"], \"cadd_phred\": 39, \"location\": \"15:48468083\", \"cadd_raw\": 8.963247, \"mutfunc\": {\"mod\": {\"ddG_sd\": 0.024741, \"dG_wt_sd\": 0.135868, \"ddG\": -0.76737, \"dG_mt\": 79.65133, \"dG_wt\": 80.4187, \"dG_mt_sd\": 0.132716}, \"exp\": {\"ddG_sd\": 0.009299, \"dG_wt_sd\": 0.003005, \"ddG\": -0.40164, \"dG_mt\": 22.2913, \"dG_wt\": 22.69294, \"dG_mt_sd\": 0.00707}}, \"allele_num\": 1, \"cds_end\": 4602, \"protein_id\": \"NP_000129.3\", \"amino_acids\": \"C/*\", \"transcript_id\": \"NM_000138.5\", \"lof\": \"HC\", \"protein_start\": 1534, \"strand\": -1, \"mastermind_counts\": \"3|3|3\", \"mutationtaster_pred\": \"A\", \"mastermind_mmid3\": \"FBN1:C1534X\", \"canonical\": 1, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"exon\": \"38/66\", \"cdna_start\": 4918, \"loeuf\": 0.105, \"ptriplo\": 0.984699554922396, \"given_ref\": \"G\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"variant_allele\": \"T\", \"impact\": \"HIGH\", \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"PERCENTILE:0.534122562674095,GERP_DIST:4810.56866104901,BP_DIST:3986,DIST_FROM_LAST_EXON:3597,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\"}], \"input\": \"NC_000015.10:g.48468083G>T\", \"most_severe_consequence\": \"stop_gained\", \"strand\": 1, \"start\": 48468083, \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.020492715198335176,\n",
       "      'pca_y': 0.0936102954779813},\n",
       "     {'t-sne_x': 30.30237356367197, 't-sne_y': 38.7467807427629},\n",
       "     {'umap_x': -10.32478141784668, 'umap_y': 2.4368224143981934}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48485483-G-T',\n",
       "      'distance': 0.0006602288922294974,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421995-G-T',\n",
       "      'distance': 0.0011445618001744151,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.0012637521140277386,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.0014975561061874032,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474565-G-T',\n",
       "      'distance': 0.0016180782113224268,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.00171054364182055,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495241-G-T',\n",
       "      'distance': 0.001765543594956398,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421601-G-T',\n",
       "      'distance': 0.0017677012365311384,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415610-G-T',\n",
       "      'distance': 0.0017934372881427407,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495130-G-T',\n",
       "      'distance': 0.0018051236402243376,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503845-G-T',\n",
       "      'distance': 0.0021758959628641605,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.0022444676142185926,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513553-G-T',\n",
       "      'distance': 0.0022744624875485897,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489921-G-T',\n",
       "      'distance': 0.002402748679742217,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421652-G-T',\n",
       "      'distance': 0.002472675871104002,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495575-G-T',\n",
       "      'distance': 0.002519500907510519,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.002759525552392006,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.00282377190887928,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425416-G-T',\n",
       "      'distance': 0.0028755611274391413,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48441797-G-T',\n",
       "      'distance': 0.0029481505043804646,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48467982-TT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48467982\\t.\\tTT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48467982',\n",
       "     'ref_allele': 'TT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4704del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48467983del',\n",
       "     'hgvs_protein': 'p.Ala1569ProfsTer12',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48467983del\", \"transcript_consequences\": [{\"strand\": -1, \"canonical\": 1, \"protein_id\": \"NP_000129.3\", \"cds_end\": 4704, \"lof\": \"HC\", \"protein_start\": 1568, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"K/X\", \"gene_symbol\": \"FBN1\", \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"PERCENTILE:0.545961002785515,GERP_DIST:4707.51904172376,BP_DIST:3886,DIST_FROM_LAST_EXON:3497,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"impact\": \"HIGH\", \"variant_allele\": \"-\", \"loeuf\": 0.105, \"cdna_start\": 5020, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"exon\": \"38/66\", \"given_ref\": \"T\", \"ptriplo\": 0.984699554922396, \"biotype\": \"protein_coding\", \"protein_end\": 1568, \"conservation\": 3.7, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"end\": 48467983, \"start\": 48467982, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Ala1569ProfsTer12\", \"used_ref\": \"T\", \"aa\": \"T\", \"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.4704del\", \"allele_num\": 1, \"consequence_terms\": [\"frameshift_variant\"], \"cdna_end\": 5020, \"codons\": \"aaA/aa\", \"spliceai\": {\"DS_DG\": 0.92, \"DP_DG\": 4, \"DS_DL\": 0.42, \"DP_AG\": 24, \"SYMBOL\": \"FBN1\", \"DP_DL\": -44, \"DS_AL\": 0, \"DP_AL\": 4, \"DS_AG\": 0}, \"cds_start\": 4704, \"mutfunc\": {\"exp\": {\"dG_wt\": 21.9628, \"dG_mt_sd\": 6.902041, \"ddG\": 46.6213, \"dG_mt\": 68.5841, \"dG_wt_sd\": 0.691188, \"ddG_sd\": 6.229275}, \"mod\": {\"dG_mt_sd\": 3.637574, \"dG_wt\": 80.61492, \"ddG\": 37.60688, \"dG_mt\": 118.2218, \"dG_wt_sd\": 1.061928, \"ddG_sd\": 3.020862}}, \"location\": \"15:48467981\", \"hgvs_offset\": -2}], \"variant_class\": \"deletion\", \"assembly_name\": \"GRCh38\", \"allele_string\": \"T/-\", \"end\": 48467983, \"vcf_string\": \"15-48467982-TT-T\", \"minimised\": 1, \"seq_region_name\": \"15\", \"start\": 48467983, \"input\": \"NC_000015.10:g.48467983del\", \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.044357579527727564,\n",
       "      'pca_y': 0.029720631438331417},\n",
       "     {'t-sne_x': 2.2897126911961734, 't-sne_y': 9.44293295752152},\n",
       "     {'umap_x': 3.86074161529541, 'umap_y': 3.218817949295044}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09058566391468048,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09144294261932373,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.09208182990550995,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.09232516586780548,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.0924287661910057,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452567-C-T',\n",
       "      'distance': 0.09245942533016205,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437069-C-T',\n",
       "      'distance': 0.09275040775537491,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.0928155928850174,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.09284880757331848,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.09286454319953918,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.09298064559698105,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468504-C-T',\n",
       "      'distance': 0.09306427091360092,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.09308856725692749,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.0932479277253151,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.09326564520597458,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.0933515802025795,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.09342887997627258,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.09343339502811432,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.09343535453081131,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.09348227083683014,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48465652-CT-C',\n",
       "    'metadata': {'vcf_string': '15\\t48465652\\t.\\tCT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48465652',\n",
       "     'ref_allele': 'CT',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4857del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48465653del',\n",
       "     'hgvs_protein': 'p.Gly1620GlufsTer20',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"start\": 48465653, \"seq_region_name\": \"15\", \"input\": \"NC_000015.10:g.48465653del\", \"strand\": 1, \"most_severe_consequence\": \"frameshift_variant\", \"id\": \"NC_000015.10:g.48465653del\", \"transcript_consequences\": [{\"location\": \"15:48465653\", \"mutfunc\": {\"mod\": {\"dG_wt\": 81.12268, \"dG_mt_sd\": 0.002441, \"ddG\": -0.22588, \"dG_mt\": 80.8968, \"dG_wt_sd\": 0.008925, \"ddG_sd\": 0.007935}, \"exp\": {\"ddG_sd\": 0.01723, \"dG_wt_sd\": 0.011917, \"ddG\": -0.15957, \"dG_mt\": 22.44741, \"dG_wt\": 22.60698, \"dG_mt_sd\": 0.008639}}, \"consequence_terms\": [\"frameshift_variant\"], \"cdna_end\": 5173, \"cds_start\": 4857, \"codons\": \"ggA/gg\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -33, \"DS_DG\": 0, \"DP_DG\": -6, \"DS_AG\": 0, \"DP_AL\": 41, \"DP_DL\": -19, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0.01}, \"allele_num\": 1, \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.4857del\", \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Gly1620GlufsTer20\", \"aa\": \"T\", \"used_ref\": \"T\", \"protein_end\": 1619, \"conservation\": -7.89, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48465653, \"type\": \"SimpleInterval\", \"start\": 48465652}, \"type\": \"SequenceLocation\"}}, \"biotype\": \"protein_coding\", \"given_ref\": \"T\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"exon\": \"40/66\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 5173, \"lof_info\": \"PERCENTILE:0.563718662952646,GERP_DIST:4511.09710225761,BP_DIST:3733,DIST_FROM_LAST_EXON:3344,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mane_select\": \"ENST00000316623.10\", \"impact\": \"HIGH\", \"variant_allele\": \"-\", \"gene_symbol\": \"FBN1\", \"protein_start\": 1619, \"lof\": \"HC\", \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"G/X\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 4857, \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:G1620fs\", \"mastermind_counts\": \"6|6|6\", \"strand\": -1}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"id\": \"CD951706\", \"phenotype_or_disease\": 1, \"start\": 48465653, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48465653}], \"variant_class\": \"deletion\", \"allele_string\": \"T/-\", \"vcf_string\": \"15-48465652-CT-C\", \"end\": 48465653, \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.020292435831535893,\n",
       "      'pca_y': 0.03198828736600256},\n",
       "     {'t-sne_x': 9.799949418437327, 't-sne_y': 15.383962854828685},\n",
       "     {'umap_x': 3.804975748062134, 'umap_y': 3.3649566173553467}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.11987154930830002,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.12020672112703323,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.12097980082035065,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.12153840810060501,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.12202739715576172,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.1224198192358017,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.12269699573516846,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.12279772013425827,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.12285541743040085,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.1229219138622284,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.1233428567647934,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.12341052293777466,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.1235651820898056,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445430-G-A',\n",
       "      'distance': 0.12364670634269714,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508612-G-A',\n",
       "      'distance': 0.12380558252334595,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425424-G-T',\n",
       "      'distance': 0.12394958734512329,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437912-G-A',\n",
       "      'distance': 0.12397307902574539,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445419-G-T',\n",
       "      'distance': 0.1239982545375824,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.12402227520942688,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.12409326434135437,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48465619-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48465619\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48465619',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4891T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48465619A>C',\n",
       "     'hgvs_protein': 'p.Cys1631Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"start\": 48465619, \"strand\": 1, \"most_severe_consequence\": \"missense_variant\", \"input\": \"NC_000015.10:g.48465619A>C\", \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"cds_end\": 4891, \"protein_start\": 1631, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/G\", \"polyphen_prediction\": \"probably_damaging\", \"polyphen_score\": 1, \"strand\": -1, \"canonical\": 1, \"clinpred\": 0.999934077262878, \"mastermind_mmid3\": \"FBN1:C1631G\", \"mutationtaster_pred\": \"D\", \"mastermind_counts\": \"6|9|9\", \"sift_score\": 0, \"loeuf\": 0.105, \"phaplo\": 0.99897615427687, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"40/66\", \"cdna_start\": 5207, \"eve_class\": \"Pathogenic\", \"given_ref\": \"A\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"blosum62\": -3, \"mane_select\": \"ENST00000316623.10\", \"impact\": \"MODERATE\", \"variant_allele\": \"C\", \"hgvsp\": \"NP_000129.3:p.Cys1631Gly\", \"aa\": \"A\", \"used_ref\": \"A\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.4891T>G\", \"biotype\": \"protein_coding\", \"protein_end\": 1631, \"conservation\": 3.95, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9966}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"end\": 48465619, \"type\": \"SimpleInterval\", \"start\": 48465618}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"eve_score\": 0.792633675269003, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 5207, \"codons\": \"Tgc/Ggc\", \"spliceai\": {\"DP_AG\": 0, \"DS_DL\": 0.02, \"DP_DG\": 27, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": -41, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -41}, \"cds_start\": 4891, \"location\": \"15:48465619\", \"mutfunc\": {\"exp\": {\"dG_mt\": 27.39528, \"ddG\": 4.70514, \"dG_wt_sd\": 0.011985, \"ddG_sd\": 0.017908, \"dG_mt_sd\": 0.01299, \"dG_wt\": 22.69014}, \"mod\": {\"dG_wt\": 80.58546, \"dG_mt_sd\": 0.0638, \"dG_mt\": 84.99242, \"ddG\": 4.40696, \"ddG_sd\": 0.019035, \"dG_wt_sd\": 0.071162}}, \"cadd_raw\": 5.285035, \"cadd_phred\": 29.7, \"allele_num\": 1}], \"id\": \"NC_000015.10:g.48465619A>C\", \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"start\": 48465619, \"phenotype_or_disease\": 1, \"id\": \"CM054701\", \"end\": 48465619, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\"}], \"variant_class\": \"SNV\", \"vcf_string\": \"15-48465619-A-C\", \"end\": 48465619, \"allele_string\": \"A/C\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.045206141503759456,\n",
       "      'pca_y': 0.09117071393004528},\n",
       "     {'t-sne_x': 19.749006929392994, 't-sne_y': 3.704916856020775},\n",
       "     {'umap_x': 3.887627124786377, 'umap_y': 6.238476753234863}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.0024390616454184055,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445432-A-C',\n",
       "      'distance': 0.0026957548689097166,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.0029505707789212465,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48436967-A-C',\n",
       "      'distance': 0.0030550858937203884,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421997-A-C',\n",
       "      'distance': 0.0031444698106497526,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483822-A-C',\n",
       "      'distance': 0.0031486304942518473,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463219-T-C',\n",
       "      'distance': 0.0031986210960894823,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.003232469316571951,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465840-C-A',\n",
       "      'distance': 0.003382243914529681,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415716-T-C',\n",
       "      'distance': 0.0034615262411534786,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468522-C-A',\n",
       "      'distance': 0.0035077915526926517,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.0036181933246552944,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463210-T-C',\n",
       "      'distance': 0.00363559415563941,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434627-C-T',\n",
       "      'distance': 0.0036442545242607594,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.003668934805318713,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.003748196642845869,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437011-T-C',\n",
       "      'distance': 0.0037518481258302927,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.0037748264148831367,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434707-T-C',\n",
       "      'distance': 0.0037769507616758347,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411228-T-C',\n",
       "      'distance': 0.003829625900834799,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48465574-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48465574\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48465574',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4936T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48465574A>G',\n",
       "     'hgvs_protein': 'p.Cys1646Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48465574A>G\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"start\": 48465574, \"seq_region_name\": \"15\", \"minimised\": 1, \"allele_string\": \"A/G\", \"vcf_string\": \"15-48465574-A-G\", \"end\": 48465574, \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48465574, \"start\": 48465574, \"phenotype_or_disease\": 1, \"id\": \"CM077927\"}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48465574A>G\", \"transcript_consequences\": [{\"gene_symbol_source\": \"EntrezGene\", \"cdna_start\": 5252, \"phaplo\": 0.99897615427687, \"exon\": \"40/66\", \"loeuf\": 0.105, \"given_ref\": \"A\", \"eve_class\": \"Pathogenic\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"impact\": \"MODERATE\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -3, \"variant_allele\": \"G\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 4936, \"protein_start\": 1646, \"amino_acids\": \"C/R\", \"transcript_id\": \"NM_000138.5\", \"polyphen_score\": 0.997, \"polyphen_prediction\": \"probably_damaging\", \"strand\": -1, \"clinpred\": 0.999980688095093, \"mastermind_mmid3\": \"FBN1:C1646R\", \"canonical\": 1, \"sift_score\": 0, \"mastermind_counts\": \"1|2|2\", \"mutationtaster_pred\": \"D\", \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.792115252819451, \"codons\": \"Tgt/Cgt\", \"cds_start\": 4936, \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DP_DL\": 2, \"DS_AL\": 0, \"DP_AL\": 4, \"DS_AG\": 0, \"DS_DG\": 0.47, \"DP_DG\": 4, \"DP_AG\": 45, \"DS_DL\": 0.01}, \"cdna_end\": 5252, \"cadd_phred\": 34, \"location\": \"15:48465574\", \"mutfunc\": {\"mod\": {\"dG_mt\": 89.82364, \"ddG\": 8.81546, \"dG_wt_sd\": 0.11403, \"ddG_sd\": 0.731962, \"dG_wt\": 81.00818, \"dG_mt_sd\": 0.789542}, \"exp\": {\"dG_mt_sd\": 0.620349, \"dG_wt\": 22.64554, \"dG_wt_sd\": 0.028796, \"ddG_sd\": 0.603312, \"dG_mt\": 30.19268, \"ddG\": 7.54714}}, \"cadd_raw\": 6.173727, \"allele_num\": 1, \"used_ref\": \"A\", \"aa\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys1646Arg\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"hgvsc\": \"NM_000138.5:c.4936T>C\", \"gene_id\": \"2200\", \"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48465573, \"end\": 48465574}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9998}, \"protein_end\": 1646, \"conservation\": 3.95}]}',\n",
       "    'coordinates': [{'pca_x': 0.051945823095375265,\n",
       "      'pca_y': 0.07808707992504985},\n",
       "     {'t-sne_x': 31.31638267160945, 't-sne_y': 4.426076172839563},\n",
       "     {'umap_x': 1.9731981754302979, 'umap_y': 6.86985445022583}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48515396-C-A',\n",
       "      'distance': 0.0016523541416972876,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434702-A-G',\n",
       "      'distance': 0.0018277252092957497,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.0018675109604373574,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.0019010677933692932,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596325-A-G',\n",
       "      'distance': 0.0019311720971018076,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.002049617236480117,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596319-A-G',\n",
       "      'distance': 0.0020550014451146126,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.002120077144354582,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434600-A-G',\n",
       "      'distance': 0.0022209486924111843,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0022392794489860535,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468418-A-G',\n",
       "      'distance': 0.0022516599856317043,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468490-A-G',\n",
       "      'distance': 0.0022763721644878387,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.002277606865391135,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0023090378381311893,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48452655-A-G',\n",
       "      'distance': 0.0023358124308288097,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0023712823167443275,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596373-A-G',\n",
       "      'distance': 0.0023779496550559998,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.0024115354754030704,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437873-A-G',\n",
       "      'distance': 0.0024265386164188385,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0024328280705958605,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48463977-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48463977\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48463977',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.4987T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48463977A>G',\n",
       "     'hgvs_protein': 'p.Cys1663Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48463977A>G\", \"transcript_consequences\": [{\"biotype\": \"protein_coding\", \"protein_end\": 1663, \"alphamissense\": {\"am_pathogenicity\": 0.9997, \"am_class\": \"likely_pathogenic\"}, \"conservation\": 3.95, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"end\": 48463977, \"type\": \"SimpleInterval\", \"start\": 48463976}}, \"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"hgvsp\": \"NP_000129.3:p.Cys1663Arg\", \"aa\": \"A\", \"used_ref\": \"A\", \"sift_prediction\": \"deleterious_low_confidence\", \"mane\": [\"MANE_Select\"], \"hgvsc\": \"NM_000138.5:c.4987T>C\", \"gene_id\": \"2200\", \"allele_num\": 1, \"eve_score\": 0.760806903171562, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 5303, \"cds_start\": 4987, \"codons\": \"Tgt/Cgt\", \"spliceai\": {\"DS_AG\": 0, \"DP_AL\": -43, \"DP_DL\": 33, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DP_AG\": 5, \"DS_DL\": 0, \"DS_DG\": 0, \"DP_DG\": -49}, \"cadd_raw\": 5.394688, \"location\": \"15:48463977\", \"cadd_phred\": 31, \"polyphen_prediction\": \"probably_damaging\", \"polyphen_score\": 0.996, \"strand\": -1, \"canonical\": 1, \"clinpred\": 0.999815404415131, \"mastermind_mmid3\": \"FBN1:C1663R\", \"mutationtaster_pred\": \"A\", \"mastermind_counts\": \"13|32|32\", \"sift_score\": 0, \"protein_id\": \"NP_000129.3\", \"cds_end\": 4987, \"protein_start\": 1663, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/R\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"mane_select\": \"ENST00000316623.10\", \"blosum62\": -3, \"impact\": \"MODERATE\", \"variant_allele\": \"G\", \"loeuf\": 0.105, \"exon\": \"41/66\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 5303, \"eve_class\": \"Pathogenic\", \"given_ref\": \"A\", \"ptriplo\": 0.984699554922396}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48463977, \"id\": \"CM920268\", \"start\": 48463977, \"phenotype_or_disease\": 1}, {\"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"A/G\", \"clin_sig_allele\": \"G:pathogenic\", \"end\": 48463977, \"pubmed\": [1301946], \"clin_sig\": [\"pathogenic\"], \"start\": 48463977, \"phenotype_or_disease\": 1, \"var_synonyms\": {\"OMIM\": [134797.0006], \"ClinVar\": [\"RCV000017888\", \"VCV000016426\"], \"UniProt\": [\"VAR_002329\"]}, \"id\": \"rs137854459\"}], \"variant_class\": \"SNV\", \"allele_string\": \"A/G\", \"end\": 48463977, \"vcf_string\": \"15-48463977-A-G\", \"minimised\": 1, \"seq_region_name\": \"15\", \"start\": 48463977, \"input\": \"NC_000015.10:g.48463977A>G\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.046810384230477925,\n",
       "      'pca_y': 0.07437048548897118},\n",
       "     {'t-sne_x': 35.68234541641207, 't-sne_y': 4.08633541790331},\n",
       "     {'umap_x': 1.1504359245300293, 'umap_y': 6.445736885070801}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.0007501522777602077,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.0007611823966726661,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0010292711667716503,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0010450499830767512,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.0010700988350436091,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0010947134578600526,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.0011073288042098284,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.0011432578321546316,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0011772153666242957,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.001185007393360138,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497317-A-G',\n",
       "      'distance': 0.0012021467555314302,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0012475065886974335,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0012480344157665968,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.0012482895981520414,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.001267171697691083,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.0012767810840159655,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468418-A-G',\n",
       "      'distance': 0.0012998025631532073,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-G',\n",
       "      'distance': 0.0013177809305489063,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.0013437523739412427,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452673-A-G',\n",
       "      'distance': 0.001367902965284884,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48463950-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48463950\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48463950',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5014T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48463950A>G',\n",
       "     'hgvs_protein': 'p.Cys1672Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"allele_string\": \"A/G\", \"vcf_string\": \"15-48463950-A-G\", \"end\": 48463950, \"variant_class\": \"SNV\", \"colocated_variants\": [{\"id\": \"CM992329\", \"start\": 48463950, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48463950}], \"assembly_name\": \"GRCh38\", \"id\": \"NC_000015.10:g.48463950A>G\", \"transcript_consequences\": [{\"variant_allele\": \"G\", \"blosum62\": -3, \"mane_select\": \"ENST00000316623.10\", \"impact\": \"MODERATE\", \"lrt_pred\": \"invalid_field\", \"gene_symbol\": \"FBN1\", \"ptriplo\": 0.984699554922396, \"given_ref\": \"A\", \"eve_class\": \"Pathogenic\", \"loeuf\": 0.105, \"exon\": \"41/66\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 5330, \"mutationtaster_pred\": \"D\", \"sift_score\": 0, \"mastermind_counts\": \"5|6|6\", \"clinpred\": 0.999852538108826, \"mastermind_mmid3\": \"FBN1:C1672R\", \"canonical\": 1, \"strand\": -1, \"polyphen_score\": 0.996, \"polyphen_prediction\": \"probably_damaging\", \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/R\", \"protein_start\": 1672, \"cds_end\": 5014, \"protein_id\": \"NP_000129.3\", \"allele_num\": 1, \"cadd_raw\": 5.466066, \"location\": \"15:48463950\", \"cadd_phred\": 32, \"cdna_end\": 5330, \"cds_start\": 5014, \"codons\": \"Tgt/Cgt\", \"spliceai\": {\"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -26, \"DP_AL\": -16, \"DS_AG\": 0, \"DP_DG\": -49, \"DS_DG\": 0, \"DS_DL\": 0, \"DP_AG\": 32}, \"eve_score\": 0.809568106919603, \"consequence_terms\": [\"missense_variant\"], \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9999}, \"protein_end\": 1672, \"conservation\": 3.95, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48463950, \"type\": \"SimpleInterval\", \"start\": 48463949}, \"type\": \"SequenceLocation\"}}, \"biotype\": \"protein_coding\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.5014T>C\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"hgvsp\": \"NP_000129.3:p.Cys1672Arg\", \"aa\": \"A\", \"used_ref\": \"A\"}], \"input\": \"NC_000015.10:g.48463950A>G\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"start\": 48463950, \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.0397617421868135, 'pca_y': 0.0981289657393424},\n",
       "     {'t-sne_x': -19.088644248758865, 't-sne_y': 34.06275138568629},\n",
       "     {'umap_x': 2.216496467590332, 'umap_y': 20.761699676513672}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463148-A-G',\n",
       "      'distance': 0.0008961630519479513,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48513627-A-G',\n",
       "      'distance': 0.0013707345351576805,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513631-A-G',\n",
       "      'distance': 0.0013707345351576805,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48513600-A-G',\n",
       "      'distance': 0.001858868170529604,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445454-A-G',\n",
       "      'distance': 0.0019044846994802356,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472644-A-G',\n",
       "      'distance': 0.0019716359674930573,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48452637-A-G',\n",
       "      'distance': 0.0021162754856050014,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495216-A-G',\n",
       "      'distance': 0.0021202596835792065,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.0021919752471148968,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.0023112366907298565,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505136-A-G',\n",
       "      'distance': 0.002315293066203594,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456643-A-G',\n",
       "      'distance': 0.002381756668910384,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510050-A-G',\n",
       "      'distance': 0.0027216593734920025,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515434-C-A',\n",
       "      'distance': 0.0029183286242187023,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494210-A-G',\n",
       "      'distance': 0.0032580802217125893,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463950-A-C',\n",
       "      'distance': 0.0032942607067525387,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472557-A-T',\n",
       "      'distance': 0.003299775067716837,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515465-G-A',\n",
       "      'distance': 0.003334225621074438,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465646-A-G',\n",
       "      'distance': 0.0035477664787322283,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463949-C-G',\n",
       "      'distance': 0.0036029675975441933,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48463944-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48463944\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48463944',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5020T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48463944A>C',\n",
       "     'hgvs_protein': 'p.Cys1674Gly',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48463944A>C\", \"seq_region_name\": \"15\", \"start\": 48463944, \"minimised\": 1, \"vcf_string\": \"15-48463944-A-C\", \"end\": 48463944, \"allele_string\": \"A/C\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48463944, \"start\": 48463944, \"phenotype_or_disease\": 1, \"id\": \"CM085411\"}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"cds_end\": 5020, \"protein_start\": 1674, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/G\", \"polyphen_prediction\": \"probably_damaging\", \"polyphen_score\": 0.985, \"strand\": -1, \"clinpred\": 0.999811351299286, \"mastermind_mmid3\": \"FBN1:C1674G\", \"canonical\": 1, \"mutationtaster_pred\": \"D\", \"mastermind_counts\": \"4|4|4\", \"sift_score\": 0, \"loeuf\": 0.105, \"exon\": \"41/66\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 5336, \"eve_class\": \"Pathogenic\", \"given_ref\": \"A\", \"ptriplo\": 0.984699554922396, \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"blosum62\": -3, \"mane_select\": \"ENST00000316623.10\", \"impact\": \"MODERATE\", \"variant_allele\": \"C\", \"hgvsp\": \"NP_000129.3:p.Cys1674Gly\", \"aa\": \"A\", \"used_ref\": \"A\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.5020T>G\", \"biotype\": \"protein_coding\", \"alphamissense\": {\"am_pathogenicity\": 0.9986, \"am_class\": \"likely_pathogenic\"}, \"protein_end\": 1674, \"conservation\": 3.95, \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48463943, \"end\": 48463944}, \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"eve_score\": 0.733754284931367, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 5336, \"cds_start\": 5020, \"codons\": \"Tgt/Ggt\", \"spliceai\": {\"DS_DG\": 0, \"DP_DG\": -45, \"DS_DL\": 0, \"DP_AG\": 38, \"SYMBOL\": \"FBN1\", \"DP_DL\": -20, \"DS_AL\": 0, \"DS_AG\": 0, \"DP_AL\": -10}, \"cadd_raw\": 5.321014, \"location\": \"15:48463944\", \"cadd_phred\": 29.9, \"allele_num\": 1}], \"id\": \"NC_000015.10:g.48463944A>C\"}',\n",
       "    'coordinates': [{'pca_x': 0.0355248384641125,\n",
       "      'pca_y': 0.10616444815706819},\n",
       "     {'t-sne_x': -16.7259409237076, 't-sne_y': 30.804105737734304},\n",
       "     {'umap_x': 2.13105845451355, 'umap_y': 21.530914306640625}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463950-A-C',\n",
       "      'distance': 0.0021466417238116264,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474345-A-C',\n",
       "      'distance': 0.0021525039337575436,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495512-A-C',\n",
       "      'distance': 0.0026685267221182585,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441764-A-C',\n",
       "      'distance': 0.0027571036480367184,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515429-A-G',\n",
       "      'distance': 0.0028459427412599325,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441765-C-T',\n",
       "      'distance': 0.004120556637644768,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.004307017661631107,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505136-A-C',\n",
       "      'distance': 0.004332143813371658,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48508633-A-C',\n",
       "      'distance': 0.004366379231214523,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48494210-A-G',\n",
       "      'distance': 0.004373620264232159,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481729-T-C',\n",
       "      'distance': 0.004430864937603474,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48421990-C-T',\n",
       "      'distance': 0.004475632682442665,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48499021-A-G',\n",
       "      'distance': 0.004539070650935173,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.004577220417559147,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495519-C-A',\n",
       "      'distance': 0.0046330224722623825,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472644-A-G',\n",
       "      'distance': 0.0047743963077664375,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.004796473775058985,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481690-C-T',\n",
       "      'distance': 0.004828731529414654,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495216-A-G',\n",
       "      'distance': 0.004952709656208754,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425401-T-C',\n",
       "      'distance': 0.004980655387043953,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48463899-C-CC',\n",
       "    'metadata': {'vcf_string': '15\\t48463899\\t.\\tC\\tCC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48463899',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'CC',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5065+1dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48463900dup',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"id\": \"NC_000015.10:g.48463900dup\", \"transcript_consequences\": [{\"gene_symbol\": \"FBN1\", \"variant_allele\": \"C\", \"allele_num\": 1, \"impact\": \"HIGH\", \"mane_select\": \"ENST00000316623.10\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"spliceai\": {\"DP_AG\": 35, \"DS_DL\": 0, \"DP_DG\": 29, \"DS_DG\": 0, \"DP_AL\": 0, \"DS_AG\": 0, \"DS_AL\": 0, \"DP_DL\": 6, \"SYMBOL\": \"FBN1\"}, \"loeuf\": 0.105, \"consequence_terms\": [\"splice_donor_variant\"], \"ptriplo\": 0.984699554922396, \"hgvs_offset\": -2, \"location\": \"15:48463897-48463898\", \"given_ref\": \"-\", \"strand\": -1, \"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48463899, \"end\": 48463899}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"conservation\": 3.95, \"canonical\": 1, \"used_ref\": \"-\", \"protein_id\": \"NP_000129.3\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.5065+1dup\", \"transcript_id\": \"NM_000138.5\", \"lof\": \"HC\", \"mane\": [\"MANE_Select\"]}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"insertion\", \"allele_string\": \"-/C\", \"end\": 48463899, \"vcf_string\": \"15-48463899-C-CC\", \"minimised\": 1, \"seq_region_name\": \"15\", \"start\": 48463900, \"input\": \"NC_000015.10:g.48463900dup\", \"strand\": 1, \"most_severe_consequence\": \"splice_donor_variant\"}',\n",
       "    'coordinates': [{'pca_x': -0.2034585528150696,\n",
       "      'pca_y': -0.03507850189694846},\n",
       "     {'t-sne_x': -22.033925655257296, 't-sne_y': -25.30465686944157},\n",
       "     {'umap_x': 8.776978492736816, 'umap_y': 12.957376480102539}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48494203-C-G',\n",
       "      'distance': 0.05359837785363197,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460244-A-C',\n",
       "      'distance': 0.05411046743392944,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425368-C-T',\n",
       "      'distance': 0.054788459092378616,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483816-A-G',\n",
       "      'distance': 0.05736719071865082,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-G',\n",
       "      'distance': 0.05739957466721535,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427566-C-A',\n",
       "      'distance': 0.057845842093229294,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488111-A-G',\n",
       "      'distance': 0.057894110679626465,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456635-A-G',\n",
       "      'distance': 0.057959675788879395,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600138-C-G',\n",
       "      'distance': 0.058842532336711884,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.0619068518280983,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48613009-C-A',\n",
       "      'distance': 0.06288778781890869,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428345-C-T',\n",
       "      'distance': 0.06323045492172241,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.06337328255176544,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.06367659568786621,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.06374171376228333,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496099-C-T',\n",
       "      'distance': 0.0642203539609909,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488112-C-T',\n",
       "      'distance': 0.0642583817243576,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.06454360485076904,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.0653284341096878,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.06571682542562485,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48463169-T-TGAAT',\n",
       "    'metadata': {'vcf_string': '15\\t48463169\\t.\\tT\\tTGAAT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48463169',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'TGAAT',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5134_5137dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48463170_48463173dup',\n",
       "     'hgvs_protein': 'p.Asn1713IlefsTer23',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"start\": 48463170, \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48463170_48463173dup\", \"transcript_consequences\": [{\"protein_start\": 1713, \"lof\": \"HC\", \"amino_acids\": \"N/IQX\", \"transcript_id\": \"NM_000138.5\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 5138, \"canonical\": 1, \"strand\": -1, \"given_ref\": \"-\", \"ptriplo\": 0.984699554922396, \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 5453, \"exon\": \"42/66\", \"loeuf\": 0.105, \"impact\": \"HIGH\", \"lof_info\": \"PERCENTILE:0.596216341689879,GERP_DIST:4116.00370168388,BP_DIST:3456,DIST_FROM_LAST_EXON:3067,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mane_select\": \"ENST00000316623.10\", \"variant_allele\": \"TGAA\", \"gene_symbol\": \"FBN1\", \"mane\": [\"MANE_Select\"], \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.5134_5137dup\", \"used_ref\": \"-\", \"hgvsp\": \"NP_000129.3:p.Asn1713IlefsTer23\", \"conservation\": 3.065, \"protein_end\": 1713, \"biotype\": \"protein_coding\", \"hgvs_offset\": -1, \"location\": \"15:48463168-48463169\", \"consequence_terms\": [\"frameshift_variant\"], \"cds_start\": 5137, \"codons\": \"aac/aTTCAac\", \"cdna_end\": 5454, \"allele_num\": 1}], \"id\": \"NC_000015.10:g.48463170_48463173dup\", \"variant_class\": \"insertion\", \"assembly_name\": \"GRCh38\", \"end\": 48463169, \"vcf_string\": \"15-48463169-T-TGAAT\", \"allele_string\": \"-/GAAT\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.031095496298068404,\n",
       "      'pca_y': 0.009592789656339022},\n",
       "     {'t-sne_x': 6.839863842248759, 't-sne_y': 2.021412013059909},\n",
       "     {'umap_x': 4.129852294921875, 'umap_y': 3.31887149810791}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48470689-G-A',\n",
       "      'distance': 0.09309937804937363,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520693-A-G',\n",
       "      'distance': 0.09390413761138916,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48613083-G-T',\n",
       "      'distance': 0.0949036255478859,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48420718-G-A',\n",
       "      'distance': 0.09544514119625092,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520675-A-G',\n",
       "      'distance': 0.09611128270626068,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48456731-G-A',\n",
       "      'distance': 0.09614528715610504,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48460259-G-A',\n",
       "      'distance': 0.09647368639707565,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48470722-G-A',\n",
       "      'distance': 0.09650324285030365,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520690-G-A',\n",
       "      'distance': 0.0965232402086258,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48427700-G-A',\n",
       "      'distance': 0.09655862301588058,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48446797-G-A',\n",
       "      'distance': 0.09673897176980972,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48487142-G-A',\n",
       "      'distance': 0.09678709506988525,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48487157-G-A',\n",
       "      'distance': 0.09683477878570557,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48460250-G-A',\n",
       "      'distance': 0.09696684032678604,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48489981-G-A',\n",
       "      'distance': 0.09728844463825226,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48427613-G-A',\n",
       "      'distance': 0.09754467010498047,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48420778-G-A',\n",
       "      'distance': 0.09757937490940094,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48420763-G-A',\n",
       "      'distance': 0.09757989645004272,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48468509-G-A',\n",
       "      'distance': 0.09789544343948364,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48610768-G-A',\n",
       "      'distance': 0.09818580001592636,\n",
       "      'pathogenicity': 'benign_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_benign',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_benign', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_benign', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_benign', 'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48456750-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48456750\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48456750',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5309G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48456750C>A',\n",
       "     'hgvs_protein': 'p.Cys1770Phe',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48456750C>A\", \"most_severe_consequence\": \"missense_variant\", \"strand\": 1, \"start\": 48456750, \"seq_region_name\": \"15\", \"minimised\": 1, \"allele_string\": \"C/A\", \"end\": 48456750, \"vcf_string\": \"15-48456750-C-A\", \"colocated_variants\": [{\"strand\": 1, \"seq_region_name\": \"15\", \"allele_string\": \"HGMD_MUTATION\", \"end\": 48456750, \"phenotype_or_disease\": 1, \"start\": 48456750, \"id\": \"CM040041\"}, {\"id\": \"CM045613\", \"phenotype_or_disease\": 1, \"start\": 48456750, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"end\": 48456750}, {\"end\": 48456750, \"allele_string\": \"COSMIC_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"somatic\": 1, \"id\": \"COSV104413900\", \"var_synonyms\": {\"COSMIC\": [\"COSM9257963\"]}, \"phenotype_or_disease\": 1, \"start\": 48456750}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48456750C>A\", \"transcript_consequences\": [{\"allele_num\": 1, \"cadd_raw\": 5.334341, \"location\": \"15:48456750\", \"cadd_phred\": 30, \"eve_score\": 0.779267234570105, \"consequence_terms\": [\"missense_variant\"], \"cdna_end\": 5625, \"codons\": \"tGc/tTc\", \"spliceai\": {\"DS_AL\": 0, \"DP_DL\": -16, \"SYMBOL\": \"FBN1\", \"DP_AL\": 6, \"DS_AG\": 0, \"DP_DG\": -31, \"DS_DG\": 0, \"DS_DL\": 0, \"DP_AG\": 12}, \"cds_start\": 5309, \"protein_end\": 1770, \"alphamissense\": {\"am_pathogenicity\": 0.9987, \"am_class\": \"likely_pathogenic\"}, \"conservation\": 3.95, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"start\": 48456749, \"end\": 48456750}, \"sequence_id\": \"refseq:NC_000015.10\"}}, \"biotype\": \"protein_coding\", \"mane\": [\"MANE_Select\"], \"sift_prediction\": \"deleterious_low_confidence\", \"hgvsc\": \"NM_000138.5:c.5309G>T\", \"gene_id\": \"2200\", \"hgvsp\": \"NP_000129.3:p.Cys1770Phe\", \"aa\": \"C\", \"used_ref\": \"C\", \"blosum62\": -2, \"mane_select\": \"ENST00000316623.10\", \"impact\": \"MODERATE\", \"variant_allele\": \"A\", \"gene_symbol\": \"FBN1\", \"lrt_pred\": \"invalid_field\", \"given_ref\": \"C\", \"eve_class\": \"Pathogenic\", \"ptriplo\": 0.984699554922396, \"loeuf\": 0.105, \"cdna_start\": 5625, \"gene_symbol_source\": \"EntrezGene\", \"exon\": \"44/66\", \"phaplo\": 0.99897615427687, \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:C1770F\", \"clinpred\": 0.999862551689148, \"mutationtaster_pred\": \"D\", \"sift_score\": 0, \"mastermind_counts\": \"5|8|8\", \"polyphen_score\": 0.996, \"polyphen_prediction\": \"probably_damaging\", \"strand\": -1, \"protein_start\": 1770, \"transcript_id\": \"NM_000138.5\", \"amino_acids\": \"C/F\", \"protein_id\": \"NP_000129.3\", \"cds_end\": 5309}]}',\n",
       "    'coordinates': [{'pca_x': 0.0380782751700344, 'pca_y': 0.0867655114607894},\n",
       "     {'t-sne_x': 21.15723057416836, 't-sne_y': 13.103621262419228},\n",
       "     {'umap_x': 4.462730407714844, 'umap_y': 7.343771457672119}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.0014647669158875942,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.0015707183629274368,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.0016028997488319874,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.00169776426628232,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.0017202876042574644,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463904-C-A',\n",
       "      'distance': 0.0017492383485659957,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444579-C-A',\n",
       "      'distance': 0.0017935482319444418,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596318-C-A',\n",
       "      'distance': 0.0018767931032925844,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485463-C-A',\n",
       "      'distance': 0.0019204984419047832,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.0020038867369294167,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.0021388856694102287,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.0021992463152855635,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415695-C-A',\n",
       "      'distance': 0.002575051039457321,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537721-C-A',\n",
       "      'distance': 0.0026578265242278576,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452595-C-A',\n",
       "      'distance': 0.002702073659747839,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497358-C-A',\n",
       "      'distance': 0.0028986497782170773,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.002901432104408741,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.0029680519364774227,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.003081848379224539,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-T',\n",
       "      'distance': 0.003210896160453558,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48456725-TT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48456725\\t.\\tTT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48456725',\n",
       "     'ref_allele': 'TT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5336del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48456726del',\n",
       "     'hgvs_protein': 'p.Asn1779MetfsTer114',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"start\": 48456726, \"most_severe_consequence\": \"frameshift_variant\", \"strand\": 1, \"input\": \"NC_000015.10:g.48456726del\", \"end\": 48456726, \"vcf_string\": \"15-48456725-TT-T\", \"allele_string\": \"T/-\", \"minimised\": 1, \"transcript_consequences\": [{\"allele_num\": 1, \"spliceai\": {\"DP_DL\": -6, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0.07, \"DP_AL\": 37, \"DS_AG\": 0, \"DS_DG\": 0, \"DP_DG\": 37, \"DP_AG\": -30, \"DS_DL\": 0}, \"codons\": \"aAt/at\", \"cds_start\": 5336, \"cdna_end\": 5652, \"consequence_terms\": [\"frameshift_variant\"], \"hgvs_offset\": -3, \"location\": \"15:48456723\", \"biotype\": \"protein_coding\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48456726, \"start\": 48456725, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"protein_end\": 1779, \"conservation\": 2.76, \"aa\": \"T\", \"used_ref\": \"T\", \"hgvsp\": \"NP_000129.3:p.Asn1779MetfsTer114\", \"hgvsc\": \"NM_000138.5:c.5336del\", \"gene_id\": \"2200\", \"mane\": [\"MANE_Select\"], \"gene_symbol\": \"FBN1\", \"variant_allele\": \"-\", \"impact\": \"HIGH\", \"mane_select\": \"ENST00000316623.10\", \"lof_info\": \"PERCENTILE:0.619312906220984,GERP_DIST:3808.40074397326,BP_DIST:3261,DIST_FROM_LAST_EXON:2872,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol_source\": \"EntrezGene\", \"phaplo\": 0.99897615427687, \"cdna_start\": 5652, \"exon\": \"44/66\", \"loeuf\": 0.105, \"ptriplo\": 0.984699554922396, \"given_ref\": \"T\", \"strand\": -1, \"canonical\": 1, \"cds_end\": 5336, \"protein_id\": \"NP_000129.3\", \"amino_acids\": \"N/X\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 1779, \"lof\": \"HC\"}], \"id\": \"NC_000015.10:g.48456726del\", \"variant_class\": \"deletion\", \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': -0.0029159250632650457,\n",
       "      'pca_y': 0.008059878827301165},\n",
       "     {'t-sne_x': -13.100315344901983, 't-sne_y': -2.566425293860169},\n",
       "     {'umap_x': 0.5641616582870483, 'umap_y': 15.356921195983887}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.08718805015087128,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.08757346868515015,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.08884083479642868,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.08942575007677078,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.09029936790466309,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.09043769538402557,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.09048357605934143,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.09052262455224991,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.09058619290590286,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.09070590883493423,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.09127411246299744,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09203046560287476,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.0920332819223404,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.09211300313472748,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.09274166822433472,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.0933031365275383,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.09339109063148499,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.09348533302545547,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.09355919063091278,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.09368261694908142,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48456680-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48456680\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48456680',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5379T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48456680A>C',\n",
       "     'hgvs_protein': 'p.Cys1793Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"A/C\", \"strand\": 1, \"id\": \"NC_000015.10:g.48456680A>C\", \"input\": \"NC_000015.10:g.48456680A>C\", \"end\": 48456680, \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"used_ref\": \"A\", \"eve_score\": 0.721013592150157, \"cds_start\": 5379, \"cdna_start\": 5695, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.999919176101685, \"aa\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys1793Trp\", \"location\": \"15:48456680\", \"cdna_end\": 5695, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_raw\": 3.665539, \"allele_num\": 1, \"mastermind_counts\": \"5|8|8\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48456680, \"start\": 48456679}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"spliceai\": {\"DP_DL\": -43, \"DS_AG\": 0, \"DP_AL\": -5, \"DS_DL\": 0, \"DP_DG\": 1, \"DP_AG\": 1, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0}, \"given_ref\": \"A\", \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"loeuf\": 0.105, \"variant_allele\": \"C\", \"canonical\": 1, \"amino_acids\": \"C/W\", \"polyphen_score\": 0.997, \"mane_select\": \"ENST00000316623.10\", \"sift_prediction\": \"deleterious_low_confidence\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"lrt_pred\": \"invalid_field\", \"cds_end\": 5379, \"protein_start\": 1793, \"codons\": \"tgT/tgG\", \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"conservation\": 1.52, \"exon\": \"44/66\", \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"cadd_phred\": 23.2, \"sift_score\": 0, \"alphamissense\": {\"am_pathogenicity\": 0.9988, \"am_class\": \"likely_pathogenic\"}, \"hgvsc\": \"NM_000138.5:c.5379T>G\", \"eve_class\": \"Pathogenic\", \"mutationtaster_pred\": \"D\", \"blosum62\": -2, \"gene_id\": \"2200\", \"mastermind_mmid3\": \"FBN1:C1793W\", \"gene_symbol\": \"FBN1\", \"protein_end\": 1793}], \"start\": 48456680, \"colocated_variants\": [{\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48456680, \"id\": \"CM040042\", \"end\": 48456680, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1}, {\"seq_region_name\": \"15\", \"start\": 48456680, \"phenotype_or_disease\": 1, \"end\": 48456680, \"id\": \"COSV100354029\", \"allele_string\": \"COSMIC_MUTATION\", \"somatic\": 1, \"strand\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM8656775\"]}}], \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"variant_class\": \"SNV\", \"vcf_string\": \"15-48456680-A-C\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.03891064073373071,\n",
       "      'pca_y': 0.08384334261566315},\n",
       "     {'t-sne_x': 18.99682188555957, 't-sne_y': 4.300197265602723},\n",
       "     {'umap_x': 3.779634475708008, 'umap_y': 6.2589826583862305}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48445432-A-C',\n",
       "      'distance': 0.003029432613402605,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483822-A-C',\n",
       "      'distance': 0.0032371319830417633,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415573-A-C',\n",
       "      'distance': 0.0037646202836185694,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468522-C-A',\n",
       "      'distance': 0.0038244822062551975,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468457-A-C',\n",
       "      'distance': 0.0038475636392831802,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48436967-A-C',\n",
       "      'distance': 0.003902195254340768,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-C',\n",
       "      'distance': 0.003926772624254227,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465840-C-A',\n",
       "      'distance': 0.003946089185774326,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421989-A-C',\n",
       "      'distance': 0.0040281047113239765,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496104-A-C',\n",
       "      'distance': 0.004115925170481205,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520677-A-C',\n",
       "      'distance': 0.004142436198890209,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430749-A-C',\n",
       "      'distance': 0.004160280805081129,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430770-A-C',\n",
       "      'distance': 0.004299226216971874,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415716-T-C',\n",
       "      'distance': 0.0044157495722174644,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.004419905133545399,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437011-T-C',\n",
       "      'distance': 0.004425971303135157,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432944-A-G',\n",
       "      'distance': 0.004509614314883947,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463219-T-C',\n",
       "      'distance': 0.004583889618515968,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.004601635038852692,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.004667364060878754,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48446708-48452687-del-9af91777',\n",
       "    'metadata': {'vcf_string': '15\\t48446708\\t.\\tATACAGTCATTGTTGTGAGAAAGGATGAAACCATGATTGCAGCGGCAGTTGAAGGAACCAATTGTGTTCCGGCAAGTTCCATTCCCACAGGCATCTCTTTCACATTCATTTATGTCTAGTAGGAAGAAAGGCCATAAAGAAACATAATTATAAGTAGAAAAAGTGGTTACACGGTTACAGTGGCTAAAGAGCATTTTAGGGTTCACCAGGCCTCATCCATAGGCTGAGAACTTCTTGATGTTGGTATCAGCTCCAGCAGCAGTCATCCCTGCAAGATACCAGGCAGCGGCAAAGCTCTAAATCTTGGGAATAAATGGTTTCCTTGGCTGGCCAGTGAAAGGAGGCAAGCTAAGGACATGAGACCTGAACAAGAATTATTTTATTCTATTTACTTCTAGAATTCTCCACCCTTGCCACTTTTCTCAAATATTTTCACTTGAAAATGGAAAAAACTCTGTGTGCATCTGGGTCTTGAGGTGTTGCCACCTTTGGCAAACTTCAACCTTCAAGATTCTCACTCCAGTTTACCAAAAAGCAGTGACTCACCTGGCATTACCCCACAATCAGCAACAAAATAACTCAGCTTTCTAGCTCACCTGTCATTTCAGACAACTTTTCCTTTTTCCCTACACCCATAAAAACATAGAACAACTTCCACCAATGATGACTTCTCTCACTTTTCTAAACTACCCCAAAGAGATGGCACTTAATTTGTTCTTTTCATACATGCTTGCATTCTTATTCGCCTTTTGATTGGTCCGTACTTTTCTCTGATGATTTTGTAAAGCTACTCTGACATCTTATATGTCTTTGCATCTCATCACAGCAGCTAGCATATTTCTTATAAAGAGTAGATGCACATTATATTAAGGAGGAAGTAATAAAAAATTTAATGAAAGAAAGTTGTTACTGCTGGTTTGAAAGAATGAGGCACCATGAAGAGGGACAAGGATGAAATGAGAGAGAGAGAGAGTGGAAGGTAACAACGGAGAATAGCAAAATGTAATGATTAGAGCTAGGATTCCCTTTCTTTTCACATTTTGGACATGAAGATAAATGATCTGTGATTCCTGAATCAGGAAGCTATTGCTAACCCAAAGGGACTCTAATGTGTGTCTGTGGTTGAAGACTACACCCAGATGAGTCATTGCTAGTAATCAAGAATTTTTGCCTTGATTCATTTTTAGCCAAACTGGTGAAGTTCTTTGGTTGGGAAGGTTCTATGTTAGGGGGCTCCAGAAAATTCAGATGAGAGATGTAGAAAGAAAGTTGGGCAGGATCATCATAGCTTTTACACAGGTTAGCTGCCCAGTGATGAGTAGGGCTGTCAGTGCTCTTGTCATTACTCTGGAGAAGATACAACTTTATGAGAAATGTTCCCAAATGGATGAACCCAGTCATAGCCAAAGTCAAGGACTTAAACATTACTATATTAATTGTTCTTTCAGCTGTGAATCTGATTTATGTTCTTAATGTCCCAAAAAGACTTAATGGTATTGCTTTAAAGTAGAAAAATAAAGGAAGGAAAAGAAGAAAATGATTCATTACACACTGAAGTTTATTCTAAATCAGCATCCTGATAATAGAGAAATCAGGAAAGAGATTTAAAATCCTGAGCTCTTTTTCCTCTAGAAAAATGATTAACTTTTCTTTGCCTTTGTAGGATTTATAAATTTGCCTTTGAGAATCTGGTACACACTATAAACCCTTCCAAAAAAATGGTCATCTCACACTCAAAAACTCAAAAGAAAATCCACATCAATATTACATATGAATAATAGCCTATTAACATTAATATTTTTGTATTAATATCTTTTTCAATTTACCCCTTAGTCAATTTTTTCTAAACTGACAGCCAAATGAATAACAAAGGAAGAGGTTTATCATGTTCAGATTGCCAAAGATTAAATAGGATATCACTGCTGCATATCTGTCTGTGTTTTTATTTTGTATATAGCAAAAATACTACTAAAAGACTTAGTATTAAATTTTATCCATATTTAGAATCAAATGAAGCTTTCAACAGCATATGAAAAAAATAATAATAATTGCATACTTACCCAAGCACATGGTTTGGTCATCATTTGTTTTAAAACCAGTGTGGCAAAGGCAATAAAAGCTTCCAACTGTGTCAATGCACTGCCCATGACTGCATATATTGGGGATTTCTTGACATTCATTACGATCTGTAAATAAGAAGCATCTTAAGTGAGAACTTAGAAGACAAAATATAATTGAATAACTTACTTCTAGCTATCATTCTCAGGAGTAATCCTAGCTCTAAACTAAGTTAGTGAAACAGATATATTTATTTTTGCATCAAAATGGATTAAAACTTACTCTCATTTACCAGGCATAGTCAGAGTATGTTAACAATACTTGACACAATAAAAAATAAGGGTAAGTAATCAAAAAATATTCTGAAAGATGAGAAAACTTAGAAGTAAAATGGGGGGATTCAAGCCGGGAGGATTTTCTGTTTGATTATCGGCCTACGCAGCACCACAGACTGTGGAATTAATTGAAGTATTTTGTCTAGTTAAGATTTGGTGCCTTAAGTCATGGATAGTTCATGGCTTCCTTTGTAAATTTATATCAAAGTCAAATCCTATGACTAAGATTAAGTTTTTCTTGATGGACAACCTACATGGCTGCTTGTTCAAGTTGCATCTCTTTGTTTTGTCTTATGCCTGTAACTTTTTGTTCCATCTTCACCAAGTATTCACGTCTACACAGGGCTCCTTTTAGCATGTGACAACCTGAACACTGTCTCTTTCTTTTTACTCTCATCCATTCAAAAGGAAGCCCTCTCATCTTGCTTCTGCCAGTCTTTAAGTTCTAAATTATATTTATCATCAACTGAATCTTTGTTGCTTTCCTGAAATCTGATTATCTGAGGGGCAGAATTTAATGCATGAATTCACATATGACATAATTATATGCCAACTTACGTCTCTAATTTTTTAAAGCTCTATGTTTAGAGTAAATGTACTTTGAAGAGCACTGTGCCATTCTGATTTTCCTACTGTCACAGAAGAAAGCAAATGGAGTGTCATCTTTGCTGCAGATGCTGTAGGGCAGATGGCATTTCGCTCTTAGTTAAATAATCCAGACTTGCCGTTTGCCTGCAATGTCTACTCAGAACACAGTAATCATGAATGGACACTTTTTGCTGCACACAGCAACCCTCCCTGTGCCAGAATCATGGACATCTCCTCCATTTCATGTGTCCGGGATGCCACTGTTATGGTTATCTCCTATTTAGTAGATGAATCATCATACCATTTAGAAGTTCCAAGTGAAGTGGAGTATATCATTATGAGGGGAAATTGAAATAGAGCAGCTTAGGAACTACTACTAGCCTATGCAGAAATGCTATTTAACAAATTATGATTATTCCCTTTTGAGAAAATGAATTTAAAGTGAGCCAACTGCTTCCCAGTATAGGCAGAGGAGACTGATTTCCGATCTTGAGGAAGGGTGTCTCAGGTCCTACCATAGTTGGCCCAAAACTTTGGCACCTTCAGGTAGGACTGGACCTGGAGGTATGCAGGGGACGAGGTCTGTCTCATATCTGAGTTCCACAAGGAAATGGAGAGCCTTAAGGATCTTATTAGAAAGGGATCCCCAATTAGGAATTTCCTTTCATCTACTGCCCTATTTCATGATGGTTTAAATAAATGAAAAATGAAACATGGGGAGTACTTTGGAGTTCTACTGGTAAAACTAGAGCAGAAATCTAGGTTTCCACAAGCACCAGAATTTGAACTCTTAGATGATCATATTTGAGGTTGGGATAGGAGGTCTGAACAGGCTACTGTGGAAAGAAACCCAAATAGTCAACAGAGAAAGAGAGGAAACCACAAGGAAGTAGCTGGGAAGAAACTGTTCCTTGTCTTTTGTGGGGCAAGACATTTCAGAAGGGGGTTGCATTTGTTCTAACTATCCAGCTTCTGCTGTCAGACTAGCATCCTGAGCTCAGATATCTAATTATTCTCTCAGCTCTCTCAGCAATAATGTGGGTGCTATGGCCAGGTGTGATTTGGGATTTGAAATAATGTGCTTATGAATTCAAATAAGGATCTCATGATGGAGAGAAAGTTTTAGTTATTTTCATTTTAAACAGATGCACAAAGGAGTACTTTGCAGCATGCCTGAAAAAAAAAATGCCAAGTATAGAATAGCATTTAAAGATATTTTAAAAGTAAGACATAAATAAAGACATTTCTAGAATGAGGTCTTCAACCCAACAGGCTGAATGGTTGCAAAACTCAAGTTAATTCCCTTTCTTCTGAGGATTTCAATGAATAAAGGAAATCAAGCTTTTGGCTGCAAATGATGCTCACTCACAACACTTATGGCTGGCTGCATTTTATGTATGATAGTCTCCTACAAGAAAACAGTAAATGACTTAAGCTCCAGCAGGCTATGGCTGAATGTCTTATCTTGATCTATAAATAGCTCTGTCCCACTTCTTACCTGAGTGTGAAATGGTGTGAAATGACTCATTCTATATTATCAGCAATTCTTTATTTTGACTGGAATTATTTCATCCAGTTCTAAAGAAATATAAATTATGAGCAAAATGCAGGGAAGTGGTATCAATGCCAAGTTTCTCTAGGGTATTGTGGTTCATATCAGAGACGTCCCTGCTTTTCCTGTCCTGTATTTTTAATACATGGATAGCAACAGCTCTGTTGACAAAATGCTAAATACTTACACATTTTCCTTTAAACTCAACAAAACAAGTCTTGATTCTCAATGCGATTTTTTTGTTTGGCAATTACACCCTCTTCCCACCAAGGTCTCATAAAACAATGCAAAATACCCTAAAGAGAACAGTGCACTTCCTACCACTTAATCACTGGTAGGCAAGGAAATCCAAGGAAAATCTGACCACGTTCCCCTGTGTTGTTTCATTCACAAATGCATAAAATATCCAGGAAACCCCCCGTGAAATGAACCGTGAGCAAAATAACACATTCTTCCTGACTAATATACAATGATTTGCTTTGCTAAATATACAGTGATTTTTGCTAAATATACAGATTCAGAATTACTTAAAAACTTACAAGAAAATCATTCTGAGATATTATTTTGTTTCTGGGCTGAAACAGCAGTCTTTTAAATTTTCACACCACTGGCTACAAGATACCATCAGATACTGAATAACTTCACCGTACACTGAAGTTGATAACATGCCAACGTGCTTCCTATTTCTATCACATCCCTGACAAAAGTGAACACCAAAAAAATCTAATCTCTAGGGGTCATGGGAATAAATAGCTAATAAAGGCTGTTGATGTTTGCTGCCTTCAGTGAGTGACCCTTTCCCTGTAAGATGACTGGGAAGAAAAGCGATGGCTTTGTTTGTTCTGTTTCTGCAGATTAATCTGACCCAGCTCAGGAGAGCTTCACTGTCTAGATTTGGGGTATGCAGCCAAGATGGTGAAAACCTCACCTTTCTGAGAGGGATTCTGTGGAAGGAAAAAAATCATTTGGTCCTAAATGTTGCTTAGGGGAATGTGTGTATGCATAATTGTGTGTGTGTGTAATGTGTGGGGGTGAGTGTTTGCAGGCAGGGCAGCAGACAGGCAAATACGAAATCTATGTATGCGAATGATTTACTTCTTGTTTTGAAAATATCTCATCCAGAAAGAGGATTAAAAAGCTAAGTGTTTAGATATTTTGACAATATCCCTTTTTCCAATTTAGTTTAATATTAGAGTTCATCAATCTAAATCTTAACTCATATATTCTTCAGCTTAACTATCTGAAAATAAATCCAGATATCTGAAGCTTCATGAAGACAAACTCTTGGGTAGGCATGTCCAGCCTGTGGGGCACTACATACCATTGCACTGTCCTGTGGAGGTGAAGCGGTAGCCGGGCTTACAGTCACAGCGGTAGCTGCCTGCAGTGTTGATGCATTCGGCGTTGCGCTGGCACACTGGGCCGTTCTGACACTCGTCAATATCTA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48446708',\n",
       "     'ref_allele': 'ATACAGTCATTGTTGTGAGAAAGGATGAAACCATGATTGCAGCGGCAGTTGAAGGAACCAATTGTGTTCCGGCAAGTTCCATTCCCACAGGCATCTCTTTCACATTCATTTATGTCTAGTAGGAAGAAAGGCCATAAAGAAACATAATTATAAGTAGAAAAAGTGGTTACACGGTTACAGTGGCTAAAGAGCATTTTAGGGTTCACCAGGCCTCATCCATAGGCTGAGAACTTCTTGATGTTGGTATCAGCTCCAGCAGCAGTCATCCCTGCAAGATACCAGGCAGCGGCAAAGCTCTAAATCTTGGGAATAAATGGTTTCCTTGGCTGGCCAGTGAAAGGAGGCAAGCTAAGGACATGAGACCTGAACAAGAATTATTTTATTCTATTTACTTCTAGAATTCTCCACCCTTGCCACTTTTCTCAAATATTTTCACTTGAAAATGGAAAAAACTCTGTGTGCATCTGGGTCTTGAGGTGTTGCCACCTTTGGCAAACTTCAACCTTCAAGATTCTCACTCCAGTTTACCAAAAAGCAGTGACTCACCTGGCATTACCCCACAATCAGCAACAAAATAACTCAGCTTTCTAGCTCACCTGTCATTTCAGACAACTTTTCCTTTTTCCCTACACCCATAAAAACATAGAACAACTTCCACCAATGATGACTTCTCTCACTTTTCTAAACTACCCCAAAGAGATGGCACTTAATTTGTTCTTTTCATACATGCTTGCATTCTTATTCGCCTTTTGATTGGTCCGTACTTTTCTCTGATGATTTTGTAAAGCTACTCTGACATCTTATATGTCTTTGCATCTCATCACAGCAGCTAGCATATTTCTTATAAAGAGTAGATGCACATTATATTAAGGAGGAAGTAATAAAAAATTTAATGAAAGAAAGTTGTTACTGCTGGTTTGAAAGAATGAGGCACCATGAAGAGGGACAAGGATGAAATGAGAGAGAGAGAGAGTGGAAGGTAACAACGGAGAATAGCAAAATGTAATGATTAGAGCTAGGATTCCCTTTCTTTTCACATTTTGGACATGAAGATAAATGATCTGTGATTCCTGAATCAGGAAGCTATTGCTAACCCAAAGGGACTCTAATGTGTGTCTGTGGTTGAAGACTACACCCAGATGAGTCATTGCTAGTAATCAAGAATTTTTGCCTTGATTCATTTTTAGCCAAACTGGTGAAGTTCTTTGGTTGGGAAGGTTCTATGTTAGGGGGCTCCAGAAAATTCAGATGAGAGATGTAGAAAGAAAGTTGGGCAGGATCATCATAGCTTTTACACAGGTTAGCTGCCCAGTGATGAGTAGGGCTGTCAGTGCTCTTGTCATTACTCTGGAGAAGATACAACTTTATGAGAAATGTTCCCAAATGGATGAACCCAGTCATAGCCAAAGTCAAGGACTTAAACATTACTATATTAATTGTTCTTTCAGCTGTGAATCTGATTTATGTTCTTAATGTCCCAAAAAGACTTAATGGTATTGCTTTAAAGTAGAAAAATAAAGGAAGGAAAAGAAGAAAATGATTCATTACACACTGAAGTTTATTCTAAATCAGCATCCTGATAATAGAGAAATCAGGAAAGAGATTTAAAATCCTGAGCTCTTTTTCCTCTAGAAAAATGATTAACTTTTCTTTGCCTTTGTAGGATTTATAAATTTGCCTTTGAGAATCTGGTACACACTATAAACCCTTCCAAAAAAATGGTCATCTCACACTCAAAAACTCAAAAGAAAATCCACATCAATATTACATATGAATAATAGCCTATTAACATTAATATTTTTGTATTAATATCTTTTTCAATTTACCCCTTAGTCAATTTTTTCTAAACTGACAGCCAAATGAATAACAAAGGAAGAGGTTTATCATGTTCAGATTGCCAAAGATTAAATAGGATATCACTGCTGCATATCTGTCTGTGTTTTTATTTTGTATATAGCAAAAATACTACTAAAAGACTTAGTATTAAATTTTATCCATATTTAGAATCAAATGAAGCTTTCAACAGCATATGAAAAAAATAATAATAATTGCATACTTACCCAAGCACATGGTTTGGTCATCATTTGTTTTAAAACCAGTGTGGCAAAGGCAATAAAAGCTTCCAACTGTGTCAATGCACTGCCCATGACTGCATATATTGGGGATTTCTTGACATTCATTACGATCTGTAAATAAGAAGCATCTTAAGTGAGAACTTAGAAGACAAAATATAATTGAATAACTTACTTCTAGCTATCATTCTCAGGAGTAATCCTAGCTCTAAACTAAGTTAGTGAAACAGATATATTTATTTTTGCATCAAAATGGATTAAAACTTACTCTCATTTACCAGGCATAGTCAGAGTATGTTAACAATACTTGACACAATAAAAAATAAGGGTAAGTAATCAAAAAATATTCTGAAAGATGAGAAAACTTAGAAGTAAAATGGGGGGATTCAAGCCGGGAGGATTTTCTGTTTGATTATCGGCCTACGCAGCACCACAGACTGTGGAATTAATTGAAGTATTTTGTCTAGTTAAGATTTGGTGCCTTAAGTCATGGATAGTTCATGGCTTCCTTTGTAAATTTATATCAAAGTCAAATCCTATGACTAAGATTAAGTTTTTCTTGATGGACAACCTACATGGCTGCTTGTTCAAGTTGCATCTCTTTGTTTTGTCTTATGCCTGTAACTTTTTGTTCCATCTTCACCAAGTATTCACGTCTACACAGGGCTCCTTTTAGCATGTGACAACCTGAACACTGTCTCTTTCTTTTTACTCTCATCCATTCAAAAGGAAGCCCTCTCATCTTGCTTCTGCCAGTCTTTAAGTTCTAAATTATATTTATCATCAACTGAATCTTTGTTGCTTTCCTGAAATCTGATTATCTGAGGGGCAGAATTTAATGCATGAATTCACATATGACATAATTATATGCCAACTTACGTCTCTAATTTTTTAAAGCTCTATGTTTAGAGTAAATGTACTTTGAAGAGCACTGTGCCATTCTGATTTTCCTACTGTCACAGAAGAAAGCAAATGGAGTGTCATCTTTGCTGCAGATGCTGTAGGGCAGATGGCATTTCGCTCTTAGTTAAATAATCCAGACTTGCCGTTTGCCTGCAATGTCTACTCAGAACACAGTAATCATGAATGGACACTTTTTGCTGCACACAGCAACCCTCCCTGTGCCAGAATCATGGACATCTCCTCCATTTCATGTGTCCGGGATGCCACTGTTATGGTTATCTCCTATTTAGTAGATGAATCATCATACCATTTAGAAGTTCCAAGTGAAGTGGAGTATATCATTATGAGGGGAAATTGAAATAGAGCAGCTTAGGAACTACTACTAGCCTATGCAGAAATGCTATTTAACAAATTATGATTATTCCCTTTTGAGAAAATGAATTTAAAGTGAGCCAACTGCTTCCCAGTATAGGCAGAGGAGACTGATTTCCGATCTTGAGGAAGGGTGTCTCAGGTCCTACCATAGTTGGCCCAAAACTTTGGCACCTTCAGGTAGGACTGGACCTGGAGGTATGCAGGGGACGAGGTCTGTCTCATATCTGAGTTCCACAAGGAAATGGAGAGCCTTAAGGATCTTATTAGAAAGGGATCCCCAATTAGGAATTTCCTTTCATCTACTGCCCTATTTCATGATGGTTTAAATAAATGAAAAATGAAACATGGGGAGTACTTTGGAGTTCTACTGGTAAAACTAGAGCAGAAATCTAGGTTTCCACAAGCACCAGAATTTGAACTCTTAGATGATCATATTTGAGGTTGGGATAGGAGGTCTGAACAGGCTACTGTGGAAAGAAACCCAAATAGTCAACAGAGAAAGAGAGGAAACCACAAGGAAGTAGCTGGGAAGAAACTGTTCCTTGTCTTTTGTGGGGCAAGACATTTCAGAAGGGGGTTGCATTTGTTCTAACTATCCAGCTTCTGCTGTCAGACTAGCATCCTGAGCTCAGATATCTAATTATTCTCTCAGCTCTCTCAGCAATAATGTGGGTGCTATGGCCAGGTGTGATTTGGGATTTGAAATAATGTGCTTATGAATTCAAATAAGGATCTCATGATGGAGAGAAAGTTTTAGTTATTTTCATTTTAAACAGATGCACAAAGGAGTACTTTGCAGCATGCCTGAAAAAAAAAATGCCAAGTATAGAATAGCATTTAAAGATATTTTAAAAGTAAGACATAAATAAAGACATTTCTAGAATGAGGTCTTCAACCCAACAGGCTGAATGGTTGCAAAACTCAAGTTAATTCCCTTTCTTCTGAGGATTTCAATGAATAAAGGAAATCAAGCTTTTGGCTGCAAATGATGCTCACTCACAACACTTATGGCTGGCTGCATTTTATGTATGATAGTCTCCTACAAGAAAACAGTAAATGACTTAAGCTCCAGCAGGCTATGGCTGAATGTCTTATCTTGATCTATAAATAGCTCTGTCCCACTTCTTACCTGAGTGTGAAATGGTGTGAAATGACTCATTCTATATTATCAGCAATTCTTTATTTTGACTGGAATTATTTCATCCAGTTCTAAAGAAATATAAATTATGAGCAAAATGCAGGGAAGTGGTATCAATGCCAAGTTTCTCTAGGGTATTGTGGTTCATATCAGAGACGTCCCTGCTTTTCCTGTCCTGTATTTTTAATACATGGATAGCAACAGCTCTGTTGACAAAATGCTAAATACTTACACATTTTCCTTTAAACTCAACAAAACAAGTCTTGATTCTCAATGCGATTTTTTTGTTTGGCAATTACACCCTCTTCCCACCAAGGTCTCATAAAACAATGCAAAATACCCTAAAGAGAACAGTGCACTTCCTACCACTTAATCACTGGTAGGCAAGGAAATCCAAGGAAAATCTGACCACGTTCCCCTGTGTTGTTTCATTCACAAATGCATAAAATATCCAGGAAACCCCCCGTGAAATGAACCGTGAGCAAAATAACACATTCTTCCTGACTAATATACAATGATTTGCTTTGCTAAATATACAGTGATTTTTGCTAAATATACAGATTCAGAATTACTTAAAAACTTACAAGAAAATCATTCTGAGATATTATTTTGTTTCTGGGCTGAAACAGCAGTCTTTTAAATTTTCACACCACTGGCTACAAGATACCATCAGATACTGAATAACTTCACCGTACACTGAAGTTGATAACATGCCAACGTGCTTCCTATTTCTATCACATCCCTGACAAAAGTGAACACCAAAAAAATCTAATCTCTAGGGGTCATGGGAATAAATAGCTAATAAAGGCTGTTGATGTTTGCTGCCTTCAGTGAGTGACCCTTTCCCTGTAAGATGACTGGGAAGAAAAGCGATGGCTTTGTTTGTTCTGTTTCTGCAGATTAATCTGACCCAGCTCAGGAGAGCTTCACTGTCTAGATTTGGGGTATGCAGCCAAGATGGTGAAAACCTCACCTTTCTGAGAGGGATTCTGTGGAAGGAAAAAAATCATTTGGTCCTAAATGTTGCTTAGGGGAATGTGTGTATGCATAATTGTGTGTGTGTGTAATGTGTGGGGGTGAGTGTTTGCAGGCAGGGCAGCAGACAGGCAAATACGAAATCTATGTATGCGAATGATTTACTTCTTGTTTTGAAAATATCTCATCCAGAAAGAGGATTAAAAAGCTAAGTGTTTAGATATTTTGACAATATCCCTTTTTCCAATTTAGTTTAATATTAGAGTTCATCAATCTAAATCTTAACTCATATATTCTTCAGCTTAACTATCTGAAAATAAATCCAGATATCTGAAGCTTCATGAAGACAAACTCTTGGGTAGGCATGTCCAGCCTGTGGGGCACTACATACCATTGCACTGTCCTGTGGAGGTGAAGCGGTAGCCGGGCTTACAGTCACAGCGGTAGCTGCCTGCAGTGTTGATGCATTCGGCGTTGCGCTGGCACACTGGGCCGTTCTGACACTCGTCAATATCTA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5423_5788del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48446709_48452687del',\n",
       "     'hgvs_protein': 'p.Ile1809_Asp1930del',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"deletion\", \"most_severe_consequence\": \"splice_acceptor_variant\", \"seq_region_name\": \"15\", \"minimised\": 1, \"vcf_string\": \"15-48446708-ATACAGTCATTGTTGTGAGAAAGGATGAAACCATGATTGCAGCGGCAGTTGAAGGAACCAATTGTGTTCCGGCAAGTTCCATTCCCACAGGCATCTCTTTCACATTCATTTATGTCTAGTAGGAAGAAAGGCCATAAAGAAACATAATTATAAGTAGAAAAAGTGGTTACACGGTTACAGTGGCTAAAGAGCATTTTAGGGTTCACCAGGCCTCATCCATAGGCTGAGAACTTCTTGATGTTGGTATCAGCTCCAGCAGCAGTCATCCCTGCAAGATACCAGGCAGCGGCAAAGCTCTAAATCTTGGGAATAAATGGTTTCCTTGGCTGGCCAGTGAAAGGAGGCAAGCTAAGGACATGAGACCTGAACAAGAATTATTTTATTCTATTTACTTCTAGAATTCTCCACCCTTGCCACTTTTCTCAAATATTTTCACTTGAAAATGGAAAAAACTCTGTGTGCATCTGGGTCTTGAGGTGTTGCCACCTTTGGCAAACTTCAACCTTCAAGATTCTCACTCCAGTTTACCAAAAAGCAGTGACTCACCTGGCATTACCCCACAATCAGCAACAAAATAACTCAGCTTTCTAGCTCACCTGTCATTTCAGACAACTTTTCCTTTTTCCCTACACCCATAAAAACATAGAACAACTTCCACCAATGATGACTTCTCTCACTTTTCTAAACTACCCCAAAGAGATGGCACTTAATTTGTTCTTTTCATACATGCTTGCATTCTTATTCGCCTTTTGATTGGTCCGTACTTTTCTCTGATGATTTTGTAAAGCTACTCTGACATCTTATATGTCTTTGCATCTCATCACAGCAGCTAGCATATTTCTTATAAAGAGTAGATGCACATTATATTAAGGAGGAAGTAATAAAAAATTTAATGAAAGAAAGTTGTTACTGCTGGTTTGAAAGAATGAGGCACCATGAAGAGGGACAAGGATGAAATGAGAGAGAGAGAGAGTGGAAGGTAACAACGGAGAATAGCAAAATGTAATGATTAGAGCTAGGATTCCCTTTCTTTTCACATTTTGGACATGAAGATAAATGATCTGTGATTCCTGAATCAGGAAGCTATTGCTAACCCAAAGGGACTCTAATGTGTGTCTGTGGTTGAAGACTACACCCAGATGAGTCATTGCTAGTAATCAAGAATTTTTGCCTTGATTCATTTTTAGCCAAACTGGTGAAGTTCTTTGGTTGGGAAGGTTCTATGTTAGGGGGCTCCAGAAAATTCAGATGAGAGATGTAGAAAGAAAGTTGGGCAGGATCATCATAGCTTTTACACAGGTTAGCTGCCCAGTGATGAGTAGGGCTGTCAGTGCTCTTGTCATTACTCTGGAGAAGATACAACTTTATGAGAAATGTTCCCAAATGGATGAACCCAGTCATAGCCAAAGTCAAGGACTTAAACATTACTATATTAATTGTTCTTTCAGCTGTGAATCTGATTTATGTTCTTAATGTCCCAAAAAGACTTAATGGTATTGCTTTAAAGTAGAAAAATAAAGGAAGGAAAAGAAGAAAATGATTCATTACACACTGAAGTTTATTCTAAATCAGCATCCTGATAATAGAGAAATCAGGAAAGAGATTTAAAATCCTGAGCTCTTTTTCCTCTAGAAAAATGATTAACTTTTCTTTGCCTTTGTAGGATTTATAAATTTGCCTTTGAGAATCTGGTACACACTATAAACCCTTCCAAAAAAATGGTCATCTCACACTCAAAAACTCAAAAGAAAATCCACATCAATATTACATATGAATAATAGCCTATTAACATTAATATTTTTGTATTAATATCTTTTTCAATTTACCCCTTAGTCAATTTTTTCTAAACTGACAGCCAAATGAATAACAAAGGAAGAGGTTTATCATGTTCAGATTGCCAAAGATTAAATAGGATATCACTGCTGCATATCTGTCTGTGTTTTTATTTTGTATATAGCAAAAATACTACTAAAAGACTTAGTATTAAATTTTATCCATATTTAGAATCAAATGAAGCTTTCAACAGCATATGAAAAAAATAATAATAATTGCATACTTACCCAAGCACATGGTTTGGTCATCATTTGTTTTAAAACCAGTGTGGCAAAGGCAATAAAAGCTTCCAACTGTGTCAATGCACTGCCCATGACTGCATATATTGGGGATTTCTTGACATTCATTACGATCTGTAAATAAGAAGCATCTTAAGTGAGAACTTAGAAGACAAAATATAATTGAATAACTTACTTCTAGCTATCATTCTCAGGAGTAATCCTAGCTCTAAACTAAGTTAGTGAAACAGATATATTTATTTTTGCATCAAAATGGATTAAAACTTACTCTCATTTACCAGGCATAGTCAGAGTATGTTAACAATACTTGACACAATAAAAAATAAGGGTAAGTAATCAAAAAATATTCTGAAAGATGAGAAAACTTAGAAGTAAAATGGGGGGATTCAAGCCGGGAGGATTTTCTGTTTGATTATCGGCCTACGCAGCACCACAGACTGTGGAATTAATTGAAGTATTTTGTCTAGTTAAGATTTGGTGCCTTAAGTCATGGATAGTTCATGGCTTCCTTTGTAAATTTATATCAAAGTCAAATCCTATGACTAAGATTAAGTTTTTCTTGATGGACAACCTACATGGCTGCTTGTTCAAGTTGCATCTCTTTGTTTTGTCTTATGCCTGTAACTTTTTGTTCCATCTTCACCAAGTATTCACGTCTACACAGGGCTCCTTTTAGCATGTGACAACCTGAACACTGTCTCTTTCTTTTTACTCTCATCCATTCAAAAGGAAGCCCTCTCATCTTGCTTCTGCCAGTCTTTAAGTTCTAAATTATATTTATCATCAACTGAATCTTTGTTGCTTTCCTGAAATCTGATTATCTGAGGGGCAGAATTTAATGCATGAATTCACATATGACATAATTATATGCCAACTTACGTCTCTAATTTTTTAAAGCTCTATGTTTAGAGTAAATGTACTTTGAAGAGCACTGTGCCATTCTGATTTTCCTACTGTCACAGAAGAAAGCAAATGGAGTGTCATCTTTGCTGCAGATGCTGTAGGGCAGATGGCATTTCGCTCTTAGTTAAATAATCCAGACTTGCCGTTTGCCTGCAATGTCTACTCAGAACACAGTAATCATGAATGGACACTTTTTGCTGCACACAGCAACCCTCCCTGTGCCAGAATCATGGACATCTCCTCCATTTCATGTGTCCGGGATGCCACTGTTATGGTTATCTCCTATTTAGTAGATGAATCATCATACCATTTAGAAGTTCCAAGTGAAGTGGAGTATATCATTATGAGGGGAAATTGAAATAGAGCAGCTTAGGAACTACTACTAGCCTATGCAGAAATGCTATTTAACAAATTATGATTATTCCCTTTTGAGAAAATGAATTTAAAGTGAGCCAACTGCTTCCCAGTATAGGCAGAGGAGACTGATTTCCGATCTTGAGGAAGGGTGTCTCAGGTCCTACCATAGTTGGCCCAAAACTTTGGCACCTTCAGGTAGGACTGGACCTGGAGGTATGCAGGGGACGAGGTCTGTCTCATATCTGAGTTCCACAAGGAAATGGAGAGCCTTAAGGATCTTATTAGAAAGGGATCCCCAATTAGGAATTTCCTTTCATCTACTGCCCTATTTCATGATGGTTTAAATAAATGAAAAATGAAACATGGGGAGTACTTTGGAGTTCTACTGGTAAAACTAGAGCAGAAATCTAGGTTTCCACAAGCACCAGAATTTGAACTCTTAGATGATCATATTTGAGGTTGGGATAGGAGGTCTGAACAGGCTACTGTGGAAAGAAACCCAAATAGTCAACAGAGAAAGAGAGGAAACCACAAGGAAGTAGCTGGGAAGAAACTGTTCCTTGTCTTTTGTGGGGCAAGACATTTCAGAAGGGGGTTGCATTTGTTCTAACTATCCAGCTTCTGCTGTCAGACTAGCATCCTGAGCTCAGATATCTAATTATTCTCTCAGCTCTCTCAGCAATAATGTGGGTGCTATGGCCAGGTGTGATTTGGGATTTGAAATAATGTGCTTATGAATTCAAATAAGGATCTCATGATGGAGAGAAAGTTTTAGTTATTTTCATTTTAAACAGATGCACAAAGGAGTACTTTGCAGCATGCCTGAAAAAAAAAATGCCAAGTATAGAATAGCATTTAAAGATATTTTAAAAGTAAGACATAAATAAAGACATTTCTAGAATGAGGTCTTCAACCCAACAGGCTGAATGGTTGCAAAACTCAAGTTAATTCCCTTTCTTCTGAGGATTTCAATGAATAAAGGAAATCAAGCTTTTGGCTGCAAATGATGCTCACTCACAACACTTATGGCTGGCTGCATTTTATGTATGATAGTCTCCTACAAGAAAACAGTAAATGACTTAAGCTCCAGCAGGCTATGGCTGAATGTCTTATCTTGATCTATAAATAGCTCTGTCCCACTTCTTACCTGAGTGTGAAATGGTGTGAAATGACTCATTCTATATTATCAGCAATTCTTTATTTTGACTGGAATTATTTCATCCAGTTCTAAAGAAATATAAATTATGAGCAAAATGCAGGGAAGTGGTATCAATGCCAAGTTTCTCTAGGGTATTGTGGTTCATATCAGAGACGTCCCTGCTTTTCCTGTCCTGTATTTTTAATACATGGATAGCAACAGCTCTGTTGACAAAATGCTAAATACTTACACATTTTCCTTTAAACTCAACAAAACAAGTCTTGATTCTCAATGCGATTTTTTTGTTTGGCAATTACACCCTCTTCCCACCAAGGTCTCATAAAACAATGCAAAATACCCTAAAGAGAACAGTGCACTTCCTACCACTTAATCACTGGTAGGCAAGGAAATCCAAGGAAAATCTGACCACGTTCCCCTGTGTTGTTTCATTCACAAATGCATAAAATATCCAGGAAACCCCCCGTGAAATGAACCGTGAGCAAAATAACACATTCTTCCTGACTAATATACAATGATTTGCTTTGCTAAATATACAGTGATTTTTGCTAAATATACAGATTCAGAATTACTTAAAAACTTACAAGAAAATCATTCTGAGATATTATTTTGTTTCTGGGCTGAAACAGCAGTCTTTTAAATTTTCACACCACTGGCTACAAGATACCATCAGATACTGAATAACTTCACCGTACACTGAAGTTGATAACATGCCAACGTGCTTCCTATTTCTATCACATCCCTGACAAAAGTGAACACCAAAAAAATCTAATCTCTAGGGGTCATGGGAATAAATAGCTAATAAAGGCTGTTGATGTTTGCTGCCTTCAGTGAGTGACCCTTTCCCTGTAAGATGACTGGGAAGAAAAGCGATGGCTTTGTTTGTTCTGTTTCTGCAGATTAATCTGACCCAGCTCAGGAGAGCTTCACTGTCTAGATTTGGGGTATGCAGCCAAGATGGTGAAAACCTCACCTTTCTGAGAGGGATTCTGTGGAAGGAAAAAAATCATTTGGTCCTAAATGTTGCTTAGGGGAATGTGTGTATGCATAATTGTGTGTGTGTGTAATGTGTGGGGGTGAGTGTTTGCAGGCAGGGCAGCAGACAGGCAAATACGAAATCTATGTATGCGAATGATTTACTTCTTGTTTTGAAAATATCTCATCCAGAAAGAGGATTAAAAAGCTAAGTGTTTAGATATTTTGACAATATCCCTTTTTCCAATTTAGTTTAATATTAGAGTTCATCAATCTAAATCTTAACTCATATATTCTTCAGCTTAACTATCTGAAAATAAATCCAGATATCTGAAGCTTCATGAAGACAAACTCTTGGGTAGGCATGTCCAGCCTGTGGGGCACTACATACCATTGCACTGTCCTGTGGAGGTGAAGCGGTAGCCGGGCTTACAGTCACAGCGGTAGCTGCCTGCAGTGTTGATGCATTCGGCGTTGCGCTGGCACACTGGGCCGTTCTGACACTCGTCAATATCTA-A\", \"id\": \"NC_000015.10:g.48446709_48452687del\", \"end\": 48452687, \"input\": \"NC_000015.10:g.48446709_48452687del\", \"strand\": 1, \"allele_string\": \"TACAGTCATTGTTGTGAGAAAGGATGAAACCATGATTGCAGCGGCAGTTGAAGGAACCAATTGTGTTCCGGCAAGTTCCATTCCCACAGGCATCTCTTTCACATTCATTTATGTCTAGTAGGAAGAAAGGCCATAAAGAAACATAATTATAAGTAGAAAAAGTGGTTACACGGTTACAGTGGCTAAAGAGCATTTTAGGGTTCACCAGGCCTCATCCATAGGCTGAGAACTTCTTGATGTTGGTATCAGCTCCAGCAGCAGTCATCCCTGCAAGATACCAGGCAGCGGCAAAGCTCTAAATCTTGGGAATAAATGGTTTCCTTGGCTGGCCAGTGAAAGGAGGCAAGCTAAGGACATGAGACCTGAACAAGAATTATTTTATTCTATTTACTTCTAGAATTCTCCACCCTTGCCACTTTTCTCAAATATTTTCACTTGAAAATGGAAAAAACTCTGTGTGCATCTGGGTCTTGAGGTGTTGCCACCTTTGGCAAACTTCAACCTTCAAGATTCTCACTCCAGTTTACCAAAAAGCAGTGACTCACCTGGCATTACCCCACAATCAGCAACAAAATAACTCAGCTTTCTAGCTCACCTGTCATTTCAGACAACTTTTCCTTTTTCCCTACACCCATAAAAACATAGAACAACTTCCACCAATGATGACTTCTCTCACTTTTCTAAACTACCCCAAAGAGATGGCACTTAATTTGTTCTTTTCATACATGCTTGCATTCTTATTCGCCTTTTGATTGGTCCGTACTTTTCTCTGATGATTTTGTAAAGCTACTCTGACATCTTATATGTCTTTGCATCTCATCACAGCAGCTAGCATATTTCTTATAAAGAGTAGATGCACATTATATTAAGGAGGAAGTAATAAAAAATTTAATGAAAGAAAGTTGTTACTGCTGGTTTGAAAGAATGAGGCACCATGAAGAGGGACAAGGATGAAATGAGAGAGAGAGAGAGTGGAAGGTAACAACGGAGAATAGCAAAATGTAATGATTAGAGCTAGGATTCCCTTTCTTTTCACATTTTGGACATGAAGATAAATGATCTGTGATTCCTGAATCAGGAAGCTATTGCTAACCCAAAGGGACTCTAATGTGTGTCTGTGGTTGAAGACTACACCCAGATGAGTCATTGCTAGTAATCAAGAATTTTTGCCTTGATTCATTTTTAGCCAAACTGGTGAAGTTCTTTGGTTGGGAAGGTTCTATGTTAGGGGGCTCCAGAAAATTCAGATGAGAGATGTAGAAAGAAAGTTGGGCAGGATCATCATAGCTTTTACACAGGTTAGCTGCCCAGTGATGAGTAGGGCTGTCAGTGCTCTTGTCATTACTCTGGAGAAGATACAACTTTATGAGAAATGTTCCCAAATGGATGAACCCAGTCATAGCCAAAGTCAAGGACTTAAACATTACTATATTAATTGTTCTTTCAGCTGTGAATCTGATTTATGTTCTTAATGTCCCAAAAAGACTTAATGGTATTGCTTTAAAGTAGAAAAATAAAGGAAGGAAAAGAAGAAAATGATTCATTACACACTGAAGTTTATTCTAAATCAGCATCCTGATAATAGAGAAATCAGGAAAGAGATTTAAAATCCTGAGCTCTTTTTCCTCTAGAAAAATGATTAACTTTTCTTTGCCTTTGTAGGATTTATAAATTTGCCTTTGAGAATCTGGTACACACTATAAACCCTTCCAAAAAAATGGTCATCTCACACTCAAAAACTCAAAAGAAAATCCACATCAATATTACATATGAATAATAGCCTATTAACATTAATATTTTTGTATTAATATCTTTTTCAATTTACCCCTTAGTCAATTTTTTCTAAACTGACAGCCAAATGAATAACAAAGGAAGAGGTTTATCATGTTCAGATTGCCAAAGATTAAATAGGATATCACTGCTGCATATCTGTCTGTGTTTTTATTTTGTATATAGCAAAAATACTACTAAAAGACTTAGTATTAAATTTTATCCATATTTAGAATCAAATGAAGCTTTCAACAGCATATGAAAAAAATAATAATAATTGCATACTTACCCAAGCACATGGTTTGGTCATCATTTGTTTTAAAACCAGTGTGGCAAAGGCAATAAAAGCTTCCAACTGTGTCAATGCACTGCCCATGACTGCATATATTGGGGATTTCTTGACATTCATTACGATCTGTAAATAAGAAGCATCTTAAGTGAGAACTTAGAAGACAAAATATAATTGAATAACTTACTTCTAGCTATCATTCTCAGGAGTAATCCTAGCTCTAAACTAAGTTAGTGAAACAGATATATTTATTTTTGCATCAAAATGGATTAAAACTTACTCTCATTTACCAGGCATAGTCAGAGTATGTTAACAATACTTGACACAATAAAAAATAAGGGTAAGTAATCAAAAAATATTCTGAAAGATGAGAAAACTTAGAAGTAAAATGGGGGGATTCAAGCCGGGAGGATTTTCTGTTTGATTATCGGCCTACGCAGCACCACAGACTGTGGAATTAATTGAAGTATTTTGTCTAGTTAAGATTTGGTGCCTTAAGTCATGGATAGTTCATGGCTTCCTTTGTAAATTTATATCAAAGTCAAATCCTATGACTAAGATTAAGTTTTTCTTGATGGACAACCTACATGGCTGCTTGTTCAAGTTGCATCTCTTTGTTTTGTCTTATGCCTGTAACTTTTTGTTCCATCTTCACCAAGTATTCACGTCTACACAGGGCTCCTTTTAGCATGTGACAACCTGAACACTGTCTCTTTCTTTTTACTCTCATCCATTCAAAAGGAAGCCCTCTCATCTTGCTTCTGCCAGTCTTTAAGTTCTAAATTATATTTATCATCAACTGAATCTTTGTTGCTTTCCTGAAATCTGATTATCTGAGGGGCAGAATTTAATGCATGAATTCACATATGACATAATTATATGCCAACTTACGTCTCTAATTTTTTAAAGCTCTATGTTTAGAGTAAATGTACTTTGAAGAGCACTGTGCCATTCTGATTTTCCTACTGTCACAGAAGAAAGCAAATGGAGTGTCATCTTTGCTGCAGATGCTGTAGGGCAGATGGCATTTCGCTCTTAGTTAAATAATCCAGACTTGCCGTTTGCCTGCAATGTCTACTCAGAACACAGTAATCATGAATGGACACTTTTTGCTGCACACAGCAACCCTCCCTGTGCCAGAATCATGGACATCTCCTCCATTTCATGTGTCCGGGATGCCACTGTTATGGTTATCTCCTATTTAGTAGATGAATCATCATACCATTTAGAAGTTCCAAGTGAAGTGGAGTATATCATTATGAGGGGAAATTGAAATAGAGCAGCTTAGGAACTACTACTAGCCTATGCAGAAATGCTATTTAACAAATTATGATTATTCCCTTTTGAGAAAATGAATTTAAAGTGAGCCAACTGCTTCCCAGTATAGGCAGAGGAGACTGATTTCCGATCTTGAGGAAGGGTGTCTCAGGTCCTACCATAGTTGGCCCAAAACTTTGGCACCTTCAGGTAGGACTGGACCTGGAGGTATGCAGGGGACGAGGTCTGTCTCATATCTGAGTTCCACAAGGAAATGGAGAGCCTTAAGGATCTTATTAGAAAGGGATCCCCAATTAGGAATTTCCTTTCATCTACTGCCCTATTTCATGATGGTTTAAATAAATGAAAAATGAAACATGGGGAGTACTTTGGAGTTCTACTGGTAAAACTAGAGCAGAAATCTAGGTTTCCACAAGCACCAGAATTTGAACTCTTAGATGATCATATTTGAGGTTGGGATAGGAGGTCTGAACAGGCTACTGTGGAAAGAAACCCAAATAGTCAACAGAGAAAGAGAGGAAACCACAAGGAAGTAGCTGGGAAGAAACTGTTCCTTGTCTTTTGTGGGGCAAGACATTTCAGAAGGGGGTTGCATTTGTTCTAACTATCCAGCTTCTGCTGTCAGACTAGCATCCTGAGCTCAGATATCTAATTATTCTCTCAGCTCTCTCAGCAATAATGTGGGTGCTATGGCCAGGTGTGATTTGGGATTTGAAATAATGTGCTTATGAATTCAAATAAGGATCTCATGATGGAGAGAAAGTTTTAGTTATTTTCATTTTAAACAGATGCACAAAGGAGTACTTTGCAGCATGCCTGAAAAAAAAAATGCCAAGTATAGAATAGCATTTAAAGATATTTTAAAAGTAAGACATAAATAAAGACATTTCTAGAATGAGGTCTTCAACCCAACAGGCTGAATGGTTGCAAAACTCAAGTTAATTCCCTTTCTTCTGAGGATTTCAATGAATAAAGGAAATCAAGCTTTTGGCTGCAAATGATGCTCACTCACAACACTTATGGCTGGCTGCATTTTATGTATGATAGTCTCCTACAAGAAAACAGTAAATGACTTAAGCTCCAGCAGGCTATGGCTGAATGTCTTATCTTGATCTATAAATAGCTCTGTCCCACTTCTTACCTGAGTGTGAAATGGTGTGAAATGACTCATTCTATATTATCAGCAATTCTTTATTTTGACTGGAATTATTTCATCCAGTTCTAAAGAAATATAAATTATGAGCAAAATGCAGGGAAGTGGTATCAATGCCAAGTTTCTCTAGGGTATTGTGGTTCATATCAGAGACGTCCCTGCTTTTCCTGTCCTGTATTTTTAATACATGGATAGCAACAGCTCTGTTGACAAAATGCTAAATACTTACACATTTTCCTTTAAACTCAACAAAACAAGTCTTGATTCTCAATGCGATTTTTTTGTTTGGCAATTACACCCTCTTCCCACCAAGGTCTCATAAAACAATGCAAAATACCCTAAAGAGAACAGTGCACTTCCTACCACTTAATCACTGGTAGGCAAGGAAATCCAAGGAAAATCTGACCACGTTCCCCTGTGTTGTTTCATTCACAAATGCATAAAATATCCAGGAAACCCCCCGTGAAATGAACCGTGAGCAAAATAACACATTCTTCCTGACTAATATACAATGATTTGCTTTGCTAAATATACAGTGATTTTTGCTAAATATACAGATTCAGAATTACTTAAAAACTTACAAGAAAATCATTCTGAGATATTATTTTGTTTCTGGGCTGAAACAGCAGTCTTTTAAATTTTCACACCACTGGCTACAAGATACCATCAGATACTGAATAACTTCACCGTACACTGAAGTTGATAACATGCCAACGTGCTTCCTATTTCTATCACATCCCTGACAAAAGTGAACACCAAAAAAATCTAATCTCTAGGGGTCATGGGAATAAATAGCTAATAAAGGCTGTTGATGTTTGCTGCCTTCAGTGAGTGACCCTTTCCCTGTAAGATGACTGGGAAGAAAAGCGATGGCTTTGTTTGTTCTGTTTCTGCAGATTAATCTGACCCAGCTCAGGAGAGCTTCACTGTCTAGATTTGGGGTATGCAGCCAAGATGGTGAAAACCTCACCTTTCTGAGAGGGATTCTGTGGAAGGAAAAAAATCATTTGGTCCTAAATGTTGCTTAGGGGAATGTGTGTATGCATAATTGTGTGTGTGTGTAATGTGTGGGGGTGAGTGTTTGCAGGCAGGGCAGCAGACAGGCAAATACGAAATCTATGTATGCGAATGATTTACTTCTTGTTTTGAAAATATCTCATCCAGAAAGAGGATTAAAAAGCTAAGTGTTTAGATATTTTGACAATATCCCTTTTTCCAATTTAGTTTAATATTAGAGTTCATCAATCTAAATCTTAACTCATATATTCTTCAGCTTAACTATCTGAAAATAAATCCAGATATCTGAAGCTTCATGAAGACAAACTCTTGGGTAGGCATGTCCAGCCTGTGGGGCACTACATACCATTGCACTGTCCTGTGGAGGTGAAGCGGTAGCCGGGCTTACAGTCACAGCGGTAGCTGCCTGCAGTGTTGATGCATTCGGCGTTGCGCTGGCACACTGGGCCGTTCTGACACTCGTCAATATCTA/-\", \"assembly_name\": \"GRCh38\", \"start\": 48446709, \"transcript_consequences\": [{\"exon\": \"45-47/66\", \"conservation\": -0.227, \"impact\": \"HIGH\", \"intron\": \"44-46/65\", \"hgvsc\": \"NM_000138.5:c.5423_5788del\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"lof_info\": \"INTRON_SIZE:3952\", \"protein_end\": 1930, \"variant_allele\": \"-\", \"loeuf\": 0.105, \"genesplicer\": \"loss/acceptor/48446818-48446819/Medium/3.554982,loss/donor/48446832-48446833/Medium/11.850327,loss/acceptor/48446923-48446922/Medium/3.930615,loss/acceptor/48447207-48447208/Medium/7.533661,loss/acceptor/48447454-48447455/Medium/2.489039,loss/donor/48447966-48447967/Medium/1.212396,loss/acceptor/48448125-48448126/Medium/3.765642,loss/donor/48448207-48448208/Medium/3.037169,loss/acceptor/48448295-48448296/Medium/4.456453,loss/acceptor/48448530-48448531/Medium/8.364010,loss/acceptor/48448867-48448868/Medium/5.321915,loss/acceptor/48449145-48449146/Medium/3.395410,loss/donor/48449158-48449157/Medium/4.459279,loss/acceptor/48449427-48449428/Medium/4.307533,loss/acceptor/48449653-48449652/Medium/3.227308,loss/acceptor/48449951-48449950/Medium/2.027501,loss/donor/48450285-48450284/Medium/1.655191,loss/acceptor/48450498-48450499/Medium/7.365027,loss/donor/48450626-48450627/Medium/2.047190,loss/acceptor/48451024-48451023/Medium/6.508151,loss/acceptor/48451185-48451186/Medium/2.670949,loss/acceptor/48451337-48451338/Medium/6.135543,loss/acceptor/48451500-48451499/Medium/1.534822,loss/donor/48451706-48451705/Medium/1.325263,loss/acceptor/48451844-48451845/Medium/1.419132,loss/acceptor/48452058-48452059/Medium/1.338555,loss/donor/48452140-48452141/Medium/6.461591,loss/acceptor/48452289-48452288/Medium/4.940221,loss/acceptor/48452416-48452417/Medium/6.069075,loss/acceptor/48452569-48452570/Medium/9.946183,gain/acceptor/48446710-48446711/Medium/7.032198\", \"amino_acids\": \"DIDECQNGPVCQRNAECINTAGSYRCDCKPGYRFTSTGQCNDRNECQEIPNICSHGQCIDTVGSFYCLCHTGFKTNDDQTMCLDINECERDACGNGTCRNTIGSFNCRCNHGFILSHNNDCID/D\", \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"cds_end\": 5788, \"codons\": \"gATATTGACGAGTGTCAGAACGGCCCAGTGTGCCAGCGCAACGCCGAATGCATCAACACTGCAGGCAGCTACCGCTGTGACTGTAAGCCCGGCTACCGCTTCACCTCCACAGGACAGTGCAATGATCGTAATGAATGTCAAGAAATCCCCAATATATGCAGTCATGGGCAGTGCATTGACACAGTTGGAAGCTTTTATTGCCTTTGCCACACTGGTTTTAAAACAAATGATGACCAAACCATGTGCTTGGACATAAATGAATGTGAAAGAGATGCCTGTGGGAATGGAACTTGCCGGAACACAATTGGTTCCTTCAACTGCCGCTGCAATCATGGTTTCATCCTTTCTCACAACAATGACTGTATAGAT/gat\", \"protein_start\": 1808, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"aa\": \"TACAGTCATTGTTGTGAGAAAGGATGAAACCATGATTGCAGCGGCAGTTGAAGGAACCAATTGTGTTCCGGCAAGTTCCATTCCCACAGGCATCTCTTTCACATTCATTTATGTCTAGTAGGAAGAAAGGCCATAAAGAAACATAATTATAAGTAGAAAAAGTGGTTACACGGTTACAGTGGCTAAAGAGCATTTTAGGGTTCACCAGGCCTCATCCATAGGCTGAGAACTTCTTGATGTTGGTATCAGCTCCAGCAGCAGTCATCCCTGCAAGACACCAGGCAGCGGCAAAGCTCTAAACCTTGGGAATAAATGGTTTCCTTGGCTGGCCAGTGAAAGGAGGCAAGCTAAGGACATGAGACCTGAACAAGAATTATTTTATTCTATTTACTTCTAGAATTCTCCACCCTTGCCACTTTTCTCAAATATTTTCACTTGAAAATGGAAAAAACTCTGTGTGCATCTGGGTCTTGAGGTGTTGCCACCTTTGGCAAACTTCAACCTTCAAGATTCTCACTCCAATTTACCAAAAAGCAGTGACTCACCTGGCATTACCCCACAATCAGCAACAAAATAACTCAGCTTTCTAGCTCACCTGTCATTTCAGACAACTTTTCCTTTTTCCCTACACCCGTAAAAACATAGAACAACTTCCACCAATGATGACTTCTCTCACTTTTCTAAACTACCCCAAaGAGATGGCACTTAATTTGTTCTTTTCATACATGCTTGCATTCTTATTCGCCTTTTGATTGGTCTGTACTTTTCTCTGATGATTTTGTAAAGCTACTCTGACATCTTATACGTCTTTGCATCTCATCACAGCAGCTAGCATATTTCTTATAAAGAGTAGATGcACATTATATTAAGGAGGAAGTAATAAAAAATTTAATGAAAGAAAGTTGTTACTGCTGGTTTGAAAGAATGAGGCACCAtGAAGAGGGACAAgGATGAAACGAGAGA--GAGAGAGTGGAAGGTAACAACGGAGAATAGCAAAATGTAATGATTAGAGCTAGGATTCCCTTTCTTTTCACATTTTGGACATGAAGATAAATGATCTGTGATTCCTGAATcAGGAAGCTATTGCTAACCCAAAGGGACTCTAATGTGTGTCTGTGGTTGAAGACTACACCCAGATGAGTCATTGCTAGTAATCAAGAATTTCTGCCTTGATTCATTTTTAGCCAAACTGGTGAAGTTCTTTGGTTGGGAAGGTTCTATGTTAGGGGGCTCCAGAAAATTCAGATGAGAgATGTAGAAAGAAAGTtgggcagggtcatCATAGCTTTTACACAGGTTAGCTGCCCAGTGATGAGTAGGGCTGTCAGTGCTCTTGTCATTACTCTGGAGAAGATACAACTTTATGAGAAATGTTCCCAAATGGATGAACCCAGTCATAGCCAAAGTCAAGGACTTAAACATTACTATATTAATTGTTCTTTCAGCTGTGAATCTGATTTATGTTCTTAATGTCCCAAAAAGACTTAATGGTATTGCTTTAAAGTAGAAAAATAAAGGAAGGAAAAGAAGAAAATGATTCATTACACACTGAAGTTTATTCTAAATCAGCATCCTGATAATAGAGAAATCAGGAAAGAGATTTAAAATCCTGAGCTCTTTTTCCTCTAGAAAAATGATTAAcTTTTCTTTGCCTTTGTAGGATTTATAAATTTGCCTTTGAGAATCTGGTACACACTATAAACCCTTCCAAAAAAATGGTCATCTCaCACTCAAAAACTCAAAAGAAAATCCACATCAATATTACATATGAATAATAGCCTATTAACATTAATATTTTTGTATTAATATCTTTTTCAATTTACCCCTTAGTCAATTTTTTCTAAACTGACAGCCAAATGAATAACA-AGGAAGAGGTTTATCATGTTCAGATTGCCAAAGATTAAATAGGATATCACTGCTGCATATCTgtctGTGTTTTTATTTTGTATATAGCAAAAATACTACTAAAAGACTTAGTATTAAATTTTATCCATATTTAGAATCAAATGAAGCTTTCAAcAGCATATGAAAAAAAGAATAATAATTGCATACTTACCCAAGCACATGGTTTGGTCATCATTTGTTTTAAAACCAGTGTGGCAAAGGCAATAAAAGCTTCCAACTGTGTCAATGCACTGCCCATGACTGCATATATTGGGgATTTCTTGACATTCATTACGATCTGTAAATAAAAAGCATCTTAAGTGAGAACTTAGAAGACAAAATATAATTGAATAACTTACTTCTAGCTATCGTTCTCAGGAGTAATCCTAGCTCTAAACTAAGTTAGTGAAACAGATATATTTATTTTTGCATCAAAATGGATTAAAACTTACTCTCATTTACCAGGCATAGTCAGAGTATGTTAACAATACTTGACACAATAAAAAATAAGGGTAAGTAATCAAAAAATATTCTGAAAGATGAGAAAACTTAGAAGTAAAATGGGGGGATTCAAGCTGGGAGGATTTTCTGTTTGATTATCGGCCTACGCAGCACCACAGACTGTGGAATTAATTGAAGTATTTTGTCTAGTTAAGATTTGGTGCCTTAAGTCATGGATAGTTCATGGCTTCCTTTGGAAATTTATATCAAAGTCAAATCCTATGACTAAGATTAAGTTTTTCTTGATGGACAACCTACATGGCtGCTTGTTCAAGTTGCATCTCTTTGTTTTGTCTTATGCCTGTAACTTTTTGTTCCATCTTCACCAAGTATTCACGTCTACACAGGGCTCCTTTTAGCATGTGACAACCTGAACACTGTCTCTTTCTTTTTACTCTCATCCATTCAAAAGGAAGCCCTCTCATCTTGCTTCTGCCAGTCTTTAAGtTCTAAATTATATTTATCATCAACTGAATCTTTGTTGCTTTCCTGAAATCTGATTATCTGAGGGGCAGAATTTaATGCATGAATTCACATATGACATAATTATATGCCAACTTAcGTCTCTAATTTTTTAAAGCTcTATGTTTAGAGTAAATGTACTTTGAAGAGCACTGTGCCATTCTGATTTTCCTACTGTCACAGAAGAAAGCAAATGGAGTGTCATCTTTGCTGCAGATGCTGTAGGGCAGATGGCATTTCGCTCTTAGTTAAATAATCCAGACTTGCCGTTTGCCTGCAATGTCTACTCAGAACACAGTAATCATGAATGGACACTTTTTGCTGCACACAGCAACCCTcCCTGTGCCAGAATCaTGGAcATCTCCTCCATTTCATGTGTCCGGGATGCCAcTGTTATGGTTATCTCCTATTTAGTAGATGAATCATCATACCATTTAGAAGTTCCAAGTGAAGTgGAGTATATCATTATGAGGGGAAATTGAAATAGAGCAGCTTAGGAACTACTACTAGCCTATGCAGAAATGCTATTTAACAAATTATGATTATTCCCTTTTGAGAAAATGAATTTAAAGTGAGCCAACTGCTTCCCAGTATAGGCAGAGGAGACTGATTTCCGATCTTGAGGAAGGGTGTCTCAgGTCCTACCATAGTTGGCCCAAAACTTTGGCACCTTCAGGTAGGACTGGACCTGGAGGTATGCAGGGGACGAGGTCTGTCTCATATCTGAGTTCCACAAGGAAATGGAGAGCCTTAAGGATCTTATTAGAAAGGGATCCCCAATTAGGAATTTCCTTTCATCTAcTGCCCTATTTCATGATGGTTTAAATAAATGAAAAATGAAACATGGGGAGTACTTTGGAGTTCTACTGGTAAAACTAGAGCAGAAATCTAGGTTTCCACAAGCACCAGAATTTGAACTCTTAGATGATCATATTTGAGGTTGGGATAGGAGGTCTGAACAGGCTACTGTGGAAAGAAACCCAAACAGTCAACAGAGAAAGAGAGGAAACCACAAGGAAGTAGCTGGGAAGAAACTGTTCCTTGTCTTTTGCGGGGCAAGACATTTCAGAAGGGGGTTGCATTTGCTCTAACTATCCAGCTTCTGCTGTCAGACTAGCATCCTGAGCTCAGATATCTAATTGTTCTCTCAGCTCTCTCAGCAATAATGTGGGTGCTATGGCCAGGTGTGATTTGGGATTTGAAATAATGTGCTTATGAATTCAAATAAAGATCTCATGATGGAGAgAAAGTTTTAGTTATTTTCATTTTAAACAGATGCACAAAGGAGTACTTTGCAGCATGCCTGAAAA--AAAATGCCAAGTATAGAATAGCATTTAAAGATATTTTAAAAGTAAGACATAAATAAAGACATTTCTAGAATGAAGTCTTCAACCCAACAGGCTGAATGGTTgCAAAACTCAagttaATTCCCTTTCTTCTGAGGATTTCAATGAAtAAAGGAAATCAAGCTTTTGGCTGCAAATGATGCTCACTCACAACACTTATAGCTGGGTGCATTTTATGTATGATAGTCTCCTACAAGAAAACAGtAAATGGCTTAAGCTCCAGCAGGCTATGGCTGAATGTCTTATCTTGATCTATAAATAGCTCTGTCCCACTTCTTACCTGAGTGTGAAATGGTATGAAATGACTCATTCTATATTATCAGCAATTCTTTATTTTGACTGGAATTATTTCATCCAGTTCTAAAgAAATATAAATTATGAGCAAAATGCAGGGAAGTGGTATCAATGCCAAGTTTCTCTAGGGTATTGTGGTTCATATCAGAGACGTCCCTGCTTTTCCTGTCCTGTATTTTTAATACATGGATAGCAACAGCTCTGTTGACAAAATGCTAAATACTTACACATTTTCCTTTAAACTCAACAAAACAAGTCTTGATTCTCAATGCGATTT-TTTGTTTGGCAATTACACCCTCTTCCCACCAAGGTCTCATAAAACAATGCAAAATACCCTAAAGAGAACAGTGCACTTCCTACCACTTAATCACTGGTAGGCAAGGAAATCCAAGGAAAATCTGACCACGTTCCCCTGTGTTGTTTCATTCACAAATGCATAAAATATCCAGGAAACCCCCCGTGAAATGAACCGTGAGCAAAATAACACATTCTTCCTGACTAATATACAATGATTTGCTTTGCTAAATATACAGTGATTTTTGCTAAATATACAGATTCAGAATTAcTTAAAAACTTACAAGAAAATCATTCTGAGATATTATTTTGCTTCTGGGCTGAAACAGCAGTCTTTTAAATTTTCACACCACTGGCTACAAGATACCATCAGATACTGAATAACTTCACCGTACACTGAAGTTGATAACATGCCAACGTGCTTCCTATTTCTATCACATCCCTGACAAAAGTGAACACCAAAAAAATCTAATCTCTAGGGGTCATGGGAATAAATAGCTAATAAAGGCTGTTGATGTTTGCTGCCTTCAGTGAGTGACCCTTTCCCTGTAAGATGACTGGGAAGAAAAGCGATGGCTTTGTTTGTTCTGTTTCTGCAGATTAATCTGACCCAGCTCAGGAGAGCTTCAcTGTCTAGATTTGGGGTATGCAGCCAAGATGGTGAAAACCTCACCTTTCTGAGAGGGATTCTGTGGAAGGAAAAAAATCATTTGGTCCTAAATGTTGCTTAGGGGAGTgTGTGTATGCATAATTGTGTGTGTGTGTAATGTGTGGGGGTGAGTGTTTGCAGGCAGGGCAGCAGACAGGCAAATACGAAATCTATGTATGCGAATGATTTACTTCTTGTTTTGAAAATATCTCATCCAGAAAGAGGATTAAAAAGCTAAGTGTTTAGATATTTTGACAATATCCCTTTTTCCAATTTaGTTTAATATTAGAGTTCATCAATCTAAATCTTAACTCATATATTCTTCAGCTTAACTATCTGAAAATAAATCCAGATATCTGAAGCTTCATGAAGACAAACTCTTGGGTAGGCATGTCCAGCCTGTGGGGCACTACATACCATTGCACTGTCCTGTGGAGGTGAAGCGGTAGCCGGGCTTACAGTCACAGCGGTAGCTGCCTGCAGTGTTGATGCATTCGGCGTTGCGCTGGCACACTGGGCCGTTCTGACACTCGTCAATATCTA\", \"location\": \"15:48446706-48452684\", \"hgvsp\": \"NP_000129.3:p.Ile1809_Asp1930del\", \"cdna_end\": 6104, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"allele_num\": 1, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48452687, \"start\": 48446708}}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}}, \"given_ref\": \"TACAGTCATTGTTGTGAGAAAGGATGAAACCATGATTGCAGCGGCAGTTGAAGGAACCAATTGTGTTCCGGCAAGTTCCATTCCCACAGGCATCTCTTTCACATTCATTTATGTCTAGTAGGAAGAAAGGCCATAAAGAAACATAATTATAAGTAGAAAAAGTGGTTACACGGTTACAGTGGCTAAAGAGCATTTTAGGGTTCACCAGGCCTCATCCATAGGCTGAGAACTTCTTGATGTTGGTATCAGCTCCAGCAGCAGTCATCCCTGCAAGATACCAGGCAGCGGCAAAGCTCTAAATCTTGGGAATAAATGGTTTCCTTGGCTGGCCAGTGAAAGGAGGCAAGCTAAGGACATGAGACCTGAACAAGAATTATTTTATTCTATTTACTTCTAGAATTCTCCACCCTTGCCACTTTTCTCAAATATTTTCACTTGAAAATGGAAAAAACTCTGTGTGCATCTGGGTCTTGAGGTGTTGCCACCTTTGGCAAACTTCAACCTTCAAGATTCTCACTCCAGTTTACCAAAAAGCAGTGACTCACCTGGCATTACCCCACAATCAGCAACAAAATAACTCAGCTTTCTAGCTCACCTGTCATTTCAGACAACTTTTCCTTTTTCCCTACACCCATAAAAACATAGAACAACTTCCACCAATGATGACTTCTCTCACTTTTCTAAACTACCCCAAAGAGATGGCACTTAATTTGTTCTTTTCATACATGCTTGCATTCTTATTCGCCTTTTGATTGGTCCGTACTTTTCTCTGATGATTTTGTAAAGCTACTCTGACATCTTATATGTCTTTGCATCTCATCACAGCAGCTAGCATATTTCTTATAAAGAGTAGATGCACATTATATTAAGGAGGAAGTAATAAAAAATTTAATGAAAGAAAGTTGTTACTGCTGGTTTGAAAGAATGAGGCACCATGAAGAGGGACAAGGATGAAATGAGAGAGAGAGAGAGTGGAAGGTAACAACGGAGAATAGCAAAATGTAATGATTAGAGCTAGGATTCCCTTTCTTTTCACATTTTGGACATGAAGATAAATGATCTGTGATTCCTGAATCAGGAAGCTATTGCTAACCCAAAGGGACTCTAATGTGTGTCTGTGGTTGAAGACTACACCCAGATGAGTCATTGCTAGTAATCAAGAATTTTTGCCTTGATTCATTTTTAGCCAAACTGGTGAAGTTCTTTGGTTGGGAAGGTTCTATGTTAGGGGGCTCCAGAAAATTCAGATGAGAGATGTAGAAAGAAAGTTGGGCAGGATCATCATAGCTTTTACACAGGTTAGCTGCCCAGTGATGAGTAGGGCTGTCAGTGCTCTTGTCATTACTCTGGAGAAGATACAACTTTATGAGAAATGTTCCCAAATGGATGAACCCAGTCATAGCCAAAGTCAAGGACTTAAACATTACTATATTAATTGTTCTTTCAGCTGTGAATCTGATTTATGTTCTTAATGTCCCAAAAAGACTTAATGGTATTGCTTTAAAGTAGAAAAATAAAGGAAGGAAAAGAAGAAAATGATTCATTACACACTGAAGTTTATTCTAAATCAGCATCCTGATAATAGAGAAATCAGGAAAGAGATTTAAAATCCTGAGCTCTTTTTCCTCTAGAAAAATGATTAACTTTTCTTTGCCTTTGTAGGATTTATAAATTTGCCTTTGAGAATCTGGTACACACTATAAACCCTTCCAAAAAAATGGTCATCTCACACTCAAAAACTCAAAAGAAAATCCACATCAATATTACATATGAATAATAGCCTATTAACATTAATATTTTTGTATTAATATCTTTTTCAATTTACCCCTTAGTCAATTTTTTCTAAACTGACAGCCAAATGAATAACAAAGGAAGAGGTTTATCATGTTCAGATTGCCAAAGATTAAATAGGATATCACTGCTGCATATCTGTCTGTGTTTTTATTTTGTATATAGCAAAAATACTACTAAAAGACTTAGTATTAAATTTTATCCATATTTAGAATCAAATGAAGCTTTCAACAGCATATGAAAAAAATAATAATAATTGCATACTTACCCAAGCACATGGTTTGGTCATCATTTGTTTTAAAACCAGTGTGGCAAAGGCAATAAAAGCTTCCAACTGTGTCAATGCACTGCCCATGACTGCATATATTGGGGATTTCTTGACATTCATTACGATCTGTAAATAAGAAGCATCTTAAGTGAGAACTTAGAAGACAAAATATAATTGAATAACTTACTTCTAGCTATCATTCTCAGGAGTAATCCTAGCTCTAAACTAAGTTAGTGAAACAGATATATTTATTTTTGCATCAAAATGGATTAAAACTTACTCTCATTTACCAGGCATAGTCAGAGTATGTTAACAATACTTGACACAATAAAAAATAAGGGTAAGTAATCAAAAAATATTCTGAAAGATGAGAAAACTTAGAAGTAAAATGGGGGGATTCAAGCCGGGAGGATTTTCTGTTTGATTATCGGCCTACGCAGCACCACAGACTGTGGAATTAATTGAAGTATTTTGTCTAGTTAAGATTTGGTGCCTTAAGTCATGGATAGTTCATGGCTTCCTTTGTAAATTTATATCAAAGTCAAATCCTATGACTAAGATTAAGTTTTTCTTGATGGACAACCTACATGGCTGCTTGTTCAAGTTGCATCTCTTTGTTTTGTCTTATGCCTGTAACTTTTTGTTCCATCTTCACCAAGTATTCACGTCTACACAGGGCTCCTTTTAGCATGTGACAACCTGAACACTGTCTCTTTCTTTTTACTCTCATCCATTCAAAAGGAAGCCCTCTCATCTTGCTTCTGCCAGTCTTTAAGTTCTAAATTATATTTATCATCAACTGAATCTTTGTTGCTTTCCTGAAATCTGATTATCTGAGGGGCAGAATTTAATGCATGAATTCACATATGACATAATTATATGCCAACTTACGTCTCTAATTTTTTAAAGCTCTATGTTTAGAGTAAATGTACTTTGAAGAGCACTGTGCCATTCTGATTTTCCTACTGTCACAGAAGAAAGCAAATGGAGTGTCATCTTTGCTGCAGATGCTGTAGGGCAGATGGCATTTCGCTCTTAGTTAAATAATCCAGACTTGCCGTTTGCCTGCAATGTCTACTCAGAACACAGTAATCATGAATGGACACTTTTTGCTGCACACAGCAACCCTCCCTGTGCCAGAATCATGGACATCTCCTCCATTTCATGTGTCCGGGATGCCACTGTTATGGTTATCTCCTATTTAGTAGATGAATCATCATACCATTTAGAAGTTCCAAGTGAAGTGGAGTATATCATTATGAGGGGAAATTGAAATAGAGCAGCTTAGGAACTACTACTAGCCTATGCAGAAATGCTATTTAACAAATTATGATTATTCCCTTTTGAGAAAATGAATTTAAAGTGAGCCAACTGCTTCCCAGTATAGGCAGAGGAGACTGATTTCCGATCTTGAGGAAGGGTGTCTCAGGTCCTACCATAGTTGGCCCAAAACTTTGGCACCTTCAGGTAGGACTGGACCTGGAGGTATGCAGGGGACGAGGTCTGTCTCATATCTGAGTTCCACAAGGAAATGGAGAGCCTTAAGGATCTTATTAGAAAGGGATCCCCAATTAGGAATTTCCTTTCATCTACTGCCCTATTTCATGATGGTTTAAATAAATGAAAAATGAAACATGGGGAGTACTTTGGAGTTCTACTGGTAAAACTAGAGCAGAAATCTAGGTTTCCACAAGCACCAGAATTTGAACTCTTAGATGATCATATTTGAGGTTGGGATAGGAGGTCTGAACAGGCTACTGTGGAAAGAAACCCAAATAGTCAACAGAGAAAGAGAGGAAACCACAAGGAAGTAGCTGGGAAGAAACTGTTCCTTGTCTTTTGTGGGGCAAGACATTTCAGAAGGGGGTTGCATTTGTTCTAACTATCCAGCTTCTGCTGTCAGACTAGCATCCTGAGCTCAGATATCTAATTATTCTCTCAGCTCTCTCAGCAATAATGTGGGTGCTATGGCCAGGTGTGATTTGGGATTTGAAATAATGTGCTTATGAATTCAAATAAGGATCTCATGATGGAGAGAAAGTTTTAGTTATTTTCATTTTAAACAGATGCACAAAGGAGTACTTTGCAGCATGCCTGAAAAAAAAAATGCCAAGTATAGAATAGCATTTAAAGATATTTTAAAAGTAAGACATAAATAAAGACATTTCTAGAATGAGGTCTTCAACCCAACAGGCTGAATGGTTGCAAAACTCAAGTTAATTCCCTTTCTTCTGAGGATTTCAATGAATAAAGGAAATCAAGCTTTTGGCTGCAAATGATGCTCACTCACAACACTTATGGCTGGCTGCATTTTATGTATGATAGTCTCCTACAAGAAAACAGTAAATGACTTAAGCTCCAGCAGGCTATGGCTGAATGTCTTATCTTGATCTATAAATAGCTCTGTCCCACTTCTTACCTGAGTGTGAAATGGTGTGAAATGACTCATTCTATATTATCAGCAATTCTTTATTTTGACTGGAATTATTTCATCCAGTTCTAAAGAAATATAAATTATGAGCAAAATGCAGGGAAGTGGTATCAATGCCAAGTTTCTCTAGGGTATTGTGGTTCATATCAGAGACGTCCCTGCTTTTCCTGTCCTGTATTTTTAATACATGGATAGCAACAGCTCTGTTGACAAAATGCTAAATACTTACACATTTTCCTTTAAACTCAACAAAACAAGTCTTGATTCTCAATGCGATTTTTTTGTTTGGCAATTACACCCTCTTCCCACCAAGGTCTCATAAAACAATGCAAAATACCCTAAAGAGAACAGTGCACTTCCTACCACTTAATCACTGGTAGGCAAGGAAATCCAAGGAAAATCTGACCACGTTCCCCTGTGTTGTTTCATTCACAAATGCATAAAATATCCAGGAAACCCCCCGTGAAATGAACCGTGAGCAAAATAACACATTCTTCCTGACTAATATACAATGATTTGCTTTGCTAAATATACAGTGATTTTTGCTAAATATACAGATTCAGAATTACTTAAAAACTTACAAGAAAATCATTCTGAGATATTATTTTGTTTCTGGGCTGAAACAGCAGTCTTTTAAATTTTCACACCACTGGCTACAAGATACCATCAGATACTGAATAACTTCACCGTACACTGAAGTTGATAACATGCCAACGTGCTTCCTATTTCTATCACATCCCTGACAAAAGTGAACACCAAAAAAATCTAATCTCTAGGGGTCATGGGAATAAATAGCTAATAAAGGCTGTTGATGTTTGCTGCCTTCAGTGAGTGACCCTTTCCCTGTAAGATGACTGGGAAGAAAAGCGATGGCTTTGTTTGTTCTGTTTCTGCAGATTAATCTGACCCAGCTCAGGAGAGCTTCACTGTCTAGATTTGGGGTATGCAGCCAAGATGGTGAAAACCTCACCTTTCTGAGAGGGATTCTGTGGAAGGAAAAAAATCATTTGGTCCTAAATGTTGCTTAGGGGAATGTGTGTATGCATAATTGTGTGTGTGTGTAATGTGTGGGGGTGAGTGTTTGCAGGCAGGGCAGCAGACAGGCAAATACGAAATCTATGTATGCGAATGATTTACTTCTTGTTTTGAAAATATCTCATCCAGAAAGAGGATTAAAAAGCTAAGTGTTTAGATATTTTGACAATATCCCTTTTTCCAATTTAGTTTAATATTAGAGTTCATCAATCTAAATCTTAACTCATATATTCTTCAGCTTAACTATCTGAAAATAAATCCAGATATCTGAAGCTTCATGAAGACAAACTCTTGGGTAGGCATGTCCAGCCTGTGGGGCACTACATACCATTGCACTGTCCTGTGGAGGTGAAGCGGTAGCCGGGCTTACAGTCACAGCGGTAGCTGCCTGCAGTGTTGATGCATTCGGCGTTGCGCTGGCACACTGGGCCGTTCTGACACTCGTCAATATCTA\", \"gene_symbol_source\": \"EntrezGene\", \"lof\": \"HC\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"CTATACAGTCATTGTTGTGAGAAAGGATGAAACCATGATTGCAGCGGCAGTTGAAGGAACCAATTGTGTTCCGGCAAGTTCCATTCCCACAGGCATCTCTTTCACATTCATTTATGTCTAGTAGGAAGAAAGGCCATAAAGAAACATAATTATAAGTAGAAAAAGTGGTTACACGGTTACAGTGGCTAAAGAGCATTTTAGGGTTCACCAGGCCTCATCCATAGGCTGAGAACTTCTTGATGTTGGTATCAGCTCCAGCAGCAGTCATCCCTGCAAGATACCAGGCAGCGGCAAAGCTCTAAATCTTGGGAATAAATGGTTTCCTTGGCTGGCCAGTGAAAGGAGGCAAGCTAAGGACATGAGACCTGAACAAGAATTATTTTATTCTATTTACTTCTAGAATTCTCCACCCTTGCCACTTTTCTCAAATATTTTCACTTGAAAATGGAAAAAACTCTGTGTGCATCTGGGTCTTGAGGTGTTGCCACCTTTGGCAAACTTCAACCTTCAAGATTCTCACTCCAGTTTACCAAAAAGCAGTGACTCACCTGGCATTACCCCACAATCAGCAACAAAATAACTCAGCTTTCTAGCTCACCTGTCATTTCAGACAACTTTTCCTTTTTCCCTACACCCATAAAAACATAGAACAACTTCCACCAATGATGACTTCTCTCACTTTTCTAAACTACCCCAAAGAGATGGCACTTAATTTGTTCTTTTCATACATGCTTGCATTCTTATTCGCCTTTTGATTGGTCCGTACTTTTCTCTGATGATTTTGTAAAGCTACTCTGACATCTTATATGTCTTTGCATCTCATCACAGCAGCTAGCATATTTCTTATAAAGAGTAGATGCACATTATATTAAGGAGGAAGTAATAAAAAATTTAATGAAAGAAAGTTGTTACTGCTGGTTTGAAAGAATGAGGCACCATGAAGAGGGACAAGGATGAAATGAGAGAGAGAGAGAGTGGAAGGTAACAACGGAGAATAGCAAAATGTAATGATTAGAGCTAGGATTCCCTTTCTTTTCACATTTTGGACATGAAGATAAATGATCTGTGATTCCTGAATCAGGAAGCTATTGCTAACCCAAAGGGACTCTAATGTGTGTCTGTGGTTGAAGACTACACCCAGATGAGTCATTGCTAGTAATCAAGAATTTTTGCCTTGATTCATTTTTAGCCAAACTGGTGAAGTTCTTTGGTTGGGAAGGTTCTATGTTAGGGGGCTCCAGAAAATTCAGATGAGAGATGTAGAAAGAAAGTTGGGCAGGATCATCATAGCTTTTACACAGGTTAGCTGCCCAGTGATGAGTAGGGCTGTCAGTGCTCTTGTCATTACTCTGGAGAAGATACAACTTTATGAGAAATGTTCCCAAATGGATGAACCCAGTCATAGCCAAAGTCAAGGACTTAAACATTACTATATTAATTGTTCTTTCAGCTGTGAATCTGATTTATGTTCTTAATGTCCCAAAAAGACTTAATGGTATTGCTTTAAAGTAGAAAAATAAAGGAAGGAAAAGAAGAAAATGATTCATTACACACTGAAGTTTATTCTAAATCAGCATCCTGATAATAGAGAAATCAGGAAAGAGATTTAAAATCCTGAGCTCTTTTTCCTCTAGAAAAATGATTAACTTTTCTTTGCCTTTGTAGGATTTATAAATTTGCCTTTGAGAATCTGGTACACACTATAAACCCTTCCAAAAAAATGGTCATCTCACACTCAAAAACTCAAAAGAAAATCCACATCAATATTACATATGAATAATAGCCTATTAACATTAATATTTTTGTATTAATATCTTTTTCAATTTACCCCTTAGTCAATTTTTTCTAAACTGACAGCCAAATGAATAACAAAGGAAGAGGTTTATCATGTTCAGATTGCCAAAGATTAAATAGGATATCACTGCTGCATATCTGTCTGTGTTTTTATTTTGTATATAGCAAAAATACTACTAAAAGACTTAGTATTAAATTTTATCCATATTTAGAATCAAATGAAGCTTTCAACAGCATATGAAAAAAATAATAATAATTGCATACTTACCCAAGCACATGGTTTGGTCATCATTTGTTTTAAAACCAGTGTGGCAAAGGCAATAAAAGCTTCCAACTGTGTCAATGCACTGCCCATGACTGCATATATTGGGGATTTCTTGACATTCATTACGATCTGTAAATAAGAAGCATCTTAAGTGAGAACTTAGAAGACAAAATATAATTGAATAACTTACTTCTAGCTATCATTCTCAGGAGTAATCCTAGCTCTAAACTAAGTTAGTGAAACAGATATATTTATTTTTGCATCAAAATGGATTAAAACTTACTCTCATTTACCAGGCATAGTCAGAGTATGTTAACAATACTTGACACAATAAAAAATAAGGGTAAGTAATCAAAAAATATTCTGAAAGATGAGAAAACTTAGAAGTAAAATGGGGGGATTCAAGCCGGGAGGATTTTCTGTTTGATTATCGGCCTACGCAGCACCACAGACTGTGGAATTAATTGAAGTATTTTGTCTAGTTAAGATTTGGTGCCTTAAGTCATGGATAGTTCATGGCTTCCTTTGTAAATTTATATCAAAGTCAAATCCTATGACTAAGATTAAGTTTTTCTTGATGGACAACCTACATGGCTGCTTGTTCAAGTTGCATCTCTTTGTTTTGTCTTATGCCTGTAACTTTTTGTTCCATCTTCACCAAGTATTCACGTCTACACAGGGCTCCTTTTAGCATGTGACAACCTGAACACTGTCTCTTTCTTTTTACTCTCATCCATTCAAAAGGAAGCCCTCTCATCTTGCTTCTGCCAGTCTTTAAGTTCTAAATTATATTTATCATCAACTGAATCTTTGTTGCTTTCCTGAAATCTGATTATCTGAGGGGCAGAATTTAATGCATGAATTCACATATGACATAATTATATGCCAACTTACGTCTCTAATTTTTTAAAGCTCTATGTTTAGAGTAAATGTACTTTGAAGAGCACTGTGCCATTCTGATTTTCCTACTGTCACAGAAGAAAGCAAATGGAGTGTCATCTTTGCTGCAGATGCTGTAGGGCAGATGGCATTTCGCTCTTAGTTAAATAATCCAGACTTGCCGTTTGCCTGCAATGTCTACTCAGAACACAGTAATCATGAATGGACACTTTTTGCTGCACACAGCAACCCTCCCTGTGCCAGAATCATGGACATCTCCTCCATTTCATGTGTCCGGGATGCCACTGTTATGGTTATCTCCTATTTAGTAGATGAATCATCATACCATTTAGAAGTTCCAAGTGAAGTGGAGTATATCATTATGAGGGGAAATTGAAATAGAGCAGCTTAGGAACTACTACTAGCCTATGCAGAAATGCTATTTAACAAATTATGATTATTCCCTTTTGAGAAAATGAATTTAAAGTGAGCCAACTGCTTCCCAGTATAGGCAGAGGAGACTGATTTCCGATCTTGAGGAAGGGTGTCTCAGGTCCTACCATAGTTGGCCCAAAACTTTGGCACCTTCAGGTAGGACTGGACCTGGAGGTATGCAGGGGACGAGGTCTGTCTCATATCTGAGTTCCACAAGGAAATGGAGAGCCTTAAGGATCTTATTAGAAAGGGATCCCCAATTAGGAATTTCCTTTCATCTACTGCCCTATTTCATGATGGTTTAAATAAATGAAAAATGAAACATGGGGAGTACTTTGGAGTTCTACTGGTAAAACTAGAGCAGAAATCTAGGTTTCCACAAGCACCAGAATTTGAACTCTTAGATGATCATATTTGAGGTTGGGATAGGAGGTCTGAACAGGCTACTGTGGAAAGAAACCCAAATAGTCAACAGAGAAAGAGAGGAAACCACAAGGAAGTAGCTGGGAAGAAACTGTTCCTTGTCTTTTGTGGGGCAAGACATTTCAGAAGGGGGTTGCATTTGTTCTAACTATCCAGCTTCTGCTGTCAGACTAGCATCCTGAGCTCAGATATCTAATTATTCTCTCAGCTCTCTCAGCAATAATGTGGGTGCTATGGCCAGGTGTGATTTGGGATTTGAAATAATGTGCTTATGAATTCAAATAAGGATCTCATGATGGAGAGAAAGTTTTAGTTATTTTCATTTTAAACAGATGCACAAAGGAGTACTTTGCAGCATGCCTGAAAAAAAAAATGCCAAGTATAGAATAGCATTTAAAGATATTTTAAAAGTAAGACATAAATAAAGACATTTCTAGAATGAGGTCTTCAACCCAACAGGCTGAATGGTTGCAAAACTCAAGTTAATTCCCTTTCTTCTGAGGATTTCAATGAATAAAGGAAATCAAGCTTTTGGCTGCAAATGATGCTCACTCACAACACTTATGGCTGGCTGCATTTTATGTATGATAGTCTCCTACAAGAAAACAGTAAATGACTTAAGCTCCAGCAGGCTATGGCTGAATGTCTTATCTTGATCTATAAATAGCTCTGTCCCACTTCTTACCTGAGTGTGAAATGGTGTGAAATGACTCATTCTATATTATCAGCAATTCTTTATTTTGACTGGAATTATTTCATCCAGTTCTAAAGAAATATAAATTATGAGCAAAATGCAGGGAAGTGGTATCAATGCCAAGTTTCTCTAGGGTATTGTGGTTCATATCAGAGACGTCCCTGCTTTTCCTGTCCTGTATTTTTAATACATGGATAGCAACAGCTCTGTTGACAAAATGCTAAATACTTACACATTTTCCTTTAAACTCAACAAAACAAGTCTTGATTCTCAATGCGATTTTTTTGTTTGGCAATTACACCCTCTTCCCACCAAGGTCTCATAAAACAATGCAAAATACCCTAAAGAGAACAGTGCACTTCCTACCACTTAATCACTGGTAGGCAAGGAAATCCAAGGAAAATCTGACCACGTTCCCCTGTGTTGTTTCATTCACAAATGCATAAAATATCCAGGAAACCCCCCGTGAAATGAACCGTGAGCAAAATAACACATTCTTCCTGACTAATATACAATGATTTGCTTTGCTAAATATACAGTGATTTTTGCTAAATATACAGATTCAGAATTACTTAAAAACTTACAAGAAAATCATTCTGAGATATTATTTTGTTTCTGGGCTGAAACAGCAGTCTTTTAAATTTTCACACCACTGGCTACAAGATACCATCAGATACTGAATAACTTCACCGTACACTGAAGTTGATAACATGCCAACGTGCTTCCTATTTCTATCACATCCCTGACAAAAGTGAACACCAAAAAAATCTAATCTCTAGGGGTCATGGGAATAAATAGCTAATAAAGGCTGTTGATGTTTGCTGCCTTCAGTGAGTGACCCTTTCCCTGTAAGATGACTGGGAAGAAAAGCGATGGCTTTGTTTGTTCTGTTTCTGCAGATTAATCTGACCCAGCTCAGGAGAGCTTCACTGTCTAGATTTGGGGTATGCAGCCAAGATGGTGAAAACCTCACCTTTCTGAGAGGGATTCTGTGGAAGGAAAAAAATCATTTGGTCCTAAATGTTGCTTAGGGGAATGTGTGTATGCATAATTGTGTGTGTGTGTAATGTGTGGGGGTGAGTGTTTGCAGGCAGGGCAGCAGACAGGCAAATACGAAATCTATGTATGCGAATGATTTACTTCTTGTTTTGAAAATATCTCATCCAGAAAGAGGATTAAAAAGCTAAGTGTTTAGATATTTTGACAATATCCCTTTTTCCAATTTAGTTTAATATTAGAGTTCATCAATCTAAATCTTAACTCATATATTCTTCAGCTTAACTATCTGAAAATAAATCCAGATATCTGAAGCTTCATGAAGACAAACTCTTGGGTAGGCATGTCCAGCCTGTGGGGCACTACATACCATTGCACTGTCCTGTGGAGGTGAAGCGGTAGCCGGGCTTACAGTCACAGCGGTAGCTGCCTGCAGTGTTGATGCATTCGGCGTTGCGCTGGCACACTGGGCCGTTCTGACACTCGTCAATAT\", \"cds_start\": 5423, \"hgvs_offset\": -3, \"cdna_start\": 5739, \"consequence_terms\": [\"splice_acceptor_variant\", \"splice_donor_variant\", \"inframe_deletion\", \"splice_donor_5th_base_variant\", \"start_retained_variant\", \"intron_variant\"]}]}',\n",
       "    'coordinates': [{'pca_x': -0.18800580117428728,\n",
       "      'pca_y': -0.026079450307635404},\n",
       "     {'t-sne_x': -38.967260348718355, 't-sne_y': -33.0887605919191},\n",
       "     {'umap_x': 17.617000579833984, 'umap_y': 14.193416595458984}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487205-C-T',\n",
       "      'distance': 0.0772230327129364,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48437393-G-A',\n",
       "      'distance': 0.07836870104074478,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48427780-G-A',\n",
       "      'distance': 0.07848323881626129,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48411383-G-A',\n",
       "      'distance': 0.07903255522251129,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48596386-A-G',\n",
       "      'distance': 0.07975286990404129,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48495267-A-G',\n",
       "      'distance': 0.0805107057094574,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48446828-A-T',\n",
       "      'distance': 0.08078651875257492,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48520825-C-T',\n",
       "      'distance': 0.08096721768379211,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48470763-G-A',\n",
       "      'distance': 0.08150942623615265,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48492593-A-G',\n",
       "      'distance': 0.08158550411462784,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48600236-T-G',\n",
       "      'distance': 0.08209043741226196,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48503943-A-G',\n",
       "      'distance': 0.0833469033241272,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48488243-T-G',\n",
       "      'distance': 0.08339027315378189,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445505-C-T',\n",
       "      'distance': 0.08369220048189163,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468110-C-G',\n",
       "      'distance': 0.08405474573373795,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48496226-C-A',\n",
       "      'distance': 0.08413304388523102,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437079-T-G',\n",
       "      'distance': 0.0842776894569397,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425735-T-C',\n",
       "      'distance': 0.08486013859510422,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48488240-G-A',\n",
       "      'distance': 0.08504341542720795,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48472544-C-A',\n",
       "      'distance': 0.08509773015975952,\n",
       "      'pathogenicity': 'likely_benign'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_benign',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_benign', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'likely_benign', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_benign', 'confidence_score': 0.75}}},\n",
       "   {'variant_id': '15-48452623-ins15-f99266b5',\n",
       "    'metadata': {'vcf_string': '15\\t48452623\\t.\\tT\\tTGCAGTGTTGATGCAT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48452623',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'TGCAGTGTTGATGCAT',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5470_5484dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48452624_48452638dup',\n",
       "     'hgvs_protein': 'p.Cys1824_Ala1828dup',\n",
       "     'most_severe_consequence': 'inframe_insertion'},\n",
       "    'annotation_raw': '{\"start\": 48452624, \"transcript_consequences\": [{\"protein_end\": 1829, \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.5470_5484dup\", \"conservation\": -1.97, \"exon\": \"45/66\", \"impact\": \"MODERATE\", \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 1828, \"cds_end\": 5485, \"codons\": \"-/TGCATCAACACTGCA\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"-/CINTA\", \"variant_allele\": \"TGCAGTGTTGATGCA\", \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"given_ref\": \"-\", \"allele_num\": 1, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cdna_end\": 5801, \"location\": \"15:48452622-48452623\", \"hgvsp\": \"NP_000129.3:p.Cys1824_Ala1828dup\", \"consequence_terms\": [\"inframe_insertion\"], \"cdna_start\": 5800, \"hgvs_offset\": -1, \"cds_start\": 5484, \"used_ref\": \"-\"}], \"assembly_name\": \"GRCh38\", \"end\": 48452623, \"input\": \"NC_000015.10:g.48452624_48452638dup\", \"id\": \"NC_000015.10:g.48452624_48452638dup\", \"allele_string\": \"-/GCAGTGTTGATGCAT\", \"strand\": 1, \"minimised\": 1, \"vcf_string\": \"15-48452623-T-TGCAGTGTTGATGCAT\", \"variant_class\": \"insertion\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"inframe_insertion\"}',\n",
       "    'coordinates': [{'pca_x': 0.0016202494673704382,\n",
       "      'pca_y': 0.023808189287798535},\n",
       "     {'t-sne_x': 6.899933416918907, 't-sne_y': 8.32249997554494},\n",
       "     {'umap_x': 3.969294309616089, 'umap_y': 3.407350540161133}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.10584219545125961,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.10631348192691803,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452595-C-A',\n",
       "      'distance': 0.1072947233915329,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.10729973018169403,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446814-C-T',\n",
       "      'distance': 0.10772409290075302,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.10786977410316467,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.10806290805339813,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.10815249383449554,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.1081533133983612,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.10827551782131195,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448851-C-T',\n",
       "      'distance': 0.1084090918302536,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.10846252739429474,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468504-C-T',\n",
       "      'distance': 0.10848832130432129,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.10854817926883698,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-T',\n",
       "      'distance': 0.10855882614850998,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452567-C-T',\n",
       "      'distance': 0.10856126993894577,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.10862899571657181,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.10867202281951904,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421981-C-T',\n",
       "      'distance': 0.10868766903877258,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520708-C-A',\n",
       "      'distance': 0.10869255661964417,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48448860-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48448860\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48448860',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5579G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48448860C>A',\n",
       "     'hgvs_protein': 'p.Cys1860Phe',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"minimised\": 1, \"vcf_string\": \"15-48448860-C-A\", \"id\": \"NC_000015.10:g.48448860C>A\", \"end\": 48448860, \"input\": \"NC_000015.10:g.48448860C>A\", \"allele_string\": \"C/A\", \"strand\": 1, \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM085423\", \"end\": 48448860, \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48448860}, {\"end\": 48448860, \"id\": \"CM098697\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48448860}], \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"protein_end\": 1860, \"gene_symbol\": \"FBN1\", \"mastermind_mmid3\": \"FBN1:C1860F\", \"gene_id\": \"2200\", \"blosum62\": -2, \"mutationtaster_pred\": \"D\", \"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.5579G>T\", \"sift_score\": 0, \"alphamissense\": {\"am_pathogenicity\": 0.9983, \"am_class\": \"likely_pathogenic\"}, \"cadd_phred\": 29, \"impact\": \"MODERATE\", \"polyphen_prediction\": \"probably_damaging\", \"conservation\": 3.95, \"exon\": \"46/66\", \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"cds_end\": 5579, \"protein_start\": 1860, \"codons\": \"tGc/tTc\", \"lrt_pred\": \"invalid_field\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"sift_prediction\": \"deleterious_low_confidence\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.996, \"canonical\": 1, \"amino_acids\": \"C/F\", \"loeuf\": 0.105, \"variant_allele\": \"A\", \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"given_ref\": \"C\", \"spliceai\": {\"DS_DL\": 0, \"DP_AL\": 15, \"DP_DG\": 31, \"DP_DL\": 5, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_DG\": 0, \"DP_AG\": 50}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48448859, \"type\": \"SimpleInterval\", \"end\": 48448860}}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}}, \"allele_num\": 1, \"mastermind_counts\": \"4|4|4\", \"cadd_raw\": 5.17256, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cdna_end\": 5895, \"hgvsp\": \"NP_000129.3:p.Cys1860Phe\", \"location\": \"15:48448860\", \"aa\": \"C\", \"clinpred\": 0.99996018409729, \"consequence_terms\": [\"missense_variant\"], \"cdna_start\": 5895, \"cds_start\": 5579, \"used_ref\": \"C\", \"eve_score\": 0.783303438171452}], \"start\": 48448860}',\n",
       "    'coordinates': [{'pca_x': 0.03993470787516179,\n",
       "      'pca_y': 0.08861325596827277},\n",
       "     {'t-sne_x': 21.677178616111213, 't-sne_y': 13.029575242037827},\n",
       "     {'umap_x': 4.410282611846924, 'umap_y': 7.312246799468994}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.001858011819422245,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48444579-C-A',\n",
       "      'distance': 0.0018882285803556442,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.0019235166255384684,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.0020546664018183947,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48463904-C-A',\n",
       "      'distance': 0.0021057557314634323,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.002193652791902423,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.0023031365126371384,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.0024195611476898193,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596318-C-A',\n",
       "      'distance': 0.002473564585670829,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.002646172186359763,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485463-C-A',\n",
       "      'distance': 0.0027452942449599504,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452595-C-A',\n",
       "      'distance': 0.0030411044135689735,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.0032206648029386997,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.003235928248614073,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537721-C-A',\n",
       "      'distance': 0.0033056368120014668,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.0035618767142295837,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497358-C-A',\n",
       "      'distance': 0.0035697852727025747,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415695-C-A',\n",
       "      'distance': 0.003643740899860859,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.0036923312582075596,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452567-C-T',\n",
       "      'distance': 0.003699346212670207,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48448766-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48448766\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48448766',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5671+2T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48448766A>C',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"splice_donor_variant\", \"minimised\": 1, \"vcf_string\": \"15-48448766-A-C\", \"end\": 48448766, \"input\": \"NC_000015.10:g.48448766A>C\", \"id\": \"NC_000015.10:g.48448766A>C\", \"allele_string\": \"A/C\", \"strand\": 1, \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48448766, \"id\": \"CS085780\", \"start\": 48448766, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}], \"start\": 48448766, \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_raw\": 6.630873, \"allele_num\": 1, \"mastermind_counts\": \"3|3|9\", \"aa\": \"A\", \"location\": \"15:48448766\", \"maxentscan_ref\": 10.239, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"start\": 48448765, \"end\": 48448766, \"type\": \"SimpleInterval\"}}, \"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"spliceai\": {\"DP_DG\": -2, \"DP_AL\": 13, \"DS_DL\": 1, \"DP_DL\": 2, \"DS_AG\": 0, \"DS_DG\": 0.86, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": -2}, \"ada_score\": 0.999988247793601, \"given_ref\": \"A\", \"used_ref\": \"A\", \"consequence_terms\": [\"splice_donor_variant\"], \"rf_score\": 0.932, \"maxentscan_diff\": 7.647, \"hgvsc\": \"NM_000138.5:c.5671+2T>G\", \"conservation\": 3.95, \"intron\": \"46/65\", \"impact\": \"HIGH\", \"cadd_phred\": 35, \"gene_symbol\": \"FBN1\", \"lof_info\": \"INTRON_SIZE:1945\", \"gene_id\": \"2200\", \"mastermind_mmid3\": \"FBN1:D1891sd\", \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"genesplicer\": \"loss/donor/48448765-48448766/Medium/2.047190\", \"variant_allele\": \"C\", \"loeuf\": 0.105, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"maxentscan_alt\": 2.591, \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396}], \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': -0.23422505793636922,\n",
       "      'pca_y': -0.0256385861147335},\n",
       "     {'t-sne_x': -15.61724969326858, 't-sne_y': -28.582835099784504},\n",
       "     {'umap_x': 8.18198299407959, 'umap_y': 13.085606575012207}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48437320-A-C',\n",
       "      'distance': 0.0011590822832658887,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48613009-C-A',\n",
       "      'distance': 0.0027366706635802984,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508581-C-A',\n",
       "      'distance': 0.0032786799129098654,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437320-A-G',\n",
       "      'distance': 0.0032962551340460777,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-A',\n",
       "      'distance': 0.0034166995901614428,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-A',\n",
       "      'distance': 0.003450455842539668,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465789-C-A',\n",
       "      'distance': 0.003541642101481557,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465788-A-G',\n",
       "      'distance': 0.003582667326554656,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-T',\n",
       "      'distance': 0.004898132756352425,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.005002220161259174,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444540-C-T',\n",
       "      'distance': 0.005119658075273037,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.0051214974373579025,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.0051458850502967834,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474254-C-T',\n",
       "      'distance': 0.005160353612154722,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.005165083333849907,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.00517635140568018,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.005361070390790701,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.005411788821220398,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441720-C-T',\n",
       "      'distance': 0.005456246435642242,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428345-C-T',\n",
       "      'distance': 0.005471107549965382,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48446748-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48446748\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48446748',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5746T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48446748A>T',\n",
       "     'hgvs_protein': 'p.Cys1916Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48446748, \"transcript_consequences\": [{\"variant_allele\": \"T\", \"loeuf\": 0.105, \"genesplicer\": \"gain/acceptor/48446748-48446749/Medium/2.100472\", \"canonical\": 1, \"amino_acids\": \"C/S\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.985, \"sift_prediction\": \"deleterious_low_confidence\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"lrt_pred\": \"invalid_field\", \"codons\": \"Tgc/Agc\", \"protein_start\": 1916, \"cds_end\": 5746, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"conservation\": 3.95, \"exon\": \"47/66\", \"impact\": \"MODERATE\", \"polyphen_prediction\": \"probably_damaging\", \"cadd_phred\": 29.4, \"sift_score\": 0, \"alphamissense\": {\"am_pathogenicity\": 0.9995, \"am_class\": \"likely_pathogenic\"}, \"hgvsc\": \"NM_000138.5:c.5746T>A\", \"eve_class\": \"Pathogenic\", \"mutationtaster_pred\": \"D\", \"gene_id\": \"2200\", \"blosum62\": -1, \"mastermind_mmid3\": \"FBN1:C1916S\", \"gene_symbol\": \"FBN1\", \"protein_end\": 1916, \"used_ref\": \"A\", \"eve_score\": 0.799665189908282, \"cds_start\": 5746, \"cdna_start\": 6062, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.999877691268921, \"aa\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys1916Ser\", \"location\": \"15:48446748\", \"cdna_end\": 6062, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_raw\": 5.237206, \"allele_num\": 1, \"mastermind_counts\": \"4|4|4\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48446747, \"type\": \"SimpleInterval\", \"end\": 48446748}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}, \"type\": \"Allele\"}, \"spliceai\": {\"DS_AG\": 0, \"DP_DL\": -42, \"DP_AL\": -43, \"DS_DL\": 0, \"DP_DG\": -44, \"DP_AG\": -2, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0}, \"given_ref\": \"A\", \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"]}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"start\": 48446748, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM098704\", \"end\": 48446748, \"strand\": 1}], \"allele_string\": \"A/T\", \"strand\": 1, \"end\": 48446748, \"input\": \"NC_000015.10:g.48446748A>T\", \"id\": \"NC_000015.10:g.48446748A>T\", \"vcf_string\": \"15-48446748-A-T\", \"minimised\": 1, \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.04332124217736527,\n",
       "      'pca_y': 0.07609760045317579},\n",
       "     {'t-sne_x': 14.506334240025886, 't-sne_y': 9.203156826247113},\n",
       "     {'umap_x': 3.334584951400757, 'umap_y': 4.692748546600342}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48446747-C-T',\n",
       "      'distance': 0.0026238271966576576,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.0028726444579660892,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430791-A-T',\n",
       "      'distance': 0.002998052630573511,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428439-A-T',\n",
       "      'distance': 0.0031582959927618504,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596325-A-T',\n",
       "      'distance': 0.003168728668242693,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445421-A-T',\n",
       "      'distance': 0.0033728484995663166,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446811-A-G',\n",
       "      'distance': 0.0035595714580267668,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.003576329443603754,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.0035821273922920227,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.0036606392823159695,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.003692941041663289,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.0037345027085393667,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.003790460526943207,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444579-C-T',\n",
       "      'distance': 0.003799008671194315,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445382-A-G',\n",
       "      'distance': 0.003943008370697498,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.00394650362432003,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452567-C-T',\n",
       "      'distance': 0.00397980073466897,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.003988791257143021,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.0039958469569683075,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432976-C-T',\n",
       "      'distance': 0.004000512417405844,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48446718-T-C',\n",
       "    'metadata': {'vcf_string': '15\\t48446718\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48446718',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5776A>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48446718T>C',\n",
       "     'hgvs_protein': 'p.Asn1926Asp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48446718-T-C\", \"minimised\": 1, \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\", \"variant_class\": \"SNV\", \"start\": 48446718, \"transcript_consequences\": [{\"cdna_start\": 6092, \"clinpred\": 0.807651400566101, \"consequence_terms\": [\"missense_variant\"], \"used_ref\": \"T\", \"eve_score\": 0.433388610249144, \"cds_start\": 5776, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"end\": 48446718, \"type\": \"SimpleInterval\", \"start\": 48446717}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"C\"}}, \"given_ref\": \"T\", \"spliceai\": {\"DS_AL\": 0, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": -1, \"DP_DG\": -45, \"DS_DL\": 0, \"DP_AL\": -13, \"DP_DL\": -6, \"DS_AG\": 0}, \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"hgvsp\": \"NP_000129.3:p.Asn1926Asp\", \"location\": \"15:48446718\", \"aa\": \"T\", \"cdna_end\": 6092, \"cadd_raw\": 4.875437, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"mastermind_counts\": \"6|7|7\", \"allele_num\": 1, \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"sift_prediction\": \"deleterious_low_confidence\", \"cds_end\": 5776, \"codons\": \"Aat/Gat\", \"protein_start\": 1926, \"lrt_pred\": \"invalid_field\", \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"loeuf\": 0.105, \"variant_allele\": \"C\", \"amino_acids\": \"N/D\", \"canonical\": 1, \"polyphen_score\": 0.988, \"mane_select\": \"ENST00000316623.10\", \"blosum62\": 1, \"gene_id\": \"2200\", \"mutationtaster_pred\": \"D\", \"mastermind_mmid3\": \"FBN1:N1926D\", \"gene_symbol\": \"FBN1\", \"protein_end\": 1926, \"impact\": \"MODERATE\", \"polyphen_prediction\": \"probably_damaging\", \"exon\": \"47/66\", \"conservation\": 3.95, \"sift_score\": 0, \"alphamissense\": {\"am_class\": \"ambiguous\", \"am_pathogenicity\": 0.3592}, \"cadd_phred\": 27.2, \"hgvsc\": \"NM_000138.5:c.5776A>G\", \"eve_class\": \"Uncertain\"}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"strand\": 1, \"id\": \"CM118914\", \"end\": 48446718, \"allele_string\": \"HGMD_MUTATION\", \"start\": 48446718, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000663816\", \"RCV000476435\", \"VCV000406371\", \"RCV001181375\", \"RCV001193546\"]}, \"clin_sig\": [\"uncertain_significance\"], \"end\": 48446718, \"frequencies\": {\"C\": {\"gnomadg_nfe\": 1.47e-05, \"gnomade_nfe\": 5.414e-06, \"gnomade_afr\": 0, \"gnomade_sas\": 0, \"gnomade_remaining\": 0, \"gnomade_fin\": 0, \"gnomadg_eas\": 0, \"gnomadg_remaining\": 0, \"gnomadg_ami\": 0, \"gnomade_asj\": 0, \"gnomadg\": 6.57e-06, \"gnomadg_asj\": 0, \"gnomadg_sas\": 0, \"gnomade\": 4.116e-06, \"gnomadg_afr\": 0, \"gnomade_eas\": 0, \"gnomadg_amr\": 0, \"gnomadg_mid\": 0, \"gnomade_amr\": 0, \"gnomadg_fin\": 0, \"gnomade_mid\": 0}}, \"id\": \"rs1060501097\", \"allele_string\": \"T/C\", \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"C:uncertain_significance\", \"start\": 48446718, \"seq_region_name\": \"15\"}], \"strand\": 1, \"allele_string\": \"T/C\", \"input\": \"NC_000015.10:g.48446718T>C\", \"end\": 48446718, \"id\": \"NC_000015.10:g.48446718T>C\"}',\n",
       "    'coordinates': [{'pca_x': 0.03614718495359051,\n",
       "      'pca_y': 0.08119740356123664},\n",
       "     {'t-sne_x': 0.021945110137406288, 't-sne_y': 1.1864146463407483},\n",
       "     {'umap_x': 5.063179969787598, 'umap_y': 2.925065040588379}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48446775-T-C',\n",
       "      'distance': 0.0010086586698889732,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446816-T-C',\n",
       "      'distance': 0.001214068615809083,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48463207-T-C',\n",
       "      'distance': 0.002416792092844844,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.0026872092857956886,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415671-T-C',\n",
       "      'distance': 0.0028301002457737923,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513581-T-C',\n",
       "      'distance': 0.002858093474060297,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415605-T-C',\n",
       "      'distance': 0.0028717368841171265,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446747-C-T',\n",
       "      'distance': 0.0029297680594027042,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420719-T-C',\n",
       "      'distance': 0.002942024264484644,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468468-T-C',\n",
       "      'distance': 0.003011156339198351,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488148-T-C',\n",
       "      'distance': 0.00315381889231503,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430727-T-C',\n",
       "      'distance': 0.003194555640220642,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505081-T-C',\n",
       "      'distance': 0.0031995209865272045,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472607-T-C',\n",
       "      'distance': 0.00320985889993608,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446795-C-T',\n",
       "      'distance': 0.0032130710314959288,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432976-C-T',\n",
       "      'distance': 0.0032719632145017385,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487431-T-C',\n",
       "      'distance': 0.003275119699537754,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.003325758734717965,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445408-T-C',\n",
       "      'distance': 0.003396733431145549,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48444579-C-T',\n",
       "      'distance': 0.0034360273275524378,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48445394-CT-C',\n",
       "    'metadata': {'vcf_string': '15\\t48445394\\t.\\tCT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48445394',\n",
       "     'ref_allele': 'CT',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5898del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48445395del',\n",
       "     'hgvs_protein': 'p.Asp1967MetfsTer13',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"deletion\", \"most_severe_consequence\": \"frameshift_variant\", \"seq_region_name\": \"15\", \"minimised\": 1, \"vcf_string\": \"15-48445394-CT-C\", \"id\": \"NC_000015.10:g.48445395del\", \"end\": 48445395, \"input\": \"NC_000015.10:g.48445395del\", \"allele_string\": \"T/-\", \"strand\": 1, \"colocated_variants\": [{\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48445395, \"id\": \"CD983902\", \"end\": 48445395, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1}, {\"allele_string\": \"COSMIC_MUTATION\", \"end\": 48445395, \"id\": \"COSV57311742\", \"strand\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM6532059\"]}, \"somatic\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48445395}, {\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48445395, \"clin_sig\": [\"pathogenic\"], \"end\": 48445395, \"id\": \"rs1555395747\", \"allele_string\": \"T/-\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000663828\", \"VCV000549313\"]}}], \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"protein_id\": \"NP_000129.3\", \"strand\": -1, \"mastermind_counts\": \"6|6|6\", \"allele_num\": 1, \"hgvsp\": \"NP_000129.3:p.Asp1967MetfsTer13\", \"location\": \"15:48445395\", \"aa\": \"T\", \"cdna_end\": 6214, \"lof\": \"HC\", \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"start\": 48445394, \"end\": 48445395, \"type\": \"SimpleInterval\"}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"given_ref\": \"T\", \"spliceai\": {\"DP_AG\": 33, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_DG\": 0, \"DP_DL\": -18, \"DS_AG\": 0.01, \"DS_DL\": 0, \"DP_AL\": 47, \"DP_DG\": -14}, \"cds_start\": 5898, \"used_ref\": \"T\", \"consequence_terms\": [\"frameshift_variant\"], \"cdna_start\": 6214, \"hgvsc\": \"NM_000138.5:c.5898del\", \"impact\": \"HIGH\", \"exon\": \"48/66\", \"conservation\": -3.37, \"lof_info\": \"PERCENTILE:0.684540389972145,GERP_DIST:3139.6147431016,BP_DIST:2700,DIST_FROM_LAST_EXON:2311,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"protein_end\": 1966, \"gene_id\": \"2200\", \"mastermind_mmid3\": \"FBN1:D1967fs\", \"amino_acids\": \"P/X\", \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"variant_allele\": \"-\", \"loeuf\": 0.105, \"cds_end\": 5898, \"codons\": \"ccA/cc\", \"protein_start\": 1966, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396}], \"start\": 48445395}',\n",
       "    'coordinates': [{'pca_x': 0.05598027841166808,\n",
       "      'pca_y': 0.033425936504634886},\n",
       "     {'t-sne_x': 0.488828375201736, 't-sne_y': 7.569103337146755},\n",
       "     {'umap_x': 3.772564172744751, 'umap_y': 3.245704174041748}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.10968112200498581,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.11008220911026001,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.11240847408771515,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.11282380670309067,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.11355346441268921,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.11392255127429962,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445419-G-T',\n",
       "      'distance': 0.11397790908813477,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.11529520153999329,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.11606629192829132,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.11611460894346237,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.11614619195461273,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.11618304997682571,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.11629997938871384,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.11644747853279114,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427646-A-T',\n",
       "      'distance': 0.11653127521276474,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.1166161298751831,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421973-G-A',\n",
       "      'distance': 0.11696277558803558,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.11708705127239227,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.11709275841712952,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.11712387204170227,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48445381-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48445381\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48445381',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5912G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48445381C>T',\n",
       "     'hgvs_protein': 'p.Cys1971Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"C/T\", \"strand\": 1, \"id\": \"NC_000015.10:g.48445381C>T\", \"end\": 48445381, \"input\": \"NC_000015.10:g.48445381C>T\", \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"eve_score\": 0.712321007526295, \"used_ref\": \"C\", \"cds_start\": 5912, \"cdna_start\": 6228, \"clinpred\": 0.999801814556122, \"consequence_terms\": [\"missense_variant\"], \"location\": \"15:48445381\", \"hgvsp\": \"NP_000129.3:p.Cys1971Tyr\", \"aa\": \"C\", \"cdna_end\": 6228, \"cadd_raw\": 5.183994, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"allele_num\": 1, \"mastermind_counts\": \"5|13|13\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"end\": 48445381, \"type\": \"SimpleInterval\", \"start\": 48445380}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"T\"}}, \"given_ref\": \"C\", \"spliceai\": {\"DP_DL\": 23, \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AL\": 46, \"DP_DG\": -1, \"DP_AG\": 12, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_DG\": 0}, \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"loeuf\": 0.105, \"variant_allele\": \"T\", \"amino_acids\": \"C/Y\", \"canonical\": 1, \"polyphen_score\": 0.996, \"mane_select\": \"ENST00000316623.10\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"sift_prediction\": \"deleterious_low_confidence\", \"protein_start\": 1971, \"cds_end\": 5912, \"codons\": \"tGt/tAt\", \"lrt_pred\": \"invalid_field\", \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"impact\": \"MODERATE\", \"polyphen_prediction\": \"probably_damaging\", \"exon\": \"48/66\", \"conservation\": 3.7, \"alphamissense\": {\"am_pathogenicity\": 0.9985, \"am_class\": \"likely_pathogenic\"}, \"sift_score\": 0, \"cadd_phred\": 29.1, \"hgvsc\": \"NM_000138.5:c.5912G>A\", \"eve_class\": \"Pathogenic\", \"blosum62\": -2, \"gene_id\": \"2200\", \"mutationtaster_pred\": \"A\", \"mastermind_mmid3\": \"FBN1:C1971Y\", \"gene_symbol\": \"FBN1\", \"protein_end\": 1971}], \"start\": 48445381, \"colocated_variants\": [{\"start\": 48445381, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM013936\", \"end\": 48445381}, {\"seq_region_name\": \"15\", \"start\": 48445381, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM1514538\", \"end\": 48445381, \"strand\": 1}, {\"pubmed\": [11700157], \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"T:likely_pathogenic\", \"start\": 48445381, \"seq_region_name\": \"15\", \"strand\": 1, \"var_synonyms\": {\"UniProt\": [\"VAR_018025\"], \"ClinVar\": [\"RCV002352081\", \"RCV000663831\", \"VCV000549316\"]}, \"clin_sig\": [\"likely_pathogenic\"], \"end\": 48445381, \"id\": \"rs111239111\", \"allele_string\": \"C/G/T\"}], \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"variant_class\": \"SNV\", \"vcf_string\": \"15-48445381-C-T\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.04471327931460184,\n",
       "      'pca_y': 0.07826915856836214},\n",
       "     {'t-sne_x': 0.8848963436578435, 't-sne_y': 8.288551815175635},\n",
       "     {'umap_x': 3.608887195587158, 'umap_y': 3.3248612880706787}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.0017198019195348024,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445492-C-T',\n",
       "      'distance': 0.0017867358401417732,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.002209173981100321,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446747-C-T',\n",
       "      'distance': 0.002523064147680998,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445408-T-C',\n",
       "      'distance': 0.0029891817830502987,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48452567-C-T',\n",
       "      'distance': 0.003209049580618739,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515393-A-G',\n",
       "      'distance': 0.003306009341031313,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456714-C-T',\n",
       "      'distance': 0.0033719465136528015,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441771-C-T',\n",
       "      'distance': 0.0034090164117515087,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.003578296396881342,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.0036176813300698996,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444579-C-T',\n",
       "      'distance': 0.0036375971976667643,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448773-C-T',\n",
       "      'distance': 0.0036420600954443216,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513581-T-C',\n",
       "      'distance': 0.0038184679578989744,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456729-C-T',\n",
       "      'distance': 0.0038395675364881754,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48537718-C-T',\n",
       "      'distance': 0.0038574328646063805,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456687-C-T',\n",
       "      'distance': 0.003912849817425013,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596318-C-T',\n",
       "      'distance': 0.0039551700465381145,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465573-C-T',\n",
       "      'distance': 0.0040549226105213165,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495576-C-T',\n",
       "      'distance': 0.004076381213963032,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48444619-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48444619\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48444619',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.5959G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48444619C>G',\n",
       "     'hgvs_protein': 'p.Gly1987Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"end\": 48444619, \"input\": \"NC_000015.10:g.48444619C>G\", \"id\": \"NC_000015.10:g.48444619C>G\", \"strand\": 1, \"allele_string\": \"C/G\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"start\": 48444619, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48444619, \"id\": \"CM012600\"}, {\"somatic\": 1, \"strand\": 1, \"var_synonyms\": {\"COSMIC\": [\"COSM3887079\"]}, \"end\": 48444619, \"id\": \"COSV57317350\", \"allele_string\": \"COSMIC_MUTATION\", \"phenotype_or_disease\": 1, \"start\": 48444619, \"seq_region_name\": \"15\"}, {\"var_synonyms\": {\"ClinVar\": [\"RCV000155904\", \"VCV000179119\", \"RCV001269653\", \"RCV002313003\", \"RCV003998292\"], \"UniProt\": [\"VAR_076107\"]}, \"strand\": 1, \"clin_sig\": [\"uncertain_significance\", \"likely_pathogenic\"], \"id\": \"rs727504642\", \"end\": 48444619, \"allele_string\": \"C/G\", \"pubmed\": [24033266], \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"G:likely_pathogenic;G:conflicting_classifications_of_pathogenicity\", \"start\": 48444619, \"seq_region_name\": \"15\"}], \"start\": 48444619, \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"given_ref\": \"C\", \"spliceai\": {\"DP_DG\": 31, \"DP_AL\": -1, \"DS_DL\": 0.08, \"DP_DL\": 0, \"DS_AG\": 0.01, \"DS_DG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": 20}, \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48444619, \"start\": 48444618}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"type\": \"Allele\"}, \"allele_num\": 1, \"mastermind_counts\": \"9|11|11\", \"cadd_raw\": 5.036977, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cdna_end\": 6275, \"hgvsp\": \"NP_000129.3:p.Gly1987Arg\", \"location\": \"15:48444619\", \"aa\": \"C\", \"clinpred\": 0.983109533786774, \"consequence_terms\": [\"missense_variant\"], \"cdna_start\": 6275, \"cds_start\": 5959, \"used_ref\": \"C\", \"eve_score\": 0.721961359143749, \"protein_end\": 1987, \"gene_symbol\": \"FBN1\", \"mastermind_mmid3\": \"FBN1:G1987R\", \"blosum62\": -2, \"gene_id\": \"2200\", \"mutationtaster_pred\": \"A\", \"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.5959G>C\", \"alphamissense\": {\"am_pathogenicity\": 0.9953, \"am_class\": \"likely_pathogenic\"}, \"sift_score\": 0, \"cadd_phred\": 28.1, \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"conservation\": 3.95, \"exon\": \"49/66\", \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"codons\": \"Ggt/Cgt\", \"protein_start\": 1987, \"cds_end\": 5959, \"lrt_pred\": \"invalid_field\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"sift_prediction\": \"deleterious_low_confidence\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 1, \"amino_acids\": \"G/R\", \"canonical\": 1, \"loeuf\": 0.105, \"variant_allele\": \"G\"}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\", \"minimised\": 1, \"vcf_string\": \"15-48444619-C-G\"}',\n",
       "    'coordinates': [{'pca_x': 0.05498306562033911,\n",
       "      'pca_y': 0.08980222317595588},\n",
       "     {'t-sne_x': 39.99886159113218, 't-sne_y': 4.850600523313533},\n",
       "     {'umap_x': 1.1486736536026, 'umap_y': 7.348292350769043}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48445493-A-G',\n",
       "      'distance': 0.010042710229754448,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495162-C-G',\n",
       "      'distance': 0.010089745745062828,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445382-A-G',\n",
       "      'distance': 0.010244786739349365,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444585-C-T',\n",
       "      'distance': 0.010375969111919403,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444611-A-C',\n",
       "      'distance': 0.010390856303274632,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508588-A-G',\n",
       "      'distance': 0.010744133964180946,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446712-A-G',\n",
       "      'distance': 0.01092679612338543,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.011635228991508484,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.011651262640953064,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.01210581511259079,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.012170873582363129,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448813-A-G',\n",
       "      'distance': 0.012301510199904442,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452589-G-A',\n",
       "      'distance': 0.012364943511784077,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505081-T-C',\n",
       "      'distance': 0.012456660158932209,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422024-A-G',\n",
       "      'distance': 0.01267323363572359,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446811-A-G',\n",
       "      'distance': 0.01298103854060173,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.013046098873019218,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596319-A-G',\n",
       "      'distance': 0.013320311903953552,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.013340507633984089,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425414-A-G',\n",
       "      'distance': 0.01340350229293108,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48441771-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48441771\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48441771',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6113G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48441771C>A',\n",
       "     'hgvs_protein': 'p.Cys2038Phe',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\", \"minimised\": 1, \"vcf_string\": \"15-48441771-C-A\", \"input\": \"NC_000015.10:g.48441771C>A\", \"end\": 48441771, \"id\": \"NC_000015.10:g.48441771C>A\", \"allele_string\": \"C/A\", \"strand\": 1, \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM055258\", \"end\": 48441771, \"phenotype_or_disease\": 1, \"start\": 48441771, \"seq_region_name\": \"15\"}, {\"id\": \"CM098707\", \"end\": 48441771, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48441771, \"phenotype_or_disease\": 1}], \"start\": 48441771, \"transcript_consequences\": [{\"cadd_phred\": 29.1, \"alphamissense\": {\"am_pathogenicity\": 0.9966, \"am_class\": \"likely_pathogenic\"}, \"sift_score\": 0, \"exon\": \"50/66\", \"conservation\": 3.95, \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.6113G>T\", \"mastermind_mmid3\": \"FBN1:C2038F\", \"mutationtaster_pred\": \"D\", \"blosum62\": -2, \"gene_id\": \"2200\", \"protein_end\": 2038, \"gene_symbol\": \"FBN1\", \"loeuf\": 0.105, \"variant_allele\": \"A\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.918, \"amino_acids\": \"C/F\", \"canonical\": 1, \"sift_prediction\": \"deleterious_low_confidence\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"lrt_pred\": \"invalid_field\", \"cds_end\": 6113, \"codons\": \"tGt/tTt\", \"protein_start\": 2038, \"cdna_end\": 6429, \"aa\": \"C\", \"location\": \"15:48441771\", \"hgvsp\": \"NP_000129.3:p.Cys2038Phe\", \"mastermind_counts\": \"3|3|3\", \"allele_num\": 1, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cadd_raw\": 5.18868, \"spliceai\": {\"DS_AG\": 0.01, \"DP_DL\": -42, \"DS_DL\": 0, \"DP_AL\": 48, \"DP_DG\": -50, \"DP_AG\": -12, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_DG\": 0}, \"given_ref\": \"C\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48441771, \"type\": \"SimpleInterval\", \"start\": 48441770}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"type\": \"Allele\"}, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"C\", \"eve_score\": 0.764943508427423, \"cds_start\": 6113, \"cdna_start\": 6429, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.999923944473267}], \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': 0.04009377788607313,\n",
       "      'pca_y': 0.08725546203796689},\n",
       "     {'t-sne_x': 21.154928727611768, 't-sne_y': 12.852438055639254},\n",
       "     {'umap_x': 4.331643581390381, 'umap_y': 7.3507399559021}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463976-C-A',\n",
       "      'distance': 0.001259177690371871,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.0012918624561280012,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537721-C-A',\n",
       "      'distance': 0.0013995019253343344,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463904-C-A',\n",
       "      'distance': 0.001478709513321519,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.001480354811064899,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.0015142718330025673,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497358-C-A',\n",
       "      'distance': 0.001531727146357298,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-A',\n",
       "      'distance': 0.0017667923821136355,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485463-C-A',\n",
       "      'distance': 0.0018295238260179758,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415582-C-A',\n",
       "      'distance': 0.0018758143996819854,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.0019365926273167133,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596318-C-A',\n",
       "      'distance': 0.0020282771438360214,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425783-C-A',\n",
       "      'distance': 0.002117781201377511,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.0022704494185745716,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48444579-C-A',\n",
       "      'distance': 0.0024079822469502687,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48452595-C-A',\n",
       "      'distance': 0.002538397442549467,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.0025753197260200977,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441771-C-T',\n",
       "      'distance': 0.002660328522324562,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415695-C-A',\n",
       "      'distance': 0.002854405902326107,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468522-C-A',\n",
       "      'distance': 0.0030833519995212555,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48441720-CC-C',\n",
       "    'metadata': {'vcf_string': '15\\t48441720\\t.\\tCC\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48441720',\n",
       "     'ref_allele': 'CC',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6163+1del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48441721del',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"allele_string\": \"C/-\", \"id\": \"NC_000015.10:g.48441721del\", \"end\": 48441721, \"input\": \"NC_000015.10:g.48441721del\", \"assembly_name\": \"GRCh38\", \"start\": 48441721, \"transcript_consequences\": [{\"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"lof\": \"HC\", \"mane\": [\"MANE_Select\"], \"ga4gh_vrs\": {\"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48441721, \"start\": 48441720}}, \"type\": \"Allele\"}, \"gene_id\": \"2200\", \"spliceai\": {\"DP_DL\": 1, \"DS_AG\": 0, \"DP_AL\": 39, \"DS_DL\": 1, \"DP_DG\": 2, \"DP_AG\": 17, \"SYMBOL\": \"FBN1\", \"DS_DG\": 1, \"DS_AL\": 0}, \"given_ref\": \"C\", \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"hgvsc\": \"NM_000138.5:c.6163+1del\", \"allele_num\": 1, \"aa\": \"C\", \"conservation\": 3.95, \"impact\": \"HIGH\", \"location\": \"15:48441720\", \"consequence_terms\": [\"splice_donor_variant\"], \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"canonical\": 1, \"hgvs_offset\": -1, \"mane_select\": \"ENST00000316623.10\", \"variant_allele\": \"-\", \"used_ref\": \"C\", \"loeuf\": 0.105, \"genesplicer\": \"gain/donor/48441721-48441722/Medium/5.394544\"}], \"most_severe_consequence\": \"splice_donor_variant\", \"seq_region_name\": \"15\", \"variant_class\": \"deletion\", \"vcf_string\": \"15-48441720-CC-C\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.197661720329902,\n",
       "      'pca_y': -0.01199973392222152},\n",
       "     {'t-sne_x': -22.090000856323865, 't-sne_y': -25.212516865141605},\n",
       "     {'umap_x': 8.668084144592285, 'umap_y': 12.954273223876953}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48425368-C-T',\n",
       "      'distance': 0.06768348067998886,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494203-C-G',\n",
       "      'distance': 0.06815940141677856,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460244-A-C',\n",
       "      'distance': 0.06917108595371246,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-G',\n",
       "      'distance': 0.07025820016860962,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488111-A-G',\n",
       "      'distance': 0.0725846141576767,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427566-C-A',\n",
       "      'distance': 0.07456070184707642,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.07456296682357788,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600138-C-G',\n",
       "      'distance': 0.0746353417634964,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483816-A-G',\n",
       "      'distance': 0.07483313977718353,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.07485158741474152,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.0750051736831665,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488112-C-T',\n",
       "      'distance': 0.07518769800662994,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.07521560788154602,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428345-C-T',\n",
       "      'distance': 0.0753747820854187,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-A',\n",
       "      'distance': 0.07569194585084915,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.07586269080638885,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.07638097554445267,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.07652126997709274,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456635-A-G',\n",
       "      'distance': 0.07680197805166245,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496099-C-T',\n",
       "      'distance': 0.07697751373052597,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48437779-G-T',\n",
       "    'metadata': {'vcf_string': '15\\t48437779\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48437779',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6302C>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48437779G>T',\n",
       "     'hgvs_protein': 'p.Thr2101Lys',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48437779-G-T\", \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"cdna_start\": 6618, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.177506121147633, \"eve_score\": 0.0413718845364996, \"used_ref\": \"G\", \"cds_start\": 6302, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"start\": 48437778, \"type\": \"SimpleInterval\", \"end\": 48437779}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}}, \"spliceai\": {\"DP_AG\": -2, \"DS_AL\": 0, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DP_DL\": -11, \"DP_DG\": -8, \"DS_DL\": 0.02, \"DP_AL\": -50}, \"given_ref\": \"G\", \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"aa\": \"G\", \"location\": \"15:48437779\", \"hgvsp\": \"NP_000129.3:p.Thr2101Lys\", \"cdna_end\": 6618, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_raw\": 2.452489, \"allele_num\": 1, \"mastermind_counts\": \"3|3|3\", \"sift_prediction\": \"tolerated_low_confidence\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"lrt_pred\": \"invalid_field\", \"codons\": \"aCg/aAg\", \"protein_start\": 2101, \"cds_end\": 6302, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"loeuf\": 0.105, \"variant_allele\": \"T\", \"canonical\": 1, \"amino_acids\": \"T/K\", \"polyphen_score\": 0.063, \"mane_select\": \"ENST00000316623.10\", \"mutationtaster_pred\": \"N\", \"blosum62\": -1, \"gene_id\": \"2200\", \"mastermind_mmid3\": \"FBN1:T2101K\", \"gene_symbol\": \"FBN1\", \"protein_end\": 2101, \"exon\": \"51/66\", \"conservation\": 3.07, \"polyphen_prediction\": \"benign\", \"impact\": \"MODERATE\", \"cadd_phred\": 18.78, \"sift_score\": 1, \"alphamissense\": {\"am_class\": \"likely_benign\", \"am_pathogenicity\": 0.0863}, \"hgvsc\": \"NM_000138.5:c.6302C>A\", \"eve_class\": \"Benign\"}], \"start\": 48437779, \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM020692\", \"end\": 48437779, \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48437779, \"phenotype_or_disease\": 1}, {\"seq_region_name\": \"15\", \"start\": 48437779, \"phenotype_or_disease\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"id\": \"COSV57326291\", \"end\": 48437779, \"var_synonyms\": {\"COSMIC\": [\"COSM3690420\"]}, \"strand\": 1, \"somatic\": 1}], \"allele_string\": \"G/T\", \"strand\": 1, \"id\": \"NC_000015.10:g.48437779G>T\", \"end\": 48437779, \"input\": \"NC_000015.10:g.48437779G>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.03916389041338736,\n",
       "      'pca_y': 0.07894394027982563},\n",
       "     {'t-sne_x': 5.748713764288847, 't-sne_y': -4.203717594357564},\n",
       "     {'umap_x': 4.226640701293945, 'umap_y': 4.120303153991699}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48434627-C-T',\n",
       "      'distance': 0.0031216368079185486,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468474-C-T',\n",
       "      'distance': 0.0036154284607619047,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.003619549795985222,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465825-C-T',\n",
       "      'distance': 0.0036480186972767115,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470679-A-G',\n",
       "      'distance': 0.003655934240669012,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430676-C-T',\n",
       "      'distance': 0.0036649000830948353,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437039-C-T',\n",
       "      'distance': 0.003694856772199273,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421594-C-T',\n",
       "      'distance': 0.0037023471668362617,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.00373273016884923,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437872-C-T',\n",
       "      'distance': 0.0037398773711174726,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437873-A-G',\n",
       "      'distance': 0.003778949147090316,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.0038186642341315746,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.0038775624707341194,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48537676-C-T',\n",
       "      'distance': 0.0038970294408500195,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.003919174894690514,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437369-C-T',\n",
       "      'distance': 0.003939145710319281,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489977-C-T',\n",
       "      'distance': 0.0039869933389127254,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48428427-G-A',\n",
       "      'distance': 0.003998923115432262,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437379-G-A',\n",
       "      'distance': 0.004002807661890984,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427768-G-A',\n",
       "      'distance': 0.004015019163489342,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48437077-T-A',\n",
       "    'metadata': {'vcf_string': '15\\t48437077\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48437077',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6380A>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48437077T>A',\n",
       "     'hgvs_protein': 'p.Asp2127Val',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"T/A\", \"strand\": 1, \"id\": \"NC_000015.10:g.48437077T>A\", \"input\": \"NC_000015.10:g.48437077T>A\", \"end\": 48437077, \"assembly_name\": \"GRCh38\", \"start\": 48437077, \"transcript_consequences\": [{\"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"maxentscan_alt\": 5.647, \"lrt_pred\": \"invalid_field\", \"cds_end\": 6380, \"codons\": \"gAt/gTt\", \"protein_start\": 2127, \"sift_prediction\": \"deleterious_low_confidence\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"polyphen_score\": 0.999, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"D/V\", \"loeuf\": 0.105, \"variant_allele\": \"A\", \"protein_end\": 2127, \"gene_symbol\": \"FBN1\", \"mastermind_mmid3\": \"FBN1:D2127V,FBN1:D2127sra\", \"mutationtaster_pred\": \"D\", \"gene_id\": \"2200\", \"blosum62\": -3, \"maxentscan_diff\": 1.091, \"hgvsc\": \"NM_000138.5:c.6380A>T\", \"cadd_phred\": 33, \"sift_score\": 0, \"alphamissense\": {\"am_pathogenicity\": 0.9875, \"am_class\": \"likely_pathogenic\"}, \"exon\": \"53/66\", \"conservation\": 3.95, \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"clinpred\": 0.99238787534897, \"rf_score\": 0.59, \"cdna_start\": 6696, \"cds_start\": 6380, \"used_ref\": \"T\", \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"spliceai\": {\"DP_AG\": -18, \"DS_AL\": 0, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -25, \"DS_AG\": 0, \"DP_DG\": 0, \"DS_DL\": 0, \"DP_AL\": 0}, \"ada_score\": 0.672499324927753, \"given_ref\": \"T\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48437077, \"start\": 48437076}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}}, \"mutfunc\": {\"exp\": {\"dG_wt_sd\": 0.086968, \"dG_mt_sd\": 0.018047, \"dG_mt\": 190.5938, \"dG_wt\": 189.434, \"ddG\": 1.1598, \"ddG_sd\": 0.082089}}, \"mastermind_counts\": \"5|18|27\", \"allele_num\": 1, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cadd_raw\": 5.641545, \"cdna_end\": 6696, \"maxentscan_ref\": 6.738, \"aa\": \"T\", \"hgvsp\": \"NP_000129.3:p.Asp2127Val\", \"location\": \"15:48437077\"}], \"colocated_variants\": [{\"seq_region_name\": \"15\", \"start\": 48437077, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM106074\", \"end\": 48437077, \"strand\": 1}, {\"var_synonyms\": {\"ClinVar\": [\"RCV001730115\", \"VCV001299642\", \"RCV001799127\", \"VCV001325439\", \"RCV002246051\", \"RCV001933590\", \"VCV001427197\"]}, \"strand\": 1, \"clin_sig\": [\"uncertain_significance\", \"likely_pathogenic\"], \"allele_string\": \"T/A/C/G\", \"id\": \"rs2141240523\", \"end\": 48437077, \"phenotype_or_disease\": 1, \"pubmed\": [8136837], \"start\": 48437077, \"clin_sig_allele\": \"G:uncertain_significance;C:likely_pathogenic;A:likely_pathogenic;A:uncertain_significance\", \"seq_region_name\": \"15\"}], \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"variant_class\": \"SNV\", \"vcf_string\": \"15-48437077-T-A\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.033212349742115246,\n",
       "      'pca_y': 0.05171766608914608},\n",
       "     {'t-sne_x': -13.266025315407534, 't-sne_y': -3.3551690178985085},\n",
       "     {'umap_x': 0.4370464086532593, 'umap_y': 15.197423934936523}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.004758917726576328,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.00530952587723732,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.0063583729788661,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.006442110054194927,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.00704836193472147,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.007593642920255661,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.007730080746114254,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.007873475551605225,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.008484319783747196,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.008578123524785042,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.008646154776215553,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.008655600249767303,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.00878666341304779,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.008905522525310516,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.008994041010737419,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.009619968011975288,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.010049618780612946,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.010330159217119217,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448768-C-G',\n",
       "      'distance': 0.011752733960747719,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.012097828090190887,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48437321-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48437321\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48437321',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6379+1G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48437321C>A',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"vcf_string\": \"15-48437321-C-A\", \"variant_class\": \"SNV\", \"most_severe_consequence\": \"splice_donor_variant\", \"seq_region_name\": \"15\", \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"start\": 48437321, \"seq_region_name\": \"15\", \"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48437321, \"id\": \"CS085782\"}, {\"phenotype_or_disease\": 1, \"start\": 48437321, \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48437321, \"id\": \"CS157717\", \"allele_string\": \"HGMD_MUTATION\"}], \"assembly_name\": \"GRCh38\", \"start\": 48437321, \"transcript_consequences\": [{\"maxentscan_ref\": 5.069, \"aa\": \"C\", \"location\": \"15:48437321\", \"allele_num\": 1, \"mastermind_counts\": \"1|1|8\", \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_raw\": 5.884975, \"spliceai\": {\"DS_AL\": 0, \"DS_DG\": 0.01, \"SYMBOL\": \"FBN1\", \"DP_AG\": -34, \"DP_DG\": -34, \"DS_DL\": 0.28, \"DP_AL\": 1, \"DS_AG\": 0, \"DP_DL\": 1}, \"ada_score\": 0.999982627682474, \"given_ref\": \"C\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48437321, \"start\": 48437320}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"used_ref\": \"C\", \"rf_score\": 0.906, \"consequence_terms\": [\"splice_donor_variant\"], \"cadd_phred\": 33, \"conservation\": 3.95, \"impact\": \"HIGH\", \"intron\": \"52/65\", \"maxentscan_diff\": 8.504, \"hgvsc\": \"NM_000138.5:c.6379+1G>T\", \"mastermind_mmid3\": \"FBN1:D2127sd\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"lof_info\": \"INTRON_SIZE:244\", \"variant_allele\": \"A\", \"loeuf\": 0.105, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"transcript_id\": \"NM_000138.5\", \"maxentscan_alt\": -3.435, \"biotype\": \"protein_coding\"}], \"id\": \"NC_000015.10:g.48437321C>A\", \"input\": \"NC_000015.10:g.48437321C>A\", \"end\": 48437321, \"allele_string\": \"C/A\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': -0.24202715653695242,\n",
       "      'pca_y': -0.025912371617958976},\n",
       "     {'t-sne_x': -16.118269821278478, 't-sne_y': -28.46515465390898},\n",
       "     {'umap_x': 8.233110427856445, 'umap_y': 13.023180961608887}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463898-C-A',\n",
       "      'distance': 0.0013523576781153679,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-A',\n",
       "      'distance': 0.001398275839164853,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465789-C-A',\n",
       "      'distance': 0.0015241826185956597,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508581-C-A',\n",
       "      'distance': 0.001528208376839757,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437320-A-C',\n",
       "      'distance': 0.001937946304678917,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.002472353633493185,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-A',\n",
       "      'distance': 0.0029622185975313187,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.0038500381633639336,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437320-A-G',\n",
       "      'distance': 0.003966790623962879,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.003973712213337421,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465788-A-G',\n",
       "      'distance': 0.004205980338156223,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-T',\n",
       "      'distance': 0.004628508351743221,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474254-C-T',\n",
       "      'distance': 0.004637399222701788,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.004666405729949474,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.004755932372063398,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.004764760844409466,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444540-C-T',\n",
       "      'distance': 0.004821518436074257,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.004956492222845554,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441720-C-T',\n",
       "      'distance': 0.005144911352545023,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488112-C-T',\n",
       "      'distance': 0.005257189739495516,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48437045-T-TACA',\n",
       "    'metadata': {'vcf_string': '15\\t48437045\\t.\\tT\\tTACA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48437045',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'TACA',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6409_6411dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48437046_48437048dup',\n",
       "     'hgvs_protein': 'p.Cys2137dup',\n",
       "     'most_severe_consequence': 'inframe_insertion'},\n",
       "    'annotation_raw': '{\"end\": 48437045, \"input\": \"NC_000015.10:g.48437046_48437048dup\", \"id\": \"NC_000015.10:g.48437046_48437048dup\", \"strand\": 1, \"allele_string\": \"-/ACA\", \"start\": 48437046, \"transcript_consequences\": [{\"cdna_start\": 6727, \"consequence_terms\": [\"inframe_insertion\"], \"used_ref\": \"-\", \"cds_start\": 6411, \"given_ref\": \"-\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48437045, \"type\": \"SimpleInterval\", \"start\": 48437045}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"ACA\"}}, \"mutfunc\": {\"exp\": {\"dG_wt_sd\": 0, \"dG_mt_sd\": 0.001516, \"dG_mt\": 191.6646, \"dG_wt\": 190.957, \"ddG\": 0.7076, \"ddG_sd\": 0.001516}}, \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"cdna_end\": 6728, \"hgvsp\": \"NP_000129.3:p.Cys2137dup\", \"location\": \"15:48437045-48437046\", \"mastermind_counts\": \"2|2|2\", \"allele_num\": 1, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"protein_start\": 2137, \"codons\": \"-/TGT\", \"cds_end\": 6412, \"loeuf\": 0.105, \"variant_allele\": \"ACA\", \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"-/C\", \"canonical\": 1, \"mastermind_mmid3\": \"FBN1:C2137dup\", \"gene_id\": \"2200\", \"protein_end\": 2138, \"gene_symbol\": \"FBN1\", \"impact\": \"MODERATE\", \"exon\": \"53/66\", \"conservation\": 2.09, \"hgvsc\": \"NM_000138.5:c.6409_6411dup\"}], \"assembly_name\": \"GRCh38\", \"variant_class\": \"insertion\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"inframe_insertion\", \"minimised\": 1, \"vcf_string\": \"15-48437045-T-TACA\"}',\n",
       "    'coordinates': [{'pca_x': 0.005394544507231508,\n",
       "      'pca_y': 0.018218095147401935},\n",
       "     {'t-sne_x': 22.36122671663029, 't-sne_y': 14.003839675204476},\n",
       "     {'umap_x': 4.401511192321777, 'umap_y': 7.250783443450928}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.10873446613550186,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.10946587473154068,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.10966319590806961,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437069-C-T',\n",
       "      'distance': 0.10974092781543732,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.1105567067861557,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446814-C-T',\n",
       "      'distance': 0.11070775985717773,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.11080073565244675,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.11083871871232986,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.11096000671386719,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488233-C-T',\n",
       "      'distance': 0.11099626123905182,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463904-C-A',\n",
       "      'distance': 0.1111862063407898,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492578-C-T',\n",
       "      'distance': 0.11122911423444748,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.11143352091312408,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520708-C-A',\n",
       "      'distance': 0.11151356250047684,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-T',\n",
       "      'distance': 0.11157295852899551,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468526-C-T',\n",
       "      'distance': 0.11181768774986267,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.1119411289691925,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.11202013492584229,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.11210864782333374,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.11218041181564331,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48437006-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48437006\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48437006',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6451T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48437006A>G',\n",
       "     'hgvs_protein': 'p.Cys2151Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"minimised\": 1, \"vcf_string\": \"15-48437006-A-G\", \"id\": \"NC_000015.10:g.48437006A>G\", \"end\": 48437006, \"input\": \"NC_000015.10:g.48437006A>G\", \"allele_string\": \"A/G\", \"strand\": 1, \"colocated_variants\": [{\"seq_region_name\": \"15\", \"start\": 48437006, \"phenotype_or_disease\": 1, \"id\": \"CM098712\", \"end\": 48437006, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1}], \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48437006, \"start\": 48437005}}}, \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 0.338602, \"dG_wt_sd\": 0.08122, \"dG_wt\": 190.8694, \"dG_mt\": 197.2458, \"ddG_sd\": 0.379996, \"ddG\": 6.3764}}, \"spliceai\": {\"DS_DG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": 14, \"DP_DG\": -41, \"DP_AL\": -28, \"DS_DL\": 0, \"DP_DL\": 0, \"DS_AG\": 0}, \"given_ref\": \"A\", \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_raw\": 5.325604, \"mastermind_counts\": \"2|2|2\", \"allele_num\": 1, \"aa\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys2151Arg\", \"location\": \"15:48437006\", \"cdna_end\": 6767, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.999815404415131, \"cdna_start\": 6767, \"cds_start\": 6451, \"eve_score\": 0.710633949118929, \"used_ref\": \"A\", \"gene_symbol\": \"FBN1\", \"protein_end\": 2151, \"mutationtaster_pred\": \"D\", \"blosum62\": -3, \"gene_id\": \"2200\", \"mastermind_mmid3\": \"FBN1:C2151R\", \"hgvsc\": \"NM_000138.5:c.6451T>C\", \"eve_class\": \"Pathogenic\", \"exon\": \"53/66\", \"conservation\": 3.95, \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"cadd_phred\": 30, \"alphamissense\": {\"am_pathogenicity\": 0.9982, \"am_class\": \"likely_pathogenic\"}, \"sift_score\": 0, \"lrt_pred\": \"invalid_field\", \"cds_end\": 6451, \"protein_start\": 2151, \"codons\": \"Tgc/Cgc\", \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"sift_prediction\": \"deleterious_low_confidence\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"amino_acids\": \"C/R\", \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.996, \"variant_allele\": \"G\", \"loeuf\": 0.105}], \"start\": 48437006}',\n",
       "    'coordinates': [{'pca_x': 0.044524346799514114,\n",
       "      'pca_y': 0.07916810806095625},\n",
       "     {'t-sne_x': 36.015751579587324, 't-sne_y': 4.222017559101931},\n",
       "     {'umap_x': 1.0290688276290894, 'umap_y': 6.550931453704834}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.0008263486670330167,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430770-A-G',\n",
       "      'distance': 0.001235438510775566,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430722-A-G',\n",
       "      'distance': 0.0012608424294739962,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0014423670945689082,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496106-A-G',\n",
       "      'distance': 0.001469202688895166,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596319-A-G',\n",
       "      'distance': 0.0015405018348246813,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.0015524061163887382,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468418-A-G',\n",
       "      'distance': 0.0016124380053952336,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.0016623956616967916,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.001664698589593172,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456688-A-G',\n",
       "      'distance': 0.0016732868971303105,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.001741782296448946,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596343-A-G',\n",
       "      'distance': 0.0017748628742992878,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.001804610714316368,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.0018278040224686265,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520677-A-G',\n",
       "      'distance': 0.0018331691389903426,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.0018376525258645415,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.0018455947283655405,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596373-A-G',\n",
       "      'distance': 0.0018578032031655312,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468490-A-G',\n",
       "      'distance': 0.0018749046139419079,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48436999-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48436999\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48436999',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6458G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48436999C>T',\n",
       "     'hgvs_protein': 'p.Cys2153Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48436999-C-T\", \"minimised\": 1, \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\", \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"cdna_start\": 6774, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.999817907810211, \"eve_score\": 0.795189823962472, \"used_ref\": \"C\", \"cds_start\": 6458, \"mutfunc\": {\"exp\": {\"ddG\": 0.4996, \"ddG_sd\": 0.535857, \"dG_wt_sd\": 0.057968, \"dG_mt_sd\": 0.500548, \"dG_wt\": 190.8586, \"dG_mt\": 191.3582}}, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"end\": 48436999, \"type\": \"SimpleInterval\", \"start\": 48436998}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"spliceai\": {\"DP_AG\": -21, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0.01, \"DS_AL\": 0, \"DS_AG\": 0, \"DP_DL\": -38, \"DP_AL\": 1, \"DS_DL\": 0, \"DP_DG\": 3}, \"given_ref\": \"C\", \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"aa\": \"C\", \"hgvsp\": \"NP_000129.3:p.Cys2153Tyr\", \"location\": \"15:48436999\", \"cdna_end\": 6774, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_raw\": 5.404336, \"allele_num\": 1, \"mastermind_counts\": \"9|10|10\", \"sift_prediction\": \"deleterious_low_confidence\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"lrt_pred\": \"invalid_field\", \"codons\": \"tGt/tAt\", \"cds_end\": 6458, \"protein_start\": 2153, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"loeuf\": 0.105, \"variant_allele\": \"T\", \"canonical\": 1, \"amino_acids\": \"C/Y\", \"polyphen_score\": 0.996, \"mane_select\": \"ENST00000316623.10\", \"mutationtaster_pred\": \"D\", \"blosum62\": -2, \"gene_id\": \"2200\", \"mastermind_mmid3\": \"FBN1:C2153Y\", \"gene_symbol\": \"FBN1\", \"protein_end\": 2153, \"exon\": \"53/66\", \"conservation\": 3.95, \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"cadd_phred\": 31, \"alphamissense\": {\"am_pathogenicity\": 0.9924, \"am_class\": \"likely_pathogenic\"}, \"sift_score\": 0, \"hgvsc\": \"NM_000138.5:c.6458G>A\", \"eve_class\": \"Pathogenic\"}], \"start\": 48436999, \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"id\": \"CM012601\", \"end\": 48436999, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48436999, \"phenotype_or_disease\": 1}], \"allele_string\": \"C/T\", \"strand\": 1, \"end\": 48436999, \"input\": \"NC_000015.10:g.48436999C>T\", \"id\": \"NC_000015.10:g.48436999C>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.0418106719195247, 'pca_y': 0.0862710379898407},\n",
       "     {'t-sne_x': 10.17925514407706, 't-sne_y': -3.0309603071701376},\n",
       "     {'umap_x': 4.596697807312012, 'umap_y': 4.509417533874512}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430676-C-T',\n",
       "      'distance': 0.002792841987684369,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437872-C-T',\n",
       "      'distance': 0.0033493819646537304,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430676-C-G',\n",
       "      'distance': 0.0033665127120912075,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470684-C-T',\n",
       "      'distance': 0.0036474093794822693,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472595-C-T',\n",
       "      'distance': 0.003798576071858406,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474299-C-T',\n",
       "      'distance': 0.0038112830370664597,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48436966-C-T',\n",
       "      'distance': 0.0038293737452477217,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428459-C-T',\n",
       "      'distance': 0.0038946056738495827,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425816-C-T',\n",
       "      'distance': 0.003924472723156214,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432916-C-T',\n",
       "      'distance': 0.004160923883318901,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437369-C-T',\n",
       "      'distance': 0.004240517970174551,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.004261130467057228,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.004324465524405241,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.004338740836828947,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432910-C-T',\n",
       "      'distance': 0.004339393228292465,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.004366425331681967,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537718-C-T',\n",
       "      'distance': 0.004379757679998875,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48537676-C-T',\n",
       "      'distance': 0.004391119349747896,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474638-C-T',\n",
       "      'distance': 0.004493676126003265,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432976-C-T',\n",
       "      'distance': 0.004510676488280296,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48434633-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48434633\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48434633',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6577G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48434633C>A',\n",
       "     'hgvs_protein': 'p.Glu2193Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48434633C>A\", \"end\": 48434633, \"id\": \"NC_000015.10:g.48434633C>A\", \"allele_string\": \"C/A\", \"strand\": 1, \"colocated_variants\": [{\"start\": 48434633, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"id\": \"CM040043\", \"end\": 48434633, \"allele_string\": \"HGMD_MUTATION\"}, {\"id\": \"CM170952\", \"end\": 48434633, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48434633}, {\"var_synonyms\": {\"COSMIC\": [\"COSM1206711\"]}, \"strand\": 1, \"somatic\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"id\": \"COSV57323942\", \"end\": 48434633, \"phenotype_or_disease\": 1, \"start\": 48434633, \"seq_region_name\": \"15\"}], \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"gene_symbol_source\": \"EntrezGene\", \"mutfunc\": {\"exp\": {\"dG_mt_sd\": 2.043039, \"dG_wt_sd\": 0.095055, \"dG_mt\": 183.075399, \"dG_wt\": 178.637, \"ddG_sd\": 1.97368, \"ddG\": 4.438399}}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"end\": 48434633, \"type\": \"SimpleInterval\", \"start\": 48434632}}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}}, \"given_ref\": \"C\", \"spliceai\": {\"DP_AL\": 30, \"DS_DL\": 0.02, \"DP_DG\": -33, \"DP_DL\": -39, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0, \"DP_AG\": -3}, \"cadd_raw\": 10.112299, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"mastermind_counts\": \"4|4|4\", \"allele_num\": 1, \"location\": \"15:48434633\", \"hgvsp\": \"NP_000129.3:p.Glu2193Ter\", \"aa\": \"C\", \"cdna_end\": 6893, \"consequence_terms\": [\"stop_gained\"], \"cdna_start\": 6893, \"cds_start\": 6577, \"used_ref\": \"C\", \"lof_info\": \"PERCENTILE:0.763347260909935,GERP_DIST:2295.48669731021,BP_DIST:2027,DIST_FROM_LAST_EXON:1638,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"protein_end\": 2193, \"gene_id\": \"2200\", \"mutationtaster_pred\": \"A\", \"mastermind_mmid3\": \"FBN1:E2193X\", \"hgvsc\": \"NM_000138.5:c.6577G>T\", \"impact\": \"HIGH\", \"exon\": \"54/66\", \"conservation\": 3.95, \"cadd_phred\": 45, \"protein_start\": 2193, \"cds_end\": 6577, \"codons\": \"Gag/Tag\", \"lrt_pred\": \"invalid_field\", \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"amino_acids\": \"E/*\", \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"variant_allele\": \"A\", \"loeuf\": 0.105}], \"start\": 48434633, \"assembly_name\": \"GRCh38\", \"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"stop_gained\", \"minimised\": 1, \"vcf_string\": \"15-48434633-C-A\"}',\n",
       "    'coordinates': [{'pca_x': 0.015079126874284893,\n",
       "      'pca_y': 0.0976436597853522},\n",
       "     {'t-sne_x': 21.458510239328533, 't-sne_y': 42.02064884365678},\n",
       "     {'umap_x': -9.33339786529541, 'umap_y': 4.763126850128174}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48428355-C-A',\n",
       "      'distance': 0.0017464577686041594,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505139-C-A',\n",
       "      'distance': 0.0020619546994566917,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425435-C-A',\n",
       "      'distance': 0.0022730298805981874,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.002274815458804369,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452676-C-A',\n",
       "      'distance': 0.0024913297966122627,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411085-C-A',\n",
       "      'distance': 0.0025196103379130363,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515397-C-T',\n",
       "      'distance': 0.0027535115368664265,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.002994145266711712,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437837-C-A',\n",
       "      'distance': 0.0030510039068758488,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415675-G-A',\n",
       "      'distance': 0.003302427940070629,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.0033430741168558598,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483878-C-A',\n",
       "      'distance': 0.003353661159053445,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-A',\n",
       "      'distance': 0.0033888351172208786,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.003524775616824627,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415540-G-A',\n",
       "      'distance': 0.0035336692817509174,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48534170-G-A',\n",
       "      'distance': 0.00355699984356761,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488233-C-A',\n",
       "      'distance': 0.0035823779180645943,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427630-G-A',\n",
       "      'distance': 0.003983153961598873,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.004001065623015165,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.004085723310709,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48434593-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48434593\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48434593',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6616+1G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48434593C>G',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48434593-C-G\", \"minimised\": 1, \"seq_region_name\": \"15\", \"most_severe_consequence\": \"splice_donor_variant\", \"variant_class\": \"SNV\", \"start\": 48434593, \"transcript_consequences\": [{\"spliceai\": {\"DP_AL\": 37, \"DS_DL\": 1, \"DP_DG\": -33, \"DP_DL\": 1, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0.29, \"DS_AL\": 0, \"DP_AG\": -17}, \"given_ref\": \"C\", \"ada_score\": 0.999990361384395, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"interval\": {\"start\": 48434592, \"type\": \"SimpleInterval\", \"end\": 48434593}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}}, \"gene_symbol_source\": \"EntrezGene\", \"lof\": \"HC\", \"mane\": [\"MANE_Select\"], \"maxentscan_ref\": 7.866, \"aa\": \"C\", \"location\": \"15:48434593\", \"allele_num\": 1, \"mastermind_counts\": \"5|5|9\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cadd_raw\": 6.370174, \"rf_score\": 0.938, \"consequence_terms\": [\"splice_donor_variant\"], \"used_ref\": \"C\", \"mastermind_mmid3\": \"FBN1:D2206sd\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"lof_info\": \"INTRON_SIZE:1605\", \"cadd_phred\": 34, \"conservation\": 3.95, \"impact\": \"HIGH\", \"intron\": \"54/65\", \"maxentscan_diff\": 8.273, \"hgvsc\": \"NM_000138.5:c.6616+1G>C\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"maxentscan_alt\": -0.407, \"variant_allele\": \"G\", \"genesplicer\": \"loss/donor/48434593-48434594/Medium/2.322491\", \"loeuf\": 0.105, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"start\": 48434593, \"phenotype_or_disease\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48434593, \"id\": \"CS098582\", \"strand\": 1}, {\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48434593, \"end\": 48434593, \"id\": \"CS157718\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1}, {\"phenotype_or_disease\": 1, \"pubmed\": [12938084, 17657824], \"start\": 48434593, \"clin_sig_allele\": \"T:likely_pathogenic;A:likely_pathogenic;G:pathogenic;T:pathogenic\", \"seq_region_name\": \"15\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000808422\", \"RCV000663884\", \"RCV000482028\", \"VCV000419595\", \"VCV001325444\", \"RCV002246056\", \"RCV000557422\", \"VCV000457247\"]}, \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"allele_string\": \"C/A/G/T\", \"end\": 48434593, \"id\": \"rs1064793980\"}], \"allele_string\": \"C/G\", \"strand\": 1, \"end\": 48434593, \"input\": \"NC_000015.10:g.48434593C>G\", \"id\": \"NC_000015.10:g.48434593C>G\"}',\n",
       "    'coordinates': [{'pca_x': -0.24824685993171092,\n",
       "      'pca_y': -0.029532460638443614},\n",
       "     {'t-sne_x': -17.924877822289993, 't-sne_y': -28.379813453512224},\n",
       "     {'umap_x': 7.9416375160217285, 'umap_y': 12.842268943786621}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48463898-C-G',\n",
       "      'distance': 0.0009070772211998701,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.0020690946839749813,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.002291348297148943,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-T',\n",
       "      'distance': 0.0025409588124603033,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.0026940517127513885,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.002799293026328087,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.002842460758984089,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.0028476284351199865,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428345-C-T',\n",
       "      'distance': 0.0028824207838624716,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488112-C-T',\n",
       "      'distance': 0.0029801116324961185,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496099-C-T',\n",
       "      'distance': 0.0030212653800845146,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.003024924313649535,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474254-C-T',\n",
       "      'distance': 0.003053647931665182,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497265-C-T',\n",
       "      'distance': 0.0032236725091934204,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.003274528542533517,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.0034269271418452263,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48444540-C-T',\n",
       "      'distance': 0.003519644495099783,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48441720-C-T',\n",
       "      'distance': 0.0035250589717179537,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48503786-C-G',\n",
       "      'distance': 0.00398209085687995,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465789-C-A',\n",
       "      'distance': 0.004531987011432648,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48432943-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48432943\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48432943',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6662G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48432943C>G',\n",
       "     'hgvs_protein': 'p.Cys2221Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"allele_string\": \"C/G\", \"strand\": 1, \"id\": \"NC_000015.10:g.48432943C>G\", \"end\": 48432943, \"input\": \"NC_000015.10:g.48432943C>G\", \"assembly_name\": \"GRCh38\", \"start\": 48432943, \"transcript_consequences\": [{\"cadd_phred\": 29.1, \"sift_score\": 0, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9991}, \"exon\": \"55/66\", \"conservation\": 3.95, \"impact\": \"MODERATE\", \"polyphen_prediction\": \"probably_damaging\", \"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.6662G>C\", \"mastermind_mmid3\": \"FBN1:C2221S\", \"mutationtaster_pred\": \"A\", \"gene_id\": \"2200\", \"blosum62\": -1, \"protein_end\": 2221, \"gene_symbol\": \"FBN1\", \"variant_allele\": \"G\", \"loeuf\": 0.105, \"polyphen_score\": 0.985, \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"C/S\", \"canonical\": 1, \"sift_prediction\": \"deleterious_low_confidence\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"lrt_pred\": \"invalid_field\", \"cds_end\": 6662, \"codons\": \"tGt/tCt\", \"protein_start\": 2221, \"cdna_end\": 6978, \"aa\": \"C\", \"hgvsp\": \"NP_000129.3:p.Cys2221Ser\", \"location\": \"15:48432943\", \"mastermind_counts\": \"7|10|10\", \"allele_num\": 1, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cadd_raw\": 5.193162, \"spliceai\": {\"DS_AL\": 0, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": 25, \"DP_DG\": 45, \"DS_DL\": 0, \"DP_AL\": 45, \"DP_DL\": 1, \"DS_AG\": 0}, \"given_ref\": \"C\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"end\": 48432943, \"type\": \"SimpleInterval\", \"start\": 48432942}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"eve_score\": 0.734878731897155, \"used_ref\": \"C\", \"cds_start\": 6662, \"cdna_start\": 6978, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.998960494995117}], \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48432943, \"id\": \"CM054722\", \"start\": 48432943, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"start\": 48432943, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48432943, \"id\": \"CM161859\", \"allele_string\": \"HGMD_MUTATION\"}, {\"start\": 48432943, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"id\": \"CM920269\", \"end\": 48432943, \"allele_string\": \"HGMD_MUTATION\"}, {\"clin_sig_allele\": \"A:likely_pathogenic;T:pathogenic;G:pathogenic\", \"start\": 48432943, \"pubmed\": [1301946], \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV001170304\", \"VCV000915547\", \"RCV000816412\", \"RCV000583145\", \"VCV000492830\", \"RCV000017889\", \"VCV000016427\"], \"OMIM\": [134797.0007], \"UniProt\": [\"VAR_002338\"]}, \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"end\": 48432943, \"id\": \"rs137854460\", \"allele_string\": \"C/A/G/T\"}], \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"variant_class\": \"SNV\", \"vcf_string\": \"15-48432943-C-G\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.034661957576603286,\n",
       "      'pca_y': 0.08801096725218643},\n",
       "     {'t-sne_x': 17.120419512067038, 't-sne_y': -5.636009960608677},\n",
       "     {'umap_x': 8.768686294555664, 'umap_y': 6.90146541595459}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48432910-C-G',\n",
       "      'distance': 0.0006944826454855502,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.002229312900453806,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.0023046426940709352,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.0024305302649736404,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437064-G-C',\n",
       "      'distance': 0.002449296647682786,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.002613690448924899,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.002678266726434231,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.0027802586555480957,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.0027961949817836285,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.002849148353561759,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.002866617403924465,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432944-A-G',\n",
       "      'distance': 0.0029971415642648935,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434627-C-T',\n",
       "      'distance': 0.0030258314218372107,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.0030740401707589626,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.003075401298701763,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.0030935676768422127,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.0031469250097870827,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.003192638047039509,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434600-A-G',\n",
       "      'distance': 0.0032089361920952797,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432910-C-T',\n",
       "      'distance': 0.0032249819487333298,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48432938-T-G',\n",
       "    'metadata': {'vcf_string': '15\\t48432938\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48432938',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6667A>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48432938T>G',\n",
       "     'hgvs_protein': 'p.Asn2223His',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48432938, \"id\": \"CM013940\", \"phenotype_or_disease\": 1, \"start\": 48432938, \"seq_region_name\": \"15\"}, {\"end\": 48432938, \"id\": \"rs1555394919\", \"allele_string\": \"T/G\", \"clin_sig\": [\"uncertain_significance\"], \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV000663889\", \"VCV000549358\"], \"UniProt\": [\"VAR_018032\"]}, \"seq_region_name\": \"15\", \"clin_sig_allele\": \"G:uncertain_significance\", \"start\": 48432938, \"phenotype_or_disease\": 1}], \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"gene_symbol\": \"FBN1\", \"protein_end\": 2223, \"mutationtaster_pred\": \"D\", \"gene_id\": \"2200\", \"blosum62\": 1, \"mastermind_mmid3\": \"FBN1:N2223H\", \"hgvsc\": \"NM_000138.5:c.6667A>C\", \"eve_class\": \"Pathogenic\", \"exon\": \"55/66\", \"conservation\": 3.95, \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"cadd_phred\": 27.2, \"alphamissense\": {\"am_pathogenicity\": 0.9832, \"am_class\": \"likely_pathogenic\"}, \"sift_score\": 0, \"lrt_pred\": \"invalid_field\", \"protein_start\": 2223, \"cds_end\": 6667, \"codons\": \"Aac/Cac\", \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"sift_prediction\": \"deleterious_low_confidence\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"canonical\": 1, \"amino_acids\": \"N/H\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.998, \"variant_allele\": \"G\", \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"start\": 48432937, \"type\": \"SimpleInterval\", \"end\": 48432938}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}}, \"spliceai\": {\"DP_AG\": 30, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": 4, \"DS_AG\": 0, \"DP_AL\": 50, \"DS_DL\": 0, \"DP_DG\": 50}, \"given_ref\": \"T\", \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_raw\": 4.872647, \"allele_num\": 1, \"mastermind_counts\": \"3|5|5\", \"aa\": \"T\", \"location\": \"15:48432938\", \"hgvsp\": \"NP_000129.3:p.Asn2223His\", \"cdna_end\": 6983, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.998197376728058, \"cdna_start\": 6983, \"cds_start\": 6667, \"eve_score\": 0.732870467873141, \"used_ref\": \"T\"}], \"start\": 48432938, \"id\": \"NC_000015.10:g.48432938T>G\", \"input\": \"NC_000015.10:g.48432938T>G\", \"end\": 48432938, \"strand\": 1, \"allele_string\": \"T/G\", \"minimised\": 1, \"vcf_string\": \"15-48432938-T-G\", \"variant_class\": \"SNV\", \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.03877030148045563,\n",
       "      'pca_y': 0.0817338011222162},\n",
       "     {'t-sne_x': 5.922963127014352, 't-sne_y': -4.202125895496796},\n",
       "     {'umap_x': 4.782320976257324, 'umap_y': 4.9449968338012695}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48434627-C-T',\n",
       "      'distance': 0.0032749406527727842,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465825-C-T',\n",
       "      'distance': 0.0033734948374330997,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463129-C-T',\n",
       "      'distance': 0.0035977831576019526,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432943-C-T',\n",
       "      'distance': 0.0036113886162638664,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432910-C-G',\n",
       "      'distance': 0.0037543880753219128,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432944-A-G',\n",
       "      'distance': 0.003848180640488863,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432911-A-G',\n",
       "      'distance': 0.0039013191126286983,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434600-A-G',\n",
       "      'distance': 0.003921646624803543,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.004104210063815117,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.004147734493017197,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.004170909523963928,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437039-C-T',\n",
       "      'distance': 0.004182717762887478,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432955-C-T',\n",
       "      'distance': 0.004185562953352928,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411243-G-A',\n",
       "      'distance': 0.004227573983371258,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48432910-C-T',\n",
       "      'distance': 0.004387209191918373,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.004445255734026432,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421680-T-C',\n",
       "      'distance': 0.004462846554815769,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.004523945972323418,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415716-T-C',\n",
       "      'distance': 0.004598585888743401,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415632-C-T',\n",
       "      'distance': 0.00464441254734993,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48430698-G-A',\n",
       "    'metadata': {'vcf_string': '15\\t48430698\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48430698',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6844C>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48430698G>A',\n",
       "     'hgvs_protein': 'p.Arg2282Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"minimised\": 1, \"vcf_string\": \"15-48430698-G-A\", \"id\": \"NC_000015.10:g.48430698G>A\", \"end\": 48430698, \"input\": \"NC_000015.10:g.48430698G>A\", \"allele_string\": \"G/A\", \"strand\": 1, \"colocated_variants\": [{\"phenotype_or_disease\": 1, \"start\": 48430698, \"seq_region_name\": \"15\", \"strand\": 1, \"end\": 48430698, \"id\": \"CM970520\", \"allele_string\": \"HGMD_MUTATION\"}, {\"seq_region_name\": \"15\", \"start\": 48430698, \"phenotype_or_disease\": 1, \"allele_string\": \"COSMIC_MUTATION\", \"id\": \"COSV57330830\", \"end\": 48430698, \"var_synonyms\": {\"COSMIC\": [\"COSM6661644\"]}, \"strand\": 1, \"somatic\": 1}, {\"phenotype_or_disease\": 1, \"pubmed\": [11700157], \"start\": 48430698, \"clin_sig_allele\": \"A:pathogenic;A:uncertain_significance\", \"seq_region_name\": \"15\", \"strand\": 1, \"var_synonyms\": {\"UniProt\": [\"VAR_002340\"], \"ClinVar\": [\"RCV004782493\", \"RCV000663905\", \"VCV000549371\", \"RCV001180130\", \"RCV001855423\"]}, \"clin_sig\": [\"uncertain_significance\", \"pathogenic\"], \"allele_string\": \"G/A\", \"end\": 48430698, \"id\": \"rs765205164\", \"frequencies\": {\"A\": {\"gnomade_remaining\": 0, \"gnomade_afr\": 2.989e-05, \"gnomade_sas\": 1.159e-05, \"gnomade_nfe\": 8.994e-07, \"gnomade\": 2.737e-06, \"gnomade_asj\": 0, \"gnomade_mid\": 0, \"gnomade_amr\": 0, \"gnomade_eas\": 2.519e-05, \"gnomade_fin\": 0}}}], \"assembly_name\": \"GRCh38\", \"start\": 48430698, \"transcript_consequences\": [{\"cdna_start\": 7160, \"clinpred\": 0.984484553337097, \"consequence_terms\": [\"missense_variant\"], \"eve_score\": 0.648965086335256, \"used_ref\": \"G\", \"cds_start\": 6844, \"given_ref\": \"G\", \"spliceai\": {\"DP_AG\": 8, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": -23, \"DS_AG\": 0, \"DP_AL\": 0, \"DS_DL\": 0.03, \"DP_DG\": 8}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"end\": 48430698, \"type\": \"SimpleInterval\", \"start\": 48430697}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}}, \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"cdna_end\": 7160, \"hgvsp\": \"NP_000129.3:p.Arg2282Trp\", \"location\": \"15:48430698\", \"aa\": \"G\", \"allele_num\": 1, \"mastermind_counts\": \"8|9|9\", \"cadd_raw\": 4.698567, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"sift_prediction\": \"deleterious_low_confidence\", \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"cds_end\": 6844, \"protein_start\": 2282, \"codons\": \"Cgg/Tgg\", \"lrt_pred\": \"invalid_field\", \"loeuf\": 0.105, \"variant_allele\": \"A\", \"polyphen_score\": 0.997, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"R/W\", \"mastermind_mmid3\": \"FBN1:R2282W\", \"gene_id\": \"2200\", \"blosum62\": -3, \"mutationtaster_pred\": \"D\", \"protein_end\": 2282, \"gene_symbol\": \"FBN1\", \"sift_score\": 0.02, \"alphamissense\": {\"am_pathogenicity\": 0.297, \"am_class\": \"likely_benign\"}, \"cadd_phred\": 26.3, \"impact\": \"MODERATE\", \"polyphen_prediction\": \"probably_damaging\", \"exon\": \"56/66\", \"conservation\": 3.95, \"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.6844C>T\"}]}',\n",
       "    'coordinates': [{'pca_x': 0.04499488047395611,\n",
       "      'pca_y': 0.08638272671493097},\n",
       "     {'t-sne_x': 31.684075288162898, 't-sne_y': 10.37372589396701},\n",
       "     {'umap_x': 2.0456109046936035, 'umap_y': 7.994078159332275}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48427768-G-A',\n",
       "      'distance': 0.0035111424513161182,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.0037406720221042633,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437009-G-A',\n",
       "      'distance': 0.0037868013605475426,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430791-A-G',\n",
       "      'distance': 0.003880253992974758,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428427-G-A',\n",
       "      'distance': 0.004169179126620293,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437379-G-A',\n",
       "      'distance': 0.004387295804917812,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430710-G-A',\n",
       "      'distance': 0.004391235299408436,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48415549-G-A',\n",
       "      'distance': 0.004983521997928619,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490013-G-A',\n",
       "      'distance': 0.004996986594051123,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.005114768166095018,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48411243-G-A',\n",
       "      'distance': 0.005250510759651661,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48470711-C-A',\n",
       "      'distance': 0.005937655456364155,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.005967798642814159,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.006229884922504425,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497332-G-A',\n",
       "      'distance': 0.0064567201770842075,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.0064759282395243645,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48481723-G-A',\n",
       "      'distance': 0.0065618762746453285,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48434702-A-G',\n",
       "      'distance': 0.006793809123337269,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.006844939198344946,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48490073-G-A',\n",
       "      'distance': 0.006873004604130983,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48428423-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48428423\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48428423',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6920G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48428423C>G',\n",
       "     'hgvs_protein': 'p.Cys2307Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"lrt_pred\": \"invalid_field\", \"protein_start\": 2307, \"codons\": \"tGc/tCc\", \"cds_end\": 6920, \"sift_prediction\": \"deleterious_low_confidence\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.989, \"amino_acids\": \"C/S\", \"canonical\": 1, \"variant_allele\": \"G\", \"loeuf\": 0.105, \"protein_end\": 2307, \"gene_symbol\": \"FBN1\", \"mastermind_mmid3\": \"FBN1:C2307S\", \"mutationtaster_pred\": \"A\", \"blosum62\": -1, \"gene_id\": \"2200\", \"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.6920G>C\", \"cadd_phred\": 29.3, \"alphamissense\": {\"am_pathogenicity\": 0.9994, \"am_class\": \"likely_pathogenic\"}, \"sift_score\": 0, \"conservation\": 3.95, \"exon\": \"57/66\", \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.999671816825867, \"cdna_start\": 7236, \"cds_start\": 6920, \"eve_score\": 0.787187888700018, \"used_ref\": \"C\", \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"spliceai\": {\"DP_DL\": -26, \"DS_AG\": 0, \"DS_DL\": 0.01, \"DP_AL\": 48, \"DP_DG\": 16, \"DP_AG\": 31, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_DG\": 0}, \"given_ref\": \"C\", \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"interval\": {\"start\": 48428422, \"type\": \"SimpleInterval\", \"end\": 48428423}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"type\": \"Allele\"}, \"allele_num\": 1, \"mastermind_counts\": \"8|13|13\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cadd_raw\": 5.214127, \"cdna_end\": 7236, \"aa\": \"C\", \"location\": \"15:48428423\", \"hgvsp\": \"NP_000129.3:p.Cys2307Ser\"}], \"start\": 48428423, \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM145795\", \"end\": 48428423, \"start\": 48428423, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"strand\": 1, \"end\": 48428423, \"id\": \"CM950451\", \"allele_string\": \"HGMD_MUTATION\", \"start\": 48428423, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"var_synonyms\": {\"OMIM\": [134797.0002], \"ClinVar\": [\"RCV000035259\", \"VCV000042414\", \"RCV000663912\", \"RCV000017884\", \"VCV000016422\"], \"UniProt\": [\"VAR_002341\"]}, \"strand\": 1, \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"id\": \"rs137854457\", \"end\": 48428423, \"allele_string\": \"C/G/T\", \"clin_sig_allele\": \"G:pathogenic;T:likely_pathogenic\", \"start\": 48428423, \"pubmed\": [24033266, 1569206, 1301946], \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}], \"allele_string\": \"C/G\", \"strand\": 1, \"end\": 48428423, \"input\": \"NC_000015.10:g.48428423C>G\", \"id\": \"NC_000015.10:g.48428423C>G\", \"vcf_string\": \"15-48428423-C-G\", \"minimised\": 1, \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.03520398632716165,\n",
       "      'pca_y': 0.08077604937076446},\n",
       "     {'t-sne_x': 20.48267347917456, 't-sne_y': -5.215575067117895},\n",
       "     {'umap_x': 8.702975273132324, 'umap_y': 7.2627339363098145}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.0008190946537069976,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.0009072953835129738,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0010786603670567274,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.001096738618798554,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.0011268563102930784,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.0019416523864492774,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.002128822263330221,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.002253651386126876,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.002302831504493952,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596360-C-G',\n",
       "      'distance': 0.002367291133850813,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.002500534523278475,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.0025642584078013897,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.002586732152849436,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-G',\n",
       "      'distance': 0.0026355432346463203,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48489991-C-G',\n",
       "      'distance': 0.0026446664705872536,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468503-G-C',\n",
       "      'distance': 0.0026864460669457912,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.0028495932929217815,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495162-C-G',\n",
       "      'distance': 0.002879528794437647,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468455-G-C',\n",
       "      'distance': 0.0028806915506720543,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-C',\n",
       "      'distance': 0.002891815733164549,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48428392-ins19-7e796f39',\n",
       "    'metadata': {'vcf_string': '15\\t48428392\\t.\\tC\\tCTCACAGGTGTAGCTCCCAC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48428392',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'CTCACAGGTGTAGCTCCCAC',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.6935_6953dup',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48428393_48428411dup',\n",
       "     'hgvs_protein': 'p.Cys2318TrpfsTer6',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"variant_class\": \"insertion\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"stop_gained\", \"minimised\": 1, \"vcf_string\": \"15-48428392-C-CTCACAGGTGTAGCTCCCAC\", \"end\": 48428392, \"input\": \"NC_000015.10:g.48428393_48428411dup\", \"id\": \"NC_000015.10:g.48428393_48428411dup\", \"strand\": 1, \"allele_string\": \"-/TCACAGGTGTAGCTCCCAC\", \"transcript_consequences\": [{\"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"cds_end\": 6954, \"codons\": \"tgt/tgGGAGCTACACCTGTGAGTGt\", \"protein_start\": 2318, \"variant_allele\": \"CACTCACAGGTGTAGCTCC\", \"loeuf\": 0.105, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"C/WELHL*VX\", \"gene_id\": \"2200\", \"protein_end\": 2318, \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.806987000928505,GERP_DIST:1924.51990754306,BP_DIST:1657,DIST_FROM_LAST_EXON:1268,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"exon\": \"57/66\", \"conservation\": 3.505, \"impact\": \"HIGH\", \"hgvsc\": \"NM_000138.5:c.6935_6953dup\", \"cdna_start\": 7269, \"consequence_terms\": [\"stop_gained\", \"frameshift_variant\"], \"used_ref\": \"-\", \"hgvs_offset\": -3, \"cds_start\": 6953, \"given_ref\": \"-\", \"gene_symbol_source\": \"EntrezGene\", \"lof\": \"HC\", \"mane\": [\"MANE_Select\"], \"cdna_end\": 7270, \"location\": \"15:48428389-48428390\", \"hgvsp\": \"NP_000129.3:p.Cys2318TrpfsTer6\", \"allele_num\": 1, \"strand\": -1, \"protein_id\": \"NP_000129.3\"}], \"start\": 48428393, \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': 0.0009766894428010486,\n",
       "      'pca_y': 0.06885799024102651},\n",
       "     {'t-sne_x': 23.86937536839346, 't-sne_y': 38.94586258815077},\n",
       "     {'umap_x': -9.613974571228027, 'umap_y': 3.895763397216797}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48427659-C-T',\n",
       "      'distance': 0.08165116608142853,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48503820-C-A',\n",
       "      'distance': 0.0818633884191513,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430785-C-A',\n",
       "      'distance': 0.08197664469480515,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420700-C-T',\n",
       "      'distance': 0.0821431428194046,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488411-A-T',\n",
       "      'distance': 0.08271873742341995,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489970-C-T',\n",
       "      'distance': 0.08293114602565765,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.08308049291372299,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.08313308656215668,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.08325951546430588,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.08327198028564453,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437805-C-T',\n",
       "      'distance': 0.08334052562713623,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.08346985280513763,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510048-A-T',\n",
       "      'distance': 0.08347673714160919,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48436970-C-A',\n",
       "      'distance': 0.08363661170005798,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425820-C-A',\n",
       "      'distance': 0.08396321535110474,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427646-A-T',\n",
       "      'distance': 0.08411365747451782,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.08411712199449539,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434705-C-A',\n",
       "      'distance': 0.08418282866477966,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.08420681208372116,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.08431774377822876,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48427749-GTG-G',\n",
       "    'metadata': {'vcf_string': '15\\t48427749\\t.\\tGTG\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48427749',\n",
       "     'ref_allele': 'GTG',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7022_7023del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48427750_48427751del',\n",
       "     'hgvs_protein': 'p.Thr2341ArgfsTer25',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"vcf_string\": \"15-48427749-GTG-G\", \"variant_class\": \"deletion\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"frameshift_variant\", \"start\": 48427750, \"transcript_consequences\": [{\"hgvsc\": \"NM_000138.5:c.7022_7023del\", \"exon\": \"58/66\", \"conservation\": 2.315, \"impact\": \"HIGH\", \"protein_end\": 2341, \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.815111420612813,GERP_DIST:1871.94088360071,BP_DIST:1587,DIST_FROM_LAST_EXON:1198,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_id\": \"2200\", \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"T/X\", \"canonical\": 1, \"variant_allele\": \"-\", \"loeuf\": 0.105, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"cds_end\": 7023, \"protein_start\": 2341, \"codons\": \"aCA/a\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"allele_num\": 1, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cdna_end\": 7339, \"aa\": \"TG\", \"location\": \"15:48427748-48427749\", \"hgvsp\": \"NP_000129.3:p.Thr2341ArgfsTer25\", \"gene_symbol_source\": \"EntrezGene\", \"lof\": \"HC\", \"mane\": [\"MANE_Select\"], \"spliceai\": {\"DP_DG\": -4, \"DS_DL\": 0, \"DP_AL\": 24, \"DP_DL\": 12, \"DS_AG\": 0, \"DS_AL\": 0, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": -37}, \"given_ref\": \"TG\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"start\": 48427749, \"type\": \"SimpleInterval\", \"end\": 48427751}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}}, \"hgvs_offset\": -2, \"cds_start\": 7022, \"used_ref\": \"TG\", \"consequence_terms\": [\"frameshift_variant\"], \"cdna_start\": 7338}], \"assembly_name\": \"GRCh38\", \"end\": 48427751, \"input\": \"NC_000015.10:g.48427750_48427751del\", \"id\": \"NC_000015.10:g.48427750_48427751del\", \"allele_string\": \"TG/-\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.034931100381498104,\n",
       "      'pca_y': 0.030244854463995725},\n",
       "     {'t-sne_x': 35.075834970175315, 't-sne_y': 7.828125368036613},\n",
       "     {'umap_x': 1.612438678741455, 'umap_y': 7.136333465576172}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.0929218977689743,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09336116164922714,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520776-G-A',\n",
       "      'distance': 0.09342841058969498,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.09353167563676834,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.09400048851966858,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.0940098837018013,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.0945480465888977,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446768-A-G',\n",
       "      'distance': 0.0945691168308258,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437807-A-G',\n",
       "      'distance': 0.09458531439304352,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09461300075054169,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.09463679790496826,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446751-G-A',\n",
       "      'distance': 0.09466379880905151,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.09469272196292877,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.09474334865808487,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.09491846710443497,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445382-A-G',\n",
       "      'distance': 0.09501665830612183,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.09507513791322708,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.0951261818408966,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445469-A-G',\n",
       "      'distance': 0.09521542489528656,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432907-G-A',\n",
       "      'distance': 0.09522589296102524,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.6},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.85}}},\n",
       "   {'variant_id': '15-48427604-GAG-G',\n",
       "    'metadata': {'vcf_string': '15\\t48427604\\t.\\tGAG\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48427604',\n",
       "     'ref_allele': 'GAG',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7167_7168del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48427605_48427606del',\n",
       "     'hgvs_protein': 'p.Cys2390SerfsTer15',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"start\": 48427605, \"transcript_consequences\": [{\"gene_id\": \"2200\", \"protein_end\": 2390, \"lof_info\": \"PERCENTILE:0.831940575673166,GERP_DIST:1662.23695383072,BP_DIST:1442,DIST_FROM_LAST_EXON:1053,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"impact\": \"HIGH\", \"exon\": \"58/66\", \"conservation\": 3.7, \"hgvsc\": \"NM_000138.5:c.7167_7168del\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"cds_end\": 7168, \"codons\": \"ctCTgt/ctgt\", \"protein_start\": 2389, \"loeuf\": 0.105, \"variant_allele\": \"-\", \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"LC/LX\", \"canonical\": 1, \"given_ref\": \"AG\", \"spliceai\": {\"DP_DG\": -33, \"DP_AL\": 23, \"DS_DL\": 0, \"DP_DL\": -37, \"DS_AG\": 0, \"DS_DG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": -37}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48427606, \"start\": 48427604}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}}, \"lof\": \"HC\", \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"cdna_end\": 7484, \"hgvsp\": \"NP_000129.3:p.Cys2390SerfsTer15\", \"location\": \"15:48427603-48427604\", \"aa\": \"AG\", \"allele_num\": 1, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cdna_start\": 7483, \"consequence_terms\": [\"frameshift_variant\"], \"used_ref\": \"AG\", \"hgvs_offset\": -2, \"cds_start\": 7167}], \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48427605_48427606del\", \"end\": 48427606, \"id\": \"NC_000015.10:g.48427605_48427606del\", \"allele_string\": \"AG/-\", \"strand\": 1, \"minimised\": 1, \"vcf_string\": \"15-48427604-GAG-G\", \"variant_class\": \"deletion\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"frameshift_variant\"}',\n",
       "    'coordinates': [{'pca_x': 0.04109681942816734,\n",
       "      'pca_y': 0.028984626519455332},\n",
       "     {'t-sne_x': 9.424496345690278, 't-sne_y': 2.8116693689342096},\n",
       "     {'umap_x': 3.892759084701538, 'umap_y': 3.5592970848083496}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.09390746057033539,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.09390837699174881,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430770-A-G',\n",
       "      'distance': 0.09423723816871643,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427603-A-G',\n",
       "      'distance': 0.09466691315174103,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446768-A-G',\n",
       "      'distance': 0.0947868824005127,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415707-C-T',\n",
       "      'distance': 0.09497559815645218,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437069-C-T',\n",
       "      'distance': 0.09498286247253418,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.09499728679656982,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474584-C-T',\n",
       "      'distance': 0.09507270157337189,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.09518206119537354,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.09528382122516632,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421981-C-T',\n",
       "      'distance': 0.09546271711587906,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48494209-C-T',\n",
       "      'distance': 0.09555543214082718,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.09559861570596695,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48467997-C-T',\n",
       "      'distance': 0.09571143239736557,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421594-C-T',\n",
       "      'distance': 0.09575888514518738,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.09582136571407318,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.09583871811628342,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48468490-A-G',\n",
       "      'distance': 0.09585009515285492,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456682-A-G',\n",
       "      'distance': 0.09597161412239075,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48425780-TT-T',\n",
       "    'metadata': {'vcf_string': '15\\t48425780\\t.\\tTT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48425780',\n",
       "     'ref_allele': 'TT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7291del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48425781del',\n",
       "     'hgvs_protein': 'p.Thr2431LeufsTer7',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"allele_string\": \"T/-\", \"end\": 48425781, \"input\": \"NC_000015.10:g.48425781del\", \"id\": \"NC_000015.10:g.48425781del\", \"transcript_consequences\": [{\"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"protein_start\": 2431, \"codons\": \"Act/ct\", \"cds_end\": 7291, \"variant_allele\": \"-\", \"loeuf\": 0.105, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"T/X\", \"gene_id\": \"2200\", \"protein_end\": 2431, \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.846216341689879,GERP_DIST:1505.08626123965,BP_DIST:1321,DIST_FROM_LAST_EXON:932,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"exon\": \"59/66\", \"conservation\": 3.95, \"impact\": \"HIGH\", \"hgvsc\": \"NM_000138.5:c.7291del\", \"cdna_start\": 7607, \"consequence_terms\": [\"frameshift_variant\"], \"used_ref\": \"T\", \"hgvs_offset\": -3, \"cds_start\": 7291, \"spliceai\": {\"DP_DG\": -41, \"DS_DL\": 0, \"DP_AL\": 5, \"DS_AG\": 0, \"DP_DL\": -37, \"DS_AL\": 0, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": -18}, \"given_ref\": \"T\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48425780, \"type\": \"SimpleInterval\", \"end\": 48425781}}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}}, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"cdna_end\": 7607, \"aa\": \"T\", \"location\": \"15:48425778\", \"hgvsp\": \"NP_000129.3:p.Thr2431LeufsTer7\", \"allele_num\": 1, \"strand\": -1, \"protein_id\": \"NP_000129.3\"}], \"start\": 48425781, \"assembly_name\": \"GRCh38\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"frameshift_variant\", \"variant_class\": \"deletion\", \"vcf_string\": \"15-48425780-TT-T\", \"minimised\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.03904282362580383,\n",
       "      'pca_y': 0.03089424259866754},\n",
       "     {'t-sne_x': 1.8910791376910059, 't-sne_y': 9.408117353583785},\n",
       "     {'umap_x': 3.6277010440826416, 'umap_y': 3.1631999015808105}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48410990-T-C',\n",
       "      'distance': 0.10772202163934708,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.10839244723320007,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.11013764142990112,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48510093-G-T',\n",
       "      'distance': 0.11018258333206177,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445467-G-T',\n",
       "      'distance': 0.11022847890853882,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.1103329062461853,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.11070417612791061,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452614-G-T',\n",
       "      'distance': 0.11099954694509506,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427646-A-T',\n",
       "      'distance': 0.11100102961063385,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.1112712174654007,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.11154484748840332,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487367-G-T',\n",
       "      'distance': 0.11167360842227936,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445419-G-T',\n",
       "      'distance': 0.1118132472038269,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427659-C-T',\n",
       "      'distance': 0.1119702011346817,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495484-T-A',\n",
       "      'distance': 0.11201528459787369,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.11201983690261841,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.11206072568893433,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48489921-G-T',\n",
       "      'distance': 0.11206351965665817,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.11207277327775955,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463098-A-T',\n",
       "      'distance': 0.11214061081409454,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48425743-A-C',\n",
       "    'metadata': {'vcf_string': '15\\t48425743\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48425743',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7326T>G',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48425743A>C',\n",
       "     'hgvs_protein': 'p.Cys2442Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM040044\", \"end\": 48425743, \"start\": 48425743, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM098628\", \"end\": 48425743, \"phenotype_or_disease\": 1, \"start\": 48425743, \"seq_region_name\": \"15\"}], \"transcript_consequences\": [{\"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.7326T>G\", \"cadd_phred\": 33, \"sift_score\": 0, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9986}, \"conservation\": 1.47, \"exon\": \"59/66\", \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"protein_end\": 2442, \"gene_symbol\": \"FBN1\", \"mastermind_mmid3\": \"FBN1:C2442W\", \"mutationtaster_pred\": \"D\", \"gene_id\": \"2200\", \"blosum62\": -2, \"polyphen_score\": 0.997, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"C/W\", \"loeuf\": 0.105, \"variant_allele\": \"C\", \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"lrt_pred\": \"invalid_field\", \"cds_end\": 7326, \"codons\": \"tgT/tgG\", \"protein_start\": 2442, \"sift_prediction\": \"deleterious_low_confidence\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"mastermind_counts\": \"2|5|5\", \"allele_num\": 1, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_raw\": 5.822614, \"cdna_end\": 7642, \"aa\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys2442Trp\", \"location\": \"15:48425743\", \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"spliceai\": {\"DP_AG\": 0, \"DS_DG\": 0.8, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DP_DL\": -4, \"DP_DG\": 0, \"DP_AL\": 19, \"DS_DL\": 0}, \"given_ref\": \"A\", \"ga4gh_vrs\": {\"state\": {\"sequence\": \"C\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48425743, \"start\": 48425742}}, \"type\": \"Allele\"}, \"cds_start\": 7326, \"used_ref\": \"A\", \"eve_score\": 0.789667329238379, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.999883651733398, \"cdna_start\": 7642}], \"start\": 48425743, \"assembly_name\": \"GRCh38\", \"end\": 48425743, \"input\": \"NC_000015.10:g.48425743A>C\", \"id\": \"NC_000015.10:g.48425743A>C\", \"strand\": 1, \"allele_string\": \"A/C\", \"minimised\": 1, \"vcf_string\": \"15-48425743-A-C\", \"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\"}',\n",
       "    'coordinates': [{'pca_x': 0.03030565469443361,\n",
       "      'pca_y': 0.10991850185450033},\n",
       "     {'t-sne_x': -16.3806981873784, 't-sne_y': 30.7142457185328},\n",
       "     {'umap_x': 1.6634498834609985, 'umap_y': 21.77474021911621}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48441764-A-C',\n",
       "      'distance': 0.0011995480163022876,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495512-A-C',\n",
       "      'distance': 0.0014993749791756272,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-G',\n",
       "      'distance': 0.0017274945275858045,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474345-A-C',\n",
       "      'distance': 0.002116811228916049,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48463950-A-C',\n",
       "      'distance': 0.002363834995776415,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.003627713769674301,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48505136-A-C',\n",
       "      'distance': 0.0037306849844753742,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.0037575303576886654,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508633-A-C',\n",
       "      'distance': 0.0037636500783264637,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.0038022149819880724,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.0038579138927161694,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425458-C-T',\n",
       "      'distance': 0.003911969251930714,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.004039324354380369,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.004107516724616289,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425401-T-C',\n",
       "      'distance': 0.004142464138567448,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437026-T-C',\n",
       "      'distance': 0.004154833033680916,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425412-G-C',\n",
       "      'distance': 0.004173942841589451,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472556-C-T',\n",
       "      'distance': 0.0042443061247467995,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515428-C-T',\n",
       "      'distance': 0.004297150298953056,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474260-C-T',\n",
       "      'distance': 0.004432994872331619,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48425413-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48425413\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48425413',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7409G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48425413C>T',\n",
       "     'hgvs_protein': 'p.Cys2470Tyr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"start\": 48425413, \"transcript_consequences\": [{\"sift_score\": 0, \"alphamissense\": {\"am_pathogenicity\": 0.9992, \"am_class\": \"likely_pathogenic\"}, \"cadd_phred\": 32, \"impact\": \"MODERATE\", \"polyphen_prediction\": \"probably_damaging\", \"exon\": \"60/66\", \"conservation\": 3.95, \"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.7409G>A\", \"mastermind_mmid3\": \"FBN1:C2470Y\", \"gene_id\": \"2200\", \"blosum62\": -2, \"mutationtaster_pred\": \"A\", \"protein_end\": 2470, \"gene_symbol\": \"FBN1\", \"variant_allele\": \"T\", \"loeuf\": 0.105, \"polyphen_score\": 0.994, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"C/Y\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"sift_prediction\": \"deleterious_low_confidence\", \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"cds_end\": 7409, \"codons\": \"tGc/tAc\", \"protein_start\": 2470, \"lrt_pred\": \"invalid_field\", \"cdna_end\": 7725, \"hgvsp\": \"NP_000129.3:p.Cys2470Tyr\", \"location\": \"15:48425413\", \"aa\": \"C\", \"mastermind_counts\": \"4|5|5\", \"allele_num\": 1, \"cadd_raw\": 5.445515, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"given_ref\": \"C\", \"spliceai\": {\"DP_AG\": -9, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_DG\": 0, \"DP_DL\": -49, \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AL\": 21, \"DP_DG\": -44}, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"start\": 48425412, \"type\": \"SimpleInterval\", \"end\": 48425413}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"eve_score\": 0.732515605664712, \"used_ref\": \"C\", \"cds_start\": 7409, \"cdna_start\": 7725, \"clinpred\": 0.999920129776001, \"consequence_terms\": [\"missense_variant\"]}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48425413, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM085425\", \"end\": 48425413, \"strand\": 1}, {\"clin_sig\": [\"uncertain_significance\", \"likely_pathogenic\", \"pathogenic\"], \"allele_string\": \"C/A/T\", \"id\": \"rs1555394398\", \"end\": 48425413, \"var_synonyms\": {\"ClinVar\": [\"RCV001221828\", \"RCV000663951\", \"VCV000549407\", \"RCV001859335\", \"RCV001814319\", \"RCV001420801\", \"VCV001098776\", \"RCV004776304\"], \"UniProt\": [\"VAR_076145\"]}, \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"pubmed\": [17657824, 17701892], \"start\": 48425413, \"clin_sig_allele\": \"T:likely_pathogenic;T:pathogenic;A:uncertain_significance;A:likely_pathogenic;A:pathogenic\"}], \"strand\": 1, \"allele_string\": \"C/T\", \"input\": \"NC_000015.10:g.48425413C>T\", \"end\": 48425413, \"id\": \"NC_000015.10:g.48425413C>T\", \"vcf_string\": \"15-48425413-C-T\", \"minimised\": 1, \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\", \"variant_class\": \"SNV\"}',\n",
       "    'coordinates': [{'pca_x': 0.029755320497168114,\n",
       "      'pca_y': 0.11019198159990048},\n",
       "     {'t-sne_x': -11.493614666149538, 't-sne_y': 32.43834755128233},\n",
       "     {'umap_x': 1.0280698537826538, 'umap_y': 22.121257781982422}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.0008709162939339876,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.0009391271159984171,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495131-C-T',\n",
       "      'distance': 0.0009751784382387996,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.0009932608809322119,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.0009953491389751434,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472628-C-T',\n",
       "      'distance': 0.0010667493334040046,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.0010847372468560934,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515429-A-C',\n",
       "      'distance': 0.0011226361384615302,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495513-C-T',\n",
       "      'distance': 0.001251916866749525,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421990-C-T',\n",
       "      'distance': 0.0012634418671950698,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.0012687959242612123,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492538-C-T',\n",
       "      'distance': 0.0013458661269396544,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472601-C-T',\n",
       "      'distance': 0.001481283688917756,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492553-C-T',\n",
       "      'distance': 0.0015413658693432808,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472556-C-T',\n",
       "      'distance': 0.001543128746561706,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48441765-C-T',\n",
       "      'distance': 0.001569810789078474,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425458-C-T',\n",
       "      'distance': 0.0016798008000478148,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452654-C-T',\n",
       "      'distance': 0.0017266911454498768,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463949-C-T',\n",
       "      'distance': 0.0018789611058309674,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474260-C-T',\n",
       "      'distance': 0.0019170991145074368,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48422068-T-A',\n",
       "    'metadata': {'vcf_string': '15\\t48422068\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48422068',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7454A>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48422068T>A',\n",
       "     'hgvs_protein': 'p.Asp2485Val',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48422068-T-A\", \"minimised\": 1, \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\", \"variant_class\": \"SNV\", \"transcript_consequences\": [{\"ada_score\": 0.416572115598939, \"given_ref\": \"T\", \"spliceai\": {\"DS_AG\": 0, \"DP_DL\": -9, \"DS_DL\": 0, \"DP_AL\": 0, \"DP_DG\": 0, \"DP_AG\": -11, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_DG\": 0}, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"start\": 48422067, \"type\": \"SimpleInterval\", \"end\": 48422068}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"maxentscan_ref\": 9.957, \"cdna_end\": 7770, \"location\": \"15:48422068\", \"hgvsp\": \"NP_000129.3:p.Asp2485Val\", \"aa\": \"T\", \"mastermind_counts\": \"8|13|18\", \"allele_num\": 1, \"cadd_raw\": 5.787488, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cdna_start\": 7770, \"rf_score\": 0.6, \"clinpred\": 0.997187077999115, \"consequence_terms\": [\"missense_variant\", \"splice_region_variant\"], \"used_ref\": \"T\", \"cds_start\": 7454, \"mastermind_mmid3\": \"FBN1:D2485V,FBN1:D2485sra\", \"gene_id\": \"2200\", \"blosum62\": -3, \"mutationtaster_pred\": \"D\", \"protein_end\": 2485, \"gene_symbol\": \"FBN1\", \"alphamissense\": {\"am_pathogenicity\": 0.9938, \"am_class\": \"likely_pathogenic\"}, \"sift_score\": 0, \"cadd_phred\": 33, \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"conservation\": 3.77, \"exon\": \"61/66\", \"hgvsc\": \"NM_000138.5:c.7454A>T\", \"maxentscan_diff\": 2.063, \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"sift_prediction\": \"deleterious_low_confidence\", \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"maxentscan_alt\": 7.894, \"codons\": \"gAt/gTt\", \"protein_start\": 2485, \"cds_end\": 7454, \"lrt_pred\": \"invalid_field\", \"loeuf\": 0.105, \"variant_allele\": \"A\", \"polyphen_score\": 0.998, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"D/V\"}], \"start\": 48422068, \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"strand\": 1, \"end\": 48422068, \"id\": \"CM086598\", \"allele_string\": \"HGMD_MUTATION\", \"start\": 48422068, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM154855\", \"end\": 48422068, \"start\": 48422068, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV001729616\", \"RCV000490210\", \"VCV000427047\"]}, \"clin_sig\": [\"likely_pathogenic\"], \"allele_string\": \"T/A/C\", \"end\": 48422068, \"id\": \"rs1085307921\", \"start\": 48422068, \"clin_sig_allele\": \"C:likely_pathogenic\", \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}], \"strand\": 1, \"allele_string\": \"T/A\", \"end\": 48422068, \"input\": \"NC_000015.10:g.48422068T>A\", \"id\": \"NC_000015.10:g.48422068T>A\"}',\n",
       "    'coordinates': [{'pca_x': -0.039996941733121835,\n",
       "      'pca_y': 0.053096523670757394},\n",
       "     {'t-sne_x': -12.436392502425443, 't-sne_y': -3.4963496380854653},\n",
       "     {'umap_x': 0.31800881028175354, 'umap_y': 15.111326217651367}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48472551-C-T',\n",
       "      'distance': 0.0054271891713142395,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465568-C-T',\n",
       "      'distance': 0.005569889210164547,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515382-C-T',\n",
       "      'distance': 0.005787667818367481,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470634-C-T',\n",
       "      'distance': 0.00651968689635396,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420687-C-T',\n",
       "      'distance': 0.0065697901882231236,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48644606-C-T',\n",
       "      'distance': 0.006673780269920826,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481655-C-T',\n",
       "      'distance': 0.0066918907687067986,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446706-C-T',\n",
       "      'distance': 0.006783891934901476,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483818-C-T',\n",
       "      'distance': 0.006999868433922529,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487312-C-T',\n",
       "      'distance': 0.00712455902248621,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48526132-A-G',\n",
       "      'distance': 0.007353785913437605,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48520659-C-T',\n",
       "      'distance': 0.007506116759032011,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48485374-C-T',\n",
       "      'distance': 0.007529441732913256,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48422068-T-C',\n",
       "      'distance': 0.007640020456165075,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497391-T-G',\n",
       "      'distance': 0.00792771391570568,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508582-C-T',\n",
       "      'distance': 0.008563077077269554,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448768-C-T',\n",
       "      'distance': 0.008842007257044315,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488493-T-C',\n",
       "      'distance': 0.009408721700310707,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48445504-T-C',\n",
       "      'distance': 0.00994732603430748,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487075-C-T',\n",
       "      'distance': 0.011073530651628971,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48415769-48422070-del-3557f46a',\n",
       "    'metadata': {'vcf_string': '15\\t48415769\\t.\\tTGCAGGCAAAATAAGAAGCGGCATGTGTGGCAGCAGCCAGAGATTTCTATTGAGGACATTGGATCTGGCCACTTGCTGCCGGCCAGCTGGCCCTCAGCTAGGCTCTCAGGAAGGTAAGACAGGCAGAGGTGGCTGGGTGGGGAGAGCAACAGCAGAGAAAGATGCGGGCATGGATGTGGCTTCTTGCTATGTTGTGAGTAAACATGGAAGATGAAACATGGAAGATGAAGTCAAATGCACCAGGCAGAGGGCATTAGAGGATGGCATTCTGAGGACCAGCAGACTCCAGTTCTCATCAGAAGTCAACACTGGTATCTTCTGCTCCATGTAAACATCCGGTCATGAACCAACATGGGGCTGGTCCGTAGCTAAGATTTAGGGTTCAAGGGGGGAGGAAAGGAATCTTGCAGTTCTCTGCGTACTTGCTTCACAAGTGCCCAGCGAGGAGACAAATCTCTCAAAAGCAGAGACCAACCATTTTCCCGCAGGCTGCTTTCCCTGTCAGCCTATTTCAGGATATGCTGATCTGGCAAAGGCTGATATCCTGCCCCAAGTGTCTGCCCCACGTGTCTGCCCCACTTGGACTTCCTTTTCAAAATTCTGCAAAAGCCCCCAGGTACCTGTTCTGGGAGATGAATAATGCCTTCTGGTATGTCTGCTCCCATCACCGAATCGGCCCCCTCCTTATTCAAACCAGCCTCCTGCCCAGAGTGTCTGAGCCCTGTCTAAAACTAAGTGTGGCTGGAAGAGATTTGGTGCAGACTTCACAAGCAGTAATTGGGGAAAACCCACAAGTGAGACAGAAGCTGCAGGGCTGGCTACCCACAGGGACCTGCCAAGTCTGATGCTAAGAAGCAGGAAAAAGAAATCTTACTTTAAAAAAATCAGCTAGAAGGGGGAAAACCATGAGTTTCTCATTATTTCTGATTTATAATTTTTGGAAGTCTCTAATATATTTTTGTAGATCTGCCTTTGCAGGGCAGGTTGGAAAGTTGCTGGTAAGACATTTCTTAACTGCCATAGTCCATTTTCAGGCATACGTTCATTTGAATGTAGATAGATTGGCCTTCTCTTCACTTAGCGAATAACCACTTCCTATGGAAGAGAGGAAGGGGAAGGGAATGAATGTTTATTAGATACCTATTTTATGATAGATAGGATACAGGTATTGACCATTTCCTTAGCTTTCCTAACAATCCTGAGAAAGAGGTATTATATTCTCTGTTTTACAGCTGAAGAAAGTGAGGCTCAGAAAGCTCTTCCAAGATCTCACAGCCAGCAGCAGCAGGGCAGAAATTCACAGCCAGGTCTGTTGGACTGCAAAGGCCATCCATGCGCTTTCCTTTATACCACAGTGCCTCTCAAGAGACGCTCCACATTTAAGCTGGTGCTTCCTTTCTCCTGTAATTAGTTTACCTTCTTTGCTAAGGCACATTCATTTATAAAGAAAAATAATGTTTCTCAACAAATCTATTTTTTCTTTTTTTTTAAATTAGAAAACAGGTGGATAAAAGTGGTGGGGAAGGATGCTCACTTGTATATACTTAGTATATATTCACATTTCCTACCTGCCTGCCTGCCTGCCTTCCCTCCTTCCATGTCTCCTTCCCTCCTACCTTCTTTCCTTCTCTCCTTCCCTTTCTTCCTTCCTTCCTTCCCCCCTCCCCTCCCCTCCCTCCTTTCCTTCCTTCCTTCCTTCCTTTCCTTCCTTCCTTCCTTCTCTCCTCCCCTCCTTCCTTCTCTCTTTCCCTCCTTCCCTTCAGCTACAGGTCTTTGATCTTAACTGCCTGTCTGAAAATCTTTTGTATCCAAAGACCTTCAAGTCTTCACCCAAGGTCACTTAGGCTCCAGGGCTCTTAAAATATAAATAATAATTTTATCTTTCTCATTCTTCCCTTTCTCTTTCTGTGCTCAACACGTTATCTGTGTTCTTTCTCCTTATTTTAAATTGAGGACGAAGTGTGCAATAGAGAGCAACAGAAACCACTGGCACCAAGCTCCATGTCAAACAAAGAATGAAGAACGAAGCATATTTACTTCATTGGAGAGAACACCCCTGGGCTGATGTCTTCTTGCTCCTCCACGTCCATCCTCCATCCTCTCCACTTGCTCTTGGCCCCAGGTGGCTGATCTATGGAGCATGAGCAAGCTCTGCTCCCCTGTCCCTCACTTTTGGTTGGCCTCAGCCATCTGCTTCTTGCTGGGACCCTGACTAATAAAACCTGCAAGGCGTGGGTATATGAGGGGGAAGACTTGTCATTCTATAGCTTAGATTTATTCTGTGTTTTTTCCTCCAAAGAGCAGAAGTAGGACCAATGGTGAGAAATTAGAGGGAGGAGAAATGATTTTTTTTATAGCAAAAAAAGACATTTCTAAGTATTAAAAGCTGTCTATACATAGATTGGGCTATCTTGGGAGAAAGTGGGCACCCTTCAAGGTGGTATTTAAGCCAAACTGGGATAAGCAAGGAAGTTAGAGAAGGATTTATATATTAGGAAAGAGAGAAGGCTAGATCTCTAAGGTTCTTTTTAACTTTGAGCATCTATCTGTGTTACTACAACTATCAAGAGTTCTCCCCCTCTCCTTTCACATTCTTTTAAATTTTATTTTACATTTTCTCCCTTCCCTGAGAATTTAATAAAGTTTAAATTAAACTTGCTTATCTAGGTTGGAGAGGTAGGATGGAGTAGAGCACAGATTAAAATAAGAAATTGGCAAATATTCAGTGTTTACAGTTTGAAAACAGCATGTTTGTGCACTCGCCCTTTTTATCAATATACAAACACCACGATTTGGTTAACATGTTTTGCACTTTTCCTGACCTTTTTCATCCTCTGCACTCTGGAGCAAATGACTCCACAAGGAATTTTTTTACACTCCAAAATATTTGTTTTATTCTTCCAGTTGGTACTTAGATTGGTGAGGGTTTTGTTTGTTTTATAAAAAGAGACAATTCGAATTTTACCAAAATAGCATTAAAGTCTGAAAACTACTCCTTAAAAGTGGATTGAAACCCACCACTCTGTCCTCTAAAATCAAAGGTTTCAGCATCCAGCTTGCACTGTTCATGTTAGCTGTGGCCTTAAGGGAGTACCGATAGGAAACTTCACAAATGCAGGATGGGCTTTCTTTTACTCATTTTCTCTAGGGGAAAATGGGTCAACTCTATCTATCTGCAAGCTGAATTTTTTGAAATTTACTTCTAGAAAGCAATGAAATTAGTGGCCCTCTTACCCCAAAGAGTGAACTTCCAGCCACCTGGGAGTGAGGATCCAGGGAGGGCTGGTCTAAGAGATCTGCTCTGTTTCTTGTGGTTCTGGGGAAGCTATTTTAGCCCACGGCAATGCTCATTCTAGATAACTCTAATTCTCAAGATTGATGAGAGCAATTTGGCATTTGGAGACATTAAGAACCCCAAACTTCTTCTGACTTCTAAATCAACTTGTTTTTTTAGAGTGTCCAGCCAGGAGATGATCTTCACGAGACCAGAGCAGAATCTCCAGGACACTAGTGCATGGCCTGGTCGCCTCTACCATCCTCAGACTGATGGTAGGAGGAGGAGGATAAGGTACAGGAGCGACATACCTTTCCTCTCTTATGTTTTTTTCACTTAGCTCCAGTGTTATTAAAATGACCTAAATAATGAGAACAGACAGGCTCATAAACTGGGTGTTGAATCAACAAAGTTCACTCTGGAAGTTCTGTGAACAATGTCAGAAACTGGTGATTAAAATCAAAGACCTCCCTGAGCTGCTAATGAACAAAGACATTTTTATTAAAACCAGATAGCTAGCCAGAGACTCACGCAGTCGAAACTCCTATAGCACAGGGGTCACTTCAGTCAACCCTACATGCCACTGCAAAACTGTCTTGTGTAGAATCAAACTAGATGACAGAGATGATTTTGATAAATGAAAACACACCAAATATGTCTTCATCTGCATGAAAGTGAACCCCTGCAATGCAGAGCCTGCACCCGGGAAGCAGTAATTCTTTGAGTTTTTGCTAAGGAGGGCTCTGTACAGAATAATGTGGCAACTAGAACAGGTTACCAAAGCACCAAGCTTGCAGTGGCACTTCACAGACGCTTGCTGAATGAACATAAACAAGTGAATGTGACACTTGAAAGGCAGCTCTACATCAGAAAGTTAAGAGTTGTGGGTTCTAGTTCCAAGCATGTCACAAAAGAACTGGGCAAGGAAATATTACCGTCCCTTTCTGGACTTCTAGGTCCTTATCTGTAAAACAAAGTGCCTGAGCTAAGTGGCTTCTGGTATCCCTCCAAACTCACAATGTTTTGATTCTCTGATGTGCATTAAGAGTTATTTTCCCATGACATTGTAGAATCAAAGATGACATCATTTCAATTTAATGAGAGAGTCAGTAGATCTCCCTTAATTATCAAAAGAATTTAGCAATCTTGAAAAACAAAACAAAACAAAACAAAACACACAAAAAAACACAACCTTGCTGTGTTTCCTGCTTGCGCTCTGTCTCAAAATGTGTAGAGCTGAACAGAACTACTTTGGCATCTTCAGGCCAAAAGCCACTCAAAATAGCCTTTGGCTGGCTGATATTTTTCAGTCATAAGTAAGATTTTTTGACCCCTGATATGAACATAACTCAAGTGTGGGCTGATGACTAGCAGTGCAGCGGGAGGGTGGAGGATATTGCGCTTCAAGAAATTCTGGGGGCATTTCATTTCAGATGCTCTCAAAGCACCCTCAATTATAATTTTTTTTCTTAGTTTGCTGTTTTGCAAAGACTGTTACTTACCTCGGGTTTCAACCAGGTTAGGGCAATTTTAAATGAGTATTTGTTGCTTCAATAACACATTTACAGCTTCAGAAAGCAAGCAGTGTTTTGCTTCATAGGACCTGATAGCCATGCATCTTGAGAGTGAGGAAAAGTTACTTGCCAACACACTGGTTCCACTGGTAGTGCTGGAGGTAGCCCTGGGGGCAGCTGCACCTGTAGCCCCCAATGATGTTCTGGCAGCCATGCTGGCAGCGGTGGTTACCCTCACACTCGTCCACGTCTGAAAAAGAAGCAGAGCCACCATGATGCCAACTCAACATCTCTCTCTGAAGCCAGATGGATTCTAATAAGAAATCTGGCCCCTACACATCCTACACATTATTCTACCACCAAAAACTTCAAGAATCTCTCTTCTGGAACTGCTGCTAAGTCCAAGGAGAAAATCATTCAGAGAGCCCTGGCAAAGTCTACAAATAAAAATTGTCAGGAAATTAGTGCAAAGCCCAAAGCACTCTCTGGGACAGCTCTGTAAGTTATAAGTTATTTGTGTTTGCTCTTTTTTTTTTTTATTTTTGGTGCCACAATTAGAAACCAAAACGTGTCAAAGCCACAGTCAACTTCTAAAAATATCATTTAAACTTCTAATTAAATGTCATCAGAAGAGCTATTCCTCATATGTTTAGTCCACATGGGGGAAATCCCTTTATGGCTCTGGACCCAGTGACCAGAAAATTAATGAATCAAAATCAAGTATATTTTTCTTGACTGAGTTATTCCCTATTAGGTTAAATTAAAAACTTAGCAGAAAAAGTCTTGATATACTTAGGAAAGGCATGTGTGTTAGACACAATCAAGCATGAAACAAAACACGACTCTCTAAGATACAGGACCTGTGCACACTATTTTAGCTGAGTGCCCCCCTGGGCTCAGATCTGCTATCTCATAGAGGCTGATGATGAAGGTGCCAATAGCCACACAGGCCACCTCCACAAGGATTCACCAGCTGGATCGCAGCTGAAGTCTCCACCCACCTTCACAGCTGGAGCCGGTCTGATCAAGTGAGAATCCCCGCTGGCATTCACAGGTGAAGCTTCCAGGAGTGTTCTGGCAAATGCCCTTAGACCCGCACAGATTGATGTCAGAGGTGCATTCATTGTTATCTATGAGAAGCAGTGGGGGCAAAGAGGGGTTAAAATTCCCCAAAGCTCTCTTTGTATCATTTAAAAGAAAATTAGAAGGATAACTCGGAAAAGGCCAAATAAGGCCAACAACGCTTCACATACATGTACAGGTGTGCTCACACATACATGCACTCATATATTTCTTTGAGCTTTTATTTTCTTATCCCAACAGCAGAGGAAATAGAAAATAATCCCTTAAAAGAATCGCTACAATCCATGTAGGATTTTTTCCTCTCCTACTCACCAATGCAGGACGTATGGTGTTGGGTAAATCCGGGAGGACATTTGCATGTGAAGCCGCCAATGGTGTTAACACATAGGAACTGGCAGTTGTGTTGCTTGGTTGCACACTCATCAAGATCT\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415769',\n",
       "     'ref_allele': 'TGCAGGCAAAATAAGAAGCGGCATGTGTGGCAGCAGCCAGAGATTTCTATTGAGGACATTGGATCTGGCCACTTGCTGCCGGCCAGCTGGCCCTCAGCTAGGCTCTCAGGAAGGTAAGACAGGCAGAGGTGGCTGGGTGGGGAGAGCAACAGCAGAGAAAGATGCGGGCATGGATGTGGCTTCTTGCTATGTTGTGAGTAAACATGGAAGATGAAACATGGAAGATGAAGTCAAATGCACCAGGCAGAGGGCATTAGAGGATGGCATTCTGAGGACCAGCAGACTCCAGTTCTCATCAGAAGTCAACACTGGTATCTTCTGCTCCATGTAAACATCCGGTCATGAACCAACATGGGGCTGGTCCGTAGCTAAGATTTAGGGTTCAAGGGGGGAGGAAAGGAATCTTGCAGTTCTCTGCGTACTTGCTTCACAAGTGCCCAGCGAGGAGACAAATCTCTCAAAAGCAGAGACCAACCATTTTCCCGCAGGCTGCTTTCCCTGTCAGCCTATTTCAGGATATGCTGATCTGGCAAAGGCTGATATCCTGCCCCAAGTGTCTGCCCCACGTGTCTGCCCCACTTGGACTTCCTTTTCAAAATTCTGCAAAAGCCCCCAGGTACCTGTTCTGGGAGATGAATAATGCCTTCTGGTATGTCTGCTCCCATCACCGAATCGGCCCCCTCCTTATTCAAACCAGCCTCCTGCCCAGAGTGTCTGAGCCCTGTCTAAAACTAAGTGTGGCTGGAAGAGATTTGGTGCAGACTTCACAAGCAGTAATTGGGGAAAACCCACAAGTGAGACAGAAGCTGCAGGGCTGGCTACCCACAGGGACCTGCCAAGTCTGATGCTAAGAAGCAGGAAAAAGAAATCTTACTTTAAAAAAATCAGCTAGAAGGGGGAAAACCATGAGTTTCTCATTATTTCTGATTTATAATTTTTGGAAGTCTCTAATATATTTTTGTAGATCTGCCTTTGCAGGGCAGGTTGGAAAGTTGCTGGTAAGACATTTCTTAACTGCCATAGTCCATTTTCAGGCATACGTTCATTTGAATGTAGATAGATTGGCCTTCTCTTCACTTAGCGAATAACCACTTCCTATGGAAGAGAGGAAGGGGAAGGGAATGAATGTTTATTAGATACCTATTTTATGATAGATAGGATACAGGTATTGACCATTTCCTTAGCTTTCCTAACAATCCTGAGAAAGAGGTATTATATTCTCTGTTTTACAGCTGAAGAAAGTGAGGCTCAGAAAGCTCTTCCAAGATCTCACAGCCAGCAGCAGCAGGGCAGAAATTCACAGCCAGGTCTGTTGGACTGCAAAGGCCATCCATGCGCTTTCCTTTATACCACAGTGCCTCTCAAGAGACGCTCCACATTTAAGCTGGTGCTTCCTTTCTCCTGTAATTAGTTTACCTTCTTTGCTAAGGCACATTCATTTATAAAGAAAAATAATGTTTCTCAACAAATCTATTTTTTCTTTTTTTTTAAATTAGAAAACAGGTGGATAAAAGTGGTGGGGAAGGATGCTCACTTGTATATACTTAGTATATATTCACATTTCCTACCTGCCTGCCTGCCTGCCTTCCCTCCTTCCATGTCTCCTTCCCTCCTACCTTCTTTCCTTCTCTCCTTCCCTTTCTTCCTTCCTTCCTTCCCCCCTCCCCTCCCCTCCCTCCTTTCCTTCCTTCCTTCCTTCCTTTCCTTCCTTCCTTCCTTCTCTCCTCCCCTCCTTCCTTCTCTCTTTCCCTCCTTCCCTTCAGCTACAGGTCTTTGATCTTAACTGCCTGTCTGAAAATCTTTTGTATCCAAAGACCTTCAAGTCTTCACCCAAGGTCACTTAGGCTCCAGGGCTCTTAAAATATAAATAATAATTTTATCTTTCTCATTCTTCCCTTTCTCTTTCTGTGCTCAACACGTTATCTGTGTTCTTTCTCCTTATTTTAAATTGAGGACGAAGTGTGCAATAGAGAGCAACAGAAACCACTGGCACCAAGCTCCATGTCAAACAAAGAATGAAGAACGAAGCATATTTACTTCATTGGAGAGAACACCCCTGGGCTGATGTCTTCTTGCTCCTCCACGTCCATCCTCCATCCTCTCCACTTGCTCTTGGCCCCAGGTGGCTGATCTATGGAGCATGAGCAAGCTCTGCTCCCCTGTCCCTCACTTTTGGTTGGCCTCAGCCATCTGCTTCTTGCTGGGACCCTGACTAATAAAACCTGCAAGGCGTGGGTATATGAGGGGGAAGACTTGTCATTCTATAGCTTAGATTTATTCTGTGTTTTTTCCTCCAAAGAGCAGAAGTAGGACCAATGGTGAGAAATTAGAGGGAGGAGAAATGATTTTTTTTATAGCAAAAAAAGACATTTCTAAGTATTAAAAGCTGTCTATACATAGATTGGGCTATCTTGGGAGAAAGTGGGCACCCTTCAAGGTGGTATTTAAGCCAAACTGGGATAAGCAAGGAAGTTAGAGAAGGATTTATATATTAGGAAAGAGAGAAGGCTAGATCTCTAAGGTTCTTTTTAACTTTGAGCATCTATCTGTGTTACTACAACTATCAAGAGTTCTCCCCCTCTCCTTTCACATTCTTTTAAATTTTATTTTACATTTTCTCCCTTCCCTGAGAATTTAATAAAGTTTAAATTAAACTTGCTTATCTAGGTTGGAGAGGTAGGATGGAGTAGAGCACAGATTAAAATAAGAAATTGGCAAATATTCAGTGTTTACAGTTTGAAAACAGCATGTTTGTGCACTCGCCCTTTTTATCAATATACAAACACCACGATTTGGTTAACATGTTTTGCACTTTTCCTGACCTTTTTCATCCTCTGCACTCTGGAGCAAATGACTCCACAAGGAATTTTTTTACACTCCAAAATATTTGTTTTATTCTTCCAGTTGGTACTTAGATTGGTGAGGGTTTTGTTTGTTTTATAAAAAGAGACAATTCGAATTTTACCAAAATAGCATTAAAGTCTGAAAACTACTCCTTAAAAGTGGATTGAAACCCACCACTCTGTCCTCTAAAATCAAAGGTTTCAGCATCCAGCTTGCACTGTTCATGTTAGCTGTGGCCTTAAGGGAGTACCGATAGGAAACTTCACAAATGCAGGATGGGCTTTCTTTTACTCATTTTCTCTAGGGGAAAATGGGTCAACTCTATCTATCTGCAAGCTGAATTTTTTGAAATTTACTTCTAGAAAGCAATGAAATTAGTGGCCCTCTTACCCCAAAGAGTGAACTTCCAGCCACCTGGGAGTGAGGATCCAGGGAGGGCTGGTCTAAGAGATCTGCTCTGTTTCTTGTGGTTCTGGGGAAGCTATTTTAGCCCACGGCAATGCTCATTCTAGATAACTCTAATTCTCAAGATTGATGAGAGCAATTTGGCATTTGGAGACATTAAGAACCCCAAACTTCTTCTGACTTCTAAATCAACTTGTTTTTTTAGAGTGTCCAGCCAGGAGATGATCTTCACGAGACCAGAGCAGAATCTCCAGGACACTAGTGCATGGCCTGGTCGCCTCTACCATCCTCAGACTGATGGTAGGAGGAGGAGGATAAGGTACAGGAGCGACATACCTTTCCTCTCTTATGTTTTTTTCACTTAGCTCCAGTGTTATTAAAATGACCTAAATAATGAGAACAGACAGGCTCATAAACTGGGTGTTGAATCAACAAAGTTCACTCTGGAAGTTCTGTGAACAATGTCAGAAACTGGTGATTAAAATCAAAGACCTCCCTGAGCTGCTAATGAACAAAGACATTTTTATTAAAACCAGATAGCTAGCCAGAGACTCACGCAGTCGAAACTCCTATAGCACAGGGGTCACTTCAGTCAACCCTACATGCCACTGCAAAACTGTCTTGTGTAGAATCAAACTAGATGACAGAGATGATTTTGATAAATGAAAACACACCAAATATGTCTTCATCTGCATGAAAGTGAACCCCTGCAATGCAGAGCCTGCACCCGGGAAGCAGTAATTCTTTGAGTTTTTGCTAAGGAGGGCTCTGTACAGAATAATGTGGCAACTAGAACAGGTTACCAAAGCACCAAGCTTGCAGTGGCACTTCACAGACGCTTGCTGAATGAACATAAACAAGTGAATGTGACACTTGAAAGGCAGCTCTACATCAGAAAGTTAAGAGTTGTGGGTTCTAGTTCCAAGCATGTCACAAAAGAACTGGGCAAGGAAATATTACCGTCCCTTTCTGGACTTCTAGGTCCTTATCTGTAAAACAAAGTGCCTGAGCTAAGTGGCTTCTGGTATCCCTCCAAACTCACAATGTTTTGATTCTCTGATGTGCATTAAGAGTTATTTTCCCATGACATTGTAGAATCAAAGATGACATCATTTCAATTTAATGAGAGAGTCAGTAGATCTCCCTTAATTATCAAAAGAATTTAGCAATCTTGAAAAACAAAACAAAACAAAACAAAACACACAAAAAAACACAACCTTGCTGTGTTTCCTGCTTGCGCTCTGTCTCAAAATGTGTAGAGCTGAACAGAACTACTTTGGCATCTTCAGGCCAAAAGCCACTCAAAATAGCCTTTGGCTGGCTGATATTTTTCAGTCATAAGTAAGATTTTTTGACCCCTGATATGAACATAACTCAAGTGTGGGCTGATGACTAGCAGTGCAGCGGGAGGGTGGAGGATATTGCGCTTCAAGAAATTCTGGGGGCATTTCATTTCAGATGCTCTCAAAGCACCCTCAATTATAATTTTTTTTCTTAGTTTGCTGTTTTGCAAAGACTGTTACTTACCTCGGGTTTCAACCAGGTTAGGGCAATTTTAAATGAGTATTTGTTGCTTCAATAACACATTTACAGCTTCAGAAAGCAAGCAGTGTTTTGCTTCATAGGACCTGATAGCCATGCATCTTGAGAGTGAGGAAAAGTTACTTGCCAACACACTGGTTCCACTGGTAGTGCTGGAGGTAGCCCTGGGGGCAGCTGCACCTGTAGCCCCCAATGATGTTCTGGCAGCCATGCTGGCAGCGGTGGTTACCCTCACACTCGTCCACGTCTGAAAAAGAAGCAGAGCCACCATGATGCCAACTCAACATCTCTCTCTGAAGCCAGATGGATTCTAATAAGAAATCTGGCCCCTACACATCCTACACATTATTCTACCACCAAAAACTTCAAGAATCTCTCTTCTGGAACTGCTGCTAAGTCCAAGGAGAAAATCATTCAGAGAGCCCTGGCAAAGTCTACAAATAAAAATTGTCAGGAAATTAGTGCAAAGCCCAAAGCACTCTCTGGGACAGCTCTGTAAGTTATAAGTTATTTGTGTTTGCTCTTTTTTTTTTTTATTTTTGGTGCCACAATTAGAAACCAAAACGTGTCAAAGCCACAGTCAACTTCTAAAAATATCATTTAAACTTCTAATTAAATGTCATCAGAAGAGCTATTCCTCATATGTTTAGTCCACATGGGGGAAATCCCTTTATGGCTCTGGACCCAGTGACCAGAAAATTAATGAATCAAAATCAAGTATATTTTTCTTGACTGAGTTATTCCCTATTAGGTTAAATTAAAAACTTAGCAGAAAAAGTCTTGATATACTTAGGAAAGGCATGTGTGTTAGACACAATCAAGCATGAAACAAAACACGACTCTCTAAGATACAGGACCTGTGCACACTATTTTAGCTGAGTGCCCCCCTGGGCTCAGATCTGCTATCTCATAGAGGCTGATGATGAAGGTGCCAATAGCCACACAGGCCACCTCCACAAGGATTCACCAGCTGGATCGCAGCTGAAGTCTCCACCCACCTTCACAGCTGGAGCCGGTCTGATCAAGTGAGAATCCCCGCTGGCATTCACAGGTGAAGCTTCCAGGAGTGTTCTGGCAAATGCCCTTAGACCCGCACAGATTGATGTCAGAGGTGCATTCATTGTTATCTATGAGAAGCAGTGGGGGCAAAGAGGGGTTAAAATTCCCCAAAGCTCTCTTTGTATCATTTAAAAGAAAATTAGAAGGATAACTCGGAAAAGGCCAAATAAGGCCAACAACGCTTCACATACATGTACAGGTGTGCTCACACATACATGCACTCATATATTTCTTTGAGCTTTTATTTTCTTATCCCAACAGCAGAGGAAATAGAAAATAATCCCTTAAAAGAATCGCTACAATCCATGTAGGATTTTTTCCTCTCCTACTCACCAATGCAGGACGTATGGTGTTGGGTAAATCCGGGAGGACATTTGCATGTGAAGCCGCCAATGGTGTTAACACATAGGAACTGGCAGTTGTGTTGCTTGGTTGCACACTCATCAAGATCT',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7456_7821del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415770_48422070del',\n",
       "     'hgvs_protein': 'p.Leu2486_Asp2607del',\n",
       "     'most_severe_consequence': 'splice_acceptor_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"deletion\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"splice_acceptor_variant\", \"minimised\": 1, \"vcf_string\": \"15-48415769-TGCAGGCAAAATAAGAAGCGGCATGTGTGGCAGCAGCCAGAGATTTCTATTGAGGACATTGGATCTGGCCACTTGCTGCCGGCCAGCTGGCCCTCAGCTAGGCTCTCAGGAAGGTAAGACAGGCAGAGGTGGCTGGGTGGGGAGAGCAACAGCAGAGAAAGATGCGGGCATGGATGTGGCTTCTTGCTATGTTGTGAGTAAACATGGAAGATGAAACATGGAAGATGAAGTCAAATGCACCAGGCAGAGGGCATTAGAGGATGGCATTCTGAGGACCAGCAGACTCCAGTTCTCATCAGAAGTCAACACTGGTATCTTCTGCTCCATGTAAACATCCGGTCATGAACCAACATGGGGCTGGTCCGTAGCTAAGATTTAGGGTTCAAGGGGGGAGGAAAGGAATCTTGCAGTTCTCTGCGTACTTGCTTCACAAGTGCCCAGCGAGGAGACAAATCTCTCAAAAGCAGAGACCAACCATTTTCCCGCAGGCTGCTTTCCCTGTCAGCCTATTTCAGGATATGCTGATCTGGCAAAGGCTGATATCCTGCCCCAAGTGTCTGCCCCACGTGTCTGCCCCACTTGGACTTCCTTTTCAAAATTCTGCAAAAGCCCCCAGGTACCTGTTCTGGGAGATGAATAATGCCTTCTGGTATGTCTGCTCCCATCACCGAATCGGCCCCCTCCTTATTCAAACCAGCCTCCTGCCCAGAGTGTCTGAGCCCTGTCTAAAACTAAGTGTGGCTGGAAGAGATTTGGTGCAGACTTCACAAGCAGTAATTGGGGAAAACCCACAAGTGAGACAGAAGCTGCAGGGCTGGCTACCCACAGGGACCTGCCAAGTCTGATGCTAAGAAGCAGGAAAAAGAAATCTTACTTTAAAAAAATCAGCTAGAAGGGGGAAAACCATGAGTTTCTCATTATTTCTGATTTATAATTTTTGGAAGTCTCTAATATATTTTTGTAGATCTGCCTTTGCAGGGCAGGTTGGAAAGTTGCTGGTAAGACATTTCTTAACTGCCATAGTCCATTTTCAGGCATACGTTCATTTGAATGTAGATAGATTGGCCTTCTCTTCACTTAGCGAATAACCACTTCCTATGGAAGAGAGGAAGGGGAAGGGAATGAATGTTTATTAGATACCTATTTTATGATAGATAGGATACAGGTATTGACCATTTCCTTAGCTTTCCTAACAATCCTGAGAAAGAGGTATTATATTCTCTGTTTTACAGCTGAAGAAAGTGAGGCTCAGAAAGCTCTTCCAAGATCTCACAGCCAGCAGCAGCAGGGCAGAAATTCACAGCCAGGTCTGTTGGACTGCAAAGGCCATCCATGCGCTTTCCTTTATACCACAGTGCCTCTCAAGAGACGCTCCACATTTAAGCTGGTGCTTCCTTTCTCCTGTAATTAGTTTACCTTCTTTGCTAAGGCACATTCATTTATAAAGAAAAATAATGTTTCTCAACAAATCTATTTTTTCTTTTTTTTTAAATTAGAAAACAGGTGGATAAAAGTGGTGGGGAAGGATGCTCACTTGTATATACTTAGTATATATTCACATTTCCTACCTGCCTGCCTGCCTGCCTTCCCTCCTTCCATGTCTCCTTCCCTCCTACCTTCTTTCCTTCTCTCCTTCCCTTTCTTCCTTCCTTCCTTCCCCCCTCCCCTCCCCTCCCTCCTTTCCTTCCTTCCTTCCTTCCTTTCCTTCCTTCCTTCCTTCTCTCCTCCCCTCCTTCCTTCTCTCTTTCCCTCCTTCCCTTCAGCTACAGGTCTTTGATCTTAACTGCCTGTCTGAAAATCTTTTGTATCCAAAGACCTTCAAGTCTTCACCCAAGGTCACTTAGGCTCCAGGGCTCTTAAAATATAAATAATAATTTTATCTTTCTCATTCTTCCCTTTCTCTTTCTGTGCTCAACACGTTATCTGTGTTCTTTCTCCTTATTTTAAATTGAGGACGAAGTGTGCAATAGAGAGCAACAGAAACCACTGGCACCAAGCTCCATGTCAAACAAAGAATGAAGAACGAAGCATATTTACTTCATTGGAGAGAACACCCCTGGGCTGATGTCTTCTTGCTCCTCCACGTCCATCCTCCATCCTCTCCACTTGCTCTTGGCCCCAGGTGGCTGATCTATGGAGCATGAGCAAGCTCTGCTCCCCTGTCCCTCACTTTTGGTTGGCCTCAGCCATCTGCTTCTTGCTGGGACCCTGACTAATAAAACCTGCAAGGCGTGGGTATATGAGGGGGAAGACTTGTCATTCTATAGCTTAGATTTATTCTGTGTTTTTTCCTCCAAAGAGCAGAAGTAGGACCAATGGTGAGAAATTAGAGGGAGGAGAAATGATTTTTTTTATAGCAAAAAAAGACATTTCTAAGTATTAAAAGCTGTCTATACATAGATTGGGCTATCTTGGGAGAAAGTGGGCACCCTTCAAGGTGGTATTTAAGCCAAACTGGGATAAGCAAGGAAGTTAGAGAAGGATTTATATATTAGGAAAGAGAGAAGGCTAGATCTCTAAGGTTCTTTTTAACTTTGAGCATCTATCTGTGTTACTACAACTATCAAGAGTTCTCCCCCTCTCCTTTCACATTCTTTTAAATTTTATTTTACATTTTCTCCCTTCCCTGAGAATTTAATAAAGTTTAAATTAAACTTGCTTATCTAGGTTGGAGAGGTAGGATGGAGTAGAGCACAGATTAAAATAAGAAATTGGCAAATATTCAGTGTTTACAGTTTGAAAACAGCATGTTTGTGCACTCGCCCTTTTTATCAATATACAAACACCACGATTTGGTTAACATGTTTTGCACTTTTCCTGACCTTTTTCATCCTCTGCACTCTGGAGCAAATGACTCCACAAGGAATTTTTTTACACTCCAAAATATTTGTTTTATTCTTCCAGTTGGTACTTAGATTGGTGAGGGTTTTGTTTGTTTTATAAAAAGAGACAATTCGAATTTTACCAAAATAGCATTAAAGTCTGAAAACTACTCCTTAAAAGTGGATTGAAACCCACCACTCTGTCCTCTAAAATCAAAGGTTTCAGCATCCAGCTTGCACTGTTCATGTTAGCTGTGGCCTTAAGGGAGTACCGATAGGAAACTTCACAAATGCAGGATGGGCTTTCTTTTACTCATTTTCTCTAGGGGAAAATGGGTCAACTCTATCTATCTGCAAGCTGAATTTTTTGAAATTTACTTCTAGAAAGCAATGAAATTAGTGGCCCTCTTACCCCAAAGAGTGAACTTCCAGCCACCTGGGAGTGAGGATCCAGGGAGGGCTGGTCTAAGAGATCTGCTCTGTTTCTTGTGGTTCTGGGGAAGCTATTTTAGCCCACGGCAATGCTCATTCTAGATAACTCTAATTCTCAAGATTGATGAGAGCAATTTGGCATTTGGAGACATTAAGAACCCCAAACTTCTTCTGACTTCTAAATCAACTTGTTTTTTTAGAGTGTCCAGCCAGGAGATGATCTTCACGAGACCAGAGCAGAATCTCCAGGACACTAGTGCATGGCCTGGTCGCCTCTACCATCCTCAGACTGATGGTAGGAGGAGGAGGATAAGGTACAGGAGCGACATACCTTTCCTCTCTTATGTTTTTTTCACTTAGCTCCAGTGTTATTAAAATGACCTAAATAATGAGAACAGACAGGCTCATAAACTGGGTGTTGAATCAACAAAGTTCACTCTGGAAGTTCTGTGAACAATGTCAGAAACTGGTGATTAAAATCAAAGACCTCCCTGAGCTGCTAATGAACAAAGACATTTTTATTAAAACCAGATAGCTAGCCAGAGACTCACGCAGTCGAAACTCCTATAGCACAGGGGTCACTTCAGTCAACCCTACATGCCACTGCAAAACTGTCTTGTGTAGAATCAAACTAGATGACAGAGATGATTTTGATAAATGAAAACACACCAAATATGTCTTCATCTGCATGAAAGTGAACCCCTGCAATGCAGAGCCTGCACCCGGGAAGCAGTAATTCTTTGAGTTTTTGCTAAGGAGGGCTCTGTACAGAATAATGTGGCAACTAGAACAGGTTACCAAAGCACCAAGCTTGCAGTGGCACTTCACAGACGCTTGCTGAATGAACATAAACAAGTGAATGTGACACTTGAAAGGCAGCTCTACATCAGAAAGTTAAGAGTTGTGGGTTCTAGTTCCAAGCATGTCACAAAAGAACTGGGCAAGGAAATATTACCGTCCCTTTCTGGACTTCTAGGTCCTTATCTGTAAAACAAAGTGCCTGAGCTAAGTGGCTTCTGGTATCCCTCCAAACTCACAATGTTTTGATTCTCTGATGTGCATTAAGAGTTATTTTCCCATGACATTGTAGAATCAAAGATGACATCATTTCAATTTAATGAGAGAGTCAGTAGATCTCCCTTAATTATCAAAAGAATTTAGCAATCTTGAAAAACAAAACAAAACAAAACAAAACACACAAAAAAACACAACCTTGCTGTGTTTCCTGCTTGCGCTCTGTCTCAAAATGTGTAGAGCTGAACAGAACTACTTTGGCATCTTCAGGCCAAAAGCCACTCAAAATAGCCTTTGGCTGGCTGATATTTTTCAGTCATAAGTAAGATTTTTTGACCCCTGATATGAACATAACTCAAGTGTGGGCTGATGACTAGCAGTGCAGCGGGAGGGTGGAGGATATTGCGCTTCAAGAAATTCTGGGGGCATTTCATTTCAGATGCTCTCAAAGCACCCTCAATTATAATTTTTTTTCTTAGTTTGCTGTTTTGCAAAGACTGTTACTTACCTCGGGTTTCAACCAGGTTAGGGCAATTTTAAATGAGTATTTGTTGCTTCAATAACACATTTACAGCTTCAGAAAGCAAGCAGTGTTTTGCTTCATAGGACCTGATAGCCATGCATCTTGAGAGTGAGGAAAAGTTACTTGCCAACACACTGGTTCCACTGGTAGTGCTGGAGGTAGCCCTGGGGGCAGCTGCACCTGTAGCCCCCAATGATGTTCTGGCAGCCATGCTGGCAGCGGTGGTTACCCTCACACTCGTCCACGTCTGAAAAAGAAGCAGAGCCACCATGATGCCAACTCAACATCTCTCTCTGAAGCCAGATGGATTCTAATAAGAAATCTGGCCCCTACACATCCTACACATTATTCTACCACCAAAAACTTCAAGAATCTCTCTTCTGGAACTGCTGCTAAGTCCAAGGAGAAAATCATTCAGAGAGCCCTGGCAAAGTCTACAAATAAAAATTGTCAGGAAATTAGTGCAAAGCCCAAAGCACTCTCTGGGACAGCTCTGTAAGTTATAAGTTATTTGTGTTTGCTCTTTTTTTTTTTTATTTTTGGTGCCACAATTAGAAACCAAAACGTGTCAAAGCCACAGTCAACTTCTAAAAATATCATTTAAACTTCTAATTAAATGTCATCAGAAGAGCTATTCCTCATATGTTTAGTCCACATGGGGGAAATCCCTTTATGGCTCTGGACCCAGTGACCAGAAAATTAATGAATCAAAATCAAGTATATTTTTCTTGACTGAGTTATTCCCTATTAGGTTAAATTAAAAACTTAGCAGAAAAAGTCTTGATATACTTAGGAAAGGCATGTGTGTTAGACACAATCAAGCATGAAACAAAACACGACTCTCTAAGATACAGGACCTGTGCACACTATTTTAGCTGAGTGCCCCCCTGGGCTCAGATCTGCTATCTCATAGAGGCTGATGATGAAGGTGCCAATAGCCACACAGGCCACCTCCACAAGGATTCACCAGCTGGATCGCAGCTGAAGTCTCCACCCACCTTCACAGCTGGAGCCGGTCTGATCAAGTGAGAATCCCCGCTGGCATTCACAGGTGAAGCTTCCAGGAGTGTTCTGGCAAATGCCCTTAGACCCGCACAGATTGATGTCAGAGGTGCATTCATTGTTATCTATGAGAAGCAGTGGGGGCAAAGAGGGGTTAAAATTCCCCAAAGCTCTCTTTGTATCATTTAAAAGAAAATTAGAAGGATAACTCGGAAAAGGCCAAATAAGGCCAACAACGCTTCACATACATGTACAGGTGTGCTCACACATACATGCACTCATATATTTCTTTGAGCTTTTATTTTCTTATCCCAACAGCAGAGGAAATAGAAAATAATCCCTTAAAAGAATCGCTACAATCCATGTAGGATTTTTTCCTCTCCTACTCACCAATGCAGGACGTATGGTGTTGGGTAAATCCGGGAGGACATTTGCATGTGAAGCCGCCAATGGTGTTAACACATAGGAACTGGCAGTTGTGTTGCTTGGTTGCACACTCATCAAGATCT-T\", \"input\": \"NC_000015.10:g.48415770_48422070del\", \"end\": 48422070, \"id\": \"NC_000015.10:g.48415770_48422070del\", \"allele_string\": \"GCAGGCAAAATAAGAAGCGGCATGTGTGGCAGCAGCCAGAGATTTCTATTGAGGACATTGGATCTGGCCACTTGCTGCCGGCCAGCTGGCCCTCAGCTAGGCTCTCAGGAAGGTAAGACAGGCAGAGGTGGCTGGGTGGGGAGAGCAACAGCAGAGAAAGATGCGGGCATGGATGTGGCTTCTTGCTATGTTGTGAGTAAACATGGAAGATGAAACATGGAAGATGAAGTCAAATGCACCAGGCAGAGGGCATTAGAGGATGGCATTCTGAGGACCAGCAGACTCCAGTTCTCATCAGAAGTCAACACTGGTATCTTCTGCTCCATGTAAACATCCGGTCATGAACCAACATGGGGCTGGTCCGTAGCTAAGATTTAGGGTTCAAGGGGGGAGGAAAGGAATCTTGCAGTTCTCTGCGTACTTGCTTCACAAGTGCCCAGCGAGGAGACAAATCTCTCAAAAGCAGAGACCAACCATTTTCCCGCAGGCTGCTTTCCCTGTCAGCCTATTTCAGGATATGCTGATCTGGCAAAGGCTGATATCCTGCCCCAAGTGTCTGCCCCACGTGTCTGCCCCACTTGGACTTCCTTTTCAAAATTCTGCAAAAGCCCCCAGGTACCTGTTCTGGGAGATGAATAATGCCTTCTGGTATGTCTGCTCCCATCACCGAATCGGCCCCCTCCTTATTCAAACCAGCCTCCTGCCCAGAGTGTCTGAGCCCTGTCTAAAACTAAGTGTGGCTGGAAGAGATTTGGTGCAGACTTCACAAGCAGTAATTGGGGAAAACCCACAAGTGAGACAGAAGCTGCAGGGCTGGCTACCCACAGGGACCTGCCAAGTCTGATGCTAAGAAGCAGGAAAAAGAAATCTTACTTTAAAAAAATCAGCTAGAAGGGGGAAAACCATGAGTTTCTCATTATTTCTGATTTATAATTTTTGGAAGTCTCTAATATATTTTTGTAGATCTGCCTTTGCAGGGCAGGTTGGAAAGTTGCTGGTAAGACATTTCTTAACTGCCATAGTCCATTTTCAGGCATACGTTCATTTGAATGTAGATAGATTGGCCTTCTCTTCACTTAGCGAATAACCACTTCCTATGGAAGAGAGGAAGGGGAAGGGAATGAATGTTTATTAGATACCTATTTTATGATAGATAGGATACAGGTATTGACCATTTCCTTAGCTTTCCTAACAATCCTGAGAAAGAGGTATTATATTCTCTGTTTTACAGCTGAAGAAAGTGAGGCTCAGAAAGCTCTTCCAAGATCTCACAGCCAGCAGCAGCAGGGCAGAAATTCACAGCCAGGTCTGTTGGACTGCAAAGGCCATCCATGCGCTTTCCTTTATACCACAGTGCCTCTCAAGAGACGCTCCACATTTAAGCTGGTGCTTCCTTTCTCCTGTAATTAGTTTACCTTCTTTGCTAAGGCACATTCATTTATAAAGAAAAATAATGTTTCTCAACAAATCTATTTTTTCTTTTTTTTTAAATTAGAAAACAGGTGGATAAAAGTGGTGGGGAAGGATGCTCACTTGTATATACTTAGTATATATTCACATTTCCTACCTGCCTGCCTGCCTGCCTTCCCTCCTTCCATGTCTCCTTCCCTCCTACCTTCTTTCCTTCTCTCCTTCCCTTTCTTCCTTCCTTCCTTCCCCCCTCCCCTCCCCTCCCTCCTTTCCTTCCTTCCTTCCTTCCTTTCCTTCCTTCCTTCCTTCTCTCCTCCCCTCCTTCCTTCTCTCTTTCCCTCCTTCCCTTCAGCTACAGGTCTTTGATCTTAACTGCCTGTCTGAAAATCTTTTGTATCCAAAGACCTTCAAGTCTTCACCCAAGGTCACTTAGGCTCCAGGGCTCTTAAAATATAAATAATAATTTTATCTTTCTCATTCTTCCCTTTCTCTTTCTGTGCTCAACACGTTATCTGTGTTCTTTCTCCTTATTTTAAATTGAGGACGAAGTGTGCAATAGAGAGCAACAGAAACCACTGGCACCAAGCTCCATGTCAAACAAAGAATGAAGAACGAAGCATATTTACTTCATTGGAGAGAACACCCCTGGGCTGATGTCTTCTTGCTCCTCCACGTCCATCCTCCATCCTCTCCACTTGCTCTTGGCCCCAGGTGGCTGATCTATGGAGCATGAGCAAGCTCTGCTCCCCTGTCCCTCACTTTTGGTTGGCCTCAGCCATCTGCTTCTTGCTGGGACCCTGACTAATAAAACCTGCAAGGCGTGGGTATATGAGGGGGAAGACTTGTCATTCTATAGCTTAGATTTATTCTGTGTTTTTTCCTCCAAAGAGCAGAAGTAGGACCAATGGTGAGAAATTAGAGGGAGGAGAAATGATTTTTTTTATAGCAAAAAAAGACATTTCTAAGTATTAAAAGCTGTCTATACATAGATTGGGCTATCTTGGGAGAAAGTGGGCACCCTTCAAGGTGGTATTTAAGCCAAACTGGGATAAGCAAGGAAGTTAGAGAAGGATTTATATATTAGGAAAGAGAGAAGGCTAGATCTCTAAGGTTCTTTTTAACTTTGAGCATCTATCTGTGTTACTACAACTATCAAGAGTTCTCCCCCTCTCCTTTCACATTCTTTTAAATTTTATTTTACATTTTCTCCCTTCCCTGAGAATTTAATAAAGTTTAAATTAAACTTGCTTATCTAGGTTGGAGAGGTAGGATGGAGTAGAGCACAGATTAAAATAAGAAATTGGCAAATATTCAGTGTTTACAGTTTGAAAACAGCATGTTTGTGCACTCGCCCTTTTTATCAATATACAAACACCACGATTTGGTTAACATGTTTTGCACTTTTCCTGACCTTTTTCATCCTCTGCACTCTGGAGCAAATGACTCCACAAGGAATTTTTTTACACTCCAAAATATTTGTTTTATTCTTCCAGTTGGTACTTAGATTGGTGAGGGTTTTGTTTGTTTTATAAAAAGAGACAATTCGAATTTTACCAAAATAGCATTAAAGTCTGAAAACTACTCCTTAAAAGTGGATTGAAACCCACCACTCTGTCCTCTAAAATCAAAGGTTTCAGCATCCAGCTTGCACTGTTCATGTTAGCTGTGGCCTTAAGGGAGTACCGATAGGAAACTTCACAAATGCAGGATGGGCTTTCTTTTACTCATTTTCTCTAGGGGAAAATGGGTCAACTCTATCTATCTGCAAGCTGAATTTTTTGAAATTTACTTCTAGAAAGCAATGAAATTAGTGGCCCTCTTACCCCAAAGAGTGAACTTCCAGCCACCTGGGAGTGAGGATCCAGGGAGGGCTGGTCTAAGAGATCTGCTCTGTTTCTTGTGGTTCTGGGGAAGCTATTTTAGCCCACGGCAATGCTCATTCTAGATAACTCTAATTCTCAAGATTGATGAGAGCAATTTGGCATTTGGAGACATTAAGAACCCCAAACTTCTTCTGACTTCTAAATCAACTTGTTTTTTTAGAGTGTCCAGCCAGGAGATGATCTTCACGAGACCAGAGCAGAATCTCCAGGACACTAGTGCATGGCCTGGTCGCCTCTACCATCCTCAGACTGATGGTAGGAGGAGGAGGATAAGGTACAGGAGCGACATACCTTTCCTCTCTTATGTTTTTTTCACTTAGCTCCAGTGTTATTAAAATGACCTAAATAATGAGAACAGACAGGCTCATAAACTGGGTGTTGAATCAACAAAGTTCACTCTGGAAGTTCTGTGAACAATGTCAGAAACTGGTGATTAAAATCAAAGACCTCCCTGAGCTGCTAATGAACAAAGACATTTTTATTAAAACCAGATAGCTAGCCAGAGACTCACGCAGTCGAAACTCCTATAGCACAGGGGTCACTTCAGTCAACCCTACATGCCACTGCAAAACTGTCTTGTGTAGAATCAAACTAGATGACAGAGATGATTTTGATAAATGAAAACACACCAAATATGTCTTCATCTGCATGAAAGTGAACCCCTGCAATGCAGAGCCTGCACCCGGGAAGCAGTAATTCTTTGAGTTTTTGCTAAGGAGGGCTCTGTACAGAATAATGTGGCAACTAGAACAGGTTACCAAAGCACCAAGCTTGCAGTGGCACTTCACAGACGCTTGCTGAATGAACATAAACAAGTGAATGTGACACTTGAAAGGCAGCTCTACATCAGAAAGTTAAGAGTTGTGGGTTCTAGTTCCAAGCATGTCACAAAAGAACTGGGCAAGGAAATATTACCGTCCCTTTCTGGACTTCTAGGTCCTTATCTGTAAAACAAAGTGCCTGAGCTAAGTGGCTTCTGGTATCCCTCCAAACTCACAATGTTTTGATTCTCTGATGTGCATTAAGAGTTATTTTCCCATGACATTGTAGAATCAAAGATGACATCATTTCAATTTAATGAGAGAGTCAGTAGATCTCCCTTAATTATCAAAAGAATTTAGCAATCTTGAAAAACAAAACAAAACAAAACAAAACACACAAAAAAACACAACCTTGCTGTGTTTCCTGCTTGCGCTCTGTCTCAAAATGTGTAGAGCTGAACAGAACTACTTTGGCATCTTCAGGCCAAAAGCCACTCAAAATAGCCTTTGGCTGGCTGATATTTTTCAGTCATAAGTAAGATTTTTTGACCCCTGATATGAACATAACTCAAGTGTGGGCTGATGACTAGCAGTGCAGCGGGAGGGTGGAGGATATTGCGCTTCAAGAAATTCTGGGGGCATTTCATTTCAGATGCTCTCAAAGCACCCTCAATTATAATTTTTTTTCTTAGTTTGCTGTTTTGCAAAGACTGTTACTTACCTCGGGTTTCAACCAGGTTAGGGCAATTTTAAATGAGTATTTGTTGCTTCAATAACACATTTACAGCTTCAGAAAGCAAGCAGTGTTTTGCTTCATAGGACCTGATAGCCATGCATCTTGAGAGTGAGGAAAAGTTACTTGCCAACACACTGGTTCCACTGGTAGTGCTGGAGGTAGCCCTGGGGGCAGCTGCACCTGTAGCCCCCAATGATGTTCTGGCAGCCATGCTGGCAGCGGTGGTTACCCTCACACTCGTCCACGTCTGAAAAAGAAGCAGAGCCACCATGATGCCAACTCAACATCTCTCTCTGAAGCCAGATGGATTCTAATAAGAAATCTGGCCCCTACACATCCTACACATTATTCTACCACCAAAAACTTCAAGAATCTCTCTTCTGGAACTGCTGCTAAGTCCAAGGAGAAAATCATTCAGAGAGCCCTGGCAAAGTCTACAAATAAAAATTGTCAGGAAATTAGTGCAAAGCCCAAAGCACTCTCTGGGACAGCTCTGTAAGTTATAAGTTATTTGTGTTTGCTCTTTTTTTTTTTTATTTTTGGTGCCACAATTAGAAACCAAAACGTGTCAAAGCCACAGTCAACTTCTAAAAATATCATTTAAACTTCTAATTAAATGTCATCAGAAGAGCTATTCCTCATATGTTTAGTCCACATGGGGGAAATCCCTTTATGGCTCTGGACCCAGTGACCAGAAAATTAATGAATCAAAATCAAGTATATTTTTCTTGACTGAGTTATTCCCTATTAGGTTAAATTAAAAACTTAGCAGAAAAAGTCTTGATATACTTAGGAAAGGCATGTGTGTTAGACACAATCAAGCATGAAACAAAACACGACTCTCTAAGATACAGGACCTGTGCACACTATTTTAGCTGAGTGCCCCCCTGGGCTCAGATCTGCTATCTCATAGAGGCTGATGATGAAGGTGCCAATAGCCACACAGGCCACCTCCACAAGGATTCACCAGCTGGATCGCAGCTGAAGTCTCCACCCACCTTCACAGCTGGAGCCGGTCTGATCAAGTGAGAATCCCCGCTGGCATTCACAGGTGAAGCTTCCAGGAGTGTTCTGGCAAATGCCCTTAGACCCGCACAGATTGATGTCAGAGGTGCATTCATTGTTATCTATGAGAAGCAGTGGGGGCAAAGAGGGGTTAAAATTCCCCAAAGCTCTCTTTGTATCATTTAAAAGAAAATTAGAAGGATAACTCGGAAAAGGCCAAATAAGGCCAACAACGCTTCACATACATGTACAGGTGTGCTCACACATACATGCACTCATATATTTCTTTGAGCTTTTATTTTCTTATCCCAACAGCAGAGGAAATAGAAAATAATCCCTTAAAAGAATCGCTACAATCCATGTAGGATTTTTTCCTCTCCTACTCACCAATGCAGGACGTATGGTGTTGGGTAAATCCGGGAGGACATTTGCATGTGAAGCCGCCAATGGTGTTAACACATAGGAACTGGCAGTTGTGTTGCTTGGTTGCACACTCATCAAGATCT/-\", \"strand\": 1, \"transcript_consequences\": [{\"used_ref\": \"ATCTGCAGGCAAAATAAGAAGCGGCATGTGTGGCAGCAGCCAGAGATTTCTATTGAGGACATTGGATCTGGCCACTTGCTGCCGGCCAGCTGGCCCTCAGCTAGGCTCTCAGGAAGGTAAGACAGGCAGAGGTGGCTGGGTGGGGAGAGCAACAGCAGAGAAAGATGCGGGCATGGATGTGGCTTCTTGCTATGTTGTGAGTAAACATGGAAGATGAAACATGGAAGATGAAGTCAAATGCACCAGGCAGAGGGCATTAGAGGATGGCATTCTGAGGACCAGCAGACTCCAGTTCTCATCAGAAGTCAACACTGGTATCTTCTGCTCCATGTAAACATCCGGTCATGAACCAACATGGGGCTGGTCCGTAGCTAAGATTTAGGGTTCAAGGGGGGAGGAAAGGAATCTTGCAGTTCTCTGCGTACTTGCTTCACAAGTGCCCAGCGAGGAGACAAATCTCTCAAAAGCAGAGACCAACCATTTTCCCGCAGGCTGCTTTCCCTGTCAGCCTATTTCAGGATATGCTGATCTGGCAAAGGCTGATATCCTGCCCCAAGTGTCTGCCCCACGTGTCTGCCCCACTTGGACTTCCTTTTCAAAATTCTGCAAAAGCCCCCAGGTACCTGTTCTGGGAGATGAATAATGCCTTCTGGTATGTCTGCTCCCATCACCGAATCGGCCCCCTCCTTATTCAAACCAGCCTCCTGCCCAGAGTGTCTGAGCCCTGTCTAAAACTAAGTGTGGCTGGAAGAGATTTGGTGCAGACTTCACAAGCAGTAATTGGGGAAAACCCACAAGTGAGACAGAAGCTGCAGGGCTGGCTACCCACAGGGACCTGCCAAGTCTGATGCTAAGAAGCAGGAAAAAGAAATCTTACTTTAAAAAAATCAGCTAGAAGGGGGAAAACCATGAGTTTCTCATTATTTCTGATTTATAATTTTTGGAAGTCTCTAATATATTTTTGTAGATCTGCCTTTGCAGGGCAGGTTGGAAAGTTGCTGGTAAGACATTTCTTAACTGCCATAGTCCATTTTCAGGCATACGTTCATTTGAATGTAGATAGATTGGCCTTCTCTTCACTTAGCGAATAACCACTTCCTATGGAAGAGAGGAAGGGGAAGGGAATGAATGTTTATTAGATACCTATTTTATGATAGATAGGATACAGGTATTGACCATTTCCTTAGCTTTCCTAACAATCCTGAGAAAGAGGTATTATATTCTCTGTTTTACAGCTGAAGAAAGTGAGGCTCAGAAAGCTCTTCCAAGATCTCACAGCCAGCAGCAGCAGGGCAGAAATTCACAGCCAGGTCTGTTGGACTGCAAAGGCCATCCATGCGCTTTCCTTTATACCACAGTGCCTCTCAAGAGACGCTCCACATTTAAGCTGGTGCTTCCTTTCTCCTGTAATTAGTTTACCTTCTTTGCTAAGGCACATTCATTTATAAAGAAAAATAATGTTTCTCAACAAATCTATTTTTTCTTTTTTTTTAAATTAGAAAACAGGTGGATAAAAGTGGTGGGGAAGGATGCTCACTTGTATATACTTAGTATATATTCACATTTCCTACCTGCCTGCCTGCCTGCCTTCCCTCCTTCCATGTCTCCTTCCCTCCTACCTTCTTTCCTTCTCTCCTTCCCTTTCTTCCTTCCTTCCTTCCCCCCTCCCCTCCCCTCCCTCCTTTCCTTCCTTCCTTCCTTCCTTTCCTTCCTTCCTTCCTTCTCTCCTCCCCTCCTTCCTTCTCTCTTTCCCTCCTTCCCTTCAGCTACAGGTCTTTGATCTTAACTGCCTGTCTGAAAATCTTTTGTATCCAAAGACCTTCAAGTCTTCACCCAAGGTCACTTAGGCTCCAGGGCTCTTAAAATATAAATAATAATTTTATCTTTCTCATTCTTCCCTTTCTCTTTCTGTGCTCAACACGTTATCTGTGTTCTTTCTCCTTATTTTAAATTGAGGACGAAGTGTGCAATAGAGAGCAACAGAAACCACTGGCACCAAGCTCCATGTCAAACAAAGAATGAAGAACGAAGCATATTTACTTCATTGGAGAGAACACCCCTGGGCTGATGTCTTCTTGCTCCTCCACGTCCATCCTCCATCCTCTCCACTTGCTCTTGGCCCCAGGTGGCTGATCTATGGAGCATGAGCAAGCTCTGCTCCCCTGTCCCTCACTTTTGGTTGGCCTCAGCCATCTGCTTCTTGCTGGGACCCTGACTAATAAAACCTGCAAGGCGTGGGTATATGAGGGGGAAGACTTGTCATTCTATAGCTTAGATTTATTCTGTGTTTTTTCCTCCAAAGAGCAGAAGTAGGACCAATGGTGAGAAATTAGAGGGAGGAGAAATGATTTTTTTTATAGCAAAAAAAGACATTTCTAAGTATTAAAAGCTGTCTATACATAGATTGGGCTATCTTGGGAGAAAGTGGGCACCCTTCAAGGTGGTATTTAAGCCAAACTGGGATAAGCAAGGAAGTTAGAGAAGGATTTATATATTAGGAAAGAGAGAAGGCTAGATCTCTAAGGTTCTTTTTAACTTTGAGCATCTATCTGTGTTACTACAACTATCAAGAGTTCTCCCCCTCTCCTTTCACATTCTTTTAAATTTTATTTTACATTTTCTCCCTTCCCTGAGAATTTAATAAAGTTTAAATTAAACTTGCTTATCTAGGTTGGAGAGGTAGGATGGAGTAGAGCACAGATTAAAATAAGAAATTGGCAAATATTCAGTGTTTACAGTTTGAAAACAGCATGTTTGTGCACTCGCCCTTTTTATCAATATACAAACACCACGATTTGGTTAACATGTTTTGCACTTTTCCTGACCTTTTTCATCCTCTGCACTCTGGAGCAAATGACTCCACAAGGAATTTTTTTACACTCCAAAATATTTGTTTTATTCTTCCAGTTGGTACTTAGATTGGTGAGGGTTTTGTTTGTTTTATAAAAAGAGACAATTCGAATTTTACCAAAATAGCATTAAAGTCTGAAAACTACTCCTTAAAAGTGGATTGAAACCCACCACTCTGTCCTCTAAAATCAAAGGTTTCAGCATCCAGCTTGCACTGTTCATGTTAGCTGTGGCCTTAAGGGAGTACCGATAGGAAACTTCACAAATGCAGGATGGGCTTTCTTTTACTCATTTTCTCTAGGGGAAAATGGGTCAACTCTATCTATCTGCAAGCTGAATTTTTTGAAATTTACTTCTAGAAAGCAATGAAATTAGTGGCCCTCTTACCCCAAAGAGTGAACTTCCAGCCACCTGGGAGTGAGGATCCAGGGAGGGCTGGTCTAAGAGATCTGCTCTGTTTCTTGTGGTTCTGGGGAAGCTATTTTAGCCCACGGCAATGCTCATTCTAGATAACTCTAATTCTCAAGATTGATGAGAGCAATTTGGCATTTGGAGACATTAAGAACCCCAAACTTCTTCTGACTTCTAAATCAACTTGTTTTTTTAGAGTGTCCAGCCAGGAGATGATCTTCACGAGACCAGAGCAGAATCTCCAGGACACTAGTGCATGGCCTGGTCGCCTCTACCATCCTCAGACTGATGGTAGGAGGAGGAGGATAAGGTACAGGAGCGACATACCTTTCCTCTCTTATGTTTTTTTCACTTAGCTCCAGTGTTATTAAAATGACCTAAATAATGAGAACAGACAGGCTCATAAACTGGGTGTTGAATCAACAAAGTTCACTCTGGAAGTTCTGTGAACAATGTCAGAAACTGGTGATTAAAATCAAAGACCTCCCTGAGCTGCTAATGAACAAAGACATTTTTATTAAAACCAGATAGCTAGCCAGAGACTCACGCAGTCGAAACTCCTATAGCACAGGGGTCACTTCAGTCAACCCTACATGCCACTGCAAAACTGTCTTGTGTAGAATCAAACTAGATGACAGAGATGATTTTGATAAATGAAAACACACCAAATATGTCTTCATCTGCATGAAAGTGAACCCCTGCAATGCAGAGCCTGCACCCGGGAAGCAGTAATTCTTTGAGTTTTTGCTAAGGAGGGCTCTGTACAGAATAATGTGGCAACTAGAACAGGTTACCAAAGCACCAAGCTTGCAGTGGCACTTCACAGACGCTTGCTGAATGAACATAAACAAGTGAATGTGACACTTGAAAGGCAGCTCTACATCAGAAAGTTAAGAGTTGTGGGTTCTAGTTCCAAGCATGTCACAAAAGAACTGGGCAAGGAAATATTACCGTCCCTTTCTGGACTTCTAGGTCCTTATCTGTAAAACAAAGTGCCTGAGCTAAGTGGCTTCTGGTATCCCTCCAAACTCACAATGTTTTGATTCTCTGATGTGCATTAAGAGTTATTTTCCCATGACATTGTAGAATCAAAGATGACATCATTTCAATTTAATGAGAGAGTCAGTAGATCTCCCTTAATTATCAAAAGAATTTAGCAATCTTGAAAAACAAAACAAAACAAAACAAAACACACAAAAAAACACAACCTTGCTGTGTTTCCTGCTTGCGCTCTGTCTCAAAATGTGTAGAGCTGAACAGAACTACTTTGGCATCTTCAGGCCAAAAGCCACTCAAAATAGCCTTTGGCTGGCTGATATTTTTCAGTCATAAGTAAGATTTTTTGACCCCTGATATGAACATAACTCAAGTGTGGGCTGATGACTAGCAGTGCAGCGGGAGGGTGGAGGATATTGCGCTTCAAGAAATTCTGGGGGCATTTCATTTCAGATGCTCTCAAAGCACCCTCAATTATAATTTTTTTTCTTAGTTTGCTGTTTTGCAAAGACTGTTACTTACCTCGGGTTTCAACCAGGTTAGGGCAATTTTAAATGAGTATTTGTTGCTTCAATAACACATTTACAGCTTCAGAAAGCAAGCAGTGTTTTGCTTCATAGGACCTGATAGCCATGCATCTTGAGAGTGAGGAAAAGTTACTTGCCAACACACTGGTTCCACTGGTAGTGCTGGAGGTAGCCCTGGGGGCAGCTGCACCTGTAGCCCCCAATGATGTTCTGGCAGCCATGCTGGCAGCGGTGGTTACCCTCACACTCGTCCACGTCTGAAAAAGAAGCAGAGCCACCATGATGCCAACTCAACATCTCTCTCTGAAGCCAGATGGATTCTAATAAGAAATCTGGCCCCTACACATCCTACACATTATTCTACCACCAAAAACTTCAAGAATCTCTCTTCTGGAACTGCTGCTAAGTCCAAGGAGAAAATCATTCAGAGAGCCCTGGCAAAGTCTACAAATAAAAATTGTCAGGAAATTAGTGCAAAGCCCAAAGCACTCTCTGGGACAGCTCTGTAAGTTATAAGTTATTTGTGTTTGCTCTTTTTTTTTTTTATTTTTGGTGCCACAATTAGAAACCAAAACGTGTCAAAGCCACAGTCAACTTCTAAAAATATCATTTAAACTTCTAATTAAATGTCATCAGAAGAGCTATTCCTCATATGTTTAGTCCACATGGGGGAAATCCCTTTATGGCTCTGGACCCAGTGACCAGAAAATTAATGAATCAAAATCAAGTATATTTTTCTTGACTGAGTTATTCCCTATTAGGTTAAATTAAAAACTTAGCAGAAAAAGTCTTGATATACTTAGGAAAGGCATGTGTGTTAGACACAATCAAGCATGAAACAAAACACGACTCTCTAAGATACAGGACCTGTGCACACTATTTTAGCTGAGTGCCCCCCTGGGCTCAGATCTGCTATCTCATAGAGGCTGATGATGAAGGTGCCAATAGCCACACAGGCCACCTCCACAAGGATTCACCAGCTGGATCGCAGCTGAAGTCTCCACCCACCTTCACAGCTGGAGCCGGTCTGATCAAGTGAGAATCCCCGCTGGCATTCACAGGTGAAGCTTCCAGGAGTGTTCTGGCAAATGCCCTTAGACCCGCACAGATTGATGTCAGAGGTGCATTCATTGTTATCTATGAGAAGCAGTGGGGGCAAAGAGGGGTTAAAATTCCCCAAAGCTCTCTTTGTATCATTTAAAAGAAAATTAGAAGGATAACTCGGAAAAGGCCAAATAAGGCCAACAACGCTTCACATACATGTACAGGTGTGCTCACACATACATGCACTCATATATTTCTTTGAGCTTTTATTTTCTTATCCCAACAGCAGAGGAAATAGAAAATAATCCCTTAAAAGAATCGCTACAATCCATGTAGGATTTTTTCCTCTCCTACTCACCAATGCAGGACGTATGGTGTTGGGTAAATCCGGGAGGACATTTGCATGTGAAGCCGCCAATGGTGTTAACACATAGGAACTGGCAGTTGTGTTGCTTGGTTGCACACTCATCAAG\", \"cds_start\": 7456, \"hgvs_offset\": -4, \"cdna_start\": 7772, \"consequence_terms\": [\"splice_acceptor_variant\", \"splice_donor_variant\", \"inframe_deletion\", \"splice_donor_5th_base_variant\", \"intron_variant\"], \"aa\": \"GCAGGCAAAATAAGAAGCGGCATGTGTGGCAGCAGCCAGAGATTTCTATTGAGGACATTGGATCTGGCCACTTGCTGCCGGCCAGCTGGCCCTCAGCTAGGCTCTCAGGAAGGTAAGACAGGCAGAGGTGGCTGGGTGGGGAGAGCAACAGCAGAGAAAGATGCGGGCATGGATGTGGCTTCTTGCTATGTTGTGAGTAAAcaTGGAAGATGAAACATGGAAGATGAAGTCAAATGCACCAGGCAGAGGGCATTAGAGGATGGCATTCTGAGGACCAGCAGACTCCAGTTCTCATCAGAAGTCAACACTGGTATCTTCTGCTCCATGTAAACATCCGGTCATGAACCAACATGGTGCTGGTCCGTAGCTAAGATTTAGGGTTCAAGGGGGGAGGAAAGGAATCTTGCAGTTCTCTGCGTACTTGCTTCACAAGTGCCCAGCGAGGAGACAAATATCTCAAAAGCAGAGACCAACCATTTTCCCGCAGGCTGCTTTCCCTGTCCGCCTATTTCAGGATATGCTGATCTGGCAAAGGCTGATATCCTGCCCCAAGTGTCTGCCCCACGTGTCTGCCCCACTTGGACTTCCTTTTCAAAATTCTGCAAAAGCCCCCAGGTACCTGTTCTGGGAGATGAATAATGCCTTCTGGTATGTCTGCTCCCATCACCGAATCGGCCCCCTCCTTATTCAAACCAGCCTCCTGCCCAGAGTGTCTGAGCCCTGTCTAAAACTAAGTGTGGCTGGAAGAGATTTGGTGCAGACTTCACAAGCAGTAATTGGGGAAAACCCACAAGAGAGACAGAAGCTGCAGGGCTGGCTACCCACAGGGACCTGCCAAGTCTGATGCTAAGAAGCAGGAAAAAGAAATCTTACTTTAAAAAAATCAACTAGAAGGGGGAAAACCATGAGTTTCTCATTATTTCTGATTTATAATTTTTGGAAGTCTCTAATATATTTTTGTAGATCTGCCTTTGCAGGGCAGGTTGGAAAGTTGCTGGTAAGACATTTCTTAACTGCCATAGtCCATTTTCAGGCATACGTTCATTTGAATGTAGATAGATTGGCCTTCTCTTCACTTAGCGAATAACCACTTCCTATGGAAGAGAGGAAGGGGAAGGGAATGAATGTTTATTAGATACCTATTTTATGATAGATAGGATACAGGTATTGCCCATTTCCTTAGCTTTCCTAACAATCCTGAGAAAGAGGTATTATATTctCTGTTTTACAGCTGAAGAAAGTGAGGCTCAGAAAGCTCTTCCAAGATCTCACAGCCAGCAGCAGCGGGGCAGAAATTCACAGCCAGGTCTGTTGGACTGCAAAGGCCATCCATGTGCTTTCCTTTATACCACAGTGCCTCTCAAGAGACGCTCCACATTTAAGCTGGTGCTTCCTTTCTCCTGTAATTAGTTTACCTTCTTTGCTAAGGCAGATTCATTTATAAAGAAAAATAATGTTTCTCAAAAAATCTATTTTTTCTTTTTTTTTAAATTAGAAAACAGGTGGATAAAAGTGGTGGGGAAGGATGCTCACTTGTATATACTTAGTATATATTCACATTTCCTACCTGCCTGCCTGCCTGCCTTCCCTCCTTCCATGTCTCCTTCCCTCCTACCTTCTTTCCTTCTCTCCTTCCCTTTCTTCCTTCCTTCCTTCCCTCC-CCCCTCCCCTCCCC---TCCCTCCTTTCCTTCCTTcctNNNNNNccttcctTCCTTCTCTCCTCCCCTCCTTCCTTCTCTCTTTCCCTCCTTCCCTTCAGCTACAGGTCTTTGATCTTAACTGCCTGTCTGAAAATCTTTTGTATCCAAAGACCTTCAAGTCTTCAcCCAAGGTCACTTAGGCTCCAGGGCTCTTAAAATAGAAATAATAATTTTATCTTTCTCATTCTTCCCTTTCTCTTTCTGTGCTCAACACGTTATCTGTGTTCTTTCTCCTTATTTTAAATTGAGGACgAAgTGTGCAATAGAGAGCAACAGAAACCACTGGCACCAAGCTCCATGTCAAACAAAGAATGAAGAACGAAGCATATTTACTTCATTGGAGAGAACACCCCTGGGCTGATGTCTTCTTGCTCCTCCAcGTCCATCCTCCATCCTCTCCACTTGCTCTTGGCCCCAGGTGGCTGATCTATGGAGCATGAGCAAGCTCTGCTCCCCTGTCCCTCACTTTTGGTTGGCCTCAGCCATCTGCTTCTTGCTGGGACCCTGACTAATAAAACCTGCAAGGCGTGGGTATATGAGGGGGAAGACTTGTCATTCTATAGCTTAGATTTATTCTGTGTTTTTTCCTCCAAAGAGCAGAAGTAGGACCAATGGTGAGAAATTAGAGGGAGGAGAAATGATTTttTTTATACCAAAAAAAGACATTTCTAAGTATTAAAAGCTGTCTATACATAGATTGGGCTATCTTGGGAGAAAGTGGGcGCCCTTCAAGGTGGTATTTAAGCCAAACTGGGATAAGCAAGGAAGTTAGAGAAGGATTTATATATTAGGAAAGAGGGAAGGCTAGATCTCTAAGGTTCTTTTTAACTTTGAGCATCTATCTGTGTTACTACAACTATCAAGAGTTCTCCCCCTCTCCTTTCACATTCTTTTAAATTTTATTTTACATTTTCTCCCTTCCCTGAGAATTTAATAAAGTTtaaATTAAACTTGCTTATCTAGGTTGGAGAGGTAGGATGGAGTAGAGCACAGATTAAAATAAGAAATTGGCAAATATTCAGTGTTTACAGTTTGAAAACAGCATGTTTGTGCACTCGCCCTTTTTATCAATATACAAACACCACGATTTGGTTAACATGTTTTGCACTTTTCCTGACCTTTTTCATCCTCTGCACTCTGGAGCAAATGACTCCACAAGGAATTTTTTTACACTCCAAAATATTTGTTTTATTCTTCCAGTTGGTACTTAGATtGGTGAGGGTTTTGTTTGTTTTATAAaaAGAGACAATTCGAATTTTACCAAAATAGCATTAAAGTCTGAAAACTACTCCTTAAAAGTGGATTGAAACCCACCACTCTGTCCTCTAAAATCAAAGGTTTCAGCATCCAGCTTGCACTGTTCATGTTAGCTGTGGCCTTAAGGGAGTACCGATAGGAAACTTCACAAATGCAGGATGGGCTTTCTTTTACTCATTTTCTCTAGGGGAAAATGGGTCAACTCTATCTATCTGCAAGCTGAATT-TTTGAAATTTACTTCTAGAAAGCAATGAAATTAGTGGCCCTCTTACCCCAAAGAGTGAACTTCCAGCCACCTGGGAGTGAGGATCCAGGGAGGGCTGGTCTAAGAGATCTGCTCTGTTTCTTGTGGTTCTGGGGAAGCTATTTTAGCCCAAGGCAATGCTCATtCTAGATAACTCTAATTCTCAAGATTGATGAGAGCAATTTGGCATTTGGAGAcATTAAGAACCCCAAACTTCTTCTGACTTCTAAATCAACTTGTTTTTTTAGAGTGTCCAGCCAGGAGATGATCTTCACGAGACCAGAGCAGAATCTCCAGGACACTAGTGCATGGCCTGGTCGCCTCTACCATCCTCAGACTGATGGTAGGAGGAGGAGGATAAGGTACAGGAGCGACATACCTTTCCTCTCTTATGTTTTTTTCACTTAGCTCCAGTGTTATTAAAATGACCTAAATAATGAGAACAGACAGGCTCATAAACTGGGTGTTGAATCAACAAAGTTCACTCTGGAAGTTCTGTGAACAATGTCAGAAACTGGTGATTAAAATCAAAGACCTCCCTGAGCTGCTAATGAACAAAGACATTTTTATTAAAACCAGATAGCTAGCCAGAGACTCACGCAGTCGAAACTCCTATAGCACAGGGGTCACTTCAGTCAACCCTACATGCCACTGCAAAACTGTCTTGTGTAGAATCAAACTAGATGACAGAGATGATTTTGATAAATGAAAACACACCAAATATGTCTTCATCTGCATGAAAGTGAACCACTGCAATGCAGAGCCTGCACCCAGGAAGCAGTAATTCTTTGAGTTTTTGCTAAGGAGGGCTCTGTACAGAATAATGTGGCAACTAGAACAGGTTACCAAAGCACCAAGCTTGCAGTGGCACTCCACAGACGCTTGCTGAATGAACATAAACAAGTGAATGTGACACTTGAAAGGCAGCTCTACaTCAGAAAGTTAAGAGTTGTGGGTTCTAGTTCCAAGCATGTCACAAAAGAACTGGgCAAGGAAATATTACCGTCCCTTTCTGGACTTCTAGGTCCTTATCTGTAAAACAAAGTGCCTGAGCTAAGTGGCTTCTGGTATCCCTCCAAACTCACAATGTTTTGATTCTCTGATGTGCATTAAGAGTTATTTTCCCATGACATTGTAGAATCAAAGATGACATCATTTCAATTTAATGAGAGAGTCAGTAGATCTCCCTTAATTATCAAAAGAATTTAGCAATCTTGAAAAACAAAACAAAACAAAacaaAACACACAAAAAAACACAACCTTGCTGTGTTTCCTGCTtGCGCTCTGTCTCAAAATGTGTAGAGCTGAACAGAACTACTTTGGCATCTTCAGGCCAAAAGCCACTCAAAATAGCCTTTGGCTGGCTGATATTTTTCAGTCATAAGTAAGATTTTTTGACCCCTGATATGAACATAACTCAAGTGTGGGCTGATGACTAGCAGTGCAGCGGGAGGGTGGAGGATATTGCGCTTCAAGAAATTCTGGGGGCATTTCATTTCAGATGCTCTCAAAGCACCCTCAATTATAATTTTTTTTCTTAGTTTGCTGTTTTGCAAAGACTGTTACTTACCTCGGGTTTCAACCAGGTTAGGGCAATTTTAAATGAGTATTTGTTGCTTCAATAACACaTTTACAGCTTCAGAAAGcAAGCAGTGTTTTGCTTCATAGGAcCTGATAGCCATGCATCTTGAGAGTGAGGAAAAGTTACTTGCCAACACACTGGTTCCACTGGTAGTGCTGGAGGTAGCCCTGGGGGCAGCTGCACCTGTAGCCCCCAATGATGTTCTGGCAGCCATGCTGGCAGCGGTGGTTACCCTCACACTCGTCCACGTCTGAAAAAGAAGCAGAGCCAcCATGATGCCAACTCAACATCTCTCTCTGAAGCCAGATGGATTCTAATAAGAAATCTGGCCCCTACACATCCTACACATTATTCTACCACCAAAAACTTCAAGAATATCTCTTCTGGAACTGCTGCTAAGTCCAAGGAGAAAATCATTCAGAGAGCCCTGGCAAAGTCTACAAATAAAAATtgTCAGGAAATTAGTGCAAAGCCCAAAGCACTCTCTGGGACAGCTCTGTAAGTTATAAGTTATTTGTGTTTGCTCTTTTTTT-TTTTATTTTTGGTGCCACAATTAGAAACCAAAACGTGTCAAAGCCACAgTCAACTTCTAAAAATATCATTTAAACTTCTAATTAAATGTCATCAGAAGAGCTATTCCTCATATGTTTAGTCCACATGGGGGAAATCCCTTTATGGCTCTGGACCCAGTGACCAGAAAATTAATGAATCAAAATCAAGTATATTTTTCTTGACTGAGTTATTCCCTATTAGGTTAAATTAAAAACTTAGCAGAAAAAGTCTTGATATACTTAGGAAAGGCATGTGTGTTAGACACAATCAAGCATGAAACAAAACACGACTCTCTAAGATACAGGACCTGTGCACACTATTTTAGCTGAGTGCCCCCCTGGGCTCAGATCTGCTATCTCATAGAGGCTGATGATGAAGGTGCCAATAGCCACACAGGCCACCTCCACAAGGATTCACCAGCTGGATCGCAGCTGAaGTCTCCACCCACCTTCACAGCTGGAGCCGGTCTGATCAAGTGAGAATCCCCGCTGGCATTCACAGGTGAAGCTTCCAGGAGTGTTCTGGCAAATGCCCTTAGACCCGCACAGATTGATGTCAGAGGTGCATTCATTGTTATCTATGAGAAGCAGTGGGGGCAAAGAGGGGTTAAAATTCCCCAAAGCTCTCTTTGTATCATTTAAAAGAAAATTAGAAGGATAACTCGGAAAAGGCCAAATAAGGCCAACAACTCTTCACATACATGTACAGGTGTGCTCACACATACATGCACTCATATATTTCTTTGAGCTTTTATTTTCTTATCCCAACAGCAGAGGAAATAGAAAATAATCCCTTaAAAGaATCGCTACAATCCATGTAGGATTTTTTCCTCTCCTACTCACCAATGCAGGACGTATGGTGTTGGGTAAATCCGGGAGGACATTTGCATGTGAAGCCGCCAATGGTGTTAACACATAGGAACTGGCAGTTGTGTTGCTTGGTTGCACACTCATCAAGATCT\", \"hgvsp\": \"NP_000129.3:p.Leu2486_Asp2607del\", \"location\": \"15:48415766-48422066\", \"cdna_end\": 8137, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"allele_num\": 1, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"start\": 48415769, \"end\": 48422070, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"given_ref\": \"GCAGGCAAAATAAGAAGCGGCATGTGTGGCAGCAGCCAGAGATTTCTATTGAGGACATTGGATCTGGCCACTTGCTGCCGGCCAGCTGGCCCTCAGCTAGGCTCTCAGGAAGGTAAGACAGGCAGAGGTGGCTGGGTGGGGAGAGCAACAGCAGAGAAAGATGCGGGCATGGATGTGGCTTCTTGCTATGTTGTGAGTAAACATGGAAGATGAAACATGGAAGATGAAGTCAAATGCACCAGGCAGAGGGCATTAGAGGATGGCATTCTGAGGACCAGCAGACTCCAGTTCTCATCAGAAGTCAACACTGGTATCTTCTGCTCCATGTAAACATCCGGTCATGAACCAACATGGGGCTGGTCCGTAGCTAAGATTTAGGGTTCAAGGGGGGAGGAAAGGAATCTTGCAGTTCTCTGCGTACTTGCTTCACAAGTGCCCAGCGAGGAGACAAATCTCTCAAAAGCAGAGACCAACCATTTTCCCGCAGGCTGCTTTCCCTGTCAGCCTATTTCAGGATATGCTGATCTGGCAAAGGCTGATATCCTGCCCCAAGTGTCTGCCCCACGTGTCTGCCCCACTTGGACTTCCTTTTCAAAATTCTGCAAAAGCCCCCAGGTACCTGTTCTGGGAGATGAATAATGCCTTCTGGTATGTCTGCTCCCATCACCGAATCGGCCCCCTCCTTATTCAAACCAGCCTCCTGCCCAGAGTGTCTGAGCCCTGTCTAAAACTAAGTGTGGCTGGAAGAGATTTGGTGCAGACTTCACAAGCAGTAATTGGGGAAAACCCACAAGTGAGACAGAAGCTGCAGGGCTGGCTACCCACAGGGACCTGCCAAGTCTGATGCTAAGAAGCAGGAAAAAGAAATCTTACTTTAAAAAAATCAGCTAGAAGGGGGAAAACCATGAGTTTCTCATTATTTCTGATTTATAATTTTTGGAAGTCTCTAATATATTTTTGTAGATCTGCCTTTGCAGGGCAGGTTGGAAAGTTGCTGGTAAGACATTTCTTAACTGCCATAGTCCATTTTCAGGCATACGTTCATTTGAATGTAGATAGATTGGCCTTCTCTTCACTTAGCGAATAACCACTTCCTATGGAAGAGAGGAAGGGGAAGGGAATGAATGTTTATTAGATACCTATTTTATGATAGATAGGATACAGGTATTGACCATTTCCTTAGCTTTCCTAACAATCCTGAGAAAGAGGTATTATATTCTCTGTTTTACAGCTGAAGAAAGTGAGGCTCAGAAAGCTCTTCCAAGATCTCACAGCCAGCAGCAGCAGGGCAGAAATTCACAGCCAGGTCTGTTGGACTGCAAAGGCCATCCATGCGCTTTCCTTTATACCACAGTGCCTCTCAAGAGACGCTCCACATTTAAGCTGGTGCTTCCTTTCTCCTGTAATTAGTTTACCTTCTTTGCTAAGGCACATTCATTTATAAAGAAAAATAATGTTTCTCAACAAATCTATTTTTTCTTTTTTTTTAAATTAGAAAACAGGTGGATAAAAGTGGTGGGGAAGGATGCTCACTTGTATATACTTAGTATATATTCACATTTCCTACCTGCCTGCCTGCCTGCCTTCCCTCCTTCCATGTCTCCTTCCCTCCTACCTTCTTTCCTTCTCTCCTTCCCTTTCTTCCTTCCTTCCTTCCCCCCTCCCCTCCCCTCCCTCCTTTCCTTCCTTCCTTCCTTCCTTTCCTTCCTTCCTTCCTTCTCTCCTCCCCTCCTTCCTTCTCTCTTTCCCTCCTTCCCTTCAGCTACAGGTCTTTGATCTTAACTGCCTGTCTGAAAATCTTTTGTATCCAAAGACCTTCAAGTCTTCACCCAAGGTCACTTAGGCTCCAGGGCTCTTAAAATATAAATAATAATTTTATCTTTCTCATTCTTCCCTTTCTCTTTCTGTGCTCAACACGTTATCTGTGTTCTTTCTCCTTATTTTAAATTGAGGACGAAGTGTGCAATAGAGAGCAACAGAAACCACTGGCACCAAGCTCCATGTCAAACAAAGAATGAAGAACGAAGCATATTTACTTCATTGGAGAGAACACCCCTGGGCTGATGTCTTCTTGCTCCTCCACGTCCATCCTCCATCCTCTCCACTTGCTCTTGGCCCCAGGTGGCTGATCTATGGAGCATGAGCAAGCTCTGCTCCCCTGTCCCTCACTTTTGGTTGGCCTCAGCCATCTGCTTCTTGCTGGGACCCTGACTAATAAAACCTGCAAGGCGTGGGTATATGAGGGGGAAGACTTGTCATTCTATAGCTTAGATTTATTCTGTGTTTTTTCCTCCAAAGAGCAGAAGTAGGACCAATGGTGAGAAATTAGAGGGAGGAGAAATGATTTTTTTTATAGCAAAAAAAGACATTTCTAAGTATTAAAAGCTGTCTATACATAGATTGGGCTATCTTGGGAGAAAGTGGGCACCCTTCAAGGTGGTATTTAAGCCAAACTGGGATAAGCAAGGAAGTTAGAGAAGGATTTATATATTAGGAAAGAGAGAAGGCTAGATCTCTAAGGTTCTTTTTAACTTTGAGCATCTATCTGTGTTACTACAACTATCAAGAGTTCTCCCCCTCTCCTTTCACATTCTTTTAAATTTTATTTTACATTTTCTCCCTTCCCTGAGAATTTAATAAAGTTTAAATTAAACTTGCTTATCTAGGTTGGAGAGGTAGGATGGAGTAGAGCACAGATTAAAATAAGAAATTGGCAAATATTCAGTGTTTACAGTTTGAAAACAGCATGTTTGTGCACTCGCCCTTTTTATCAATATACAAACACCACGATTTGGTTAACATGTTTTGCACTTTTCCTGACCTTTTTCATCCTCTGCACTCTGGAGCAAATGACTCCACAAGGAATTTTTTTACACTCCAAAATATTTGTTTTATTCTTCCAGTTGGTACTTAGATTGGTGAGGGTTTTGTTTGTTTTATAAAAAGAGACAATTCGAATTTTACCAAAATAGCATTAAAGTCTGAAAACTACTCCTTAAAAGTGGATTGAAACCCACCACTCTGTCCTCTAAAATCAAAGGTTTCAGCATCCAGCTTGCACTGTTCATGTTAGCTGTGGCCTTAAGGGAGTACCGATAGGAAACTTCACAAATGCAGGATGGGCTTTCTTTTACTCATTTTCTCTAGGGGAAAATGGGTCAACTCTATCTATCTGCAAGCTGAATTTTTTGAAATTTACTTCTAGAAAGCAATGAAATTAGTGGCCCTCTTACCCCAAAGAGTGAACTTCCAGCCACCTGGGAGTGAGGATCCAGGGAGGGCTGGTCTAAGAGATCTGCTCTGTTTCTTGTGGTTCTGGGGAAGCTATTTTAGCCCACGGCAATGCTCATTCTAGATAACTCTAATTCTCAAGATTGATGAGAGCAATTTGGCATTTGGAGACATTAAGAACCCCAAACTTCTTCTGACTTCTAAATCAACTTGTTTTTTTAGAGTGTCCAGCCAGGAGATGATCTTCACGAGACCAGAGCAGAATCTCCAGGACACTAGTGCATGGCCTGGTCGCCTCTACCATCCTCAGACTGATGGTAGGAGGAGGAGGATAAGGTACAGGAGCGACATACCTTTCCTCTCTTATGTTTTTTTCACTTAGCTCCAGTGTTATTAAAATGACCTAAATAATGAGAACAGACAGGCTCATAAACTGGGTGTTGAATCAACAAAGTTCACTCTGGAAGTTCTGTGAACAATGTCAGAAACTGGTGATTAAAATCAAAGACCTCCCTGAGCTGCTAATGAACAAAGACATTTTTATTAAAACCAGATAGCTAGCCAGAGACTCACGCAGTCGAAACTCCTATAGCACAGGGGTCACTTCAGTCAACCCTACATGCCACTGCAAAACTGTCTTGTGTAGAATCAAACTAGATGACAGAGATGATTTTGATAAATGAAAACACACCAAATATGTCTTCATCTGCATGAAAGTGAACCCCTGCAATGCAGAGCCTGCACCCGGGAAGCAGTAATTCTTTGAGTTTTTGCTAAGGAGGGCTCTGTACAGAATAATGTGGCAACTAGAACAGGTTACCAAAGCACCAAGCTTGCAGTGGCACTTCACAGACGCTTGCTGAATGAACATAAACAAGTGAATGTGACACTTGAAAGGCAGCTCTACATCAGAAAGTTAAGAGTTGTGGGTTCTAGTTCCAAGCATGTCACAAAAGAACTGGGCAAGGAAATATTACCGTCCCTTTCTGGACTTCTAGGTCCTTATCTGTAAAACAAAGTGCCTGAGCTAAGTGGCTTCTGGTATCCCTCCAAACTCACAATGTTTTGATTCTCTGATGTGCATTAAGAGTTATTTTCCCATGACATTGTAGAATCAAAGATGACATCATTTCAATTTAATGAGAGAGTCAGTAGATCTCCCTTAATTATCAAAAGAATTTAGCAATCTTGAAAAACAAAACAAAACAAAACAAAACACACAAAAAAACACAACCTTGCTGTGTTTCCTGCTTGCGCTCTGTCTCAAAATGTGTAGAGCTGAACAGAACTACTTTGGCATCTTCAGGCCAAAAGCCACTCAAAATAGCCTTTGGCTGGCTGATATTTTTCAGTCATAAGTAAGATTTTTTGACCCCTGATATGAACATAACTCAAGTGTGGGCTGATGACTAGCAGTGCAGCGGGAGGGTGGAGGATATTGCGCTTCAAGAAATTCTGGGGGCATTTCATTTCAGATGCTCTCAAAGCACCCTCAATTATAATTTTTTTTCTTAGTTTGCTGTTTTGCAAAGACTGTTACTTACCTCGGGTTTCAACCAGGTTAGGGCAATTTTAAATGAGTATTTGTTGCTTCAATAACACATTTACAGCTTCAGAAAGCAAGCAGTGTTTTGCTTCATAGGACCTGATAGCCATGCATCTTGAGAGTGAGGAAAAGTTACTTGCCAACACACTGGTTCCACTGGTAGTGCTGGAGGTAGCCCTGGGGGCAGCTGCACCTGTAGCCCCCAATGATGTTCTGGCAGCCATGCTGGCAGCGGTGGTTACCCTCACACTCGTCCACGTCTGAAAAAGAAGCAGAGCCACCATGATGCCAACTCAACATCTCTCTCTGAAGCCAGATGGATTCTAATAAGAAATCTGGCCCCTACACATCCTACACATTATTCTACCACCAAAAACTTCAAGAATCTCTCTTCTGGAACTGCTGCTAAGTCCAAGGAGAAAATCATTCAGAGAGCCCTGGCAAAGTCTACAAATAAAAATTGTCAGGAAATTAGTGCAAAGCCCAAAGCACTCTCTGGGACAGCTCTGTAAGTTATAAGTTATTTGTGTTTGCTCTTTTTTTTTTTTATTTTTGGTGCCACAATTAGAAACCAAAACGTGTCAAAGCCACAGTCAACTTCTAAAAATATCATTTAAACTTCTAATTAAATGTCATCAGAAGAGCTATTCCTCATATGTTTAGTCCACATGGGGGAAATCCCTTTATGGCTCTGGACCCAGTGACCAGAAAATTAATGAATCAAAATCAAGTATATTTTTCTTGACTGAGTTATTCCCTATTAGGTTAAATTAAAAACTTAGCAGAAAAAGTCTTGATATACTTAGGAAAGGCATGTGTGTTAGACACAATCAAGCATGAAACAAAACACGACTCTCTAAGATACAGGACCTGTGCACACTATTTTAGCTGAGTGCCCCCCTGGGCTCAGATCTGCTATCTCATAGAGGCTGATGATGAAGGTGCCAATAGCCACACAGGCCACCTCCACAAGGATTCACCAGCTGGATCGCAGCTGAAGTCTCCACCCACCTTCACAGCTGGAGCCGGTCTGATCAAGTGAGAATCCCCGCTGGCATTCACAGGTGAAGCTTCCAGGAGTGTTCTGGCAAATGCCCTTAGACCCGCACAGATTGATGTCAGAGGTGCATTCATTGTTATCTATGAGAAGCAGTGGGGGCAAAGAGGGGTTAAAATTCCCCAAAGCTCTCTTTGTATCATTTAAAAGAAAATTAGAAGGATAACTCGGAAAAGGCCAAATAAGGCCAACAACGCTTCACATACATGTACAGGTGTGCTCACACATACATGCACTCATATATTTCTTTGAGCTTTTATTTTCTTATCCCAACAGCAGAGGAAATAGAAAATAATCCCTTAAAAGAATCGCTACAATCCATGTAGGATTTTTTCCTCTCCTACTCACCAATGCAGGACGTATGGTGTTGGGTAAATCCGGGAGGACATTTGCATGTGAAGCCGCCAATGGTGTTAACACATAGGAACTGGCAGTTGTGTTGCTTGGTTGCACACTCATCAAGATCT\", \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"genesplicer\": \"loss/acceptor/48415878-48415877/Medium/5.531441,loss/donor/48415885-48415886/Medium/6.229576,loss/acceptor/48416148-48416149/Medium/9.828930,loss/acceptor/48416272-48416271/Medium/2.266671,loss/donor/48416279-48416280/Medium/6.891307,loss/acceptor/48416527-48416526/Medium/1.388597,loss/acceptor/48417028-48417029/Medium/10.303534,loss/donor/48417097-48417098/Medium/3.634750,loss/acceptor/48417195-48417194/Medium/2.638052,loss/donor/48417292-48417293/Medium/3.000407,loss/acceptor/48417647-48417648/Medium/6.573641,loss/acceptor/48417839-48417838/Medium/4.901527,loss/acceptor/48417975-48417976/Medium/2.309903,loss/acceptor/48418181-48418182/Medium/3.754612,loss/acceptor/48418455-48418454/Medium/7.321602,loss/donor/48418518-48418517/Medium/4.296170,loss/acceptor/48418615-48418614/Medium/7.499184,loss/donor/48418834-48418835/Medium/1.483408,loss/acceptor/48418921-48418920/Medium/2.559974,loss/acceptor/48419403-48419404/Medium/1.081218,loss/acceptor/48419612-48419613/Medium/2.404654,loss/acceptor/48419770-48419769/Medium/6.647516,loss/acceptor/48419947-48419946/Medium/9.172028,loss/donor/48420031-48420032/Medium/0.711372,loss/acceptor/48420306-48420305/Medium/12.830047,loss/donor/48420452-48420453/Medium/3.098467,loss/acceptor/48420573-48420572/Medium/3.701447,loss/acceptor/48420714-48420713/Medium/3.226489,loss/acceptor/48420837-48420836/Medium/6.361984,loss/acceptor/48420971-48420972/Medium/10.421684,loss/acceptor/48421219-48421220/Medium/4.579688,loss/acceptor/48421372-48421371/Medium/2.045553,loss/donor/48421448-48421449/Medium/6.802227,loss/donor/48421756-48421755/Medium/0.613517,loss/acceptor/48421934-48421935/Medium/5.283292,loss/donor/48421954-48421953/Medium/7.001884,loss/acceptor/48422067-48422068/Medium/10.899395,gain/acceptor/48415770-48415771/Medium/13.000279\", \"variant_allele\": \"-\", \"loeuf\": 0.105, \"amino_acids\": \"LDECATKQHNCQFLCVNTIGGFTCKCPPGFTQHHTSCIDNNECTSDINLCGSKGICQNTPGSFTCECQRGFSLDQTGSSCEDVDECEGNHRCQHGCQNIIGGYRCSCPQGYLQHYQWNQCVD/-\", \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"protein_start\": 2486, \"cds_end\": 7821, \"codons\": \"CTTGATGAGTGTGCAACCAAGCAACACAACTGCCAGTTCCTATGTGTTAACACCATTGGCGGCTTCACATGCAAATGTCCTCCCGGATTTACCCAACACCATACGTCCTGCATTGATAACAATGAATGCACCTCTGACATCAATCTGTGCGGGTCTAAGGGCATTTGCCAGAACACTCCTGGAAGCTTCACCTGTGAATGCCAGCGGGGATTCTCACTTGATCAGACCGGCTCCAGCTGTGAAGACGTGGACGAGTGTGAGGGTAACCACCGCTGCCAGCATGGCTGCCAGAACATCATTGGGGGCTACAGGTGCAGCTGCCCCCAGGGCTACCTCCAGCACTACCAGTGGAACCAGTGTGTTGAT/-\", \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"exon\": \"61-63/66\", \"conservation\": -0.57, \"impact\": \"HIGH\", \"intron\": \"60-63/65\", \"hgvsc\": \"NM_000138.5:c.7456_7821del\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"lof_info\": \"INTRON_SIZE:3300\", \"protein_end\": 2607}], \"start\": 48415770, \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': -0.19242028783033255,\n",
       "      'pca_y': -0.03919398737456886},\n",
       "     {'t-sne_x': -38.882404456042956, 't-sne_y': -33.34639278761933},\n",
       "     {'umap_x': 17.517858505249023, 'umap_y': 14.450155258178711}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48437393-G-A',\n",
       "      'distance': 0.08528511226177216,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48411383-G-A',\n",
       "      'distance': 0.08579934388399124,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48487205-C-T',\n",
       "      'distance': 0.08591736108064651,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48427780-G-A',\n",
       "      'distance': 0.08617277443408966,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48596386-A-G',\n",
       "      'distance': 0.0861923098564148,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48470763-G-A',\n",
       "      'distance': 0.08678159862756729,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48495267-A-G',\n",
       "      'distance': 0.0873333290219307,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48446828-A-T',\n",
       "      'distance': 0.08772784471511841,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48508577-C-T',\n",
       "      'distance': 0.08863256871700287,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520825-C-T',\n",
       "      'distance': 0.08888830244541168,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48460241-C-T',\n",
       "      'distance': 0.08892408013343811,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492593-A-G',\n",
       "      'distance': 0.08917123079299927,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48503943-A-G',\n",
       "      'distance': 0.08954540640115738,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48600236-T-G',\n",
       "      'distance': 0.08959618955850601,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465861-A-G',\n",
       "      'distance': 0.0899098664522171,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48446701-C-A',\n",
       "      'distance': 0.0900353342294693,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474658-A-G',\n",
       "      'distance': 0.09044831991195679,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48515385-A-G',\n",
       "      'distance': 0.09078855812549591,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472544-C-A',\n",
       "      'distance': 0.09111566841602325,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48488243-T-G',\n",
       "      'distance': 0.09125198423862457,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_benign',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_benign', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_benign', 'confidence_score': 0.8},\n",
       "     '20': {'pred_result': 'likely_benign', 'confidence_score': 0.7}}},\n",
       "   {'variant_id': '15-48422057-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48422057\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48422057',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7465T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48422057A>G',\n",
       "     'hgvs_protein': 'p.Cys2489Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"id\": \"CM020373\", \"end\": 48422057, \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48422057}, {\"strand\": 1, \"end\": 48422057, \"id\": \"CM970521\", \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"start\": 48422057, \"seq_region_name\": \"15\"}, {\"seq_region_name\": \"15\", \"pubmed\": [17701892], \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"T:likely_pathogenic;C:likely_pathogenic;G:pathogenic\", \"start\": 48422057, \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"id\": \"rs1057520728\", \"end\": 48422057, \"allele_string\": \"A/C/G/T\", \"var_synonyms\": {\"UniProt\": [\"VAR_002343\"], \"ClinVar\": [\"RCV000440702\", \"VCV000379778\", \"RCV000663956\", \"VCV000549412\", \"RCV001219717\", \"VCV000948459\"]}, \"strand\": 1}], \"start\": 48422057, \"transcript_consequences\": [{\"loeuf\": 0.105, \"variant_allele\": \"G\", \"polyphen_score\": 0.994, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"C/R\", \"sift_prediction\": \"deleterious_low_confidence\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"lrt_pred\": \"invalid_field\", \"codons\": \"Tgt/Cgt\", \"protein_start\": 2489, \"cds_end\": 7465, \"cadd_phred\": 31, \"alphamissense\": {\"am_pathogenicity\": 0.9996, \"am_class\": \"likely_pathogenic\"}, \"sift_score\": 0, \"exon\": \"61/66\", \"conservation\": 3.77, \"impact\": \"MODERATE\", \"polyphen_prediction\": \"probably_damaging\", \"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.7465T>C\", \"mastermind_mmid3\": \"FBN1:C2489R\", \"mutationtaster_pred\": \"A\", \"gene_id\": \"2200\", \"blosum62\": -3, \"protein_end\": 2489, \"gene_symbol\": \"FBN1\", \"used_ref\": \"A\", \"eve_score\": 0.736377709618199, \"cds_start\": 7465, \"cdna_start\": 7781, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.999889254570007, \"cdna_end\": 7781, \"aa\": \"A\", \"location\": \"15:48422057\", \"hgvsp\": \"NP_000129.3:p.Cys2489Arg\", \"allele_num\": 1, \"mastermind_counts\": \"14|18|18\", \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_raw\": 5.430223, \"spliceai\": {\"DP_AL\": 0, \"DS_DL\": 0, \"DP_DG\": 4, \"DS_AG\": 0, \"DP_DL\": 2, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0, \"DP_AG\": 11}, \"given_ref\": \"A\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"end\": 48422057, \"type\": \"SimpleInterval\", \"start\": 48422056}}}, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"]}], \"assembly_name\": \"GRCh38\", \"input\": \"NC_000015.10:g.48422057A>G\", \"end\": 48422057, \"id\": \"NC_000015.10:g.48422057A>G\", \"allele_string\": \"A/G\", \"strand\": 1, \"minimised\": 1, \"vcf_string\": \"15-48422057-A-G\", \"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\"}',\n",
       "    'coordinates': [{'pca_x': 0.04380161838631447,\n",
       "      'pca_y': 0.08017243448149422},\n",
       "     {'t-sne_x': 28.900008704025325, 't-sne_y': 5.190628644710475},\n",
       "     {'umap_x': 2.5408213138580322, 'umap_y': 7.045839309692383}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.0006506353383883834,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.000760562950745225,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.0008869782323017716,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432944-A-G',\n",
       "      'distance': 0.0009051563101820648,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.0013419894967228174,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481733-A-G',\n",
       "      'distance': 0.0013788716169074178,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434600-A-G',\n",
       "      'distance': 0.0013898512115702033,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.001557481475174427,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415723-A-G',\n",
       "      'distance': 0.0015733197797089815,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434702-A-G',\n",
       "      'distance': 0.0016386593924835324,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432911-A-G',\n",
       "      'distance': 0.001679136767052114,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472665-A-G',\n",
       "      'distance': 0.00185263785533607,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420795-A-G',\n",
       "      'distance': 0.0018568229861557484,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.001930772326886654,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.002173353685066104,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596325-A-G',\n",
       "      'distance': 0.002294630976393819,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48503850-A-G',\n",
       "      'distance': 0.00231110118329525,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474567-A-G',\n",
       "      'distance': 0.0023449091240763664,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48515396-C-A',\n",
       "      'distance': 0.0023575597442686558,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470679-A-G',\n",
       "      'distance': 0.0023726574145257473,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48421957-C-G',\n",
       "    'metadata': {'vcf_string': '15\\t48421957\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48421957',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7565G>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48421957C>G',\n",
       "     'hgvs_protein': 'p.Cys2522Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\", \"variant_class\": \"SNV\", \"vcf_string\": \"15-48421957-C-G\", \"minimised\": 1, \"allele_string\": \"C/G\", \"strand\": 1, \"end\": 48421957, \"input\": \"NC_000015.10:g.48421957C>G\", \"id\": \"NC_000015.10:g.48421957C>G\", \"transcript_consequences\": [{\"variant_allele\": \"G\", \"loeuf\": 0.105, \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.968, \"amino_acids\": \"C/S\", \"canonical\": 1, \"sift_prediction\": \"deleterious_low_confidence\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"lrt_pred\": \"invalid_field\", \"cds_end\": 7565, \"protein_start\": 2522, \"codons\": \"tGc/tCc\", \"cadd_phred\": 27, \"sift_score\": 0, \"alphamissense\": {\"am_pathogenicity\": 0.999, \"am_class\": \"likely_pathogenic\"}, \"conservation\": 3.95, \"exon\": \"61/66\", \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"eve_class\": \"Uncertain\", \"hgvsc\": \"NM_000138.5:c.7565G>C\", \"mastermind_mmid3\": \"FBN1:C2522S\", \"mutationtaster_pred\": \"D\", \"gene_id\": \"2200\", \"blosum62\": -1, \"protein_end\": 2522, \"gene_symbol\": \"FBN1\", \"used_ref\": \"C\", \"eve_score\": 0.636002067978648, \"cds_start\": 7565, \"cdna_start\": 7881, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.994962930679321, \"cdna_end\": 7881, \"aa\": \"C\", \"location\": \"15:48421957\", \"hgvsp\": \"NP_000129.3:p.Cys2522Ser\", \"mastermind_counts\": \"3|3|3\", \"allele_num\": 1, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cadd_raw\": 4.839717, \"spliceai\": {\"DP_DL\": -5, \"DS_AG\": 0, \"DS_DL\": 0, \"DP_AL\": 5, \"DP_DG\": -9, \"DP_AG\": -14, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_DG\": 0}, \"given_ref\": \"C\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"start\": 48421956, \"type\": \"SimpleInterval\", \"end\": 48421957}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"type\": \"Allele\"}, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"]}], \"start\": 48421957, \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48421957, \"id\": \"CM000514\", \"phenotype_or_disease\": 1, \"start\": 48421957, \"seq_region_name\": \"15\"}, {\"end\": 48421957, \"id\": \"CM093617\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48421957, \"phenotype_or_disease\": 1}]}',\n",
       "    'coordinates': [{'pca_x': 0.040156538011522774,\n",
       "      'pca_y': 0.0904779850945909},\n",
       "     {'t-sne_x': 17.503815085653763, 't-sne_y': -5.073302593237229},\n",
       "     {'umap_x': 8.597886085510254, 'umap_y': 7.071327209472656}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48415722-C-G',\n",
       "      'distance': 0.0010198845993727446,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432910-C-G',\n",
       "      'distance': 0.0016417470760643482,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489988-C-G',\n",
       "      'distance': 0.0018390221521258354,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489991-C-G',\n",
       "      'distance': 0.001844950718805194,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474299-C-G',\n",
       "      'distance': 0.0018518619472160935,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474599-C-G',\n",
       "      'distance': 0.0019202050752937794,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48494209-C-G',\n",
       "      'distance': 0.002666436368599534,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434599-C-G',\n",
       "      'distance': 0.002738820854574442,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487362-C-G',\n",
       "      'distance': 0.00278969737701118,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48483862-C-G',\n",
       "      'distance': 0.002795260166749358,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48428390-C-G',\n",
       "      'distance': 0.0029529447201639414,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437064-G-C',\n",
       "      'distance': 0.003047143341973424,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470711-C-G',\n",
       "      'distance': 0.0030499096028506756,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421652-G-C',\n",
       "      'distance': 0.0031749499030411243,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.003236908931285143,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.003487425623461604,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.00356295146048069,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420736-G-C',\n",
       "      'distance': 0.003754677716642618,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425816-C-T',\n",
       "      'distance': 0.003758404403924942,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.003861419390887022,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48421600-G-A',\n",
       "    'metadata': {'vcf_string': '15\\t48421600\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48421600',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7657C>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48421600G>A',\n",
       "     'hgvs_protein': 'p.Gln2553Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48421600, \"id\": \"CM127886\", \"phenotype_or_disease\": 1, \"start\": 48421600, \"seq_region_name\": \"15\"}], \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"gene_symbol_source\": \"EntrezGene\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"interval\": {\"start\": 48421599, \"type\": \"SimpleInterval\", \"end\": 48421600}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}}, \"given_ref\": \"G\", \"spliceai\": {\"DP_DL\": -42, \"DS_AG\": 0, \"DP_AL\": 42, \"DS_DL\": 0.04, \"DP_DG\": 9, \"DP_AG\": -24, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0}, \"cadd_raw\": 10.742064, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"mastermind_counts\": \"3|3|3\", \"allele_num\": 1, \"location\": \"15:48421600\", \"hgvsp\": \"NP_000129.3:p.Gln2553Ter\", \"aa\": \"G\", \"cdna_end\": 7973, \"consequence_terms\": [\"stop_gained\"], \"cdna_start\": 7973, \"cds_start\": 7657, \"used_ref\": \"G\", \"lof_info\": \"PERCENTILE:0.888695450324977,GERP_DIST:1049.23628667146,BP_DIST:955,DIST_FROM_LAST_EXON:566,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"protein_end\": 2553, \"gene_id\": \"2200\", \"mutationtaster_pred\": \"A\", \"mastermind_mmid3\": \"FBN1:Q2553X\", \"hgvsc\": \"NM_000138.5:c.7657C>T\", \"impact\": \"HIGH\", \"conservation\": 3.76, \"exon\": \"62/66\", \"cadd_phred\": 49, \"protein_start\": 2553, \"cds_end\": 7657, \"codons\": \"Cag/Tag\", \"lrt_pred\": \"invalid_field\", \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"amino_acids\": \"Q/*\", \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"variant_allele\": \"A\", \"loeuf\": 0.105}], \"start\": 48421600, \"id\": \"NC_000015.10:g.48421600G>A\", \"end\": 48421600, \"input\": \"NC_000015.10:g.48421600G>A\", \"allele_string\": \"G/A\", \"strand\": 1, \"minimised\": 1, \"vcf_string\": \"15-48421600-G-A\", \"variant_class\": \"SNV\", \"most_severe_consequence\": \"stop_gained\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.014888766367534778,\n",
       "      'pca_y': 0.09709869181689254},\n",
       "     {'t-sne_x': 17.521584179575324, 't-sne_y': 46.532210378599586},\n",
       "     {'umap_x': -9.239957809448242, 'umap_y': 5.877508163452148}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48425423-G-A',\n",
       "      'distance': 0.0006890645017847419,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472632-G-A',\n",
       "      'distance': 0.0007705326424911618,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495237-G-A',\n",
       "      'distance': 0.0008121182327158749,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485416-G-A',\n",
       "      'distance': 0.0008406268316321075,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415675-G-A',\n",
       "      'distance': 0.0008673583506606519,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48534170-G-A',\n",
       "      'distance': 0.00120441650506109,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513615-G-A',\n",
       "      'distance': 0.001294954214245081,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520764-G-A',\n",
       "      'distance': 0.00129976449534297,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465622-G-A',\n",
       "      'distance': 0.0013371609384194016,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452670-G-A',\n",
       "      'distance': 0.0013511305442079902,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415540-G-A',\n",
       "      'distance': 0.0014044995186850429,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.0014548260951414704,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520737-G-A',\n",
       "      'distance': 0.0015523636247962713,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487129-G-A',\n",
       "      'distance': 0.0015701863449066877,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.0015938773285597563,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428361-G-A',\n",
       "      'distance': 0.0016484425868839025,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437376-G-A',\n",
       "      'distance': 0.0016505590174347162,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427630-G-A',\n",
       "      'distance': 0.0016650102334097028,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488197-G-A',\n",
       "      'distance': 0.001737675629556179,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.0018335620407015085,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48421596-C-T',\n",
       "    'metadata': {'vcf_string': '15\\t48421596\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48421596',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7661G>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48421596C>T',\n",
       "     'hgvs_protein': 'p.Arg2554Gln',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\", \"minimised\": 1, \"vcf_string\": \"15-48421596-C-T\", \"input\": \"NC_000015.10:g.48421596C>T\", \"end\": 48421596, \"id\": \"NC_000015.10:g.48421596C>T\", \"allele_string\": \"C/T\", \"strand\": 1, \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM127269\", \"end\": 48421596, \"start\": 48421596, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"clin_sig_allele\": \"T:likely_benign;T:conflicting_classifications_of_pathogenicity;T:uncertain_significance\", \"start\": 48421596, \"pubmed\": [24033266, 24941995], \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\", \"var_synonyms\": {\"ClinVar\": [\"RCV001081111\", \"RCV000766961\", \"RCV000347326\", \"RCV000382117\", \"RCV000400646\", \"RCV000396029\", \"RCV000454621\", \"RCV000290065\", \"VCV000316364\", \"RCV000313030\", \"RCV000297567\"]}, \"strand\": 1, \"clin_sig\": [\"uncertain_significance\", \"likely_benign\"], \"id\": \"rs199522781\", \"frequencies\": {\"T\": {\"gnomadg_remaining\": 0, \"gnomade_fin\": 0.0002809, \"gnomadg_eas\": 0, \"gnomade_afr\": 2.988e-05, \"gnomade_sas\": 1.162e-05, \"gnomade_remaining\": 6.626e-05, \"gnomadg_nfe\": 0.0002499, \"gnomade_nfe\": 5.126e-05, \"gnomadg_fin\": 0.0001882, \"gnomade_amr\": 2.238e-05, \"gnomade_mid\": 0, \"gnomadg_mid\": 0, \"gnomade_eas\": 5.045e-05, \"gnomadg_amr\": 0.0001309, \"gnomadg_afr\": 0, \"gnomadg_asj\": 0, \"gnomadg\": 0.000138, \"gnomade\": 5.543e-05, \"gnomadg_sas\": 0, \"gnomadg_ami\": 0, \"gnomade_asj\": 0}}, \"end\": 48421596, \"allele_string\": \"C/G/T\"}], \"transcript_consequences\": [{\"cdna_end\": 7977, \"aa\": \"C\", \"hgvsp\": \"NP_000129.3:p.Arg2554Gln\", \"location\": \"15:48421596\", \"allele_num\": 1, \"mastermind_counts\": \"12|13|13\", \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_raw\": 5.372964, \"spliceai\": {\"DP_AL\": 46, \"DS_DL\": 0.02, \"DP_DG\": -2, \"DP_DL\": -38, \"DS_AG\": 0, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0, \"DP_AG\": -2}, \"given_ref\": \"C\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48421596, \"start\": 48421595}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"geno2mp_hpo_count\": 4, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"used_ref\": \"C\", \"eve_score\": 0.167144359662773, \"cds_start\": 7661, \"cdna_start\": 7977, \"geno2mp_url\": \"https://geno2mp.gs.washington.edu/Geno2MP/#/variant/15/48713793/C%3ET/snp\", \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.225451802316178, \"cadd_phred\": 31, \"alphamissense\": {\"am_pathogenicity\": 0.1876, \"am_class\": \"likely_benign\"}, \"sift_score\": 0.02, \"exon\": \"62/66\", \"conservation\": 3.76, \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"eve_class\": \"Benign\", \"hgvsc\": \"NM_000138.5:c.7661G>A\", \"mastermind_mmid3\": \"FBN1:R2554Q\", \"mutationtaster_pred\": \"N\", \"gene_id\": \"2200\", \"blosum62\": 1, \"protein_end\": 2554, \"gene_symbol\": \"FBN1\", \"variant_allele\": \"T\", \"loeuf\": 0.105, \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.985, \"canonical\": 1, \"amino_acids\": \"R/Q\", \"sift_prediction\": \"deleterious_low_confidence\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"lrt_pred\": \"invalid_field\", \"protein_start\": 2554, \"codons\": \"cGg/cAg\", \"cds_end\": 7661}], \"start\": 48421596, \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': 0.03410892889898498,\n",
       "      'pca_y': 0.08319496301967658},\n",
       "     {'t-sne_x': 9.629803518675732, 't-sne_y': 3.9328350202788007},\n",
       "     {'umap_x': 3.828916311264038, 'umap_y': 2.879464626312256}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.0016970316646620631,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421594-C-T',\n",
       "      'distance': 0.0017439804505556822,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485379-C-T',\n",
       "      'distance': 0.002192070474848151,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.0022019429598003626,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415684-C-T',\n",
       "      'distance': 0.0022134198807179928,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.0023184390738606453,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.0023834891617298126,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432976-C-T',\n",
       "      'distance': 0.0024052108637988567,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495176-C-T',\n",
       "      'distance': 0.002429964952170849,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.002464527962729335,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421982-C-T',\n",
       "      'distance': 0.0024825807195156813,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487188-C-T',\n",
       "      'distance': 0.002512318082153797,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420731-C-T',\n",
       "      'distance': 0.0025252392515540123,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496213-C-T',\n",
       "      'distance': 0.002528798533603549,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495162-C-T',\n",
       "      'distance': 0.002538779517635703,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489946-C-T',\n",
       "      'distance': 0.002561685163527727,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48496177-C-T',\n",
       "      'distance': 0.002601626329123974,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481768-C-T',\n",
       "      'distance': 0.002603897824883461,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489980-C-T',\n",
       "      'distance': 0.002607507398352027,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487119-C-T',\n",
       "      'distance': 0.0026482893154025078,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48421564-48421590-del-5d3c8606',\n",
       "    'metadata': {'vcf_string': '15\\t48421564\\t.\\tAGCTGGAGCCGGTCTGATCAAGTGAGA\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48421564',\n",
       "     'ref_allele': 'AGCTGGAGCCGGTCTGATCAAGTGAGA',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7668_7693del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48421565_48421590del',\n",
       "     'hgvs_protein': 'p.Phe2556LeufsTer2',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"frameshift_variant\", \"seq_region_name\": \"15\", \"variant_class\": \"deletion\", \"vcf_string\": \"15-48421564-AGCTGGAGCCGGTCTGATCAAGTGAGA-A\", \"minimised\": 1, \"allele_string\": \"GCTGGAGCCGGTCTGATCAAGTGAGA/-\", \"strand\": 1, \"id\": \"NC_000015.10:g.48421565_48421590del\", \"input\": \"NC_000015.10:g.48421565_48421590del\", \"end\": 48421590, \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"given_ref\": \"GCTGGAGCCGGTCTGATCAAGTGAGA\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48421590, \"start\": 48421564}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}}, \"allele_num\": 1, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cdna_end\": 8009, \"aa\": \"GCTGGAGCCGGTCTGATCAAGTGAGA\", \"location\": \"15:48421564-48421589\", \"hgvsp\": \"NP_000129.3:p.Phe2556LeufsTer2\", \"consequence_terms\": [\"frameshift_variant\"], \"cdna_start\": 7984, \"hgvs_offset\": -1, \"cds_start\": 7668, \"used_ref\": \"AGCTGGAGCCGGTCTGATCAAGTGAG\", \"protein_end\": 2565, \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.892873723305478,GERP_DIST:1006.97418853045,BP_DIST:920,DIST_FROM_LAST_EXON:531,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.7668_7693del\", \"exon\": \"62/66\", \"conservation\": 1.59, \"impact\": \"HIGH\", \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"cds_end\": 7693, \"codons\": \"ttCTCACTTGATCAGACCGGCTCCAGCTgt/ttgt\", \"protein_start\": 2556, \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"FSLDQTGSSC/LX\", \"canonical\": 1, \"genesplicer\": \"gain/donor/48421572-48421573/Medium/4.803673\", \"variant_allele\": \"-\", \"loeuf\": 0.105}], \"start\": 48421565}',\n",
       "    'coordinates': [{'pca_x': 0.027209274594555022,\n",
       "      'pca_y': 0.04702332019080862},\n",
       "     {'t-sne_x': 8.846175288892338, 't-sne_y': 2.912611437825335},\n",
       "     {'umap_x': 4.135585308074951, 'umap_y': 3.6589770317077637}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.11647334694862366,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.1168881505727768,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.11709821224212646,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508602-G-T',\n",
       "      'distance': 0.11784219741821289,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48600173-G-T',\n",
       "      'distance': 0.11800600588321686,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.11819471418857574,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.11830183118581772,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.1183202862739563,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.11832915246486664,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48516225-G-A',\n",
       "      'distance': 0.11865565180778503,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.11873690783977509,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.11896926164627075,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.11906404793262482,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.11915890872478485,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425802-C-A',\n",
       "      'distance': 0.11948895454406738,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420718-G-T',\n",
       "      'distance': 0.11954379081726074,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490047-G-T',\n",
       "      'distance': 0.11954507231712341,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.11959167569875717,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427646-A-T',\n",
       "      'distance': 0.11970051378011703,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48436970-C-A',\n",
       "      'distance': 0.11975783109664917,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48421576-T-G',\n",
       "    'metadata': {'vcf_string': '15\\t48421576\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48421576',\n",
       "     'ref_allele': 'T',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7681A>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48421576T>G',\n",
       "     'hgvs_protein': 'p.Thr2561Pro',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"variant_class\": \"SNV\", \"vcf_string\": \"15-48421576-T-G\", \"minimised\": 1, \"allele_string\": \"T/G\", \"strand\": 1, \"id\": \"NC_000015.10:g.48421576T>G\", \"end\": 48421576, \"input\": \"NC_000015.10:g.48421576T>G\", \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"lrt_pred\": \"invalid_field\", \"cds_end\": 7681, \"protein_start\": 2561, \"codons\": \"Acc/Ccc\", \"sift_prediction\": \"deleterious_low_confidence\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.085, \"canonical\": 1, \"amino_acids\": \"T/P\", \"loeuf\": 0.105, \"variant_allele\": \"G\", \"protein_end\": 2561, \"gene_symbol\": \"FBN1\", \"mastermind_mmid3\": \"FBN1:T2561P\", \"mutationtaster_pred\": \"D\", \"gene_id\": \"2200\", \"blosum62\": -1, \"eve_class\": \"Benign\", \"hgvsc\": \"NM_000138.5:c.7681A>C\", \"cadd_phred\": 21.7, \"alphamissense\": {\"am_pathogenicity\": 0.1985, \"am_class\": \"likely_benign\"}, \"sift_score\": 0, \"exon\": \"62/66\", \"conservation\": 3.76, \"impact\": \"MODERATE\", \"polyphen_prediction\": \"benign\", \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.835597276687622, \"cdna_start\": 7997, \"cds_start\": 7681, \"used_ref\": \"T\", \"eve_score\": 0.318964803792405, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"spliceai\": {\"DP_DG\": -49, \"DP_AL\": 0, \"DS_DL\": 0, \"DP_DL\": -18, \"DS_AG\": 0, \"DS_DG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": -11}, \"given_ref\": \"T\", \"ga4gh_vrs\": {\"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"start\": 48421575, \"end\": 48421576, \"type\": \"SimpleInterval\"}}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"type\": \"Allele\"}, \"mastermind_counts\": \"2|2|2\", \"allele_num\": 1, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cadd_raw\": 3.172372, \"cdna_end\": 7997, \"aa\": \"T\", \"location\": \"15:48421576\", \"hgvsp\": \"NP_000129.3:p.Thr2561Pro\"}], \"start\": 48421576, \"colocated_variants\": [{\"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48421576, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48421576, \"id\": \"CM085413\", \"strand\": 1}]}',\n",
       "    'coordinates': [{'pca_x': 0.03299528108129296,\n",
       "      'pca_y': 0.07018732733029007},\n",
       "     {'t-sne_x': 31.718334035627095, 't-sne_y': -1.1354721012867117},\n",
       "     {'umap_x': 3.953951358795166, 'umap_y': 3.37601637840271}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.0019201540853828192,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421594-C-T',\n",
       "      'distance': 0.002659630961716175,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415708-C-T',\n",
       "      'distance': 0.0028803632594645023,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.003142036497592926,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48415684-C-T',\n",
       "      'distance': 0.0031562154181301594,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48495162-C-T',\n",
       "      'distance': 0.0032037703786045313,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487155-C-T',\n",
       "      'distance': 0.00344069954007864,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.0034582773223519325,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-G',\n",
       "      'distance': 0.0034724934957921505,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.0034919108729809523,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.0034963947255164385,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.003507159184664488,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.003530772402882576,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415707-C-T',\n",
       "      'distance': 0.0035500144585967064,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496145-A-G',\n",
       "      'distance': 0.0035842563956975937,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-G',\n",
       "      'distance': 0.0036141148302704096,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432976-C-T',\n",
       "      'distance': 0.0036178682930767536,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48537706-C-T',\n",
       "      'distance': 0.003647776087746024,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468096-C-G',\n",
       "      'distance': 0.0036915182135999203,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.0037891115061938763,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48420777-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48420777\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48420777',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7729T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48420777A>G',\n",
       "     'hgvs_protein': 'p.Cys2577Arg',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"minimised\": 1, \"vcf_string\": \"15-48420777-A-G\", \"id\": \"NC_000015.10:g.48420777A>G\", \"end\": 48420777, \"input\": \"NC_000015.10:g.48420777A>G\", \"allele_string\": \"A/G\", \"strand\": 1, \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"end\": 48420777, \"id\": \"CM972859\", \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"start\": 48420777}], \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"mastermind_mmid3\": \"FBN1:C2577R\", \"blosum62\": -3, \"gene_id\": \"2200\", \"mutationtaster_pred\": \"D\", \"protein_end\": 2577, \"gene_symbol\": \"FBN1\", \"alphamissense\": {\"am_pathogenicity\": 0.9997, \"am_class\": \"likely_pathogenic\"}, \"sift_score\": 0, \"cadd_phred\": 29.5, \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"conservation\": 3.77, \"exon\": \"63/66\", \"eve_class\": \"Pathogenic\", \"hgvsc\": \"NM_000138.5:c.7729T>C\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"sift_prediction\": \"deleterious_low_confidence\", \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"cds_end\": 7729, \"codons\": \"Tgc/Cgc\", \"protein_start\": 2577, \"lrt_pred\": \"invalid_field\", \"loeuf\": 0.105, \"variant_allele\": \"G\", \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.996, \"amino_acids\": \"C/R\", \"canonical\": 1, \"given_ref\": \"A\", \"spliceai\": {\"DP_AG\": 29, \"DS_DG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DS_AG\": 0, \"DP_DL\": 12, \"DP_DG\": -37, \"DP_AL\": 50, \"DS_DL\": 0}, \"ga4gh_vrs\": {\"state\": {\"sequence\": \"G\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48420777, \"start\": 48420776}}, \"type\": \"Allele\"}, \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"cdna_end\": 8045, \"hgvsp\": \"NP_000129.3:p.Cys2577Arg\", \"location\": \"15:48420777\", \"aa\": \"A\", \"allele_num\": 1, \"mastermind_counts\": \"3|4|4\", \"cadd_raw\": 5.253789, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cdna_start\": 8045, \"clinpred\": 0.998970508575439, \"consequence_terms\": [\"missense_variant\"], \"used_ref\": \"A\", \"eve_score\": 0.786834287984379, \"cds_start\": 7729}], \"start\": 48420777}',\n",
       "    'coordinates': [{'pca_x': 0.04867173019680501,\n",
       "      'pca_y': 0.08187166793414247},\n",
       "     {'t-sne_x': 30.718573454937992, 't-sne_y': 2.609139430741003},\n",
       "     {'umap_x': 2.3396403789520264, 'umap_y': 6.870666027069092}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48430791-A-G',\n",
       "      'distance': 0.0014805699465796351,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420795-A-G',\n",
       "      'distance': 0.0026364082004874945,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437873-A-G',\n",
       "      'distance': 0.002714807167649269,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.0027889697812497616,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470679-A-G',\n",
       "      'distance': 0.002981546800583601,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434702-A-G',\n",
       "      'distance': 0.003060423769056797,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434669-A-G',\n",
       "      'distance': 0.003174207638949156,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470745-A-G',\n",
       "      'distance': 0.003288702107965946,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474567-A-G',\n",
       "      'distance': 0.0034362245351076126,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430676-C-G',\n",
       "      'distance': 0.0035405417438596487,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425375-A-G',\n",
       "      'distance': 0.0036245775409042835,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.0036501088179647923,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415690-A-G',\n",
       "      'distance': 0.0036508250050246716,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48432944-A-G',\n",
       "      'distance': 0.004050186835229397,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596325-A-G',\n",
       "      'distance': 0.004092090763151646,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474534-A-G',\n",
       "      'distance': 0.004162090364843607,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520677-A-G',\n",
       "      'distance': 0.004170053638517857,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48497272-A-G',\n",
       "      'distance': 0.004231063649058342,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48434600-A-G',\n",
       "      'distance': 0.004259348846971989,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428427-G-A',\n",
       "      'distance': 0.004274506121873856,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48420765-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48420765\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48420765',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7741T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48420765A>T',\n",
       "     'hgvs_protein': 'p.Cys2581Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48420765-A-T\", \"minimised\": 1, \"seq_region_name\": \"15\", \"most_severe_consequence\": \"missense_variant\", \"variant_class\": \"SNV\", \"start\": 48420765, \"transcript_consequences\": [{\"impact\": \"MODERATE\", \"polyphen_prediction\": \"probably_damaging\", \"exon\": \"63/66\", \"conservation\": 3.77, \"alphamissense\": {\"am_class\": \"likely_pathogenic\", \"am_pathogenicity\": 0.9995}, \"sift_score\": 0, \"cadd_phred\": 28.9, \"hgvsc\": \"NM_000138.5:c.7741T>A\", \"eve_class\": \"Pathogenic\", \"gene_id\": \"2200\", \"blosum62\": -1, \"mutationtaster_pred\": \"D\", \"mastermind_mmid3\": \"FBN1:C2581S\", \"gene_symbol\": \"FBN1\", \"protein_end\": 2581, \"loeuf\": 0.105, \"variant_allele\": \"T\", \"amino_acids\": \"C/S\", \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0.985, \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"sift_prediction\": \"deleterious_low_confidence\", \"codons\": \"Tgc/Agc\", \"cds_end\": 7741, \"protein_start\": 2581, \"lrt_pred\": \"invalid_field\", \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"hgvsp\": \"NP_000129.3:p.Cys2581Ser\", \"location\": \"15:48420765\", \"aa\": \"A\", \"cdna_end\": 8057, \"cadd_raw\": 5.16487, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"mastermind_counts\": \"2|3|3\", \"allele_num\": 1, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48420765, \"start\": 48420764}}}, \"given_ref\": \"A\", \"spliceai\": {\"DP_DG\": -25, \"DS_DL\": 0, \"DP_AL\": 41, \"DP_DL\": 24, \"DS_AG\": 0, \"DS_AL\": 0, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_AG\": 30}, \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"eve_score\": 0.739232693736857, \"used_ref\": \"A\", \"cds_start\": 7741, \"cdna_start\": 8057, \"clinpred\": 0.999260485172272, \"consequence_terms\": [\"missense_variant\"]}], \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"id\": \"CM086599\", \"end\": 48420765, \"start\": 48420765, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"clin_sig\": [\"pathogenic\"], \"allele_string\": \"A/T\", \"id\": \"rs1165503481\", \"end\": 48420765, \"var_synonyms\": {\"ClinVar\": [\"RCV002400779\", \"VCV001760403\"]}, \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48420765, \"clin_sig_allele\": \"T:pathogenic\", \"phenotype_or_disease\": 1, \"pubmed\": [11826022, 11700157]}], \"strand\": 1, \"allele_string\": \"A/T\", \"input\": \"NC_000015.10:g.48420765A>T\", \"end\": 48420765, \"id\": \"NC_000015.10:g.48420765A>T\"}',\n",
       "    'coordinates': [{'pca_x': 0.04122243882908405,\n",
       "      'pca_y': 0.08465866769337857},\n",
       "     {'t-sne_x': 15.707488630933007, 't-sne_y': 8.744954687166322},\n",
       "     {'umap_x': 3.5938825607299805, 'umap_y': 4.738768577575684}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48428439-A-T',\n",
       "      'distance': 0.0007920658681541681,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421609-A-T',\n",
       "      'distance': 0.001361575210466981,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48596325-A-T',\n",
       "      'distance': 0.0019096191972494125,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427684-A-T',\n",
       "      'distance': 0.001914007356390357,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48427603-A-T',\n",
       "      'distance': 0.0023293565027415752,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430791-A-T',\n",
       "      'distance': 0.002948567271232605,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425783-C-A',\n",
       "      'distance': 0.002989754779264331,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437369-C-T',\n",
       "      'distance': 0.0031832889653742313,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-T',\n",
       "      'distance': 0.0034102932550013065,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496213-C-T',\n",
       "      'distance': 0.003442584304139018,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48434702-A-G',\n",
       "      'distance': 0.003480997635051608,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48436966-C-T',\n",
       "      'distance': 0.0034829736687242985,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425479-C-T',\n",
       "      'distance': 0.003501609433442354,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472595-C-T',\n",
       "      'distance': 0.00355557631701231,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.0035591227933764458,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.003581680590286851,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474299-C-T',\n",
       "      'distance': 0.0036332544405013323,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421675-A-G',\n",
       "      'distance': 0.003659619949758053,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428459-C-T',\n",
       "      'distance': 0.003691127523779869,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48474572-C-T',\n",
       "      'distance': 0.0036932569928467274,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48420750-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48420750\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48420750',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7756G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48420750C>A',\n",
       "     'hgvs_protein': 'p.Gly2586Trp',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"minimised\": 1, \"vcf_string\": \"15-48420750-C-A\", \"variant_class\": \"SNV\", \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"colocated_variants\": [{\"strand\": 1, \"end\": 48420750, \"id\": \"CM098727\", \"allele_string\": \"HGMD_MUTATION\", \"start\": 48420750, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}], \"assembly_name\": \"GRCh38\", \"start\": 48420750, \"transcript_consequences\": [{\"cds_end\": 7756, \"codons\": \"Ggg/Tgg\", \"protein_start\": 2586, \"lrt_pred\": \"invalid_field\", \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"sift_prediction\": \"deleterious_low_confidence\", \"canonical\": 1, \"amino_acids\": \"G/W\", \"polyphen_score\": 1, \"mane_select\": \"ENST00000316623.10\", \"loeuf\": 0.105, \"variant_allele\": \"A\", \"gene_symbol\": \"FBN1\", \"protein_end\": 2586, \"gene_id\": \"2200\", \"blosum62\": -2, \"mutationtaster_pred\": \"D\", \"mastermind_mmid3\": \"FBN1:G2586W\", \"hgvsc\": \"NM_000138.5:c.7756G>T\", \"eve_class\": \"Pathogenic\", \"polyphen_prediction\": \"probably_damaging\", \"impact\": \"MODERATE\", \"exon\": \"63/66\", \"conservation\": 3.95, \"alphamissense\": {\"am_pathogenicity\": 0.9937, \"am_class\": \"likely_pathogenic\"}, \"sift_score\": 0, \"cadd_phred\": 32, \"clinpred\": 0.999256551265717, \"consequence_terms\": [\"missense_variant\"], \"cdna_start\": 8072, \"cds_start\": 7756, \"eve_score\": 0.701816953695869, \"used_ref\": \"C\", \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48420750, \"start\": 48420749}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"given_ref\": \"C\", \"spliceai\": {\"DP_DL\": 45, \"DS_AG\": 0, \"DP_DG\": -10, \"DP_AL\": -27, \"DS_DL\": 0, \"DP_AG\": -11, \"DS_DG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\"}, \"cadd_raw\": 5.589592, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"allele_num\": 1, \"mastermind_counts\": \"1|1|1\", \"hgvsp\": \"NP_000129.3:p.Gly2586Trp\", \"location\": \"15:48420750\", \"aa\": \"C\", \"cdna_end\": 8072}], \"id\": \"NC_000015.10:g.48420750C>A\", \"end\": 48420750, \"input\": \"NC_000015.10:g.48420750C>A\", \"allele_string\": \"C/A\", \"strand\": 1}',\n",
       "    'coordinates': [{'pca_x': 0.035461707225741435,\n",
       "      'pca_y': 0.090036643359174},\n",
       "     {'t-sne_x': 30.344048277399903, 't-sne_y': 11.783005115994122},\n",
       "     {'umap_x': 2.315354347229004, 'umap_y': 7.906872272491455}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.0010773236863315105,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.0018522345926612616,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428427-G-A',\n",
       "      'distance': 0.00229041650891304,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437379-G-A',\n",
       "      'distance': 0.002369247144088149,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437009-G-A',\n",
       "      'distance': 0.0025398884899914265,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415582-C-A',\n",
       "      'distance': 0.0026173002552241087,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430710-G-A',\n",
       "      'distance': 0.002646037843078375,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48496150-C-A',\n",
       "      'distance': 0.0028019973542541265,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48490050-C-A',\n",
       "      'distance': 0.0028291703201830387,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48470726-C-A',\n",
       "      'distance': 0.002831793390214443,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48470744-C-A',\n",
       "      'distance': 0.0028594378381967545,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488403-C-A',\n",
       "      'distance': 0.002877937164157629,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48420731-C-A',\n",
       "      'distance': 0.00294036278501153,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421603-A-G',\n",
       "      'distance': 0.0030513512901961803,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48425783-C-A',\n",
       "      'distance': 0.0031261732801795006,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415581-C-A',\n",
       "      'distance': 0.0031579071655869484,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490013-G-A',\n",
       "      'distance': 0.0031673468183726072,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495173-C-A',\n",
       "      'distance': 0.003187076188623905,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463904-C-A',\n",
       "      'distance': 0.0031913956627249718,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48490073-G-A',\n",
       "      'distance': 0.003367571160197258,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48415723-A-T',\n",
       "    'metadata': {'vcf_string': '15\\t48415723\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415723',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'T',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7864T>A',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415723A>T',\n",
       "     'hgvs_protein': 'p.Cys2622Ser',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48415723-A-T\", \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"cds_start\": 7864, \"eve_score\": 0.789073753857352, \"used_ref\": \"A\", \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.999791204929352, \"cdna_start\": 8180, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cadd_raw\": 4.958299, \"mastermind_counts\": \"2|2|3\", \"allele_num\": 1, \"aa\": \"A\", \"hgvsp\": \"NP_000129.3:p.Cys2622Ser\", \"location\": \"15:48415723\", \"cdna_end\": 8180, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"ga4gh_vrs\": {\"state\": {\"sequence\": \"T\", \"type\": \"SequenceState\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"type\": \"SimpleInterval\", \"end\": 48415723, \"start\": 48415722}}, \"type\": \"Allele\"}, \"spliceai\": {\"DP_AL\": 25, \"DS_DL\": 0, \"DP_DG\": -49, \"DS_AG\": 0, \"DP_DL\": 44, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0, \"DP_AG\": 44}, \"given_ref\": \"A\", \"canonical\": 1, \"amino_acids\": \"C/S\", \"polyphen_score\": 0.487, \"mane_select\": \"ENST00000316623.10\", \"loeuf\": 0.105, \"variant_allele\": \"T\", \"lrt_pred\": \"invalid_field\", \"protein_start\": 2622, \"codons\": \"Tgt/Agt\", \"cds_end\": 7864, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"sift_prediction\": \"deleterious_low_confidence\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"hgvsc\": \"NM_000138.5:c.7864T>A\", \"eve_class\": \"Pathogenic\", \"conservation\": 5.35, \"exon\": \"64/66\", \"impact\": \"MODERATE\", \"polyphen_prediction\": \"possibly_damaging\", \"cadd_phred\": 27.7, \"alphamissense\": {\"am_pathogenicity\": 0.9994, \"am_class\": \"likely_pathogenic\"}, \"sift_score\": 0, \"gene_symbol\": \"FBN1\", \"protein_end\": 2622, \"mutationtaster_pred\": \"A\", \"gene_id\": \"2200\", \"blosum62\": -1, \"mastermind_mmid3\": \"FBN1:C2622S\"}], \"start\": 48415723, \"colocated_variants\": [{\"strand\": 1, \"end\": 48415723, \"id\": \"CM042043\", \"allele_string\": \"HGMD_MUTATION\", \"start\": 48415723, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}, {\"strand\": 1, \"id\": \"CM157314\", \"end\": 48415723, \"allele_string\": \"HGMD_MUTATION\", \"phenotype_or_disease\": 1, \"start\": 48415723, \"seq_region_name\": \"15\"}, {\"allele_string\": \"A/G/T\", \"id\": \"rs1555393882\", \"end\": 48415723, \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"], \"var_synonyms\": {\"ClinVar\": [\"RCV000663988\", \"VCV000549436\", \"RCV002530617\", \"RCV000663987\", \"VCV000549435\"]}, \"strand\": 1, \"seq_region_name\": \"15\", \"phenotype_or_disease\": 1, \"pubmed\": [17701892], \"start\": 48415723, \"clin_sig_allele\": \"T:likely_pathogenic;G:pathogenic;G:likely_pathogenic\"}], \"allele_string\": \"A/T\", \"strand\": 1, \"id\": \"NC_000015.10:g.48415723A>T\", \"input\": \"NC_000015.10:g.48415723A>T\", \"end\": 48415723}',\n",
       "    'coordinates': [{'pca_x': 0.03185167081695087,\n",
       "      'pca_y': 0.10440495050782952},\n",
       "     {'t-sne_x': -14.640928189031328, 't-sne_y': 33.30536073626149},\n",
       "     {'umap_x': 0.9800006747245789, 'umap_y': 21.876340866088867}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48425745-A-T',\n",
       "      'distance': 0.0010916779283434153,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495243-A-T',\n",
       "      'distance': 0.0027259881608188152,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48465619-A-T',\n",
       "      'distance': 0.002787819132208824,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48472557-A-T',\n",
       "      'distance': 0.0030866118613630533,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415755-C-T',\n",
       "      'distance': 0.0037725237198174,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495140-C-T',\n",
       "      'distance': 0.003777154488489032,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474260-C-T',\n",
       "      'distance': 0.0038389258552342653,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48474365-C-T',\n",
       "      'distance': 0.003995127510279417,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425458-C-T',\n",
       "      'distance': 0.004138837568461895,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513554-C-T',\n",
       "      'distance': 0.004217902198433876,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48495215-C-T',\n",
       "      'distance': 0.0042576114647090435,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425744-C-T',\n",
       "      'distance': 0.004276819061487913,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513661-A-G',\n",
       "      'distance': 0.004302408546209335,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48422024-A-T',\n",
       "      'distance': 0.00440339557826519,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48515440-T-T',\n",
       "      'distance': 0.004405587911605835,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48456643-A-G',\n",
       "      'distance': 0.004406552761793137,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48422023-C-T',\n",
       "      'distance': 0.004431076813489199,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437047-C-T',\n",
       "      'distance': 0.004578270949423313,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492539-A-G',\n",
       "      'distance': 0.004660270642489195,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48492538-C-T',\n",
       "      'distance': 0.004722803831100464,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48415613-GGGGGCCTGCGCAG-G',\n",
       "    'metadata': {'vcf_string': '15\\t48415613\\t.\\tGGGGGCCTGCGCAG\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415613',\n",
       "     'ref_allele': 'GGGGGCCTGCGCAG',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7965_7977del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415614_48415626del',\n",
       "     'hgvs_protein': 'p.Gln2656AlafsTer22',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"deletion\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"frameshift_variant\", \"minimised\": 1, \"vcf_string\": \"15-48415613-GGGGGCCTGCGCAG-G\", \"input\": \"NC_000015.10:g.48415614_48415626del\", \"end\": 48415626, \"id\": \"NC_000015.10:g.48415614_48415626del\", \"allele_string\": \"GGGGCCTGCGCAG/-\", \"strand\": 1, \"start\": 48415614, \"transcript_consequences\": [{\"exon\": \"64/66\", \"conservation\": 1.578, \"impact\": \"HIGH\", \"hgvsc\": \"NM_000138.5:c.7965_7977del\", \"gene_id\": \"2200\", \"protein_end\": 2659, \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.9258356545961,GERP_DIST:622.53373208642,BP_DIST:641,DIST_FROM_LAST_EXON:252,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"-\", \"loeuf\": 0.105, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"AQAPC/X\", \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"cds_end\": 7977, \"codons\": \"gcGCAGGCCCCCTGC/gc\", \"protein_start\": 2655, \"cdna_end\": 8293, \"aa\": \"GGGGCCTGCGCAG\", \"location\": \"15:48415610-48415622\", \"hgvsp\": \"NP_000129.3:p.Gln2656AlafsTer22\", \"allele_num\": 1, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"given_ref\": \"GGGGCCTGCGCAG\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48415626, \"type\": \"SimpleInterval\", \"start\": 48415613}}}, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"used_ref\": \"GCAGGGGGCCTGC\", \"hgvs_offset\": -4, \"cds_start\": 7965, \"cdna_start\": 8281, \"consequence_terms\": [\"frameshift_variant\"]}], \"assembly_name\": \"GRCh38\"}',\n",
       "    'coordinates': [{'pca_x': 0.03424685815637199,\n",
       "      'pca_y': 0.042534479481095185},\n",
       "     {'t-sne_x': 34.4544370344156, 't-sne_y': 8.492495922831527},\n",
       "     {'umap_x': 1.541014313697815, 'umap_y': 7.269289016723633}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48492509-G-A',\n",
       "      'distance': 0.10912247002124786,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430758-G-A',\n",
       "      'distance': 0.10920678824186325,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48487402-G-A',\n",
       "      'distance': 0.11059804260730743,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520755-G-A',\n",
       "      'distance': 0.11068783700466156,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456691-G-A',\n",
       "      'distance': 0.11075194925069809,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48437912-G-A',\n",
       "      'distance': 0.11130964010953903,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537755-G-A',\n",
       "      'distance': 0.11137336492538452,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472594-G-T',\n",
       "      'distance': 0.11152169108390808,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48596328-G-A',\n",
       "      'distance': 0.11195121705532074,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420714-G-A',\n",
       "      'distance': 0.11205318570137024,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445424-G-A',\n",
       "      'distance': 0.11220405995845795,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520716-G-A',\n",
       "      'distance': 0.11225158721208572,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520764-G-A',\n",
       "      'distance': 0.11258435994386673,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.11278261244297028,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48520795-G-T',\n",
       "      'distance': 0.11295223236083984,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48516225-G-A',\n",
       "      'distance': 0.11342082917690277,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425393-G-A',\n",
       "      'distance': 0.11351746320724487,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48513591-G-A',\n",
       "      'distance': 0.11374553292989731,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48537704-G-A',\n",
       "      'distance': 0.11376563459634781,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468427-G-A',\n",
       "      'distance': 0.1137838140130043,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48415621-G-A',\n",
       "    'metadata': {'vcf_string': '15\\t48415621\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415621',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.7966C>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415621G>A',\n",
       "     'hgvs_protein': 'p.Gln2656Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"end\": 48415621, \"id\": \"CM098620\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48415621, \"phenotype_or_disease\": 1}, {\"end\": 48415621, \"id\": \"rs2141214772\", \"allele_string\": \"G/A\", \"clin_sig\": [\"pathogenic\"], \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV001383756\", \"VCV001071314\"]}, \"seq_region_name\": \"15\", \"clin_sig_allele\": \"A:pathogenic\", \"start\": 48415621, \"pubmed\": [17657824], \"phenotype_or_disease\": 1}], \"assembly_name\": \"GRCh38\", \"start\": 48415621, \"transcript_consequences\": [{\"hgvsc\": \"NM_000138.5:c.7966C>T\", \"cadd_phred\": 43, \"conservation\": 4.43, \"exon\": \"64/66\", \"impact\": \"HIGH\", \"protein_end\": 2656, \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.92455896007428,GERP_DIST:583.419414436817,BP_DIST:648,DIST_FROM_LAST_EXON:259,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"mastermind_mmid3\": \"FBN1:Q2656X\", \"mutationtaster_pred\": \"A\", \"gene_id\": \"2200\", \"mane_select\": \"ENST00000316623.10\", \"amino_acids\": \"Q/*\", \"canonical\": 1, \"variant_allele\": \"A\", \"genesplicer\": \"gain/donor/48415620-48415621/Medium/1.062081\", \"loeuf\": 0.105, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"lrt_pred\": \"invalid_field\", \"cds_end\": 7966, \"protein_start\": 2656, \"codons\": \"Cag/Tag\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"allele_num\": 1, \"mastermind_counts\": \"2|2|2\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cadd_raw\": 9.786658, \"cdna_end\": 8282, \"aa\": \"G\", \"hgvsp\": \"NP_000129.3:p.Gln2656Ter\", \"location\": \"15:48415621\", \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"spliceai\": {\"DP_AG\": -14, \"DS_AL\": 0, \"DS_DG\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": -3, \"DS_AG\": 0, \"DP_DG\": 2, \"DS_DL\": 0, \"DP_AL\": -3}, \"given_ref\": \"G\", \"ga4gh_vrs\": {\"location\": {\"interval\": {\"start\": 48415620, \"type\": \"SimpleInterval\", \"end\": 48415621}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"type\": \"Allele\"}, \"cds_start\": 7966, \"used_ref\": \"G\", \"consequence_terms\": [\"stop_gained\"], \"cdna_start\": 8282}], \"id\": \"NC_000015.10:g.48415621G>A\", \"end\": 48415621, \"input\": \"NC_000015.10:g.48415621G>A\", \"allele_string\": \"G/A\", \"strand\": 1, \"minimised\": 1, \"vcf_string\": \"15-48415621-G-A\", \"variant_class\": \"SNV\", \"most_severe_consequence\": \"stop_gained\", \"seq_region_name\": \"15\"}',\n",
       "    'coordinates': [{'pca_x': 0.020602431820033573,\n",
       "      'pca_y': 0.09616319253778097},\n",
       "     {'t-sne_x': 17.70714822220623, 't-sne_y': 46.28751343184786},\n",
       "     {'umap_x': -9.038549423217773, 'umap_y': 5.945611953735352}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48415675-G-A',\n",
       "      'distance': 0.000537922082003206,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425423-G-A',\n",
       "      'distance': 0.0007801054744049907,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485416-G-A',\n",
       "      'distance': 0.0007856826996430755,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472632-G-A',\n",
       "      'distance': 0.0008418536162935197,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465622-G-A',\n",
       "      'distance': 0.0008736391318961978,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495237-G-A',\n",
       "      'distance': 0.0009686797275207937,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48415540-G-A',\n",
       "      'distance': 0.001085874391719699,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48534170-G-A',\n",
       "      'distance': 0.001173479249700904,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48465679-G-A',\n",
       "      'distance': 0.0013476407621055841,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452670-G-A',\n",
       "      'distance': 0.0013793522957712412,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437376-G-A',\n",
       "      'distance': 0.0015584086067974567,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48428361-G-A',\n",
       "      'distance': 0.0016494934679940343,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513615-G-A',\n",
       "      'distance': 0.0017361303325742483,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48427630-G-A',\n",
       "      'distance': 0.0017433364409953356,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48520737-G-A',\n",
       "      'distance': 0.0017969534965232015,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.001841292018070817,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48432971-G-A',\n",
       "      'distance': 0.0018580476753413677,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48432947-G-A',\n",
       "      'distance': 0.0018633282743394375,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513579-G-A',\n",
       "      'distance': 0.001966959098353982,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48425829-G-A',\n",
       "      'distance': 0.001966991927474737,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48415163-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48415163\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48415163',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8051+373G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48415163C>A',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'intron_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"most_severe_consequence\": \"intron_variant\", \"seq_region_name\": \"15\", \"minimised\": 1, \"vcf_string\": \"15-48415163-C-A\", \"id\": \"NC_000015.10:g.48415163C>A\", \"input\": \"NC_000015.10:g.48415163C>A\", \"end\": 48415163, \"strand\": 1, \"allele_string\": \"C/A\", \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"type\": \"SequenceState\", \"sequence\": \"A\"}, \"location\": {\"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\", \"interval\": {\"end\": 48415163, \"type\": \"SimpleInterval\", \"start\": 48415162}}}, \"given_ref\": \"C\", \"spliceai\": {\"DP_AG\": 47, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0, \"DP_DL\": 0, \"DS_AG\": 0, \"DP_AL\": -1, \"DS_DL\": 0, \"DP_DG\": -31}, \"cadd_raw\": 0.475611, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"mastermind_counts\": \"1|1|22\", \"allele_num\": 1, \"location\": \"15:48415163\", \"aa\": \"C\", \"consequence_terms\": [\"intron_variant\"], \"used_ref\": \"C\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"mastermind_mmid3\": \"FBN1:G2684int,FBN1:G2684intd\", \"hgvsc\": \"NM_000138.5:c.8051+373G>T\", \"impact\": \"MODIFIER\", \"intron\": \"64/65\", \"conservation\": -0.42, \"cadd_phred\": 5.21, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"genesplicer\": \"loss/acceptor/48415162-48415163/Medium/2.182468\", \"variant_allele\": \"A\", \"loeuf\": 0.105}], \"start\": 48415163}',\n",
       "    'coordinates': [{'pca_x': -0.23735198269034355,\n",
       "      'pca_y': -0.03086106177079003},\n",
       "     {'t-sne_x': -44.76565329067227, 't-sne_y': -1.3219654147907016},\n",
       "     {'umap_x': 20.023286819458008, 'umap_y': 6.612325191497803}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48497410-C-A',\n",
       "      'distance': 0.002342433901503682,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48460233-C-T',\n",
       "      'distance': 0.002397761680185795,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48434577-C-T',\n",
       "      'distance': 0.002887480426579714,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48428327-C-T',\n",
       "      'distance': 0.00314552104100585,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48446691-A-G',\n",
       "      'distance': 0.0032741003669798374,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48489838-C-T',\n",
       "      'distance': 0.0033051937352865934,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48437311-A-G',\n",
       "      'distance': 0.0035353596322238445,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48503768-T-A',\n",
       "      'distance': 0.0035596739035099745,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48428074-C-T',\n",
       "      'distance': 0.0036844233982264996,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48429432-T-C',\n",
       "      'distance': 0.003771673422306776,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610929-A-G',\n",
       "      'distance': 0.003878265619277954,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48472694-C-T',\n",
       "      'distance': 0.004014857113361359,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48460336-G-A',\n",
       "      'distance': 0.0040909466333687305,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48444487-A-T',\n",
       "      'distance': 0.004170330241322517,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48430659-G-T',\n",
       "      'distance': 0.004375871736556292,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48488093-T-C',\n",
       "      'distance': 0.004522699862718582,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48474244-C-T',\n",
       "      'distance': 0.004779433831572533,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48600253-G-T',\n",
       "      'distance': 0.004816202912479639,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48425729-A-C',\n",
       "      'distance': 0.004851971752941608,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48428329-G-C',\n",
       "      'distance': 0.004984241910278797,\n",
       "      'pathogenicity': 'benign'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_benign',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_benign', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_benign',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_benign', 'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48412699-GG-G',\n",
       "    'metadata': {'vcf_string': '15\\t48412699\\t.\\tGG\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48412699',\n",
       "     'ref_allele': 'GG',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8096del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48412700del',\n",
       "     'hgvs_protein': 'p.Pro2699HisfsTer53',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"start\": 48412700, \"transcript_consequences\": [{\"protein_end\": 2699, \"lof_info\": \"PERCENTILE:0.939647168059424,GERP_DIST:508.250833535194,BP_DIST:520,DIST_FROM_LAST_EXON:131,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"gene_symbol\": \"FBN1\", \"gene_id\": \"2200\", \"hgvsc\": \"NM_000138.5:c.8096del\", \"impact\": \"HIGH\", \"exon\": \"65/66\", \"conservation\": 2.47, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"protein_start\": 2699, \"cds_end\": 8096, \"codons\": \"cCa/ca\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"amino_acids\": \"P/X\", \"loeuf\": 0.105, \"variant_allele\": \"-\", \"lof\": \"HC\", \"mane\": [\"MANE_Select\"], \"gene_symbol_source\": \"EntrezGene\", \"given_ref\": \"G\", \"spliceai\": {\"DS_AG\": 0, \"DP_DL\": -4, \"DP_AL\": -42, \"DS_DL\": 0, \"DP_DG\": 44, \"DP_AG\": -44, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"interval\": {\"end\": 48412700, \"type\": \"SimpleInterval\", \"start\": 48412699}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"type\": \"SequenceState\", \"sequence\": \"\"}}, \"allele_num\": 1, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"cdna_end\": 8412, \"location\": \"15:48412699\", \"hgvsp\": \"NP_000129.3:p.Pro2699HisfsTer53\", \"aa\": \"G\", \"consequence_terms\": [\"frameshift_variant\"], \"cdna_start\": 8412, \"hgvs_offset\": -1, \"cds_start\": 8096, \"used_ref\": \"G\"}], \"assembly_name\": \"GRCh38\", \"end\": 48412700, \"input\": \"NC_000015.10:g.48412700del\", \"id\": \"NC_000015.10:g.48412700del\", \"allele_string\": \"G/-\", \"strand\": 1, \"minimised\": 1, \"vcf_string\": \"15-48412699-GG-G\", \"variant_class\": \"deletion\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"frameshift_variant\"}',\n",
       "    'coordinates': [{'pca_x': 0.04657356942831026,\n",
       "      'pca_y': 0.037277796870805756},\n",
       "     {'t-sne_x': 31.743084646700844, 't-sne_y': 11.88867968168215},\n",
       "     {'umap_x': 2.036073923110962, 'umap_y': 7.82841682434082}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.0950985997915268,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505106-G-A',\n",
       "      'distance': 0.09516517072916031,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48515427-G-C',\n",
       "      'distance': 0.09629818797111511,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420780-G-A',\n",
       "      'distance': 0.09633275866508484,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48430710-G-A',\n",
       "      'distance': 0.0964578241109848,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48613057-C-A',\n",
       "      'distance': 0.09666536748409271,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.09684288501739502,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613073-G-A',\n",
       "      'distance': 0.0969277024269104,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09699202328920364,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48421593-C-A',\n",
       "      'distance': 0.09719307720661163,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48497336-G-T',\n",
       "      'distance': 0.0972486138343811,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48411243-G-A',\n",
       "      'distance': 0.09725591540336609,\n",
       "      'pathogenicity': 'benign'},\n",
       "     {'variant_id': '15-48503850-A-G',\n",
       "      'distance': 0.09731756150722504,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.09743830561637878,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.09748607873916626,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437779-G-A',\n",
       "      'distance': 0.0975278913974762,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.09753093868494034,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48481723-G-A',\n",
       "      'distance': 0.09767146408557892,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48600217-G-A',\n",
       "      'distance': 0.09767739474773407,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.09768856316804886,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8666666666666667},\n",
       "     '20': {'pred_result': 'likely_pathogenic', 'confidence_score': 0.8}}},\n",
       "   {'variant_id': '15-48412638-CTT-C',\n",
       "    'metadata': {'vcf_string': '15\\t48412638\\t.\\tCTT\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48412638',\n",
       "     'ref_allele': 'CTT',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8155_8156del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48412639_48412640del',\n",
       "     'hgvs_protein': 'p.Lys2719AspfsTer18',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"transcript_consequences\": [{\"hgvsc\": \"NM_000138.5:c.8155_8156del\", \"conservation\": 5.35, \"exon\": \"65/66\", \"impact\": \"HIGH\", \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.94661095636026,GERP_DIST:448.571940228343,BP_DIST:459,DIST_FROM_LAST_EXON:70,50_BP_RULE:PASS,PHYLOCSF_TOO_SHORT\", \"protein_end\": 2719, \"gene_id\": \"2200\", \"mastermind_mmid3\": \"FBN1:K2719fs\", \"amino_acids\": \"K/X\", \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"variant_allele\": \"-\", \"loeuf\": 0.105, \"cds_end\": 8156, \"protein_start\": 2719, \"codons\": \"AAg/g\", \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"allele_num\": 1, \"mastermind_counts\": \"11|11|12\", \"aa\": \"TT\", \"hgvsp\": \"NP_000129.3:p.Lys2719AspfsTer18\", \"location\": \"15:48412639-48412640\", \"cdna_end\": 8472, \"gene_symbol_source\": \"EntrezGene\", \"lof\": \"HC\", \"mane\": [\"MANE_Select\"], \"ga4gh_vrs\": {\"type\": \"Allele\", \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"end\": 48412640, \"type\": \"SimpleInterval\", \"start\": 48412638}}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}}, \"spliceai\": {\"DP_AG\": 19, \"DS_DG\": 0, \"DS_AL\": 0, \"SYMBOL\": \"FBN1\", \"DP_DL\": 5, \"DS_AG\": 0.03, \"DP_DG\": 17, \"DP_AL\": 5, \"DS_DL\": 0}, \"given_ref\": \"TT\", \"cds_start\": 8155, \"used_ref\": \"TT\", \"consequence_terms\": [\"frameshift_variant\"], \"cdna_start\": 8471}], \"start\": 48412639, \"assembly_name\": \"GRCh38\", \"colocated_variants\": [{\"strand\": 1, \"var_synonyms\": {\"OMIM\": [134797.0064], \"ClinVar\": [\"RCV000033241\", \"VCV000040243\"]}, \"end\": 48412640, \"id\": \"rs398122831\", \"allele_string\": \"TT/-\", \"clin_sig\": [\"pathogenic\"], \"pubmed\": [20979188], \"phenotype_or_disease\": 1, \"start\": 48412639, \"seq_region_name\": \"15\"}], \"allele_string\": \"TT/-\", \"strand\": 1, \"end\": 48412640, \"input\": \"NC_000015.10:g.48412639_48412640del\", \"id\": \"NC_000015.10:g.48412639_48412640del\", \"vcf_string\": \"15-48412638-CTT-C\", \"minimised\": 1, \"seq_region_name\": \"15\", \"most_severe_consequence\": \"frameshift_variant\", \"variant_class\": \"deletion\"}',\n",
       "    'coordinates': [{'pca_x': 0.03540552241568784,\n",
       "      'pca_y': 0.03803314877725663},\n",
       "     {'t-sne_x': 2.2908083403470876, 't-sne_y': 9.581141783941186},\n",
       "     {'umap_x': 3.64292311668396, 'umap_y': 3.336642026901245}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.08575981855392456,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.08602406084537506,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.08684200048446655,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48456750-C-T',\n",
       "      'distance': 0.08684240281581879,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.08777397871017456,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.08784262835979462,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468504-C-T',\n",
       "      'distance': 0.08808024227619171,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.08820824325084686,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.088272824883461,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446711-C-T',\n",
       "      'distance': 0.0882844477891922,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48452567-C-T',\n",
       "      'distance': 0.08831462264060974,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48596360-C-T',\n",
       "      'distance': 0.08836884051561356,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48520676-C-T',\n",
       "      'distance': 0.08837386220693588,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.08843923360109329,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487146-T-G',\n",
       "      'distance': 0.08849141746759415,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48496150-C-T',\n",
       "      'distance': 0.08878161758184433,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.08879093825817108,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48421608-C-T',\n",
       "      'distance': 0.08884158730506897,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.08904827386140823,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48613054-C-T',\n",
       "      'distance': 0.08906638622283936,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48412573-A-G',\n",
       "    'metadata': {'vcf_string': '15\\t48412573\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48412573',\n",
       "     'ref_allele': 'A',\n",
       "     'alt_allele': 'G',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8222T>C',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48412573A>G',\n",
       "     'hgvs_protein': 'p.Ile2741Thr',\n",
       "     'most_severe_consequence': 'missense_variant'},\n",
       "    'annotation_raw': '{\"vcf_string\": \"15-48412573-A-G\", \"minimised\": 1, \"most_severe_consequence\": \"missense_variant\", \"seq_region_name\": \"15\", \"variant_class\": \"SNV\", \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"gene_symbol\": \"FBN1\", \"protein_end\": 2741, \"mutationtaster_pred\": \"N\", \"gene_id\": \"2200\", \"blosum62\": -1, \"mastermind_mmid3\": \"FBN1:I2741T\", \"hgvsc\": \"NM_000138.5:c.8222T>C\", \"exon\": \"65/66\", \"conservation\": 1.7, \"impact\": \"MODERATE\", \"polyphen_prediction\": \"benign\", \"cadd_phred\": 5.456, \"sift_score\": 0.19, \"alphamissense\": {\"am_pathogenicity\": 0.0477, \"am_class\": \"likely_benign\"}, \"lrt_pred\": \"invalid_field\", \"codons\": \"aTc/aCc\", \"cds_end\": 8222, \"protein_start\": 2741, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"sift_prediction\": \"tolerated_low_confidence\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"amino_acids\": \"I/T\", \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"polyphen_score\": 0, \"variant_allele\": \"G\", \"loeuf\": 0.105, \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"ga4gh_vrs\": {\"state\": {\"type\": \"SequenceState\", \"sequence\": \"G\"}, \"location\": {\"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\", \"interval\": {\"start\": 48412572, \"type\": \"SimpleInterval\", \"end\": 48412573}}, \"type\": \"Allele\"}, \"spliceai\": {\"SYMBOL\": \"FBN1\", \"DS_DG\": 0, \"DS_AL\": 0, \"DP_AG\": 38, \"DP_AL\": -5, \"DS_DL\": 0, \"DP_DG\": -15, \"DS_AG\": 0, \"DP_DL\": -4}, \"given_ref\": \"A\", \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"cadd_raw\": 0.499324, \"mastermind_counts\": \"3|3|3\", \"allele_num\": 1, \"aa\": \"A\", \"hgvsp\": \"NP_000129.3:p.Ile2741Thr\", \"location\": \"15:48412573\", \"cdna_end\": 8538, \"consequence_terms\": [\"missense_variant\"], \"clinpred\": 0.0166746487056132, \"cdna_start\": 8538, \"cds_start\": 8222, \"used_ref\": \"A\"}], \"start\": 48412573, \"colocated_variants\": [{\"var_synonyms\": {\"UniProt\": [\"VAR_076160\"], \"ClinVar\": [\"RCV003790281\", \"VCV002933163\", \"RCV004805081\"]}, \"strand\": 1, \"clin_sig\": [\"uncertain_significance\", \"likely_benign\"], \"id\": \"rs1409931715\", \"frequencies\": {\"G\": {\"gnomade_amr\": 0, \"gnomade_mid\": 0, \"gnomade_fin\": 0, \"gnomade_eas\": 0, \"gnomade_afr\": 0, \"gnomade_sas\": 0, \"gnomade_remaining\": 0, \"gnomade_nfe\": 3.597e-06, \"gnomade\": 2.736e-06, \"gnomade_asj\": 0}}, \"end\": 48412573, \"allele_string\": \"A/G\", \"clin_sig_allele\": \"G:uncertain_significance;G:likely_benign\", \"start\": 48412573, \"phenotype_or_disease\": 1, \"seq_region_name\": \"15\"}], \"allele_string\": \"A/G\", \"strand\": 1, \"id\": \"NC_000015.10:g.48412573A>G\", \"input\": \"NC_000015.10:g.48412573A>G\", \"end\": 48412573}',\n",
       "    'coordinates': [{'pca_x': 0.039674548747305804,\n",
       "      'pca_y': 0.07072389330852216},\n",
       "     {'t-sne_x': 34.799254664045705, 't-sne_y': 4.496831635259905},\n",
       "     {'umap_x': 1.3945266008377075, 'umap_y': 6.685816764831543}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48432977-A-G',\n",
       "      'distance': 0.0019309250637888908,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487189-A-G',\n",
       "      'distance': 0.002033204771578312,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488433-A-G',\n",
       "      'distance': 0.0021156612783670425,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420752-A-G',\n",
       "      'distance': 0.002147279679775238,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48481661-A-G',\n",
       "      'distance': 0.0021600178442895412,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48485397-A-G',\n",
       "      'distance': 0.0022305394522845745,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48496214-A-G',\n",
       "      'distance': 0.002283109351992607,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430736-A-G',\n",
       "      'distance': 0.002293273340910673,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487318-A-G',\n",
       "      'distance': 0.0024091852828860283,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48446768-A-G',\n",
       "      'distance': 0.0024312331806868315,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48428391-A-G',\n",
       "      'distance': 0.0024510780349373817,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468463-A-G',\n",
       "      'distance': 0.002497277222573757,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437900-A-G',\n",
       "      'distance': 0.002503269584849477,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430749-A-G',\n",
       "      'distance': 0.0025371627416461706,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427756-A-G',\n",
       "      'distance': 0.002561857458204031,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48489856-A-G',\n",
       "      'distance': 0.0025668961461633444,\n",
       "      'pathogenicity': 'benign_or_likely'},\n",
       "     {'variant_id': '15-48421991-A-G',\n",
       "      'distance': 0.002568902913480997,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427603-A-G',\n",
       "      'distance': 0.00258844718337059,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496178-A-G',\n",
       "      'distance': 0.002608352340757847,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495562-A-G',\n",
       "      'distance': 0.002615994540974498,\n",
       "      'pathogenicity': 'pathogenic_or_likely'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9}}},\n",
       "   {'variant_id': '15-48412568-C-A',\n",
       "    'metadata': {'vcf_string': '15\\t48412568\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48412568',\n",
       "     'ref_allele': 'C',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8226+1G>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48412568C>A',\n",
       "     'hgvs_protein': '',\n",
       "     'most_severe_consequence': 'splice_donor_variant'},\n",
       "    'annotation_raw': '{\"variant_class\": \"SNV\", \"most_severe_consequence\": \"splice_donor_variant\", \"seq_region_name\": \"15\", \"minimised\": 1, \"vcf_string\": \"15-48412568-C-A\", \"id\": \"NC_000015.10:g.48412568C>A\", \"input\": \"NC_000015.10:g.48412568C>A\", \"end\": 48412568, \"strand\": 1, \"allele_string\": \"C/A\", \"colocated_variants\": [{\"allele_string\": \"HGMD_MUTATION\", \"end\": 48412568, \"id\": \"CS112157\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48412568, \"phenotype_or_disease\": 1}, {\"end\": 48412568, \"id\": \"CS144890\", \"allele_string\": \"HGMD_MUTATION\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48412568, \"phenotype_or_disease\": 1}, {\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"end\": 48412568, \"id\": \"CX144691\", \"phenotype_or_disease\": 1, \"start\": 48412568, \"seq_region_name\": \"15\"}, {\"pubmed\": [21594993, 24613577, 27087445], \"phenotype_or_disease\": 1, \"clin_sig_allele\": \"A:pathogenic;T:pathogenic;T:likely_pathogenic\", \"start\": 48412568, \"seq_region_name\": \"15\", \"strand\": 1, \"var_synonyms\": {\"ClinVar\": [\"RCV001566353\", \"RCV000664017\", \"RCV000631959\", \"RCV000210934\", \"VCV000225629\", \"RCV000033243\", \"VCV000040245\", \"RCV001386016\"], \"OMIM\": [134797.0066, 134797.007]}, \"end\": 48412568, \"id\": \"rs398122833\", \"allele_string\": \"C/A/T\", \"clin_sig\": [\"likely_pathogenic\", \"pathogenic\"]}], \"assembly_name\": \"GRCh38\", \"transcript_consequences\": [{\"loeuf\": 0.105, \"variant_allele\": \"A\", \"genesplicer\": \"loss/donor/48412568-48412569/Medium/4.899993\", \"mane_select\": \"ENST00000316623.10\", \"canonical\": 1, \"ptriplo\": 0.984699554922396, \"phaplo\": 0.99897615427687, \"maxentscan_alt\": -1.429, \"transcript_id\": \"NM_000138.5\", \"biotype\": \"protein_coding\", \"cadd_phred\": 26.3, \"intron\": \"65/65\", \"impact\": \"HIGH\", \"conservation\": 5.11, \"hgvsc\": \"NM_000138.5:c.8226+1G>T\", \"maxentscan_diff\": 8.504, \"mastermind_mmid3\": \"FBN1:E2742sd\", \"gene_id\": \"2200\", \"lof_info\": \"INTRON_SIZE:1189\", \"gene_symbol\": \"FBN1\", \"used_ref\": \"C\", \"rf_score\": 0.862, \"consequence_terms\": [\"splice_donor_variant\"], \"maxentscan_ref\": 7.075, \"location\": \"15:48412568\", \"aa\": \"C\", \"mastermind_counts\": \"8|8|11\", \"allele_num\": 1, \"cadd_raw\": 4.690744, \"strand\": -1, \"protein_id\": \"NP_000129.3\", \"given_ref\": \"C\", \"ada_score\": 0.999969620222061, \"spliceai\": {\"DP_AG\": -45, \"SYMBOL\": \"FBN1\", \"DS_DG\": 0.11, \"DS_AL\": 0, \"DS_AG\": 0, \"DP_DL\": 1, \"DP_AL\": 43, \"DS_DL\": 0.99, \"DP_DG\": -3}, \"ga4gh_vrs\": {\"type\": \"Allele\", \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"location\": {\"interval\": {\"type\": \"SimpleInterval\", \"end\": 48412568, \"start\": 48412567}, \"type\": \"SequenceLocation\", \"sequence_id\": \"refseq:NC_000015.10\"}}, \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"gene_symbol_source\": \"EntrezGene\"}], \"start\": 48412568}',\n",
       "    'coordinates': [{'pca_x': -0.24677768325176636,\n",
       "      'pca_y': -0.031117101561583444},\n",
       "     {'t-sne_x': -16.166689134266147, 't-sne_y': -28.28319769239965},\n",
       "     {'umap_x': 8.258520126342773, 'umap_y': 13.040166854858398}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48613009-C-A',\n",
       "      'distance': 0.0011429083533585072,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465789-C-A',\n",
       "      'distance': 0.001232837326824665,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48508581-C-A',\n",
       "      'distance': 0.0017782177310436964,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-A',\n",
       "      'distance': 0.0018140600295737386,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48445375-C-A',\n",
       "      'distance': 0.0021369056776165962,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437321-C-T',\n",
       "      'distance': 0.0029558632522821426,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48495468-C-T',\n",
       "      'distance': 0.0032571894116699696,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48437320-A-C',\n",
       "      'distance': 0.0033149858936667442,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48412568-C-T',\n",
       "      'distance': 0.003318441566079855,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420686-C-T',\n",
       "      'distance': 0.0035421450156718493,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48434593-C-T',\n",
       "      'distance': 0.003690273966640234,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48644605-C-T',\n",
       "      'distance': 0.0037544015794992447,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48468411-C-T',\n",
       "      'distance': 0.003850196022540331,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48463898-C-T',\n",
       "      'distance': 0.003995127975940704,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48445375-C-T',\n",
       "      'distance': 0.004115610383450985,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48488112-C-T',\n",
       "      'distance': 0.00415384816005826,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48613009-C-T',\n",
       "      'distance': 0.004178881645202637,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48441720-C-T',\n",
       "      'distance': 0.00420570932328701,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48474254-C-T',\n",
       "      'distance': 0.004269919823855162,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48496099-C-T',\n",
       "      'distance': 0.004275956656783819,\n",
       "      'pathogenicity': 'likely_pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}},\n",
       "   {'variant_id': '15-48411368-CTC-C',\n",
       "    'metadata': {'vcf_string': '15\\t48411368\\t.\\tCTC\\tC\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48411368',\n",
       "     'ref_allele': 'CTC',\n",
       "     'alt_allele': 'C',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8238_8239del',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48411369_48411370del',\n",
       "     'hgvs_protein': 'p.Glu2746AspfsTer13',\n",
       "     'most_severe_consequence': 'frameshift_variant'},\n",
       "    'annotation_raw': '{\"input\": \"NC_000015.10:g.48411369_48411370del\", \"end\": 48411370, \"id\": \"NC_000015.10:g.48411369_48411370del\", \"allele_string\": \"TC/-\", \"strand\": 1, \"transcript_consequences\": [{\"exon\": \"66/66\", \"conservation\": 4.445, \"impact\": \"HIGH\", \"hgvsc\": \"NM_000138.5:c.8238_8239del\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"lof_info\": \"PERCENTILE:0.956244196843083,GERP_DIST:419.379658579826,BP_DIST:379,DIST_FROM_LAST_EXON:-10,50_BP_RULE:FAIL,PHYLOCSF_TOO_SHORT\", \"protein_end\": 2747, \"variant_allele\": \"-\", \"loeuf\": 0.105, \"amino_acids\": \"ET/DX\", \"canonical\": 1, \"mane_select\": \"ENST00000316623.10\", \"phaplo\": 0.99897615427687, \"ptriplo\": 0.984699554922396, \"cds_end\": 8239, \"codons\": \"gaGAca/gaca\", \"protein_start\": 2746, \"biotype\": \"protein_coding\", \"transcript_id\": \"NM_000138.5\", \"aa\": \"TC\", \"hgvsp\": \"NP_000129.3:p.Glu2746AspfsTer13\", \"location\": \"15:48411367-48411368\", \"cdna_end\": 8555, \"protein_id\": \"NP_000129.3\", \"strand\": -1, \"allele_num\": 1, \"ga4gh_vrs\": {\"location\": {\"interval\": {\"end\": 48411370, \"type\": \"SimpleInterval\", \"start\": 48411368}, \"sequence_id\": \"refseq:NC_000015.10\", \"type\": \"SequenceLocation\"}, \"state\": {\"sequence\": \"\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"spliceai\": {\"DP_AG\": 8, \"SYMBOL\": \"FBN1\", \"DS_AL\": 0, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_DL\": 5, \"DS_DL\": 0, \"DP_AL\": 5, \"DP_DG\": -12}, \"given_ref\": \"TC\", \"gene_symbol_source\": \"EntrezGene\", \"mane\": [\"MANE_Select\"], \"lof\": \"HC\", \"used_ref\": \"TC\", \"nmd\": \"NMD_escaping_variant\", \"cds_start\": 8238, \"hgvs_offset\": -2, \"cdna_start\": 8554, \"consequence_terms\": [\"frameshift_variant\"]}], \"start\": 48411369, \"assembly_name\": \"GRCh38\", \"variant_class\": \"deletion\", \"seq_region_name\": \"15\", \"most_severe_consequence\": \"frameshift_variant\", \"minimised\": 1, \"vcf_string\": \"15-48411368-CTC-C\"}',\n",
       "    'coordinates': [{'pca_x': 0.04065361965751746,\n",
       "      'pca_y': 0.03078795737263782},\n",
       "     {'t-sne_x': 8.061256758206236, 't-sne_y': 2.908799454877673},\n",
       "     {'umap_x': 4.131190776824951, 'umap_y': 3.207465887069702}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48526133-T-C',\n",
       "      'distance': 0.08840025216341019,\n",
       "      'pathogenicity': 'likely_benign'},\n",
       "     {'variant_id': '15-48460258-C-T',\n",
       "      'distance': 0.08841118216514587,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48460281-C-T',\n",
       "      'distance': 0.08883924037218094,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48437069-C-T',\n",
       "      'distance': 0.08943548798561096,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48487110-C-T',\n",
       "      'distance': 0.08945459127426147,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48421981-C-T',\n",
       "      'distance': 0.08955693244934082,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48505102-C-T',\n",
       "      'distance': 0.08956398069858551,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430748-C-T',\n",
       "      'distance': 0.09001344442367554,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48610808-C-T',\n",
       "      'distance': 0.09002786874771118,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48446753-C-T',\n",
       "      'distance': 0.09011851996183395,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48448860-C-T',\n",
       "      'distance': 0.09016552567481995,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48446814-C-T',\n",
       "      'distance': 0.09029452502727509,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48492578-C-T',\n",
       "      'distance': 0.09036803245544434,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48468504-C-T',\n",
       "      'distance': 0.09039212018251419,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48448851-C-T',\n",
       "      'distance': 0.09041400253772736,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48492505-C-T',\n",
       "      'distance': 0.09047219157218933,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452567-C-T',\n",
       "      'distance': 0.09049661457538605,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48430790-C-T',\n",
       "      'distance': 0.0905425027012825,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48470726-C-T',\n",
       "      'distance': 0.09054651856422424,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48488160-C-T',\n",
       "      'distance': 0.0906239002943039,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'likely_pathogenic',\n",
       "      'confidence_score': 0.8},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 0.9},\n",
       "     '15': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.9333333333333333},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 0.95}}},\n",
       "   {'variant_id': '15-48411007-G-A',\n",
       "    'metadata': {'vcf_string': '15\\t48411007\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "     'chromosome': '15',\n",
       "     'position': '48411007',\n",
       "     'ref_allele': 'G',\n",
       "     'alt_allele': 'A',\n",
       "     'gene_symbol': 'FBN1',\n",
       "     'hgvs_coding': 'NM_000138.5:c.8599C>T',\n",
       "     'hgvs_genomic_38': 'NC_000015.10:g.48411007G>A',\n",
       "     'hgvs_protein': 'p.Gln2867Ter',\n",
       "     'most_severe_consequence': 'stop_gained'},\n",
       "    'annotation_raw': '{\"strand\": 1, \"assembly_name\": \"GRCh38\", \"allele_string\": \"G/A\", \"vcf_string\": \"15-48411007-G-A\", \"input\": \"NC_000015.10:g.48411007G>A\", \"seq_region_name\": \"15\", \"minimised\": 1, \"colocated_variants\": [{\"strand\": 1, \"allele_string\": \"HGMD_MUTATION\", \"seq_region_name\": \"15\", \"start\": 48411007, \"end\": 48411007, \"id\": \"CM098621\", \"phenotype_or_disease\": 1}, {\"allele_string\": \"G/A\", \"strand\": 1, \"seq_region_name\": \"15\", \"start\": 48411007, \"end\": 48411007, \"id\": \"rs1555393486\"}], \"variant_class\": \"SNV\", \"id\": \"NC_000015.10:g.48411007G>A\", \"start\": 48411007, \"end\": 48411007, \"most_severe_consequence\": \"stop_gained\", \"transcript_consequences\": [{\"lof_info\": \"PERCENTILE:0.998026926648097,GERP_DIST:40.8479999899864,BP_DIST:17,DIST_FROM_LAST_EXON:-372,50_BP_RULE:FAIL,PHYLOCSF_TOO_SHORT\", \"variant_allele\": \"A\", \"gene_id\": \"2200\", \"gene_symbol\": \"FBN1\", \"gene_symbol_source\": \"EntrezGene\", \"mastermind_counts\": \"2|3|3\", \"cadd_phred\": 42, \"codons\": \"Cag/Tag\", \"canonical\": 1, \"hgvsc\": \"NM_000138.5:c.8599C>T\", \"biotype\": \"protein_coding\", \"exon\": \"66/66\", \"aa\": \"G\", \"spliceai\": {\"DS_DL\": 0, \"DP_AG\": -30, \"SYMBOL\": \"FBN1\", \"DP_DL\": -2, \"DS_AL\": 0, \"DS_DG\": 0, \"DS_AG\": 0, \"DP_AL\": -2, \"DP_DG\": 24}, \"lof\": \"HC\", \"location\": \"15:48411007\", \"consequence_terms\": [\"stop_gained\"], \"allele_num\": 1, \"cadd_raw\": 9.56318, \"given_ref\": \"G\", \"ga4gh_vrs\": {\"location\": {\"type\": \"SequenceLocation\", \"interval\": {\"start\": 48411006, \"end\": 48411007, \"type\": \"SimpleInterval\"}, \"sequence_id\": \"refseq:NC_000015.10\"}, \"state\": {\"sequence\": \"A\", \"type\": \"SequenceState\"}, \"type\": \"Allele\"}, \"hgvsp\": \"NP_000129.3:p.Gln2867Ter\", \"amino_acids\": \"Q/*\", \"ptriplo\": 0.984699554922396, \"protein_id\": \"NP_000129.3\", \"impact\": \"HIGH\", \"cds_start\": 8599, \"nmd\": \"NMD_escaping_variant\", \"strand\": -1, \"mutationtaster_pred\": \"D\", \"protein_start\": 2867, \"cds_end\": 8599, \"mane_select\": \"ENST00000316623.10\", \"used_ref\": \"G\", \"loeuf\": 0.105, \"conservation\": 5.35, \"phaplo\": 0.99897615427687, \"protein_end\": 2867, \"mane\": [\"MANE_Select\"], \"mastermind_mmid3\": \"FBN1:Q2867X\", \"cdna_end\": 8915, \"cdna_start\": 8915, \"lrt_pred\": \"invalid_field\", \"transcript_id\": \"NM_000138.5\"}]}',\n",
       "    'coordinates': [{'pca_x': 0.016711562379454564,\n",
       "      'pca_y': 0.09381009438485854},\n",
       "     {'t-sne_x': 17.828451379645337, 't-sne_y': 46.30368542869221},\n",
       "     {'umap_x': -9.068187713623047, 'umap_y': 5.936969757080078}],\n",
       "    'nearest_training_variants': [{'variant_id': '15-48415675-G-A',\n",
       "      'distance': 0.0006689414149150252,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425423-G-A',\n",
       "      'distance': 0.0007177521474659443,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48534170-G-A',\n",
       "      'distance': 0.0010818240698426962,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48472632-G-A',\n",
       "      'distance': 0.0011409311555325985,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48495237-G-A',\n",
       "      'distance': 0.00115022377576679,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48485416-G-A',\n",
       "      'distance': 0.0011568842455744743,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48415540-G-A',\n",
       "      'distance': 0.0011686822399497032,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411118-G-A',\n",
       "      'distance': 0.0014309320831671357,\n",
       "      'pathogenicity': 'likely_pathogenic'},\n",
       "     {'variant_id': '15-48487129-G-A',\n",
       "      'distance': 0.00157882459461689,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48427630-G-A',\n",
       "      'distance': 0.0016019013710319996,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48513615-G-A',\n",
       "      'distance': 0.0016037283930927515,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48465622-G-A',\n",
       "      'distance': 0.0016678048996254802,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48452670-G-A',\n",
       "      'distance': 0.001697299536317587,\n",
       "      'pathogenicity': 'pathogenic_or_likely'},\n",
       "     {'variant_id': '15-48420705-G-A',\n",
       "      'distance': 0.00176932942122221,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48411280-G-A',\n",
       "      'distance': 0.00178077956661582,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48600142-G-A',\n",
       "      'distance': 0.0019319376442581415,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48488197-G-A',\n",
       "      'distance': 0.001945936237461865,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48425829-G-A',\n",
       "      'distance': 0.0019564565736800432,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48468064-G-A',\n",
       "      'distance': 0.0020032497122883797,\n",
       "      'pathogenicity': 'pathogenic'},\n",
       "     {'variant_id': '15-48412715-G-A',\n",
       "      'distance': 0.0020974022336304188,\n",
       "      'pathogenicity': 'pathogenic'}],\n",
       "    'prediction_result': {'5': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0},\n",
       "     '10': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '15': {'pred_result': 'potentially_pathogenic', 'confidence_score': 1.0},\n",
       "     '20': {'pred_result': 'potentially_pathogenic',\n",
       "      'confidence_score': 1.0}}}]},\n",
       " 'unsuccessful': {'results_count': 0, 'results': []}}"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pred_result_path = \"data_user/user_query/results/FBN1/all-mpnet-base-v2_vep/prediction_results.json\"\n",
    "pred_result = json.load(open(pred_result_path, 'r'))\n",
    "pred_result"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "4e46d241",
   "metadata": {},
   "outputs": [],
   "source": [
    "pred_result_all = pred_result['successful'].get('results', [])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "83991a24",
   "metadata": {},
   "outputs": [],
   "source": [
    "k = 5\n",
    "for result in pred_result_all:\n",
    "    pred_result = result['prediction_result'][str(k)]['pred_result']\n",
    "    confidence_score = result['prediction_result'][str(k)]['confidence_score']\n",
    "    top_k_result = result['nearest_training_variants'][:k]\n",
    "    top_k_variant_ids = [nv['variant_id'] for nv in top_k_result]\n",
    "    break\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "f6d2e982",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1.0"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "confidence_score"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "ac51d1d3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['15-48610775-C-T',\n",
       " '15-48534131-A-C',\n",
       " '15-48600174-C-T',\n",
       " '15-48600147-C-T',\n",
       " '15-48526207-C-T']"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "top_k_variant_ids"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "924299ad",
   "metadata": {},
   "outputs": [],
   "source": [
    "pred_result"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "63790d96",
   "metadata": {},
   "outputs": [],
   "source": [
    "def parse_prediction_results(json_file_path: str, \n",
    "                             df_all_training_variants: pd.DataFrame,\n",
    "                             k: Optional[int] = None,\n",
    "                             model_name: Optional[str] = None) -> pd.DataFrame:\n",
    "    \"\"\"\n",
    "    Parse prediction_results.json and generate a DataFrame with specified columns.\n",
    "    \n",
    "    Args:\n",
    "        json_file_path: Path to the prediction_results.json file\n",
    "        df_all_training_variants: DataFrame containing all training variants with \n",
    "                                  variant_id and hgvs_genomic_38 columns\n",
    "        k: Optional k value to select specific prediction result and top k neighbors.\n",
    "           If provided, uses prediction_result[str(k)] for pred_result and confidence_score,\n",
    "           and selects top k neighbors from nearest_training_variants.\n",
    "           If None, uses the first available k value or falls back to old structure.\n",
    "        model_name: Optional model name to include in column names (e.g., 'all-mpnet-base-v2').\n",
    "                    If provided, confidence_score column will be named 'confidence_score {model_name}'.\n",
    "        \n",
    "    Returns:\n",
    "        DataFrame with columns: vcf_string, chromosome, position, ref_allele, \n",
    "        alt_allele, gene_symbol, hgvs_coding, hgvs_genomic_38, protein_change, \n",
    "        most_severe_consequence, confidence_score, pred_result, top_similar_variants\n",
    "    \"\"\"\n",
    "    # Read JSON file\n",
    "    with open(json_file_path, 'r') as f:\n",
    "        data = json.load(f)\n",
    "    \n",
    "    # Handle new structure with \"successful\" wrapper\n",
    "    if 'successful' in data:\n",
    "        results = data['successful'].get('results', [])\n",
    "    else:\n",
    "        # Fall back to old structure\n",
    "        results = data.get('results', [])\n",
    "    \n",
    "    if not results:\n",
    "        print(f\"Warning: No results found in {json_file_path}\")\n",
    "        return pd.DataFrame()\n",
    "    \n",
    "    # Create a mapping from variant_id to hgvs_genomic_38 from training variants\n",
    "    # Check if variant_id column exists in df_all_training_variants\n",
    "    if 'variant_id' not in df_all_training_variants.columns:\n",
    "        raise ValueError(\"df_all_training_variants must contain 'variant_id' column\")\n",
    "    if 'hgvs_genomic_38' not in df_all_training_variants.columns:\n",
    "        raise ValueError(\"df_all_training_variants must contain 'hgvs_genomic_38' column\")\n",
    "    \n",
    "    # Create variant_id to hgvs_genomic_38 mapping\n",
    "    variant_id_to_hgvs = dict(zip(\n",
    "        df_all_training_variants['variant_id'], \n",
    "        df_all_training_variants['hgvs_genomic_38']\n",
    "    ))\n",
    "    \n",
    "    # Parse each result\n",
    "    parsed_results = []\n",
    "    for result in results:\n",
    "        metadata = result.get('metadata', {})\n",
    "        \n",
    "        # Extract nearest_training_variants\n",
    "        nearest_variants = result.get('nearest_training_variants', [])\n",
    "        \n",
    "        # If k is specified, select top k neighbors\n",
    "        if k is not None:\n",
    "            nearest_variants = nearest_variants[:k]\n",
    "        \n",
    "        # Extract variant_ids from nearest_training_variants\n",
    "        variant_ids = [nv.get('variant_id') for nv in nearest_variants if 'variant_id' in nv]\n",
    "        \n",
    "        # Map variant_ids to hgvs_genomic_38 annotations\n",
    "        top_similar_variants = [\n",
    "            variant_id_to_hgvs.get(vid) \n",
    "            for vid in variant_ids \n",
    "            if vid in variant_id_to_hgvs and pd.notna(variant_id_to_hgvs.get(vid))\n",
    "        ]\n",
    "        \n",
    "        # Get prediction result and confidence score\n",
    "        # Handle new structure with prediction_result dictionary\n",
    "        prediction_result = result.get('prediction_result', {})\n",
    "        if prediction_result and isinstance(prediction_result, dict):\n",
    "            # If k is specified, use the corresponding k value\n",
    "            if k is not None:\n",
    "                k_str = str(k)\n",
    "                if k_str in prediction_result:\n",
    "                    pred_result = prediction_result[k_str].get('pred_result', '')\n",
    "                    confidence_score = prediction_result[k_str].get('confidence_score', '')\n",
    "                else:\n",
    "                    print(f\"Warning: k={k} not found in prediction_result, using first available\")\n",
    "                    # Use first available k value\n",
    "                    first_k = list(prediction_result.keys())[0] if prediction_result else None\n",
    "                    if first_k:\n",
    "                        pred_result = prediction_result[first_k].get('pred_result', '')\n",
    "                        confidence_score = prediction_result[first_k].get('confidence_score', '')\n",
    "                    else:\n",
    "                        pred_result = ''\n",
    "                        confidence_score = ''\n",
    "            else:\n",
    "                # Use first available k value if k not specified\n",
    "                first_k = list(prediction_result.keys())[0] if prediction_result else None\n",
    "                if first_k:\n",
    "                    pred_result = prediction_result[first_k].get('pred_result', '')\n",
    "                    confidence_score = prediction_result[first_k].get('confidence_score', '')\n",
    "                else:\n",
    "                    pred_result = ''\n",
    "                    confidence_score = ''\n",
    "        else:\n",
    "            # Fall back to old structure (direct fields)\n",
    "            pred_result = result.get('pred_result', '')\n",
    "            confidence_score = result.get('confidence_score', '')\n",
    "        \n",
    "        # Build the row data\n",
    "        # Determine column names based on model_name\n",
    "        confidence_col = f'confidence_score {model_name}' if model_name else 'confidence_score'\n",
    "        pred_result_col = f'pred_result {model_name}' if model_name else 'pred_result'\n",
    "        similar_variants_col = f'top_similar_variants {model_name}' if model_name else 'top_similar_variants'\n",
    "        \n",
    "        row = {\n",
    "            # 'vcf_string': metadata.get('vcf_string', ''),\n",
    "            'chromosome': metadata.get('chromosome', ''),\n",
    "            'position': metadata.get('position', ''),\n",
    "            'ref_allele': metadata.get('ref_allele', ''),\n",
    "            'alt_allele': metadata.get('alt_allele', ''),\n",
    "            'gene_symbol': metadata.get('gene_symbol', ''),\n",
    "            'hgvs_coding': metadata.get('hgvs_coding', ''),\n",
    "            'hgvs_genomic_38': metadata.get('hgvs_genomic_38', ''),\n",
    "            'protein_change': metadata.get('protein_change', ''),\n",
    "            'most_severe_consequence': metadata.get('most_severe_consequence', ''),\n",
    "            confidence_col: confidence_score,\n",
    "            pred_result_col: pred_result,\n",
    "            similar_variants_col: top_similar_variants  # List of hgvs_genomic_38\n",
    "        }\n",
    "        parsed_results.append(row)\n",
    "    \n",
    "    # Create DataFrame\n",
    "    df = pd.DataFrame(parsed_results)\n",
    "    \n",
    "    k_info = f\" (k={k})\" if k is not None else \"\"\n",
    "    print(f\"Successfully parsed {len(df)} prediction results{k_info}\")\n",
    "    print(f\"DataFrame shape: {df.shape}\")\n",
    "    print(f\"Columns: {list(df.columns)}\")\n",
    "    \n",
    "    return df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4a6efa4f",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_predictions = parse_prediction_results(\n",
    "    prediction_results_path, \n",
    "    df_training_variants, \n",
    "    k=k_value,\n",
    "    model_name=model_name\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9cbf4fcf",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9c79bc6a",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "83b35410",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "bfba1f44",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'variant_id': '15-48408466-T-C',\n",
       "  'variant_hash': '25ec24a26ce39b7f',\n",
       "  'vcf_string': '15\\t48408466\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48408466T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*2524A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48408466,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48408832-C-T',\n",
       "  'variant_hash': '3c7a24079bb8b721',\n",
       "  'vcf_string': '15\\t48408832\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48408832C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*2158G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48408832,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48408837-T-C',\n",
       "  'variant_hash': 'e8ba082cac65aa92',\n",
       "  'vcf_string': '15\\t48408837\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48408837T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*2153A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48408837,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48408950-A-G',\n",
       "  'variant_hash': 'cb541f7aa1e4ade8',\n",
       "  'vcf_string': '15\\t48408950\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48408950A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*2040T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48408950,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48409001-G-A',\n",
       "  'variant_hash': '0b888da0f746e600',\n",
       "  'vcf_string': '15\\t48409001\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48409001G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*1989C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48409001,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48409031-C-T',\n",
       "  'variant_hash': '5873a4e607dc63d9',\n",
       "  'vcf_string': '15\\t48409031\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48409031C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*1959G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48409031,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48409415-A-C',\n",
       "  'variant_hash': 'f47a671cc9f9785d',\n",
       "  'vcf_string': '15\\t48409415\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48409415A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*1575T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48409415,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48409428-C-A',\n",
       "  'variant_hash': '37fd237d73fb80af',\n",
       "  'vcf_string': '15\\t48409428\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48409428C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*1562G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48409428,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48409622-T-C',\n",
       "  'variant_hash': '75ab93d320c2dac8',\n",
       "  'vcf_string': '15\\t48409622\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48409622T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*1368A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48409622,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48409762-A-G',\n",
       "  'variant_hash': '234422e16ba5455d',\n",
       "  'vcf_string': '15\\t48409762\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48409762A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*1228T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48409762,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48410123-C-A',\n",
       "  'variant_hash': '434aef223ad368c3',\n",
       "  'vcf_string': '15\\t48410123\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48410123C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*867G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48410123,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48410260-C-A',\n",
       "  'variant_hash': 'c6c8d7fa0ad7209e',\n",
       "  'vcf_string': '15\\t48410260\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48410260C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*730G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48410260,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48410676-G-A',\n",
       "  'variant_hash': 'd45f76a151f903a2',\n",
       "  'vcf_string': '15\\t48410676\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48410676G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*314C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48410676,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48410933-T-C',\n",
       "  'variant_hash': '71346ce7515b639d',\n",
       "  'vcf_string': '15\\t48410933\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48410933T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*57A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48410933,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48410990-T-C',\n",
       "  'variant_hash': '468c8756f6e9a904',\n",
       "  'vcf_string': '15\\t48410990\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48410990T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8616A>G',\n",
       "  'hgvs_protein': 'p.Ter2872=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48410990,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_retained_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411008-G-T',\n",
       "  'variant_hash': '3697072459541090',\n",
       "  'vcf_string': '15\\t48411008\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411008G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8598C>A',\n",
       "  'hgvs_protein': 'p.Ile2866=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411008,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411020-C-T',\n",
       "  'variant_hash': '6811f606a66fb1aa',\n",
       "  'vcf_string': '15\\t48411020\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411020C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8586G>A',\n",
       "  'hgvs_protein': 'p.Leu2862=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411020,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411059-A-G',\n",
       "  'variant_hash': 'd126baaf9176a2ad',\n",
       "  'vcf_string': '15\\t48411059\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411059A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8547T>C',\n",
       "  'hgvs_protein': 'p.Tyr2849=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411059,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411064-T-G',\n",
       "  'variant_hash': 'ca0075f13e2068d9',\n",
       "  'vcf_string': '15\\t48411064\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411064T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8542A>C',\n",
       "  'hgvs_protein': 'p.Lys2848Gln',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411064,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411073-G-A',\n",
       "  'variant_hash': '39c23d87cae62c8c',\n",
       "  'vcf_string': '15\\t48411073\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411073G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8533C>T',\n",
       "  'hgvs_protein': 'p.Leu2845=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411073,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411085-C-A',\n",
       "  'variant_hash': 'e9418960be23625e',\n",
       "  'vcf_string': '15\\t48411085\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411085C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8521G>T',\n",
       "  'hgvs_protein': 'p.Glu2841Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411085,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48411110-A-G',\n",
       "  'variant_hash': '4b47112b9ed166f9',\n",
       "  'vcf_string': '15\\t48411110\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411110A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8496T>C',\n",
       "  'hgvs_protein': 'p.Ser2832=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411110,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411118-G-A',\n",
       "  'variant_hash': '6534897248c9a721',\n",
       "  'vcf_string': '15\\t48411118\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411118G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8488C>T',\n",
       "  'hgvs_protein': 'p.Gln2830Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411118,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48411128-G-C',\n",
       "  'variant_hash': '5df890ecf7ac2f9b',\n",
       "  'vcf_string': '15\\t48411128\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411128G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8478C>G',\n",
       "  'hgvs_protein': 'p.Thr2826=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411128,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411140-T-C',\n",
       "  'variant_hash': '9d8f0266cfd0571b',\n",
       "  'vcf_string': '15\\t48411140\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411140T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8466A>G',\n",
       "  'hgvs_protein': 'p.Pro2822=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411140,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411161-G-A',\n",
       "  'variant_hash': '624da23bbb220e28',\n",
       "  'vcf_string': '15\\t48411161\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411161G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8445C>T',\n",
       "  'hgvs_protein': 'p.Leu2815=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411161,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48411173-C-T',\n",
       "  'variant_hash': '2d1787e6493020d2',\n",
       "  'vcf_string': '15\\t48411173\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411173C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8433G>A',\n",
       "  'hgvs_protein': 'p.Gly2811=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411173,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411221-G-A',\n",
       "  'variant_hash': '25653480a14aa233',\n",
       "  'vcf_string': '15\\t48411221\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411221G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8385C>T',\n",
       "  'hgvs_protein': 'p.Ile2795=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411221,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48411228-T-C',\n",
       "  'variant_hash': '79a8ddabf39b394e',\n",
       "  'vcf_string': '15\\t48411228\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411228T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8378A>G',\n",
       "  'hgvs_protein': 'p.Tyr2793Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411228,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48411243-G-A',\n",
       "  'variant_hash': 'becd876b9ad3c083',\n",
       "  'vcf_string': '15\\t48411243\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411243G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8363C>T',\n",
       "  'hgvs_protein': 'p.Thr2788Met',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411243,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48411245-C-G',\n",
       "  'variant_hash': '39eab76887d2447b',\n",
       "  'vcf_string': '15\\t48411245\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411245C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8361G>C',\n",
       "  'hgvs_protein': 'p.Leu2787=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411245,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411257-A-G',\n",
       "  'variant_hash': '3350af1939046bb5',\n",
       "  'vcf_string': '15\\t48411257\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411257A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8349T>C',\n",
       "  'hgvs_protein': 'p.Ala2783=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411257,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411274-G-A',\n",
       "  'variant_hash': '98d7771226d49a30',\n",
       "  'vcf_string': '15\\t48411274\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411274G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8332C>T',\n",
       "  'hgvs_protein': 'p.Leu2778=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411274,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411280-G-A',\n",
       "  'variant_hash': '993c84e847fe446a',\n",
       "  'vcf_string': '15\\t48411280\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411280G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8326C>T',\n",
       "  'hgvs_protein': 'p.Arg2776Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411280,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48411311-A-T',\n",
       "  'variant_hash': '081cee7bacd3a990',\n",
       "  'vcf_string': '15\\t48411311\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411311A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8295T>A',\n",
       "  'hgvs_protein': 'p.Ala2765=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411311,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411323-T-A',\n",
       "  'variant_hash': 'a3bd60eb498e51c3',\n",
       "  'vcf_string': '15\\t48411323\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411323T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8283A>T',\n",
       "  'hgvs_protein': 'p.Thr2761=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411323,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48411338-C-T',\n",
       "  'variant_hash': 'aae9bd56ec0aac23',\n",
       "  'vcf_string': '15\\t48411338\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411338C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8268G>A',\n",
       "  'hgvs_protein': 'p.Trp2756Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411338,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48411350-A-G',\n",
       "  'variant_hash': '323b90011dc0028e',\n",
       "  'vcf_string': '15\\t48411350\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411350A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8256T>C',\n",
       "  'hgvs_protein': 'p.Ser2752=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411350,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411365-T-C',\n",
       "  'variant_hash': '1ffab58b6dcd7d41',\n",
       "  'vcf_string': '15\\t48411365\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411365T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8241A>G',\n",
       "  'hgvs_protein': 'p.Thr2747=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411365,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411374-C-T',\n",
       "  'variant_hash': '8168324abd30d4c7',\n",
       "  'vcf_string': '15\\t48411374\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411374C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8232G>A',\n",
       "  'hgvs_protein': 'p.Gln2744=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411374,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48411383-G-A',\n",
       "  'variant_hash': '3528213345cabe79',\n",
       "  'vcf_string': '15\\t48411383\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411383G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8227-4C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411383,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48411389-C-T',\n",
       "  'variant_hash': 'a610f40a1a34f14d',\n",
       "  'vcf_string': '15\\t48411389\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411389C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8227-10G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411389,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411482-G-A',\n",
       "  'variant_hash': 'e0cb9a895eacc9eb',\n",
       "  'vcf_string': '15\\t48411482\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411482G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8227-103C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411482,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412564-C-T',\n",
       "  'variant_hash': '9c3c30981a9748af',\n",
       "  'vcf_string': '15\\t48412564\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412564C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8226+5G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412564,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_5th_base_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48412568-C-T',\n",
       "  'variant_hash': '278f0ad4490b691d',\n",
       "  'vcf_string': '15\\t48412568\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412568C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8226+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412568,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48412596-T-A',\n",
       "  'variant_hash': 'd684d505585043f7',\n",
       "  'vcf_string': '15\\t48412596\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412596T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8199A>T',\n",
       "  'hgvs_protein': 'p.Thr2733=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412596,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412626-G-A',\n",
       "  'variant_hash': 'c90225f8e46767d8',\n",
       "  'vcf_string': '15\\t48412626\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412626G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8169C>T',\n",
       "  'hgvs_protein': 'p.Tyr2723=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412626,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412638-C-T',\n",
       "  'variant_hash': '9269268d1ddd43ea',\n",
       "  'vcf_string': '15\\t48412638\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412638C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8157G>A',\n",
       "  'hgvs_protein': 'p.Lys2719=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412638,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412647-G-C',\n",
       "  'variant_hash': 'bcec1a5d5140ff28',\n",
       "  'vcf_string': '15\\t48412647\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412647G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8148C>G',\n",
       "  'hgvs_protein': 'p.Tyr2716Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412647,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48412647-G-A',\n",
       "  'variant_hash': 'db2ac6fe2b1e8b7b',\n",
       "  'vcf_string': '15\\t48412647\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412647G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8148C>T',\n",
       "  'hgvs_protein': 'p.Tyr2716=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412647,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412665-G-A',\n",
       "  'variant_hash': '6364157ce29f55d6',\n",
       "  'vcf_string': '15\\t48412665\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412665G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8130C>T',\n",
       "  'hgvs_protein': 'p.Leu2710=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412665,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412674-G-A',\n",
       "  'variant_hash': 'f865b60683f48ca0',\n",
       "  'vcf_string': '15\\t48412674\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412674G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8121C>T',\n",
       "  'hgvs_protein': 'p.Asp2707=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412674,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412692-G-A',\n",
       "  'variant_hash': 'b56608d079ecd001',\n",
       "  'vcf_string': '15\\t48412692\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412692G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8103C>T',\n",
       "  'hgvs_protein': 'p.Val2701=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412692,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412713-T-G',\n",
       "  'variant_hash': '7fd52c7d190202a2',\n",
       "  'vcf_string': '15\\t48412713\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412713T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8082A>C',\n",
       "  'hgvs_protein': 'p.Arg2694=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412713,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412715-G-A',\n",
       "  'variant_hash': '781ff1571689f69e',\n",
       "  'vcf_string': '15\\t48412715\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412715G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8080C>T',\n",
       "  'hgvs_protein': 'p.Arg2694Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412715,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48412740-G-A',\n",
       "  'variant_hash': '54e3376754516310',\n",
       "  'vcf_string': '15\\t48412740\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412740G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8055C>T',\n",
       "  'hgvs_protein': 'p.His2685=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412740,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415516-C-A',\n",
       "  'variant_hash': 'd202f6ea392b33e3',\n",
       "  'vcf_string': '15\\t48415516\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415516C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8051+20G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415516,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415529-T-C',\n",
       "  'variant_hash': '43ddbcc860f654f5',\n",
       "  'vcf_string': '15\\t48415529\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415529T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8051+7A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415529,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415538-T-C',\n",
       "  'variant_hash': '0441ea9854248e45',\n",
       "  'vcf_string': '15\\t48415538\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415538T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8049A>G',\n",
       "  'hgvs_protein': 'p.Gln2683=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415538,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415540-G-A',\n",
       "  'variant_hash': 'c754dfd9fd49f104',\n",
       "  'vcf_string': '15\\t48415540\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415540G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8047C>T',\n",
       "  'hgvs_protein': 'p.Gln2683Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415540,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48415545-A-C',\n",
       "  'variant_hash': '7a1e7f5c72cac5a8',\n",
       "  'vcf_string': '15\\t48415545\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415545A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8042T>G',\n",
       "  'hgvs_protein': 'p.Ile2681Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415545,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48415549-G-A',\n",
       "  'variant_hash': '27dc8db741cb6699',\n",
       "  'vcf_string': '15\\t48415549\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415549G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8038C>T',\n",
       "  'hgvs_protein': 'p.Arg2680Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415549,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415550-G-A',\n",
       "  'variant_hash': '9ba6378e5e82f7d6',\n",
       "  'vcf_string': '15\\t48415550\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415550G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8037C>T',\n",
       "  'hgvs_protein': 'p.Phe2679=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415550,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415553-G-A',\n",
       "  'variant_hash': '83bb80360d778a16',\n",
       "  'vcf_string': '15\\t48415553\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415553G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8034C>T',\n",
       "  'hgvs_protein': 'p.Tyr2678=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415553,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415566-C-T',\n",
       "  'variant_hash': '36e595d48f055eb0',\n",
       "  'vcf_string': '15\\t48415566\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415566C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8021G>A',\n",
       "  'hgvs_protein': 'p.Cys2674Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415566,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415567-A-G',\n",
       "  'variant_hash': '44b1a6379c9a90df',\n",
       "  'vcf_string': '15\\t48415567\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415567A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8020T>C',\n",
       "  'hgvs_protein': 'p.Cys2674Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415567,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415573-A-C',\n",
       "  'variant_hash': 'c04a6023a48961f5',\n",
       "  'vcf_string': '15\\t48415573\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415573A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8014T>G',\n",
       "  'hgvs_protein': 'p.Cys2672Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415573,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48415581-C-A',\n",
       "  'variant_hash': 'd5c12cdc3fc9da9b',\n",
       "  'vcf_string': '15\\t48415581\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415581C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8006G>T',\n",
       "  'hgvs_protein': 'p.Gly2669Val',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415581,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415582-C-A',\n",
       "  'variant_hash': 'affa4d7740e22530',\n",
       "  'vcf_string': '15\\t48415582\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415582C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8005G>T',\n",
       "  'hgvs_protein': 'p.Gly2669Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415582,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415583-G-A',\n",
       "  'variant_hash': '84347cd72e17e59c',\n",
       "  'vcf_string': '15\\t48415583\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415583G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8004C>T',\n",
       "  'hgvs_protein': 'p.Gly2668=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415583,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415585-C-A',\n",
       "  'variant_hash': '6ee4307d50f0b1fc',\n",
       "  'vcf_string': '15\\t48415585\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415585C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8002G>T',\n",
       "  'hgvs_protein': 'p.Gly2668Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415585,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48415589-G-A',\n",
       "  'variant_hash': '274f622cfadf0cd8',\n",
       "  'vcf_string': '15\\t48415589\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415589G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7998C>T',\n",
       "  'hgvs_protein': 'p.Thr2666=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415589,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415605-T-C',\n",
       "  'variant_hash': '239a6a713e9fabca',\n",
       "  'vcf_string': '15\\t48415605\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415605T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7982A>G',\n",
       "  'hgvs_protein': 'p.Tyr2661Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415605,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415610-G-T',\n",
       "  'variant_hash': '49eef4aa4e022c19',\n",
       "  'vcf_string': '15\\t48415610\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415610G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7977C>A',\n",
       "  'hgvs_protein': 'p.Cys2659Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415610,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48415619-C-T',\n",
       "  'variant_hash': '552682e8385de420',\n",
       "  'vcf_string': '15\\t48415619\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415619C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7968G>A',\n",
       "  'hgvs_protein': 'p.Gln2656=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415619,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415622-C-T',\n",
       "  'variant_hash': '130943e189812347',\n",
       "  'vcf_string': '15\\t48415622\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415622C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7965G>A',\n",
       "  'hgvs_protein': 'p.Ala2655=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415622,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415632-C-T',\n",
       "  'variant_hash': '18be341e2c8303dd',\n",
       "  'vcf_string': '15\\t48415632\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415632C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7955G>A',\n",
       "  'hgvs_protein': 'p.Cys2652Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415632,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415637-A-G',\n",
       "  'variant_hash': 'e1dcbae09f742563',\n",
       "  'vcf_string': '15\\t48415637\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415637A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7950T>C',\n",
       "  'hgvs_protein': 'p.Asn2650=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415637,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48415655-T-A',\n",
       "  'variant_hash': '46703132bc16b7b8',\n",
       "  'vcf_string': '15\\t48415655\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415655T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7932A>T',\n",
       "  'hgvs_protein': 'p.Gly2644=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415655,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415658-A-G',\n",
       "  'variant_hash': 'cfc9345143326fbf',\n",
       "  'vcf_string': '15\\t48415658\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415658A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7929T>C',\n",
       "  'hgvs_protein': 'p.Ser2643=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415658,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415671-T-C',\n",
       "  'variant_hash': 'ecdf5756d2fbd932',\n",
       "  'vcf_string': '15\\t48415671\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415671T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7916A>G',\n",
       "  'hgvs_protein': 'p.Tyr2639Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415671,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415675-G-A',\n",
       "  'variant_hash': '65f007b83fdc5c29',\n",
       "  'vcf_string': '15\\t48415675\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415675G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7912C>T',\n",
       "  'hgvs_protein': 'p.Gln2638Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415675,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48415682-G-A',\n",
       "  'variant_hash': 'edc6e3efc1d6123f',\n",
       "  'vcf_string': '15\\t48415682\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415682G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7905C>T',\n",
       "  'hgvs_protein': 'p.Ala2635=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415682,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415684-C-T',\n",
       "  'variant_hash': 'a8c92ae828334745',\n",
       "  'vcf_string': '15\\t48415684\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415684C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7903G>A',\n",
       "  'hgvs_protein': 'p.Ala2635Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415684,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48415690-A-G',\n",
       "  'variant_hash': '578628593cfc8743',\n",
       "  'vcf_string': '15\\t48415690\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415690A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7897T>C',\n",
       "  'hgvs_protein': 'p.Cys2633Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415690,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415694-G-A',\n",
       "  'variant_hash': 'd350af886e66e070',\n",
       "  'vcf_string': '15\\t48415694\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415694G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7893C>T',\n",
       "  'hgvs_protein': 'p.Cys2631=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415694,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415695-C-T',\n",
       "  'variant_hash': 'd660f76476200c8c',\n",
       "  'vcf_string': '15\\t48415695\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415695C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7892G>A',\n",
       "  'hgvs_protein': 'p.Cys2631Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415695,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48415695-C-A',\n",
       "  'variant_hash': 'dc78ccffc830753b',\n",
       "  'vcf_string': '15\\t48415695\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415695C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7892G>T',\n",
       "  'hgvs_protein': 'p.Cys2631Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415695,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415701-T-C',\n",
       "  'variant_hash': 'ad4762b0a0d479c9',\n",
       "  'vcf_string': '15\\t48415701\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415701T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7886A>G',\n",
       "  'hgvs_protein': 'p.Tyr2629Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415701,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415706-C-T',\n",
       "  'variant_hash': 'd3c1be8eef67e694',\n",
       "  'vcf_string': '15\\t48415706\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415706C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7881G>A',\n",
       "  'hgvs_protein': 'p.Gly2627=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415706,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415707-C-T',\n",
       "  'variant_hash': 'fd4dc6ece4804b20',\n",
       "  'vcf_string': '15\\t48415707\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415707C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7880G>A',\n",
       "  'hgvs_protein': 'p.Gly2627Glu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415707,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48415708-C-T',\n",
       "  'variant_hash': '35e5cf4d77428252',\n",
       "  'vcf_string': '15\\t48415708\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415708C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7879G>A',\n",
       "  'hgvs_protein': 'p.Gly2627Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415708,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415708-C-G',\n",
       "  'variant_hash': 'fb5747d3e9bfbe9e',\n",
       "  'vcf_string': '15\\t48415708\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415708C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7879G>C',\n",
       "  'hgvs_protein': 'p.Gly2627Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415708,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48415712-G-A',\n",
       "  'variant_hash': '861a1fae03d9106c',\n",
       "  'vcf_string': '15\\t48415712\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415712G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7875C>T',\n",
       "  'hgvs_protein': 'p.Thr2625=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415712,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415715-G-A',\n",
       "  'variant_hash': 'cd2b1df5c2dcc4f4',\n",
       "  'vcf_string': '15\\t48415715\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415715G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7872C>T',\n",
       "  'hgvs_protein': 'p.Asn2624=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415715,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415716-T-C',\n",
       "  'variant_hash': 'c8781651fe8f4806',\n",
       "  'vcf_string': '15\\t48415716\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415716T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7871A>G',\n",
       "  'hgvs_protein': 'p.Asn2624Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415716,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415722-C-G',\n",
       "  'variant_hash': '1cf889f31f97d8f9',\n",
       "  'vcf_string': '15\\t48415722\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415722C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7865G>C',\n",
       "  'hgvs_protein': 'p.Cys2622Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415722,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415723-A-G',\n",
       "  'variant_hash': 'f55acc581bf834fc',\n",
       "  'vcf_string': '15\\t48415723\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415723A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7864T>C',\n",
       "  'hgvs_protein': 'p.Cys2622Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415723,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48415736-G-A',\n",
       "  'variant_hash': '66e9516d68eb1578',\n",
       "  'vcf_string': '15\\t48415736\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415736G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7851C>T',\n",
       "  'hgvs_protein': 'p.Cys2617=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415736,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415745-A-T',\n",
       "  'variant_hash': '48b07305a0a9d835',\n",
       "  'vcf_string': '15\\t48415745\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415745A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7842T>A',\n",
       "  'hgvs_protein': 'p.Ala2614=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415745,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415745-A-C',\n",
       "  'variant_hash': '7e1df03f208df6db',\n",
       "  'vcf_string': '15\\t48415745\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415745A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7842T>G',\n",
       "  'hgvs_protein': 'p.Ala2614=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415745,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415748-G-A',\n",
       "  'variant_hash': 'e605c27fd412432f',\n",
       "  'vcf_string': '15\\t48415748\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415748G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7839C>T',\n",
       "  'hgvs_protein': 'p.Ser2613=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415748,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415751-G-A',\n",
       "  'variant_hash': '73be9109574d7afe',\n",
       "  'vcf_string': '15\\t48415751\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415751G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7836C>T',\n",
       "  'hgvs_protein': 'p.Leu2612=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415751,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415755-C-T',\n",
       "  'variant_hash': '664f512ae4e3cfda',\n",
       "  'vcf_string': '15\\t48415755\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415755C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7832G>A',\n",
       "  'hgvs_protein': 'p.Cys2611Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415755,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415760-G-A',\n",
       "  'variant_hash': '3a545661008fd1ba',\n",
       "  'vcf_string': '15\\t48415760\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415760G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7827C>T',\n",
       "  'hgvs_protein': 'p.Asn2609=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415760,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415765-C-A',\n",
       "  'variant_hash': 'af5c755c1cb95aed',\n",
       "  'vcf_string': '15\\t48415765\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415765C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7822G>T',\n",
       "  'hgvs_protein': 'p.Glu2608Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415765,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420545-C-T',\n",
       "  'variant_hash': '171a7ad55ff37a90',\n",
       "  'vcf_string': '15\\t48420545\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420545C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7819+142G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420545,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420547-C-T',\n",
       "  'variant_hash': 'c443e77e9c136832',\n",
       "  'vcf_string': '15\\t48420547\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420547C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7819+140G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420547,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420679-T-G',\n",
       "  'variant_hash': 'c8f5d695f47ce447',\n",
       "  'vcf_string': '15\\t48420679\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420679T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7819+8A>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420679,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48420686-C-T',\n",
       "  'variant_hash': '6f78ccb3a5bf6a85',\n",
       "  'vcf_string': '15\\t48420686\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420686C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7819+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420686,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420687-C-T',\n",
       "  'variant_hash': 'a64370cabe0bad93',\n",
       "  'vcf_string': '15\\t48420687\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420687C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7819G>A',\n",
       "  'hgvs_protein': 'p.Asp2607Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420687,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48420700-C-T',\n",
       "  'variant_hash': '944a2672daaabe4e',\n",
       "  'vcf_string': '15\\t48420700\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420700C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7806G>A',\n",
       "  'hgvs_protein': 'p.Trp2602Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420700,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420705-G-A',\n",
       "  'variant_hash': 'e0cd04128b6b0ff7',\n",
       "  'vcf_string': '15\\t48420705\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420705G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7801C>T',\n",
       "  'hgvs_protein': 'p.Gln2601Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420705,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420706-G-C',\n",
       "  'variant_hash': 'd22220199a31802c',\n",
       "  'vcf_string': '15\\t48420706\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420706G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7800C>G',\n",
       "  'hgvs_protein': 'p.Tyr2600Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420706,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420714-G-A',\n",
       "  'variant_hash': 'c765d2fc11795cca',\n",
       "  'vcf_string': '15\\t48420714\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420714G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7792C>T',\n",
       "  'hgvs_protein': 'p.Gln2598Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420714,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420718-G-T',\n",
       "  'variant_hash': 'dadc019010006a43',\n",
       "  'vcf_string': '15\\t48420718\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420718G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7788C>A',\n",
       "  'hgvs_protein': 'p.Tyr2596Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420718,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420718-G-A',\n",
       "  'variant_hash': '19ec0657906b294a',\n",
       "  'vcf_string': '15\\t48420718\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420718G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7788C>T',\n",
       "  'hgvs_protein': 'p.Tyr2596=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420718,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420719-T-C',\n",
       "  'variant_hash': '7962c506ada0f091',\n",
       "  'vcf_string': '15\\t48420719\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420719T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7787A>G',\n",
       "  'hgvs_protein': 'p.Tyr2596Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420719,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420731-C-T',\n",
       "  'variant_hash': 'a667eceb65040f56',\n",
       "  'vcf_string': '15\\t48420731\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420731C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7775G>A',\n",
       "  'hgvs_protein': 'p.Cys2592Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420731,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420731-C-A',\n",
       "  'variant_hash': 'e83ce1f43523866a',\n",
       "  'vcf_string': '15\\t48420731\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420731C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7775G>T',\n",
       "  'hgvs_protein': 'p.Cys2592Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420731,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420733-G-A',\n",
       "  'variant_hash': '4d5b4aaf774979da',\n",
       "  'vcf_string': '15\\t48420733\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420733G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7773C>T',\n",
       "  'hgvs_protein': 'p.Ser2591=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420733,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420736-G-C',\n",
       "  'variant_hash': 'eee965a4e34307ab',\n",
       "  'vcf_string': '15\\t48420736\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420736G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7770C>G',\n",
       "  'hgvs_protein': 'p.Cys2590Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420736,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48420737-C-T',\n",
       "  'variant_hash': '1b638d7ca57d632d',\n",
       "  'vcf_string': '15\\t48420737\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420737C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7769G>A',\n",
       "  'hgvs_protein': 'p.Cys2590Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420737,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420752-A-G',\n",
       "  'variant_hash': 'd9a98c08813c11bd',\n",
       "  'vcf_string': '15\\t48420752\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420752A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7754T>C',\n",
       "  'hgvs_protein': 'p.Ile2585Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420752,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420757-G-A',\n",
       "  'variant_hash': '00a8cdabce2bb538',\n",
       "  'vcf_string': '15\\t48420757\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420757G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7749C>T',\n",
       "  'hgvs_protein': 'p.Asn2583=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420757,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48420763-G-A',\n",
       "  'variant_hash': '7de6776d3ebebfd8',\n",
       "  'vcf_string': '15\\t48420763\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420763G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7743C>T',\n",
       "  'hgvs_protein': 'p.Cys2581=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420763,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48420778-G-C',\n",
       "  'variant_hash': '211ee5755026a11b',\n",
       "  'vcf_string': '15\\t48420778\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420778G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7728C>G',\n",
       "  'hgvs_protein': 'p.Arg2576=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420778,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420778-G-A',\n",
       "  'variant_hash': 'c2a002478ee20aad',\n",
       "  'vcf_string': '15\\t48420778\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420778G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7728C>T',\n",
       "  'hgvs_protein': 'p.Arg2576=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420778,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420780-G-A',\n",
       "  'variant_hash': '84ac46dda0dfed69',\n",
       "  'vcf_string': '15\\t48420780\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420780G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7726C>T',\n",
       "  'hgvs_protein': 'p.Arg2576Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420780,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420781-G-A',\n",
       "  'variant_hash': '2986ff5d2b5b3824',\n",
       "  'vcf_string': '15\\t48420781\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420781G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7725C>T',\n",
       "  'hgvs_protein': 'p.His2575=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420781,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420794-C-T',\n",
       "  'variant_hash': 'ada190ebb2ae9fcf',\n",
       "  'vcf_string': '15\\t48420794\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420794C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7712G>A',\n",
       "  'hgvs_protein': 'p.Cys2571Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420794,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420795-A-G',\n",
       "  'variant_hash': '115e34e3f4c66e70',\n",
       "  'vcf_string': '15\\t48420795\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420795A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7711T>C',\n",
       "  'hgvs_protein': 'p.Cys2571Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420795,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420796-C-T',\n",
       "  'variant_hash': 'fec31f4c66b89658',\n",
       "  'vcf_string': '15\\t48420796\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420796C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7710G>A',\n",
       "  'hgvs_protein': 'p.Glu2570=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420796,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420798-C-T',\n",
       "  'variant_hash': '0c819141140561d1',\n",
       "  'vcf_string': '15\\t48420798\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420798C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7708G>A',\n",
       "  'hgvs_protein': 'p.Glu2570Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420798,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420799-G-A',\n",
       "  'variant_hash': '53d8096949b45ca7',\n",
       "  'vcf_string': '15\\t48420799\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420799G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7707C>T',\n",
       "  'hgvs_protein': 'p.Asp2569=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420799,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420815-G-A',\n",
       "  'variant_hash': 'b8184af11e1a064a',\n",
       "  'vcf_string': '15\\t48420815\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420815G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7700-9C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420815,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420819-C-G',\n",
       "  'variant_hash': 'ba223d5b54135f7c',\n",
       "  'vcf_string': '15\\t48420819\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420819C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7700-13G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420819,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421540-G-A',\n",
       "  'variant_hash': '8957fca455cf05d2',\n",
       "  'vcf_string': '15\\t48421540\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421540G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7699+18C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421540,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48421543-G-A',\n",
       "  'variant_hash': '2adc65e43198ae3d',\n",
       "  'vcf_string': '15\\t48421543\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421543G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7699+15C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421543,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421557-C-T',\n",
       "  'variant_hash': '1a015bbcada3d541',\n",
       "  'vcf_string': '15\\t48421557\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421557C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7699+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421557,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421574-G-A',\n",
       "  'variant_hash': '76f171251f50c716',\n",
       "  'vcf_string': '15\\t48421574\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421574G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7683C>T',\n",
       "  'hgvs_protein': 'p.Thr2561=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421574,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421580-A-G',\n",
       "  'variant_hash': '64df07e4b55b055c',\n",
       "  'vcf_string': '15\\t48421580\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421580A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7677T>C',\n",
       "  'hgvs_protein': 'p.Asp2559=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421580,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421593-C-A',\n",
       "  'variant_hash': '2c1d369bcd0b0de2',\n",
       "  'vcf_string': '15\\t48421593\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421593C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7664G>T',\n",
       "  'hgvs_protein': 'p.Gly2555Val',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421593,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421594-C-T',\n",
       "  'variant_hash': '0e5e687741a2e073',\n",
       "  'vcf_string': '15\\t48421594\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421594C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7663G>A',\n",
       "  'hgvs_protein': 'p.Gly2555Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421594,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421601-G-T',\n",
       "  'variant_hash': '17866ede0ac03538',\n",
       "  'vcf_string': '15\\t48421601\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421601G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7656C>A',\n",
       "  'hgvs_protein': 'p.Cys2552Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421601,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421603-A-G',\n",
       "  'variant_hash': '94316ce547240140',\n",
       "  'vcf_string': '15\\t48421603\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421603A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7654T>C',\n",
       "  'hgvs_protein': 'p.Cys2552Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421603,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48421608-C-T',\n",
       "  'variant_hash': 'c82b6fb402724538',\n",
       "  'vcf_string': '15\\t48421608\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421608C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7649G>A',\n",
       "  'hgvs_protein': 'p.Cys2550Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421608,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421609-A-T',\n",
       "  'variant_hash': 'b5f10c5aeca14e29',\n",
       "  'vcf_string': '15\\t48421609\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421609A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7648T>A',\n",
       "  'hgvs_protein': 'p.Cys2550Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421609,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48421625-A-G',\n",
       "  'variant_hash': 'c2669fa552696bb1',\n",
       "  'vcf_string': '15\\t48421625\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421625A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7632T>C',\n",
       "  'hgvs_protein': 'p.Thr2544=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421625,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421651-C-T',\n",
       "  'variant_hash': '6a710c64525ec606',\n",
       "  'vcf_string': '15\\t48421651\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421651C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7606G>A',\n",
       "  'hgvs_protein': 'p.Gly2536Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421651,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421652-G-T',\n",
       "  'variant_hash': '3bb26e494e5731b8',\n",
       "  'vcf_string': '15\\t48421652\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421652G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7605C>A',\n",
       "  'hgvs_protein': 'p.Cys2535Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421652,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421652-G-C',\n",
       "  'variant_hash': '66cebd9d1a480acb',\n",
       "  'vcf_string': '15\\t48421652\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421652G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7605C>G',\n",
       "  'hgvs_protein': 'p.Cys2535Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421652,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421652-G-A',\n",
       "  'variant_hash': '086d4529a282d89f',\n",
       "  'vcf_string': '15\\t48421652\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421652G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7605C>T',\n",
       "  'hgvs_protein': 'p.Cys2535=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421652,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421653-C-T',\n",
       "  'variant_hash': '14ccc2ffdb57f0df',\n",
       "  'vcf_string': '15\\t48421653\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421653C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7604G>A',\n",
       "  'hgvs_protein': 'p.Cys2535Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421653,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421655-C-T',\n",
       "  'variant_hash': '871eda844e519e03',\n",
       "  'vcf_string': '15\\t48421655\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421655C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7602G>A',\n",
       "  'hgvs_protein': 'p.Leu2534=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421655,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421675-A-G',\n",
       "  'variant_hash': '2d48855827740eb0',\n",
       "  'vcf_string': '15\\t48421675\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421675A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7582T>C',\n",
       "  'hgvs_protein': 'p.Cys2528Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421675,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421680-T-C',\n",
       "  'variant_hash': '5b063864f0be3393',\n",
       "  'vcf_string': '15\\t48421680\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421680T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7577A>G',\n",
       "  'hgvs_protein': 'p.Asn2526Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421680,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48421682-G-A',\n",
       "  'variant_hash': 'b60c7f86c9c9d8a1',\n",
       "  'vcf_string': '15\\t48421682\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421682G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7575C>T',\n",
       "  'hgvs_protein': 'p.Asn2525=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421682,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421687-C-T',\n",
       "  'variant_hash': '7c498f95c8743c09',\n",
       "  'vcf_string': '15\\t48421687\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421687C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7571-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421687,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48421762-A-T',\n",
       "  'variant_hash': '4b9748df345814d5',\n",
       "  'vcf_string': '15\\t48421762\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421762A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7571-76T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421762,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48421799-G-T',\n",
       "  'variant_hash': '1c24435950fee4b6',\n",
       "  'vcf_string': '15\\t48421799\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421799G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7571-113C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421799,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48421939-C-T',\n",
       "  'variant_hash': 'c5f408d842e193d8',\n",
       "  'vcf_string': '15\\t48421939\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421939C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7570+13G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421939,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421973-G-A',\n",
       "  'variant_hash': '31644dba23f966fb',\n",
       "  'vcf_string': '15\\t48421973\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421973G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7549C>T',\n",
       "  'hgvs_protein': 'p.Gln2517Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421973,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421981-C-T',\n",
       "  'variant_hash': 'ce97621862880e29',\n",
       "  'vcf_string': '15\\t48421981\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421981C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7541G>A',\n",
       "  'hgvs_protein': 'p.Gly2514Glu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421981,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48421982-C-T',\n",
       "  'variant_hash': '68cd739bbc90c779',\n",
       "  'vcf_string': '15\\t48421982\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421982C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7540G>A',\n",
       "  'hgvs_protein': 'p.Gly2514Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421982,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421982-C-A',\n",
       "  'variant_hash': '4a1b10457c353e7f',\n",
       "  'vcf_string': '15\\t48421982\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421982C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7540G>T',\n",
       "  'hgvs_protein': 'p.Gly2514Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421982,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421983-G-A',\n",
       "  'variant_hash': '9e2dde939447deab',\n",
       "  'vcf_string': '15\\t48421983\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421983G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7539C>T',\n",
       "  'hgvs_protein': 'p.Pro2513=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421983,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421989-A-G',\n",
       "  'variant_hash': 'c181c097a7ae2496',\n",
       "  'vcf_string': '15\\t48421989\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421989A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7533T>C',\n",
       "  'hgvs_protein': 'p.Cys2511=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421989,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421989-A-C',\n",
       "  'variant_hash': '362ee0519c207cab',\n",
       "  'vcf_string': '15\\t48421989\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421989A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7533T>G',\n",
       "  'hgvs_protein': 'p.Cys2511Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421989,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421990-C-T',\n",
       "  'variant_hash': 'af127d42d150e575',\n",
       "  'vcf_string': '15\\t48421990\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421990C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7532G>A',\n",
       "  'hgvs_protein': 'p.Cys2511Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421990,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421991-A-G',\n",
       "  'variant_hash': '52910bd2a4d4debe',\n",
       "  'vcf_string': '15\\t48421991\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421991A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7531T>C',\n",
       "  'hgvs_protein': 'p.Cys2511Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421991,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421992-T-C',\n",
       "  'variant_hash': 'a32de820b534543d',\n",
       "  'vcf_string': '15\\t48421992\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421992T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7530A>G',\n",
       "  'hgvs_protein': 'p.Lys2510=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421992,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421995-G-T',\n",
       "  'variant_hash': '28a145e4ee9ab3d2',\n",
       "  'vcf_string': '15\\t48421995\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421995G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7527C>A',\n",
       "  'hgvs_protein': 'p.Cys2509Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421995,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421997-A-C',\n",
       "  'variant_hash': 'a5ff04f91680ab09',\n",
       "  'vcf_string': '15\\t48421997\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421997A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7525T>G',\n",
       "  'hgvs_protein': 'p.Cys2509Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421997,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48421998-T-C',\n",
       "  'variant_hash': 'cbdfbf3df19760ac',\n",
       "  'vcf_string': '15\\t48421998\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421998T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7524A>G',\n",
       "  'hgvs_protein': 'p.Thr2508=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421998,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421998-T-A',\n",
       "  'variant_hash': '671d042c24093983',\n",
       "  'vcf_string': '15\\t48421998\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421998T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7524A>T',\n",
       "  'hgvs_protein': 'p.Thr2508=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421998,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48422001-G-A',\n",
       "  'variant_hash': '6726883aa1ca40ae',\n",
       "  'vcf_string': '15\\t48422001\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422001G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7521C>T',\n",
       "  'hgvs_protein': 'p.Phe2507=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422001,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48422007-G-A',\n",
       "  'variant_hash': '8ef0c2336574f849',\n",
       "  'vcf_string': '15\\t48422007\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422007G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7515C>T',\n",
       "  'hgvs_protein': 'p.Gly2505=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422007,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48422023-C-T',\n",
       "  'variant_hash': '29c3fd2f3654cc07',\n",
       "  'vcf_string': '15\\t48422023\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422023C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7499G>A',\n",
       "  'hgvs_protein': 'p.Cys2500Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422023,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48422024-A-T',\n",
       "  'variant_hash': '582427b2ad0b4fb6',\n",
       "  'vcf_string': '15\\t48422024\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422024A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7498T>A',\n",
       "  'hgvs_protein': 'p.Cys2500Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422024,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48422024-A-G',\n",
       "  'variant_hash': 'e3180748801d1821',\n",
       "  'vcf_string': '15\\t48422024\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422024A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7498T>C',\n",
       "  'hgvs_protein': 'p.Cys2500Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422024,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48422025-T-C',\n",
       "  'variant_hash': '6b4dd5c24d82e225',\n",
       "  'vcf_string': '15\\t48422025\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422025T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7497A>G',\n",
       "  'hgvs_protein': 'p.Leu2499=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422025,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48422046-C-T',\n",
       "  'variant_hash': '53a50eedf6a1cf33',\n",
       "  'vcf_string': '15\\t48422046\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422046C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7476G>A',\n",
       "  'hgvs_protein': 'p.Lys2492=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422046,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48422068-T-C',\n",
       "  'variant_hash': '21bf32835aa1a28e',\n",
       "  'vcf_string': '15\\t48422068\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422068T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7454A>G',\n",
       "  'hgvs_protein': 'p.Asp2485Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422068,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48422069-C-A',\n",
       "  'variant_hash': '1fe8751e994a5f1f',\n",
       "  'vcf_string': '15\\t48422069\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422069C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7454-1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422069,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48422074-A-G',\n",
       "  'variant_hash': 'cac151c816777307',\n",
       "  'vcf_string': '15\\t48422074\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422074A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7454-6T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422074,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425368-C-T',\n",
       "  'variant_hash': '63404e53d261bd6c',\n",
       "  'vcf_string': '15\\t48425368\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425368C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7453+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425368,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425374-C-T',\n",
       "  'variant_hash': 'bc4a61ad98280d52',\n",
       "  'vcf_string': '15\\t48425374\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425374C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7448G>A',\n",
       "  'hgvs_protein': 'p.Cys2483Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425374,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425375-A-G',\n",
       "  'variant_hash': 'b87786f6021f0334',\n",
       "  'vcf_string': '15\\t48425375\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425375A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7447T>C',\n",
       "  'hgvs_protein': 'p.Cys2483Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425375,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425393-G-A',\n",
       "  'variant_hash': '3b5091f4d0d02f23',\n",
       "  'vcf_string': '15\\t48425393\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425393G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7429C>T',\n",
       "  'hgvs_protein': 'p.Gln2477Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425393,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425394-C-T',\n",
       "  'variant_hash': '12fbb69f9fff6fb6',\n",
       "  'vcf_string': '15\\t48425394\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425394C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7428G>A',\n",
       "  'hgvs_protein': 'p.Leu2476=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425394,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425401-T-C',\n",
       "  'variant_hash': '89e6452f989bf1d2',\n",
       "  'vcf_string': '15\\t48425401\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425401T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7421A>G',\n",
       "  'hgvs_protein': 'p.Tyr2474Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425401,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48425409-C-T',\n",
       "  'variant_hash': '4dfa402f37a0e7db',\n",
       "  'vcf_string': '15\\t48425409\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425409C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7413G>A',\n",
       "  'hgvs_protein': 'p.Pro2471=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425409,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425412-G-C',\n",
       "  'variant_hash': '9a30b38daa3bf731',\n",
       "  'vcf_string': '15\\t48425412\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425412G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7410C>G',\n",
       "  'hgvs_protein': 'p.Cys2470Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425412,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425413-C-A',\n",
       "  'variant_hash': '09b707cd25afd6dd',\n",
       "  'vcf_string': '15\\t48425413\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425413C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7409G>T',\n",
       "  'hgvs_protein': 'p.Cys2470Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425413,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48425414-A-G',\n",
       "  'variant_hash': 'a1ece309f807be3c',\n",
       "  'vcf_string': '15\\t48425414\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425414A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7408T>C',\n",
       "  'hgvs_protein': 'p.Cys2470Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425414,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48425415-T-C',\n",
       "  'variant_hash': '1d5a7151b5f5279a',\n",
       "  'vcf_string': '15\\t48425415\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425415T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7407A>G',\n",
       "  'hgvs_protein': 'p.Ser2469=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425415,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425416-G-T',\n",
       "  'variant_hash': '802cff98f9bcc549',\n",
       "  'vcf_string': '15\\t48425416\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425416G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7406C>A',\n",
       "  'hgvs_protein': 'p.Ser2469Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425416,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425419-C-T',\n",
       "  'variant_hash': 'bf0d94837ee8709c',\n",
       "  'vcf_string': '15\\t48425419\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425419C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7403G>A',\n",
       "  'hgvs_protein': 'p.Cys2468Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425419,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425420-A-G',\n",
       "  'variant_hash': 'a1a335525fbd5c7a',\n",
       "  'vcf_string': '15\\t48425420\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425420A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7402T>C',\n",
       "  'hgvs_protein': 'p.Cys2468Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425420,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425423-G-A',\n",
       "  'variant_hash': '92b3a65459d68cad',\n",
       "  'vcf_string': '15\\t48425423\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425423G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7399C>T',\n",
       "  'hgvs_protein': 'p.Gln2467Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425423,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425424-G-T',\n",
       "  'variant_hash': '4dc27a0230e14d70',\n",
       "  'vcf_string': '15\\t48425424\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425424G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7398C>A',\n",
       "  'hgvs_protein': 'p.Tyr2466Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425424,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425427-A-G',\n",
       "  'variant_hash': 'dece7a1962d77fc2',\n",
       "  'vcf_string': '15\\t48425427\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425427A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7395T>C',\n",
       "  'hgvs_protein': 'p.Ser2465=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425427,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425435-C-A',\n",
       "  'variant_hash': 'de304dfb4d998abe',\n",
       "  'vcf_string': '15\\t48425435\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425435C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7387G>T',\n",
       "  'hgvs_protein': 'p.Glu2463Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425435,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425446-C-T',\n",
       "  'variant_hash': 'dbb9c7b2c98936e5',\n",
       "  'vcf_string': '15\\t48425446\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425446C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7376G>A',\n",
       "  'hgvs_protein': 'p.Cys2459Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425446,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48425458-C-T',\n",
       "  'variant_hash': '35cbf4206cfedb35',\n",
       "  'vcf_string': '15\\t48425458\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425458C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7364G>A',\n",
       "  'hgvs_protein': 'p.Cys2455Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425458,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425474-G-A',\n",
       "  'variant_hash': '8450cf06f349ca43',\n",
       "  'vcf_string': '15\\t48425474\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425474G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7348C>T',\n",
       "  'hgvs_protein': 'p.Gln2450Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425474,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425479-C-T',\n",
       "  'variant_hash': '40e55ee2959dc543',\n",
       "  'vcf_string': '15\\t48425479\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425479C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7343G>A',\n",
       "  'hgvs_protein': 'p.Cys2448Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425479,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425483-C-T',\n",
       "  'variant_hash': '17b26d478df11415',\n",
       "  'vcf_string': '15\\t48425483\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425483C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7339G>A',\n",
       "  'hgvs_protein': 'p.Glu2447Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425483,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425484-G-A',\n",
       "  'variant_hash': '47f10e3b69cbb089',\n",
       "  'vcf_string': '15\\t48425484\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425484G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7338C>T',\n",
       "  'hgvs_protein': 'p.Asn2446=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425484,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425487-C-G',\n",
       "  'variant_hash': '4548431078781190',\n",
       "  'vcf_string': '15\\t48425487\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425487C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7335G>C',\n",
       "  'hgvs_protein': 'p.Leu2445=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425487,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425487-C-A',\n",
       "  'variant_hash': '4d3e96475ee1e590',\n",
       "  'vcf_string': '15\\t48425487\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425487C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7335G>T',\n",
       "  'hgvs_protein': 'p.Leu2445=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425487,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425729-A-C',\n",
       "  'variant_hash': '2e1bda42fd696061',\n",
       "  'vcf_string': '15\\t48425729\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425729A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7330+10T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425729,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425735-T-C',\n",
       "  'variant_hash': '1c60f98b4df33305',\n",
       "  'vcf_string': '15\\t48425735\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425735T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7330+4A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425735,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48425738-C-T',\n",
       "  'variant_hash': 'e6fc8a97cd497f51',\n",
       "  'vcf_string': '15\\t48425738\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425738C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7330+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425738,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425738-C-A',\n",
       "  'variant_hash': 'a394e8f0350cdf5c',\n",
       "  'vcf_string': '15\\t48425738\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425738C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7330+1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425738,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425744-C-T',\n",
       "  'variant_hash': 'ad0917749b2c5543',\n",
       "  'vcf_string': '15\\t48425744\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425744C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7325G>A',\n",
       "  'hgvs_protein': 'p.Cys2442Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425744,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425745-A-T',\n",
       "  'variant_hash': '09af6544166f7a1a',\n",
       "  'vcf_string': '15\\t48425745\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425745A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7324T>A',\n",
       "  'hgvs_protein': 'p.Cys2442Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425745,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425752-C-G',\n",
       "  'variant_hash': 'f352e3d8f79fbf05',\n",
       "  'vcf_string': '15\\t48425752\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425752C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7317G>C',\n",
       "  'hgvs_protein': 'p.Gly2439=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425752,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425764-T-C',\n",
       "  'variant_hash': 'b197f1e249319dfb',\n",
       "  'vcf_string': '15\\t48425764\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425764T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7305A>G',\n",
       "  'hgvs_protein': 'p.Pro2435=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425764,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425770-G-A',\n",
       "  'variant_hash': 'b0915328dc62e6b8',\n",
       "  'vcf_string': '15\\t48425770\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425770G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7299C>T',\n",
       "  'hgvs_protein': 'p.Tyr2433=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425770,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425783-C-A',\n",
       "  'variant_hash': '93e06a137764883e',\n",
       "  'vcf_string': '15\\t48425783\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425783C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7286G>T',\n",
       "  'hgvs_protein': 'p.Cys2429Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425783,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425797-T-C',\n",
       "  'variant_hash': 'd855a644abca457c',\n",
       "  'vcf_string': '15\\t48425797\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425797T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7272A>G',\n",
       "  'hgvs_protein': 'p.Ser2424=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425797,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425802-C-A',\n",
       "  'variant_hash': '10982efaf35ee0a5',\n",
       "  'vcf_string': '15\\t48425802\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425802C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7267G>T',\n",
       "  'hgvs_protein': 'p.Gly2423Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425802,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425803-T-C',\n",
       "  'variant_hash': 'f8402c7de1dbd862',\n",
       "  'vcf_string': '15\\t48425803\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425803T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7266A>G',\n",
       "  'hgvs_protein': 'p.Arg2422=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425803,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425816-C-T',\n",
       "  'variant_hash': 'f831701ca580038c',\n",
       "  'vcf_string': '15\\t48425816\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425816C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7253G>A',\n",
       "  'hgvs_protein': 'p.Cys2418Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425816,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425820-C-A',\n",
       "  'variant_hash': '43878a6cbbbdcc4e',\n",
       "  'vcf_string': '15\\t48425820\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425820C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7249G>T',\n",
       "  'hgvs_protein': 'p.Glu2417Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425820,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425823-C-T',\n",
       "  'variant_hash': '3dca26e0357e45f5',\n",
       "  'vcf_string': '15\\t48425823\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425823C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7246G>A',\n",
       "  'hgvs_protein': 'p.Gly2416Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425823,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425829-G-A',\n",
       "  'variant_hash': '3432f2cbf515d118',\n",
       "  'vcf_string': '15\\t48425829\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425829G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7240C>T',\n",
       "  'hgvs_protein': 'p.Arg2414Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425829,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425830-G-A',\n",
       "  'variant_hash': '133f4450cb009016',\n",
       "  'vcf_string': '15\\t48425830\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425830G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7239C>T',\n",
       "  'hgvs_protein': 'p.Cys2413=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425830,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425833-A-G',\n",
       "  'variant_hash': '23ae98dcfd6abb44',\n",
       "  'vcf_string': '15\\t48425833\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425833A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7236T>C',\n",
       "  'hgvs_protein': 'p.Val2412=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425833,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425839-G-A',\n",
       "  'variant_hash': '7d64646fdc04c2ab',\n",
       "  'vcf_string': '15\\t48425839\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425839G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7230C>T',\n",
       "  'hgvs_protein': 'p.His2410=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425839,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425842-A-T',\n",
       "  'variant_hash': 'ef1f18f4880d3201',\n",
       "  'vcf_string': '15\\t48425842\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425842A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7227T>A',\n",
       "  'hgvs_protein': 'p.Ile2409=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425842,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425852-C-T',\n",
       "  'variant_hash': '8ec6c058bcc8a155',\n",
       "  'vcf_string': '15\\t48425852\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425852C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7217G>A',\n",
       "  'hgvs_protein': 'p.Cys2406Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425852,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425860-G-A',\n",
       "  'variant_hash': 'e58bc65f2ba1328e',\n",
       "  'vcf_string': '15\\t48425860\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425860G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7209C>T',\n",
       "  'hgvs_protein': 'p.Ile2403=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425860,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425865-C-T',\n",
       "  'variant_hash': '65da85ff5c343023',\n",
       "  'vcf_string': '15\\t48425865\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425865C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7205-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425865,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425866-T-C',\n",
       "  'variant_hash': '423ec7c130513151',\n",
       "  'vcf_string': '15\\t48425866\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425866T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7205-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425866,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425876-C-G',\n",
       "  'variant_hash': '57d214ca6804a513',\n",
       "  'vcf_string': '15\\t48425876\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425876C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7205-12G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425876,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425882-T-C',\n",
       "  'variant_hash': 'c925a6f67238c03b',\n",
       "  'vcf_string': '15\\t48425882\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425882T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7205-18A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425882,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427504-G-T',\n",
       "  'variant_hash': '2b293fd12774a614',\n",
       "  'vcf_string': '15\\t48427504\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427504G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7204+63C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427504,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48427560-G-C',\n",
       "  'variant_hash': '0dc4c0aca67494f3',\n",
       "  'vcf_string': '15\\t48427560\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427560G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7204+7C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427560,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48427561-A-C',\n",
       "  'variant_hash': '36c1089f0aa1f05f',\n",
       "  'vcf_string': '15\\t48427561\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427561A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7204+6T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427561,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427566-C-T',\n",
       "  'variant_hash': 'f0225e216a6e7365',\n",
       "  'vcf_string': '15\\t48427566\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427566C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7204+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427566,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48427566-C-A',\n",
       "  'variant_hash': '1395c97ef7a12aa9',\n",
       "  'vcf_string': '15\\t48427566\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427566C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7204+1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427566,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48427591-G-A',\n",
       "  'variant_hash': 'b4c8984f354221b4',\n",
       "  'vcf_string': '15\\t48427591\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427591G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7180C>T',\n",
       "  'hgvs_protein': 'p.Arg2394Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427591,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48427602-C-T',\n",
       "  'variant_hash': 'c0f4c7e6c5c5068f',\n",
       "  'vcf_string': '15\\t48427602\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427602C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7169G>A',\n",
       "  'hgvs_protein': 'p.Cys2390Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427602,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48427603-A-T',\n",
       "  'variant_hash': '5de3b0b810601ea2',\n",
       "  'vcf_string': '15\\t48427603\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427603A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7168T>A',\n",
       "  'hgvs_protein': 'p.Cys2390Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427603,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48427603-A-G',\n",
       "  'variant_hash': 'd04274c9793f502b',\n",
       "  'vcf_string': '15\\t48427603\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427603A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7168T>C',\n",
       "  'hgvs_protein': 'p.Cys2390Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427603,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48427613-G-A',\n",
       "  'variant_hash': '28d1038095439d92',\n",
       "  'vcf_string': '15\\t48427613\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427613G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7158C>T',\n",
       "  'hgvs_protein': 'p.Phe2386=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427613,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427630-G-A',\n",
       "  'variant_hash': '2031e69a7ff803bc',\n",
       "  'vcf_string': '15\\t48427630\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427630G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7141C>T',\n",
       "  'hgvs_protein': 'p.Gln2381Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427630,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48427646-A-T',\n",
       "  'variant_hash': 'b57d5b255c0cb1da',\n",
       "  'vcf_string': '15\\t48427646\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427646A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7125T>A',\n",
       "  'hgvs_protein': 'p.Cys2375Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427646,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48427659-C-T',\n",
       "  'variant_hash': '7b087039763388e1',\n",
       "  'vcf_string': '15\\t48427659\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427659C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7112G>A',\n",
       "  'hgvs_protein': 'p.Trp2371Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427659,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48427673-G-A',\n",
       "  'variant_hash': '7187ad9382fc51f4',\n",
       "  'vcf_string': '15\\t48427673\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427673G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7098C>T',\n",
       "  'hgvs_protein': 'p.Asp2366=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427673,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48427684-A-T',\n",
       "  'variant_hash': 'ef52275a3dfb5ade',\n",
       "  'vcf_string': '15\\t48427684\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427684A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7087T>A',\n",
       "  'hgvs_protein': 'p.Cys2363Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427684,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48427684-A-C',\n",
       "  'variant_hash': '4c423364b25068f5',\n",
       "  'vcf_string': '15\\t48427684\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427684A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7087T>G',\n",
       "  'hgvs_protein': 'p.Cys2363Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427684,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48427688-C-T',\n",
       "  'variant_hash': '21eade2e835c77ee',\n",
       "  'vcf_string': '15\\t48427688\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427688C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7083G>A',\n",
       "  'hgvs_protein': 'p.Ser2361=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427688,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427700-G-A',\n",
       "  'variant_hash': '2eefd8f686d4bfbd',\n",
       "  'vcf_string': '15\\t48427700\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427700G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7071C>T',\n",
       "  'hgvs_protein': 'p.Pro2357=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427700,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427721-G-A',\n",
       "  'variant_hash': 'e0b66cbd6a6e85f7',\n",
       "  'vcf_string': '15\\t48427721\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427721G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7050C>T',\n",
       "  'hgvs_protein': 'p.Ile2350=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427721,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427745-C-T',\n",
       "  'variant_hash': 'ffca1c4d79f74dc1',\n",
       "  'vcf_string': '15\\t48427745\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427745C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7026G>A',\n",
       "  'hgvs_protein': 'p.Glu2342=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427745,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427755-C-T',\n",
       "  'variant_hash': '031013375d390d3c',\n",
       "  'vcf_string': '15\\t48427755\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427755C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7016G>A',\n",
       "  'hgvs_protein': 'p.Cys2339Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427755,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48427756-A-G',\n",
       "  'variant_hash': '8e7a87ae639e2289',\n",
       "  'vcf_string': '15\\t48427756\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427756A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7015T>C',\n",
       "  'hgvs_protein': 'p.Cys2339Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427756,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48427768-G-A',\n",
       "  'variant_hash': '15d8f696d0d65d68',\n",
       "  'vcf_string': '15\\t48427768\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427768G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7003C>T',\n",
       "  'hgvs_protein': 'p.Arg2335Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427768,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48427780-G-A',\n",
       "  'variant_hash': 'eeb98af74792a26d',\n",
       "  'vcf_string': '15\\t48427780\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427780G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6998-7C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427780,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48427787-T-G',\n",
       "  'variant_hash': '4d88c196cc349f07',\n",
       "  'vcf_string': '15\\t48427787\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427787T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6998-14A>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427787,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48428074-C-T',\n",
       "  'variant_hash': 'c18ed16936777432',\n",
       "  'vcf_string': '15\\t48428074\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428074C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6997+272G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428074,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48428327-C-T',\n",
       "  'variant_hash': '5b3f94f2c1896aca',\n",
       "  'vcf_string': '15\\t48428327\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428327C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6997+19G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428327,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48428328-G-A',\n",
       "  'variant_hash': 'b7ddddc8b6bb65e3',\n",
       "  'vcf_string': '15\\t48428328\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428328G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6997+18C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428328,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48428329-G-C',\n",
       "  'variant_hash': '911323c95ffa93d9',\n",
       "  'vcf_string': '15\\t48428329\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428329G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6997+17C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428329,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48428345-C-T',\n",
       "  'variant_hash': 'e96fb64a936b900a',\n",
       "  'vcf_string': '15\\t48428345\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428345C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6997+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428345,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48428355-C-A',\n",
       "  'variant_hash': '101dc01eea0d8266',\n",
       "  'vcf_string': '15\\t48428355\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428355C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6988G>T',\n",
       "  'hgvs_protein': 'p.Glu2330Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428355,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48428356-G-C',\n",
       "  'variant_hash': '00cd1a208e972485',\n",
       "  'vcf_string': '15\\t48428356\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428356G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6987C>G',\n",
       "  'hgvs_protein': 'p.Asp2329Glu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428356,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48428356-G-A',\n",
       "  'variant_hash': 'c62f46eca31331b2',\n",
       "  'vcf_string': '15\\t48428356\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428356G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6987C>T',\n",
       "  'hgvs_protein': 'p.Asp2329=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428356,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48428361-G-A',\n",
       "  'variant_hash': '8f5b4292d6128535',\n",
       "  'vcf_string': '15\\t48428361\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428361G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6982C>T',\n",
       "  'hgvs_protein': 'p.Gln2328Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428361,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48428374-G-A',\n",
       "  'variant_hash': '91e82200fe30c8b4',\n",
       "  'vcf_string': '15\\t48428374\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428374G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6969C>T',\n",
       "  'hgvs_protein': 'p.Thr2323=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428374,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48428390-C-T',\n",
       "  'variant_hash': '2a26342dd6617379',\n",
       "  'vcf_string': '15\\t48428390\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428390C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6953G>A',\n",
       "  'hgvs_protein': 'p.Cys2318Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428390,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48428390-C-G',\n",
       "  'variant_hash': '67502910aa01c2a3',\n",
       "  'vcf_string': '15\\t48428390\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428390C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6953G>C',\n",
       "  'hgvs_protein': 'p.Cys2318Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428390,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48428391-A-G',\n",
       "  'variant_hash': '8cb1a7f15c881fbe',\n",
       "  'vcf_string': '15\\t48428391\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428391A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6952T>C',\n",
       "  'hgvs_protein': 'p.Cys2318Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428391,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48428401-G-A',\n",
       "  'variant_hash': '01016d3af9ffcfd8',\n",
       "  'vcf_string': '15\\t48428401\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428401G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6942C>T',\n",
       "  'hgvs_protein': 'p.Tyr2314=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428401,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48428427-G-A',\n",
       "  'variant_hash': 'fca97119f2bf661f',\n",
       "  'vcf_string': '15\\t48428427\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428427G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6916C>T',\n",
       "  'hgvs_protein': 'p.Arg2306Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428427,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48428439-A-T',\n",
       "  'variant_hash': 'd392cc369aa9e8e1',\n",
       "  'vcf_string': '15\\t48428439\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428439A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6904T>A',\n",
       "  'hgvs_protein': 'p.Cys2302Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428439,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48428440-G-A',\n",
       "  'variant_hash': 'f96f1738a1ee2805',\n",
       "  'vcf_string': '15\\t48428440\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428440G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6903C>T',\n",
       "  'hgvs_protein': 'p.Ile2301=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428440,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48428452-C-T',\n",
       "  'variant_hash': '557fb4386df096f3',\n",
       "  'vcf_string': '15\\t48428452\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428452C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6891G>A',\n",
       "  'hgvs_protein': 'p.Thr2297=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428452,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48428455-C-T',\n",
       "  'variant_hash': '315e8658d485da02',\n",
       "  'vcf_string': '15\\t48428455\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428455C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6888G>A',\n",
       "  'hgvs_protein': 'p.Gln2296=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428455,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48428459-C-T',\n",
       "  'variant_hash': '0ccc454534bb0d52',\n",
       "  'vcf_string': '15\\t48428459\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428459C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6884G>A',\n",
       "  'hgvs_protein': 'p.Cys2295Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428459,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48428483-T-C',\n",
       "  'variant_hash': '16c46b36b46d95c8',\n",
       "  'vcf_string': '15\\t48428483\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428483T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6872-12A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428483,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48429432-T-C',\n",
       "  'variant_hash': '25a166ee5e7dea58',\n",
       "  'vcf_string': '15\\t48429432\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48429432T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6872-961A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48429432,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430659-G-T',\n",
       "  'variant_hash': '2dd0480270d52a53',\n",
       "  'vcf_string': '15\\t48430659\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430659G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6871+12C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430659,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48430676-C-T',\n",
       "  'variant_hash': 'fa54d615960591c8',\n",
       "  'vcf_string': '15\\t48430676\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430676C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6866G>A',\n",
       "  'hgvs_protein': 'p.Cys2289Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430676,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48430676-C-G',\n",
       "  'variant_hash': 'b309d88e77654407',\n",
       "  'vcf_string': '15\\t48430676\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430676C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6866G>C',\n",
       "  'hgvs_protein': 'p.Cys2289Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430676,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48430687-A-G',\n",
       "  'variant_hash': '33df7322a8e03367',\n",
       "  'vcf_string': '15\\t48430687\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430687A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6855T>C',\n",
       "  'hgvs_protein': 'p.Asp2285=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430687,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48430705-C-T',\n",
       "  'variant_hash': '8e6382cf70a3d5c4',\n",
       "  'vcf_string': '15\\t48430705\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430705C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6837G>A',\n",
       "  'hgvs_protein': 'p.Gly2279=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430705,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48430708-G-A',\n",
       "  'variant_hash': '09924c9a37aa2b1c',\n",
       "  'vcf_string': '15\\t48430708\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430708G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6834C>T',\n",
       "  'hgvs_protein': 'p.Pro2278=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430708,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48430710-G-A',\n",
       "  'variant_hash': 'bd820bc456f5f660',\n",
       "  'vcf_string': '15\\t48430710\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430710G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6832C>T',\n",
       "  'hgvs_protein': 'p.Pro2278Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430710,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48430722-A-G',\n",
       "  'variant_hash': '4aab8670414a7007',\n",
       "  'vcf_string': '15\\t48430722\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430722A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6820T>C',\n",
       "  'hgvs_protein': 'p.Cys2274Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430722,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48430727-T-C',\n",
       "  'variant_hash': 'b4e478f5165bfbf6',\n",
       "  'vcf_string': '15\\t48430727\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430727T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6815A>G',\n",
       "  'hgvs_protein': 'p.Tyr2272Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430727,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48430735-A-T',\n",
       "  'variant_hash': '7145bacc06e238ba',\n",
       "  'vcf_string': '15\\t48430735\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430735A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6807T>A',\n",
       "  'hgvs_protein': 'p.Ile2269=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430735,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48430736-A-T',\n",
       "  'variant_hash': 'dddaac792b81279e',\n",
       "  'vcf_string': '15\\t48430736\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430736A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6806T>A',\n",
       "  'hgvs_protein': 'p.Ile2269Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430736,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430736-A-G',\n",
       "  'variant_hash': '4b6576d56b700beb',\n",
       "  'vcf_string': '15\\t48430736\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430736A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6806T>C',\n",
       "  'hgvs_protein': 'p.Ile2269Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430736,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430748-C-T',\n",
       "  'variant_hash': 'fa63cf259a5a487e',\n",
       "  'vcf_string': '15\\t48430748\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430748C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6794G>A',\n",
       "  'hgvs_protein': 'p.Cys2265Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430748,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48430749-A-G',\n",
       "  'variant_hash': '3b0386ba19db0a42',\n",
       "  'vcf_string': '15\\t48430749\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430749A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6793T>C',\n",
       "  'hgvs_protein': 'p.Cys2265Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430749,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48430749-A-C',\n",
       "  'variant_hash': '54be68e6a440ec25',\n",
       "  'vcf_string': '15\\t48430749\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430749A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6793T>G',\n",
       "  'hgvs_protein': 'p.Cys2265Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430749,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430758-G-A',\n",
       "  'variant_hash': 'e7665b54d0224ebd',\n",
       "  'vcf_string': '15\\t48430758\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430758G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6784C>T',\n",
       "  'hgvs_protein': 'p.Gln2262Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430758,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48430770-A-G',\n",
       "  'variant_hash': '62d2d1e66383d245',\n",
       "  'vcf_string': '15\\t48430770\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430770A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6772T>C',\n",
       "  'hgvs_protein': 'p.Cys2258Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430770,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430770-A-C',\n",
       "  'variant_hash': '1de3457a7a7eecfc',\n",
       "  'vcf_string': '15\\t48430770\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430770A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6772T>G',\n",
       "  'hgvs_protein': 'p.Cys2258Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430770,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430774-A-G',\n",
       "  'variant_hash': 'e3f90af8b0cedc87',\n",
       "  'vcf_string': '15\\t48430774\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430774A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6768T>C',\n",
       "  'hgvs_protein': 'p.His2256=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430774,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48430780-T-C',\n",
       "  'variant_hash': 'f34cc3d4b1e70990',\n",
       "  'vcf_string': '15\\t48430780\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430780T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6762A>G',\n",
       "  'hgvs_protein': 'p.Gly2254=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430780,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48430785-C-A',\n",
       "  'variant_hash': '06e3c195584c14a1',\n",
       "  'vcf_string': '15\\t48430785\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430785C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6757G>T',\n",
       "  'hgvs_protein': 'p.Glu2253Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430785,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48430790-C-T',\n",
       "  'variant_hash': '6376d0353955dc3d',\n",
       "  'vcf_string': '15\\t48430790\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430790C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6752G>A',\n",
       "  'hgvs_protein': 'p.Cys2251Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430790,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430791-A-T',\n",
       "  'variant_hash': '7ad83929367ce6eb',\n",
       "  'vcf_string': '15\\t48430791\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430791A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6751T>A',\n",
       "  'hgvs_protein': 'p.Cys2251Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430791,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430791-A-G',\n",
       "  'variant_hash': 'bd751b8744b42c69',\n",
       "  'vcf_string': '15\\t48430791\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430791A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6751T>C',\n",
       "  'hgvs_protein': 'p.Cys2251Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430791,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430803-C-T',\n",
       "  'variant_hash': '0c394766937f8f49',\n",
       "  'vcf_string': '15\\t48430803\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430803C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6740-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430803,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48432850-G-A',\n",
       "  'variant_hash': '102a800fedbacdb4',\n",
       "  'vcf_string': '15\\t48432850\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432850G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6739+16C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432850,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432855-C-T',\n",
       "  'variant_hash': '941d7e348e713cb2',\n",
       "  'vcf_string': '15\\t48432855\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432855C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6739+11G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432855,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432865-C-T',\n",
       "  'variant_hash': 'ccacf817fcae4744',\n",
       "  'vcf_string': '15\\t48432865\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432865C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6739+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432865,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432894-C-A',\n",
       "  'variant_hash': '940f612b2705d8e9',\n",
       "  'vcf_string': '15\\t48432894\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432894C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6711G>T',\n",
       "  'hgvs_protein': 'p.Val2237=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432894,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432897-A-G',\n",
       "  'variant_hash': 'a72f48fe9fd72567',\n",
       "  'vcf_string': '15\\t48432897\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432897A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6708T>C',\n",
       "  'hgvs_protein': 'p.Tyr2236=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432897,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432900-T-G',\n",
       "  'variant_hash': 'e24ccb05acb51d36',\n",
       "  'vcf_string': '15\\t48432900\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432900T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6705A>C',\n",
       "  'hgvs_protein': 'p.Gly2235=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432900,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48432906-G-A',\n",
       "  'variant_hash': 'd0269c747074ee7a',\n",
       "  'vcf_string': '15\\t48432906\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432906G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6699C>T',\n",
       "  'hgvs_protein': 'p.Pro2233=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432906,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432907-G-A',\n",
       "  'variant_hash': '84bdcb2722f9ded6',\n",
       "  'vcf_string': '15\\t48432907\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432907G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6698C>T',\n",
       "  'hgvs_protein': 'p.Pro2233Leu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432907,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48432910-C-T',\n",
       "  'variant_hash': '9b87adb57c9ed55d',\n",
       "  'vcf_string': '15\\t48432910\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432910C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6695G>A',\n",
       "  'hgvs_protein': 'p.Cys2232Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432910,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432910-C-G',\n",
       "  'variant_hash': 'c41fdf72fb23ab2f',\n",
       "  'vcf_string': '15\\t48432910\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432910C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6695G>C',\n",
       "  'hgvs_protein': 'p.Cys2232Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432910,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432911-A-G',\n",
       "  'variant_hash': '4777df22a3645923',\n",
       "  'vcf_string': '15\\t48432911\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432911A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6694T>C',\n",
       "  'hgvs_protein': 'p.Cys2232Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432911,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432916-C-T',\n",
       "  'variant_hash': '3bf1f3015c87e807',\n",
       "  'vcf_string': '15\\t48432916\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432916C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6689G>A',\n",
       "  'hgvs_protein': 'p.Cys2230Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432916,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432924-T-A',\n",
       "  'variant_hash': '6f8557b9066742a3',\n",
       "  'vcf_string': '15\\t48432924\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432924T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6681A>T',\n",
       "  'hgvs_protein': 'p.Ser2227=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432924,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432930-A-G',\n",
       "  'variant_hash': '6a0bba330a67f0c8',\n",
       "  'vcf_string': '15\\t48432930\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432930A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6675T>C',\n",
       "  'hgvs_protein': 'p.Tyr2225=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432930,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432943-C-T',\n",
       "  'variant_hash': '05f05da0bb82f44f',\n",
       "  'vcf_string': '15\\t48432943\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432943C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6662G>A',\n",
       "  'hgvs_protein': 'p.Cys2221Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432943,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48432944-A-G',\n",
       "  'variant_hash': 'c5cdeb6a30607d89',\n",
       "  'vcf_string': '15\\t48432944\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432944A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6661T>C',\n",
       "  'hgvs_protein': 'p.Cys2221Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432944,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48432947-G-A',\n",
       "  'variant_hash': 'b64a19db35126b1d',\n",
       "  'vcf_string': '15\\t48432947\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432947G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6658C>T',\n",
       "  'hgvs_protein': 'p.Arg2220Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432947,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432955-C-T',\n",
       "  'variant_hash': '7b4f9231bc34782b',\n",
       "  'vcf_string': '15\\t48432955\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432955C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6650G>A',\n",
       "  'hgvs_protein': 'p.Cys2217Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432955,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432957-G-T',\n",
       "  'variant_hash': '5356bc74f392d116',\n",
       "  'vcf_string': '15\\t48432957\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432957G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6648C>A',\n",
       "  'hgvs_protein': 'p.Leu2216=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432957,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432962-G-A',\n",
       "  'variant_hash': 'b44431414fe82e38',\n",
       "  'vcf_string': '15\\t48432962\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432962G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6643C>T',\n",
       "  'hgvs_protein': 'p.Leu2215=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432962,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432971-G-A',\n",
       "  'variant_hash': '480e6e678a8aa9a3',\n",
       "  'vcf_string': '15\\t48432971\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432971G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6634C>T',\n",
       "  'hgvs_protein': 'p.Gln2212Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432971,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48432976-C-T',\n",
       "  'variant_hash': '25e744401480f69f',\n",
       "  'vcf_string': '15\\t48432976\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432976C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6629G>A',\n",
       "  'hgvs_protein': 'p.Cys2210Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432976,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432977-A-G',\n",
       "  'variant_hash': '920bed443ce1c1eb',\n",
       "  'vcf_string': '15\\t48432977\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432977A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6628T>C',\n",
       "  'hgvs_protein': 'p.Cys2210Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432977,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432999-G-T',\n",
       "  'variant_hash': '31fa58d331a873ce',\n",
       "  'vcf_string': '15\\t48432999\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432999G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6617-11C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432999,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48433000-G-A',\n",
       "  'variant_hash': '6c98475217db8c46',\n",
       "  'vcf_string': '15\\t48433000\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48433000G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6617-12C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48433000,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48433009-T-A',\n",
       "  'variant_hash': 'f9c55af7417eca6d',\n",
       "  'vcf_string': '15\\t48433009\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48433009T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6617-21A>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48433009,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48434577-C-T',\n",
       "  'variant_hash': 'b757d4f7f7a516cc',\n",
       "  'vcf_string': '15\\t48434577\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434577C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6616+17G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434577,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48434587-G-C',\n",
       "  'variant_hash': 'b031fa6f0a1949be',\n",
       "  'vcf_string': '15\\t48434587\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434587G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6616+7C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434587,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48434593-C-T',\n",
       "  'variant_hash': '71698bc854d05eb3',\n",
       "  'vcf_string': '15\\t48434593\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434593C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6616+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434593,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434599-C-T',\n",
       "  'variant_hash': '35957d2f64570227',\n",
       "  'vcf_string': '15\\t48434599\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434599C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6611G>A',\n",
       "  'hgvs_protein': 'p.Cys2204Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434599,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434599-C-G',\n",
       "  'variant_hash': 'b3e7553d072ac27a',\n",
       "  'vcf_string': '15\\t48434599\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434599C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6611G>C',\n",
       "  'hgvs_protein': 'p.Cys2204Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434599,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48434600-A-G',\n",
       "  'variant_hash': 'e37c07a1ef238252',\n",
       "  'vcf_string': '15\\t48434600\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434600A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6610T>C',\n",
       "  'hgvs_protein': 'p.Cys2204Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434600,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434601-T-C',\n",
       "  'variant_hash': '7f7789f82ac2187a',\n",
       "  'vcf_string': '15\\t48434601\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434601T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6609A>G',\n",
       "  'hgvs_protein': 'p.Thr2203=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434601,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48434616-G-A',\n",
       "  'variant_hash': '5e4d547db5a173e1',\n",
       "  'vcf_string': '15\\t48434616\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434616G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6594C>T',\n",
       "  'hgvs_protein': 'p.Pro2198=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434616,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48434627-C-T',\n",
       "  'variant_hash': 'c18994ab2b15dcb8',\n",
       "  'vcf_string': '15\\t48434627\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434627C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6583G>A',\n",
       "  'hgvs_protein': 'p.Gly2195Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434627,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434634-G-A',\n",
       "  'variant_hash': 'f6316420abff4688',\n",
       "  'vcf_string': '15\\t48434634\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434634G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6576C>T',\n",
       "  'hgvs_protein': 'p.Cys2192=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434634,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48434669-A-G',\n",
       "  'variant_hash': 'f76355dd345c1135',\n",
       "  'vcf_string': '15\\t48434669\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434669A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6541T>C',\n",
       "  'hgvs_protein': 'p.Cys2181Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434669,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48434701-C-T',\n",
       "  'variant_hash': '2074733e70da0186',\n",
       "  'vcf_string': '15\\t48434701\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434701C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6509G>A',\n",
       "  'hgvs_protein': 'p.Cys2170Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434701,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434702-A-G',\n",
       "  'variant_hash': 'c5647b4b7cea9ce1',\n",
       "  'vcf_string': '15\\t48434702\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434702A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6508T>C',\n",
       "  'hgvs_protein': 'p.Cys2170Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434702,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434705-C-A',\n",
       "  'variant_hash': 'edfca620be0da2e7',\n",
       "  'vcf_string': '15\\t48434705\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434705C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6505G>T',\n",
       "  'hgvs_protein': 'p.Glu2169Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434705,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48434707-T-C',\n",
       "  'variant_hash': 'fbbe81dc14a9e52e',\n",
       "  'vcf_string': '15\\t48434707\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434707T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6503A>G',\n",
       "  'hgvs_protein': 'p.Asp2168Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434707,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434724-C-T',\n",
       "  'variant_hash': 'ddbac26d9bd948f5',\n",
       "  'vcf_string': '15\\t48434724\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434724C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6497-11G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434724,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48434725-C-G',\n",
       "  'variant_hash': '951914f60086cacd',\n",
       "  'vcf_string': '15\\t48434725\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434725C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6497-12G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434725,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48434728-G-A',\n",
       "  'variant_hash': '62b82f2fe1c77d65',\n",
       "  'vcf_string': '15\\t48434728\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434728G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6497-15C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434728,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48434736-C-G',\n",
       "  'variant_hash': '8e1090f5f8bd3bb4',\n",
       "  'vcf_string': '15\\t48434736\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434736C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6497-23G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434736,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48434762-C-T',\n",
       "  'variant_hash': 'd6857953b5786fd7',\n",
       "  'vcf_string': '15\\t48434762\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434762C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6497-49G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434762,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48436966-C-T',\n",
       "  'variant_hash': '81bf910dd30ab978',\n",
       "  'vcf_string': '15\\t48436966\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48436966C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6491G>A',\n",
       "  'hgvs_protein': 'p.Cys2164Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48436966,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48436967-A-C',\n",
       "  'variant_hash': 'c3e4cbc81f252463',\n",
       "  'vcf_string': '15\\t48436967\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48436967A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6490T>G',\n",
       "  'hgvs_protein': 'p.Cys2164Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48436967,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48436970-C-A',\n",
       "  'variant_hash': '26214a531331badd',\n",
       "  'vcf_string': '15\\t48436970\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48436970C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6487G>T',\n",
       "  'hgvs_protein': 'p.Glu2163Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48436970,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48436995-G-C',\n",
       "  'variant_hash': '9b5fa8415f484956',\n",
       "  'vcf_string': '15\\t48436995\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48436995G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6462C>G',\n",
       "  'hgvs_protein': 'p.Pro2154=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48436995,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437004-G-C',\n",
       "  'variant_hash': '2b29bac8f007e83c',\n",
       "  'vcf_string': '15\\t48437004\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437004G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6453C>G',\n",
       "  'hgvs_protein': 'p.Cys2151Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437004,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437004-G-A',\n",
       "  'variant_hash': '9f1c18fe0bc997b7',\n",
       "  'vcf_string': '15\\t48437004\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437004G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6453C>T',\n",
       "  'hgvs_protein': 'p.Cys2151=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437004,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437009-G-A',\n",
       "  'variant_hash': 'ff09cdc4703dc42d',\n",
       "  'vcf_string': '15\\t48437009\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437009G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6448C>T',\n",
       "  'hgvs_protein': 'p.Arg2150Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437009,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437011-T-C',\n",
       "  'variant_hash': 'c8d5e9fa1bb99b05',\n",
       "  'vcf_string': '15\\t48437011\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437011T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6446A>G',\n",
       "  'hgvs_protein': 'p.Tyr2149Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437011,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437013-G-A',\n",
       "  'variant_hash': '391857e44d488580',\n",
       "  'vcf_string': '15\\t48437013\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437013G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6444C>T',\n",
       "  'hgvs_protein': 'p.Ser2148=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437013,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437019-A-G',\n",
       "  'variant_hash': 'fd129540b2ea93cf',\n",
       "  'vcf_string': '15\\t48437019\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437019A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6438T>C',\n",
       "  'hgvs_protein': 'p.Asp2146=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437019,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437026-T-C',\n",
       "  'variant_hash': '3e164082d05192b3',\n",
       "  'vcf_string': '15\\t48437026\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437026T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6431A>G',\n",
       "  'hgvs_protein': 'p.Asn2144Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437026,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437027-T-G',\n",
       "  'variant_hash': 'b23555db55a0158d',\n",
       "  'vcf_string': '15\\t48437027\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437027T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6430A>C',\n",
       "  'hgvs_protein': 'p.Asn2144His',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437027,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437032-C-T',\n",
       "  'variant_hash': 'b202d5c300fb4331',\n",
       "  'vcf_string': '15\\t48437032\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437032C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6425G>A',\n",
       "  'hgvs_protein': 'p.Cys2142Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437032,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437038-C-T',\n",
       "  'variant_hash': '0ef75f1230f9f333',\n",
       "  'vcf_string': '15\\t48437038\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437038C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6419G>A',\n",
       "  'hgvs_protein': 'p.Gly2140Glu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437038,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437039-C-T',\n",
       "  'variant_hash': 'afb1f3d6e95fbeaf',\n",
       "  'vcf_string': '15\\t48437039\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437039C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6418G>A',\n",
       "  'hgvs_protein': 'p.Gly2140Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437039,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437040-A-G',\n",
       "  'variant_hash': '97ee733ca3eb31a0',\n",
       "  'vcf_string': '15\\t48437040\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437040A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6417T>C',\n",
       "  'hgvs_protein': 'p.His2139=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437040,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437047-C-T',\n",
       "  'variant_hash': '06d325a9283f49b9',\n",
       "  'vcf_string': '15\\t48437047\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437047C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6410G>A',\n",
       "  'hgvs_protein': 'p.Cys2137Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437047,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437055-G-A',\n",
       "  'variant_hash': '77aded8df890488b',\n",
       "  'vcf_string': '15\\t48437055\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437055G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6402C>T',\n",
       "  'hgvs_protein': 'p.Pro2134=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437055,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437064-G-C',\n",
       "  'variant_hash': '600e0354173ddfcc',\n",
       "  'vcf_string': '15\\t48437064\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437064G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6393C>G',\n",
       "  'hgvs_protein': 'p.Cys2131Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437064,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437064-G-A',\n",
       "  'variant_hash': '9e6e12e8f80b65e7',\n",
       "  'vcf_string': '15\\t48437064\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437064G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6393C>T',\n",
       "  'hgvs_protein': 'p.Cys2131=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437064,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48437069-C-T',\n",
       "  'variant_hash': '9d515ba17ffda9c2',\n",
       "  'vcf_string': '15\\t48437069\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437069C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6388G>A',\n",
       "  'hgvs_protein': 'p.Glu2130Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437069,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437070-G-A',\n",
       "  'variant_hash': '43f911e52c1d4433',\n",
       "  'vcf_string': '15\\t48437070\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437070G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6387C>T',\n",
       "  'hgvs_protein': 'p.Asp2129=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437070,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437079-T-G',\n",
       "  'variant_hash': 'fd44ae96d50e8ac2',\n",
       "  'vcf_string': '15\\t48437079\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437079T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6380-2A>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437079,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437097-A-G',\n",
       "  'variant_hash': '849781a5b02e52d3',\n",
       "  'vcf_string': '15\\t48437097\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437097A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6380-20T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437097,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437162-A-T',\n",
       "  'variant_hash': '872c4188d62fd3a5',\n",
       "  'vcf_string': '15\\t48437162\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437162A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6380-85T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437162,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48437207-G-A',\n",
       "  'variant_hash': 'cac248341e5c2e69',\n",
       "  'vcf_string': '15\\t48437207\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437207G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+115C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437207,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48437311-A-G',\n",
       "  'variant_hash': '60d6e23c957ff3d7',\n",
       "  'vcf_string': '15\\t48437311\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437311A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+11T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437311,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437313-G-A',\n",
       "  'variant_hash': '48b35b51e3f3ce3a',\n",
       "  'vcf_string': '15\\t48437313\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437313G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+9C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437313,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437318-T-C',\n",
       "  'variant_hash': '38d0b21c6e5c00d6',\n",
       "  'vcf_string': '15\\t48437318\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437318T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+4A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437318,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48437320-A-G',\n",
       "  'variant_hash': '601482ba61352bbc',\n",
       "  'vcf_string': '15\\t48437320\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437320A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+2T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437320,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437320-A-C',\n",
       "  'variant_hash': '037bf7c66ca55a4b',\n",
       "  'vcf_string': '15\\t48437320\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437320A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+2T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437320,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437321-C-T',\n",
       "  'variant_hash': 'be0c98425c2da098',\n",
       "  'vcf_string': '15\\t48437321\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437321C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437321,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437326-T-C',\n",
       "  'variant_hash': 'c92d134b0fb60545',\n",
       "  'vcf_string': '15\\t48437326\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437326T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6375A>G',\n",
       "  'hgvs_protein': 'p.Ala2125=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437326,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437329-T-C',\n",
       "  'variant_hash': 'a6207db4e8c3e537',\n",
       "  'vcf_string': '15\\t48437329\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437329T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6372A>G',\n",
       "  'hgvs_protein': 'p.Ser2124=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437329,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437347-G-C',\n",
       "  'variant_hash': 'bfcae4c74832ff3a',\n",
       "  'vcf_string': '15\\t48437347\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437347G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6354C>G',\n",
       "  'hgvs_protein': 'p.Ile2118Met',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437347,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437347-G-A',\n",
       "  'variant_hash': '8bfb584dafae485e',\n",
       "  'vcf_string': '15\\t48437347\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437347G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6354C>T',\n",
       "  'hgvs_protein': 'p.Ile2118=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437347,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437359-T-G',\n",
       "  'variant_hash': '7a03cb3c62f92e98',\n",
       "  'vcf_string': '15\\t48437359\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437359T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6342A>C',\n",
       "  'hgvs_protein': 'p.Gly2114=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437359,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437362-A-C',\n",
       "  'variant_hash': '95e9366f62b85656',\n",
       "  'vcf_string': '15\\t48437362\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437362A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6339T>G',\n",
       "  'hgvs_protein': 'p.Tyr2113Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437362,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437365-A-G',\n",
       "  'variant_hash': '4e21d2a9ff2c8b69',\n",
       "  'vcf_string': '15\\t48437365\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437365A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6336T>C',\n",
       "  'hgvs_protein': 'p.Pro2112=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437365,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437369-C-T',\n",
       "  'variant_hash': '929bb230b11eab82',\n",
       "  'vcf_string': '15\\t48437369\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437369C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6332G>A',\n",
       "  'hgvs_protein': 'p.Cys2111Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437369,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437370-A-G',\n",
       "  'variant_hash': 'c82cab5cf9304570',\n",
       "  'vcf_string': '15\\t48437370\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437370A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6331T>C',\n",
       "  'hgvs_protein': 'p.Cys2111Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437370,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437376-G-A',\n",
       "  'variant_hash': 'f7ac39535ee29752',\n",
       "  'vcf_string': '15\\t48437376\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437376G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6325C>T',\n",
       "  'hgvs_protein': 'p.Gln2109Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437376,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437379-G-A',\n",
       "  'variant_hash': 'bbe9e6ae55f253b0',\n",
       "  'vcf_string': '15\\t48437379\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437379G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6322C>T',\n",
       "  'hgvs_protein': 'p.Arg2108Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437379,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437393-G-A',\n",
       "  'variant_hash': '0a9acc9501c60cae',\n",
       "  'vcf_string': '15\\t48437393\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437393G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6314-6C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437393,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437403-T-C',\n",
       "  'variant_hash': '8ffc0331955f5d3b',\n",
       "  'vcf_string': '15\\t48437403\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437403T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6314-16A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437403,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437451-T-C',\n",
       "  'variant_hash': 'f1948443db6d5ea9',\n",
       "  'vcf_string': '15\\t48437451\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437451T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6314-64A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437451,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48437765-T-C',\n",
       "  'variant_hash': '00b4c885ef35057a',\n",
       "  'vcf_string': '15\\t48437765\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437765T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6313+3A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437765,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437768-C-A',\n",
       "  'variant_hash': 'e1698bf7c2e9ef1e',\n",
       "  'vcf_string': '15\\t48437768\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437768C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6313G>T',\n",
       "  'hgvs_protein': 'p.Glu2105Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437768,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437778-C-T',\n",
       "  'variant_hash': '350e78e8e0330934',\n",
       "  'vcf_string': '15\\t48437778\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437778C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6303G>A',\n",
       "  'hgvs_protein': 'p.Thr2101=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437778,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437779-G-A',\n",
       "  'variant_hash': '4448945f75bfa05a',\n",
       "  'vcf_string': '15\\t48437779\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437779G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6302C>T',\n",
       "  'hgvs_protein': 'p.Thr2101Met',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437779,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48437784-G-C',\n",
       "  'variant_hash': '74a5ea48ab6e4651',\n",
       "  'vcf_string': '15\\t48437784\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437784G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6297C>G',\n",
       "  'hgvs_protein': 'p.Cys2099Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437784,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437793-G-A',\n",
       "  'variant_hash': '37a6a769ab95404c',\n",
       "  'vcf_string': '15\\t48437793\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437793G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6288C>T',\n",
       "  'hgvs_protein': 'p.Cys2096=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437793,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48437805-C-T',\n",
       "  'variant_hash': '9115ac899b22eee7',\n",
       "  'vcf_string': '15\\t48437805\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437805C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6276G>A',\n",
       "  'hgvs_protein': 'p.Trp2092Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437805,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437807-A-G',\n",
       "  'variant_hash': '8200626dda786e1c',\n",
       "  'vcf_string': '15\\t48437807\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437807A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6274T>C',\n",
       "  'hgvs_protein': 'p.Trp2092Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437807,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437827-C-T',\n",
       "  'variant_hash': '7bf3f0a1b9ddfbd8',\n",
       "  'vcf_string': '15\\t48437827\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437827C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6254G>A',\n",
       "  'hgvs_protein': 'p.Cys2085Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437827,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437837-C-A',\n",
       "  'variant_hash': '6481759627163733',\n",
       "  'vcf_string': '15\\t48437837\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437837C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6244G>T',\n",
       "  'hgvs_protein': 'p.Glu2082Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437837,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437862-G-A',\n",
       "  'variant_hash': '346bf87822e45e32',\n",
       "  'vcf_string': '15\\t48437862\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437862G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6219C>T',\n",
       "  'hgvs_protein': 'p.Pro2073=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437862,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437872-C-T',\n",
       "  'variant_hash': 'b1731817c5287ce6',\n",
       "  'vcf_string': '15\\t48437872\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437872C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6209G>A',\n",
       "  'hgvs_protein': 'p.Cys2070Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437872,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437873-A-G',\n",
       "  'variant_hash': 'e2dc96e331221545',\n",
       "  'vcf_string': '15\\t48437873\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437873A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6208T>C',\n",
       "  'hgvs_protein': 'p.Cys2070Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437873,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437874-C-T',\n",
       "  'variant_hash': '6c14b3be8263546a',\n",
       "  'vcf_string': '15\\t48437874\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437874C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6207G>A',\n",
       "  'hgvs_protein': 'p.Lys2069=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437874,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437900-A-G',\n",
       "  'variant_hash': '18458fc5b185c1c4',\n",
       "  'vcf_string': '15\\t48437900\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437900A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6181T>C',\n",
       "  'hgvs_protein': 'p.Cys2061Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437900,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437901-G-C',\n",
       "  'variant_hash': '7f8d4fae2cc159e7',\n",
       "  'vcf_string': '15\\t48437901\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437901G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6180C>G',\n",
       "  'hgvs_protein': 'p.Tyr2060Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437901,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437912-G-A',\n",
       "  'variant_hash': 'f069a16de1ed6dc7',\n",
       "  'vcf_string': '15\\t48437912\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437912G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6169C>T',\n",
       "  'hgvs_protein': 'p.Arg2057Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437912,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437919-T-C',\n",
       "  'variant_hash': 'd391c8add4c9f11e',\n",
       "  'vcf_string': '15\\t48437919\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437919T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6164-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437919,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437929-A-G',\n",
       "  'variant_hash': 'c8459cb26179dcaf',\n",
       "  'vcf_string': '15\\t48437929\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437929A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6164-12T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437929,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441591-G-A',\n",
       "  'variant_hash': 'c1e4d9721da48833',\n",
       "  'vcf_string': '15\\t48441591\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441591G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6163+130C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441591,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441712-G-C',\n",
       "  'variant_hash': 'cb2cc480e3680e0c',\n",
       "  'vcf_string': '15\\t48441712\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441712G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6163+9C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441712,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441720-C-T',\n",
       "  'variant_hash': '4453c0df99bae68d',\n",
       "  'vcf_string': '15\\t48441720\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441720C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6163+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441720,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48441728-C-T',\n",
       "  'variant_hash': '2bdce4db48755f3e',\n",
       "  'vcf_string': '15\\t48441728\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441728C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6156G>A',\n",
       "  'hgvs_protein': 'p.Arg2052=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441728,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441737-A-G',\n",
       "  'variant_hash': 'f9f27141ef905966',\n",
       "  'vcf_string': '15\\t48441737\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441737A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6147T>C',\n",
       "  'hgvs_protein': 'p.Ser2049=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441737,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441740-G-A',\n",
       "  'variant_hash': 'e05063d3dfcfaf37',\n",
       "  'vcf_string': '15\\t48441740\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441740G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6144C>T',\n",
       "  'hgvs_protein': 'p.Ser2048=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441740,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441743-G-A',\n",
       "  'variant_hash': '44c5a78acaaecd27',\n",
       "  'vcf_string': '15\\t48441743\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441743G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6141C>T',\n",
       "  'hgvs_protein': 'p.Ser2047=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441743,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441755-C-G',\n",
       "  'variant_hash': '45f15d8ce1e16aeb',\n",
       "  'vcf_string': '15\\t48441755\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441755C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6129G>C',\n",
       "  'hgvs_protein': 'p.Gly2043=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441755,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441755-C-A',\n",
       "  'variant_hash': 'a14eaf214c2f72bd',\n",
       "  'vcf_string': '15\\t48441755\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441755C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6129G>T',\n",
       "  'hgvs_protein': 'p.Gly2043=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441755,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441764-A-G',\n",
       "  'variant_hash': 'e14acfac952ea98e',\n",
       "  'vcf_string': '15\\t48441764\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441764A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6120T>C',\n",
       "  'hgvs_protein': 'p.Cys2040=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441764,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441764-A-C',\n",
       "  'variant_hash': '130188d7ce8431d0',\n",
       "  'vcf_string': '15\\t48441764\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441764A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6120T>G',\n",
       "  'hgvs_protein': 'p.Cys2040Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441764,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48441765-C-T',\n",
       "  'variant_hash': 'a8c9b1779d22f37c',\n",
       "  'vcf_string': '15\\t48441765\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441765C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6119G>A',\n",
       "  'hgvs_protein': 'p.Cys2040Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441765,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48441771-C-T',\n",
       "  'variant_hash': '7e4fc1d27dc9673e',\n",
       "  'vcf_string': '15\\t48441771\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441771C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6113G>A',\n",
       "  'hgvs_protein': 'p.Cys2038Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441771,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48441773-T-C',\n",
       "  'variant_hash': '31de9045f8804ded',\n",
       "  'vcf_string': '15\\t48441773\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441773T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6111A>G',\n",
       "  'hgvs_protein': 'p.Lys2037=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441773,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441785-T-C',\n",
       "  'variant_hash': '259aef14ddc04c42',\n",
       "  'vcf_string': '15\\t48441785\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441785T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6099A>G',\n",
       "  'hgvs_protein': 'p.Glu2033=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441785,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441797-G-T',\n",
       "  'variant_hash': '0964b805d28952f1',\n",
       "  'vcf_string': '15\\t48441797\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441797G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6087C>A',\n",
       "  'hgvs_protein': 'p.Cys2029Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441797,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48441798-C-T',\n",
       "  'variant_hash': 'fcb3d401f7dcb10d',\n",
       "  'vcf_string': '15\\t48441798\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441798C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6086G>A',\n",
       "  'hgvs_protein': 'p.Cys2029Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441798,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48441800-T-C',\n",
       "  'variant_hash': '4f92136d8bd062d0',\n",
       "  'vcf_string': '15\\t48441800\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441800T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6084A>G',\n",
       "  'hgvs_protein': 'p.Thr2028=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441800,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441811-C-A',\n",
       "  'variant_hash': '24235f4275cd1ed3',\n",
       "  'vcf_string': '15\\t48441811\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441811C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6073G>T',\n",
       "  'hgvs_protein': 'p.Ala2025Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441811,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48441813-C-T',\n",
       "  'variant_hash': '52e560f9f9bcd508',\n",
       "  'vcf_string': '15\\t48441813\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441813C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6071G>A',\n",
       "  'hgvs_protein': 'p.Cys2024Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441813,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48441830-G-C',\n",
       "  'variant_hash': '1beb96ae7ff095b5',\n",
       "  'vcf_string': '15\\t48441830\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441830G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6054C>G',\n",
       "  'hgvs_protein': 'p.Val2018=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441830,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441833-A-C',\n",
       "  'variant_hash': 'b4f0ed4b37f2aa09',\n",
       "  'vcf_string': '15\\t48441833\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441833A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6051T>G',\n",
       "  'hgvs_protein': 'p.Cys2017Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441833,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48441834-C-T',\n",
       "  'variant_hash': '6d9a3da09b3429da',\n",
       "  'vcf_string': '15\\t48441834\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441834C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6050G>A',\n",
       "  'hgvs_protein': 'p.Cys2017Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441834,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48441835-A-G',\n",
       "  'variant_hash': 'd1fa9e20fea109ee',\n",
       "  'vcf_string': '15\\t48441835\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441835A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6049T>C',\n",
       "  'hgvs_protein': 'p.Cys2017Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441835,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48441836-C-T',\n",
       "  'variant_hash': '673447160c87a653',\n",
       "  'vcf_string': '15\\t48441836\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441836C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6048G>A',\n",
       "  'hgvs_protein': 'p.Glu2016=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441836,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441854-A-T',\n",
       "  'variant_hash': '8265461a2dbf247f',\n",
       "  'vcf_string': '15\\t48441854\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441854A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6038-8T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441854,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48441859-C-A',\n",
       "  'variant_hash': '55e29edac54302cd',\n",
       "  'vcf_string': '15\\t48441859\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441859C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6038-13G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441859,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441864-G-A',\n",
       "  'variant_hash': 'e0b5750fc7ee093a',\n",
       "  'vcf_string': '15\\t48441864\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441864G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6038-18C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441864,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48444418-T-C',\n",
       "  'variant_hash': 'e394270ca7dfaeee',\n",
       "  'vcf_string': '15\\t48444418\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444418T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6037+123A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444418,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48444487-A-T',\n",
       "  'variant_hash': '3d8c44e41bac050c',\n",
       "  'vcf_string': '15\\t48444487\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444487A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6037+54T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444487,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48444540-C-T',\n",
       "  'variant_hash': '3a96f687894abcbc',\n",
       "  'vcf_string': '15\\t48444540\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444540C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6037+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444540,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48444546-C-T',\n",
       "  'variant_hash': '0d2bbd010be9d2c4',\n",
       "  'vcf_string': '15\\t48444546\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444546C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6032G>A',\n",
       "  'hgvs_protein': 'p.Cys2011Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444546,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48444547-A-G',\n",
       "  'variant_hash': 'eb341c1fe1cd8df8',\n",
       "  'vcf_string': '15\\t48444547\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444547A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6031T>C',\n",
       "  'hgvs_protein': 'p.Cys2011Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444547,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444566-G-C',\n",
       "  'variant_hash': '7f15c7f4c056924a',\n",
       "  'vcf_string': '15\\t48444566\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444566G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6012C>G',\n",
       "  'hgvs_protein': 'p.Tyr2004Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444566,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444579-C-T',\n",
       "  'variant_hash': 'b0e312abdd5711d7',\n",
       "  'vcf_string': '15\\t48444579\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444579C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5999G>A',\n",
       "  'hgvs_protein': 'p.Cys2000Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444579,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444579-C-A',\n",
       "  'variant_hash': '509fdbc1d88fff54',\n",
       "  'vcf_string': '15\\t48444579\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444579C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5999G>T',\n",
       "  'hgvs_protein': 'p.Cys2000Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444579,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48444584-G-A',\n",
       "  'variant_hash': 'beef76f425b1d59b',\n",
       "  'vcf_string': '15\\t48444584\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444584G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5994C>T',\n",
       "  'hgvs_protein': 'p.Cys1998=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444584,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48444585-C-T',\n",
       "  'variant_hash': 'e2974dc8800a7ea1',\n",
       "  'vcf_string': '15\\t48444585\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444585C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5993G>A',\n",
       "  'hgvs_protein': 'p.Cys1998Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444585,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444587-T-C',\n",
       "  'variant_hash': 'bbbddd956455a8bb',\n",
       "  'vcf_string': '15\\t48444587\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444587T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5991A>G',\n",
       "  'hgvs_protein': 'p.Arg1997=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444587,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48444611-A-C',\n",
       "  'variant_hash': '46cc85d43ee8486c',\n",
       "  'vcf_string': '15\\t48444611\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444611A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5967T>G',\n",
       "  'hgvs_protein': 'p.Cys1989Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444611,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444612-C-A',\n",
       "  'variant_hash': 'dd13990e47083276',\n",
       "  'vcf_string': '15\\t48444612\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444612C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5966G>T',\n",
       "  'hgvs_protein': 'p.Cys1989Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444612,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48444627-C-T',\n",
       "  'variant_hash': 'd67c3b39913962be',\n",
       "  'vcf_string': '15\\t48444627\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444627C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5951G>A',\n",
       "  'hgvs_protein': 'p.Cys1984Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444627,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444628-A-G',\n",
       "  'variant_hash': '1ba04f7166341485',\n",
       "  'vcf_string': '15\\t48444628\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444628A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5950T>C',\n",
       "  'hgvs_protein': 'p.Cys1984Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444628,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444648-C-T',\n",
       "  'variant_hash': '5ac0463132c00ac0',\n",
       "  'vcf_string': '15\\t48444648\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444648C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5930G>A',\n",
       "  'hgvs_protein': 'p.Cys1977Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444648,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444653-A-G',\n",
       "  'variant_hash': 'f1cd1a8eabe4c413',\n",
       "  'vcf_string': '15\\t48444653\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444653A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5925T>C',\n",
       "  'hgvs_protein': 'p.Asn1975=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444653,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48444662-T-C',\n",
       "  'variant_hash': '682350d4d9920bdf',\n",
       "  'vcf_string': '15\\t48444662\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444662T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5918-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444662,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445168-A-G',\n",
       "  'variant_hash': '1cb1e2053487a0aa',\n",
       "  'vcf_string': '15\\t48445168\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445168A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5917+208T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445168,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445370-A-G',\n",
       "  'variant_hash': '9f90da6bdc752725',\n",
       "  'vcf_string': '15\\t48445370\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445370A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5917+6T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445370,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445375-C-T',\n",
       "  'variant_hash': '7c64d8401a6e85f5',\n",
       "  'vcf_string': '15\\t48445375\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445375C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5917+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445375,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445375-C-A',\n",
       "  'variant_hash': '8a6369eb7ca8d2cc',\n",
       "  'vcf_string': '15\\t48445375\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445375C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5917+1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445375,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445382-A-G',\n",
       "  'variant_hash': '02561fdac0903b77',\n",
       "  'vcf_string': '15\\t48445382\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445382A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5911T>C',\n",
       "  'hgvs_protein': 'p.Cys1971Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445382,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445408-T-C',\n",
       "  'variant_hash': '11be2f5136edd89c',\n",
       "  'vcf_string': '15\\t48445408\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445408T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5885A>G',\n",
       "  'hgvs_protein': 'p.Tyr1962Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445408,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48445419-G-T',\n",
       "  'variant_hash': 'b770b62f594e55d0',\n",
       "  'vcf_string': '15\\t48445419\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445419G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5874C>A',\n",
       "  'hgvs_protein': 'p.Cys1958Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445419,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445421-A-T',\n",
       "  'variant_hash': 'bb19c0af44799d81',\n",
       "  'vcf_string': '15\\t48445421\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445421A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5872T>A',\n",
       "  'hgvs_protein': 'p.Cys1958Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445421,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445424-G-A',\n",
       "  'variant_hash': '13062342ac55f80f',\n",
       "  'vcf_string': '15\\t48445424\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445424G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5869C>T',\n",
       "  'hgvs_protein': 'p.Gln1957Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445424,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445430-G-A',\n",
       "  'variant_hash': '536dc6d2efacdb99',\n",
       "  'vcf_string': '15\\t48445430\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445430G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5863C>T',\n",
       "  'hgvs_protein': 'p.Gln1955Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445430,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48445432-A-C',\n",
       "  'variant_hash': '4eb1620c11debe59',\n",
       "  'vcf_string': '15\\t48445432\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445432A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5861T>G',\n",
       "  'hgvs_protein': 'p.Phe1954Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445432,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48445437-C-T',\n",
       "  'variant_hash': 'efccae0ca637f929',\n",
       "  'vcf_string': '15\\t48445437\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445437C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5856G>A',\n",
       "  'hgvs_protein': 'p.Gly1952=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445437,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445443-T-C',\n",
       "  'variant_hash': '1204a6772965572a',\n",
       "  'vcf_string': '15\\t48445443\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445443T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5850A>G',\n",
       "  'hgvs_protein': 'p.Thr1950=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445443,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445452-G-A',\n",
       "  'variant_hash': 'df26fbf94e5955b0',\n",
       "  'vcf_string': '15\\t48445452\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445452G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5841C>T',\n",
       "  'hgvs_protein': 'p.Cys1947=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445452,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445453-C-T',\n",
       "  'variant_hash': 'c85c160a7b08fc8a',\n",
       "  'vcf_string': '15\\t48445453\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445453C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5840G>A',\n",
       "  'hgvs_protein': 'p.Cys1947Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445453,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445454-A-G',\n",
       "  'variant_hash': 'fefb3ea3f7084c3e',\n",
       "  'vcf_string': '15\\t48445454\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445454A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5839T>C',\n",
       "  'hgvs_protein': 'p.Cys1947Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445454,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48445455-T-C',\n",
       "  'variant_hash': '0bdd2b9fe48efa45',\n",
       "  'vcf_string': '15\\t48445455\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445455T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5838A>G',\n",
       "  'hgvs_protein': 'p.Gln1946=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445455,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445467-G-T',\n",
       "  'variant_hash': 'a4419fee5fc076d0',\n",
       "  'vcf_string': '15\\t48445467\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445467G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5826C>A',\n",
       "  'hgvs_protein': 'p.Cys1942Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445467,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48445469-A-G',\n",
       "  'variant_hash': '9fc261592d885c80',\n",
       "  'vcf_string': '15\\t48445469\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445469A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5824T>C',\n",
       "  'hgvs_protein': 'p.Cys1942Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445469,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445476-C-T',\n",
       "  'variant_hash': '6f99a0fe69fd4f76',\n",
       "  'vcf_string': '15\\t48445476\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445476C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5817G>A',\n",
       "  'hgvs_protein': 'p.Gly1939=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445476,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445488-T-G',\n",
       "  'variant_hash': 'bd2dcd2114102294',\n",
       "  'vcf_string': '15\\t48445488\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445488T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5805A>C',\n",
       "  'hgvs_protein': 'p.Ala1935=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445488,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445492-C-T',\n",
       "  'variant_hash': '9c8a7e57216e626d',\n",
       "  'vcf_string': '15\\t48445492\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445492C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5801G>A',\n",
       "  'hgvs_protein': 'p.Cys1934Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445492,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48445493-A-G',\n",
       "  'variant_hash': 'c605be31d168548d',\n",
       "  'vcf_string': '15\\t48445493\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445493A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5800T>C',\n",
       "  'hgvs_protein': 'p.Cys1934Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445493,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445504-T-C',\n",
       "  'variant_hash': '688eca7921121f3d',\n",
       "  'vcf_string': '15\\t48445504\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445504T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5789A>G',\n",
       "  'hgvs_protein': 'p.Asp1930Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445504,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48445505-C-T',\n",
       "  'variant_hash': '0912852182a3acd3',\n",
       "  'vcf_string': '15\\t48445505\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445505C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5789-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445505,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446687-C-T',\n",
       "  'variant_hash': '9265702bd414be49',\n",
       "  'vcf_string': '15\\t48446687\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446687C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5788+19G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446687,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48446691-A-G',\n",
       "  'variant_hash': '698bd08ec64c6a64',\n",
       "  'vcf_string': '15\\t48446691\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446691A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5788+15T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446691,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48446696-G-T',\n",
       "  'variant_hash': 'ae84ecbcab770c84',\n",
       "  'vcf_string': '15\\t48446696\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446696G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5788+10C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446696,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48446701-C-T',\n",
       "  'variant_hash': '682d34777733d33d',\n",
       "  'vcf_string': '15\\t48446701\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446701C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5788+5G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446701,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_5th_base_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446701-C-A',\n",
       "  'variant_hash': 'c53960afbc10e379',\n",
       "  'vcf_string': '15\\t48446701\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446701C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5788+5G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446701,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_5th_base_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446706-C-T',\n",
       "  'variant_hash': '82709d7f90b1a07a',\n",
       "  'vcf_string': '15\\t48446706\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446706C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5788G>A',\n",
       "  'hgvs_protein': 'p.Asp1930Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446706,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446711-C-T',\n",
       "  'variant_hash': '3061c37c4df7f5df',\n",
       "  'vcf_string': '15\\t48446711\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446711C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5783G>A',\n",
       "  'hgvs_protein': 'p.Cys1928Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446711,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446712-A-G',\n",
       "  'variant_hash': '4c90e054f5ac221d',\n",
       "  'vcf_string': '15\\t48446712\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446712A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5782T>C',\n",
       "  'hgvs_protein': 'p.Cys1928Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446712,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446747-C-T',\n",
       "  'variant_hash': 'a55d56115f21e812',\n",
       "  'vcf_string': '15\\t48446747\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446747C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5747G>A',\n",
       "  'hgvs_protein': 'p.Cys1916Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446747,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48446751-G-A',\n",
       "  'variant_hash': 'a9b30ee29b341bfe',\n",
       "  'vcf_string': '15\\t48446751\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446751G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5743C>T',\n",
       "  'hgvs_protein': 'p.Arg1915Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446751,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48446753-C-T',\n",
       "  'variant_hash': '477ea7e342ed6535',\n",
       "  'vcf_string': '15\\t48446753\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446753C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5741G>A',\n",
       "  'hgvs_protein': 'p.Cys1914Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446753,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48446765-C-A',\n",
       "  'variant_hash': 'bc691d45b4bd7a75',\n",
       "  'vcf_string': '15\\t48446765\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446765C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5729G>T',\n",
       "  'hgvs_protein': 'p.Gly1910Val',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446765,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48446768-A-G',\n",
       "  'variant_hash': 'beafa552af3ebae4',\n",
       "  'vcf_string': '15\\t48446768\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446768A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5726T>C',\n",
       "  'hgvs_protein': 'p.Ile1909Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446768,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446775-T-C',\n",
       "  'variant_hash': '09106bf0a5a14182',\n",
       "  'vcf_string': '15\\t48446775\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446775T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5719A>G',\n",
       "  'hgvs_protein': 'p.Asn1907Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446775,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48446776-C-G',\n",
       "  'variant_hash': '4aa62ae425535424',\n",
       "  'vcf_string': '15\\t48446776\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446776C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5718G>C',\n",
       "  'hgvs_protein': 'p.Arg1906=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446776,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48446795-C-T',\n",
       "  'variant_hash': '257fed0735de0226',\n",
       "  'vcf_string': '15\\t48446795\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446795C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5699G>A',\n",
       "  'hgvs_protein': 'p.Cys1900Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446795,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48446797-G-A',\n",
       "  'variant_hash': '030e1098800e32f1',\n",
       "  'vcf_string': '15\\t48446797\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446797G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5697C>T',\n",
       "  'hgvs_protein': 'p.Ala1899=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446797,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48446811-A-G',\n",
       "  'variant_hash': '4123060547f1e834',\n",
       "  'vcf_string': '15\\t48446811\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446811A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5683T>C',\n",
       "  'hgvs_protein': 'p.Cys1895Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446811,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446814-C-T',\n",
       "  'variant_hash': '47ef48ef4dae6e7d',\n",
       "  'vcf_string': '15\\t48446814\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446814C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5680G>A',\n",
       "  'hgvs_protein': 'p.Glu1894Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446814,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48446816-T-C',\n",
       "  'variant_hash': '5d20b4cc64cbe99b',\n",
       "  'vcf_string': '15\\t48446816\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446816T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5678A>G',\n",
       "  'hgvs_protein': 'p.Asn1893Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446816,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48446823-C-T',\n",
       "  'variant_hash': '43deced036587ed5',\n",
       "  'vcf_string': '15\\t48446823\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446823C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5672-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446823,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446823-C-G',\n",
       "  'variant_hash': 'a2097a144e38638a',\n",
       "  'vcf_string': '15\\t48446823\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446823C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5672-1G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446823,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446828-A-T',\n",
       "  'variant_hash': '0f0db93646a338a7',\n",
       "  'vcf_string': '15\\t48446828\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446828A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5672-6T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446828,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48446830-G-C',\n",
       "  'variant_hash': 'c7787f6044a326ad',\n",
       "  'vcf_string': '15\\t48446830\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446830G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5672-8C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446830,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48446885-C-A',\n",
       "  'variant_hash': '6ba4f78d687feddc',\n",
       "  'vcf_string': '15\\t48446885\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446885C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5672-63G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446885,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48446909-T-C',\n",
       "  'variant_hash': '0644fe5f86aacff2',\n",
       "  'vcf_string': '15\\t48446909\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446909T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5672-87A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446909,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48448768-C-T',\n",
       "  'variant_hash': '2257c2a1e7f68491',\n",
       "  'vcf_string': '15\\t48448768\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448768C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5671G>A',\n",
       "  'hgvs_protein': 'p.Asp1891Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448768,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48448768-C-G',\n",
       "  'variant_hash': '9748b6075d4c10ca',\n",
       "  'vcf_string': '15\\t48448768\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448768C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5671G>C',\n",
       "  'hgvs_protein': 'p.Asp1891His',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448768,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48448773-C-T',\n",
       "  'variant_hash': '68a300cfb66c1935',\n",
       "  'vcf_string': '15\\t48448773\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448773C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5666G>A',\n",
       "  'hgvs_protein': 'p.Cys1889Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448773,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48448790-A-G',\n",
       "  'variant_hash': '73b55dc567094aec',\n",
       "  'vcf_string': '15\\t48448790\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448790A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5649T>C',\n",
       "  'hgvs_protein': 'p.Asn1883=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448790,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48448813-A-G',\n",
       "  'variant_hash': 'dcbc24fe7d496682',\n",
       "  'vcf_string': '15\\t48448813\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448813A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5626T>C',\n",
       "  'hgvs_protein': 'p.Cys1876Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448813,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48448832-A-G',\n",
       "  'variant_hash': 'cfde5968433d842d',\n",
       "  'vcf_string': '15\\t48448832\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448832A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5607T>C',\n",
       "  'hgvs_protein': 'p.Val1869=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448832,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48448851-C-T',\n",
       "  'variant_hash': '807d3ea5c091d639',\n",
       "  'vcf_string': '15\\t48448851\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448851C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5588G>A',\n",
       "  'hgvs_protein': 'p.Gly1863Glu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448851,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48448853-A-G',\n",
       "  'variant_hash': '20479d50f8d4a711',\n",
       "  'vcf_string': '15\\t48448853\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448853A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5586T>C',\n",
       "  'hgvs_protein': 'p.His1862=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448853,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48448860-C-T',\n",
       "  'variant_hash': '46885208f28a0ed3',\n",
       "  'vcf_string': '15\\t48448860\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448860C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5579G>A',\n",
       "  'hgvs_protein': 'p.Cys1860Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448860,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48448879-G-A',\n",
       "  'variant_hash': '91b79f24a9f27cb2',\n",
       "  'vcf_string': '15\\t48448879\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448879G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5560C>T',\n",
       "  'hgvs_protein': 'p.Gln1854Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448879,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48448881-C-T',\n",
       "  'variant_hash': 'f98e3d03ebbb2fe1',\n",
       "  'vcf_string': '15\\t48448881\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448881C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5558G>A',\n",
       "  'hgvs_protein': 'p.Cys1853Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448881,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48448882-A-G',\n",
       "  'variant_hash': '35cdca661924c0ce',\n",
       "  'vcf_string': '15\\t48448882\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448882A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5557T>C',\n",
       "  'hgvs_protein': 'p.Cys1853Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448882,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48448894-C-T',\n",
       "  'variant_hash': '34bc7dda59fe38c2',\n",
       "  'vcf_string': '15\\t48448894\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448894C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5546-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448894,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452550-G-A',\n",
       "  'variant_hash': 'b738472dcb137651',\n",
       "  'vcf_string': '15\\t48452550\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452550G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5545+12C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452550,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452567-C-T',\n",
       "  'variant_hash': '148f00d474619a76',\n",
       "  'vcf_string': '15\\t48452567\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452567C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5540G>A',\n",
       "  'hgvs_protein': 'p.Cys1847Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452567,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452575-T-C',\n",
       "  'variant_hash': '2fadeb5aa29054dc',\n",
       "  'vcf_string': '15\\t48452575\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452575T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5532A>G',\n",
       "  'hgvs_protein': 'p.Thr1844=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452575,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452589-G-A',\n",
       "  'variant_hash': '826a4eec0c553e3a',\n",
       "  'vcf_string': '15\\t48452589\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452589G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5518C>T',\n",
       "  'hgvs_protein': 'p.Arg1840Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452589,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48452590-G-A',\n",
       "  'variant_hash': '47dc99ff5e005461',\n",
       "  'vcf_string': '15\\t48452590\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452590G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5517C>T',\n",
       "  'hgvs_protein': 'p.Tyr1839=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452590,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452595-C-A',\n",
       "  'variant_hash': 'd5aa158a8a4efa1e',\n",
       "  'vcf_string': '15\\t48452595\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452595C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5512G>T',\n",
       "  'hgvs_protein': 'p.Gly1838Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452595,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48452596-G-A',\n",
       "  'variant_hash': '2f9b26c4993e6b7b',\n",
       "  'vcf_string': '15\\t48452596\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452596G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5511C>T',\n",
       "  'hgvs_protein': 'p.Pro1837=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452596,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452603-C-T',\n",
       "  'variant_hash': '151a0dfe392065d4',\n",
       "  'vcf_string': '15\\t48452603\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452603C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5504G>A',\n",
       "  'hgvs_protein': 'p.Cys1835Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452603,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452614-G-T',\n",
       "  'variant_hash': '53bf7c0b182bc591',\n",
       "  'vcf_string': '15\\t48452614\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452614G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5493C>A',\n",
       "  'hgvs_protein': 'p.Tyr1831Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452614,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48452626-A-C',\n",
       "  'variant_hash': 'd56762821c758c8f',\n",
       "  'vcf_string': '15\\t48452626\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452626A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5481T>G',\n",
       "  'hgvs_protein': 'p.Thr1827=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452626,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48452637-A-G',\n",
       "  'variant_hash': 'f5fd77a155cc2928',\n",
       "  'vcf_string': '15\\t48452637\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452637A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5470T>C',\n",
       "  'hgvs_protein': 'p.Cys1824Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452637,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452641-G-A',\n",
       "  'variant_hash': 'e5b5fa4bfe4e05c2',\n",
       "  'vcf_string': '15\\t48452641\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452641G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5466C>T',\n",
       "  'hgvs_protein': 'p.Ala1822=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452641,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452644-G-A',\n",
       "  'variant_hash': '90d5029b42dc16b6',\n",
       "  'vcf_string': '15\\t48452644\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452644G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5463C>T',\n",
       "  'hgvs_protein': 'p.Asn1821=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452644,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452654-C-T',\n",
       "  'variant_hash': 'e8b772739563384e',\n",
       "  'vcf_string': '15\\t48452654\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452654C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5453G>A',\n",
       "  'hgvs_protein': 'p.Cys1818Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452654,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452655-A-G',\n",
       "  'variant_hash': 'f19c95b77f7a5c77',\n",
       "  'vcf_string': '15\\t48452655\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452655A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5452T>C',\n",
       "  'hgvs_protein': 'p.Cys1818Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452655,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48452665-G-A',\n",
       "  'variant_hash': '8971ab9a3a40441f',\n",
       "  'vcf_string': '15\\t48452665\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452665G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5442C>T',\n",
       "  'hgvs_protein': 'p.Asn1814=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452665,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48452670-G-A',\n",
       "  'variant_hash': 'e34d203b9ecd97e9',\n",
       "  'vcf_string': '15\\t48452670\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452670G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5437C>T',\n",
       "  'hgvs_protein': 'p.Gln1813Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452670,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452673-A-G',\n",
       "  'variant_hash': '9338a102c49bedf7',\n",
       "  'vcf_string': '15\\t48452673\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452673A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5434T>C',\n",
       "  'hgvs_protein': 'p.Cys1812Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452673,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48452676-C-T',\n",
       "  'variant_hash': '2bd1bb39a557db88',\n",
       "  'vcf_string': '15\\t48452676\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452676C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5431G>A',\n",
       "  'hgvs_protein': 'p.Glu1811Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452676,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452676-C-A',\n",
       "  'variant_hash': 'd19a1276e43eb8ba',\n",
       "  'vcf_string': '15\\t48452676\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452676C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5431G>T',\n",
       "  'hgvs_protein': 'p.Glu1811Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452676,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48452677-G-A',\n",
       "  'variant_hash': 'b93d4bdc0663ae0c',\n",
       "  'vcf_string': '15\\t48452677\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452677G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5430C>T',\n",
       "  'hgvs_protein': 'p.Asp1810=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452677,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452685-C-G',\n",
       "  'variant_hash': 'f3eeec0b9636bbcc',\n",
       "  'vcf_string': '15\\t48452685\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452685C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5423-1G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452685,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452686-T-C',\n",
       "  'variant_hash': '68852d716f1f4fb4',\n",
       "  'vcf_string': '15\\t48452686\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452686T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5423-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452686,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452699-G-T',\n",
       "  'variant_hash': '6b1928de0fd0041f',\n",
       "  'vcf_string': '15\\t48452699\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452699G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5423-15C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452699,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456635-A-G',\n",
       "  'variant_hash': '7b71d11730ee9cee',\n",
       "  'vcf_string': '15\\t48456635\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456635A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5422+2T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456635,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48456636-C-T',\n",
       "  'variant_hash': '1b091ee498bbd535',\n",
       "  'vcf_string': '15\\t48456636\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456636C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5422+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456636,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48456642-C-A',\n",
       "  'variant_hash': '7706ec1bfbc8e1ab',\n",
       "  'vcf_string': '15\\t48456642\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456642C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5417G>T',\n",
       "  'hgvs_protein': 'p.Cys1806Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456642,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48456643-A-G',\n",
       "  'variant_hash': '4208078819dcdc66',\n",
       "  'vcf_string': '15\\t48456643\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456643A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5416T>C',\n",
       "  'hgvs_protein': 'p.Cys1806Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456643,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48456647-C-T',\n",
       "  'variant_hash': '31a744dfef5eeccf',\n",
       "  'vcf_string': '15\\t48456647\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456647C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5412G>A',\n",
       "  'hgvs_protein': 'p.Leu1804=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456647,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456650-C-T',\n",
       "  'variant_hash': 'bbbe100625a143e1',\n",
       "  'vcf_string': '15\\t48456650\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456650C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5409G>A',\n",
       "  'hgvs_protein': 'p.Leu1803=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456650,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456674-C-G',\n",
       "  'variant_hash': '5e01ae31c20acfd2',\n",
       "  'vcf_string': '15\\t48456674\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456674C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5385G>C',\n",
       "  'hgvs_protein': 'p.Val1795=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456674,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456677-T-C',\n",
       "  'variant_hash': '52b62da23300aa7a',\n",
       "  'vcf_string': '15\\t48456677\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456677T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5382A>G',\n",
       "  'hgvs_protein': 'p.Pro1794=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456677,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456682-A-G',\n",
       "  'variant_hash': '3a826ccefc0151f3',\n",
       "  'vcf_string': '15\\t48456682\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456682A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5377T>C',\n",
       "  'hgvs_protein': 'p.Cys1793Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456682,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48456687-C-T',\n",
       "  'variant_hash': '82fa0e1efb12d4a6',\n",
       "  'vcf_string': '15\\t48456687\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456687C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5372G>A',\n",
       "  'hgvs_protein': 'p.Cys1791Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456687,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48456688-A-G',\n",
       "  'variant_hash': 'a9cb639aa336c0a9',\n",
       "  'vcf_string': '15\\t48456688\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456688A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5371T>C',\n",
       "  'hgvs_protein': 'p.Cys1791Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456688,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48456691-G-A',\n",
       "  'variant_hash': 'cce9ac3a6920a544',\n",
       "  'vcf_string': '15\\t48456691\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456691G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5368C>T',\n",
       "  'hgvs_protein': 'p.Arg1790Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456691,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48456714-C-T',\n",
       "  'variant_hash': '3df6cb2a6b7cebb1',\n",
       "  'vcf_string': '15\\t48456714\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456714C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5345G>A',\n",
       "  'hgvs_protein': 'p.Cys1782Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456714,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48456716-C-T',\n",
       "  'variant_hash': '8e3534d6c10130b6',\n",
       "  'vcf_string': '15\\t48456716\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456716C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5343G>A',\n",
       "  'hgvs_protein': 'p.Val1781=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456716,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48456719-T-G',\n",
       "  'variant_hash': '75dcb06cadc8e6d3',\n",
       "  'vcf_string': '15\\t48456719\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456719T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5340A>C',\n",
       "  'hgvs_protein': 'p.Gly1780=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456719,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456729-C-T',\n",
       "  'variant_hash': '1d4f6cb01c219abb',\n",
       "  'vcf_string': '15\\t48456729\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456729C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5330G>A',\n",
       "  'hgvs_protein': 'p.Cys1777Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456729,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48456731-G-A',\n",
       "  'variant_hash': '7aae48d441623547',\n",
       "  'vcf_string': '15\\t48456731\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456731G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5328C>T',\n",
       "  'hgvs_protein': 'p.Val1776=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456731,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456750-C-T',\n",
       "  'variant_hash': '9d63d1818020bea7',\n",
       "  'vcf_string': '15\\t48456750\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456750C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5309G>A',\n",
       "  'hgvs_protein': 'p.Cys1770Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456750,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48456758-A-G',\n",
       "  'variant_hash': '39c24bc16c59f963',\n",
       "  'vcf_string': '15\\t48456758\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456758A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5301T>C',\n",
       "  'hgvs_protein': 'p.Ile1767=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456758,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456772-G-A',\n",
       "  'variant_hash': '0a69ad91be3818e5',\n",
       "  'vcf_string': '15\\t48456772\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456772G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5297-10C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456772,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456781-T-C',\n",
       "  'variant_hash': 'ed24ae0569c6ea16',\n",
       "  'vcf_string': '15\\t48456781\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456781T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5297-19A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456781,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460232-C-T',\n",
       "  'variant_hash': '8effcd7c50fd6e83',\n",
       "  'vcf_string': '15\\t48460232\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460232C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5296+14G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460232,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48460233-C-T',\n",
       "  'variant_hash': '143bcb2a51b31e34',\n",
       "  'vcf_string': '15\\t48460233\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460233C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5296+13G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460233,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460234-G-A',\n",
       "  'variant_hash': '08f5a522fd0a9900',\n",
       "  'vcf_string': '15\\t48460234\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460234G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5296+12C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460234,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460241-C-T',\n",
       "  'variant_hash': '961189ad72d3b944',\n",
       "  'vcf_string': '15\\t48460241\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460241C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5296+5G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460241,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_5th_base_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48460244-A-C',\n",
       "  'variant_hash': '42cf203b15173947',\n",
       "  'vcf_string': '15\\t48460244\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460244A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5296+2T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460244,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48460250-G-A',\n",
       "  'variant_hash': 'cce4f462fa65eb39',\n",
       "  'vcf_string': '15\\t48460250\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460250G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5292C>T',\n",
       "  'hgvs_protein': 'p.Pro1764=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460250,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460258-C-T',\n",
       "  'variant_hash': '8edb1055943790c2',\n",
       "  'vcf_string': '15\\t48460258\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460258C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5284G>A',\n",
       "  'hgvs_protein': 'p.Gly1762Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460258,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48460259-G-A',\n",
       "  'variant_hash': '1f2c2fdcd63b09d8',\n",
       "  'vcf_string': '15\\t48460259\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460259G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5283C>T',\n",
       "  'hgvs_protein': 'p.Thr1761=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460259,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460262-A-G',\n",
       "  'variant_hash': 'd25ff756bbd8b580',\n",
       "  'vcf_string': '15\\t48460262\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460262A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5280T>C',\n",
       "  'hgvs_protein': 'p.Tyr1760=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460262,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460271-G-A',\n",
       "  'variant_hash': '9e34f7cde68871a2',\n",
       "  'vcf_string': '15\\t48460271\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460271G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5271C>T',\n",
       "  'hgvs_protein': 'p.Ile1757=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460271,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460274-G-T',\n",
       "  'variant_hash': '8f4178e7df234e53',\n",
       "  'vcf_string': '15\\t48460274\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460274G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5268C>A',\n",
       "  'hgvs_protein': 'p.Val1756=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460274,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460280-G-T',\n",
       "  'variant_hash': 'ab57f31625b45d18',\n",
       "  'vcf_string': '15\\t48460280\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460280G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5262C>A',\n",
       "  'hgvs_protein': 'p.Gly1754=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460280,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460281-C-T',\n",
       "  'variant_hash': 'ed1add3f9813668f',\n",
       "  'vcf_string': '15\\t48460281\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460281C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5261G>A',\n",
       "  'hgvs_protein': 'p.Gly1754Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460281,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48460283-T-C',\n",
       "  'variant_hash': 'd42b3af9dd5af574',\n",
       "  'vcf_string': '15\\t48460283\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460283T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5259A>G',\n",
       "  'hgvs_protein': 'p.Pro1753=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460283,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48460319-T-C',\n",
       "  'variant_hash': '44d8c1276b992d9f',\n",
       "  'vcf_string': '15\\t48460319\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460319T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5225-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460319,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48460327-T-A',\n",
       "  'variant_hash': 'fbdacbc2cbc878b5',\n",
       "  'vcf_string': '15\\t48460327\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460327T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5225-10A>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460327,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460330-A-C',\n",
       "  'variant_hash': '38adce18681cb055',\n",
       "  'vcf_string': '15\\t48460330\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460330A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5225-13T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460330,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460336-G-A',\n",
       "  'variant_hash': '88a0d1942ebd9371',\n",
       "  'vcf_string': '15\\t48460336\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460336G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5225-19C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460336,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48462971-T-C',\n",
       "  'variant_hash': '3f95ae7e0043539b',\n",
       "  'vcf_string': '15\\t48462971\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48462971T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5224+111A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48462971,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48463098-A-T',\n",
       "  'variant_hash': 'b12045cb40062215',\n",
       "  'vcf_string': '15\\t48463098\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463098A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5208T>A',\n",
       "  'hgvs_protein': 'p.Cys1736Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463098,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463116-G-A',\n",
       "  'variant_hash': '9156b6e364a375d4',\n",
       "  'vcf_string': '15\\t48463116\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463116G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5190C>T',\n",
       "  'hgvs_protein': 'p.Asn1730=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463116,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463122-C-T',\n",
       "  'variant_hash': 'a589cc72b50e5150',\n",
       "  'vcf_string': '15\\t48463122\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463122C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5184G>A',\n",
       "  'hgvs_protein': 'p.Ala1728=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463122,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463123-G-A',\n",
       "  'variant_hash': '22f68ba180b1f76f',\n",
       "  'vcf_string': '15\\t48463123\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463123G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5183C>T',\n",
       "  'hgvs_protein': 'p.Ala1728Val',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463123,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463129-C-T',\n",
       "  'variant_hash': '846d1f2b84565fe7',\n",
       "  'vcf_string': '15\\t48463129\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463129C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5177G>A',\n",
       "  'hgvs_protein': 'p.Gly1726Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463129,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463148-A-G',\n",
       "  'variant_hash': 'a77390078db9e3c7',\n",
       "  'vcf_string': '15\\t48463148\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463148A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5158T>C',\n",
       "  'hgvs_protein': 'p.Cys1720Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463148,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48463185-A-G',\n",
       "  'variant_hash': '100e6179a70308ec',\n",
       "  'vcf_string': '15\\t48463185\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463185A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5121T>C',\n",
       "  'hgvs_protein': 'p.Asp1707=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463185,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463197-G-A',\n",
       "  'variant_hash': '5b81592604770668',\n",
       "  'vcf_string': '15\\t48463197\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463197G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5109C>T',\n",
       "  'hgvs_protein': 'p.Asn1703=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463197,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463203-A-C',\n",
       "  'variant_hash': 'f7b89dab8f15a146',\n",
       "  'vcf_string': '15\\t48463203\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463203A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5103T>G',\n",
       "  'hgvs_protein': 'p.Ala1701=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463203,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463206-A-G',\n",
       "  'variant_hash': '853a33fdf338b0ba',\n",
       "  'vcf_string': '15\\t48463206\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463206A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5100T>C',\n",
       "  'hgvs_protein': 'p.Tyr1700=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463206,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463207-T-C',\n",
       "  'variant_hash': '8e825445d7038ec3',\n",
       "  'vcf_string': '15\\t48463207\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463207T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5099A>G',\n",
       "  'hgvs_protein': 'p.Tyr1700Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463207,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463209-G-C',\n",
       "  'variant_hash': 'e8b0a0dd8f9743eb',\n",
       "  'vcf_string': '15\\t48463209\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463209G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5097C>G',\n",
       "  'hgvs_protein': 'p.Tyr1699Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463209,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463209-G-A',\n",
       "  'variant_hash': '8a6d3a4998863a33',\n",
       "  'vcf_string': '15\\t48463209\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463209G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5097C>T',\n",
       "  'hgvs_protein': 'p.Tyr1699=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463209,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48463210-T-C',\n",
       "  'variant_hash': '0d639b7dca96831b',\n",
       "  'vcf_string': '15\\t48463210\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463210T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5096A>G',\n",
       "  'hgvs_protein': 'p.Tyr1699Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463210,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463219-T-C',\n",
       "  'variant_hash': '6057b5e06b1e2c2c',\n",
       "  'vcf_string': '15\\t48463219\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463219T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5087A>G',\n",
       "  'hgvs_protein': 'p.Tyr1696Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463219,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463226-A-G',\n",
       "  'variant_hash': '1dc0589786233c09',\n",
       "  'vcf_string': '15\\t48463226\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463226A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5080T>C',\n",
       "  'hgvs_protein': 'p.Leu1694=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463226,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463230-T-C',\n",
       "  'variant_hash': '1b38a95c623163ba',\n",
       "  'vcf_string': '15\\t48463230\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463230T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5076A>G',\n",
       "  'hgvs_protein': 'p.Arg1692=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463230,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463241-C-T',\n",
       "  'variant_hash': 'd9f7300739c85dcd',\n",
       "  'vcf_string': '15\\t48463241\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463241C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463241,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463241-C-A',\n",
       "  'variant_hash': 'ee62f4f3537a0c17',\n",
       "  'vcf_string': '15\\t48463241\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463241C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463241,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463247-G-C',\n",
       "  'variant_hash': '0b011a9704d94a80',\n",
       "  'vcf_string': '15\\t48463247\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463247G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-7C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463247,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463252-G-T',\n",
       "  'variant_hash': '7662c893ec493261',\n",
       "  'vcf_string': '15\\t48463252\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463252G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-12C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463252,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48463252-G-A',\n",
       "  'variant_hash': 'e26cccee22bd1b3e',\n",
       "  'vcf_string': '15\\t48463252\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463252G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-12C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463252,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463253-T-A',\n",
       "  'variant_hash': '6b4cce2a454e4dc0',\n",
       "  'vcf_string': '15\\t48463253\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463253T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-13A>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463253,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463259-A-G',\n",
       "  'variant_hash': '353cfdfb4b62ea98',\n",
       "  'vcf_string': '15\\t48463259\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463259A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-19T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463259,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463275-G-A',\n",
       "  'variant_hash': 'eda534dba07213f1',\n",
       "  'vcf_string': '15\\t48463275\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463275G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-35C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463275,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48463333-T-C',\n",
       "  'variant_hash': '45572db5803079b0',\n",
       "  'vcf_string': '15\\t48463333\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463333T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-93A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463333,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48463889-T-C',\n",
       "  'variant_hash': '88436ecd7c7c6631',\n",
       "  'vcf_string': '15\\t48463889\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463889T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5065+10A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463889,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48463898-C-T',\n",
       "  'variant_hash': '395623db7180d6f5',\n",
       "  'vcf_string': '15\\t48463898\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463898C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5065+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463898,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463898-C-G',\n",
       "  'variant_hash': '6b9ff489ca9d4853',\n",
       "  'vcf_string': '15\\t48463898\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463898C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5065+1G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463898,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463898-C-A',\n",
       "  'variant_hash': 'cbafeb8a2305c632',\n",
       "  'vcf_string': '15\\t48463898\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463898C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5065+1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463898,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463904-C-A',\n",
       "  'variant_hash': '6536ba5cab562b47',\n",
       "  'vcf_string': '15\\t48463904\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463904C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5060G>T',\n",
       "  'hgvs_protein': 'p.Cys1687Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463904,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463933-G-A',\n",
       "  'variant_hash': '58193d0f496fce24',\n",
       "  'vcf_string': '15\\t48463933\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463933G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5031C>T',\n",
       "  'hgvs_protein': 'p.Asp1677=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463933,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463943-C-T',\n",
       "  'variant_hash': '48e7af65e034e830',\n",
       "  'vcf_string': '15\\t48463943\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463943C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5021G>A',\n",
       "  'hgvs_protein': 'p.Cys1674Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463943,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463949-C-T',\n",
       "  'variant_hash': '987d65fe712c3b06',\n",
       "  'vcf_string': '15\\t48463949\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463949C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5015G>A',\n",
       "  'hgvs_protein': 'p.Cys1672Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463949,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463949-C-G',\n",
       "  'variant_hash': '2a219d579cae3881',\n",
       "  'vcf_string': '15\\t48463949\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463949C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5015G>C',\n",
       "  'hgvs_protein': 'p.Cys1672Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463949,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463949-C-A',\n",
       "  'variant_hash': '1052ad6d02d0745e',\n",
       "  'vcf_string': '15\\t48463949\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463949C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5015G>T',\n",
       "  'hgvs_protein': 'p.Cys1672Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463949,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463950-A-C',\n",
       "  'variant_hash': '1041ceb2f8d58904',\n",
       "  'vcf_string': '15\\t48463950\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463950A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5014T>G',\n",
       "  'hgvs_protein': 'p.Cys1672Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463950,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463955-T-C',\n",
       "  'variant_hash': 'd7a9d94e7cba5c0a',\n",
       "  'vcf_string': '15\\t48463955\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463955T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5009A>G',\n",
       "  'hgvs_protein': 'p.Tyr1670Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463955,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48463957-G-A',\n",
       "  'variant_hash': '22f19ae1f46de1b8',\n",
       "  'vcf_string': '15\\t48463957\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463957G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5007C>T',\n",
       "  'hgvs_protein': 'p.Asn1669=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463957,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463966-G-C',\n",
       "  'variant_hash': '308c53db1b916cdb',\n",
       "  'vcf_string': '15\\t48463966\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463966G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4998C>G',\n",
       "  'hgvs_protein': 'p.Thr1666=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463966,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48463976-C-A',\n",
       "  'variant_hash': '4c3cc5fbfcf28177',\n",
       "  'vcf_string': '15\\t48463976\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463976C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4988G>T',\n",
       "  'hgvs_protein': 'p.Cys1663Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463976,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463978-T-C',\n",
       "  'variant_hash': 'e310e654f5da1884',\n",
       "  'vcf_string': '15\\t48463978\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463978T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4986A>G',\n",
       "  'hgvs_protein': 'p.Thr1662=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463978,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463984-T-C',\n",
       "  'variant_hash': '3baefaaa4182a4c4',\n",
       "  'vcf_string': '15\\t48463984\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463984T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4980A>G',\n",
       "  'hgvs_protein': 'p.Pro1660=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463984,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48464009-C-T',\n",
       "  'variant_hash': '3ea0d935d47f13df',\n",
       "  'vcf_string': '15\\t48464009\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48464009C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4955G>A',\n",
       "  'hgvs_protein': 'p.Cys1652Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48464009,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48464022-C-G',\n",
       "  'variant_hash': 'c786f81526349b49',\n",
       "  'vcf_string': '15\\t48464022\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48464022C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4943-1G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48464022,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48464025-T-C',\n",
       "  'variant_hash': 'b4fb5f6f46fc66c2',\n",
       "  'vcf_string': '15\\t48464025\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48464025T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4943-4A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48464025,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48464034-G-A',\n",
       "  'variant_hash': '6d31a8f4659fa679',\n",
       "  'vcf_string': '15\\t48464034\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48464034G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4943-13C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48464034,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48464090-T-C',\n",
       "  'variant_hash': '8ba3c191300dd3cd',\n",
       "  'vcf_string': '15\\t48464090\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48464090T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4943-69A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48464090,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465467-T-A',\n",
       "  'variant_hash': 'c835c2997b3d9847',\n",
       "  'vcf_string': '15\\t48465467\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465467T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4942+101A>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465467,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465568-C-T',\n",
       "  'variant_hash': '8c7680d8e923c24c',\n",
       "  'vcf_string': '15\\t48465568\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465568C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4942G>A',\n",
       "  'hgvs_protein': 'p.Asp1648Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465568,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465573-C-T',\n",
       "  'variant_hash': '52146fa0b8f9303f',\n",
       "  'vcf_string': '15\\t48465573\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465573C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4937G>A',\n",
       "  'hgvs_protein': 'p.Cys1646Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465573,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465574-A-T',\n",
       "  'variant_hash': '8e94ee2375bda140',\n",
       "  'vcf_string': '15\\t48465574\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465574A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4936T>A',\n",
       "  'hgvs_protein': 'p.Cys1646Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465574,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465580-G-T',\n",
       "  'variant_hash': 'f72a52bcca5c9246',\n",
       "  'vcf_string': '15\\t48465580\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465580G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4930C>A',\n",
       "  'hgvs_protein': 'p.Arg1644=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465580,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465580-G-A',\n",
       "  'variant_hash': '2b97c3b16222b499',\n",
       "  'vcf_string': '15\\t48465580\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465580G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4930C>T',\n",
       "  'hgvs_protein': 'p.Arg1644Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465580,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465584-A-G',\n",
       "  'variant_hash': 'c76958b3911fc9c6',\n",
       "  'vcf_string': '15\\t48465584\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465584A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4926T>C',\n",
       "  'hgvs_protein': 'p.Asp1642=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465584,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465605-G-C',\n",
       "  'variant_hash': 'dd37d5182e364ce3',\n",
       "  'vcf_string': '15\\t48465605\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465605G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4905C>G',\n",
       "  'hgvs_protein': 'p.Thr1635=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465605,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48465605-G-A',\n",
       "  'variant_hash': '0cc2fa6e2ea6ffda',\n",
       "  'vcf_string': '15\\t48465605\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465605G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4905C>T',\n",
       "  'hgvs_protein': 'p.Thr1635=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465605,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465619-A-T',\n",
       "  'variant_hash': '9a57cf9f20164d0c',\n",
       "  'vcf_string': '15\\t48465619\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465619A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4891T>A',\n",
       "  'hgvs_protein': 'p.Cys1631Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465619,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48465622-G-A',\n",
       "  'variant_hash': '67df1e48a74f2f49',\n",
       "  'vcf_string': '15\\t48465622\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465622G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4888C>T',\n",
       "  'hgvs_protein': 'p.Gln1630Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465622,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48465635-G-C',\n",
       "  'variant_hash': 'a7d6ecccc8508f3a',\n",
       "  'vcf_string': '15\\t48465635\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465635G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4875C>G',\n",
       "  'hgvs_protein': 'p.Thr1625=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465635,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465646-A-G',\n",
       "  'variant_hash': '7ec076a0b7e8832c',\n",
       "  'vcf_string': '15\\t48465646\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465646A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4864T>C',\n",
       "  'hgvs_protein': 'p.Cys1622Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465646,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465647-T-C',\n",
       "  'variant_hash': 'bb467447c2cb2a7b',\n",
       "  'vcf_string': '15\\t48465647\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465647T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4863A>G',\n",
       "  'hgvs_protein': 'p.Lys1621=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465647,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465679-G-A',\n",
       "  'variant_hash': '5a9ecb0a5d537f20',\n",
       "  'vcf_string': '15\\t48465679\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465679G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4831C>T',\n",
       "  'hgvs_protein': 'p.Gln1611Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465679,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465686-A-G',\n",
       "  'variant_hash': 'a3b7423f7167193b',\n",
       "  'vcf_string': '15\\t48465686\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465686A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4824T>C',\n",
       "  'hgvs_protein': 'p.Asp1608=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465686,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465695-T-C',\n",
       "  'variant_hash': '3c5c86ded724da77',\n",
       "  'vcf_string': '15\\t48465695\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465695T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4817-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465695,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48465788-A-G',\n",
       "  'variant_hash': '64c4ed6840071634',\n",
       "  'vcf_string': '15\\t48465788\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465788A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4816+2T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465788,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465789-C-A',\n",
       "  'variant_hash': 'fc596da1ae690599',\n",
       "  'vcf_string': '15\\t48465789\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465789C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4816+1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465789,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465803-G-A',\n",
       "  'variant_hash': 'fbb3c0bc3dc6b576',\n",
       "  'vcf_string': '15\\t48465803\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465803G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4803C>T',\n",
       "  'hgvs_protein': 'p.Thr1601=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465803,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465815-T-G',\n",
       "  'variant_hash': 'cc65c5b3deeef524',\n",
       "  'vcf_string': '15\\t48465815\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465815T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4791A>C',\n",
       "  'hgvs_protein': 'p.Pro1597=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465815,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465820-G-A',\n",
       "  'variant_hash': '845c15904220312e',\n",
       "  'vcf_string': '15\\t48465820\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465820G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4786C>T',\n",
       "  'hgvs_protein': 'p.Arg1596Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465820,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48465825-C-T',\n",
       "  'variant_hash': 'c0227e6e574fb152',\n",
       "  'vcf_string': '15\\t48465825\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465825C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4781G>A',\n",
       "  'hgvs_protein': 'p.Gly1594Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465825,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465840-C-A',\n",
       "  'variant_hash': 'fd18b58eb7875a07',\n",
       "  'vcf_string': '15\\t48465840\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465840C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4766G>T',\n",
       "  'hgvs_protein': 'p.Cys1589Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465840,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465857-G-A',\n",
       "  'variant_hash': '607db87701b1f48e',\n",
       "  'vcf_string': '15\\t48465857\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465857G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4749C>T',\n",
       "  'hgvs_protein': 'p.Ser1583=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465857,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465861-A-G',\n",
       "  'variant_hash': 'b3a28ddbe4d325d4',\n",
       "  'vcf_string': '15\\t48465861\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465861A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4748-3T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465861,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465864-G-A',\n",
       "  'variant_hash': '3896b2a96bb99e3f',\n",
       "  'vcf_string': '15\\t48465864\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465864G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4748-6C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465864,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465869-A-G',\n",
       "  'variant_hash': '6d8410a8dda2f1cc',\n",
       "  'vcf_string': '15\\t48465869\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465869A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4748-11T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465869,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48465870-A-T',\n",
       "  'variant_hash': 'c1ef7aad44d1c0d6',\n",
       "  'vcf_string': '15\\t48465870\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465870A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4748-12T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465870,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465935-A-C',\n",
       "  'variant_hash': 'ac7cdb9e913a2400',\n",
       "  'vcf_string': '15\\t48465935\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465935A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4748-77T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465935,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48467870-G-A',\n",
       "  'variant_hash': '3ae7af9c98da1503',\n",
       "  'vcf_string': '15\\t48467870\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467870G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4747+68C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467870,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48467954-A-T',\n",
       "  'variant_hash': '152547831d8ee034',\n",
       "  'vcf_string': '15\\t48467954\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467954A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4731T>A',\n",
       "  'hgvs_protein': 'p.Cys1577Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467954,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48467966-A-G',\n",
       "  'variant_hash': '08fddad90f850f95',\n",
       "  'vcf_string': '15\\t48467966\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467966A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4719T>C',\n",
       "  'hgvs_protein': 'p.Pro1573=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467966,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48467972-A-T',\n",
       "  'variant_hash': '457e19590f1bd67b',\n",
       "  'vcf_string': '15\\t48467972\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467972A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4713T>A',\n",
       "  'hgvs_protein': 'p.Gly1571=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467972,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48467975-C-T',\n",
       "  'variant_hash': 'f93b38d281ef3f94',\n",
       "  'vcf_string': '15\\t48467975\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467975C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4710G>A',\n",
       "  'hgvs_protein': 'p.Trp1570Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467975,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48467978-G-C',\n",
       "  'variant_hash': '474601fe992c868d',\n",
       "  'vcf_string': '15\\t48467978\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467978G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4707C>G',\n",
       "  'hgvs_protein': 'p.Ala1569=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467978,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48467990-A-T',\n",
       "  'variant_hash': '5f0419266fd7e0b2',\n",
       "  'vcf_string': '15\\t48467990\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467990A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4695T>A',\n",
       "  'hgvs_protein': 'p.Ser1565=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467990,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48467996-G-C',\n",
       "  'variant_hash': 'da3cfdb2e8861e77',\n",
       "  'vcf_string': '15\\t48467996\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467996G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4689C>G',\n",
       "  'hgvs_protein': 'p.Cys1563Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467996,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48467997-C-T',\n",
       "  'variant_hash': 'b3c7540650a0324f',\n",
       "  'vcf_string': '15\\t48467997\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467997C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4688G>A',\n",
       "  'hgvs_protein': 'p.Cys1563Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467997,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468001-A-G',\n",
       "  'variant_hash': '1237ab7696cc6eda',\n",
       "  'vcf_string': '15\\t48468001\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468001A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4684T>C',\n",
       "  'hgvs_protein': 'p.Cys1562Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468001,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468002-G-A',\n",
       "  'variant_hash': '84f9287dde95ce72',\n",
       "  'vcf_string': '15\\t48468002\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468002G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4683C>T',\n",
       "  'hgvs_protein': 'p.Ser1561=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468002,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48468020-A-G',\n",
       "  'variant_hash': '368dd5d2421f2d33',\n",
       "  'vcf_string': '15\\t48468020\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468020A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4665T>C',\n",
       "  'hgvs_protein': 'p.Val1555=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468020,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48468023-T-G',\n",
       "  'variant_hash': 'be3fcea41ca7546b',\n",
       "  'vcf_string': '15\\t48468023\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468023T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4662A>C',\n",
       "  'hgvs_protein': 'p.Gly1554=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468023,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48468035-G-A',\n",
       "  'variant_hash': '2d904cc046a53663',\n",
       "  'vcf_string': '15\\t48468035\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468035G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4650C>T',\n",
       "  'hgvs_protein': 'p.Ser1550=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468035,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468045-G-A',\n",
       "  'variant_hash': '4e1c4f8e04903e17',\n",
       "  'vcf_string': '15\\t48468045\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468045G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4640C>T',\n",
       "  'hgvs_protein': 'p.Thr1547Ile',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468045,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48468064-G-A',\n",
       "  'variant_hash': '75e08399fc954280',\n",
       "  'vcf_string': '15\\t48468064\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468064G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4621C>T',\n",
       "  'hgvs_protein': 'p.Arg1541Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468064,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48468065-A-C',\n",
       "  'variant_hash': '7cc0e2b3746e85cf',\n",
       "  'vcf_string': '15\\t48468065\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468065A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4620T>G',\n",
       "  'hgvs_protein': 'p.Pro1540=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468065,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468070-G-A',\n",
       "  'variant_hash': '8b0bb973ea6cc63b',\n",
       "  'vcf_string': '15\\t48468070\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468070G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4615C>T',\n",
       "  'hgvs_protein': 'p.Arg1539Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468070,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48468080-A-C',\n",
       "  'variant_hash': 'd243ff9adb8b5a3e',\n",
       "  'vcf_string': '15\\t48468080\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468080A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4605T>G',\n",
       "  'hgvs_protein': 'p.Tyr1535Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468080,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48468092-A-G',\n",
       "  'variant_hash': 'd8053905401d03a6',\n",
       "  'vcf_string': '15\\t48468092\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468092A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4593T>C',\n",
       "  'hgvs_protein': 'p.Ser1531=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468092,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468096-C-G',\n",
       "  'variant_hash': '268434c3977700a2',\n",
       "  'vcf_string': '15\\t48468096\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468096C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4589G>C',\n",
       "  'hgvs_protein': 'p.Arg1530Pro',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468096,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48468097-G-A',\n",
       "  'variant_hash': 'be3f1c9a58c39bd8',\n",
       "  'vcf_string': '15\\t48468097\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468097G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4588C>T',\n",
       "  'hgvs_protein': 'p.Arg1530Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468097,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468107-T-C',\n",
       "  'variant_hash': 'a81aba4d9ced414d',\n",
       "  'vcf_string': '15\\t48468107\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468107T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4583-5A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468107,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48468110-C-G',\n",
       "  'variant_hash': 'a3710a1951683540',\n",
       "  'vcf_string': '15\\t48468110\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468110C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4583-8G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468110,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48468113-T-C',\n",
       "  'variant_hash': '28428e4177e82e80',\n",
       "  'vcf_string': '15\\t48468113\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468113T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4583-11A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468113,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468114-A-G',\n",
       "  'variant_hash': '0dfa8507441984b1',\n",
       "  'vcf_string': '15\\t48468114\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468114A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4583-12T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468114,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468115-T-C',\n",
       "  'variant_hash': '989be7a1fffa60b1',\n",
       "  'vcf_string': '15\\t48468115\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468115T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4583-13A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468115,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468194-G-A',\n",
       "  'variant_hash': '18df21506df47cde',\n",
       "  'vcf_string': '15\\t48468194\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468194G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4583-92C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468194,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468348-T-C',\n",
       "  'variant_hash': 'fe6c41b61edf422f',\n",
       "  'vcf_string': '15\\t48468348\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468348T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4582+64A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468348,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48468411-C-T',\n",
       "  'variant_hash': 'ec51f34479475f00',\n",
       "  'vcf_string': '15\\t48468411\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468411C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4582+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468411,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468418-A-G',\n",
       "  'variant_hash': '4a7e6a672cca2ec2',\n",
       "  'vcf_string': '15\\t48468418\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468418A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4576T>C',\n",
       "  'hgvs_protein': 'p.Cys1526Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468418,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468425-T-A',\n",
       "  'variant_hash': '7d922de341e58f48',\n",
       "  'vcf_string': '15\\t48468425\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468425T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4569A>T',\n",
       "  'hgvs_protein': 'p.Arg1523=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468425,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468427-G-A',\n",
       "  'variant_hash': 'a9ea487841853a78',\n",
       "  'vcf_string': '15\\t48468427\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468427G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4567C>T',\n",
       "  'hgvs_protein': 'p.Arg1523Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468427,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468455-G-C',\n",
       "  'variant_hash': 'df04245f0016fde5',\n",
       "  'vcf_string': '15\\t48468455\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468455G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4539C>G',\n",
       "  'hgvs_protein': 'p.Cys1513Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468455,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468456-C-T',\n",
       "  'variant_hash': 'bd19a31c893d6c7a',\n",
       "  'vcf_string': '15\\t48468456\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468456C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4538G>A',\n",
       "  'hgvs_protein': 'p.Cys1513Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468456,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48468457-A-C',\n",
       "  'variant_hash': '43f28b67aed6306e',\n",
       "  'vcf_string': '15\\t48468457\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468457A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4537T>G',\n",
       "  'hgvs_protein': 'p.Cys1513Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468457,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468462-C-T',\n",
       "  'variant_hash': '7025873c7615cd03',\n",
       "  'vcf_string': '15\\t48468462\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468462C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4532G>A',\n",
       "  'hgvs_protein': 'p.Cys1511Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468462,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468463-A-G',\n",
       "  'variant_hash': '2b2caf359d5e9fd8',\n",
       "  'vcf_string': '15\\t48468463\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468463A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4531T>C',\n",
       "  'hgvs_protein': 'p.Cys1511Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468463,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468464-G-A',\n",
       "  'variant_hash': '1452080d3d212673',\n",
       "  'vcf_string': '15\\t48468464\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468464G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4530C>T',\n",
       "  'hgvs_protein': 'p.Ile1510=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468464,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468467-A-G',\n",
       "  'variant_hash': 'b16dd4680d19e758',\n",
       "  'vcf_string': '15\\t48468467\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468467A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4527T>C',\n",
       "  'hgvs_protein': 'p.Tyr1509=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468467,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468468-T-C',\n",
       "  'variant_hash': '2cfaf5b9d145eb09',\n",
       "  'vcf_string': '15\\t48468468\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468468T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4526A>G',\n",
       "  'hgvs_protein': 'p.Tyr1509Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468468,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48468474-C-T',\n",
       "  'variant_hash': 'a1ffdccc02cf75f3',\n",
       "  'vcf_string': '15\\t48468474\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468474C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4520G>A',\n",
       "  'hgvs_protein': 'p.Gly1507Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468474,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468482-G-A',\n",
       "  'variant_hash': 'aa165c0dc4a384d7',\n",
       "  'vcf_string': '15\\t48468482\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468482G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4512C>T',\n",
       "  'hgvs_protein': 'p.Asn1504=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468482,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468489-C-T',\n",
       "  'variant_hash': 'b2750b9f383d6700',\n",
       "  'vcf_string': '15\\t48468489\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468489C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4505G>A',\n",
       "  'hgvs_protein': 'p.Cys1502Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468489,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468490-A-G',\n",
       "  'variant_hash': '4299d47deff80bf5',\n",
       "  'vcf_string': '15\\t48468490\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468490A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4504T>C',\n",
       "  'hgvs_protein': 'p.Cys1502Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468490,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468491-G-A',\n",
       "  'variant_hash': 'f5f85d3d07c0c253',\n",
       "  'vcf_string': '15\\t48468491\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468491G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4503C>T',\n",
       "  'hgvs_protein': 'p.Asn1501=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468491,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468503-G-C',\n",
       "  'variant_hash': 'bae116545302f1b7',\n",
       "  'vcf_string': '15\\t48468503\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468503G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4491C>G',\n",
       "  'hgvs_protein': 'p.Cys1497Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468503,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48468504-C-T',\n",
       "  'variant_hash': 'b7ac43c90e048763',\n",
       "  'vcf_string': '15\\t48468504\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468504C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4490G>A',\n",
       "  'hgvs_protein': 'p.Cys1497Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468504,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468506-C-T',\n",
       "  'variant_hash': '92f9f3093c3cc9b1',\n",
       "  'vcf_string': '15\\t48468506\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468506C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4488G>A',\n",
       "  'hgvs_protein': 'p.Thr1496=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468506,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468509-G-A',\n",
       "  'variant_hash': '770bafd0537512e6',\n",
       "  'vcf_string': '15\\t48468509\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468509G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4485C>T',\n",
       "  'hgvs_protein': 'p.Thr1495=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468509,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468522-C-A',\n",
       "  'variant_hash': 'a4812ff9f07d510c',\n",
       "  'vcf_string': '15\\t48468522\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468522C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4472G>T',\n",
       "  'hgvs_protein': 'p.Cys1491Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468522,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468526-C-T',\n",
       "  'variant_hash': 'af41e58771851ee9',\n",
       "  'vcf_string': '15\\t48468526\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468526C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4468G>A',\n",
       "  'hgvs_protein': 'p.Glu1490Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468526,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468527-A-T',\n",
       "  'variant_hash': '72ded8e896e27b1c',\n",
       "  'vcf_string': '15\\t48468527\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468527A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4467T>A',\n",
       "  'hgvs_protein': 'p.Asn1489Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468527,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468534-T-C',\n",
       "  'variant_hash': '215b9646262fcbfc',\n",
       "  'vcf_string': '15\\t48468534\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468534T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4460A>G',\n",
       "  'hgvs_protein': 'p.Asp1487Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468534,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468542-C-T',\n",
       "  'variant_hash': 'a24a35451c082939',\n",
       "  'vcf_string': '15\\t48468542\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468542C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4460-8G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468542,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468543-G-A',\n",
       "  'variant_hash': '8a819441c13e28b5',\n",
       "  'vcf_string': '15\\t48468543\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468543G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4460-9C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468543,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468553-T-C',\n",
       "  'variant_hash': '286c2d37811b5b10',\n",
       "  'vcf_string': '15\\t48468553\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468553T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4460-19A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468553,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48468700-G-A',\n",
       "  'variant_hash': '3ac45391ebef82e6',\n",
       "  'vcf_string': '15\\t48468700\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468700G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4460-166C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468700,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48470634-C-T',\n",
       "  'variant_hash': 'dd3642ad2b8070ed',\n",
       "  'vcf_string': '15\\t48470634\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470634C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4459G>A',\n",
       "  'hgvs_protein': 'p.Asp1487Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470634,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48470647-G-A',\n",
       "  'variant_hash': 'd2d9281ad517dd1a',\n",
       "  'vcf_string': '15\\t48470647\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470647G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4446C>T',\n",
       "  'hgvs_protein': 'p.Gly1482=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470647,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470661-G-A',\n",
       "  'variant_hash': '187190c1460aa766',\n",
       "  'vcf_string': '15\\t48470661\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470661G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4432C>T',\n",
       "  'hgvs_protein': 'p.Leu1478=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470661,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470665-G-A',\n",
       "  'variant_hash': 'f5e006452c2d2ce6',\n",
       "  'vcf_string': '15\\t48470665\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470665G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4428C>T',\n",
       "  'hgvs_protein': 'p.Tyr1476=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470665,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470679-A-G',\n",
       "  'variant_hash': '3bd221f9b6bb0a46',\n",
       "  'vcf_string': '15\\t48470679\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470679A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4414T>C',\n",
       "  'hgvs_protein': 'p.Cys1472Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470679,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48470680-C-T',\n",
       "  'variant_hash': '2f5a5f0574a60644',\n",
       "  'vcf_string': '15\\t48470680\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470680C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4413G>A',\n",
       "  'hgvs_protein': 'p.Glu1471=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470680,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470684-C-T',\n",
       "  'variant_hash': '0d05f8dba29601ae',\n",
       "  'vcf_string': '15\\t48470684\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470684C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4409G>A',\n",
       "  'hgvs_protein': 'p.Cys1470Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470684,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48470689-G-A',\n",
       "  'variant_hash': 'e0d87ee7367ffb59',\n",
       "  'vcf_string': '15\\t48470689\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470689G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4404C>T',\n",
       "  'hgvs_protein': 'p.Phe1468=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470689,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470692-C-A',\n",
       "  'variant_hash': '786723d638967d35',\n",
       "  'vcf_string': '15\\t48470692\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470692C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4401G>T',\n",
       "  'hgvs_protein': 'p.Leu1467=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470692,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470711-C-G',\n",
       "  'variant_hash': '354bd4df11a4b5da',\n",
       "  'vcf_string': '15\\t48470711\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470711C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4382G>C',\n",
       "  'hgvs_protein': 'p.Cys1461Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470711,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470711-C-A',\n",
       "  'variant_hash': '09fcd24a71a5bcb4',\n",
       "  'vcf_string': '15\\t48470711\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470711C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4382G>T',\n",
       "  'hgvs_protein': 'p.Cys1461Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470711,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470722-G-A',\n",
       "  'variant_hash': 'd9d59f719389c076',\n",
       "  'vcf_string': '15\\t48470722\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470722G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4371C>T',\n",
       "  'hgvs_protein': 'p.Val1457=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470722,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470726-C-T',\n",
       "  'variant_hash': '7aa35ef36b8f30e8',\n",
       "  'vcf_string': '15\\t48470726\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470726C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4367G>A',\n",
       "  'hgvs_protein': 'p.Cys1456Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470726,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48470726-C-G',\n",
       "  'variant_hash': 'b92a37bf03311cc2',\n",
       "  'vcf_string': '15\\t48470726\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470726C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4367G>C',\n",
       "  'hgvs_protein': 'p.Cys1456Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470726,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470726-C-A',\n",
       "  'variant_hash': 'e3ea3b50c3dcc74c',\n",
       "  'vcf_string': '15\\t48470726\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470726C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4367G>T',\n",
       "  'hgvs_protein': 'p.Cys1456Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470726,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48470734-C-T',\n",
       "  'variant_hash': 'a7a7cf9ecda9f80c',\n",
       "  'vcf_string': '15\\t48470734\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470734C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4359G>A',\n",
       "  'hgvs_protein': 'p.Pro1453=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470734,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470744-C-A',\n",
       "  'variant_hash': '017d757955514e93',\n",
       "  'vcf_string': '15\\t48470744\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470744C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4349G>T',\n",
       "  'hgvs_protein': 'p.Cys1450Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470744,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470745-A-G',\n",
       "  'variant_hash': '7129571835b76752',\n",
       "  'vcf_string': '15\\t48470745\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470745A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4348T>C',\n",
       "  'hgvs_protein': 'p.Cys1450Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470745,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470757-C-T',\n",
       "  'variant_hash': '698357c052284d40',\n",
       "  'vcf_string': '15\\t48470757\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470757C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470757,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470758-T-C',\n",
       "  'variant_hash': '8fa365f1b44ba706',\n",
       "  'vcf_string': '15\\t48470758\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470758T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470758,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470763-G-A',\n",
       "  'variant_hash': '45ff0e3eb017e4ec',\n",
       "  'vcf_string': '15\\t48470763\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470763G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-7C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470763,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470765-G-A',\n",
       "  'variant_hash': 'df1dfde3cfb7a8f5',\n",
       "  'vcf_string': '15\\t48470765\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470765G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-9C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470765,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470768-G-A',\n",
       "  'variant_hash': '5efc87a18aa7d890',\n",
       "  'vcf_string': '15\\t48470768\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470768G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-12C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470768,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470770-A-G',\n",
       "  'variant_hash': '0fd3a2acfd310455',\n",
       "  'vcf_string': '15\\t48470770\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470770A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-14T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470770,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470811-A-T',\n",
       "  'variant_hash': 'dd85062938ec40f0',\n",
       "  'vcf_string': '15\\t48470811\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470811A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-55T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470811,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48472544-C-A',\n",
       "  'variant_hash': 'fcef3dfcc9cc0c9c',\n",
       "  'vcf_string': '15\\t48472544\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472544C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4336+7G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472544,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472551-C-T',\n",
       "  'variant_hash': 'eeaca513de416074',\n",
       "  'vcf_string': '15\\t48472551\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472551C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4336G>A',\n",
       "  'hgvs_protein': 'p.Asp1446Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472551,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472556-C-T',\n",
       "  'variant_hash': 'bdd7d7e8d6133f4b',\n",
       "  'vcf_string': '15\\t48472556\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472556C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4331G>A',\n",
       "  'hgvs_protein': 'p.Cys1444Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472556,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472557-A-T',\n",
       "  'variant_hash': '83127a9bea85466a',\n",
       "  'vcf_string': '15\\t48472557\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472557A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4330T>A',\n",
       "  'hgvs_protein': 'p.Cys1444Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472557,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48472567-G-A',\n",
       "  'variant_hash': 'd7a1968181c35ded',\n",
       "  'vcf_string': '15\\t48472567\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472567G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4320C>T',\n",
       "  'hgvs_protein': 'p.Asp1440=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472567,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472579-C-T',\n",
       "  'variant_hash': '89431db5d1487e1d',\n",
       "  'vcf_string': '15\\t48472579\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472579C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4308G>A',\n",
       "  'hgvs_protein': 'p.Val1436=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472579,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472594-G-T',\n",
       "  'variant_hash': '02b6bc30cf4de63e',\n",
       "  'vcf_string': '15\\t48472594\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472594G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4293C>A',\n",
       "  'hgvs_protein': 'p.Cys1431Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472594,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48472594-G-C',\n",
       "  'variant_hash': 'b706597d30e370e5',\n",
       "  'vcf_string': '15\\t48472594\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472594G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4293C>G',\n",
       "  'hgvs_protein': 'p.Cys1431Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472594,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472594-G-A',\n",
       "  'variant_hash': '9d2d3a8bd3719861',\n",
       "  'vcf_string': '15\\t48472594\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472594G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4293C>T',\n",
       "  'hgvs_protein': 'p.Cys1431=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472594,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472595-C-T',\n",
       "  'variant_hash': '7b5864c4b7a0931d',\n",
       "  'vcf_string': '15\\t48472595\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472595C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4292G>A',\n",
       "  'hgvs_protein': 'p.Cys1431Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472595,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48472601-C-T',\n",
       "  'variant_hash': '5a912fe76dae5e5d',\n",
       "  'vcf_string': '15\\t48472601\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472601C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4286G>A',\n",
       "  'hgvs_protein': 'p.Cys1429Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472601,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472606-G-A',\n",
       "  'variant_hash': '37912a3ee88700c0',\n",
       "  'vcf_string': '15\\t48472606\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472606G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4281C>T',\n",
       "  'hgvs_protein': 'p.Tyr1427=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472606,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472607-T-C',\n",
       "  'variant_hash': '4cda1b9916668174',\n",
       "  'vcf_string': '15\\t48472607\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472607T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4280A>G',\n",
       "  'hgvs_protein': 'p.Tyr1427Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472607,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48472615-T-C',\n",
       "  'variant_hash': 'dd2b73298a98dc2a',\n",
       "  'vcf_string': '15\\t48472615\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472615T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4272A>G',\n",
       "  'hgvs_protein': 'p.Pro1424=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472615,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472618-T-A',\n",
       "  'variant_hash': 'c8bbd29267327747',\n",
       "  'vcf_string': '15\\t48472618\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472618T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4269A>T',\n",
       "  'hgvs_protein': 'p.Ala1423=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472618,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472628-C-T',\n",
       "  'variant_hash': '3af43e133de1e8ca',\n",
       "  'vcf_string': '15\\t48472628\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472628C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4259G>A',\n",
       "  'hgvs_protein': 'p.Cys1420Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472628,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472628-C-A',\n",
       "  'variant_hash': 'a4be30c986e2b604',\n",
       "  'vcf_string': '15\\t48472628\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472628C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4259G>T',\n",
       "  'hgvs_protein': 'p.Cys1420Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472628,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472632-G-A',\n",
       "  'variant_hash': 'e810b1d218d1409a',\n",
       "  'vcf_string': '15\\t48472632\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472632G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4255C>T',\n",
       "  'hgvs_protein': 'p.Gln1419Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472632,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48472644-A-G',\n",
       "  'variant_hash': 'bd25538d192561c5',\n",
       "  'vcf_string': '15\\t48472644\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472644A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4243T>C',\n",
       "  'hgvs_protein': 'p.Cys1415Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472644,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48472654-G-A',\n",
       "  'variant_hash': 'dac1dab9815937cb',\n",
       "  'vcf_string': '15\\t48472654\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472654G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4233C>T',\n",
       "  'hgvs_protein': 'p.Asn1411=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472654,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472665-A-G',\n",
       "  'variant_hash': '4c77ec6f5d437e70',\n",
       "  'vcf_string': '15\\t48472665\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472665A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4222T>C',\n",
       "  'hgvs_protein': 'p.Cys1408Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472665,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48472669-A-G',\n",
       "  'variant_hash': '8373eb07f5e36b11',\n",
       "  'vcf_string': '15\\t48472669\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472669A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4218T>C',\n",
       "  'hgvs_protein': 'p.Asp1406=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472669,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472670-T-C',\n",
       "  'variant_hash': '8bcc6e02f0f756e3',\n",
       "  'vcf_string': '15\\t48472670\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472670T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4217A>G',\n",
       "  'hgvs_protein': 'p.Asp1406Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472670,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48472677-C-T',\n",
       "  'variant_hash': '7afa5916878c3c2b',\n",
       "  'vcf_string': '15\\t48472677\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472677C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4211-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472677,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472684-C-G',\n",
       "  'variant_hash': 'c58b89ca985cf326',\n",
       "  'vcf_string': '15\\t48472684\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472684C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4211-8G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472684,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472694-C-T',\n",
       "  'variant_hash': 'd9f0d24ee8695958',\n",
       "  'vcf_string': '15\\t48472694\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472694C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4211-18G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472694,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472745-A-G',\n",
       "  'variant_hash': '4707ce67185f4d08',\n",
       "  'vcf_string': '15\\t48472745\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472745A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4211-69T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472745,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48472836-A-C',\n",
       "  'variant_hash': 'befefdf5fec7a173',\n",
       "  'vcf_string': '15\\t48472836\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472836A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4211-160T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472836,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48472847-T-G',\n",
       "  'variant_hash': '12ef333ae67df609',\n",
       "  'vcf_string': '15\\t48472847\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472847T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4211-171A>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472847,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48474053-G-C',\n",
       "  'variant_hash': '050e09c5844b22df',\n",
       "  'vcf_string': '15\\t48474053\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474053G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4210+202C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474053,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48474244-C-T',\n",
       "  'variant_hash': '657e8d6e68e755d6',\n",
       "  'vcf_string': '15\\t48474244\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474244C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4210+11G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474244,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474245-G-A',\n",
       "  'variant_hash': 'd80d70ef4e5e123e',\n",
       "  'vcf_string': '15\\t48474245\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474245G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4210+10C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474245,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48474254-C-T',\n",
       "  'variant_hash': 'c44d24267b4a6459',\n",
       "  'vcf_string': '15\\t48474254\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474254C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4210+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474254,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474260-C-T',\n",
       "  'variant_hash': '33d3c462276a2236',\n",
       "  'vcf_string': '15\\t48474260\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474260C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4205G>A',\n",
       "  'hgvs_protein': 'p.Cys1402Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474260,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474271-A-G',\n",
       "  'variant_hash': '7e638923d54847bd',\n",
       "  'vcf_string': '15\\t48474271\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474271A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4194T>C',\n",
       "  'hgvs_protein': 'p.Asp1398=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474271,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474280-G-A',\n",
       "  'variant_hash': '40010e4301501064',\n",
       "  'vcf_string': '15\\t48474280\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474280G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4185C>T',\n",
       "  'hgvs_protein': 'p.Tyr1395=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474280,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474286-T-C',\n",
       "  'variant_hash': '526333ac5b0daa34',\n",
       "  'vcf_string': '15\\t48474286\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474286T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4179A>G',\n",
       "  'hgvs_protein': 'p.Glu1393=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474286,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474289-C-T',\n",
       "  'variant_hash': 'd551a54a00279111',\n",
       "  'vcf_string': '15\\t48474289\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474289C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4176G>A',\n",
       "  'hgvs_protein': 'p.Lys1392=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474289,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48474293-C-T',\n",
       "  'variant_hash': '9dbc38d6c971c956',\n",
       "  'vcf_string': '15\\t48474293\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474293C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4172G>A',\n",
       "  'hgvs_protein': 'p.Cys1391Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474293,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474295-C-T',\n",
       "  'variant_hash': '8f4289a392cf7e99',\n",
       "  'vcf_string': '15\\t48474295\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474295C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4170G>A',\n",
       "  'hgvs_protein': 'p.Leu1390=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474295,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474299-C-T',\n",
       "  'variant_hash': '80eabf9050578b92',\n",
       "  'vcf_string': '15\\t48474299\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474299C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4166G>A',\n",
       "  'hgvs_protein': 'p.Cys1389Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474299,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474299-C-G',\n",
       "  'variant_hash': 'b10f74a0442d5611',\n",
       "  'vcf_string': '15\\t48474299\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474299C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4166G>C',\n",
       "  'hgvs_protein': 'p.Cys1389Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474299,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474301-G-T',\n",
       "  'variant_hash': '69d5d9c5ab06157f',\n",
       "  'vcf_string': '15\\t48474301\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474301G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4164C>A',\n",
       "  'hgvs_protein': 'p.Arg1388=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474301,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474304-G-A',\n",
       "  'variant_hash': 'b1a26523477ba210',\n",
       "  'vcf_string': '15\\t48474304\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474304G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4161C>T',\n",
       "  'hgvs_protein': 'p.Tyr1387=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474304,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474310-T-A',\n",
       "  'variant_hash': 'af580186148342f2',\n",
       "  'vcf_string': '15\\t48474310\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474310T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4155A>T',\n",
       "  'hgvs_protein': 'p.Gly1385=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474310,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474343-G-A',\n",
       "  'variant_hash': '0a3ba8356ad9bb91',\n",
       "  'vcf_string': '15\\t48474343\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474343G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4122C>T',\n",
       "  'hgvs_protein': 'p.Cys1374=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474343,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474344-C-T',\n",
       "  'variant_hash': '9020c787b6e51206',\n",
       "  'vcf_string': '15\\t48474344\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474344C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4121G>A',\n",
       "  'hgvs_protein': 'p.Cys1374Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474344,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474345-A-C',\n",
       "  'variant_hash': 'dff6dd9a88b380ba',\n",
       "  'vcf_string': '15\\t48474345\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474345A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4120T>G',\n",
       "  'hgvs_protein': 'p.Cys1374Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474345,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474365-C-T',\n",
       "  'variant_hash': '7c7c5bd5d851a1e0',\n",
       "  'vcf_string': '15\\t48474365\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474365C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4100G>A',\n",
       "  'hgvs_protein': 'p.Cys1367Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474365,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474369-C-T',\n",
       "  'variant_hash': 'a6459e2606859a11',\n",
       "  'vcf_string': '15\\t48474369\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474369C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4096G>A',\n",
       "  'hgvs_protein': 'p.Glu1366Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474369,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474376-A-G',\n",
       "  'variant_hash': 'b18099292f9f974b',\n",
       "  'vcf_string': '15\\t48474376\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474376A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4089T>C',\n",
       "  'hgvs_protein': 'p.Asp1363=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474376,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474385-A-G',\n",
       "  'variant_hash': '300e6f879626d974',\n",
       "  'vcf_string': '15\\t48474385\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474385A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4088-8T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474385,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474394-A-G',\n",
       "  'variant_hash': '8a4a0cc95043d018',\n",
       "  'vcf_string': '15\\t48474394\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474394A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4088-17T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474394,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474533-C-T',\n",
       "  'variant_hash': '0f9144e6f16bc7df',\n",
       "  'vcf_string': '15\\t48474533\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474533C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4082G>A',\n",
       "  'hgvs_protein': 'p.Cys1361Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474533,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474534-A-G',\n",
       "  'variant_hash': 'e3781552cd591371',\n",
       "  'vcf_string': '15\\t48474534\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474534A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4081T>C',\n",
       "  'hgvs_protein': 'p.Cys1361Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474534,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48474550-A-T',\n",
       "  'variant_hash': 'c7ca878bf085a254',\n",
       "  'vcf_string': '15\\t48474550\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474550A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4065T>A',\n",
       "  'hgvs_protein': 'p.Ile1355=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474550,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474554-C-T',\n",
       "  'variant_hash': '30896f8b54ac6007',\n",
       "  'vcf_string': '15\\t48474554\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474554C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4061G>A',\n",
       "  'hgvs_protein': 'p.Trp1354Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474554,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474556-C-A',\n",
       "  'variant_hash': '3fc051fc97ea7b9a',\n",
       "  'vcf_string': '15\\t48474556\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474556C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4059G>T',\n",
       "  'hgvs_protein': 'p.Gly1353=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474556,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474559-G-A',\n",
       "  'variant_hash': 'bab5bd663f494379',\n",
       "  'vcf_string': '15\\t48474559\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474559G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4056C>T',\n",
       "  'hgvs_protein': 'p.Pro1352=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474559,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474565-G-T',\n",
       "  'variant_hash': 'd2896a16207513f2',\n",
       "  'vcf_string': '15\\t48474565\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474565G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4050C>A',\n",
       "  'hgvs_protein': 'p.Cys1350Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474565,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474565-G-A',\n",
       "  'variant_hash': '87bb30bd802fbff3',\n",
       "  'vcf_string': '15\\t48474565\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474565G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4050C>T',\n",
       "  'hgvs_protein': 'p.Cys1350=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474565,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474566-C-A',\n",
       "  'variant_hash': '7d8465993c1b22aa',\n",
       "  'vcf_string': '15\\t48474566\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474566C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4049G>T',\n",
       "  'hgvs_protein': 'p.Cys1350Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474566,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474567-A-G',\n",
       "  'variant_hash': '9f199e83805c254e',\n",
       "  'vcf_string': '15\\t48474567\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474567A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4048T>C',\n",
       "  'hgvs_protein': 'p.Cys1350Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474567,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474572-C-T',\n",
       "  'variant_hash': 'e38f7b42631f7de8',\n",
       "  'vcf_string': '15\\t48474572\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474572C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4043G>A',\n",
       "  'hgvs_protein': 'p.Cys1348Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474572,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474580-G-A',\n",
       "  'variant_hash': '21885cc3950da826',\n",
       "  'vcf_string': '15\\t48474580\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474580G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4035C>T',\n",
       "  'hgvs_protein': 'p.Ser1345=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474580,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474584-C-T',\n",
       "  'variant_hash': 'a0607e98129413b7',\n",
       "  'vcf_string': '15\\t48474584\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474584C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4031G>A',\n",
       "  'hgvs_protein': 'p.Gly1344Glu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474584,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474589-T-C',\n",
       "  'variant_hash': 'cb8b41494f3a37fd',\n",
       "  'vcf_string': '15\\t48474589\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474589T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4026A>G',\n",
       "  'hgvs_protein': 'p.Thr1342=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474589,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474592-A-G',\n",
       "  'variant_hash': 'e80d37bae20c5ebb',\n",
       "  'vcf_string': '15\\t48474592\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474592A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4023T>C',\n",
       "  'hgvs_protein': 'p.Asn1341=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474592,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474599-C-G',\n",
       "  'variant_hash': '7ab684589a8d878a',\n",
       "  'vcf_string': '15\\t48474599\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474599C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4016G>C',\n",
       "  'hgvs_protein': 'p.Cys1339Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474599,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474601-T-C',\n",
       "  'variant_hash': '23587f4d12b0db17',\n",
       "  'vcf_string': '15\\t48474601\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474601T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4014A>G',\n",
       "  'hgvs_protein': 'p.Val1338=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474601,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474638-C-T',\n",
       "  'variant_hash': 'b245ab634a33b7f0',\n",
       "  'vcf_string': '15\\t48474638\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474638C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3977G>A',\n",
       "  'hgvs_protein': 'p.Cys1326Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474638,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474640-T-C',\n",
       "  'variant_hash': 'fe71d98cb52a2d7d',\n",
       "  'vcf_string': '15\\t48474640\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474640T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3975A>G',\n",
       "  'hgvs_protein': 'p.Glu1325=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474640,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474646-G-A',\n",
       "  'variant_hash': '94212aa632c44fe8',\n",
       "  'vcf_string': '15\\t48474646\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474646G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3969C>T',\n",
       "  'hgvs_protein': 'p.Ile1323=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474646,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48474658-A-G',\n",
       "  'variant_hash': '2c0744c49b247d40',\n",
       "  'vcf_string': '15\\t48474658\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474658A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3965-8T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474658,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48481636-T-C',\n",
       "  'variant_hash': '5f1d43c7ca5b222d',\n",
       "  'vcf_string': '15\\t48481636\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481636T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3964+19A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481636,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481643-G-A',\n",
       "  'variant_hash': 'f4a44890202d1bd1',\n",
       "  'vcf_string': '15\\t48481643\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481643G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3964+12C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481643,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481655-C-T',\n",
       "  'variant_hash': '08293a705a9e6c78',\n",
       "  'vcf_string': '15\\t48481655\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481655C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3964G>A',\n",
       "  'hgvs_protein': 'p.Asp1322Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481655,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48481656-T-C',\n",
       "  'variant_hash': '81362f0c19c82c5b',\n",
       "  'vcf_string': '15\\t48481656\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481656T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3963A>G',\n",
       "  'hgvs_protein': 'p.Thr1321=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481656,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48481661-A-G',\n",
       "  'variant_hash': '3c183e206188c6b1',\n",
       "  'vcf_string': '15\\t48481661\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481661A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3958T>C',\n",
       "  'hgvs_protein': 'p.Cys1320Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481661,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48481668-T-C',\n",
       "  'variant_hash': 'd1935cd51482ab9e',\n",
       "  'vcf_string': '15\\t48481668\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481668T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3951A>G',\n",
       "  'hgvs_protein': 'p.Lys1317=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481668,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481680-G-A',\n",
       "  'variant_hash': 'ab52560bff759f20',\n",
       "  'vcf_string': '15\\t48481680\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481680G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3939C>T',\n",
       "  'hgvs_protein': 'p.Gly1313=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481680,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481687-T-C',\n",
       "  'variant_hash': 'ba23853330c2f0ab',\n",
       "  'vcf_string': '15\\t48481687\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481687T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3932A>G',\n",
       "  'hgvs_protein': 'p.Tyr1311Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481687,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48481690-C-T',\n",
       "  'variant_hash': '67eb5593f4a57536',\n",
       "  'vcf_string': '15\\t48481690\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481690C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3929G>A',\n",
       "  'hgvs_protein': 'p.Gly1310Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481690,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48481699-C-T',\n",
       "  'variant_hash': '5a6cf978f98a5b29',\n",
       "  'vcf_string': '15\\t48481699\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481699C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3920G>A',\n",
       "  'hgvs_protein': 'p.Cys1307Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481699,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48481713-T-C',\n",
       "  'variant_hash': 'c0963ed4dd769682',\n",
       "  'vcf_string': '15\\t48481713\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481713T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3906A>G',\n",
       "  'hgvs_protein': 'p.Ser1302=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481713,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481716-G-A',\n",
       "  'variant_hash': 'd14b76e86e12deb3',\n",
       "  'vcf_string': '15\\t48481716\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481716G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3903C>T',\n",
       "  'hgvs_protein': 'p.Gly1301=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481716,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481719-T-C',\n",
       "  'variant_hash': '995759e741b144c4',\n",
       "  'vcf_string': '15\\t48481719\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481719T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3900A>G',\n",
       "  'hgvs_protein': 'p.Lys1300=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481719,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48481722-C-T',\n",
       "  'variant_hash': '561d69c3781bd895',\n",
       "  'vcf_string': '15\\t48481722\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481722C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3897G>A',\n",
       "  'hgvs_protein': 'p.Thr1299=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481722,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481722-C-G',\n",
       "  'variant_hash': 'b1151f699d58aac2',\n",
       "  'vcf_string': '15\\t48481722\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481722C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3897G>C',\n",
       "  'hgvs_protein': 'p.Thr1299=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481722,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481723-G-A',\n",
       "  'variant_hash': '36e08338f44fdd16',\n",
       "  'vcf_string': '15\\t48481723\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481723G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3896C>T',\n",
       "  'hgvs_protein': 'p.Thr1299Met',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481723,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48481729-T-C',\n",
       "  'variant_hash': 'c3f7afeef47865e2',\n",
       "  'vcf_string': '15\\t48481729\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481729T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3890A>G',\n",
       "  'hgvs_protein': 'p.Glu1297Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481729,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48481733-A-G',\n",
       "  'variant_hash': '08270c3712f6fe54',\n",
       "  'vcf_string': '15\\t48481733\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481733A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3886T>C',\n",
       "  'hgvs_protein': 'p.Cys1296Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481733,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48481743-T-G',\n",
       "  'variant_hash': 'b20badec5d64a977',\n",
       "  'vcf_string': '15\\t48481743\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481743T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3876A>C',\n",
       "  'hgvs_protein': 'p.Leu1292=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481743,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481768-C-T',\n",
       "  'variant_hash': 'a20dd887bd4d6051',\n",
       "  'vcf_string': '15\\t48481768\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481768C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3851G>A',\n",
       "  'hgvs_protein': 'p.Cys1284Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481768,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48483798-A-G',\n",
       "  'variant_hash': '70fc8e021d0fb898',\n",
       "  'vcf_string': '15\\t48483798\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483798A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3838+20T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483798,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483816-A-G',\n",
       "  'variant_hash': 'b5ecc26e2e6bb2e1',\n",
       "  'vcf_string': '15\\t48483816\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483816A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3838+2T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483816,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48483818-C-T',\n",
       "  'variant_hash': 'dfc24d425aa00fb7',\n",
       "  'vcf_string': '15\\t48483818\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483818C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3838G>A',\n",
       "  'hgvs_protein': 'p.Asp1280Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483818,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48483822-A-G',\n",
       "  'variant_hash': '0fa825827fa391c0',\n",
       "  'vcf_string': '15\\t48483822\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483822A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3834T>C',\n",
       "  'hgvs_protein': 'p.Cys1278=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483822,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483822-A-C',\n",
       "  'variant_hash': '52c00d9adc0b28f4',\n",
       "  'vcf_string': '15\\t48483822\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483822A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3834T>G',\n",
       "  'hgvs_protein': 'p.Cys1278Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483822,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48483823-C-G',\n",
       "  'variant_hash': '9f3f5959d91b4789',\n",
       "  'vcf_string': '15\\t48483823\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483823C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3833G>C',\n",
       "  'hgvs_protein': 'p.Cys1278Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483823,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48483855-A-G',\n",
       "  'variant_hash': 'cdb3146e25115cb5',\n",
       "  'vcf_string': '15\\t48483855\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483855A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3801T>C',\n",
       "  'hgvs_protein': 'p.Asp1267=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483855,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483858-A-G',\n",
       "  'variant_hash': '5b5bb501f476b668',\n",
       "  'vcf_string': '15\\t48483858\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483858A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3798T>C',\n",
       "  'hgvs_protein': 'p.Tyr1266=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483858,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48483862-C-T',\n",
       "  'variant_hash': '61e1c75811da3038',\n",
       "  'vcf_string': '15\\t48483862\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483862C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3794G>A',\n",
       "  'hgvs_protein': 'p.Cys1265Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483862,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48483862-C-G',\n",
       "  'variant_hash': '9025c7bb1071c057',\n",
       "  'vcf_string': '15\\t48483862\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483862C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3794G>C',\n",
       "  'hgvs_protein': 'p.Cys1265Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483862,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48483876-C-T',\n",
       "  'variant_hash': 'ab86c442d4b430e0',\n",
       "  'vcf_string': '15\\t48483876\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483876C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3780G>A',\n",
       "  'hgvs_protein': 'p.Glu1260=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483876,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483878-C-A',\n",
       "  'variant_hash': 'd807716d42032a8b',\n",
       "  'vcf_string': '15\\t48483878\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483878C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3778G>T',\n",
       "  'hgvs_protein': 'p.Glu1260Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483878,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48483879-T-G',\n",
       "  'variant_hash': 'e878524b99513ded',\n",
       "  'vcf_string': '15\\t48483879\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483879T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3777A>C',\n",
       "  'hgvs_protein': 'p.Gly1259=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483879,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483894-G-A',\n",
       "  'variant_hash': 'f5cececf06bd40ce',\n",
       "  'vcf_string': '15\\t48483894\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483894G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3762C>T',\n",
       "  'hgvs_protein': 'p.Cys1254=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483894,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483899-G-A',\n",
       "  'variant_hash': 'c586d6fa035db753',\n",
       "  'vcf_string': '15\\t48483899\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483899G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3757C>T',\n",
       "  'hgvs_protein': 'p.Gln1253Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483899,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48483900-A-C',\n",
       "  'variant_hash': '6acc17e14f5e7bd8',\n",
       "  'vcf_string': '15\\t48483900\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483900A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3756T>G',\n",
       "  'hgvs_protein': 'p.Gly1252=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483900,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483921-A-G',\n",
       "  'variant_hash': '1f3d9b9c128a846d',\n",
       "  'vcf_string': '15\\t48483921\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483921A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3735T>C',\n",
       "  'hgvs_protein': 'p.Asn1245=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483921,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483931-C-T',\n",
       "  'variant_hash': 'de1810c7335d4bed',\n",
       "  'vcf_string': '15\\t48483931\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483931C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3725G>A',\n",
       "  'hgvs_protein': 'p.Cys1242Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483931,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48483935-C-T',\n",
       "  'variant_hash': '3a84357d480c04ae',\n",
       "  'vcf_string': '15\\t48483935\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483935C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3721G>A',\n",
       "  'hgvs_protein': 'p.Glu1241Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483935,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48483939-G-A',\n",
       "  'variant_hash': 'e049ae6265663b1f',\n",
       "  'vcf_string': '15\\t48483939\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483939G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3717C>T',\n",
       "  'hgvs_protein': 'p.Ile1239=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483939,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48485374-C-T',\n",
       "  'variant_hash': 'ab482243510985a7',\n",
       "  'vcf_string': '15\\t48485374\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485374C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3712G>A',\n",
       "  'hgvs_protein': 'p.Asp1238Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485374,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48485379-C-T',\n",
       "  'variant_hash': 'ef6167125eabf530',\n",
       "  'vcf_string': '15\\t48485379\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485379C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3707G>A',\n",
       "  'hgvs_protein': 'p.Cys1236Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485379,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48485393-A-G',\n",
       "  'variant_hash': '7d9faed37c228e53',\n",
       "  'vcf_string': '15\\t48485393\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485393A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3693T>C',\n",
       "  'hgvs_protein': 'p.Pro1231=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485393,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48485397-A-G',\n",
       "  'variant_hash': 'be73a692b6ebc8fe',\n",
       "  'vcf_string': '15\\t48485397\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485397A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3689T>C',\n",
       "  'hgvs_protein': 'p.Met1230Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485397,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48485411-C-T',\n",
       "  'variant_hash': '00a9f1482021510e',\n",
       "  'vcf_string': '15\\t48485411\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485411C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3675G>A',\n",
       "  'hgvs_protein': 'p.Pro1225=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485411,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48485416-G-A',\n",
       "  'variant_hash': '84414afc1aa10a47',\n",
       "  'vcf_string': '15\\t48485416\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485416G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3670C>T',\n",
       "  'hgvs_protein': 'p.Gln1224Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485416,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48485430-T-C',\n",
       "  'variant_hash': '8641856aca41d151',\n",
       "  'vcf_string': '15\\t48485430\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485430T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3656A>G',\n",
       "  'hgvs_protein': 'p.Tyr1219Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485430,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48485432-G-A',\n",
       "  'variant_hash': '3dd87cf1fb9f90fb',\n",
       "  'vcf_string': '15\\t48485432\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485432G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3654C>T',\n",
       "  'hgvs_protein': 'p.Ser1218=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485432,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48485435-G-T',\n",
       "  'variant_hash': '4255efc81cca6ba2',\n",
       "  'vcf_string': '15\\t48485435\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485435G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3651C>A',\n",
       "  'hgvs_protein': 'p.Gly1217=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485435,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48485436-C-T',\n",
       "  'variant_hash': '9e42689289369b72',\n",
       "  'vcf_string': '15\\t48485436\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485436C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3650G>A',\n",
       "  'hgvs_protein': 'p.Gly1217Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485436,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48485463-C-A',\n",
       "  'variant_hash': 'e7298acd3ac0659a',\n",
       "  'vcf_string': '15\\t48485463\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485463C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3623G>T',\n",
       "  'hgvs_protein': 'p.Cys1208Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485463,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48485483-G-T',\n",
       "  'variant_hash': '26ed38a33cfb1694',\n",
       "  'vcf_string': '15\\t48485483\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485483G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3603C>A',\n",
       "  'hgvs_protein': 'p.Cys1201Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485483,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48485486-T-C',\n",
       "  'variant_hash': '6abe3fec2075c63b',\n",
       "  'vcf_string': '15\\t48485486\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485486T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3600A>G',\n",
       "  'hgvs_protein': 'p.Glu1200=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485486,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48485490-T-C',\n",
       "  'variant_hash': 'ee02c0326ccae0d3',\n",
       "  'vcf_string': '15\\t48485490\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485490T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3596A>G',\n",
       "  'hgvs_protein': 'p.Asp1199Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485490,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48485507-T-C',\n",
       "  'variant_hash': '65426e0cd7e8e82d',\n",
       "  'vcf_string': '15\\t48485507\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485507T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3590-11A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485507,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487075-C-T',\n",
       "  'variant_hash': '1a46642266f28e22',\n",
       "  'vcf_string': '15\\t48487075\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487075C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3589G>A',\n",
       "  'hgvs_protein': 'p.Asp1197Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487075,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48487085-T-C',\n",
       "  'variant_hash': '851d32530ca3e22f',\n",
       "  'vcf_string': '15\\t48487085\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487085T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3579A>G',\n",
       "  'hgvs_protein': 'p.Leu1193=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487085,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487091-A-G',\n",
       "  'variant_hash': 'a48057876d468e0d',\n",
       "  'vcf_string': '15\\t48487091\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487091A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3573T>C',\n",
       "  'hgvs_protein': 'p.Asp1191=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487091,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487094-G-A',\n",
       "  'variant_hash': '161c9cc9cb7cbbb3',\n",
       "  'vcf_string': '15\\t48487094\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487094G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3570C>T',\n",
       "  'hgvs_protein': 'p.Pro1190=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487094,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487106-G-C',\n",
       "  'variant_hash': '1c8f22bdffd33216',\n",
       "  'vcf_string': '15\\t48487106\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487106G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3558C>G',\n",
       "  'hgvs_protein': 'p.Tyr1186Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487106,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487107-T-C',\n",
       "  'variant_hash': '6b2000d53f88414c',\n",
       "  'vcf_string': '15\\t48487107\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487107T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3557A>G',\n",
       "  'hgvs_protein': 'p.Tyr1186Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487107,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48487110-C-T',\n",
       "  'variant_hash': 'a92efa6111aedff2',\n",
       "  'vcf_string': '15\\t48487110\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487110C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3554G>A',\n",
       "  'hgvs_protein': 'p.Gly1185Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487110,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48487119-C-T',\n",
       "  'variant_hash': 'a43d1c4c192883b0',\n",
       "  'vcf_string': '15\\t48487119\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487119C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3545G>A',\n",
       "  'hgvs_protein': 'p.Cys1182Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487119,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487129-G-A',\n",
       "  'variant_hash': '7ba7202122053e18',\n",
       "  'vcf_string': '15\\t48487129\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487129G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3535C>T',\n",
       "  'hgvs_protein': 'p.Gln1179Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487129,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48487131-T-C',\n",
       "  'variant_hash': '73546cee3e5d6ad7',\n",
       "  'vcf_string': '15\\t48487131\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487131T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3533A>G',\n",
       "  'hgvs_protein': 'p.Tyr1178Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487131,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48487142-G-A',\n",
       "  'variant_hash': '2670657eeb8e744a',\n",
       "  'vcf_string': '15\\t48487142\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487142G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3522C>T',\n",
       "  'hgvs_protein': 'p.Leu1174=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487142,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487146-T-G',\n",
       "  'variant_hash': '8a23c9499eaec155',\n",
       "  'vcf_string': '15\\t48487146\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487146T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3518A>C',\n",
       "  'hgvs_protein': 'p.Asn1173Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487146,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48487151-G-A',\n",
       "  'variant_hash': '8996cdc0b27f01fe',\n",
       "  'vcf_string': '15\\t48487151\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487151G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3513C>T',\n",
       "  'hgvs_protein': 'p.Cys1171=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487151,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487155-C-T',\n",
       "  'variant_hash': 'e5a2a4c2015123a2',\n",
       "  'vcf_string': '15\\t48487155\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487155C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3509G>A',\n",
       "  'hgvs_protein': 'p.Arg1170His',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487155,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48487157-G-A',\n",
       "  'variant_hash': '582da053cd736f7e',\n",
       "  'vcf_string': '15\\t48487157\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487157G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3507C>T',\n",
       "  'hgvs_protein': 'p.Gly1169=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487157,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487160-A-G',\n",
       "  'variant_hash': 'd475b6ab78dcfeb1',\n",
       "  'vcf_string': '15\\t48487160\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487160A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3504T>C',\n",
       "  'hgvs_protein': 'p.Asn1168=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487160,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487167-C-T',\n",
       "  'variant_hash': 'cbc34da0e559ef7d',\n",
       "  'vcf_string': '15\\t48487167\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487167C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3497G>A',\n",
       "  'hgvs_protein': 'p.Cys1166Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487167,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48487171-G-A',\n",
       "  'variant_hash': 'b8207041803bda92',\n",
       "  'vcf_string': '15\\t48487171\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487171G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3493C>T',\n",
       "  'hgvs_protein': 'p.Leu1165=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487171,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487184-C-T',\n",
       "  'variant_hash': 'cc0531fcf90b59c9',\n",
       "  'vcf_string': '15\\t48487184\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487184C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3480G>A',\n",
       "  'hgvs_protein': 'p.Glu1160=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487184,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487188-C-T',\n",
       "  'variant_hash': '8fb415d816cbf41f',\n",
       "  'vcf_string': '15\\t48487188\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487188C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3476G>A',\n",
       "  'hgvs_protein': 'p.Cys1159Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487188,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48487189-A-G',\n",
       "  'variant_hash': 'acd3dc4522e496b4',\n",
       "  'vcf_string': '15\\t48487189\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487189A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3475T>C',\n",
       "  'hgvs_protein': 'p.Cys1159Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487189,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487199-G-A',\n",
       "  'variant_hash': 'ebb8953eabf0ba72',\n",
       "  'vcf_string': '15\\t48487199\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487199G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3465C>T',\n",
       "  'hgvs_protein': 'p.Asp1155=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487199,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487205-C-T',\n",
       "  'variant_hash': 'dfa0f0db264d78c8',\n",
       "  'vcf_string': '15\\t48487205\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487205C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3464-5G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487205,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48487219-G-C',\n",
       "  'variant_hash': '5e4c74606771f32b',\n",
       "  'vcf_string': '15\\t48487219\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487219G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3464-19C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487219,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487297-A-T',\n",
       "  'variant_hash': 'd2b19e877a24b26f',\n",
       "  'vcf_string': '15\\t48487297\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487297A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3463+15T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487297,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487309-T-C',\n",
       "  'variant_hash': 'a8a427647ba2d9e5',\n",
       "  'vcf_string': '15\\t48487309\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487309T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3463+3A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487309,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48487312-C-T',\n",
       "  'variant_hash': 'ba13fb9fbd3f82a3',\n",
       "  'vcf_string': '15\\t48487312\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487312C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3463G>A',\n",
       "  'hgvs_protein': 'p.Asp1155Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487312,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487317-C-T',\n",
       "  'variant_hash': '5d3dc55637b6c9c8',\n",
       "  'vcf_string': '15\\t48487317\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487317C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3458G>A',\n",
       "  'hgvs_protein': 'p.Cys1153Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487317,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487318-A-G',\n",
       "  'variant_hash': '74de6820d6699216',\n",
       "  'vcf_string': '15\\t48487318\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487318A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3457T>C',\n",
       "  'hgvs_protein': 'p.Cys1153Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487318,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48487319-C-T',\n",
       "  'variant_hash': 'dc83bb19728928bf',\n",
       "  'vcf_string': '15\\t48487319\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487319C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3456G>A',\n",
       "  'hgvs_protein': 'p.Ala1152=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487319,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487319-C-G',\n",
       "  'variant_hash': '6234640b46e125e8',\n",
       "  'vcf_string': '15\\t48487319\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487319C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3456G>C',\n",
       "  'hgvs_protein': 'p.Ala1152=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487319,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487322-G-A',\n",
       "  'variant_hash': '237f74e9f6a5dc5e',\n",
       "  'vcf_string': '15\\t48487322\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487322G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3453C>T',\n",
       "  'hgvs_protein': 'p.Ser1151=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487322,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487333-G-C',\n",
       "  'variant_hash': '97bfb42ce2f3d65a',\n",
       "  'vcf_string': '15\\t48487333\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487333G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3442C>G',\n",
       "  'hgvs_protein': 'p.Pro1148Ala',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487333,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48487334-G-A',\n",
       "  'variant_hash': '954fa832b71e15ba',\n",
       "  'vcf_string': '15\\t48487334\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487334G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3441C>T',\n",
       "  'hgvs_protein': 'p.Ser1147=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487334,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487346-G-T',\n",
       "  'variant_hash': '6ec34bfad89fd1cd',\n",
       "  'vcf_string': '15\\t48487346\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487346G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3429C>A',\n",
       "  'hgvs_protein': 'p.Gly1143=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487346,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487353-G-A',\n",
       "  'variant_hash': 'de37ce0e60745f1f',\n",
       "  'vcf_string': '15\\t48487353\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487353G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3422C>T',\n",
       "  'hgvs_protein': 'p.Pro1141Leu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487353,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48487362-C-G',\n",
       "  'variant_hash': 'acd74aa4befc9675',\n",
       "  'vcf_string': '15\\t48487362\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487362C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3413G>C',\n",
       "  'hgvs_protein': 'p.Cys1138Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487362,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487362-C-A',\n",
       "  'variant_hash': 'cd3cd87709596480',\n",
       "  'vcf_string': '15\\t48487362\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487362C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3413G>T',\n",
       "  'hgvs_protein': 'p.Cys1138Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487362,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487364-G-A',\n",
       "  'variant_hash': '8bf4a61c52b57280',\n",
       "  'vcf_string': '15\\t48487364\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487364G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3411C>T',\n",
       "  'hgvs_protein': 'p.Arg1137=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487364,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487367-G-T',\n",
       "  'variant_hash': 'af0bf3cf57036ed1',\n",
       "  'vcf_string': '15\\t48487367\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487367G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3408C>A',\n",
       "  'hgvs_protein': 'p.Tyr1136Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487367,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48487402-G-A',\n",
       "  'variant_hash': '0b8d91ba4b4dd7fb',\n",
       "  'vcf_string': '15\\t48487402\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487402G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3373C>T',\n",
       "  'hgvs_protein': 'p.Arg1125Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487402,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487406-T-C',\n",
       "  'variant_hash': 'a0a8a82eb521b265',\n",
       "  'vcf_string': '15\\t48487406\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487406T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3369A>G',\n",
       "  'hgvs_protein': 'p.Leu1123=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487406,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487431-T-C',\n",
       "  'variant_hash': 'c7530d9f18c66227',\n",
       "  'vcf_string': '15\\t48487431\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487431T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3344A>G',\n",
       "  'hgvs_protein': 'p.Asp1115Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487431,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487437-T-C',\n",
       "  'variant_hash': '7c122680d40f3603',\n",
       "  'vcf_string': '15\\t48487437\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487437T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3338A>G',\n",
       "  'hgvs_protein': 'p.Asp1113Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487437,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487438-C-G',\n",
       "  'variant_hash': '68952ef545ca138e',\n",
       "  'vcf_string': '15\\t48487438\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487438C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3338-1G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487438,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487639-C-A',\n",
       "  'variant_hash': 'afba005185c9a8cd',\n",
       "  'vcf_string': '15\\t48487639\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487639C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3338-202G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487639,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488093-T-C',\n",
       "  'variant_hash': 'f244d18d0f24af3d',\n",
       "  'vcf_string': '15\\t48488093\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488093T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3337+20A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488093,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488111-A-G',\n",
       "  'variant_hash': '77c3fee78f9b482b',\n",
       "  'vcf_string': '15\\t48488111\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488111A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3337+2T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488111,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48488112-C-T',\n",
       "  'variant_hash': '51c082851bbedb9f',\n",
       "  'vcf_string': '15\\t48488112\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488112C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3337+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488112,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488148-T-C',\n",
       "  'variant_hash': '7db88f0f1bee1fca',\n",
       "  'vcf_string': '15\\t48488148\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488148T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3302A>G',\n",
       "  'hgvs_protein': 'p.Tyr1101Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488148,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488156-G-A',\n",
       "  'variant_hash': '752d68448151fd00',\n",
       "  'vcf_string': '15\\t48488156\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488156G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3294C>T',\n",
       "  'hgvs_protein': 'p.Asp1098=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488156,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48488160-C-T',\n",
       "  'variant_hash': '35c79d1a5ff076e7',\n",
       "  'vcf_string': '15\\t48488160\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488160C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3290G>A',\n",
       "  'hgvs_protein': 'p.Cys1097Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488160,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48488177-C-T',\n",
       "  'variant_hash': '24a65ebbff94da81',\n",
       "  'vcf_string': '15\\t48488177\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488177C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3273G>A',\n",
       "  'hgvs_protein': 'p.Gly1091=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488177,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488195-C-T',\n",
       "  'variant_hash': 'd1b0d398f44d9719',\n",
       "  'vcf_string': '15\\t48488195\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488195C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3255G>A',\n",
       "  'hgvs_protein': 'p.Gln1085=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488195,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488197-G-A',\n",
       "  'variant_hash': '9af5184cf0cba574',\n",
       "  'vcf_string': '15\\t48488197\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488197G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3253C>T',\n",
       "  'hgvs_protein': 'p.Gln1085Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488197,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48488204-G-A',\n",
       "  'variant_hash': '9271cbaa878a7827',\n",
       "  'vcf_string': '15\\t48488204\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488204G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3246C>T',\n",
       "  'hgvs_protein': 'p.Gly1082=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488204,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488233-C-T',\n",
       "  'variant_hash': '9d92637e511d90a3',\n",
       "  'vcf_string': '15\\t48488233\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488233C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3217G>A',\n",
       "  'hgvs_protein': 'p.Glu1073Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488233,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48488233-C-A',\n",
       "  'variant_hash': '811c1244d1f98d30',\n",
       "  'vcf_string': '15\\t48488233\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488233C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3217G>T',\n",
       "  'hgvs_protein': 'p.Glu1073Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488233,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48488240-G-A',\n",
       "  'variant_hash': '5fdbea8da664b9fb',\n",
       "  'vcf_string': '15\\t48488240\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488240G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3210C>T',\n",
       "  'hgvs_protein': 'p.Asp1070=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488240,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488242-C-G',\n",
       "  'variant_hash': '9b4bcd0c937382e5',\n",
       "  'vcf_string': '15\\t48488242\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488242C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3209-1G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488242,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488243-T-G',\n",
       "  'variant_hash': '8c8e119f4dfa770f',\n",
       "  'vcf_string': '15\\t48488243\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488243T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3209-2A>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488243,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48488250-A-G',\n",
       "  'variant_hash': '2bdcd1eb79a54ba3',\n",
       "  'vcf_string': '15\\t48488250\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488250A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3209-9T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488250,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488254-A-G',\n",
       "  'variant_hash': '290c3ce8689d6f85',\n",
       "  'vcf_string': '15\\t48488254\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488254A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3209-13T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488254,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488355-C-T',\n",
       "  'variant_hash': 'f4799bf625a80724',\n",
       "  'vcf_string': '15\\t48488355\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488355C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3208+13G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488355,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488403-C-A',\n",
       "  'variant_hash': '6dd64d8ad0475c5b',\n",
       "  'vcf_string': '15\\t48488403\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488403C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3173G>T',\n",
       "  'hgvs_protein': 'p.Gly1058Val',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488403,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48488405-G-A',\n",
       "  'variant_hash': '95ce5ee3a6e57268',\n",
       "  'vcf_string': '15\\t48488405\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488405G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3171C>T',\n",
       "  'hgvs_protein': 'p.Ser1057=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488405,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488411-A-T',\n",
       "  'variant_hash': 'd87804816b434e0c',\n",
       "  'vcf_string': '15\\t48488411\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488411A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3165T>A',\n",
       "  'hgvs_protein': 'p.Cys1055Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488411,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488412-C-T',\n",
       "  'variant_hash': 'ffcfa53659eb6ce5',\n",
       "  'vcf_string': '15\\t48488412\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488412C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3164G>A',\n",
       "  'hgvs_protein': 'p.Cys1055Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488412,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488419-A-C',\n",
       "  'variant_hash': '3c65f9500a57fc2e',\n",
       "  'vcf_string': '15\\t48488419\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488419A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3157T>G',\n",
       "  'hgvs_protein': 'p.Cys1053Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488419,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48488432-A-T',\n",
       "  'variant_hash': 'd6e8f502146aa208',\n",
       "  'vcf_string': '15\\t48488432\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488432A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3144T>A',\n",
       "  'hgvs_protein': 'p.Ile1048=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488432,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488433-A-G',\n",
       "  'variant_hash': '4ec12b14d16f495a',\n",
       "  'vcf_string': '15\\t48488433\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488433A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3143T>C',\n",
       "  'hgvs_protein': 'p.Ile1048Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488433,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488439-T-C',\n",
       "  'variant_hash': '975dfd258129e472',\n",
       "  'vcf_string': '15\\t48488439\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488439T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3137A>G',\n",
       "  'hgvs_protein': 'p.Asn1046Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488439,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48488443-T-G',\n",
       "  'variant_hash': '0a5ee582070a442d',\n",
       "  'vcf_string': '15\\t48488443\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488443T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3133A>C',\n",
       "  'hgvs_protein': 'p.Arg1045=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488443,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488451-C-T',\n",
       "  'variant_hash': '59f3b70494aeb458',\n",
       "  'vcf_string': '15\\t48488451\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488451C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3125G>A',\n",
       "  'hgvs_protein': 'p.Gly1042Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488451,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48488452-C-T',\n",
       "  'variant_hash': '424af1df130b318d',\n",
       "  'vcf_string': '15\\t48488452\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488452C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3124G>A',\n",
       "  'hgvs_protein': 'p.Gly1042Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488452,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488453-G-A',\n",
       "  'variant_hash': '3196a1f088cde790',\n",
       "  'vcf_string': '15\\t48488453\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488453G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3123C>T',\n",
       "  'hgvs_protein': 'p.His1041=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488453,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488465-G-A',\n",
       "  'variant_hash': '3ff39ec2eb289ef4',\n",
       "  'vcf_string': '15\\t48488465\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488465G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3111C>T',\n",
       "  'hgvs_protein': 'p.Ser1037=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488465,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488493-T-C',\n",
       "  'variant_hash': 'b8041a35664157f3',\n",
       "  'vcf_string': '15\\t48488493\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488493T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3083A>G',\n",
       "  'hgvs_protein': 'p.Asp1028Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488493,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48488502-A-G',\n",
       "  'variant_hash': '40599036c2e0048f',\n",
       "  'vcf_string': '15\\t48488502\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488502A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3083-9T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488502,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489838-C-T',\n",
       "  'variant_hash': '32780963d8534edb',\n",
       "  'vcf_string': '15\\t48489838\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489838C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3082+13G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489838,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489842-A-T',\n",
       "  'variant_hash': 'e72d04818e4aacae',\n",
       "  'vcf_string': '15\\t48489842\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489842A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3082+9T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489842,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489842-A-G',\n",
       "  'variant_hash': 'e1a26f456fb23ac0',\n",
       "  'vcf_string': '15\\t48489842\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489842A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3082+9T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489842,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489845-T-C',\n",
       "  'variant_hash': '9875842df05e593d',\n",
       "  'vcf_string': '15\\t48489845\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489845T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3082+6A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489845,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48489855-G-A',\n",
       "  'variant_hash': '78dc08c490430264',\n",
       "  'vcf_string': '15\\t48489855\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489855G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3078C>T',\n",
       "  'hgvs_protein': 'p.Phe1026=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489855,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489856-A-G',\n",
       "  'variant_hash': '6a51e9226dcbab23',\n",
       "  'vcf_string': '15\\t48489856\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489856A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3077T>C',\n",
       "  'hgvs_protein': 'p.Phe1026Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489856,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48489864-C-T',\n",
       "  'variant_hash': 'a6e6bca1ada55071',\n",
       "  'vcf_string': '15\\t48489864\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489864C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3069G>A',\n",
       "  'hgvs_protein': 'p.Lys1023=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489864,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48489869-C-A',\n",
       "  'variant_hash': '4548541dae41f4ef',\n",
       "  'vcf_string': '15\\t48489869\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489869C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3064G>T',\n",
       "  'hgvs_protein': 'p.Gly1022Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489869,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48489875-T-C',\n",
       "  'variant_hash': 'd86f18bce88f499f',\n",
       "  'vcf_string': '15\\t48489875\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489875T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3058A>G',\n",
       "  'hgvs_protein': 'p.Thr1020Ala',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489875,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489896-C-T',\n",
       "  'variant_hash': '739c4aeb8aecca92',\n",
       "  'vcf_string': '15\\t48489896\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489896C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3037G>A',\n",
       "  'hgvs_protein': 'p.Gly1013Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489896,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48489897-G-A',\n",
       "  'variant_hash': '4eb06795ba551ffc',\n",
       "  'vcf_string': '15\\t48489897\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489897G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3036C>T',\n",
       "  'hgvs_protein': 'p.Pro1012=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489897,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489906-C-T',\n",
       "  'variant_hash': 'a1dcdf2126e6e781',\n",
       "  'vcf_string': '15\\t48489906\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489906C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3027G>A',\n",
       "  'hgvs_protein': 'p.Pro1009=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489906,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489909-A-G',\n",
       "  'variant_hash': '66777176344bcc95',\n",
       "  'vcf_string': '15\\t48489909\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489909A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3024T>C',\n",
       "  'hgvs_protein': 'p.Cys1008=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489909,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489911-A-G',\n",
       "  'variant_hash': 'ba01b3d4f012f96d',\n",
       "  'vcf_string': '15\\t48489911\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489911A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3022T>C',\n",
       "  'hgvs_protein': 'p.Cys1008Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489911,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48489912-C-T',\n",
       "  'variant_hash': '62487a52680ee6ae',\n",
       "  'vcf_string': '15\\t48489912\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489912C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3021G>A',\n",
       "  'hgvs_protein': 'p.Leu1007=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489912,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489921-G-T',\n",
       "  'variant_hash': '4ae4d57b83cc5a43',\n",
       "  'vcf_string': '15\\t48489921\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489921G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3012C>A',\n",
       "  'hgvs_protein': 'p.Tyr1004Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489921,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48489921-G-C',\n",
       "  'variant_hash': 'c90004088067ba7f',\n",
       "  'vcf_string': '15\\t48489921\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489921G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3012C>G',\n",
       "  'hgvs_protein': 'p.Tyr1004Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489921,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48489921-G-A',\n",
       "  'variant_hash': 'c5a767601e650f73',\n",
       "  'vcf_string': '15\\t48489921\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489921G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3012C>T',\n",
       "  'hgvs_protein': 'p.Tyr1004=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489921,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489927-A-G',\n",
       "  'variant_hash': '60fdfde317b60576',\n",
       "  'vcf_string': '15\\t48489927\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489927A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3006T>C',\n",
       "  'hgvs_protein': 'p.Pro1002=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489927,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489946-C-T',\n",
       "  'variant_hash': '70497546aa6a786c',\n",
       "  'vcf_string': '15\\t48489946\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489946C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2987G>A',\n",
       "  'hgvs_protein': 'p.Cys996Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489946,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48489951-C-T',\n",
       "  'variant_hash': '4f2fd92d48aa8c1b',\n",
       "  'vcf_string': '15\\t48489951\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489951C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2982G>A',\n",
       "  'hgvs_protein': 'p.Glu994=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489951,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489956-A-G',\n",
       "  'variant_hash': '1885bbfc470c21dc',\n",
       "  'vcf_string': '15\\t48489956\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489956A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2977T>C',\n",
       "  'hgvs_protein': 'p.Cys993Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489956,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48489969-C-T',\n",
       "  'variant_hash': 'b31dc340ff5733bf',\n",
       "  'vcf_string': '15\\t48489969\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489969C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2964G>A',\n",
       "  'hgvs_protein': 'p.Trp988Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489969,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48489970-C-T',\n",
       "  'variant_hash': '76b15198492d1d0d',\n",
       "  'vcf_string': '15\\t48489970\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489970C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2963G>A',\n",
       "  'hgvs_protein': 'p.Trp988Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489970,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48489977-C-T',\n",
       "  'variant_hash': '1e5586c5af16f948',\n",
       "  'vcf_string': '15\\t48489977\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489977C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2956G>A',\n",
       "  'hgvs_protein': 'p.Ala986Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489977,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48489980-C-T',\n",
       "  'variant_hash': '3dcb8e2ba880fad7',\n",
       "  'vcf_string': '15\\t48489980\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489980C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2953G>A',\n",
       "  'hgvs_protein': 'p.Gly985Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489980,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48489981-G-T',\n",
       "  'variant_hash': 'f962704ec0eb39d4',\n",
       "  'vcf_string': '15\\t48489981\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489981G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2952C>A',\n",
       "  'hgvs_protein': 'p.Val984=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489981,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489981-G-A',\n",
       "  'variant_hash': '02103a2dff190ac9',\n",
       "  'vcf_string': '15\\t48489981\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489981G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2952C>T',\n",
       "  'hgvs_protein': 'p.Val984=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489981,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489984-G-A',\n",
       "  'variant_hash': 'f53db5c5728b4181',\n",
       "  'vcf_string': '15\\t48489984\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489984G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2949C>T',\n",
       "  'hgvs_protein': 'p.Ser983=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489984,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489988-C-G',\n",
       "  'variant_hash': '86e0e13faa758367',\n",
       "  'vcf_string': '15\\t48489988\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489988C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2945G>C',\n",
       "  'hgvs_protein': 'p.Cys982Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489988,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48489991-C-G',\n",
       "  'variant_hash': 'e8ea85d07d719abe',\n",
       "  'vcf_string': '15\\t48489991\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489991C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2942G>C',\n",
       "  'hgvs_protein': 'p.Cys981Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489991,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48489999-G-A',\n",
       "  'variant_hash': '7d55ad5c39bafe0d',\n",
       "  'vcf_string': '15\\t48489999\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489999G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2934C>T',\n",
       "  'hgvs_protein': 'p.Asp978=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489999,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48490005-G-A',\n",
       "  'variant_hash': 'dddec78e2eec29de',\n",
       "  'vcf_string': '15\\t48490005\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48490005G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2928C>T',\n",
       "  'hgvs_protein': 'p.Arg976=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48490005,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48490013-G-A',\n",
       "  'variant_hash': '5c5acd7be3688c6e',\n",
       "  'vcf_string': '15\\t48490013\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48490013G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2920C>T',\n",
       "  'hgvs_protein': 'p.Arg974Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48490013,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48490020-A-G',\n",
       "  'variant_hash': '07d2c84436935cd2',\n",
       "  'vcf_string': '15\\t48490020\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48490020A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2913T>C',\n",
       "  'hgvs_protein': 'p.Ile971=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48490020,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " ...]"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0c75c4ef",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Check data types of variant fields\n",
    "variant_id_to_find = '15-48408466-T-C'\n",
    "variant = next((v for v in metadata_variants if v.get('variant_id') == variant_id_to_find), None)\n",
    "\n",
    "if variant:\n",
    "    print(\"=== Variant Structure and Data Types ===\")\n",
    "    print(f\"variant_id: {variant.get('variant_id')} (type: {type(variant.get('variant_id')).__name__})\")\n",
    "    print(f\"pathogenicity_original: {variant.get('pathogenicity_original')} (type: {type(variant.get('pathogenicity_original')).__name__})\")\n",
    "    \n",
    "    metadata = variant.get('metadata', {})\n",
    "    print(f\"\\n=== Metadata Fields and Types ===\")\n",
    "    print(f\"chromosome: {metadata.get('chromosome')} (type: {type(metadata.get('chromosome')).__name__})\")\n",
    "    print(f\"position: {metadata.get('position')} (type: {type(metadata.get('position')).__name__})\")\n",
    "    print(f\"ref_allele: {metadata.get('ref_allele')} (type: {type(metadata.get('ref_allele')).__name__})\")\n",
    "    print(f\"alt_allele: {metadata.get('alt_allele')} (type: {type(metadata.get('alt_allele')).__name__})\")\n",
    "    print(f\"gene_symbol: {metadata.get('gene_symbol')} (type: {type(metadata.get('gene_symbol')).__name__})\")\n",
    "    print(f\"hgvs_protein: '{metadata.get('hgvs_protein')}' (type: {type(metadata.get('hgvs_protein')).__name__})\")\n",
    "    \n",
    "    # Example: Accessing with type awareness\n",
    "    chromosome = metadata.get('chromosome')  # Returns string '15'\n",
    "    position = metadata.get('position')      # Returns int 48408466\n",
    "    \n",
    "    # Type conversions if needed\n",
    "    chromosome_int = int(chromosome) if chromosome else None\n",
    "    position_str = str(position) if position else None\n",
    "    \n",
    "    print(f\"\\n=== Type Conversions ===\")\n",
    "    print(f\"chromosome as int: {chromosome_int} (type: {type(chromosome_int).__name__})\")\n",
    "    print(f\"position as string: {position_str} (type: {type(position_str).__name__})\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b3c58db7",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "aee52457",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5127811c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Chromosome: 15\n",
      "Chromosome: 15\n",
      "Pathogenicity original: benign_or_likely\n",
      "Chromosome: 15\n"
     ]
    }
   ],
   "source": [
    "# Method 1: Using a loop to find the variant\n",
    "variant_id_to_find = '15-48408466-T-C'\n",
    "chromosome = None\n",
    "\n",
    "for variant in metadata_variants:\n",
    "    if variant.get('variant_id') == variant_id_to_find:\n",
    "        chromosome = variant.get('metadata', {}).get('chromosome')\n",
    "        break\n",
    "\n",
    "print(f\"Chromosome: {chromosome}\")\n",
    "\n",
    "# Method 2: Using list comprehension (returns first match or None)\n",
    "variant = next((v for v in metadata_variants if v.get('variant_id') == variant_id_to_find), None)\n",
    "if variant:\n",
    "    chromosome = variant.get('metadata', {}).get('chromosome')\n",
    "    print(f\"Chromosome: {chromosome}\")\n",
    "\n",
    "# Method 3: Create a dictionary for faster lookups (if you need to search multiple times)\n",
    "variant_dict = {v.get('variant_id'): v for v in metadata_variants}\n",
    "target_variant = variant_dict.get('15-48408466-T-C')\n",
    "if target_variant:\n",
    "    pathogenicity_original = target_variant.get('pathogenicity_original')\n",
    "    chromosome = target_variant.get('metadata', {}).get('chromosome')\n",
    "    print(f\"Pathogenicity original: {pathogenicity_original}\")\n",
    "    print(f\"Chromosome: {chromosome}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0937136e",
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def json_to_df(json_file_path: str, \n",
    "field_to_convert: str = 'variants',\n",
    "drop_columns: Optional[list] = None) -> pd.DataFrame:\n",
    "    \"\"\"\n",
    "    Convert existing variants JSON file to pandas DataFrame.\n",
    "    \n",
    "    Args:\n",
    "        json_file_path: Path to the JSON file containing existing variants\n",
    "        drop_columns: Optional list of column names to drop after conversion\n",
    "        \n",
    "    Returns:\n",
    "        DataFrame with one row per variant\n",
    "    \"\"\"\n",
    "    # Read JSON file\n",
    "    with open(json_file_path, 'r') as f:\n",
    "        data = json.load(f)\n",
    "    \n",
    "    # Extract variants list\n",
    "    variants = data.get(field_to_convert, [])\n",
    "    \n",
    "    if not variants:\n",
    "        print(f\"Warning: No variants found in {json_file_path}\")\n",
    "        return pd.DataFrame()\n",
    "    \n",
    "    # Convert to DataFrame\n",
    "    df = pd.DataFrame(variants)\n",
    "    \n",
    "    # Add gene_symbol and count as columns if not already present\n",
    "    if 'gene_symbol' in data and 'gene_symbol' not in df.columns:\n",
    "        df['gene_symbol'] = data['gene_symbol']\n",
    "    \n",
    "    # Drop specified columns if they exist\n",
    "    if drop_columns:\n",
    "        columns_to_drop = [col for col in drop_columns if col in df.columns]\n",
    "        if columns_to_drop:\n",
    "            df = df.drop(columns=columns_to_drop)\n",
    "            print(f\"Dropped columns: {columns_to_drop}\")\n",
    "    \n",
    "    print(f\"Successfully converted {len(df)} variants to DataFrame\")\n",
    "    print(f\"DataFrame shape: {df.shape}\")\n",
    "    print(f\"Columns: {list(df.columns)}\")\n",
    "    \n",
    "    return df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "d625df0e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'variant_id': '15-48408466-T-C',\n",
       "  'variant_hash': '25ec24a26ce39b7f',\n",
       "  'vcf_string': '15\\t48408466\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48408466T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*2524A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48408466,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48408832-C-T',\n",
       "  'variant_hash': '3c7a24079bb8b721',\n",
       "  'vcf_string': '15\\t48408832\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48408832C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*2158G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48408832,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48408837-T-C',\n",
       "  'variant_hash': 'e8ba082cac65aa92',\n",
       "  'vcf_string': '15\\t48408837\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48408837T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*2153A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48408837,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48408950-A-G',\n",
       "  'variant_hash': 'cb541f7aa1e4ade8',\n",
       "  'vcf_string': '15\\t48408950\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48408950A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*2040T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48408950,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48409001-G-A',\n",
       "  'variant_hash': '0b888da0f746e600',\n",
       "  'vcf_string': '15\\t48409001\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48409001G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*1989C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48409001,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48409031-C-T',\n",
       "  'variant_hash': '5873a4e607dc63d9',\n",
       "  'vcf_string': '15\\t48409031\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48409031C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*1959G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48409031,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48409415-A-C',\n",
       "  'variant_hash': 'f47a671cc9f9785d',\n",
       "  'vcf_string': '15\\t48409415\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48409415A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*1575T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48409415,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48409428-C-A',\n",
       "  'variant_hash': '37fd237d73fb80af',\n",
       "  'vcf_string': '15\\t48409428\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48409428C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*1562G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48409428,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48409622-T-C',\n",
       "  'variant_hash': '75ab93d320c2dac8',\n",
       "  'vcf_string': '15\\t48409622\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48409622T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*1368A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48409622,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48409762-A-G',\n",
       "  'variant_hash': '234422e16ba5455d',\n",
       "  'vcf_string': '15\\t48409762\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48409762A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*1228T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48409762,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48410123-C-A',\n",
       "  'variant_hash': '434aef223ad368c3',\n",
       "  'vcf_string': '15\\t48410123\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48410123C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*867G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48410123,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48410260-C-A',\n",
       "  'variant_hash': 'c6c8d7fa0ad7209e',\n",
       "  'vcf_string': '15\\t48410260\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48410260C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*730G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48410260,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48410676-G-A',\n",
       "  'variant_hash': 'd45f76a151f903a2',\n",
       "  'vcf_string': '15\\t48410676\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48410676G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*314C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48410676,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48410933-T-C',\n",
       "  'variant_hash': '71346ce7515b639d',\n",
       "  'vcf_string': '15\\t48410933\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48410933T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.*57A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48410933,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': '3_prime_UTR_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48410990-T-C',\n",
       "  'variant_hash': '468c8756f6e9a904',\n",
       "  'vcf_string': '15\\t48410990\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48410990T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8616A>G',\n",
       "  'hgvs_protein': 'p.Ter2872=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48410990,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_retained_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411008-G-T',\n",
       "  'variant_hash': '3697072459541090',\n",
       "  'vcf_string': '15\\t48411008\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411008G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8598C>A',\n",
       "  'hgvs_protein': 'p.Ile2866=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411008,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411020-C-T',\n",
       "  'variant_hash': '6811f606a66fb1aa',\n",
       "  'vcf_string': '15\\t48411020\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411020C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8586G>A',\n",
       "  'hgvs_protein': 'p.Leu2862=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411020,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411059-A-G',\n",
       "  'variant_hash': 'd126baaf9176a2ad',\n",
       "  'vcf_string': '15\\t48411059\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411059A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8547T>C',\n",
       "  'hgvs_protein': 'p.Tyr2849=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411059,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411064-T-G',\n",
       "  'variant_hash': 'ca0075f13e2068d9',\n",
       "  'vcf_string': '15\\t48411064\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411064T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8542A>C',\n",
       "  'hgvs_protein': 'p.Lys2848Gln',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411064,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411073-G-A',\n",
       "  'variant_hash': '39c23d87cae62c8c',\n",
       "  'vcf_string': '15\\t48411073\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411073G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8533C>T',\n",
       "  'hgvs_protein': 'p.Leu2845=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411073,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411085-C-A',\n",
       "  'variant_hash': 'e9418960be23625e',\n",
       "  'vcf_string': '15\\t48411085\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411085C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8521G>T',\n",
       "  'hgvs_protein': 'p.Glu2841Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411085,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48411110-A-G',\n",
       "  'variant_hash': '4b47112b9ed166f9',\n",
       "  'vcf_string': '15\\t48411110\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411110A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8496T>C',\n",
       "  'hgvs_protein': 'p.Ser2832=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411110,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411118-G-A',\n",
       "  'variant_hash': '6534897248c9a721',\n",
       "  'vcf_string': '15\\t48411118\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411118G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8488C>T',\n",
       "  'hgvs_protein': 'p.Gln2830Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411118,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48411128-G-C',\n",
       "  'variant_hash': '5df890ecf7ac2f9b',\n",
       "  'vcf_string': '15\\t48411128\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411128G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8478C>G',\n",
       "  'hgvs_protein': 'p.Thr2826=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411128,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411140-T-C',\n",
       "  'variant_hash': '9d8f0266cfd0571b',\n",
       "  'vcf_string': '15\\t48411140\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411140T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8466A>G',\n",
       "  'hgvs_protein': 'p.Pro2822=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411140,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411161-G-A',\n",
       "  'variant_hash': '624da23bbb220e28',\n",
       "  'vcf_string': '15\\t48411161\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411161G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8445C>T',\n",
       "  'hgvs_protein': 'p.Leu2815=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411161,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48411173-C-T',\n",
       "  'variant_hash': '2d1787e6493020d2',\n",
       "  'vcf_string': '15\\t48411173\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411173C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8433G>A',\n",
       "  'hgvs_protein': 'p.Gly2811=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411173,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411221-G-A',\n",
       "  'variant_hash': '25653480a14aa233',\n",
       "  'vcf_string': '15\\t48411221\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411221G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8385C>T',\n",
       "  'hgvs_protein': 'p.Ile2795=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411221,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48411228-T-C',\n",
       "  'variant_hash': '79a8ddabf39b394e',\n",
       "  'vcf_string': '15\\t48411228\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411228T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8378A>G',\n",
       "  'hgvs_protein': 'p.Tyr2793Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411228,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48411243-G-A',\n",
       "  'variant_hash': 'becd876b9ad3c083',\n",
       "  'vcf_string': '15\\t48411243\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411243G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8363C>T',\n",
       "  'hgvs_protein': 'p.Thr2788Met',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411243,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48411245-C-G',\n",
       "  'variant_hash': '39eab76887d2447b',\n",
       "  'vcf_string': '15\\t48411245\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411245C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8361G>C',\n",
       "  'hgvs_protein': 'p.Leu2787=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411245,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411257-A-G',\n",
       "  'variant_hash': '3350af1939046bb5',\n",
       "  'vcf_string': '15\\t48411257\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411257A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8349T>C',\n",
       "  'hgvs_protein': 'p.Ala2783=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411257,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411274-G-A',\n",
       "  'variant_hash': '98d7771226d49a30',\n",
       "  'vcf_string': '15\\t48411274\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411274G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8332C>T',\n",
       "  'hgvs_protein': 'p.Leu2778=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411274,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411280-G-A',\n",
       "  'variant_hash': '993c84e847fe446a',\n",
       "  'vcf_string': '15\\t48411280\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411280G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8326C>T',\n",
       "  'hgvs_protein': 'p.Arg2776Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411280,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48411311-A-T',\n",
       "  'variant_hash': '081cee7bacd3a990',\n",
       "  'vcf_string': '15\\t48411311\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411311A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8295T>A',\n",
       "  'hgvs_protein': 'p.Ala2765=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411311,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411323-T-A',\n",
       "  'variant_hash': 'a3bd60eb498e51c3',\n",
       "  'vcf_string': '15\\t48411323\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411323T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8283A>T',\n",
       "  'hgvs_protein': 'p.Thr2761=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411323,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48411338-C-T',\n",
       "  'variant_hash': 'aae9bd56ec0aac23',\n",
       "  'vcf_string': '15\\t48411338\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411338C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8268G>A',\n",
       "  'hgvs_protein': 'p.Trp2756Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411338,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48411350-A-G',\n",
       "  'variant_hash': '323b90011dc0028e',\n",
       "  'vcf_string': '15\\t48411350\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411350A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8256T>C',\n",
       "  'hgvs_protein': 'p.Ser2752=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411350,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411365-T-C',\n",
       "  'variant_hash': '1ffab58b6dcd7d41',\n",
       "  'vcf_string': '15\\t48411365\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411365T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8241A>G',\n",
       "  'hgvs_protein': 'p.Thr2747=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411365,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411374-C-T',\n",
       "  'variant_hash': '8168324abd30d4c7',\n",
       "  'vcf_string': '15\\t48411374\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411374C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8232G>A',\n",
       "  'hgvs_protein': 'p.Gln2744=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411374,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48411383-G-A',\n",
       "  'variant_hash': '3528213345cabe79',\n",
       "  'vcf_string': '15\\t48411383\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411383G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8227-4C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411383,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48411389-C-T',\n",
       "  'variant_hash': 'a610f40a1a34f14d',\n",
       "  'vcf_string': '15\\t48411389\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411389C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8227-10G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411389,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48411482-G-A',\n",
       "  'variant_hash': 'e0cb9a895eacc9eb',\n",
       "  'vcf_string': '15\\t48411482\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48411482G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8227-103C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48411482,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412564-C-T',\n",
       "  'variant_hash': '9c3c30981a9748af',\n",
       "  'vcf_string': '15\\t48412564\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412564C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8226+5G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412564,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_5th_base_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48412568-C-T',\n",
       "  'variant_hash': '278f0ad4490b691d',\n",
       "  'vcf_string': '15\\t48412568\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412568C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8226+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412568,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48412596-T-A',\n",
       "  'variant_hash': 'd684d505585043f7',\n",
       "  'vcf_string': '15\\t48412596\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412596T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8199A>T',\n",
       "  'hgvs_protein': 'p.Thr2733=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412596,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412626-G-A',\n",
       "  'variant_hash': 'c90225f8e46767d8',\n",
       "  'vcf_string': '15\\t48412626\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412626G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8169C>T',\n",
       "  'hgvs_protein': 'p.Tyr2723=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412626,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412638-C-T',\n",
       "  'variant_hash': '9269268d1ddd43ea',\n",
       "  'vcf_string': '15\\t48412638\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412638C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8157G>A',\n",
       "  'hgvs_protein': 'p.Lys2719=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412638,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412647-G-C',\n",
       "  'variant_hash': 'bcec1a5d5140ff28',\n",
       "  'vcf_string': '15\\t48412647\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412647G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8148C>G',\n",
       "  'hgvs_protein': 'p.Tyr2716Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412647,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48412647-G-A',\n",
       "  'variant_hash': 'db2ac6fe2b1e8b7b',\n",
       "  'vcf_string': '15\\t48412647\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412647G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8148C>T',\n",
       "  'hgvs_protein': 'p.Tyr2716=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412647,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412665-G-A',\n",
       "  'variant_hash': '6364157ce29f55d6',\n",
       "  'vcf_string': '15\\t48412665\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412665G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8130C>T',\n",
       "  'hgvs_protein': 'p.Leu2710=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412665,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412674-G-A',\n",
       "  'variant_hash': 'f865b60683f48ca0',\n",
       "  'vcf_string': '15\\t48412674\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412674G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8121C>T',\n",
       "  'hgvs_protein': 'p.Asp2707=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412674,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412692-G-A',\n",
       "  'variant_hash': 'b56608d079ecd001',\n",
       "  'vcf_string': '15\\t48412692\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412692G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8103C>T',\n",
       "  'hgvs_protein': 'p.Val2701=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412692,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412713-T-G',\n",
       "  'variant_hash': '7fd52c7d190202a2',\n",
       "  'vcf_string': '15\\t48412713\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412713T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8082A>C',\n",
       "  'hgvs_protein': 'p.Arg2694=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412713,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48412715-G-A',\n",
       "  'variant_hash': '781ff1571689f69e',\n",
       "  'vcf_string': '15\\t48412715\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412715G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8080C>T',\n",
       "  'hgvs_protein': 'p.Arg2694Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412715,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48412740-G-A',\n",
       "  'variant_hash': '54e3376754516310',\n",
       "  'vcf_string': '15\\t48412740\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48412740G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8055C>T',\n",
       "  'hgvs_protein': 'p.His2685=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48412740,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415516-C-A',\n",
       "  'variant_hash': 'd202f6ea392b33e3',\n",
       "  'vcf_string': '15\\t48415516\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415516C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8051+20G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415516,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415529-T-C',\n",
       "  'variant_hash': '43ddbcc860f654f5',\n",
       "  'vcf_string': '15\\t48415529\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415529T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8051+7A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415529,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415538-T-C',\n",
       "  'variant_hash': '0441ea9854248e45',\n",
       "  'vcf_string': '15\\t48415538\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415538T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8049A>G',\n",
       "  'hgvs_protein': 'p.Gln2683=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415538,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415540-G-A',\n",
       "  'variant_hash': 'c754dfd9fd49f104',\n",
       "  'vcf_string': '15\\t48415540\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415540G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8047C>T',\n",
       "  'hgvs_protein': 'p.Gln2683Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415540,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48415545-A-C',\n",
       "  'variant_hash': '7a1e7f5c72cac5a8',\n",
       "  'vcf_string': '15\\t48415545\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415545A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8042T>G',\n",
       "  'hgvs_protein': 'p.Ile2681Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415545,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48415549-G-A',\n",
       "  'variant_hash': '27dc8db741cb6699',\n",
       "  'vcf_string': '15\\t48415549\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415549G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8038C>T',\n",
       "  'hgvs_protein': 'p.Arg2680Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415549,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415550-G-A',\n",
       "  'variant_hash': '9ba6378e5e82f7d6',\n",
       "  'vcf_string': '15\\t48415550\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415550G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8037C>T',\n",
       "  'hgvs_protein': 'p.Phe2679=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415550,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415553-G-A',\n",
       "  'variant_hash': '83bb80360d778a16',\n",
       "  'vcf_string': '15\\t48415553\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415553G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8034C>T',\n",
       "  'hgvs_protein': 'p.Tyr2678=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415553,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415566-C-T',\n",
       "  'variant_hash': '36e595d48f055eb0',\n",
       "  'vcf_string': '15\\t48415566\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415566C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8021G>A',\n",
       "  'hgvs_protein': 'p.Cys2674Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415566,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415567-A-G',\n",
       "  'variant_hash': '44b1a6379c9a90df',\n",
       "  'vcf_string': '15\\t48415567\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415567A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8020T>C',\n",
       "  'hgvs_protein': 'p.Cys2674Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415567,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415573-A-C',\n",
       "  'variant_hash': 'c04a6023a48961f5',\n",
       "  'vcf_string': '15\\t48415573\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415573A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8014T>G',\n",
       "  'hgvs_protein': 'p.Cys2672Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415573,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48415581-C-A',\n",
       "  'variant_hash': 'd5c12cdc3fc9da9b',\n",
       "  'vcf_string': '15\\t48415581\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415581C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8006G>T',\n",
       "  'hgvs_protein': 'p.Gly2669Val',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415581,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415582-C-A',\n",
       "  'variant_hash': 'affa4d7740e22530',\n",
       "  'vcf_string': '15\\t48415582\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415582C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8005G>T',\n",
       "  'hgvs_protein': 'p.Gly2669Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415582,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415583-G-A',\n",
       "  'variant_hash': '84347cd72e17e59c',\n",
       "  'vcf_string': '15\\t48415583\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415583G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8004C>T',\n",
       "  'hgvs_protein': 'p.Gly2668=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415583,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415585-C-A',\n",
       "  'variant_hash': '6ee4307d50f0b1fc',\n",
       "  'vcf_string': '15\\t48415585\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415585C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.8002G>T',\n",
       "  'hgvs_protein': 'p.Gly2668Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415585,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48415589-G-A',\n",
       "  'variant_hash': '274f622cfadf0cd8',\n",
       "  'vcf_string': '15\\t48415589\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415589G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7998C>T',\n",
       "  'hgvs_protein': 'p.Thr2666=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415589,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415605-T-C',\n",
       "  'variant_hash': '239a6a713e9fabca',\n",
       "  'vcf_string': '15\\t48415605\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415605T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7982A>G',\n",
       "  'hgvs_protein': 'p.Tyr2661Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415605,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415610-G-T',\n",
       "  'variant_hash': '49eef4aa4e022c19',\n",
       "  'vcf_string': '15\\t48415610\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415610G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7977C>A',\n",
       "  'hgvs_protein': 'p.Cys2659Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415610,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48415619-C-T',\n",
       "  'variant_hash': '552682e8385de420',\n",
       "  'vcf_string': '15\\t48415619\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415619C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7968G>A',\n",
       "  'hgvs_protein': 'p.Gln2656=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415619,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415622-C-T',\n",
       "  'variant_hash': '130943e189812347',\n",
       "  'vcf_string': '15\\t48415622\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415622C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7965G>A',\n",
       "  'hgvs_protein': 'p.Ala2655=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415622,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415632-C-T',\n",
       "  'variant_hash': '18be341e2c8303dd',\n",
       "  'vcf_string': '15\\t48415632\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415632C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7955G>A',\n",
       "  'hgvs_protein': 'p.Cys2652Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415632,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415637-A-G',\n",
       "  'variant_hash': 'e1dcbae09f742563',\n",
       "  'vcf_string': '15\\t48415637\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415637A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7950T>C',\n",
       "  'hgvs_protein': 'p.Asn2650=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415637,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48415655-T-A',\n",
       "  'variant_hash': '46703132bc16b7b8',\n",
       "  'vcf_string': '15\\t48415655\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415655T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7932A>T',\n",
       "  'hgvs_protein': 'p.Gly2644=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415655,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415658-A-G',\n",
       "  'variant_hash': 'cfc9345143326fbf',\n",
       "  'vcf_string': '15\\t48415658\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415658A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7929T>C',\n",
       "  'hgvs_protein': 'p.Ser2643=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415658,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415671-T-C',\n",
       "  'variant_hash': 'ecdf5756d2fbd932',\n",
       "  'vcf_string': '15\\t48415671\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415671T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7916A>G',\n",
       "  'hgvs_protein': 'p.Tyr2639Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415671,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415675-G-A',\n",
       "  'variant_hash': '65f007b83fdc5c29',\n",
       "  'vcf_string': '15\\t48415675\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415675G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7912C>T',\n",
       "  'hgvs_protein': 'p.Gln2638Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415675,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48415682-G-A',\n",
       "  'variant_hash': 'edc6e3efc1d6123f',\n",
       "  'vcf_string': '15\\t48415682\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415682G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7905C>T',\n",
       "  'hgvs_protein': 'p.Ala2635=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415682,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415684-C-T',\n",
       "  'variant_hash': 'a8c92ae828334745',\n",
       "  'vcf_string': '15\\t48415684\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415684C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7903G>A',\n",
       "  'hgvs_protein': 'p.Ala2635Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415684,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48415690-A-G',\n",
       "  'variant_hash': '578628593cfc8743',\n",
       "  'vcf_string': '15\\t48415690\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415690A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7897T>C',\n",
       "  'hgvs_protein': 'p.Cys2633Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415690,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415694-G-A',\n",
       "  'variant_hash': 'd350af886e66e070',\n",
       "  'vcf_string': '15\\t48415694\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415694G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7893C>T',\n",
       "  'hgvs_protein': 'p.Cys2631=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415694,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415695-C-T',\n",
       "  'variant_hash': 'd660f76476200c8c',\n",
       "  'vcf_string': '15\\t48415695\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415695C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7892G>A',\n",
       "  'hgvs_protein': 'p.Cys2631Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415695,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48415695-C-A',\n",
       "  'variant_hash': 'dc78ccffc830753b',\n",
       "  'vcf_string': '15\\t48415695\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415695C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7892G>T',\n",
       "  'hgvs_protein': 'p.Cys2631Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415695,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415701-T-C',\n",
       "  'variant_hash': 'ad4762b0a0d479c9',\n",
       "  'vcf_string': '15\\t48415701\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415701T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7886A>G',\n",
       "  'hgvs_protein': 'p.Tyr2629Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415701,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415706-C-T',\n",
       "  'variant_hash': 'd3c1be8eef67e694',\n",
       "  'vcf_string': '15\\t48415706\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415706C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7881G>A',\n",
       "  'hgvs_protein': 'p.Gly2627=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415706,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415707-C-T',\n",
       "  'variant_hash': 'fd4dc6ece4804b20',\n",
       "  'vcf_string': '15\\t48415707\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415707C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7880G>A',\n",
       "  'hgvs_protein': 'p.Gly2627Glu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415707,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48415708-C-T',\n",
       "  'variant_hash': '35e5cf4d77428252',\n",
       "  'vcf_string': '15\\t48415708\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415708C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7879G>A',\n",
       "  'hgvs_protein': 'p.Gly2627Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415708,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415708-C-G',\n",
       "  'variant_hash': 'fb5747d3e9bfbe9e',\n",
       "  'vcf_string': '15\\t48415708\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415708C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7879G>C',\n",
       "  'hgvs_protein': 'p.Gly2627Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415708,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48415712-G-A',\n",
       "  'variant_hash': '861a1fae03d9106c',\n",
       "  'vcf_string': '15\\t48415712\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415712G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7875C>T',\n",
       "  'hgvs_protein': 'p.Thr2625=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415712,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415715-G-A',\n",
       "  'variant_hash': 'cd2b1df5c2dcc4f4',\n",
       "  'vcf_string': '15\\t48415715\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415715G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7872C>T',\n",
       "  'hgvs_protein': 'p.Asn2624=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415715,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415716-T-C',\n",
       "  'variant_hash': 'c8781651fe8f4806',\n",
       "  'vcf_string': '15\\t48415716\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415716T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7871A>G',\n",
       "  'hgvs_protein': 'p.Asn2624Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415716,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415722-C-G',\n",
       "  'variant_hash': '1cf889f31f97d8f9',\n",
       "  'vcf_string': '15\\t48415722\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415722C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7865G>C',\n",
       "  'hgvs_protein': 'p.Cys2622Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415722,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415723-A-G',\n",
       "  'variant_hash': 'f55acc581bf834fc',\n",
       "  'vcf_string': '15\\t48415723\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415723A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7864T>C',\n",
       "  'hgvs_protein': 'p.Cys2622Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415723,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48415736-G-A',\n",
       "  'variant_hash': '66e9516d68eb1578',\n",
       "  'vcf_string': '15\\t48415736\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415736G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7851C>T',\n",
       "  'hgvs_protein': 'p.Cys2617=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415736,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415745-A-T',\n",
       "  'variant_hash': '48b07305a0a9d835',\n",
       "  'vcf_string': '15\\t48415745\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415745A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7842T>A',\n",
       "  'hgvs_protein': 'p.Ala2614=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415745,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415745-A-C',\n",
       "  'variant_hash': '7e1df03f208df6db',\n",
       "  'vcf_string': '15\\t48415745\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415745A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7842T>G',\n",
       "  'hgvs_protein': 'p.Ala2614=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415745,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415748-G-A',\n",
       "  'variant_hash': 'e605c27fd412432f',\n",
       "  'vcf_string': '15\\t48415748\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415748G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7839C>T',\n",
       "  'hgvs_protein': 'p.Ser2613=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415748,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415751-G-A',\n",
       "  'variant_hash': '73be9109574d7afe',\n",
       "  'vcf_string': '15\\t48415751\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415751G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7836C>T',\n",
       "  'hgvs_protein': 'p.Leu2612=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415751,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415755-C-T',\n",
       "  'variant_hash': '664f512ae4e3cfda',\n",
       "  'vcf_string': '15\\t48415755\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415755C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7832G>A',\n",
       "  'hgvs_protein': 'p.Cys2611Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415755,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48415760-G-A',\n",
       "  'variant_hash': '3a545661008fd1ba',\n",
       "  'vcf_string': '15\\t48415760\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415760G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7827C>T',\n",
       "  'hgvs_protein': 'p.Asn2609=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415760,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48415765-C-A',\n",
       "  'variant_hash': 'af5c755c1cb95aed',\n",
       "  'vcf_string': '15\\t48415765\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48415765C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7822G>T',\n",
       "  'hgvs_protein': 'p.Glu2608Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48415765,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420545-C-T',\n",
       "  'variant_hash': '171a7ad55ff37a90',\n",
       "  'vcf_string': '15\\t48420545\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420545C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7819+142G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420545,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420547-C-T',\n",
       "  'variant_hash': 'c443e77e9c136832',\n",
       "  'vcf_string': '15\\t48420547\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420547C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7819+140G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420547,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420679-T-G',\n",
       "  'variant_hash': 'c8f5d695f47ce447',\n",
       "  'vcf_string': '15\\t48420679\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420679T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7819+8A>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420679,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48420686-C-T',\n",
       "  'variant_hash': '6f78ccb3a5bf6a85',\n",
       "  'vcf_string': '15\\t48420686\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420686C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7819+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420686,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420687-C-T',\n",
       "  'variant_hash': 'a64370cabe0bad93',\n",
       "  'vcf_string': '15\\t48420687\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420687C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7819G>A',\n",
       "  'hgvs_protein': 'p.Asp2607Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420687,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48420700-C-T',\n",
       "  'variant_hash': '944a2672daaabe4e',\n",
       "  'vcf_string': '15\\t48420700\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420700C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7806G>A',\n",
       "  'hgvs_protein': 'p.Trp2602Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420700,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420705-G-A',\n",
       "  'variant_hash': 'e0cd04128b6b0ff7',\n",
       "  'vcf_string': '15\\t48420705\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420705G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7801C>T',\n",
       "  'hgvs_protein': 'p.Gln2601Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420705,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420706-G-C',\n",
       "  'variant_hash': 'd22220199a31802c',\n",
       "  'vcf_string': '15\\t48420706\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420706G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7800C>G',\n",
       "  'hgvs_protein': 'p.Tyr2600Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420706,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420714-G-A',\n",
       "  'variant_hash': 'c765d2fc11795cca',\n",
       "  'vcf_string': '15\\t48420714\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420714G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7792C>T',\n",
       "  'hgvs_protein': 'p.Gln2598Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420714,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420718-G-T',\n",
       "  'variant_hash': 'dadc019010006a43',\n",
       "  'vcf_string': '15\\t48420718\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420718G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7788C>A',\n",
       "  'hgvs_protein': 'p.Tyr2596Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420718,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420718-G-A',\n",
       "  'variant_hash': '19ec0657906b294a',\n",
       "  'vcf_string': '15\\t48420718\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420718G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7788C>T',\n",
       "  'hgvs_protein': 'p.Tyr2596=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420718,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420719-T-C',\n",
       "  'variant_hash': '7962c506ada0f091',\n",
       "  'vcf_string': '15\\t48420719\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420719T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7787A>G',\n",
       "  'hgvs_protein': 'p.Tyr2596Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420719,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420731-C-T',\n",
       "  'variant_hash': 'a667eceb65040f56',\n",
       "  'vcf_string': '15\\t48420731\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420731C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7775G>A',\n",
       "  'hgvs_protein': 'p.Cys2592Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420731,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420731-C-A',\n",
       "  'variant_hash': 'e83ce1f43523866a',\n",
       "  'vcf_string': '15\\t48420731\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420731C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7775G>T',\n",
       "  'hgvs_protein': 'p.Cys2592Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420731,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420733-G-A',\n",
       "  'variant_hash': '4d5b4aaf774979da',\n",
       "  'vcf_string': '15\\t48420733\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420733G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7773C>T',\n",
       "  'hgvs_protein': 'p.Ser2591=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420733,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420736-G-C',\n",
       "  'variant_hash': 'eee965a4e34307ab',\n",
       "  'vcf_string': '15\\t48420736\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420736G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7770C>G',\n",
       "  'hgvs_protein': 'p.Cys2590Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420736,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48420737-C-T',\n",
       "  'variant_hash': '1b638d7ca57d632d',\n",
       "  'vcf_string': '15\\t48420737\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420737C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7769G>A',\n",
       "  'hgvs_protein': 'p.Cys2590Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420737,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420752-A-G',\n",
       "  'variant_hash': 'd9a98c08813c11bd',\n",
       "  'vcf_string': '15\\t48420752\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420752A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7754T>C',\n",
       "  'hgvs_protein': 'p.Ile2585Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420752,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420757-G-A',\n",
       "  'variant_hash': '00a8cdabce2bb538',\n",
       "  'vcf_string': '15\\t48420757\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420757G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7749C>T',\n",
       "  'hgvs_protein': 'p.Asn2583=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420757,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48420763-G-A',\n",
       "  'variant_hash': '7de6776d3ebebfd8',\n",
       "  'vcf_string': '15\\t48420763\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420763G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7743C>T',\n",
       "  'hgvs_protein': 'p.Cys2581=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420763,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48420778-G-C',\n",
       "  'variant_hash': '211ee5755026a11b',\n",
       "  'vcf_string': '15\\t48420778\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420778G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7728C>G',\n",
       "  'hgvs_protein': 'p.Arg2576=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420778,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420778-G-A',\n",
       "  'variant_hash': 'c2a002478ee20aad',\n",
       "  'vcf_string': '15\\t48420778\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420778G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7728C>T',\n",
       "  'hgvs_protein': 'p.Arg2576=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420778,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420780-G-A',\n",
       "  'variant_hash': '84ac46dda0dfed69',\n",
       "  'vcf_string': '15\\t48420780\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420780G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7726C>T',\n",
       "  'hgvs_protein': 'p.Arg2576Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420780,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48420781-G-A',\n",
       "  'variant_hash': '2986ff5d2b5b3824',\n",
       "  'vcf_string': '15\\t48420781\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420781G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7725C>T',\n",
       "  'hgvs_protein': 'p.His2575=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420781,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420794-C-T',\n",
       "  'variant_hash': 'ada190ebb2ae9fcf',\n",
       "  'vcf_string': '15\\t48420794\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420794C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7712G>A',\n",
       "  'hgvs_protein': 'p.Cys2571Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420794,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420795-A-G',\n",
       "  'variant_hash': '115e34e3f4c66e70',\n",
       "  'vcf_string': '15\\t48420795\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420795A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7711T>C',\n",
       "  'hgvs_protein': 'p.Cys2571Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420795,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420796-C-T',\n",
       "  'variant_hash': 'fec31f4c66b89658',\n",
       "  'vcf_string': '15\\t48420796\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420796C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7710G>A',\n",
       "  'hgvs_protein': 'p.Glu2570=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420796,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420798-C-T',\n",
       "  'variant_hash': '0c819141140561d1',\n",
       "  'vcf_string': '15\\t48420798\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420798C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7708G>A',\n",
       "  'hgvs_protein': 'p.Glu2570Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420798,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48420799-G-A',\n",
       "  'variant_hash': '53d8096949b45ca7',\n",
       "  'vcf_string': '15\\t48420799\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420799G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7707C>T',\n",
       "  'hgvs_protein': 'p.Asp2569=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420799,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420815-G-A',\n",
       "  'variant_hash': 'b8184af11e1a064a',\n",
       "  'vcf_string': '15\\t48420815\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420815G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7700-9C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420815,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48420819-C-G',\n",
       "  'variant_hash': 'ba223d5b54135f7c',\n",
       "  'vcf_string': '15\\t48420819\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48420819C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7700-13G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48420819,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421540-G-A',\n",
       "  'variant_hash': '8957fca455cf05d2',\n",
       "  'vcf_string': '15\\t48421540\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421540G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7699+18C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421540,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48421543-G-A',\n",
       "  'variant_hash': '2adc65e43198ae3d',\n",
       "  'vcf_string': '15\\t48421543\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421543G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7699+15C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421543,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421557-C-T',\n",
       "  'variant_hash': '1a015bbcada3d541',\n",
       "  'vcf_string': '15\\t48421557\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421557C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7699+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421557,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421574-G-A',\n",
       "  'variant_hash': '76f171251f50c716',\n",
       "  'vcf_string': '15\\t48421574\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421574G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7683C>T',\n",
       "  'hgvs_protein': 'p.Thr2561=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421574,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421580-A-G',\n",
       "  'variant_hash': '64df07e4b55b055c',\n",
       "  'vcf_string': '15\\t48421580\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421580A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7677T>C',\n",
       "  'hgvs_protein': 'p.Asp2559=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421580,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421593-C-A',\n",
       "  'variant_hash': '2c1d369bcd0b0de2',\n",
       "  'vcf_string': '15\\t48421593\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421593C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7664G>T',\n",
       "  'hgvs_protein': 'p.Gly2555Val',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421593,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421594-C-T',\n",
       "  'variant_hash': '0e5e687741a2e073',\n",
       "  'vcf_string': '15\\t48421594\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421594C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7663G>A',\n",
       "  'hgvs_protein': 'p.Gly2555Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421594,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421601-G-T',\n",
       "  'variant_hash': '17866ede0ac03538',\n",
       "  'vcf_string': '15\\t48421601\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421601G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7656C>A',\n",
       "  'hgvs_protein': 'p.Cys2552Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421601,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421603-A-G',\n",
       "  'variant_hash': '94316ce547240140',\n",
       "  'vcf_string': '15\\t48421603\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421603A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7654T>C',\n",
       "  'hgvs_protein': 'p.Cys2552Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421603,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48421608-C-T',\n",
       "  'variant_hash': 'c82b6fb402724538',\n",
       "  'vcf_string': '15\\t48421608\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421608C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7649G>A',\n",
       "  'hgvs_protein': 'p.Cys2550Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421608,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421609-A-T',\n",
       "  'variant_hash': 'b5f10c5aeca14e29',\n",
       "  'vcf_string': '15\\t48421609\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421609A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7648T>A',\n",
       "  'hgvs_protein': 'p.Cys2550Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421609,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48421625-A-G',\n",
       "  'variant_hash': 'c2669fa552696bb1',\n",
       "  'vcf_string': '15\\t48421625\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421625A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7632T>C',\n",
       "  'hgvs_protein': 'p.Thr2544=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421625,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421651-C-T',\n",
       "  'variant_hash': '6a710c64525ec606',\n",
       "  'vcf_string': '15\\t48421651\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421651C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7606G>A',\n",
       "  'hgvs_protein': 'p.Gly2536Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421651,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421652-G-T',\n",
       "  'variant_hash': '3bb26e494e5731b8',\n",
       "  'vcf_string': '15\\t48421652\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421652G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7605C>A',\n",
       "  'hgvs_protein': 'p.Cys2535Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421652,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421652-G-C',\n",
       "  'variant_hash': '66cebd9d1a480acb',\n",
       "  'vcf_string': '15\\t48421652\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421652G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7605C>G',\n",
       "  'hgvs_protein': 'p.Cys2535Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421652,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421652-G-A',\n",
       "  'variant_hash': '086d4529a282d89f',\n",
       "  'vcf_string': '15\\t48421652\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421652G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7605C>T',\n",
       "  'hgvs_protein': 'p.Cys2535=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421652,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421653-C-T',\n",
       "  'variant_hash': '14ccc2ffdb57f0df',\n",
       "  'vcf_string': '15\\t48421653\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421653C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7604G>A',\n",
       "  'hgvs_protein': 'p.Cys2535Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421653,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421655-C-T',\n",
       "  'variant_hash': '871eda844e519e03',\n",
       "  'vcf_string': '15\\t48421655\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421655C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7602G>A',\n",
       "  'hgvs_protein': 'p.Leu2534=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421655,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421675-A-G',\n",
       "  'variant_hash': '2d48855827740eb0',\n",
       "  'vcf_string': '15\\t48421675\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421675A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7582T>C',\n",
       "  'hgvs_protein': 'p.Cys2528Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421675,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421680-T-C',\n",
       "  'variant_hash': '5b063864f0be3393',\n",
       "  'vcf_string': '15\\t48421680\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421680T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7577A>G',\n",
       "  'hgvs_protein': 'p.Asn2526Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421680,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48421682-G-A',\n",
       "  'variant_hash': 'b60c7f86c9c9d8a1',\n",
       "  'vcf_string': '15\\t48421682\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421682G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7575C>T',\n",
       "  'hgvs_protein': 'p.Asn2525=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421682,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421687-C-T',\n",
       "  'variant_hash': '7c498f95c8743c09',\n",
       "  'vcf_string': '15\\t48421687\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421687C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7571-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421687,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48421762-A-T',\n",
       "  'variant_hash': '4b9748df345814d5',\n",
       "  'vcf_string': '15\\t48421762\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421762A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7571-76T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421762,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48421799-G-T',\n",
       "  'variant_hash': '1c24435950fee4b6',\n",
       "  'vcf_string': '15\\t48421799\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421799G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7571-113C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421799,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48421939-C-T',\n",
       "  'variant_hash': 'c5f408d842e193d8',\n",
       "  'vcf_string': '15\\t48421939\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421939C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7570+13G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421939,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421973-G-A',\n",
       "  'variant_hash': '31644dba23f966fb',\n",
       "  'vcf_string': '15\\t48421973\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421973G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7549C>T',\n",
       "  'hgvs_protein': 'p.Gln2517Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421973,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421981-C-T',\n",
       "  'variant_hash': 'ce97621862880e29',\n",
       "  'vcf_string': '15\\t48421981\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421981C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7541G>A',\n",
       "  'hgvs_protein': 'p.Gly2514Glu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421981,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48421982-C-T',\n",
       "  'variant_hash': '68cd739bbc90c779',\n",
       "  'vcf_string': '15\\t48421982\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421982C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7540G>A',\n",
       "  'hgvs_protein': 'p.Gly2514Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421982,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421982-C-A',\n",
       "  'variant_hash': '4a1b10457c353e7f',\n",
       "  'vcf_string': '15\\t48421982\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421982C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7540G>T',\n",
       "  'hgvs_protein': 'p.Gly2514Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421982,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421983-G-A',\n",
       "  'variant_hash': '9e2dde939447deab',\n",
       "  'vcf_string': '15\\t48421983\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421983G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7539C>T',\n",
       "  'hgvs_protein': 'p.Pro2513=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421983,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421989-A-G',\n",
       "  'variant_hash': 'c181c097a7ae2496',\n",
       "  'vcf_string': '15\\t48421989\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421989A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7533T>C',\n",
       "  'hgvs_protein': 'p.Cys2511=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421989,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421989-A-C',\n",
       "  'variant_hash': '362ee0519c207cab',\n",
       "  'vcf_string': '15\\t48421989\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421989A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7533T>G',\n",
       "  'hgvs_protein': 'p.Cys2511Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421989,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421990-C-T',\n",
       "  'variant_hash': 'af127d42d150e575',\n",
       "  'vcf_string': '15\\t48421990\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421990C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7532G>A',\n",
       "  'hgvs_protein': 'p.Cys2511Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421990,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48421991-A-G',\n",
       "  'variant_hash': '52910bd2a4d4debe',\n",
       "  'vcf_string': '15\\t48421991\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421991A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7531T>C',\n",
       "  'hgvs_protein': 'p.Cys2511Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421991,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421992-T-C',\n",
       "  'variant_hash': 'a32de820b534543d',\n",
       "  'vcf_string': '15\\t48421992\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421992T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7530A>G',\n",
       "  'hgvs_protein': 'p.Lys2510=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421992,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421995-G-T',\n",
       "  'variant_hash': '28a145e4ee9ab3d2',\n",
       "  'vcf_string': '15\\t48421995\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421995G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7527C>A',\n",
       "  'hgvs_protein': 'p.Cys2509Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421995,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48421997-A-C',\n",
       "  'variant_hash': 'a5ff04f91680ab09',\n",
       "  'vcf_string': '15\\t48421997\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421997A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7525T>G',\n",
       "  'hgvs_protein': 'p.Cys2509Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421997,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48421998-T-C',\n",
       "  'variant_hash': 'cbdfbf3df19760ac',\n",
       "  'vcf_string': '15\\t48421998\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421998T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7524A>G',\n",
       "  'hgvs_protein': 'p.Thr2508=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421998,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48421998-T-A',\n",
       "  'variant_hash': '671d042c24093983',\n",
       "  'vcf_string': '15\\t48421998\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48421998T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7524A>T',\n",
       "  'hgvs_protein': 'p.Thr2508=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48421998,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48422001-G-A',\n",
       "  'variant_hash': '6726883aa1ca40ae',\n",
       "  'vcf_string': '15\\t48422001\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422001G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7521C>T',\n",
       "  'hgvs_protein': 'p.Phe2507=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422001,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48422007-G-A',\n",
       "  'variant_hash': '8ef0c2336574f849',\n",
       "  'vcf_string': '15\\t48422007\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422007G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7515C>T',\n",
       "  'hgvs_protein': 'p.Gly2505=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422007,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48422023-C-T',\n",
       "  'variant_hash': '29c3fd2f3654cc07',\n",
       "  'vcf_string': '15\\t48422023\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422023C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7499G>A',\n",
       "  'hgvs_protein': 'p.Cys2500Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422023,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48422024-A-T',\n",
       "  'variant_hash': '582427b2ad0b4fb6',\n",
       "  'vcf_string': '15\\t48422024\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422024A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7498T>A',\n",
       "  'hgvs_protein': 'p.Cys2500Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422024,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48422024-A-G',\n",
       "  'variant_hash': 'e3180748801d1821',\n",
       "  'vcf_string': '15\\t48422024\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422024A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7498T>C',\n",
       "  'hgvs_protein': 'p.Cys2500Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422024,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48422025-T-C',\n",
       "  'variant_hash': '6b4dd5c24d82e225',\n",
       "  'vcf_string': '15\\t48422025\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422025T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7497A>G',\n",
       "  'hgvs_protein': 'p.Leu2499=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422025,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48422046-C-T',\n",
       "  'variant_hash': '53a50eedf6a1cf33',\n",
       "  'vcf_string': '15\\t48422046\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422046C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7476G>A',\n",
       "  'hgvs_protein': 'p.Lys2492=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422046,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48422068-T-C',\n",
       "  'variant_hash': '21bf32835aa1a28e',\n",
       "  'vcf_string': '15\\t48422068\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422068T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7454A>G',\n",
       "  'hgvs_protein': 'p.Asp2485Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422068,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48422069-C-A',\n",
       "  'variant_hash': '1fe8751e994a5f1f',\n",
       "  'vcf_string': '15\\t48422069\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422069C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7454-1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422069,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48422074-A-G',\n",
       "  'variant_hash': 'cac151c816777307',\n",
       "  'vcf_string': '15\\t48422074\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48422074A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7454-6T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48422074,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425368-C-T',\n",
       "  'variant_hash': '63404e53d261bd6c',\n",
       "  'vcf_string': '15\\t48425368\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425368C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7453+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425368,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425374-C-T',\n",
       "  'variant_hash': 'bc4a61ad98280d52',\n",
       "  'vcf_string': '15\\t48425374\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425374C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7448G>A',\n",
       "  'hgvs_protein': 'p.Cys2483Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425374,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425375-A-G',\n",
       "  'variant_hash': 'b87786f6021f0334',\n",
       "  'vcf_string': '15\\t48425375\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425375A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7447T>C',\n",
       "  'hgvs_protein': 'p.Cys2483Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425375,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425393-G-A',\n",
       "  'variant_hash': '3b5091f4d0d02f23',\n",
       "  'vcf_string': '15\\t48425393\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425393G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7429C>T',\n",
       "  'hgvs_protein': 'p.Gln2477Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425393,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425394-C-T',\n",
       "  'variant_hash': '12fbb69f9fff6fb6',\n",
       "  'vcf_string': '15\\t48425394\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425394C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7428G>A',\n",
       "  'hgvs_protein': 'p.Leu2476=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425394,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425401-T-C',\n",
       "  'variant_hash': '89e6452f989bf1d2',\n",
       "  'vcf_string': '15\\t48425401\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425401T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7421A>G',\n",
       "  'hgvs_protein': 'p.Tyr2474Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425401,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48425409-C-T',\n",
       "  'variant_hash': '4dfa402f37a0e7db',\n",
       "  'vcf_string': '15\\t48425409\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425409C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7413G>A',\n",
       "  'hgvs_protein': 'p.Pro2471=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425409,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425412-G-C',\n",
       "  'variant_hash': '9a30b38daa3bf731',\n",
       "  'vcf_string': '15\\t48425412\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425412G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7410C>G',\n",
       "  'hgvs_protein': 'p.Cys2470Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425412,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425413-C-A',\n",
       "  'variant_hash': '09b707cd25afd6dd',\n",
       "  'vcf_string': '15\\t48425413\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425413C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7409G>T',\n",
       "  'hgvs_protein': 'p.Cys2470Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425413,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48425414-A-G',\n",
       "  'variant_hash': 'a1ece309f807be3c',\n",
       "  'vcf_string': '15\\t48425414\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425414A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7408T>C',\n",
       "  'hgvs_protein': 'p.Cys2470Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425414,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48425415-T-C',\n",
       "  'variant_hash': '1d5a7151b5f5279a',\n",
       "  'vcf_string': '15\\t48425415\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425415T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7407A>G',\n",
       "  'hgvs_protein': 'p.Ser2469=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425415,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425416-G-T',\n",
       "  'variant_hash': '802cff98f9bcc549',\n",
       "  'vcf_string': '15\\t48425416\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425416G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7406C>A',\n",
       "  'hgvs_protein': 'p.Ser2469Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425416,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425419-C-T',\n",
       "  'variant_hash': 'bf0d94837ee8709c',\n",
       "  'vcf_string': '15\\t48425419\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425419C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7403G>A',\n",
       "  'hgvs_protein': 'p.Cys2468Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425419,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425420-A-G',\n",
       "  'variant_hash': 'a1a335525fbd5c7a',\n",
       "  'vcf_string': '15\\t48425420\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425420A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7402T>C',\n",
       "  'hgvs_protein': 'p.Cys2468Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425420,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425423-G-A',\n",
       "  'variant_hash': '92b3a65459d68cad',\n",
       "  'vcf_string': '15\\t48425423\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425423G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7399C>T',\n",
       "  'hgvs_protein': 'p.Gln2467Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425423,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425424-G-T',\n",
       "  'variant_hash': '4dc27a0230e14d70',\n",
       "  'vcf_string': '15\\t48425424\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425424G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7398C>A',\n",
       "  'hgvs_protein': 'p.Tyr2466Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425424,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425427-A-G',\n",
       "  'variant_hash': 'dece7a1962d77fc2',\n",
       "  'vcf_string': '15\\t48425427\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425427A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7395T>C',\n",
       "  'hgvs_protein': 'p.Ser2465=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425427,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425435-C-A',\n",
       "  'variant_hash': 'de304dfb4d998abe',\n",
       "  'vcf_string': '15\\t48425435\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425435C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7387G>T',\n",
       "  'hgvs_protein': 'p.Glu2463Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425435,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425446-C-T',\n",
       "  'variant_hash': 'dbb9c7b2c98936e5',\n",
       "  'vcf_string': '15\\t48425446\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425446C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7376G>A',\n",
       "  'hgvs_protein': 'p.Cys2459Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425446,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48425458-C-T',\n",
       "  'variant_hash': '35cbf4206cfedb35',\n",
       "  'vcf_string': '15\\t48425458\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425458C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7364G>A',\n",
       "  'hgvs_protein': 'p.Cys2455Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425458,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425474-G-A',\n",
       "  'variant_hash': '8450cf06f349ca43',\n",
       "  'vcf_string': '15\\t48425474\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425474G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7348C>T',\n",
       "  'hgvs_protein': 'p.Gln2450Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425474,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425479-C-T',\n",
       "  'variant_hash': '40e55ee2959dc543',\n",
       "  'vcf_string': '15\\t48425479\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425479C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7343G>A',\n",
       "  'hgvs_protein': 'p.Cys2448Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425479,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425483-C-T',\n",
       "  'variant_hash': '17b26d478df11415',\n",
       "  'vcf_string': '15\\t48425483\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425483C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7339G>A',\n",
       "  'hgvs_protein': 'p.Glu2447Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425483,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425484-G-A',\n",
       "  'variant_hash': '47f10e3b69cbb089',\n",
       "  'vcf_string': '15\\t48425484\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425484G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7338C>T',\n",
       "  'hgvs_protein': 'p.Asn2446=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425484,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425487-C-G',\n",
       "  'variant_hash': '4548431078781190',\n",
       "  'vcf_string': '15\\t48425487\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425487C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7335G>C',\n",
       "  'hgvs_protein': 'p.Leu2445=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425487,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425487-C-A',\n",
       "  'variant_hash': '4d3e96475ee1e590',\n",
       "  'vcf_string': '15\\t48425487\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425487C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7335G>T',\n",
       "  'hgvs_protein': 'p.Leu2445=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425487,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425729-A-C',\n",
       "  'variant_hash': '2e1bda42fd696061',\n",
       "  'vcf_string': '15\\t48425729\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425729A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7330+10T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425729,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425735-T-C',\n",
       "  'variant_hash': '1c60f98b4df33305',\n",
       "  'vcf_string': '15\\t48425735\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425735T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7330+4A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425735,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48425738-C-T',\n",
       "  'variant_hash': 'e6fc8a97cd497f51',\n",
       "  'vcf_string': '15\\t48425738\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425738C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7330+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425738,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425738-C-A',\n",
       "  'variant_hash': 'a394e8f0350cdf5c',\n",
       "  'vcf_string': '15\\t48425738\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425738C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7330+1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425738,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425744-C-T',\n",
       "  'variant_hash': 'ad0917749b2c5543',\n",
       "  'vcf_string': '15\\t48425744\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425744C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7325G>A',\n",
       "  'hgvs_protein': 'p.Cys2442Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425744,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425745-A-T',\n",
       "  'variant_hash': '09af6544166f7a1a',\n",
       "  'vcf_string': '15\\t48425745\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425745A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7324T>A',\n",
       "  'hgvs_protein': 'p.Cys2442Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425745,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425752-C-G',\n",
       "  'variant_hash': 'f352e3d8f79fbf05',\n",
       "  'vcf_string': '15\\t48425752\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425752C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7317G>C',\n",
       "  'hgvs_protein': 'p.Gly2439=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425752,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425764-T-C',\n",
       "  'variant_hash': 'b197f1e249319dfb',\n",
       "  'vcf_string': '15\\t48425764\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425764T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7305A>G',\n",
       "  'hgvs_protein': 'p.Pro2435=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425764,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425770-G-A',\n",
       "  'variant_hash': 'b0915328dc62e6b8',\n",
       "  'vcf_string': '15\\t48425770\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425770G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7299C>T',\n",
       "  'hgvs_protein': 'p.Tyr2433=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425770,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425783-C-A',\n",
       "  'variant_hash': '93e06a137764883e',\n",
       "  'vcf_string': '15\\t48425783\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425783C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7286G>T',\n",
       "  'hgvs_protein': 'p.Cys2429Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425783,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425797-T-C',\n",
       "  'variant_hash': 'd855a644abca457c',\n",
       "  'vcf_string': '15\\t48425797\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425797T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7272A>G',\n",
       "  'hgvs_protein': 'p.Ser2424=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425797,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425802-C-A',\n",
       "  'variant_hash': '10982efaf35ee0a5',\n",
       "  'vcf_string': '15\\t48425802\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425802C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7267G>T',\n",
       "  'hgvs_protein': 'p.Gly2423Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425802,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425803-T-C',\n",
       "  'variant_hash': 'f8402c7de1dbd862',\n",
       "  'vcf_string': '15\\t48425803\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425803T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7266A>G',\n",
       "  'hgvs_protein': 'p.Arg2422=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425803,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425816-C-T',\n",
       "  'variant_hash': 'f831701ca580038c',\n",
       "  'vcf_string': '15\\t48425816\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425816C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7253G>A',\n",
       "  'hgvs_protein': 'p.Cys2418Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425816,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425820-C-A',\n",
       "  'variant_hash': '43878a6cbbbdcc4e',\n",
       "  'vcf_string': '15\\t48425820\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425820C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7249G>T',\n",
       "  'hgvs_protein': 'p.Glu2417Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425820,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425823-C-T',\n",
       "  'variant_hash': '3dca26e0357e45f5',\n",
       "  'vcf_string': '15\\t48425823\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425823C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7246G>A',\n",
       "  'hgvs_protein': 'p.Gly2416Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425823,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425829-G-A',\n",
       "  'variant_hash': '3432f2cbf515d118',\n",
       "  'vcf_string': '15\\t48425829\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425829G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7240C>T',\n",
       "  'hgvs_protein': 'p.Arg2414Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425829,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425830-G-A',\n",
       "  'variant_hash': '133f4450cb009016',\n",
       "  'vcf_string': '15\\t48425830\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425830G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7239C>T',\n",
       "  'hgvs_protein': 'p.Cys2413=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425830,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425833-A-G',\n",
       "  'variant_hash': '23ae98dcfd6abb44',\n",
       "  'vcf_string': '15\\t48425833\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425833A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7236T>C',\n",
       "  'hgvs_protein': 'p.Val2412=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425833,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425839-G-A',\n",
       "  'variant_hash': '7d64646fdc04c2ab',\n",
       "  'vcf_string': '15\\t48425839\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425839G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7230C>T',\n",
       "  'hgvs_protein': 'p.His2410=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425839,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425842-A-T',\n",
       "  'variant_hash': 'ef1f18f4880d3201',\n",
       "  'vcf_string': '15\\t48425842\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425842A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7227T>A',\n",
       "  'hgvs_protein': 'p.Ile2409=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425842,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425852-C-T',\n",
       "  'variant_hash': '8ec6c058bcc8a155',\n",
       "  'vcf_string': '15\\t48425852\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425852C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7217G>A',\n",
       "  'hgvs_protein': 'p.Cys2406Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425852,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425860-G-A',\n",
       "  'variant_hash': 'e58bc65f2ba1328e',\n",
       "  'vcf_string': '15\\t48425860\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425860G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7209C>T',\n",
       "  'hgvs_protein': 'p.Ile2403=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425860,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425865-C-T',\n",
       "  'variant_hash': '65da85ff5c343023',\n",
       "  'vcf_string': '15\\t48425865\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425865C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7205-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425865,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48425866-T-C',\n",
       "  'variant_hash': '423ec7c130513151',\n",
       "  'vcf_string': '15\\t48425866\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425866T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7205-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425866,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48425876-C-G',\n",
       "  'variant_hash': '57d214ca6804a513',\n",
       "  'vcf_string': '15\\t48425876\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425876C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7205-12G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425876,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48425882-T-C',\n",
       "  'variant_hash': 'c925a6f67238c03b',\n",
       "  'vcf_string': '15\\t48425882\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48425882T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7205-18A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48425882,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427504-G-T',\n",
       "  'variant_hash': '2b293fd12774a614',\n",
       "  'vcf_string': '15\\t48427504\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427504G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7204+63C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427504,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48427560-G-C',\n",
       "  'variant_hash': '0dc4c0aca67494f3',\n",
       "  'vcf_string': '15\\t48427560\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427560G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7204+7C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427560,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48427561-A-C',\n",
       "  'variant_hash': '36c1089f0aa1f05f',\n",
       "  'vcf_string': '15\\t48427561\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427561A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7204+6T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427561,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427566-C-T',\n",
       "  'variant_hash': 'f0225e216a6e7365',\n",
       "  'vcf_string': '15\\t48427566\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427566C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7204+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427566,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48427566-C-A',\n",
       "  'variant_hash': '1395c97ef7a12aa9',\n",
       "  'vcf_string': '15\\t48427566\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427566C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7204+1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427566,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48427591-G-A',\n",
       "  'variant_hash': 'b4c8984f354221b4',\n",
       "  'vcf_string': '15\\t48427591\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427591G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7180C>T',\n",
       "  'hgvs_protein': 'p.Arg2394Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427591,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48427602-C-T',\n",
       "  'variant_hash': 'c0f4c7e6c5c5068f',\n",
       "  'vcf_string': '15\\t48427602\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427602C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7169G>A',\n",
       "  'hgvs_protein': 'p.Cys2390Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427602,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48427603-A-T',\n",
       "  'variant_hash': '5de3b0b810601ea2',\n",
       "  'vcf_string': '15\\t48427603\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427603A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7168T>A',\n",
       "  'hgvs_protein': 'p.Cys2390Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427603,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48427603-A-G',\n",
       "  'variant_hash': 'd04274c9793f502b',\n",
       "  'vcf_string': '15\\t48427603\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427603A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7168T>C',\n",
       "  'hgvs_protein': 'p.Cys2390Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427603,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48427613-G-A',\n",
       "  'variant_hash': '28d1038095439d92',\n",
       "  'vcf_string': '15\\t48427613\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427613G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7158C>T',\n",
       "  'hgvs_protein': 'p.Phe2386=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427613,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427630-G-A',\n",
       "  'variant_hash': '2031e69a7ff803bc',\n",
       "  'vcf_string': '15\\t48427630\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427630G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7141C>T',\n",
       "  'hgvs_protein': 'p.Gln2381Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427630,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48427646-A-T',\n",
       "  'variant_hash': 'b57d5b255c0cb1da',\n",
       "  'vcf_string': '15\\t48427646\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427646A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7125T>A',\n",
       "  'hgvs_protein': 'p.Cys2375Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427646,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48427659-C-T',\n",
       "  'variant_hash': '7b087039763388e1',\n",
       "  'vcf_string': '15\\t48427659\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427659C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7112G>A',\n",
       "  'hgvs_protein': 'p.Trp2371Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427659,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48427673-G-A',\n",
       "  'variant_hash': '7187ad9382fc51f4',\n",
       "  'vcf_string': '15\\t48427673\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427673G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7098C>T',\n",
       "  'hgvs_protein': 'p.Asp2366=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427673,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48427684-A-T',\n",
       "  'variant_hash': 'ef52275a3dfb5ade',\n",
       "  'vcf_string': '15\\t48427684\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427684A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7087T>A',\n",
       "  'hgvs_protein': 'p.Cys2363Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427684,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48427684-A-C',\n",
       "  'variant_hash': '4c423364b25068f5',\n",
       "  'vcf_string': '15\\t48427684\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427684A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7087T>G',\n",
       "  'hgvs_protein': 'p.Cys2363Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427684,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48427688-C-T',\n",
       "  'variant_hash': '21eade2e835c77ee',\n",
       "  'vcf_string': '15\\t48427688\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427688C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7083G>A',\n",
       "  'hgvs_protein': 'p.Ser2361=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427688,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427700-G-A',\n",
       "  'variant_hash': '2eefd8f686d4bfbd',\n",
       "  'vcf_string': '15\\t48427700\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427700G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7071C>T',\n",
       "  'hgvs_protein': 'p.Pro2357=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427700,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427721-G-A',\n",
       "  'variant_hash': 'e0b66cbd6a6e85f7',\n",
       "  'vcf_string': '15\\t48427721\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427721G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7050C>T',\n",
       "  'hgvs_protein': 'p.Ile2350=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427721,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427745-C-T',\n",
       "  'variant_hash': 'ffca1c4d79f74dc1',\n",
       "  'vcf_string': '15\\t48427745\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427745C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7026G>A',\n",
       "  'hgvs_protein': 'p.Glu2342=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427745,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48427755-C-T',\n",
       "  'variant_hash': '031013375d390d3c',\n",
       "  'vcf_string': '15\\t48427755\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427755C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7016G>A',\n",
       "  'hgvs_protein': 'p.Cys2339Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427755,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48427756-A-G',\n",
       "  'variant_hash': '8e7a87ae639e2289',\n",
       "  'vcf_string': '15\\t48427756\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427756A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7015T>C',\n",
       "  'hgvs_protein': 'p.Cys2339Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427756,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48427768-G-A',\n",
       "  'variant_hash': '15d8f696d0d65d68',\n",
       "  'vcf_string': '15\\t48427768\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427768G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.7003C>T',\n",
       "  'hgvs_protein': 'p.Arg2335Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427768,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48427780-G-A',\n",
       "  'variant_hash': 'eeb98af74792a26d',\n",
       "  'vcf_string': '15\\t48427780\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427780G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6998-7C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427780,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48427787-T-G',\n",
       "  'variant_hash': '4d88c196cc349f07',\n",
       "  'vcf_string': '15\\t48427787\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48427787T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6998-14A>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48427787,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48428074-C-T',\n",
       "  'variant_hash': 'c18ed16936777432',\n",
       "  'vcf_string': '15\\t48428074\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428074C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6997+272G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428074,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48428327-C-T',\n",
       "  'variant_hash': '5b3f94f2c1896aca',\n",
       "  'vcf_string': '15\\t48428327\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428327C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6997+19G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428327,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48428328-G-A',\n",
       "  'variant_hash': 'b7ddddc8b6bb65e3',\n",
       "  'vcf_string': '15\\t48428328\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428328G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6997+18C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428328,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48428329-G-C',\n",
       "  'variant_hash': '911323c95ffa93d9',\n",
       "  'vcf_string': '15\\t48428329\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428329G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6997+17C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428329,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48428345-C-T',\n",
       "  'variant_hash': 'e96fb64a936b900a',\n",
       "  'vcf_string': '15\\t48428345\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428345C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6997+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428345,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48428355-C-A',\n",
       "  'variant_hash': '101dc01eea0d8266',\n",
       "  'vcf_string': '15\\t48428355\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428355C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6988G>T',\n",
       "  'hgvs_protein': 'p.Glu2330Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428355,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48428356-G-C',\n",
       "  'variant_hash': '00cd1a208e972485',\n",
       "  'vcf_string': '15\\t48428356\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428356G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6987C>G',\n",
       "  'hgvs_protein': 'p.Asp2329Glu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428356,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48428356-G-A',\n",
       "  'variant_hash': 'c62f46eca31331b2',\n",
       "  'vcf_string': '15\\t48428356\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428356G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6987C>T',\n",
       "  'hgvs_protein': 'p.Asp2329=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428356,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48428361-G-A',\n",
       "  'variant_hash': '8f5b4292d6128535',\n",
       "  'vcf_string': '15\\t48428361\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428361G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6982C>T',\n",
       "  'hgvs_protein': 'p.Gln2328Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428361,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48428374-G-A',\n",
       "  'variant_hash': '91e82200fe30c8b4',\n",
       "  'vcf_string': '15\\t48428374\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428374G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6969C>T',\n",
       "  'hgvs_protein': 'p.Thr2323=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428374,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48428390-C-T',\n",
       "  'variant_hash': '2a26342dd6617379',\n",
       "  'vcf_string': '15\\t48428390\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428390C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6953G>A',\n",
       "  'hgvs_protein': 'p.Cys2318Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428390,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48428390-C-G',\n",
       "  'variant_hash': '67502910aa01c2a3',\n",
       "  'vcf_string': '15\\t48428390\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428390C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6953G>C',\n",
       "  'hgvs_protein': 'p.Cys2318Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428390,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48428391-A-G',\n",
       "  'variant_hash': '8cb1a7f15c881fbe',\n",
       "  'vcf_string': '15\\t48428391\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428391A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6952T>C',\n",
       "  'hgvs_protein': 'p.Cys2318Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428391,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48428401-G-A',\n",
       "  'variant_hash': '01016d3af9ffcfd8',\n",
       "  'vcf_string': '15\\t48428401\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428401G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6942C>T',\n",
       "  'hgvs_protein': 'p.Tyr2314=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428401,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48428427-G-A',\n",
       "  'variant_hash': 'fca97119f2bf661f',\n",
       "  'vcf_string': '15\\t48428427\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428427G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6916C>T',\n",
       "  'hgvs_protein': 'p.Arg2306Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428427,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48428439-A-T',\n",
       "  'variant_hash': 'd392cc369aa9e8e1',\n",
       "  'vcf_string': '15\\t48428439\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428439A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6904T>A',\n",
       "  'hgvs_protein': 'p.Cys2302Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428439,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48428440-G-A',\n",
       "  'variant_hash': 'f96f1738a1ee2805',\n",
       "  'vcf_string': '15\\t48428440\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428440G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6903C>T',\n",
       "  'hgvs_protein': 'p.Ile2301=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428440,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48428452-C-T',\n",
       "  'variant_hash': '557fb4386df096f3',\n",
       "  'vcf_string': '15\\t48428452\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428452C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6891G>A',\n",
       "  'hgvs_protein': 'p.Thr2297=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428452,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48428455-C-T',\n",
       "  'variant_hash': '315e8658d485da02',\n",
       "  'vcf_string': '15\\t48428455\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428455C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6888G>A',\n",
       "  'hgvs_protein': 'p.Gln2296=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428455,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48428459-C-T',\n",
       "  'variant_hash': '0ccc454534bb0d52',\n",
       "  'vcf_string': '15\\t48428459\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428459C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6884G>A',\n",
       "  'hgvs_protein': 'p.Cys2295Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428459,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48428483-T-C',\n",
       "  'variant_hash': '16c46b36b46d95c8',\n",
       "  'vcf_string': '15\\t48428483\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48428483T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6872-12A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48428483,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48429432-T-C',\n",
       "  'variant_hash': '25a166ee5e7dea58',\n",
       "  'vcf_string': '15\\t48429432\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48429432T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6872-961A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48429432,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430659-G-T',\n",
       "  'variant_hash': '2dd0480270d52a53',\n",
       "  'vcf_string': '15\\t48430659\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430659G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6871+12C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430659,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48430676-C-T',\n",
       "  'variant_hash': 'fa54d615960591c8',\n",
       "  'vcf_string': '15\\t48430676\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430676C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6866G>A',\n",
       "  'hgvs_protein': 'p.Cys2289Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430676,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48430676-C-G',\n",
       "  'variant_hash': 'b309d88e77654407',\n",
       "  'vcf_string': '15\\t48430676\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430676C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6866G>C',\n",
       "  'hgvs_protein': 'p.Cys2289Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430676,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48430687-A-G',\n",
       "  'variant_hash': '33df7322a8e03367',\n",
       "  'vcf_string': '15\\t48430687\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430687A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6855T>C',\n",
       "  'hgvs_protein': 'p.Asp2285=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430687,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48430705-C-T',\n",
       "  'variant_hash': '8e6382cf70a3d5c4',\n",
       "  'vcf_string': '15\\t48430705\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430705C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6837G>A',\n",
       "  'hgvs_protein': 'p.Gly2279=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430705,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48430708-G-A',\n",
       "  'variant_hash': '09924c9a37aa2b1c',\n",
       "  'vcf_string': '15\\t48430708\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430708G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6834C>T',\n",
       "  'hgvs_protein': 'p.Pro2278=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430708,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48430710-G-A',\n",
       "  'variant_hash': 'bd820bc456f5f660',\n",
       "  'vcf_string': '15\\t48430710\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430710G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6832C>T',\n",
       "  'hgvs_protein': 'p.Pro2278Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430710,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48430722-A-G',\n",
       "  'variant_hash': '4aab8670414a7007',\n",
       "  'vcf_string': '15\\t48430722\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430722A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6820T>C',\n",
       "  'hgvs_protein': 'p.Cys2274Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430722,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48430727-T-C',\n",
       "  'variant_hash': 'b4e478f5165bfbf6',\n",
       "  'vcf_string': '15\\t48430727\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430727T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6815A>G',\n",
       "  'hgvs_protein': 'p.Tyr2272Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430727,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48430735-A-T',\n",
       "  'variant_hash': '7145bacc06e238ba',\n",
       "  'vcf_string': '15\\t48430735\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430735A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6807T>A',\n",
       "  'hgvs_protein': 'p.Ile2269=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430735,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48430736-A-T',\n",
       "  'variant_hash': 'dddaac792b81279e',\n",
       "  'vcf_string': '15\\t48430736\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430736A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6806T>A',\n",
       "  'hgvs_protein': 'p.Ile2269Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430736,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430736-A-G',\n",
       "  'variant_hash': '4b6576d56b700beb',\n",
       "  'vcf_string': '15\\t48430736\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430736A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6806T>C',\n",
       "  'hgvs_protein': 'p.Ile2269Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430736,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430748-C-T',\n",
       "  'variant_hash': 'fa63cf259a5a487e',\n",
       "  'vcf_string': '15\\t48430748\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430748C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6794G>A',\n",
       "  'hgvs_protein': 'p.Cys2265Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430748,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48430749-A-G',\n",
       "  'variant_hash': '3b0386ba19db0a42',\n",
       "  'vcf_string': '15\\t48430749\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430749A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6793T>C',\n",
       "  'hgvs_protein': 'p.Cys2265Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430749,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48430749-A-C',\n",
       "  'variant_hash': '54be68e6a440ec25',\n",
       "  'vcf_string': '15\\t48430749\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430749A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6793T>G',\n",
       "  'hgvs_protein': 'p.Cys2265Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430749,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430758-G-A',\n",
       "  'variant_hash': 'e7665b54d0224ebd',\n",
       "  'vcf_string': '15\\t48430758\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430758G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6784C>T',\n",
       "  'hgvs_protein': 'p.Gln2262Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430758,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48430770-A-G',\n",
       "  'variant_hash': '62d2d1e66383d245',\n",
       "  'vcf_string': '15\\t48430770\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430770A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6772T>C',\n",
       "  'hgvs_protein': 'p.Cys2258Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430770,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430770-A-C',\n",
       "  'variant_hash': '1de3457a7a7eecfc',\n",
       "  'vcf_string': '15\\t48430770\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430770A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6772T>G',\n",
       "  'hgvs_protein': 'p.Cys2258Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430770,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430774-A-G',\n",
       "  'variant_hash': 'e3f90af8b0cedc87',\n",
       "  'vcf_string': '15\\t48430774\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430774A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6768T>C',\n",
       "  'hgvs_protein': 'p.His2256=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430774,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48430780-T-C',\n",
       "  'variant_hash': 'f34cc3d4b1e70990',\n",
       "  'vcf_string': '15\\t48430780\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430780T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6762A>G',\n",
       "  'hgvs_protein': 'p.Gly2254=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430780,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48430785-C-A',\n",
       "  'variant_hash': '06e3c195584c14a1',\n",
       "  'vcf_string': '15\\t48430785\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430785C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6757G>T',\n",
       "  'hgvs_protein': 'p.Glu2253Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430785,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48430790-C-T',\n",
       "  'variant_hash': '6376d0353955dc3d',\n",
       "  'vcf_string': '15\\t48430790\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430790C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6752G>A',\n",
       "  'hgvs_protein': 'p.Cys2251Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430790,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430791-A-T',\n",
       "  'variant_hash': '7ad83929367ce6eb',\n",
       "  'vcf_string': '15\\t48430791\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430791A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6751T>A',\n",
       "  'hgvs_protein': 'p.Cys2251Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430791,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430791-A-G',\n",
       "  'variant_hash': 'bd751b8744b42c69',\n",
       "  'vcf_string': '15\\t48430791\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430791A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6751T>C',\n",
       "  'hgvs_protein': 'p.Cys2251Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430791,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48430803-C-T',\n",
       "  'variant_hash': '0c394766937f8f49',\n",
       "  'vcf_string': '15\\t48430803\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48430803C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6740-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48430803,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48432850-G-A',\n",
       "  'variant_hash': '102a800fedbacdb4',\n",
       "  'vcf_string': '15\\t48432850\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432850G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6739+16C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432850,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432855-C-T',\n",
       "  'variant_hash': '941d7e348e713cb2',\n",
       "  'vcf_string': '15\\t48432855\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432855C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6739+11G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432855,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432865-C-T',\n",
       "  'variant_hash': 'ccacf817fcae4744',\n",
       "  'vcf_string': '15\\t48432865\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432865C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6739+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432865,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432894-C-A',\n",
       "  'variant_hash': '940f612b2705d8e9',\n",
       "  'vcf_string': '15\\t48432894\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432894C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6711G>T',\n",
       "  'hgvs_protein': 'p.Val2237=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432894,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432897-A-G',\n",
       "  'variant_hash': 'a72f48fe9fd72567',\n",
       "  'vcf_string': '15\\t48432897\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432897A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6708T>C',\n",
       "  'hgvs_protein': 'p.Tyr2236=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432897,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432900-T-G',\n",
       "  'variant_hash': 'e24ccb05acb51d36',\n",
       "  'vcf_string': '15\\t48432900\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432900T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6705A>C',\n",
       "  'hgvs_protein': 'p.Gly2235=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432900,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48432906-G-A',\n",
       "  'variant_hash': 'd0269c747074ee7a',\n",
       "  'vcf_string': '15\\t48432906\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432906G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6699C>T',\n",
       "  'hgvs_protein': 'p.Pro2233=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432906,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432907-G-A',\n",
       "  'variant_hash': '84bdcb2722f9ded6',\n",
       "  'vcf_string': '15\\t48432907\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432907G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6698C>T',\n",
       "  'hgvs_protein': 'p.Pro2233Leu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432907,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48432910-C-T',\n",
       "  'variant_hash': '9b87adb57c9ed55d',\n",
       "  'vcf_string': '15\\t48432910\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432910C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6695G>A',\n",
       "  'hgvs_protein': 'p.Cys2232Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432910,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432910-C-G',\n",
       "  'variant_hash': 'c41fdf72fb23ab2f',\n",
       "  'vcf_string': '15\\t48432910\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432910C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6695G>C',\n",
       "  'hgvs_protein': 'p.Cys2232Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432910,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432911-A-G',\n",
       "  'variant_hash': '4777df22a3645923',\n",
       "  'vcf_string': '15\\t48432911\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432911A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6694T>C',\n",
       "  'hgvs_protein': 'p.Cys2232Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432911,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432916-C-T',\n",
       "  'variant_hash': '3bf1f3015c87e807',\n",
       "  'vcf_string': '15\\t48432916\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432916C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6689G>A',\n",
       "  'hgvs_protein': 'p.Cys2230Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432916,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432924-T-A',\n",
       "  'variant_hash': '6f8557b9066742a3',\n",
       "  'vcf_string': '15\\t48432924\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432924T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6681A>T',\n",
       "  'hgvs_protein': 'p.Ser2227=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432924,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432930-A-G',\n",
       "  'variant_hash': '6a0bba330a67f0c8',\n",
       "  'vcf_string': '15\\t48432930\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432930A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6675T>C',\n",
       "  'hgvs_protein': 'p.Tyr2225=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432930,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432943-C-T',\n",
       "  'variant_hash': '05f05da0bb82f44f',\n",
       "  'vcf_string': '15\\t48432943\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432943C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6662G>A',\n",
       "  'hgvs_protein': 'p.Cys2221Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432943,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48432944-A-G',\n",
       "  'variant_hash': 'c5cdeb6a30607d89',\n",
       "  'vcf_string': '15\\t48432944\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432944A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6661T>C',\n",
       "  'hgvs_protein': 'p.Cys2221Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432944,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48432947-G-A',\n",
       "  'variant_hash': 'b64a19db35126b1d',\n",
       "  'vcf_string': '15\\t48432947\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432947G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6658C>T',\n",
       "  'hgvs_protein': 'p.Arg2220Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432947,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432955-C-T',\n",
       "  'variant_hash': '7b4f9231bc34782b',\n",
       "  'vcf_string': '15\\t48432955\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432955C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6650G>A',\n",
       "  'hgvs_protein': 'p.Cys2217Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432955,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432957-G-T',\n",
       "  'variant_hash': '5356bc74f392d116',\n",
       "  'vcf_string': '15\\t48432957\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432957G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6648C>A',\n",
       "  'hgvs_protein': 'p.Leu2216=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432957,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432962-G-A',\n",
       "  'variant_hash': 'b44431414fe82e38',\n",
       "  'vcf_string': '15\\t48432962\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432962G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6643C>T',\n",
       "  'hgvs_protein': 'p.Leu2215=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432962,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48432971-G-A',\n",
       "  'variant_hash': '480e6e678a8aa9a3',\n",
       "  'vcf_string': '15\\t48432971\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432971G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6634C>T',\n",
       "  'hgvs_protein': 'p.Gln2212Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432971,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48432976-C-T',\n",
       "  'variant_hash': '25e744401480f69f',\n",
       "  'vcf_string': '15\\t48432976\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432976C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6629G>A',\n",
       "  'hgvs_protein': 'p.Cys2210Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432976,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432977-A-G',\n",
       "  'variant_hash': '920bed443ce1c1eb',\n",
       "  'vcf_string': '15\\t48432977\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432977A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6628T>C',\n",
       "  'hgvs_protein': 'p.Cys2210Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432977,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48432999-G-T',\n",
       "  'variant_hash': '31fa58d331a873ce',\n",
       "  'vcf_string': '15\\t48432999\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48432999G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6617-11C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48432999,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48433000-G-A',\n",
       "  'variant_hash': '6c98475217db8c46',\n",
       "  'vcf_string': '15\\t48433000\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48433000G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6617-12C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48433000,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48433009-T-A',\n",
       "  'variant_hash': 'f9c55af7417eca6d',\n",
       "  'vcf_string': '15\\t48433009\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48433009T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6617-21A>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48433009,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48434577-C-T',\n",
       "  'variant_hash': 'b757d4f7f7a516cc',\n",
       "  'vcf_string': '15\\t48434577\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434577C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6616+17G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434577,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48434587-G-C',\n",
       "  'variant_hash': 'b031fa6f0a1949be',\n",
       "  'vcf_string': '15\\t48434587\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434587G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6616+7C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434587,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48434593-C-T',\n",
       "  'variant_hash': '71698bc854d05eb3',\n",
       "  'vcf_string': '15\\t48434593\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434593C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6616+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434593,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434599-C-T',\n",
       "  'variant_hash': '35957d2f64570227',\n",
       "  'vcf_string': '15\\t48434599\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434599C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6611G>A',\n",
       "  'hgvs_protein': 'p.Cys2204Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434599,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434599-C-G',\n",
       "  'variant_hash': 'b3e7553d072ac27a',\n",
       "  'vcf_string': '15\\t48434599\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434599C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6611G>C',\n",
       "  'hgvs_protein': 'p.Cys2204Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434599,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48434600-A-G',\n",
       "  'variant_hash': 'e37c07a1ef238252',\n",
       "  'vcf_string': '15\\t48434600\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434600A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6610T>C',\n",
       "  'hgvs_protein': 'p.Cys2204Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434600,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434601-T-C',\n",
       "  'variant_hash': '7f7789f82ac2187a',\n",
       "  'vcf_string': '15\\t48434601\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434601T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6609A>G',\n",
       "  'hgvs_protein': 'p.Thr2203=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434601,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48434616-G-A',\n",
       "  'variant_hash': '5e4d547db5a173e1',\n",
       "  'vcf_string': '15\\t48434616\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434616G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6594C>T',\n",
       "  'hgvs_protein': 'p.Pro2198=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434616,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48434627-C-T',\n",
       "  'variant_hash': 'c18994ab2b15dcb8',\n",
       "  'vcf_string': '15\\t48434627\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434627C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6583G>A',\n",
       "  'hgvs_protein': 'p.Gly2195Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434627,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434634-G-A',\n",
       "  'variant_hash': 'f6316420abff4688',\n",
       "  'vcf_string': '15\\t48434634\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434634G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6576C>T',\n",
       "  'hgvs_protein': 'p.Cys2192=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434634,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48434669-A-G',\n",
       "  'variant_hash': 'f76355dd345c1135',\n",
       "  'vcf_string': '15\\t48434669\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434669A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6541T>C',\n",
       "  'hgvs_protein': 'p.Cys2181Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434669,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48434701-C-T',\n",
       "  'variant_hash': '2074733e70da0186',\n",
       "  'vcf_string': '15\\t48434701\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434701C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6509G>A',\n",
       "  'hgvs_protein': 'p.Cys2170Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434701,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434702-A-G',\n",
       "  'variant_hash': 'c5647b4b7cea9ce1',\n",
       "  'vcf_string': '15\\t48434702\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434702A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6508T>C',\n",
       "  'hgvs_protein': 'p.Cys2170Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434702,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434705-C-A',\n",
       "  'variant_hash': 'edfca620be0da2e7',\n",
       "  'vcf_string': '15\\t48434705\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434705C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6505G>T',\n",
       "  'hgvs_protein': 'p.Glu2169Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434705,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48434707-T-C',\n",
       "  'variant_hash': 'fbbe81dc14a9e52e',\n",
       "  'vcf_string': '15\\t48434707\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434707T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6503A>G',\n",
       "  'hgvs_protein': 'p.Asp2168Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434707,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48434724-C-T',\n",
       "  'variant_hash': 'ddbac26d9bd948f5',\n",
       "  'vcf_string': '15\\t48434724\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434724C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6497-11G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434724,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48434725-C-G',\n",
       "  'variant_hash': '951914f60086cacd',\n",
       "  'vcf_string': '15\\t48434725\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434725C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6497-12G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434725,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48434728-G-A',\n",
       "  'variant_hash': '62b82f2fe1c77d65',\n",
       "  'vcf_string': '15\\t48434728\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434728G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6497-15C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434728,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48434736-C-G',\n",
       "  'variant_hash': '8e1090f5f8bd3bb4',\n",
       "  'vcf_string': '15\\t48434736\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434736C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6497-23G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434736,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48434762-C-T',\n",
       "  'variant_hash': 'd6857953b5786fd7',\n",
       "  'vcf_string': '15\\t48434762\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48434762C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6497-49G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48434762,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48436966-C-T',\n",
       "  'variant_hash': '81bf910dd30ab978',\n",
       "  'vcf_string': '15\\t48436966\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48436966C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6491G>A',\n",
       "  'hgvs_protein': 'p.Cys2164Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48436966,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48436967-A-C',\n",
       "  'variant_hash': 'c3e4cbc81f252463',\n",
       "  'vcf_string': '15\\t48436967\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48436967A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6490T>G',\n",
       "  'hgvs_protein': 'p.Cys2164Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48436967,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48436970-C-A',\n",
       "  'variant_hash': '26214a531331badd',\n",
       "  'vcf_string': '15\\t48436970\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48436970C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6487G>T',\n",
       "  'hgvs_protein': 'p.Glu2163Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48436970,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48436995-G-C',\n",
       "  'variant_hash': '9b5fa8415f484956',\n",
       "  'vcf_string': '15\\t48436995\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48436995G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6462C>G',\n",
       "  'hgvs_protein': 'p.Pro2154=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48436995,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437004-G-C',\n",
       "  'variant_hash': '2b29bac8f007e83c',\n",
       "  'vcf_string': '15\\t48437004\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437004G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6453C>G',\n",
       "  'hgvs_protein': 'p.Cys2151Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437004,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437004-G-A',\n",
       "  'variant_hash': '9f1c18fe0bc997b7',\n",
       "  'vcf_string': '15\\t48437004\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437004G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6453C>T',\n",
       "  'hgvs_protein': 'p.Cys2151=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437004,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437009-G-A',\n",
       "  'variant_hash': 'ff09cdc4703dc42d',\n",
       "  'vcf_string': '15\\t48437009\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437009G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6448C>T',\n",
       "  'hgvs_protein': 'p.Arg2150Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437009,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437011-T-C',\n",
       "  'variant_hash': 'c8d5e9fa1bb99b05',\n",
       "  'vcf_string': '15\\t48437011\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437011T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6446A>G',\n",
       "  'hgvs_protein': 'p.Tyr2149Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437011,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437013-G-A',\n",
       "  'variant_hash': '391857e44d488580',\n",
       "  'vcf_string': '15\\t48437013\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437013G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6444C>T',\n",
       "  'hgvs_protein': 'p.Ser2148=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437013,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437019-A-G',\n",
       "  'variant_hash': 'fd129540b2ea93cf',\n",
       "  'vcf_string': '15\\t48437019\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437019A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6438T>C',\n",
       "  'hgvs_protein': 'p.Asp2146=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437019,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437026-T-C',\n",
       "  'variant_hash': '3e164082d05192b3',\n",
       "  'vcf_string': '15\\t48437026\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437026T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6431A>G',\n",
       "  'hgvs_protein': 'p.Asn2144Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437026,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437027-T-G',\n",
       "  'variant_hash': 'b23555db55a0158d',\n",
       "  'vcf_string': '15\\t48437027\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437027T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6430A>C',\n",
       "  'hgvs_protein': 'p.Asn2144His',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437027,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437032-C-T',\n",
       "  'variant_hash': 'b202d5c300fb4331',\n",
       "  'vcf_string': '15\\t48437032\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437032C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6425G>A',\n",
       "  'hgvs_protein': 'p.Cys2142Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437032,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437038-C-T',\n",
       "  'variant_hash': '0ef75f1230f9f333',\n",
       "  'vcf_string': '15\\t48437038\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437038C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6419G>A',\n",
       "  'hgvs_protein': 'p.Gly2140Glu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437038,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437039-C-T',\n",
       "  'variant_hash': 'afb1f3d6e95fbeaf',\n",
       "  'vcf_string': '15\\t48437039\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437039C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6418G>A',\n",
       "  'hgvs_protein': 'p.Gly2140Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437039,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437040-A-G',\n",
       "  'variant_hash': '97ee733ca3eb31a0',\n",
       "  'vcf_string': '15\\t48437040\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437040A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6417T>C',\n",
       "  'hgvs_protein': 'p.His2139=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437040,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437047-C-T',\n",
       "  'variant_hash': '06d325a9283f49b9',\n",
       "  'vcf_string': '15\\t48437047\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437047C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6410G>A',\n",
       "  'hgvs_protein': 'p.Cys2137Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437047,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437055-G-A',\n",
       "  'variant_hash': '77aded8df890488b',\n",
       "  'vcf_string': '15\\t48437055\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437055G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6402C>T',\n",
       "  'hgvs_protein': 'p.Pro2134=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437055,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437064-G-C',\n",
       "  'variant_hash': '600e0354173ddfcc',\n",
       "  'vcf_string': '15\\t48437064\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437064G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6393C>G',\n",
       "  'hgvs_protein': 'p.Cys2131Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437064,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437064-G-A',\n",
       "  'variant_hash': '9e6e12e8f80b65e7',\n",
       "  'vcf_string': '15\\t48437064\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437064G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6393C>T',\n",
       "  'hgvs_protein': 'p.Cys2131=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437064,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48437069-C-T',\n",
       "  'variant_hash': '9d515ba17ffda9c2',\n",
       "  'vcf_string': '15\\t48437069\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437069C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6388G>A',\n",
       "  'hgvs_protein': 'p.Glu2130Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437069,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437070-G-A',\n",
       "  'variant_hash': '43f911e52c1d4433',\n",
       "  'vcf_string': '15\\t48437070\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437070G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6387C>T',\n",
       "  'hgvs_protein': 'p.Asp2129=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437070,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437079-T-G',\n",
       "  'variant_hash': 'fd44ae96d50e8ac2',\n",
       "  'vcf_string': '15\\t48437079\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437079T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6380-2A>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437079,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437097-A-G',\n",
       "  'variant_hash': '849781a5b02e52d3',\n",
       "  'vcf_string': '15\\t48437097\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437097A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6380-20T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437097,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437162-A-T',\n",
       "  'variant_hash': '872c4188d62fd3a5',\n",
       "  'vcf_string': '15\\t48437162\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437162A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6380-85T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437162,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48437207-G-A',\n",
       "  'variant_hash': 'cac248341e5c2e69',\n",
       "  'vcf_string': '15\\t48437207\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437207G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+115C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437207,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48437311-A-G',\n",
       "  'variant_hash': '60d6e23c957ff3d7',\n",
       "  'vcf_string': '15\\t48437311\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437311A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+11T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437311,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437313-G-A',\n",
       "  'variant_hash': '48b35b51e3f3ce3a',\n",
       "  'vcf_string': '15\\t48437313\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437313G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+9C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437313,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437318-T-C',\n",
       "  'variant_hash': '38d0b21c6e5c00d6',\n",
       "  'vcf_string': '15\\t48437318\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437318T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+4A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437318,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48437320-A-G',\n",
       "  'variant_hash': '601482ba61352bbc',\n",
       "  'vcf_string': '15\\t48437320\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437320A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+2T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437320,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437320-A-C',\n",
       "  'variant_hash': '037bf7c66ca55a4b',\n",
       "  'vcf_string': '15\\t48437320\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437320A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+2T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437320,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437321-C-T',\n",
       "  'variant_hash': 'be0c98425c2da098',\n",
       "  'vcf_string': '15\\t48437321\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437321C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6379+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437321,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437326-T-C',\n",
       "  'variant_hash': 'c92d134b0fb60545',\n",
       "  'vcf_string': '15\\t48437326\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437326T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6375A>G',\n",
       "  'hgvs_protein': 'p.Ala2125=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437326,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437329-T-C',\n",
       "  'variant_hash': 'a6207db4e8c3e537',\n",
       "  'vcf_string': '15\\t48437329\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437329T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6372A>G',\n",
       "  'hgvs_protein': 'p.Ser2124=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437329,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437347-G-C',\n",
       "  'variant_hash': 'bfcae4c74832ff3a',\n",
       "  'vcf_string': '15\\t48437347\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437347G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6354C>G',\n",
       "  'hgvs_protein': 'p.Ile2118Met',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437347,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437347-G-A',\n",
       "  'variant_hash': '8bfb584dafae485e',\n",
       "  'vcf_string': '15\\t48437347\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437347G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6354C>T',\n",
       "  'hgvs_protein': 'p.Ile2118=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437347,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437359-T-G',\n",
       "  'variant_hash': '7a03cb3c62f92e98',\n",
       "  'vcf_string': '15\\t48437359\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437359T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6342A>C',\n",
       "  'hgvs_protein': 'p.Gly2114=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437359,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437362-A-C',\n",
       "  'variant_hash': '95e9366f62b85656',\n",
       "  'vcf_string': '15\\t48437362\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437362A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6339T>G',\n",
       "  'hgvs_protein': 'p.Tyr2113Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437362,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437365-A-G',\n",
       "  'variant_hash': '4e21d2a9ff2c8b69',\n",
       "  'vcf_string': '15\\t48437365\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437365A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6336T>C',\n",
       "  'hgvs_protein': 'p.Pro2112=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437365,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437369-C-T',\n",
       "  'variant_hash': '929bb230b11eab82',\n",
       "  'vcf_string': '15\\t48437369\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437369C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6332G>A',\n",
       "  'hgvs_protein': 'p.Cys2111Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437369,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437370-A-G',\n",
       "  'variant_hash': 'c82cab5cf9304570',\n",
       "  'vcf_string': '15\\t48437370\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437370A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6331T>C',\n",
       "  'hgvs_protein': 'p.Cys2111Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437370,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437376-G-A',\n",
       "  'variant_hash': 'f7ac39535ee29752',\n",
       "  'vcf_string': '15\\t48437376\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437376G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6325C>T',\n",
       "  'hgvs_protein': 'p.Gln2109Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437376,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437379-G-A',\n",
       "  'variant_hash': 'bbe9e6ae55f253b0',\n",
       "  'vcf_string': '15\\t48437379\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437379G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6322C>T',\n",
       "  'hgvs_protein': 'p.Arg2108Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437379,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437393-G-A',\n",
       "  'variant_hash': '0a9acc9501c60cae',\n",
       "  'vcf_string': '15\\t48437393\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437393G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6314-6C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437393,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437403-T-C',\n",
       "  'variant_hash': '8ffc0331955f5d3b',\n",
       "  'vcf_string': '15\\t48437403\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437403T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6314-16A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437403,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437451-T-C',\n",
       "  'variant_hash': 'f1948443db6d5ea9',\n",
       "  'vcf_string': '15\\t48437451\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437451T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6314-64A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437451,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48437765-T-C',\n",
       "  'variant_hash': '00b4c885ef35057a',\n",
       "  'vcf_string': '15\\t48437765\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437765T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6313+3A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437765,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437768-C-A',\n",
       "  'variant_hash': 'e1698bf7c2e9ef1e',\n",
       "  'vcf_string': '15\\t48437768\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437768C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6313G>T',\n",
       "  'hgvs_protein': 'p.Glu2105Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437768,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437778-C-T',\n",
       "  'variant_hash': '350e78e8e0330934',\n",
       "  'vcf_string': '15\\t48437778\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437778C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6303G>A',\n",
       "  'hgvs_protein': 'p.Thr2101=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437778,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437779-G-A',\n",
       "  'variant_hash': '4448945f75bfa05a',\n",
       "  'vcf_string': '15\\t48437779\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437779G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6302C>T',\n",
       "  'hgvs_protein': 'p.Thr2101Met',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437779,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48437784-G-C',\n",
       "  'variant_hash': '74a5ea48ab6e4651',\n",
       "  'vcf_string': '15\\t48437784\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437784G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6297C>G',\n",
       "  'hgvs_protein': 'p.Cys2099Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437784,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437793-G-A',\n",
       "  'variant_hash': '37a6a769ab95404c',\n",
       "  'vcf_string': '15\\t48437793\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437793G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6288C>T',\n",
       "  'hgvs_protein': 'p.Cys2096=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437793,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48437805-C-T',\n",
       "  'variant_hash': '9115ac899b22eee7',\n",
       "  'vcf_string': '15\\t48437805\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437805C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6276G>A',\n",
       "  'hgvs_protein': 'p.Trp2092Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437805,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437807-A-G',\n",
       "  'variant_hash': '8200626dda786e1c',\n",
       "  'vcf_string': '15\\t48437807\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437807A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6274T>C',\n",
       "  'hgvs_protein': 'p.Trp2092Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437807,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437827-C-T',\n",
       "  'variant_hash': '7bf3f0a1b9ddfbd8',\n",
       "  'vcf_string': '15\\t48437827\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437827C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6254G>A',\n",
       "  'hgvs_protein': 'p.Cys2085Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437827,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437837-C-A',\n",
       "  'variant_hash': '6481759627163733',\n",
       "  'vcf_string': '15\\t48437837\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437837C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6244G>T',\n",
       "  'hgvs_protein': 'p.Glu2082Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437837,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437862-G-A',\n",
       "  'variant_hash': '346bf87822e45e32',\n",
       "  'vcf_string': '15\\t48437862\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437862G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6219C>T',\n",
       "  'hgvs_protein': 'p.Pro2073=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437862,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437872-C-T',\n",
       "  'variant_hash': 'b1731817c5287ce6',\n",
       "  'vcf_string': '15\\t48437872\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437872C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6209G>A',\n",
       "  'hgvs_protein': 'p.Cys2070Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437872,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437873-A-G',\n",
       "  'variant_hash': 'e2dc96e331221545',\n",
       "  'vcf_string': '15\\t48437873\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437873A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6208T>C',\n",
       "  'hgvs_protein': 'p.Cys2070Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437873,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48437874-C-T',\n",
       "  'variant_hash': '6c14b3be8263546a',\n",
       "  'vcf_string': '15\\t48437874\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437874C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6207G>A',\n",
       "  'hgvs_protein': 'p.Lys2069=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437874,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48437900-A-G',\n",
       "  'variant_hash': '18458fc5b185c1c4',\n",
       "  'vcf_string': '15\\t48437900\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437900A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6181T>C',\n",
       "  'hgvs_protein': 'p.Cys2061Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437900,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437901-G-C',\n",
       "  'variant_hash': '7f8d4fae2cc159e7',\n",
       "  'vcf_string': '15\\t48437901\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437901G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6180C>G',\n",
       "  'hgvs_protein': 'p.Tyr2060Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437901,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437912-G-A',\n",
       "  'variant_hash': 'f069a16de1ed6dc7',\n",
       "  'vcf_string': '15\\t48437912\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437912G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6169C>T',\n",
       "  'hgvs_protein': 'p.Arg2057Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437912,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48437919-T-C',\n",
       "  'variant_hash': 'd391c8add4c9f11e',\n",
       "  'vcf_string': '15\\t48437919\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437919T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6164-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437919,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48437929-A-G',\n",
       "  'variant_hash': 'c8459cb26179dcaf',\n",
       "  'vcf_string': '15\\t48437929\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48437929A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6164-12T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48437929,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441591-G-A',\n",
       "  'variant_hash': 'c1e4d9721da48833',\n",
       "  'vcf_string': '15\\t48441591\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441591G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6163+130C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441591,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441712-G-C',\n",
       "  'variant_hash': 'cb2cc480e3680e0c',\n",
       "  'vcf_string': '15\\t48441712\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441712G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6163+9C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441712,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441720-C-T',\n",
       "  'variant_hash': '4453c0df99bae68d',\n",
       "  'vcf_string': '15\\t48441720\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441720C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6163+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441720,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48441728-C-T',\n",
       "  'variant_hash': '2bdce4db48755f3e',\n",
       "  'vcf_string': '15\\t48441728\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441728C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6156G>A',\n",
       "  'hgvs_protein': 'p.Arg2052=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441728,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441737-A-G',\n",
       "  'variant_hash': 'f9f27141ef905966',\n",
       "  'vcf_string': '15\\t48441737\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441737A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6147T>C',\n",
       "  'hgvs_protein': 'p.Ser2049=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441737,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441740-G-A',\n",
       "  'variant_hash': 'e05063d3dfcfaf37',\n",
       "  'vcf_string': '15\\t48441740\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441740G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6144C>T',\n",
       "  'hgvs_protein': 'p.Ser2048=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441740,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441743-G-A',\n",
       "  'variant_hash': '44c5a78acaaecd27',\n",
       "  'vcf_string': '15\\t48441743\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441743G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6141C>T',\n",
       "  'hgvs_protein': 'p.Ser2047=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441743,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441755-C-G',\n",
       "  'variant_hash': '45f15d8ce1e16aeb',\n",
       "  'vcf_string': '15\\t48441755\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441755C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6129G>C',\n",
       "  'hgvs_protein': 'p.Gly2043=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441755,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441755-C-A',\n",
       "  'variant_hash': 'a14eaf214c2f72bd',\n",
       "  'vcf_string': '15\\t48441755\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441755C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6129G>T',\n",
       "  'hgvs_protein': 'p.Gly2043=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441755,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441764-A-G',\n",
       "  'variant_hash': 'e14acfac952ea98e',\n",
       "  'vcf_string': '15\\t48441764\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441764A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6120T>C',\n",
       "  'hgvs_protein': 'p.Cys2040=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441764,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441764-A-C',\n",
       "  'variant_hash': '130188d7ce8431d0',\n",
       "  'vcf_string': '15\\t48441764\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441764A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6120T>G',\n",
       "  'hgvs_protein': 'p.Cys2040Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441764,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48441765-C-T',\n",
       "  'variant_hash': 'a8c9b1779d22f37c',\n",
       "  'vcf_string': '15\\t48441765\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441765C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6119G>A',\n",
       "  'hgvs_protein': 'p.Cys2040Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441765,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48441771-C-T',\n",
       "  'variant_hash': '7e4fc1d27dc9673e',\n",
       "  'vcf_string': '15\\t48441771\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441771C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6113G>A',\n",
       "  'hgvs_protein': 'p.Cys2038Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441771,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48441773-T-C',\n",
       "  'variant_hash': '31de9045f8804ded',\n",
       "  'vcf_string': '15\\t48441773\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441773T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6111A>G',\n",
       "  'hgvs_protein': 'p.Lys2037=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441773,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441785-T-C',\n",
       "  'variant_hash': '259aef14ddc04c42',\n",
       "  'vcf_string': '15\\t48441785\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441785T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6099A>G',\n",
       "  'hgvs_protein': 'p.Glu2033=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441785,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441797-G-T',\n",
       "  'variant_hash': '0964b805d28952f1',\n",
       "  'vcf_string': '15\\t48441797\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441797G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6087C>A',\n",
       "  'hgvs_protein': 'p.Cys2029Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441797,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48441798-C-T',\n",
       "  'variant_hash': 'fcb3d401f7dcb10d',\n",
       "  'vcf_string': '15\\t48441798\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441798C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6086G>A',\n",
       "  'hgvs_protein': 'p.Cys2029Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441798,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48441800-T-C',\n",
       "  'variant_hash': '4f92136d8bd062d0',\n",
       "  'vcf_string': '15\\t48441800\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441800T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6084A>G',\n",
       "  'hgvs_protein': 'p.Thr2028=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441800,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441811-C-A',\n",
       "  'variant_hash': '24235f4275cd1ed3',\n",
       "  'vcf_string': '15\\t48441811\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441811C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6073G>T',\n",
       "  'hgvs_protein': 'p.Ala2025Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441811,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48441813-C-T',\n",
       "  'variant_hash': '52e560f9f9bcd508',\n",
       "  'vcf_string': '15\\t48441813\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441813C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6071G>A',\n",
       "  'hgvs_protein': 'p.Cys2024Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441813,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48441830-G-C',\n",
       "  'variant_hash': '1beb96ae7ff095b5',\n",
       "  'vcf_string': '15\\t48441830\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441830G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6054C>G',\n",
       "  'hgvs_protein': 'p.Val2018=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441830,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441833-A-C',\n",
       "  'variant_hash': 'b4f0ed4b37f2aa09',\n",
       "  'vcf_string': '15\\t48441833\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441833A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6051T>G',\n",
       "  'hgvs_protein': 'p.Cys2017Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441833,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48441834-C-T',\n",
       "  'variant_hash': '6d9a3da09b3429da',\n",
       "  'vcf_string': '15\\t48441834\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441834C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6050G>A',\n",
       "  'hgvs_protein': 'p.Cys2017Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441834,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48441835-A-G',\n",
       "  'variant_hash': 'd1fa9e20fea109ee',\n",
       "  'vcf_string': '15\\t48441835\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441835A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6049T>C',\n",
       "  'hgvs_protein': 'p.Cys2017Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441835,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48441836-C-T',\n",
       "  'variant_hash': '673447160c87a653',\n",
       "  'vcf_string': '15\\t48441836\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441836C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6048G>A',\n",
       "  'hgvs_protein': 'p.Glu2016=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441836,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441854-A-T',\n",
       "  'variant_hash': '8265461a2dbf247f',\n",
       "  'vcf_string': '15\\t48441854\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441854A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6038-8T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441854,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48441859-C-A',\n",
       "  'variant_hash': '55e29edac54302cd',\n",
       "  'vcf_string': '15\\t48441859\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441859C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6038-13G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441859,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48441864-G-A',\n",
       "  'variant_hash': 'e0b5750fc7ee093a',\n",
       "  'vcf_string': '15\\t48441864\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48441864G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6038-18C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48441864,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48444418-T-C',\n",
       "  'variant_hash': 'e394270ca7dfaeee',\n",
       "  'vcf_string': '15\\t48444418\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444418T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6037+123A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444418,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48444487-A-T',\n",
       "  'variant_hash': '3d8c44e41bac050c',\n",
       "  'vcf_string': '15\\t48444487\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444487A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6037+54T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444487,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48444540-C-T',\n",
       "  'variant_hash': '3a96f687894abcbc',\n",
       "  'vcf_string': '15\\t48444540\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444540C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6037+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444540,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48444546-C-T',\n",
       "  'variant_hash': '0d2bbd010be9d2c4',\n",
       "  'vcf_string': '15\\t48444546\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444546C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6032G>A',\n",
       "  'hgvs_protein': 'p.Cys2011Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444546,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48444547-A-G',\n",
       "  'variant_hash': 'eb341c1fe1cd8df8',\n",
       "  'vcf_string': '15\\t48444547\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444547A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6031T>C',\n",
       "  'hgvs_protein': 'p.Cys2011Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444547,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444566-G-C',\n",
       "  'variant_hash': '7f15c7f4c056924a',\n",
       "  'vcf_string': '15\\t48444566\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444566G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.6012C>G',\n",
       "  'hgvs_protein': 'p.Tyr2004Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444566,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444579-C-T',\n",
       "  'variant_hash': 'b0e312abdd5711d7',\n",
       "  'vcf_string': '15\\t48444579\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444579C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5999G>A',\n",
       "  'hgvs_protein': 'p.Cys2000Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444579,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444579-C-A',\n",
       "  'variant_hash': '509fdbc1d88fff54',\n",
       "  'vcf_string': '15\\t48444579\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444579C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5999G>T',\n",
       "  'hgvs_protein': 'p.Cys2000Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444579,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48444584-G-A',\n",
       "  'variant_hash': 'beef76f425b1d59b',\n",
       "  'vcf_string': '15\\t48444584\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444584G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5994C>T',\n",
       "  'hgvs_protein': 'p.Cys1998=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444584,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48444585-C-T',\n",
       "  'variant_hash': 'e2974dc8800a7ea1',\n",
       "  'vcf_string': '15\\t48444585\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444585C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5993G>A',\n",
       "  'hgvs_protein': 'p.Cys1998Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444585,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444587-T-C',\n",
       "  'variant_hash': 'bbbddd956455a8bb',\n",
       "  'vcf_string': '15\\t48444587\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444587T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5991A>G',\n",
       "  'hgvs_protein': 'p.Arg1997=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444587,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48444611-A-C',\n",
       "  'variant_hash': '46cc85d43ee8486c',\n",
       "  'vcf_string': '15\\t48444611\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444611A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5967T>G',\n",
       "  'hgvs_protein': 'p.Cys1989Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444611,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444612-C-A',\n",
       "  'variant_hash': 'dd13990e47083276',\n",
       "  'vcf_string': '15\\t48444612\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444612C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5966G>T',\n",
       "  'hgvs_protein': 'p.Cys1989Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444612,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48444627-C-T',\n",
       "  'variant_hash': 'd67c3b39913962be',\n",
       "  'vcf_string': '15\\t48444627\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444627C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5951G>A',\n",
       "  'hgvs_protein': 'p.Cys1984Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444627,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444628-A-G',\n",
       "  'variant_hash': '1ba04f7166341485',\n",
       "  'vcf_string': '15\\t48444628\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444628A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5950T>C',\n",
       "  'hgvs_protein': 'p.Cys1984Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444628,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444648-C-T',\n",
       "  'variant_hash': '5ac0463132c00ac0',\n",
       "  'vcf_string': '15\\t48444648\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444648C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5930G>A',\n",
       "  'hgvs_protein': 'p.Cys1977Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444648,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48444653-A-G',\n",
       "  'variant_hash': 'f1cd1a8eabe4c413',\n",
       "  'vcf_string': '15\\t48444653\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444653A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5925T>C',\n",
       "  'hgvs_protein': 'p.Asn1975=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444653,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48444662-T-C',\n",
       "  'variant_hash': '682350d4d9920bdf',\n",
       "  'vcf_string': '15\\t48444662\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48444662T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5918-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48444662,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445168-A-G',\n",
       "  'variant_hash': '1cb1e2053487a0aa',\n",
       "  'vcf_string': '15\\t48445168\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445168A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5917+208T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445168,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445370-A-G',\n",
       "  'variant_hash': '9f90da6bdc752725',\n",
       "  'vcf_string': '15\\t48445370\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445370A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5917+6T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445370,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445375-C-T',\n",
       "  'variant_hash': '7c64d8401a6e85f5',\n",
       "  'vcf_string': '15\\t48445375\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445375C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5917+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445375,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445375-C-A',\n",
       "  'variant_hash': '8a6369eb7ca8d2cc',\n",
       "  'vcf_string': '15\\t48445375\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445375C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5917+1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445375,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445382-A-G',\n",
       "  'variant_hash': '02561fdac0903b77',\n",
       "  'vcf_string': '15\\t48445382\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445382A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5911T>C',\n",
       "  'hgvs_protein': 'p.Cys1971Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445382,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445408-T-C',\n",
       "  'variant_hash': '11be2f5136edd89c',\n",
       "  'vcf_string': '15\\t48445408\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445408T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5885A>G',\n",
       "  'hgvs_protein': 'p.Tyr1962Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445408,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48445419-G-T',\n",
       "  'variant_hash': 'b770b62f594e55d0',\n",
       "  'vcf_string': '15\\t48445419\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445419G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5874C>A',\n",
       "  'hgvs_protein': 'p.Cys1958Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445419,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445421-A-T',\n",
       "  'variant_hash': 'bb19c0af44799d81',\n",
       "  'vcf_string': '15\\t48445421\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445421A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5872T>A',\n",
       "  'hgvs_protein': 'p.Cys1958Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445421,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445424-G-A',\n",
       "  'variant_hash': '13062342ac55f80f',\n",
       "  'vcf_string': '15\\t48445424\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445424G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5869C>T',\n",
       "  'hgvs_protein': 'p.Gln1957Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445424,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445430-G-A',\n",
       "  'variant_hash': '536dc6d2efacdb99',\n",
       "  'vcf_string': '15\\t48445430\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445430G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5863C>T',\n",
       "  'hgvs_protein': 'p.Gln1955Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445430,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48445432-A-C',\n",
       "  'variant_hash': '4eb1620c11debe59',\n",
       "  'vcf_string': '15\\t48445432\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445432A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5861T>G',\n",
       "  'hgvs_protein': 'p.Phe1954Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445432,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48445437-C-T',\n",
       "  'variant_hash': 'efccae0ca637f929',\n",
       "  'vcf_string': '15\\t48445437\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445437C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5856G>A',\n",
       "  'hgvs_protein': 'p.Gly1952=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445437,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445443-T-C',\n",
       "  'variant_hash': '1204a6772965572a',\n",
       "  'vcf_string': '15\\t48445443\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445443T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5850A>G',\n",
       "  'hgvs_protein': 'p.Thr1950=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445443,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445452-G-A',\n",
       "  'variant_hash': 'df26fbf94e5955b0',\n",
       "  'vcf_string': '15\\t48445452\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445452G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5841C>T',\n",
       "  'hgvs_protein': 'p.Cys1947=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445452,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445453-C-T',\n",
       "  'variant_hash': 'c85c160a7b08fc8a',\n",
       "  'vcf_string': '15\\t48445453\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445453C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5840G>A',\n",
       "  'hgvs_protein': 'p.Cys1947Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445453,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445454-A-G',\n",
       "  'variant_hash': 'fefb3ea3f7084c3e',\n",
       "  'vcf_string': '15\\t48445454\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445454A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5839T>C',\n",
       "  'hgvs_protein': 'p.Cys1947Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445454,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48445455-T-C',\n",
       "  'variant_hash': '0bdd2b9fe48efa45',\n",
       "  'vcf_string': '15\\t48445455\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445455T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5838A>G',\n",
       "  'hgvs_protein': 'p.Gln1946=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445455,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445467-G-T',\n",
       "  'variant_hash': 'a4419fee5fc076d0',\n",
       "  'vcf_string': '15\\t48445467\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445467G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5826C>A',\n",
       "  'hgvs_protein': 'p.Cys1942Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445467,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48445469-A-G',\n",
       "  'variant_hash': '9fc261592d885c80',\n",
       "  'vcf_string': '15\\t48445469\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445469A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5824T>C',\n",
       "  'hgvs_protein': 'p.Cys1942Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445469,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445476-C-T',\n",
       "  'variant_hash': '6f99a0fe69fd4f76',\n",
       "  'vcf_string': '15\\t48445476\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445476C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5817G>A',\n",
       "  'hgvs_protein': 'p.Gly1939=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445476,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445488-T-G',\n",
       "  'variant_hash': 'bd2dcd2114102294',\n",
       "  'vcf_string': '15\\t48445488\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445488T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5805A>C',\n",
       "  'hgvs_protein': 'p.Ala1935=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445488,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48445492-C-T',\n",
       "  'variant_hash': '9c8a7e57216e626d',\n",
       "  'vcf_string': '15\\t48445492\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445492C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5801G>A',\n",
       "  'hgvs_protein': 'p.Cys1934Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445492,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48445493-A-G',\n",
       "  'variant_hash': 'c605be31d168548d',\n",
       "  'vcf_string': '15\\t48445493\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445493A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5800T>C',\n",
       "  'hgvs_protein': 'p.Cys1934Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445493,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48445504-T-C',\n",
       "  'variant_hash': '688eca7921121f3d',\n",
       "  'vcf_string': '15\\t48445504\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445504T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5789A>G',\n",
       "  'hgvs_protein': 'p.Asp1930Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445504,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48445505-C-T',\n",
       "  'variant_hash': '0912852182a3acd3',\n",
       "  'vcf_string': '15\\t48445505\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48445505C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5789-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48445505,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446687-C-T',\n",
       "  'variant_hash': '9265702bd414be49',\n",
       "  'vcf_string': '15\\t48446687\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446687C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5788+19G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446687,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48446691-A-G',\n",
       "  'variant_hash': '698bd08ec64c6a64',\n",
       "  'vcf_string': '15\\t48446691\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446691A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5788+15T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446691,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48446696-G-T',\n",
       "  'variant_hash': 'ae84ecbcab770c84',\n",
       "  'vcf_string': '15\\t48446696\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446696G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5788+10C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446696,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48446701-C-T',\n",
       "  'variant_hash': '682d34777733d33d',\n",
       "  'vcf_string': '15\\t48446701\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446701C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5788+5G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446701,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_5th_base_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446701-C-A',\n",
       "  'variant_hash': 'c53960afbc10e379',\n",
       "  'vcf_string': '15\\t48446701\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446701C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5788+5G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446701,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_5th_base_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446706-C-T',\n",
       "  'variant_hash': '82709d7f90b1a07a',\n",
       "  'vcf_string': '15\\t48446706\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446706C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5788G>A',\n",
       "  'hgvs_protein': 'p.Asp1930Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446706,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446711-C-T',\n",
       "  'variant_hash': '3061c37c4df7f5df',\n",
       "  'vcf_string': '15\\t48446711\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446711C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5783G>A',\n",
       "  'hgvs_protein': 'p.Cys1928Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446711,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446712-A-G',\n",
       "  'variant_hash': '4c90e054f5ac221d',\n",
       "  'vcf_string': '15\\t48446712\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446712A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5782T>C',\n",
       "  'hgvs_protein': 'p.Cys1928Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446712,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446747-C-T',\n",
       "  'variant_hash': 'a55d56115f21e812',\n",
       "  'vcf_string': '15\\t48446747\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446747C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5747G>A',\n",
       "  'hgvs_protein': 'p.Cys1916Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446747,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48446751-G-A',\n",
       "  'variant_hash': 'a9b30ee29b341bfe',\n",
       "  'vcf_string': '15\\t48446751\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446751G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5743C>T',\n",
       "  'hgvs_protein': 'p.Arg1915Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446751,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48446753-C-T',\n",
       "  'variant_hash': '477ea7e342ed6535',\n",
       "  'vcf_string': '15\\t48446753\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446753C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5741G>A',\n",
       "  'hgvs_protein': 'p.Cys1914Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446753,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48446765-C-A',\n",
       "  'variant_hash': 'bc691d45b4bd7a75',\n",
       "  'vcf_string': '15\\t48446765\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446765C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5729G>T',\n",
       "  'hgvs_protein': 'p.Gly1910Val',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446765,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48446768-A-G',\n",
       "  'variant_hash': 'beafa552af3ebae4',\n",
       "  'vcf_string': '15\\t48446768\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446768A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5726T>C',\n",
       "  'hgvs_protein': 'p.Ile1909Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446768,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446775-T-C',\n",
       "  'variant_hash': '09106bf0a5a14182',\n",
       "  'vcf_string': '15\\t48446775\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446775T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5719A>G',\n",
       "  'hgvs_protein': 'p.Asn1907Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446775,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48446776-C-G',\n",
       "  'variant_hash': '4aa62ae425535424',\n",
       "  'vcf_string': '15\\t48446776\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446776C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5718G>C',\n",
       "  'hgvs_protein': 'p.Arg1906=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446776,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48446795-C-T',\n",
       "  'variant_hash': '257fed0735de0226',\n",
       "  'vcf_string': '15\\t48446795\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446795C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5699G>A',\n",
       "  'hgvs_protein': 'p.Cys1900Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446795,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48446797-G-A',\n",
       "  'variant_hash': '030e1098800e32f1',\n",
       "  'vcf_string': '15\\t48446797\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446797G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5697C>T',\n",
       "  'hgvs_protein': 'p.Ala1899=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446797,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48446811-A-G',\n",
       "  'variant_hash': '4123060547f1e834',\n",
       "  'vcf_string': '15\\t48446811\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446811A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5683T>C',\n",
       "  'hgvs_protein': 'p.Cys1895Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446811,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446814-C-T',\n",
       "  'variant_hash': '47ef48ef4dae6e7d',\n",
       "  'vcf_string': '15\\t48446814\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446814C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5680G>A',\n",
       "  'hgvs_protein': 'p.Glu1894Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446814,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48446816-T-C',\n",
       "  'variant_hash': '5d20b4cc64cbe99b',\n",
       "  'vcf_string': '15\\t48446816\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446816T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5678A>G',\n",
       "  'hgvs_protein': 'p.Asn1893Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446816,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48446823-C-T',\n",
       "  'variant_hash': '43deced036587ed5',\n",
       "  'vcf_string': '15\\t48446823\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446823C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5672-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446823,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446823-C-G',\n",
       "  'variant_hash': 'a2097a144e38638a',\n",
       "  'vcf_string': '15\\t48446823\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446823C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5672-1G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446823,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48446828-A-T',\n",
       "  'variant_hash': '0f0db93646a338a7',\n",
       "  'vcf_string': '15\\t48446828\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446828A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5672-6T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446828,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48446830-G-C',\n",
       "  'variant_hash': 'c7787f6044a326ad',\n",
       "  'vcf_string': '15\\t48446830\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446830G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5672-8C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446830,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48446885-C-A',\n",
       "  'variant_hash': '6ba4f78d687feddc',\n",
       "  'vcf_string': '15\\t48446885\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446885C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5672-63G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446885,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48446909-T-C',\n",
       "  'variant_hash': '0644fe5f86aacff2',\n",
       "  'vcf_string': '15\\t48446909\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48446909T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5672-87A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48446909,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48448768-C-T',\n",
       "  'variant_hash': '2257c2a1e7f68491',\n",
       "  'vcf_string': '15\\t48448768\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448768C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5671G>A',\n",
       "  'hgvs_protein': 'p.Asp1891Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448768,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48448768-C-G',\n",
       "  'variant_hash': '9748b6075d4c10ca',\n",
       "  'vcf_string': '15\\t48448768\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448768C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5671G>C',\n",
       "  'hgvs_protein': 'p.Asp1891His',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448768,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48448773-C-T',\n",
       "  'variant_hash': '68a300cfb66c1935',\n",
       "  'vcf_string': '15\\t48448773\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448773C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5666G>A',\n",
       "  'hgvs_protein': 'p.Cys1889Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448773,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48448790-A-G',\n",
       "  'variant_hash': '73b55dc567094aec',\n",
       "  'vcf_string': '15\\t48448790\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448790A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5649T>C',\n",
       "  'hgvs_protein': 'p.Asn1883=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448790,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48448813-A-G',\n",
       "  'variant_hash': 'dcbc24fe7d496682',\n",
       "  'vcf_string': '15\\t48448813\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448813A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5626T>C',\n",
       "  'hgvs_protein': 'p.Cys1876Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448813,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48448832-A-G',\n",
       "  'variant_hash': 'cfde5968433d842d',\n",
       "  'vcf_string': '15\\t48448832\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448832A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5607T>C',\n",
       "  'hgvs_protein': 'p.Val1869=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448832,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48448851-C-T',\n",
       "  'variant_hash': '807d3ea5c091d639',\n",
       "  'vcf_string': '15\\t48448851\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448851C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5588G>A',\n",
       "  'hgvs_protein': 'p.Gly1863Glu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448851,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48448853-A-G',\n",
       "  'variant_hash': '20479d50f8d4a711',\n",
       "  'vcf_string': '15\\t48448853\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448853A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5586T>C',\n",
       "  'hgvs_protein': 'p.His1862=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448853,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48448860-C-T',\n",
       "  'variant_hash': '46885208f28a0ed3',\n",
       "  'vcf_string': '15\\t48448860\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448860C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5579G>A',\n",
       "  'hgvs_protein': 'p.Cys1860Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448860,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48448879-G-A',\n",
       "  'variant_hash': '91b79f24a9f27cb2',\n",
       "  'vcf_string': '15\\t48448879\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448879G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5560C>T',\n",
       "  'hgvs_protein': 'p.Gln1854Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448879,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48448881-C-T',\n",
       "  'variant_hash': 'f98e3d03ebbb2fe1',\n",
       "  'vcf_string': '15\\t48448881\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448881C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5558G>A',\n",
       "  'hgvs_protein': 'p.Cys1853Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448881,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48448882-A-G',\n",
       "  'variant_hash': '35cdca661924c0ce',\n",
       "  'vcf_string': '15\\t48448882\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448882A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5557T>C',\n",
       "  'hgvs_protein': 'p.Cys1853Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448882,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48448894-C-T',\n",
       "  'variant_hash': '34bc7dda59fe38c2',\n",
       "  'vcf_string': '15\\t48448894\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48448894C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5546-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48448894,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452550-G-A',\n",
       "  'variant_hash': 'b738472dcb137651',\n",
       "  'vcf_string': '15\\t48452550\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452550G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5545+12C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452550,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452567-C-T',\n",
       "  'variant_hash': '148f00d474619a76',\n",
       "  'vcf_string': '15\\t48452567\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452567C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5540G>A',\n",
       "  'hgvs_protein': 'p.Cys1847Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452567,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452575-T-C',\n",
       "  'variant_hash': '2fadeb5aa29054dc',\n",
       "  'vcf_string': '15\\t48452575\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452575T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5532A>G',\n",
       "  'hgvs_protein': 'p.Thr1844=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452575,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452589-G-A',\n",
       "  'variant_hash': '826a4eec0c553e3a',\n",
       "  'vcf_string': '15\\t48452589\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452589G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5518C>T',\n",
       "  'hgvs_protein': 'p.Arg1840Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452589,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48452590-G-A',\n",
       "  'variant_hash': '47dc99ff5e005461',\n",
       "  'vcf_string': '15\\t48452590\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452590G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5517C>T',\n",
       "  'hgvs_protein': 'p.Tyr1839=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452590,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452595-C-A',\n",
       "  'variant_hash': 'd5aa158a8a4efa1e',\n",
       "  'vcf_string': '15\\t48452595\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452595C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5512G>T',\n",
       "  'hgvs_protein': 'p.Gly1838Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452595,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48452596-G-A',\n",
       "  'variant_hash': '2f9b26c4993e6b7b',\n",
       "  'vcf_string': '15\\t48452596\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452596G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5511C>T',\n",
       "  'hgvs_protein': 'p.Pro1837=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452596,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452603-C-T',\n",
       "  'variant_hash': '151a0dfe392065d4',\n",
       "  'vcf_string': '15\\t48452603\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452603C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5504G>A',\n",
       "  'hgvs_protein': 'p.Cys1835Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452603,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452614-G-T',\n",
       "  'variant_hash': '53bf7c0b182bc591',\n",
       "  'vcf_string': '15\\t48452614\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452614G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5493C>A',\n",
       "  'hgvs_protein': 'p.Tyr1831Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452614,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48452626-A-C',\n",
       "  'variant_hash': 'd56762821c758c8f',\n",
       "  'vcf_string': '15\\t48452626\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452626A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5481T>G',\n",
       "  'hgvs_protein': 'p.Thr1827=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452626,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48452637-A-G',\n",
       "  'variant_hash': 'f5fd77a155cc2928',\n",
       "  'vcf_string': '15\\t48452637\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452637A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5470T>C',\n",
       "  'hgvs_protein': 'p.Cys1824Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452637,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452641-G-A',\n",
       "  'variant_hash': 'e5b5fa4bfe4e05c2',\n",
       "  'vcf_string': '15\\t48452641\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452641G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5466C>T',\n",
       "  'hgvs_protein': 'p.Ala1822=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452641,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452644-G-A',\n",
       "  'variant_hash': '90d5029b42dc16b6',\n",
       "  'vcf_string': '15\\t48452644\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452644G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5463C>T',\n",
       "  'hgvs_protein': 'p.Asn1821=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452644,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452654-C-T',\n",
       "  'variant_hash': 'e8b772739563384e',\n",
       "  'vcf_string': '15\\t48452654\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452654C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5453G>A',\n",
       "  'hgvs_protein': 'p.Cys1818Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452654,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452655-A-G',\n",
       "  'variant_hash': 'f19c95b77f7a5c77',\n",
       "  'vcf_string': '15\\t48452655\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452655A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5452T>C',\n",
       "  'hgvs_protein': 'p.Cys1818Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452655,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48452665-G-A',\n",
       "  'variant_hash': '8971ab9a3a40441f',\n",
       "  'vcf_string': '15\\t48452665\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452665G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5442C>T',\n",
       "  'hgvs_protein': 'p.Asn1814=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452665,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48452670-G-A',\n",
       "  'variant_hash': 'e34d203b9ecd97e9',\n",
       "  'vcf_string': '15\\t48452670\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452670G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5437C>T',\n",
       "  'hgvs_protein': 'p.Gln1813Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452670,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452673-A-G',\n",
       "  'variant_hash': '9338a102c49bedf7',\n",
       "  'vcf_string': '15\\t48452673\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452673A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5434T>C',\n",
       "  'hgvs_protein': 'p.Cys1812Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452673,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48452676-C-T',\n",
       "  'variant_hash': '2bd1bb39a557db88',\n",
       "  'vcf_string': '15\\t48452676\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452676C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5431G>A',\n",
       "  'hgvs_protein': 'p.Glu1811Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452676,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452676-C-A',\n",
       "  'variant_hash': 'd19a1276e43eb8ba',\n",
       "  'vcf_string': '15\\t48452676\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452676C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5431G>T',\n",
       "  'hgvs_protein': 'p.Glu1811Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452676,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48452677-G-A',\n",
       "  'variant_hash': 'b93d4bdc0663ae0c',\n",
       "  'vcf_string': '15\\t48452677\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452677G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5430C>T',\n",
       "  'hgvs_protein': 'p.Asp1810=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452677,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48452685-C-G',\n",
       "  'variant_hash': 'f3eeec0b9636bbcc',\n",
       "  'vcf_string': '15\\t48452685\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452685C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5423-1G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452685,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452686-T-C',\n",
       "  'variant_hash': '68852d716f1f4fb4',\n",
       "  'vcf_string': '15\\t48452686\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452686T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5423-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452686,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48452699-G-T',\n",
       "  'variant_hash': '6b1928de0fd0041f',\n",
       "  'vcf_string': '15\\t48452699\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48452699G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5423-15C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48452699,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456635-A-G',\n",
       "  'variant_hash': '7b71d11730ee9cee',\n",
       "  'vcf_string': '15\\t48456635\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456635A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5422+2T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456635,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48456636-C-T',\n",
       "  'variant_hash': '1b091ee498bbd535',\n",
       "  'vcf_string': '15\\t48456636\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456636C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5422+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456636,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48456642-C-A',\n",
       "  'variant_hash': '7706ec1bfbc8e1ab',\n",
       "  'vcf_string': '15\\t48456642\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456642C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5417G>T',\n",
       "  'hgvs_protein': 'p.Cys1806Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456642,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48456643-A-G',\n",
       "  'variant_hash': '4208078819dcdc66',\n",
       "  'vcf_string': '15\\t48456643\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456643A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5416T>C',\n",
       "  'hgvs_protein': 'p.Cys1806Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456643,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48456647-C-T',\n",
       "  'variant_hash': '31a744dfef5eeccf',\n",
       "  'vcf_string': '15\\t48456647\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456647C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5412G>A',\n",
       "  'hgvs_protein': 'p.Leu1804=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456647,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456650-C-T',\n",
       "  'variant_hash': 'bbbe100625a143e1',\n",
       "  'vcf_string': '15\\t48456650\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456650C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5409G>A',\n",
       "  'hgvs_protein': 'p.Leu1803=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456650,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456674-C-G',\n",
       "  'variant_hash': '5e01ae31c20acfd2',\n",
       "  'vcf_string': '15\\t48456674\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456674C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5385G>C',\n",
       "  'hgvs_protein': 'p.Val1795=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456674,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456677-T-C',\n",
       "  'variant_hash': '52b62da23300aa7a',\n",
       "  'vcf_string': '15\\t48456677\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456677T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5382A>G',\n",
       "  'hgvs_protein': 'p.Pro1794=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456677,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456682-A-G',\n",
       "  'variant_hash': '3a826ccefc0151f3',\n",
       "  'vcf_string': '15\\t48456682\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456682A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5377T>C',\n",
       "  'hgvs_protein': 'p.Cys1793Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456682,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48456687-C-T',\n",
       "  'variant_hash': '82fa0e1efb12d4a6',\n",
       "  'vcf_string': '15\\t48456687\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456687C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5372G>A',\n",
       "  'hgvs_protein': 'p.Cys1791Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456687,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48456688-A-G',\n",
       "  'variant_hash': 'a9cb639aa336c0a9',\n",
       "  'vcf_string': '15\\t48456688\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456688A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5371T>C',\n",
       "  'hgvs_protein': 'p.Cys1791Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456688,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48456691-G-A',\n",
       "  'variant_hash': 'cce9ac3a6920a544',\n",
       "  'vcf_string': '15\\t48456691\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456691G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5368C>T',\n",
       "  'hgvs_protein': 'p.Arg1790Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456691,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48456714-C-T',\n",
       "  'variant_hash': '3df6cb2a6b7cebb1',\n",
       "  'vcf_string': '15\\t48456714\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456714C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5345G>A',\n",
       "  'hgvs_protein': 'p.Cys1782Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456714,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48456716-C-T',\n",
       "  'variant_hash': '8e3534d6c10130b6',\n",
       "  'vcf_string': '15\\t48456716\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456716C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5343G>A',\n",
       "  'hgvs_protein': 'p.Val1781=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456716,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48456719-T-G',\n",
       "  'variant_hash': '75dcb06cadc8e6d3',\n",
       "  'vcf_string': '15\\t48456719\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456719T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5340A>C',\n",
       "  'hgvs_protein': 'p.Gly1780=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456719,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456729-C-T',\n",
       "  'variant_hash': '1d4f6cb01c219abb',\n",
       "  'vcf_string': '15\\t48456729\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456729C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5330G>A',\n",
       "  'hgvs_protein': 'p.Cys1777Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456729,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48456731-G-A',\n",
       "  'variant_hash': '7aae48d441623547',\n",
       "  'vcf_string': '15\\t48456731\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456731G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5328C>T',\n",
       "  'hgvs_protein': 'p.Val1776=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456731,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456750-C-T',\n",
       "  'variant_hash': '9d63d1818020bea7',\n",
       "  'vcf_string': '15\\t48456750\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456750C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5309G>A',\n",
       "  'hgvs_protein': 'p.Cys1770Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456750,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48456758-A-G',\n",
       "  'variant_hash': '39c24bc16c59f963',\n",
       "  'vcf_string': '15\\t48456758\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456758A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5301T>C',\n",
       "  'hgvs_protein': 'p.Ile1767=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456758,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456772-G-A',\n",
       "  'variant_hash': '0a69ad91be3818e5',\n",
       "  'vcf_string': '15\\t48456772\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456772G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5297-10C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456772,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48456781-T-C',\n",
       "  'variant_hash': 'ed24ae0569c6ea16',\n",
       "  'vcf_string': '15\\t48456781\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48456781T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5297-19A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48456781,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460232-C-T',\n",
       "  'variant_hash': '8effcd7c50fd6e83',\n",
       "  'vcf_string': '15\\t48460232\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460232C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5296+14G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460232,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48460233-C-T',\n",
       "  'variant_hash': '143bcb2a51b31e34',\n",
       "  'vcf_string': '15\\t48460233\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460233C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5296+13G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460233,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460234-G-A',\n",
       "  'variant_hash': '08f5a522fd0a9900',\n",
       "  'vcf_string': '15\\t48460234\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460234G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5296+12C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460234,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460241-C-T',\n",
       "  'variant_hash': '961189ad72d3b944',\n",
       "  'vcf_string': '15\\t48460241\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460241C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5296+5G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460241,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_5th_base_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48460244-A-C',\n",
       "  'variant_hash': '42cf203b15173947',\n",
       "  'vcf_string': '15\\t48460244\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460244A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5296+2T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460244,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48460250-G-A',\n",
       "  'variant_hash': 'cce4f462fa65eb39',\n",
       "  'vcf_string': '15\\t48460250\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460250G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5292C>T',\n",
       "  'hgvs_protein': 'p.Pro1764=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460250,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460258-C-T',\n",
       "  'variant_hash': '8edb1055943790c2',\n",
       "  'vcf_string': '15\\t48460258\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460258C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5284G>A',\n",
       "  'hgvs_protein': 'p.Gly1762Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460258,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48460259-G-A',\n",
       "  'variant_hash': '1f2c2fdcd63b09d8',\n",
       "  'vcf_string': '15\\t48460259\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460259G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5283C>T',\n",
       "  'hgvs_protein': 'p.Thr1761=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460259,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460262-A-G',\n",
       "  'variant_hash': 'd25ff756bbd8b580',\n",
       "  'vcf_string': '15\\t48460262\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460262A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5280T>C',\n",
       "  'hgvs_protein': 'p.Tyr1760=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460262,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460271-G-A',\n",
       "  'variant_hash': '9e34f7cde68871a2',\n",
       "  'vcf_string': '15\\t48460271\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460271G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5271C>T',\n",
       "  'hgvs_protein': 'p.Ile1757=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460271,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460274-G-T',\n",
       "  'variant_hash': '8f4178e7df234e53',\n",
       "  'vcf_string': '15\\t48460274\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460274G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5268C>A',\n",
       "  'hgvs_protein': 'p.Val1756=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460274,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460280-G-T',\n",
       "  'variant_hash': 'ab57f31625b45d18',\n",
       "  'vcf_string': '15\\t48460280\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460280G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5262C>A',\n",
       "  'hgvs_protein': 'p.Gly1754=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460280,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460281-C-T',\n",
       "  'variant_hash': 'ed1add3f9813668f',\n",
       "  'vcf_string': '15\\t48460281\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460281C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5261G>A',\n",
       "  'hgvs_protein': 'p.Gly1754Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460281,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48460283-T-C',\n",
       "  'variant_hash': 'd42b3af9dd5af574',\n",
       "  'vcf_string': '15\\t48460283\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460283T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5259A>G',\n",
       "  'hgvs_protein': 'p.Pro1753=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460283,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48460319-T-C',\n",
       "  'variant_hash': '44d8c1276b992d9f',\n",
       "  'vcf_string': '15\\t48460319\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460319T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5225-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460319,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48460327-T-A',\n",
       "  'variant_hash': 'fbdacbc2cbc878b5',\n",
       "  'vcf_string': '15\\t48460327\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460327T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5225-10A>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460327,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460330-A-C',\n",
       "  'variant_hash': '38adce18681cb055',\n",
       "  'vcf_string': '15\\t48460330\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460330A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5225-13T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460330,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48460336-G-A',\n",
       "  'variant_hash': '88a0d1942ebd9371',\n",
       "  'vcf_string': '15\\t48460336\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48460336G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5225-19C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48460336,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48462971-T-C',\n",
       "  'variant_hash': '3f95ae7e0043539b',\n",
       "  'vcf_string': '15\\t48462971\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48462971T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5224+111A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48462971,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48463098-A-T',\n",
       "  'variant_hash': 'b12045cb40062215',\n",
       "  'vcf_string': '15\\t48463098\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463098A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5208T>A',\n",
       "  'hgvs_protein': 'p.Cys1736Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463098,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463116-G-A',\n",
       "  'variant_hash': '9156b6e364a375d4',\n",
       "  'vcf_string': '15\\t48463116\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463116G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5190C>T',\n",
       "  'hgvs_protein': 'p.Asn1730=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463116,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463122-C-T',\n",
       "  'variant_hash': 'a589cc72b50e5150',\n",
       "  'vcf_string': '15\\t48463122\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463122C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5184G>A',\n",
       "  'hgvs_protein': 'p.Ala1728=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463122,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463123-G-A',\n",
       "  'variant_hash': '22f68ba180b1f76f',\n",
       "  'vcf_string': '15\\t48463123\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463123G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5183C>T',\n",
       "  'hgvs_protein': 'p.Ala1728Val',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463123,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463129-C-T',\n",
       "  'variant_hash': '846d1f2b84565fe7',\n",
       "  'vcf_string': '15\\t48463129\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463129C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5177G>A',\n",
       "  'hgvs_protein': 'p.Gly1726Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463129,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463148-A-G',\n",
       "  'variant_hash': 'a77390078db9e3c7',\n",
       "  'vcf_string': '15\\t48463148\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463148A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5158T>C',\n",
       "  'hgvs_protein': 'p.Cys1720Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463148,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48463185-A-G',\n",
       "  'variant_hash': '100e6179a70308ec',\n",
       "  'vcf_string': '15\\t48463185\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463185A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5121T>C',\n",
       "  'hgvs_protein': 'p.Asp1707=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463185,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463197-G-A',\n",
       "  'variant_hash': '5b81592604770668',\n",
       "  'vcf_string': '15\\t48463197\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463197G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5109C>T',\n",
       "  'hgvs_protein': 'p.Asn1703=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463197,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463203-A-C',\n",
       "  'variant_hash': 'f7b89dab8f15a146',\n",
       "  'vcf_string': '15\\t48463203\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463203A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5103T>G',\n",
       "  'hgvs_protein': 'p.Ala1701=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463203,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463206-A-G',\n",
       "  'variant_hash': '853a33fdf338b0ba',\n",
       "  'vcf_string': '15\\t48463206\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463206A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5100T>C',\n",
       "  'hgvs_protein': 'p.Tyr1700=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463206,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463207-T-C',\n",
       "  'variant_hash': '8e825445d7038ec3',\n",
       "  'vcf_string': '15\\t48463207\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463207T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5099A>G',\n",
       "  'hgvs_protein': 'p.Tyr1700Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463207,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463209-G-C',\n",
       "  'variant_hash': 'e8b0a0dd8f9743eb',\n",
       "  'vcf_string': '15\\t48463209\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463209G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5097C>G',\n",
       "  'hgvs_protein': 'p.Tyr1699Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463209,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463209-G-A',\n",
       "  'variant_hash': '8a6d3a4998863a33',\n",
       "  'vcf_string': '15\\t48463209\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463209G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5097C>T',\n",
       "  'hgvs_protein': 'p.Tyr1699=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463209,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48463210-T-C',\n",
       "  'variant_hash': '0d639b7dca96831b',\n",
       "  'vcf_string': '15\\t48463210\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463210T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5096A>G',\n",
       "  'hgvs_protein': 'p.Tyr1699Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463210,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463219-T-C',\n",
       "  'variant_hash': '6057b5e06b1e2c2c',\n",
       "  'vcf_string': '15\\t48463219\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463219T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5087A>G',\n",
       "  'hgvs_protein': 'p.Tyr1696Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463219,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463226-A-G',\n",
       "  'variant_hash': '1dc0589786233c09',\n",
       "  'vcf_string': '15\\t48463226\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463226A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5080T>C',\n",
       "  'hgvs_protein': 'p.Leu1694=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463226,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463230-T-C',\n",
       "  'variant_hash': '1b38a95c623163ba',\n",
       "  'vcf_string': '15\\t48463230\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463230T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5076A>G',\n",
       "  'hgvs_protein': 'p.Arg1692=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463230,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463241-C-T',\n",
       "  'variant_hash': 'd9f7300739c85dcd',\n",
       "  'vcf_string': '15\\t48463241\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463241C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463241,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463241-C-A',\n",
       "  'variant_hash': 'ee62f4f3537a0c17',\n",
       "  'vcf_string': '15\\t48463241\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463241C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463241,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463247-G-C',\n",
       "  'variant_hash': '0b011a9704d94a80',\n",
       "  'vcf_string': '15\\t48463247\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463247G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-7C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463247,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463252-G-T',\n",
       "  'variant_hash': '7662c893ec493261',\n",
       "  'vcf_string': '15\\t48463252\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463252G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-12C>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463252,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48463252-G-A',\n",
       "  'variant_hash': 'e26cccee22bd1b3e',\n",
       "  'vcf_string': '15\\t48463252\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463252G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-12C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463252,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463253-T-A',\n",
       "  'variant_hash': '6b4cce2a454e4dc0',\n",
       "  'vcf_string': '15\\t48463253\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463253T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-13A>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463253,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463259-A-G',\n",
       "  'variant_hash': '353cfdfb4b62ea98',\n",
       "  'vcf_string': '15\\t48463259\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463259A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-19T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463259,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463275-G-A',\n",
       "  'variant_hash': 'eda534dba07213f1',\n",
       "  'vcf_string': '15\\t48463275\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463275G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-35C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463275,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48463333-T-C',\n",
       "  'variant_hash': '45572db5803079b0',\n",
       "  'vcf_string': '15\\t48463333\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463333T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5066-93A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463333,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48463889-T-C',\n",
       "  'variant_hash': '88436ecd7c7c6631',\n",
       "  'vcf_string': '15\\t48463889\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463889T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5065+10A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463889,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48463898-C-T',\n",
       "  'variant_hash': '395623db7180d6f5',\n",
       "  'vcf_string': '15\\t48463898\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463898C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5065+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463898,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463898-C-G',\n",
       "  'variant_hash': '6b9ff489ca9d4853',\n",
       "  'vcf_string': '15\\t48463898\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463898C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5065+1G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463898,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463898-C-A',\n",
       "  'variant_hash': 'cbafeb8a2305c632',\n",
       "  'vcf_string': '15\\t48463898\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463898C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5065+1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463898,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463904-C-A',\n",
       "  'variant_hash': '6536ba5cab562b47',\n",
       "  'vcf_string': '15\\t48463904\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463904C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5060G>T',\n",
       "  'hgvs_protein': 'p.Cys1687Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463904,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463933-G-A',\n",
       "  'variant_hash': '58193d0f496fce24',\n",
       "  'vcf_string': '15\\t48463933\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463933G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5031C>T',\n",
       "  'hgvs_protein': 'p.Asp1677=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463933,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463943-C-T',\n",
       "  'variant_hash': '48e7af65e034e830',\n",
       "  'vcf_string': '15\\t48463943\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463943C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5021G>A',\n",
       "  'hgvs_protein': 'p.Cys1674Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463943,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463949-C-T',\n",
       "  'variant_hash': '987d65fe712c3b06',\n",
       "  'vcf_string': '15\\t48463949\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463949C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5015G>A',\n",
       "  'hgvs_protein': 'p.Cys1672Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463949,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463949-C-G',\n",
       "  'variant_hash': '2a219d579cae3881',\n",
       "  'vcf_string': '15\\t48463949\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463949C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5015G>C',\n",
       "  'hgvs_protein': 'p.Cys1672Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463949,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463949-C-A',\n",
       "  'variant_hash': '1052ad6d02d0745e',\n",
       "  'vcf_string': '15\\t48463949\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463949C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5015G>T',\n",
       "  'hgvs_protein': 'p.Cys1672Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463949,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463950-A-C',\n",
       "  'variant_hash': '1041ceb2f8d58904',\n",
       "  'vcf_string': '15\\t48463950\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463950A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5014T>G',\n",
       "  'hgvs_protein': 'p.Cys1672Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463950,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48463955-T-C',\n",
       "  'variant_hash': 'd7a9d94e7cba5c0a',\n",
       "  'vcf_string': '15\\t48463955\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463955T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5009A>G',\n",
       "  'hgvs_protein': 'p.Tyr1670Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463955,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48463957-G-A',\n",
       "  'variant_hash': '22f19ae1f46de1b8',\n",
       "  'vcf_string': '15\\t48463957\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463957G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.5007C>T',\n",
       "  'hgvs_protein': 'p.Asn1669=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463957,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463966-G-C',\n",
       "  'variant_hash': '308c53db1b916cdb',\n",
       "  'vcf_string': '15\\t48463966\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463966G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4998C>G',\n",
       "  'hgvs_protein': 'p.Thr1666=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463966,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48463976-C-A',\n",
       "  'variant_hash': '4c3cc5fbfcf28177',\n",
       "  'vcf_string': '15\\t48463976\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463976C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4988G>T',\n",
       "  'hgvs_protein': 'p.Cys1663Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463976,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48463978-T-C',\n",
       "  'variant_hash': 'e310e654f5da1884',\n",
       "  'vcf_string': '15\\t48463978\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463978T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4986A>G',\n",
       "  'hgvs_protein': 'p.Thr1662=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463978,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48463984-T-C',\n",
       "  'variant_hash': '3baefaaa4182a4c4',\n",
       "  'vcf_string': '15\\t48463984\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48463984T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4980A>G',\n",
       "  'hgvs_protein': 'p.Pro1660=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48463984,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48464009-C-T',\n",
       "  'variant_hash': '3ea0d935d47f13df',\n",
       "  'vcf_string': '15\\t48464009\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48464009C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4955G>A',\n",
       "  'hgvs_protein': 'p.Cys1652Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48464009,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48464022-C-G',\n",
       "  'variant_hash': 'c786f81526349b49',\n",
       "  'vcf_string': '15\\t48464022\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48464022C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4943-1G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48464022,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48464025-T-C',\n",
       "  'variant_hash': 'b4fb5f6f46fc66c2',\n",
       "  'vcf_string': '15\\t48464025\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48464025T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4943-4A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48464025,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48464034-G-A',\n",
       "  'variant_hash': '6d31a8f4659fa679',\n",
       "  'vcf_string': '15\\t48464034\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48464034G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4943-13C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48464034,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48464090-T-C',\n",
       "  'variant_hash': '8ba3c191300dd3cd',\n",
       "  'vcf_string': '15\\t48464090\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48464090T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4943-69A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48464090,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465467-T-A',\n",
       "  'variant_hash': 'c835c2997b3d9847',\n",
       "  'vcf_string': '15\\t48465467\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465467T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4942+101A>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465467,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465568-C-T',\n",
       "  'variant_hash': '8c7680d8e923c24c',\n",
       "  'vcf_string': '15\\t48465568\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465568C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4942G>A',\n",
       "  'hgvs_protein': 'p.Asp1648Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465568,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465573-C-T',\n",
       "  'variant_hash': '52146fa0b8f9303f',\n",
       "  'vcf_string': '15\\t48465573\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465573C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4937G>A',\n",
       "  'hgvs_protein': 'p.Cys1646Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465573,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465574-A-T',\n",
       "  'variant_hash': '8e94ee2375bda140',\n",
       "  'vcf_string': '15\\t48465574\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465574A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4936T>A',\n",
       "  'hgvs_protein': 'p.Cys1646Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465574,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465580-G-T',\n",
       "  'variant_hash': 'f72a52bcca5c9246',\n",
       "  'vcf_string': '15\\t48465580\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465580G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4930C>A',\n",
       "  'hgvs_protein': 'p.Arg1644=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465580,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465580-G-A',\n",
       "  'variant_hash': '2b97c3b16222b499',\n",
       "  'vcf_string': '15\\t48465580\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465580G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4930C>T',\n",
       "  'hgvs_protein': 'p.Arg1644Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465580,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465584-A-G',\n",
       "  'variant_hash': 'c76958b3911fc9c6',\n",
       "  'vcf_string': '15\\t48465584\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465584A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4926T>C',\n",
       "  'hgvs_protein': 'p.Asp1642=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465584,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465605-G-C',\n",
       "  'variant_hash': 'dd37d5182e364ce3',\n",
       "  'vcf_string': '15\\t48465605\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465605G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4905C>G',\n",
       "  'hgvs_protein': 'p.Thr1635=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465605,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48465605-G-A',\n",
       "  'variant_hash': '0cc2fa6e2ea6ffda',\n",
       "  'vcf_string': '15\\t48465605\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465605G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4905C>T',\n",
       "  'hgvs_protein': 'p.Thr1635=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465605,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465619-A-T',\n",
       "  'variant_hash': '9a57cf9f20164d0c',\n",
       "  'vcf_string': '15\\t48465619\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465619A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4891T>A',\n",
       "  'hgvs_protein': 'p.Cys1631Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465619,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48465622-G-A',\n",
       "  'variant_hash': '67df1e48a74f2f49',\n",
       "  'vcf_string': '15\\t48465622\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465622G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4888C>T',\n",
       "  'hgvs_protein': 'p.Gln1630Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465622,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48465635-G-C',\n",
       "  'variant_hash': 'a7d6ecccc8508f3a',\n",
       "  'vcf_string': '15\\t48465635\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465635G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4875C>G',\n",
       "  'hgvs_protein': 'p.Thr1625=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465635,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465646-A-G',\n",
       "  'variant_hash': '7ec076a0b7e8832c',\n",
       "  'vcf_string': '15\\t48465646\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465646A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4864T>C',\n",
       "  'hgvs_protein': 'p.Cys1622Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465646,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465647-T-C',\n",
       "  'variant_hash': 'bb467447c2cb2a7b',\n",
       "  'vcf_string': '15\\t48465647\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465647T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4863A>G',\n",
       "  'hgvs_protein': 'p.Lys1621=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465647,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465679-G-A',\n",
       "  'variant_hash': '5a9ecb0a5d537f20',\n",
       "  'vcf_string': '15\\t48465679\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465679G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4831C>T',\n",
       "  'hgvs_protein': 'p.Gln1611Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465679,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465686-A-G',\n",
       "  'variant_hash': 'a3b7423f7167193b',\n",
       "  'vcf_string': '15\\t48465686\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465686A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4824T>C',\n",
       "  'hgvs_protein': 'p.Asp1608=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465686,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465695-T-C',\n",
       "  'variant_hash': '3c5c86ded724da77',\n",
       "  'vcf_string': '15\\t48465695\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465695T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4817-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465695,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48465788-A-G',\n",
       "  'variant_hash': '64c4ed6840071634',\n",
       "  'vcf_string': '15\\t48465788\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465788A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4816+2T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465788,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465789-C-A',\n",
       "  'variant_hash': 'fc596da1ae690599',\n",
       "  'vcf_string': '15\\t48465789\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465789C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4816+1G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465789,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465803-G-A',\n",
       "  'variant_hash': 'fbb3c0bc3dc6b576',\n",
       "  'vcf_string': '15\\t48465803\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465803G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4803C>T',\n",
       "  'hgvs_protein': 'p.Thr1601=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465803,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465815-T-G',\n",
       "  'variant_hash': 'cc65c5b3deeef524',\n",
       "  'vcf_string': '15\\t48465815\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465815T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4791A>C',\n",
       "  'hgvs_protein': 'p.Pro1597=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465815,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465820-G-A',\n",
       "  'variant_hash': '845c15904220312e',\n",
       "  'vcf_string': '15\\t48465820\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465820G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4786C>T',\n",
       "  'hgvs_protein': 'p.Arg1596Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465820,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48465825-C-T',\n",
       "  'variant_hash': 'c0227e6e574fb152',\n",
       "  'vcf_string': '15\\t48465825\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465825C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4781G>A',\n",
       "  'hgvs_protein': 'p.Gly1594Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465825,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465840-C-A',\n",
       "  'variant_hash': 'fd18b58eb7875a07',\n",
       "  'vcf_string': '15\\t48465840\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465840C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4766G>T',\n",
       "  'hgvs_protein': 'p.Cys1589Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465840,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48465857-G-A',\n",
       "  'variant_hash': '607db87701b1f48e',\n",
       "  'vcf_string': '15\\t48465857\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465857G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4749C>T',\n",
       "  'hgvs_protein': 'p.Ser1583=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465857,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465861-A-G',\n",
       "  'variant_hash': 'b3a28ddbe4d325d4',\n",
       "  'vcf_string': '15\\t48465861\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465861A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4748-3T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465861,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465864-G-A',\n",
       "  'variant_hash': '3896b2a96bb99e3f',\n",
       "  'vcf_string': '15\\t48465864\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465864G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4748-6C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465864,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465869-A-G',\n",
       "  'variant_hash': '6d8410a8dda2f1cc',\n",
       "  'vcf_string': '15\\t48465869\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465869A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4748-11T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465869,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48465870-A-T',\n",
       "  'variant_hash': 'c1ef7aad44d1c0d6',\n",
       "  'vcf_string': '15\\t48465870\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465870A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4748-12T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465870,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48465935-A-C',\n",
       "  'variant_hash': 'ac7cdb9e913a2400',\n",
       "  'vcf_string': '15\\t48465935\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48465935A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4748-77T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48465935,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48467870-G-A',\n",
       "  'variant_hash': '3ae7af9c98da1503',\n",
       "  'vcf_string': '15\\t48467870\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467870G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4747+68C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467870,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48467954-A-T',\n",
       "  'variant_hash': '152547831d8ee034',\n",
       "  'vcf_string': '15\\t48467954\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467954A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4731T>A',\n",
       "  'hgvs_protein': 'p.Cys1577Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467954,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48467966-A-G',\n",
       "  'variant_hash': '08fddad90f850f95',\n",
       "  'vcf_string': '15\\t48467966\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467966A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4719T>C',\n",
       "  'hgvs_protein': 'p.Pro1573=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467966,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48467972-A-T',\n",
       "  'variant_hash': '457e19590f1bd67b',\n",
       "  'vcf_string': '15\\t48467972\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467972A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4713T>A',\n",
       "  'hgvs_protein': 'p.Gly1571=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467972,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48467975-C-T',\n",
       "  'variant_hash': 'f93b38d281ef3f94',\n",
       "  'vcf_string': '15\\t48467975\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467975C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4710G>A',\n",
       "  'hgvs_protein': 'p.Trp1570Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467975,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48467978-G-C',\n",
       "  'variant_hash': '474601fe992c868d',\n",
       "  'vcf_string': '15\\t48467978\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467978G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4707C>G',\n",
       "  'hgvs_protein': 'p.Ala1569=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467978,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48467990-A-T',\n",
       "  'variant_hash': '5f0419266fd7e0b2',\n",
       "  'vcf_string': '15\\t48467990\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467990A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4695T>A',\n",
       "  'hgvs_protein': 'p.Ser1565=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467990,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48467996-G-C',\n",
       "  'variant_hash': 'da3cfdb2e8861e77',\n",
       "  'vcf_string': '15\\t48467996\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467996G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4689C>G',\n",
       "  'hgvs_protein': 'p.Cys1563Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467996,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48467997-C-T',\n",
       "  'variant_hash': 'b3c7540650a0324f',\n",
       "  'vcf_string': '15\\t48467997\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48467997C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4688G>A',\n",
       "  'hgvs_protein': 'p.Cys1563Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48467997,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468001-A-G',\n",
       "  'variant_hash': '1237ab7696cc6eda',\n",
       "  'vcf_string': '15\\t48468001\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468001A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4684T>C',\n",
       "  'hgvs_protein': 'p.Cys1562Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468001,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468002-G-A',\n",
       "  'variant_hash': '84f9287dde95ce72',\n",
       "  'vcf_string': '15\\t48468002\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468002G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4683C>T',\n",
       "  'hgvs_protein': 'p.Ser1561=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468002,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48468020-A-G',\n",
       "  'variant_hash': '368dd5d2421f2d33',\n",
       "  'vcf_string': '15\\t48468020\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468020A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4665T>C',\n",
       "  'hgvs_protein': 'p.Val1555=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468020,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48468023-T-G',\n",
       "  'variant_hash': 'be3fcea41ca7546b',\n",
       "  'vcf_string': '15\\t48468023\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468023T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4662A>C',\n",
       "  'hgvs_protein': 'p.Gly1554=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468023,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48468035-G-A',\n",
       "  'variant_hash': '2d904cc046a53663',\n",
       "  'vcf_string': '15\\t48468035\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468035G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4650C>T',\n",
       "  'hgvs_protein': 'p.Ser1550=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468035,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468045-G-A',\n",
       "  'variant_hash': '4e1c4f8e04903e17',\n",
       "  'vcf_string': '15\\t48468045\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468045G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4640C>T',\n",
       "  'hgvs_protein': 'p.Thr1547Ile',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468045,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48468064-G-A',\n",
       "  'variant_hash': '75e08399fc954280',\n",
       "  'vcf_string': '15\\t48468064\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468064G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4621C>T',\n",
       "  'hgvs_protein': 'p.Arg1541Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468064,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48468065-A-C',\n",
       "  'variant_hash': '7cc0e2b3746e85cf',\n",
       "  'vcf_string': '15\\t48468065\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468065A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4620T>G',\n",
       "  'hgvs_protein': 'p.Pro1540=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468065,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468070-G-A',\n",
       "  'variant_hash': '8b0bb973ea6cc63b',\n",
       "  'vcf_string': '15\\t48468070\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468070G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4615C>T',\n",
       "  'hgvs_protein': 'p.Arg1539Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468070,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48468080-A-C',\n",
       "  'variant_hash': 'd243ff9adb8b5a3e',\n",
       "  'vcf_string': '15\\t48468080\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468080A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4605T>G',\n",
       "  'hgvs_protein': 'p.Tyr1535Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468080,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48468092-A-G',\n",
       "  'variant_hash': 'd8053905401d03a6',\n",
       "  'vcf_string': '15\\t48468092\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468092A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4593T>C',\n",
       "  'hgvs_protein': 'p.Ser1531=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468092,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468096-C-G',\n",
       "  'variant_hash': '268434c3977700a2',\n",
       "  'vcf_string': '15\\t48468096\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468096C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4589G>C',\n",
       "  'hgvs_protein': 'p.Arg1530Pro',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468096,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48468097-G-A',\n",
       "  'variant_hash': 'be3f1c9a58c39bd8',\n",
       "  'vcf_string': '15\\t48468097\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468097G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4588C>T',\n",
       "  'hgvs_protein': 'p.Arg1530Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468097,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468107-T-C',\n",
       "  'variant_hash': 'a81aba4d9ced414d',\n",
       "  'vcf_string': '15\\t48468107\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468107T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4583-5A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468107,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48468110-C-G',\n",
       "  'variant_hash': 'a3710a1951683540',\n",
       "  'vcf_string': '15\\t48468110\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468110C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4583-8G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468110,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48468113-T-C',\n",
       "  'variant_hash': '28428e4177e82e80',\n",
       "  'vcf_string': '15\\t48468113\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468113T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4583-11A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468113,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468114-A-G',\n",
       "  'variant_hash': '0dfa8507441984b1',\n",
       "  'vcf_string': '15\\t48468114\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468114A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4583-12T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468114,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468115-T-C',\n",
       "  'variant_hash': '989be7a1fffa60b1',\n",
       "  'vcf_string': '15\\t48468115\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468115T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4583-13A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468115,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468194-G-A',\n",
       "  'variant_hash': '18df21506df47cde',\n",
       "  'vcf_string': '15\\t48468194\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468194G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4583-92C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468194,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468348-T-C',\n",
       "  'variant_hash': 'fe6c41b61edf422f',\n",
       "  'vcf_string': '15\\t48468348\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468348T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4582+64A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468348,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48468411-C-T',\n",
       "  'variant_hash': 'ec51f34479475f00',\n",
       "  'vcf_string': '15\\t48468411\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468411C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4582+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468411,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468418-A-G',\n",
       "  'variant_hash': '4a7e6a672cca2ec2',\n",
       "  'vcf_string': '15\\t48468418\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468418A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4576T>C',\n",
       "  'hgvs_protein': 'p.Cys1526Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468418,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468425-T-A',\n",
       "  'variant_hash': '7d922de341e58f48',\n",
       "  'vcf_string': '15\\t48468425\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468425T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4569A>T',\n",
       "  'hgvs_protein': 'p.Arg1523=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468425,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468427-G-A',\n",
       "  'variant_hash': 'a9ea487841853a78',\n",
       "  'vcf_string': '15\\t48468427\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468427G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4567C>T',\n",
       "  'hgvs_protein': 'p.Arg1523Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468427,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468455-G-C',\n",
       "  'variant_hash': 'df04245f0016fde5',\n",
       "  'vcf_string': '15\\t48468455\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468455G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4539C>G',\n",
       "  'hgvs_protein': 'p.Cys1513Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468455,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468456-C-T',\n",
       "  'variant_hash': 'bd19a31c893d6c7a',\n",
       "  'vcf_string': '15\\t48468456\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468456C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4538G>A',\n",
       "  'hgvs_protein': 'p.Cys1513Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468456,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48468457-A-C',\n",
       "  'variant_hash': '43f28b67aed6306e',\n",
       "  'vcf_string': '15\\t48468457\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468457A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4537T>G',\n",
       "  'hgvs_protein': 'p.Cys1513Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468457,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468462-C-T',\n",
       "  'variant_hash': '7025873c7615cd03',\n",
       "  'vcf_string': '15\\t48468462\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468462C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4532G>A',\n",
       "  'hgvs_protein': 'p.Cys1511Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468462,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468463-A-G',\n",
       "  'variant_hash': '2b2caf359d5e9fd8',\n",
       "  'vcf_string': '15\\t48468463\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468463A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4531T>C',\n",
       "  'hgvs_protein': 'p.Cys1511Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468463,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468464-G-A',\n",
       "  'variant_hash': '1452080d3d212673',\n",
       "  'vcf_string': '15\\t48468464\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468464G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4530C>T',\n",
       "  'hgvs_protein': 'p.Ile1510=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468464,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468467-A-G',\n",
       "  'variant_hash': 'b16dd4680d19e758',\n",
       "  'vcf_string': '15\\t48468467\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468467A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4527T>C',\n",
       "  'hgvs_protein': 'p.Tyr1509=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468467,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468468-T-C',\n",
       "  'variant_hash': '2cfaf5b9d145eb09',\n",
       "  'vcf_string': '15\\t48468468\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468468T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4526A>G',\n",
       "  'hgvs_protein': 'p.Tyr1509Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468468,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48468474-C-T',\n",
       "  'variant_hash': 'a1ffdccc02cf75f3',\n",
       "  'vcf_string': '15\\t48468474\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468474C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4520G>A',\n",
       "  'hgvs_protein': 'p.Gly1507Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468474,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468482-G-A',\n",
       "  'variant_hash': 'aa165c0dc4a384d7',\n",
       "  'vcf_string': '15\\t48468482\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468482G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4512C>T',\n",
       "  'hgvs_protein': 'p.Asn1504=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468482,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468489-C-T',\n",
       "  'variant_hash': 'b2750b9f383d6700',\n",
       "  'vcf_string': '15\\t48468489\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468489C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4505G>A',\n",
       "  'hgvs_protein': 'p.Cys1502Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468489,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468490-A-G',\n",
       "  'variant_hash': '4299d47deff80bf5',\n",
       "  'vcf_string': '15\\t48468490\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468490A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4504T>C',\n",
       "  'hgvs_protein': 'p.Cys1502Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468490,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468491-G-A',\n",
       "  'variant_hash': 'f5f85d3d07c0c253',\n",
       "  'vcf_string': '15\\t48468491\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468491G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4503C>T',\n",
       "  'hgvs_protein': 'p.Asn1501=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468491,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468503-G-C',\n",
       "  'variant_hash': 'bae116545302f1b7',\n",
       "  'vcf_string': '15\\t48468503\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468503G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4491C>G',\n",
       "  'hgvs_protein': 'p.Cys1497Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468503,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48468504-C-T',\n",
       "  'variant_hash': 'b7ac43c90e048763',\n",
       "  'vcf_string': '15\\t48468504\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468504C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4490G>A',\n",
       "  'hgvs_protein': 'p.Cys1497Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468504,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468506-C-T',\n",
       "  'variant_hash': '92f9f3093c3cc9b1',\n",
       "  'vcf_string': '15\\t48468506\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468506C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4488G>A',\n",
       "  'hgvs_protein': 'p.Thr1496=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468506,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468509-G-A',\n",
       "  'variant_hash': '770bafd0537512e6',\n",
       "  'vcf_string': '15\\t48468509\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468509G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4485C>T',\n",
       "  'hgvs_protein': 'p.Thr1495=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468509,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468522-C-A',\n",
       "  'variant_hash': 'a4812ff9f07d510c',\n",
       "  'vcf_string': '15\\t48468522\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468522C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4472G>T',\n",
       "  'hgvs_protein': 'p.Cys1491Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468522,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468526-C-T',\n",
       "  'variant_hash': 'af41e58771851ee9',\n",
       "  'vcf_string': '15\\t48468526\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468526C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4468G>A',\n",
       "  'hgvs_protein': 'p.Glu1490Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468526,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468527-A-T',\n",
       "  'variant_hash': '72ded8e896e27b1c',\n",
       "  'vcf_string': '15\\t48468527\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468527A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4467T>A',\n",
       "  'hgvs_protein': 'p.Asn1489Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468527,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468534-T-C',\n",
       "  'variant_hash': '215b9646262fcbfc',\n",
       "  'vcf_string': '15\\t48468534\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468534T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4460A>G',\n",
       "  'hgvs_protein': 'p.Asp1487Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468534,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468542-C-T',\n",
       "  'variant_hash': 'a24a35451c082939',\n",
       "  'vcf_string': '15\\t48468542\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468542C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4460-8G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468542,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48468543-G-A',\n",
       "  'variant_hash': '8a819441c13e28b5',\n",
       "  'vcf_string': '15\\t48468543\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468543G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4460-9C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468543,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48468553-T-C',\n",
       "  'variant_hash': '286c2d37811b5b10',\n",
       "  'vcf_string': '15\\t48468553\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468553T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4460-19A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468553,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48468700-G-A',\n",
       "  'variant_hash': '3ac45391ebef82e6',\n",
       "  'vcf_string': '15\\t48468700\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48468700G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4460-166C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48468700,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48470634-C-T',\n",
       "  'variant_hash': 'dd3642ad2b8070ed',\n",
       "  'vcf_string': '15\\t48470634\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470634C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4459G>A',\n",
       "  'hgvs_protein': 'p.Asp1487Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470634,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48470647-G-A',\n",
       "  'variant_hash': 'd2d9281ad517dd1a',\n",
       "  'vcf_string': '15\\t48470647\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470647G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4446C>T',\n",
       "  'hgvs_protein': 'p.Gly1482=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470647,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470661-G-A',\n",
       "  'variant_hash': '187190c1460aa766',\n",
       "  'vcf_string': '15\\t48470661\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470661G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4432C>T',\n",
       "  'hgvs_protein': 'p.Leu1478=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470661,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470665-G-A',\n",
       "  'variant_hash': 'f5e006452c2d2ce6',\n",
       "  'vcf_string': '15\\t48470665\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470665G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4428C>T',\n",
       "  'hgvs_protein': 'p.Tyr1476=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470665,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470679-A-G',\n",
       "  'variant_hash': '3bd221f9b6bb0a46',\n",
       "  'vcf_string': '15\\t48470679\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470679A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4414T>C',\n",
       "  'hgvs_protein': 'p.Cys1472Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470679,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48470680-C-T',\n",
       "  'variant_hash': '2f5a5f0574a60644',\n",
       "  'vcf_string': '15\\t48470680\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470680C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4413G>A',\n",
       "  'hgvs_protein': 'p.Glu1471=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470680,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470684-C-T',\n",
       "  'variant_hash': '0d05f8dba29601ae',\n",
       "  'vcf_string': '15\\t48470684\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470684C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4409G>A',\n",
       "  'hgvs_protein': 'p.Cys1470Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470684,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48470689-G-A',\n",
       "  'variant_hash': 'e0d87ee7367ffb59',\n",
       "  'vcf_string': '15\\t48470689\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470689G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4404C>T',\n",
       "  'hgvs_protein': 'p.Phe1468=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470689,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470692-C-A',\n",
       "  'variant_hash': '786723d638967d35',\n",
       "  'vcf_string': '15\\t48470692\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470692C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4401G>T',\n",
       "  'hgvs_protein': 'p.Leu1467=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470692,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470711-C-G',\n",
       "  'variant_hash': '354bd4df11a4b5da',\n",
       "  'vcf_string': '15\\t48470711\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470711C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4382G>C',\n",
       "  'hgvs_protein': 'p.Cys1461Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470711,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470711-C-A',\n",
       "  'variant_hash': '09fcd24a71a5bcb4',\n",
       "  'vcf_string': '15\\t48470711\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470711C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4382G>T',\n",
       "  'hgvs_protein': 'p.Cys1461Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470711,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470722-G-A',\n",
       "  'variant_hash': 'd9d59f719389c076',\n",
       "  'vcf_string': '15\\t48470722\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470722G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4371C>T',\n",
       "  'hgvs_protein': 'p.Val1457=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470722,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470726-C-T',\n",
       "  'variant_hash': '7aa35ef36b8f30e8',\n",
       "  'vcf_string': '15\\t48470726\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470726C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4367G>A',\n",
       "  'hgvs_protein': 'p.Cys1456Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470726,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48470726-C-G',\n",
       "  'variant_hash': 'b92a37bf03311cc2',\n",
       "  'vcf_string': '15\\t48470726\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470726C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4367G>C',\n",
       "  'hgvs_protein': 'p.Cys1456Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470726,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470726-C-A',\n",
       "  'variant_hash': 'e3ea3b50c3dcc74c',\n",
       "  'vcf_string': '15\\t48470726\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470726C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4367G>T',\n",
       "  'hgvs_protein': 'p.Cys1456Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470726,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48470734-C-T',\n",
       "  'variant_hash': 'a7a7cf9ecda9f80c',\n",
       "  'vcf_string': '15\\t48470734\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470734C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4359G>A',\n",
       "  'hgvs_protein': 'p.Pro1453=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470734,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470744-C-A',\n",
       "  'variant_hash': '017d757955514e93',\n",
       "  'vcf_string': '15\\t48470744\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470744C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4349G>T',\n",
       "  'hgvs_protein': 'p.Cys1450Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470744,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470745-A-G',\n",
       "  'variant_hash': '7129571835b76752',\n",
       "  'vcf_string': '15\\t48470745\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470745A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4348T>C',\n",
       "  'hgvs_protein': 'p.Cys1450Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470745,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470757-C-T',\n",
       "  'variant_hash': '698357c052284d40',\n",
       "  'vcf_string': '15\\t48470757\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470757C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470757,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470758-T-C',\n",
       "  'variant_hash': '8fa365f1b44ba706',\n",
       "  'vcf_string': '15\\t48470758\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470758T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-2A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470758,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48470763-G-A',\n",
       "  'variant_hash': '45ff0e3eb017e4ec',\n",
       "  'vcf_string': '15\\t48470763\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470763G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-7C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470763,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470765-G-A',\n",
       "  'variant_hash': 'df1dfde3cfb7a8f5',\n",
       "  'vcf_string': '15\\t48470765\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470765G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-9C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470765,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470768-G-A',\n",
       "  'variant_hash': '5efc87a18aa7d890',\n",
       "  'vcf_string': '15\\t48470768\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470768G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-12C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470768,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470770-A-G',\n",
       "  'variant_hash': '0fd3a2acfd310455',\n",
       "  'vcf_string': '15\\t48470770\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470770A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-14T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470770,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48470811-A-T',\n",
       "  'variant_hash': 'dd85062938ec40f0',\n",
       "  'vcf_string': '15\\t48470811\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48470811A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4337-55T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48470811,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48472544-C-A',\n",
       "  'variant_hash': 'fcef3dfcc9cc0c9c',\n",
       "  'vcf_string': '15\\t48472544\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472544C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4336+7G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472544,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472551-C-T',\n",
       "  'variant_hash': 'eeaca513de416074',\n",
       "  'vcf_string': '15\\t48472551\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472551C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4336G>A',\n",
       "  'hgvs_protein': 'p.Asp1446Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472551,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472556-C-T',\n",
       "  'variant_hash': 'bdd7d7e8d6133f4b',\n",
       "  'vcf_string': '15\\t48472556\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472556C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4331G>A',\n",
       "  'hgvs_protein': 'p.Cys1444Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472556,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472557-A-T',\n",
       "  'variant_hash': '83127a9bea85466a',\n",
       "  'vcf_string': '15\\t48472557\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472557A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4330T>A',\n",
       "  'hgvs_protein': 'p.Cys1444Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472557,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48472567-G-A',\n",
       "  'variant_hash': 'd7a1968181c35ded',\n",
       "  'vcf_string': '15\\t48472567\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472567G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4320C>T',\n",
       "  'hgvs_protein': 'p.Asp1440=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472567,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472579-C-T',\n",
       "  'variant_hash': '89431db5d1487e1d',\n",
       "  'vcf_string': '15\\t48472579\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472579C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4308G>A',\n",
       "  'hgvs_protein': 'p.Val1436=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472579,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472594-G-T',\n",
       "  'variant_hash': '02b6bc30cf4de63e',\n",
       "  'vcf_string': '15\\t48472594\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472594G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4293C>A',\n",
       "  'hgvs_protein': 'p.Cys1431Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472594,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48472594-G-C',\n",
       "  'variant_hash': 'b706597d30e370e5',\n",
       "  'vcf_string': '15\\t48472594\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472594G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4293C>G',\n",
       "  'hgvs_protein': 'p.Cys1431Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472594,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472594-G-A',\n",
       "  'variant_hash': '9d2d3a8bd3719861',\n",
       "  'vcf_string': '15\\t48472594\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472594G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4293C>T',\n",
       "  'hgvs_protein': 'p.Cys1431=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472594,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472595-C-T',\n",
       "  'variant_hash': '7b5864c4b7a0931d',\n",
       "  'vcf_string': '15\\t48472595\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472595C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4292G>A',\n",
       "  'hgvs_protein': 'p.Cys1431Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472595,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48472601-C-T',\n",
       "  'variant_hash': '5a912fe76dae5e5d',\n",
       "  'vcf_string': '15\\t48472601\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472601C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4286G>A',\n",
       "  'hgvs_protein': 'p.Cys1429Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472601,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472606-G-A',\n",
       "  'variant_hash': '37912a3ee88700c0',\n",
       "  'vcf_string': '15\\t48472606\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472606G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4281C>T',\n",
       "  'hgvs_protein': 'p.Tyr1427=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472606,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472607-T-C',\n",
       "  'variant_hash': '4cda1b9916668174',\n",
       "  'vcf_string': '15\\t48472607\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472607T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4280A>G',\n",
       "  'hgvs_protein': 'p.Tyr1427Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472607,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48472615-T-C',\n",
       "  'variant_hash': 'dd2b73298a98dc2a',\n",
       "  'vcf_string': '15\\t48472615\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472615T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4272A>G',\n",
       "  'hgvs_protein': 'p.Pro1424=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472615,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472618-T-A',\n",
       "  'variant_hash': 'c8bbd29267327747',\n",
       "  'vcf_string': '15\\t48472618\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472618T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4269A>T',\n",
       "  'hgvs_protein': 'p.Ala1423=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472618,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472628-C-T',\n",
       "  'variant_hash': '3af43e133de1e8ca',\n",
       "  'vcf_string': '15\\t48472628\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472628C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4259G>A',\n",
       "  'hgvs_protein': 'p.Cys1420Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472628,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472628-C-A',\n",
       "  'variant_hash': 'a4be30c986e2b604',\n",
       "  'vcf_string': '15\\t48472628\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472628C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4259G>T',\n",
       "  'hgvs_protein': 'p.Cys1420Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472628,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472632-G-A',\n",
       "  'variant_hash': 'e810b1d218d1409a',\n",
       "  'vcf_string': '15\\t48472632\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472632G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4255C>T',\n",
       "  'hgvs_protein': 'p.Gln1419Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472632,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48472644-A-G',\n",
       "  'variant_hash': 'bd25538d192561c5',\n",
       "  'vcf_string': '15\\t48472644\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472644A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4243T>C',\n",
       "  'hgvs_protein': 'p.Cys1415Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472644,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48472654-G-A',\n",
       "  'variant_hash': 'dac1dab9815937cb',\n",
       "  'vcf_string': '15\\t48472654\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472654G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4233C>T',\n",
       "  'hgvs_protein': 'p.Asn1411=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472654,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472665-A-G',\n",
       "  'variant_hash': '4c77ec6f5d437e70',\n",
       "  'vcf_string': '15\\t48472665\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472665A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4222T>C',\n",
       "  'hgvs_protein': 'p.Cys1408Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472665,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48472669-A-G',\n",
       "  'variant_hash': '8373eb07f5e36b11',\n",
       "  'vcf_string': '15\\t48472669\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472669A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4218T>C',\n",
       "  'hgvs_protein': 'p.Asp1406=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472669,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472670-T-C',\n",
       "  'variant_hash': '8bcc6e02f0f756e3',\n",
       "  'vcf_string': '15\\t48472670\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472670T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4217A>G',\n",
       "  'hgvs_protein': 'p.Asp1406Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472670,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48472677-C-T',\n",
       "  'variant_hash': '7afa5916878c3c2b',\n",
       "  'vcf_string': '15\\t48472677\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472677C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4211-1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472677,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48472684-C-G',\n",
       "  'variant_hash': 'c58b89ca985cf326',\n",
       "  'vcf_string': '15\\t48472684\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472684C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4211-8G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472684,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472694-C-T',\n",
       "  'variant_hash': 'd9f0d24ee8695958',\n",
       "  'vcf_string': '15\\t48472694\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472694C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4211-18G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472694,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48472745-A-G',\n",
       "  'variant_hash': '4707ce67185f4d08',\n",
       "  'vcf_string': '15\\t48472745\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472745A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4211-69T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472745,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48472836-A-C',\n",
       "  'variant_hash': 'befefdf5fec7a173',\n",
       "  'vcf_string': '15\\t48472836\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472836A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4211-160T>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472836,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48472847-T-G',\n",
       "  'variant_hash': '12ef333ae67df609',\n",
       "  'vcf_string': '15\\t48472847\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48472847T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4211-171A>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48472847,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48474053-G-C',\n",
       "  'variant_hash': '050e09c5844b22df',\n",
       "  'vcf_string': '15\\t48474053\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474053G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4210+202C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474053,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48474244-C-T',\n",
       "  'variant_hash': '657e8d6e68e755d6',\n",
       "  'vcf_string': '15\\t48474244\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474244C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4210+11G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474244,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474245-G-A',\n",
       "  'variant_hash': 'd80d70ef4e5e123e',\n",
       "  'vcf_string': '15\\t48474245\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474245G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4210+10C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474245,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48474254-C-T',\n",
       "  'variant_hash': 'c44d24267b4a6459',\n",
       "  'vcf_string': '15\\t48474254\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474254C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4210+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474254,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474260-C-T',\n",
       "  'variant_hash': '33d3c462276a2236',\n",
       "  'vcf_string': '15\\t48474260\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474260C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4205G>A',\n",
       "  'hgvs_protein': 'p.Cys1402Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474260,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474271-A-G',\n",
       "  'variant_hash': '7e638923d54847bd',\n",
       "  'vcf_string': '15\\t48474271\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474271A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4194T>C',\n",
       "  'hgvs_protein': 'p.Asp1398=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474271,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474280-G-A',\n",
       "  'variant_hash': '40010e4301501064',\n",
       "  'vcf_string': '15\\t48474280\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474280G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4185C>T',\n",
       "  'hgvs_protein': 'p.Tyr1395=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474280,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474286-T-C',\n",
       "  'variant_hash': '526333ac5b0daa34',\n",
       "  'vcf_string': '15\\t48474286\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474286T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4179A>G',\n",
       "  'hgvs_protein': 'p.Glu1393=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474286,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474289-C-T',\n",
       "  'variant_hash': 'd551a54a00279111',\n",
       "  'vcf_string': '15\\t48474289\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474289C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4176G>A',\n",
       "  'hgvs_protein': 'p.Lys1392=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474289,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48474293-C-T',\n",
       "  'variant_hash': '9dbc38d6c971c956',\n",
       "  'vcf_string': '15\\t48474293\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474293C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4172G>A',\n",
       "  'hgvs_protein': 'p.Cys1391Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474293,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474295-C-T',\n",
       "  'variant_hash': '8f4289a392cf7e99',\n",
       "  'vcf_string': '15\\t48474295\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474295C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4170G>A',\n",
       "  'hgvs_protein': 'p.Leu1390=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474295,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474299-C-T',\n",
       "  'variant_hash': '80eabf9050578b92',\n",
       "  'vcf_string': '15\\t48474299\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474299C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4166G>A',\n",
       "  'hgvs_protein': 'p.Cys1389Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474299,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474299-C-G',\n",
       "  'variant_hash': 'b10f74a0442d5611',\n",
       "  'vcf_string': '15\\t48474299\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474299C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4166G>C',\n",
       "  'hgvs_protein': 'p.Cys1389Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474299,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474301-G-T',\n",
       "  'variant_hash': '69d5d9c5ab06157f',\n",
       "  'vcf_string': '15\\t48474301\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474301G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4164C>A',\n",
       "  'hgvs_protein': 'p.Arg1388=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474301,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474304-G-A',\n",
       "  'variant_hash': 'b1a26523477ba210',\n",
       "  'vcf_string': '15\\t48474304\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474304G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4161C>T',\n",
       "  'hgvs_protein': 'p.Tyr1387=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474304,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474310-T-A',\n",
       "  'variant_hash': 'af580186148342f2',\n",
       "  'vcf_string': '15\\t48474310\\t.\\tT\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474310T>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4155A>T',\n",
       "  'hgvs_protein': 'p.Gly1385=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474310,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474343-G-A',\n",
       "  'variant_hash': '0a3ba8356ad9bb91',\n",
       "  'vcf_string': '15\\t48474343\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474343G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4122C>T',\n",
       "  'hgvs_protein': 'p.Cys1374=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474343,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474344-C-T',\n",
       "  'variant_hash': '9020c787b6e51206',\n",
       "  'vcf_string': '15\\t48474344\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474344C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4121G>A',\n",
       "  'hgvs_protein': 'p.Cys1374Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474344,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474345-A-C',\n",
       "  'variant_hash': 'dff6dd9a88b380ba',\n",
       "  'vcf_string': '15\\t48474345\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474345A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4120T>G',\n",
       "  'hgvs_protein': 'p.Cys1374Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474345,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474365-C-T',\n",
       "  'variant_hash': '7c7c5bd5d851a1e0',\n",
       "  'vcf_string': '15\\t48474365\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474365C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4100G>A',\n",
       "  'hgvs_protein': 'p.Cys1367Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474365,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474369-C-T',\n",
       "  'variant_hash': 'a6459e2606859a11',\n",
       "  'vcf_string': '15\\t48474369\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474369C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4096G>A',\n",
       "  'hgvs_protein': 'p.Glu1366Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474369,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474376-A-G',\n",
       "  'variant_hash': 'b18099292f9f974b',\n",
       "  'vcf_string': '15\\t48474376\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474376A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4089T>C',\n",
       "  'hgvs_protein': 'p.Asp1363=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474376,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474385-A-G',\n",
       "  'variant_hash': '300e6f879626d974',\n",
       "  'vcf_string': '15\\t48474385\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474385A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4088-8T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474385,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474394-A-G',\n",
       "  'variant_hash': '8a4a0cc95043d018',\n",
       "  'vcf_string': '15\\t48474394\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474394A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4088-17T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474394,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474533-C-T',\n",
       "  'variant_hash': '0f9144e6f16bc7df',\n",
       "  'vcf_string': '15\\t48474533\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474533C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4082G>A',\n",
       "  'hgvs_protein': 'p.Cys1361Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474533,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474534-A-G',\n",
       "  'variant_hash': 'e3781552cd591371',\n",
       "  'vcf_string': '15\\t48474534\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474534A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4081T>C',\n",
       "  'hgvs_protein': 'p.Cys1361Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474534,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48474550-A-T',\n",
       "  'variant_hash': 'c7ca878bf085a254',\n",
       "  'vcf_string': '15\\t48474550\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474550A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4065T>A',\n",
       "  'hgvs_protein': 'p.Ile1355=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474550,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474554-C-T',\n",
       "  'variant_hash': '30896f8b54ac6007',\n",
       "  'vcf_string': '15\\t48474554\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474554C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4061G>A',\n",
       "  'hgvs_protein': 'p.Trp1354Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474554,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474556-C-A',\n",
       "  'variant_hash': '3fc051fc97ea7b9a',\n",
       "  'vcf_string': '15\\t48474556\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474556C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4059G>T',\n",
       "  'hgvs_protein': 'p.Gly1353=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474556,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474559-G-A',\n",
       "  'variant_hash': 'bab5bd663f494379',\n",
       "  'vcf_string': '15\\t48474559\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474559G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4056C>T',\n",
       "  'hgvs_protein': 'p.Pro1352=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474559,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474565-G-T',\n",
       "  'variant_hash': 'd2896a16207513f2',\n",
       "  'vcf_string': '15\\t48474565\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474565G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4050C>A',\n",
       "  'hgvs_protein': 'p.Cys1350Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474565,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474565-G-A',\n",
       "  'variant_hash': '87bb30bd802fbff3',\n",
       "  'vcf_string': '15\\t48474565\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474565G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4050C>T',\n",
       "  'hgvs_protein': 'p.Cys1350=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474565,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474566-C-A',\n",
       "  'variant_hash': '7d8465993c1b22aa',\n",
       "  'vcf_string': '15\\t48474566\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474566C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4049G>T',\n",
       "  'hgvs_protein': 'p.Cys1350Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474566,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48474567-A-G',\n",
       "  'variant_hash': '9f199e83805c254e',\n",
       "  'vcf_string': '15\\t48474567\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474567A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4048T>C',\n",
       "  'hgvs_protein': 'p.Cys1350Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474567,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474572-C-T',\n",
       "  'variant_hash': 'e38f7b42631f7de8',\n",
       "  'vcf_string': '15\\t48474572\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474572C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4043G>A',\n",
       "  'hgvs_protein': 'p.Cys1348Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474572,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474580-G-A',\n",
       "  'variant_hash': '21885cc3950da826',\n",
       "  'vcf_string': '15\\t48474580\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474580G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4035C>T',\n",
       "  'hgvs_protein': 'p.Ser1345=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474580,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474584-C-T',\n",
       "  'variant_hash': 'a0607e98129413b7',\n",
       "  'vcf_string': '15\\t48474584\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474584C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4031G>A',\n",
       "  'hgvs_protein': 'p.Gly1344Glu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474584,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474589-T-C',\n",
       "  'variant_hash': 'cb8b41494f3a37fd',\n",
       "  'vcf_string': '15\\t48474589\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474589T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4026A>G',\n",
       "  'hgvs_protein': 'p.Thr1342=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474589,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474592-A-G',\n",
       "  'variant_hash': 'e80d37bae20c5ebb',\n",
       "  'vcf_string': '15\\t48474592\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474592A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4023T>C',\n",
       "  'hgvs_protein': 'p.Asn1341=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474592,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474599-C-G',\n",
       "  'variant_hash': '7ab684589a8d878a',\n",
       "  'vcf_string': '15\\t48474599\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474599C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4016G>C',\n",
       "  'hgvs_protein': 'p.Cys1339Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474599,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474601-T-C',\n",
       "  'variant_hash': '23587f4d12b0db17',\n",
       "  'vcf_string': '15\\t48474601\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474601T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.4014A>G',\n",
       "  'hgvs_protein': 'p.Val1338=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474601,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474638-C-T',\n",
       "  'variant_hash': 'b245ab634a33b7f0',\n",
       "  'vcf_string': '15\\t48474638\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474638C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3977G>A',\n",
       "  'hgvs_protein': 'p.Cys1326Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474638,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48474640-T-C',\n",
       "  'variant_hash': 'fe71d98cb52a2d7d',\n",
       "  'vcf_string': '15\\t48474640\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474640T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3975A>G',\n",
       "  'hgvs_protein': 'p.Glu1325=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474640,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48474646-G-A',\n",
       "  'variant_hash': '94212aa632c44fe8',\n",
       "  'vcf_string': '15\\t48474646\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474646G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3969C>T',\n",
       "  'hgvs_protein': 'p.Ile1323=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474646,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48474658-A-G',\n",
       "  'variant_hash': '2c0744c49b247d40',\n",
       "  'vcf_string': '15\\t48474658\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48474658A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3965-8T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48474658,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48481636-T-C',\n",
       "  'variant_hash': '5f1d43c7ca5b222d',\n",
       "  'vcf_string': '15\\t48481636\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481636T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3964+19A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481636,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481643-G-A',\n",
       "  'variant_hash': 'f4a44890202d1bd1',\n",
       "  'vcf_string': '15\\t48481643\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481643G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3964+12C>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481643,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481655-C-T',\n",
       "  'variant_hash': '08293a705a9e6c78',\n",
       "  'vcf_string': '15\\t48481655\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481655C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3964G>A',\n",
       "  'hgvs_protein': 'p.Asp1322Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481655,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48481656-T-C',\n",
       "  'variant_hash': '81362f0c19c82c5b',\n",
       "  'vcf_string': '15\\t48481656\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481656T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3963A>G',\n",
       "  'hgvs_protein': 'p.Thr1321=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481656,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48481661-A-G',\n",
       "  'variant_hash': '3c183e206188c6b1',\n",
       "  'vcf_string': '15\\t48481661\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481661A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3958T>C',\n",
       "  'hgvs_protein': 'p.Cys1320Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481661,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48481668-T-C',\n",
       "  'variant_hash': 'd1935cd51482ab9e',\n",
       "  'vcf_string': '15\\t48481668\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481668T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3951A>G',\n",
       "  'hgvs_protein': 'p.Lys1317=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481668,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481680-G-A',\n",
       "  'variant_hash': 'ab52560bff759f20',\n",
       "  'vcf_string': '15\\t48481680\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481680G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3939C>T',\n",
       "  'hgvs_protein': 'p.Gly1313=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481680,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481687-T-C',\n",
       "  'variant_hash': 'ba23853330c2f0ab',\n",
       "  'vcf_string': '15\\t48481687\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481687T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3932A>G',\n",
       "  'hgvs_protein': 'p.Tyr1311Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481687,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48481690-C-T',\n",
       "  'variant_hash': '67eb5593f4a57536',\n",
       "  'vcf_string': '15\\t48481690\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481690C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3929G>A',\n",
       "  'hgvs_protein': 'p.Gly1310Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481690,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48481699-C-T',\n",
       "  'variant_hash': '5a6cf978f98a5b29',\n",
       "  'vcf_string': '15\\t48481699\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481699C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3920G>A',\n",
       "  'hgvs_protein': 'p.Cys1307Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481699,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48481713-T-C',\n",
       "  'variant_hash': 'c0963ed4dd769682',\n",
       "  'vcf_string': '15\\t48481713\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481713T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3906A>G',\n",
       "  'hgvs_protein': 'p.Ser1302=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481713,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481716-G-A',\n",
       "  'variant_hash': 'd14b76e86e12deb3',\n",
       "  'vcf_string': '15\\t48481716\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481716G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3903C>T',\n",
       "  'hgvs_protein': 'p.Gly1301=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481716,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481719-T-C',\n",
       "  'variant_hash': '995759e741b144c4',\n",
       "  'vcf_string': '15\\t48481719\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481719T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3900A>G',\n",
       "  'hgvs_protein': 'p.Lys1300=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481719,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48481722-C-T',\n",
       "  'variant_hash': '561d69c3781bd895',\n",
       "  'vcf_string': '15\\t48481722\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481722C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3897G>A',\n",
       "  'hgvs_protein': 'p.Thr1299=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481722,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481722-C-G',\n",
       "  'variant_hash': 'b1151f699d58aac2',\n",
       "  'vcf_string': '15\\t48481722\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481722C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3897G>C',\n",
       "  'hgvs_protein': 'p.Thr1299=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481722,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481723-G-A',\n",
       "  'variant_hash': '36e08338f44fdd16',\n",
       "  'vcf_string': '15\\t48481723\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481723G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3896C>T',\n",
       "  'hgvs_protein': 'p.Thr1299Met',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481723,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48481729-T-C',\n",
       "  'variant_hash': 'c3f7afeef47865e2',\n",
       "  'vcf_string': '15\\t48481729\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481729T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3890A>G',\n",
       "  'hgvs_protein': 'p.Glu1297Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481729,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48481733-A-G',\n",
       "  'variant_hash': '08270c3712f6fe54',\n",
       "  'vcf_string': '15\\t48481733\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481733A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3886T>C',\n",
       "  'hgvs_protein': 'p.Cys1296Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481733,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48481743-T-G',\n",
       "  'variant_hash': 'b20badec5d64a977',\n",
       "  'vcf_string': '15\\t48481743\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481743T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3876A>C',\n",
       "  'hgvs_protein': 'p.Leu1292=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481743,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48481768-C-T',\n",
       "  'variant_hash': 'a20dd887bd4d6051',\n",
       "  'vcf_string': '15\\t48481768\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48481768C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3851G>A',\n",
       "  'hgvs_protein': 'p.Cys1284Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48481768,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48483798-A-G',\n",
       "  'variant_hash': '70fc8e021d0fb898',\n",
       "  'vcf_string': '15\\t48483798\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483798A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3838+20T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483798,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483816-A-G',\n",
       "  'variant_hash': 'b5ecc26e2e6bb2e1',\n",
       "  'vcf_string': '15\\t48483816\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483816A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3838+2T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483816,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48483818-C-T',\n",
       "  'variant_hash': 'dfc24d425aa00fb7',\n",
       "  'vcf_string': '15\\t48483818\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483818C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3838G>A',\n",
       "  'hgvs_protein': 'p.Asp1280Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483818,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48483822-A-G',\n",
       "  'variant_hash': '0fa825827fa391c0',\n",
       "  'vcf_string': '15\\t48483822\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483822A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3834T>C',\n",
       "  'hgvs_protein': 'p.Cys1278=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483822,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483822-A-C',\n",
       "  'variant_hash': '52c00d9adc0b28f4',\n",
       "  'vcf_string': '15\\t48483822\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483822A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3834T>G',\n",
       "  'hgvs_protein': 'p.Cys1278Trp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483822,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48483823-C-G',\n",
       "  'variant_hash': '9f3f5959d91b4789',\n",
       "  'vcf_string': '15\\t48483823\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483823C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3833G>C',\n",
       "  'hgvs_protein': 'p.Cys1278Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483823,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48483855-A-G',\n",
       "  'variant_hash': 'cdb3146e25115cb5',\n",
       "  'vcf_string': '15\\t48483855\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483855A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3801T>C',\n",
       "  'hgvs_protein': 'p.Asp1267=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483855,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483858-A-G',\n",
       "  'variant_hash': '5b5bb501f476b668',\n",
       "  'vcf_string': '15\\t48483858\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483858A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3798T>C',\n",
       "  'hgvs_protein': 'p.Tyr1266=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483858,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48483862-C-T',\n",
       "  'variant_hash': '61e1c75811da3038',\n",
       "  'vcf_string': '15\\t48483862\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483862C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3794G>A',\n",
       "  'hgvs_protein': 'p.Cys1265Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483862,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48483862-C-G',\n",
       "  'variant_hash': '9025c7bb1071c057',\n",
       "  'vcf_string': '15\\t48483862\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483862C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3794G>C',\n",
       "  'hgvs_protein': 'p.Cys1265Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483862,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48483876-C-T',\n",
       "  'variant_hash': 'ab86c442d4b430e0',\n",
       "  'vcf_string': '15\\t48483876\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483876C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3780G>A',\n",
       "  'hgvs_protein': 'p.Glu1260=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483876,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483878-C-A',\n",
       "  'variant_hash': 'd807716d42032a8b',\n",
       "  'vcf_string': '15\\t48483878\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483878C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3778G>T',\n",
       "  'hgvs_protein': 'p.Glu1260Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483878,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48483879-T-G',\n",
       "  'variant_hash': 'e878524b99513ded',\n",
       "  'vcf_string': '15\\t48483879\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483879T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3777A>C',\n",
       "  'hgvs_protein': 'p.Gly1259=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483879,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483894-G-A',\n",
       "  'variant_hash': 'f5cececf06bd40ce',\n",
       "  'vcf_string': '15\\t48483894\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483894G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3762C>T',\n",
       "  'hgvs_protein': 'p.Cys1254=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483894,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483899-G-A',\n",
       "  'variant_hash': 'c586d6fa035db753',\n",
       "  'vcf_string': '15\\t48483899\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483899G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3757C>T',\n",
       "  'hgvs_protein': 'p.Gln1253Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483899,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48483900-A-C',\n",
       "  'variant_hash': '6acc17e14f5e7bd8',\n",
       "  'vcf_string': '15\\t48483900\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483900A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3756T>G',\n",
       "  'hgvs_protein': 'p.Gly1252=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483900,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483921-A-G',\n",
       "  'variant_hash': '1f3d9b9c128a846d',\n",
       "  'vcf_string': '15\\t48483921\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483921A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3735T>C',\n",
       "  'hgvs_protein': 'p.Asn1245=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483921,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48483931-C-T',\n",
       "  'variant_hash': 'de1810c7335d4bed',\n",
       "  'vcf_string': '15\\t48483931\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483931C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3725G>A',\n",
       "  'hgvs_protein': 'p.Cys1242Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483931,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48483935-C-T',\n",
       "  'variant_hash': '3a84357d480c04ae',\n",
       "  'vcf_string': '15\\t48483935\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483935C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3721G>A',\n",
       "  'hgvs_protein': 'p.Glu1241Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483935,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48483939-G-A',\n",
       "  'variant_hash': 'e049ae6265663b1f',\n",
       "  'vcf_string': '15\\t48483939\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48483939G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3717C>T',\n",
       "  'hgvs_protein': 'p.Ile1239=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48483939,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48485374-C-T',\n",
       "  'variant_hash': 'ab482243510985a7',\n",
       "  'vcf_string': '15\\t48485374\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485374C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3712G>A',\n",
       "  'hgvs_protein': 'p.Asp1238Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485374,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48485379-C-T',\n",
       "  'variant_hash': 'ef6167125eabf530',\n",
       "  'vcf_string': '15\\t48485379\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485379C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3707G>A',\n",
       "  'hgvs_protein': 'p.Cys1236Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485379,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48485393-A-G',\n",
       "  'variant_hash': '7d9faed37c228e53',\n",
       "  'vcf_string': '15\\t48485393\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485393A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3693T>C',\n",
       "  'hgvs_protein': 'p.Pro1231=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485393,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48485397-A-G',\n",
       "  'variant_hash': 'be73a692b6ebc8fe',\n",
       "  'vcf_string': '15\\t48485397\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485397A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3689T>C',\n",
       "  'hgvs_protein': 'p.Met1230Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485397,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48485411-C-T',\n",
       "  'variant_hash': '00a9f1482021510e',\n",
       "  'vcf_string': '15\\t48485411\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485411C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3675G>A',\n",
       "  'hgvs_protein': 'p.Pro1225=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485411,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48485416-G-A',\n",
       "  'variant_hash': '84414afc1aa10a47',\n",
       "  'vcf_string': '15\\t48485416\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485416G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3670C>T',\n",
       "  'hgvs_protein': 'p.Gln1224Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485416,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48485430-T-C',\n",
       "  'variant_hash': '8641856aca41d151',\n",
       "  'vcf_string': '15\\t48485430\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485430T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3656A>G',\n",
       "  'hgvs_protein': 'p.Tyr1219Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485430,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48485432-G-A',\n",
       "  'variant_hash': '3dd87cf1fb9f90fb',\n",
       "  'vcf_string': '15\\t48485432\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485432G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3654C>T',\n",
       "  'hgvs_protein': 'p.Ser1218=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485432,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48485435-G-T',\n",
       "  'variant_hash': '4255efc81cca6ba2',\n",
       "  'vcf_string': '15\\t48485435\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485435G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3651C>A',\n",
       "  'hgvs_protein': 'p.Gly1217=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485435,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48485436-C-T',\n",
       "  'variant_hash': '9e42689289369b72',\n",
       "  'vcf_string': '15\\t48485436\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485436C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3650G>A',\n",
       "  'hgvs_protein': 'p.Gly1217Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485436,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48485463-C-A',\n",
       "  'variant_hash': 'e7298acd3ac0659a',\n",
       "  'vcf_string': '15\\t48485463\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485463C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3623G>T',\n",
       "  'hgvs_protein': 'p.Cys1208Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485463,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48485483-G-T',\n",
       "  'variant_hash': '26ed38a33cfb1694',\n",
       "  'vcf_string': '15\\t48485483\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485483G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3603C>A',\n",
       "  'hgvs_protein': 'p.Cys1201Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485483,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48485486-T-C',\n",
       "  'variant_hash': '6abe3fec2075c63b',\n",
       "  'vcf_string': '15\\t48485486\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485486T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3600A>G',\n",
       "  'hgvs_protein': 'p.Glu1200=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485486,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48485490-T-C',\n",
       "  'variant_hash': 'ee02c0326ccae0d3',\n",
       "  'vcf_string': '15\\t48485490\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485490T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3596A>G',\n",
       "  'hgvs_protein': 'p.Asp1199Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485490,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48485507-T-C',\n",
       "  'variant_hash': '65426e0cd7e8e82d',\n",
       "  'vcf_string': '15\\t48485507\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48485507T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3590-11A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48485507,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487075-C-T',\n",
       "  'variant_hash': '1a46642266f28e22',\n",
       "  'vcf_string': '15\\t48487075\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487075C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3589G>A',\n",
       "  'hgvs_protein': 'p.Asp1197Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487075,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48487085-T-C',\n",
       "  'variant_hash': '851d32530ca3e22f',\n",
       "  'vcf_string': '15\\t48487085\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487085T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3579A>G',\n",
       "  'hgvs_protein': 'p.Leu1193=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487085,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487091-A-G',\n",
       "  'variant_hash': 'a48057876d468e0d',\n",
       "  'vcf_string': '15\\t48487091\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487091A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3573T>C',\n",
       "  'hgvs_protein': 'p.Asp1191=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487091,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487094-G-A',\n",
       "  'variant_hash': '161c9cc9cb7cbbb3',\n",
       "  'vcf_string': '15\\t48487094\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487094G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3570C>T',\n",
       "  'hgvs_protein': 'p.Pro1190=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487094,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487106-G-C',\n",
       "  'variant_hash': '1c8f22bdffd33216',\n",
       "  'vcf_string': '15\\t48487106\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487106G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3558C>G',\n",
       "  'hgvs_protein': 'p.Tyr1186Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487106,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487107-T-C',\n",
       "  'variant_hash': '6b2000d53f88414c',\n",
       "  'vcf_string': '15\\t48487107\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487107T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3557A>G',\n",
       "  'hgvs_protein': 'p.Tyr1186Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487107,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48487110-C-T',\n",
       "  'variant_hash': 'a92efa6111aedff2',\n",
       "  'vcf_string': '15\\t48487110\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487110C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3554G>A',\n",
       "  'hgvs_protein': 'p.Gly1185Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487110,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48487119-C-T',\n",
       "  'variant_hash': 'a43d1c4c192883b0',\n",
       "  'vcf_string': '15\\t48487119\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487119C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3545G>A',\n",
       "  'hgvs_protein': 'p.Cys1182Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487119,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487129-G-A',\n",
       "  'variant_hash': '7ba7202122053e18',\n",
       "  'vcf_string': '15\\t48487129\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487129G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3535C>T',\n",
       "  'hgvs_protein': 'p.Gln1179Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487129,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48487131-T-C',\n",
       "  'variant_hash': '73546cee3e5d6ad7',\n",
       "  'vcf_string': '15\\t48487131\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487131T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3533A>G',\n",
       "  'hgvs_protein': 'p.Tyr1178Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487131,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48487142-G-A',\n",
       "  'variant_hash': '2670657eeb8e744a',\n",
       "  'vcf_string': '15\\t48487142\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487142G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3522C>T',\n",
       "  'hgvs_protein': 'p.Leu1174=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487142,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487146-T-G',\n",
       "  'variant_hash': '8a23c9499eaec155',\n",
       "  'vcf_string': '15\\t48487146\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487146T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3518A>C',\n",
       "  'hgvs_protein': 'p.Asn1173Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487146,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48487151-G-A',\n",
       "  'variant_hash': '8996cdc0b27f01fe',\n",
       "  'vcf_string': '15\\t48487151\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487151G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3513C>T',\n",
       "  'hgvs_protein': 'p.Cys1171=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487151,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487155-C-T',\n",
       "  'variant_hash': 'e5a2a4c2015123a2',\n",
       "  'vcf_string': '15\\t48487155\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487155C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3509G>A',\n",
       "  'hgvs_protein': 'p.Arg1170His',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487155,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48487157-G-A',\n",
       "  'variant_hash': '582da053cd736f7e',\n",
       "  'vcf_string': '15\\t48487157\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487157G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3507C>T',\n",
       "  'hgvs_protein': 'p.Gly1169=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487157,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487160-A-G',\n",
       "  'variant_hash': 'd475b6ab78dcfeb1',\n",
       "  'vcf_string': '15\\t48487160\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487160A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3504T>C',\n",
       "  'hgvs_protein': 'p.Asn1168=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487160,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487167-C-T',\n",
       "  'variant_hash': 'cbc34da0e559ef7d',\n",
       "  'vcf_string': '15\\t48487167\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487167C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3497G>A',\n",
       "  'hgvs_protein': 'p.Cys1166Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487167,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48487171-G-A',\n",
       "  'variant_hash': 'b8207041803bda92',\n",
       "  'vcf_string': '15\\t48487171\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487171G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3493C>T',\n",
       "  'hgvs_protein': 'p.Leu1165=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487171,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487184-C-T',\n",
       "  'variant_hash': 'cc0531fcf90b59c9',\n",
       "  'vcf_string': '15\\t48487184\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487184C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3480G>A',\n",
       "  'hgvs_protein': 'p.Glu1160=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487184,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487188-C-T',\n",
       "  'variant_hash': '8fb415d816cbf41f',\n",
       "  'vcf_string': '15\\t48487188\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487188C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3476G>A',\n",
       "  'hgvs_protein': 'p.Cys1159Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487188,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48487189-A-G',\n",
       "  'variant_hash': 'acd3dc4522e496b4',\n",
       "  'vcf_string': '15\\t48487189\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487189A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3475T>C',\n",
       "  'hgvs_protein': 'p.Cys1159Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487189,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487199-G-A',\n",
       "  'variant_hash': 'ebb8953eabf0ba72',\n",
       "  'vcf_string': '15\\t48487199\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487199G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3465C>T',\n",
       "  'hgvs_protein': 'p.Asp1155=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487199,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487205-C-T',\n",
       "  'variant_hash': 'dfa0f0db264d78c8',\n",
       "  'vcf_string': '15\\t48487205\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487205C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3464-5G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487205,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48487219-G-C',\n",
       "  'variant_hash': '5e4c74606771f32b',\n",
       "  'vcf_string': '15\\t48487219\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487219G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3464-19C>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487219,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487297-A-T',\n",
       "  'variant_hash': 'd2b19e877a24b26f',\n",
       "  'vcf_string': '15\\t48487297\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487297A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3463+15T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487297,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487309-T-C',\n",
       "  'variant_hash': 'a8a427647ba2d9e5',\n",
       "  'vcf_string': '15\\t48487309\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487309T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3463+3A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487309,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48487312-C-T',\n",
       "  'variant_hash': 'ba13fb9fbd3f82a3',\n",
       "  'vcf_string': '15\\t48487312\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487312C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3463G>A',\n",
       "  'hgvs_protein': 'p.Asp1155Asn',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487312,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487317-C-T',\n",
       "  'variant_hash': '5d3dc55637b6c9c8',\n",
       "  'vcf_string': '15\\t48487317\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487317C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3458G>A',\n",
       "  'hgvs_protein': 'p.Cys1153Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487317,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487318-A-G',\n",
       "  'variant_hash': '74de6820d6699216',\n",
       "  'vcf_string': '15\\t48487318\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487318A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3457T>C',\n",
       "  'hgvs_protein': 'p.Cys1153Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487318,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48487319-C-T',\n",
       "  'variant_hash': 'dc83bb19728928bf',\n",
       "  'vcf_string': '15\\t48487319\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487319C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3456G>A',\n",
       "  'hgvs_protein': 'p.Ala1152=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487319,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487319-C-G',\n",
       "  'variant_hash': '6234640b46e125e8',\n",
       "  'vcf_string': '15\\t48487319\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487319C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3456G>C',\n",
       "  'hgvs_protein': 'p.Ala1152=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487319,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487322-G-A',\n",
       "  'variant_hash': '237f74e9f6a5dc5e',\n",
       "  'vcf_string': '15\\t48487322\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487322G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3453C>T',\n",
       "  'hgvs_protein': 'p.Ser1151=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487322,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487333-G-C',\n",
       "  'variant_hash': '97bfb42ce2f3d65a',\n",
       "  'vcf_string': '15\\t48487333\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487333G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3442C>G',\n",
       "  'hgvs_protein': 'p.Pro1148Ala',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487333,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48487334-G-A',\n",
       "  'variant_hash': '954fa832b71e15ba',\n",
       "  'vcf_string': '15\\t48487334\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487334G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3441C>T',\n",
       "  'hgvs_protein': 'p.Ser1147=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487334,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487346-G-T',\n",
       "  'variant_hash': '6ec34bfad89fd1cd',\n",
       "  'vcf_string': '15\\t48487346\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487346G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3429C>A',\n",
       "  'hgvs_protein': 'p.Gly1143=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487346,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487353-G-A',\n",
       "  'variant_hash': 'de37ce0e60745f1f',\n",
       "  'vcf_string': '15\\t48487353\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487353G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3422C>T',\n",
       "  'hgvs_protein': 'p.Pro1141Leu',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487353,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48487362-C-G',\n",
       "  'variant_hash': 'acd74aa4befc9675',\n",
       "  'vcf_string': '15\\t48487362\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487362C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3413G>C',\n",
       "  'hgvs_protein': 'p.Cys1138Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487362,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487362-C-A',\n",
       "  'variant_hash': 'cd3cd87709596480',\n",
       "  'vcf_string': '15\\t48487362\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487362C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3413G>T',\n",
       "  'hgvs_protein': 'p.Cys1138Phe',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487362,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487364-G-A',\n",
       "  'variant_hash': '8bf4a61c52b57280',\n",
       "  'vcf_string': '15\\t48487364\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487364G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3411C>T',\n",
       "  'hgvs_protein': 'p.Arg1137=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487364,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487367-G-T',\n",
       "  'variant_hash': 'af0bf3cf57036ed1',\n",
       "  'vcf_string': '15\\t48487367\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487367G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3408C>A',\n",
       "  'hgvs_protein': 'p.Tyr1136Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487367,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48487402-G-A',\n",
       "  'variant_hash': '0b8d91ba4b4dd7fb',\n",
       "  'vcf_string': '15\\t48487402\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487402G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3373C>T',\n",
       "  'hgvs_protein': 'p.Arg1125Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487402,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487406-T-C',\n",
       "  'variant_hash': 'a0a8a82eb521b265',\n",
       "  'vcf_string': '15\\t48487406\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487406T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3369A>G',\n",
       "  'hgvs_protein': 'p.Leu1123=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487406,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48487431-T-C',\n",
       "  'variant_hash': 'c7530d9f18c66227',\n",
       "  'vcf_string': '15\\t48487431\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487431T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3344A>G',\n",
       "  'hgvs_protein': 'p.Asp1115Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487431,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487437-T-C',\n",
       "  'variant_hash': '7c122680d40f3603',\n",
       "  'vcf_string': '15\\t48487437\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487437T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3338A>G',\n",
       "  'hgvs_protein': 'p.Asp1113Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487437,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487438-C-G',\n",
       "  'variant_hash': '68952ef545ca138e',\n",
       "  'vcf_string': '15\\t48487438\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487438C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3338-1G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487438,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48487639-C-A',\n",
       "  'variant_hash': 'afba005185c9a8cd',\n",
       "  'vcf_string': '15\\t48487639\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48487639C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3338-202G>T',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48487639,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488093-T-C',\n",
       "  'variant_hash': 'f244d18d0f24af3d',\n",
       "  'vcf_string': '15\\t48488093\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488093T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3337+20A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488093,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488111-A-G',\n",
       "  'variant_hash': '77c3fee78f9b482b',\n",
       "  'vcf_string': '15\\t48488111\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488111A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3337+2T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488111,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48488112-C-T',\n",
       "  'variant_hash': '51c082851bbedb9f',\n",
       "  'vcf_string': '15\\t48488112\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488112C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3337+1G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488112,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488148-T-C',\n",
       "  'variant_hash': '7db88f0f1bee1fca',\n",
       "  'vcf_string': '15\\t48488148\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488148T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3302A>G',\n",
       "  'hgvs_protein': 'p.Tyr1101Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488148,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488156-G-A',\n",
       "  'variant_hash': '752d68448151fd00',\n",
       "  'vcf_string': '15\\t48488156\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488156G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3294C>T',\n",
       "  'hgvs_protein': 'p.Asp1098=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488156,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48488160-C-T',\n",
       "  'variant_hash': '35c79d1a5ff076e7',\n",
       "  'vcf_string': '15\\t48488160\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488160C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3290G>A',\n",
       "  'hgvs_protein': 'p.Cys1097Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488160,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48488177-C-T',\n",
       "  'variant_hash': '24a65ebbff94da81',\n",
       "  'vcf_string': '15\\t48488177\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488177C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3273G>A',\n",
       "  'hgvs_protein': 'p.Gly1091=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488177,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488195-C-T',\n",
       "  'variant_hash': 'd1b0d398f44d9719',\n",
       "  'vcf_string': '15\\t48488195\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488195C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3255G>A',\n",
       "  'hgvs_protein': 'p.Gln1085=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488195,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488197-G-A',\n",
       "  'variant_hash': '9af5184cf0cba574',\n",
       "  'vcf_string': '15\\t48488197\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488197G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3253C>T',\n",
       "  'hgvs_protein': 'p.Gln1085Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488197,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48488204-G-A',\n",
       "  'variant_hash': '9271cbaa878a7827',\n",
       "  'vcf_string': '15\\t48488204\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488204G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3246C>T',\n",
       "  'hgvs_protein': 'p.Gly1082=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488204,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488233-C-T',\n",
       "  'variant_hash': '9d92637e511d90a3',\n",
       "  'vcf_string': '15\\t48488233\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488233C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3217G>A',\n",
       "  'hgvs_protein': 'p.Glu1073Lys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488233,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48488233-C-A',\n",
       "  'variant_hash': '811c1244d1f98d30',\n",
       "  'vcf_string': '15\\t48488233\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488233C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3217G>T',\n",
       "  'hgvs_protein': 'p.Glu1073Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488233,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48488240-G-A',\n",
       "  'variant_hash': '5fdbea8da664b9fb',\n",
       "  'vcf_string': '15\\t48488240\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488240G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3210C>T',\n",
       "  'hgvs_protein': 'p.Asp1070=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488240,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_region_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488242-C-G',\n",
       "  'variant_hash': '9b4bcd0c937382e5',\n",
       "  'vcf_string': '15\\t48488242\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488242C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3209-1G>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488242,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488243-T-G',\n",
       "  'variant_hash': '8c8e119f4dfa770f',\n",
       "  'vcf_string': '15\\t48488243\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488243T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3209-2A>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488243,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_acceptor_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48488250-A-G',\n",
       "  'variant_hash': '2bdcd1eb79a54ba3',\n",
       "  'vcf_string': '15\\t48488250\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488250A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3209-9T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488250,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488254-A-G',\n",
       "  'variant_hash': '290c3ce8689d6f85',\n",
       "  'vcf_string': '15\\t48488254\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488254A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3209-13T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488254,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488355-C-T',\n",
       "  'variant_hash': 'f4799bf625a80724',\n",
       "  'vcf_string': '15\\t48488355\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488355C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3208+13G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488355,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488403-C-A',\n",
       "  'variant_hash': '6dd64d8ad0475c5b',\n",
       "  'vcf_string': '15\\t48488403\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488403C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3173G>T',\n",
       "  'hgvs_protein': 'p.Gly1058Val',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488403,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48488405-G-A',\n",
       "  'variant_hash': '95ce5ee3a6e57268',\n",
       "  'vcf_string': '15\\t48488405\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488405G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3171C>T',\n",
       "  'hgvs_protein': 'p.Ser1057=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488405,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488411-A-T',\n",
       "  'variant_hash': 'd87804816b434e0c',\n",
       "  'vcf_string': '15\\t48488411\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488411A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3165T>A',\n",
       "  'hgvs_protein': 'p.Cys1055Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488411,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488412-C-T',\n",
       "  'variant_hash': 'ffcfa53659eb6ce5',\n",
       "  'vcf_string': '15\\t48488412\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488412C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3164G>A',\n",
       "  'hgvs_protein': 'p.Cys1055Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488412,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488419-A-C',\n",
       "  'variant_hash': '3c65f9500a57fc2e',\n",
       "  'vcf_string': '15\\t48488419\\t.\\tA\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488419A>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3157T>G',\n",
       "  'hgvs_protein': 'p.Cys1053Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488419,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48488432-A-T',\n",
       "  'variant_hash': 'd6e8f502146aa208',\n",
       "  'vcf_string': '15\\t48488432\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488432A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3144T>A',\n",
       "  'hgvs_protein': 'p.Ile1048=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488432,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488433-A-G',\n",
       "  'variant_hash': '4ec12b14d16f495a',\n",
       "  'vcf_string': '15\\t48488433\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488433A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3143T>C',\n",
       "  'hgvs_protein': 'p.Ile1048Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488433,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488439-T-C',\n",
       "  'variant_hash': '975dfd258129e472',\n",
       "  'vcf_string': '15\\t48488439\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488439T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3137A>G',\n",
       "  'hgvs_protein': 'p.Asn1046Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488439,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48488443-T-G',\n",
       "  'variant_hash': '0a5ee582070a442d',\n",
       "  'vcf_string': '15\\t48488443\\t.\\tT\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488443T>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3133A>C',\n",
       "  'hgvs_protein': 'p.Arg1045=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488443,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488451-C-T',\n",
       "  'variant_hash': '59f3b70494aeb458',\n",
       "  'vcf_string': '15\\t48488451\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488451C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3125G>A',\n",
       "  'hgvs_protein': 'p.Gly1042Asp',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488451,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48488452-C-T',\n",
       "  'variant_hash': '424af1df130b318d',\n",
       "  'vcf_string': '15\\t48488452\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488452C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3124G>A',\n",
       "  'hgvs_protein': 'p.Gly1042Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488452,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48488453-G-A',\n",
       "  'variant_hash': '3196a1f088cde790',\n",
       "  'vcf_string': '15\\t48488453\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488453G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3123C>T',\n",
       "  'hgvs_protein': 'p.His1041=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488453,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488465-G-A',\n",
       "  'variant_hash': '3ff39ec2eb289ef4',\n",
       "  'vcf_string': '15\\t48488465\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488465G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3111C>T',\n",
       "  'hgvs_protein': 'p.Ser1037=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488465,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48488493-T-C',\n",
       "  'variant_hash': 'b8041a35664157f3',\n",
       "  'vcf_string': '15\\t48488493\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488493T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3083A>G',\n",
       "  'hgvs_protein': 'p.Asp1028Gly',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488493,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48488502-A-G',\n",
       "  'variant_hash': '40599036c2e0048f',\n",
       "  'vcf_string': '15\\t48488502\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48488502A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3083-9T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48488502,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_polypyrimidine_tract_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489838-C-T',\n",
       "  'variant_hash': '32780963d8534edb',\n",
       "  'vcf_string': '15\\t48489838\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489838C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3082+13G>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489838,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489842-A-T',\n",
       "  'variant_hash': 'e72d04818e4aacae',\n",
       "  'vcf_string': '15\\t48489842\\t.\\tA\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489842A>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3082+9T>A',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489842,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489842-A-G',\n",
       "  'variant_hash': 'e1a26f456fb23ac0',\n",
       "  'vcf_string': '15\\t48489842\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489842A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3082+9T>C',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489842,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'intron_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489845-T-C',\n",
       "  'variant_hash': '9875842df05e593d',\n",
       "  'vcf_string': '15\\t48489845\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489845T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3082+6A>G',\n",
       "  'hgvs_protein': '',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489845,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'splice_donor_region_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48489855-G-A',\n",
       "  'variant_hash': '78dc08c490430264',\n",
       "  'vcf_string': '15\\t48489855\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489855G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3078C>T',\n",
       "  'hgvs_protein': 'p.Phe1026=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489855,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489856-A-G',\n",
       "  'variant_hash': '6a51e9226dcbab23',\n",
       "  'vcf_string': '15\\t48489856\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489856A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3077T>C',\n",
       "  'hgvs_protein': 'p.Phe1026Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489856,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48489864-C-T',\n",
       "  'variant_hash': 'a6e6bca1ada55071',\n",
       "  'vcf_string': '15\\t48489864\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489864C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3069G>A',\n",
       "  'hgvs_protein': 'p.Lys1023=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489864,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'benign_or_likely'},\n",
       " {'variant_id': '15-48489869-C-A',\n",
       "  'variant_hash': '4548541dae41f4ef',\n",
       "  'vcf_string': '15\\t48489869\\t.\\tC\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489869C>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3064G>T',\n",
       "  'hgvs_protein': 'p.Gly1022Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489869,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48489875-T-C',\n",
       "  'variant_hash': 'd86f18bce88f499f',\n",
       "  'vcf_string': '15\\t48489875\\t.\\tT\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489875T>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3058A>G',\n",
       "  'hgvs_protein': 'p.Thr1020Ala',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489875,\n",
       "  'ref_allele': 'T',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489896-C-T',\n",
       "  'variant_hash': '739c4aeb8aecca92',\n",
       "  'vcf_string': '15\\t48489896\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489896C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3037G>A',\n",
       "  'hgvs_protein': 'p.Gly1013Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489896,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48489897-G-A',\n",
       "  'variant_hash': '4eb06795ba551ffc',\n",
       "  'vcf_string': '15\\t48489897\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489897G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3036C>T',\n",
       "  'hgvs_protein': 'p.Pro1012=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489897,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489906-C-T',\n",
       "  'variant_hash': 'a1dcdf2126e6e781',\n",
       "  'vcf_string': '15\\t48489906\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489906C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3027G>A',\n",
       "  'hgvs_protein': 'p.Pro1009=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489906,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489909-A-G',\n",
       "  'variant_hash': '66777176344bcc95',\n",
       "  'vcf_string': '15\\t48489909\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489909A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3024T>C',\n",
       "  'hgvs_protein': 'p.Cys1008=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489909,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489911-A-G',\n",
       "  'variant_hash': 'ba01b3d4f012f96d',\n",
       "  'vcf_string': '15\\t48489911\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489911A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3022T>C',\n",
       "  'hgvs_protein': 'p.Cys1008Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489911,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48489912-C-T',\n",
       "  'variant_hash': '62487a52680ee6ae',\n",
       "  'vcf_string': '15\\t48489912\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489912C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3021G>A',\n",
       "  'hgvs_protein': 'p.Leu1007=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489912,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489921-G-T',\n",
       "  'variant_hash': '4ae4d57b83cc5a43',\n",
       "  'vcf_string': '15\\t48489921\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489921G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3012C>A',\n",
       "  'hgvs_protein': 'p.Tyr1004Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489921,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48489921-G-C',\n",
       "  'variant_hash': 'c90004088067ba7f',\n",
       "  'vcf_string': '15\\t48489921\\t.\\tG\\tC\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489921G>C',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3012C>G',\n",
       "  'hgvs_protein': 'p.Tyr1004Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489921,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'C',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48489921-G-A',\n",
       "  'variant_hash': 'c5a767601e650f73',\n",
       "  'vcf_string': '15\\t48489921\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489921G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3012C>T',\n",
       "  'hgvs_protein': 'p.Tyr1004=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489921,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489927-A-G',\n",
       "  'variant_hash': '60fdfde317b60576',\n",
       "  'vcf_string': '15\\t48489927\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489927A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.3006T>C',\n",
       "  'hgvs_protein': 'p.Pro1002=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489927,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489946-C-T',\n",
       "  'variant_hash': '70497546aa6a786c',\n",
       "  'vcf_string': '15\\t48489946\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489946C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2987G>A',\n",
       "  'hgvs_protein': 'p.Cys996Tyr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489946,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48489951-C-T',\n",
       "  'variant_hash': '4f2fd92d48aa8c1b',\n",
       "  'vcf_string': '15\\t48489951\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489951C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2982G>A',\n",
       "  'hgvs_protein': 'p.Glu994=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489951,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489956-A-G',\n",
       "  'variant_hash': '1885bbfc470c21dc',\n",
       "  'vcf_string': '15\\t48489956\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489956A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2977T>C',\n",
       "  'hgvs_protein': 'p.Cys993Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489956,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48489969-C-T',\n",
       "  'variant_hash': 'b31dc340ff5733bf',\n",
       "  'vcf_string': '15\\t48489969\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489969C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2964G>A',\n",
       "  'hgvs_protein': 'p.Trp988Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489969,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48489970-C-T',\n",
       "  'variant_hash': '76b15198492d1d0d',\n",
       "  'vcf_string': '15\\t48489970\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489970C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2963G>A',\n",
       "  'hgvs_protein': 'p.Trp988Ter',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489970,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'stop_gained',\n",
       "  'pathogenicity_original': 'pathogenic'},\n",
       " {'variant_id': '15-48489977-C-T',\n",
       "  'variant_hash': '1e5586c5af16f948',\n",
       "  'vcf_string': '15\\t48489977\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489977C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2956G>A',\n",
       "  'hgvs_protein': 'p.Ala986Thr',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489977,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'benign'},\n",
       " {'variant_id': '15-48489980-C-T',\n",
       "  'variant_hash': '3dcb8e2ba880fad7',\n",
       "  'vcf_string': '15\\t48489980\\t.\\tC\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489980C>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2953G>A',\n",
       "  'hgvs_protein': 'p.Gly985Arg',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489980,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48489981-G-T',\n",
       "  'variant_hash': 'f962704ec0eb39d4',\n",
       "  'vcf_string': '15\\t48489981\\t.\\tG\\tT\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489981G>T',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2952C>A',\n",
       "  'hgvs_protein': 'p.Val984=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489981,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'T',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489981-G-A',\n",
       "  'variant_hash': '02103a2dff190ac9',\n",
       "  'vcf_string': '15\\t48489981\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489981G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2952C>T',\n",
       "  'hgvs_protein': 'p.Val984=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489981,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489984-G-A',\n",
       "  'variant_hash': 'f53db5c5728b4181',\n",
       "  'vcf_string': '15\\t48489984\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489984G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2949C>T',\n",
       "  'hgvs_protein': 'p.Ser983=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489984,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48489988-C-G',\n",
       "  'variant_hash': '86e0e13faa758367',\n",
       "  'vcf_string': '15\\t48489988\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489988C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2945G>C',\n",
       "  'hgvs_protein': 'p.Cys982Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489988,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48489991-C-G',\n",
       "  'variant_hash': 'e8ea85d07d719abe',\n",
       "  'vcf_string': '15\\t48489991\\t.\\tC\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489991C>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2942G>C',\n",
       "  'hgvs_protein': 'p.Cys981Ser',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489991,\n",
       "  'ref_allele': 'C',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'likely_pathogenic'},\n",
       " {'variant_id': '15-48489999-G-A',\n",
       "  'variant_hash': '7d55ad5c39bafe0d',\n",
       "  'vcf_string': '15\\t48489999\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48489999G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2934C>T',\n",
       "  'hgvs_protein': 'p.Asp978=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48489999,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48490005-G-A',\n",
       "  'variant_hash': 'dddec78e2eec29de',\n",
       "  'vcf_string': '15\\t48490005\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48490005G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2928C>T',\n",
       "  'hgvs_protein': 'p.Arg976=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48490005,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " {'variant_id': '15-48490013-G-A',\n",
       "  'variant_hash': '5c5acd7be3688c6e',\n",
       "  'vcf_string': '15\\t48490013\\t.\\tG\\tA\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48490013G>A',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2920C>T',\n",
       "  'hgvs_protein': 'p.Arg974Cys',\n",
       "  'chromosome': '15',\n",
       "  'position': 48490013,\n",
       "  'ref_allele': 'G',\n",
       "  'alt_allele': 'A',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'missense_variant',\n",
       "  'pathogenicity_original': 'pathogenic_or_likely'},\n",
       " {'variant_id': '15-48490020-A-G',\n",
       "  'variant_hash': '07d2c84436935cd2',\n",
       "  'vcf_string': '15\\t48490020\\t.\\tA\\tG\\t.\\t.\\t.',\n",
       "  'hgvs_genomic_38': 'NC_000015.10:g.48490020A>G',\n",
       "  'hgvs_coding': 'NM_000138.5:c.2913T>C',\n",
       "  'hgvs_protein': 'p.Ile971=',\n",
       "  'chromosome': '15',\n",
       "  'position': 48490020,\n",
       "  'ref_allele': 'A',\n",
       "  'alt_allele': 'G',\n",
       "  'gene_symbol': 'FBN1',\n",
       "  'most_severe_consequence': 'synonymous_variant',\n",
       "  'pathogenicity_original': 'likely_benign'},\n",
       " ...]"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "metadata_variants"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6545cb6b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Created lookup dictionary with 1532 variants\n",
      "\n",
      "=== Debug: Inspecting variant structure ===\n",
      "Variant keys: ['variant_id', 'variant_hash', 'vcf_string', 'hgvs_genomic_38', 'hgvs_coding', 'hgvs_protein', 'chromosome', 'position', 'ref_allele', 'alt_allele', 'gene_symbol', 'most_severe_consequence', 'pathogenicity_original']\n",
      "Full variant structure:\n",
      "{\n",
      "  \"variant_id\": \"15-48411280-G-A\",\n",
      "  \"variant_hash\": \"993c84e847fe446a\",\n",
      "  \"vcf_string\": \"15\\t48411280\\t.\\tG\\tA\\t.\\t.\\t.\",\n",
      "  \"hgvs_genomic_38\": \"NC_000015.10:g.48411280G>A\",\n",
      "  \"hgvs_coding\": \"NM_000138.5:c.8326C>T\",\n",
      "  \"hgvs_protein\": \"p.Arg2776Ter\",\n",
      "  \"chromosome\": \"15\",\n",
      "  \"position\": 48411280,\n",
      "  \"ref_allele\": \"G\",\n",
      "  \"alt_allele\": \"A\",\n",
      "  \"gene_symbol\": \"FBN1\",\n",
      "  \"most_severe_consequence\": \"stop_gained\",\n",
      "  \"pathogenicity_original\": \"pathogenic\"\n",
      "}\n",
      "\n",
      "=== Found 3 variants ===\n",
      "\n",
      "Variant ID: 15-48411280-G-A\n",
      "  chromosome: 15 (type: str)\n",
      "  position: 48411280 (type: int)\n",
      "  ref_allele: G (type: str)\n",
      "  alt_allele: A (type: str)\n",
      "  pathogenicity_original: pathogenic (type: str)\n",
      "\n",
      "Variant ID: 15-48411338-C-T\n",
      "  chromosome: 15 (type: str)\n",
      "  position: 48411338 (type: int)\n",
      "  ref_allele: C (type: str)\n",
      "  alt_allele: T (type: str)\n",
      "  pathogenicity_original: pathogenic (type: str)\n",
      "\n",
      "Variant ID: 15-48412715-G-A\n",
      "  chromosome: 15 (type: str)\n",
      "  position: 48412715 (type: int)\n",
      "  ref_allele: G (type: str)\n",
      "  alt_allele: A (type: str)\n",
      "  pathogenicity_original: pathogenic (type: str)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "5ccb6c82",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Retrieved 3 variants\n",
      "\n",
      "DataFrame info:\n",
      "<class 'pandas.core.frame.DataFrame'>\n",
      "RangeIndex: 3 entries, 0 to 2\n",
      "Data columns (total 2 columns):\n",
      " #   Column                  Non-Null Count  Dtype \n",
      "---  ------                  --------------  ----- \n",
      " 0   variant_id              3 non-null      object\n",
      " 1   pathogenicity_original  3 non-null      object\n",
      "dtypes: object(2)\n",
      "memory usage: 180.0+ bytes\n",
      "None\n",
      "\n",
      "First few rows:\n",
      "        variant_id pathogenicity_original\n",
      "0  15-48411280-G-A             pathogenic\n",
      "1  15-48411338-C-T             pathogenic\n",
      "2  15-48412715-G-A             pathogenic\n",
      "\n",
      "=== Data Types Summary ===\n",
      "variant_id: str (example: 15-48411280-G-A)\n",
      "pathogenicity_original: str (example: pathogenic)\n"
     ]
    }
   ],
   "source": [
    "# Alternative: More compact function for batch queries\n",
    "def get_variants_metadata(variant_ids: list, variant_dict: dict) -> list:\n",
    "    \"\"\"\n",
    "    Retrieve metadata for multiple variant IDs.\n",
    "    \n",
    "    Args:\n",
    "        variant_ids: List of variant IDs to query\n",
    "        variant_dict: Dictionary mapping variant_id to variant data\n",
    "        \n",
    "    Returns:\n",
    "        List of dictionaries containing variant metadata\n",
    "    \"\"\"\n",
    "    results = []\n",
    "    for variant_id in variant_ids:\n",
    "        variant = variant_dict.get(variant_id)\n",
    "        if variant:\n",
    "            metadata = variant.get('metadata', {})\n",
    "            # Flatten the structure: combine top-level and metadata fields\n",
    "            result = {\n",
    "                'variant_id': variant_id,\n",
    "                'pathogenicity_original': variant.get('pathogenicity_original'),\n",
    "                **metadata  # Unpack all metadata fields\n",
    "            }\n",
    "            results.append(result)\n",
    "        else:\n",
    "            results.append({'variant_id': variant_id, 'found': False})\n",
    "    return results\n",
    "\n",
    "# Usage example\n",
    "variant_ids_to_find = [\n",
    "    '15-48411280-G-A',\n",
    "    '15-48411338-C-T',\n",
    "    '15-48412715-G-A',\n",
    "]\n",
    "\n",
    "# Get all metadata at once\n",
    "variants_metadata = get_variants_metadata(variant_ids_to_find, variant_dict)\n",
    "\n",
    "# Convert to DataFrame for easier analysis\n",
    "df_variants = pd.DataFrame(variants_metadata)\n",
    "print(f\"Retrieved {len(df_variants)} variants\")\n",
    "print(f\"\\nDataFrame info:\")\n",
    "print(df_variants.info())\n",
    "print(f\"\\nFirst few rows:\")\n",
    "print(df_variants.head())\n",
    "\n",
    "# Data types summary\n",
    "print(f\"\\n=== Data Types Summary ===\")\n",
    "for col in df_variants.columns:\n",
    "    if df_variants[col].notna().any():\n",
    "        sample_value = df_variants[col].dropna().iloc[0]\n",
    "        print(f\"{col}: {type(sample_value).__name__} (example: {sample_value})\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "62f102a8",
   "metadata": {},
   "outputs": [],
   "source": [
    "pred_result = "
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "bio-ai",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
